{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[{"file_id":"174Qh22KCga8E4EiJ9fc8AUAbsQzOP11b","timestamp":1729147288297}],"gpuType":"T4","machine_shape":"hm"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"},"accelerator":"GPU","widgets":{"application/vnd.jupyter.widget-state+json":{"8aca7e3d8c5441f087f084f632bab0e5":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_c7ac9660b133439db3f42a7ac88ddb13","IPY_MODEL_1f0af1dee80c4530a216795a67c44de3","IPY_MODEL_e4c12ad68bed436ca9cfc17006953991"],"layout":"IPY_MODEL_bd808bee84f546eca5837cb789c865f6"}},"c7ac9660b133439db3f42a7ac88ddb13":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_bf8ae96cdef447e8b274e81850e64e4a","placeholder":"​","style":"IPY_MODEL_798292909e834f1398e162aa7c4cf88d","value":"tokenizer_config.json: 100%"}},"1f0af1dee80c4530a216795a67c44de3":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_06f8030bd44a4bbab55d0c3cbf011404","max":86,"min":0,"orientation":"horizontal","style":"IPY_MODEL_7958f8d1a83a44a084ddf15190ec5100","value":86}},"e4c12ad68bed436ca9cfc17006953991":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_7551e5083e1b43ff859228f61c1969df","placeholder":"​","style":"IPY_MODEL_da9ad0b570804b19bfa7a618c87cccfd","value":" 86.0/86.0 [00:00&lt;00:00, 7.29kB/s]"}},"bd808bee84f546eca5837cb789c865f6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"bf8ae96cdef447e8b274e81850e64e4a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"798292909e834f1398e162aa7c4cf88d":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"06f8030bd44a4bbab55d0c3cbf011404":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"7958f8d1a83a44a084ddf15190ec5100":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"7551e5083e1b43ff859228f61c1969df":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"da9ad0b570804b19bfa7a618c87cccfd":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"3c53870ad5984829bacfd49a0214bc8d":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_aff18f8042984b07a31ee64ecce8fdb6","IPY_MODEL_174d286408e248629c2e5cc5016602d7","IPY_MODEL_e3d1f92c485240759840ca8f0ba163ce"],"layout":"IPY_MODEL_75bcdcd1cfe24ca08f4e6c35402eea5b"}},"aff18f8042984b07a31ee64ecce8fdb6":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_881a5dded8e44102b0d15b14585ef09b","placeholder":"​","style":"IPY_MODEL_1fef4a3f39ec423989fa5e0499cb928b","value":"config.json: 100%"}},"174d286408e248629c2e5cc5016602d7":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_e2e1c058bf004dacba6237de36fb9e08","max":361,"min":0,"orientation":"horizontal","style":"IPY_MODEL_02f107c9457547a2a34efa23e466896d","value":361}},"e3d1f92c485240759840ca8f0ba163ce":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_2b0b0ef079b243f9879c48c79e3d0dc2","placeholder":"​","style":"IPY_MODEL_0c24c7fb581446bca188a9ac4006764a","value":" 361/361 [00:00&lt;00:00, 32.1kB/s]"}},"75bcdcd1cfe24ca08f4e6c35402eea5b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"881a5dded8e44102b0d15b14585ef09b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"1fef4a3f39ec423989fa5e0499cb928b":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"e2e1c058bf004dacba6237de36fb9e08":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"02f107c9457547a2a34efa23e466896d":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"2b0b0ef079b243f9879c48c79e3d0dc2":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"0c24c7fb581446bca188a9ac4006764a":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"6ce7405f091f494c87369df393cce96c":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_33f803786a74420e98c364ecd775b497","IPY_MODEL_7aac207ee63d4c6b8190a3c818ccc9e0","IPY_MODEL_4da994018b144174a1afe1a54bb5e2d6"],"layout":"IPY_MODEL_0edad9e943984baf9d7f37c1640fee75"}},"33f803786a74420e98c364ecd775b497":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_8e1512c5a9584acea8b7a2cc502487b1","placeholder":"​","style":"IPY_MODEL_00aba44f26554a9087e473825616e47d","value":"vocab.txt: 100%"}},"7aac207ee63d4c6b8190a3c818ccc9e0":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_97dc10663885448f8bd6f68bce415856","max":81,"min":0,"orientation":"horizontal","style":"IPY_MODEL_bde5c315085545dc8d574c74f586d7b0","value":81}},"4da994018b144174a1afe1a54bb5e2d6":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_282faa48f09a437a9e9fb553d5bd513a","placeholder":"​","style":"IPY_MODEL_561c4f64026a4a00afc058303a8bdc0a","value":" 81.0/81.0 [00:00&lt;00:00, 7.18kB/s]"}},"0edad9e943984baf9d7f37c1640fee75":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"8e1512c5a9584acea8b7a2cc502487b1":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"00aba44f26554a9087e473825616e47d":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"97dc10663885448f8bd6f68bce415856":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"bde5c315085545dc8d574c74f586d7b0":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"282faa48f09a437a9e9fb553d5bd513a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"561c4f64026a4a00afc058303a8bdc0a":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"f51813427eb24c178f88a9c9788b2385":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_c598e40f285746f4a6fc058f9f97f1ad","IPY_MODEL_17172415be384375a09b7a4ecfba02f8","IPY_MODEL_264f7a8109a942e4a1fa49881c2cbe85"],"layout":"IPY_MODEL_838afc4ada694e309a436f99b1ae3ed4"}},"c598e40f285746f4a6fc058f9f97f1ad":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_4f0e0a4c99e14aafa3abe44c8c8a5cc9","placeholder":"​","style":"IPY_MODEL_3f288447d1714d3bb22bf4fa238481bd","value":"special_tokens_map.json: 100%"}},"17172415be384375a09b7a4ecfba02f8":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_a36703a03c5149b690c1bb525bded337","max":112,"min":0,"orientation":"horizontal","style":"IPY_MODEL_0d7438de152449639c0c8cb0dc960dc9","value":112}},"264f7a8109a942e4a1fa49881c2cbe85":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_210a206e59a9422bb085cce6982c7628","placeholder":"​","style":"IPY_MODEL_1a4b4ede6c424471bfa8b26a41684cfc","value":" 112/112 [00:00&lt;00:00, 9.74kB/s]"}},"838afc4ada694e309a436f99b1ae3ed4":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"4f0e0a4c99e14aafa3abe44c8c8a5cc9":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"3f288447d1714d3bb22bf4fa238481bd":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"a36703a03c5149b690c1bb525bded337":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"0d7438de152449639c0c8cb0dc960dc9":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"210a206e59a9422bb085cce6982c7628":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"1a4b4ede6c424471bfa8b26a41684cfc":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"43403696d8f948fcb8bfdaa922686a00":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_c4f8e28467f041d4baaef90e2a264b02","IPY_MODEL_f575a47e1aca4b498f3cef7f459667c3","IPY_MODEL_8cd81d9e97f3462a86aae6fa10681169"],"layout":"IPY_MODEL_e634f0c7416d48cf940aa41f98c24877"}},"c4f8e28467f041d4baaef90e2a264b02":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_c7cd34c5376c4156a121b2d4c5cd9966","placeholder":"​","style":"IPY_MODEL_fa0aa80ea4c44d9fa7a40acf6535aac3","value":"pytorch_model.bin: 100%"}},"f575a47e1aca4b498f3cef7f459667c3":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_ff12d7c242cf4ca78bb2716ef003d84c","max":1684058277,"min":0,"orientation":"horizontal","style":"IPY_MODEL_0a7f07678ce140a48dcc1fc9559ff90d","value":1684058277}},"8cd81d9e97f3462a86aae6fa10681169":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_64779eff82ad4dbc9bf73a8d51cc3801","placeholder":"​","style":"IPY_MODEL_67730cdb4c0d44b3bd99bc523e69371d","value":" 1.68G/1.68G [00:07&lt;00:00, 210MB/s]"}},"e634f0c7416d48cf940aa41f98c24877":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"c7cd34c5376c4156a121b2d4c5cd9966":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"fa0aa80ea4c44d9fa7a40acf6535aac3":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"ff12d7c242cf4ca78bb2716ef003d84c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"0a7f07678ce140a48dcc1fc9559ff90d":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"64779eff82ad4dbc9bf73a8d51cc3801":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"67730cdb4c0d44b3bd99bc523e69371d":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"6586ffa129d94cb487b38ccaae9bc13d":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_924de148a0534375a65cf98f135fe073","IPY_MODEL_508e757a7fa046fa94e7095dd72efb07","IPY_MODEL_1e567aaf117740fcba8dcfeb98268faf"],"layout":"IPY_MODEL_f4d92dcd15b44072aba5cd0fe6d83ce0"}},"924de148a0534375a65cf98f135fe073":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_c737a4d7cf1046e6aac983bd8cb3d0c1","placeholder":"​","style":"IPY_MODEL_14c65e20a0044c54901cb22321712e90","value":"tokenizer_config.json: 100%"}},"508e757a7fa046fa94e7095dd72efb07":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_2c63d0288d934080abd2b2bbdf6b77d6","max":86,"min":0,"orientation":"horizontal","style":"IPY_MODEL_ef15df0fbadc49ddb1162b5cbec64ea1","value":86}},"1e567aaf117740fcba8dcfeb98268faf":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_a4e55bf562334e8fb97d0dd0d0278c68","placeholder":"​","style":"IPY_MODEL_fe193f31597b4922bcc2ef9b2505e3a6","value":" 86.0/86.0 [00:00&lt;00:00, 7.73kB/s]"}},"f4d92dcd15b44072aba5cd0fe6d83ce0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"c737a4d7cf1046e6aac983bd8cb3d0c1":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"14c65e20a0044c54901cb22321712e90":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"2c63d0288d934080abd2b2bbdf6b77d6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"ef15df0fbadc49ddb1162b5cbec64ea1":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"a4e55bf562334e8fb97d0dd0d0278c68":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"fe193f31597b4922bcc2ef9b2505e3a6":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"375b4c229f40489884ea897f0afc128b":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_1f2bf479ea254179ae77c9e8bb0099e4","IPY_MODEL_53a9d84dea7b4f8eae9c087c751820a0","IPY_MODEL_5d80e2b2208941c288018accb9f1278f"],"layout":"IPY_MODEL_8d62f7741c574e22b1a4067393389d18"}},"1f2bf479ea254179ae77c9e8bb0099e4":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_7cab74916ba146ceafda5d15c3136d24","placeholder":"​","style":"IPY_MODEL_ad6cf9afa720434f8e3292f26221cc74","value":"config.json: 100%"}},"53a9d84dea7b4f8eae9c087c751820a0":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_5866698c491d482ea50b73e1eb6ee7bf","max":361,"min":0,"orientation":"horizontal","style":"IPY_MODEL_a7619dc67e1c478d958265fb0fb94f5b","value":361}},"5d80e2b2208941c288018accb9f1278f":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_612262b39bfa46a6b59f327398ef8b7c","placeholder":"​","style":"IPY_MODEL_7c1d37d0888a4119b40114051ab2fbef","value":" 361/361 [00:00&lt;00:00, 34.8kB/s]"}},"8d62f7741c574e22b1a4067393389d18":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"7cab74916ba146ceafda5d15c3136d24":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"ad6cf9afa720434f8e3292f26221cc74":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"5866698c491d482ea50b73e1eb6ee7bf":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"a7619dc67e1c478d958265fb0fb94f5b":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"612262b39bfa46a6b59f327398ef8b7c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"7c1d37d0888a4119b40114051ab2fbef":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"016bf80638c443a29bf69001705993ce":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_9bd17bddf09a42fb9f2feb655f9fff31","IPY_MODEL_f6cad4d462bb44dfb00d3509c16af6a6","IPY_MODEL_479c8aeea3654f75be883ad488ad63aa"],"layout":"IPY_MODEL_22deb523667a490382f98364617e55b6"}},"9bd17bddf09a42fb9f2feb655f9fff31":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_158c42d7f3754e5c99c23ebab040e04d","placeholder":"​","style":"IPY_MODEL_6203665749044e7ea6450c3b257b6b4a","value":"vocab.txt: 100%"}},"f6cad4d462bb44dfb00d3509c16af6a6":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_9d1c27828c1c477c90462b977845e5eb","max":81,"min":0,"orientation":"horizontal","style":"IPY_MODEL_6536a8a861014d44afdf8ea4d704512b","value":81}},"479c8aeea3654f75be883ad488ad63aa":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_596be27d9bfc4feba9d49fcd7599213c","placeholder":"​","style":"IPY_MODEL_54d2f3f3c5e743c58646d9e261f70746","value":" 81.0/81.0 [00:00&lt;00:00, 6.78kB/s]"}},"22deb523667a490382f98364617e55b6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"158c42d7f3754e5c99c23ebab040e04d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"6203665749044e7ea6450c3b257b6b4a":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"9d1c27828c1c477c90462b977845e5eb":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"6536a8a861014d44afdf8ea4d704512b":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"596be27d9bfc4feba9d49fcd7599213c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"54d2f3f3c5e743c58646d9e261f70746":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"66adbb15bae2492ba1afc9aadf005076":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_39fad12de7454ed2b5a794a04bf92f00","IPY_MODEL_0c3377e47bed4043adc83fa001a9c89d","IPY_MODEL_f5dbfee410914ef6b2f64760625d5d0e"],"layout":"IPY_MODEL_9cd09430cf774f9cbee545ab0c6d0ebb"}},"39fad12de7454ed2b5a794a04bf92f00":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_23fc6ce9e1f94d5391446d1965dbf9ee","placeholder":"​","style":"IPY_MODEL_3aa4b8594b9643f5b0697b7cc52240c5","value":"special_tokens_map.json: 100%"}},"0c3377e47bed4043adc83fa001a9c89d":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_9ee3cdb5605d4d578ca3911051005228","max":112,"min":0,"orientation":"horizontal","style":"IPY_MODEL_09eebdf57aca48a9a23175ad7cdaccce","value":112}},"f5dbfee410914ef6b2f64760625d5d0e":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_267cb4a8fa0742a0b9ecb5b2d1cdc2e6","placeholder":"​","style":"IPY_MODEL_6bcdcd33b5fd496b819fc8c2242740ac","value":" 112/112 [00:00&lt;00:00, 9.10kB/s]"}},"9cd09430cf774f9cbee545ab0c6d0ebb":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"23fc6ce9e1f94d5391446d1965dbf9ee":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"3aa4b8594b9643f5b0697b7cc52240c5":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"9ee3cdb5605d4d578ca3911051005228":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"09eebdf57aca48a9a23175ad7cdaccce":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"267cb4a8fa0742a0b9ecb5b2d1cdc2e6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"6bcdcd33b5fd496b819fc8c2242740ac":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}}}}},"cells":[{"cell_type":"markdown","source":["# MODELO DE APRENDIZAJE BERT PARA PREDICCIÓN DE NEUROPÉPTIDOS"],"metadata":{"id":"c1rR4D8qhj65"}},{"cell_type":"markdown","source":["## INSTALACIÓN E IMPORTACIÓN DE PAQUETES DE TRABAJO"],"metadata":{"id":"iavOzGT3hvsg"}},{"cell_type":"code","execution_count":null,"metadata":{"id":"cYhGE2zhpcgo","colab":{"base_uri":"https://localhost:8080/"},"outputId":"785da899-5aba-4d60-c58b-690ef242c181","executionInfo":{"status":"ok","timestamp":1734253409002,"user_tz":-60,"elapsed":2588,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}}},"outputs":[{"output_type":"stream","name":"stdout","text":["Requirement already satisfied: transformers in /usr/local/lib/python3.10/dist-packages (4.46.3)\n","Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from transformers) (3.16.1)\n","Requirement already satisfied: huggingface-hub<1.0,>=0.23.2 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.26.5)\n","Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.10/dist-packages (from transformers) (1.26.4)\n","Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.10/dist-packages (from transformers) (24.2)\n","Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.10/dist-packages (from transformers) (6.0.2)\n","Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.10/dist-packages (from transformers) (2024.9.11)\n","Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from transformers) (2.32.3)\n","Requirement already satisfied: tokenizers<0.21,>=0.20 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.20.3)\n","Requirement already satisfied: safetensors>=0.4.1 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.4.5)\n","Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.10/dist-packages (from transformers) (4.66.6)\n","Requirement already satisfied: fsspec>=2023.5.0 in /usr/local/lib/python3.10/dist-packages (from huggingface-hub<1.0,>=0.23.2->transformers) (2024.10.0)\n","Requirement already satisfied: typing-extensions>=3.7.4.3 in /usr/local/lib/python3.10/dist-packages (from huggingface-hub<1.0,>=0.23.2->transformers) (4.12.2)\n","Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (3.4.0)\n","Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (3.10)\n","Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (2.2.3)\n","Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (2024.8.30)\n"]}],"source":["!pip install transformers"]},{"cell_type":"code","source":["import os\n","import pandas as pd\n","import numpy as np\n","import torch\n","import re\n","from pprint import pprint\n","from sklearn.metrics import confusion_matrix\n","from sklearn.metrics import accuracy_score, precision_recall_fscore_support\n","from torch.utils.data import Dataset, DataLoader\n","\n","torch.cuda.is_available()\n","device = torch.device(\"cuda\" if torch.cuda.is_available() else \"cpu\")\n","print(device)\n","\n","from transformers import AutoTokenizer, Trainer, TrainingArguments, BertForSequenceClassification, AdamW"],"metadata":{"id":"c8wIZMbgrdGa","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1734253435191,"user_tz":-60,"elapsed":15804,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"7ab89ab1-4269-4405-c053-938f48367004"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["cuda\n"]}]},{"cell_type":"code","source":["os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'  # or any {'0', '1', '2'}\n","import tensorflow as tf"],"metadata":{"id":"-0vtPTdQG90E"},"execution_count":null,"outputs":[]},{"cell_type":"markdown","source":["## FUNCIÓN TOKENIZACIÓN"],"metadata":{"id":"LGHdTS1YiCQd"}},{"cell_type":"code","source":["# Clase mix_data para procesar cualquier DataFrame (df o ddf)\n","class mix_data():\n","    def __init__(self, df, tokenizer_name='Rostlab/prot_bert_bfd', max_len=200):\n","        # Inicializa el tokenizador de BERT y otros parámetros\n","        self.tokenizer = AutoTokenizer.from_pretrained(tokenizer_name, do_lower_case=False)\n","        self.max_len = max_len\n","        self.seqs, self.labels = self.get_seqs_labels(df)  # Aquí pasamos df o ddf como parámetro\n","\n","    def get_seqs_labels(self, df):  # Aquí recibimos el DataFrame que puede ser df o ddf\n","        # Aislar las secuencias de aminoácidos y sus etiquetas\n","        seqs = list(df['aa_seq'])  # Suponiendo que 'aa_seq' tiene las secuencias\n","        labels = list(df['label'].astype(int))  # Suponiendo que 'label' tiene las etiquetas\n","        return seqs, labels\n","\n","    def __len__(self):\n","        return len(self.labels)\n","\n","    def __getitem__(self, idx):\n","        seq = \" \".join(\"\".join(self.seqs[idx].split()))  # Tokeniza la secuencia\n","        seq_ids = self.tokenizer(seq, truncation=True, padding='max_length', max_length=self.max_len)\n","\n","        sample = {key: torch.tensor(val) for key, val in seq_ids.items()}\n","        sample['labels'] = torch.tensor(self.labels[idx])  # Añadir la etiqueta correspondiente\n","        return sample\n","    def __setitem__(self, idx, sample):\n","        # Update the elements of the mix_data object at the given index\n","        for key in sample:\n","            if key in self.__dict__:  # Or any other logic to determine which attributes to update\n","                if isinstance(self.__dict__[key], list): # Assuming your data is stored in lists\n","                    self.__dict__[key][idx] = sample[key]\n","                # Add other conditions for different data structures if needed\n","                else:\n","                    # Handle cases where the attribute is not a list\n","                    pass  # Or raise an error if this should not happen\n"],"metadata":{"id":"S2k-QWVu6Og8"},"execution_count":null,"outputs":[]},{"cell_type":"markdown","source":["## CARGADO DE DATOS"],"metadata":{"id":"HbiI-VuezmVa"}},{"cell_type":"code","source":["# read in the train dataset\n","# create the dataset\n","\n","data_url = 'https://raw.githubusercontent.com/JavierColubi/Javier-Colubi-github/refs/heads/main/train_neuropep-BERT_2.0.txt'\n","df = pd.read_csv(data_url, index_col = 0)\n","df = df.sample(frac=1, random_state = 0)\n","print(df.head(7))\n","print(f'El número de secuencias totales en el archivo es: {len(df)}')\n","\n","train_dataset = mix_data(df)"],"metadata":{"id":"BBymtbXgrdOc","colab":{"base_uri":"https://localhost:8080/","height":601,"referenced_widgets":["8aca7e3d8c5441f087f084f632bab0e5","c7ac9660b133439db3f42a7ac88ddb13","1f0af1dee80c4530a216795a67c44de3","e4c12ad68bed436ca9cfc17006953991","bd808bee84f546eca5837cb789c865f6","bf8ae96cdef447e8b274e81850e64e4a","798292909e834f1398e162aa7c4cf88d","06f8030bd44a4bbab55d0c3cbf011404","7958f8d1a83a44a084ddf15190ec5100","7551e5083e1b43ff859228f61c1969df","da9ad0b570804b19bfa7a618c87cccfd","3c53870ad5984829bacfd49a0214bc8d","aff18f8042984b07a31ee64ecce8fdb6","174d286408e248629c2e5cc5016602d7","e3d1f92c485240759840ca8f0ba163ce","75bcdcd1cfe24ca08f4e6c35402eea5b","881a5dded8e44102b0d15b14585ef09b","1fef4a3f39ec423989fa5e0499cb928b","e2e1c058bf004dacba6237de36fb9e08","02f107c9457547a2a34efa23e466896d","2b0b0ef079b243f9879c48c79e3d0dc2","0c24c7fb581446bca188a9ac4006764a","6ce7405f091f494c87369df393cce96c","33f803786a74420e98c364ecd775b497","7aac207ee63d4c6b8190a3c818ccc9e0","4da994018b144174a1afe1a54bb5e2d6","0edad9e943984baf9d7f37c1640fee75","8e1512c5a9584acea8b7a2cc502487b1","00aba44f26554a9087e473825616e47d","97dc10663885448f8bd6f68bce415856","bde5c315085545dc8d574c74f586d7b0","282faa48f09a437a9e9fb553d5bd513a","561c4f64026a4a00afc058303a8bdc0a","f51813427eb24c178f88a9c9788b2385","c598e40f285746f4a6fc058f9f97f1ad","17172415be384375a09b7a4ecfba02f8","264f7a8109a942e4a1fa49881c2cbe85","838afc4ada694e309a436f99b1ae3ed4","4f0e0a4c99e14aafa3abe44c8c8a5cc9","3f288447d1714d3bb22bf4fa238481bd","a36703a03c5149b690c1bb525bded337","0d7438de152449639c0c8cb0dc960dc9","210a206e59a9422bb085cce6982c7628","1a4b4ede6c424471bfa8b26a41684cfc"]},"executionInfo":{"status":"ok","timestamp":1733490675735,"user_tz":-60,"elapsed":4713,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"b42a4641-e5c1-4d8e-b377-7f69b2261a2b"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["                                                            aa_seq  aa_len  \\\n","non_AMPEP63746                               MAKINELLRESTTTNSNSIGR      21   \n","IDEN1606         LYLKQADFDDPRMFTSSFGKRSAIESEPQAYPKSYRAIRIQRRSMD...      99   \n","IDEN1310                                                  KVKFSAWG       8   \n","UniRef50_Q9ZTK5                  CVEKTFIFPPEAIEKLKSKAVEFGIEKPTRVEV      33   \n","IDEN135                       APLEPMYPGDYATPEQMAQYETQLRRYINTLTRPRY      36   \n","IDEN265                                                  DGRMYSFGL       9   \n","non_AMPEP34652                            THPSVLPFIKQLIGTMDSVRGLPR      24   \n","\n","                 label  \n","non_AMPEP63746       0  \n","IDEN1606             2  \n","IDEN1310             2  \n","UniRef50_Q9ZTK5      0  \n","IDEN135              2  \n","IDEN265              2  \n","non_AMPEP34652       0  \n","El número de secuencias totales en el archivo es: 6266\n"]},{"output_type":"stream","name":"stderr","text":["/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_auth.py:94: UserWarning: \n","The secret `HF_TOKEN` does not exist in your Colab secrets.\n","To authenticate with the Hugging Face Hub, create a token in your settings tab (https://huggingface.co/settings/tokens), set it as secret in your Google Colab and restart your session.\n","You will be able to reuse this secret in all of your notebooks.\n","Please note that authentication is recommended but still optional to access public models or datasets.\n","  warnings.warn(\n"]},{"output_type":"display_data","data":{"text/plain":["tokenizer_config.json:   0%|          | 0.00/86.0 [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"8aca7e3d8c5441f087f084f632bab0e5"}},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["config.json:   0%|          | 0.00/361 [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"3c53870ad5984829bacfd49a0214bc8d"}},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["vocab.txt:   0%|          | 0.00/81.0 [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"6ce7405f091f494c87369df393cce96c"}},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["special_tokens_map.json:   0%|          | 0.00/112 [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"f51813427eb24c178f88a9c9788b2385"}},"metadata":{}}]},{"cell_type":"code","source":["# read in the train dataset\n","# create the dataset\n","\n","data_url2 = 'https://raw.githubusercontent.com/JavierColubi/Javier-Colubi-github/refs/heads/main/validacion_20.txt'\n","ddf = pd.read_csv(data_url2, index_col = 0)\n","ddf = ddf.sample(frac=1, random_state = 0)\n","print(ddf.head(7))\n","print(f'El número de secuencias totales en el archivo es: {len(ddf)}')\n","\n","val_dataset = mix_data(ddf)"],"metadata":{"id":"KGgW4gVL6eCJ","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1733490680736,"user_tz":-60,"elapsed":833,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"51907b02-dabb-48ce-d89d-1b8d311dab6a"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["                                                            aa_seq  aa_len  \\\n","DRAMP04081                                     AKKVFKRLEKLFSKIFNFK      19   \n","IDEN2055                                           QPPYLDLTPAYFHIR      15   \n","non_AMPEP52184                                     AMTEELEAVENNGVR      15   \n","non_AMPEP149113                                      PFDRISNSAFSDF      13   \n","IDEN1925                   LVMAGYERRGIQKRHGEQGITCECCYNHCSFRELVQYCN      39   \n","IDEN2021                     EAVPSQVLSEQNEEAGAPLSPLSEMPPWMGEVNPAQR      37   \n","DRAMP00144       NRWTNAYSAALGCAVPGVKYGKKLGGVWGAVIGGVGGAAVCGLAGY...      50   \n","\n","                 label  \n","DRAMP04081           1  \n","IDEN2055             2  \n","non_AMPEP52184       0  \n","non_AMPEP149113      0  \n","IDEN1925             2  \n","IDEN2021             2  \n","DRAMP00144           1  \n","El número de secuencias totales en el archivo es: 1065\n"]}]},{"cell_type":"markdown","source":["## FUNCIÓN DE EVALUCIÓN DEL ENTRENAMIENTO"],"metadata":{"id":"emIbL4LPiTh9"}},{"cell_type":"code","source":["def compute_metrics(pred):\n","    labels = pred.label_ids\n","    preds = pred.predictions.argmax(-1)\n","\n","    # Calcular métricas globales (macro)\n","    precision_macro, recall_macro, f1_macro, _ = precision_recall_fscore_support(labels, preds, average='weighted')\n","    acc = accuracy_score(labels, preds)\n","\n","    # Calcular métricas por clase\n","    precision_per_label, recall_per_label, f1_per_label, support_per_label = precision_recall_fscore_support(\n","        labels, preds, average=None\n","    )\n","\n","    # Confusion matrix\n","    conf_matrix = confusion_matrix(labels, preds).tolist()  # Convertir a lista para serialización en JSON si es necesario\n","\n","    # Crear el diccionario de métricas\n","    metrics = {\n","        'accuracy': acc,\n","        'precision_macro': precision_macro,\n","        'recall_macro': recall_macro,\n","        'f1_macro': f1_macro,\n","    }\n","\n","    # Agregar métricas por clase al diccionario\n","    for i, precision in enumerate(precision_per_label):\n","        metrics[f'precision_label_{i}'] = precision  # Unique key for each label's precision\n","    for i, recall in enumerate(recall_per_label):\n","        metrics[f'recall_label_{i}'] = recall  # Unique key for each label's recall\n","    for i, f1 in enumerate(f1_per_label):\n","        metrics[f'f1_label_{i}'] = f1  # Unique key for each label's F1-score\n","    # ... (add other metrics as needed) ...\n","\n","    return metrics"],"metadata":{"id":"n4jrGQT6rdRA"},"execution_count":null,"outputs":[]},{"cell_type":"markdown","source":["## MODELO DE ENTRENAMIENTO"],"metadata":{"id":"-IDmLYa6iadf"}},{"cell_type":"code","source":["# define the model initializing function for Trainer in huggingface\n","\n","def model_init():\n","    model = BertForSequenceClassification.from_pretrained('Rostlab/prot_bert_bfd', num_labels=3)\n","    #dropout_rate = 0.5  # Cambia el valor según lo que quieras (0.1 a 0.5 suele ser común)\n","    #model.bert.encoder.config.hidden_dropout_prob = dropout_rate  # Dropout en las capas internas\n","    #model.bert.encoder.config.attention_probs_dropout_prob = dropout_rate  # Dropout en la atención\n","    #model.dropout = torch.nn.Dropout(p=dropout_rate)  # Dropout en la capa final (si está definida)\n","    return model"],"metadata":{"id":"j-pQako9rdTQ"},"execution_count":null,"outputs":[]},{"cell_type":"code","source":["# training on entire data\n","# no evaluation/validation\n","\n","training_args = TrainingArguments(\n","    output_dir='./results',\n","    num_train_epochs=10,\n","    learning_rate = 5e-5,\n","    per_device_train_batch_size=1,\n","    warmup_steps=0,\n","    weight_decay=0.1,\n","    logging_dir='./logs',\n","    do_train=True,\n","    do_eval=True,\n","    evaluation_strategy=\"epoch\",\n","    save_strategy='epoch',\n","    logging_strategy='epoch',\n","    gradient_accumulation_steps=64,\n","    fp16=True,\n","    fp16_opt_level=\"O2\",\n","    run_name=\"neuropep_BERT\",\n","    seed=0,\n","    load_best_model_at_end = True,\n","    metric_for_best_model=\"eval_accuracy\",  # Seleccionar el mejor modelo basado en precisión\n","    greater_is_better=True        # La precisión es mejor si es mayor\n",")\n","\n","trainer = Trainer(\n","    model_init=model_init,\n","    args=training_args,\n","    train_dataset=train_dataset,\n","    eval_dataset=val_dataset,\n","    compute_metrics = compute_metrics,\n",")\n","trainer.train()"],"metadata":{"id":"hk0ETN8brdVR","colab":{"base_uri":"https://localhost:8080/","height":799,"referenced_widgets":["43403696d8f948fcb8bfdaa922686a00","c4f8e28467f041d4baaef90e2a264b02","f575a47e1aca4b498f3cef7f459667c3","8cd81d9e97f3462a86aae6fa10681169","e634f0c7416d48cf940aa41f98c24877","c7cd34c5376c4156a121b2d4c5cd9966","fa0aa80ea4c44d9fa7a40acf6535aac3","ff12d7c242cf4ca78bb2716ef003d84c","0a7f07678ce140a48dcc1fc9559ff90d","64779eff82ad4dbc9bf73a8d51cc3801","67730cdb4c0d44b3bd99bc523e69371d"]},"outputId":"f3d3aa2f-35d5-47c1-9ac6-729dcd394fbe","executionInfo":{"status":"ok","timestamp":1733497141499,"user_tz":-60,"elapsed":6434634,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}}},"execution_count":null,"outputs":[{"output_type":"stream","name":"stderr","text":["/usr/local/lib/python3.10/dist-packages/transformers/training_args.py:1568: FutureWarning: `evaluation_strategy` is deprecated and will be removed in version 4.46 of 🤗 Transformers. Use `eval_strategy` instead\n","  warnings.warn(\n"]},{"output_type":"display_data","data":{"text/plain":["pytorch_model.bin:   0%|          | 0.00/1.68G [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"43403696d8f948fcb8bfdaa922686a00"}},"metadata":{}},{"output_type":"stream","name":"stderr","text":["Some weights of BertForSequenceClassification were not initialized from the model checkpoint at Rostlab/prot_bert_bfd and are newly initialized: ['classifier.bias', 'classifier.weight']\n","You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.\n","Some weights of BertForSequenceClassification were not initialized from the model checkpoint at Rostlab/prot_bert_bfd and are newly initialized: ['classifier.bias', 'classifier.weight']\n","You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.\n","\u001b[34m\u001b[1mwandb\u001b[0m: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.\n"]},{"output_type":"display_data","data":{"text/plain":["<IPython.core.display.Javascript object>"],"application/javascript":["\n","        window._wandbApiKey = new Promise((resolve, reject) => {\n","            function loadScript(url) {\n","            return new Promise(function(resolve, reject) {\n","                let newScript = document.createElement(\"script\");\n","                newScript.onerror = reject;\n","                newScript.onload = resolve;\n","                document.body.appendChild(newScript);\n","                newScript.src = url;\n","            });\n","            }\n","            loadScript(\"https://cdn.jsdelivr.net/npm/postmate/build/postmate.min.js\").then(() => {\n","            const iframe = document.createElement('iframe')\n","            iframe.style.cssText = \"width:0;height:0;border:none\"\n","            document.body.appendChild(iframe)\n","            const handshake = new Postmate({\n","                container: iframe,\n","                url: 'https://wandb.ai/authorize'\n","            });\n","            const timeout = setTimeout(() => reject(\"Couldn't auto authenticate\"), 5000)\n","            handshake.then(function(child) {\n","                child.on('authorize', data => {\n","                    clearTimeout(timeout)\n","                    resolve(data)\n","                });\n","            });\n","            })\n","        });\n","    "]},"metadata":{}},{"output_type":"stream","name":"stderr","text":["\u001b[34m\u001b[1mwandb\u001b[0m: Logging into wandb.ai. (Learn how to deploy a W&B server locally: https://wandb.me/wandb-server)\n","\u001b[34m\u001b[1mwandb\u001b[0m: You can find your API key in your browser here: https://wandb.ai/authorize\n","wandb: Paste an API key from your profile and hit enter, or press ctrl+c to quit:"]},{"name":"stdout","output_type":"stream","text":[" ··········\n"]},{"output_type":"stream","name":"stderr","text":["\u001b[34m\u001b[1mwandb\u001b[0m: Appending key for api.wandb.ai to your netrc file: /root/.netrc\n"]},{"output_type":"display_data","data":{"text/plain":["<IPython.core.display.HTML object>"],"text/html":["Tracking run with wandb version 0.18.7"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["<IPython.core.display.HTML object>"],"text/html":["Run data is saved locally in <code>/content/wandb/run-20241206_131244-kpobcmtv</code>"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["<IPython.core.display.HTML object>"],"text/html":["Syncing run <strong><a href='https://wandb.ai/jacoldom4201-universidad-politecnica-de-madrid/huggingface/runs/kpobcmtv' target=\"_blank\">neuropep_BERT</a></strong> to <a href='https://wandb.ai/jacoldom4201-universidad-politecnica-de-madrid/huggingface' target=\"_blank\">Weights & Biases</a> (<a href='https://wandb.me/developer-guide' target=\"_blank\">docs</a>)<br/>"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["<IPython.core.display.HTML object>"],"text/html":[" View project at <a href='https://wandb.ai/jacoldom4201-universidad-politecnica-de-madrid/huggingface' target=\"_blank\">https://wandb.ai/jacoldom4201-universidad-politecnica-de-madrid/huggingface</a>"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["<IPython.core.display.HTML object>"],"text/html":[" View run at <a href='https://wandb.ai/jacoldom4201-universidad-politecnica-de-madrid/huggingface/runs/kpobcmtv' target=\"_blank\">https://wandb.ai/jacoldom4201-universidad-politecnica-de-madrid/huggingface/runs/kpobcmtv</a>"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["<IPython.core.display.HTML object>"],"text/html":["\n","    <div>\n","      \n","      <progress value='970' max='970' style='width:300px; height:20px; vertical-align: middle;'></progress>\n","      [970/970 1:46:04, Epoch 9/10]\n","    </div>\n","    <table border=\"1\" class=\"dataframe\">\n","  <thead>\n"," <tr style=\"text-align: left;\">\n","      <th>Epoch</th>\n","      <th>Training Loss</th>\n","      <th>Validation Loss</th>\n","      <th>Accuracy</th>\n","      <th>Precision Macro</th>\n","      <th>Recall Macro</th>\n","      <th>F1 Macro</th>\n","      <th>Precision Label 0</th>\n","      <th>Precision Label 1</th>\n","      <th>Precision Label 2</th>\n","      <th>Recall Label 0</th>\n","      <th>Recall Label 1</th>\n","      <th>Recall Label 2</th>\n","      <th>F1 Label 0</th>\n","      <th>F1 Label 1</th>\n","      <th>F1 Label 2</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <td>0</td>\n","      <td>0.846400</td>\n","      <td>1.051599</td>\n","      <td>0.520188</td>\n","      <td>0.524410</td>\n","      <td>0.520188</td>\n","      <td>0.521466</td>\n","      <td>0.369509</td>\n","      <td>0.665746</td>\n","      <td>0.537975</td>\n","      <td>0.402817</td>\n","      <td>0.678873</td>\n","      <td>0.478873</td>\n","      <td>0.385445</td>\n","      <td>0.672245</td>\n","      <td>0.506706</td>\n","    </tr>\n","    <tr>\n","      <td>1</td>\n","      <td>0.624800</td>\n","      <td>1.031513</td>\n","      <td>0.584038</td>\n","      <td>0.566439</td>\n","      <td>0.584038</td>\n","      <td>0.564214</td>\n","      <td>0.429864</td>\n","      <td>0.713896</td>\n","      <td>0.555556</td>\n","      <td>0.267606</td>\n","      <td>0.738028</td>\n","      <td>0.746479</td>\n","      <td>0.329861</td>\n","      <td>0.725762</td>\n","      <td>0.637019</td>\n","    </tr>\n","    <tr>\n","      <td>2</td>\n","      <td>0.526000</td>\n","      <td>1.154910</td>\n","      <td>0.568075</td>\n","      <td>0.574182</td>\n","      <td>0.568075</td>\n","      <td>0.567723</td>\n","      <td>0.408669</td>\n","      <td>0.773463</td>\n","      <td>0.540416</td>\n","      <td>0.371831</td>\n","      <td>0.673239</td>\n","      <td>0.659155</td>\n","      <td>0.389381</td>\n","      <td>0.719880</td>\n","      <td>0.593909</td>\n","    </tr>\n","    <tr>\n","      <td>3</td>\n","      <td>0.446100</td>\n","      <td>1.015441</td>\n","      <td>0.610329</td>\n","      <td>0.609560</td>\n","      <td>0.610329</td>\n","      <td>0.605642</td>\n","      <td>0.471761</td>\n","      <td>0.763240</td>\n","      <td>0.593679</td>\n","      <td>0.400000</td>\n","      <td>0.690141</td>\n","      <td>0.740845</td>\n","      <td>0.432927</td>\n","      <td>0.724852</td>\n","      <td>0.659148</td>\n","    </tr>\n","    <tr>\n","      <td>4</td>\n","      <td>0.369300</td>\n","      <td>1.304759</td>\n","      <td>0.590610</td>\n","      <td>0.583127</td>\n","      <td>0.590610</td>\n","      <td>0.575300</td>\n","      <td>0.434426</td>\n","      <td>0.749216</td>\n","      <td>0.565737</td>\n","      <td>0.298592</td>\n","      <td>0.673239</td>\n","      <td>0.800000</td>\n","      <td>0.353923</td>\n","      <td>0.709199</td>\n","      <td>0.662777</td>\n","    </tr>\n","    <tr>\n","      <td>5</td>\n","      <td>0.289700</td>\n","      <td>1.304990</td>\n","      <td>0.619718</td>\n","      <td>0.615037</td>\n","      <td>0.619718</td>\n","      <td>0.615362</td>\n","      <td>0.466238</td>\n","      <td>0.756598</td>\n","      <td>0.622276</td>\n","      <td>0.408451</td>\n","      <td>0.726761</td>\n","      <td>0.723944</td>\n","      <td>0.435435</td>\n","      <td>0.741379</td>\n","      <td>0.669271</td>\n","    </tr>\n","    <tr>\n","      <td>6</td>\n","      <td>0.236000</td>\n","      <td>1.459364</td>\n","      <td>0.606573</td>\n","      <td>0.595702</td>\n","      <td>0.606573</td>\n","      <td>0.597202</td>\n","      <td>0.456835</td>\n","      <td>0.729805</td>\n","      <td>0.600467</td>\n","      <td>0.357746</td>\n","      <td>0.738028</td>\n","      <td>0.723944</td>\n","      <td>0.401264</td>\n","      <td>0.733894</td>\n","      <td>0.656450</td>\n","    </tr>\n","    <tr>\n","      <td>7</td>\n","      <td>0.194000</td>\n","      <td>1.474323</td>\n","      <td>0.630047</td>\n","      <td>0.618560</td>\n","      <td>0.630047</td>\n","      <td>0.612738</td>\n","      <td>0.506494</td>\n","      <td>0.742938</td>\n","      <td>0.606250</td>\n","      <td>0.329577</td>\n","      <td>0.740845</td>\n","      <td>0.819718</td>\n","      <td>0.399317</td>\n","      <td>0.741890</td>\n","      <td>0.697006</td>\n","    </tr>\n","    <tr>\n","      <td>8</td>\n","      <td>0.169500</td>\n","      <td>1.558082</td>\n","      <td>0.620657</td>\n","      <td>0.609900</td>\n","      <td>0.620657</td>\n","      <td>0.610528</td>\n","      <td>0.485294</td>\n","      <td>0.727778</td>\n","      <td>0.616628</td>\n","      <td>0.371831</td>\n","      <td>0.738028</td>\n","      <td>0.752113</td>\n","      <td>0.421053</td>\n","      <td>0.732867</td>\n","      <td>0.677665</td>\n","    </tr>\n","    <tr>\n","      <td>9</td>\n","      <td>0.146800</td>\n","      <td>1.620241</td>\n","      <td>0.624413</td>\n","      <td>0.611995</td>\n","      <td>0.624413</td>\n","      <td>0.613578</td>\n","      <td>0.494424</td>\n","      <td>0.711230</td>\n","      <td>0.630332</td>\n","      <td>0.374648</td>\n","      <td>0.749296</td>\n","      <td>0.749296</td>\n","      <td>0.426282</td>\n","      <td>0.729767</td>\n","      <td>0.684685</td>\n","    </tr>\n","  </tbody>\n","</table><p>"]},"metadata":{}},{"output_type":"execute_result","data":{"text/plain":["TrainOutput(global_step=970, training_loss=0.38658978275417055, metrics={'train_runtime': 6417.909, 'train_samples_per_second': 9.763, 'train_steps_per_second': 0.151, 'total_flos': 2.8229621737728e+16, 'train_loss': 0.38658978275417055, 'epoch': 9.90743696137887})"]},"metadata":{},"execution_count":9}]},{"cell_type":"markdown","source":["## MÉTRICAS DEL MODELO"],"metadata":{"id":"RfUEpHRkiiTh"}},{"cell_type":"code","source":["# Imprimir el historial de logs del entrenador\n","print(trainer.state.log_history)\n","\n","# Analizar la relación entre pasos y épocas\n","for log in trainer.state.log_history:\n","    if 'epoch' in log:\n","        print(f\"Step: {log['step']}, Epoch: {log['epoch']}, Metrics: {log}\")"],"metadata":{"id":"uBWgBQkdrLrb","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1733497822521,"user_tz":-60,"elapsed":470,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"debe461b-2199-41ac-97ea-8d903974a25e"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["[{'loss': 0.8464, 'grad_norm': 239.1932373046875, 'learning_rate': 4.5463917525773195e-05, 'epoch': 0.990743696137887, 'step': 97}, {'eval_loss': 1.0515987873077393, 'eval_accuracy': 0.5201877934272301, 'eval_precision_macro': 0.5244098612751812, 'eval_recall_macro': 0.5201877934272301, 'eval_f1_macro': 0.521465539835203, 'eval_precision_label_0': 0.3695090439276486, 'eval_precision_label_1': 0.6657458563535912, 'eval_precision_label_2': 0.5379746835443038, 'eval_recall_label_0': 0.4028169014084507, 'eval_recall_label_1': 0.6788732394366197, 'eval_recall_label_2': 0.4788732394366197, 'eval_f1_label_0': 0.38544474393531, 'eval_f1_label_1': 0.6722454672245467, 'eval_f1_label_2': 0.5067064083457526, 'eval_runtime': 12.7988, 'eval_samples_per_second': 83.211, 'eval_steps_per_second': 10.47, 'epoch': 0.990743696137887, 'step': 97}, {'loss': 0.6248, 'grad_norm': 3447.231689453125, 'learning_rate': 4.041237113402062e-05, 'epoch': 1.991701244813278, 'step': 195}, {'eval_loss': 1.0315132141113281, 'eval_accuracy': 0.584037558685446, 'eval_precision_macro': 0.5664387555716295, 'eval_recall_macro': 0.584037558685446, 'eval_f1_macro': 0.5642140382445092, 'eval_precision_label_0': 0.4298642533936652, 'eval_precision_label_1': 0.7138964577656676, 'eval_precision_label_2': 0.5555555555555556, 'eval_recall_label_0': 0.2676056338028169, 'eval_recall_label_1': 0.7380281690140845, 'eval_recall_label_2': 0.7464788732394366, 'eval_f1_label_0': 0.3298611111111111, 'eval_f1_label_1': 0.7257617728531855, 'eval_f1_label_2': 0.6370192307692307, 'eval_runtime': 12.7696, 'eval_samples_per_second': 83.401, 'eval_steps_per_second': 10.494, 'epoch': 1.991701244813278, 'step': 195}, {'loss': 0.526, 'grad_norm': 437.5106506347656, 'learning_rate': 3.5360824742268044e-05, 'epoch': 2.992658793488669, 'step': 293}, {'eval_loss': 1.1549098491668701, 'eval_accuracy': 0.568075117370892, 'eval_precision_macro': 0.5741824060698889, 'eval_recall_macro': 0.568075117370892, 'eval_f1_macro': 0.5677228928291216, 'eval_precision_label_0': 0.4086687306501548, 'eval_precision_label_1': 0.7734627831715211, 'eval_precision_label_2': 0.5404157043879908, 'eval_recall_label_0': 0.37183098591549296, 'eval_recall_label_1': 0.6732394366197183, 'eval_recall_label_2': 0.6591549295774648, 'eval_f1_label_0': 0.3893805309734513, 'eval_f1_label_1': 0.7198795180722891, 'eval_f1_label_2': 0.5939086294416244, 'eval_runtime': 12.8081, 'eval_samples_per_second': 83.151, 'eval_steps_per_second': 10.462, 'epoch': 2.992658793488669, 'step': 293}, {'loss': 0.4461, 'grad_norm': 360.1712951660156, 'learning_rate': 3.036082474226804e-05, 'epoch': 3.99361634216406, 'step': 391}, {'eval_loss': 1.0154414176940918, 'eval_accuracy': 0.6103286384976526, 'eval_precision_macro': 0.6095600436569596, 'eval_recall_macro': 0.6103286384976526, 'eval_f1_macro': 0.6056422566494651, 'eval_precision_label_0': 0.4717607973421927, 'eval_precision_label_1': 0.7632398753894081, 'eval_precision_label_2': 0.5936794582392777, 'eval_recall_label_0': 0.4, 'eval_recall_label_1': 0.6901408450704225, 'eval_recall_label_2': 0.7408450704225352, 'eval_f1_label_0': 0.4329268292682927, 'eval_f1_label_1': 0.7248520710059172, 'eval_f1_label_2': 0.6591478696741855, 'eval_runtime': 12.8344, 'eval_samples_per_second': 82.98, 'eval_steps_per_second': 10.441, 'epoch': 3.99361634216406, 'step': 391}, {'loss': 0.3693, 'grad_norm': 249.04336547851562, 'learning_rate': 2.5309278350515462e-05, 'epoch': 4.994573890839451, 'step': 489}, {'eval_loss': 1.3047586679458618, 'eval_accuracy': 0.5906103286384976, 'eval_precision_macro': 0.5831265274138214, 'eval_recall_macro': 0.5906103286384976, 'eval_f1_macro': 0.5752997159734013, 'eval_precision_label_0': 0.4344262295081967, 'eval_precision_label_1': 0.7492163009404389, 'eval_precision_label_2': 0.5657370517928287, 'eval_recall_label_0': 0.29859154929577464, 'eval_recall_label_1': 0.6732394366197183, 'eval_recall_label_2': 0.8, 'eval_f1_label_0': 0.35392320534223703, 'eval_f1_label_1': 0.7091988130563798, 'eval_f1_label_2': 0.662777129521587, 'eval_runtime': 12.5637, 'eval_samples_per_second': 84.768, 'eval_steps_per_second': 10.666, 'epoch': 4.994573890839451, 'step': 489}, {'loss': 0.2897, 'grad_norm': 524.452392578125, 'learning_rate': 2.0257731958762887e-05, 'epoch': 5.995531439514842, 'step': 587}, {'eval_loss': 1.3049896955490112, 'eval_accuracy': 0.6197183098591549, 'eval_precision_macro': 0.6150374038823616, 'eval_recall_macro': 0.6197183098591549, 'eval_f1_macro': 0.6153618597045322, 'eval_precision_label_0': 0.4662379421221865, 'eval_precision_label_1': 0.7565982404692082, 'eval_precision_label_2': 0.6222760290556901, 'eval_recall_label_0': 0.4084507042253521, 'eval_recall_label_1': 0.7267605633802817, 'eval_recall_label_2': 0.723943661971831, 'eval_f1_label_0': 0.43543543543543545, 'eval_f1_label_1': 0.7413793103448276, 'eval_f1_label_2': 0.6692708333333334, 'eval_runtime': 12.7813, 'eval_samples_per_second': 83.325, 'eval_steps_per_second': 10.484, 'epoch': 5.995531439514842, 'step': 587}, {'loss': 0.236, 'grad_norm': 589.4967651367188, 'learning_rate': 1.5206185567010308e-05, 'epoch': 6.996488988190233, 'step': 685}, {'eval_loss': 1.4593641757965088, 'eval_accuracy': 0.6065727699530516, 'eval_precision_macro': 0.5957022786737678, 'eval_recall_macro': 0.6065727699530516, 'eval_f1_macro': 0.5972023111630057, 'eval_precision_label_0': 0.4568345323741007, 'eval_precision_label_1': 0.7298050139275766, 'eval_precision_label_2': 0.6004672897196262, 'eval_recall_label_0': 0.35774647887323946, 'eval_recall_label_1': 0.7380281690140845, 'eval_recall_label_2': 0.723943661971831, 'eval_f1_label_0': 0.40126382306477093, 'eval_f1_label_1': 0.7338935574229691, 'eval_f1_label_2': 0.6564495530012772, 'eval_runtime': 12.7497, 'eval_samples_per_second': 83.531, 'eval_steps_per_second': 10.51, 'epoch': 6.996488988190233, 'step': 685}, {'loss': 0.194, 'grad_norm': 586.6455078125, 'learning_rate': 1.0154639175257733e-05, 'epoch': 7.997446536865624, 'step': 783}, {'eval_loss': 1.4743232727050781, 'eval_accuracy': 0.6300469483568075, 'eval_precision_macro': 0.6185604532002836, 'eval_recall_macro': 0.6300469483568075, 'eval_f1_macro': 0.6127377933543082, 'eval_precision_label_0': 0.5064935064935064, 'eval_precision_label_1': 0.7429378531073446, 'eval_precision_label_2': 0.60625, 'eval_recall_label_0': 0.3295774647887324, 'eval_recall_label_1': 0.7408450704225352, 'eval_recall_label_2': 0.819718309859155, 'eval_f1_label_0': 0.3993174061433447, 'eval_f1_label_1': 0.7418899858956276, 'eval_f1_label_2': 0.6970059880239521, 'eval_runtime': 12.7629, 'eval_samples_per_second': 83.445, 'eval_steps_per_second': 10.499, 'epoch': 7.997446536865624, 'step': 783}, {'loss': 0.1695, 'grad_norm': 396.6549987792969, 'learning_rate': 5.1030927835051544e-06, 'epoch': 8.998404085541015, 'step': 881}, {'eval_loss': 1.558082103729248, 'eval_accuracy': 0.6206572769953052, 'eval_precision_macro': 0.6099000236481557, 'eval_recall_macro': 0.6206572769953052, 'eval_f1_macro': 0.6105282463551231, 'eval_precision_label_0': 0.4852941176470588, 'eval_precision_label_1': 0.7277777777777777, 'eval_precision_label_2': 0.6166281755196305, 'eval_recall_label_0': 0.37183098591549296, 'eval_recall_label_1': 0.7380281690140845, 'eval_recall_label_2': 0.752112676056338, 'eval_f1_label_0': 0.42105263157894735, 'eval_f1_label_1': 0.7328671328671329, 'eval_f1_label_2': 0.6776649746192893, 'eval_runtime': 12.7986, 'eval_samples_per_second': 83.212, 'eval_steps_per_second': 10.47, 'epoch': 8.998404085541015, 'step': 881}, {'loss': 0.1468, 'grad_norm': 77.30679321289062, 'learning_rate': 5.154639175257732e-07, 'epoch': 9.90743696137887, 'step': 970}, {'eval_loss': 1.6202409267425537, 'eval_accuracy': 0.6244131455399061, 'eval_precision_macro': 0.6119951639667093, 'eval_recall_macro': 0.6244131455399061, 'eval_f1_macro': 0.6135778466025379, 'eval_precision_label_0': 0.4944237918215613, 'eval_precision_label_1': 0.7112299465240641, 'eval_precision_label_2': 0.6303317535545023, 'eval_recall_label_0': 0.37464788732394366, 'eval_recall_label_1': 0.7492957746478873, 'eval_recall_label_2': 0.7492957746478873, 'eval_f1_label_0': 0.42628205128205127, 'eval_f1_label_1': 0.7297668038408779, 'eval_f1_label_2': 0.6846846846846847, 'eval_runtime': 12.8073, 'eval_samples_per_second': 83.156, 'eval_steps_per_second': 10.463, 'epoch': 9.90743696137887, 'step': 970}, {'train_runtime': 6417.909, 'train_samples_per_second': 9.763, 'train_steps_per_second': 0.151, 'total_flos': 2.8229621737728e+16, 'train_loss': 0.38658978275417055, 'epoch': 9.90743696137887, 'step': 970}]\n","Step: 97, Epoch: 0.990743696137887, Metrics: {'loss': 0.8464, 'grad_norm': 239.1932373046875, 'learning_rate': 4.5463917525773195e-05, 'epoch': 0.990743696137887, 'step': 97}\n","Step: 97, Epoch: 0.990743696137887, Metrics: {'eval_loss': 1.0515987873077393, 'eval_accuracy': 0.5201877934272301, 'eval_precision_macro': 0.5244098612751812, 'eval_recall_macro': 0.5201877934272301, 'eval_f1_macro': 0.521465539835203, 'eval_precision_label_0': 0.3695090439276486, 'eval_precision_label_1': 0.6657458563535912, 'eval_precision_label_2': 0.5379746835443038, 'eval_recall_label_0': 0.4028169014084507, 'eval_recall_label_1': 0.6788732394366197, 'eval_recall_label_2': 0.4788732394366197, 'eval_f1_label_0': 0.38544474393531, 'eval_f1_label_1': 0.6722454672245467, 'eval_f1_label_2': 0.5067064083457526, 'eval_runtime': 12.7988, 'eval_samples_per_second': 83.211, 'eval_steps_per_second': 10.47, 'epoch': 0.990743696137887, 'step': 97}\n","Step: 195, Epoch: 1.991701244813278, Metrics: {'loss': 0.6248, 'grad_norm': 3447.231689453125, 'learning_rate': 4.041237113402062e-05, 'epoch': 1.991701244813278, 'step': 195}\n","Step: 195, Epoch: 1.991701244813278, Metrics: {'eval_loss': 1.0315132141113281, 'eval_accuracy': 0.584037558685446, 'eval_precision_macro': 0.5664387555716295, 'eval_recall_macro': 0.584037558685446, 'eval_f1_macro': 0.5642140382445092, 'eval_precision_label_0': 0.4298642533936652, 'eval_precision_label_1': 0.7138964577656676, 'eval_precision_label_2': 0.5555555555555556, 'eval_recall_label_0': 0.2676056338028169, 'eval_recall_label_1': 0.7380281690140845, 'eval_recall_label_2': 0.7464788732394366, 'eval_f1_label_0': 0.3298611111111111, 'eval_f1_label_1': 0.7257617728531855, 'eval_f1_label_2': 0.6370192307692307, 'eval_runtime': 12.7696, 'eval_samples_per_second': 83.401, 'eval_steps_per_second': 10.494, 'epoch': 1.991701244813278, 'step': 195}\n","Step: 293, Epoch: 2.992658793488669, Metrics: {'loss': 0.526, 'grad_norm': 437.5106506347656, 'learning_rate': 3.5360824742268044e-05, 'epoch': 2.992658793488669, 'step': 293}\n","Step: 293, Epoch: 2.992658793488669, Metrics: {'eval_loss': 1.1549098491668701, 'eval_accuracy': 0.568075117370892, 'eval_precision_macro': 0.5741824060698889, 'eval_recall_macro': 0.568075117370892, 'eval_f1_macro': 0.5677228928291216, 'eval_precision_label_0': 0.4086687306501548, 'eval_precision_label_1': 0.7734627831715211, 'eval_precision_label_2': 0.5404157043879908, 'eval_recall_label_0': 0.37183098591549296, 'eval_recall_label_1': 0.6732394366197183, 'eval_recall_label_2': 0.6591549295774648, 'eval_f1_label_0': 0.3893805309734513, 'eval_f1_label_1': 0.7198795180722891, 'eval_f1_label_2': 0.5939086294416244, 'eval_runtime': 12.8081, 'eval_samples_per_second': 83.151, 'eval_steps_per_second': 10.462, 'epoch': 2.992658793488669, 'step': 293}\n","Step: 391, Epoch: 3.99361634216406, Metrics: {'loss': 0.4461, 'grad_norm': 360.1712951660156, 'learning_rate': 3.036082474226804e-05, 'epoch': 3.99361634216406, 'step': 391}\n","Step: 391, Epoch: 3.99361634216406, Metrics: {'eval_loss': 1.0154414176940918, 'eval_accuracy': 0.6103286384976526, 'eval_precision_macro': 0.6095600436569596, 'eval_recall_macro': 0.6103286384976526, 'eval_f1_macro': 0.6056422566494651, 'eval_precision_label_0': 0.4717607973421927, 'eval_precision_label_1': 0.7632398753894081, 'eval_precision_label_2': 0.5936794582392777, 'eval_recall_label_0': 0.4, 'eval_recall_label_1': 0.6901408450704225, 'eval_recall_label_2': 0.7408450704225352, 'eval_f1_label_0': 0.4329268292682927, 'eval_f1_label_1': 0.7248520710059172, 'eval_f1_label_2': 0.6591478696741855, 'eval_runtime': 12.8344, 'eval_samples_per_second': 82.98, 'eval_steps_per_second': 10.441, 'epoch': 3.99361634216406, 'step': 391}\n","Step: 489, Epoch: 4.994573890839451, Metrics: {'loss': 0.3693, 'grad_norm': 249.04336547851562, 'learning_rate': 2.5309278350515462e-05, 'epoch': 4.994573890839451, 'step': 489}\n","Step: 489, Epoch: 4.994573890839451, Metrics: {'eval_loss': 1.3047586679458618, 'eval_accuracy': 0.5906103286384976, 'eval_precision_macro': 0.5831265274138214, 'eval_recall_macro': 0.5906103286384976, 'eval_f1_macro': 0.5752997159734013, 'eval_precision_label_0': 0.4344262295081967, 'eval_precision_label_1': 0.7492163009404389, 'eval_precision_label_2': 0.5657370517928287, 'eval_recall_label_0': 0.29859154929577464, 'eval_recall_label_1': 0.6732394366197183, 'eval_recall_label_2': 0.8, 'eval_f1_label_0': 0.35392320534223703, 'eval_f1_label_1': 0.7091988130563798, 'eval_f1_label_2': 0.662777129521587, 'eval_runtime': 12.5637, 'eval_samples_per_second': 84.768, 'eval_steps_per_second': 10.666, 'epoch': 4.994573890839451, 'step': 489}\n","Step: 587, Epoch: 5.995531439514842, Metrics: {'loss': 0.2897, 'grad_norm': 524.452392578125, 'learning_rate': 2.0257731958762887e-05, 'epoch': 5.995531439514842, 'step': 587}\n","Step: 587, Epoch: 5.995531439514842, Metrics: {'eval_loss': 1.3049896955490112, 'eval_accuracy': 0.6197183098591549, 'eval_precision_macro': 0.6150374038823616, 'eval_recall_macro': 0.6197183098591549, 'eval_f1_macro': 0.6153618597045322, 'eval_precision_label_0': 0.4662379421221865, 'eval_precision_label_1': 0.7565982404692082, 'eval_precision_label_2': 0.6222760290556901, 'eval_recall_label_0': 0.4084507042253521, 'eval_recall_label_1': 0.7267605633802817, 'eval_recall_label_2': 0.723943661971831, 'eval_f1_label_0': 0.43543543543543545, 'eval_f1_label_1': 0.7413793103448276, 'eval_f1_label_2': 0.6692708333333334, 'eval_runtime': 12.7813, 'eval_samples_per_second': 83.325, 'eval_steps_per_second': 10.484, 'epoch': 5.995531439514842, 'step': 587}\n","Step: 685, Epoch: 6.996488988190233, Metrics: {'loss': 0.236, 'grad_norm': 589.4967651367188, 'learning_rate': 1.5206185567010308e-05, 'epoch': 6.996488988190233, 'step': 685}\n","Step: 685, Epoch: 6.996488988190233, Metrics: {'eval_loss': 1.4593641757965088, 'eval_accuracy': 0.6065727699530516, 'eval_precision_macro': 0.5957022786737678, 'eval_recall_macro': 0.6065727699530516, 'eval_f1_macro': 0.5972023111630057, 'eval_precision_label_0': 0.4568345323741007, 'eval_precision_label_1': 0.7298050139275766, 'eval_precision_label_2': 0.6004672897196262, 'eval_recall_label_0': 0.35774647887323946, 'eval_recall_label_1': 0.7380281690140845, 'eval_recall_label_2': 0.723943661971831, 'eval_f1_label_0': 0.40126382306477093, 'eval_f1_label_1': 0.7338935574229691, 'eval_f1_label_2': 0.6564495530012772, 'eval_runtime': 12.7497, 'eval_samples_per_second': 83.531, 'eval_steps_per_second': 10.51, 'epoch': 6.996488988190233, 'step': 685}\n","Step: 783, Epoch: 7.997446536865624, Metrics: {'loss': 0.194, 'grad_norm': 586.6455078125, 'learning_rate': 1.0154639175257733e-05, 'epoch': 7.997446536865624, 'step': 783}\n","Step: 783, Epoch: 7.997446536865624, Metrics: {'eval_loss': 1.4743232727050781, 'eval_accuracy': 0.6300469483568075, 'eval_precision_macro': 0.6185604532002836, 'eval_recall_macro': 0.6300469483568075, 'eval_f1_macro': 0.6127377933543082, 'eval_precision_label_0': 0.5064935064935064, 'eval_precision_label_1': 0.7429378531073446, 'eval_precision_label_2': 0.60625, 'eval_recall_label_0': 0.3295774647887324, 'eval_recall_label_1': 0.7408450704225352, 'eval_recall_label_2': 0.819718309859155, 'eval_f1_label_0': 0.3993174061433447, 'eval_f1_label_1': 0.7418899858956276, 'eval_f1_label_2': 0.6970059880239521, 'eval_runtime': 12.7629, 'eval_samples_per_second': 83.445, 'eval_steps_per_second': 10.499, 'epoch': 7.997446536865624, 'step': 783}\n","Step: 881, Epoch: 8.998404085541015, Metrics: {'loss': 0.1695, 'grad_norm': 396.6549987792969, 'learning_rate': 5.1030927835051544e-06, 'epoch': 8.998404085541015, 'step': 881}\n","Step: 881, Epoch: 8.998404085541015, Metrics: {'eval_loss': 1.558082103729248, 'eval_accuracy': 0.6206572769953052, 'eval_precision_macro': 0.6099000236481557, 'eval_recall_macro': 0.6206572769953052, 'eval_f1_macro': 0.6105282463551231, 'eval_precision_label_0': 0.4852941176470588, 'eval_precision_label_1': 0.7277777777777777, 'eval_precision_label_2': 0.6166281755196305, 'eval_recall_label_0': 0.37183098591549296, 'eval_recall_label_1': 0.7380281690140845, 'eval_recall_label_2': 0.752112676056338, 'eval_f1_label_0': 0.42105263157894735, 'eval_f1_label_1': 0.7328671328671329, 'eval_f1_label_2': 0.6776649746192893, 'eval_runtime': 12.7986, 'eval_samples_per_second': 83.212, 'eval_steps_per_second': 10.47, 'epoch': 8.998404085541015, 'step': 881}\n","Step: 970, Epoch: 9.90743696137887, Metrics: {'loss': 0.1468, 'grad_norm': 77.30679321289062, 'learning_rate': 5.154639175257732e-07, 'epoch': 9.90743696137887, 'step': 970}\n","Step: 970, Epoch: 9.90743696137887, Metrics: {'eval_loss': 1.6202409267425537, 'eval_accuracy': 0.6244131455399061, 'eval_precision_macro': 0.6119951639667093, 'eval_recall_macro': 0.6244131455399061, 'eval_f1_macro': 0.6135778466025379, 'eval_precision_label_0': 0.4944237918215613, 'eval_precision_label_1': 0.7112299465240641, 'eval_precision_label_2': 0.6303317535545023, 'eval_recall_label_0': 0.37464788732394366, 'eval_recall_label_1': 0.7492957746478873, 'eval_recall_label_2': 0.7492957746478873, 'eval_f1_label_0': 0.42628205128205127, 'eval_f1_label_1': 0.7297668038408779, 'eval_f1_label_2': 0.6846846846846847, 'eval_runtime': 12.8073, 'eval_samples_per_second': 83.156, 'eval_steps_per_second': 10.463, 'epoch': 9.90743696137887, 'step': 970}\n","Step: 970, Epoch: 9.90743696137887, Metrics: {'train_runtime': 6417.909, 'train_samples_per_second': 9.763, 'train_steps_per_second': 0.151, 'total_flos': 2.8229621737728e+16, 'train_loss': 0.38658978275417055, 'epoch': 9.90743696137887, 'step': 970}\n"]}]},{"cell_type":"code","source":["ls ./results"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"LifHJxeJaqXi","executionInfo":{"status":"ok","timestamp":1733497903764,"user_tz":-60,"elapsed":435,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"f9771608-55ed-462b-f68a-d19d5bfb6d75"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["\u001b[0m\u001b[01;34mcheckpoint-195\u001b[0m/  \u001b[01;34mcheckpoint-391\u001b[0m/  \u001b[01;34mcheckpoint-587\u001b[0m/  \u001b[01;34mcheckpoint-783\u001b[0m/  \u001b[01;34mcheckpoint-97\u001b[0m/\n","\u001b[01;34mcheckpoint-293\u001b[0m/  \u001b[01;34mcheckpoint-489\u001b[0m/  \u001b[01;34mcheckpoint-685\u001b[0m/  \u001b[01;34mcheckpoint-881\u001b[0m/  \u001b[01;34mcheckpoint-970\u001b[0m/\n"]}]},{"cell_type":"code","source":["#Este script permite elegir que te muestre las metricas de entrenamiento del epoch que quieras que en este\n","#caso será aquel que haya dado un valor de accuracy mayor en la validación, en este caso es el epoch 7\n","from transformers import AutoModelForSequenceClassification\n","\n","# Ruta del checkpoint del epoch deseado\n","specific_epoch_checkpoint = \"./results/checkpoint-783\"  # Cambia X por el número del checkpoint\n","\n","# Cargar el modelo desde el checkpoint\n","# Add the 'local_files_only=True' argument to load from local path.\n","model = AutoModelForSequenceClassification.from_pretrained(specific_epoch_checkpoint, local_files_only=True)\n","# Move the model to the GPU\n","model.to(device) # where 'device' is your cuda device if available\n","trainer.model = model  # Actualiza el modelo del trainer\n","\n","# Calcular métricas para el conjunto de entrenamiento\n","train_predictions, train_label_ids, train_metrics = trainer.predict(train_dataset)\n","print(f\"Metrics for epoch X:\", train_metrics)"],"metadata":{"id":"m-lPeJHtFjD0","colab":{"base_uri":"https://localhost:8080/","height":55},"executionInfo":{"status":"ok","timestamp":1733498248869,"user_tz":-60,"elapsed":320763,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"3c2d5a7c-06c1-4d05-a725-f5ff1f5bd464"},"execution_count":null,"outputs":[{"output_type":"display_data","data":{"text/plain":["<IPython.core.display.HTML object>"],"text/html":[]},"metadata":{}},{"output_type":"stream","name":"stdout","text":["Metrics for epoch X: {'test_loss': 0.1606423258781433, 'test_accuracy': 0.9549952122566231, 'test_precision_macro': 0.9571567616403104, 'test_recall_macro': 0.9549952122566231, 'test_f1_macro': 0.9551918111741575, 'test_precision_label_0': 0.9893533123028391, 'test_precision_label_1': 0.9006243496357961, 'test_precision_label_2': 0.9646017699115044, 'test_recall_label_0': 0.9254887495389156, 'test_recall_label_1': 0.9735658042744657, 'test_recall_label_2': 0.9814293753517164, 'test_f1_label_0': 0.9563560129597866, 'test_f1_label_1': 0.9356756756756757, 'test_f1_label_2': 0.9729428172942818, 'test_runtime': 319.302, 'test_samples_per_second': 19.624, 'test_steps_per_second': 2.455}\n"]}]},{"cell_type":"code","source":["val_predictions, val_label_ids, val_metrics = trainer.predict(val_dataset)\n","print(\"Validation Metrics:\", val_metrics)"],"metadata":{"id":"iG3MWj5y4KvS","colab":{"base_uri":"https://localhost:8080/","height":55},"executionInfo":{"status":"ok","timestamp":1733498984940,"user_tz":-60,"elapsed":55979,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"da3b6d4e-6216-4499-d1ec-0c34f5f27601"},"execution_count":null,"outputs":[{"output_type":"display_data","data":{"text/plain":["<IPython.core.display.HTML object>"],"text/html":[]},"metadata":{}},{"output_type":"stream","name":"stdout","text":["Validation Metrics: {'test_loss': 1.4744986295700073, 'test_accuracy': 0.6300469483568075, 'test_precision_macro': 0.6188743700902875, 'test_recall_macro': 0.6300469483568075, 'test_f1_macro': 0.6126874115664851, 'test_precision_label_0': 0.508695652173913, 'test_precision_label_1': 0.7429378531073446, 'test_precision_label_2': 0.604989604989605, 'test_recall_label_0': 0.3295774647887324, 'test_recall_label_1': 0.7408450704225352, 'test_recall_label_2': 0.819718309859155, 'test_f1_label_0': 0.4, 'test_f1_label_1': 0.7418899858956276, 'test_f1_label_2': 0.6961722488038278, 'test_runtime': 55.6909, 'test_samples_per_second': 19.123, 'test_steps_per_second': 2.406}\n"]}]},{"cell_type":"markdown","source":["## GUARDADO Y CARGADO DEL MODELO"],"metadata":{"id":"sFSDPmnFin-8"}},{"cell_type":"code","source":["# save the model, if desired\n","\n","from google.colab import drive\n","drive.mount('/content/drive')\n","# Call .contiguous() on the model's parameters before saving\n","for name, param in trainer.model.named_parameters():\n","    if not param.is_contiguous():\n","        param.data = param.data.contiguous()\n","trainer.save_model('/content/drive/MyDrive/Datos Finales para Memoria/modelos/neuropep-BERT/Fine-tuned_model/')"],"metadata":{"id":"f6JgOpbw9q4Y","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1733499200569,"user_tz":-60,"elapsed":92942,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"52c87cc2-cc8d-4867-9107-e75c2f68fc41"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Mounted at /content/drive\n"]}]},{"cell_type":"code","source":["# predict AMP/non-AMP for a single example\n","\n","# IMPORTANT:\n","# one must mount their Google Drive and load their own fine-tuned model before running the below cell for individual predictions\n","from google.colab import drive\n","drive.mount('/content/drive')\n","\n","# load appropriate tokenizer and fine-tuned model\n","tokenizer = AutoTokenizer.from_pretrained('Rostlab/prot_bert_bfd', do_lower_case=False)\n","model = BertForSequenceClassification.from_pretrained(\"/content/drive/MyDrive/Datos Finales para Memoria/modelos/neuropep-BERT/Fine-tuned_model\")"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":292,"referenced_widgets":["6586ffa129d94cb487b38ccaae9bc13d","924de148a0534375a65cf98f135fe073","508e757a7fa046fa94e7095dd72efb07","1e567aaf117740fcba8dcfeb98268faf","f4d92dcd15b44072aba5cd0fe6d83ce0","c737a4d7cf1046e6aac983bd8cb3d0c1","14c65e20a0044c54901cb22321712e90","2c63d0288d934080abd2b2bbdf6b77d6","ef15df0fbadc49ddb1162b5cbec64ea1","a4e55bf562334e8fb97d0dd0d0278c68","fe193f31597b4922bcc2ef9b2505e3a6","375b4c229f40489884ea897f0afc128b","1f2bf479ea254179ae77c9e8bb0099e4","53a9d84dea7b4f8eae9c087c751820a0","5d80e2b2208941c288018accb9f1278f","8d62f7741c574e22b1a4067393389d18","7cab74916ba146ceafda5d15c3136d24","ad6cf9afa720434f8e3292f26221cc74","5866698c491d482ea50b73e1eb6ee7bf","a7619dc67e1c478d958265fb0fb94f5b","612262b39bfa46a6b59f327398ef8b7c","7c1d37d0888a4119b40114051ab2fbef","016bf80638c443a29bf69001705993ce","9bd17bddf09a42fb9f2feb655f9fff31","f6cad4d462bb44dfb00d3509c16af6a6","479c8aeea3654f75be883ad488ad63aa","22deb523667a490382f98364617e55b6","158c42d7f3754e5c99c23ebab040e04d","6203665749044e7ea6450c3b257b6b4a","9d1c27828c1c477c90462b977845e5eb","6536a8a861014d44afdf8ea4d704512b","596be27d9bfc4feba9d49fcd7599213c","54d2f3f3c5e743c58646d9e261f70746","66adbb15bae2492ba1afc9aadf005076","39fad12de7454ed2b5a794a04bf92f00","0c3377e47bed4043adc83fa001a9c89d","f5dbfee410914ef6b2f64760625d5d0e","9cd09430cf774f9cbee545ab0c6d0ebb","23fc6ce9e1f94d5391446d1965dbf9ee","3aa4b8594b9643f5b0697b7cc52240c5","9ee3cdb5605d4d578ca3911051005228","09eebdf57aca48a9a23175ad7cdaccce","267cb4a8fa0742a0b9ecb5b2d1cdc2e6","6bcdcd33b5fd496b819fc8c2242740ac"]},"id":"xqdIvYee4i_v","executionInfo":{"status":"ok","timestamp":1734253532380,"user_tz":-60,"elapsed":84905,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"fd62ce34-b3be-44cc-942f-146b9d421490"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Mounted at /content/drive\n"]},{"output_type":"stream","name":"stderr","text":["/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_auth.py:94: UserWarning: \n","The secret `HF_TOKEN` does not exist in your Colab secrets.\n","To authenticate with the Hugging Face Hub, create a token in your settings tab (https://huggingface.co/settings/tokens), set it as secret in your Google Colab and restart your session.\n","You will be able to reuse this secret in all of your notebooks.\n","Please note that authentication is recommended but still optional to access public models or datasets.\n","  warnings.warn(\n"]},{"output_type":"display_data","data":{"text/plain":["tokenizer_config.json:   0%|          | 0.00/86.0 [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"6586ffa129d94cb487b38ccaae9bc13d"}},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["config.json:   0%|          | 0.00/361 [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"375b4c229f40489884ea897f0afc128b"}},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["vocab.txt:   0%|          | 0.00/81.0 [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"016bf80638c443a29bf69001705993ce"}},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["special_tokens_map.json:   0%|          | 0.00/112 [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"66adbb15bae2492ba1afc9aadf005076"}},"metadata":{}}]},{"cell_type":"markdown","source":["## TESTS DE VERIFICACIÓN DEL ENTRENAMIENTO DEL MODELO"],"metadata":{"id":"cR1vbk3mghYP"}},{"cell_type":"markdown","source":["### TEST DE VERIFICACIÓN CON NEUROPÉPTIDOS HUMANOS"],"metadata":{"id":"bFEap8XTmkFW"}},{"cell_type":"code","source":["import re\n","import torch\n","\n","# Función para leer un archivo FASTA\n","def read_fasta(file_path):\n","    sequences = []\n","    with open(file_path, 'r') as f:\n","        identifier = \"\"\n","        sequence = \"\"\n","        for line in f:\n","            line = line.strip()\n","            if line.startswith(\">\"):  # Es un identificador de secuencia\n","                if sequence:  # Guarda la secuencia anterior si existe\n","                    sequences.append((identifier, sequence))\n","                    sequence = \"\"\n","                identifier = line[1:]  # Guarda el identificador sin el \">\"\n","            else:\n","                sequence += line  # Agrega las líneas de la secuencia\n","        if sequence:  # Guarda la última secuencia\n","            sequences.append((identifier, sequence))\n","    return sequences\n","# Función para hacer predicciones de AMP/non-AMP para múltiples secuencias y guardar probabilidades\n","def predict_amp(sequences, output_file):\n","    i=0\n","    n=0\n","    with open(output_file, 'w') as out_f:\n","        out_f.write(\"Identifier\\tSequence\\tPrediction\\tProbability\\n\")  # Encabezados\n","\n","        for identifier, input_seq in sequences:\n","            input_seq_spaced = ' '.join([input_seq[i:i+1] for i in range(0, len(input_seq), 1)])\n","            input_seq_spaced = re.sub(r'[UZOB]', 'X', input_seq_spaced)  # Reemplazar aminoácidos no estándar\n","            input_seq_tok = tokenizer(input_seq_spaced, return_tensors='pt')\n","            #print(input_seq_tok)\n","\n","            output = model(**input_seq_tok)\n","            logits = output[0]\n","           # print(f\"Logits for {identifier}: {logits}\")\n","\n","            # Extraer la probabilidad de clase AMP\n","            #y_prob = torch.sigmoid(logits)[:, 1].detach().numpy()\n","            #y_pred = y_prob > 0.84  # Predicción binaria con umbral de 0.5\n","            #prob = y_prob[0]  # Probabilidad real (sin umbral)\n","\n","            # Aplicar softmax para obtener probabilidades de cada clase\n","            y_prob = torch.softmax(logits, dim=1).detach().numpy()\n","\n","            # Obtener la clase con la probabilidad más alta\n","            y_pred_class = y_prob.argmax(axis=1)[0]\n","            prob_class1, prob_class2, prob_class3 = y_prob[0]  # Probabilidades para las tres clases\n","            # Determinar la etiqueta de la predicción según la clase\n","            if y_pred_class == 0:\n","                input_class = 'non-AMP/OTHERS'  # Cambia estos nombres según tus etiquetas\n","                n=n+1\n","            elif y_pred_class == 1:\n","                input_class = 'AMP'\n","                n=n+1\n","            else:\n","                input_class = 'Neuropeptide'\n","                i=i+1\n","            # Escribir el identificador, secuencia, predicción y probabilidades en el archivo de salida\n","            out_f.write(f\"{identifier}\\t{input_seq}\\t{input_class}\\t{prob_class1:.4f}\\t{prob_class2:.4f}\\t{prob_class3:.4f}\\n\")\n","            print(f\"Processed {identifier}: {input_class} (Probabilities: {prob_class1:.4f}, {prob_class2:.4f}, {prob_class3:.4f})\")\n","    r=i/(i+n)\n","    print(f\"El número de secuencias de Neuropeptidos es: {i}\")\n","    print(f\"El número de secuencias de AMP y non-AMP es: {n}\")\n","    print(f\"El ratio de acierto de neuropéptidos es de: {r}\")\n","\n","            # Determinar clase (AMP o non-AMP)\n","            #if y_pred:\n","            #    input_class = 'AMP'\n","            #else:\n","            #    input_class = 'non-AMP'\n","\n","            # Escribir el identificador, secuencia, predicción y probabilidad en el archivo de salida\n","           # out_f.write(f\"{identifier}\\t{input_seq}\\t{input_class}\\t{prob:.4f}\\n\")\n","           # print(f\"Processed {identifier}: {input_class} (Probability: {prob:.4f})\")\n","\n","# Ruta del archivo FASTA de entrada y del archivo de salida\n","fasta_file = '/content/drive/MyDrive/data_final/Human_neuropep_Neuropedia_test_NO_REPES.fasta'  # Nombre del archivo FASTA con secuencias\n","output_file = '/content/drive/MyDrive/Datos Finales para Memoria/resultados/RESULTADO_verificacion_neuropep_humanos.tsv'  # Archivo de salida con predicciones y probabilidades\n","\n","# Leer las secuencias del archivo FASTA\n","sequences = read_fasta(fasta_file)\n","\n","# Realizar predicciones y escribir resultados en un archivo\n","predict_amp(sequences, output_file)\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"QKpTLyVeBZrr","executionInfo":{"status":"ok","timestamp":1733821204492,"user_tz":-60,"elapsed":75134,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"beb8b2fd-4ba5-4af2-e5c9-ca964b71d809"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Processed Homo_sapiens_(Human)_CRH-related_gene_family_Urotensin-2B: AMP (Probabilities: 0.0856, 0.9015, 0.0129)\n","Processed Homo_sapiens_(Human)_Somatostatin_gene_family_Somatostatin-14: AMP (Probabilities: 0.0794, 0.9075, 0.0131)\n","Processed Homo_sapiens_(Human)_Neurexophilins_Neurexophilin-1: AMP (Probabilities: 0.0712, 0.9152, 0.0137)\n","Processed Homo_sapiens_(Human)_Galanin_family_Galanin-like_peptide: Neuropeptide (Probabilities: 0.0077, 0.0091, 0.9832)\n","Processed Homo_sapiens_(Human)_Bombesin-like_peptide_gene_family_Neuromedin-B-32: Neuropeptide (Probabilities: 0.0274, 0.0168, 0.9558)\n","Processed Homo_sapiens_(Human)_Granins_Neurosecretory_protein_VGF: Neuropeptide (Probabilities: 0.0491, 0.0187, 0.9322)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Agouti-related_protein: Neuropeptide (Probabilities: 0.2055, 0.2946, 0.4999)\n","Processed Homo_sapiens_(Human)_Calcitonin_gene_family_Proadrenomedullin_N-20_terminal_peptide: Neuropeptide (Probabilities: 0.0246, 0.0304, 0.9451)\n","Processed Homo_sapiens_(Human)_Vasopressin/oxytocin_gene_family_Copeptin_: Neuropeptide (Probabilities: 0.0070, 0.0090, 0.9840)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_PTHrP[38-94]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0042, 0.0035)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_PTHrP[1-36]: non-AMP/OTHERS (Probabilities: 0.9040, 0.0147, 0.0813)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Parathyroid_hormone-related_protein: non-AMP/OTHERS (Probabilities: 0.9902, 0.0053, 0.0046)\n","Processed Homo_sapiens_(Human)_Granins_AL-11: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed Homo_sapiens_(Human)_Insulin_family_Insulin-like_growth_factor_II_Ala-25_Del: Neuropeptide (Probabilities: 0.0077, 0.0092, 0.9831)\n","Processed Homo_sapiens_(Human)_Calcitonin_gene_family_Calcitonin: non-AMP/OTHERS (Probabilities: 0.8594, 0.1325, 0.0081)\n","Processed Homo_sapiens_(Human)_Endothelin_gene_family_Endothelin-1: Neuropeptide (Probabilities: 0.0133, 0.0117, 0.9750)\n","Processed Homo_sapiens_(Human)_Endothelin_gene_family_Endothelin-3: non-AMP/OTHERS (Probabilities: 0.8931, 0.0766, 0.0303)\n","Processed Homo_sapiens_(Human)_Vasopressin/oxytocin_gene_family_Arg-vasopressin_: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed Homo_sapiens_(Human)_Kinin_and_tensin_gene_family_Neuropeptide_K: Neuropeptide (Probabilities: 0.2237, 0.0431, 0.7332)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_PENK(143-183)_: Neuropeptide (Probabilities: 0.0124, 0.0105, 0.9770)\n","Processed Homo_sapiens_(Human)_GnRH_family_GnRH-associated_peptide_1: Neuropeptide (Probabilities: 0.0096, 0.0096, 0.9807)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Melanotropin_beta: non-AMP/OTHERS (Probabilities: 0.5292, 0.2017, 0.2692)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Gastrin-52: Neuropeptide (Probabilities: 0.0072, 0.0090, 0.9838)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Kisspeptin-14: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_CNP-53: Neuropeptide (Probabilities: 0.0184, 0.0132, 0.9684)\n","Processed Homo_sapiens_(Human)_Kinin_and_tensin_gene_family_Neurokinin-B: Neuropeptide (Probabilities: 0.1709, 0.1448, 0.6843)\n","Processed Homo_sapiens_(Human)_Calcitonin_gene_family_Katacalcin: Neuropeptide (Probabilities: 0.0090, 0.0091, 0.9819)\n","Processed Homo_sapiens_(Human)_CRH-related_gene_family_Urocortin: non-AMP/OTHERS (Probabilities: 0.9934, 0.0036, 0.0030)\n","Processed Homo_sapiens_(Human)_Somatostatin_gene_family_Cortistatin-17: AMP (Probabilities: 0.1946, 0.4705, 0.3349)\n","Processed Homo_sapiens_(Human)_Tensins_and_Kinins_Angiotensin-2: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed Homo_sapiens_(Human)_Tensins_and_Kinins_Tail_peptide_: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed Homo_sapiens_(Human)_Glucagon/secretin_gene_family_PACAP-related_peptide: Neuropeptide (Probabilities: 0.0095, 0.0099, 0.9806)\n","Processed Homo_sapiens_(Human)_Insulin_family_Relaxin-3_A_chain_: Neuropeptide (Probabilities: 0.0411, 0.0209, 0.9380)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Cholecystokinin-8: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed Homo_sapiens_(Human)_Insulin_family_C_peptide: Neuropeptide (Probabilities: 0.0083, 0.0093, 0.9824)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Synenkephalin: Neuropeptide (Probabilities: 0.0077, 0.0091, 0.9832)\n","Processed Homo_sapiens_(Human)_Granins_EA-92: Neuropeptide (Probabilities: 0.0093, 0.0098, 0.9809)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Potential_peptide: Neuropeptide (Probabilities: 0.0089, 0.0093, 0.9818)\n","Processed Homo_sapiens_(Human)_Granins_ES-43: Neuropeptide (Probabilities: 0.0935, 0.0189, 0.8876)\n","Processed Homo_sapiens_(Human)_Galanin_family_Galanin_message-associated_peptide: Neuropeptide (Probabilities: 0.0100, 0.0098, 0.9803)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Lipotropin_beta: Neuropeptide (Probabilities: 0.0613, 0.0230, 0.9157)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Metastasis-suppressor_KiSS-1: non-AMP/OTHERS (Probabilities: 0.9801, 0.0057, 0.0142)\n","Processed Homo_sapiens_(Human)_CRH-related_gene_family_Urotensin-2: Neuropeptide (Probabilities: 0.0068, 0.0085, 0.9847)\n","Processed Homo_sapiens_(Human)_Adipose_neuroepeptides_Adiponectin: non-AMP/OTHERS (Probabilities: 0.9140, 0.0721, 0.0139)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_PENK(237-258)_: Neuropeptide (Probabilities: 0.0127, 0.0109, 0.9764)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Nociceptin: non-AMP/OTHERS (Probabilities: 0.9904, 0.0069, 0.0026)\n","Processed Homo_sapiens_(Human)_F-_and_Y-amide_gene_family_Neuropeptide_FF: Neuropeptide (Probabilities: 0.0095, 0.0106, 0.9798)\n","Processed Homo_sapiens_(Human)_Granins_GAWK_peptide_: Neuropeptide (Probabilities: 0.0138, 0.0119, 0.9742)\n","Processed Homo_sapiens_(Human)_Motilin_family_Obestatin: non-AMP/OTHERS (Probabilities: 0.8776, 0.0376, 0.0848)\n","Processed Homo_sapiens_(Human)_Granins_Secretogranin-3: Neuropeptide (Probabilities: 0.1333, 0.0484, 0.8183)\n","Processed Homo_sapiens_(Human)_Neuromedins_Neuromedin-U-25: Neuropeptide (Probabilities: 0.1616, 0.0737, 0.7647)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Neuropeptide_2_: non-AMP/OTHERS (Probabilities: 0.9567, 0.0199, 0.0234)\n","Processed Homo_sapiens_(Human)_CRH-related_gene_family_Urocortin-3: Neuropeptide (Probabilities: 0.0807, 0.0333, 0.8860)\n","Processed Homo_sapiens_(Human)_Motilin_family_Motilin: Neuropeptide (Probabilities: 0.0103, 0.0099, 0.9799)\n","Processed Homo_sapiens_(Human)_Motilin_family_Promotilin: Neuropeptide (Probabilities: 0.0470, 0.0169, 0.9361)\n","Processed Homo_sapiens_(Human)_Kinin_and_tensin_gene_family_Neuropeptide_gamma,_2nd_part: Neuropeptide (Probabilities: 0.2088, 0.0393, 0.7519)\n","Processed Homo_sapiens_(Human)_Bombesin-like_peptide_gene_family_Neuromedin-C: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed Homo_sapiens_(Human)_Bombesin-like_peptide_gene_family_Neuromedin-B: Neuropeptide (Probabilities: 0.0126, 0.0129, 0.9746)\n","Processed Homo_sapiens_(Human)_Cerebellins_[des-Ser1]-cerebellin: Neuropeptide (Probabilities: 0.0093, 0.0100, 0.9807)\n","Processed Homo_sapiens_(Human)_Cerebellins_[des-Ser1]-cerebellin: Neuropeptide (Probabilities: 0.0093, 0.0100, 0.9807)\n","Processed Homo_sapiens_(Human)_Motilin_family_Ghrelins-27: Neuropeptide (Probabilities: 0.0759, 0.0227, 0.9014)\n","Processed Homo_sapiens_(Human)_Motilin_family_Ghrelins-28: Neuropeptide (Probabilities: 0.0719, 0.0229, 0.9052)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Metastin: non-AMP/OTHERS (Probabilities: 0.9419, 0.0082, 0.0499)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Cholecystokinin-5: Neuropeptide (Probabilities: 0.0058, 0.0094, 0.9848)\n","Processed Homo_sapiens_(Human)_Granins_GV-19: Neuropeptide (Probabilities: 0.0183, 0.0123, 0.9694)\n","Processed Homo_sapiens_(Human)_Galanin_family_Galanin_: Neuropeptide (Probabilities: 0.0103, 0.0099, 0.9798)\n","Processed Homo_sapiens_(Human)_Glucagon/secretin_gene_family_Intestinal_peptide_PHM-27: Neuropeptide (Probabilities: 0.0071, 0.0089, 0.9839)\n","Processed Homo_sapiens_(Human)_Glucagon/secretin_gene_family_Glucagon-like_peptide_1(7-36)_: Neuropeptide (Probabilities: 0.0104, 0.0113, 0.9784)\n","Processed Homo_sapiens_(Human)_Glucagon/secretin_gene_family_Glucagon-like_peptide_1(7-37): Neuropeptide (Probabilities: 0.0092, 0.0102, 0.9806)\n","Processed Homo_sapiens_(Human)_Glucagon/secretin_gene_family_Glucagon-like_peptide_1: Neuropeptide (Probabilities: 0.0072, 0.0090, 0.9838)\n","Processed Homo_sapiens_(Human)_F-_and_Y-amide_gene_family_Pancreatic_icosapeptide_: Neuropeptide (Probabilities: 0.2514, 0.0283, 0.7203)\n","Processed Homo_sapiens_(Human)_Kinin_and_tensin_gene_family_Neurokinin_A_: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed Homo_sapiens_(Human)_Glucagon/secretin_gene_family_Vasoactive_intestinal_peptide: Neuropeptide (Probabilities: 0.0111, 0.0110, 0.9779)\n","Processed Homo_sapiens_(Human)_Glucagon/secretin_gene_family_Pituitary_adenylate_cyclase-activating_polypeptide: Neuropeptide (Probabilities: 0.0089, 0.0098, 0.9813)\n","Processed Homo_sapiens_(Human)_Glucagon/secretin_gene_family_Secretin_: Neuropeptide (Probabilities: 0.0081, 0.0094, 0.9825)\n","Processed Homo_sapiens_(Human)_F-_and_Y-amide_gene_family_Peptide_YY(3-36): Neuropeptide (Probabilities: 0.0328, 0.0206, 0.9466)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Pro-MCH: Neuropeptide (Probabilities: 0.2738, 0.0319, 0.6943)\n","Processed Homo_sapiens_(Human)_Neuromedins_Neuromedin-S: Neuropeptide (Probabilities: 0.1772, 0.1918, 0.6310)\n","Processed Homo_sapiens_(Human)_Tensins_and_Kinins_Neuromedin_N: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Cholecystokinin-12: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Cholecystokinin-25: Neuropeptide (Probabilities: 0.0474, 0.0248, 0.9278)\n","Processed Homo_sapiens_(Human)_CRH-related_gene_family_Urocortin-2: non-AMP/OTHERS (Probabilities: 0.9861, 0.0072, 0.0067)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Cholecystokinin-33: Neuropeptide (Probabilities: 0.0109, 0.0101, 0.9789)\n","Processed Homo_sapiens_(Human)_Calcitonin_gene_family_Islet_amyloid_polypeptide_: AMP (Probabilities: 0.1028, 0.8804, 0.0168)\n","Processed Homo_sapiens_(Human)_Neurexophilins_Neurexophilin-2: AMP (Probabilities: 0.0754, 0.9115, 0.0131)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(3-29): non-AMP/OTHERS (Probabilities: 0.9497, 0.0276, 0.0227)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(3-30): Neuropeptide (Probabilities: 0.3204, 0.1669, 0.5127)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(3-32): Neuropeptide (Probabilities: 0.0232, 0.0174, 0.9595)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Cholecystokinin-18: Neuropeptide (Probabilities: 0.0072, 0.0089, 0.9840)\n","Processed Homo_sapiens_(Human)_Granins_LF-19: Neuropeptide (Probabilities: 0.0234, 0.0156, 0.9610)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Prolactin: non-AMP/OTHERS (Probabilities: 0.8934, 0.0223, 0.0842)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Kisspeptin-13: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed Homo_sapiens_(Human)_Granins_Vasostatin-1: AMP (Probabilities: 0.1317, 0.8544, 0.0138)\n","Processed Homo_sapiens_(Human)_Granins_Vasostatin-2: AMP (Probabilities: 0.4236, 0.5203, 0.0560)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_PENK(114-133)_: Neuropeptide (Probabilities: 0.0071, 0.0089, 0.9840)\n","Processed Homo_sapiens_(Human)_Adipose_neuroepeptides_Nicotinamide_phosphoribosyltransferase: non-AMP/OTHERS (Probabilities: 0.9715, 0.0205, 0.0081)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Neuropeptide_1_: Neuropeptide (Probabilities: 0.0089, 0.0097, 0.9814)\n","Processed Homo_sapiens_(Human)_Granins_Secretogranin-1: Neuropeptide (Probabilities: 0.1671, 0.0223, 0.8106)\n","Processed Homo_sapiens_(Human)_Cerebellins_Cerebellin-2: non-AMP/OTHERS (Probabilities: 0.7132, 0.2283, 0.0585)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(4-27): non-AMP/OTHERS (Probabilities: 0.9014, 0.0095, 0.0891)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(4-29): non-AMP/OTHERS (Probabilities: 0.9873, 0.0049, 0.0078)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(4-30): non-AMP/OTHERS (Probabilities: 0.8292, 0.0180, 0.1527)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(4-31): non-AMP/OTHERS (Probabilities: 0.5730, 0.0341, 0.3928)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(4-32): non-AMP/OTHERS (Probabilities: 0.7397, 0.0600, 0.2003)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Apelin-28_: Neuropeptide (Probabilities: 0.1646, 0.0420, 0.7934)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_Cardiodilatin-related_peptide: Neuropeptide (Probabilities: 0.0082, 0.0094, 0.9824)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(2-31): Neuropeptide (Probabilities: 0.0757, 0.0444, 0.8799)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Corticotropin-like_intermediary_peptide: non-AMP/OTHERS (Probabilities: 0.9056, 0.0110, 0.0834)\n","Processed Homo_sapiens_(Human)_Insulin_family_Relaxin_A_chain_: AMP (Probabilities: 0.0698, 0.9159, 0.0143)\n","Processed Homo_sapiens_(Human)_Adipose_neuroepeptides_Resistin-like_beta: Neuropeptide (Probabilities: 0.0090, 0.0099, 0.9811)\n","Processed Homo_sapiens_(Human)_Neurexophilins_Neurexophilin-3: AMP (Probabilities: 0.1503, 0.7895, 0.0601)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_CART(1-39): non-AMP/OTHERS (Probabilities: 0.9933, 0.0037, 0.0030)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Cocaine-_and_amphetamine-regulated_transcript_protein: AMP (Probabilities: 0.1373, 0.8487, 0.0140)\n","Processed Homo_sapiens_(Human)_Cerebellins_Cerebellin-3: non-AMP/OTHERS (Probabilities: 0.6933, 0.2840, 0.0227)\n","Processed Homo_sapiens_(Human)_Tensins_and_Kinins_Kininogen-1_heavy_chain: AMP (Probabilities: 0.0855, 0.9012, 0.0133)\n","Processed Homo_sapiens_(Human)_Tensins_and_Kinins_Kininogen-1_: AMP (Probabilities: 0.0792, 0.9062, 0.0146)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Gastrin_: Neuropeptide (Probabilities: 0.0838, 0.0235, 0.8927)\n","Processed Homo_sapiens_(Human)_GnRH_family_Gonadoliberin-2: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed Homo_sapiens_(Human)_GnRH_family_Progonadoliberin-1: Neuropeptide (Probabilities: 0.0078, 0.0090, 0.9832)\n","Processed Homo_sapiens_(Human)_Neurexophilins_Neurexophilin-4: AMP (Probabilities: 0.0659, 0.9189, 0.0152)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Big_gastrin: Neuropeptide (Probabilities: 0.0083, 0.0093, 0.9824)\n","Processed Homo_sapiens_(Human)_Tensins_and_Kinins_Neurotesin_: Neuropeptide (Probabilities: 0.0692, 0.0474, 0.8834)\n","Processed Homo_sapiens_(Human)_Cerebellins_Cerebellin-4: AMP (Probabilities: 0.4060, 0.5835, 0.0105)\n","Processed Homo_sapiens_(Human)_Cerebellins_Cerebellin-1_: AMP (Probabilities: 0.3829, 0.6062, 0.0109)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Prothyroliberin: Neuropeptide (Probabilities: 0.0195, 0.0139, 0.9666)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Cholecystokinin: Neuropeptide (Probabilities: 0.0085, 0.0095, 0.9820)\n","Processed Homo_sapiens_(Human)_Granins_Secretoneurin-2: Neuropeptide (Probabilities: 0.0113, 0.0111, 0.9776)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Apelin-13_: Neuropeptide (Probabilities: 0.0075, 0.0092, 0.9832)\n","Processed Homo_sapiens_(Human)_Insulin_family_Relaxin-3_B_chain_: AMP (Probabilities: 0.2929, 0.6544, 0.0527)\n","Processed Homo_sapiens_(Human)_Insulin_family_E_peptide: Neuropeptide (Probabilities: 0.0240, 0.0180, 0.9580)\n","Processed Homo_sapiens_(Human)_Kinin_and_tensin_gene_family_Substance_P: AMP (Probabilities: 0.1583, 0.7066, 0.1351)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Orexin-B: Neuropeptide (Probabilities: 0.0361, 0.0202, 0.9437)\n","Processed Homo_sapiens_(Human)_Glucagon/secretin_gene_family_Glicentin_: Neuropeptide (Probabilities: 0.1778, 0.0234, 0.7988)\n","Processed Homo_sapiens_(Human)_Insulin_family_E_peptide: non-AMP/OTHERS (Probabilities: 0.9909, 0.0058, 0.0033)\n","Processed Homo_sapiens_(Human)_Insulin_family_E_peptide: AMP (Probabilities: 0.0791, 0.9081, 0.0129)\n","Processed Homo_sapiens_(Human)_Insulin_family_Connecting_peptide: Neuropeptide (Probabilities: 0.0086, 0.0093, 0.9821)\n","Processed Homo_sapiens_(Human)_Tensins_and_Kinins_Large_neuromedin_N: non-AMP/OTHERS (Probabilities: 0.9057, 0.0097, 0.0846)\n","Processed Homo_sapiens_(Human)_Granins_Pancreastatin: Neuropeptide (Probabilities: 0.0105, 0.0100, 0.9795)\n","Processed Homo_sapiens_(Human)_Granins_SS-18: Neuropeptide (Probabilities: 0.1402, 0.0243, 0.8355)\n","Processed Homo_sapiens_(Human)_Insulin_family_Connecting_peptide_: non-AMP/OTHERS (Probabilities: 0.6676, 0.0510, 0.2814)\n","Processed Homo_sapiens_(Human)_Motilin_family_Motilin-associated_peptide: Neuropeptide (Probabilities: 0.0105, 0.0101, 0.9794)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(1-28): non-AMP/OTHERS (Probabilities: 0.9871, 0.0056, 0.0073)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(1-29): non-AMP/OTHERS (Probabilities: 0.9840, 0.0076, 0.0084)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(1-30): non-AMP/OTHERS (Probabilities: 0.6924, 0.0352, 0.2724)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_Brain_natriuretic_peptide_32_: Neuropeptide (Probabilities: 0.1307, 0.0542, 0.8151)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Diazepam-binding_inhibitor: Neuropeptide (Probabilities: 0.0107, 0.0114, 0.9779)\n","Processed Homo_sapiens_(Human)_F-_and_Y-amide_gene_family_Prolactin-releasing_peptide_PrRP31: Neuropeptide (Probabilities: 0.0811, 0.0323, 0.8866)\n","Processed Homo_sapiens_(Human)_F-_and_Y-amide_gene_family_C-flanking_peptide_of_NPY: Neuropeptide (Probabilities: 0.0107, 0.0100, 0.9793)\n","Processed Homo_sapiens_(Human)_Tensins_and_Kinins_Kininogen-1_light_chain: non-AMP/OTHERS (Probabilities: 0.9426, 0.0153, 0.0421)\n","Processed Homo_sapiens_(Human)_Granins_C-terminal_peptide_: Neuropeptide (Probabilities: 0.0110, 0.0096, 0.9794)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Gastrin-71: Neuropeptide (Probabilities: 0.0077, 0.0090, 0.9833)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Melanotropin_alpha: AMP (Probabilities: 0.2590, 0.5718, 0.1692)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Corticotropin: Neuropeptide (Probabilities: 0.1656, 0.0389, 0.7955)\n","Processed Homo_sapiens_(Human)_F-_and_Y-amide_gene_family_Prolactin-releasing_peptide_PrRP20: Neuropeptide (Probabilities: 0.0093, 0.0101, 0.9806)\n","Processed Homo_sapiens_(Human)_Calcitonin_gene_family_Adrenomedullin-2_: Neuropeptide (Probabilities: 0.2387, 0.0424, 0.7189)\n","Processed Homo_sapiens_(Human)_Calcitonin_gene_family_Intermedin-short_: Neuropeptide (Probabilities: 0.0427, 0.0187, 0.9386)\n","Processed Homo_sapiens_(Human)_Adipose_neuroepeptides_Nucleobindin-2: non-AMP/OTHERS (Probabilities: 0.9778, 0.0150, 0.0072)\n","Processed Homo_sapiens_(Human)_Adipose_neuroepeptides_Leptin_: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0034)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_CART(42-89): AMP (Probabilities: 0.0717, 0.9141, 0.0142)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(5-29): non-AMP/OTHERS (Probabilities: 0.9918, 0.0045, 0.0036)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(5-31): Neuropeptide (Probabilities: 0.1871, 0.0623, 0.7505)\n","Processed Homo_sapiens_(Human)_Natriuretic_factor_gene_family_BNP(5-32): Neuropeptide (Probabilities: 0.2079, 0.0648, 0.7273)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Cholecystokinin-58_desnonopeptide: Neuropeptide (Probabilities: 0.0170, 0.0132, 0.9698)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Cholecystokinin-58: Neuropeptide (Probabilities: 0.0110, 0.0110, 0.9780)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_NPP: Neuropeptide (Probabilities: 0.0131, 0.0112, 0.9758)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Gastrin-14: non-AMP/OTHERS (Probabilities: 0.6345, 0.0155, 0.3500)\n","Processed Homo_sapiens_(Human)_Granins_WA-8: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed Homo_sapiens_(Human)_Granins_WE-14_: non-AMP/OTHERS (Probabilities: 0.9659, 0.0105, 0.0236)\n","Processed Homo_sapiens_(Human)_Neuropeptide_B/W_family_Neuropeptide_W-23: AMP (Probabilities: 0.2473, 0.5636, 0.1891)\n","Processed Homo_sapiens_(Human)_Neuropeptide_B/W_family_Neuropeptide_W-30: Neuropeptide (Probabilities: 0.0293, 0.0229, 0.9478)\n","Processed Homo_sapiens_(Human)_Neuropeptide_B/W_family_Neuropeptide_B-29: Neuropeptide (Probabilities: 0.2031, 0.3220, 0.4749)\n","Processed Homo_sapiens_(Human)_Neuropeptide_B/W_family_Neuropeptide_B-23: Neuropeptide (Probabilities: 0.0909, 0.0591, 0.8500)\n","Processed Homo_sapiens_(Human)_Glucagon/secretin_gene_family_Gastric_inhibitory_polypeptide_: Neuropeptide (Probabilities: 0.0412, 0.0237, 0.9351)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Leu-enkephalin: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Leu-enkephalin: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Leu-enkephalin_: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Leu-enkephalin_: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Beta-neoendorphin: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Alpha-neoendorphin: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Dynorphin_A(1-8)_: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Dynorphin_A(1-13)_: AMP (Probabilities: 0.0750, 0.9121, 0.0128)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Dynorphin_A(1-17): AMP (Probabilities: 0.2429, 0.5829, 0.1742)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Rimorphin: AMP (Probabilities: 0.0679, 0.9172, 0.0149)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Leumorphin: Neuropeptide (Probabilities: 0.0104, 0.0106, 0.9790)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Met-enkephalin: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Met-enkephalin: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Met-enkephalin: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Met-enkephalin: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Met-enkephalin: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Met-enkephalin-Arg-Phe: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Met-enkephalin-Arg-Gly-Leu: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9850)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Beta-endorphin: Neuropeptide (Probabilities: 0.2094, 0.0716, 0.7189)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Gastrin-6: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Cholecystokinin-39: Neuropeptide (Probabilities: 0.0168, 0.0137, 0.9695)\n","Processed Homo_sapiens_(Human)_CCK/gastrin_gene_family_Cholecystokinin-7: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed Homo_sapiens_(Human)_No-family_neuropeptides_Kisspeptin-10: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed Homo_sapiens_(Human)_F-_and_Y-amide_gene_family_Peptide_YY_: Neuropeptide (Probabilities: 0.0383, 0.0237, 0.9380)\n","Processed Homo_sapiens_(Human)_Insulin_family_Insulin-like_growth_factor_II_Ala-25_Del: Neuropeptide (Probabilities: 0.0081, 0.0093, 0.9826)\n","Processed Homo_sapiens_(Human)_Granins_N-terminal_peptide: Neuropeptide (Probabilities: 0.0423, 0.0253, 0.9324)\n","Processed Homo_sapiens_(Human)_Granins_Neuroendocrine_protein_7B2: Neuropeptide (Probabilities: 0.0368, 0.0204, 0.9428)\n","Processed Homo_sapiens_(Human)_Opioid_gene_family_Melanotropin_gamma: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","El número de secuencias de Neuropeptidos es: 138\n","El número de secuencias de AMP y non-AMP es: 63\n","El ratio de acierto de neuropéptidos es de: 0.6865671641791045\n"]}]},{"cell_type":"markdown","source":["### TEST DE VERIFICACIÓN CON AMPs DIFERENTES A LOS DEL ENTRENAMIENTO"],"metadata":{"id":"inof80U3mwvu"}},{"cell_type":"code","source":["import re\n","import torch\n","\n","# Función para leer un archivo FASTA\n","def read_fasta(file_path):\n","    sequences = []\n","    with open(file_path, 'r') as f:\n","        identifier = \"\"\n","        sequence = \"\"\n","        for line in f:\n","            line = line.strip()\n","            if line.startswith(\">\"):  # Es un identificador de secuencia\n","                if sequence:  # Guarda la secuencia anterior si existe\n","                    sequences.append((identifier, sequence))\n","                    sequence = \"\"\n","                identifier = line[1:]  # Guarda el identificador sin el \">\"\n","            else:\n","                sequence += line  # Agrega las líneas de la secuencia\n","        if sequence:  # Guarda la última secuencia\n","            sequences.append((identifier, sequence))\n","    return sequences\n","# Función para hacer predicciones de AMP/non-AMP para múltiples secuencias y guardar probabilidades\n","def predict_amp(sequences, output_file):\n","    i=0\n","    n=0\n","    with open(output_file, 'w') as out_f:\n","        out_f.write(\"Identifier\\tSequence\\tPrediction\\tProbability\\n\")  # Encabezados\n","\n","        for identifier, input_seq in sequences:\n","            input_seq_spaced = ' '.join([input_seq[i:i+1] for i in range(0, len(input_seq), 1)])\n","            input_seq_spaced = re.sub(r'[UZOB]', 'X', input_seq_spaced)  # Reemplazar aminoácidos no estándar\n","            input_seq_tok = tokenizer(input_seq_spaced, return_tensors='pt')\n","            #print(input_seq_tok)\n","\n","            output = model(**input_seq_tok)\n","            logits = output[0]\n","           # print(f\"Logits for {identifier}: {logits}\")\n","\n","            # Extraer la probabilidad de clase AMP\n","            #y_prob = torch.sigmoid(logits)[:, 1].detach().numpy()\n","            #y_pred = y_prob > 0.84  # Predicción binaria con umbral de 0.5\n","            #prob = y_prob[0]  # Probabilidad real (sin umbral)\n","\n","            # Aplicar softmax para obtener probabilidades de cada clase\n","            y_prob = torch.softmax(logits, dim=1).detach().numpy()\n","\n","            # Obtener la clase con la probabilidad más alta\n","            y_pred_class = y_prob.argmax(axis=1)[0]\n","            prob_class1, prob_class2, prob_class3 = y_prob[0]  # Probabilidades para las tres clases\n","            # Determinar la etiqueta de la predicción según la clase\n","            if y_pred_class == 0:\n","                input_class = 'non-AMP/OTHERS'  # Cambia estos nombres según tus etiquetas\n","                n=n+1\n","            elif y_pred_class == 1:\n","                input_class = 'AMP'\n","                i=i+1\n","            else:\n","                input_class = 'Neuropeptide'\n","                n=n+1\n","            # Escribir el identificador, secuencia, predicción y probabilidades en el archivo de salida\n","            out_f.write(f\"{identifier}\\t{input_seq}\\t{input_class}\\t{prob_class1:.4f}\\t{prob_class2:.4f}\\t{prob_class3:.4f}\\n\")\n","            print(f\"Processed {identifier}: {input_class} (Probabilities: {prob_class1:.4f}, {prob_class2:.4f}, {prob_class3:.4f})\")\n","    print(f\"El número de secuencias de AMPs es: {i}\")\n","    print(f\"El número de secuencias de neuropéptidos y non-AMP es: {n}\")\n","\n","            # Determinar clase (AMP o non-AMP)\n","            #if y_pred:\n","            #    input_class = 'AMP'\n","            #else:\n","            #    input_class = 'non-AMP'\n","\n","            # Escribir el identificador, secuencia, predicción y probabilidad en el archivo de salida\n","           # out_f.write(f\"{identifier}\\t{input_seq}\\t{input_class}\\t{prob:.4f}\\n\")\n","           # print(f\"Processed {identifier}: {input_class} (Probability: {prob:.4f})\")\n","\n","# Ruta del archivo FASTA de entrada y del archivo de salida\n","fasta_file = '/content/drive/MyDrive/data_final/general_amps_DRAMP_cdhit_80_NO_REPES_recortado1892seq.fasta'  # Nombre del archivo FASTA con secuencias\n","output_file = '/content/drive/MyDrive/Datos Finales para Memoria/resultados_verificacion_general_AMPs.tsv'  # Archivo de salida con predicciones y probabilidades\n","\n","# Leer las secuencias del archivo FASTA\n","sequences = read_fasta(fasta_file)\n","\n","# Realizar predicciones y escribir resultados en un archivo\n","predict_amp(sequences, output_file)\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1733916109118,"user_tz":-60,"elapsed":560533,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"6f53b6f5-7331-4cf8-be44-dbdb8f046809","collapsed":true,"id":"nOfQOr2YWSew"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Processed DRAMP00032: AMP (Probabilities: 0.0667, 0.9188, 0.0145)\n","Processed DRAMP00089: AMP (Probabilities: 0.0658, 0.9180, 0.0162)\n","Processed DRAMP00106: AMP (Probabilities: 0.0718, 0.9145, 0.0138)\n","Processed DRAMP00127: AMP (Probabilities: 0.0966, 0.8812, 0.0222)\n","Processed DRAMP00129: AMP (Probabilities: 0.0670, 0.9182, 0.0148)\n","Processed DRAMP00189: AMP (Probabilities: 0.0700, 0.9155, 0.0145)\n","Processed DRAMP00190: non-AMP/OTHERS (Probabilities: 0.5189, 0.4695, 0.0116)\n","Processed DRAMP00191: non-AMP/OTHERS (Probabilities: 0.8212, 0.0918, 0.0871)\n","Processed DRAMP00204: AMP (Probabilities: 0.1470, 0.7889, 0.0641)\n","Processed DRAMP00254: AMP (Probabilities: 0.0963, 0.8922, 0.0115)\n","Processed DRAMP00384: non-AMP/OTHERS (Probabilities: 0.6807, 0.0819, 0.2373)\n","Processed DRAMP00425: non-AMP/OTHERS (Probabilities: 0.9761, 0.0151, 0.0088)\n","Processed DRAMP00437: AMP (Probabilities: 0.0679, 0.9165, 0.0156)\n","Processed DRAMP00454: AMP (Probabilities: 0.0747, 0.9116, 0.0138)\n","Processed DRAMP00766: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed DRAMP01004: non-AMP/OTHERS (Probabilities: 0.9904, 0.0039, 0.0057)\n","Processed DRAMP01064: Neuropeptide (Probabilities: 0.0063, 0.0088, 0.9849)\n","Processed DRAMP01066: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed DRAMP01088: AMP (Probabilities: 0.0748, 0.9111, 0.0141)\n","Processed DRAMP01188: AMP (Probabilities: 0.1118, 0.8763, 0.0119)\n","Processed DRAMP00931: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed DRAMP00929: Neuropeptide (Probabilities: 0.0059, 0.0087, 0.9854)\n","Processed DRAMP03542: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed DRAMP02344: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed DRAMP01457: AMP (Probabilities: 0.0669, 0.9178, 0.0153)\n","Processed DRAMP01546: AMP (Probabilities: 0.0743, 0.9112, 0.0145)\n","Processed DRAMP18391: AMP (Probabilities: 0.1466, 0.8390, 0.0144)\n","Processed DRAMP01590: AMP (Probabilities: 0.1703, 0.8151, 0.0146)\n","Processed DRAMP01779: Neuropeptide (Probabilities: 0.0061, 0.0093, 0.9846)\n","Processed DRAMP01885: AMP (Probabilities: 0.0680, 0.9165, 0.0155)\n","Processed DRAMP01886: AMP (Probabilities: 0.0742, 0.9115, 0.0143)\n","Processed DRAMP01887: AMP (Probabilities: 0.0728, 0.9128, 0.0144)\n","Processed DRAMP01890: AMP (Probabilities: 0.0729, 0.9126, 0.0145)\n","Processed DRAMP01892: AMP (Probabilities: 0.0696, 0.9156, 0.0148)\n","Processed DRAMP18383: Neuropeptide (Probabilities: 0.0058, 0.0094, 0.9848)\n","Processed DRAMP01934: AMP (Probabilities: 0.0713, 0.9138, 0.0149)\n","Processed DRAMP01970: AMP (Probabilities: 0.0777, 0.9085, 0.0138)\n","Processed DRAMP01971: AMP (Probabilities: 0.1667, 0.8193, 0.0140)\n","Processed DRAMP01974: AMP (Probabilities: 0.0705, 0.9142, 0.0153)\n","Processed DRAMP02006: AMP (Probabilities: 0.0738, 0.9119, 0.0143)\n","Processed DRAMP18378: non-AMP/OTHERS (Probabilities: 0.9908, 0.0046, 0.0046)\n","Processed DRAMP04238: Neuropeptide (Probabilities: 0.0071, 0.0105, 0.9824)\n","Processed DRAMP02129: AMP (Probabilities: 0.0718, 0.9136, 0.0146)\n","Processed DRAMP03002: AMP (Probabilities: 0.0687, 0.9171, 0.0142)\n","Processed DRAMP03998: Neuropeptide (Probabilities: 0.0059, 0.0097, 0.9845)\n","Processed DRAMP02318: AMP (Probabilities: 0.0678, 0.9168, 0.0154)\n","Processed DRAMP02321: AMP (Probabilities: 0.0672, 0.9173, 0.0155)\n","Processed DRAMP02376: AMP (Probabilities: 0.0697, 0.9151, 0.0151)\n","Processed DRAMP02377: AMP (Probabilities: 0.0731, 0.9124, 0.0145)\n","Processed DRAMP02378: AMP (Probabilities: 0.0674, 0.9173, 0.0153)\n","Processed DRAMP02380: AMP (Probabilities: 0.0647, 0.9194, 0.0160)\n","Processed DRAMP02397: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9853)\n","Processed DRAMP02409: AMP (Probabilities: 0.0678, 0.9173, 0.0148)\n","Processed DRAMP02421: AMP (Probabilities: 0.0687, 0.9163, 0.0150)\n","Processed DRAMP02422: AMP (Probabilities: 0.0695, 0.9159, 0.0146)\n","Processed DRAMP02430: non-AMP/OTHERS (Probabilities: 0.9716, 0.0229, 0.0055)\n","Processed DRAMP02434: non-AMP/OTHERS (Probabilities: 0.7688, 0.0241, 0.2071)\n","Processed DRAMP02439: AMP (Probabilities: 0.0793, 0.9073, 0.0134)\n","Processed DRAMP02445: AMP (Probabilities: 0.1695, 0.8153, 0.0152)\n","Processed DRAMP02449: Neuropeptide (Probabilities: 0.0065, 0.0093, 0.9842)\n","Processed DRAMP02456: AMP (Probabilities: 0.4656, 0.5205, 0.0139)\n","Processed DRAMP02478: non-AMP/OTHERS (Probabilities: 0.9896, 0.0056, 0.0048)\n","Processed DRAMP02522: non-AMP/OTHERS (Probabilities: 0.9843, 0.0043, 0.0114)\n","Processed DRAMP18371: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9849)\n","Processed DRAMP02586: AMP (Probabilities: 0.0664, 0.9181, 0.0155)\n","Processed DRAMP02603: AMP (Probabilities: 0.0682, 0.9171, 0.0147)\n","Processed DRAMP02650: AMP (Probabilities: 0.2465, 0.7179, 0.0356)\n","Processed DRAMP02776: AMP (Probabilities: 0.0668, 0.9184, 0.0148)\n","Processed DRAMP02777: AMP (Probabilities: 0.2685, 0.7069, 0.0246)\n","Processed DRAMP02802: AMP (Probabilities: 0.0651, 0.9195, 0.0154)\n","Processed DRAMP02813: AMP (Probabilities: 0.1730, 0.8121, 0.0149)\n","Processed DRAMP02844: AMP (Probabilities: 0.1531, 0.8330, 0.0139)\n","Processed DRAMP02845: Neuropeptide (Probabilities: 0.0328, 0.0296, 0.9375)\n","Processed DRAMP02877: AMP (Probabilities: 0.0725, 0.9128, 0.0147)\n","Processed DRAMP18368: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed DRAMP02934: AMP (Probabilities: 0.0722, 0.9144, 0.0134)\n","Processed DRAMP02951: AMP (Probabilities: 0.1436, 0.8439, 0.0125)\n","Processed DRAMP02952: AMP (Probabilities: 0.0731, 0.9131, 0.0138)\n","Processed DRAMP02966: non-AMP/OTHERS (Probabilities: 0.4901, 0.1096, 0.4003)\n","Processed DRAMP02441: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed DRAMP03000: Neuropeptide (Probabilities: 0.0358, 0.0529, 0.9113)\n","Processed DRAMP03040: AMP (Probabilities: 0.0701, 0.9145, 0.0154)\n","Processed DRAMP03041: AMP (Probabilities: 0.0701, 0.9145, 0.0154)\n","Processed DRAMP03820: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9851)\n","Processed DRAMP03069: non-AMP/OTHERS (Probabilities: 0.9924, 0.0049, 0.0027)\n","Processed DRAMP03153: non-AMP/OTHERS (Probabilities: 0.9900, 0.0037, 0.0064)\n","Processed DRAMP03166: AMP (Probabilities: 0.0701, 0.9160, 0.0139)\n","Processed DRAMP03187: AMP (Probabilities: 0.0693, 0.9167, 0.0140)\n","Processed DRAMP03321: AMP (Probabilities: 0.0894, 0.8945, 0.0160)\n","Processed DRAMP03465: non-AMP/OTHERS (Probabilities: 0.9820, 0.0056, 0.0124)\n","Processed DRAMP03467: AMP (Probabilities: 0.1621, 0.7778, 0.0601)\n","Processed DRAMP03474: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP03539: Neuropeptide (Probabilities: 0.0109, 0.0103, 0.9787)\n","Processed DRAMP03540: AMP (Probabilities: 0.0708, 0.9153, 0.0139)\n","Processed DRAMP03562: AMP (Probabilities: 0.0717, 0.9132, 0.0151)\n","Processed DRAMP03647: AMP (Probabilities: 0.0699, 0.9163, 0.0138)\n","Processed DRAMP03676: non-AMP/OTHERS (Probabilities: 0.4791, 0.3190, 0.2019)\n","Processed DRAMP03682: AMP (Probabilities: 0.0675, 0.9163, 0.0162)\n","Processed DRAMP03683: AMP (Probabilities: 0.1280, 0.8507, 0.0213)\n","Processed DRAMP03684: AMP (Probabilities: 0.0876, 0.8981, 0.0143)\n","Processed DRAMP18366: Neuropeptide (Probabilities: 0.0441, 0.0357, 0.9202)\n","Processed DRAMP03826: AMP (Probabilities: 0.0754, 0.9106, 0.0139)\n","Processed DRAMP03852: AMP (Probabilities: 0.0913, 0.8956, 0.0131)\n","Processed DRAMP03858: AMP (Probabilities: 0.1456, 0.8409, 0.0135)\n","Processed DRAMP03865: AMP (Probabilities: 0.0814, 0.9052, 0.0134)\n","Processed DRAMP03867: AMP (Probabilities: 0.0791, 0.9085, 0.0125)\n","Processed DRAMP03868: AMP (Probabilities: 0.0875, 0.8997, 0.0128)\n","Processed DRAMP03877: AMP (Probabilities: 0.1499, 0.8366, 0.0135)\n","Processed DRAMP03879: AMP (Probabilities: 0.1397, 0.8473, 0.0130)\n","Processed DRAMP03880: AMP (Probabilities: 0.1330, 0.8543, 0.0127)\n","Processed DRAMP03885: non-AMP/OTHERS (Probabilities: 0.9727, 0.0198, 0.0075)\n","Processed DRAMP03899: AMP (Probabilities: 0.3939, 0.4302, 0.1759)\n","Processed DRAMP03909: non-AMP/OTHERS (Probabilities: 0.5388, 0.4329, 0.0283)\n","Processed DRAMP03920: AMP (Probabilities: 0.0674, 0.9167, 0.0160)\n","Processed DRAMP03930: AMP (Probabilities: 0.0685, 0.9173, 0.0142)\n","Processed DRAMP03954: AMP (Probabilities: 0.1635, 0.8237, 0.0128)\n","Processed DRAMP03984: AMP (Probabilities: 0.1310, 0.8564, 0.0126)\n","Processed DRAMP03992: AMP (Probabilities: 0.1715, 0.8129, 0.0156)\n","Processed DRAMP03999: AMP (Probabilities: 0.0707, 0.9138, 0.0155)\n","Processed DRAMP04002: AMP (Probabilities: 0.0709, 0.9136, 0.0155)\n","Processed DRAMP04003: AMP (Probabilities: 0.0686, 0.9161, 0.0153)\n","Processed DRAMP04005: AMP (Probabilities: 0.1723, 0.7364, 0.0914)\n","Processed DRAMP04019: AMP (Probabilities: 0.1158, 0.8721, 0.0121)\n","Processed DRAMP04027: AMP (Probabilities: 0.1620, 0.8238, 0.0142)\n","Processed DRAMP04030: AMP (Probabilities: 0.1179, 0.8696, 0.0126)\n","Processed DRAMP04032: AMP (Probabilities: 0.2680, 0.7146, 0.0174)\n","Processed DRAMP04034: AMP (Probabilities: 0.1150, 0.8727, 0.0123)\n","Processed DRAMP04051: AMP (Probabilities: 0.0822, 0.9047, 0.0131)\n","Processed DRAMP04052: AMP (Probabilities: 0.0876, 0.9001, 0.0123)\n","Processed DRAMP04055: AMP (Probabilities: 0.0998, 0.8879, 0.0123)\n","Processed DRAMP18361: Neuropeptide (Probabilities: 0.2729, 0.1510, 0.5761)\n","Processed DRAMP04064: AMP (Probabilities: 0.0705, 0.9159, 0.0136)\n","Processed DRAMP04065: AMP (Probabilities: 0.0690, 0.9158, 0.0152)\n","Processed DRAMP04068: AMP (Probabilities: 0.0720, 0.9142, 0.0138)\n","Processed DRAMP04069: AMP (Probabilities: 0.0682, 0.9169, 0.0149)\n","Processed DRAMP04070: AMP (Probabilities: 0.0690, 0.9164, 0.0146)\n","Processed DRAMP04096: non-AMP/OTHERS (Probabilities: 0.9930, 0.0041, 0.0030)\n","Processed DRAMP04097: non-AMP/OTHERS (Probabilities: 0.9925, 0.0038, 0.0037)\n","Processed DRAMP04098: AMP (Probabilities: 0.1289, 0.8170, 0.0541)\n","Processed DRAMP04099: non-AMP/OTHERS (Probabilities: 0.9705, 0.0234, 0.0061)\n","Processed DRAMP04100: non-AMP/OTHERS (Probabilities: 0.9913, 0.0055, 0.0032)\n","Processed DRAMP04101: non-AMP/OTHERS (Probabilities: 0.9908, 0.0056, 0.0036)\n","Processed DRAMP04102: AMP (Probabilities: 0.0728, 0.9125, 0.0147)\n","Processed DRAMP04108: AMP (Probabilities: 0.0732, 0.9123, 0.0145)\n","Processed DRAMP04124: AMP (Probabilities: 0.1844, 0.6843, 0.1313)\n","Processed DRAMP04125: AMP (Probabilities: 0.0739, 0.9137, 0.0125)\n","Processed DRAMP04128: AMP (Probabilities: 0.1251, 0.8626, 0.0123)\n","Processed DRAMP04137: AMP (Probabilities: 0.1511, 0.8358, 0.0131)\n","Processed DRAMP04141: Neuropeptide (Probabilities: 0.0058, 0.0088, 0.9854)\n","Processed DRAMP04142: Neuropeptide (Probabilities: 0.0867, 0.1158, 0.7975)\n","Processed DRAMP04143: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed DRAMP04144: AMP (Probabilities: 0.2570, 0.3902, 0.3528)\n","Processed DRAMP04146: AMP (Probabilities: 0.0914, 0.8955, 0.0131)\n","Processed DRAMP04181: AMP (Probabilities: 0.1410, 0.8456, 0.0135)\n","Processed DRAMP04183: AMP (Probabilities: 0.1466, 0.8396, 0.0138)\n","Processed DRAMP04184: AMP (Probabilities: 0.0671, 0.9176, 0.0153)\n","Processed DRAMP04234: AMP (Probabilities: 0.0716, 0.9142, 0.0142)\n","Processed DRAMP04235: AMP (Probabilities: 0.0703, 0.9143, 0.0154)\n","Processed DRAMP04237: AMP (Probabilities: 0.0750, 0.9109, 0.0141)\n","Processed DRAMP04241: AMP (Probabilities: 0.0734, 0.9131, 0.0134)\n","Processed DRAMP04244: AMP (Probabilities: 0.1055, 0.8821, 0.0124)\n","Processed DRAMP04265: AMP (Probabilities: 0.0714, 0.9140, 0.0146)\n","Processed DRAMP04365: AMP (Probabilities: 0.1644, 0.8218, 0.0138)\n","Processed DRAMP04372: AMP (Probabilities: 0.0838, 0.9026, 0.0136)\n","Processed DRAMP04389: AMP (Probabilities: 0.0691, 0.9151, 0.0158)\n","Processed DRAMP04390: AMP (Probabilities: 0.3442, 0.6424, 0.0134)\n","Processed DRAMP04671: non-AMP/OTHERS (Probabilities: 0.9608, 0.0091, 0.0302)\n","Processed DRAMP04676: AMP (Probabilities: 0.1386, 0.8493, 0.0121)\n","Processed DRAMP00004: AMP (Probabilities: 0.0673, 0.9181, 0.0146)\n","Processed DRAMP00006: AMP (Probabilities: 0.0674, 0.9178, 0.0148)\n","Processed DRAMP18350: AMP (Probabilities: 0.0734, 0.9135, 0.0131)\n","Processed DRAMP18351: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9852)\n","Processed DRAMP00009: AMP (Probabilities: 0.1133, 0.8756, 0.0111)\n","Processed DRAMP00011: AMP (Probabilities: 0.0761, 0.9107, 0.0132)\n","Processed DRAMP00013: AMP (Probabilities: 0.0734, 0.9125, 0.0141)\n","Processed DRAMP00019: AMP (Probabilities: 0.0772, 0.9092, 0.0136)\n","Processed DRAMP00023: AMP (Probabilities: 0.0782, 0.9088, 0.0130)\n","Processed DRAMP00025: AMP (Probabilities: 0.0643, 0.9199, 0.0158)\n","Processed DRAMP00031: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed DRAMP00038: AMP (Probabilities: 0.1509, 0.8359, 0.0132)\n","Processed DRAMP18346: non-AMP/OTHERS (Probabilities: 0.9919, 0.0056, 0.0025)\n","Processed DRAMP18348: Neuropeptide (Probabilities: 0.0078, 0.0090, 0.9832)\n","Processed DRAMP00051: AMP (Probabilities: 0.2378, 0.5114, 0.2509)\n","Processed DRAMP00064: non-AMP/OTHERS (Probabilities: 0.9891, 0.0081, 0.0028)\n","Processed DRAMP00065: AMP (Probabilities: 0.0657, 0.9191, 0.0152)\n","Processed DRAMP00071: AMP (Probabilities: 0.0655, 0.9203, 0.0142)\n","Processed DRAMP00073: AMP (Probabilities: 0.0818, 0.9050, 0.0132)\n","Processed DRAMP00093: non-AMP/OTHERS (Probabilities: 0.9909, 0.0057, 0.0034)\n","Processed DRAMP00094: AMP (Probabilities: 0.0839, 0.9037, 0.0125)\n","Processed DRAMP18343: AMP (Probabilities: 0.3638, 0.4940, 0.1423)\n","Processed DRAMP18341: non-AMP/OTHERS (Probabilities: 0.9821, 0.0043, 0.0136)\n","Processed DRAMP00125: AMP (Probabilities: 0.1625, 0.6603, 0.1773)\n","Processed DRAMP00131: AMP (Probabilities: 0.0676, 0.9164, 0.0160)\n","Processed DRAMP00137: AMP (Probabilities: 0.0719, 0.9141, 0.0140)\n","Processed DRAMP00139: AMP (Probabilities: 0.0950, 0.8920, 0.0130)\n","Processed DRAMP00140: AMP (Probabilities: 0.0636, 0.9207, 0.0157)\n","Processed DRAMP00142: AMP (Probabilities: 0.0639, 0.9201, 0.0160)\n","Processed DRAMP00144: AMP (Probabilities: 0.0646, 0.9195, 0.0159)\n","Processed DRAMP00147: AMP (Probabilities: 0.0635, 0.9200, 0.0165)\n","Processed DRAMP18339: non-AMP/OTHERS (Probabilities: 0.9648, 0.0274, 0.0078)\n","Processed DRAMP00152: AMP (Probabilities: 0.0665, 0.9182, 0.0152)\n","Processed DRAMP00154: AMP (Probabilities: 0.0663, 0.9189, 0.0148)\n","Processed DRAMP00156: AMP (Probabilities: 0.0624, 0.9212, 0.0164)\n","Processed DRAMP00159: AMP (Probabilities: 0.0684, 0.9166, 0.0150)\n","Processed DRAMP00160: AMP (Probabilities: 0.0650, 0.9204, 0.0146)\n","Processed DRAMP00161: AMP (Probabilities: 0.0704, 0.9145, 0.0151)\n","Processed DRAMP00163: AMP (Probabilities: 0.2626, 0.5543, 0.1832)\n","Processed DRAMP00168: AMP (Probabilities: 0.0777, 0.9095, 0.0128)\n","Processed DRAMP18337: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed DRAMP00180: AMP (Probabilities: 0.0702, 0.9164, 0.0134)\n","Processed DRAMP00181: AMP (Probabilities: 0.1413, 0.8439, 0.0148)\n","Processed DRAMP00185: AMP (Probabilities: 0.0668, 0.9182, 0.0150)\n","Processed DRAMP00192: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed DRAMP00197: AMP (Probabilities: 0.0962, 0.8909, 0.0129)\n","Processed DRAMP00208: Neuropeptide (Probabilities: 0.0421, 0.0391, 0.9188)\n","Processed DRAMP00209: AMP (Probabilities: 0.0688, 0.9178, 0.0134)\n","Processed DRAMP00212: non-AMP/OTHERS (Probabilities: 0.9905, 0.0034, 0.0061)\n","Processed DRAMP00214: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0030)\n","Processed DRAMP00217: Neuropeptide (Probabilities: 0.0080, 0.0089, 0.9831)\n","Processed DRAMP00221: AMP (Probabilities: 0.0650, 0.9195, 0.0155)\n","Processed DRAMP00224: non-AMP/OTHERS (Probabilities: 0.9408, 0.0083, 0.0509)\n","Processed DRAMP00225: non-AMP/OTHERS (Probabilities: 0.9811, 0.0162, 0.0027)\n","Processed DRAMP00226: non-AMP/OTHERS (Probabilities: 0.5047, 0.2324, 0.2629)\n","Processed DRAMP00227: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed DRAMP00228: Neuropeptide (Probabilities: 0.0073, 0.0087, 0.9840)\n","Processed DRAMP00233: Neuropeptide (Probabilities: 0.0221, 0.0160, 0.9619)\n","Processed DRAMP00234: Neuropeptide (Probabilities: 0.0072, 0.0093, 0.9835)\n","Processed DRAMP00235: non-AMP/OTHERS (Probabilities: 0.6568, 0.3300, 0.0131)\n","Processed DRAMP00238: AMP (Probabilities: 0.0789, 0.9085, 0.0125)\n","Processed DRAMP00242: non-AMP/OTHERS (Probabilities: 0.8701, 0.0785, 0.0514)\n","Processed DRAMP00250: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed DRAMP00252: AMP (Probabilities: 0.2364, 0.7175, 0.0461)\n","Processed DRAMP00256: AMP (Probabilities: 0.0702, 0.9153, 0.0145)\n","Processed DRAMP00259: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed DRAMP00261: AMP (Probabilities: 0.0679, 0.9186, 0.0135)\n","Processed DRAMP00262: Neuropeptide (Probabilities: 0.0236, 0.0163, 0.9602)\n","Processed DRAMP00263: AMP (Probabilities: 0.0622, 0.9212, 0.0166)\n","Processed DRAMP00264: Neuropeptide (Probabilities: 0.0070, 0.0087, 0.9843)\n","Processed DRAMP00265: Neuropeptide (Probabilities: 0.0573, 0.0186, 0.9241)\n","Processed DRAMP00266: Neuropeptide (Probabilities: 0.0272, 0.0131, 0.9597)\n","Processed DRAMP00267: non-AMP/OTHERS (Probabilities: 0.9913, 0.0036, 0.0051)\n","Processed DRAMP00268: Neuropeptide (Probabilities: 0.4715, 0.0410, 0.4875)\n","Processed DRAMP00269: Neuropeptide (Probabilities: 0.0095, 0.0099, 0.9806)\n","Processed DRAMP00270: AMP (Probabilities: 0.3228, 0.6659, 0.0113)\n","Processed DRAMP00271: AMP (Probabilities: 0.2799, 0.6992, 0.0208)\n","Processed DRAMP00272: non-AMP/OTHERS (Probabilities: 0.7065, 0.2817, 0.0118)\n","Processed DRAMP00273: non-AMP/OTHERS (Probabilities: 0.9900, 0.0067, 0.0033)\n","Processed DRAMP00274: non-AMP/OTHERS (Probabilities: 0.9920, 0.0041, 0.0040)\n","Processed DRAMP00277: AMP (Probabilities: 0.1201, 0.8679, 0.0120)\n","Processed DRAMP00278: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed DRAMP00279: non-AMP/OTHERS (Probabilities: 0.9835, 0.0138, 0.0027)\n","Processed DRAMP00280: AMP (Probabilities: 0.0680, 0.9181, 0.0138)\n","Processed DRAMP00281: AMP (Probabilities: 0.0787, 0.9075, 0.0138)\n","Processed DRAMP00282: AMP (Probabilities: 0.0720, 0.9139, 0.0141)\n","Processed DRAMP00283: AMP (Probabilities: 0.0668, 0.9178, 0.0154)\n","Processed DRAMP00284: AMP (Probabilities: 0.0652, 0.9191, 0.0157)\n","Processed DRAMP00285: AMP (Probabilities: 0.0649, 0.9196, 0.0155)\n","Processed DRAMP00289: AMP (Probabilities: 0.0683, 0.9175, 0.0142)\n","Processed DRAMP00291: AMP (Probabilities: 0.0647, 0.9194, 0.0159)\n","Processed DRAMP00292: AMP (Probabilities: 0.0659, 0.9186, 0.0155)\n","Processed DRAMP00293: AMP (Probabilities: 0.0676, 0.9176, 0.0148)\n","Processed DRAMP00294: AMP (Probabilities: 0.0670, 0.9166, 0.0164)\n","Processed DRAMP00295: non-AMP/OTHERS (Probabilities: 0.9795, 0.0152, 0.0053)\n","Processed DRAMP00296: non-AMP/OTHERS (Probabilities: 0.9926, 0.0044, 0.0030)\n","Processed DRAMP00297: AMP (Probabilities: 0.1069, 0.8812, 0.0119)\n","Processed DRAMP00298: AMP (Probabilities: 0.0697, 0.9160, 0.0143)\n","Processed DRAMP00299: non-AMP/OTHERS (Probabilities: 0.9847, 0.0047, 0.0106)\n","Processed DRAMP00300: Neuropeptide (Probabilities: 0.0343, 0.0165, 0.9492)\n","Processed DRAMP00301: Neuropeptide (Probabilities: 0.0114, 0.0106, 0.9780)\n","Processed DRAMP00302: Neuropeptide (Probabilities: 0.0072, 0.0087, 0.9841)\n","Processed DRAMP00303: Neuropeptide (Probabilities: 0.0137, 0.0129, 0.9734)\n","Processed DRAMP00304: non-AMP/OTHERS (Probabilities: 0.5872, 0.0239, 0.3889)\n","Processed DRAMP00305: non-AMP/OTHERS (Probabilities: 0.9859, 0.0107, 0.0034)\n","Processed DRAMP00306: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed DRAMP00307: non-AMP/OTHERS (Probabilities: 0.5808, 0.1260, 0.2932)\n","Processed DRAMP00309: non-AMP/OTHERS (Probabilities: 0.9915, 0.0034, 0.0051)\n","Processed DRAMP00310: Neuropeptide (Probabilities: 0.0167, 0.0164, 0.9669)\n","Processed DRAMP00311: non-AMP/OTHERS (Probabilities: 0.9897, 0.0071, 0.0032)\n","Processed DRAMP00312: non-AMP/OTHERS (Probabilities: 0.9764, 0.0211, 0.0026)\n","Processed DRAMP00313: non-AMP/OTHERS (Probabilities: 0.9924, 0.0035, 0.0042)\n","Processed DRAMP00314: Neuropeptide (Probabilities: 0.0102, 0.0099, 0.9799)\n","Processed DRAMP00315: non-AMP/OTHERS (Probabilities: 0.9927, 0.0046, 0.0027)\n","Processed DRAMP00320: non-AMP/OTHERS (Probabilities: 0.7528, 0.2327, 0.0145)\n","Processed DRAMP00321: AMP (Probabilities: 0.1087, 0.8795, 0.0118)\n","Processed DRAMP00322: Neuropeptide (Probabilities: 0.0066, 0.0091, 0.9843)\n","Processed DRAMP00323: Neuropeptide (Probabilities: 0.0077, 0.0097, 0.9826)\n","Processed DRAMP00324: non-AMP/OTHERS (Probabilities: 0.9918, 0.0035, 0.0047)\n","Processed DRAMP00325: Neuropeptide (Probabilities: 0.0080, 0.0091, 0.9830)\n","Processed DRAMP00327: non-AMP/OTHERS (Probabilities: 0.9898, 0.0049, 0.0053)\n","Processed DRAMP00328: non-AMP/OTHERS (Probabilities: 0.9485, 0.0109, 0.0406)\n","Processed DRAMP00330: non-AMP/OTHERS (Probabilities: 0.9917, 0.0052, 0.0031)\n","Processed DRAMP00331: non-AMP/OTHERS (Probabilities: 0.9920, 0.0032, 0.0048)\n","Processed DRAMP00332: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed DRAMP00333: non-AMP/OTHERS (Probabilities: 0.9926, 0.0034, 0.0040)\n","Processed DRAMP00334: non-AMP/OTHERS (Probabilities: 0.9751, 0.0068, 0.0181)\n","Processed DRAMP00335: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0031)\n","Processed DRAMP00339: non-AMP/OTHERS (Probabilities: 0.9926, 0.0034, 0.0040)\n","Processed DRAMP00340: Neuropeptide (Probabilities: 0.1422, 0.1671, 0.6907)\n","Processed DRAMP00342: non-AMP/OTHERS (Probabilities: 0.9823, 0.0147, 0.0030)\n","Processed DRAMP00343: non-AMP/OTHERS (Probabilities: 0.9792, 0.0167, 0.0041)\n","Processed DRAMP00344: AMP (Probabilities: 0.0845, 0.9035, 0.0120)\n","Processed DRAMP00345: AMP (Probabilities: 0.1126, 0.8748, 0.0126)\n","Processed DRAMP00346: AMP (Probabilities: 0.0876, 0.9002, 0.0122)\n","Processed DRAMP00347: AMP (Probabilities: 0.0714, 0.9156, 0.0130)\n","Processed DRAMP00348: AMP (Probabilities: 0.1131, 0.8746, 0.0124)\n","Processed DRAMP00351: AMP (Probabilities: 0.0791, 0.9085, 0.0124)\n","Processed DRAMP00352: AMP (Probabilities: 0.1015, 0.8864, 0.0121)\n","Processed DRAMP00353: AMP (Probabilities: 0.0792, 0.9085, 0.0123)\n","Processed DRAMP00354: AMP (Probabilities: 0.1190, 0.8684, 0.0126)\n","Processed DRAMP00356: AMP (Probabilities: 0.0683, 0.9183, 0.0134)\n","Processed DRAMP00358: AMP (Probabilities: 0.0960, 0.8917, 0.0124)\n","Processed DRAMP00360: AMP (Probabilities: 0.3304, 0.6489, 0.0207)\n","Processed DRAMP00361: non-AMP/OTHERS (Probabilities: 0.9909, 0.0065, 0.0026)\n","Processed DRAMP00362: AMP (Probabilities: 0.1194, 0.8680, 0.0126)\n","Processed DRAMP00363: AMP (Probabilities: 0.1046, 0.8828, 0.0126)\n","Processed DRAMP00369: AMP (Probabilities: 0.0753, 0.9121, 0.0126)\n","Processed DRAMP00370: Neuropeptide (Probabilities: 0.0182, 0.0151, 0.9667)\n","Processed DRAMP00371: AMP (Probabilities: 0.0695, 0.9155, 0.0150)\n","Processed DRAMP00373: AMP (Probabilities: 0.0978, 0.8900, 0.0122)\n","Processed DRAMP00374: AMP (Probabilities: 0.1085, 0.8786, 0.0129)\n","Processed DRAMP00375: Neuropeptide (Probabilities: 0.0069, 0.0087, 0.9844)\n","Processed DRAMP00377: non-AMP/OTHERS (Probabilities: 0.8617, 0.0107, 0.1276)\n","Processed DRAMP00378: non-AMP/OTHERS (Probabilities: 0.6701, 0.0222, 0.3076)\n","Processed DRAMP00380: Neuropeptide (Probabilities: 0.0580, 0.0328, 0.9092)\n","Processed DRAMP00381: AMP (Probabilities: 0.1272, 0.8318, 0.0411)\n","Processed DRAMP00382: AMP (Probabilities: 0.1250, 0.8624, 0.0126)\n","Processed DRAMP00387: AMP (Probabilities: 0.0689, 0.9174, 0.0137)\n","Processed DRAMP00393: AMP (Probabilities: 0.0678, 0.9177, 0.0145)\n","Processed DRAMP00394: Neuropeptide (Probabilities: 0.0327, 0.0259, 0.9415)\n","Processed DRAMP00401: AMP (Probabilities: 0.0649, 0.9197, 0.0155)\n","Processed DRAMP00402: non-AMP/OTHERS (Probabilities: 0.6576, 0.3014, 0.0410)\n","Processed DRAMP00407: Neuropeptide (Probabilities: 0.2024, 0.3956, 0.4020)\n","Processed DRAMP00419: AMP (Probabilities: 0.0628, 0.9207, 0.0165)\n","Processed DRAMP00426: AMP (Probabilities: 0.0684, 0.9166, 0.0150)\n","Processed DRAMP00427: AMP (Probabilities: 0.0676, 0.9168, 0.0156)\n","Processed DRAMP00428: AMP (Probabilities: 0.1453, 0.8416, 0.0131)\n","Processed DRAMP00429: AMP (Probabilities: 0.0693, 0.9157, 0.0150)\n","Processed DRAMP00430: AMP (Probabilities: 0.0665, 0.9181, 0.0154)\n","Processed DRAMP00438: AMP (Probabilities: 0.0629, 0.9211, 0.0160)\n","Processed DRAMP00440: AMP (Probabilities: 0.0694, 0.9156, 0.0150)\n","Processed DRAMP00441: AMP (Probabilities: 0.0674, 0.9178, 0.0148)\n","Processed DRAMP00442: AMP (Probabilities: 0.0654, 0.9193, 0.0153)\n","Processed DRAMP00445: AMP (Probabilities: 0.0677, 0.9173, 0.0150)\n","Processed DRAMP00446: AMP (Probabilities: 0.0642, 0.9205, 0.0152)\n","Processed DRAMP00451: AMP (Probabilities: 0.0674, 0.9167, 0.0160)\n","Processed DRAMP00452: AMP (Probabilities: 0.0660, 0.9183, 0.0156)\n","Processed DRAMP00453: AMP (Probabilities: 0.0672, 0.9177, 0.0151)\n","Processed DRAMP00457: AMP (Probabilities: 0.0677, 0.9178, 0.0145)\n","Processed DRAMP00459: AMP (Probabilities: 0.0640, 0.9203, 0.0157)\n","Processed DRAMP00460: AMP (Probabilities: 0.0682, 0.9167, 0.0150)\n","Processed DRAMP00461: AMP (Probabilities: 0.0652, 0.9193, 0.0155)\n","Processed DRAMP00462: AMP (Probabilities: 0.0660, 0.9188, 0.0152)\n","Processed DRAMP00463: AMP (Probabilities: 0.0693, 0.9161, 0.0146)\n","Processed DRAMP00464: AMP (Probabilities: 0.0701, 0.9150, 0.0149)\n","Processed DRAMP00465: AMP (Probabilities: 0.0664, 0.9185, 0.0151)\n","Processed DRAMP00466: AMP (Probabilities: 0.0652, 0.9191, 0.0157)\n","Processed DRAMP00467: AMP (Probabilities: 0.0650, 0.9191, 0.0159)\n","Processed DRAMP00468: Neuropeptide (Probabilities: 0.0288, 0.0181, 0.9531)\n","Processed DRAMP18329: AMP (Probabilities: 0.0650, 0.9207, 0.0143)\n","Processed DRAMP18330: AMP (Probabilities: 0.1522, 0.8128, 0.0350)\n","Processed DRAMP18328: AMP (Probabilities: 0.0627, 0.9211, 0.0163)\n","Processed DRAMP00474: AMP (Probabilities: 0.0655, 0.9189, 0.0157)\n","Processed DRAMP00475: AMP (Probabilities: 0.0684, 0.9161, 0.0155)\n","Processed DRAMP00476: AMP (Probabilities: 0.0660, 0.9185, 0.0155)\n","Processed DRAMP00479: AMP (Probabilities: 0.0777, 0.9090, 0.0134)\n","Processed DRAMP00480: Neuropeptide (Probabilities: 0.0243, 0.0158, 0.9599)\n","Processed DRAMP00482: AMP (Probabilities: 0.0690, 0.9162, 0.0148)\n","Processed DRAMP00483: AMP (Probabilities: 0.0680, 0.9178, 0.0142)\n","Processed DRAMP00484: Neuropeptide (Probabilities: 0.0095, 0.0100, 0.9804)\n","Processed DRAMP00485: AMP (Probabilities: 0.0679, 0.9169, 0.0152)\n","Processed DRAMP00486: AMP (Probabilities: 0.0669, 0.9177, 0.0155)\n","Processed DRAMP00487: AMP (Probabilities: 0.0683, 0.9178, 0.0139)\n","Processed DRAMP00488: AMP (Probabilities: 0.1112, 0.8506, 0.0382)\n","Processed DRAMP00489: AMP (Probabilities: 0.0818, 0.9002, 0.0179)\n","Processed DRAMP00490: AMP (Probabilities: 0.3759, 0.5968, 0.0273)\n","Processed DRAMP00491: AMP (Probabilities: 0.0695, 0.9171, 0.0134)\n","Processed DRAMP00492: AMP (Probabilities: 0.4063, 0.4703, 0.1234)\n","Processed DRAMP00493: AMP (Probabilities: 0.0667, 0.9175, 0.0157)\n","Processed DRAMP00494: AMP (Probabilities: 0.0676, 0.9177, 0.0147)\n","Processed DRAMP00495: AMP (Probabilities: 0.0688, 0.9162, 0.0150)\n","Processed DRAMP00496: AMP (Probabilities: 0.0903, 0.8967, 0.0130)\n","Processed DRAMP00497: AMP (Probabilities: 0.0672, 0.9181, 0.0147)\n","Processed DRAMP00498: AMP (Probabilities: 0.0661, 0.9186, 0.0153)\n","Processed DRAMP00499: AMP (Probabilities: 0.0720, 0.9136, 0.0144)\n","Processed DRAMP00501: AMP (Probabilities: 0.0728, 0.9140, 0.0132)\n","Processed DRAMP00502: AMP (Probabilities: 0.0658, 0.9190, 0.0153)\n","Processed DRAMP00503: AMP (Probabilities: 0.2005, 0.5769, 0.2225)\n","Processed DRAMP00505: AMP (Probabilities: 0.0675, 0.9175, 0.0150)\n","Processed DRAMP00506: AMP (Probabilities: 0.0721, 0.9148, 0.0131)\n","Processed DRAMP00508: AMP (Probabilities: 0.0677, 0.9173, 0.0150)\n","Processed DRAMP00509: AMP (Probabilities: 0.0715, 0.9157, 0.0129)\n","Processed DRAMP00510: AMP (Probabilities: 0.0649, 0.9197, 0.0154)\n","Processed DRAMP00512: AMP (Probabilities: 0.2071, 0.6786, 0.1142)\n","Processed DRAMP00513: AMP (Probabilities: 0.0671, 0.9185, 0.0144)\n","Processed DRAMP00515: Neuropeptide (Probabilities: 0.1591, 0.0618, 0.7791)\n","Processed DRAMP00516: AMP (Probabilities: 0.0655, 0.9192, 0.0153)\n","Processed DRAMP00517: AMP (Probabilities: 0.0685, 0.9166, 0.0149)\n","Processed DRAMP00518: non-AMP/OTHERS (Probabilities: 0.9417, 0.0394, 0.0189)\n","Processed DRAMP00519: AMP (Probabilities: 0.0774, 0.9088, 0.0138)\n","Processed DRAMP00520: AMP (Probabilities: 0.0714, 0.9151, 0.0135)\n","Processed DRAMP00521: AMP (Probabilities: 0.0676, 0.9184, 0.0140)\n","Processed DRAMP00522: Neuropeptide (Probabilities: 0.3845, 0.1073, 0.5082)\n","Processed DRAMP00523: Neuropeptide (Probabilities: 0.3509, 0.2565, 0.3926)\n","Processed DRAMP00524: AMP (Probabilities: 0.0694, 0.9160, 0.0146)\n","Processed DRAMP00525: AMP (Probabilities: 0.0660, 0.9182, 0.0158)\n","Processed DRAMP00526: AMP (Probabilities: 0.0680, 0.9172, 0.0148)\n","Processed DRAMP00527: AMP (Probabilities: 0.0664, 0.9185, 0.0151)\n","Processed DRAMP00528: AMP (Probabilities: 0.0639, 0.9201, 0.0160)\n","Processed DRAMP00529: AMP (Probabilities: 0.0717, 0.9155, 0.0128)\n","Processed DRAMP00530: AMP (Probabilities: 0.0759, 0.9106, 0.0134)\n","Processed DRAMP00531: AMP (Probabilities: 0.0663, 0.9182, 0.0155)\n","Processed DRAMP00532: AMP (Probabilities: 0.0713, 0.9154, 0.0133)\n","Processed DRAMP00533: AMP (Probabilities: 0.0662, 0.9190, 0.0149)\n","Processed DRAMP00534: AMP (Probabilities: 0.0677, 0.9170, 0.0153)\n","Processed DRAMP00535: AMP (Probabilities: 0.0663, 0.9199, 0.0138)\n","Processed DRAMP00536: AMP (Probabilities: 0.0667, 0.9190, 0.0143)\n","Processed DRAMP00537: AMP (Probabilities: 0.0654, 0.9187, 0.0160)\n","Processed DRAMP00538: AMP (Probabilities: 0.0651, 0.9193, 0.0155)\n","Processed DRAMP00539: AMP (Probabilities: 0.0737, 0.9126, 0.0137)\n","Processed DRAMP00540: AMP (Probabilities: 0.0701, 0.9142, 0.0157)\n","Processed DRAMP00541: AMP (Probabilities: 0.0673, 0.9182, 0.0145)\n","Processed DRAMP00542: AMP (Probabilities: 0.3257, 0.5930, 0.0813)\n","Processed DRAMP00543: AMP (Probabilities: 0.0663, 0.9179, 0.0158)\n","Processed DRAMP00544: AMP (Probabilities: 0.0669, 0.9179, 0.0152)\n","Processed DRAMP00545: AMP (Probabilities: 0.0658, 0.9192, 0.0150)\n","Processed DRAMP00546: AMP (Probabilities: 0.2508, 0.4985, 0.2507)\n","Processed DRAMP00547: Neuropeptide (Probabilities: 0.0917, 0.0497, 0.8586)\n","Processed DRAMP00548: AMP (Probabilities: 0.4077, 0.4569, 0.1353)\n","Processed DRAMP00549: AMP (Probabilities: 0.0697, 0.9160, 0.0143)\n","Processed DRAMP00550: AMP (Probabilities: 0.0751, 0.9125, 0.0124)\n","Processed DRAMP00551: AMP (Probabilities: 0.0642, 0.9196, 0.0162)\n","Processed DRAMP00552: AMP (Probabilities: 0.0770, 0.9094, 0.0136)\n","Processed DRAMP00553: AMP (Probabilities: 0.0688, 0.9169, 0.0143)\n","Processed DRAMP00554: AMP (Probabilities: 0.1126, 0.8752, 0.0122)\n","Processed DRAMP00555: AMP (Probabilities: 0.0725, 0.9112, 0.0162)\n","Processed DRAMP00556: AMP (Probabilities: 0.0687, 0.9180, 0.0133)\n","Processed DRAMP00557: Neuropeptide (Probabilities: 0.1319, 0.0999, 0.7682)\n","Processed DRAMP00558: AMP (Probabilities: 0.0686, 0.9175, 0.0138)\n","Processed DRAMP00559: AMP (Probabilities: 0.3126, 0.4771, 0.2104)\n","Processed DRAMP00560: AMP (Probabilities: 0.0705, 0.9156, 0.0140)\n","Processed DRAMP00561: AMP (Probabilities: 0.0665, 0.9183, 0.0152)\n","Processed DRAMP00562: AMP (Probabilities: 0.0886, 0.8989, 0.0125)\n","Processed DRAMP00563: AMP (Probabilities: 0.0678, 0.9181, 0.0140)\n","Processed DRAMP00564: AMP (Probabilities: 0.0714, 0.9146, 0.0140)\n","Processed DRAMP00565: Neuropeptide (Probabilities: 0.0077, 0.0091, 0.9832)\n","Processed DRAMP00566: AMP (Probabilities: 0.0688, 0.9162, 0.0150)\n","Processed DRAMP00567: AMP (Probabilities: 0.0676, 0.9172, 0.0153)\n","Processed DRAMP00569: AMP (Probabilities: 0.0695, 0.9159, 0.0147)\n","Processed DRAMP00570: AMP (Probabilities: 0.1021, 0.8701, 0.0277)\n","Processed DRAMP00571: AMP (Probabilities: 0.0661, 0.9180, 0.0159)\n","Processed DRAMP00572: AMP (Probabilities: 0.0656, 0.9188, 0.0156)\n","Processed DRAMP00573: AMP (Probabilities: 0.0739, 0.9120, 0.0141)\n","Processed DRAMP00574: non-AMP/OTHERS (Probabilities: 0.9880, 0.0081, 0.0039)\n","Processed DRAMP00575: AMP (Probabilities: 0.0680, 0.9182, 0.0137)\n","Processed DRAMP00576: AMP (Probabilities: 0.0659, 0.9182, 0.0159)\n","Processed DRAMP00577: AMP (Probabilities: 0.0658, 0.9186, 0.0156)\n","Processed DRAMP00578: non-AMP/OTHERS (Probabilities: 0.8495, 0.1059, 0.0446)\n","Processed DRAMP00579: AMP (Probabilities: 0.2244, 0.4062, 0.3694)\n","Processed DRAMP00580: non-AMP/OTHERS (Probabilities: 0.8370, 0.1326, 0.0303)\n","Processed DRAMP00581: AMP (Probabilities: 0.0805, 0.9068, 0.0127)\n","Processed DRAMP00582: AMP (Probabilities: 0.0664, 0.9179, 0.0158)\n","Processed DRAMP00583: AMP (Probabilities: 0.0676, 0.9174, 0.0149)\n","Processed DRAMP00584: AMP (Probabilities: 0.0689, 0.9165, 0.0146)\n","Processed DRAMP00585: AMP (Probabilities: 0.0663, 0.9185, 0.0152)\n","Processed DRAMP00586: AMP (Probabilities: 0.2110, 0.7236, 0.0654)\n","Processed DRAMP00587: AMP (Probabilities: 0.0649, 0.9200, 0.0151)\n","Processed DRAMP00588: AMP (Probabilities: 0.0686, 0.9164, 0.0150)\n","Processed DRAMP00589: AMP (Probabilities: 0.0691, 0.9162, 0.0147)\n","Processed DRAMP00590: AMP (Probabilities: 0.0659, 0.9191, 0.0150)\n","Processed DRAMP00591: AMP (Probabilities: 0.0758, 0.9110, 0.0132)\n","Processed DRAMP00592: AMP (Probabilities: 0.0673, 0.9175, 0.0153)\n","Processed DRAMP00593: AMP (Probabilities: 0.0692, 0.9162, 0.0146)\n","Processed DRAMP00594: AMP (Probabilities: 0.0695, 0.9159, 0.0145)\n","Processed DRAMP00595: AMP (Probabilities: 0.0665, 0.9183, 0.0152)\n","Processed DRAMP00596: AMP (Probabilities: 0.0678, 0.9167, 0.0155)\n","Processed DRAMP00597: AMP (Probabilities: 0.0672, 0.9175, 0.0152)\n","Processed DRAMP00598: AMP (Probabilities: 0.0685, 0.9168, 0.0147)\n","Processed DRAMP00599: AMP (Probabilities: 0.0820, 0.9051, 0.0129)\n","Processed DRAMP00600: AMP (Probabilities: 0.0825, 0.9029, 0.0146)\n","Processed DRAMP00601: AMP (Probabilities: 0.0694, 0.9175, 0.0131)\n","Processed DRAMP00602: AMP (Probabilities: 0.0676, 0.9169, 0.0155)\n","Processed DRAMP00603: AMP (Probabilities: 0.0682, 0.9172, 0.0145)\n","Processed DRAMP00605: AMP (Probabilities: 0.1866, 0.7101, 0.1033)\n","Processed DRAMP00606: AMP (Probabilities: 0.3508, 0.5542, 0.0950)\n","Processed DRAMP00607: AMP (Probabilities: 0.0690, 0.9162, 0.0148)\n","Processed DRAMP00608: AMP (Probabilities: 0.0673, 0.9180, 0.0147)\n","Processed DRAMP00609: AMP (Probabilities: 0.0686, 0.9181, 0.0133)\n","Processed DRAMP00610: AMP (Probabilities: 0.0659, 0.9188, 0.0153)\n","Processed DRAMP00611: AMP (Probabilities: 0.0694, 0.9166, 0.0140)\n","Processed DRAMP00612: AMP (Probabilities: 0.0659, 0.9187, 0.0154)\n","Processed DRAMP00613: AMP (Probabilities: 0.0695, 0.9158, 0.0147)\n","Processed DRAMP00614: AMP (Probabilities: 0.0823, 0.9020, 0.0156)\n","Processed DRAMP00615: AMP (Probabilities: 0.0676, 0.9181, 0.0143)\n","Processed DRAMP00616: AMP (Probabilities: 0.0677, 0.9187, 0.0136)\n","Processed DRAMP00617: AMP (Probabilities: 0.0680, 0.9169, 0.0151)\n","Processed DRAMP00618: AMP (Probabilities: 0.0685, 0.9174, 0.0141)\n","Processed DRAMP00620: AMP (Probabilities: 0.0662, 0.9201, 0.0137)\n","Processed DRAMP00621: AMP (Probabilities: 0.0675, 0.9174, 0.0151)\n","Processed DRAMP00622: AMP (Probabilities: 0.0894, 0.8984, 0.0123)\n","Processed DRAMP00623: AMP (Probabilities: 0.0654, 0.9190, 0.0156)\n","Processed DRAMP00624: AMP (Probabilities: 0.0653, 0.9194, 0.0153)\n","Processed DRAMP00625: AMP (Probabilities: 0.0666, 0.9188, 0.0146)\n","Processed DRAMP00627: AMP (Probabilities: 0.0653, 0.9193, 0.0154)\n","Processed DRAMP00628: AMP (Probabilities: 0.2904, 0.4966, 0.2131)\n","Processed DRAMP00629: AMP (Probabilities: 0.0675, 0.9184, 0.0141)\n","Processed DRAMP00630: AMP (Probabilities: 0.0649, 0.9200, 0.0151)\n","Processed DRAMP00631: AMP (Probabilities: 0.0659, 0.9192, 0.0148)\n","Processed DRAMP00632: AMP (Probabilities: 0.0664, 0.9188, 0.0148)\n","Processed DRAMP00633: AMP (Probabilities: 0.0673, 0.9176, 0.0151)\n","Processed DRAMP00634: AMP (Probabilities: 0.0650, 0.9194, 0.0156)\n","Processed DRAMP00636: AMP (Probabilities: 0.0660, 0.9186, 0.0154)\n","Processed DRAMP00637: AMP (Probabilities: 0.0669, 0.9182, 0.0149)\n","Processed DRAMP00638: AMP (Probabilities: 0.2169, 0.4139, 0.3692)\n","Processed DRAMP00639: non-AMP/OTHERS (Probabilities: 0.5558, 0.2827, 0.1615)\n","Processed DRAMP00640: AMP (Probabilities: 0.0721, 0.9152, 0.0127)\n","Processed DRAMP00641: AMP (Probabilities: 0.0689, 0.9178, 0.0133)\n","Processed DRAMP00642: AMP (Probabilities: 0.0686, 0.9169, 0.0146)\n","Processed DRAMP00643: AMP (Probabilities: 0.0954, 0.8898, 0.0148)\n","Processed DRAMP00644: AMP (Probabilities: 0.0662, 0.9179, 0.0159)\n","Processed DRAMP00646: AMP (Probabilities: 0.0655, 0.9187, 0.0159)\n","Processed DRAMP00647: AMP (Probabilities: 0.1137, 0.8690, 0.0173)\n","Processed DRAMP00649: AMP (Probabilities: 0.0754, 0.9121, 0.0126)\n","Processed DRAMP00650: AMP (Probabilities: 0.1122, 0.8731, 0.0147)\n","Processed DRAMP00655: AMP (Probabilities: 0.2688, 0.4007, 0.3305)\n","Processed DRAMP00656: AMP (Probabilities: 0.0656, 0.9187, 0.0157)\n","Processed DRAMP00657: non-AMP/OTHERS (Probabilities: 0.9633, 0.0074, 0.0293)\n","Processed DRAMP00660: Neuropeptide (Probabilities: 0.0891, 0.0470, 0.8639)\n","Processed DRAMP00661: AMP (Probabilities: 0.0675, 0.9177, 0.0148)\n","Processed DRAMP00662: AMP (Probabilities: 0.0661, 0.9183, 0.0156)\n","Processed DRAMP00663: AMP (Probabilities: 0.0661, 0.9185, 0.0154)\n","Processed DRAMP00664: AMP (Probabilities: 0.0678, 0.9172, 0.0150)\n","Processed DRAMP00665: AMP (Probabilities: 0.0680, 0.9169, 0.0152)\n","Processed DRAMP00666: AMP (Probabilities: 0.0661, 0.9185, 0.0154)\n","Processed DRAMP00667: AMP (Probabilities: 0.0662, 0.9181, 0.0157)\n","Processed DRAMP00668: AMP (Probabilities: 0.0683, 0.9170, 0.0146)\n","Processed DRAMP00669: AMP (Probabilities: 0.0695, 0.9163, 0.0142)\n","Processed DRAMP00670: AMP (Probabilities: 0.0705, 0.9152, 0.0143)\n","Processed DRAMP00671: AMP (Probabilities: 0.0682, 0.9166, 0.0152)\n","Processed DRAMP00672: AMP (Probabilities: 0.0685, 0.9169, 0.0146)\n","Processed DRAMP00673: AMP (Probabilities: 0.0679, 0.9171, 0.0150)\n","Processed DRAMP00674: AMP (Probabilities: 0.0668, 0.9184, 0.0148)\n","Processed DRAMP00675: AMP (Probabilities: 0.0689, 0.9158, 0.0153)\n","Processed DRAMP00676: Neuropeptide (Probabilities: 0.1515, 0.1348, 0.7137)\n","Processed DRAMP00677: AMP (Probabilities: 0.0660, 0.9185, 0.0156)\n","Processed DRAMP00678: AMP (Probabilities: 0.0637, 0.9204, 0.0159)\n","Processed DRAMP00679: Neuropeptide (Probabilities: 0.0089, 0.0098, 0.9813)\n","Processed DRAMP00680: Neuropeptide (Probabilities: 0.0698, 0.0346, 0.8955)\n","Processed DRAMP00681: AMP (Probabilities: 0.0689, 0.9163, 0.0148)\n","Processed DRAMP00682: Neuropeptide (Probabilities: 0.0094, 0.0099, 0.9807)\n","Processed DRAMP00683: Neuropeptide (Probabilities: 0.0077, 0.0091, 0.9832)\n","Processed DRAMP00684: Neuropeptide (Probabilities: 0.0079, 0.0094, 0.9827)\n","Processed DRAMP00685: Neuropeptide (Probabilities: 0.0842, 0.0515, 0.8642)\n","Processed DRAMP00686: AMP (Probabilities: 0.0778, 0.9059, 0.0163)\n","Processed DRAMP00687: AMP (Probabilities: 0.0698, 0.9160, 0.0142)\n","Processed DRAMP00688: AMP (Probabilities: 0.0677, 0.9174, 0.0149)\n","Processed DRAMP00689: AMP (Probabilities: 0.0680, 0.9173, 0.0148)\n","Processed DRAMP00690: Neuropeptide (Probabilities: 0.0071, 0.0091, 0.9839)\n","Processed DRAMP00691: Neuropeptide (Probabilities: 0.0223, 0.0154, 0.9624)\n","Processed DRAMP00692: AMP (Probabilities: 0.0716, 0.9152, 0.0132)\n","Processed DRAMP00693: Neuropeptide (Probabilities: 0.1192, 0.0502, 0.8306)\n","Processed DRAMP00694: Neuropeptide (Probabilities: 0.0972, 0.0781, 0.8247)\n","Processed DRAMP00695: Neuropeptide (Probabilities: 0.0505, 0.0284, 0.9211)\n","Processed DRAMP00696: AMP (Probabilities: 0.3443, 0.5088, 0.1469)\n","Processed DRAMP00697: AMP (Probabilities: 0.2510, 0.4479, 0.3011)\n","Processed DRAMP00698: Neuropeptide (Probabilities: 0.0391, 0.0222, 0.9387)\n","Processed DRAMP00699: Neuropeptide (Probabilities: 0.2815, 0.0944, 0.6240)\n","Processed DRAMP00700: AMP (Probabilities: 0.2960, 0.5303, 0.1737)\n","Processed DRAMP00701: Neuropeptide (Probabilities: 0.0087, 0.0099, 0.9814)\n","Processed DRAMP00702: AMP (Probabilities: 0.0683, 0.9173, 0.0144)\n","Processed DRAMP00703: AMP (Probabilities: 0.1015, 0.8798, 0.0187)\n","Processed DRAMP00704: AMP (Probabilities: 0.0649, 0.9203, 0.0148)\n","Processed DRAMP00705: AMP (Probabilities: 0.0712, 0.9155, 0.0133)\n","Processed DRAMP00706: AMP (Probabilities: 0.4142, 0.4647, 0.1211)\n","Processed DRAMP00707: AMP (Probabilities: 0.0673, 0.9174, 0.0153)\n","Processed DRAMP00708: Neuropeptide (Probabilities: 0.0118, 0.0107, 0.9774)\n","Processed DRAMP00709: AMP (Probabilities: 0.0655, 0.9192, 0.0153)\n","Processed DRAMP00710: AMP (Probabilities: 0.0680, 0.9169, 0.0151)\n","Processed DRAMP00711: AMP (Probabilities: 0.0685, 0.9182, 0.0133)\n","Processed DRAMP00712: AMP (Probabilities: 0.0705, 0.9153, 0.0142)\n","Processed DRAMP00714: AMP (Probabilities: 0.2327, 0.6383, 0.1291)\n","Processed DRAMP00715: AMP (Probabilities: 0.0658, 0.9186, 0.0156)\n","Processed DRAMP00716: AMP (Probabilities: 0.0697, 0.9167, 0.0136)\n","Processed DRAMP00717: AMP (Probabilities: 0.0670, 0.9179, 0.0152)\n","Processed DRAMP00718: Neuropeptide (Probabilities: 0.2148, 0.0534, 0.7318)\n","Processed DRAMP00719: AMP (Probabilities: 0.3263, 0.3603, 0.3134)\n","Processed DRAMP00720: AMP (Probabilities: 0.0666, 0.9189, 0.0145)\n","Processed DRAMP00721: AMP (Probabilities: 0.0661, 0.9191, 0.0148)\n","Processed DRAMP00722: non-AMP/OTHERS (Probabilities: 0.9749, 0.0136, 0.0115)\n","Processed DRAMP00723: AMP (Probabilities: 0.1408, 0.8236, 0.0356)\n","Processed DRAMP00724: Neuropeptide (Probabilities: 0.0603, 0.0292, 0.9105)\n","Processed DRAMP00726: AMP (Probabilities: 0.3932, 0.4492, 0.1576)\n","Processed DRAMP00727: AMP (Probabilities: 0.0886, 0.8965, 0.0149)\n","Processed DRAMP00728: AMP (Probabilities: 0.4337, 0.5379, 0.0284)\n","Processed DRAMP00729: non-AMP/OTHERS (Probabilities: 0.9795, 0.0168, 0.0037)\n","Processed DRAMP00730: non-AMP/OTHERS (Probabilities: 0.7139, 0.2555, 0.0306)\n","Processed DRAMP00731: AMP (Probabilities: 0.4369, 0.5512, 0.0118)\n","Processed DRAMP00732: AMP (Probabilities: 0.0687, 0.9178, 0.0135)\n","Processed DRAMP00733: AMP (Probabilities: 0.0650, 0.9199, 0.0151)\n","Processed DRAMP00734: AMP (Probabilities: 0.0653, 0.9195, 0.0152)\n","Processed DRAMP00735: AMP (Probabilities: 0.0672, 0.9182, 0.0147)\n","Processed DRAMP00736: AMP (Probabilities: 0.0653, 0.9192, 0.0154)\n","Processed DRAMP00738: AMP (Probabilities: 0.0673, 0.9186, 0.0142)\n","Processed DRAMP00739: AMP (Probabilities: 0.0694, 0.9158, 0.0149)\n","Processed DRAMP00740: non-AMP/OTHERS (Probabilities: 0.9846, 0.0057, 0.0097)\n","Processed DRAMP00741: Neuropeptide (Probabilities: 0.0388, 0.0177, 0.9435)\n","Processed DRAMP00742: Neuropeptide (Probabilities: 0.1319, 0.0703, 0.7978)\n","Processed DRAMP00743: Neuropeptide (Probabilities: 0.3328, 0.1039, 0.5634)\n","Processed DRAMP00744: AMP (Probabilities: 0.0670, 0.9183, 0.0147)\n","Processed DRAMP00745: AMP (Probabilities: 0.0699, 0.9156, 0.0146)\n","Processed DRAMP00749: AMP (Probabilities: 0.0649, 0.9186, 0.0164)\n","Processed DRAMP00751: AMP (Probabilities: 0.0658, 0.9184, 0.0158)\n","Processed DRAMP00753: non-AMP/OTHERS (Probabilities: 0.8313, 0.0234, 0.1453)\n","Processed DRAMP00754: non-AMP/OTHERS (Probabilities: 0.9925, 0.0034, 0.0041)\n","Processed DRAMP00755: non-AMP/OTHERS (Probabilities: 0.9841, 0.0080, 0.0078)\n","Processed DRAMP00756: AMP (Probabilities: 0.1964, 0.7660, 0.0375)\n","Processed DRAMP00757: non-AMP/OTHERS (Probabilities: 0.9467, 0.0076, 0.0457)\n","Processed DRAMP00758: AMP (Probabilities: 0.1621, 0.8011, 0.0368)\n","Processed DRAMP00759: non-AMP/OTHERS (Probabilities: 0.9879, 0.0065, 0.0056)\n","Processed DRAMP00760: AMP (Probabilities: 0.0664, 0.9175, 0.0160)\n","Processed DRAMP00767: non-AMP/OTHERS (Probabilities: 0.9766, 0.0195, 0.0038)\n","Processed DRAMP00768: AMP (Probabilities: 0.1353, 0.8526, 0.0121)\n","Processed DRAMP00770: AMP (Probabilities: 0.0710, 0.9156, 0.0134)\n","Processed DRAMP00771: AMP (Probabilities: 0.0993, 0.8883, 0.0124)\n","Processed DRAMP00776: AMP (Probabilities: 0.1395, 0.8473, 0.0131)\n","Processed DRAMP00780: AMP (Probabilities: 0.1638, 0.8221, 0.0141)\n","Processed DRAMP18326: AMP (Probabilities: 0.2400, 0.5671, 0.1929)\n","Processed DRAMP00787: AMP (Probabilities: 0.0925, 0.8944, 0.0131)\n","Processed DRAMP00789: AMP (Probabilities: 0.1116, 0.8764, 0.0121)\n","Processed DRAMP00790: non-AMP/OTHERS (Probabilities: 0.6817, 0.3110, 0.0073)\n","Processed DRAMP00793: AMP (Probabilities: 0.0716, 0.9139, 0.0145)\n","Processed DRAMP00796: AMP (Probabilities: 0.0751, 0.9116, 0.0133)\n","Processed DRAMP00797: AMP (Probabilities: 0.3575, 0.6298, 0.0127)\n","Processed DRAMP00800: AMP (Probabilities: 0.3675, 0.4534, 0.1790)\n","Processed DRAMP00803: AMP (Probabilities: 0.0795, 0.9066, 0.0140)\n","Processed DRAMP00817: AMP (Probabilities: 0.1592, 0.8273, 0.0135)\n","Processed DRAMP00819: AMP (Probabilities: 0.0797, 0.9068, 0.0136)\n","Processed DRAMP00821: AMP (Probabilities: 0.1525, 0.8344, 0.0131)\n","Processed DRAMP00826: non-AMP/OTHERS (Probabilities: 0.9544, 0.0419, 0.0037)\n","Processed DRAMP18197: AMP (Probabilities: 0.0650, 0.9196, 0.0154)\n","Processed DRAMP00830: AMP (Probabilities: 0.0706, 0.9159, 0.0135)\n","Processed DRAMP00833: AMP (Probabilities: 0.1468, 0.8090, 0.0442)\n","Processed DRAMP00836: AMP (Probabilities: 0.4606, 0.4795, 0.0599)\n","Processed DRAMP00838: AMP (Probabilities: 0.0677, 0.9177, 0.0147)\n","Processed DRAMP00841: AMP (Probabilities: 0.1203, 0.8674, 0.0123)\n","Processed DRAMP00849: AMP (Probabilities: 0.0667, 0.9182, 0.0151)\n","Processed DRAMP00855: AMP (Probabilities: 0.0812, 0.9062, 0.0126)\n","Processed DRAMP00860: non-AMP/OTHERS (Probabilities: 0.9492, 0.0462, 0.0046)\n","Processed DRAMP00862: AMP (Probabilities: 0.1502, 0.8362, 0.0136)\n","Processed DRAMP00863: AMP (Probabilities: 0.0693, 0.9162, 0.0145)\n","Processed DRAMP00865: non-AMP/OTHERS (Probabilities: 0.5638, 0.0581, 0.3781)\n","Processed DRAMP00868: non-AMP/OTHERS (Probabilities: 0.9857, 0.0114, 0.0029)\n","Processed DRAMP00872: AMP (Probabilities: 0.1291, 0.8591, 0.0119)\n","Processed DRAMP00875: Neuropeptide (Probabilities: 0.2375, 0.3509, 0.4116)\n","Processed DRAMP00876: AMP (Probabilities: 0.2943, 0.6833, 0.0223)\n","Processed DRAMP00880: AMP (Probabilities: 0.0722, 0.9141, 0.0137)\n","Processed DRAMP00882: AMP (Probabilities: 0.0633, 0.9202, 0.0164)\n","Processed DRAMP00883: AMP (Probabilities: 0.0864, 0.9008, 0.0128)\n","Processed DRAMP00884: AMP (Probabilities: 0.0749, 0.9116, 0.0135)\n","Processed DRAMP00885: AMP (Probabilities: 0.0708, 0.9161, 0.0131)\n","Processed DRAMP00888: AMP (Probabilities: 0.2996, 0.3774, 0.3231)\n","Processed DRAMP00889: AMP (Probabilities: 0.0749, 0.9117, 0.0134)\n","Processed DRAMP00890: AMP (Probabilities: 0.1763, 0.7672, 0.0564)\n","Processed DRAMP00891: AMP (Probabilities: 0.0705, 0.9152, 0.0143)\n","Processed DRAMP00893: AMP (Probabilities: 0.0774, 0.9089, 0.0137)\n","Processed DRAMP00895: AMP (Probabilities: 0.0814, 0.9063, 0.0123)\n","Processed DRAMP00896: AMP (Probabilities: 0.4502, 0.4759, 0.0739)\n","Processed DRAMP00897: AMP (Probabilities: 0.0646, 0.9200, 0.0154)\n","Processed DRAMP00900: AMP (Probabilities: 0.0682, 0.9167, 0.0152)\n","Processed DRAMP00901: non-AMP/OTHERS (Probabilities: 0.9038, 0.0861, 0.0102)\n","Processed DRAMP00902: AMP (Probabilities: 0.2442, 0.4264, 0.3294)\n","Processed DRAMP00903: AMP (Probabilities: 0.0690, 0.9157, 0.0153)\n","Processed DRAMP00904: AMP (Probabilities: 0.0694, 0.9158, 0.0148)\n","Processed DRAMP00907: AMP (Probabilities: 0.0690, 0.9161, 0.0149)\n","Processed DRAMP00911: AMP (Probabilities: 0.0699, 0.9162, 0.0139)\n","Processed DRAMP00920: AMP (Probabilities: 0.0894, 0.8978, 0.0128)\n","Processed DRAMP00924: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed DRAMP00925: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed DRAMP00926: AMP (Probabilities: 0.1222, 0.8656, 0.0121)\n","Processed DRAMP00927: AMP (Probabilities: 0.1385, 0.8488, 0.0127)\n","Processed DRAMP00932: AMP (Probabilities: 0.0650, 0.9189, 0.0161)\n","Processed DRAMP00934: AMP (Probabilities: 0.0691, 0.9165, 0.0144)\n","Processed DRAMP00935: non-AMP/OTHERS (Probabilities: 0.9665, 0.0210, 0.0125)\n","Processed DRAMP00936: non-AMP/OTHERS (Probabilities: 0.9919, 0.0055, 0.0027)\n","Processed DRAMP00939: AMP (Probabilities: 0.0904, 0.8923, 0.0173)\n","Processed DRAMP00940: non-AMP/OTHERS (Probabilities: 0.8925, 0.0968, 0.0107)\n","Processed DRAMP00941: AMP (Probabilities: 0.0675, 0.9167, 0.0158)\n","Processed DRAMP00942: AMP (Probabilities: 0.0693, 0.9160, 0.0146)\n","Processed DRAMP00945: AMP (Probabilities: 0.0698, 0.9153, 0.0150)\n","Processed DRAMP00946: AMP (Probabilities: 0.0666, 0.9182, 0.0152)\n","Processed DRAMP00947: AMP (Probabilities: 0.0655, 0.9188, 0.0157)\n","Processed DRAMP00950: AMP (Probabilities: 0.0683, 0.9168, 0.0149)\n","Processed DRAMP00959: AMP (Probabilities: 0.0677, 0.9176, 0.0146)\n","Processed DRAMP00963: AMP (Probabilities: 0.0663, 0.9182, 0.0156)\n","Processed DRAMP00964: AMP (Probabilities: 0.1049, 0.8671, 0.0281)\n","Processed DRAMP00966: AMP (Probabilities: 0.0670, 0.9179, 0.0151)\n","Processed DRAMP00971: Neuropeptide (Probabilities: 0.0089, 0.0097, 0.9813)\n","Processed DRAMP00972: Neuropeptide (Probabilities: 0.0107, 0.0107, 0.9786)\n","Processed DRAMP00973: AMP (Probabilities: 0.0708, 0.9157, 0.0135)\n","Processed DRAMP00974: AMP (Probabilities: 0.0689, 0.9162, 0.0149)\n","Processed DRAMP00978: AMP (Probabilities: 0.0645, 0.9192, 0.0163)\n","Processed DRAMP00990: AMP (Probabilities: 0.0638, 0.9195, 0.0167)\n","Processed DRAMP00991: AMP (Probabilities: 0.0798, 0.9061, 0.0141)\n","Processed DRAMP00993: AMP (Probabilities: 0.2625, 0.7254, 0.0121)\n","Processed DRAMP01000: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP01001: non-AMP/OTHERS (Probabilities: 0.9928, 0.0030, 0.0042)\n","Processed DRAMP01002: non-AMP/OTHERS (Probabilities: 0.9888, 0.0043, 0.0069)\n","Processed DRAMP01005: non-AMP/OTHERS (Probabilities: 0.9856, 0.0050, 0.0094)\n","Processed DRAMP01006: non-AMP/OTHERS (Probabilities: 0.5049, 0.0303, 0.4649)\n","Processed DRAMP01007: AMP (Probabilities: 0.0772, 0.9102, 0.0126)\n","Processed DRAMP01008: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP01009: Neuropeptide (Probabilities: 0.0061, 0.0089, 0.9850)\n","Processed DRAMP01010: AMP (Probabilities: 0.1390, 0.8367, 0.0243)\n","Processed DRAMP01026: AMP (Probabilities: 0.1105, 0.8757, 0.0138)\n","Processed DRAMP01027: non-AMP/OTHERS (Probabilities: 0.8581, 0.1315, 0.0104)\n","Processed DRAMP01031: non-AMP/OTHERS (Probabilities: 0.9592, 0.0366, 0.0042)\n","Processed DRAMP01043: AMP (Probabilities: 0.0741, 0.9114, 0.0145)\n","Processed DRAMP01048: AMP (Probabilities: 0.0663, 0.9184, 0.0153)\n","Processed DRAMP01049: AMP (Probabilities: 0.0647, 0.9193, 0.0160)\n","Processed DRAMP01050: AMP (Probabilities: 0.0636, 0.9208, 0.0156)\n","Processed DRAMP01053: non-AMP/OTHERS (Probabilities: 0.9569, 0.0382, 0.0049)\n","Processed DRAMP01054: AMP (Probabilities: 0.0669, 0.9178, 0.0153)\n","Processed DRAMP01057: Neuropeptide (Probabilities: 0.0087, 0.0096, 0.9816)\n","Processed DRAMP01058: non-AMP/OTHERS (Probabilities: 0.5566, 0.3968, 0.0466)\n","Processed DRAMP01059: non-AMP/OTHERS (Probabilities: 0.9869, 0.0032, 0.0099)\n","Processed DRAMP01060: Neuropeptide (Probabilities: 0.0716, 0.0169, 0.9115)\n","Processed DRAMP01062: non-AMP/OTHERS (Probabilities: 0.9913, 0.0030, 0.0057)\n","Processed DRAMP01065: non-AMP/OTHERS (Probabilities: 0.9879, 0.0041, 0.0080)\n","Processed DRAMP01067: non-AMP/OTHERS (Probabilities: 0.9907, 0.0029, 0.0064)\n","Processed DRAMP01068: AMP (Probabilities: 0.1259, 0.8611, 0.0130)\n","Processed DRAMP01069: AMP (Probabilities: 0.0712, 0.9154, 0.0134)\n","Processed DRAMP01070: AMP (Probabilities: 0.1109, 0.8768, 0.0123)\n","Processed DRAMP01074: AMP (Probabilities: 0.0869, 0.9011, 0.0121)\n","Processed DRAMP01080: AMP (Probabilities: 0.0845, 0.9034, 0.0122)\n","Processed DRAMP01084: AMP (Probabilities: 0.0769, 0.9096, 0.0135)\n","Processed DRAMP01092: Neuropeptide (Probabilities: 0.0067, 0.0089, 0.9844)\n","Processed DRAMP01128: AMP (Probabilities: 0.0703, 0.9150, 0.0147)\n","Processed DRAMP01156: AMP (Probabilities: 0.1759, 0.8092, 0.0149)\n","Processed DRAMP01158: non-AMP/OTHERS (Probabilities: 0.8379, 0.0434, 0.1187)\n","Processed DRAMP01159: AMP (Probabilities: 0.0692, 0.9165, 0.0143)\n","Processed DRAMP01166: AMP (Probabilities: 0.0719, 0.9130, 0.0150)\n","Processed DRAMP01168: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9849)\n","Processed DRAMP01169: non-AMP/OTHERS (Probabilities: 0.9008, 0.0922, 0.0070)\n","Processed DRAMP01193: AMP (Probabilities: 0.0688, 0.9156, 0.0156)\n","Processed DRAMP01196: AMP (Probabilities: 0.0761, 0.9112, 0.0126)\n","Processed DRAMP01197: non-AMP/OTHERS (Probabilities: 0.9278, 0.0088, 0.0635)\n","Processed DRAMP01198: non-AMP/OTHERS (Probabilities: 0.9889, 0.0056, 0.0055)\n","Processed DRAMP01210: AMP (Probabilities: 0.2122, 0.7730, 0.0148)\n","Processed DRAMP01240: AMP (Probabilities: 0.0832, 0.9038, 0.0129)\n","Processed DRAMP01242: non-AMP/OTHERS (Probabilities: 0.9536, 0.0417, 0.0047)\n","Processed DRAMP01247: AMP (Probabilities: 0.1807, 0.8066, 0.0127)\n","Processed DRAMP01266: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP01268: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP01269: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed DRAMP01270: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP18403: AMP (Probabilities: 0.1193, 0.8658, 0.0150)\n","Processed DRAMP01279: non-AMP/OTHERS (Probabilities: 0.9798, 0.0174, 0.0028)\n","Processed DRAMP01280: non-AMP/OTHERS (Probabilities: 0.8073, 0.1837, 0.0090)\n","Processed DRAMP01285: AMP (Probabilities: 0.1318, 0.8551, 0.0131)\n","Processed DRAMP01294: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed DRAMP01295: AMP (Probabilities: 0.0731, 0.9134, 0.0135)\n","Processed DRAMP01299: AMP (Probabilities: 0.1506, 0.8202, 0.0292)\n","Processed DRAMP01300: AMP (Probabilities: 0.2207, 0.7676, 0.0117)\n","Processed DRAMP01316: non-AMP/OTHERS (Probabilities: 0.5900, 0.3992, 0.0108)\n","Processed DRAMP01323: AMP (Probabilities: 0.4714, 0.5182, 0.0103)\n","Processed DRAMP01329: non-AMP/OTHERS (Probabilities: 0.9891, 0.0080, 0.0029)\n","Processed DRAMP01330: non-AMP/OTHERS (Probabilities: 0.9807, 0.0141, 0.0052)\n","Processed DRAMP01332: AMP (Probabilities: 0.1418, 0.8436, 0.0146)\n","Processed DRAMP01334: AMP (Probabilities: 0.0724, 0.9140, 0.0136)\n","Processed DRAMP01336: AMP (Probabilities: 0.0707, 0.9144, 0.0149)\n","Processed DRAMP01343: AMP (Probabilities: 0.0720, 0.9137, 0.0143)\n","Processed DRAMP01349: AMP (Probabilities: 0.1271, 0.8463, 0.0266)\n","Processed DRAMP01388: Neuropeptide (Probabilities: 0.0061, 0.0092, 0.9847)\n","Processed DRAMP01363: AMP (Probabilities: 0.0739, 0.9121, 0.0140)\n","Processed DRAMP01379: AMP (Probabilities: 0.0706, 0.9132, 0.0161)\n","Processed DRAMP01382: non-AMP/OTHERS (Probabilities: 0.8038, 0.1274, 0.0688)\n","Processed DRAMP01385: AMP (Probabilities: 0.3413, 0.6146, 0.0441)\n","Processed DRAMP01404: AMP (Probabilities: 0.1096, 0.8778, 0.0126)\n","Processed DRAMP01407: AMP (Probabilities: 0.1425, 0.8452, 0.0123)\n","Processed DRAMP18317: AMP (Probabilities: 0.0711, 0.9152, 0.0136)\n","Processed DRAMP01446: AMP (Probabilities: 0.0740, 0.9122, 0.0138)\n","Processed DRAMP18316: AMP (Probabilities: 0.0691, 0.9161, 0.0148)\n","Processed DRAMP18315: AMP (Probabilities: 0.0662, 0.9180, 0.0159)\n","Processed DRAMP18314: AMP (Probabilities: 0.0671, 0.9174, 0.0155)\n","Processed DRAMP01481: AMP (Probabilities: 0.0776, 0.9080, 0.0144)\n","Processed DRAMP18313: non-AMP/OTHERS (Probabilities: 0.9906, 0.0070, 0.0024)\n","Processed DRAMP18312: non-AMP/OTHERS (Probabilities: 0.9813, 0.0157, 0.0030)\n","Processed DRAMP01527: non-AMP/OTHERS (Probabilities: 0.9007, 0.0923, 0.0070)\n","Processed DRAMP01535: AMP (Probabilities: 0.0717, 0.9141, 0.0142)\n","Processed DRAMP01537: non-AMP/OTHERS (Probabilities: 0.9530, 0.0423, 0.0047)\n","Processed DRAMP01541: non-AMP/OTHERS (Probabilities: 0.6023, 0.0371, 0.3606)\n","Processed DRAMP01544: AMP (Probabilities: 0.1231, 0.8563, 0.0206)\n","Processed DRAMP01545: AMP (Probabilities: 0.1466, 0.8320, 0.0215)\n","Processed DRAMP01548: non-AMP/OTHERS (Probabilities: 0.9917, 0.0057, 0.0026)\n","Processed DRAMP01554: Neuropeptide (Probabilities: 0.0064, 0.0093, 0.9843)\n","Processed DRAMP01580: AMP (Probabilities: 0.0841, 0.9026, 0.0133)\n","Processed DRAMP01582: AMP (Probabilities: 0.0788, 0.9076, 0.0136)\n","Processed DRAMP18310: AMP (Probabilities: 0.0681, 0.9174, 0.0145)\n","Processed DRAMP18311: AMP (Probabilities: 0.0815, 0.9051, 0.0134)\n","Processed DRAMP01605: AMP (Probabilities: 0.0731, 0.9128, 0.0141)\n","Processed DRAMP18309: Neuropeptide (Probabilities: 0.0058, 0.0094, 0.9848)\n","Processed DRAMP18308: Neuropeptide (Probabilities: 0.3260, 0.0438, 0.6302)\n","Processed DRAMP01630: AMP (Probabilities: 0.0689, 0.9160, 0.0151)\n","Processed DRAMP01633: Neuropeptide (Probabilities: 0.2244, 0.0614, 0.7142)\n","Processed DRAMP01634: AMP (Probabilities: 0.3960, 0.5925, 0.0115)\n","Processed DRAMP01635: Neuropeptide (Probabilities: 0.3592, 0.1337, 0.5071)\n","Processed DRAMP01636: AMP (Probabilities: 0.1621, 0.8153, 0.0226)\n","Processed DRAMP01642: AMP (Probabilities: 0.0689, 0.9162, 0.0148)\n","Processed DRAMP01656: AMP (Probabilities: 0.0673, 0.9175, 0.0153)\n","Processed DRAMP01667: AMP (Probabilities: 0.0682, 0.9167, 0.0151)\n","Processed DRAMP18306: Neuropeptide (Probabilities: 0.4082, 0.0840, 0.5078)\n","Processed DRAMP01680: AMP (Probabilities: 0.0699, 0.9149, 0.0153)\n","Processed DRAMP01686: AMP (Probabilities: 0.0711, 0.9138, 0.0151)\n","Processed DRAMP01689: AMP (Probabilities: 0.0714, 0.9139, 0.0148)\n","Processed DRAMP01690: AMP (Probabilities: 0.0677, 0.9165, 0.0157)\n","Processed DRAMP01692: AMP (Probabilities: 0.0696, 0.9154, 0.0150)\n","Processed DRAMP01693: AMP (Probabilities: 0.0708, 0.9143, 0.0149)\n","Processed DRAMP01695: AMP (Probabilities: 0.1636, 0.8223, 0.0140)\n","Processed DRAMP01696: AMP (Probabilities: 0.1451, 0.8421, 0.0128)\n","Processed DRAMP01699: AMP (Probabilities: 0.0703, 0.9152, 0.0145)\n","Processed DRAMP01705: AMP (Probabilities: 0.0716, 0.9138, 0.0147)\n","Processed DRAMP01706: AMP (Probabilities: 0.1696, 0.8162, 0.0142)\n","Processed DRAMP01708: AMP (Probabilities: 0.0718, 0.9135, 0.0147)\n","Processed DRAMP01709: AMP (Probabilities: 0.0714, 0.9133, 0.0153)\n","Processed DRAMP01713: AMP (Probabilities: 0.2237, 0.7476, 0.0287)\n","Processed DRAMP01714: non-AMP/OTHERS (Probabilities: 0.6447, 0.3444, 0.0109)\n","Processed DRAMP02819: AMP (Probabilities: 0.0678, 0.9171, 0.0152)\n","Processed DRAMP04395: Neuropeptide (Probabilities: 0.0058, 0.0094, 0.9848)\n","Processed DRAMP02396: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed DRAMP01757: AMP (Probabilities: 0.1573, 0.8250, 0.0177)\n","Processed DRAMP01767: AMP (Probabilities: 0.0665, 0.9182, 0.0153)\n","Processed DRAMP01770: AMP (Probabilities: 0.0722, 0.9135, 0.0143)\n","Processed DRAMP18302: AMP (Probabilities: 0.2064, 0.6746, 0.1190)\n","Processed DRAMP01792: AMP (Probabilities: 0.1450, 0.8416, 0.0134)\n","Processed DRAMP01794: AMP (Probabilities: 0.0677, 0.9168, 0.0155)\n","Processed DRAMP01805: AMP (Probabilities: 0.0692, 0.9157, 0.0151)\n","Processed DRAMP18301: non-AMP/OTHERS (Probabilities: 0.5735, 0.3838, 0.0427)\n","Processed DRAMP18298: AMP (Probabilities: 0.0824, 0.9049, 0.0127)\n","Processed DRAMP18297: AMP (Probabilities: 0.0948, 0.8926, 0.0127)\n","Processed DRAMP18296: AMP (Probabilities: 0.1630, 0.8251, 0.0119)\n","Processed DRAMP01854: AMP (Probabilities: 0.0708, 0.9144, 0.0148)\n","Processed DRAMP18295: AMP (Probabilities: 0.1588, 0.8274, 0.0138)\n","Processed DRAMP01860: AMP (Probabilities: 0.1686, 0.8146, 0.0169)\n","Processed DRAMP01861: AMP (Probabilities: 0.1578, 0.8284, 0.0137)\n","Processed DRAMP01864: AMP (Probabilities: 0.0730, 0.9122, 0.0147)\n","Processed DRAMP01865: non-AMP/OTHERS (Probabilities: 0.9831, 0.0138, 0.0030)\n","Processed DRAMP01868: AMP (Probabilities: 0.2769, 0.7070, 0.0161)\n","Processed DRAMP01871: AMP (Probabilities: 0.1534, 0.8324, 0.0142)\n","Processed DRAMP01882: AMP (Probabilities: 0.0949, 0.8927, 0.0124)\n","Processed DRAMP01883: AMP (Probabilities: 0.1057, 0.8806, 0.0137)\n","Processed DRAMP01884: AMP (Probabilities: 0.0859, 0.8960, 0.0181)\n","Processed DRAMP01917: AMP (Probabilities: 0.0744, 0.9110, 0.0146)\n","Processed DRAMP01923: AMP (Probabilities: 0.0759, 0.9101, 0.0140)\n","Processed DRAMP01936: AMP (Probabilities: 0.0804, 0.9065, 0.0130)\n","Processed DRAMP01945: AMP (Probabilities: 0.0710, 0.9144, 0.0146)\n","Processed DRAMP18294: non-AMP/OTHERS (Probabilities: 0.6679, 0.3237, 0.0084)\n","Processed DRAMP01979: AMP (Probabilities: 0.0700, 0.9146, 0.0154)\n","Processed DRAMP01980: AMP (Probabilities: 0.0716, 0.9135, 0.0149)\n","Processed DRAMP18291: AMP (Probabilities: 0.0687, 0.9167, 0.0145)\n","Processed DRAMP01984: AMP (Probabilities: 0.0955, 0.8916, 0.0130)\n","Processed DRAMP01993: AMP (Probabilities: 0.1595, 0.8262, 0.0143)\n","Processed DRAMP18287: non-AMP/OTHERS (Probabilities: 0.7495, 0.2354, 0.0152)\n","Processed DRAMP02002: AMP (Probabilities: 0.0810, 0.9055, 0.0136)\n","Processed DRAMP02003: AMP (Probabilities: 0.0737, 0.9118, 0.0145)\n","Processed DRAMP02017: AMP (Probabilities: 0.0717, 0.9134, 0.0149)\n","Processed DRAMP18282: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9850)\n","Processed DRAMP02079: AMP (Probabilities: 0.1348, 0.8524, 0.0127)\n","Processed DRAMP02080: AMP (Probabilities: 0.1797, 0.8059, 0.0144)\n","Processed DRAMP18278: AMP (Probabilities: 0.0682, 0.9181, 0.0137)\n","Processed DRAMP02123: AMP (Probabilities: 0.0701, 0.9148, 0.0151)\n","Processed DRAMP02138: AMP (Probabilities: 0.0689, 0.9152, 0.0159)\n","Processed DRAMP02139: AMP (Probabilities: 0.0711, 0.9134, 0.0154)\n","Processed DRAMP02145: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed DRAMP02146: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed DRAMP02147: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP02148: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9850)\n","Processed DRAMP02149: non-AMP/OTHERS (Probabilities: 0.9469, 0.0063, 0.0469)\n","Processed DRAMP02151: AMP (Probabilities: 0.4195, 0.5692, 0.0113)\n","Processed DRAMP02152: non-AMP/OTHERS (Probabilities: 0.6198, 0.3707, 0.0095)\n","Processed DRAMP02154: AMP (Probabilities: 0.0686, 0.9156, 0.0157)\n","Processed DRAMP02155: non-AMP/OTHERS (Probabilities: 0.9034, 0.0099, 0.0867)\n","Processed DRAMP02158: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed DRAMP02159: AMP (Probabilities: 0.0677, 0.9169, 0.0154)\n","Processed DRAMP02160: Neuropeptide (Probabilities: 0.0173, 0.0134, 0.9694)\n","Processed DRAMP02161: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed DRAMP02165: Neuropeptide (Probabilities: 0.1080, 0.0694, 0.8225)\n","Processed DRAMP02166: non-AMP/OTHERS (Probabilities: 0.9239, 0.0137, 0.0623)\n","Processed DRAMP02167: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed DRAMP02168: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed DRAMP02171: Neuropeptide (Probabilities: 0.0058, 0.0094, 0.9848)\n","Processed DRAMP02172: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed DRAMP02173: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed DRAMP02175: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed DRAMP02176: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9852)\n","Processed DRAMP02179: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed DRAMP02182: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed DRAMP02184: Neuropeptide (Probabilities: 0.0062, 0.0087, 0.9851)\n","Processed DRAMP02185: Neuropeptide (Probabilities: 0.0083, 0.0105, 0.9812)\n","Processed DRAMP02186: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9850)\n","Processed DRAMP02188: AMP (Probabilities: 0.0721, 0.9136, 0.0144)\n","Processed DRAMP02192: AMP (Probabilities: 0.0757, 0.9112, 0.0131)\n","Processed DRAMP02195: AMP (Probabilities: 0.0694, 0.9156, 0.0150)\n","Processed DRAMP02197: AMP (Probabilities: 0.0697, 0.9161, 0.0142)\n","Processed DRAMP02200: AMP (Probabilities: 0.1412, 0.8469, 0.0119)\n","Processed DRAMP02201: AMP (Probabilities: 0.0955, 0.8922, 0.0122)\n","Processed DRAMP02205: AMP (Probabilities: 0.1605, 0.8263, 0.0131)\n","Processed DRAMP02206: AMP (Probabilities: 0.0718, 0.9148, 0.0134)\n","Processed DRAMP02207: AMP (Probabilities: 0.0780, 0.9089, 0.0131)\n","Processed DRAMP02208: AMP (Probabilities: 0.1478, 0.8399, 0.0124)\n","Processed DRAMP02211: AMP (Probabilities: 0.1190, 0.8686, 0.0124)\n","Processed DRAMP02212: AMP (Probabilities: 0.3186, 0.6649, 0.0165)\n","Processed DRAMP02213: AMP (Probabilities: 0.0967, 0.8905, 0.0128)\n","Processed DRAMP02217: AMP (Probabilities: 0.1589, 0.8278, 0.0133)\n","Processed DRAMP01384: Neuropeptide (Probabilities: 0.0361, 0.0248, 0.9391)\n","Processed DRAMP02260: AMP (Probabilities: 0.0675, 0.9165, 0.0160)\n","Processed DRAMP02261: AMP (Probabilities: 0.0657, 0.9185, 0.0158)\n","Processed DRAMP02266: AMP (Probabilities: 0.0726, 0.9140, 0.0133)\n","Processed DRAMP02267: AMP (Probabilities: 0.0692, 0.9158, 0.0150)\n","Processed DRAMP02297: non-AMP/OTHERS (Probabilities: 0.9381, 0.0565, 0.0054)\n","Processed DRAMP02301: non-AMP/OTHERS (Probabilities: 0.9264, 0.0639, 0.0097)\n","Processed DRAMP02302: AMP (Probabilities: 0.0721, 0.9139, 0.0140)\n","Processed DRAMP02311: non-AMP/OTHERS (Probabilities: 0.4002, 0.3091, 0.2907)\n","Processed DRAMP02313: AMP (Probabilities: 0.0727, 0.9135, 0.0138)\n","Processed DRAMP02319: AMP (Probabilities: 0.0675, 0.9169, 0.0156)\n","Processed DRAMP18268: AMP (Probabilities: 0.0679, 0.9167, 0.0155)\n","Processed DRAMP02334: AMP (Probabilities: 0.0735, 0.9126, 0.0139)\n","Processed DRAMP02338: AMP (Probabilities: 0.0703, 0.9148, 0.0150)\n","Processed DRAMP02340: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed DRAMP02343: non-AMP/OTHERS (Probabilities: 0.9687, 0.0190, 0.0123)\n","Processed DRAMP02345: AMP (Probabilities: 0.2689, 0.4972, 0.2339)\n","Processed DRAMP02346: AMP (Probabilities: 0.2337, 0.6691, 0.0972)\n","Processed DRAMP02353: non-AMP/OTHERS (Probabilities: 0.9888, 0.0069, 0.0043)\n","Processed DRAMP18267: non-AMP/OTHERS (Probabilities: 0.4887, 0.0300, 0.4813)\n","Processed DRAMP02369: AMP (Probabilities: 0.2247, 0.7479, 0.0275)\n","Processed DRAMP02370: AMP (Probabilities: 0.0720, 0.9150, 0.0130)\n","Processed DRAMP02371: AMP (Probabilities: 0.1031, 0.8821, 0.0147)\n","Processed DRAMP02399: Neuropeptide (Probabilities: 0.0101, 0.0097, 0.9801)\n","Processed DRAMP02400: non-AMP/OTHERS (Probabilities: 0.5040, 0.4768, 0.0192)\n","Processed DRAMP02404: AMP (Probabilities: 0.2167, 0.7648, 0.0185)\n","Processed DRAMP02405: Neuropeptide (Probabilities: 0.0146, 0.0121, 0.9733)\n","Processed DRAMP02406: Neuropeptide (Probabilities: 0.2272, 0.3576, 0.4152)\n","Processed DRAMP02407: non-AMP/OTHERS (Probabilities: 0.6216, 0.2873, 0.0911)\n","Processed DRAMP02408: AMP (Probabilities: 0.0870, 0.9011, 0.0119)\n","Processed DRAMP02417: AMP (Probabilities: 0.0688, 0.9167, 0.0145)\n","Processed DRAMP02431: AMP (Probabilities: 0.0861, 0.9017, 0.0122)\n","Processed DRAMP02435: AMP (Probabilities: 0.0682, 0.9168, 0.0150)\n","Processed DRAMP02436: non-AMP/OTHERS (Probabilities: 0.9856, 0.0054, 0.0090)\n","Processed DRAMP02440: AMP (Probabilities: 0.0679, 0.9185, 0.0136)\n","Processed DRAMP18264: AMP (Probabilities: 0.0764, 0.9106, 0.0130)\n","Processed DRAMP02448: non-AMP/OTHERS (Probabilities: 0.5262, 0.1897, 0.2841)\n","Processed DRAMP02450: Neuropeptide (Probabilities: 0.0374, 0.0259, 0.9367)\n","Processed DRAMP02451: AMP (Probabilities: 0.1879, 0.8006, 0.0115)\n","Processed DRAMP02452: non-AMP/OTHERS (Probabilities: 0.8680, 0.1241, 0.0079)\n","Processed DRAMP02455: non-AMP/OTHERS (Probabilities: 0.9898, 0.0035, 0.0067)\n","Processed DRAMP02457: non-AMP/OTHERS (Probabilities: 0.9908, 0.0037, 0.0054)\n","Processed DRAMP02458: non-AMP/OTHERS (Probabilities: 0.9915, 0.0042, 0.0043)\n","Processed DRAMP02462: Neuropeptide (Probabilities: 0.0381, 0.0186, 0.9433)\n","Processed DRAMP02464: non-AMP/OTHERS (Probabilities: 0.9387, 0.0316, 0.0297)\n","Processed DRAMP02465: non-AMP/OTHERS (Probabilities: 0.9859, 0.0050, 0.0091)\n","Processed DRAMP02467: non-AMP/OTHERS (Probabilities: 0.9849, 0.0085, 0.0066)\n","Processed DRAMP02468: Neuropeptide (Probabilities: 0.0063, 0.0087, 0.9849)\n","Processed DRAMP02469: non-AMP/OTHERS (Probabilities: 0.7927, 0.1937, 0.0135)\n","Processed DRAMP02471: AMP (Probabilities: 0.1316, 0.8546, 0.0138)\n","Processed DRAMP02472: non-AMP/OTHERS (Probabilities: 0.9918, 0.0035, 0.0047)\n","Processed DRAMP02475: AMP (Probabilities: 0.3174, 0.6383, 0.0443)\n","Processed DRAMP02476: AMP (Probabilities: 0.0654, 0.9202, 0.0145)\n","Processed DRAMP02477: AMP (Probabilities: 0.1342, 0.8397, 0.0261)\n","Processed DRAMP02479: AMP (Probabilities: 0.0668, 0.9189, 0.0143)\n","Processed DRAMP02484: non-AMP/OTHERS (Probabilities: 0.7088, 0.1796, 0.1115)\n","Processed DRAMP02485: AMP (Probabilities: 0.0948, 0.8800, 0.0252)\n","Processed DRAMP02486: AMP (Probabilities: 0.0709, 0.9163, 0.0129)\n","Processed DRAMP02487: AMP (Probabilities: 0.0651, 0.9189, 0.0160)\n","Processed DRAMP18263: AMP (Probabilities: 0.0687, 0.9168, 0.0145)\n","Processed DRAMP02512: non-AMP/OTHERS (Probabilities: 0.9931, 0.0037, 0.0032)\n","Processed DRAMP02523: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0036)\n","Processed DRAMP02524: non-AMP/OTHERS (Probabilities: 0.9929, 0.0037, 0.0035)\n","Processed DRAMP02525: non-AMP/OTHERS (Probabilities: 0.9920, 0.0046, 0.0034)\n","Processed DRAMP02526: non-AMP/OTHERS (Probabilities: 0.9919, 0.0041, 0.0040)\n","Processed DRAMP02527: non-AMP/OTHERS (Probabilities: 0.9927, 0.0041, 0.0033)\n","Processed DRAMP02528: non-AMP/OTHERS (Probabilities: 0.9925, 0.0039, 0.0036)\n","Processed DRAMP02529: non-AMP/OTHERS (Probabilities: 0.9925, 0.0038, 0.0037)\n","Processed DRAMP02530: non-AMP/OTHERS (Probabilities: 0.9918, 0.0044, 0.0038)\n","Processed DRAMP02531: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0036)\n","Processed DRAMP02532: non-AMP/OTHERS (Probabilities: 0.9927, 0.0040, 0.0034)\n","Processed DRAMP02533: non-AMP/OTHERS (Probabilities: 0.9926, 0.0037, 0.0037)\n","Processed DRAMP02534: non-AMP/OTHERS (Probabilities: 0.9929, 0.0038, 0.0033)\n","Processed DRAMP02535: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0035)\n","Processed DRAMP02536: non-AMP/OTHERS (Probabilities: 0.9923, 0.0036, 0.0041)\n","Processed DRAMP02537: non-AMP/OTHERS (Probabilities: 0.9923, 0.0039, 0.0038)\n","Processed DRAMP02538: non-AMP/OTHERS (Probabilities: 0.9925, 0.0038, 0.0038)\n","Processed DRAMP02539: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed DRAMP02540: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0038)\n","Processed DRAMP02541: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed DRAMP02542: non-AMP/OTHERS (Probabilities: 0.9924, 0.0036, 0.0040)\n","Processed DRAMP02543: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed DRAMP02544: non-AMP/OTHERS (Probabilities: 0.9923, 0.0039, 0.0038)\n","Processed DRAMP02545: non-AMP/OTHERS (Probabilities: 0.9929, 0.0036, 0.0035)\n","Processed DRAMP02546: non-AMP/OTHERS (Probabilities: 0.9919, 0.0038, 0.0043)\n","Processed DRAMP02547: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed DRAMP02548: non-AMP/OTHERS (Probabilities: 0.9922, 0.0039, 0.0039)\n","Processed DRAMP02549: non-AMP/OTHERS (Probabilities: 0.9922, 0.0047, 0.0031)\n","Processed DRAMP02550: non-AMP/OTHERS (Probabilities: 0.9927, 0.0037, 0.0036)\n","Processed DRAMP02551: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0034)\n","Processed DRAMP02552: non-AMP/OTHERS (Probabilities: 0.9927, 0.0038, 0.0034)\n","Processed DRAMP02553: non-AMP/OTHERS (Probabilities: 0.9925, 0.0039, 0.0036)\n","Processed DRAMP02554: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed DRAMP02555: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0036)\n","Processed DRAMP02556: non-AMP/OTHERS (Probabilities: 0.9921, 0.0040, 0.0039)\n","Processed DRAMP02558: non-AMP/OTHERS (Probabilities: 0.9920, 0.0043, 0.0036)\n","Processed DRAMP02559: non-AMP/OTHERS (Probabilities: 0.9920, 0.0039, 0.0040)\n","Processed DRAMP02560: non-AMP/OTHERS (Probabilities: 0.9923, 0.0040, 0.0038)\n","Processed DRAMP02561: non-AMP/OTHERS (Probabilities: 0.9918, 0.0041, 0.0041)\n","Processed DRAMP02562: non-AMP/OTHERS (Probabilities: 0.9923, 0.0041, 0.0036)\n","Processed DRAMP02563: non-AMP/OTHERS (Probabilities: 0.9922, 0.0041, 0.0037)\n","Processed DRAMP02564: non-AMP/OTHERS (Probabilities: 0.9926, 0.0040, 0.0034)\n","Processed DRAMP02565: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed DRAMP18253: AMP (Probabilities: 0.0861, 0.8998, 0.0141)\n","Processed DRAMP02567: non-AMP/OTHERS (Probabilities: 0.9926, 0.0037, 0.0037)\n","Processed DRAMP02568: AMP (Probabilities: 0.0661, 0.9182, 0.0158)\n","Processed DRAMP02569: AMP (Probabilities: 0.0674, 0.9185, 0.0141)\n","Processed DRAMP18252: AMP (Probabilities: 0.0723, 0.9141, 0.0136)\n","Processed DRAMP02607: AMP (Probabilities: 0.0733, 0.9126, 0.0141)\n","Processed DRAMP02610: non-AMP/OTHERS (Probabilities: 0.4189, 0.3242, 0.2568)\n","Processed DRAMP02616: AMP (Probabilities: 0.0701, 0.9153, 0.0146)\n","Processed DRAMP02617: AMP (Probabilities: 0.0695, 0.9167, 0.0138)\n","Processed DRAMP02618: AMP (Probabilities: 0.0720, 0.9133, 0.0147)\n","Processed DRAMP02619: AMP (Probabilities: 0.0667, 0.9195, 0.0139)\n","Processed DRAMP02620: AMP (Probabilities: 0.0706, 0.9155, 0.0140)\n","Processed DRAMP18248: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed DRAMP18249: non-AMP/OTHERS (Probabilities: 0.9929, 0.0031, 0.0040)\n","Processed DRAMP02632: AMP (Probabilities: 0.0678, 0.9178, 0.0143)\n","Processed DRAMP02633: AMP (Probabilities: 0.0674, 0.9178, 0.0149)\n","Processed DRAMP02634: AMP (Probabilities: 0.0748, 0.9116, 0.0137)\n","Processed DRAMP18246: non-AMP/OTHERS (Probabilities: 0.4903, 0.4744, 0.0353)\n","Processed DRAMP18247: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed DRAMP02639: Neuropeptide (Probabilities: 0.0641, 0.1307, 0.8052)\n","Processed DRAMP02649: non-AMP/OTHERS (Probabilities: 0.9923, 0.0036, 0.0041)\n","Processed DRAMP02651: AMP (Probabilities: 0.0714, 0.9144, 0.0141)\n","Processed DRAMP02655: non-AMP/OTHERS (Probabilities: 0.9885, 0.0075, 0.0040)\n","Processed DRAMP02656: AMP (Probabilities: 0.0948, 0.8914, 0.0138)\n","Processed DRAMP02665: non-AMP/OTHERS (Probabilities: 0.4123, 0.3070, 0.2807)\n","Processed DRAMP02668: AMP (Probabilities: 0.0670, 0.9177, 0.0153)\n","Processed DRAMP02678: non-AMP/OTHERS (Probabilities: 0.4919, 0.4045, 0.1036)\n","Processed DRAMP02679: AMP (Probabilities: 0.0663, 0.9184, 0.0153)\n","Processed DRAMP02680: AMP (Probabilities: 0.0661, 0.9186, 0.0153)\n","Processed DRAMP02681: AMP (Probabilities: 0.0675, 0.9179, 0.0146)\n","Processed DRAMP02682: Neuropeptide (Probabilities: 0.2977, 0.1194, 0.5828)\n","Processed DRAMP02684: AMP (Probabilities: 0.0655, 0.9188, 0.0157)\n","Processed DRAMP02685: AMP (Probabilities: 0.0672, 0.9177, 0.0151)\n","Processed DRAMP02687: Neuropeptide (Probabilities: 0.1472, 0.2601, 0.5928)\n","Processed DRAMP02688: AMP (Probabilities: 0.0673, 0.9177, 0.0150)\n","Processed DRAMP02694: AMP (Probabilities: 0.0831, 0.9039, 0.0130)\n","Processed DRAMP02695: AMP (Probabilities: 0.0664, 0.9193, 0.0143)\n","Processed DRAMP02697: Neuropeptide (Probabilities: 0.2336, 0.2720, 0.4944)\n","Processed DRAMP18242: Neuropeptide (Probabilities: 0.0693, 0.0263, 0.9044)\n","Processed DRAMP18240: Neuropeptide (Probabilities: 0.1106, 0.0341, 0.8554)\n","Processed DRAMP18239: Neuropeptide (Probabilities: 0.0504, 0.0240, 0.9256)\n","Processed DRAMP02732: AMP (Probabilities: 0.0708, 0.9147, 0.0145)\n","Processed DRAMP18236: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP02737: AMP (Probabilities: 0.0687, 0.9167, 0.0147)\n","Processed DRAMP02743: AMP (Probabilities: 0.0748, 0.9112, 0.0140)\n","Processed DRAMP02744: AMP (Probabilities: 0.0669, 0.9173, 0.0158)\n","Processed DRAMP02746: AMP (Probabilities: 0.0712, 0.9144, 0.0144)\n","Processed DRAMP02747: AMP (Probabilities: 0.0690, 0.9161, 0.0149)\n","Processed DRAMP02748: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed DRAMP02750: AMP (Probabilities: 0.0670, 0.9183, 0.0147)\n","Processed DRAMP02769: AMP (Probabilities: 0.0717, 0.9134, 0.0149)\n","Processed DRAMP02782: AMP (Probabilities: 0.0694, 0.9153, 0.0153)\n","Processed DRAMP02788: Neuropeptide (Probabilities: 0.2118, 0.0266, 0.7616)\n","Processed DRAMP02790: AMP (Probabilities: 0.0799, 0.9074, 0.0127)\n","Processed DRAMP02792: AMP (Probabilities: 0.1253, 0.8346, 0.0402)\n","Processed DRAMP02793: AMP (Probabilities: 0.0696, 0.9160, 0.0144)\n","Processed DRAMP02794: AMP (Probabilities: 0.0687, 0.9169, 0.0145)\n","Processed DRAMP02795: AMP (Probabilities: 0.0728, 0.9139, 0.0133)\n","Processed DRAMP02796: AMP (Probabilities: 0.0720, 0.9142, 0.0139)\n","Processed DRAMP02797: AMP (Probabilities: 0.0695, 0.9162, 0.0143)\n","Processed DRAMP02798: AMP (Probabilities: 0.0792, 0.9087, 0.0122)\n","Processed DRAMP02801: Neuropeptide (Probabilities: 0.0682, 0.0362, 0.8956)\n","Processed DRAMP02807: AMP (Probabilities: 0.0679, 0.9162, 0.0159)\n","Processed DRAMP02810: AMP (Probabilities: 0.0640, 0.9202, 0.0158)\n","Processed DRAMP02812: non-AMP/OTHERS (Probabilities: 0.9850, 0.0042, 0.0108)\n","Processed DRAMP02821: AMP (Probabilities: 0.0682, 0.9179, 0.0139)\n","Processed DRAMP02822: AMP (Probabilities: 0.0708, 0.9155, 0.0137)\n","Processed DRAMP02831: AMP (Probabilities: 0.1560, 0.8306, 0.0134)\n","Processed DRAMP02833: non-AMP/OTHERS (Probabilities: 0.9778, 0.0145, 0.0077)\n","Processed DRAMP02836: non-AMP/OTHERS (Probabilities: 0.7117, 0.0372, 0.2511)\n","Processed DRAMP02839: AMP (Probabilities: 0.0696, 0.9155, 0.0149)\n","Processed DRAMP02847: AMP (Probabilities: 0.0705, 0.9153, 0.0142)\n","Processed DRAMP02871: AMP (Probabilities: 0.0672, 0.9180, 0.0148)\n","Processed DRAMP02883: AMP (Probabilities: 0.0650, 0.9196, 0.0154)\n","Processed DRAMP02885: AMP (Probabilities: 0.0703, 0.9158, 0.0140)\n","Processed DRAMP02887: AMP (Probabilities: 0.0676, 0.9176, 0.0148)\n","Processed DRAMP02888: AMP (Probabilities: 0.0994, 0.8885, 0.0122)\n","Processed DRAMP02889: AMP (Probabilities: 0.0715, 0.9152, 0.0134)\n","Processed DRAMP02890: AMP (Probabilities: 0.0857, 0.9020, 0.0123)\n","Processed DRAMP02891: AMP (Probabilities: 0.0859, 0.9013, 0.0128)\n","Processed DRAMP02892: AMP (Probabilities: 0.1919, 0.7882, 0.0200)\n","Processed DRAMP02893: AMP (Probabilities: 0.1426, 0.8391, 0.0183)\n","Processed DRAMP02894: non-AMP/OTHERS (Probabilities: 0.6263, 0.3484, 0.0253)\n","Processed DRAMP02895: AMP (Probabilities: 0.1768, 0.7998, 0.0234)\n","Processed DRAMP02898: non-AMP/OTHERS (Probabilities: 0.9275, 0.0680, 0.0046)\n","Processed DRAMP02904: non-AMP/OTHERS (Probabilities: 0.6249, 0.3611, 0.0140)\n","Processed DRAMP02909: AMP (Probabilities: 0.0782, 0.9086, 0.0132)\n","Processed DRAMP02917: AMP (Probabilities: 0.1320, 0.8558, 0.0123)\n","Processed DRAMP02918: AMP (Probabilities: 0.0684, 0.9173, 0.0143)\n","Processed DRAMP02919: AMP (Probabilities: 0.0802, 0.9064, 0.0135)\n","Processed DRAMP02920: Neuropeptide (Probabilities: 0.2096, 0.2513, 0.5391)\n","Processed DRAMP02921: AMP (Probabilities: 0.0689, 0.9170, 0.0140)\n","Processed DRAMP02928: AMP (Probabilities: 0.2093, 0.4712, 0.3195)\n","Processed DRAMP02930: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9848)\n","Processed DRAMP18228: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP02957: AMP (Probabilities: 0.1180, 0.8691, 0.0129)\n","Processed DRAMP02958: non-AMP/OTHERS (Probabilities: 0.9917, 0.0032, 0.0050)\n","Processed DRAMP02987: non-AMP/OTHERS (Probabilities: 0.5839, 0.3544, 0.0617)\n","Processed DRAMP04531: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP04530: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed DRAMP03013: AMP (Probabilities: 0.0684, 0.9149, 0.0167)\n","Processed DRAMP03014: AMP (Probabilities: 0.0711, 0.9147, 0.0142)\n","Processed DRAMP03015: AMP (Probabilities: 0.0706, 0.9149, 0.0146)\n","Processed DRAMP03023: AMP (Probabilities: 0.1789, 0.8058, 0.0153)\n","Processed DRAMP03026: AMP (Probabilities: 0.0717, 0.9131, 0.0152)\n","Processed DRAMP03032: AMP (Probabilities: 0.0735, 0.9125, 0.0140)\n","Processed DRAMP03048: non-AMP/OTHERS (Probabilities: 0.9634, 0.0334, 0.0032)\n","Processed DRAMP03049: AMP (Probabilities: 0.0687, 0.9159, 0.0154)\n","Processed DRAMP03062: AMP (Probabilities: 0.0659, 0.9186, 0.0155)\n","Processed DRAMP03064: Neuropeptide (Probabilities: 0.0273, 0.0152, 0.9575)\n","Processed DRAMP03068: Neuropeptide (Probabilities: 0.0121, 0.0116, 0.9762)\n","Processed DRAMP03071: AMP (Probabilities: 0.0682, 0.9179, 0.0139)\n","Processed DRAMP03105: AMP (Probabilities: 0.0736, 0.9116, 0.0149)\n","Processed DRAMP03107: non-AMP/OTHERS (Probabilities: 0.9294, 0.0639, 0.0067)\n","Processed DRAMP03108: non-AMP/OTHERS (Probabilities: 0.9910, 0.0066, 0.0024)\n","Processed DRAMP03111: non-AMP/OTHERS (Probabilities: 0.9842, 0.0055, 0.0102)\n","Processed DRAMP03118: AMP (Probabilities: 0.0681, 0.9165, 0.0154)\n","Processed DRAMP03119: AMP (Probabilities: 0.0707, 0.9150, 0.0143)\n","Processed DRAMP03120: AMP (Probabilities: 0.0696, 0.9161, 0.0143)\n","Processed DRAMP03123: AMP (Probabilities: 0.0691, 0.9157, 0.0152)\n","Processed DRAMP03124: AMP (Probabilities: 0.0692, 0.9165, 0.0144)\n","Processed DRAMP03126: AMP (Probabilities: 0.0723, 0.9139, 0.0138)\n","Processed DRAMP03128: AMP (Probabilities: 0.0692, 0.9162, 0.0146)\n","Processed DRAMP03139: AMP (Probabilities: 0.0629, 0.9205, 0.0165)\n","Processed DRAMP03141: AMP (Probabilities: 0.0651, 0.9194, 0.0155)\n","Processed DRAMP03142: AMP (Probabilities: 0.0666, 0.9181, 0.0153)\n","Processed DRAMP03143: AMP (Probabilities: 0.0667, 0.9189, 0.0144)\n","Processed DRAMP03149: AMP (Probabilities: 0.1167, 0.8675, 0.0157)\n","Processed DRAMP03155: AMP (Probabilities: 0.0850, 0.9017, 0.0133)\n","Processed DRAMP03167: AMP (Probabilities: 0.0744, 0.9119, 0.0137)\n","Processed DRAMP03168: non-AMP/OTHERS (Probabilities: 0.9808, 0.0067, 0.0125)\n","Processed DRAMP03189: AMP (Probabilities: 0.0718, 0.9139, 0.0143)\n","Processed DRAMP18221: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed DRAMP03192: AMP (Probabilities: 0.0690, 0.9160, 0.0150)\n","Processed DRAMP18220: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed DRAMP18219: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed DRAMP18218: Neuropeptide (Probabilities: 0.0061, 0.0086, 0.9853)\n","Processed DRAMP18217: non-AMP/OTHERS (Probabilities: 0.9695, 0.0165, 0.0139)\n","Processed DRAMP03234: Neuropeptide (Probabilities: 0.1295, 0.1278, 0.7427)\n","Processed DRAMP03235: non-AMP/OTHERS (Probabilities: 0.6653, 0.3258, 0.0089)\n","Processed DRAMP03243: AMP (Probabilities: 0.0698, 0.9157, 0.0145)\n","Processed DRAMP03250: AMP (Probabilities: 0.0702, 0.9152, 0.0145)\n","Processed DRAMP03255: non-AMP/OTHERS (Probabilities: 0.9808, 0.0084, 0.0108)\n","Processed DRAMP03256: AMP (Probabilities: 0.0708, 0.9147, 0.0144)\n","Processed DRAMP03257: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed DRAMP03258: non-AMP/OTHERS (Probabilities: 0.9589, 0.0155, 0.0255)\n","Processed DRAMP03260: AMP (Probabilities: 0.0665, 0.9194, 0.0141)\n","Processed DRAMP03266: AMP (Probabilities: 0.0680, 0.9169, 0.0151)\n","Processed DRAMP18213: AMP (Probabilities: 0.1129, 0.8756, 0.0115)\n","Processed DRAMP18211: Neuropeptide (Probabilities: 0.0068, 0.0085, 0.9847)\n","Processed DRAMP03284: AMP (Probabilities: 0.1361, 0.8158, 0.0481)\n","Processed DRAMP03289: AMP (Probabilities: 0.0663, 0.9186, 0.0151)\n","Processed DRAMP03292: AMP (Probabilities: 0.0686, 0.9169, 0.0145)\n","Processed DRAMP03293: AMP (Probabilities: 0.0718, 0.9142, 0.0139)\n","Processed DRAMP03294: AMP (Probabilities: 0.0735, 0.9138, 0.0127)\n","Processed DRAMP03295: AMP (Probabilities: 0.0660, 0.9200, 0.0140)\n","Processed DRAMP03296: AMP (Probabilities: 0.0671, 0.9178, 0.0151)\n","Processed DRAMP03297: AMP (Probabilities: 0.0680, 0.9176, 0.0144)\n","Processed DRAMP03298: AMP (Probabilities: 0.0715, 0.9142, 0.0142)\n","Processed DRAMP03299: AMP (Probabilities: 0.0721, 0.9134, 0.0146)\n","Processed DRAMP03300: AMP (Probabilities: 0.0653, 0.9187, 0.0159)\n","Processed DRAMP03301: AMP (Probabilities: 0.0749, 0.9123, 0.0128)\n","Processed DRAMP03316: AMP (Probabilities: 0.0699, 0.9151, 0.0150)\n","Processed DRAMP03317: AMP (Probabilities: 0.0705, 0.9150, 0.0145)\n","Processed DRAMP03322: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed DRAMP03323: non-AMP/OTHERS (Probabilities: 0.9882, 0.0068, 0.0050)\n","Processed DRAMP03324: non-AMP/OTHERS (Probabilities: 0.9912, 0.0053, 0.0035)\n","Processed DRAMP03325: Neuropeptide (Probabilities: 0.2442, 0.2024, 0.5534)\n","Processed DRAMP18204: AMP (Probabilities: 0.1355, 0.8333, 0.0311)\n","Processed DRAMP03327: non-AMP/OTHERS (Probabilities: 0.9598, 0.0371, 0.0031)\n","Processed DRAMP18206: AMP (Probabilities: 0.1222, 0.8637, 0.0141)\n","Processed DRAMP18205: Neuropeptide (Probabilities: 0.1539, 0.1191, 0.7270)\n","Processed DRAMP03333: AMP (Probabilities: 0.0671, 0.9181, 0.0149)\n","Processed DRAMP03345: AMP (Probabilities: 0.0665, 0.9184, 0.0150)\n","Processed DRAMP03346: AMP (Probabilities: 0.0687, 0.9173, 0.0140)\n","Processed DRAMP03350: AMP (Probabilities: 0.0691, 0.9171, 0.0138)\n","Processed DRAMP03351: AMP (Probabilities: 0.0715, 0.9149, 0.0136)\n","Processed DRAMP03352: AMP (Probabilities: 0.1002, 0.8877, 0.0122)\n","Processed DRAMP03356: AMP (Probabilities: 0.1323, 0.8549, 0.0128)\n","Processed DRAMP03364: AMP (Probabilities: 0.0752, 0.9111, 0.0136)\n","Processed DRAMP03367: AMP (Probabilities: 0.0929, 0.8943, 0.0128)\n","Processed DRAMP03371: AMP (Probabilities: 0.0676, 0.9175, 0.0149)\n","Processed DRAMP03372: AMP (Probabilities: 0.1325, 0.8539, 0.0136)\n","Processed DRAMP03374: AMP (Probabilities: 0.0816, 0.9051, 0.0132)\n","Processed DRAMP03375: AMP (Probabilities: 0.0823, 0.9044, 0.0133)\n","Processed DRAMP03376: AMP (Probabilities: 0.0676, 0.9180, 0.0144)\n","Processed DRAMP03377: AMP (Probabilities: 0.0693, 0.9163, 0.0144)\n","Processed DRAMP03378: AMP (Probabilities: 0.0707, 0.9154, 0.0139)\n","Processed DRAMP03381: AMP (Probabilities: 0.0679, 0.9172, 0.0149)\n","Processed DRAMP03382: AMP (Probabilities: 0.0701, 0.9162, 0.0137)\n","Processed DRAMP03384: AMP (Probabilities: 0.0663, 0.9194, 0.0144)\n","Processed DRAMP03385: AMP (Probabilities: 0.0661, 0.9186, 0.0153)\n","Processed DRAMP03386: AMP (Probabilities: 0.0739, 0.9121, 0.0140)\n","Processed DRAMP03387: AMP (Probabilities: 0.1825, 0.7226, 0.0949)\n","Processed DRAMP03388: AMP (Probabilities: 0.0691, 0.9165, 0.0144)\n","Processed DRAMP03389: AMP (Probabilities: 0.0657, 0.9190, 0.0154)\n","Processed DRAMP03392: AMP (Probabilities: 0.0709, 0.9151, 0.0141)\n","Processed DRAMP03393: AMP (Probabilities: 0.0685, 0.9172, 0.0143)\n","Processed DRAMP03394: AMP (Probabilities: 0.0716, 0.9148, 0.0136)\n","Processed DRAMP03395: AMP (Probabilities: 0.0695, 0.9160, 0.0144)\n","Processed DRAMP03396: AMP (Probabilities: 0.0676, 0.9186, 0.0138)\n","Processed DRAMP03398: AMP (Probabilities: 0.0675, 0.9180, 0.0145)\n","Processed DRAMP03399: AMP (Probabilities: 0.0687, 0.9169, 0.0143)\n","Processed DRAMP03400: non-AMP/OTHERS (Probabilities: 0.5433, 0.1376, 0.3191)\n","Processed DRAMP03401: AMP (Probabilities: 0.0675, 0.9178, 0.0146)\n","Processed DRAMP03402: AMP (Probabilities: 0.0672, 0.9182, 0.0146)\n","Processed DRAMP03412: AMP (Probabilities: 0.0662, 0.9185, 0.0153)\n","Processed DRAMP03413: AMP (Probabilities: 0.0834, 0.9045, 0.0122)\n","Processed DRAMP03414: AMP (Probabilities: 0.0933, 0.8928, 0.0139)\n","Processed DRAMP03415: non-AMP/OTHERS (Probabilities: 0.4924, 0.4773, 0.0304)\n","Processed DRAMP03416: AMP (Probabilities: 0.1328, 0.8468, 0.0204)\n","Processed DRAMP03417: AMP (Probabilities: 0.1597, 0.8197, 0.0206)\n","Processed DRAMP03418: AMP (Probabilities: 0.0928, 0.8941, 0.0131)\n","Processed DRAMP03425: AMP (Probabilities: 0.0674, 0.9174, 0.0152)\n","Processed DRAMP03426: AMP (Probabilities: 0.0710, 0.9145, 0.0146)\n","Processed DRAMP03427: AMP (Probabilities: 0.0735, 0.9128, 0.0137)\n","Processed DRAMP03428: AMP (Probabilities: 0.0874, 0.8996, 0.0130)\n","Processed DRAMP03429: AMP (Probabilities: 0.0676, 0.9182, 0.0142)\n","Processed DRAMP03430: AMP (Probabilities: 0.0675, 0.9175, 0.0150)\n","Processed DRAMP03432: AMP (Probabilities: 0.0757, 0.9110, 0.0134)\n","Processed DRAMP03434: AMP (Probabilities: 0.0703, 0.9129, 0.0168)\n","Processed DRAMP03436: AMP (Probabilities: 0.2083, 0.6847, 0.1070)\n","Processed DRAMP03437: AMP (Probabilities: 0.0665, 0.9181, 0.0153)\n","Processed DRAMP03439: AMP (Probabilities: 0.0667, 0.9186, 0.0147)\n","Processed DRAMP03440: AMP (Probabilities: 0.0655, 0.9192, 0.0153)\n","Processed DRAMP03442: AMP (Probabilities: 0.0686, 0.9176, 0.0138)\n","Processed DRAMP03443: AMP (Probabilities: 0.0662, 0.9192, 0.0146)\n","Processed DRAMP03444: AMP (Probabilities: 0.0707, 0.9152, 0.0141)\n","Processed DRAMP03447: AMP (Probabilities: 0.0739, 0.9129, 0.0131)\n","Processed DRAMP03448: AMP (Probabilities: 0.0713, 0.9148, 0.0139)\n","Processed DRAMP03449: AMP (Probabilities: 0.0692, 0.9165, 0.0143)\n","Processed DRAMP03450: AMP (Probabilities: 0.0648, 0.9200, 0.0153)\n","Processed DRAMP03451: AMP (Probabilities: 0.0650, 0.9196, 0.0154)\n","Processed DRAMP03452: AMP (Probabilities: 0.0668, 0.9179, 0.0153)\n","Processed DRAMP03453: AMP (Probabilities: 0.0757, 0.9111, 0.0132)\n","Processed DRAMP03454: AMP (Probabilities: 0.0681, 0.9165, 0.0154)\n","Processed DRAMP03455: AMP (Probabilities: 0.0666, 0.9182, 0.0152)\n","Processed DRAMP03456: AMP (Probabilities: 0.0709, 0.9153, 0.0138)\n","Processed DRAMP03457: AMP (Probabilities: 0.0680, 0.9170, 0.0150)\n","Processed DRAMP03458: AMP (Probabilities: 0.0664, 0.9179, 0.0157)\n","Processed DRAMP03460: AMP (Probabilities: 0.2699, 0.5724, 0.1576)\n","Processed DRAMP03462: non-AMP/OTHERS (Probabilities: 0.9666, 0.0301, 0.0033)\n","Processed DRAMP03466: Neuropeptide (Probabilities: 0.0253, 0.0148, 0.9599)\n","Processed DRAMP03468: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed DRAMP03469: non-AMP/OTHERS (Probabilities: 0.9924, 0.0044, 0.0032)\n","Processed DRAMP03473: Neuropeptide (Probabilities: 0.0063, 0.0088, 0.9849)\n","Processed DRAMP03478: non-AMP/OTHERS (Probabilities: 0.8220, 0.1675, 0.0105)\n","Processed DRAMP03482: AMP (Probabilities: 0.3930, 0.5924, 0.0146)\n","Processed DRAMP03483: AMP (Probabilities: 0.4062, 0.5841, 0.0097)\n","Processed DRAMP03484: AMP (Probabilities: 0.2721, 0.7077, 0.0202)\n","Processed DRAMP03488: non-AMP/OTHERS (Probabilities: 0.4888, 0.4792, 0.0320)\n","Processed DRAMP03489: AMP (Probabilities: 0.0683, 0.9163, 0.0155)\n","Processed DRAMP03490: non-AMP/OTHERS (Probabilities: 0.9796, 0.0086, 0.0118)\n","Processed DRAMP03491: non-AMP/OTHERS (Probabilities: 0.9867, 0.0106, 0.0027)\n","Processed DRAMP03494: AMP (Probabilities: 0.0668, 0.9184, 0.0148)\n","Processed DRAMP03501: AMP (Probabilities: 0.0746, 0.9127, 0.0126)\n","Processed DRAMP03506: Neuropeptide (Probabilities: 0.0632, 0.0236, 0.9132)\n","Processed DRAMP03508: Neuropeptide (Probabilities: 0.0058, 0.0094, 0.9848)\n","Processed DRAMP03522: AMP (Probabilities: 0.0726, 0.9141, 0.0133)\n","Processed DRAMP03531: AMP (Probabilities: 0.0677, 0.9172, 0.0152)\n","Processed DRAMP18208: non-AMP/OTHERS (Probabilities: 0.9611, 0.0261, 0.0128)\n","Processed DRAMP03545: AMP (Probabilities: 0.0727, 0.9133, 0.0140)\n","Processed DRAMP03548: AMP (Probabilities: 0.0788, 0.9086, 0.0126)\n","Processed DRAMP03552: AMP (Probabilities: 0.0766, 0.9106, 0.0127)\n","Processed DRAMP03558: AMP (Probabilities: 0.0685, 0.9182, 0.0133)\n","Processed DRAMP03576: AMP (Probabilities: 0.1073, 0.8787, 0.0140)\n","Processed DRAMP03614: AMP (Probabilities: 0.0713, 0.9149, 0.0138)\n","Processed DRAMP03621: AMP (Probabilities: 0.0689, 0.9166, 0.0145)\n","Processed DRAMP03629: AMP (Probabilities: 0.0703, 0.9164, 0.0132)\n","Processed DRAMP03630: AMP (Probabilities: 0.0685, 0.9169, 0.0146)\n","Processed DRAMP18140: AMP (Probabilities: 0.0708, 0.9153, 0.0139)\n","Processed DRAMP03633: AMP (Probabilities: 0.1265, 0.8593, 0.0142)\n","Processed DRAMP03637: non-AMP/OTHERS (Probabilities: 0.9920, 0.0049, 0.0031)\n","Processed DRAMP03656: AMP (Probabilities: 0.0714, 0.9148, 0.0138)\n","Processed DRAMP03661: AMP (Probabilities: 0.0697, 0.9161, 0.0142)\n","Processed DRAMP03662: AMP (Probabilities: 0.0757, 0.9107, 0.0136)\n","Processed DRAMP03665: Neuropeptide (Probabilities: 0.0997, 0.0508, 0.8494)\n","Processed DRAMP03666: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed DRAMP03667: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed DRAMP03668: AMP (Probabilities: 0.0665, 0.9186, 0.0149)\n","Processed DRAMP03672: AMP (Probabilities: 0.0898, 0.8937, 0.0165)\n","Processed DRAMP03674: non-AMP/OTHERS (Probabilities: 0.9895, 0.0032, 0.0073)\n","Processed DRAMP03690: AMP (Probabilities: 0.0700, 0.9151, 0.0150)\n","Processed DRAMP03692: AMP (Probabilities: 0.0693, 0.9161, 0.0145)\n","Processed DRAMP03695: AMP (Probabilities: 0.0674, 0.9178, 0.0148)\n","Processed DRAMP03696: AMP (Probabilities: 0.0716, 0.9143, 0.0141)\n","Processed DRAMP03697: AMP (Probabilities: 0.0671, 0.9184, 0.0145)\n","Processed DRAMP03698: AMP (Probabilities: 0.0670, 0.9174, 0.0156)\n","Processed DRAMP03699: AMP (Probabilities: 0.0648, 0.9195, 0.0157)\n","Processed DRAMP03701: non-AMP/OTHERS (Probabilities: 0.9875, 0.0075, 0.0051)\n","Processed DRAMP03703: AMP (Probabilities: 0.0761, 0.9099, 0.0140)\n","Processed DRAMP03708: AMP (Probabilities: 0.0676, 0.9172, 0.0152)\n","Processed DRAMP03709: AMP (Probabilities: 0.0697, 0.9155, 0.0149)\n","Processed DRAMP03711: AMP (Probabilities: 0.0687, 0.9159, 0.0154)\n","Processed DRAMP03713: AMP (Probabilities: 0.0677, 0.9168, 0.0155)\n","Processed DRAMP03717: AMP (Probabilities: 0.0942, 0.8938, 0.0120)\n","Processed DRAMP03718: AMP (Probabilities: 0.0923, 0.8961, 0.0116)\n","Processed DRAMP18417: AMP (Probabilities: 0.0710, 0.9141, 0.0149)\n","Processed DRAMP03725: AMP (Probabilities: 0.0693, 0.9156, 0.0151)\n","Processed DRAMP03726: AMP (Probabilities: 0.0696, 0.9143, 0.0161)\n","Processed DRAMP03728: AMP (Probabilities: 0.0667, 0.9176, 0.0157)\n","Processed DRAMP03733: non-AMP/OTHERS (Probabilities: 0.9849, 0.0071, 0.0080)\n","Processed DRAMP03737: AMP (Probabilities: 0.0695, 0.9161, 0.0143)\n","Processed DRAMP03741: AMP (Probabilities: 0.0709, 0.9148, 0.0143)\n","Processed DRAMP03749: Neuropeptide (Probabilities: 0.0178, 0.0149, 0.9673)\n","Processed DRAMP03756: AMP (Probabilities: 0.0675, 0.9171, 0.0154)\n","Processed DRAMP03757: AMP (Probabilities: 0.0714, 0.9142, 0.0144)\n","Processed DRAMP03759: AMP (Probabilities: 0.0775, 0.9088, 0.0137)\n","Processed DRAMP03761: AMP (Probabilities: 0.0669, 0.9180, 0.0151)\n","Processed DRAMP03762: Neuropeptide (Probabilities: 0.0076, 0.0090, 0.9834)\n","Processed DRAMP03769: AMP (Probabilities: 0.0727, 0.9125, 0.0148)\n","Processed DRAMP03771: AMP (Probabilities: 0.0678, 0.9168, 0.0154)\n","Processed DRAMP03772: AMP (Probabilities: 0.0680, 0.9164, 0.0156)\n","Processed DRAMP03773: AMP (Probabilities: 0.0712, 0.9147, 0.0141)\n","Processed DRAMP03778: Neuropeptide (Probabilities: 0.0893, 0.0354, 0.8753)\n","Processed DRAMP03780: AMP (Probabilities: 0.0652, 0.9188, 0.0160)\n","Processed DRAMP03781: AMP (Probabilities: 0.0671, 0.9174, 0.0155)\n","Processed DRAMP03782: AMP (Probabilities: 0.0735, 0.9133, 0.0131)\n","Processed DRAMP03783: Neuropeptide (Probabilities: 0.0148, 0.0158, 0.9695)\n","Processed DRAMP03788: AMP (Probabilities: 0.2178, 0.5268, 0.2554)\n","Processed DRAMP03789: Neuropeptide (Probabilities: 0.0122, 0.0124, 0.9754)\n","Processed DRAMP03791: AMP (Probabilities: 0.1834, 0.6306, 0.1860)\n","Processed DRAMP03792: Neuropeptide (Probabilities: 0.1060, 0.1224, 0.7716)\n","Processed DRAMP03793: AMP (Probabilities: 0.1961, 0.4200, 0.3838)\n","Processed DRAMP03795: AMP (Probabilities: 0.1823, 0.7253, 0.0924)\n","Processed DRAMP03808: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed DRAMP03817: AMP (Probabilities: 0.1229, 0.8640, 0.0130)\n","Processed DRAMP03850: AMP (Probabilities: 0.1676, 0.8179, 0.0146)\n","Processed DRAMP03873: Neuropeptide (Probabilities: 0.0463, 0.0313, 0.9224)\n","Processed DRAMP03874: non-AMP/OTHERS (Probabilities: 0.5347, 0.3919, 0.0734)\n","Processed DRAMP03915: AMP (Probabilities: 0.1497, 0.8372, 0.0131)\n","Processed DRAMP03919: AMP (Probabilities: 0.1063, 0.8818, 0.0119)\n","Processed DRAMP03926: AMP (Probabilities: 0.0676, 0.9171, 0.0153)\n","Processed DRAMP03932: AMP (Probabilities: 0.1659, 0.8203, 0.0138)\n","Processed DRAMP03942: AMP (Probabilities: 0.1425, 0.8444, 0.0131)\n","Processed DRAMP03943: AMP (Probabilities: 0.0827, 0.9051, 0.0122)\n","Processed DRAMP03962: AMP (Probabilities: 0.0707, 0.9145, 0.0149)\n","Processed DRAMP04006: AMP (Probabilities: 0.1597, 0.8265, 0.0137)\n","Processed DRAMP04041: AMP (Probabilities: 0.0717, 0.9137, 0.0146)\n","Processed DRAMP04046: AMP (Probabilities: 0.0697, 0.9154, 0.0149)\n","Processed DRAMP04072: AMP (Probabilities: 0.0658, 0.9183, 0.0159)\n","Processed DRAMP04134: AMP (Probabilities: 0.1750, 0.8104, 0.0146)\n","Processed DRAMP04149: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed DRAMP04150: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed DRAMP04198: non-AMP/OTHERS (Probabilities: 0.9523, 0.0408, 0.0069)\n","Processed DRAMP04204: AMP (Probabilities: 0.1235, 0.8645, 0.0120)\n","Processed DRAMP04206: AMP (Probabilities: 0.0764, 0.9105, 0.0131)\n","Processed DRAMP04207: AMP (Probabilities: 0.1151, 0.8727, 0.0122)\n","Processed DRAMP02126: AMP (Probabilities: 0.0667, 0.9175, 0.0158)\n","Processed DRAMP04212: AMP (Probabilities: 0.0710, 0.9137, 0.0153)\n","Processed DRAMP04214: AMP (Probabilities: 0.1619, 0.8240, 0.0141)\n","Processed DRAMP04216: non-AMP/OTHERS (Probabilities: 0.8601, 0.0965, 0.0434)\n","Processed DRAMP04217: non-AMP/OTHERS (Probabilities: 0.9836, 0.0098, 0.0066)\n","Processed DRAMP04219: non-AMP/OTHERS (Probabilities: 0.9584, 0.0208, 0.0208)\n","Processed DRAMP04222: Neuropeptide (Probabilities: 0.0399, 0.0203, 0.9398)\n","Processed DRAMP04226: AMP (Probabilities: 0.1121, 0.8695, 0.0183)\n","Processed DRAMP04228: AMP (Probabilities: 0.0723, 0.9138, 0.0140)\n","Processed DRAMP04229: AMP (Probabilities: 0.0720, 0.9139, 0.0142)\n","Processed DRAMP04230: AMP (Probabilities: 0.0723, 0.9130, 0.0148)\n","Processed DRAMP04236: AMP (Probabilities: 0.0707, 0.9150, 0.0142)\n","Processed DRAMP04245: AMP (Probabilities: 0.1125, 0.8759, 0.0116)\n","Processed DRAMP04246: AMP (Probabilities: 0.0746, 0.9111, 0.0143)\n","Processed DRAMP04248: AMP (Probabilities: 0.0752, 0.9116, 0.0132)\n","Processed DRAMP04249: AMP (Probabilities: 0.0706, 0.9145, 0.0149)\n","Processed DRAMP04250: AMP (Probabilities: 0.0731, 0.9125, 0.0144)\n","Processed DRAMP04257: AMP (Probabilities: 0.0689, 0.9166, 0.0145)\n","Processed DRAMP04260: AMP (Probabilities: 0.1653, 0.8204, 0.0143)\n","Processed DRAMP04261: AMP (Probabilities: 0.0742, 0.9120, 0.0138)\n","Processed DRAMP04263: AMP (Probabilities: 0.0975, 0.8905, 0.0119)\n","Processed DRAMP04267: AMP (Probabilities: 0.0702, 0.9153, 0.0145)\n","Processed DRAMP04268: AMP (Probabilities: 0.0697, 0.9164, 0.0140)\n","Processed DRAMP04269: AMP (Probabilities: 0.0684, 0.9158, 0.0158)\n","Processed DRAMP04274: AMP (Probabilities: 0.0665, 0.9185, 0.0150)\n","Processed DRAMP04276: AMP (Probabilities: 0.1302, 0.8590, 0.0108)\n","Processed DRAMP04277: AMP (Probabilities: 0.0726, 0.9129, 0.0145)\n","Processed DRAMP04280: AMP (Probabilities: 0.0710, 0.9142, 0.0149)\n","Processed DRAMP04281: AMP (Probabilities: 0.0743, 0.9116, 0.0140)\n","Processed DRAMP04282: AMP (Probabilities: 0.0710, 0.9141, 0.0149)\n","Processed DRAMP04287: AMP (Probabilities: 0.1830, 0.8013, 0.0157)\n","Processed DRAMP04288: AMP (Probabilities: 0.0700, 0.9147, 0.0154)\n","Processed DRAMP04289: AMP (Probabilities: 0.1778, 0.8070, 0.0152)\n","Processed DRAMP04290: AMP (Probabilities: 0.0723, 0.9132, 0.0144)\n","Processed DRAMP04291: AMP (Probabilities: 0.2502, 0.5619, 0.1878)\n","Processed DRAMP04293: non-AMP/OTHERS (Probabilities: 0.7762, 0.1484, 0.0754)\n","Processed DRAMP04294: non-AMP/OTHERS (Probabilities: 0.9813, 0.0127, 0.0060)\n","Processed DRAMP04295: non-AMP/OTHERS (Probabilities: 0.9652, 0.0196, 0.0152)\n","Processed DRAMP04296: non-AMP/OTHERS (Probabilities: 0.8064, 0.1750, 0.0187)\n","Processed DRAMP04297: AMP (Probabilities: 0.1602, 0.8257, 0.0141)\n","Processed DRAMP04299: AMP (Probabilities: 0.1485, 0.8385, 0.0130)\n","Processed DRAMP04300: AMP (Probabilities: 0.2427, 0.7465, 0.0109)\n","Processed DRAMP04301: non-AMP/OTHERS (Probabilities: 0.7146, 0.2690, 0.0163)\n","Processed DRAMP04302: non-AMP/OTHERS (Probabilities: 0.9727, 0.0142, 0.0132)\n","Processed DRAMP04303: non-AMP/OTHERS (Probabilities: 0.9695, 0.0223, 0.0083)\n","Processed DRAMP04304: AMP (Probabilities: 0.1676, 0.8186, 0.0138)\n","Processed DRAMP04306: AMP (Probabilities: 0.0709, 0.9137, 0.0155)\n","Processed DRAMP04307: AMP (Probabilities: 0.0763, 0.9096, 0.0141)\n","Processed DRAMP04308: AMP (Probabilities: 0.1020, 0.8859, 0.0121)\n","Processed DRAMP04309: non-AMP/OTHERS (Probabilities: 0.9844, 0.0103, 0.0053)\n","Processed DRAMP04310: AMP (Probabilities: 0.1698, 0.8155, 0.0147)\n","Processed DRAMP04312: AMP (Probabilities: 0.0721, 0.9131, 0.0149)\n","Processed DRAMP04313: AMP (Probabilities: 0.0782, 0.9088, 0.0129)\n","Processed DRAMP04314: AMP (Probabilities: 0.1711, 0.8144, 0.0145)\n","Processed DRAMP04315: AMP (Probabilities: 0.1763, 0.8088, 0.0150)\n","Processed DRAMP04316: non-AMP/OTHERS (Probabilities: 0.9852, 0.0071, 0.0077)\n","Processed DRAMP04317: AMP (Probabilities: 0.0699, 0.9143, 0.0159)\n","Processed DRAMP04318: AMP (Probabilities: 0.1625, 0.8230, 0.0145)\n","Processed DRAMP04320: AMP (Probabilities: 0.0846, 0.9035, 0.0119)\n","Processed DRAMP04325: AMP (Probabilities: 0.0903, 0.8978, 0.0119)\n","Processed DRAMP04327: AMP (Probabilities: 0.1589, 0.6053, 0.2358)\n","Processed DRAMP04329: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed DRAMP04330: Neuropeptide (Probabilities: 0.0720, 0.0833, 0.8447)\n","Processed DRAMP04340: AMP (Probabilities: 0.0764, 0.9101, 0.0135)\n","Processed DRAMP04347: AMP (Probabilities: 0.0845, 0.8997, 0.0158)\n","Processed DRAMP04349: AMP (Probabilities: 0.0721, 0.9140, 0.0139)\n","Processed DRAMP04350: AMP (Probabilities: 0.0754, 0.9106, 0.0140)\n","Processed DRAMP04351: AMP (Probabilities: 0.0736, 0.9124, 0.0140)\n","Processed DRAMP04353: AMP (Probabilities: 0.1061, 0.8793, 0.0147)\n","Processed DRAMP04356: AMP (Probabilities: 0.0683, 0.9177, 0.0141)\n","Processed DRAMP04396: AMP (Probabilities: 0.0691, 0.9151, 0.0158)\n","Processed DRAMP04398: AMP (Probabilities: 0.0674, 0.9177, 0.0148)\n","Processed DRAMP04403: non-AMP/OTHERS (Probabilities: 0.7728, 0.2199, 0.0073)\n","Processed DRAMP04406: AMP (Probabilities: 0.1439, 0.8441, 0.0119)\n","Processed DRAMP04407: AMP (Probabilities: 0.1628, 0.8229, 0.0143)\n","Processed DRAMP04408: AMP (Probabilities: 0.0670, 0.9179, 0.0151)\n","Processed DRAMP04410: AMP (Probabilities: 0.0842, 0.9039, 0.0119)\n","Processed DRAMP04411: AMP (Probabilities: 0.0723, 0.9132, 0.0145)\n","Processed DRAMP04412: AMP (Probabilities: 0.0798, 0.9067, 0.0135)\n","Processed DRAMP04413: AMP (Probabilities: 0.0676, 0.9169, 0.0155)\n","Processed DRAMP04418: AMP (Probabilities: 0.0735, 0.9127, 0.0139)\n","Processed DRAMP04419: AMP (Probabilities: 0.0802, 0.9063, 0.0136)\n","Processed DRAMP04425: AMP (Probabilities: 0.0702, 0.9153, 0.0145)\n","Processed DRAMP04426: AMP (Probabilities: 0.0747, 0.9112, 0.0141)\n","Processed DRAMP04430: AMP (Probabilities: 0.2165, 0.7469, 0.0367)\n","Processed DRAMP04432: AMP (Probabilities: 0.0709, 0.9155, 0.0136)\n","Processed DRAMP04433: AMP (Probabilities: 0.0673, 0.9177, 0.0149)\n","Processed DRAMP04435: AMP (Probabilities: 0.1191, 0.8429, 0.0380)\n","Processed DRAMP04437: AMP (Probabilities: 0.0639, 0.9206, 0.0155)\n","Processed DRAMP04440: AMP (Probabilities: 0.0671, 0.9178, 0.0151)\n","Processed DRAMP04442: AMP (Probabilities: 0.0691, 0.9165, 0.0144)\n","Processed DRAMP04443: AMP (Probabilities: 0.0666, 0.9183, 0.0151)\n","Processed DRAMP04444: AMP (Probabilities: 0.0653, 0.9192, 0.0155)\n","Processed DRAMP04445: AMP (Probabilities: 0.0844, 0.9030, 0.0126)\n","Processed DRAMP04449: AMP (Probabilities: 0.0680, 0.9172, 0.0148)\n","Processed DRAMP04451: AMP (Probabilities: 0.0656, 0.9190, 0.0154)\n","Processed DRAMP04452: AMP (Probabilities: 0.0673, 0.9185, 0.0143)\n","Processed DRAMP04453: AMP (Probabilities: 0.0988, 0.8890, 0.0123)\n","Processed DRAMP04454: AMP (Probabilities: 0.0699, 0.9163, 0.0138)\n","Processed DRAMP04457: AMP (Probabilities: 0.1336, 0.8536, 0.0128)\n","Processed DRAMP04458: AMP (Probabilities: 0.0758, 0.9117, 0.0124)\n","Processed DRAMP04459: AMP (Probabilities: 0.0656, 0.9192, 0.0152)\n","Processed DRAMP04460: AMP (Probabilities: 0.1462, 0.8348, 0.0190)\n","Processed DRAMP04461: AMP (Probabilities: 0.0670, 0.9186, 0.0144)\n","Processed DRAMP04462: AMP (Probabilities: 0.0966, 0.8908, 0.0126)\n","Processed DRAMP04463: AMP (Probabilities: 0.4295, 0.5602, 0.0103)\n","Processed DRAMP04466: AMP (Probabilities: 0.0758, 0.9116, 0.0126)\n","Processed DRAMP04467: AMP (Probabilities: 0.1072, 0.8806, 0.0122)\n","Processed DRAMP04468: AMP (Probabilities: 0.1505, 0.8323, 0.0172)\n","Processed DRAMP04469: AMP (Probabilities: 0.1328, 0.8549, 0.0123)\n","Processed DRAMP04470: non-AMP/OTHERS (Probabilities: 0.9823, 0.0146, 0.0031)\n","Processed DRAMP04471: AMP (Probabilities: 0.0985, 0.8888, 0.0127)\n","Processed DRAMP04473: AMP (Probabilities: 0.0708, 0.9152, 0.0140)\n","Processed DRAMP04474: AMP (Probabilities: 0.1571, 0.7643, 0.0786)\n","Processed DRAMP04475: AMP (Probabilities: 0.0661, 0.9178, 0.0161)\n","Processed DRAMP04476: AMP (Probabilities: 0.0640, 0.9205, 0.0155)\n","Processed DRAMP04477: AMP (Probabilities: 0.0700, 0.9161, 0.0139)\n","Processed DRAMP04478: AMP (Probabilities: 0.1537, 0.8308, 0.0155)\n","Processed DRAMP04479: AMP (Probabilities: 0.0702, 0.9157, 0.0141)\n","Processed DRAMP04480: non-AMP/OTHERS (Probabilities: 0.9831, 0.0065, 0.0105)\n","Processed DRAMP04481: AMP (Probabilities: 0.0703, 0.9159, 0.0138)\n","Processed DRAMP04482: AMP (Probabilities: 0.0962, 0.8914, 0.0124)\n","Processed DRAMP04483: AMP (Probabilities: 0.0640, 0.9201, 0.0159)\n","Processed DRAMP04484: AMP (Probabilities: 0.3005, 0.4270, 0.2725)\n","Processed DRAMP04485: AMP (Probabilities: 0.0662, 0.9185, 0.0153)\n","Processed DRAMP04487: AMP (Probabilities: 0.0679, 0.9175, 0.0146)\n","Processed DRAMP04489: non-AMP/OTHERS (Probabilities: 0.8242, 0.0140, 0.1618)\n","Processed DRAMP04491: AMP (Probabilities: 0.1230, 0.8645, 0.0125)\n","Processed DRAMP04493: AMP (Probabilities: 0.0670, 0.9179, 0.0151)\n","Processed DRAMP04496: AMP (Probabilities: 0.0742, 0.9117, 0.0141)\n","Processed DRAMP04497: AMP (Probabilities: 0.0688, 0.9162, 0.0150)\n","Processed DRAMP04498: Neuropeptide (Probabilities: 0.0638, 0.0318, 0.9044)\n","Processed DRAMP04500: AMP (Probabilities: 0.0690, 0.9163, 0.0147)\n","Processed DRAMP04503: AMP (Probabilities: 0.0741, 0.9115, 0.0143)\n","Processed DRAMP04505: AMP (Probabilities: 0.0705, 0.9148, 0.0147)\n","Processed DRAMP04506: AMP (Probabilities: 0.1104, 0.8767, 0.0129)\n","Processed DRAMP04512: AMP (Probabilities: 0.0819, 0.9035, 0.0146)\n","Processed DRAMP04526: AMP (Probabilities: 0.0759, 0.9108, 0.0133)\n","Processed DRAMP04561: AMP (Probabilities: 0.0685, 0.9167, 0.0148)\n","Processed DRAMP04569: AMP (Probabilities: 0.0737, 0.9127, 0.0136)\n","Processed DRAMP04571: non-AMP/OTHERS (Probabilities: 0.9913, 0.0062, 0.0024)\n","Processed DRAMP04575: AMP (Probabilities: 0.0699, 0.9155, 0.0146)\n","Processed DRAMP04586: AMP (Probabilities: 0.0693, 0.9160, 0.0147)\n","Processed DRAMP04597: AMP (Probabilities: 0.0751, 0.9109, 0.0141)\n","Processed DRAMP04598: AMP (Probabilities: 0.0818, 0.9047, 0.0134)\n","Processed DRAMP04599: AMP (Probabilities: 0.0692, 0.9155, 0.0153)\n","Processed DRAMP04605: AMP (Probabilities: 0.0846, 0.9024, 0.0130)\n","Processed DRAMP04607: Neuropeptide (Probabilities: 0.0205, 0.0207, 0.9588)\n","Processed DRAMP04608: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed DRAMP04615: AMP (Probabilities: 0.1020, 0.8774, 0.0207)\n","Processed DRAMP04616: non-AMP/OTHERS (Probabilities: 0.9913, 0.0053, 0.0034)\n","Processed DRAMP04617: AMP (Probabilities: 0.0691, 0.9155, 0.0154)\n","Processed DRAMP04624: AMP (Probabilities: 0.0709, 0.9142, 0.0149)\n","Processed DRAMP04625: AMP (Probabilities: 0.0719, 0.9138, 0.0143)\n","Processed DRAMP04627: AMP (Probabilities: 0.0672, 0.9174, 0.0155)\n","Processed DRAMP04628: AMP (Probabilities: 0.0713, 0.9139, 0.0147)\n","Processed DRAMP04629: AMP (Probabilities: 0.0701, 0.9148, 0.0151)\n","Processed DRAMP04630: AMP (Probabilities: 0.0670, 0.9167, 0.0163)\n","Processed DRAMP04634: AMP (Probabilities: 0.0651, 0.9185, 0.0163)\n","Processed DRAMP04659: AMP (Probabilities: 0.0725, 0.9133, 0.0141)\n","Processed DRAMP04663: AMP (Probabilities: 0.0836, 0.9034, 0.0130)\n","Processed DRAMP04666: AMP (Probabilities: 0.0699, 0.9155, 0.0146)\n","Processed DRAMP04668: AMP (Probabilities: 0.0684, 0.9173, 0.0144)\n","Processed DRAMP04669: non-AMP/OTHERS (Probabilities: 0.9806, 0.0078, 0.0115)\n","Processed DRAMP04674: non-AMP/OTHERS (Probabilities: 0.9891, 0.0080, 0.0028)\n","Processed DRAMP04675: AMP (Probabilities: 0.0923, 0.8956, 0.0121)\n","Processed DRAMP04678: AMP (Probabilities: 0.0659, 0.9183, 0.0158)\n","Processed DRAMP04681: AMP (Probabilities: 0.1440, 0.7970, 0.0590)\n","Processed DRAMP04682: Neuropeptide (Probabilities: 0.1487, 0.0272, 0.8242)\n","Processed DRAMP04683: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed DRAMP04684: non-AMP/OTHERS (Probabilities: 0.9876, 0.0038, 0.0086)\n","Processed DRAMP04685: non-AMP/OTHERS (Probabilities: 0.9906, 0.0031, 0.0063)\n","Processed DRAMP04686: non-AMP/OTHERS (Probabilities: 0.9929, 0.0037, 0.0034)\n","Processed DRAMP04687: non-AMP/OTHERS (Probabilities: 0.9884, 0.0042, 0.0074)\n","Processed DRAMP04688: non-AMP/OTHERS (Probabilities: 0.9933, 0.0036, 0.0031)\n","Processed DRAMP04689: AMP (Probabilities: 0.1129, 0.8754, 0.0117)\n","Processed DRAMP04690: non-AMP/OTHERS (Probabilities: 0.8271, 0.0159, 0.1570)\n","Processed DRAMP04691: AMP (Probabilities: 0.0912, 0.8964, 0.0124)\n","Processed DRAMP04692: non-AMP/OTHERS (Probabilities: 0.9907, 0.0039, 0.0054)\n","Processed DRAMP04693: AMP (Probabilities: 0.0832, 0.9046, 0.0122)\n","Processed DRAMP04694: non-AMP/OTHERS (Probabilities: 0.9882, 0.0063, 0.0055)\n","Processed DRAMP04695: non-AMP/OTHERS (Probabilities: 0.9921, 0.0038, 0.0041)\n","Processed DRAMP04696: non-AMP/OTHERS (Probabilities: 0.9902, 0.0067, 0.0030)\n","Processed DRAMP04697: non-AMP/OTHERS (Probabilities: 0.9907, 0.0059, 0.0034)\n","Processed DRAMP04698: AMP (Probabilities: 0.0693, 0.9154, 0.0152)\n","Processed DRAMP04699: AMP (Probabilities: 0.0686, 0.9155, 0.0159)\n","Processed DRAMP04700: AMP (Probabilities: 0.1059, 0.8813, 0.0127)\n","Processed DRAMP04701: Neuropeptide (Probabilities: 0.0261, 0.0189, 0.9550)\n","Processed DRAMP04702: non-AMP/OTHERS (Probabilities: 0.9888, 0.0037, 0.0075)\n","Processed DRAMP04708: AMP (Probabilities: 0.0707, 0.9145, 0.0148)\n","Processed DRAMP18097: AMP (Probabilities: 0.0703, 0.9160, 0.0138)\n","Processed DRAMP18098: AMP (Probabilities: 0.1017, 0.8864, 0.0119)\n","Processed DRAMP18099: AMP (Probabilities: 0.0688, 0.9161, 0.0151)\n","Processed DRAMP18100: AMP (Probabilities: 0.0691, 0.9164, 0.0145)\n","Processed DRAMP18101: AMP (Probabilities: 0.0669, 0.9181, 0.0150)\n","Processed DRAMP18104: AMP (Probabilities: 0.0698, 0.9153, 0.0149)\n","Processed DRAMP18106: AMP (Probabilities: 0.0732, 0.9132, 0.0136)\n","Processed DRAMP18108: non-AMP/OTHERS (Probabilities: 0.5212, 0.2394, 0.2393)\n","Processed DRAMP18109: AMP (Probabilities: 0.0658, 0.9195, 0.0147)\n","Processed DRAMP18110: non-AMP/OTHERS (Probabilities: 0.5627, 0.4036, 0.0336)\n","Processed DRAMP18111: AMP (Probabilities: 0.0645, 0.9193, 0.0161)\n","Processed DRAMP18112: AMP (Probabilities: 0.0754, 0.9114, 0.0133)\n","Processed DRAMP18114: AMP (Probabilities: 0.0675, 0.9183, 0.0142)\n","Processed DRAMP18115: AMP (Probabilities: 0.0764, 0.9107, 0.0129)\n","Processed DRAMP18116: non-AMP/OTHERS (Probabilities: 0.8995, 0.0351, 0.0654)\n","Processed DRAMP18117: AMP (Probabilities: 0.4014, 0.5126, 0.0861)\n","Processed DRAMP18118: non-AMP/OTHERS (Probabilities: 0.9908, 0.0043, 0.0050)\n","Processed DRAMP18119: AMP (Probabilities: 0.0693, 0.9169, 0.0139)\n","Processed DRAMP18186: AMP (Probabilities: 0.0675, 0.9178, 0.0148)\n","Processed DRAMP18122: AMP (Probabilities: 0.0710, 0.9156, 0.0135)\n","Processed DRAMP18129: AMP (Probabilities: 0.1326, 0.8425, 0.0250)\n","Processed DRAMP18130: non-AMP/OTHERS (Probabilities: 0.5625, 0.0504, 0.3871)\n","Processed DRAMP18133: Neuropeptide (Probabilities: 0.0164, 0.0146, 0.9690)\n","Processed DRAMP18134: AMP (Probabilities: 0.1033, 0.8852, 0.0115)\n","Processed DRAMP18135: AMP (Probabilities: 0.0963, 0.8917, 0.0120)\n","Processed DRAMP18137: AMP (Probabilities: 0.0704, 0.9148, 0.0148)\n","Processed DRAMP18143: Neuropeptide (Probabilities: 0.0083, 0.0091, 0.9826)\n","Processed DRAMP18144: AMP (Probabilities: 0.0691, 0.9159, 0.0149)\n","Processed DRAMP18145: AMP (Probabilities: 0.0661, 0.9200, 0.0139)\n","Processed DRAMP18146: AMP (Probabilities: 0.1544, 0.8318, 0.0138)\n","Processed DRAMP18147: AMP (Probabilities: 0.0750, 0.9113, 0.0137)\n","Processed DRAMP18148: AMP (Probabilities: 0.0684, 0.9169, 0.0147)\n","Processed DRAMP18149: AMP (Probabilities: 0.0703, 0.9160, 0.0137)\n","Processed DRAMP01274: non-AMP/OTHERS (Probabilities: 0.8379, 0.1538, 0.0083)\n","Processed DRAMP01296: Neuropeptide (Probabilities: 0.3043, 0.1620, 0.5337)\n","Processed DRAMP01297: non-AMP/OTHERS (Probabilities: 0.9754, 0.0090, 0.0156)\n","Processed DRAMP18184: AMP (Probabilities: 0.0676, 0.9187, 0.0137)\n","Processed DRAMP18406: AMP (Probabilities: 0.0688, 0.9163, 0.0149)\n","Processed DRAMP18410: non-AMP/OTHERS (Probabilities: 0.9910, 0.0036, 0.0054)\n","Processed DRAMP18421: AMP (Probabilities: 0.0693, 0.9153, 0.0154)\n","Processed DRAMP18422: AMP (Probabilities: 0.0679, 0.9172, 0.0149)\n","Processed DRAMP18426: AMP (Probabilities: 0.0754, 0.9100, 0.0146)\n","Processed DRAMP18430: AMP (Probabilities: 0.0732, 0.9125, 0.0143)\n","Processed DRAMP18434: AMP (Probabilities: 0.0678, 0.9172, 0.0150)\n","Processed DRAMP18435: Neuropeptide (Probabilities: 0.0248, 0.0204, 0.9547)\n","Processed DRAMP18444: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP18457: AMP (Probabilities: 0.1564, 0.8298, 0.0138)\n","Processed DRAMP18419: AMP (Probabilities: 0.0702, 0.9152, 0.0146)\n","Processed DRAMP18471: AMP (Probabilities: 0.0716, 0.9140, 0.0145)\n","Processed DRAMP18473: AMP (Probabilities: 0.0841, 0.9028, 0.0131)\n","Processed DRAMP18475: AMP (Probabilities: 0.1473, 0.8393, 0.0134)\n","Processed DRAMP18476: AMP (Probabilities: 0.0723, 0.9137, 0.0141)\n","Processed DRAMP18480: AMP (Probabilities: 0.0767, 0.9096, 0.0136)\n","Processed DRAMP18484: AMP (Probabilities: 0.0730, 0.9132, 0.0138)\n","Processed DRAMP18500: AMP (Probabilities: 0.1172, 0.8709, 0.0119)\n","Processed DRAMP18504: AMP (Probabilities: 0.0702, 0.9150, 0.0148)\n","Processed DRAMP18505: AMP (Probabilities: 0.0734, 0.9124, 0.0142)\n","Processed DRAMP18508: AMP (Probabilities: 0.0720, 0.9137, 0.0143)\n","Processed DRAMP18512: AMP (Probabilities: 0.0727, 0.9132, 0.0141)\n","Processed DRAMP18514: AMP (Probabilities: 0.0741, 0.9117, 0.0142)\n","Processed DRAMP18515: AMP (Probabilities: 0.0701, 0.9148, 0.0151)\n","Processed DRAMP18558: AMP (Probabilities: 0.0745, 0.9114, 0.0141)\n","Processed DRAMP18561: AMP (Probabilities: 0.1598, 0.8258, 0.0144)\n","Processed DRAMP18563: AMP (Probabilities: 0.0810, 0.9072, 0.0118)\n","Processed DRAMP18566: AMP (Probabilities: 0.0766, 0.9096, 0.0138)\n","Processed DRAMP18569: non-AMP/OTHERS (Probabilities: 0.9242, 0.0642, 0.0117)\n","Processed DRAMP18576: AMP (Probabilities: 0.0691, 0.9157, 0.0152)\n","Processed DRAMP18578: AMP (Probabilities: 0.0810, 0.9057, 0.0133)\n","Processed DRAMP18583: AMP (Probabilities: 0.0729, 0.9131, 0.0140)\n","Processed DRAMP18586: AMP (Probabilities: 0.0811, 0.9060, 0.0129)\n","Processed DRAMP18589: AMP (Probabilities: 0.0682, 0.9165, 0.0153)\n","Processed DRAMP18601: AMP (Probabilities: 0.1650, 0.8204, 0.0145)\n","Processed DRAMP18613: AMP (Probabilities: 0.1212, 0.8662, 0.0127)\n","Processed DRAMP18627: AMP (Probabilities: 0.1012, 0.8860, 0.0128)\n","Processed DRAMP18614: AMP (Probabilities: 0.0856, 0.9017, 0.0128)\n","Processed DRAMP18615: AMP (Probabilities: 0.0929, 0.8951, 0.0120)\n","Processed DRAMP18619: AMP (Probabilities: 0.1443, 0.8428, 0.0129)\n","Processed DRAMP18622: AMP (Probabilities: 0.0784, 0.9092, 0.0124)\n","Processed DRAMP18623: AMP (Probabilities: 0.0803, 0.9061, 0.0136)\n","Processed DRAMP18631: non-AMP/OTHERS (Probabilities: 0.9847, 0.0128, 0.0025)\n","Processed DRAMP18635: non-AMP/OTHERS (Probabilities: 0.7694, 0.2188, 0.0119)\n","Processed DRAMP18636: AMP (Probabilities: 0.0723, 0.9130, 0.0147)\n","Processed DRAMP18638: non-AMP/OTHERS (Probabilities: 0.9766, 0.0167, 0.0067)\n","Processed DRAMP18640: AMP (Probabilities: 0.0775, 0.9102, 0.0123)\n","Processed DRAMP18642: AMP (Probabilities: 0.0765, 0.9098, 0.0137)\n","Processed DRAMP18644: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed DRAMP18645: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed DRAMP18646: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9850)\n","Processed DRAMP18647: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed DRAMP18651: Neuropeptide (Probabilities: 0.2586, 0.0674, 0.6740)\n","Processed DRAMP18654: AMP (Probabilities: 0.0742, 0.9114, 0.0144)\n","Processed DRAMP18655: AMP (Probabilities: 0.0694, 0.9157, 0.0149)\n","Processed DRAMP18657: Neuropeptide (Probabilities: 0.0076, 0.0086, 0.9839)\n","Processed DRAMP18668: non-AMP/OTHERS (Probabilities: 0.9844, 0.0046, 0.0110)\n","Processed DRAMP18669: non-AMP/OTHERS (Probabilities: 0.9525, 0.0331, 0.0144)\n","Processed DRAMP18670: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed DRAMP18672: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed DRAMP18673: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0039)\n","Processed DRAMP18674: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed DRAMP18675: non-AMP/OTHERS (Probabilities: 0.7716, 0.0196, 0.2088)\n","Processed DRAMP18676: Neuropeptide (Probabilities: 0.0066, 0.0088, 0.9846)\n","Processed DRAMP18677: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed DRAMP18678: non-AMP/OTHERS (Probabilities: 0.8791, 0.0099, 0.1110)\n","Processed DRAMP18680: AMP (Probabilities: 0.0678, 0.9164, 0.0158)\n","Processed DRAMP18682: AMP (Probabilities: 0.0681, 0.9170, 0.0149)\n","Processed DRAMP18683: Neuropeptide (Probabilities: 0.0067, 0.0085, 0.9848)\n","Processed DRAMP18684: AMP (Probabilities: 0.0711, 0.9144, 0.0144)\n","Processed DRAMP18685: AMP (Probabilities: 0.0734, 0.9123, 0.0143)\n","Processed DRAMP18686: AMP (Probabilities: 0.0738, 0.9120, 0.0142)\n","Processed DRAMP18687: AMP (Probabilities: 0.0722, 0.9134, 0.0144)\n","Processed DRAMP18689: AMP (Probabilities: 0.0852, 0.9014, 0.0134)\n","Processed DRAMP18691: AMP (Probabilities: 0.2071, 0.5986, 0.1943)\n","Processed DRAMP18692: AMP (Probabilities: 0.1609, 0.8067, 0.0324)\n","Processed DRAMP18695: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DRAMP18714: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed DRAMP18715: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed DRAMP20770: AMP (Probabilities: 0.0755, 0.9120, 0.0125)\n","Processed DRAMP20775: AMP (Probabilities: 0.0658, 0.9179, 0.0163)\n","Processed DRAMP20776: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed DRAMP20803: AMP (Probabilities: 0.0687, 0.9159, 0.0155)\n","Processed DRAMP20804: non-AMP/OTHERS (Probabilities: 0.9829, 0.0048, 0.0123)\n","Processed DRAMP20817: AMP (Probabilities: 0.1467, 0.8394, 0.0138)\n","Processed DRAMP20821: AMP (Probabilities: 0.0739, 0.9124, 0.0137)\n","Processed DRAMP20822: AMP (Probabilities: 0.0784, 0.9090, 0.0125)\n","Processed DRAMP20823: AMP (Probabilities: 0.0866, 0.9008, 0.0126)\n","Processed DRAMP20827: AMP (Probabilities: 0.0695, 0.9154, 0.0151)\n","Processed DRAMP20828: AMP (Probabilities: 0.0714, 0.9140, 0.0146)\n","Processed DRAMP20831: AMP (Probabilities: 0.0711, 0.9136, 0.0153)\n","Processed DRAMP20834: AMP (Probabilities: 0.1056, 0.8827, 0.0117)\n","Processed DRAMP20835: non-AMP/OTHERS (Probabilities: 0.9523, 0.0415, 0.0062)\n","Processed DRAMP20836: non-AMP/OTHERS (Probabilities: 0.5406, 0.4438, 0.0157)\n","Processed DRAMP20837: AMP (Probabilities: 0.0734, 0.9128, 0.0138)\n","Processed DRAMP20838: AMP (Probabilities: 0.0709, 0.9144, 0.0146)\n","Processed DRAMP20840: non-AMP/OTHERS (Probabilities: 0.9910, 0.0036, 0.0054)\n","Processed DRAMP20841: AMP (Probabilities: 0.1498, 0.8370, 0.0132)\n","Processed DRAMP20849: AMP (Probabilities: 0.0921, 0.8952, 0.0127)\n","Processed DRAMP20850: AMP (Probabilities: 0.0962, 0.8913, 0.0125)\n","Processed DRAMP20856: Neuropeptide (Probabilities: 0.2411, 0.3630, 0.3959)\n","Processed DRAMP20860: AMP (Probabilities: 0.1485, 0.8295, 0.0220)\n","Processed DRAMP20861: AMP (Probabilities: 0.0753, 0.9122, 0.0125)\n","Processed DRAMP20863: AMP (Probabilities: 0.0717, 0.9153, 0.0131)\n","Processed DRAMP20864: AMP (Probabilities: 0.0696, 0.9170, 0.0133)\n","Processed DRAMP20868: AMP (Probabilities: 0.0728, 0.9129, 0.0143)\n","Processed DRAMP20880: AMP (Probabilities: 0.0676, 0.9172, 0.0152)\n","Processed DRAMP20881: AMP (Probabilities: 0.0730, 0.9140, 0.0130)\n","Processed DRAMP20882: AMP (Probabilities: 0.1121, 0.8757, 0.0122)\n","Processed DRAMP20883: AMP (Probabilities: 0.0770, 0.9100, 0.0130)\n","Processed DRAMP20890: AMP (Probabilities: 0.0687, 0.9161, 0.0152)\n","Processed DRAMP20908: AMP (Probabilities: 0.0761, 0.9108, 0.0131)\n","Processed DRAMP20911: non-AMP/OTHERS (Probabilities: 0.9906, 0.0045, 0.0048)\n","Processed DRAMP20912: AMP (Probabilities: 0.0683, 0.9158, 0.0159)\n","Processed DRAMP20927: AMP (Probabilities: 0.1488, 0.8381, 0.0131)\n","Processed DRAMP20937: AMP (Probabilities: 0.0722, 0.9137, 0.0141)\n","Processed DRAMP20938: AMP (Probabilities: 0.0741, 0.9115, 0.0144)\n","Processed DRAMP20945: AMP (Probabilities: 0.0712, 0.9143, 0.0144)\n","Processed DRAMP20953: AMP (Probabilities: 0.0685, 0.9164, 0.0151)\n","Processed DRAMP20956: AMP (Probabilities: 0.0714, 0.9143, 0.0143)\n","Processed DRAMP20957: AMP (Probabilities: 0.0711, 0.9135, 0.0154)\n","Processed DRAMP20959: AMP (Probabilities: 0.0719, 0.9131, 0.0150)\n","Processed DRAMP20963: AMP (Probabilities: 0.1751, 0.7835, 0.0414)\n","Processed DRAMP20965: AMP (Probabilities: 0.1476, 0.8415, 0.0109)\n","Processed DRAMP20966: AMP (Probabilities: 0.0957, 0.8918, 0.0124)\n","Processed DRAMP20967: Neuropeptide (Probabilities: 0.0085, 0.0096, 0.9819)\n","Processed DRAMP20968: non-AMP/OTHERS (Probabilities: 0.9901, 0.0066, 0.0033)\n","Processed DRAMP20969: AMP (Probabilities: 0.1247, 0.8630, 0.0123)\n","Processed DRAMP20970: non-AMP/OTHERS (Probabilities: 0.9546, 0.0419, 0.0035)\n","Processed DRAMP20972: AMP (Probabilities: 0.1640, 0.8107, 0.0253)\n","Processed DRAMP20975: AMP (Probabilities: 0.0736, 0.9125, 0.0139)\n","Processed DRAMP20976: AMP (Probabilities: 0.1685, 0.8174, 0.0142)\n","Processed DRAMP20977: AMP (Probabilities: 0.1441, 0.8425, 0.0134)\n","Processed DRAMP20979: AMP (Probabilities: 0.1324, 0.8541, 0.0135)\n","Processed DRAMP20982: AMP (Probabilities: 0.0695, 0.9152, 0.0153)\n","Processed DRAMP20987: AMP (Probabilities: 0.1333, 0.8531, 0.0136)\n","Processed DRAMP20988: AMP (Probabilities: 0.0748, 0.9109, 0.0144)\n","Processed DRAMP20991: AMP (Probabilities: 0.1434, 0.8434, 0.0132)\n","Processed DRAMP20994: AMP (Probabilities: 0.0694, 0.9151, 0.0155)\n","Processed DRAMP21004: AMP (Probabilities: 0.0695, 0.9165, 0.0140)\n","Processed DRAMP21007: AMP (Probabilities: 0.1069, 0.8617, 0.0314)\n","Processed DRAMP21010: AMP (Probabilities: 0.0722, 0.9147, 0.0131)\n","Processed DRAMP21025: AMP (Probabilities: 0.0699, 0.9146, 0.0155)\n","Processed DRAMP21029: Neuropeptide (Probabilities: 0.1062, 0.0421, 0.8517)\n","Processed DRAMP21040: Neuropeptide (Probabilities: 0.0821, 0.1053, 0.8126)\n","Processed DRAMP21044: AMP (Probabilities: 0.1383, 0.8489, 0.0129)\n","Processed DRAMP21045: AMP (Probabilities: 0.1248, 0.8629, 0.0123)\n","Processed DRAMP21046: AMP (Probabilities: 0.1344, 0.8527, 0.0129)\n","Processed DRAMP21048: AMP (Probabilities: 0.0687, 0.9166, 0.0147)\n","Processed DRAMP21049: AMP (Probabilities: 0.0728, 0.9129, 0.0143)\n","Processed DRAMP21052: AMP (Probabilities: 0.0781, 0.9091, 0.0128)\n","Processed DRAMP21068: AMP (Probabilities: 0.0690, 0.9168, 0.0142)\n","Processed DRAMP21070: AMP (Probabilities: 0.0708, 0.9147, 0.0146)\n","Processed DRAMP21076: AMP (Probabilities: 0.0703, 0.9154, 0.0142)\n","Processed DRAMP21086: AMP (Probabilities: 0.0712, 0.9147, 0.0141)\n","Processed DRAMP21087: AMP (Probabilities: 0.0717, 0.9147, 0.0136)\n","Processed DRAMP21089: AMP (Probabilities: 0.1396, 0.8477, 0.0128)\n","Processed DRAMP21090: AMP (Probabilities: 0.0769, 0.9105, 0.0125)\n","Processed DRAMP21091: AMP (Probabilities: 0.1495, 0.8373, 0.0132)\n","Processed DRAMP21092: AMP (Probabilities: 0.1536, 0.8321, 0.0144)\n","Processed DRAMP21096: AMP (Probabilities: 0.1209, 0.8670, 0.0121)\n","Processed DRAMP21098: AMP (Probabilities: 0.1345, 0.8533, 0.0123)\n","Processed DRAMP21100: AMP (Probabilities: 0.0744, 0.9127, 0.0129)\n","Processed DRAMP21102: AMP (Probabilities: 0.1222, 0.8657, 0.0121)\n","Processed DRAMP21165: AMP (Probabilities: 0.0789, 0.9078, 0.0133)\n","Processed DRAMP21166: AMP (Probabilities: 0.0741, 0.9116, 0.0144)\n","Processed DRAMP21153: AMP (Probabilities: 0.1468, 0.8402, 0.0131)\n","Processed DRAMP21151: AMP (Probabilities: 0.0774, 0.9093, 0.0134)\n","Processed DRAMP21149: AMP (Probabilities: 0.0743, 0.9116, 0.0141)\n","Processed DRAMP21145: AMP (Probabilities: 0.1384, 0.8485, 0.0131)\n","Processed DRAMP21142: AMP (Probabilities: 0.0730, 0.9134, 0.0136)\n","Processed DRAMP21141: AMP (Probabilities: 0.0709, 0.9149, 0.0142)\n","Processed DRAMP21144: AMP (Probabilities: 0.1547, 0.8314, 0.0140)\n","Processed DRAMP21140: AMP (Probabilities: 0.0707, 0.9153, 0.0140)\n","Processed DRAMP21138: AMP (Probabilities: 0.0913, 0.8961, 0.0126)\n","Processed DRAMP21137: AMP (Probabilities: 0.1607, 0.8249, 0.0144)\n","Processed DRAMP21135: AMP (Probabilities: 0.1494, 0.8368, 0.0138)\n","Processed DRAMP21136: AMP (Probabilities: 0.1356, 0.8510, 0.0135)\n","Processed DRAMP21132: AMP (Probabilities: 0.1404, 0.8462, 0.0133)\n","Processed DRAMP21131: AMP (Probabilities: 0.1358, 0.8512, 0.0129)\n","Processed DRAMP21130: AMP (Probabilities: 0.1434, 0.8432, 0.0135)\n","Processed DRAMP21129: AMP (Probabilities: 0.1400, 0.8467, 0.0133)\n","Processed DRAMP21123: AMP (Probabilities: 0.1540, 0.8320, 0.0140)\n","Processed DRAMP21111: AMP (Probabilities: 0.1536, 0.8329, 0.0136)\n","Processed DRAMP21113: AMP (Probabilities: 0.1304, 0.8570, 0.0126)\n","Processed DRAMP21103: Neuropeptide (Probabilities: 0.0622, 0.0787, 0.8590)\n","Processed DRAMP21105: AMP (Probabilities: 0.1282, 0.8585, 0.0132)\n","Processed DRAMP21106: AMP (Probabilities: 0.0681, 0.9165, 0.0154)\n","Processed DRAMP21231: AMP (Probabilities: 0.1077, 0.8722, 0.0201)\n","Processed DRAMP21229: AMP (Probabilities: 0.0936, 0.8912, 0.0153)\n","Processed DRAMP21227: AMP (Probabilities: 0.0739, 0.9117, 0.0144)\n","Processed DRAMP21225: AMP (Probabilities: 0.1418, 0.8450, 0.0132)\n","Processed DRAMP21224: AMP (Probabilities: 0.1435, 0.8431, 0.0134)\n","Processed DRAMP21219: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed DRAMP21211: AMP (Probabilities: 0.1468, 0.8402, 0.0130)\n","Processed DRAMP21208: AMP (Probabilities: 0.1068, 0.8762, 0.0170)\n","Processed DRAMP21202: AMP (Probabilities: 0.0753, 0.9114, 0.0133)\n","Processed DRAMP21200: AMP (Probabilities: 0.0704, 0.9148, 0.0148)\n","Processed DRAMP21199: AMP (Probabilities: 0.0743, 0.9112, 0.0145)\n","Processed DRAMP21198: AMP (Probabilities: 0.0703, 0.9149, 0.0149)\n","Processed DRAMP21197: AMP (Probabilities: 0.0719, 0.9131, 0.0150)\n","Processed DRAMP21196: AMP (Probabilities: 0.0716, 0.9137, 0.0147)\n","Processed DRAMP21195: AMP (Probabilities: 0.0687, 0.9155, 0.0158)\n","Processed DRAMP21194: AMP (Probabilities: 0.0706, 0.9139, 0.0155)\n","Processed DRAMP21193: AMP (Probabilities: 0.0680, 0.9165, 0.0155)\n","Processed DRAMP21192: AMP (Probabilities: 0.0668, 0.9178, 0.0154)\n","Processed DRAMP21191: AMP (Probabilities: 0.0723, 0.9130, 0.0147)\n","Processed DRAMP21190: AMP (Probabilities: 0.0697, 0.9145, 0.0157)\n","Processed DRAMP21189: AMP (Probabilities: 0.0720, 0.9132, 0.0149)\n","Processed DRAMP21188: AMP (Probabilities: 0.0731, 0.9123, 0.0146)\n","Processed DRAMP21187: AMP (Probabilities: 0.1197, 0.8673, 0.0130)\n","Processed DRAMP21184: Neuropeptide (Probabilities: 0.0292, 0.0366, 0.9341)\n","Processed DRAMP21178: AMP (Probabilities: 0.0742, 0.9119, 0.0140)\n","Processed DRAMP21240: AMP (Probabilities: 0.1543, 0.8320, 0.0138)\n","Processed DRAMP21238: AMP (Probabilities: 0.1000, 0.8872, 0.0128)\n","Processed DRAMP21236: AMP (Probabilities: 0.0824, 0.9044, 0.0131)\n","Processed DRAMP21235: AMP (Probabilities: 0.0749, 0.9111, 0.0140)\n","Processed DRAMP21244: AMP (Probabilities: 0.0715, 0.9138, 0.0147)\n","Processed DRAMP21245: AMP (Probabilities: 0.0734, 0.9122, 0.0144)\n","Processed DRAMP21247: AMP (Probabilities: 0.0737, 0.9119, 0.0144)\n","Processed DRAMP21248: AMP (Probabilities: 0.1142, 0.8736, 0.0122)\n","Processed DRAMP21249: AMP (Probabilities: 0.0780, 0.9089, 0.0132)\n","Processed DRAMP21253: AMP (Probabilities: 0.0725, 0.9126, 0.0149)\n","Processed DRAMP21257: AMP (Probabilities: 0.1484, 0.8377, 0.0139)\n","Processed DRAMP21261: AMP (Probabilities: 0.0718, 0.9129, 0.0152)\n","Processed DRAMP21262: AMP (Probabilities: 0.0687, 0.9157, 0.0156)\n","Processed DRAMP21264: AMP (Probabilities: 0.0713, 0.9141, 0.0146)\n","Processed DRAMP21306: AMP (Probabilities: 0.0689, 0.9155, 0.0156)\n","Processed DRAMP21307: AMP (Probabilities: 0.1617, 0.8237, 0.0146)\n","Processed DRAMP21309: AMP (Probabilities: 0.0713, 0.9134, 0.0154)\n","Processed DRAMP21311: AMP (Probabilities: 0.1710, 0.8157, 0.0132)\n","Processed DRAMP21316: AMP (Probabilities: 0.1609, 0.8250, 0.0140)\n","Processed DRAMP21319: AMP (Probabilities: 0.1418, 0.8455, 0.0126)\n","Processed DRAMP21320: AMP (Probabilities: 0.1408, 0.8468, 0.0124)\n","Processed DRAMP21321: AMP (Probabilities: 0.1541, 0.8328, 0.0131)\n","Processed DRAMP21322: AMP (Probabilities: 0.1390, 0.8484, 0.0126)\n","Processed DRAMP21325: AMP (Probabilities: 0.1223, 0.8656, 0.0120)\n","Processed DRAMP21327: AMP (Probabilities: 0.1314, 0.8560, 0.0125)\n","Processed DRAMP21329: AMP (Probabilities: 0.0982, 0.8899, 0.0119)\n","Processed DRAMP21330: AMP (Probabilities: 0.1429, 0.8444, 0.0127)\n","Processed DRAMP21331: AMP (Probabilities: 0.1443, 0.8431, 0.0126)\n","Processed DRAMP21332: AMP (Probabilities: 0.1196, 0.8683, 0.0120)\n","Processed DRAMP21350: AMP (Probabilities: 0.1173, 0.8706, 0.0121)\n","Processed DRAMP21355: AMP (Probabilities: 0.0917, 0.8965, 0.0119)\n","Processed DRAMP21358: AMP (Probabilities: 0.0694, 0.9155, 0.0151)\n","Processed DRAMP21360: AMP (Probabilities: 0.0904, 0.8976, 0.0120)\n","Processed DRAMP21362: AMP (Probabilities: 0.1575, 0.8291, 0.0134)\n","Processed DRAMP21364: AMP (Probabilities: 0.0701, 0.9151, 0.0148)\n","Processed DRAMP21369: AMP (Probabilities: 0.0706, 0.9144, 0.0150)\n","Processed DRAMP21373: AMP (Probabilities: 0.1627, 0.8232, 0.0141)\n","Processed DRAMP21375: AMP (Probabilities: 0.0728, 0.9126, 0.0146)\n","Processed DRAMP21376: AMP (Probabilities: 0.0741, 0.9116, 0.0143)\n","Processed DRAMP21377: AMP (Probabilities: 0.1390, 0.8485, 0.0126)\n","Processed DRAMP21378: AMP (Probabilities: 0.1105, 0.8774, 0.0122)\n","Processed DRAMP21379: Neuropeptide (Probabilities: 0.0227, 0.0239, 0.9534)\n","Processed DRAMP21380: AMP (Probabilities: 0.0716, 0.9144, 0.0140)\n","Processed DRAMP21382: AMP (Probabilities: 0.1060, 0.8669, 0.0271)\n","Processed DRAMP21383: AMP (Probabilities: 0.1198, 0.8650, 0.0153)\n","Processed DRAMP21384: AMP (Probabilities: 0.1502, 0.8362, 0.0137)\n","Processed DRAMP21385: AMP (Probabilities: 0.1392, 0.8482, 0.0126)\n","Processed DRAMP21386: AMP (Probabilities: 0.1609, 0.8247, 0.0145)\n","Processed DRAMP21387: AMP (Probabilities: 0.0785, 0.9084, 0.0131)\n","Processed DRAMP21391: non-AMP/OTHERS (Probabilities: 0.9737, 0.0174, 0.0089)\n","Processed DRAMP21400: AMP (Probabilities: 0.0753, 0.9112, 0.0135)\n","Processed DRAMP21401: AMP (Probabilities: 0.0775, 0.9102, 0.0124)\n","Processed DRAMP21403: non-AMP/OTHERS (Probabilities: 0.9920, 0.0055, 0.0025)\n","Processed DRAMP21404: AMP (Probabilities: 0.1709, 0.8142, 0.0149)\n","Processed DRAMP21407: AMP (Probabilities: 0.0832, 0.9037, 0.0131)\n","Processed DRAMP21408: AMP (Probabilities: 0.0660, 0.9194, 0.0147)\n","Processed DRAMP21415: AMP (Probabilities: 0.0698, 0.9153, 0.0149)\n","Processed DRAMP21416: AMP (Probabilities: 0.0743, 0.9112, 0.0145)\n","Processed DRAMP21424: AMP (Probabilities: 0.2305, 0.6256, 0.1439)\n","Processed DRAMP21426: AMP (Probabilities: 0.1276, 0.8430, 0.0294)\n","Processed DRAMP21427: AMP (Probabilities: 0.2858, 0.3675, 0.3467)\n","Processed DRAMP21428: Neuropeptide (Probabilities: 0.0061, 0.0086, 0.9853)\n","Processed DRAMP21430: Neuropeptide (Probabilities: 0.0077, 0.0113, 0.9810)\n","Processed DRAMP21433: Neuropeptide (Probabilities: 0.0742, 0.1270, 0.7988)\n","Processed DRAMP21435: AMP (Probabilities: 0.0982, 0.8830, 0.0187)\n","Processed DRAMP21446: Neuropeptide (Probabilities: 0.0182, 0.0267, 0.9551)\n","Processed DRAMP21454: AMP (Probabilities: 0.0746, 0.9113, 0.0141)\n","Processed DRAMP21455: AMP (Probabilities: 0.0678, 0.9160, 0.0161)\n","Processed DRAMP21456: AMP (Probabilities: 0.0673, 0.9171, 0.0156)\n","Processed DRAMP21457: AMP (Probabilities: 0.0685, 0.9162, 0.0152)\n","Processed DRAMP21458: AMP (Probabilities: 0.1667, 0.8189, 0.0144)\n","Processed DRAMP21459: AMP (Probabilities: 0.0688, 0.9165, 0.0148)\n","Processed DRAMP21460: AMP (Probabilities: 0.1648, 0.8207, 0.0145)\n","Processed DRAMP21461: AMP (Probabilities: 0.0688, 0.9154, 0.0158)\n","Processed DRAMP21464: AMP (Probabilities: 0.1373, 0.8505, 0.0122)\n","Processed DRAMP21465: Neuropeptide (Probabilities: 0.0248, 0.0182, 0.9570)\n","Processed DRAMP21467: AMP (Probabilities: 0.0722, 0.9135, 0.0143)\n","Processed DRAMP21577: non-AMP/OTHERS (Probabilities: 0.9210, 0.0431, 0.0359)\n","Processed DRAMP21579: non-AMP/OTHERS (Probabilities: 0.9480, 0.0231, 0.0289)\n","Processed DRAMP21581: AMP (Probabilities: 0.2250, 0.5232, 0.2519)\n","Processed DRAMP21604: Neuropeptide (Probabilities: 0.0138, 0.0186, 0.9677)\n","Processed DRAMP21606: AMP (Probabilities: 0.1325, 0.8547, 0.0128)\n","Processed DRAMP21608: AMP (Probabilities: 0.0894, 0.8983, 0.0123)\n","Processed DRAMP21610: AMP (Probabilities: 0.0976, 0.8905, 0.0119)\n","Processed DRAMP21612: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed DRAMP21614: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed DRAMP21616: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed DRAMP21618: AMP (Probabilities: 0.1273, 0.8298, 0.0429)\n","Processed DRAMP21620: non-AMP/OTHERS (Probabilities: 0.9652, 0.0300, 0.0048)\n","El número de secuencias de AMPs es: 1351\n","El número de secuencias de neuropéptidos y non-AMP es: 541\n"]}]},{"cell_type":"markdown","source":["### TEST DE VERIFICACIÓN CON NON-AMPs DIFERENTES A LOS DEL ENTRENAMIENTO"],"metadata":{"id":"oQVj99B8n_Cq"}},{"cell_type":"code","source":["import re\n","import torch\n","\n","# Función para leer un archivo FASTA\n","def read_fasta(file_path):\n","    sequences = []\n","    with open(file_path, 'r') as f:\n","        identifier = \"\"\n","        sequence = \"\"\n","        for line in f:\n","            line = line.strip()\n","            if line.startswith(\">\"):  # Es un identificador de secuencia\n","                if sequence:  # Guarda la secuencia anterior si existe\n","                    sequences.append((identifier, sequence))\n","                    sequence = \"\"\n","                identifier = line[1:]  # Guarda el identificador sin el \">\"\n","            else:\n","                sequence += line  # Agrega las líneas de la secuencia\n","        if sequence:  # Guarda la última secuencia\n","            sequences.append((identifier, sequence))\n","    return sequences\n","# Función para hacer predicciones de AMP/non-AMP para múltiples secuencias y guardar probabilidades\n","def predict_amp(sequences, output_file):\n","    i=0\n","    n=0\n","    with open(output_file, 'w') as out_f:\n","        out_f.write(\"Identifier\\tSequence\\tPrediction\\tProbability\\n\")  # Encabezados\n","\n","        for identifier, input_seq in sequences:\n","            input_seq_spaced = ' '.join([input_seq[i:i+1] for i in range(0, len(input_seq), 1)])\n","            input_seq_spaced = re.sub(r'[UZOB]', 'X', input_seq_spaced)  # Reemplazar aminoácidos no estándar\n","            input_seq_tok = tokenizer(input_seq_spaced, return_tensors='pt')\n","            #print(input_seq_tok)\n","\n","            output = model(**input_seq_tok)\n","            logits = output[0]\n","           # print(f\"Logits for {identifier}: {logits}\")\n","\n","            # Extraer la probabilidad de clase AMP\n","            #y_prob = torch.sigmoid(logits)[:, 1].detach().numpy()\n","            #y_pred = y_prob > 0.84  # Predicción binaria con umbral de 0.5\n","            #prob = y_prob[0]  # Probabilidad real (sin umbral)\n","\n","            # Aplicar softmax para obtener probabilidades de cada clase\n","            y_prob = torch.softmax(logits, dim=1).detach().numpy()\n","\n","            # Obtener la clase con la probabilidad más alta\n","            y_pred_class = y_prob.argmax(axis=1)[0]\n","            prob_class1, prob_class2, prob_class3 = y_prob[0]  # Probabilidades para las tres clases\n","            # Determinar la etiqueta de la predicción según la clase\n","            if y_pred_class == 0:\n","                input_class = 'non-AMP/OTHERS'  # Cambia estos nombres según tus etiquetas\n","                i=i+1\n","            elif y_pred_class == 1:\n","                input_class = 'AMP'\n","                n=n+1\n","            else:\n","                input_class = 'Neuropeptide'\n","                n=n+1\n","            # Escribir el identificador, secuencia, predicción y probabilidades en el archivo de salida\n","            out_f.write(f\"{identifier}\\t{input_seq}\\t{input_class}\\t{prob_class1:.4f}\\t{prob_class2:.4f}\\t{prob_class3:.4f}\\n\")\n","            print(f\"Processed {identifier}: {input_class} (Probabilities: {prob_class1:.4f}, {prob_class2:.4f}, {prob_class3:.4f})\")\n","    print(f\"El número de secuencias de Non-AMPs es: {i}\")\n","    print(f\"El número de secuencias de AMP y neuropéptidos es: {n}\")\n","\n","            # Determinar clase (AMP o non-AMP)\n","            #if y_pred:\n","            #    input_class = 'AMP'\n","            #else:\n","            #    input_class = 'non-AMP'\n","\n","            # Escribir el identificador, secuencia, predicción y probabilidad en el archivo de salida\n","           # out_f.write(f\"{identifier}\\t{input_seq}\\t{input_class}\\t{prob:.4f}\\n\")\n","           # print(f\"Processed {identifier}: {input_class} (Probability: {prob:.4f})\")\n","\n","# Ruta del archivo FASTA de entrada y del archivo de salida\n","fasta_file = '/content/drive/MyDrive/data/non_amp_verificacion.fasta'  # Nombre del archivo FASTA con secuencias...\n","output_file = '/content/drive/MyDrive/Datos Finales para Memoria/resultados/RESULTADO_verificacion_non-AMPs.tsv'  # Archivo de salida con predicciones y probabilidades\n","\n","# Leer las secuencias del archivo FASTA\n","sequences = read_fasta(fasta_file)\n","\n","# Realizar predicciones y escribir resultados en un archivo\n","predict_amp(sequences, output_file)\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1733823375555,"user_tz":-60,"elapsed":284846,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"4445805b-e437-4ba1-84bb-68826fefe767","id":"SD1T_uH7oF1M"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Processed non_AMPEP82130: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9849)\n","Processed non_AMPEP54142: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed non_AMPEP137679: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed non_AMPEP77131: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed non_AMPEP100173: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed non_AMPEP49224: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed non_AMPEP9780: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9850)\n","Processed non_AMPEP100390: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9851)\n","Processed non_AMPEP119666: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9850)\n","Processed non_AMPEP2676: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed non_AMPEP1965: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed non_AMPEP133341: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed non_AMPEP144088: Neuropeptide (Probabilities: 0.0121, 0.0156, 0.9723)\n","Processed non_AMPEP162641: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed non_AMPEP91917: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9849)\n","Processed non_AMPEP20482: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed non_AMPEP43516: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed non_AMPEP110561: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed non_AMPEP105491: Neuropeptide (Probabilities: 0.0789, 0.0789, 0.8422)\n","Processed non_AMPEP36699: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed non_AMPEP79757: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed non_AMPEP146506: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed non_AMPEP128290: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed non_AMPEP41609: AMP (Probabilities: 0.2071, 0.5986, 0.1943)\n","Processed non_AMPEP93479: Neuropeptide (Probabilities: 0.0066, 0.0088, 0.9846)\n","Processed non_AMPEP146877: non-AMP/OTHERS (Probabilities: 0.9865, 0.0093, 0.0043)\n","Processed non_AMPEP48525: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed non_AMPEP60572: non-AMP/OTHERS (Probabilities: 0.9876, 0.0047, 0.0077)\n","Processed non_AMPEP153436: Neuropeptide (Probabilities: 0.4421, 0.1126, 0.4453)\n","Processed non_AMPEP103050: Neuropeptide (Probabilities: 0.0063, 0.0086, 0.9851)\n","Processed non_AMPEP18731: non-AMP/OTHERS (Probabilities: 0.9929, 0.0038, 0.0032)\n","Processed non_AMPEP67786: AMP (Probabilities: 0.0697, 0.9154, 0.0149)\n","Processed non_AMPEP136654: Neuropeptide (Probabilities: 0.0132, 0.0121, 0.9747)\n","Processed non_AMPEP39565: AMP (Probabilities: 0.0647, 0.9200, 0.0153)\n","Processed non_AMPEP142377: non-AMP/OTHERS (Probabilities: 0.9368, 0.0074, 0.0558)\n","Processed non_AMPEP64127: non-AMP/OTHERS (Probabilities: 0.9694, 0.0051, 0.0254)\n","Processed non_AMPEP126743: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed non_AMPEP42233: non-AMP/OTHERS (Probabilities: 0.7575, 0.1154, 0.1272)\n","Processed non_AMPEP93971: non-AMP/OTHERS (Probabilities: 0.9879, 0.0031, 0.0090)\n","Processed non_AMPEP57347: non-AMP/OTHERS (Probabilities: 0.9894, 0.0057, 0.0049)\n","Processed non_AMPEP119303: AMP (Probabilities: 0.1746, 0.8020, 0.0234)\n","Processed non_AMPEP71007: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed non_AMPEP86896: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed non_AMPEP164292: Neuropeptide (Probabilities: 0.0247, 0.0142, 0.9612)\n","Processed non_AMPEP123655: Neuropeptide (Probabilities: 0.1834, 0.0275, 0.7890)\n","Processed non_AMPEP13673: Neuropeptide (Probabilities: 0.0101, 0.0095, 0.9804)\n","Processed non_AMPEP43195: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed non_AMPEP125275: AMP (Probabilities: 0.2260, 0.4414, 0.3327)\n","Processed non_AMPEP19664: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9851)\n","Processed non_AMPEP78324: AMP (Probabilities: 0.1573, 0.7969, 0.0458)\n","Processed non_AMPEP129160: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed non_AMPEP30281: Neuropeptide (Probabilities: 0.0069, 0.0085, 0.9846)\n","Processed non_AMPEP7232: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed non_AMPEP67360: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed non_AMPEP78968: Neuropeptide (Probabilities: 0.0073, 0.0087, 0.9840)\n","Processed non_AMPEP107398: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9852)\n","Processed non_AMPEP66648: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed non_AMPEP46216: Neuropeptide (Probabilities: 0.2379, 0.0783, 0.6838)\n","Processed non_AMPEP142514: Neuropeptide (Probabilities: 0.0071, 0.0089, 0.9840)\n","Processed non_AMPEP89101: Neuropeptide (Probabilities: 0.0181, 0.0126, 0.9693)\n","Processed non_AMPEP78245: Neuropeptide (Probabilities: 0.0068, 0.0096, 0.9836)\n","Processed non_AMPEP72707: non-AMP/OTHERS (Probabilities: 0.9848, 0.0103, 0.0049)\n","Processed non_AMPEP141745: Neuropeptide (Probabilities: 0.0177, 0.0122, 0.9701)\n","Processed non_AMPEP156987: non-AMP/OTHERS (Probabilities: 0.7372, 0.2268, 0.0359)\n","Processed non_AMPEP28234: non-AMP/OTHERS (Probabilities: 0.5213, 0.0173, 0.4614)\n","Processed non_AMPEP58969: Neuropeptide (Probabilities: 0.0101, 0.0095, 0.9804)\n","Processed non_AMPEP144909: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed non_AMPEP74343: AMP (Probabilities: 0.1926, 0.7633, 0.0441)\n","Processed non_AMPEP104693: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed non_AMPEP28403: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed non_AMPEP75445: AMP (Probabilities: 0.0845, 0.9034, 0.0120)\n","Processed non_AMPEP51852: non-AMP/OTHERS (Probabilities: 0.9884, 0.0052, 0.0064)\n","Processed non_AMPEP137432: non-AMP/OTHERS (Probabilities: 0.9903, 0.0047, 0.0050)\n","Processed non_AMPEP164663: Neuropeptide (Probabilities: 0.0063, 0.0089, 0.9848)\n","Processed non_AMPEP8371: non-AMP/OTHERS (Probabilities: 0.9600, 0.0225, 0.0174)\n","Processed non_AMPEP137246: non-AMP/OTHERS (Probabilities: 0.9907, 0.0035, 0.0058)\n","Processed non_AMPEP11798: Neuropeptide (Probabilities: 0.0065, 0.0085, 0.9849)\n","Processed non_AMPEP132505: Neuropeptide (Probabilities: 0.0062, 0.0087, 0.9851)\n","Processed non_AMPEP137481: Neuropeptide (Probabilities: 0.3983, 0.0388, 0.5629)\n","Processed non_AMPEP137748: non-AMP/OTHERS (Probabilities: 0.8406, 0.0649, 0.0945)\n","Processed non_AMPEP91738: non-AMP/OTHERS (Probabilities: 0.9921, 0.0032, 0.0047)\n","Processed non_AMPEP151116: Neuropeptide (Probabilities: 0.0060, 0.0090, 0.9850)\n","Processed non_AMPEP98525: non-AMP/OTHERS (Probabilities: 0.5363, 0.0655, 0.3982)\n","Processed non_AMPEP83040: AMP (Probabilities: 0.1626, 0.8235, 0.0139)\n","Processed non_AMPEP126126: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed non_AMPEP137535: AMP (Probabilities: 0.1678, 0.8184, 0.0137)\n","Processed non_AMPEP43306: AMP (Probabilities: 0.0797, 0.9074, 0.0129)\n","Processed non_AMPEP35019: Neuropeptide (Probabilities: 0.0062, 0.0087, 0.9851)\n","Processed non_AMPEP165264: non-AMP/OTHERS (Probabilities: 0.7765, 0.0412, 0.1823)\n","Processed non_AMPEP25815: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed non_AMPEP107426: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed non_AMPEP122359: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed non_AMPEP51869: Neuropeptide (Probabilities: 0.0195, 0.0174, 0.9631)\n","Processed non_AMPEP124114: Neuropeptide (Probabilities: 0.0064, 0.0085, 0.9850)\n","Processed non_AMPEP6140: non-AMP/OTHERS (Probabilities: 0.5953, 0.3805, 0.0241)\n","Processed non_AMPEP113213: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed non_AMPEP148876: AMP (Probabilities: 0.0684, 0.9159, 0.0157)\n","Processed non_AMPEP56104: non-AMP/OTHERS (Probabilities: 0.9907, 0.0062, 0.0031)\n","Processed non_AMPEP10745: Neuropeptide (Probabilities: 0.0109, 0.0099, 0.9792)\n","Processed non_AMPEP34833: Neuropeptide (Probabilities: 0.1347, 0.1688, 0.6965)\n","Processed non_AMPEP51456: Neuropeptide (Probabilities: 0.0138, 0.0139, 0.9722)\n","Processed non_AMPEP11064: Neuropeptide (Probabilities: 0.0088, 0.0092, 0.9820)\n","Processed non_AMPEP68483: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed non_AMPEP154934: Neuropeptide (Probabilities: 0.0081, 0.0090, 0.9829)\n","Processed non_AMPEP48567: Neuropeptide (Probabilities: 0.0062, 0.0085, 0.9853)\n","Processed non_AMPEP43390: AMP (Probabilities: 0.2400, 0.5760, 0.1840)\n","Processed non_AMPEP71630: Neuropeptide (Probabilities: 0.0154, 0.0123, 0.9722)\n","Processed non_AMPEP93821: Neuropeptide (Probabilities: 0.0767, 0.0579, 0.8654)\n","Processed non_AMPEP48905: non-AMP/OTHERS (Probabilities: 0.9410, 0.0077, 0.0513)\n","Processed non_AMPEP10485: Neuropeptide (Probabilities: 0.0061, 0.0086, 0.9853)\n","Processed non_AMPEP52122: Neuropeptide (Probabilities: 0.0066, 0.0091, 0.9844)\n","Processed non_AMPEP130946: non-AMP/OTHERS (Probabilities: 0.9601, 0.0063, 0.0335)\n","Processed non_AMPEP134790: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed non_AMPEP74023: Neuropeptide (Probabilities: 0.0073, 0.0089, 0.9838)\n","Processed non_AMPEP136664: AMP (Probabilities: 0.0861, 0.9015, 0.0123)\n","Processed non_AMPEP135992: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed non_AMPEP154081: AMP (Probabilities: 0.0774, 0.9082, 0.0144)\n","Processed non_AMPEP127089: AMP (Probabilities: 0.0857, 0.9024, 0.0120)\n","Processed non_AMPEP103622: non-AMP/OTHERS (Probabilities: 0.4943, 0.4282, 0.0775)\n","Processed non_AMPEP25809: Neuropeptide (Probabilities: 0.0366, 0.0187, 0.9447)\n","Processed non_AMPEP154558: Neuropeptide (Probabilities: 0.0096, 0.0121, 0.9783)\n","Processed non_AMPEP29171: Neuropeptide (Probabilities: 0.0070, 0.0089, 0.9842)\n","Processed non_AMPEP69532: Neuropeptide (Probabilities: 0.0108, 0.0100, 0.9791)\n","Processed non_AMPEP96205: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed non_AMPEP109705: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9852)\n","Processed non_AMPEP146914: non-AMP/OTHERS (Probabilities: 0.9356, 0.0316, 0.0328)\n","Processed non_AMPEP111388: Neuropeptide (Probabilities: 0.0063, 0.0087, 0.9850)\n","Processed non_AMPEP154616: non-AMP/OTHERS (Probabilities: 0.7686, 0.0167, 0.2147)\n","Processed non_AMPEP65949: AMP (Probabilities: 0.1020, 0.8857, 0.0123)\n","Processed non_AMPEP52319: non-AMP/OTHERS (Probabilities: 0.9922, 0.0048, 0.0030)\n","Processed non_AMPEP37310: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed non_AMPEP68668: Neuropeptide (Probabilities: 0.0083, 0.0105, 0.9812)\n","Processed non_AMPEP11235: Neuropeptide (Probabilities: 0.0109, 0.0096, 0.9795)\n","Processed non_AMPEP47288: non-AMP/OTHERS (Probabilities: 0.6241, 0.0265, 0.3494)\n","Processed non_AMPEP41523: AMP (Probabilities: 0.0675, 0.9173, 0.0152)\n","Processed non_AMPEP113499: AMP (Probabilities: 0.0677, 0.9166, 0.0157)\n","Processed non_AMPEP19052: non-AMP/OTHERS (Probabilities: 0.9543, 0.0091, 0.0365)\n","Processed non_AMPEP155281: AMP (Probabilities: 0.1338, 0.8340, 0.0322)\n","Processed non_AMPEP126421: Neuropeptide (Probabilities: 0.0062, 0.0087, 0.9851)\n","Processed non_AMPEP58958: AMP (Probabilities: 0.0876, 0.8981, 0.0143)\n","Processed non_AMPEP63666: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed non_AMPEP15561: AMP (Probabilities: 0.0711, 0.9139, 0.0149)\n","Processed non_AMPEP147044: AMP (Probabilities: 0.0706, 0.9148, 0.0146)\n","Processed non_AMPEP24317: AMP (Probabilities: 0.1280, 0.8507, 0.0213)\n","Processed non_AMPEP103637: Neuropeptide (Probabilities: 0.0167, 0.0164, 0.9669)\n","Processed non_AMPEP77825: Neuropeptide (Probabilities: 0.0075, 0.0090, 0.9836)\n","Processed non_AMPEP130635: AMP (Probabilities: 0.0699, 0.9149, 0.0152)\n","Processed non_AMPEP103794: AMP (Probabilities: 0.1804, 0.6531, 0.1664)\n","Processed non_AMPEP31306: non-AMP/OTHERS (Probabilities: 0.9905, 0.0034, 0.0061)\n","Processed non_AMPEP129109: AMP (Probabilities: 0.0678, 0.9164, 0.0158)\n","Processed non_AMPEP129083: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed non_AMPEP99682: AMP (Probabilities: 0.2439, 0.5947, 0.1615)\n","Processed non_AMPEP16193: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0034)\n","Processed non_AMPEP73195: non-AMP/OTHERS (Probabilities: 0.9917, 0.0032, 0.0051)\n","Processed non_AMPEP32286: Neuropeptide (Probabilities: 0.0308, 0.0205, 0.9487)\n","Processed non_AMPEP82143: non-AMP/OTHERS (Probabilities: 0.9899, 0.0050, 0.0050)\n","Processed non_AMPEP72437: non-AMP/OTHERS (Probabilities: 0.9848, 0.0080, 0.0072)\n","Processed non_AMPEP4574: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed non_AMPEP157396: non-AMP/OTHERS (Probabilities: 0.8943, 0.0134, 0.0923)\n","Processed non_AMPEP164710: AMP (Probabilities: 0.0688, 0.9154, 0.0158)\n","Processed non_AMPEP12313: AMP (Probabilities: 0.0678, 0.9160, 0.0162)\n","Processed non_AMPEP164509: AMP (Probabilities: 0.2276, 0.4515, 0.3208)\n","Processed non_AMPEP156613: non-AMP/OTHERS (Probabilities: 0.9903, 0.0031, 0.0065)\n","Processed non_AMPEP159505: AMP (Probabilities: 0.0681, 0.9164, 0.0155)\n","Processed non_AMPEP106486: Neuropeptide (Probabilities: 0.0084, 0.0092, 0.9824)\n","Processed non_AMPEP164589: AMP (Probabilities: 0.1439, 0.8428, 0.0133)\n","Processed non_AMPEP149113: Neuropeptide (Probabilities: 0.0327, 0.0185, 0.9488)\n","Processed non_AMPEP102363: non-AMP/OTHERS (Probabilities: 0.9830, 0.0058, 0.0112)\n","Processed non_AMPEP134555: AMP (Probabilities: 0.0806, 0.9058, 0.0136)\n","Processed non_AMPEP142360: AMP (Probabilities: 0.0686, 0.9156, 0.0157)\n","Processed non_AMPEP58272: Neuropeptide (Probabilities: 0.0636, 0.0279, 0.9085)\n","Processed non_AMPEP105596: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed non_AMPEP86459: AMP (Probabilities: 0.1527, 0.8338, 0.0134)\n","Processed non_AMPEP75158: Neuropeptide (Probabilities: 0.0108, 0.0108, 0.9783)\n","Processed non_AMPEP113426: non-AMP/OTHERS (Probabilities: 0.9349, 0.0087, 0.0564)\n","Processed non_AMPEP114112: non-AMP/OTHERS (Probabilities: 0.9883, 0.0069, 0.0048)\n","Processed non_AMPEP80609: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed non_AMPEP55063: AMP (Probabilities: 0.0777, 0.9082, 0.0141)\n","Processed non_AMPEP13118: Neuropeptide (Probabilities: 0.2182, 0.0401, 0.7417)\n","Processed non_AMPEP142010: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed non_AMPEP122652: AMP (Probabilities: 0.0684, 0.9168, 0.0148)\n","Processed non_AMPEP93851: non-AMP/OTHERS (Probabilities: 0.9911, 0.0034, 0.0056)\n","Processed non_AMPEP62588: AMP (Probabilities: 0.3422, 0.3669, 0.2909)\n","Processed non_AMPEP87320: AMP (Probabilities: 0.0675, 0.9163, 0.0162)\n","Processed non_AMPEP165780: AMP (Probabilities: 0.0675, 0.9169, 0.0156)\n","Processed non_AMPEP155001: Neuropeptide (Probabilities: 0.0123, 0.0136, 0.9742)\n","Processed non_AMPEP9661: non-AMP/OTHERS (Probabilities: 0.9885, 0.0065, 0.0050)\n","Processed non_AMPEP80582: Neuropeptide (Probabilities: 0.0063, 0.0089, 0.9848)\n","Processed non_AMPEP137696: non-AMP/OTHERS (Probabilities: 0.9921, 0.0031, 0.0048)\n","Processed non_AMPEP99497: non-AMP/OTHERS (Probabilities: 0.9469, 0.0063, 0.0469)\n","Processed non_AMPEP12712: AMP (Probabilities: 0.0705, 0.9151, 0.0145)\n","Processed non_AMPEP153850: AMP (Probabilities: 0.0674, 0.9168, 0.0158)\n","Processed non_AMPEP13815: non-AMP/OTHERS (Probabilities: 0.9209, 0.0603, 0.0189)\n","Processed non_AMPEP165199: non-AMP/OTHERS (Probabilities: 0.7794, 0.0108, 0.2098)\n","Processed non_AMPEP103613: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed non_AMPEP152171: non-AMP/OTHERS (Probabilities: 0.8974, 0.0098, 0.0928)\n","Processed non_AMPEP54525: AMP (Probabilities: 0.0665, 0.9177, 0.0157)\n","Processed non_AMPEP12457: AMP (Probabilities: 0.1000, 0.8881, 0.0119)\n","Processed non_AMPEP66501: AMP (Probabilities: 0.0708, 0.9142, 0.0150)\n","Processed non_AMPEP50190: AMP (Probabilities: 0.0690, 0.9155, 0.0155)\n","Processed non_AMPEP77661: Neuropeptide (Probabilities: 0.0080, 0.0091, 0.9829)\n","Processed non_AMPEP4070: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed non_AMPEP144453: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed non_AMPEP156669: AMP (Probabilities: 0.0711, 0.9140, 0.0149)\n","Processed non_AMPEP50629: Neuropeptide (Probabilities: 0.0261, 0.0133, 0.9606)\n","Processed non_AMPEP121365: AMP (Probabilities: 0.2882, 0.6727, 0.0391)\n","Processed non_AMPEP152707: AMP (Probabilities: 0.0688, 0.9154, 0.0157)\n","Processed non_AMPEP15796: Neuropeptide (Probabilities: 0.0061, 0.0086, 0.9853)\n","Processed non_AMPEP115353: AMP (Probabilities: 0.0817, 0.9047, 0.0136)\n","Processed non_AMPEP44049: non-AMP/OTHERS (Probabilities: 0.9416, 0.0481, 0.0102)\n","Processed non_AMPEP7884: Neuropeptide (Probabilities: 0.0080, 0.0090, 0.9830)\n","Processed non_AMPEP63124: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed non_AMPEP134306: Neuropeptide (Probabilities: 0.2000, 0.0651, 0.7350)\n","Processed non_AMPEP76735: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed non_AMPEP37570: non-AMP/OTHERS (Probabilities: 0.9911, 0.0047, 0.0042)\n","Processed non_AMPEP121053: Neuropeptide (Probabilities: 0.0082, 0.0098, 0.9820)\n","Processed non_AMPEP8546: non-AMP/OTHERS (Probabilities: 0.5548, 0.0212, 0.4240)\n","Processed non_AMPEP113749: Neuropeptide (Probabilities: 0.0154, 0.0141, 0.9705)\n","Processed non_AMPEP44966: non-AMP/OTHERS (Probabilities: 0.9839, 0.0041, 0.0121)\n","Processed non_AMPEP96654: non-AMP/OTHERS (Probabilities: 0.9860, 0.0047, 0.0093)\n","Processed non_AMPEP87759: AMP (Probabilities: 0.1789, 0.8058, 0.0153)\n","Processed non_AMPEP83919: non-AMP/OTHERS (Probabilities: 0.8040, 0.0564, 0.1396)\n","Processed non_AMPEP146832: AMP (Probabilities: 0.1726, 0.8124, 0.0150)\n","Processed non_AMPEP60295: AMP (Probabilities: 0.1634, 0.8221, 0.0145)\n","Processed non_AMPEP84996: non-AMP/OTHERS (Probabilities: 0.9895, 0.0035, 0.0070)\n","Processed non_AMPEP61872: non-AMP/OTHERS (Probabilities: 0.9910, 0.0049, 0.0041)\n","Processed non_AMPEP78403: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed non_AMPEP44252: non-AMP/OTHERS (Probabilities: 0.9765, 0.0040, 0.0194)\n","Processed non_AMPEP89574: non-AMP/OTHERS (Probabilities: 0.8514, 0.0346, 0.1140)\n","Processed non_AMPEP101160: non-AMP/OTHERS (Probabilities: 0.5766, 0.3967, 0.0267)\n","Processed non_AMPEP53551: AMP (Probabilities: 0.0733, 0.9122, 0.0145)\n","Processed non_AMPEP20703: non-AMP/OTHERS (Probabilities: 0.9902, 0.0046, 0.0051)\n","Processed non_AMPEP136400: AMP (Probabilities: 0.0764, 0.9097, 0.0138)\n","Processed non_AMPEP82416: non-AMP/OTHERS (Probabilities: 0.9928, 0.0034, 0.0038)\n","Processed non_AMPEP28940: Neuropeptide (Probabilities: 0.0066, 0.0089, 0.9846)\n","Processed non_AMPEP8912: Neuropeptide (Probabilities: 0.1981, 0.3153, 0.4866)\n","Processed non_AMPEP139513: non-AMP/OTHERS (Probabilities: 0.9081, 0.0763, 0.0156)\n","Processed non_AMPEP28749: non-AMP/OTHERS (Probabilities: 0.9416, 0.0137, 0.0447)\n","Processed non_AMPEP52883: Neuropeptide (Probabilities: 0.0470, 0.0296, 0.9234)\n","Processed non_AMPEP113075: non-AMP/OTHERS (Probabilities: 0.8908, 0.0156, 0.0937)\n","Processed non_AMPEP106705: AMP (Probabilities: 0.0701, 0.9145, 0.0154)\n","Processed non_AMPEP29861: AMP (Probabilities: 0.0710, 0.9138, 0.0152)\n","Processed non_AMPEP148544: AMP (Probabilities: 0.0765, 0.9095, 0.0141)\n","Processed non_AMPEP94829: AMP (Probabilities: 0.1473, 0.8399, 0.0127)\n","Processed non_AMPEP30414: non-AMP/OTHERS (Probabilities: 0.9206, 0.0105, 0.0689)\n","Processed non_AMPEP140337: Neuropeptide (Probabilities: 0.0151, 0.0129, 0.9720)\n","Processed non_AMPEP145167: AMP (Probabilities: 0.0666, 0.9179, 0.0154)\n","Processed non_AMPEP66800: non-AMP/OTHERS (Probabilities: 0.9911, 0.0038, 0.0051)\n","Processed non_AMPEP23417: Neuropeptide (Probabilities: 0.0068, 0.0092, 0.9840)\n","Processed non_AMPEP93316: AMP (Probabilities: 0.1891, 0.6267, 0.1842)\n","Processed non_AMPEP70205: AMP (Probabilities: 0.2310, 0.7538, 0.0151)\n","Processed non_AMPEP106085: Neuropeptide (Probabilities: 0.0234, 0.0130, 0.9637)\n","Processed non_AMPEP124419: non-AMP/OTHERS (Probabilities: 0.7210, 0.0219, 0.2570)\n","Processed non_AMPEP150981: Neuropeptide (Probabilities: 0.0158, 0.0116, 0.9726)\n","Processed non_AMPEP51380: Neuropeptide (Probabilities: 0.2588, 0.1681, 0.5730)\n","Processed non_AMPEP14807: non-AMP/OTHERS (Probabilities: 0.9055, 0.0351, 0.0595)\n","Processed non_AMPEP67362: Neuropeptide (Probabilities: 0.0132, 0.0128, 0.9740)\n","Processed non_AMPEP125963: non-AMP/OTHERS (Probabilities: 0.3768, 0.3718, 0.2514)\n","Processed non_AMPEP161119: Neuropeptide (Probabilities: 0.0109, 0.0119, 0.9771)\n","Processed non_AMPEP135109: non-AMP/OTHERS (Probabilities: 0.9708, 0.0238, 0.0054)\n","Processed non_AMPEP94923: non-AMP/OTHERS (Probabilities: 0.9919, 0.0042, 0.0039)\n","Processed non_AMPEP64371: Neuropeptide (Probabilities: 0.0152, 0.0167, 0.9680)\n","Processed non_AMPEP120842: AMP (Probabilities: 0.2601, 0.7157, 0.0241)\n","Processed non_AMPEP140265: Neuropeptide (Probabilities: 0.0069, 0.0088, 0.9842)\n","Processed non_AMPEP60204: non-AMP/OTHERS (Probabilities: 0.7747, 0.0230, 0.2023)\n","Processed non_AMPEP128277: AMP (Probabilities: 0.4624, 0.4755, 0.0621)\n","Processed non_AMPEP59541: AMP (Probabilities: 0.0720, 0.9136, 0.0145)\n","Processed non_AMPEP155075: non-AMP/OTHERS (Probabilities: 0.9552, 0.0143, 0.0306)\n","Processed non_AMPEP127634: AMP (Probabilities: 0.0730, 0.9122, 0.0148)\n","Processed non_AMPEP110786: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed non_AMPEP74194: non-AMP/OTHERS (Probabilities: 0.9872, 0.0033, 0.0095)\n","Processed non_AMPEP8133: Neuropeptide (Probabilities: 0.0185, 0.0117, 0.9698)\n","Processed non_AMPEP115914: Neuropeptide (Probabilities: 0.1080, 0.0151, 0.8768)\n","Processed non_AMPEP142403: Neuropeptide (Probabilities: 0.0639, 0.0173, 0.9188)\n","Processed non_AMPEP106295: Neuropeptide (Probabilities: 0.0063, 0.0085, 0.9852)\n","Processed non_AMPEP4392: Neuropeptide (Probabilities: 0.0083, 0.0095, 0.9822)\n","Processed non_AMPEP38769: non-AMP/OTHERS (Probabilities: 0.4935, 0.4787, 0.0277)\n","Processed non_AMPEP154965: Neuropeptide (Probabilities: 0.0075, 0.0088, 0.9837)\n","Processed non_AMPEP147299: Neuropeptide (Probabilities: 0.0125, 0.0109, 0.9766)\n","Processed non_AMPEP110784: Neuropeptide (Probabilities: 0.0085, 0.0091, 0.9824)\n","Processed non_AMPEP127471: non-AMP/OTHERS (Probabilities: 0.7018, 0.2641, 0.0341)\n","Processed non_AMPEP132524: Neuropeptide (Probabilities: 0.0121, 0.0108, 0.9771)\n","Processed non_AMPEP81147: non-AMP/OTHERS (Probabilities: 0.9466, 0.0337, 0.0197)\n","Processed non_AMPEP54001: non-AMP/OTHERS (Probabilities: 0.9908, 0.0043, 0.0049)\n","Processed non_AMPEP150638: Neuropeptide (Probabilities: 0.1722, 0.0267, 0.8012)\n","Processed non_AMPEP35673: non-AMP/OTHERS (Probabilities: 0.9598, 0.0285, 0.0117)\n","Processed non_AMPEP11301: non-AMP/OTHERS (Probabilities: 0.5006, 0.3149, 0.1844)\n","Processed non_AMPEP89804: AMP (Probabilities: 0.2075, 0.7397, 0.0528)\n","Processed non_AMPEP121525: non-AMP/OTHERS (Probabilities: 0.7083, 0.0219, 0.2699)\n","Processed non_AMPEP98145: non-AMP/OTHERS (Probabilities: 0.9371, 0.0087, 0.0541)\n","Processed non_AMPEP80081: non-AMP/OTHERS (Probabilities: 0.9905, 0.0050, 0.0045)\n","Processed non_AMPEP46270: non-AMP/OTHERS (Probabilities: 0.9888, 0.0078, 0.0035)\n","Processed non_AMPEP117206: Neuropeptide (Probabilities: 0.0804, 0.0796, 0.8399)\n","Processed non_AMPEP14462: non-AMP/OTHERS (Probabilities: 0.7078, 0.1895, 0.1028)\n","Processed non_AMPEP13244: non-AMP/OTHERS (Probabilities: 0.9573, 0.0067, 0.0360)\n","Processed non_AMPEP114900: Neuropeptide (Probabilities: 0.0137, 0.0118, 0.9745)\n","Processed non_AMPEP6320: non-AMP/OTHERS (Probabilities: 0.9896, 0.0049, 0.0055)\n","Processed non_AMPEP129245: non-AMP/OTHERS (Probabilities: 0.9641, 0.0121, 0.0238)\n","Processed non_AMPEP154374: non-AMP/OTHERS (Probabilities: 0.9902, 0.0046, 0.0052)\n","Processed non_AMPEP80924: Neuropeptide (Probabilities: 0.0165, 0.0148, 0.9687)\n","Processed non_AMPEP69914: Neuropeptide (Probabilities: 0.3020, 0.0786, 0.6194)\n","Processed non_AMPEP58303: non-AMP/OTHERS (Probabilities: 0.9918, 0.0033, 0.0049)\n","Processed non_AMPEP31091: AMP (Probabilities: 0.2911, 0.3648, 0.3441)\n","Processed non_AMPEP159613: non-AMP/OTHERS (Probabilities: 0.9913, 0.0040, 0.0047)\n","Processed non_AMPEP102431: non-AMP/OTHERS (Probabilities: 0.9910, 0.0043, 0.0048)\n","Processed non_AMPEP153819: Neuropeptide (Probabilities: 0.3579, 0.1191, 0.5230)\n","Processed non_AMPEP76362: Neuropeptide (Probabilities: 0.0085, 0.0091, 0.9824)\n","Processed non_AMPEP135696: AMP (Probabilities: 0.1714, 0.8140, 0.0146)\n","Processed non_AMPEP146752: AMP (Probabilities: 0.0722, 0.9137, 0.0141)\n","Processed non_AMPEP13883: non-AMP/OTHERS (Probabilities: 0.8927, 0.0811, 0.0263)\n","Processed non_AMPEP112801: Neuropeptide (Probabilities: 0.0206, 0.0146, 0.9648)\n","Processed non_AMPEP81554: Neuropeptide (Probabilities: 0.0094, 0.0096, 0.9811)\n","Processed non_AMPEP120941: AMP (Probabilities: 0.1238, 0.8447, 0.0315)\n","Processed non_AMPEP152953: non-AMP/OTHERS (Probabilities: 0.8739, 0.0113, 0.1148)\n","Processed non_AMPEP18037: AMP (Probabilities: 0.1670, 0.8189, 0.0141)\n","Processed non_AMPEP114725: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed non_AMPEP69241: non-AMP/OTHERS (Probabilities: 0.9900, 0.0042, 0.0058)\n","Processed non_AMPEP38805: non-AMP/OTHERS (Probabilities: 0.7841, 0.1893, 0.0266)\n","Processed non_AMPEP120768: non-AMP/OTHERS (Probabilities: 0.9884, 0.0056, 0.0060)\n","Processed non_AMPEP4257: non-AMP/OTHERS (Probabilities: 0.9918, 0.0038, 0.0045)\n","Processed non_AMPEP82618: non-AMP/OTHERS (Probabilities: 0.9914, 0.0056, 0.0030)\n","Processed non_AMPEP53340: Neuropeptide (Probabilities: 0.0893, 0.0354, 0.8753)\n","Processed non_AMPEP42699: AMP (Probabilities: 0.0697, 0.9145, 0.0159)\n","Processed non_AMPEP55811: Neuropeptide (Probabilities: 0.2311, 0.0685, 0.7004)\n","Processed non_AMPEP21094: non-AMP/OTHERS (Probabilities: 0.9901, 0.0050, 0.0049)\n","Processed non_AMPEP36638: AMP (Probabilities: 0.0714, 0.9139, 0.0147)\n","Processed non_AMPEP12676: Neuropeptide (Probabilities: 0.0270, 0.0140, 0.9590)\n","Processed non_AMPEP93143: Neuropeptide (Probabilities: 0.0365, 0.0168, 0.9467)\n","Processed non_AMPEP69187: non-AMP/OTHERS (Probabilities: 0.9848, 0.0083, 0.0069)\n","Processed non_AMPEP18904: non-AMP/OTHERS (Probabilities: 0.9907, 0.0047, 0.0046)\n","Processed non_AMPEP23675: non-AMP/OTHERS (Probabilities: 0.5193, 0.0693, 0.4114)\n","Processed non_AMPEP90604: non-AMP/OTHERS (Probabilities: 0.9703, 0.0194, 0.0103)\n","Processed non_AMPEP52184: Neuropeptide (Probabilities: 0.0548, 0.0169, 0.9283)\n","Processed non_AMPEP45922: AMP (Probabilities: 0.1180, 0.7724, 0.1095)\n","Processed non_AMPEP149580: non-AMP/OTHERS (Probabilities: 0.7100, 0.0257, 0.2643)\n","Processed non_AMPEP7726: AMP (Probabilities: 0.1684, 0.8174, 0.0142)\n","Processed non_AMPEP106698: non-AMP/OTHERS (Probabilities: 0.9922, 0.0032, 0.0046)\n","Processed non_AMPEP126790: non-AMP/OTHERS (Probabilities: 0.9837, 0.0099, 0.0064)\n","Processed non_AMPEP147573: non-AMP/OTHERS (Probabilities: 0.9665, 0.0210, 0.0125)\n","Processed non_AMPEP30442: non-AMP/OTHERS (Probabilities: 0.5730, 0.0302, 0.3968)\n","Processed non_AMPEP138542: Neuropeptide (Probabilities: 0.0070, 0.0088, 0.9842)\n","Processed non_AMPEP105081: Neuropeptide (Probabilities: 0.0135, 0.0122, 0.9743)\n","Processed non_AMPEP118740: non-AMP/OTHERS (Probabilities: 0.9799, 0.0053, 0.0148)\n","Processed non_AMPEP105028: non-AMP/OTHERS (Probabilities: 0.9569, 0.0128, 0.0303)\n","Processed non_AMPEP158112: non-AMP/OTHERS (Probabilities: 0.9802, 0.0052, 0.0146)\n","Processed non_AMPEP129395: non-AMP/OTHERS (Probabilities: 0.9900, 0.0066, 0.0034)\n","Processed non_AMPEP28836: Neuropeptide (Probabilities: 0.0074, 0.0090, 0.9836)\n","Processed non_AMPEP16212: Neuropeptide (Probabilities: 0.0064, 0.0087, 0.9849)\n","Processed non_AMPEP54341: Neuropeptide (Probabilities: 0.0070, 0.0092, 0.9837)\n","Processed non_AMPEP43255: non-AMP/OTHERS (Probabilities: 0.9923, 0.0035, 0.0041)\n","Processed non_AMPEP44585: non-AMP/OTHERS (Probabilities: 0.9927, 0.0040, 0.0033)\n","Processed non_AMPEP55742: non-AMP/OTHERS (Probabilities: 0.9919, 0.0028, 0.0053)\n","Processed non_AMPEP17715: non-AMP/OTHERS (Probabilities: 0.9745, 0.0210, 0.0046)\n","Processed non_AMPEP84934: Neuropeptide (Probabilities: 0.0516, 0.0251, 0.9233)\n","Processed non_AMPEP112211: non-AMP/OTHERS (Probabilities: 0.9906, 0.0028, 0.0066)\n","Processed non_AMPEP80106: non-AMP/OTHERS (Probabilities: 0.9853, 0.0050, 0.0097)\n","Processed non_AMPEP46130: non-AMP/OTHERS (Probabilities: 0.9908, 0.0051, 0.0041)\n","Processed non_AMPEP123469: non-AMP/OTHERS (Probabilities: 0.6185, 0.3650, 0.0164)\n","Processed non_AMPEP9682: non-AMP/OTHERS (Probabilities: 0.9923, 0.0044, 0.0033)\n","Processed non_AMPEP118736: non-AMP/OTHERS (Probabilities: 0.9927, 0.0034, 0.0039)\n","Processed non_AMPEP53255: non-AMP/OTHERS (Probabilities: 0.9926, 0.0044, 0.0030)\n","Processed non_AMPEP102930: Neuropeptide (Probabilities: 0.0119, 0.0107, 0.9773)\n","Processed non_AMPEP94925: non-AMP/OTHERS (Probabilities: 0.9927, 0.0030, 0.0042)\n","Processed non_AMPEP89371: non-AMP/OTHERS (Probabilities: 0.9916, 0.0033, 0.0051)\n","Processed non_AMPEP149425: non-AMP/OTHERS (Probabilities: 0.9781, 0.0061, 0.0158)\n","Processed non_AMPEP68936: non-AMP/OTHERS (Probabilities: 0.9934, 0.0032, 0.0034)\n","Processed non_AMPEP90150: non-AMP/OTHERS (Probabilities: 0.9930, 0.0029, 0.0041)\n","Processed non_AMPEP75574: Neuropeptide (Probabilities: 0.0073, 0.0091, 0.9836)\n","Processed non_AMPEP80583: non-AMP/OTHERS (Probabilities: 0.9928, 0.0043, 0.0029)\n","Processed non_AMPEP51392: AMP (Probabilities: 0.0746, 0.9127, 0.0127)\n","Processed non_AMPEP98030: AMP (Probabilities: 0.0761, 0.9114, 0.0124)\n","Processed non_AMPEP106502: AMP (Probabilities: 0.0877, 0.9001, 0.0122)\n","Processed non_AMPEP14150: non-AMP/OTHERS (Probabilities: 0.9877, 0.0081, 0.0042)\n","Processed non_AMPEP134115: non-AMP/OTHERS (Probabilities: 0.9920, 0.0037, 0.0043)\n","Processed non_AMPEP85859: non-AMP/OTHERS (Probabilities: 0.9924, 0.0041, 0.0035)\n","Processed non_AMPEP24172: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed non_AMPEP10214: AMP (Probabilities: 0.0701, 0.9157, 0.0142)\n","Processed non_AMPEP140778: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0037)\n","Processed non_AMPEP21588: non-AMP/OTHERS (Probabilities: 0.9929, 0.0030, 0.0041)\n","Processed non_AMPEP59921: non-AMP/OTHERS (Probabilities: 0.9920, 0.0048, 0.0032)\n","Processed non_AMPEP116185: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0037)\n","Processed non_AMPEP113094: non-AMP/OTHERS (Probabilities: 0.9933, 0.0032, 0.0035)\n","Processed non_AMPEP32151: non-AMP/OTHERS (Probabilities: 0.9907, 0.0039, 0.0054)\n","Processed non_AMPEP153528: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0030)\n","Processed non_AMPEP101991: non-AMP/OTHERS (Probabilities: 0.9922, 0.0040, 0.0039)\n","Processed non_AMPEP33287: non-AMP/OTHERS (Probabilities: 0.9909, 0.0044, 0.0047)\n","Processed non_AMPEP141421: non-AMP/OTHERS (Probabilities: 0.9932, 0.0030, 0.0038)\n","Processed non_AMPEP147603: non-AMP/OTHERS (Probabilities: 0.9932, 0.0030, 0.0038)\n","Processed non_AMPEP77379: AMP (Probabilities: 0.1270, 0.8568, 0.0162)\n","Processed non_AMPEP161217: AMP (Probabilities: 0.1031, 0.8851, 0.0119)\n","Processed non_AMPEP85124: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0035)\n","Processed non_AMPEP154213: AMP (Probabilities: 0.0920, 0.8948, 0.0131)\n","Processed non_AMPEP142355: non-AMP/OTHERS (Probabilities: 0.8783, 0.0110, 0.1106)\n","Processed non_AMPEP113271: non-AMP/OTHERS (Probabilities: 0.9889, 0.0050, 0.0061)\n","Processed non_AMPEP143642: AMP (Probabilities: 0.0926, 0.8947, 0.0128)\n","Processed non_AMPEP119844: AMP (Probabilities: 0.2115, 0.7750, 0.0136)\n","Processed non_AMPEP94475: AMP (Probabilities: 0.3174, 0.6383, 0.0443)\n","Processed non_AMPEP116076: AMP (Probabilities: 0.1436, 0.8432, 0.0132)\n","Processed non_AMPEP75635: AMP (Probabilities: 0.3644, 0.6213, 0.0142)\n","Processed non_AMPEP113425: non-AMP/OTHERS (Probabilities: 0.9921, 0.0051, 0.0028)\n","Processed non_AMPEP90100: non-AMP/OTHERS (Probabilities: 0.7211, 0.2700, 0.0088)\n","Processed non_AMPEP113193: non-AMP/OTHERS (Probabilities: 0.9931, 0.0031, 0.0038)\n","Processed non_AMPEP123451: non-AMP/OTHERS (Probabilities: 0.9850, 0.0106, 0.0044)\n","Processed non_AMPEP95030: non-AMP/OTHERS (Probabilities: 0.9933, 0.0032, 0.0034)\n","Processed non_AMPEP11290: AMP (Probabilities: 0.1073, 0.8757, 0.0169)\n","Processed non_AMPEP153763: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0032)\n","Processed non_AMPEP106470: non-AMP/OTHERS (Probabilities: 0.9330, 0.0623, 0.0046)\n","Processed non_AMPEP6345: non-AMP/OTHERS (Probabilities: 0.4948, 0.4893, 0.0159)\n","Processed non_AMPEP41410: AMP (Probabilities: 0.0709, 0.9160, 0.0131)\n","Processed non_AMPEP124053: non-AMP/OTHERS (Probabilities: 0.9933, 0.0032, 0.0035)\n","Processed non_AMPEP81948: AMP (Probabilities: 0.0715, 0.9139, 0.0146)\n","Processed non_AMPEP105413: non-AMP/OTHERS (Probabilities: 0.9932, 0.0032, 0.0036)\n","Processed non_AMPEP102297: non-AMP/OTHERS (Probabilities: 0.8800, 0.1098, 0.0102)\n","Processed non_AMPEP39633: non-AMP/OTHERS (Probabilities: 0.9060, 0.0876, 0.0064)\n","Processed non_AMPEP131717: non-AMP/OTHERS (Probabilities: 0.9934, 0.0032, 0.0034)\n","Processed non_AMPEP137108: AMP (Probabilities: 0.2622, 0.7244, 0.0133)\n","Processed non_AMPEP152091: non-AMP/OTHERS (Probabilities: 0.9918, 0.0053, 0.0029)\n","Processed non_AMPEP56304: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0035)\n","Processed non_AMPEP156049: non-AMP/OTHERS (Probabilities: 0.9925, 0.0040, 0.0035)\n","Processed non_AMPEP96647: non-AMP/OTHERS (Probabilities: 0.6472, 0.0408, 0.3121)\n","Processed non_AMPEP7851: non-AMP/OTHERS (Probabilities: 0.9932, 0.0040, 0.0028)\n","Processed non_AMPEP61509: non-AMP/OTHERS (Probabilities: 0.9409, 0.0552, 0.0038)\n","Processed non_AMPEP127562: non-AMP/OTHERS (Probabilities: 0.9927, 0.0040, 0.0034)\n","Processed non_AMPEP22612: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0030)\n","Processed non_AMPEP146013: non-AMP/OTHERS (Probabilities: 0.5699, 0.0343, 0.3958)\n","Processed non_AMPEP59020: AMP (Probabilities: 0.1062, 0.8815, 0.0123)\n","Processed non_AMPEP114865: non-AMP/OTHERS (Probabilities: 0.9923, 0.0043, 0.0035)\n","Processed non_AMPEP23660: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0043)\n","Processed non_AMPEP92434: AMP (Probabilities: 0.1696, 0.8171, 0.0132)\n","Processed non_AMPEP106453: non-AMP/OTHERS (Probabilities: 0.9915, 0.0034, 0.0051)\n","Processed non_AMPEP79195: non-AMP/OTHERS (Probabilities: 0.9910, 0.0061, 0.0029)\n","Processed non_AMPEP41398: AMP (Probabilities: 0.0871, 0.8972, 0.0157)\n","Processed non_AMPEP7867: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0034)\n","Processed non_AMPEP48082: non-AMP/OTHERS (Probabilities: 0.9598, 0.0356, 0.0046)\n","Processed non_AMPEP101032: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed non_AMPEP130363: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed non_AMPEP117003: non-AMP/OTHERS (Probabilities: 0.9925, 0.0031, 0.0043)\n","Processed non_AMPEP58333: Neuropeptide (Probabilities: 0.0122, 0.0101, 0.9777)\n","Processed non_AMPEP117850: non-AMP/OTHERS (Probabilities: 0.7428, 0.2467, 0.0105)\n","Processed non_AMPEP115697: non-AMP/OTHERS (Probabilities: 0.9918, 0.0039, 0.0043)\n","Processed non_AMPEP70611: non-AMP/OTHERS (Probabilities: 0.9896, 0.0078, 0.0025)\n","Processed non_AMPEP109475: non-AMP/OTHERS (Probabilities: 0.9916, 0.0035, 0.0049)\n","Processed non_AMPEP30690: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed non_AMPEP90276: non-AMP/OTHERS (Probabilities: 0.9800, 0.0056, 0.0145)\n","Processed non_AMPEP108090: non-AMP/OTHERS (Probabilities: 0.9907, 0.0056, 0.0037)\n","Processed non_AMPEP114998: non-AMP/OTHERS (Probabilities: 0.9614, 0.0085, 0.0301)\n","Processed non_AMPEP6509: non-AMP/OTHERS (Probabilities: 0.9928, 0.0041, 0.0030)\n","Processed non_AMPEP34232: non-AMP/OTHERS (Probabilities: 0.9927, 0.0035, 0.0038)\n","Processed non_AMPEP98547: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed non_AMPEP143039: non-AMP/OTHERS (Probabilities: 0.9108, 0.0810, 0.0082)\n","Processed non_AMPEP83797: non-AMP/OTHERS (Probabilities: 0.9918, 0.0053, 0.0029)\n","Processed non_AMPEP131954: non-AMP/OTHERS (Probabilities: 0.9933, 0.0031, 0.0036)\n","Processed non_AMPEP99190: AMP (Probabilities: 0.1746, 0.8085, 0.0169)\n","Processed non_AMPEP68243: non-AMP/OTHERS (Probabilities: 0.9566, 0.0065, 0.0368)\n","Processed non_AMPEP154133: non-AMP/OTHERS (Probabilities: 0.9930, 0.0030, 0.0040)\n","Processed non_AMPEP56070: non-AMP/OTHERS (Probabilities: 0.9288, 0.0101, 0.0611)\n","Processed non_AMPEP157044: non-AMP/OTHERS (Probabilities: 0.9916, 0.0036, 0.0048)\n","Processed non_AMPEP118760: non-AMP/OTHERS (Probabilities: 0.9931, 0.0030, 0.0039)\n","Processed non_AMPEP101467: non-AMP/OTHERS (Probabilities: 0.8203, 0.1524, 0.0273)\n","Processed non_AMPEP76669: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0035)\n","Processed non_AMPEP43929: non-AMP/OTHERS (Probabilities: 0.9192, 0.0721, 0.0088)\n","Processed non_AMPEP9536: non-AMP/OTHERS (Probabilities: 0.9917, 0.0037, 0.0046)\n","Processed non_AMPEP88818: non-AMP/OTHERS (Probabilities: 0.9930, 0.0039, 0.0031)\n","Processed non_AMPEP98015: non-AMP/OTHERS (Probabilities: 0.9911, 0.0034, 0.0055)\n","Processed non_AMPEP6920: non-AMP/OTHERS (Probabilities: 0.9902, 0.0039, 0.0059)\n","Processed non_AMPEP31709: non-AMP/OTHERS (Probabilities: 0.9493, 0.0451, 0.0055)\n","Processed non_AMPEP123862: non-AMP/OTHERS (Probabilities: 0.9924, 0.0037, 0.0040)\n","Processed non_AMPEP28052: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0035)\n","Processed non_AMPEP101954: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0034)\n","Processed non_AMPEP48445: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0034)\n","Processed non_AMPEP118317: non-AMP/OTHERS (Probabilities: 0.9932, 0.0036, 0.0032)\n","Processed non_AMPEP107452: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0036)\n","Processed non_AMPEP22451: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed non_AMPEP66224: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0034)\n","Processed non_AMPEP7766: non-AMP/OTHERS (Probabilities: 0.9928, 0.0031, 0.0041)\n","Processed non_AMPEP9803: non-AMP/OTHERS (Probabilities: 0.9925, 0.0045, 0.0029)\n","Processed non_AMPEP156001: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0035)\n","Processed non_AMPEP112252: non-AMP/OTHERS (Probabilities: 0.9930, 0.0033, 0.0036)\n","Processed non_AMPEP4773: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0036)\n","Processed non_AMPEP43145: non-AMP/OTHERS (Probabilities: 0.8745, 0.0679, 0.0576)\n","Processed non_AMPEP33462: non-AMP/OTHERS (Probabilities: 0.9932, 0.0036, 0.0032)\n","Processed non_AMPEP132732: non-AMP/OTHERS (Probabilities: 0.9779, 0.0183, 0.0038)\n","Processed non_AMPEP96568: non-AMP/OTHERS (Probabilities: 0.9917, 0.0056, 0.0027)\n","Processed non_AMPEP115381: non-AMP/OTHERS (Probabilities: 0.9919, 0.0048, 0.0033)\n","Processed non_AMPEP87804: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0034)\n","Processed non_AMPEP137668: non-AMP/OTHERS (Probabilities: 0.9930, 0.0030, 0.0040)\n","Processed non_AMPEP54026: AMP (Probabilities: 0.0650, 0.9195, 0.0156)\n","Processed non_AMPEP161356: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0039)\n","Processed non_AMPEP60003: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0040)\n","Processed non_AMPEP125124: non-AMP/OTHERS (Probabilities: 0.9928, 0.0030, 0.0042)\n","Processed non_AMPEP133556: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0035)\n","Processed non_AMPEP71891: AMP (Probabilities: 0.0807, 0.9064, 0.0129)\n","Processed non_AMPEP165467: non-AMP/OTHERS (Probabilities: 0.9932, 0.0035, 0.0034)\n","Processed non_AMPEP123624: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0034)\n","Processed non_AMPEP157807: non-AMP/OTHERS (Probabilities: 0.9921, 0.0039, 0.0040)\n","Processed non_AMPEP137825: non-AMP/OTHERS (Probabilities: 0.9440, 0.0471, 0.0089)\n","Processed non_AMPEP46387: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0037)\n","Processed non_AMPEP121893: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0035)\n","Processed non_AMPEP120765: non-AMP/OTHERS (Probabilities: 0.9786, 0.0174, 0.0040)\n","Processed non_AMPEP76936: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0036)\n","Processed non_AMPEP122603: non-AMP/OTHERS (Probabilities: 0.9576, 0.0387, 0.0037)\n","Processed non_AMPEP13467: non-AMP/OTHERS (Probabilities: 0.9911, 0.0037, 0.0052)\n","Processed non_AMPEP6162: AMP (Probabilities: 0.0775, 0.9093, 0.0132)\n","Processed non_AMPEP93488: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0035)\n","Processed non_AMPEP158042: non-AMP/OTHERS (Probabilities: 0.9928, 0.0034, 0.0038)\n","Processed non_AMPEP76932: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0037)\n","Processed non_AMPEP130391: non-AMP/OTHERS (Probabilities: 0.7245, 0.2679, 0.0076)\n","Processed non_AMPEP103201: non-AMP/OTHERS (Probabilities: 0.9220, 0.0726, 0.0054)\n","Processed non_AMPEP149661: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0037)\n","Processed non_AMPEP108388: Neuropeptide (Probabilities: 0.0168, 0.0129, 0.9703)\n","Processed non_AMPEP64501: non-AMP/OTHERS (Probabilities: 0.9857, 0.0083, 0.0060)\n","Processed non_AMPEP115790: non-AMP/OTHERS (Probabilities: 0.9927, 0.0044, 0.0029)\n","Processed non_AMPEP77749: non-AMP/OTHERS (Probabilities: 0.9924, 0.0043, 0.0033)\n","Processed non_AMPEP102701: non-AMP/OTHERS (Probabilities: 0.9920, 0.0045, 0.0035)\n","Processed non_AMPEP131911: non-AMP/OTHERS (Probabilities: 0.9932, 0.0035, 0.0033)\n","Processed non_AMPEP85778: non-AMP/OTHERS (Probabilities: 0.9891, 0.0037, 0.0072)\n","Processed non_AMPEP157411: non-AMP/OTHERS (Probabilities: 0.9926, 0.0045, 0.0029)\n","Processed non_AMPEP47754: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0033)\n","Processed non_AMPEP93879: non-AMP/OTHERS (Probabilities: 0.4882, 0.0307, 0.4810)\n","Processed non_AMPEP102468: non-AMP/OTHERS (Probabilities: 0.9744, 0.0205, 0.0051)\n","Processed non_AMPEP156960: non-AMP/OTHERS (Probabilities: 0.9923, 0.0041, 0.0036)\n","Processed non_AMPEP80685: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0036)\n","Processed non_AMPEP125835: non-AMP/OTHERS (Probabilities: 0.9824, 0.0075, 0.0101)\n","Processed non_AMPEP104436: Neuropeptide (Probabilities: 0.3255, 0.0227, 0.6518)\n","Processed non_AMPEP58097: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0037)\n","Processed non_AMPEP136230: non-AMP/OTHERS (Probabilities: 0.9878, 0.0093, 0.0029)\n","Processed non_AMPEP115885: non-AMP/OTHERS (Probabilities: 0.9933, 0.0031, 0.0035)\n","Processed non_AMPEP106726: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0035)\n","Processed non_AMPEP125467: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed non_AMPEP37555: non-AMP/OTHERS (Probabilities: 0.5723, 0.4159, 0.0118)\n","Processed non_AMPEP160234: non-AMP/OTHERS (Probabilities: 0.9929, 0.0041, 0.0030)\n","Processed non_AMPEP57499: non-AMP/OTHERS (Probabilities: 0.9917, 0.0038, 0.0045)\n","Processed non_AMPEP161875: non-AMP/OTHERS (Probabilities: 0.9924, 0.0047, 0.0029)\n","Processed non_AMPEP45064: non-AMP/OTHERS (Probabilities: 0.5234, 0.4523, 0.0242)\n","Processed non_AMPEP37209: non-AMP/OTHERS (Probabilities: 0.9330, 0.0582, 0.0088)\n","Processed non_AMPEP125769: non-AMP/OTHERS (Probabilities: 0.7461, 0.2470, 0.0069)\n","Processed non_AMPEP90837: AMP (Probabilities: 0.0794, 0.9082, 0.0124)\n","Processed non_AMPEP90820: non-AMP/OTHERS (Probabilities: 0.9917, 0.0039, 0.0044)\n","Processed non_AMPEP66160: AMP (Probabilities: 0.0825, 0.9049, 0.0126)\n","Processed non_AMPEP41867: Neuropeptide (Probabilities: 0.0199, 0.0134, 0.9667)\n","Processed non_AMPEP130139: non-AMP/OTHERS (Probabilities: 0.9905, 0.0064, 0.0032)\n","Processed non_AMPEP79873: non-AMP/OTHERS (Probabilities: 0.9873, 0.0076, 0.0051)\n","Processed non_AMPEP55172: non-AMP/OTHERS (Probabilities: 0.9907, 0.0039, 0.0054)\n","Processed non_AMPEP157871: non-AMP/OTHERS (Probabilities: 0.9610, 0.0352, 0.0037)\n","Processed non_AMPEP133767: non-AMP/OTHERS (Probabilities: 0.9933, 0.0035, 0.0031)\n","Processed non_AMPEP104204: AMP (Probabilities: 0.0671, 0.9174, 0.0155)\n","Processed non_AMPEP120815: non-AMP/OTHERS (Probabilities: 0.9865, 0.0085, 0.0050)\n","Processed non_AMPEP117973: non-AMP/OTHERS (Probabilities: 0.9911, 0.0037, 0.0052)\n","Processed non_AMPEP45073: non-AMP/OTHERS (Probabilities: 0.9906, 0.0062, 0.0032)\n","Processed non_AMPEP67296: non-AMP/OTHERS (Probabilities: 0.9926, 0.0037, 0.0038)\n","Processed non_AMPEP4180: non-AMP/OTHERS (Probabilities: 0.9921, 0.0033, 0.0046)\n","Processed non_AMPEP24763: non-AMP/OTHERS (Probabilities: 0.9919, 0.0037, 0.0044)\n","Processed non_AMPEP119204: non-AMP/OTHERS (Probabilities: 0.9908, 0.0041, 0.0051)\n","Processed non_AMPEP23025: AMP (Probabilities: 0.0880, 0.8980, 0.0140)\n","Processed non_AMPEP153840: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0034)\n","Processed non_AMPEP59315: AMP (Probabilities: 0.0738, 0.9130, 0.0133)\n","Processed non_AMPEP101840: non-AMP/OTHERS (Probabilities: 0.9927, 0.0046, 0.0027)\n","Processed non_AMPEP120011: non-AMP/OTHERS (Probabilities: 0.9927, 0.0030, 0.0043)\n","Processed non_AMPEP50019: non-AMP/OTHERS (Probabilities: 0.9928, 0.0036, 0.0036)\n","Processed non_AMPEP50309: non-AMP/OTHERS (Probabilities: 0.9932, 0.0030, 0.0038)\n","Processed non_AMPEP110448: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed non_AMPEP67331: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed non_AMPEP135709: non-AMP/OTHERS (Probabilities: 0.6433, 0.3445, 0.0123)\n","Processed non_AMPEP101524: non-AMP/OTHERS (Probabilities: 0.9922, 0.0035, 0.0044)\n","Processed non_AMPEP108198: AMP (Probabilities: 0.0667, 0.9177, 0.0156)\n","Processed non_AMPEP148169: AMP (Probabilities: 0.0787, 0.9085, 0.0127)\n","Processed non_AMPEP150213: non-AMP/OTHERS (Probabilities: 0.9918, 0.0037, 0.0045)\n","Processed non_AMPEP6157: non-AMP/OTHERS (Probabilities: 0.9924, 0.0036, 0.0039)\n","Processed non_AMPEP148517: non-AMP/OTHERS (Probabilities: 0.9898, 0.0031, 0.0071)\n","Processed non_AMPEP79861: non-AMP/OTHERS (Probabilities: 0.9923, 0.0035, 0.0042)\n","Processed non_AMPEP81944: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0037)\n","Processed non_AMPEP32387: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0034)\n","Processed non_AMPEP60702: non-AMP/OTHERS (Probabilities: 0.9923, 0.0038, 0.0039)\n","Processed non_AMPEP115247: non-AMP/OTHERS (Probabilities: 0.9919, 0.0036, 0.0044)\n","Processed non_AMPEP71644: non-AMP/OTHERS (Probabilities: 0.9874, 0.0092, 0.0035)\n","Processed non_AMPEP106949: AMP (Probabilities: 0.1084, 0.8795, 0.0121)\n","Processed non_AMPEP10143: non-AMP/OTHERS (Probabilities: 0.9929, 0.0033, 0.0038)\n","Processed non_AMPEP78850: non-AMP/OTHERS (Probabilities: 0.9933, 0.0032, 0.0035)\n","Processed non_AMPEP1264: non-AMP/OTHERS (Probabilities: 0.9924, 0.0037, 0.0039)\n","Processed non_AMPEP4598: non-AMP/OTHERS (Probabilities: 0.9749, 0.0087, 0.0164)\n","Processed non_AMPEP83135: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0037)\n","Processed non_AMPEP69965: non-AMP/OTHERS (Probabilities: 0.9928, 0.0034, 0.0037)\n","Processed non_AMPEP23885: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed non_AMPEP106860: non-AMP/OTHERS (Probabilities: 0.9927, 0.0037, 0.0036)\n","Processed non_AMPEP51533: AMP (Probabilities: 0.1069, 0.8810, 0.0121)\n","Processed non_AMPEP107557: non-AMP/OTHERS (Probabilities: 0.9924, 0.0038, 0.0039)\n","Processed non_AMPEP27675: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0039)\n","Processed non_AMPEP145866: non-AMP/OTHERS (Probabilities: 0.9927, 0.0034, 0.0038)\n","Processed non_AMPEP3231: non-AMP/OTHERS (Probabilities: 0.9928, 0.0034, 0.0037)\n","Processed non_AMPEP14594: AMP (Probabilities: 0.0696, 0.9159, 0.0145)\n","Processed non_AMPEP142580: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0035)\n","Processed non_AMPEP165968: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0037)\n","Processed non_AMPEP144895: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0036)\n","Processed non_AMPEP115055: non-AMP/OTHERS (Probabilities: 0.9745, 0.0087, 0.0168)\n","Processed non_AMPEP36964: non-AMP/OTHERS (Probabilities: 0.9917, 0.0046, 0.0037)\n","Processed non_AMPEP48591: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0038)\n","Processed non_AMPEP86798: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0038)\n","Processed non_AMPEP6823: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed non_AMPEP151860: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0035)\n","Processed non_AMPEP122827: non-AMP/OTHERS (Probabilities: 0.9916, 0.0038, 0.0046)\n","Processed non_AMPEP48731: non-AMP/OTHERS (Probabilities: 0.9918, 0.0055, 0.0027)\n","Processed non_AMPEP30052: non-AMP/OTHERS (Probabilities: 0.9920, 0.0043, 0.0038)\n","Processed non_AMPEP133933: non-AMP/OTHERS (Probabilities: 0.9913, 0.0059, 0.0028)\n","Processed non_AMPEP7418: non-AMP/OTHERS (Probabilities: 0.9925, 0.0036, 0.0039)\n","Processed non_AMPEP83937: non-AMP/OTHERS (Probabilities: 0.9900, 0.0072, 0.0028)\n","Processed non_AMPEP98317: non-AMP/OTHERS (Probabilities: 0.9933, 0.0035, 0.0032)\n","Processed non_AMPEP88500: non-AMP/OTHERS (Probabilities: 0.9923, 0.0038, 0.0038)\n","Processed non_AMPEP81402: non-AMP/OTHERS (Probabilities: 0.9906, 0.0068, 0.0026)\n","Processed non_AMPEP59722: non-AMP/OTHERS (Probabilities: 0.9921, 0.0048, 0.0031)\n","Processed non_AMPEP150247: non-AMP/OTHERS (Probabilities: 0.9929, 0.0036, 0.0035)\n","Processed non_AMPEP20646: non-AMP/OTHERS (Probabilities: 0.9911, 0.0048, 0.0041)\n","Processed non_AMPEP74649: non-AMP/OTHERS (Probabilities: 0.9922, 0.0042, 0.0036)\n","Processed non_AMPEP78184: AMP (Probabilities: 0.0874, 0.9002, 0.0124)\n","Processed non_AMPEP5032: AMP (Probabilities: 0.0776, 0.9100, 0.0125)\n","Processed non_AMPEP133376: non-AMP/OTHERS (Probabilities: 0.9917, 0.0049, 0.0034)\n","Processed non_AMPEP98218: non-AMP/OTHERS (Probabilities: 0.9927, 0.0041, 0.0032)\n","Processed non_AMPEP41089: non-AMP/OTHERS (Probabilities: 0.9917, 0.0046, 0.0037)\n","Processed non_AMPEP129270: non-AMP/OTHERS (Probabilities: 0.9932, 0.0037, 0.0031)\n","Processed non_AMPEP4919: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0036)\n","Processed non_AMPEP124428: non-AMP/OTHERS (Probabilities: 0.9105, 0.0843, 0.0052)\n","Processed non_AMPEP123324: non-AMP/OTHERS (Probabilities: 0.9922, 0.0045, 0.0033)\n","Processed non_AMPEP64670: AMP (Probabilities: 0.1457, 0.8424, 0.0119)\n","Processed non_AMPEP164936: non-AMP/OTHERS (Probabilities: 0.9920, 0.0037, 0.0043)\n","Processed non_AMPEP61998: non-AMP/OTHERS (Probabilities: 0.9917, 0.0052, 0.0032)\n","Processed non_AMPEP161436: non-AMP/OTHERS (Probabilities: 0.9861, 0.0047, 0.0092)\n","Processed non_AMPEP119988: AMP (Probabilities: 0.1244, 0.8635, 0.0121)\n","Processed non_AMPEP79841: non-AMP/OTHERS (Probabilities: 0.9759, 0.0177, 0.0064)\n","Processed non_AMPEP93091: non-AMP/OTHERS (Probabilities: 0.9916, 0.0054, 0.0030)\n","Processed non_AMPEP22018: non-AMP/OTHERS (Probabilities: 0.9927, 0.0041, 0.0033)\n","Processed non_AMPEP36954: non-AMP/OTHERS (Probabilities: 0.9921, 0.0036, 0.0044)\n","Processed non_AMPEP79047: non-AMP/OTHERS (Probabilities: 0.9912, 0.0062, 0.0026)\n","Processed non_AMPEP96982: non-AMP/OTHERS (Probabilities: 0.9884, 0.0061, 0.0055)\n","Processed non_AMPEP116836: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0034)\n","Processed non_AMPEP52991: non-AMP/OTHERS (Probabilities: 0.9919, 0.0033, 0.0048)\n","Processed non_AMPEP45582: non-AMP/OTHERS (Probabilities: 0.9927, 0.0041, 0.0032)\n","Processed non_AMPEP23736: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed non_AMPEP126720: non-AMP/OTHERS (Probabilities: 0.9707, 0.0097, 0.0196)\n","Processed non_AMPEP21039: non-AMP/OTHERS (Probabilities: 0.9926, 0.0041, 0.0033)\n","Processed non_AMPEP152650: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed non_AMPEP74617: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed non_AMPEP128268: non-AMP/OTHERS (Probabilities: 0.9907, 0.0053, 0.0040)\n","Processed non_AMPEP109410: non-AMP/OTHERS (Probabilities: 0.9902, 0.0053, 0.0045)\n","Processed non_AMPEP43211: non-AMP/OTHERS (Probabilities: 0.9933, 0.0036, 0.0031)\n","Processed non_AMPEP84611: non-AMP/OTHERS (Probabilities: 0.9928, 0.0040, 0.0032)\n","Processed non_AMPEP113771: non-AMP/OTHERS (Probabilities: 0.9926, 0.0040, 0.0033)\n","Processed non_AMPEP156431: non-AMP/OTHERS (Probabilities: 0.9920, 0.0031, 0.0049)\n","Processed non_AMPEP2560: AMP (Probabilities: 0.0657, 0.9192, 0.0150)\n","Processed non_AMPEP124264: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed non_AMPEP65377: non-AMP/OTHERS (Probabilities: 0.9522, 0.0313, 0.0166)\n","Processed non_AMPEP153689: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0031)\n","Processed non_AMPEP124126: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed non_AMPEP88455: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0032)\n","Processed non_AMPEP147828: non-AMP/OTHERS (Probabilities: 0.9926, 0.0036, 0.0038)\n","Processed non_AMPEP22529: non-AMP/OTHERS (Probabilities: 0.9926, 0.0042, 0.0031)\n","Processed non_AMPEP119539: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0032)\n","Processed non_AMPEP109235: non-AMP/OTHERS (Probabilities: 0.9929, 0.0040, 0.0031)\n","Processed non_AMPEP98216: non-AMP/OTHERS (Probabilities: 0.9929, 0.0040, 0.0031)\n","Processed non_AMPEP82401: non-AMP/OTHERS (Probabilities: 0.9914, 0.0035, 0.0051)\n","Processed non_AMPEP70674: non-AMP/OTHERS (Probabilities: 0.9927, 0.0043, 0.0030)\n","Processed non_AMPEP107111: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0031)\n","Processed non_AMPEP151430: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0033)\n","Processed non_AMPEP127585: non-AMP/OTHERS (Probabilities: 0.9921, 0.0048, 0.0031)\n","Processed non_AMPEP136883: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed non_AMPEP150605: non-AMP/OTHERS (Probabilities: 0.9923, 0.0042, 0.0035)\n","Processed non_AMPEP95209: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0037)\n","Processed non_AMPEP158335: non-AMP/OTHERS (Probabilities: 0.9922, 0.0040, 0.0038)\n","Processed non_AMPEP90785: non-AMP/OTHERS (Probabilities: 0.9934, 0.0033, 0.0033)\n","Processed non_AMPEP148680: non-AMP/OTHERS (Probabilities: 0.9894, 0.0062, 0.0044)\n","Processed non_AMPEP5022: non-AMP/OTHERS (Probabilities: 0.9917, 0.0039, 0.0044)\n","Processed non_AMPEP119902: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0036)\n","Processed non_AMPEP166003: AMP (Probabilities: 0.1148, 0.8725, 0.0126)\n","Processed non_AMPEP114089: non-AMP/OTHERS (Probabilities: 0.9910, 0.0047, 0.0043)\n","Processed non_AMPEP152437: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0036)\n","Processed non_AMPEP126006: non-AMP/OTHERS (Probabilities: 0.9884, 0.0084, 0.0032)\n","Processed non_AMPEP97840: non-AMP/OTHERS (Probabilities: 0.9933, 0.0034, 0.0034)\n","Processed non_AMPEP141778: non-AMP/OTHERS (Probabilities: 0.9908, 0.0062, 0.0030)\n","Processed non_AMPEP48339: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0032)\n","Processed non_AMPEP76412: non-AMP/OTHERS (Probabilities: 0.9915, 0.0037, 0.0047)\n","Processed non_AMPEP75783: non-AMP/OTHERS (Probabilities: 0.9905, 0.0060, 0.0035)\n","Processed non_AMPEP45808: non-AMP/OTHERS (Probabilities: 0.9913, 0.0056, 0.0031)\n","Processed non_AMPEP30856: non-AMP/OTHERS (Probabilities: 0.9909, 0.0062, 0.0029)\n","Processed non_AMPEP133384: non-AMP/OTHERS (Probabilities: 0.9905, 0.0040, 0.0055)\n","Processed non_AMPEP8235: non-AMP/OTHERS (Probabilities: 0.7342, 0.0288, 0.2370)\n","Processed non_AMPEP35516: non-AMP/OTHERS (Probabilities: 0.9924, 0.0044, 0.0032)\n","Processed non_AMPEP142908: non-AMP/OTHERS (Probabilities: 0.9031, 0.0901, 0.0068)\n","Processed non_AMPEP91848: non-AMP/OTHERS (Probabilities: 0.9843, 0.0090, 0.0067)\n","Processed non_AMPEP71588: non-AMP/OTHERS (Probabilities: 0.9933, 0.0036, 0.0031)\n","Processed non_AMPEP128723: non-AMP/OTHERS (Probabilities: 0.5031, 0.0286, 0.4683)\n","Processed non_AMPEP122166: non-AMP/OTHERS (Probabilities: 0.9848, 0.0048, 0.0104)\n","Processed non_AMPEP162614: non-AMP/OTHERS (Probabilities: 0.9917, 0.0042, 0.0040)\n","Processed non_AMPEP66656: non-AMP/OTHERS (Probabilities: 0.9922, 0.0046, 0.0032)\n","Processed non_AMPEP143494: non-AMP/OTHERS (Probabilities: 0.9848, 0.0037, 0.0115)\n","Processed non_AMPEP126240: non-AMP/OTHERS (Probabilities: 0.9881, 0.0054, 0.0065)\n","Processed non_AMPEP101407: non-AMP/OTHERS (Probabilities: 0.9908, 0.0045, 0.0047)\n","Processed non_AMPEP150385: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0036)\n","Processed non_AMPEP22831: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0037)\n","Processed non_AMPEP53248: non-AMP/OTHERS (Probabilities: 0.9908, 0.0056, 0.0036)\n","Processed non_AMPEP16031: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed non_AMPEP65087: non-AMP/OTHERS (Probabilities: 0.9907, 0.0047, 0.0046)\n","Processed non_AMPEP162884: non-AMP/OTHERS (Probabilities: 0.9919, 0.0046, 0.0035)\n","Processed non_AMPEP87907: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed non_AMPEP163198: non-AMP/OTHERS (Probabilities: 0.9911, 0.0047, 0.0042)\n","Processed non_AMPEP125514: non-AMP/OTHERS (Probabilities: 0.9904, 0.0048, 0.0049)\n","Processed non_AMPEP31774: non-AMP/OTHERS (Probabilities: 0.9511, 0.0425, 0.0064)\n","Processed non_AMPEP150633: non-AMP/OTHERS (Probabilities: 0.9912, 0.0034, 0.0054)\n","Processed non_AMPEP16560: AMP (Probabilities: 0.4763, 0.5031, 0.0205)\n","Processed non_AMPEP102484: non-AMP/OTHERS (Probabilities: 0.9895, 0.0061, 0.0044)\n","Processed non_AMPEP111686: non-AMP/OTHERS (Probabilities: 0.9903, 0.0053, 0.0044)\n","Processed non_AMPEP139241: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0039)\n","Processed non_AMPEP91998: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed non_AMPEP40367: non-AMP/OTHERS (Probabilities: 0.9766, 0.0192, 0.0041)\n","Processed non_AMPEP152162: non-AMP/OTHERS (Probabilities: 0.8086, 0.1814, 0.0100)\n","Processed non_AMPEP99417: non-AMP/OTHERS (Probabilities: 0.8641, 0.1294, 0.0065)\n","Processed non_AMPEP46680: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed non_AMPEP92387: non-AMP/OTHERS (Probabilities: 0.9907, 0.0063, 0.0030)\n","Processed non_AMPEP144871: non-AMP/OTHERS (Probabilities: 0.9510, 0.0093, 0.0397)\n","Processed non_AMPEP99822: non-AMP/OTHERS (Probabilities: 0.9923, 0.0034, 0.0043)\n","Processed non_AMPEP70022: non-AMP/OTHERS (Probabilities: 0.9906, 0.0036, 0.0058)\n","Processed non_AMPEP8982: non-AMP/OTHERS (Probabilities: 0.9919, 0.0045, 0.0036)\n","Processed non_AMPEP66939: AMP (Probabilities: 0.1199, 0.8675, 0.0127)\n","Processed non_AMPEP58539: non-AMP/OTHERS (Probabilities: 0.9926, 0.0041, 0.0033)\n","Processed non_AMPEP99727: non-AMP/OTHERS (Probabilities: 0.9898, 0.0070, 0.0032)\n","Processed non_AMPEP4715: non-AMP/OTHERS (Probabilities: 0.9898, 0.0067, 0.0036)\n","Processed non_AMPEP141969: non-AMP/OTHERS (Probabilities: 0.9919, 0.0046, 0.0036)\n","Processed non_AMPEP47299: non-AMP/OTHERS (Probabilities: 0.9890, 0.0049, 0.0060)\n","Processed non_AMPEP162796: non-AMP/OTHERS (Probabilities: 0.9918, 0.0045, 0.0037)\n","Processed non_AMPEP118833: non-AMP/OTHERS (Probabilities: 0.9929, 0.0033, 0.0038)\n","Processed non_AMPEP5945: non-AMP/OTHERS (Probabilities: 0.9867, 0.0092, 0.0041)\n","Processed non_AMPEP73613: non-AMP/OTHERS (Probabilities: 0.9875, 0.0089, 0.0036)\n","Processed non_AMPEP60613: non-AMP/OTHERS (Probabilities: 0.9920, 0.0043, 0.0037)\n","Processed non_AMPEP150058: non-AMP/OTHERS (Probabilities: 0.9848, 0.0096, 0.0056)\n","Processed non_AMPEP44197: non-AMP/OTHERS (Probabilities: 0.9929, 0.0032, 0.0039)\n","Processed non_AMPEP94765: non-AMP/OTHERS (Probabilities: 0.9906, 0.0062, 0.0032)\n","Processed non_AMPEP72216: non-AMP/OTHERS (Probabilities: 0.9864, 0.0072, 0.0063)\n","Processed non_AMPEP84506: non-AMP/OTHERS (Probabilities: 0.9504, 0.0451, 0.0045)\n","Processed non_AMPEP134026: non-AMP/OTHERS (Probabilities: 0.9922, 0.0041, 0.0037)\n","Processed non_AMPEP73401: non-AMP/OTHERS (Probabilities: 0.9503, 0.0453, 0.0044)\n","Processed non_AMPEP99122: non-AMP/OTHERS (Probabilities: 0.9919, 0.0045, 0.0036)\n","Processed non_AMPEP56927: non-AMP/OTHERS (Probabilities: 0.8906, 0.0127, 0.0966)\n","Processed non_AMPEP118759: non-AMP/OTHERS (Probabilities: 0.9920, 0.0040, 0.0040)\n","Processed non_AMPEP69944: non-AMP/OTHERS (Probabilities: 0.9921, 0.0041, 0.0038)\n","Processed non_AMPEP67487: non-AMP/OTHERS (Probabilities: 0.9900, 0.0068, 0.0032)\n","Processed non_AMPEP50381: non-AMP/OTHERS (Probabilities: 0.9827, 0.0118, 0.0055)\n","Processed non_AMPEP8632: non-AMP/OTHERS (Probabilities: 0.9869, 0.0103, 0.0028)\n","Processed non_AMPEP141236: non-AMP/OTHERS (Probabilities: 0.9920, 0.0042, 0.0038)\n","Processed non_AMPEP114328: non-AMP/OTHERS (Probabilities: 0.9919, 0.0043, 0.0038)\n","Processed non_AMPEP111112: non-AMP/OTHERS (Probabilities: 0.9916, 0.0045, 0.0039)\n","Processed non_AMPEP157250: non-AMP/OTHERS (Probabilities: 0.9899, 0.0065, 0.0036)\n","Processed non_AMPEP12933: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0035)\n","Processed non_AMPEP147221: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0035)\n","Processed non_AMPEP99945: non-AMP/OTHERS (Probabilities: 0.9907, 0.0056, 0.0037)\n","Processed non_AMPEP13940: AMP (Probabilities: 0.0707, 0.9151, 0.0142)\n","Processed non_AMPEP16388: non-AMP/OTHERS (Probabilities: 0.9896, 0.0036, 0.0068)\n","Processed non_AMPEP96181: AMP (Probabilities: 0.1404, 0.8478, 0.0118)\n","Processed non_AMPEP83950: non-AMP/OTHERS (Probabilities: 0.9893, 0.0062, 0.0045)\n","Processed non_AMPEP60871: non-AMP/OTHERS (Probabilities: 0.9921, 0.0038, 0.0041)\n","Processed non_AMPEP144957: non-AMP/OTHERS (Probabilities: 0.9914, 0.0042, 0.0044)\n","Processed non_AMPEP84408: non-AMP/OTHERS (Probabilities: 0.9920, 0.0038, 0.0041)\n","Processed non_AMPEP137503: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed non_AMPEP109101: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed non_AMPEP137702: non-AMP/OTHERS (Probabilities: 0.9927, 0.0044, 0.0029)\n","Processed non_AMPEP61699: non-AMP/OTHERS (Probabilities: 0.9915, 0.0048, 0.0037)\n","Processed non_AMPEP22143: non-AMP/OTHERS (Probabilities: 0.9766, 0.0205, 0.0029)\n","Processed non_AMPEP86722: AMP (Probabilities: 0.1242, 0.8636, 0.0122)\n","Processed non_AMPEP128771: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed non_AMPEP133208: non-AMP/OTHERS (Probabilities: 0.8422, 0.1497, 0.0081)\n","Processed non_AMPEP61218: AMP (Probabilities: 0.1019, 0.8848, 0.0133)\n","Processed non_AMPEP28395: AMP (Probabilities: 0.1606, 0.8200, 0.0194)\n","Processed non_AMPEP110648: Neuropeptide (Probabilities: 0.0154, 0.0129, 0.9716)\n","Processed non_AMPEP15784: non-AMP/OTHERS (Probabilities: 0.9878, 0.0090, 0.0032)\n","Processed non_AMPEP82144: non-AMP/OTHERS (Probabilities: 0.9899, 0.0058, 0.0043)\n","Processed non_AMPEP122080: non-AMP/OTHERS (Probabilities: 0.9917, 0.0042, 0.0040)\n","Processed non_AMPEP23326: AMP (Probabilities: 0.0827, 0.9022, 0.0151)\n","Processed non_AMPEP12223: non-AMP/OTHERS (Probabilities: 0.9689, 0.0267, 0.0044)\n","Processed non_AMPEP94812: non-AMP/OTHERS (Probabilities: 0.9896, 0.0064, 0.0039)\n","Processed non_AMPEP87308: non-AMP/OTHERS (Probabilities: 0.9858, 0.0055, 0.0086)\n","Processed non_AMPEP150343: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0040)\n","Processed non_AMPEP72492: non-AMP/OTHERS (Probabilities: 0.9854, 0.0114, 0.0032)\n","Processed non_AMPEP49742: non-AMP/OTHERS (Probabilities: 0.9609, 0.0341, 0.0050)\n","Processed non_AMPEP4318: non-AMP/OTHERS (Probabilities: 0.9851, 0.0073, 0.0076)\n","Processed non_AMPEP75216: non-AMP/OTHERS (Probabilities: 0.9918, 0.0047, 0.0034)\n","Processed non_AMPEP72506: non-AMP/OTHERS (Probabilities: 0.9873, 0.0048, 0.0079)\n","Processed non_AMPEP82902: non-AMP/OTHERS (Probabilities: 0.9922, 0.0051, 0.0027)\n","Processed non_AMPEP28613: non-AMP/OTHERS (Probabilities: 0.9916, 0.0049, 0.0036)\n","Processed non_AMPEP61780: non-AMP/OTHERS (Probabilities: 0.9142, 0.0792, 0.0066)\n","Processed non_AMPEP63477: non-AMP/OTHERS (Probabilities: 0.9929, 0.0045, 0.0026)\n","Processed non_AMPEP50251: non-AMP/OTHERS (Probabilities: 0.9483, 0.0465, 0.0053)\n","Processed non_AMPEP26043: non-AMP/OTHERS (Probabilities: 0.9921, 0.0039, 0.0041)\n","Processed non_AMPEP113282: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0037)\n","Processed non_AMPEP116727: non-AMP/OTHERS (Probabilities: 0.9390, 0.0540, 0.0070)\n","Processed non_AMPEP90166: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0032)\n","Processed non_AMPEP52272: non-AMP/OTHERS (Probabilities: 0.9924, 0.0043, 0.0034)\n","Processed non_AMPEP5753: non-AMP/OTHERS (Probabilities: 0.9924, 0.0032, 0.0044)\n","Processed non_AMPEP140940: AMP (Probabilities: 0.1023, 0.8854, 0.0123)\n","Processed non_AMPEP142620: Neuropeptide (Probabilities: 0.0179, 0.0143, 0.9678)\n","Processed non_AMPEP35499: non-AMP/OTHERS (Probabilities: 0.9903, 0.0066, 0.0031)\n","Processed non_AMPEP139317: non-AMP/OTHERS (Probabilities: 0.9780, 0.0093, 0.0127)\n","Processed non_AMPEP165592: non-AMP/OTHERS (Probabilities: 0.9919, 0.0040, 0.0041)\n","Processed non_AMPEP63663: non-AMP/OTHERS (Probabilities: 0.9888, 0.0076, 0.0036)\n","Processed non_AMPEP128165: non-AMP/OTHERS (Probabilities: 0.9544, 0.0214, 0.0243)\n","Processed non_AMPEP43599: non-AMP/OTHERS (Probabilities: 0.9804, 0.0159, 0.0037)\n","Processed non_AMPEP123180: non-AMP/OTHERS (Probabilities: 0.9733, 0.0052, 0.0215)\n","Processed non_AMPEP24610: non-AMP/OTHERS (Probabilities: 0.9919, 0.0037, 0.0044)\n","Processed non_AMPEP65612: AMP (Probabilities: 0.1942, 0.7933, 0.0125)\n","Processed non_AMPEP141581: non-AMP/OTHERS (Probabilities: 0.9918, 0.0046, 0.0035)\n","Processed non_AMPEP153589: non-AMP/OTHERS (Probabilities: 0.9906, 0.0051, 0.0043)\n","Processed non_AMPEP163651: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0037)\n","Processed non_AMPEP64554: non-AMP/OTHERS (Probabilities: 0.9886, 0.0071, 0.0043)\n","Processed non_AMPEP88479: non-AMP/OTHERS (Probabilities: 0.9865, 0.0090, 0.0045)\n","Processed non_AMPEP154898: non-AMP/OTHERS (Probabilities: 0.9575, 0.0086, 0.0339)\n","Processed non_AMPEP119484: non-AMP/OTHERS (Probabilities: 0.9927, 0.0037, 0.0036)\n","Processed non_AMPEP118892: non-AMP/OTHERS (Probabilities: 0.9607, 0.0203, 0.0190)\n","Processed non_AMPEP57645: non-AMP/OTHERS (Probabilities: 0.9909, 0.0047, 0.0044)\n","Processed non_AMPEP31422: non-AMP/OTHERS (Probabilities: 0.9918, 0.0039, 0.0043)\n","Processed non_AMPEP40141: AMP (Probabilities: 0.0657, 0.9199, 0.0144)\n","Processed non_AMPEP131329: non-AMP/OTHERS (Probabilities: 0.9927, 0.0037, 0.0036)\n","Processed non_AMPEP27846: non-AMP/OTHERS (Probabilities: 0.9915, 0.0054, 0.0031)\n","Processed non_AMPEP34752: Neuropeptide (Probabilities: 0.1953, 0.0562, 0.7485)\n","Processed non_AMPEP94391: non-AMP/OTHERS (Probabilities: 0.9893, 0.0033, 0.0075)\n","Processed non_AMPEP101737: non-AMP/OTHERS (Probabilities: 0.9895, 0.0063, 0.0041)\n","Processed non_AMPEP150301: non-AMP/OTHERS (Probabilities: 0.9919, 0.0041, 0.0040)\n","Processed non_AMPEP116207: non-AMP/OTHERS (Probabilities: 0.9926, 0.0038, 0.0036)\n","Processed non_AMPEP7394: AMP (Probabilities: 0.0923, 0.8959, 0.0117)\n","Processed non_AMPEP111831: non-AMP/OTHERS (Probabilities: 0.9912, 0.0044, 0.0044)\n","Processed non_AMPEP161763: non-AMP/OTHERS (Probabilities: 0.9888, 0.0083, 0.0030)\n","Processed non_AMPEP101118: non-AMP/OTHERS (Probabilities: 0.9865, 0.0097, 0.0039)\n","Processed non_AMPEP76065: non-AMP/OTHERS (Probabilities: 0.9928, 0.0044, 0.0028)\n","Processed non_AMPEP6015: non-AMP/OTHERS (Probabilities: 0.9907, 0.0056, 0.0037)\n","Processed non_AMPEP71653: non-AMP/OTHERS (Probabilities: 0.9904, 0.0050, 0.0046)\n","Processed non_AMPEP58730: non-AMP/OTHERS (Probabilities: 0.9508, 0.0451, 0.0041)\n","Processed non_AMPEP100380: AMP (Probabilities: 0.1279, 0.8598, 0.0123)\n","Processed non_AMPEP3012: non-AMP/OTHERS (Probabilities: 0.9901, 0.0054, 0.0045)\n","Processed non_AMPEP100485: non-AMP/OTHERS (Probabilities: 0.9926, 0.0045, 0.0029)\n","Processed non_AMPEP103714: non-AMP/OTHERS (Probabilities: 0.9924, 0.0039, 0.0036)\n","Processed non_AMPEP144449: non-AMP/OTHERS (Probabilities: 0.9912, 0.0055, 0.0033)\n","Processed non_AMPEP107436: non-AMP/OTHERS (Probabilities: 0.9894, 0.0054, 0.0052)\n","Processed non_AMPEP120502: non-AMP/OTHERS (Probabilities: 0.9921, 0.0050, 0.0029)\n","Processed non_AMPEP70626: AMP (Probabilities: 0.1627, 0.8237, 0.0136)\n","Processed non_AMPEP131069: non-AMP/OTHERS (Probabilities: 0.9900, 0.0063, 0.0036)\n","Processed non_AMPEP24067: non-AMP/OTHERS (Probabilities: 0.9892, 0.0054, 0.0053)\n","Processed non_AMPEP69932: non-AMP/OTHERS (Probabilities: 0.9907, 0.0060, 0.0033)\n","Processed non_AMPEP4527: non-AMP/OTHERS (Probabilities: 0.9877, 0.0092, 0.0031)\n","Processed non_AMPEP76093: non-AMP/OTHERS (Probabilities: 0.9814, 0.0159, 0.0027)\n","Processed non_AMPEP97204: non-AMP/OTHERS (Probabilities: 0.9927, 0.0039, 0.0034)\n","Processed non_AMPEP28819: non-AMP/OTHERS (Probabilities: 0.9895, 0.0072, 0.0033)\n","Processed non_AMPEP133950: non-AMP/OTHERS (Probabilities: 0.9909, 0.0056, 0.0035)\n","Processed non_AMPEP55849: non-AMP/OTHERS (Probabilities: 0.9724, 0.0238, 0.0038)\n","Processed non_AMPEP128291: AMP (Probabilities: 0.1275, 0.8591, 0.0134)\n","Processed non_AMPEP97708: non-AMP/OTHERS (Probabilities: 0.9868, 0.0097, 0.0035)\n","Processed non_AMPEP118933: non-AMP/OTHERS (Probabilities: 0.9928, 0.0034, 0.0037)\n","Processed non_AMPEP24176: non-AMP/OTHERS (Probabilities: 0.8533, 0.1390, 0.0077)\n","Processed non_AMPEP134688: non-AMP/OTHERS (Probabilities: 0.8889, 0.0116, 0.0995)\n","Processed non_AMPEP26578: non-AMP/OTHERS (Probabilities: 0.9875, 0.0102, 0.0023)\n","Processed non_AMPEP56718: non-AMP/OTHERS (Probabilities: 0.9933, 0.0035, 0.0032)\n","Processed non_AMPEP41495: non-AMP/OTHERS (Probabilities: 0.9801, 0.0173, 0.0026)\n","Processed non_AMPEP10227: non-AMP/OTHERS (Probabilities: 0.9131, 0.0825, 0.0044)\n","Processed non_AMPEP28486: non-AMP/OTHERS (Probabilities: 0.9929, 0.0040, 0.0031)\n","Processed non_AMPEP44848: non-AMP/OTHERS (Probabilities: 0.9873, 0.0096, 0.0031)\n","Processed non_AMPEP31671: non-AMP/OTHERS (Probabilities: 0.9918, 0.0035, 0.0047)\n","Processed non_AMPEP35498: non-AMP/OTHERS (Probabilities: 0.9932, 0.0036, 0.0032)\n","Processed non_AMPEP157508: non-AMP/OTHERS (Probabilities: 0.9927, 0.0040, 0.0034)\n","Processed non_AMPEP14531: non-AMP/OTHERS (Probabilities: 0.9919, 0.0045, 0.0037)\n","Processed non_AMPEP145318: non-AMP/OTHERS (Probabilities: 0.9928, 0.0036, 0.0037)\n","Processed non_AMPEP88612: non-AMP/OTHERS (Probabilities: 0.9869, 0.0096, 0.0035)\n","Processed non_AMPEP15252: non-AMP/OTHERS (Probabilities: 0.9832, 0.0136, 0.0032)\n","Processed non_AMPEP16807: non-AMP/OTHERS (Probabilities: 0.9927, 0.0038, 0.0036)\n","Processed non_AMPEP139354: non-AMP/OTHERS (Probabilities: 0.6192, 0.3557, 0.0251)\n","Processed non_AMPEP108242: non-AMP/OTHERS (Probabilities: 0.9917, 0.0041, 0.0042)\n","Processed non_AMPEP115925: non-AMP/OTHERS (Probabilities: 0.7243, 0.2664, 0.0094)\n","Processed non_AMPEP100513: non-AMP/OTHERS (Probabilities: 0.9928, 0.0034, 0.0038)\n","Processed non_AMPEP152897: Neuropeptide (Probabilities: 0.2315, 0.0219, 0.7466)\n","Processed non_AMPEP16725: non-AMP/OTHERS (Probabilities: 0.9885, 0.0055, 0.0060)\n","Processed non_AMPEP95642: non-AMP/OTHERS (Probabilities: 0.9919, 0.0052, 0.0029)\n","Processed non_AMPEP133242: non-AMP/OTHERS (Probabilities: 0.9925, 0.0036, 0.0039)\n","Processed non_AMPEP51847: non-AMP/OTHERS (Probabilities: 0.9922, 0.0043, 0.0034)\n","Processed non_AMPEP59412: non-AMP/OTHERS (Probabilities: 0.9744, 0.0095, 0.0160)\n","Processed non_AMPEP46188: non-AMP/OTHERS (Probabilities: 0.9237, 0.0131, 0.0632)\n","Processed non_AMPEP143431: non-AMP/OTHERS (Probabilities: 0.9916, 0.0039, 0.0045)\n","Processed non_AMPEP158036: non-AMP/OTHERS (Probabilities: 0.9925, 0.0039, 0.0036)\n","Processed non_AMPEP89635: non-AMP/OTHERS (Probabilities: 0.9578, 0.0378, 0.0043)\n","Processed non_AMPEP14603: non-AMP/OTHERS (Probabilities: 0.9911, 0.0059, 0.0030)\n","Processed non_AMPEP94372: non-AMP/OTHERS (Probabilities: 0.9921, 0.0040, 0.0039)\n","Processed non_AMPEP8850: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed non_AMPEP100261: non-AMP/OTHERS (Probabilities: 0.9910, 0.0059, 0.0031)\n","Processed non_AMPEP6807: non-AMP/OTHERS (Probabilities: 0.9916, 0.0056, 0.0028)\n","Processed non_AMPEP156343: non-AMP/OTHERS (Probabilities: 0.9924, 0.0035, 0.0041)\n","Processed non_AMPEP36813: non-AMP/OTHERS (Probabilities: 0.9202, 0.0730, 0.0068)\n","Processed non_AMPEP74085: AMP (Probabilities: 0.3869, 0.5973, 0.0158)\n","Processed non_AMPEP43369: non-AMP/OTHERS (Probabilities: 0.9925, 0.0037, 0.0038)\n","Processed non_AMPEP147180: non-AMP/OTHERS (Probabilities: 0.9925, 0.0038, 0.0037)\n","Processed non_AMPEP41401: non-AMP/OTHERS (Probabilities: 0.9896, 0.0063, 0.0041)\n","Processed non_AMPEP11499: non-AMP/OTHERS (Probabilities: 0.9920, 0.0049, 0.0030)\n","Processed non_AMPEP38756: non-AMP/OTHERS (Probabilities: 0.9912, 0.0058, 0.0030)\n","Processed non_AMPEP37530: non-AMP/OTHERS (Probabilities: 0.9846, 0.0116, 0.0038)\n","Processed non_AMPEP1375: non-AMP/OTHERS (Probabilities: 0.9310, 0.0643, 0.0047)\n","Processed non_AMPEP3910: non-AMP/OTHERS (Probabilities: 0.9681, 0.0281, 0.0038)\n","Processed non_AMPEP122721: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed non_AMPEP101472: non-AMP/OTHERS (Probabilities: 0.9703, 0.0238, 0.0059)\n","Processed non_AMPEP147235: Neuropeptide (Probabilities: 0.0142, 0.0119, 0.9740)\n","Processed non_AMPEP165790: non-AMP/OTHERS (Probabilities: 0.9644, 0.0224, 0.0132)\n","Processed non_AMPEP104117: non-AMP/OTHERS (Probabilities: 0.9123, 0.0383, 0.0493)\n","Processed non_AMPEP143788: non-AMP/OTHERS (Probabilities: 0.9913, 0.0047, 0.0040)\n","Processed non_AMPEP78790: non-AMP/OTHERS (Probabilities: 0.9705, 0.0236, 0.0058)\n","Processed non_AMPEP4516: non-AMP/OTHERS (Probabilities: 0.9918, 0.0037, 0.0045)\n","Processed non_AMPEP136426: non-AMP/OTHERS (Probabilities: 0.9926, 0.0042, 0.0031)\n","Processed non_AMPEP20389: non-AMP/OTHERS (Probabilities: 0.9879, 0.0090, 0.0031)\n","Processed non_AMPEP44000: non-AMP/OTHERS (Probabilities: 0.9911, 0.0053, 0.0036)\n","Processed non_AMPEP125943: non-AMP/OTHERS (Probabilities: 0.9530, 0.0405, 0.0064)\n","Processed non_AMPEP124687: non-AMP/OTHERS (Probabilities: 0.9924, 0.0041, 0.0035)\n","Processed non_AMPEP154478: non-AMP/OTHERS (Probabilities: 0.9898, 0.0068, 0.0034)\n","Processed non_AMPEP101100: non-AMP/OTHERS (Probabilities: 0.9700, 0.0260, 0.0040)\n","Processed non_AMPEP59273: non-AMP/OTHERS (Probabilities: 0.9868, 0.0041, 0.0091)\n","Processed non_AMPEP101309: non-AMP/OTHERS (Probabilities: 0.9842, 0.0104, 0.0055)\n","Processed non_AMPEP140241: non-AMP/OTHERS (Probabilities: 0.9903, 0.0064, 0.0033)\n","Processed non_AMPEP31397: non-AMP/OTHERS (Probabilities: 0.9884, 0.0078, 0.0038)\n","Processed non_AMPEP28121: non-AMP/OTHERS (Probabilities: 0.9703, 0.0260, 0.0037)\n","Processed non_AMPEP78201: non-AMP/OTHERS (Probabilities: 0.9926, 0.0036, 0.0038)\n","Processed non_AMPEP102728: non-AMP/OTHERS (Probabilities: 0.9925, 0.0043, 0.0031)\n","Processed non_AMPEP115149: non-AMP/OTHERS (Probabilities: 0.9888, 0.0080, 0.0031)\n","Processed non_AMPEP122929: non-AMP/OTHERS (Probabilities: 0.9910, 0.0057, 0.0033)\n","Processed non_AMPEP130580: non-AMP/OTHERS (Probabilities: 0.9770, 0.0196, 0.0035)\n","Processed non_AMPEP137874: non-AMP/OTHERS (Probabilities: 0.9921, 0.0048, 0.0031)\n","Processed non_AMPEP19488: non-AMP/OTHERS (Probabilities: 0.9895, 0.0067, 0.0038)\n","Processed non_AMPEP147149: non-AMP/OTHERS (Probabilities: 0.9898, 0.0066, 0.0035)\n","Processed non_AMPEP101627: non-AMP/OTHERS (Probabilities: 0.9920, 0.0036, 0.0044)\n","Processed non_AMPEP62421: non-AMP/OTHERS (Probabilities: 0.4844, 0.4823, 0.0333)\n","Processed non_AMPEP131546: non-AMP/OTHERS (Probabilities: 0.9905, 0.0058, 0.0037)\n","Processed non_AMPEP11597: non-AMP/OTHERS (Probabilities: 0.9927, 0.0043, 0.0030)\n","Processed non_AMPEP139833: non-AMP/OTHERS (Probabilities: 0.7927, 0.0294, 0.1779)\n","Processed non_AMPEP148284: non-AMP/OTHERS (Probabilities: 0.9914, 0.0040, 0.0046)\n","Processed non_AMPEP130734: non-AMP/OTHERS (Probabilities: 0.9796, 0.0171, 0.0033)\n","Processed non_AMPEP57513: non-AMP/OTHERS (Probabilities: 0.9844, 0.0091, 0.0065)\n","Processed non_AMPEP152770: Neuropeptide (Probabilities: 0.3642, 0.1309, 0.5049)\n","Processed non_AMPEP24487: non-AMP/OTHERS (Probabilities: 0.9912, 0.0050, 0.0038)\n","Processed non_AMPEP34427: non-AMP/OTHERS (Probabilities: 0.9894, 0.0070, 0.0036)\n","Processed non_AMPEP41202: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0036)\n","Processed non_AMPEP128808: non-AMP/OTHERS (Probabilities: 0.8728, 0.1179, 0.0092)\n","Processed non_AMPEP29942: non-AMP/OTHERS (Probabilities: 0.9903, 0.0056, 0.0041)\n","Processed non_AMPEP155297: non-AMP/OTHERS (Probabilities: 0.7254, 0.1139, 0.1607)\n","Processed non_AMPEP58968: non-AMP/OTHERS (Probabilities: 0.9893, 0.0063, 0.0044)\n","Processed non_AMPEP62030: non-AMP/OTHERS (Probabilities: 0.9909, 0.0044, 0.0046)\n","Processed non_AMPEP119275: non-AMP/OTHERS (Probabilities: 0.9929, 0.0041, 0.0030)\n","Processed non_AMPEP51602: non-AMP/OTHERS (Probabilities: 0.9886, 0.0079, 0.0035)\n","Processed non_AMPEP136449: non-AMP/OTHERS (Probabilities: 0.9471, 0.0487, 0.0042)\n","Processed non_AMPEP81971: AMP (Probabilities: 0.1112, 0.8699, 0.0189)\n","Processed non_AMPEP148238: AMP (Probabilities: 0.0664, 0.9185, 0.0152)\n","Processed non_AMPEP76736: non-AMP/OTHERS (Probabilities: 0.9912, 0.0057, 0.0031)\n","Processed non_AMPEP141234: AMP (Probabilities: 0.0915, 0.8955, 0.0130)\n","Processed non_AMPEP56879: non-AMP/OTHERS (Probabilities: 0.9897, 0.0066, 0.0037)\n","Processed non_AMPEP109176: non-AMP/OTHERS (Probabilities: 0.9919, 0.0038, 0.0043)\n","Processed non_AMPEP73693: non-AMP/OTHERS (Probabilities: 0.7923, 0.2012, 0.0066)\n","Processed non_AMPEP4396: non-AMP/OTHERS (Probabilities: 0.9903, 0.0065, 0.0033)\n","Processed non_AMPEP45688: non-AMP/OTHERS (Probabilities: 0.9910, 0.0054, 0.0037)\n","Processed non_AMPEP17167: non-AMP/OTHERS (Probabilities: 0.9925, 0.0041, 0.0034)\n","Processed non_AMPEP83666: non-AMP/OTHERS (Probabilities: 0.6628, 0.0313, 0.3058)\n","Processed non_AMPEP7295: non-AMP/OTHERS (Probabilities: 0.9906, 0.0054, 0.0040)\n","Processed non_AMPEP107240: non-AMP/OTHERS (Probabilities: 0.9777, 0.0186, 0.0038)\n","Processed non_AMPEP163504: non-AMP/OTHERS (Probabilities: 0.9906, 0.0039, 0.0054)\n","Processed non_AMPEP72775: non-AMP/OTHERS (Probabilities: 0.9920, 0.0041, 0.0039)\n","Processed non_AMPEP34376: non-AMP/OTHERS (Probabilities: 0.9924, 0.0041, 0.0034)\n","Processed non_AMPEP52920: AMP (Probabilities: 0.1284, 0.8605, 0.0111)\n","Processed non_AMPEP51209: non-AMP/OTHERS (Probabilities: 0.9889, 0.0078, 0.0033)\n","Processed non_AMPEP101562: non-AMP/OTHERS (Probabilities: 0.9865, 0.0096, 0.0040)\n","Processed non_AMPEP102131: non-AMP/OTHERS (Probabilities: 0.8903, 0.0235, 0.0862)\n","Processed non_AMPEP9818: non-AMP/OTHERS (Probabilities: 0.9926, 0.0038, 0.0036)\n","Processed non_AMPEP147835: non-AMP/OTHERS (Probabilities: 0.8004, 0.1911, 0.0085)\n","Processed non_AMPEP56174: non-AMP/OTHERS (Probabilities: 0.9903, 0.0048, 0.0049)\n","Processed non_AMPEP83560: non-AMP/OTHERS (Probabilities: 0.9884, 0.0079, 0.0037)\n","Processed non_AMPEP137151: non-AMP/OTHERS (Probabilities: 0.9877, 0.0093, 0.0030)\n","Processed non_AMPEP145586: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed non_AMPEP30413: non-AMP/OTHERS (Probabilities: 0.9924, 0.0049, 0.0027)\n","Processed non_AMPEP112337: non-AMP/OTHERS (Probabilities: 0.9908, 0.0054, 0.0038)\n","Processed non_AMPEP62660: non-AMP/OTHERS (Probabilities: 0.9900, 0.0064, 0.0036)\n","Processed non_AMPEP117563: non-AMP/OTHERS (Probabilities: 0.9903, 0.0055, 0.0042)\n","Processed non_AMPEP112090: non-AMP/OTHERS (Probabilities: 0.9927, 0.0044, 0.0029)\n","El número de secuencias de Non-AMPs es: 666\n","El número de secuencias de AMP y neuropéptidos es: 309\n"]}]},{"cell_type":"markdown","source":["## TEST DE PREDICCIÓN DE NEUROPEPTIDOS EN BACTERIOCINAS"],"metadata":{"id":"0_m4J2x7g34l"}},{"cell_type":"code","source":["import re\n","import torch\n","\n","# Función para leer un archivo FASTA\n","def read_fasta(file_path):\n","    sequences = []\n","    with open(file_path, 'r') as f:\n","        identifier = \"\"\n","        sequence = \"\"\n","        for line in f:\n","            line = line.strip()\n","            if line.startswith(\">\"):  # Es un identificador de secuencia\n","                if sequence:  # Guarda la secuencia anterior si existe\n","                    sequences.append((identifier, sequence))\n","                    sequence = \"\"\n","                identifier = line[1:]  # Guarda el identificador sin el \">\"\n","            else:\n","                sequence += line  # Agrega las líneas de la secuencia\n","        if sequence:  # Guarda la última secuencia\n","            sequences.append((identifier, sequence))\n","    return sequences\n","\n","# Función para hacer predicciones de AMP/non-AMP para múltiples secuencias y guardar probabilidades\n","def predict_amp(sequences, output_file):\n","    with open(output_file, 'w') as out_f:\n","        out_f.write(\"Identifier\\tSequence\\tPrediction\\tProbability\\n\")  # Encabezados\n","        for identifier, input_seq in sequences:\n","            input_seq_spaced = ' '.join([input_seq[i:i+1] for i in range(0, len(input_seq), 1)])\n","            input_seq_spaced = re.sub(r'[UZOB]', 'X', input_seq_spaced)  # Reemplazar aminoácidos no estándar\n","            input_seq_tok = tokenizer(input_seq_spaced, return_tensors='pt')\n","            #print(input_seq_tok)\n","\n","            output = model(**input_seq_tok)\n","            logits = output[0]\n","           # print(f\"Logits for {identifier}: {logits}\")\n","\n","            # Extraer la probabilidad de clase AMP\n","            #y_prob = torch.sigmoid(logits)[:, 1].detach().numpy()\n","            #y_pred = y_prob > 0.84  # Predicción binaria con umbral de 0.5\n","            #prob = y_prob[0]  # Probabilidad real (sin umbral)\n","\n","            # Aplicar softmax para obtener probabilidades de cada clase\n","            y_prob = torch.softmax(logits, dim=1).detach().numpy()\n","\n","            # Obtener la clase con la probabilidad más alta\n","            y_pred_class = y_prob.argmax(axis=1)[0]\n","            prob_class1, prob_class2, prob_class3 = y_prob[0]  # Probabilidades para las tres clases\n","            # Determinar la etiqueta de la predicción según la clase\n","            if y_pred_class == 0:\n","                input_class = 'non-AMP/OTHERS'  # Cambia estos nombres según tus etiquetas\n","            elif y_pred_class == 1:\n","                input_class = 'AMP'\n","            else:\n","                input_class = 'Neuropeptide'\n","\n","            # Escribir el identificador, secuencia, predicción y probabilidades en el archivo de salida\n","            out_f.write(f\"{identifier}\\t{input_seq}\\t{input_class}\\t{prob_class1:.4f}\\t{prob_class2:.4f}\\t{prob_class3:.4f}\\n\")\n","            print(f\"Processed {identifier}: {input_class} (Probabilities: {prob_class1:.4f}, {prob_class2:.4f}, {prob_class3:.4f})\")\n","\n","\n","            # Determinar clase (AMP o non-AMP)\n","            #if y_pred:\n","            #    input_class = 'AMP'\n","            #else:\n","            #    input_class = 'non-AMP'\n","\n","            # Escribir el identificador, secuencia, predicción y probabilidad en el archivo de salida\n","           # out_f.write(f\"{identifier}\\t{input_seq}\\t{input_class}\\t{prob:.4f}\\n\")\n","           # print(f\"Processed {identifier}: {input_class} (Probability: {prob:.4f})\")\n","\n","# Ruta del archivo FASTA de entrada y del archivo de salida\n","fasta_file = '/content/drive/MyDrive/data/bacteriocin_all_class_and_colicin_and_microcin_protlevel_200.fasta'  # Nombre del archivo FASTA con secuencias\n","output_file = '/content/drive/MyDrive/Datos Finales para Memoria/resultados/RESULTADO_prediccion_bacteriocinas.tsv'  # Archivo de salida con predicciones y probabilidades\n","\n","# Leer las secuencias del archivo FASTA\n","sequences = read_fasta(fasta_file)\n","\n","# Realizar predicciones y escribir resultados en un archivo\n","predict_amp(sequences, output_file)\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"3q3GujE3_csL","executionInfo":{"status":"ok","timestamp":1733928525293,"user_tz":-60,"elapsed":1500486,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"199af4a2-35fe-4b50-b9dc-d50b0340d892"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Processed 1.1;Anacyclamide_(AcyE): non-AMP/OTHERS (Probabilities: 0.9920, 0.0034, 0.0046)\n","Processed 10.1;Bovicin_HJ50: non-AMP/OTHERS (Probabilities: 0.9901, 0.0041, 0.0058)\n","Processed 100.1;Ruminococcin_A_: non-AMP/OTHERS (Probabilities: 0.9928, 0.0030, 0.0043)\n","Processed 101.1;Salivaricin_9: non-AMP/OTHERS (Probabilities: 0.8151, 0.1751, 0.0098)\n","Processed 102.1;Salivaricin_A: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0040)\n","Processed 103.1;Salivaricin_A2: non-AMP/OTHERS (Probabilities: 0.9924, 0.0033, 0.0043)\n","Processed 104.1;Salivaricin_A3: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0035)\n","Processed 105.1;Salivaricin_A4: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed 106.1;Salivaricin_A5: non-AMP/OTHERS (Probabilities: 0.9926, 0.0035, 0.0039)\n","Processed 107.1;Salivaricin_D: AMP (Probabilities: 0.2185, 0.7641, 0.0174)\n","Processed 108.1;Salivaricin_G32: non-AMP/OTHERS (Probabilities: 0.9878, 0.0064, 0.0058)\n","Processed 109.1;SalivaricinA: non-AMP/OTHERS (Probabilities: 0.9934, 0.0033, 0.0033)\n","Processed 11.1;BsaA2: non-AMP/OTHERS (Probabilities: 0.8897, 0.0135, 0.0968)\n","Processed 110.1;Sap_B: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0044)\n","Processed 111.1;SCAB_84151_putative_Linaridin: non-AMP/OTHERS (Probabilities: 0.9905, 0.0065, 0.0030)\n","Processed 112.1;SCAB_84201_putativeLinaridin: AMP (Probabilities: 0.3670, 0.6222, 0.0109)\n","Processed 113.1;SGR_1511_putative_Linaridin: non-AMP/OTHERS (Probabilities: 0.9796, 0.0173, 0.0031)\n","Processed 114.1;SGR_1512_putative_Linaridin: non-AMP/OTHERS (Probabilities: 0.9868, 0.0069, 0.0063)\n","Processed 115.1;SGR_1513_putative_Linaridin: AMP (Probabilities: 0.2724, 0.7074, 0.0202)\n","Processed 116.1;SGR_1514_putative_Linaridin: Neuropeptide (Probabilities: 0.2041, 0.0295, 0.7664)\n","Processed 117.1;siamycin_I_(BMY-29304): AMP (Probabilities: 0.0857, 0.9020, 0.0123)\n","Processed 118.1;siamycin_II_(BMY-29303): AMP (Probabilities: 0.0734, 0.9135, 0.0131)\n","Processed 119.1;SmbA: non-AMP/OTHERS (Probabilities: 0.9863, 0.0106, 0.0030)\n","Processed 12.1;Butyrivibriocin: non-AMP/OTHERS (Probabilities: 0.4632, 0.2730, 0.2638)\n","Processed 120.1;SmbB: AMP (Probabilities: 0.0766, 0.9110, 0.0124)\n","Processed 121.1;Sporulation-killingfactor_skfA: AMP (Probabilities: 0.0679, 0.9185, 0.0136)\n","Processed 122.1;Staphylococcins_C55a_SacaA: non-AMP/OTHERS (Probabilities: 0.6542, 0.0297, 0.3161)\n","Processed 123.1;Staphylococcins_C55b_SacbA: non-AMP/OTHERS (Probabilities: 0.9648, 0.0314, 0.0038)\n","Processed 124.1;Streptin: AMP (Probabilities: 0.0673, 0.9181, 0.0146)\n","Processed 125.1;Streptococcin_A-FF22: non-AMP/OTHERS (Probabilities: 0.9883, 0.0039, 0.0077)\n","Processed 126.1;StreptococcinA-M49: non-AMP/OTHERS (Probabilities: 0.9878, 0.0038, 0.0084)\n","Processed 127.1;Sublancin_168: AMP (Probabilities: 0.0650, 0.9201, 0.0150)\n","Processed 128.1;Subtilin: AMP (Probabilities: 0.0738, 0.9136, 0.0126)\n","Processed 129.1;SvirD4_22614_putative_Linaridin: non-AMP/OTHERS (Probabilities: 0.9896, 0.0040, 0.0065)\n","Processed 13.1;Butyrivibriocin: Neuropeptide (Probabilities: 0.3448, 0.2952, 0.3600)\n","Processed 130.1;Thermophilin1277: non-AMP/OTHERS (Probabilities: 0.9901, 0.0041, 0.0058)\n","Processed 131.1;Thiocillin_(MicrococcinP1)_(MicrococcinP2)_(ThiocillinI)_(ThiocillinII)_(ThiocillinIII)_(ThiocillinIV)_(AntibioticYM-266183)_(AntibioticYM-266184): non-AMP/OTHERS (Probabilities: 0.9919, 0.0043, 0.0038)\n","Processed 132.1;Thiocillin_GE37468: non-AMP/OTHERS (Probabilities: 0.8200, 0.1058, 0.0742)\n","Processed 133.1;Thiomuracin_A_(Thiomuracin_B)_(Thiomuracin_C)_(Thiomuracin_D)_(Thiomuracin_E)_(Thiomuracin_F)_(Thiomuracin_G)_(Thiomuracin_H)_(Thiomuracin_I): Neuropeptide (Probabilities: 0.3444, 0.0719, 0.5838)\n","Processed 134.1;Thiostrepton_A: non-AMP/OTHERS (Probabilities: 0.9807, 0.0126, 0.0068)\n","Processed 135.1;Thuricin_CD_beta: AMP (Probabilities: 0.2190, 0.7654, 0.0155)\n","Processed 136.1;Thurincin_H_(thuricin17): non-AMP/OTHERS (Probabilities: 0.9904, 0.0035, 0.0061)\n","Processed 137.1;ThuricinCD_alfa: non-AMP/OTHERS (Probabilities: 0.9790, 0.0177, 0.0032)\n","Processed 138.1;Trichamide: non-AMP/OTHERS (Probabilities: 0.7228, 0.0156, 0.2617)\n","Processed 139.1;Trunkamide: Neuropeptide (Probabilities: 0.2497, 0.0261, 0.7242)\n","Processed 14.1;Butyrivibriocin_OR79: non-AMP/OTHERS (Probabilities: 0.5338, 0.4431, 0.0231)\n","Processed 140.1;Ulithiacyclamide: non-AMP/OTHERS (Probabilities: 0.5055, 0.0463, 0.4482)\n","Processed 141.1;Variacin: non-AMP/OTHERS (Probabilities: 0.6014, 0.2000, 0.1986)\n","Processed 142.1;Venezuelin: non-AMP/OTHERS (Probabilities: 0.9917, 0.0035, 0.0048)\n","Processed 143.1;Lasso Peptide: AMP (Probabilities: 0.0857, 0.9020, 0.0123)\n","Processed 144.1;Thiocillin_GE2270: Neuropeptide (Probabilities: 0.0634, 0.0666, 0.8699)\n","Processed 145.1;Subtilosin_A: non-AMP/OTHERS (Probabilities: 0.9935, 0.0033, 0.0033)\n","Processed 146.1;Capistruin: non-AMP/OTHERS (Probabilities: 0.6352, 0.0438, 0.3210)\n","Processed 147.1;Circularin_A: AMP (Probabilities: 0.0680, 0.9176, 0.0144)\n","Processed 148.1;Carnocyclin-A: AMP (Probabilities: 0.0701, 0.9155, 0.0144)\n","Processed 149.1;Butyrivibriocin_AR10: non-AMP/OTHERS (Probabilities: 0.9914, 0.0058, 0.0028)\n","Processed 15.1;Catenulipeptin: Neuropeptide (Probabilities: 0.2156, 0.0248, 0.7596)\n","Processed 150.1;Enterocin_AS-48: AMP (Probabilities: 0.1408, 0.8476, 0.0116)\n","Processed 151.1;Enterocin_AS-48RJ: AMP (Probabilities: 0.0678, 0.9178, 0.0145)\n","Processed 152.1;Garvicin_ML: AMP (Probabilities: 0.0657, 0.9189, 0.0154)\n","Processed 153.1;Gassericin_A: non-AMP/OTHERS (Probabilities: 0.9597, 0.0365, 0.0038)\n","Processed 154.1;Lactocyclicin_Q: AMP (Probabilities: 0.0714, 0.9151, 0.0135)\n","Processed 155.1;Leucocyclicin_Q: AMP (Probabilities: 0.0687, 0.9164, 0.0148)\n","Processed 156.1;Microcin_J25(MccJ25): AMP (Probabilities: 0.0895, 0.8990, 0.0115)\n","Processed 157.1;uberolysin: AMP (Probabilities: 0.2395, 0.7490, 0.0114)\n","Processed 158.1;Acidocin_B: non-AMP/OTHERS (Probabilities: 0.9427, 0.0529, 0.0043)\n","Processed 159.1;carnolysins A1: AMP (Probabilities: 0.0672, 0.9191, 0.0137)\n","Processed 16.1;Cinnamycin_(Ro09-0198): non-AMP/OTHERS (Probabilities: 0.7859, 0.0834, 0.1307)\n","Processed 160.1;carnolysins A2: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0034)\n","Processed 161.1;Flavucin: AMP (Probabilities: 0.0690, 0.9178, 0.0132)\n","Processed 162.1;Mathermycin: non-AMP/OTHERS (Probabilities: 0.8364, 0.0408, 0.1228)\n","Processed 163.1;Hyicin 3682: non-AMP/OTHERS (Probabilities: 0.9920, 0.0041, 0.0039)\n","Processed 164.1;Pneumolancidin PldA1: AMP (Probabilities: 0.0647, 0.9205, 0.0148)\n","Processed 165.1;Pneumolancidin PldA2: AMP (Probabilities: 0.0664, 0.9190, 0.0145)\n","Processed 166.1;Pneumolancidin PldA3: non-AMP/OTHERS (Probabilities: 0.7019, 0.2805, 0.0176)\n","Processed 167.1;Pneumolancidin PldA4: AMP (Probabilities: 0.0671, 0.9192, 0.0137)\n","Processed 168.1;Cerecidin (CerA1): non-AMP/OTHERS (Probabilities: 0.9859, 0.0095, 0.0046)\n","Processed 169.1;Cerecidin (CerA2): non-AMP/OTHERS (Probabilities: 0.9859, 0.0095, 0.0046)\n","Processed 17.1;Cypemycin: non-AMP/OTHERS (Probabilities: 0.9931, 0.0042, 0.0027)\n","Processed 170.1;Cerecidin (CerA3): non-AMP/OTHERS (Probabilities: 0.9872, 0.0074, 0.0054)\n","Processed 171.1;Cerecidin (CerA4): non-AMP/OTHERS (Probabilities: 0.9859, 0.0095, 0.0046)\n","Processed 172.1;Cerecidin (CerA5): non-AMP/OTHERS (Probabilities: 0.9872, 0.0074, 0.0054)\n","Processed 173.1;Cerecidin (CerA6): non-AMP/OTHERS (Probabilities: 0.9872, 0.0074, 0.0054)\n","Processed 174.1;Cerecidin (CerA7): non-AMP/OTHERS (Probabilities: 0.9868, 0.0093, 0.0039)\n","Processed 175.1;FlvA1: AMP (Probabilities: 0.0647, 0.9195, 0.0158)\n","Processed 176.1;FlvA2a: non-AMP/OTHERS (Probabilities: 0.9920, 0.0038, 0.0042)\n","Processed 177.1;FlvA2b: AMP (Probabilities: 0.2032, 0.7780, 0.0188)\n","Processed 178.1;FlvA2c: AMP (Probabilities: 0.3977, 0.5890, 0.0134)\n","Processed 179.1;FlvA2d: non-AMP/OTHERS (Probabilities: 0.8878, 0.1060, 0.0063)\n","Processed 18.1;Cytolysin_ClyLl: non-AMP/OTHERS (Probabilities: 0.9875, 0.0095, 0.0030)\n","Processed 180.1;FlvA2e: AMP (Probabilities: 0.0825, 0.9043, 0.0131)\n","Processed 181.1;FlvA2f: non-AMP/OTHERS (Probabilities: 0.9925, 0.0033, 0.0042)\n","Processed 182.1;FlvA2g: non-AMP/OTHERS (Probabilities: 0.9793, 0.0184, 0.0024)\n","Processed 183.1;FlvA2h: non-AMP/OTHERS (Probabilities: 0.9930, 0.0041, 0.0029)\n","Processed 184.1;Macedovicin: non-AMP/OTHERS (Probabilities: 0.9901, 0.0041, 0.0058)\n","Processed 185.1;BacCH91: non-AMP/OTHERS (Probabilities: 0.8595, 0.0378, 0.1027)\n","Processed 186.1;Streptocollin: non-AMP/OTHERS (Probabilities: 0.9849, 0.0126, 0.0025)\n","Processed 187.1;Thurandacin A: non-AMP/OTHERS (Probabilities: 0.5939, 0.0798, 0.3263)\n","Processed 188.1;Thurandacin B: non-AMP/OTHERS (Probabilities: 0.5939, 0.0798, 0.3263)\n","Processed 189.1;Enterocin F4-9: AMP (Probabilities: 0.1527, 0.7144, 0.1330)\n","Processed 19.1;Cytolysin_ClyLs: non-AMP/OTHERS (Probabilities: 0.8130, 0.0151, 0.1719)\n","Processed 190.1;Paenicidin A: AMP (Probabilities: 0.2193, 0.7274, 0.0533)\n","Processed 191.1;Curvopeptin: non-AMP/OTHERS (Probabilities: 0.9893, 0.0044, 0.0064)\n","Processed 192.1;Mutacin I: non-AMP/OTHERS (Probabilities: 0.9923, 0.0035, 0.0042)\n","Processed 193.1;Nai_112: non-AMP/OTHERS (Probabilities: 0.9119, 0.0089, 0.0792)\n","Processed 194.1;Paenicidin B: AMP (Probabilities: 0.0673, 0.9180, 0.0147)\n","Processed 195.1;Pinensin A: non-AMP/OTHERS (Probabilities: 0.9926, 0.0046, 0.0028)\n","Processed 196.1;Pinensin B: non-AMP/OTHERS (Probabilities: 0.9926, 0.0046, 0.0028)\n","Processed 197.1;Planosporicin: Neuropeptide (Probabilities: 0.3260, 0.0438, 0.6302)\n","Processed 198.1;Prochlorosin 1.1: non-AMP/OTHERS (Probabilities: 0.6214, 0.3309, 0.0477)\n","Processed 199.1;Prochlorosin 1.7: non-AMP/OTHERS (Probabilities: 0.6534, 0.2513, 0.0952)\n","Processed 2.1;Ancovenin: non-AMP/OTHERS (Probabilities: 0.9923, 0.0036, 0.0041)\n","Processed 20.1;Duramycin_(Leucopeptin)_(AntibioticPA48009): AMP (Probabilities: 0.0791, 0.9076, 0.0133)\n","Processed 200.1;Prochlorosin 2.11: non-AMP/OTHERS (Probabilities: 0.4697, 0.1127, 0.4176)\n","Processed 201.1;Prochlorosin 2.8: non-AMP/OTHERS (Probabilities: 0.4977, 0.0356, 0.4667)\n","Processed 202.1;Prochlorosin 3.3: non-AMP/OTHERS (Probabilities: 0.4758, 0.2672, 0.2570)\n","Processed 203.1;Prochlorosin 4.3: Neuropeptide (Probabilities: 0.4571, 0.0554, 0.4875)\n","Processed 204.1;SAL 2242: non-AMP/OTHERS (Probabilities: 0.9925, 0.0036, 0.0039)\n","Processed 205.1;Sap T: AMP (Probabilities: 0.0636, 0.9203, 0.0160)\n","Processed 206.1;SRO15 2212: non-AMP/OTHERS (Probabilities: 0.9923, 0.0029, 0.0048)\n","Processed 207.1;Stackepeptin A: Neuropeptide (Probabilities: 0.0160, 0.0118, 0.9722)\n","Processed 208.1;Stackepeptin B: Neuropeptide (Probabilities: 0.0162, 0.0118, 0.9720)\n","Processed 209.1;Stackepeptin C: Neuropeptide (Probabilities: 0.0160, 0.0118, 0.9722)\n","Processed 21.1;Duramycin_B: AMP (Probabilities: 0.1797, 0.7990, 0.0213)\n","Processed 210.1;Stackepeptin D: Neuropeptide (Probabilities: 0.0160, 0.0118, 0.9722)\n","Processed 211.1;Bottromycin B2: Neuropeptide (Probabilities: 0.0141, 0.0118, 0.9741)\n","Processed 212.1;Bottromycin D2: Neuropeptide (Probabilities: 0.0141, 0.0118, 0.9741)\n","Processed 213.1;Bottromycin E2: Neuropeptide (Probabilities: 0.0141, 0.0118, 0.9741)\n","Processed 214.1;Venturamide A: Neuropeptide (Probabilities: 0.0069, 0.0100, 0.9831)\n","Processed 215.1;Venturamide B: AMP (Probabilities: 0.1476, 0.7261, 0.1263)\n","Processed 216.1;Westiellamide: AMP (Probabilities: 0.0971, 0.8899, 0.0130)\n","Processed 217.1;Legonaridin: non-AMP/OTHERS (Probabilities: 0.9925, 0.0042, 0.0033)\n","Processed 218.1;Sonorensin: AMP (Probabilities: 0.0733, 0.9136, 0.0131)\n","Processed 219.1;Streptolysin S: AMP (Probabilities: 0.2093, 0.7757, 0.0150)\n","Processed 22.1;Duramycin_C: non-AMP/OTHERS (Probabilities: 0.7907, 0.2032, 0.0061)\n","Processed 220.1;Cyclothiazomycin: Neuropeptide (Probabilities: 0.0360, 0.0158, 0.9482)\n","Processed 221.1;GE2270_A_(GE2270_B1)_(GE2270_B2)_(GE2270_C1)_(GE2270_C2a)_(GE2270_C2b)_(GE2270_D1)_(GE2270_D2)_(GE2270_E)_(GE2270_T): Neuropeptide (Probabilities: 0.1507, 0.0300, 0.8193)\n","Processed 222.1;Thiocillin_GE37468_A: non-AMP/OTHERS (Probabilities: 0.8200, 0.1058, 0.0742)\n","Processed 223.1;mb-OB3b: AMP (Probabilities: 0.0728, 0.9140, 0.0132)\n","Processed 224.1;Nocathiacin_I_(Nocathiacin_II)_(Nocathiacin_III)_(Nocathiacin_IV): non-AMP/OTHERS (Probabilities: 0.8507, 0.0168, 0.1324)\n","Processed 225.1;Siomycin_A_(Siomycin_B)_(Siomycin_C)_(Siomycin_D): non-AMP/OTHERS (Probabilities: 0.9898, 0.0064, 0.0038)\n","Processed 226.1;Thiostrepton_B: non-AMP/OTHERS (Probabilities: 0.9824, 0.0151, 0.0025)\n","Processed 227.1;TP-1161: non-AMP/OTHERS (Probabilities: 0.8134, 0.0166, 0.1700)\n","Processed 228.1;A10255_B_(A10255_E)_(A10255_G): non-AMP/OTHERS (Probabilities: 0.9623, 0.0307, 0.0070)\n","Processed 229.1;A10255_J: non-AMP/OTHERS (Probabilities: 0.7549, 0.2349, 0.0102)\n","Processed 23.1;Enterocin_W_alfa: Neuropeptide (Probabilities: 0.0936, 0.0261, 0.8804)\n","Processed 230.1;Amythiamicin_A_(Amythiamicin_B)_(Amythiamicin_C): AMP (Probabilities: 0.2401, 0.6552, 0.1047)\n","Processed 231.1;Amythiamicin_D: Neuropeptide (Probabilities: 0.0090, 0.0122, 0.9788)\n","Processed 232.1;Berninamycin_A_(Berninamycin_B): AMP (Probabilities: 0.4081, 0.5780, 0.0139)\n","Processed 233.1;Berninamycin_C: AMP (Probabilities: 0.3508, 0.6342, 0.0149)\n","Processed 234.1;Berninamycin_D: non-AMP/OTHERS (Probabilities: 0.8096, 0.1777, 0.0127)\n","Processed 235.1;Geninthiocin_(Val-geninthiocin): AMP (Probabilities: 0.4081, 0.5780, 0.0139)\n","Processed 236.1;Glycothiohexide_alpha: AMP (Probabilities: 0.2847, 0.4900, 0.2253)\n","Processed 237.1;JBIR-83: AMP (Probabilities: 0.1703, 0.7910, 0.0387)\n","Processed 238.1;JBIR-84: non-AMP/OTHERS (Probabilities: 0.5500, 0.3249, 0.1250)\n","Processed 239.1;Methylsulfomycin: non-AMP/OTHERS (Probabilities: 0.6518, 0.3301, 0.0181)\n","Processed 24.1;Enterocin_W_beta: non-AMP/OTHERS (Probabilities: 0.9901, 0.0067, 0.0032)\n","Processed 240.1;Thiopeptin_A1a_(Thiopeptin_A1b)_(Thiopeptin_Ba)_(Thiopeptin_Bb): AMP (Probabilities: 0.1558, 0.8315, 0.0127)\n","Processed 241.1;Nocardithiocin: non-AMP/OTHERS (Probabilities: 0.8935, 0.0940, 0.0124)\n","Processed 242.1;MJ347-81F4_A_(MJ347-81F4_B): non-AMP/OTHERS (Probabilities: 0.4942, 0.4403, 0.0655)\n","Processed 243.1;O-methyl-glycothiohexide_alpha: AMP (Probabilities: 0.2847, 0.4900, 0.2253)\n","Processed 244.1;Promoinducin: non-AMP/OTHERS (Probabilities: 0.6706, 0.3101, 0.0193)\n","Processed 245.1;Promothiocin_A: Neuropeptide (Probabilities: 0.0168, 0.0131, 0.9701)\n","Processed 246.1;Promothiocin_B: Neuropeptide (Probabilities: 0.3246, 0.1756, 0.4998)\n","Processed 247.1;Radamycin: non-AMP/OTHERS (Probabilities: 0.6120, 0.3711, 0.0169)\n","Processed 248.1;Sch_40832: AMP (Probabilities: 0.1084, 0.8793, 0.0123)\n","Processed 249.1;Sulfomycin_I_(Sulfomycin_II)_(Sulfomycin_III): AMP (Probabilities: 0.3279, 0.6443, 0.0279)\n","Processed 25.1;Entianin: AMP (Probabilities: 0.0786, 0.9091, 0.0123)\n","Processed 250.1;Thioactin: non-AMP/OTHERS (Probabilities: 0.9860, 0.0094, 0.0046)\n","Processed 251.1;Thiopeptin_A3a_(Thiopeptin_A3b): AMP (Probabilities: 0.1096, 0.8778, 0.0126)\n","Processed 252.1;Thiopeptin_A4a_(Thiopeptin_A4b): AMP (Probabilities: 0.1254, 0.8611, 0.0135)\n","Processed 253.1;Thioxamycin: non-AMP/OTHERS (Probabilities: 0.8559, 0.1026, 0.0415)\n","Processed 254.1;QN3323_A_(QN3323_B)_(QN3323_Y): AMP (Probabilities: 0.0734, 0.9119, 0.0148)\n","Processed 255.1;QN3323_A-I: non-AMP/OTHERS (Probabilities: 0.4942, 0.4403, 0.0655)\n","Processed 256.1;Thiotipin: non-AMP/OTHERS (Probabilities: 0.6518, 0.3301, 0.0181)\n","Processed 257.1;68-1147: Neuropeptide (Probabilities: 0.0645, 0.0252, 0.9102)\n","Processed 258.1;Bicereucin_BsjA1: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0034)\n","Processed 259.1;Bicereucin_BsjA2: non-AMP/OTHERS (Probabilities: 0.9897, 0.0080, 0.0024)\n","Processed 26.1;Epicidin280: AMP (Probabilities: 0.0899, 0.8978, 0.0123)\n","Processed 260.1;Thusin_alpha: AMP (Probabilities: 0.0887, 0.8993, 0.0120)\n","Processed 261.1;Thusin_beta: non-AMP/OTHERS (Probabilities: 0.9917, 0.0055, 0.0028)\n","Processed 262.1;Thusin_beta2: non-AMP/OTHERS (Probabilities: 0.9920, 0.0050, 0.0030)\n","Processed 263.1;Gassericin E: AMP (Probabilities: 0.0691, 0.9174, 0.0135)\n","Processed 264.1;Cinnamycin_B: non-AMP/OTHERS (Probabilities: 0.9101, 0.0350, 0.0550)\n","Processed 265.1;Chaxapeptin: Neuropeptide (Probabilities: 0.0324, 0.0162, 0.9514)\n","Processed 266.1;Amylocyclicin: non-AMP/OTHERS (Probabilities: 0.5824, 0.4081, 0.0094)\n","Processed 267.1;Burhizin: Neuropeptide (Probabilities: 0.0158, 0.0121, 0.9720)\n","Processed 269.1;Corynazolicin: non-AMP/OTHERS (Probabilities: 0.8797, 0.1153, 0.0051)\n","Processed 27.1;Epidermin: non-AMP/OTHERS (Probabilities: 0.9918, 0.0053, 0.0029)\n","Processed 270.1;Cyclothiazomycin_B: Neuropeptide (Probabilities: 0.0199, 0.0127, 0.9674)\n","Processed 271.1;Cyclothiazomycin_C: Neuropeptide (Probabilities: 0.1858, 0.0279, 0.7863)\n","Processed 272.1;Enterocin_Nkr-5-3B: non-AMP/OTHERS (Probabilities: 0.5040, 0.4856, 0.0104)\n","Processed 273.1;Goadsporin: non-AMP/OTHERS (Probabilities: 0.9898, 0.0076, 0.0025)\n","Processed 274.1;Informatipeptin: non-AMP/OTHERS (Probabilities: 0.9912, 0.0031, 0.0057)\n","Processed 275.1;Lactazole: Neuropeptide (Probabilities: 0.1833, 0.0210, 0.7957)\n","Processed 276.1;Lactocillin: Neuropeptide (Probabilities: 0.0628, 0.0229, 0.9144)\n","Processed 277.1;Listeriolysin_S: non-AMP/OTHERS (Probabilities: 0.9726, 0.0247, 0.0027)\n","Processed 278.1;Marinostatin: Neuropeptide (Probabilities: 0.1229, 0.0230, 0.8542)\n","Processed 279.1;Microcin_C7: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed 28.1;Epilancin_15X: AMP (Probabilities: 0.0750, 0.9124, 0.0126)\n","Processed 280.1;Microcin_N: AMP (Probabilities: 0.0786, 0.9088, 0.0126)\n","Processed 282.1;Microviridin_K: non-AMP/OTHERS (Probabilities: 0.9345, 0.0088, 0.0567)\n","Processed 284.1;Patellin_2_(Patellin_3): non-AMP/OTHERS (Probabilities: 0.6495, 0.0229, 0.3276)\n","Processed 286.1;Penisin_I: AMP (Probabilities: 0.0876, 0.8987, 0.0137)\n","Processed 287.1;Penisin_II: AMP (Probabilities: 0.1501, 0.7747, 0.0752)\n","Processed 289.1;Pinensin: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed 29.1;EpilancinK7(ElkA): AMP (Probabilities: 0.0675, 0.9183, 0.0142)\n","Processed 290.1;Piricyclamide_7005E1: non-AMP/OTHERS (Probabilities: 0.9906, 0.0029, 0.0065)\n","Processed 291.1;Piricyclamide_7005E2: non-AMP/OTHERS (Probabilities: 0.9925, 0.0032, 0.0044)\n","Processed 292.1;Piricyclamide_7005E3: non-AMP/OTHERS (Probabilities: 0.9842, 0.0040, 0.0118)\n","Processed 293.1;Piricyclamide_7005E4: AMP (Probabilities: 0.0925, 0.8952, 0.0123)\n","Processed 294.1;Plantathiazolicin_(Plantazolicin): non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed 295.1;Streptide: non-AMP/OTHERS (Probabilities: 0.9930, 0.0033, 0.0037)\n","Processed 296.1;Streptomonomicin: non-AMP/OTHERS (Probabilities: 0.9933, 0.0038, 0.0029)\n","Processed 297.1;Subtilomycin: AMP (Probabilities: 0.0775, 0.9096, 0.0129)\n","Processed 298.1;Thiomuracins: Neuropeptide (Probabilities: 0.3444, 0.0719, 0.5838)\n","Processed 299.1;Thioviridamide: non-AMP/OTHERS (Probabilities: 0.9922, 0.0036, 0.0042)\n","Processed 3.1;Astexin: non-AMP/OTHERS (Probabilities: 0.5002, 0.0238, 0.4760)\n","Processed 30.1;Ericin_A: AMP (Probabilities: 0.1090, 0.8796, 0.0114)\n","Processed 300.1;Zucinodin: non-AMP/OTHERS (Probabilities: 0.9894, 0.0058, 0.0048)\n","Processed 301.1;Rhodanodin: non-AMP/OTHERS (Probabilities: 0.9706, 0.0049, 0.0245)\n","Processed 302.1;Syanodin: Neuropeptide (Probabilities: 0.0273, 0.0150, 0.9576)\n","Processed 303.1;Caulonodin_I: Neuropeptide (Probabilities: 0.0078, 0.0090, 0.9833)\n","Processed 304.1;Caulonodin_II: non-AMP/OTHERS (Probabilities: 0.9897, 0.0035, 0.0067)\n","Processed 305.1;Caulonodin_III: Neuropeptide (Probabilities: 0.0096, 0.0095, 0.9809)\n","Processed 306.1;Rubrivinodin: Neuropeptide (Probabilities: 0.1059, 0.0293, 0.8647)\n","Processed 307.1;Sphingonodin_I: non-AMP/OTHERS (Probabilities: 0.9909, 0.0034, 0.0057)\n","Processed 308.1;Sphingonodin_II: Neuropeptide (Probabilities: 0.0110, 0.0096, 0.9795)\n","Processed 309.1;Sphingopyxin_I: non-AMP/OTHERS (Probabilities: 0.9921, 0.0035, 0.0043)\n","Processed 31.1;Ericin_S: non-AMP/OTHERS (Probabilities: 0.9815, 0.0153, 0.0032)\n","Processed 310.1;Sphingopyxin_II: non-AMP/OTHERS (Probabilities: 0.9803, 0.0048, 0.0150)\n","Processed 311.1;Caulosegnin_I: non-AMP/OTHERS (Probabilities: 0.9719, 0.0091, 0.0190)\n","Processed 312.1;Caulosegnin_II: non-AMP/OTHERS (Probabilities: 0.9855, 0.0047, 0.0098)\n","Processed 313.1;Caulosegnin_III: Neuropeptide (Probabilities: 0.0604, 0.0355, 0.9041)\n","Processed 314.1;Auto_Inducing_Peptide_I: AMP (Probabilities: 0.1039, 0.8841, 0.0120)\n","Processed 315.1;Auto_Inducing_Peptide_II: non-AMP/OTHERS (Probabilities: 0.9926, 0.0036, 0.0037)\n","Processed 316.1;Auto_Inducing_Peptide_III: AMP (Probabilities: 0.4305, 0.5576, 0.0119)\n","Processed 317.1;Auto_Inducing_Peptide_IV: AMP (Probabilities: 0.2596, 0.7286, 0.0118)\n","Processed 318.1;ComX1: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0037)\n","Processed 319.1;ComX2: non-AMP/OTHERS (Probabilities: 0.9924, 0.0049, 0.0027)\n","Processed 32.1;Erythreapeptin: Neuropeptide (Probabilities: 0.4656, 0.0250, 0.5094)\n","Processed 320.1;ComX3: non-AMP/OTHERS (Probabilities: 0.9928, 0.0043, 0.0029)\n","Processed 321.1;ComX4: non-AMP/OTHERS (Probabilities: 0.9922, 0.0051, 0.0028)\n","Processed 322.1;Aborycin: AMP (Probabilities: 0.0708, 0.9158, 0.0134)\n","Processed 323.1;Anantin: non-AMP/OTHERS (Probabilities: 0.8383, 0.0636, 0.0981)\n","Processed 324.1;Astexin_1_19_(Astexin_1_23): non-AMP/OTHERS (Probabilities: 0.5002, 0.0238, 0.4760)\n","Processed 325.1;Astexin_2: non-AMP/OTHERS (Probabilities: 0.9902, 0.0068, 0.0030)\n","Processed 326.1;Astexin_3: Neuropeptide (Probabilities: 0.0101, 0.0096, 0.9803)\n","Processed 327.1;BI_32169: AMP (Probabilities: 0.3638, 0.4940, 0.1423)\n","Processed 328.1;Caulonodin_IV: non-AMP/OTHERS (Probabilities: 0.9909, 0.0042, 0.0049)\n","Processed 329.1;Caulonodin_V: Neuropeptide (Probabilities: 0.1906, 0.0220, 0.7873)\n","Processed 33.1;FrEUN1f_DRAFT_0188_putative_Linaridin: non-AMP/OTHERS (Probabilities: 0.5497, 0.0217, 0.4287)\n","Processed 330.1;Caulonodin_VI: non-AMP/OTHERS (Probabilities: 0.8689, 0.0188, 0.1123)\n","Processed 331.1;Caulonodin_VII: non-AMP/OTHERS (Probabilities: 0.9297, 0.0101, 0.0602)\n","Processed 332.1;Humidimycin_MDN_0010: AMP (Probabilities: 0.1131, 0.8280, 0.0589)\n","Processed 333.1;Lassomycin: non-AMP/OTHERS (Probabilities: 0.8383, 0.0636, 0.0981)\n","Processed 334.1;Paeninodin: Neuropeptide (Probabilities: 0.0127, 0.0112, 0.9761)\n","Processed 335.1;Propeptin: non-AMP/OTHERS (Probabilities: 0.8717, 0.0209, 0.1074)\n","Processed 336.1;Propeptin_2: non-AMP/OTHERS (Probabilities: 0.8717, 0.0209, 0.1074)\n","Processed 337.1;RES_701_1_(RES_701_2): Neuropeptide (Probabilities: 0.0078, 0.0090, 0.9832)\n","Processed 338.1;RES_701_3_9RES_701_4): non-AMP/OTHERS (Probabilities: 0.7975, 0.0132, 0.1894)\n","Processed 339.1;xanthomonin_I: non-AMP/OTHERS (Probabilities: 0.9857, 0.0042, 0.0101)\n","Processed 34.1;Gallidermin: non-AMP/OTHERS (Probabilities: 0.9789, 0.0117, 0.0094)\n","Processed 340.1;xanthomonin_II: Neuropeptide (Probabilities: 0.0349, 0.0166, 0.9485)\n","Processed 341.1;xanthomonin_III: non-AMP/OTHERS (Probabilities: 0.9170, 0.0087, 0.0743)\n","Processed 342.1;SSV-2083: non-AMP/OTHERS (Probabilities: 0.9875, 0.0042, 0.0084)\n","Processed 343.1;Sungsanpin: Neuropeptide (Probabilities: 0.0441, 0.0357, 0.9202)\n","Processed 344.1;SRO15-2005: non-AMP/OTHERS (Probabilities: 0.8832, 0.0949, 0.0219)\n","Processed 345.1;A_3802_IV_4_(A_3802_IV_7): non-AMP/OTHERS (Probabilities: 0.8982, 0.0911, 0.0107)\n","Processed 346.1;Carnocin_UI49: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed 347.1;Hominicin: AMP (Probabilities: 0.1333, 0.8541, 0.0126)\n","Processed 348.1;Aerucyclamide_A_McaE1_(Aerucyclamide_B_McaE1)_(Aerucyclamide_C_McaE1)_(Aerucyclamide_D_McaE1): Neuropeptide (Probabilities: 0.1354, 0.0264, 0.8382)\n","Processed 349.1;aeruginosamide_B_AgeE: Neuropeptide (Probabilities: 0.0725, 0.0213, 0.9062)\n","Processed 35.1;Gardimycin_(actagardine): non-AMP/OTHERS (Probabilities: 0.9871, 0.0076, 0.0053)\n","Processed 350.1;aeruginosamide_C_AgeE: Neuropeptide (Probabilities: 0.0725, 0.0213, 0.9062)\n","Processed 351.1;Aesturamide_A_LynE_(Aesturamide_B_LynE)_(Aesturamide_C_LynE)_(Aesturamide_D_LynE)_(Aesturamide_E_LynE)_(Aesturamide_F_LynE): Neuropeptide (Probabilities: 0.3919, 0.0281, 0.5800)\n","Processed 352.1;Aesturamide_G_LynE: Neuropeptide (Probabilities: 0.3919, 0.0281, 0.5800)\n","Processed 353.1;Aesturamide_H_LynE_(Aesturamide_I_LynE)_(Aesturamide_J_LynE)_(Aesturamide_K_LynE)_(Aesturamide_L_LynE): Neuropeptide (Probabilities: 0.3548, 0.0287, 0.6165)\n","Processed 354.1;Agardhipeptin_A: Neuropeptide (Probabilities: 0.0058, 0.0094, 0.9848)\n","Processed 355.1;Agardhipeptin_B: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed 356.1;Anacyclamide_A10_AcyE: non-AMP/OTHERS (Probabilities: 0.9908, 0.0037, 0.0055)\n","Processed 357.1;Anacyclamide_A11_AcyE: non-AMP/OTHERS (Probabilities: 0.9832, 0.0053, 0.0115)\n","Processed 358.1;Anacyclamide_A15_AcyE_(Anacyclamide_A15G_AcyE): non-AMP/OTHERS (Probabilities: 0.9678, 0.0063, 0.0259)\n","Processed 359.1;Anacyclamide_A20P_AcyE_(Anacyclamide_A20PP_AcyE): non-AMP/OTHERS (Probabilities: 0.9887, 0.0071, 0.0042)\n","Processed 36.1;geobacillin_I: non-AMP/OTHERS (Probabilities: 0.8143, 0.1597, 0.0259)\n","Processed 360.1;Anacyclamide_A7_AcyE: non-AMP/OTHERS (Probabilities: 0.9913, 0.0046, 0.0041)\n","Processed 361.1;Anacyclamide_A8P_AcyE: non-AMP/OTHERS (Probabilities: 0.9897, 0.0038, 0.0066)\n","Processed 362.1;Anacyclamide_A9P_AcyE: non-AMP/OTHERS (Probabilities: 0.9751, 0.0041, 0.0207)\n","Processed 363.1;Anacyclamide_B10_AcyE: non-AMP/OTHERS (Probabilities: 0.9920, 0.0034, 0.0046)\n","Processed 364.1;Anacyclamide_B7_AcyE: non-AMP/OTHERS (Probabilities: 0.9915, 0.0033, 0.0052)\n","Processed 365.1;Anacyclamide_B8_AcyE: non-AMP/OTHERS (Probabilities: 0.9880, 0.0041, 0.0079)\n","Processed 366.1;Anacyclamide_C10_AcyE: non-AMP/OTHERS (Probabilities: 0.9892, 0.0039, 0.0069)\n","Processed 367.1;Anacyclamide_C8_AcyE: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed 368.1;Anacyclamide_D10_AcyE: non-AMP/OTHERS (Probabilities: 0.9898, 0.0041, 0.0061)\n","Processed 369.1;Anacyclamide_E10_AcyE_(Anacyclamide_E10P_AcyE): non-AMP/OTHERS (Probabilities: 0.9896, 0.0037, 0.0067)\n","Processed 37.1;Geobacillin_I_like: non-AMP/OTHERS (Probabilities: 0.8310, 0.1301, 0.0389)\n","Processed 370.1;Anacyclamide_F10P_AcyE: non-AMP/OTHERS (Probabilities: 0.9927, 0.0032, 0.0041)\n","Processed 371.1;arthrospiramides_A_ArtE3: Neuropeptide (Probabilities: 0.1494, 0.0211, 0.8295)\n","Processed 372.1;arthrospiramides_B_ArtE5: Neuropeptide (Probabilities: 0.1920, 0.0209, 0.7871)\n","Processed 373.1;Ascidiacyclamide: AMP (Probabilities: 0.1170, 0.8701, 0.0129)\n","Processed 374.1;Axinastatin_1: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 375.1;Axinastatin_2: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed 376.1;Axinastatin_3: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed 377.1;Axinellin_C: Neuropeptide (Probabilities: 0.0060, 0.0090, 0.9850)\n","Processed 378.1;Banyascyclamide_A: AMP (Probabilities: 0.0660, 0.9194, 0.0145)\n","Processed 379.1;Banyascyclamide_B: AMP (Probabilities: 0.0661, 0.9193, 0.0146)\n","Processed 38.1;geobacillin_II: AMP (Probabilities: 0.1205, 0.8642, 0.0154)\n","Processed 380.1;Banyascyclamide_C: AMP (Probabilities: 0.0660, 0.9194, 0.0145)\n","Processed 381.1;Bistratamide_A_(Bistratamide_B): AMP (Probabilities: 0.0713, 0.9131, 0.0156)\n","Processed 382.1;Bistratamide_C: Neuropeptide (Probabilities: 0.0069, 0.0102, 0.9829)\n","Processed 383.1;Bistratamide_D_(Bistratamide_F)_(Bistratamide_G)_(Bistratamide_I): Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 384.1;Bistratamide_E_(Bistratamide_H)_(Bistratamide_J): AMP (Probabilities: 0.0712, 0.9148, 0.0141)\n","Processed 385.1;Comoramide_A: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9851)\n","Processed 386.1;Cyanothecamide_A_ThcE2: Neuropeptide (Probabilities: 0.0277, 0.0149, 0.9574)\n","Processed 387.1;Cyanothecamide_B_ThcE2: Neuropeptide (Probabilities: 0.0277, 0.0149, 0.9574)\n","Processed 388.1;Cyanothecamide_C_ThcE3: non-AMP/OTHERS (Probabilities: 0.9575, 0.0049, 0.0375)\n","Processed 389.1;Cyclodidemnamide: AMP (Probabilities: 0.0897, 0.8915, 0.0188)\n","Processed 39.1;Glycocin_F: AMP (Probabilities: 0.0658, 0.9190, 0.0152)\n","Processed 390.1;Cyclodidemnamide_B: Neuropeptide (Probabilities: 0.0063, 0.0087, 0.9850)\n","Processed 391.1;Cyclonellin: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed 392.1;Cycloxazoline: AMP (Probabilities: 0.0971, 0.8899, 0.0130)\n","Processed 393.1;Dendroamide_A: AMP (Probabilities: 0.0840, 0.8916, 0.0243)\n","Processed 394.1;Dendroamide_B: non-AMP/OTHERS (Probabilities: 0.7549, 0.0574, 0.1877)\n","Processed 395.1;Dendroamide_C: non-AMP/OTHERS (Probabilities: 0.7549, 0.0574, 0.1877)\n","Processed 396.1;Didmolamide_A_(Didmolamide_B): AMP (Probabilities: 0.0660, 0.9194, 0.0145)\n","Processed 397.1;Dolastatin_3: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed 398.1;Dolastatin_E: Neuropeptide (Probabilities: 0.0063, 0.0096, 0.9842)\n","Processed 399.1;Dolastatin_I: Neuropeptide (Probabilities: 0.2364, 0.1504, 0.6133)\n","Processed 4.1;Avermipeptin: non-AMP/OTHERS (Probabilities: 0.9499, 0.0075, 0.0426)\n","Processed 40.1;Grisemycin: non-AMP/OTHERS (Probabilities: 0.9917, 0.0046, 0.0036)\n","Processed 400.1;Haliclonamide_A_(Haliclonamide_B)_(Haliclonamide_D)_(Haliclonamide_E): Neuropeptide (Probabilities: 0.0061, 0.0089, 0.9850)\n","Processed 401.1;Hexamollamide: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed 402.1;Homodolastatin_3: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed 403.1;Hymenamide_A: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 404.1;Hymenamide_B: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed 405.1;Hymenamide_C: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed 406.1;Hymenamide_D: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed 407.1;Hymenamide_E: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 408.1;Hymenamide_F: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed 409.1;Hymenamide_G: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed 41.1;Griseopeptin: non-AMP/OTHERS (Probabilities: 0.9923, 0.0029, 0.0048)\n","Processed 410.1;Hymenamide_H: Neuropeptide (Probabilities: 0.0063, 0.0087, 0.9850)\n","Processed 411.1;Hymenamide_J: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed 412.1;Hymenamide_K: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed 413.1;Kawaguchipeptin_A: non-AMP/OTHERS (Probabilities: 0.5847, 0.0343, 0.3810)\n","Processed 414.1;Kawaguchipeptin_B: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed 415.1;Kororamide: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed 416.1;Lissoclinamide_1: Neuropeptide (Probabilities: 0.1396, 0.3799, 0.4805)\n","Processed 417.1;Lissoclinamide_10: AMP (Probabilities: 0.0773, 0.9098, 0.0129)\n","Processed 418.1;Lissoclinamide_4_(Lissoclinamide_5)_(Lissoclinamide_7)_(Lissoclinamide_8): Neuropeptide (Probabilities: 0.0069, 0.0107, 0.9824)\n","Processed 419.1;Lissoclinamide_9: Neuropeptide (Probabilities: 0.0796, 0.2249, 0.6955)\n","Processed 42.1;Haloduracin_alpha: non-AMP/OTHERS (Probabilities: 0.5536, 0.0457, 0.4007)\n","Processed 420.1;lissoclinamide2/3_patE3: non-AMP/OTHERS (Probabilities: 0.8612, 0.0273, 0.1115)\n","Processed 421.1;lissoclinamide4/5_patE3: non-AMP/OTHERS (Probabilities: 0.8612, 0.0273, 0.1115)\n","Processed 422.1;Mayotamide_A: AMP (Probabilities: 0.0800, 0.9074, 0.0127)\n","Processed 423.1;Mayotamide_B: AMP (Probabilities: 0.0748, 0.9117, 0.0135)\n","Processed 424.1;microcyclamide_McaE2: Neuropeptide (Probabilities: 0.0311, 0.0178, 0.9511)\n","Processed 425.1;Microphycin_AL828: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed 426.1;Mollamide: Neuropeptide (Probabilities: 0.0060, 0.0090, 0.9850)\n","Processed 427.1;Mollamide_B: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed 428.1;Mollamide_C: Neuropeptide (Probabilities: 0.0058, 0.0094, 0.9848)\n","Processed 429.1;Muscoride_A: Neuropeptide (Probabilities: 0.0058, 0.0094, 0.9848)\n","Processed 43.1;Haloduracin_beta: non-AMP/OTHERS (Probabilities: 0.9906, 0.0063, 0.0031)\n","Processed 430.1;Nairaiamide_A: AMP (Probabilities: 0.0915, 0.8964, 0.0121)\n","Processed 431.1;Nairaiamide_B: AMP (Probabilities: 0.0800, 0.9078, 0.0122)\n","Processed 432.1;Nostocyclamide: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed 433.1;Nostocyclamide_M: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9852)\n","Processed 434.1;Oscillacyclin: Neuropeptide (Probabilities: 0.0074, 0.0090, 0.9836)\n","Processed 435.1;patellamide_A_patE1: non-AMP/OTHERS (Probabilities: 0.7664, 0.0341, 0.1995)\n","Processed 436.1;patellamide_A_patE4: Neuropeptide (Probabilities: 0.4687, 0.0360, 0.4953)\n","Processed 437.1;patellamide_B_patE4: Neuropeptide (Probabilities: 0.4687, 0.0360, 0.4953)\n","Processed 438.1;patellamide_B_patE5: Neuropeptide (Probabilities: 0.3022, 0.0297, 0.6681)\n","Processed 439.1;patellamide_C_patE1: non-AMP/OTHERS (Probabilities: 0.7664, 0.0341, 0.1995)\n","Processed 44.1;Htur_3018_putative_Linaridin: non-AMP/OTHERS (Probabilities: 0.9920, 0.0041, 0.0040)\n","Processed 440.1;patellamide_C_patE2: non-AMP/OTHERS (Probabilities: 0.7910, 0.0428, 0.1662)\n","Processed 441.1;Patellamide_D: AMP (Probabilities: 0.0718, 0.9136, 0.0146)\n","Processed 442.1;Patellamide_E: AMP (Probabilities: 0.1306, 0.8561, 0.0134)\n","Processed 443.1;Patellamide_F: AMP (Probabilities: 0.0657, 0.9199, 0.0144)\n","Processed 444.1;Patellamide_G: AMP (Probabilities: 0.0803, 0.9066, 0.0131)\n","Processed 445.1;Patellin_1: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed 446.1;patellin_2_TruE1: non-AMP/OTHERS (Probabilities: 0.6495, 0.0229, 0.3276)\n","Processed 447.1;patellin_3_TruE1: non-AMP/OTHERS (Probabilities: 0.6495, 0.0229, 0.3276)\n","Processed 448.1;Patellin_4: Neuropeptide (Probabilities: 0.0116, 0.0108, 0.9776)\n","Processed 449.1;Patellin_5: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 45.1;Labyrinthopeptin_A1/A3: Neuropeptide (Probabilities: 0.4407, 0.0257, 0.5336)\n","Processed 450.1;patellin_6_TruE2: non-AMP/OTHERS (Probabilities: 0.8955, 0.0127, 0.0917)\n","Processed 451.1;Phakellistatin_1: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed 452.1;Phakellistatin_12: AMP (Probabilities: 0.2249, 0.6741, 0.1011)\n","Processed 453.1;Phakellistatin_13: Neuropeptide (Probabilities: 0.0059, 0.0092, 0.9850)\n","Processed 454.1;Phakellistatin_2: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed 455.1;Phakellistatin_7: Neuropeptide (Probabilities: 0.1560, 0.2800, 0.5640)\n","Processed 456.1;Phakellistatin_8: AMP (Probabilities: 0.2550, 0.6167, 0.1283)\n","Processed 457.1;Phakellistatin_9: AMP (Probabilities: 0.1374, 0.8291, 0.0335)\n","Processed 458.1;Planktocyclin: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed 459.1;Prelissoclinamide_2: AMP (Probabilities: 0.0687, 0.9164, 0.0149)\n","Processed 46.1;Labyrinthopeptin_A2: Neuropeptide (Probabilities: 0.0445, 0.0192, 0.9363)\n","Processed 460.1;Prenylagaramide_A: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed 461.1;prenylagaramide_B_PagE6: non-AMP/OTHERS (Probabilities: 0.9923, 0.0036, 0.0041)\n","Processed 462.1;prenylagaramide_C_PagE7: non-AMP/OTHERS (Probabilities: 0.9927, 0.0038, 0.0035)\n","Processed 463.1;Prepatellamide_A: AMP (Probabilities: 0.0681, 0.9174, 0.0145)\n","Processed 464.1;Prepatellamide_B_formate: AMP (Probabilities: 0.0684, 0.9166, 0.0149)\n","Processed 465.1;Preulicyclamide: AMP (Probabilities: 0.0687, 0.9164, 0.0149)\n","Processed 466.1;Preulithiacyclamide: AMP (Probabilities: 0.0833, 0.9031, 0.0136)\n","Processed 467.1;Raocyclamide_A_(Raocyclamide_B): Neuropeptide (Probabilities: 0.0063, 0.0098, 0.9839)\n","Processed 468.1;Stylisin_1: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 469.1;Stylisin_2: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 47.1;Lacticin_3147_A1: Neuropeptide (Probabilities: 0.0296, 0.0182, 0.9522)\n","Processed 470.1;Stylissamide_A: Neuropeptide (Probabilities: 0.0062, 0.0089, 0.9849)\n","Processed 471.1;Stylissamide_B: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed 472.1;Stylissamide_C: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9850)\n","Processed 473.1;Stylissamide_D: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 474.1;Tawicyclamide_A_(Tawicyclamide_B): AMP (Probabilities: 0.0684, 0.9165, 0.0151)\n","Processed 475.1;tenuecyclamide_A_tenE: non-AMP/OTHERS (Probabilities: 0.9121, 0.0679, 0.0201)\n","Processed 476.1;Tenuecyclamide_C: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9852)\n","Processed 477.1;tenuecyclamide_C_tenE: non-AMP/OTHERS (Probabilities: 0.9121, 0.0679, 0.0201)\n","Processed 478.1;Tenuecyclamide_D: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9852)\n","Processed 479.1;ThcE4: Neuropeptide (Probabilities: 0.0227, 0.0136, 0.9637)\n","Processed 48.1;Lacticin_3147_A2: non-AMP/OTHERS (Probabilities: 0.9925, 0.0046, 0.0029)\n","Processed 480.1;trichamide_TriG: non-AMP/OTHERS (Probabilities: 0.7228, 0.0156, 0.2617)\n","Processed 481.1;trunkamide_TruE2: non-AMP/OTHERS (Probabilities: 0.8955, 0.0127, 0.0917)\n","Processed 482.1;trunkamide_TruE3: Neuropeptide (Probabilities: 0.2497, 0.0261, 0.7242)\n","Processed 483.1;Ulithiacyclamide_B_(Ulithiacyclamide_E)_(Ulithiacyclamide_F)_(Ulithiacyclamide_G): AMP (Probabilities: 0.0833, 0.9031, 0.0136)\n","Processed 484.1;ulithiacyclamide_patE2: non-AMP/OTHERS (Probabilities: 0.7910, 0.0428, 0.1662)\n","Processed 485.1;ulithiacyclamide_patE5: Neuropeptide (Probabilities: 0.3022, 0.0297, 0.6681)\n","Processed 486.1;ulithiacyclamide_patE6: non-AMP/OTHERS (Probabilities: 0.8772, 0.0380, 0.0848)\n","Processed 487.1;Ulithiacyclamide: AMP (Probabilities: 0.0687, 0.9164, 0.0149)\n","Processed 488.1;viridisamide_A_AgeE: non-AMP/OTHERS (Probabilities: 0.6823, 0.2948, 0.0229)\n","Processed 489.1;PoyA: AMP (Probabilities: 0.3094, 0.6773, 0.0133)\n","Processed 49.1;Lacticin_481_(LactococcinDR): non-AMP/OTHERS (Probabilities: 0.9819, 0.0151, 0.0030)\n","Processed 490.1;Pumilarin: non-AMP/OTHERS (Probabilities: 0.8669, 0.1267, 0.0064)\n","Processed 491.1;Lantibiotic nukacin: Neuropeptide (Probabilities: 0.1436, 0.0233, 0.8331)\n","Processed 492.1;Competence pheromone: non-AMP/OTHERS (Probabilities: 0.9933, 0.0035, 0.0032)\n","Processed 493.1;Mejucin: non-AMP/OTHERS (Probabilities: 0.9897, 0.0069, 0.0034)\n","Processed 5.1;BhtA1: AMP (Probabilities: 0.0799, 0.9080, 0.0121)\n","Processed 50.1;Lactocin_S: non-AMP/OTHERS (Probabilities: 0.9878, 0.0056, 0.0066)\n","Processed 51.1;Lantibiotic107891: AMP (Probabilities: 0.2794, 0.7066, 0.0140)\n","Processed 52.1;Lariatin_A: non-AMP/OTHERS (Probabilities: 0.9892, 0.0076, 0.0032)\n","Processed 53.1;Lariatin_B: non-AMP/OTHERS (Probabilities: 0.9888, 0.0083, 0.0029)\n","Processed 54.1;LichenicidinVK21A1_(Lichenicidin_A1): non-AMP/OTHERS (Probabilities: 0.9256, 0.0101, 0.0642)\n","Processed 55.1;LichenicidinVK21A2_(Lichenicidin_A2): AMP (Probabilities: 0.0872, 0.9008, 0.0120)\n","Processed 56.1;Lxx19440_putative_Linaridin: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed 57.1;Lxx19470_putative_Linaridin: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0033)\n","Processed 58.1;Macedocin: non-AMP/OTHERS (Probabilities: 0.9894, 0.0047, 0.0058)\n","Processed 59.1;Macedovicin: non-AMP/OTHERS (Probabilities: 0.9901, 0.0041, 0.0058)\n","Processed 6.1;BhtA2: non-AMP/OTHERS (Probabilities: 0.9810, 0.0163, 0.0027)\n","Processed 60.1;MccI47_mchS2: AMP (Probabilities: 0.0710, 0.9159, 0.0130)\n","Processed 61.1;McdA1: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0034)\n","Processed 62.1;Mersacidin: Neuropeptide (Probabilities: 0.0206, 0.0141, 0.9652)\n","Processed 63.1;Michiganin-A: non-AMP/OTHERS (Probabilities: 0.5413, 0.2778, 0.1809)\n","Processed 64.1;Microbisporicin_(NAI-107)_(Microbisporicin_A1)_(Microbisporicin_A2): Neuropeptide (Probabilities: 0.0589, 0.0203, 0.9208)\n","Processed 65.1;Microcin_B17_(MccB17): AMP (Probabilities: 0.4763, 0.5031, 0.0205)\n","Processed 66.1;Microcin_E492: AMP (Probabilities: 0.0673, 0.9188, 0.0139)\n","Processed 67.1;Microcin_H47_(MccH47): AMP (Probabilities: 0.1600, 0.8281, 0.0118)\n","Processed 68.1;Microcin_L: AMP (Probabilities: 0.1738, 0.7976, 0.0286)\n","Processed 69.1;Microcin_M: AMP (Probabilities: 0.0908, 0.8967, 0.0125)\n","Processed 7.1;BLD_1648: non-AMP/OTHERS (Probabilities: 0.9922, 0.0040, 0.0038)\n","Processed 70.1;Microcin_V_(Colicin_V): AMP (Probabilities: 0.0665, 0.9201, 0.0134)\n","Processed 71.1;Microcyclamide: Neuropeptide (Probabilities: 0.1715, 0.0274, 0.8011)\n","Processed 72.1;Mutacin1140_(Mutacin_III): non-AMP/OTHERS (Probabilities: 0.8189, 0.0247, 0.1564)\n","Processed 73.1;MutacinB-Ny266: AMP (Probabilities: 0.0658, 0.9188, 0.0154)\n","Processed 74.1;Mutacin_II(MutacinH-29B): AMP (Probabilities: 0.0736, 0.9138, 0.0125)\n","Processed 75.1;Mvan_2782_putative_Linaridin: non-AMP/OTHERS (Probabilities: 0.9908, 0.0062, 0.0029)\n","Processed 76.1;NAI-802: AMP (Probabilities: 0.0725, 0.9135, 0.0140)\n","Processed 77.1;Nisin_A: AMP (Probabilities: 0.0669, 0.9183, 0.0147)\n","Processed 78.1;Nisin_F: AMP (Probabilities: 0.0664, 0.9189, 0.0147)\n","Processed 79.1;Nisin_Q: AMP (Probabilities: 0.0662, 0.9186, 0.0152)\n","Processed 8.1;Bottromycin_A2: Neuropeptide (Probabilities: 0.0258, 0.0143, 0.9600)\n","Processed 80.1;Nisin_U: non-AMP/OTHERS (Probabilities: 0.7318, 0.2325, 0.0357)\n","Processed 81.1;Nisin_Z: AMP (Probabilities: 0.0697, 0.9163, 0.0140)\n","Processed 82.1;Nmul_A1818_putative_Linaridin: Neuropeptide (Probabilities: 0.1158, 0.0244, 0.8598)\n","Processed 83.1;Nosiheptide: Neuropeptide (Probabilities: 0.0119, 0.0107, 0.9773)\n","Processed 84.1;Nukacin_A_(NukacinISK-1): non-AMP/OTHERS (Probabilities: 0.4882, 0.0307, 0.4810)\n","Processed 85.1;Paenibacillin: AMP (Probabilities: 0.0702, 0.9142, 0.0155)\n","Processed 86.1;Patellamide: non-AMP/OTHERS (Probabilities: 0.7918, 0.0384, 0.1698)\n","Processed 87.1;Patellamide_B: Neuropeptide (Probabilities: 0.3022, 0.0297, 0.6681)\n","Processed 88.1;Pep5: AMP (Probabilities: 0.0685, 0.9177, 0.0138)\n","Processed 89.1;Plantaricin_W_alfa: AMP (Probabilities: 0.0699, 0.9167, 0.0133)\n","Processed 9.1;Bottromycin_D: Neuropeptide (Probabilities: 0.0141, 0.0118, 0.9741)\n","Processed 90.1;Plantaricin_W_beta: AMP (Probabilities: 0.0664, 0.9191, 0.0146)\n","Processed 91.1;Plantazolicin: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed 92.1;Prenylagaramide_B: non-AMP/OTHERS (Probabilities: 0.9923, 0.0036, 0.0041)\n","Processed 93.1;Prenylagaramide_C: non-AMP/OTHERS (Probabilities: 0.9927, 0.0038, 0.0035)\n","Processed 94.1;putative_lantibiotic: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0031)\n","Processed 95.1;putative_lantibiotic: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0040)\n","Processed 96.1;putative_lantibiotic: Neuropeptide (Probabilities: 0.0201, 0.0129, 0.9670)\n","Processed 97.1;putative_lantibiotic_precursor: non-AMP/OTHERS (Probabilities: 0.9919, 0.0043, 0.0038)\n","Processed 98.1;Rp71,955: AMP (Probabilities: 0.0708, 0.9158, 0.0134)\n","Processed 99.1;Ruminococcin_A: non-AMP/OTHERS (Probabilities: 0.9928, 0.0030, 0.0042)\n","Processed 489.1;Pinensin_01: Neuropeptide (Probabilities: 0.3175, 0.0340, 0.6485)\n","Processed 490.1;Pinensin_02: Neuropeptide (Probabilities: 0.0600, 0.0211, 0.9189)\n","Processed 491.1;Pinensin_03: Neuropeptide (Probabilities: 0.3275, 0.0355, 0.6370)\n","Processed 492.1;Pinensin_04: non-AMP/OTHERS (Probabilities: 0.9716, 0.0063, 0.0221)\n","Processed 493.1;Pinensin_PedA17: non-AMP/OTHERS (Probabilities: 0.8744, 0.0185, 0.1071)\n","Processed 494.1;Pinensin_06: Neuropeptide (Probabilities: 0.0911, 0.0218, 0.8872)\n","Processed 495.1;Pinensin_07: Neuropeptide (Probabilities: 0.2258, 0.0243, 0.7499)\n","Processed 496.1;Pinensin_PinA: non-AMP/OTHERS (Probabilities: 0.9857, 0.0093, 0.0051)\n","Processed 497.1;Pinensin_09: Neuropeptide (Probabilities: 0.1016, 0.0208, 0.8777)\n","Processed 498.1;Pinensin_10: non-AMP/OTHERS (Probabilities: 0.9097, 0.0325, 0.0578)\n","Processed 499.1;Pinensin_11: non-AMP/OTHERS (Probabilities: 0.8810, 0.0697, 0.0493)\n","Processed 1.2;Abp118_alpha: AMP (Probabilities: 0.0679, 0.9175, 0.0145)\n","Processed 10.2;AFP-1: non-AMP/OTHERS (Probabilities: 0.6568, 0.3300, 0.0131)\n","Processed 100.2;EnterocinP: AMP (Probabilities: 0.0802, 0.9077, 0.0121)\n","Processed 101.2;EP-20: Neuropeptide (Probabilities: 0.0101, 0.0097, 0.9801)\n","Processed 102.2;Fulvocin-C: AMP (Probabilities: 0.0688, 0.9178, 0.0134)\n","Processed 103.2;garvieacinQ_garQ: non-AMP/OTHERS (Probabilities: 0.9717, 0.0229, 0.0054)\n","Processed 104.2;Gassericin_T: AMP (Probabilities: 0.0661, 0.9197, 0.0142)\n","Processed 105.2;GP-19: AMP (Probabilities: 0.4505, 0.5400, 0.0095)\n","Processed 106.2;Halocin-C8: AMP (Probabilities: 0.0625, 0.9208, 0.0167)\n","Processed 107.2;Hiracin_JM79: AMP (Probabilities: 0.0664, 0.9186, 0.0150)\n","Processed 108.2;hominicin: AMP (Probabilities: 0.1260, 0.8616, 0.0124)\n","Processed 109.2;Enterocin_96: AMP (Probabilities: 0.2564, 0.7300, 0.0136)\n","Processed 11.2;AFP-1: AMP (Probabilities: 0.2468, 0.7418, 0.0114)\n","Processed 110.2;Ipomicin: AMP (Probabilities: 0.1312, 0.8554, 0.0134)\n","Processed 111.2;Lactacin_F_(fragment): AMP (Probabilities: 0.1131, 0.8729, 0.0139)\n","Processed 112.2;Lactacin_F,subunit_lafA: AMP (Probabilities: 0.0663, 0.9196, 0.0141)\n","Processed 113.2;Lactacin_F,subunit_lafX: AMP (Probabilities: 0.0842, 0.9030, 0.0128)\n","Processed 114.2;Lacticin_Q: AMP (Probabilities: 0.0685, 0.9167, 0.0148)\n","Processed 115.2;Lacticin_Z: AMP (Probabilities: 0.0686, 0.9167, 0.0147)\n","Processed 116.2;Lactocin_705_alpha: AMP (Probabilities: 0.0791, 0.9089, 0.0121)\n","Processed 117.2;Lactocin_705_beta: AMP (Probabilities: 0.1325, 0.8557, 0.0118)\n","Processed 118.2;Lactococcin_972: AMP (Probabilities: 0.0676, 0.9187, 0.0136)\n","Processed 119.2;Lactococcin_972_(Lcn972): AMP (Probabilities: 0.0919, 0.8955, 0.0126)\n","Processed 12.2;Amylovorin: AMP (Probabilities: 0.0685, 0.9179, 0.0136)\n","Processed 120.2;lactococcin_A: AMP (Probabilities: 0.1056, 0.8798, 0.0146)\n","Processed 121.2;Lactococcin_A_(LCN-A): AMP (Probabilities: 0.2400, 0.7466, 0.0133)\n","Processed 122.2;Lactococcin_B_(LCN-B): non-AMP/OTHERS (Probabilities: 0.8641, 0.1267, 0.0092)\n","Processed 123.2;Lactococcin_G_peptide_alpha_2: AMP (Probabilities: 0.0704, 0.9154, 0.0142)\n","Processed 124.2;lactococcin_G,_alphasubunit: AMP (Probabilities: 0.0705, 0.9152, 0.0143)\n","Processed 125.2;lactococcin_G,_betasubunit: AMP (Probabilities: 0.0665, 0.9173, 0.0162)\n","Processed 126.2;Lactococcin_MMFII: AMP (Probabilities: 0.1104, 0.8722, 0.0175)\n","Processed 127.2;Lactococcin_MMFII: AMP (Probabilities: 0.1104, 0.8722, 0.0175)\n","Processed 128.2;lactococcin_Q_chain_a: AMP (Probabilities: 0.0818, 0.9050, 0.0132)\n","Processed 129.2;lactococcin_Q_chain_b: AMP (Probabilities: 0.0712, 0.9158, 0.0130)\n","Processed 13.2;Aureocin_A53: AMP (Probabilities: 0.0715, 0.9139, 0.0146)\n","Processed 131.2;laterosporulin: AMP (Probabilities: 0.0722, 0.9145, 0.0133)\n","Processed 132.2;LCI: AMP (Probabilities: 0.0689, 0.9173, 0.0139)\n","Processed 133.2;Leucocin_A_(LeucocinA-UAL187): AMP (Probabilities: 0.0677, 0.9184, 0.0139)\n","Processed 134.2;Leucocin_B: AMP (Probabilities: 0.0668, 0.9182, 0.0150)\n","Processed 135.2;Leucocin_B_(LeucocinB-Ta11a): AMP (Probabilities: 0.0675, 0.9183, 0.0142)\n","Processed 136.2;Leucocin_C: AMP (Probabilities: 0.0709, 0.9148, 0.0142)\n","Processed 137.2;Leucocin_K: non-AMP/OTHERS (Probabilities: 0.9803, 0.0166, 0.0031)\n","Processed 139.2;Lichenin: Neuropeptide (Probabilities: 0.0221, 0.0160, 0.9619)\n","Processed 14.2;Bac32: AMP (Probabilities: 0.0934, 0.8942, 0.0124)\n","Processed 140.2;lichenin: Neuropeptide (Probabilities: 0.0221, 0.0160, 0.9619)\n","Processed 141.2;listeriocin_743A: AMP (Probabilities: 0.0796, 0.9075, 0.0129)\n","Processed 142.2;LSEI_2163: non-AMP/OTHERS (Probabilities: 0.9932, 0.0040, 0.0028)\n","Processed 143.2;LSEI_2386: Neuropeptide (Probabilities: 0.1113, 0.0465, 0.8422)\n","Processed 144.2;Mesentericin: AMP (Probabilities: 0.0679, 0.9175, 0.0146)\n","Processed 145.2;Mesentericin_B105: non-AMP/OTHERS (Probabilities: 0.9773, 0.0139, 0.0088)\n","Processed 146.2;MesentericinY105(anti-Listeria): AMP (Probabilities: 0.0708, 0.9156, 0.0136)\n","Processed 147.2;Microcin_24: non-AMP/OTHERS (Probabilities: 0.9913, 0.0047, 0.0040)\n","Processed 148.2;Microcin_C7: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed 149.2;Microcin_I47: AMP (Probabilities: 0.0745, 0.9126, 0.0129)\n","Processed 15.2;Bac43: AMP (Probabilities: 0.0664, 0.9186, 0.0150)\n","Processed 150.2;Microcin_PDI: AMP (Probabilities: 0.0701, 0.9164, 0.0135)\n","Processed 151.2;Microcin_S: AMP (Probabilities: 0.0669, 0.9184, 0.0147)\n","Processed 152.2;MR10A: AMP (Probabilities: 0.0699, 0.9150, 0.0152)\n","Processed 153.2;MR10B: AMP (Probabilities: 0.0680, 0.9166, 0.0154)\n","Processed 154.2;Mundticin: AMP (Probabilities: 0.0683, 0.9159, 0.0157)\n","Processed 155.2;Mundticin_ATO6: AMP (Probabilities: 0.0674, 0.9167, 0.0160)\n","Processed 156.2;Mundticin_KS: AMP (Probabilities: 0.0658, 0.9183, 0.0160)\n","Processed 157.2;Mutacin_IV: non-AMP/OTHERS (Probabilities: 0.9862, 0.0111, 0.0027)\n","Processed 158.2;Pediocin: AMP (Probabilities: 0.0649, 0.9192, 0.0159)\n","Processed 159.2;Pediocin: non-AMP/OTHERS (Probabilities: 0.9856, 0.0105, 0.0039)\n","Processed 16.2;Bacteriocin_31: AMP (Probabilities: 0.0728, 0.9135, 0.0136)\n","Processed 160.2;Pediocin_L50_{C-terminal}: AMP (Probabilities: 0.0668, 0.9175, 0.0157)\n","Processed 161.2;Pediocin_PA: AMP (Probabilities: 0.0653, 0.9187, 0.0160)\n","Processed 162.2;Pediocin_PA-1: AMP (Probabilities: 0.0654, 0.9180, 0.0166)\n","Processed 163.2;Penocin_A: AMP (Probabilities: 0.0655, 0.9192, 0.0153)\n","Processed 164.2;Piscicolin_126: AMP (Probabilities: 0.0662, 0.9181, 0.0157)\n","Processed 165.2;Plantaricin_125_beta_(plnB): AMP (Probabilities: 0.0663, 0.9191, 0.0147)\n","Processed 166.2;Plantaricin_423: AMP (Probabilities: 0.0659, 0.9183, 0.0158)\n","Processed 167.2;Plantaricin_A: AMP (Probabilities: 0.0899, 0.8985, 0.0116)\n","Processed 168.2;Plantaricin_ASM1: AMP (Probabilities: 0.0658, 0.9190, 0.0152)\n","Processed 169.2;Plantaricin_C19: AMP (Probabilities: 0.0674, 0.9170, 0.0156)\n","Processed 17.2;bacteriocin_J46: non-AMP/OTHERS (Probabilities: 0.9907, 0.0043, 0.0051)\n","Processed 170.2;Plantaricin_E: AMP (Probabilities: 0.0702, 0.9155, 0.0143)\n","Processed 171.2;Plantaricin_F: non-AMP/OTHERS (Probabilities: 0.9920, 0.0053, 0.0027)\n","Processed 172.2;Plantaricin_J: AMP (Probabilities: 0.1094, 0.8781, 0.0125)\n","Processed 173.2;Plantaricin_K: non-AMP/OTHERS (Probabilities: 0.7488, 0.2438, 0.0074)\n","Processed 174.2;Plantaricin_N: AMP (Probabilities: 0.0697, 0.9158, 0.0145)\n","Processed 175.2;Plantaricin_NC8-alpha: non-AMP/OTHERS (Probabilities: 0.9082, 0.0750, 0.0168)\n","Processed 176.2;Plantaricin_NC8-beta: non-AMP/OTHERS (Probabilities: 0.9823, 0.0146, 0.0031)\n","Processed 177.2;Plantaricin_S_alpha_chain_{N-terminal}: non-AMP/OTHERS (Probabilities: 0.9047, 0.0663, 0.0290)\n","Processed 178.2;Plantaricin_S_alpha_chain_N-terminal: non-AMP/OTHERS (Probabilities: 0.9047, 0.0663, 0.0290)\n","Processed 179.2;Plantaricin_S_beta_chain_{N-terminal}: AMP (Probabilities: 0.0734, 0.9133, 0.0133)\n","Processed 18.2;bacteriocin_like_peptide_associated: AMP (Probabilities: 0.0648, 0.9208, 0.0144)\n","Processed 180.2;Plantaricin_S-alpha: non-AMP/OTHERS (Probabilities: 0.9851, 0.0050, 0.0099)\n","Processed 181.2;Plantaricin_S-beta: AMP (Probabilities: 0.0972, 0.8910, 0.0118)\n","Processed 182.2;Plantaricin_SA6: AMP (Probabilities: 0.3913, 0.5651, 0.0436)\n","Processed 183.2;Plantaricin_W: AMP (Probabilities: 0.0664, 0.9191, 0.0146)\n","Processed 184.2;Plantaricin_W: AMP (Probabilities: 0.0699, 0.9167, 0.0133)\n","Processed 185.2;PlnK_(putative): non-AMP/OTHERS (Probabilities: 0.9401, 0.0553, 0.0046)\n","Processed 186.2;Propionicin_SM1: AMP (Probabilities: 0.1153, 0.8728, 0.0119)\n","Processed 187.2;Propionicin_T1: AMP (Probabilities: 0.0859, 0.9017, 0.0124)\n","Processed 188.2;Propionicin-F: non-AMP/OTHERS (Probabilities: 0.9821, 0.0148, 0.0031)\n","Processed 189.2;protease-activatedantimicrobialprotein(PAMP): AMP (Probabilities: 0.1283, 0.8597, 0.0120)\n","Processed 19.2;bacteriocin_likepeptide_associated: non-AMP/OTHERS (Probabilities: 0.8339, 0.1601, 0.0060)\n","Processed 190.2;putative_bacteriocin: non-AMP/OTHERS (Probabilities: 0.9908, 0.0061, 0.0031)\n","Processed 191.2;putative_bacteriocin: AMP (Probabilities: 0.0662, 0.9191, 0.0147)\n","Processed 192.2;putative_bacteriocin: AMP (Probabilities: 0.1462, 0.8424, 0.0114)\n","Processed 193.2;putative_bacteriocin: AMP (Probabilities: 0.1039, 0.8836, 0.0125)\n","Processed 194.2;putative_bacteriocin: AMP (Probabilities: 0.1162, 0.8712, 0.0127)\n","Processed 195.2;putative_bacteriocin: AMP (Probabilities: 0.0664, 0.9191, 0.0145)\n","Processed 196.2;putative_bacteriocin: AMP (Probabilities: 0.1109, 0.8761, 0.0129)\n","Processed 197.2;putative_bacteriocin: AMP (Probabilities: 0.0835, 0.9035, 0.0129)\n","Processed 198.2;putative_bacteriocin: non-AMP/OTHERS (Probabilities: 0.9830, 0.0140, 0.0030)\n","Processed 199.2;putative_bacteriocin: AMP (Probabilities: 0.0655, 0.9181, 0.0164)\n","Processed 2.2;AcdB_acidocin_B: non-AMP/OTHERS (Probabilities: 0.9910, 0.0061, 0.0028)\n","Processed 20.2;bacteriocin_LS2chaina: AMP (Probabilities: 0.0974, 0.8901, 0.0125)\n","Processed 200.2;putative_bacteriocin: AMP (Probabilities: 0.1614, 0.8167, 0.0219)\n","Processed 201.2;Rhamnosin_A: non-AMP/OTHERS (Probabilities: 0.9905, 0.0034, 0.0061)\n","Processed 202.2;Sakacin_5XSakX: AMP (Probabilities: 0.0659, 0.9184, 0.0156)\n","Processed 203.2;sakacin_A: AMP (Probabilities: 0.0701, 0.9164, 0.0135)\n","Processed 204.2;sakacin_G_skgA1: AMP (Probabilities: 0.0659, 0.9187, 0.0154)\n","Processed 205.2;sakacin_G_skgA2: AMP (Probabilities: 0.0670, 0.9173, 0.0157)\n","Processed 206.2;Sakacin_P: AMP (Probabilities: 0.0646, 0.9194, 0.0159)\n","Processed 207.2;Sakacin_P_(Sakacin674): AMP (Probabilities: 0.0644, 0.9197, 0.0159)\n","Processed 208.2;Sakacin_Q: AMP (Probabilities: 0.1556, 0.8263, 0.0181)\n","Processed 209.2;Sakacin_Q: AMP (Probabilities: 0.3272, 0.6586, 0.0142)\n","Processed 21.2;bacteriocin_LS2chainb: non-AMP/OTHERS (Probabilities: 0.7495, 0.2354, 0.0152)\n","Processed 210.2;SakT_alpha: AMP (Probabilities: 0.0782, 0.9094, 0.0124)\n","Processed 211.2;SakT_beta: AMP (Probabilities: 0.1740, 0.8140, 0.0120)\n","Processed 212.2;salivaricin_P_chain_a: AMP (Probabilities: 0.0679, 0.9175, 0.0145)\n","Processed 213.2;salivaricin_P_chain_b: AMP (Probabilities: 0.0641, 0.9209, 0.0151)\n","Processed 214.2;Serracin-P: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed 215.2;Subpeptin_JM4-B: non-AMP/OTHERS (Probabilities: 0.8701, 0.0785, 0.0514)\n","Processed 216.2;Subtilosin_(SboX): non-AMP/OTHERS (Probabilities: 0.9928, 0.0043, 0.0029)\n","Processed 217.2;Thermophilin_13_ChainA(ThmA): AMP (Probabilities: 0.0667, 0.9194, 0.0139)\n","Processed 218.2;Thermophilin_13_chainB_(ThmB): AMP (Probabilities: 0.0648, 0.9205, 0.0146)\n","Processed 219.2;Thermophilin_A: AMP (Probabilities: 0.0667, 0.9194, 0.0139)\n","Processed 22.2;Bacteriocin_T8: AMP (Probabilities: 0.0664, 0.9186, 0.0150)\n","Processed 220.2;Thuricin-S: AMP (Probabilities: 0.1152, 0.8721, 0.0127)\n","Processed 221.2;Trifolitoxin: AMP (Probabilities: 0.0691, 0.9174, 0.0135)\n","Processed 222.2;ubericin-A: AMP (Probabilities: 0.0650, 0.9194, 0.0156)\n","Processed 223.2;UviB: non-AMP/OTHERS (Probabilities: 0.9921, 0.0046, 0.0033)\n","Processed 224.2;UviB: non-AMP/OTHERS (Probabilities: 0.9920, 0.0047, 0.0034)\n","Processed 225.2;UviB: non-AMP/OTHERS (Probabilities: 0.9909, 0.0063, 0.0027)\n","Processed 226.2;warnericin_RC: AMP (Probabilities: 0.0815, 0.9057, 0.0128)\n","Processed 227.2;Weisselin_A: AMP (Probabilities: 0.0818, 0.9050, 0.0132)\n","Processed 228.2;Weissellicin_M: AMP (Probabilities: 0.0775, 0.9083, 0.0142)\n","Processed 229.2;Weissellicin_Y: AMP (Probabilities: 0.0705, 0.9151, 0.0144)\n","Processed 23.2;Bavaricin_A: AMP (Probabilities: 0.0704, 0.9165, 0.0131)\n","Processed 230.2;Pentocin KCA1: non-AMP/OTHERS (Probabilities: 0.9907, 0.0062, 0.0032)\n","Processed 231.2;BlpD / Thermophilin 9: non-AMP/OTHERS (Probabilities: 0.9686, 0.0273, 0.0041)\n","Processed 24.2;Bavaricin_MN: AMP (Probabilities: 0.0650, 0.9193, 0.0157)\n","Processed 25.2;BhtB: AMP (Probabilities: 0.0652, 0.9192, 0.0156)\n","Processed 26.2;BlpI: AMP (Probabilities: 0.2694, 0.7203, 0.0103)\n","Processed 27.2;BlpJ: AMP (Probabilities: 0.0647, 0.9210, 0.0143)\n","Processed 28.2;BlpK: AMP (Probabilities: 0.0664, 0.9192, 0.0144)\n","Processed 29.2;BlpK: AMP (Probabilities: 0.0637, 0.9215, 0.0148)\n","Processed 3.2;Acidocin_8912: Neuropeptide (Probabilities: 0.0845, 0.0404, 0.8751)\n","Processed 30.2;BlpM: AMP (Probabilities: 0.0656, 0.9198, 0.0146)\n","Processed 31.2;BlpM: AMP (Probabilities: 0.0668, 0.9187, 0.0144)\n","Processed 32.2;BlpM: AMP (Probabilities: 0.0663, 0.9192, 0.0144)\n","Processed 33.2;BlpM: AMP (Probabilities: 0.0656, 0.9197, 0.0147)\n","Processed 34.2;BlpN: AMP (Probabilities: 0.3892, 0.5883, 0.0224)\n","Processed 35.2;BlpN: AMP (Probabilities: 0.3067, 0.6746, 0.0187)\n","Processed 36.2;BlpN: AMP (Probabilities: 0.1829, 0.8032, 0.0139)\n","Processed 37.2;BlpN: AMP (Probabilities: 0.3750, 0.6058, 0.0192)\n","Processed 38.2;BlpO: Neuropeptide (Probabilities: 0.0271, 0.0160, 0.9570)\n","Processed 39.2;BlpU: AMP (Probabilities: 0.0732, 0.9140, 0.0128)\n","Processed 4.2;Acidocin_8912_N-terminal: Neuropeptide (Probabilities: 0.3211, 0.3394, 0.3395)\n","Processed 40.2;BlpU: AMP (Probabilities: 0.0669, 0.9182, 0.0149)\n","Processed 41.2;Boticin_B: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed 42.2;Bovicin_255_peptide: AMP (Probabilities: 0.0659, 0.9189, 0.0152)\n","Processed 43.2;Bovicin_255_variant: AMP (Probabilities: 0.0665, 0.9184, 0.0151)\n","Processed 44.2;Bovicin_255_variant: AMP (Probabilities: 0.0656, 0.9194, 0.0151)\n","Processed 45.2;Brochocin_C_BrcA: AMP (Probabilities: 0.0669, 0.9181, 0.0150)\n","Processed 46.2;Brochocin_C_BrcB: AMP (Probabilities: 0.0663, 0.9187, 0.0149)\n","Processed 47.2;BTL_(N-terminus): non-AMP/OTHERS (Probabilities: 0.9408, 0.0083, 0.0509)\n","Processed 48.2;Carno_bacteriocin_A_(Piscicolin61): AMP (Probabilities: 0.0669, 0.9187, 0.0145)\n","Processed 49.2;Carnobacteriocin_B2_(CarnocinCP52): AMP (Probabilities: 0.0667, 0.9180, 0.0153)\n","Processed 5.2;Acidocin_J1132_alpha_peptide_N-terminal: AMP (Probabilities: 0.1080, 0.8787, 0.0133)\n","Processed 50.2;Carnobacteriocin_BM1_(CarnobacteriocinB1): AMP (Probabilities: 0.0829, 0.9049, 0.0122)\n","Processed 51.2;Carnocin_CP52: non-AMP/OTHERS (Probabilities: 0.9931, 0.0041, 0.0028)\n","Processed 52.2;cerein_7B: AMP (Probabilities: 0.0743, 0.9116, 0.0141)\n","Processed 53.2;Coagulin_CoaA: AMP (Probabilities: 0.0653, 0.9187, 0.0161)\n","Processed 54.2;CoagulinA: AMP (Probabilities: 0.0653, 0.9187, 0.0161)\n","Processed 55.2;Colicin_V: AMP (Probabilities: 0.0811, 0.9064, 0.0126)\n","Processed 56.2;Colicin_V: AMP (Probabilities: 0.0672, 0.9192, 0.0137)\n","Processed 57.2;Curvacin_A: AMP (Probabilities: 0.0701, 0.9164, 0.0135)\n","Processed 58.2;Curvalicin-28a: Neuropeptide (Probabilities: 0.0073, 0.0087, 0.9840)\n","Processed 59.2;Curvalicin-28b: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed 6.2;Acidocin_J1132_beta_peptide_N-terminal: AMP (Probabilities: 0.0950, 0.8920, 0.0130)\n","Processed 60.2;Curvalicin-28c: non-AMP/OTHERS (Probabilities: 0.5047, 0.2324, 0.2629)\n","Processed 61.2;Curvaticin: AMP (Probabilities: 0.0983, 0.8886, 0.0131)\n","Processed 62.2;curvaticin_FS47: AMP (Probabilities: 0.0789, 0.9085, 0.0125)\n","Processed 63.2;Curvaticin_L442: AMP (Probabilities: 0.0983, 0.8886, 0.0131)\n","Processed 64.2;Delta-lysinI: AMP (Probabilities: 0.0806, 0.9059, 0.0135)\n","Processed 65.2;Divercin_RV41(mature): AMP (Probabilities: 0.0680, 0.9169, 0.0151)\n","Processed 66.2;Divercin_V41_(DvnV41): AMP (Probabilities: 0.0644, 0.9197, 0.0159)\n","Processed 67.2;Divergicin_750: AMP (Probabilities: 0.1022, 0.8860, 0.0118)\n","Processed 68.2;Divergicin_A: AMP (Probabilities: 0.1653, 0.8231, 0.0116)\n","Processed 69.2;Divergicin_M35: non-AMP/OTHERS (Probabilities: 0.9893, 0.0079, 0.0028)\n","Processed 7.2;Acidocin_LF221B(GassericinK7B): AMP (Probabilities: 0.0686, 0.9173, 0.0140)\n","Processed 70.2;Divergicin_M35: AMP (Probabilities: 0.0660, 0.9179, 0.0161)\n","Processed 71.2;Divergicin_M35: AMP (Probabilities: 0.0660, 0.9179, 0.0161)\n","Processed 72.2;Durancin_Q: AMP (Probabilities: 0.0985, 0.8888, 0.0127)\n","Processed 73.2;Durancin_TW-49M: AMP (Probabilities: 0.0772, 0.9101, 0.0127)\n","Processed 74.2;E50-52: AMP (Probabilities: 0.0658, 0.9180, 0.0162)\n","Processed 75.2;Eenterocin_P-like: AMP (Probabilities: 0.0685, 0.9184, 0.0131)\n","Processed 76.2;EJ97enterocin: AMP (Probabilities: 0.0697, 0.9160, 0.0142)\n","Processed 77.2;EntA: AMP (Probabilities: 0.0663, 0.9176, 0.0162)\n","Processed 78.2;Enterocin_1071A: non-AMP/OTHERS (Probabilities: 0.5147, 0.4761, 0.0092)\n","Processed 79.2;Enterocin_1071A: AMP (Probabilities: 0.4295, 0.5602, 0.0103)\n","Processed 8.2;AcidocinA: AMP (Probabilities: 0.0666, 0.9186, 0.0148)\n","Processed 80.2;Enterocin_1071B: AMP (Probabilities: 0.0660, 0.9189, 0.0151)\n","Processed 81.2;Enterocin_A: AMP (Probabilities: 0.0661, 0.9176, 0.0163)\n","Processed 82.2;Enterocin_B: AMP (Probabilities: 0.0699, 0.9165, 0.0137)\n","Processed 83.2;Enterocin_CRL35: AMP (Probabilities: 0.0672, 0.9168, 0.0161)\n","Processed 84.2;Enterocin_CRL35_(pediocin-like)_{N-terminal}: non-AMP/OTHERS (Probabilities: 0.5324, 0.3260, 0.1416)\n","Processed 85.2;Enterocin_E-760: AMP (Probabilities: 0.0643, 0.9197, 0.0160)\n","Processed 86.2;Enterocin_L50a: AMP (Probabilities: 0.0702, 0.9147, 0.0151)\n","Processed 87.2;Enterocin_L50b: AMP (Probabilities: 0.0682, 0.9165, 0.0153)\n","Processed 88.2;Enterocin_NKR-5-3A: AMP (Probabilities: 0.0669, 0.9189, 0.0142)\n","Processed 89.2;Enterocin_NKR-5-3D: AMP (Probabilities: 0.1288, 0.8576, 0.0136)\n","Processed 9.2;AdDLP: AMP (Probabilities: 0.2364, 0.7175, 0.0461)\n","Processed 90.2;Enterocin_NKR-5-3Z: AMP (Probabilities: 0.0651, 0.9200, 0.0149)\n","Processed 91.2;Enterocin_P: AMP (Probabilities: 0.1219, 0.8661, 0.0121)\n","Processed 92.2;Enterocin_P-like: AMP (Probabilities: 0.0868, 0.8996, 0.0136)\n","Processed 93.2;Enterocin_Q: AMP (Probabilities: 0.0701, 0.9166, 0.0133)\n","Processed 94.2;Enterocin_RJ-11: AMP (Probabilities: 0.0736, 0.9118, 0.0146)\n","Processed 95.2;Enterocin_SE-K4: AMP (Probabilities: 0.0696, 0.9162, 0.0142)\n","Processed 96.2;Enterocin_X_chain_alpha: AMP (Probabilities: 0.0828, 0.9053, 0.0119)\n","Processed 97.2;Enterocin_X_chain_beta: non-AMP/OTHERS (Probabilities: 0.9573, 0.0390, 0.0037)\n","Processed 98.2;Enterocin-HF: AMP (Probabilities: 0.0673, 0.9166, 0.0162)\n","Processed 99.2;Enterocin-M: non-AMP/OTHERS (Probabilities: 0.9916, 0.0060, 0.0024)\n","Processed 1.3;alveicin_A_Bacteriocintoxin: AMP (Probabilities: 0.0664, 0.9197, 0.0139)\n","Processed 10.3;Colicin: non-AMP/OTHERS (Probabilities: 0.8996, 0.0938, 0.0066)\n","Processed 11.3;Colicin: AMP (Probabilities: 0.0885, 0.8919, 0.0196)\n","Processed 12.3;Colicin_B: non-AMP/OTHERS (Probabilities: 0.9533, 0.0390, 0.0077)\n","Processed 13.3;Colicin_E1: non-AMP/OTHERS (Probabilities: 0.9319, 0.0629, 0.0052)\n","Processed 14.3;Colicin_E1: non-AMP/OTHERS (Probabilities: 0.9340, 0.0609, 0.0051)\n","Processed 15.3;Colicin_E1: AMP (Probabilities: 0.2313, 0.7555, 0.0131)\n","Processed 16.3;Colicin_E1_protein: non-AMP/OTHERS (Probabilities: 0.9266, 0.0680, 0.0054)\n","Processed 17.3;Colicin_E5: AMP (Probabilities: 0.0778, 0.9089, 0.0133)\n","Processed 18.3;Colicin_E6: AMP (Probabilities: 0.0814, 0.9067, 0.0119)\n","Processed 19.3;Colicin_E7: AMP (Probabilities: 0.0736, 0.9137, 0.0127)\n","Processed 2.3;alveicin_B_Bacteriocintoxin: AMP (Probabilities: 0.0649, 0.9190, 0.0160)\n","Processed 20.3;Colicin_E7: AMP (Probabilities: 0.0736, 0.9137, 0.0127)\n","Processed 21.3;Colicin_E9: AMP (Probabilities: 0.0805, 0.9065, 0.0129)\n","Processed 22.3;Colicin_Ia: non-AMP/OTHERS (Probabilities: 0.4920, 0.4917, 0.0163)\n","Processed 23.3;Colicin_Ia: AMP (Probabilities: 0.1696, 0.8136, 0.0168)\n","Processed 24.3;Colicin_IB: non-AMP/OTHERS (Probabilities: 0.6919, 0.2951, 0.0131)\n","Processed 25.3;Colicin_IB: non-AMP/OTHERS (Probabilities: 0.6919, 0.2951, 0.0131)\n","Processed 26.3;Colicin_Ib: non-AMP/OTHERS (Probabilities: 0.4920, 0.4917, 0.0163)\n","Processed 27.3;Colicin_Ib: non-AMP/OTHERS (Probabilities: 0.4920, 0.4917, 0.0163)\n","Processed 28.3;Colicin_Ib: non-AMP/OTHERS (Probabilities: 0.4920, 0.4917, 0.0163)\n","Processed 29.3;Colicin_IB: non-AMP/OTHERS (Probabilities: 0.6920, 0.2951, 0.0129)\n","Processed 3.3;Bacteriocin: non-AMP/OTHERS (Probabilities: 0.9138, 0.0738, 0.0124)\n","Processed 30.3;Colicin_K: AMP (Probabilities: 0.1022, 0.8858, 0.0120)\n","Processed 31.3;Colicin_K: AMP (Probabilities: 0.1022, 0.8858, 0.0120)\n","Processed 32.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.9340, 0.0609, 0.0051)\n","Processed 33.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.4920, 0.4917, 0.0163)\n","Processed 34.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.4920, 0.4917, 0.0163)\n","Processed 35.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.4920, 0.4917, 0.0163)\n","Processed 36.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.4920, 0.4917, 0.0163)\n","Processed 37.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.4920, 0.4917, 0.0163)\n","Processed 38.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.9325, 0.0598, 0.0076)\n","Processed 39.3;Colicin_protein: AMP (Probabilities: 0.4641, 0.5188, 0.0172)\n","Processed 4.3;Bacteriocin_28b: AMP (Probabilities: 0.0680, 0.9184, 0.0137)\n","Processed 40.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.9340, 0.0609, 0.0051)\n","Processed 41.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.9340, 0.0609, 0.0051)\n","Processed 42.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.9340, 0.0609, 0.0051)\n","Processed 43.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.9340, 0.0609, 0.0051)\n","Processed 44.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.9340, 0.0609, 0.0051)\n","Processed 45.3;Colicin_protein: non-AMP/OTHERS (Probabilities: 0.9340, 0.0609, 0.0051)\n","Processed 46.3;Colicin_U: AMP (Probabilities: 0.0695, 0.9176, 0.0129)\n","Processed 47.3;Colicin_Y: AMP (Probabilities: 0.0661, 0.9200, 0.0139)\n","Processed 48.3;Colicin_Y: AMP (Probabilities: 0.0661, 0.9200, 0.0139)\n","Processed 49.3;Colicin-10: AMP (Probabilities: 0.1543, 0.8331, 0.0127)\n","Processed 5.3;Bacteriocin_BCN5: AMP (Probabilities: 0.2347, 0.7526, 0.0128)\n","Processed 50.3;Colicin-A: AMP (Probabilities: 0.0668, 0.9180, 0.0152)\n","Processed 51.3;Colicin-A: AMP (Probabilities: 0.0659, 0.9186, 0.0155)\n","Processed 52.3;Colicin-A: AMP (Probabilities: 0.0688, 0.9176, 0.0137)\n","Processed 53.3;Colicin-B: non-AMP/OTHERS (Probabilities: 0.9533, 0.0390, 0.0077)\n","Processed 54.3;Colicin-D: non-AMP/OTHERS (Probabilities: 0.9677, 0.0238, 0.0085)\n","Processed 55.3;Colicin-E1: non-AMP/OTHERS (Probabilities: 0.9362, 0.0586, 0.0052)\n","Processed 56.3;Colicin-E1: non-AMP/OTHERS (Probabilities: 0.9385, 0.0564, 0.0050)\n","Processed 57.3;Colicin-E1: non-AMP/OTHERS (Probabilities: 0.9340, 0.0609, 0.0051)\n","Processed 58.3;Colicin-E9: AMP (Probabilities: 0.1052, 0.8827, 0.0121)\n","Processed 59.3;Colicin-M: AMP (Probabilities: 0.1173, 0.8713, 0.0114)\n","Processed 6.3;Bacteriocin_helveticin_J: non-AMP/OTHERS (Probabilities: 0.9477, 0.0474, 0.0049)\n","Processed 60.3;Colicin-N: AMP (Probabilities: 0.0646, 0.9195, 0.0158)\n","Processed 61.3;Dysgalacticin: AMP (Probabilities: 0.1617, 0.8270, 0.0113)\n","Processed 62.3;enterolysin_A: AMP (Probabilities: 0.3026, 0.6861, 0.0114)\n","Processed 63.3;Enterolysin_A: AMP (Probabilities: 0.3026, 0.6861, 0.0114)\n","Processed 64.3;Enterolysin_A: non-AMP/OTHERS (Probabilities: 0.8882, 0.1060, 0.0058)\n","Processed 65.3;Halocin_H4: AMP (Probabilities: 0.1858, 0.7943, 0.0199)\n","Processed 66.3;Halocin_H4_: AMP (Probabilities: 0.1887, 0.7878, 0.0235)\n","Processed 67.3;Halocin_S8: non-AMP/OTHERS (Probabilities: 0.5150, 0.4739, 0.0111)\n","Processed 68.3;Halocin-S8: AMP (Probabilities: 0.1056, 0.8828, 0.0116)\n","Processed 69.3;Helveticin: non-AMP/OTHERS (Probabilities: 0.9700, 0.0260, 0.0041)\n","Processed 7.3;Bacteriocinalbusin_B: non-AMP/OTHERS (Probabilities: 0.6711, 0.3208, 0.0081)\n","Processed 70.3;Helveticin-J: non-AMP/OTHERS (Probabilities: 0.9595, 0.0344, 0.0060)\n","Processed 71.3;klebicin_C_activity: non-AMP/OTHERS (Probabilities: 0.9888, 0.0076, 0.0036)\n","Processed 72.3;klebicin_C_activity: non-AMP/OTHERS (Probabilities: 0.9886, 0.0075, 0.0039)\n","Processed 73.3;klebicin_C_phage_associated_protein: AMP (Probabilities: 0.1273, 0.8581, 0.0146)\n","Processed 74.3;klebicin_C_phage_associated_protein: AMP (Probabilities: 0.2172, 0.7698, 0.0130)\n","Processed 75.3;klebicin_D_activity_protein: AMP (Probabilities: 0.0638, 0.9210, 0.0152)\n","Processed 76.3;klebicin_D_phage-associated_protein: AMP (Probabilities: 0.0804, 0.9070, 0.0126)\n","Processed 77.3;lin_M18: non-AMP/OTHERS (Probabilities: 0.9472, 0.0438, 0.0090)\n","Processed 78.3;linocin_CFP29_homolog: non-AMP/OTHERS (Probabilities: 0.9601, 0.0314, 0.0086)\n","Processed 79.3;LINOCIN_M18: non-AMP/OTHERS (Probabilities: 0.9275, 0.0620, 0.0105)\n","Processed 8.3;Carocin_D: AMP (Probabilities: 0.0728, 0.9142, 0.0130)\n","Processed 80.3;Linocin_M18: non-AMP/OTHERS (Probabilities: 0.9532, 0.0361, 0.0108)\n","Processed 81.3;Linocin_M18Bacteriocin: non-AMP/OTHERS (Probabilities: 0.9553, 0.0356, 0.0091)\n","Processed 82.3;Linocin-M18: non-AMP/OTHERS (Probabilities: 0.9498, 0.0407, 0.0095)\n","Processed 83.3;PaeM: AMP (Probabilities: 0.1821, 0.7933, 0.0246)\n","Processed 84.3;pesticin: non-AMP/OTHERS (Probabilities: 0.7528, 0.0410, 0.2062)\n","Processed 85.3;pesticin: non-AMP/OTHERS (Probabilities: 0.8633, 0.0321, 0.1046)\n","Processed 86.3;putative_Bacteriocin_family_protein: non-AMP/OTHERS (Probabilities: 0.9544, 0.0355, 0.0101)\n","Processed 87.3;putidacin_L1: AMP (Probabilities: 0.2506, 0.6920, 0.0574)\n","Processed 88.3;pyocin_AP41_subunit: AMP (Probabilities: 0.3617, 0.6258, 0.0124)\n","Processed 89.3;Pyocin_S1: AMP (Probabilities: 0.1885, 0.6969, 0.1146)\n","Processed 9.3;Closticin_574: AMP (Probabilities: 0.2966, 0.6925, 0.0109)\n","Processed 90.3;Pyocin_S2: AMP (Probabilities: 0.1630, 0.8176, 0.0195)\n","Processed 91.3;Pyocin-S1: AMP (Probabilities: 0.2092, 0.6781, 0.1127)\n","Processed 92.3;Pyocin-S2: AMP (Probabilities: 0.1250, 0.8534, 0.0216)\n","Processed 93.3;Zoocin_A: non-AMP/OTHERS (Probabilities: 0.9492, 0.0475, 0.0033)\n","Processed sp|Q9X2V7|MCJA_ECOLX Microcin J25 OS=Escherichia coli OX=562 GN=mcjA PE=1 SV=1: AMP (Probabilities: 0.0895, 0.8990, 0.0115)\n","Processed sp|Q46971|MCNN_ECOLX Microcin N OS=Escherichia coli OX=562 GN=mcnN PE=1 SV=2: AMP (Probabilities: 0.0786, 0.9088, 0.0126)\n","Processed sp|Q9Z4N4|MCEA_KLEPN Microcin E492 OS=Klebsiella pneumoniae OX=573 GN=mceA PE=1 SV=2: AMP (Probabilities: 0.0673, 0.9188, 0.0139)\n","Processed sp|P02984|IMM3_ECOLX Colicin E3 immunity protein OS=Escherichia coli OX=562 GN=imm PE=1 SV=3: non-AMP/OTHERS (Probabilities: 0.9911, 0.0048, 0.0042)\n","Processed sp|P05834|MCBA_ECOLX Bacteriocin microcin B17 OS=Escherichia coli OX=562 GN=mcbA PE=1 SV=1: AMP (Probabilities: 0.4763, 0.5031, 0.0205)\n","Processed sp|P08550|CVPA_ECOLI Colicin V production protein OS=Escherichia coli (strain K12) OX=83333 GN=cvpA PE=1 SV=1: non-AMP/OTHERS (Probabilities: 0.9090, 0.0865, 0.0046)\n","Processed sp|P13479|IMM9_ECOLX Colicin-E9 immunity protein OS=Escherichia coli OX=562 GN=imm PE=1 SV=2: non-AMP/OTHERS (Probabilities: 0.9064, 0.0114, 0.0822)\n","Processed sp|P22522|CEAV_ECOLX Colicin-V OS=Escherichia coli OX=562 GN=cvaC PE=1 SV=1: AMP (Probabilities: 0.0665, 0.9201, 0.0134)\n","Processed sp|Q47505|MCCC7_ECOLX Microcin C7 OS=Escherichia coli OX=562 GN=mccA PE=1 SV=1: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed sp|O86200|MCHI_ECOLX Microcin H47 immunity protein MchI OS=Escherichia coli OX=562 GN=mchI PE=1 SV=1: non-AMP/OTHERS (Probabilities: 0.9910, 0.0059, 0.0031)\n","Processed sp|P04482|IMM2_ECOLX Colicin-E2 immunity protein OS=Escherichia coli OX=562 GN=imm PE=1 SV=1: non-AMP/OTHERS (Probabilities: 0.6825, 0.0261, 0.2914)\n","Processed sp|P05701|IMMA_CITFR Colicin-A immunity protein OS=Citrobacter freundii OX=546 GN=cai PE=1 SV=1: non-AMP/OTHERS (Probabilities: 0.9912, 0.0058, 0.0030)\n","Processed sp|P09181|LYS4_ECOLX Lysis protein for colicin N OS=Escherichia coli OX=562 GN=cnl PE=1 SV=1: non-AMP/OTHERS (Probabilities: 0.9920, 0.0047, 0.0033)\n","Processed sp|P09881|IMM8_ECOLX Colicin-E8 immunity protein OS=Escherichia coli OX=562 GN=imm PE=1 SV=1: non-AMP/OTHERS (Probabilities: 0.9820, 0.0052, 0.0128)\n","Processed sp|P11899|IMMD_ECOLX Colicin-D immunity protein OS=Escherichia coli OX=562 GN=cdi PE=1 SV=2: non-AMP/OTHERS (Probabilities: 0.9920, 0.0042, 0.0039)\n","Processed sp|P13476|IMM5_ECOLX Colicin-E5 immunity protein OS=Escherichia coli OX=562 GN=imm PE=1 SV=1: non-AMP/OTHERS (Probabilities: 0.9929, 0.0036, 0.0035)\n","Processed sp|P18000|CEA5_ECOLX Colicin-E5 (Fragment) OS=Escherichia coli OX=562 GN=col PE=1 SV=1: AMP (Probabilities: 0.0778, 0.9089, 0.0133)\n","Processed sp|P18002|IMMM_ECOLX Colicin-M immunity protein OS=Escherichia coli OX=562 GN=cmi PE=1 SV=2: non-AMP/OTHERS (Probabilities: 0.6097, 0.3787, 0.0115)\n","Processed sp|P62530|MCHB_ECOLX Microcin H47 OS=Escherichia coli OX=562 GN=mchB PE=1 SV=1: AMP (Probabilities: 0.1600, 0.8281, 0.0118)\n","Processed sp|Q03708|IMM7_ECOLX Colicin-E7 immunity protein OS=Escherichia coli OX=562 GN=imm PE=1 SV=2: non-AMP/OTHERS (Probabilities: 0.9883, 0.0057, 0.0060)\n","Processed WP_029376348.1 type A lantibiotic [Staphylococcus epidermidis]: AMP (Probabilities: 0.0685, 0.9177, 0.0138)\n","Processed WP_042739435.1 gallidermin family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.9789, 0.0117, 0.0094)\n","Processed WP_015057295.1 type A2 lantipeptide [Streptococcus dysgalactiae group]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0034, 0.0046)\n","Processed WP_023416956.1 FxLD family lanthipeptide [unclassified Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9470, 0.0097, 0.0434)\n","Processed WP_155961898.1 type A2 lantipeptide [Streptococcus dysgalactiae]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0038)\n","Processed WP_012380187.1 FxLD family lanthipeptide [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9845, 0.0074, 0.0081)\n","Processed WP_137578055.1 gallidermin/nisin family lantibiotic [Staphylococcus argenteus]: non-AMP/OTHERS (Probabilities: 0.9764, 0.0050, 0.0186)\n","Processed WP_072324115.1 gallidermin/nisin family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.9511, 0.0110, 0.0379)\n","Processed WP_167533993.1 VenA family class IV lanthipeptide [Streptomyces gardneri]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0039, 0.0049)\n","Processed WP_015609023.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0040, 0.0030)\n","Processed WP_020449898.1 plantaricin C family lantibiotic [Bacillus]: Neuropeptide (Probabilities: 0.0336, 0.0186, 0.9478)\n","Processed WP_029325769.1 plantaricin C family lantibiotic [Bacillus]: Neuropeptide (Probabilities: 0.0206, 0.0141, 0.9652)\n","Processed WP_011229795.1 gallidermin/nisin family lantibiotic [Bacillaceae]: non-AMP/OTHERS (Probabilities: 0.8310, 0.1301, 0.0389)\n","Processed WP_003186379.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus]: AMP (Probabilities: 0.0872, 0.9008, 0.0120)\n","Processed WP_003186381.1 plantaricin C family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9256, 0.0101, 0.0642)\n","Processed WP_001286517.1 plantaricin C family lantibiotic [Streptococcus]: Neuropeptide (Probabilities: 0.1363, 0.0460, 0.8177)\n","Processed WP_000416757.1 gallidermin/nisin family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.5091, 0.0554, 0.4355)\n","Processed WP_002995525.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0040)\n","Processed WP_002469996.1 gallidermin family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.9863, 0.0036, 0.0101)\n","Processed WP_003220055.1 gallidermin/nisin family lantibiotic [Bacillus]: AMP (Probabilities: 0.0738, 0.9136, 0.0126)\n","Processed WP_012038654.1 lantibiotic michiganin-A [Clavibacter michiganensis]: non-AMP/OTHERS (Probabilities: 0.5413, 0.2778, 0.1809)\n","Processed WP_015425978.1 gallidermin/nisin family lantibiotic [Lactococcus]: AMP (Probabilities: 0.0697, 0.9163, 0.0140)\n","Processed WP_031787916.1 gallidermin/nisin family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.9154, 0.0108, 0.0738)\n","Processed WP_000432510.1 gallidermin/nisin family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.8595, 0.0378, 0.1027)\n","Processed WP_001058290.1 gallidermin/nisin family lantibiotic [Bacilli]: AMP (Probabilities: 0.1408, 0.8241, 0.0351)\n","Processed WP_032726862.1 gallidermin/nisin family lantibiotic [Bacillus]: AMP (Probabilities: 0.0786, 0.9088, 0.0126)\n","Processed WP_006639978.1 gallidermin/nisin family lantibiotic [Bacillus]: AMP (Probabilities: 0.0688, 0.9172, 0.0141)\n","Processed WP_016898795.1 gallidermin family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.9831, 0.0045, 0.0124)\n","Processed WP_010824504.1 plantaricin C family lantibiotic [Enterococcus faecalis]: Neuropeptide (Probabilities: 0.0142, 0.0130, 0.9728)\n","Processed WP_000410306.1 type A2 lantipeptide [Streptococcaceae]: non-AMP/OTHERS (Probabilities: 0.9894, 0.0047, 0.0058)\n","Processed WP_000606628.1 gallidermin/nisin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0037, 0.0035)\n","Processed WP_016090584.1 type A2 lantipeptide [Bacillus]: AMP (Probabilities: 0.2117, 0.7448, 0.0435)\n","Processed WP_002498697.1 gallidermin family lantibiotic [Staphylococcus epidermidis]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0053, 0.0029)\n","Processed WP_000416760.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.2171, 0.0323, 0.7507)\n","Processed WP_014570405.1 gallidermin/nisin family lantibiotic [Lactococcus lactis]: AMP (Probabilities: 0.0669, 0.9183, 0.0147)\n","Processed WP_034655941.1 mersacidin family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9936, 0.0034, 0.0030)\n","Processed WP_034655938.1 mersacidin family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0042, 0.0030)\n","Processed WP_002403042.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9887, 0.0031, 0.0082)\n","Processed WP_002268802.1 lantibiotic mutacin 1140 [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.8189, 0.0247, 0.1564)\n","Processed WP_049522134.1 plantaricin C family lantibiotic [Streptococcus]: Neuropeptide (Probabilities: 0.0260, 0.0166, 0.9574)\n","Processed WP_085613746.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9878, 0.0038, 0.0084)\n","Processed WP_023612114.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9883, 0.0039, 0.0077)\n","Processed WP_009967544.1 bacteriocin sublancin-168 [Bacillus]: AMP (Probabilities: 0.0650, 0.9201, 0.0150)\n","Processed WP_079267797.1 gallidermin/nisin family lantibiotic [Streptococcus]: AMP (Probabilities: 0.0783, 0.9082, 0.0135)\n","Processed WP_000794682.1 type A2 lantipeptide [Streptococcaceae]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0034)\n","Processed WP_074790748.1 lichenicidin A2 family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0050, 0.0036)\n","Processed WP_074790746.1 mersacidin family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.2129, 0.7609, 0.0262)\n","Processed WP_050197451.1 plantaricin C family lantibiotic [Streptococcus]: Neuropeptide (Probabilities: 0.0134, 0.0117, 0.9750)\n","Processed WP_030065142.1 gallidermin family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0036, 0.0077)\n","Processed WP_002984803.1 lantibiotic streptin [Streptococcus pyogenes]: AMP (Probabilities: 0.0673, 0.9181, 0.0146)\n","Processed WP_002982773.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0031)\n","Processed WP_000244521.1 hypothetical protein [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0064, 0.0025)\n","Processed WP_001245107.1 lichenicidin A2 family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0038, 0.0028)\n","Processed WP_002358485.1 lantipeptide cytolysin subunit CylL-L [Bacteria]: non-AMP/OTHERS (Probabilities: 0.9875, 0.0095, 0.0030)\n","Processed WP_087440185.1 lichenicidin A2 family type 2 lantibiotic [Paenibacillus thiaminolyticus]: non-AMP/OTHERS (Probabilities: 0.9838, 0.0123, 0.0038)\n","Processed WP_036623852.1 mersacidin/lichenicidin family type 2 lantibiotic [Paenibacillus macerans]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0036, 0.0046)\n","Processed WP_016820476.1 FDLD family class I lanthipeptide [Paenibacillus]: AMP (Probabilities: 0.1153, 0.8720, 0.0128)\n","Processed WP_004635387.1 plantaricin C family lantibiotic [Dolosigranulum pigrum]: Neuropeptide (Probabilities: 0.1118, 0.0296, 0.8586)\n","Processed WP_000181087.1 two-peptide bacteriocin subunit PneA1 [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0078, 0.0024)\n","Processed WP_001031160.1 type A lantibiotic [Bacillus]: AMP (Probabilities: 0.0697, 0.9166, 0.0137)\n","Processed WP_002401134.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus faecalis]: AMP (Probabilities: 0.2231, 0.7642, 0.0127)\n","Processed WP_000064151.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0045, 0.0040)\n","Processed WP_010994049.1 plantaricin C family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.6542, 0.0297, 0.3161)\n","Processed WP_002396812.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9871, 0.0038, 0.0091)\n","Processed WP_017644334.1 plantaricin C family lantibiotic [Firmicutes]: non-AMP/OTHERS (Probabilities: 0.5755, 0.0490, 0.3755)\n","Processed WP_001031159.1 type A lantibiotic [Bacillus]: AMP (Probabilities: 0.0857, 0.9018, 0.0125)\n","Processed WP_002358179.1 lantipeptide cytolysin subunit CylL-L [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9819, 0.0150, 0.0031)\n","Processed WP_115313665.1 gallidermin family lantibiotic [Staphylococcus saccharolyticus]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0033, 0.0050)\n","Processed WP_079533552.1 lantibiotic michiganin-A [Clavibacter michiganensis]: non-AMP/OTHERS (Probabilities: 0.4932, 0.3893, 0.1175)\n","Processed WP_002887181.1 type A2 lantipeptide [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0035, 0.0057)\n","Processed WP_010994050.1 class II lanthipeptide, LchA2/BrtA2 family [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9648, 0.0314, 0.0038)\n","Processed WP_069995861.1 plantaricin C family lantibiotic [Staphylococcus]: Neuropeptide (Probabilities: 0.1286, 0.0280, 0.8434)\n","Processed WP_002358181.1 lantipeptide cytolysin subunit CylL-S [Bacteria]: non-AMP/OTHERS (Probabilities: 0.8130, 0.0151, 0.1719)\n","Processed WP_116100560.1 type A lantibiotic [Bacillus mycoides]: AMP (Probabilities: 0.1671, 0.8219, 0.0110)\n","Processed WP_107519564.1 gallidermin family lantibiotic [Staphylococcus epidermidis]: non-AMP/OTHERS (Probabilities: 0.9893, 0.0065, 0.0042)\n","Processed WP_024785188.1 lantibiotic mutacin I [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0035, 0.0042)\n","Processed WP_113970549.1 mersacidin family lantibiotic [Rossellomorea aquimaris]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0047, 0.0062)\n","Processed WP_113970544.1 mersacidin family lantibiotic [Rossellomorea aquimaris]: non-AMP/OTHERS (Probabilities: 0.9725, 0.0174, 0.0101)\n","Processed WP_113970550.1 mersacidin family lantibiotic [Rossellomorea aquimaris]: non-AMP/OTHERS (Probabilities: 0.9876, 0.0096, 0.0028)\n","Processed WP_113970543.1 mersacidin family lantibiotic [Rossellomorea aquimaris]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0045, 0.0032)\n","Processed WP_002935048.1 type A2 lantipeptide [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0039, 0.0043)\n","Processed WP_011528609.1 lantibiotic streptin [Streptococcus pyogenes]: AMP (Probabilities: 0.0663, 0.9187, 0.0150)\n","Processed WP_000416758.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.5511, 0.0514, 0.3975)\n","Processed WP_002935049.1 type A2 lantipeptide [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0045, 0.0055)\n","Processed WP_094938127.1 mersacidin family lantibiotic [Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0031, 0.0049)\n","Processed WP_002296231.1 type A2 lantipeptide [Streptococcus mutans]: Neuropeptide (Probabilities: 0.4253, 0.0557, 0.5190)\n","Processed WP_002268660.1 type A2 lantipeptide [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.7014, 0.0407, 0.2579)\n","Processed WP_074619581.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.6682, 0.1246, 0.2072)\n","Processed WP_016078320.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0031, 0.0035)\n","Processed WP_098878099.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9670, 0.0277, 0.0053)\n","Processed WP_016078319.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0033, 0.0070)\n","Processed WP_010772180.1 mersacidin family lantibiotic [Enterococcus caccae]: AMP (Probabilities: 0.0682, 0.9180, 0.0138)\n","Processed WP_001031158.1 type A lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.0738, 0.9132, 0.0131)\n","Processed WP_010772179.1 mersacidin family lantibiotic [Enterococcus caccae]: AMP (Probabilities: 0.0704, 0.9161, 0.0135)\n","Processed WP_000786759.1 two-peptide bacteriocin subunit PneA2 [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9771, 0.0127, 0.0102)\n","Processed WP_074619582.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0061, 0.0031)\n","Processed WP_010772181.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus caccae]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0037, 0.0049)\n","Processed WP_038675129.1 gallidermin/nisin family lantibiotic [Streptococcus salivarius]: AMP (Probabilities: 0.0922, 0.8956, 0.0122)\n","Processed WP_088777683.1 gallidermin/nisin family lantibiotic [Streptococcus]: AMP (Probabilities: 0.0971, 0.8897, 0.0132)\n","Processed WP_072903626.1 gallidermin/nisin family lantibiotic [Streptococcus salivarius]: AMP (Probabilities: 0.0939, 0.8932, 0.0129)\n","Processed WP_086505100.1 lantibiotic michiganin-A [Clavibacter michiganensis]: non-AMP/OTHERS (Probabilities: 0.5457, 0.3289, 0.1254)\n","Processed WP_087197770.1 lantibiotic michiganin-A [Clavibacter michiganensis]: non-AMP/OTHERS (Probabilities: 0.6189, 0.2036, 0.1775)\n","Processed WP_003968635.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0032)\n","Processed WP_008881441.1 gallidermin/nisin family lantibiotic [Geobacillus]: non-AMP/OTHERS (Probabilities: 0.8143, 0.1597, 0.0259)\n","Processed WP_010772178.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0049, 0.0048)\n","Processed WP_190107705.1 cinnamycin family lantibiotic [Streptomyces cinnamoneus]: non-AMP/OTHERS (Probabilities: 0.7859, 0.0834, 0.1307)\n","Processed WP_072526671.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9935, 0.0033, 0.0032)\n","Processed ACY11712.1 epidermin immunity protein F [Staphylococcus aureus subsp. aureus ED98 ED98]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0060, 0.0027)\n","Processed WP_010081696.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium longum]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0040, 0.0038)\n","Processed WP_079152910.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9256, 0.0091, 0.0653)\n","Processed P0DQM1.1 Lantibiotic Flvalpha.a [Ruminococcus flavefaciens]: AMP (Probabilities: 0.0923, 0.8957, 0.0120)\n","Processed WP_084839579.1 plantaricin C family lantibiotic [Williamsia]: non-AMP/OTHERS (Probabilities: 0.4564, 0.4468, 0.0968)\n","Processed WP_006839087.1 gallidermin/nisin family lantibiotic [Corynebacterium lipophiloflavum]: AMP (Probabilities: 0.0690, 0.9178, 0.0132)\n","Processed WP_017237622.1 plantaricin C family lantibiotic [Streptomyces]: Neuropeptide (Probabilities: 0.3528, 0.0454, 0.6018)\n","Processed WP_090951383.1 gallidermin/nisin family lantibiotic [Parageobacillus thermantarcticus]: non-AMP/OTHERS (Probabilities: 0.6409, 0.3272, 0.0319)\n","Processed AGN34611.1 lichenicidin prepeptide LanA-like protein [Bacillus paralicheniformis ATCC 9945a ATCC 9945a]: non-AMP/OTHERS (Probabilities: 0.9164, 0.0127, 0.0709)\n","Processed WP_047426098.1 gallidermin/nisin family lantibiotic [Staphylococcus schweitzeri]: non-AMP/OTHERS (Probabilities: 0.9905, 0.0067, 0.0029)\n","Processed WP_094751000.1 gallidermin/nisin family lantibiotic [Apilactobacillus kunkeei]: AMP (Probabilities: 0.0684, 0.9167, 0.0149)\n","Processed WP_003459268.1 lichenicidin A2 family type 2 lantibiotic [Clostridium perfringens]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0065, 0.0028)\n","Processed ESA54865.1 lantibiotic salivaricin-A [Streptococcus pyogenes GA40468 GA40468]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0040, 0.0032)\n","Processed WP_047448451.1 gallidermin/nisin family lantibiotic [Staphylococcus schweitzeri]: non-AMP/OTHERS (Probabilities: 0.6044, 0.0334, 0.3622)\n","Processed WP_004456307.1 lichenicidin A2 family type 2 lantibiotic [Clostridium perfringens]: AMP (Probabilities: 0.0735, 0.9137, 0.0128)\n","Processed SUO59891.1 lantibiotic processing peptidase / lantibiotic transport permease / lantibiotic transport ATP-binding protein [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0034)\n","Processed WP_144989376.1 mersacidin family lantibiotic [Paenibacillus sp. N4]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0034)\n","Processed VHE61093.1 lantibiotic processing peptidase / lantibiotic transport permease / lantibiotic transport ATP-binding protein [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0065, 0.0026)\n","Processed WP_003955817.1 FxLD family lanthipeptide [Streptomyces clavuligerus]: Neuropeptide (Probabilities: 0.4631, 0.0218, 0.5152)\n","Processed WP_011889147.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9427, 0.0534, 0.0039)\n","Processed WP_010824505.1 class II lanthipeptide, LchA2/BrtA2 family [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0032)\n","Processed WP_073259420.1 gallidermin/nisin family lantibiotic [Streptococcus suis]: AMP (Probabilities: 0.1874, 0.7716, 0.0410)\n","Processed WP_044774497.1 plantaricin C family lantibiotic [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.7100, 0.2765, 0.0134)\n","Processed WP_044668600.1 lichenicidin A2 family type 2 lantibiotic [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0037, 0.0053)\n","Processed AGI88404.1 Lantibiotic protein [Streptomyces albidoflavus]: non-AMP/OTHERS (Probabilities: 0.9757, 0.0086, 0.0158)\n","Processed WP_024388713.1 plantaricin C family lantibiotic [Streptococcus suis]: Neuropeptide (Probabilities: 0.4152, 0.0631, 0.5216)\n","Processed WP_135019183.1 type A lantibiotic [Staphylococcus]: AMP (Probabilities: 0.0749, 0.9125, 0.0126)\n","Processed WP_023612118.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0030, 0.0037)\n","Processed WP_024405530.1 lichenicidin A2 family type 2 lantibiotic [Streptococcus suis]: AMP (Probabilities: 0.3680, 0.6225, 0.0096)\n","Processed WP_097981416.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9849, 0.0109, 0.0042)\n","Processed WP_111270220.1 plantaricin C family lantibiotic [Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0040, 0.0044)\n","Processed WP_164965719.1 hypothetical protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0073, 0.0028)\n","Processed WP_097845479.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.6850, 0.1491, 0.1659)\n","Processed WP_078207080.1 gallidermin/nisin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0039)\n","Processed WP_033669113.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0064, 0.0028)\n","Processed WP_021495220.1 plantaricin C family lantibiotic [Bacillus]: AMP (Probabilities: 0.0703, 0.9160, 0.0136)\n","Processed WP_000790339.1 mersacidin family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.1269, 0.8616, 0.0115)\n","Processed WP_097850306.1 plantaricin C family lantibiotic [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9437, 0.0134, 0.0429)\n","Processed WP_017658367.1 lichenicidin A2 family type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0045, 0.0049)\n","Processed WP_113050054.1 gallidermin/nisin family lantibiotic [Paenibacillus]: AMP (Probabilities: 0.2193, 0.7274, 0.0533)\n","Processed WP_069013017.1 gallidermin/nisin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9815, 0.0153, 0.0032)\n","Processed WP_026048345.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9702, 0.0261, 0.0037)\n","Processed WP_076431873.1 mersacidin/lichenicidin family type 2 lantibiotic [Microbispora]: non-AMP/OTHERS (Probabilities: 0.9896, 0.0052, 0.0052)\n","Processed WP_087440233.1 mersacidin family lantibiotic [Paenibacillus thiaminolyticus]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0040, 0.0058)\n","Processed WP_000790337.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0090, 0.0024)\n","Processed WP_017417595.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0029)\n","Processed WP_185551091.1 lichenicidin A2 family type 2 lantibiotic [Listeria innocua]: non-AMP/OTHERS (Probabilities: 0.9892, 0.0080, 0.0028)\n","Processed WP_000033704.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9859, 0.0095, 0.0046)\n","Processed WP_080561132.1 FDLD family class I lanthipeptide [Paenibacillus]: AMP (Probabilities: 0.0747, 0.9122, 0.0131)\n","Processed WP_050353280.1 plantaricin C family lantibiotic [Virgibacillus]: non-AMP/OTHERS (Probabilities: 0.9883, 0.0083, 0.0034)\n","Processed WP_125532690.1 plantaricin C family lantibiotic [Streptomyces sp. WAC 06725]: Neuropeptide (Probabilities: 0.2064, 0.0399, 0.7537)\n","Processed WP_185561830.1 lichenicidin A2 family type 2 lantibiotic [Listeria innocua]: AMP (Probabilities: 0.4690, 0.5217, 0.0094)\n","Processed WP_033685035.1 type A2 lantipeptide [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0043, 0.0035)\n","Processed WP_020998765.1 gallidermin/nisin family lantibiotic [Streptococcus intermedius]: AMP (Probabilities: 0.0850, 0.9025, 0.0125)\n","Processed WP_001798253.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0059, 0.0031)\n","Processed WP_000416759.1 gallidermin/nisin family lantibiotic [Staphylococcus]: Neuropeptide (Probabilities: 0.0201, 0.0129, 0.9670)\n","Processed WP_000271788.1 hypothetical protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0072, 0.0027)\n","Processed WP_000032577.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0071, 0.0039)\n","Processed WP_098250217.1 lichenicidin A2 family type 2 lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0064, 0.0032)\n","Processed WP_098250218.1 mersacidin family lantibiotic [Priestia megaterium]: AMP (Probabilities: 0.1568, 0.8311, 0.0121)\n","Processed WP_002917611.1 lantibiotic streptin [Streptococcus]: AMP (Probabilities: 0.0676, 0.9176, 0.0148)\n","Processed WP_123965962.1 plantaricin C family lantibiotic [Streptomyces sp. TLI_185]: Neuropeptide (Probabilities: 0.0652, 0.0229, 0.9119)\n","Processed WP_087929258.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0047, 0.0041)\n","Processed WP_000356689.1 type A lantibiotic [Bacillus]: AMP (Probabilities: 0.1460, 0.8402, 0.0137)\n","Processed WP_080684896.1 gallidermin/nisin family lantibiotic [Bifidobacterium]: AMP (Probabilities: 0.1189, 0.8564, 0.0247)\n","Processed WP_094041300.1 mersacidin family lantibiotic [Bacillaceae]: AMP (Probabilities: 0.0935, 0.8930, 0.0135)\n","Processed WP_094041302.1 mersacidin family lantibiotic [Bacillaceae]: AMP (Probabilities: 0.2432, 0.7387, 0.0182)\n","Processed WP_025580957.1 plantaricin C family lantibiotic [Eubacteriales]: AMP (Probabilities: 0.1352, 0.8305, 0.0343)\n","Processed WP_012578505.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium longum]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0037, 0.0035)\n","Processed WP_040060799.1 mersacidin family lantibiotic [Bacillaceae]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0040)\n","Processed WP_023016180.1 mersacidin family lantibiotic [Staphylococcus simulans]: non-AMP/OTHERS (Probabilities: 0.4952, 0.4938, 0.0109)\n","Processed WP_107368141.1 mersacidin family lantibiotic [Staphylococcus chromogenes]: Neuropeptide (Probabilities: 0.3306, 0.0309, 0.6385)\n","Processed WP_024343838.1 gallidermin/nisin family lantibiotic [Streptococcus equinus]: Neuropeptide (Probabilities: 0.3379, 0.1440, 0.5181)\n","Processed WP_118095861.1 lichenicidin A2 family type 2 lantibiotic [Streptococcus parasanguinis]: AMP (Probabilities: 0.1144, 0.8734, 0.0122)\n","Processed WP_118095860.1 lichenicidin A2 family type 2 lantibiotic [Streptococcus parasanguinis]: AMP (Probabilities: 0.1365, 0.8522, 0.0113)\n","Processed WP_024408876.1 plantaricin C family lantibiotic [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9777, 0.0182, 0.0041)\n","Processed WP_076514039.1 mersacidin family lantibiotic [Priestia flexa]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0041, 0.0034)\n","Processed WP_073304601.1 gallidermin/nisin family lantibiotic [Alkalihalobacillus]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0059, 0.0027)\n","Processed WP_023016179.1 mersacidin family lantibiotic [Staphylococcus simulans]: AMP (Probabilities: 0.2323, 0.7561, 0.0116)\n","Processed WP_077322205.1 plantaricin C family lantibiotic [Streptococcus pseudoporcinus]: non-AMP/OTHERS (Probabilities: 0.6820, 0.0960, 0.2220)\n","Processed WP_013373086.1 mersacidin family lantibiotic [Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0034, 0.0032)\n","Processed WP_069995862.1 class II lanthipeptide, LchA2/BrtA2 family [Staphylococcus caeli]: non-AMP/OTHERS (Probabilities: 0.9357, 0.0194, 0.0449)\n","Processed WP_012775117.1 type A2 lantipeptide [Streptococcus]: AMP (Probabilities: 0.0721, 0.9149, 0.0130)\n","Processed WP_113937374.1 type A lantibiotic [Bacillus]: AMP (Probabilities: 0.0736, 0.9133, 0.0131)\n","Processed WP_117040689.1 gallidermin family lantibiotic [Klebsiella pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0041, 0.0050)\n","Processed WP_048520746.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0037, 0.0048)\n","Processed WP_113937375.1 type A lantibiotic [Bacillus sp. DB-2]: AMP (Probabilities: 0.1897, 0.7998, 0.0105)\n","Processed WP_181537221.1 mersacidin family lantibiotic [Anoxybacillus calidus]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0035)\n","Processed WP_088718128.1 mutacin-like lantipeptide [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0034, 0.0043)\n","Processed WP_111152704.1 mersacidin family lantibiotic [Paenibacillus dendritiformis]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0044, 0.0032)\n","Processed WP_181537222.1 mersacidin family lantibiotic [Anoxybacillus calidus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0037, 0.0041)\n","Processed WP_181537223.1 mersacidin family lantibiotic [Anoxybacillus calidus]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0036)\n","Processed WP_111152706.1 lichenicidin A2 family type 2 lantibiotic [Paenibacillus dendritiformis]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0063, 0.0030)\n","Processed WP_094938128.1 mersacidin family lantibiotic [Paenibacillus]: AMP (Probabilities: 0.2555, 0.7339, 0.0106)\n","Processed WP_074562140.1 mersacidin family lantibiotic [Bacillus]: AMP (Probabilities: 0.0738, 0.9124, 0.0138)\n","Processed WP_091399741.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora]: AMP (Probabilities: 0.1114, 0.8679, 0.0207)\n","Processed WP_091399738.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora]: non-AMP/OTHERS (Probabilities: 0.4918, 0.3426, 0.1656)\n","Processed WP_111247033.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus plantarum]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0037, 0.0060)\n","Processed WP_060674405.1 plantaricin C family lantibiotic [Bacillus]: Neuropeptide (Probabilities: 0.0111, 0.0106, 0.9784)\n","Processed WP_218011708.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9248, 0.0703, 0.0049)\n","Processed WP_218924004.1 mersacidin family lantibiotic, partial [Bacillus cereus]: AMP (Probabilities: 0.1249, 0.8634, 0.0117)\n","Processed WP_218924629.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9893, 0.0077, 0.0030)\n","Processed WP_218652305.1 mersacidin family lantibiotic, partial [Bacillus paramycoides]: non-AMP/OTHERS (Probabilities: 0.7967, 0.1533, 0.0499)\n","Processed WP_217695698.1 mersacidin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9777, 0.0192, 0.0031)\n","Processed WP_217992723.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0032, 0.0049)\n","Processed WP_217992724.1 mersacidin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9123, 0.0261, 0.0616)\n","Processed WP_218922882.1 mersacidin family lantibiotic, partial [Bacillus cereus]: AMP (Probabilities: 0.1232, 0.8646, 0.0122)\n","Processed WP_013450107.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.7619, 0.2301, 0.0080)\n","Processed WP_044276571.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus]: Neuropeptide (Probabilities: 0.0148, 0.0125, 0.9727)\n","Processed WP_215561387.1 type A lantibiotic [Bacillus mycoides]: AMP (Probabilities: 0.0690, 0.9171, 0.0139)\n","Processed WP_217700541.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.5630, 0.4235, 0.0135)\n","Processed WP_011552917.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus]: Neuropeptide (Probabilities: 0.2098, 0.0639, 0.7264)\n","Processed WP_215561388.1 type A lantibiotic [Bacillus mycoides]: AMP (Probabilities: 0.2921, 0.6980, 0.0100)\n","Processed WP_008783414.1 lichenicidin A2 family type 2 lantibiotic [Terrabacteria group]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed WP_013309465.1 gallidermin family lantibiotic [Paenibacillus]: AMP (Probabilities: 0.0745, 0.9127, 0.0129)\n","Processed WP_217994269.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9832, 0.0088, 0.0080)\n","Processed WP_215561389.1 type A lantibiotic [Bacillus mycoides]: AMP (Probabilities: 0.0691, 0.9174, 0.0135)\n","Processed WP_011552916.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus]: non-AMP/OTHERS (Probabilities: 0.5745, 0.0358, 0.3898)\n","Processed WP_105407464.1 plantaricin C family lantibiotic [Paenibacillus sp. PCH8]: non-AMP/OTHERS (Probabilities: 0.9834, 0.0113, 0.0054)\n","Processed WP_140461879.1 mersacidin family lantibiotic [Bacillaceae]: AMP (Probabilities: 0.1165, 0.8717, 0.0119)\n","Processed WP_204956245.1 mersacidin family lantibiotic [Metabacillus crassostreae]: non-AMP/OTHERS (Probabilities: 0.9838, 0.0119, 0.0042)\n","Processed WP_204956246.1 mersacidin family lantibiotic [Metabacillus crassostreae]: non-AMP/OTHERS (Probabilities: 0.9863, 0.0098, 0.0040)\n","Processed WP_204956247.1 mersacidin family lantibiotic [Metabacillus crassostreae]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0054, 0.0044)\n","Processed WP_204730760.1 mersacidin family lantibiotic [Jeotgalibacillus terrae]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0071, 0.0031)\n","Processed WP_194380275.1 gallidermin family lantibiotic [Staphylococcus capitis]: non-AMP/OTHERS (Probabilities: 0.9845, 0.0044, 0.0111)\n","Processed WP_104661400.1 lichenicidin alpha family lanthipeptide [Enterococcus]: non-AMP/OTHERS (Probabilities: 0.5668, 0.0264, 0.4068)\n","Processed WP_204956244.1 mersacidin family lantibiotic [Metabacillus crassostreae]: non-AMP/OTHERS (Probabilities: 0.9848, 0.0106, 0.0046)\n","Processed WP_196302624.1 lantibiotic dehydratase, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0041, 0.0030)\n","Processed WP_196302629.1 lantibiotic dehydratase, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0039, 0.0038)\n","Processed WP_191701775.1 mersacidin family lantibiotic [Solibacillus sp. A46]: non-AMP/OTHERS (Probabilities: 0.9418, 0.0548, 0.0034)\n","Processed WP_104255168.1 plantaricin C family lantibiotic [unclassified Rathayibacter]: AMP (Probabilities: 0.0784, 0.9092, 0.0124)\n","Processed WP_191714744.1 mersacidin family lantibiotic [Planococcaceae]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0034, 0.0051)\n","Processed WP_191701774.1 mersacidin family lantibiotic [Solibacillus sp. A46]: non-AMP/OTHERS (Probabilities: 0.8898, 0.0190, 0.0912)\n","Processed WP_194786482.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.2396, 0.7489, 0.0115)\n","Processed WP_178941437.1 mersacidin family lantibiotic, partial [Bacillus paramycoides]: AMP (Probabilities: 0.4039, 0.5864, 0.0097)\n","Processed WP_000429839.1 gallidermin/nisin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9798, 0.0095, 0.0108)\n","Processed WP_191714745.1 mersacidin family lantibiotic [Planococcaceae]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0060, 0.0029)\n","Processed WP_175369819.1 mersacidin family lantibiotic [Paenibacillus sp. JW14]: non-AMP/OTHERS (Probabilities: 0.9840, 0.0042, 0.0117)\n","Processed NRT74868.1 type 2 lantibiotic (TIGR03893 family) [Clostridium beijerinckii]: AMP (Probabilities: 0.2055, 0.7820, 0.0125)\n","Processed WP_030075698.1 plantaricin C family lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9339, 0.0147, 0.0515)\n","Processed WP_000429840.1 gallidermin/nisin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9573, 0.0223, 0.0205)\n","Processed NRT74867.1 type 2 lantibiotic (TIGR03893 family) [Clostridium beijerinckii]: non-AMP/OTHERS (Probabilities: 0.9893, 0.0078, 0.0029)\n","Processed WP_175369820.1 mersacidin family lantibiotic [Paenibacillus sp. JW14]: non-AMP/OTHERS (Probabilities: 0.5036, 0.4866, 0.0098)\n","Processed WP_153592933.1 mersacidin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.5249, 0.4651, 0.0099)\n","Processed WP_148992457.1 mersacidin family lantibiotic [Rossellomorea aquimaris]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0038, 0.0028)\n","Processed WP_148992456.1 mersacidin family lantibiotic [Rossellomorea aquimaris]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0035, 0.0031)\n","Processed WP_190316885.1 lantibiotic dehydratase, partial [Pseudomonas aeruginosa]: non-AMP/OTHERS (Probabilities: 0.9871, 0.0079, 0.0050)\n","Processed WP_172295955.1 mersacidin family lantibiotic [Bacillus sp. WMMC1349]: non-AMP/OTHERS (Probabilities: 0.9935, 0.0036, 0.0029)\n","Processed WP_168455240.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0056, 0.0027)\n","Processed WP_144167514.1 mersacidin family lantibiotic [Eubacteriales]: AMP (Probabilities: 0.0897, 0.8983, 0.0121)\n","Processed WP_137077046.1 mersacidin family lantibiotic [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0047, 0.0068)\n","Processed WP_151525233.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9867, 0.0102, 0.0032)\n","Processed WP_137077338.1 mersacidin family lantibiotic, partial [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9787, 0.0101, 0.0112)\n","Processed WP_137077047.1 mersacidin family lantibiotic [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9892, 0.0058, 0.0050)\n","Processed WP_133377366.1 mersacidin family lantibiotic [Jeotgalibacillus sp. S-D1]: non-AMP/OTHERS (Probabilities: 0.9892, 0.0074, 0.0034)\n","Processed WP_130547104.1 gallidermin family lantibiotic [Staphylococcus saccharolyticus]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0036)\n","Processed WP_135782397.1 mersacidin family lantibiotic [Streptococcus rubneri]: non-AMP/OTHERS (Probabilities: 0.8872, 0.1041, 0.0087)\n","Processed WP_120028178.1 mersacidin family lantibiotic [Bacillus subtilis]: AMP (Probabilities: 0.1191, 0.8689, 0.0120)\n","Processed WP_115871133.1 mersacidin family lantibiotic [Staphylococcus felis]: AMP (Probabilities: 0.2527, 0.7351, 0.0122)\n","Processed WP_115871615.1 gallidermin family lantibiotic [Staphylococcus felis]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0061, 0.0049)\n","Processed WP_143802894.1 mersacidin family lantibiotic [Paenibacillus thiaminolyticus]: AMP (Probabilities: 0.1052, 0.8831, 0.0117)\n","Processed WP_115871132.1 mersacidin family lantibiotic [Staphylococcus felis]: non-AMP/OTHERS (Probabilities: 0.7465, 0.2452, 0.0083)\n","Processed WP_133377367.1 mersacidin family lantibiotic [Jeotgalibacillus sp. S-D1]: Neuropeptide (Probabilities: 0.1163, 0.0347, 0.8490)\n","Processed WP_113715809.1 mersacidin family lantibiotic [Priestia endophytica]: AMP (Probabilities: 0.0947, 0.8929, 0.0124)\n","Processed WP_116078335.1 mersacidin family lantibiotic [Priestia megaterium]: AMP (Probabilities: 0.2257, 0.7622, 0.0121)\n","Processed WP_107516539.1 gallidermin family lantibiotic, partial [Staphylococcus epidermidis]: non-AMP/OTHERS (Probabilities: 0.8613, 0.1335, 0.0052)\n","Processed WP_106383562.1 mersacidin family lantibiotic [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9871, 0.0068, 0.0061)\n","Processed WP_106383563.1 mersacidin family lantibiotic [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9763, 0.0082, 0.0156)\n","Processed WP_106383558.1 mersacidin family lantibiotic [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0037, 0.0032)\n","Processed WP_101485617.1 gallidermin family lantibiotic [Actinomyces sp. UMB0138]: Neuropeptide (Probabilities: 0.2131, 0.1407, 0.6462)\n","Processed WP_103050334.1 mersacidin family lantibiotic [Paenibacillus sp. F4]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0039, 0.0044)\n","Processed WP_157804272.1 mersacidin family lantibiotic, partial [Lysinibacillus xylanilyticus]: AMP (Probabilities: 0.2681, 0.7197, 0.0121)\n","Processed WP_100772596.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9773, 0.0172, 0.0055)\n","Processed WP_098858778.1 mersacidin family lantibiotic [Bacillus sp. AFS054943]: non-AMP/OTHERS (Probabilities: 0.9871, 0.0090, 0.0040)\n","Processed WP_098858917.1 mersacidin family lantibiotic, partial [Bacillus sp. AFS054943]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0041, 0.0029)\n","Processed WP_141549500.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.2222, 0.7655, 0.0123)\n","Processed WP_141549521.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9080, 0.0874, 0.0046)\n","Processed WP_100616757.1 mersacidin family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.2392, 0.7335, 0.0273)\n","Processed WP_100616759.1 mersacidin family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.0885, 0.8996, 0.0119)\n","Processed WP_100544957.1 mersacidin family lantibiotic [Lysinibacillus xylanilyticus]: non-AMP/OTHERS (Probabilities: 0.9697, 0.0229, 0.0074)\n","Processed WP_141544246.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.8071, 0.1871, 0.0058)\n","Processed WP_098603249.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.7309, 0.1337, 0.1355)\n","Processed WP_098433818.1 mersacidin family lantibiotic [Priestia megaterium]: AMP (Probabilities: 0.0801, 0.9056, 0.0142)\n","Processed WP_098315639.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0042, 0.0038)\n","Processed WP_098434429.1 mersacidin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.3889, 0.2512, 0.3600)\n","Processed WP_098546696.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.4060, 0.5833, 0.0107)\n","Processed WP_098400124.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9785, 0.0176, 0.0039)\n","Processed WP_142321486.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.6324, 0.3558, 0.0119)\n","Processed WP_098589521.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9787, 0.0181, 0.0032)\n","Processed WP_098322665.1 mersacidin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9639, 0.0245, 0.0116)\n","Processed WP_098353557.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9179, 0.0499, 0.0322)\n","Processed WP_142322364.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9630, 0.0335, 0.0035)\n","Processed WP_098377703.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.6436, 0.2704, 0.0860)\n","Processed WP_141529804.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.3735, 0.6074, 0.0191)\n","Processed WP_141527744.1 mersacidin family lantibiotic, partial [Bacillus sp. AFS012607]: non-AMP/OTHERS (Probabilities: 0.7622, 0.2247, 0.0131)\n","Processed WP_098271355.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0035, 0.0031)\n","Processed WP_098353559.1 mersacidin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0052, 0.0026)\n","Processed WP_097888491.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9269, 0.0687, 0.0044)\n","Processed WP_098271354.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0048, 0.0042)\n","Processed WP_097997863.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0040, 0.0028)\n","Processed WP_097999697.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9272, 0.0248, 0.0480)\n","Processed WP_097845728.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.6881, 0.0194, 0.2925)\n","Processed WP_141520739.1 lichenicidin alpha family lanthipeptide, partial [Bacillus]: Neuropeptide (Probabilities: 0.1202, 0.0303, 0.8495)\n","Processed WP_097999698.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.6948, 0.0258, 0.2794)\n","Processed WP_097821240.1 mersacidin family lantibiotic [Bacillus toyonensis]: AMP (Probabilities: 0.1455, 0.8356, 0.0189)\n","Processed WP_095325772.1 gallidermin family lantibiotic [Staphylococcus capitis]: non-AMP/OTHERS (Probabilities: 0.9749, 0.0056, 0.0196)\n","Processed WP_088310180.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0068, 0.0046)\n","Processed WP_089200865.1 mersacidin family lantibiotic [Paenibacillus xerothermodurans]: non-AMP/OTHERS (Probabilities: 0.9297, 0.0415, 0.0288)\n","Processed WP_088309981.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0060, 0.0040)\n","Processed WP_097821239.1 mersacidin family lantibiotic [Bacillus toyonensis]: AMP (Probabilities: 0.0800, 0.9078, 0.0122)\n","Processed WP_098033454.1 gallidermin family lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0675, 0.9187, 0.0138)\n","Processed WP_089200864.1 mersacidin family lantibiotic [Paenibacillus xerothermodurans]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0084, 0.0025)\n","Processed WP_088303392.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9802, 0.0165, 0.0033)\n","Processed WP_087966624.1 mersacidin family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9854, 0.0105, 0.0041)\n","Processed WP_097842226.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.7740, 0.2177, 0.0084)\n","Processed WP_088045763.1 mersacidin family lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.1949, 0.7929, 0.0122)\n","Processed WP_088045759.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0049, 0.0036)\n","Processed WP_087944968.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9883, 0.0034, 0.0082)\n","Processed WP_087966626.1 mersacidin family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9935, 0.0034, 0.0031)\n","Processed WP_080351401.1 mersacidin family lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9474, 0.0489, 0.0037)\n","Processed WP_082147438.1 lichenicidin A2 family type 2 lantibiotic [Gemella sanguinis]: AMP (Probabilities: 0.1717, 0.8147, 0.0136)\n","Processed WP_070831962.1 gallidermin family lantibiotic [Staphylococcus sp. HMSC036D05]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed WP_062688910.1 gallidermin family lantibiotic [Lactiplantibacillus plantarum]: non-AMP/OTHERS (Probabilities: 0.7356, 0.0218, 0.2425)\n","Processed WP_143564843.1 gallidermin family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0041, 0.0039)\n","Processed WP_055991772.1 mersacidin family lantibiotic [Bacillus sp. Leaf75]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0032, 0.0040)\n","Processed WP_068779519.1 mersacidin family lantibiotic [unclassified Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.7802, 0.2135, 0.0063)\n","Processed WP_088045715.1 mersacidin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0079, 0.0026)\n","Processed WP_055991766.1 mersacidin family lantibiotic [Bacillaceae]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0035, 0.0043)\n","Processed WP_055991769.1 mersacidin family lantibiotic [Bacillus sp. Leaf75]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0039, 0.0037)\n","Processed WP_050845638.1 mersacidin family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0041, 0.0048)\n","Processed WP_068779678.1 mersacidin family lantibiotic [Paenibacillus sp. GM2]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0039, 0.0051)\n","Processed WP_048548711.1 mersacidin family lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0889, 0.8994, 0.0117)\n","Processed WP_063209861.1 gallidermin family lantibiotic [Paenibacillus]: AMP (Probabilities: 0.0988, 0.8892, 0.0121)\n","Processed WP_048548713.1 mersacidin family lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.1840, 0.7937, 0.0223)\n","Processed WP_074619573.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0054, 0.0045)\n","Processed WP_043981766.1 gallidermin family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0036, 0.0045)\n","Processed WP_055991775.1 mersacidin family lantibiotic [Bacillaceae]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0040, 0.0031)\n","Processed WP_039269802.1 gallidermin family lantibiotic [Paenibacillus polymyxa]: AMP (Probabilities: 0.0742, 0.9129, 0.0129)\n","Processed WP_025723957.1 gallidermin family lantibiotic [Paenibacillus]: AMP (Probabilities: 0.0701, 0.9167, 0.0132)\n","Processed WP_016078396.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0032, 0.0036)\n","Processed WP_002192306.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0034, 0.0048)\n","Processed WP_043938505.1 mersacidin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0035, 0.0032)\n","Processed WP_002192304.1 mersacidin family lantibiotic, partial [Bacillus cereus]: AMP (Probabilities: 0.0688, 0.9171, 0.0141)\n","Processed WP_000790338.1 mersacidin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.7468, 0.2464, 0.0067)\n","Processed WP_050427133.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0032, 0.0047)\n","Processed WP_043938504.1 mersacidin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0039, 0.0046)\n","Processed WP_000790340.1 mersacidin family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.1662, 0.8217, 0.0121)\n","Processed WP_002182160.1 mersacidin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0031, 0.0052)\n","Processed WP_002182159.1 mersacidin family lantibiotic, partial [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9765, 0.0190, 0.0045)\n","Processed WP_002170361.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.5382, 0.4514, 0.0103)\n","Processed WP_000790335.1 mersacidin family lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0732, 0.9142, 0.0126)\n","Processed WP_219574878.1 lantibiotic (srt) production protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0055, 0.0027)\n","Processed WP_002192303.1 mersacidin family lantibiotic, partial [Bacillus cereus]: Neuropeptide (Probabilities: 0.0147, 0.0117, 0.9736)\n","Processed WP_004167060.1 plantaricin C family lantibiotic [Peptoniphilaceae]: non-AMP/OTHERS (Probabilities: 0.4886, 0.0446, 0.4668)\n","Processed WP_033692916.1 lichenicidin alpha family lanthipeptide [Terrabacteria group]: non-AMP/OTHERS (Probabilities: 0.6938, 0.0222, 0.2840)\n","Processed WP_130118375.1 gallidermin/nisin family lantibiotic, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9868, 0.0069, 0.0063)\n","Processed WP_000790336.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.6761, 0.3159, 0.0080)\n","Processed WP_047437384.1 gallidermin/nisin family lantibiotic [Staphylococcus argenteus]: Neuropeptide (Probabilities: 0.2218, 0.0248, 0.7534)\n","Processed WP_139183657.1 lantibiotic dehydratase [Algoriphagus alkaliphilus]: non-AMP/OTHERS (Probabilities: 0.9418, 0.0125, 0.0456)\n","Processed WP_010896632.1 plantaricin C family lantibiotic [Alkalihalobacillus]: non-AMP/OTHERS (Probabilities: 0.5536, 0.0457, 0.4007)\n","Processed EJH02380.1 hypothetical protein [Streptococcus pneumoniae GA56348 GA56348]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0039, 0.0029)\n","Processed WP_099000710.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9770, 0.0066, 0.0165)\n","Processed WP_098425099.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9516, 0.0155, 0.0329)\n","Processed WP_080035512.1 type A2 lantipeptide [Streptococcus mutans]: AMP (Probabilities: 0.0736, 0.9138, 0.0125)\n","Processed WP_098764200.1 plantaricin C family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9825, 0.0071, 0.0104)\n","Processed WP_005522761.1 lichenicidin alpha family lanthipeptide [Corynebacterium matruchotii]: Neuropeptide (Probabilities: 0.0084, 0.0092, 0.9824)\n","Processed WP_098018805.1 plantaricin C family lantibiotic [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9881, 0.0062, 0.0057)\n","Processed WP_098777887.1 type A lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0689, 0.9173, 0.0138)\n","Processed WP_098690898.1 plantaricin C family lantibiotic, partial [Bacillus toyonensis]: Neuropeptide (Probabilities: 0.1985, 0.0272, 0.7743)\n","Processed WP_010896631.1 class II lanthipeptide, LchA2/BrtA2 family [Alkalihalobacillus]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0063, 0.0031)\n","Processed WP_098598719.1 gallidermin/nisin family lantibiotic [Priestia megaterium]: AMP (Probabilities: 0.1938, 0.7888, 0.0174)\n","Processed WP_060489082.1 type A2 lantipeptide [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0035)\n","Processed WP_097787302.1 plantaricin C family lantibiotic [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9444, 0.0152, 0.0404)\n","Processed PFJ19200.1 lantibiotic paenibacillin [Bacillus cereus]: AMP (Probabilities: 0.0816, 0.9063, 0.0122)\n","Processed WP_098437008.1 type A lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0754, 0.9116, 0.0130)\n","Processed WP_141530342.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9788, 0.0117, 0.0096)\n","Processed WP_098877058.1 type A lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0721, 0.9151, 0.0127)\n","Processed WP_098023387.1 plantaricin C family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8866, 0.0934, 0.0200)\n","Processed WP_098437849.1 plantaricin C family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.6101, 0.3564, 0.0335)\n","Processed WP_097797583.1 type A lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.0740, 0.9134, 0.0126)\n","Processed WP_098318779.1 plantaricin C family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9853, 0.0075, 0.0072)\n","Processed WP_128267342.1 type A lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0827, 0.9044, 0.0130)\n","Processed WP_098292646.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9477, 0.0268, 0.0255)\n","Processed WP_098488025.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9649, 0.0247, 0.0103)\n","Processed WP_098293299.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.8840, 0.0160, 0.1001)\n","Processed WP_043354960.1 type A lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0962, 0.8918, 0.0120)\n","Processed WP_098218277.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9193, 0.0115, 0.0692)\n","Processed WP_098288436.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9092, 0.0135, 0.0773)\n","Processed WP_141533902.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9658, 0.0096, 0.0245)\n","Processed WP_098139887.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9078, 0.0235, 0.0687)\n","Processed WP_097913742.1 lichenicidin A2 family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.8373, 0.0132, 0.1495)\n","Processed WP_142318766.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0050, 0.0035)\n","Processed WP_006096541.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.5500, 0.1448, 0.3052)\n","Processed WP_098138748.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9865, 0.0061, 0.0074)\n","Processed WP_095980838.1 mersacidin/lichenicidin family type 2 lantibiotic [Melittangium boletus]: non-AMP/OTHERS (Probabilities: 0.9196, 0.0504, 0.0299)\n","Processed WP_078994195.1 lichenicidin A2 family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9761, 0.0098, 0.0141)\n","Processed WP_078994196.1 lichenicidin A2 family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.4952, 0.0265, 0.4783)\n","Processed WP_000041628.1 class II lanthipeptide, LchA2/BrtA2 family [Terrabacteria group]: non-AMP/OTHERS (Probabilities: 0.9822, 0.0150, 0.0028)\n","Processed WP_036592998.1 plantaricin C family lantibiotic [Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0049, 0.0031)\n","Processed WP_098596258.1 type A lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0749, 0.9121, 0.0130)\n","Processed WP_142316653.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9790, 0.0058, 0.0152)\n","Processed WP_098886147.1 type A lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0721, 0.9147, 0.0132)\n","Processed WP_086398127.1 type A lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0678, 0.9182, 0.0140)\n","Processed WP_098300520.1 type A lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0724, 0.9143, 0.0133)\n","Processed WP_080117353.1 type A lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0759, 0.9118, 0.0124)\n","Processed WP_024405531.1 lichenicidin A2 family type 2 lantibiotic [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9880, 0.0090, 0.0030)\n","Processed WP_016083135.1 type A lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.1679, 0.8212, 0.0109)\n","Processed WP_016084570.1 type A lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0698, 0.9166, 0.0136)\n","Processed WP_097891850.1 type A lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0719, 0.9149, 0.0132)\n","Processed WP_037580734.1 type A2 lantipeptide [Streptococcus equi]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0039, 0.0045)\n","Processed WP_218757245.1 plantaricin C family lantibiotic [Streptococcus mitis]: non-AMP/OTHERS (Probabilities: 0.6010, 0.0434, 0.3556)\n","Processed EGS1166385.1 type 2 lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9894, 0.0033, 0.0073)\n","Processed WP_077996071.1 lantibiotic dehydratase [Paenibacillus larvae]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0067, 0.0045)\n","Processed WP_009345567.1 lantibiotic dehydratase [Peptoniphilus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0050, 0.0029)\n","Processed WP_148712530.1 plantaricin C family lantibiotic [Enterococcus avium]: non-AMP/OTHERS (Probabilities: 0.7939, 0.0635, 0.1426)\n","Processed WP_072973636.1 plantaricin C family lantibiotic [Tissierella praeacuta]: non-AMP/OTHERS (Probabilities: 0.7894, 0.0360, 0.1746)\n","Processed WP_178940990.1 lichenicidin A2 family type 2 lantibiotic, partial [Bacillus paramycoides]: non-AMP/OTHERS (Probabilities: 0.6151, 0.3770, 0.0078)\n","Processed WP_001289699.1 thiocillin/thiostrepton family thiazolyl peptide [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0043, 0.0038)\n","Processed WP_115347074.1 gallidermin/nisin family lantibiotic [Nocardia brasiliensis]: non-AMP/OTHERS (Probabilities: 0.9461, 0.0480, 0.0059)\n","Processed WP_000160316.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9866, 0.0090, 0.0044)\n","Processed WP_001118874.1 type A2 lantipeptide [Streptococcus mitis]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0038, 0.0037)\n","Processed WP_093792134.1 plantaricin C family lantibiotic [Streptomyces sp. cf124]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0054, 0.0032)\n","Processed WP_143035056.1 gallidermin/nisin family lantibiotic [Marininema mesophilum]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0036, 0.0055)\n","Processed WP_055501949.1 mersacidin/lichenicidin family type 2 lantibiotic [Nonomuraea pusilla]: non-AMP/OTHERS (Probabilities: 0.9564, 0.0065, 0.0371)\n","Processed WP_091738768.1 gallidermin/nisin family lantibiotic [Marininema mesophilum]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0055, 0.0028)\n","Processed WP_091040467.1 mersacidin/lichenicidin family type 2 lantibiotic [Glycomyces harbinensis]: Neuropeptide (Probabilities: 0.3612, 0.1035, 0.5354)\n","Processed WP_091040466.1 mersacidin/lichenicidin family type 2 lantibiotic [Glycomyces harbinensis]: AMP (Probabilities: 0.1526, 0.7800, 0.0674)\n","Processed WP_058265998.1 lichenicidin A2 family type 2 lantibiotic [Exiguobacterium enclense]: non-AMP/OTHERS (Probabilities: 0.9887, 0.0047, 0.0066)\n","Processed WP_048517722.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9845, 0.0096, 0.0059)\n","Processed WP_072526965.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.5359, 0.4490, 0.0151)\n","Processed WP_091094588.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora citrea]: non-AMP/OTHERS (Probabilities: 0.3962, 0.2410, 0.3628)\n","Processed WP_030881377.1 FxLD family lanthipeptide [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9474, 0.0084, 0.0443)\n","Processed WP_091633868.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora peucetia]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0044, 0.0057)\n","Processed WP_126639118.1 plantaricin C family lantibiotic [Embleya hyalina]: Neuropeptide (Probabilities: 0.4641, 0.0431, 0.4928)\n","Processed WP_141549493.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0056, 0.0039)\n","Processed WP_142952917.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.8581, 0.0241, 0.1178)\n","Processed WP_093830922.1 plantaricin C family lantibiotic [unclassified Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9180, 0.0111, 0.0709)\n","Processed WP_065957033.1 plantaricin C family lantibiotic [Streptococcus dysgalactiae]: Neuropeptide (Probabilities: 0.0200, 0.0156, 0.9643)\n","Processed WP_142953167.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.6166, 0.0264, 0.3570)\n","Processed WP_098033722.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.8576, 0.0166, 0.1258)\n","Processed WP_098361625.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.8747, 0.0164, 0.1089)\n","Processed WP_098472342.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9864, 0.0076, 0.0060)\n","Processed WP_098361590.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9812, 0.0071, 0.0117)\n","Processed WP_095690008.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.3039, 0.6830, 0.0132)\n","Processed WP_002103816.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9883, 0.0076, 0.0041)\n","Processed WP_097995638.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9905, 0.0056, 0.0039)\n","Processed WP_098327068.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9683, 0.0083, 0.0233)\n","Processed WP_142952809.1 lichenicidin A2 family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8693, 0.1260, 0.0047)\n","Processed WP_098033951.1 plantaricin C family lantibiotic, partial [Bacillus thuringiensis]: Neuropeptide (Probabilities: 0.2110, 0.0310, 0.7580)\n","Processed WP_086444718.1 plantaricin C family lantibiotic [Enterococcus sp. 12C11_DIV0727]: Neuropeptide (Probabilities: 0.0369, 0.0228, 0.9402)\n","Processed WP_029443584.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9842, 0.0110, 0.0047)\n","Processed WP_000033705.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9868, 0.0093, 0.0039)\n","Processed WP_093230868.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermoflavimicrobium dichotomicum]: Neuropeptide (Probabilities: 0.3921, 0.0410, 0.5669)\n","Processed WP_093230867.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermoflavimicrobium dichotomicum]: Neuropeptide (Probabilities: 0.0906, 0.0336, 0.8758)\n","Processed WP_079936422.1 gallidermin/nisin family lantibiotic [Geobacillus sp. LEMMY01]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0040, 0.0037)\n","Processed WP_091562712.1 mersacidin/lichenicidin family type 2 lantibiotic [Arthrobacter sp. ok362]: non-AMP/OTHERS (Probabilities: 0.9198, 0.0264, 0.0538)\n","Processed WP_074466923.1 gallidermin/nisin family lantibiotic [Streptomyces sp. WMMB 322]: non-AMP/OTHERS (Probabilities: 0.9878, 0.0085, 0.0038)\n","Processed WP_069329679.1 gallidermin/nisin family lantibiotic [Paenibacillus nuruki]: AMP (Probabilities: 0.2965, 0.6759, 0.0276)\n","Processed WP_098488434.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0042, 0.0040)\n","Processed WP_091839400.1 gallidermin/nisin family lantibiotic [Marininema halotolerans]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0033)\n","Processed WP_098375653.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.1648, 0.8216, 0.0137)\n","Processed WP_002396811.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus faecalis]: AMP (Probabilities: 0.1682, 0.8204, 0.0114)\n","Processed WP_141549522.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9043, 0.0377, 0.0580)\n","Processed WP_086396028.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: Neuropeptide (Probabilities: 0.0223, 0.0157, 0.9620)\n","Processed WP_175063858.1 gallidermin/nisin family lantibiotic [Streptococcus salivarius]: AMP (Probabilities: 0.1081, 0.8789, 0.0130)\n","Processed WP_100353943.1 plantaricin C family lantibiotic [Chryseomicrobium excrementi]: non-AMP/OTHERS (Probabilities: 0.9650, 0.0073, 0.0278)\n","Processed WP_141727352.1 lantibiotic paenibacillin [Staphylococcus xylosus]: non-AMP/OTHERS (Probabilities: 0.9794, 0.0176, 0.0030)\n","Processed WP_111743423.1 lichenicidin A2 family type 2 lantibiotic [Gemella morbillorum]: AMP (Probabilities: 0.1298, 0.8569, 0.0132)\n","Processed WP_054975888.1 elgicin/penisin family lantibiotic [unclassified Paenibacillus]: AMP (Probabilities: 0.0876, 0.8987, 0.0137)\n","Processed WP_082248051.1 gallidermin/nisin family lantibiotic [Bifidobacterium longum]: non-AMP/OTHERS (Probabilities: 0.6761, 0.2866, 0.0373)\n","Processed WP_160468797.1 lantibiotic michiganin-A [Clavibacter michiganensis]: AMP (Probabilities: 0.4102, 0.4684, 0.1214)\n","Processed KMR21201.1 lantibiotic ABC transporter permease, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0078, 0.0027)\n","Processed MBU7114174.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0063, 0.0034)\n","Processed WP_077996054.1 mersacidin/lichenicidin family type 2 lantibiotic [Paenibacillus larvae]: non-AMP/OTHERS (Probabilities: 0.5804, 0.0795, 0.3401)\n","Processed WP_216558237.1 lichenicidin A2 family type 2 lantibiotic [Roseburia sp. MSJ-14]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0053, 0.0033)\n","Processed WP_015182925.1 mersacidin/lichenicidin family type 2 lantibiotic [Microcoleus sp. PCC 7113]: non-AMP/OTHERS (Probabilities: 0.9818, 0.0059, 0.0123)\n","Processed WP_015136862.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc sp. PCC 7524]: non-AMP/OTHERS (Probabilities: 0.9778, 0.0127, 0.0096)\n","Processed WP_064671706.1 plantaricin C family lantibiotic [Bacillus subtilis]: non-AMP/OTHERS (Probabilities: 0.5575, 0.4281, 0.0145)\n","Processed WP_060698897.1 plantaricin C family lantibiotic [Bacillus australimaris]: non-AMP/OTHERS (Probabilities: 0.9878, 0.0051, 0.0072)\n","Processed WP_172295953.1 lichenicidin A2 family type 2 lantibiotic [Bacillus sp. WMMC1349]: AMP (Probabilities: 0.2016, 0.7773, 0.0211)\n","Processed WP_044741771.1 lichenicidin A2 family type 2 lantibiotic [unclassified Anoxybacillus]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0045, 0.0036)\n","Processed WP_081189986.1 gallidermin/nisin family lantibiotic [Bacillaceae]: AMP (Probabilities: 0.4751, 0.5161, 0.0089)\n","Processed WP_120028177.1 lichenicidin A2 family type 2 lantibiotic [Bacillus subtilis]: non-AMP/OTHERS (Probabilities: 0.9851, 0.0110, 0.0038)\n","Processed WP_015715148.1 FDLD family class I lanthipeptide [Bacillaceae]: AMP (Probabilities: 0.0775, 0.9096, 0.0129)\n","Processed WP_211752103.1 lichenicidin A2 family type 2 lantibiotic [Lactococcus lactis]: AMP (Probabilities: 0.2623, 0.7263, 0.0114)\n","Processed WP_211752102.1 lichenicidin A2 family type 2 lantibiotic [Lactococcus lactis]: non-AMP/OTHERS (Probabilities: 0.9887, 0.0045, 0.0068)\n","Processed WP_126195691.1 gallidermin/nisin family lantibiotic [Tsukamurella paurometabola]: AMP (Probabilities: 0.1124, 0.8737, 0.0139)\n","Processed WP_071581717.1 plantaricin C family lantibiotic [Bacillus subtilis]: non-AMP/OTHERS (Probabilities: 0.5777, 0.4075, 0.0148)\n","Processed P0DQM3.1 Lantibiotic Flvalpha.c [Ruminococcus flavefaciens]: AMP (Probabilities: 0.0901, 0.8977, 0.0122)\n","Processed P0DQM2.1 Lantibiotic Flvalpha.b [Ruminococcus flavefaciens]: AMP (Probabilities: 0.0862, 0.9017, 0.0121)\n","Processed WP_211768001.1 cinnamycin family lantibiotic [Kutzneria sp. CA-103260]: non-AMP/OTHERS (Probabilities: 0.8901, 0.0909, 0.0190)\n","Processed WP_081539212.1 gallidermin/nisin family lantibiotic [Ligilactobacillus salivarius]: AMP (Probabilities: 0.1073, 0.8807, 0.0121)\n","Processed WP_141927650.1 plantaricin C family lantibiotic [Yimella lutea]: non-AMP/OTHERS (Probabilities: 0.9763, 0.0102, 0.0135)\n","Processed WP_169925446.1 plantaricin C family lantibiotic [Sanguibacter antarcticus]: non-AMP/OTHERS (Probabilities: 0.8878, 0.0174, 0.0948)\n","Processed WP_041521073.1 gallidermin/nisin family lantibiotic [Bacillus]: AMP (Probabilities: 0.0786, 0.9091, 0.0123)\n","Processed WP_197683974.1 mersacidin/lichenicidin family type 2 lantibiotic [Allokutzneria albata]: Neuropeptide (Probabilities: 0.0089, 0.0093, 0.9818)\n","Processed WP_055056069.1 plantaricin C family lantibiotic [Eubacteriales]: Neuropeptide (Probabilities: 0.0115, 0.0107, 0.9778)\n","Processed WP_170082055.1 lichenicidin A2 family type 2 lantibiotic [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0038, 0.0049)\n","Processed WP_100902213.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc flagelliforme]: non-AMP/OTHERS (Probabilities: 0.9734, 0.0164, 0.0102)\n","Processed WP_003826899.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium angulatum]: AMP (Probabilities: 0.4439, 0.5373, 0.0188)\n","Processed WP_010071700.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces]: AMP (Probabilities: 0.2867, 0.5109, 0.2024)\n","Processed WP_006126649.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces]: AMP (Probabilities: 0.0852, 0.9026, 0.0122)\n","Processed WP_174257142.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus]: Neuropeptide (Probabilities: 0.0540, 0.0230, 0.9230)\n","Processed MBR1822729.1 plantaricin C family lantibiotic [Ruminococcus sp.]: AMP (Probabilities: 0.0901, 0.8978, 0.0121)\n","Processed WP_090495580.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus virescens]: non-AMP/OTHERS (Probabilities: 0.6352, 0.0351, 0.3297)\n","Processed WP_062708545.1 plantaricin C family lantibiotic [Streptomyces regalis]: non-AMP/OTHERS (Probabilities: 0.8928, 0.1000, 0.0073)\n","Processed WP_144167516.1 lichenicidin A2 family type 2 lantibiotic [Eubacteriales]: non-AMP/OTHERS (Probabilities: 0.9894, 0.0035, 0.0071)\n","Processed WP_016249978.1 gallidermin/nisin family lantibiotic [Enterococcus]: AMP (Probabilities: 0.0674, 0.9184, 0.0142)\n","Processed NRT74866.1 type 2 lantibiotic (TIGR03893 family) [Clostridium beijerinckii]: AMP (Probabilities: 0.0861, 0.9015, 0.0124)\n","Processed WP_055261650.1 plantaricin C family lantibiotic [Roseburia faecis]: non-AMP/OTHERS (Probabilities: 0.6777, 0.0683, 0.2540)\n","Processed WP_004223616.1 plantaricin C family lantibiotic [Lachnospiraceae]: non-AMP/OTHERS (Probabilities: 0.9855, 0.0116, 0.0029)\n","Processed NRT74865.1 type 2 lantibiotic (TIGR03893 family) [Clostridium beijerinckii]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0035, 0.0033)\n","Processed NRT74870.1 type 2 lantibiotic (TIGR03893 family) [Clostridium beijerinckii]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0034)\n","Processed WP_120686896.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. XB500-5]: non-AMP/OTHERS (Probabilities: 0.8722, 0.1230, 0.0048)\n","Processed WP_120686900.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. XB500-5]: non-AMP/OTHERS (Probabilities: 0.9635, 0.0064, 0.0300)\n","Processed WP_120686897.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. XB500-5]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0033, 0.0046)\n","Processed WP_120686901.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. XB500-5]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0035, 0.0038)\n","Processed WP_120658122.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. X503]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0059, 0.0034)\n","Processed WP_120658121.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. X503]: non-AMP/OTHERS (Probabilities: 0.9856, 0.0094, 0.0050)\n","Processed WP_120658124.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. X503]: non-AMP/OTHERS (Probabilities: 0.9860, 0.0109, 0.0031)\n","Processed WP_120686898.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. XB500-5]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0038, 0.0035)\n","Processed WP_120686894.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. XB500-5]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0082, 0.0029)\n","Processed WP_118013842.1 lichenicidin A2 family type 2 lantibiotic [[Ruminococcus] gnavus]: non-AMP/OTHERS (Probabilities: 0.7767, 0.0176, 0.2057)\n","Processed WP_120658120.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. X503]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0040, 0.0043)\n","Processed WP_120658116.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. X503]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0035)\n","Processed WP_120658117.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. X503]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0035, 0.0063)\n","Processed WP_115642317.1 plantaricin C family lantibiotic [Clostridium putrefaciens]: Neuropeptide (Probabilities: 0.1059, 0.0278, 0.8663)\n","Processed WP_003470161.1 gallidermin/nisin family lantibiotic [Clostridium perfringens]: Neuropeptide (Probabilities: 0.0137, 0.0114, 0.9749)\n","Processed WP_106059053.1 plantaricin C family lantibiotic [Clostridium vincentii]: non-AMP/OTHERS (Probabilities: 0.9846, 0.0078, 0.0075)\n","Processed WP_106899368.1 lichenicidin A2 family type 2 lantibiotic [Clostridium botulinum]: AMP (Probabilities: 0.0635, 0.9207, 0.0157)\n","Processed WP_106899367.1 lichenicidin A2 family type 2 lantibiotic [Clostridium botulinum]: AMP (Probabilities: 0.0680, 0.9181, 0.0140)\n","Processed WP_106059058.1 plantaricin C family lantibiotic [Clostridium vincentii]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0061, 0.0025)\n","Processed WP_110290824.1 lichenicidin A2 family type 2 lantibiotic [Lachnotalea glycerini]: AMP (Probabilities: 0.1062, 0.8819, 0.0119)\n","Processed WP_090043248.1 plantaricin C family lantibiotic [Clostridium frigidicarnis]: AMP (Probabilities: 0.0940, 0.8940, 0.0120)\n","Processed WP_118013846.1 lichenicidin A2 family type 2 lantibiotic [[Ruminococcus] gnavus]: AMP (Probabilities: 0.0950, 0.8911, 0.0140)\n","Processed WP_014519822.1 plantaricin C family lantibiotic [Clostridium botulinum]: AMP (Probabilities: 0.0714, 0.9155, 0.0131)\n","Processed WP_103565569.1 cinnamycin family lantibiotic [Actinomadura rubteroloni]: Neuropeptide (Probabilities: 0.4664, 0.0488, 0.4848)\n","Processed WP_092249247.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. INlla21]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0034)\n","Processed WP_092249241.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. INlla21]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0050, 0.0053)\n","Processed WP_092249238.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. INlla21]: AMP (Probabilities: 0.4400, 0.5486, 0.0115)\n","Processed WP_092249244.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. INlla21]: AMP (Probabilities: 0.0943, 0.8933, 0.0124)\n","Processed WP_090169704.1 gallidermin/nisin family lantibiotic [unclassified Pseudobutyrivibrio]: non-AMP/OTHERS (Probabilities: 0.4950, 0.4931, 0.0118)\n","Processed WP_092249232.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. INlla21]: AMP (Probabilities: 0.1157, 0.8704, 0.0140)\n","Processed WP_092249223.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. INlla21]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0034, 0.0033)\n","Processed WP_091822691.1 plantaricin C family lantibiotic [Butyrivibrio sp. ob235]: Neuropeptide (Probabilities: 0.4073, 0.0418, 0.5510)\n","Processed WP_090169698.1 gallidermin/nisin family lantibiotic [unclassified Pseudobutyrivibrio]: AMP (Probabilities: 0.0638, 0.9200, 0.0162)\n","Processed WP_092249229.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. INlla21]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0029, 0.0049)\n","Processed WP_092249226.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. INlla21]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0031, 0.0067)\n","Processed WP_090169700.1 gallidermin/nisin family lantibiotic [unclassified Pseudobutyrivibrio]: AMP (Probabilities: 0.0641, 0.9198, 0.0161)\n","Processed WP_090169702.1 gallidermin/nisin family lantibiotic [unclassified Pseudobutyrivibrio]: AMP (Probabilities: 0.0671, 0.9179, 0.0150)\n","Processed WP_008429200.1 gallidermin/nisin family lantibiotic [Clostridium sp. LS]: AMP (Probabilities: 0.0667, 0.9179, 0.0153)\n","Processed WP_207486580.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus avium]: AMP (Probabilities: 0.3456, 0.6441, 0.0103)\n","Processed WP_004065092.1 lichenicidin A2 family type 2 lantibiotic [Eubacterium plexicaudatum]: non-AMP/OTHERS (Probabilities: 0.9298, 0.0647, 0.0055)\n","Processed WP_093261184.1 gallidermin/nisin family lantibiotic [Thermostaphylospora chromogena]: non-AMP/OTHERS (Probabilities: 0.9805, 0.0155, 0.0040)\n","Processed WP_111330805.1 plantaricin C family lantibiotic [unclassified Streptomyces]: Neuropeptide (Probabilities: 0.4030, 0.1487, 0.4483)\n","Processed WP_087986582.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus tropicus]: non-AMP/OTHERS (Probabilities: 0.9869, 0.0096, 0.0035)\n","Processed WP_078787334.1 plantaricin C family lantibiotic [Eubacterium ruminantium]: Neuropeptide (Probabilities: 0.0142, 0.0124, 0.9734)\n","Processed WP_207486581.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus avium]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0039, 0.0040)\n","Processed WP_207486578.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus avium]: Neuropeptide (Probabilities: 0.3080, 0.0267, 0.6654)\n","Processed WP_107537066.1 lichenicidin A2 family type 2 lantibiotic [Mammaliicoccus vitulinus]: non-AMP/OTHERS (Probabilities: 0.9896, 0.0067, 0.0037)\n","Processed WP_107537067.1 lichenicidin A2 family type 2 lantibiotic [Mammaliicoccus vitulinus]: non-AMP/OTHERS (Probabilities: 0.9852, 0.0115, 0.0033)\n","Processed WP_113657277.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermoflavimicrobium sp. FBKL4.011]: non-AMP/OTHERS (Probabilities: 0.9777, 0.0095, 0.0128)\n","Processed WP_129300415.1 lichenicidin A2 family type 2 lantibiotic [Lactococcus lactis]: AMP (Probabilities: 0.1061, 0.8819, 0.0120)\n","Processed WP_143802365.1 lichenicidin A2 family type 2 lantibiotic [Paenibacillus thiaminolyticus]: non-AMP/OTHERS (Probabilities: 0.9849, 0.0124, 0.0028)\n","Processed WP_129300414.1 lichenicidin A2 family type 2 lantibiotic [Lactococcus lactis]: AMP (Probabilities: 0.2864, 0.6955, 0.0181)\n","Processed WP_155279846.1 lichenicidin A2 family type 2 lantibiotic [Streptococcus parasanguinis]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0048, 0.0033)\n","Processed WP_128799169.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus coralloides]: Neuropeptide (Probabilities: 0.4182, 0.0964, 0.4854)\n","Processed WP_134189469.1 lichenicidin A2 family type 2 lantibiotic [Bacillus sp. BH2]: non-AMP/OTHERS (Probabilities: 0.9563, 0.0407, 0.0030)\n","Processed WP_137077048.1 lichenicidin A2 family type 2 lantibiotic, partial [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9875, 0.0096, 0.0029)\n","Processed WP_143567378.1 plantaricin C family lantibiotic [Streptococcus salivarius]: Neuropeptide (Probabilities: 0.0205, 0.0160, 0.9635)\n","Processed WP_046198765.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9862, 0.0092, 0.0046)\n","Processed WP_031924236.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9625, 0.0081, 0.0293)\n","Processed WP_143567471.1 lantibiotic streptin [Streptococcus salivarius]: AMP (Probabilities: 0.0772, 0.9105, 0.0124)\n","Processed WP_056937919.1 lichenicidin A2 family type 2 lantibiotic [Limosilactobacillus gastricus]: AMP (Probabilities: 0.1186, 0.8687, 0.0126)\n","Processed WP_056937913.1 lichenicidin A2 family type 2 lantibiotic [Limosilactobacillus gastricus]: Neuropeptide (Probabilities: 0.1869, 0.0229, 0.7902)\n","Processed WP_084946468.1 plantaricin C family lantibiotic [Streptococcus mitis]: Neuropeptide (Probabilities: 0.0721, 0.0336, 0.8943)\n","Processed WP_162238550.1 elgicin/penisin family lantibiotic [Paenibacillus sp. A3]: AMP (Probabilities: 0.1501, 0.7747, 0.0752)\n","Processed WP_048523969.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.5418, 0.0338, 0.4244)\n","Processed WP_033521883.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium merycicum]: AMP (Probabilities: 0.1700, 0.8107, 0.0193)\n","Processed WP_116078332.1 lichenicidin A2 family type 2 lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9808, 0.0162, 0.0030)\n","Processed WP_002289977.1 lantibiotic mutacin [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.8027, 0.0323, 0.1650)\n","Processed WP_202231258.1 lichenicidin A2 family type 2 lantibiotic [Lactococcus garvieae]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0039, 0.0066)\n","Processed WP_010890686.1 lantibiotic lacticin 3147 A2 [Lactococcus lactis]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0046, 0.0029)\n","Processed WP_034530520.1 gallidermin/nisin family lantibiotic, partial [Apilactobacillus kunkeei]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0038, 0.0031)\n","Processed WP_142219303.1 gallidermin/nisin family lantibiotic, partial [Streptococcus salivarius]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0037)\n","Processed WP_078806208.1 lichenicidin A2 family type 2 lantibiotic [Pilibacter termitis]: AMP (Probabilities: 0.0703, 0.9167, 0.0130)\n","Processed WP_010890685.1 plantaricin C family lantibiotic [Lactococcus lactis]: Neuropeptide (Probabilities: 0.0296, 0.0182, 0.9522)\n","Processed WP_202231257.1 lichenicidin A2 family type 2 lantibiotic [Lactococcus garvieae]: non-AMP/OTHERS (Probabilities: 0.9872, 0.0072, 0.0056)\n","Processed WP_139555170.1 plantaricin C family lantibiotic [Fructilactobacillus sanfranciscensis]: Neuropeptide (Probabilities: 0.0708, 0.0252, 0.9040)\n","Processed WP_078806213.1 lichenicidin A2 family type 2 lantibiotic [Pilibacter termitis]: non-AMP/OTHERS (Probabilities: 0.6145, 0.0282, 0.3574)\n","Processed WP_070046034.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.8548, 0.0535, 0.0917)\n","Processed WP_070005024.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9790, 0.0119, 0.0091)\n","Processed WP_110933144.1 gallidermin/nisin family lantibiotic [Paenibacillus bouchesdurhonensis]: AMP (Probabilities: 0.0782, 0.9087, 0.0131)\n","Processed WP_036103587.1 plantaricin C family lantibiotic [Listeria grayi]: AMP (Probabilities: 0.1080, 0.8760, 0.0160)\n","Processed WP_100544254.1 mersacidin/lichenicidin family type 2 lantibiotic [Lysinibacillus xylanilyticus]: non-AMP/OTHERS (Probabilities: 0.9641, 0.0277, 0.0082)\n","Processed WP_157829307.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0045, 0.0046)\n","Processed WP_100544958.1 mersacidin/lichenicidin family type 2 lantibiotic [Lysinibacillus xylanilyticus]: non-AMP/OTHERS (Probabilities: 0.9809, 0.0116, 0.0075)\n","Processed WP_100544960.1 mersacidin/lichenicidin family type 2 lantibiotic [Lysinibacillus xylanilyticus]: non-AMP/OTHERS (Probabilities: 0.9820, 0.0100, 0.0081)\n","Processed WP_100544956.1 mersacidin/lichenicidin family type 2 lantibiotic [Lysinibacillus xylanilyticus]: non-AMP/OTHERS (Probabilities: 0.9640, 0.0144, 0.0217)\n","Processed WP_100544955.1 mersacidin/lichenicidin family type 2 lantibiotic [Lysinibacillus xylanilyticus]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0051, 0.0061)\n","Processed WP_100544961.1 mersacidin/lichenicidin family type 2 lantibiotic [Lysinibacillus xylanilyticus]: non-AMP/OTHERS (Probabilities: 0.9729, 0.0190, 0.0081)\n","Processed WP_100544962.1 mersacidin/lichenicidin family type 2 lantibiotic [Lysinibacillus xylanilyticus]: non-AMP/OTHERS (Probabilities: 0.9857, 0.0104, 0.0039)\n","Processed WP_100544959.1 mersacidin/lichenicidin family type 2 lantibiotic [Lysinibacillus xylanilyticus]: non-AMP/OTHERS (Probabilities: 0.9641, 0.0159, 0.0200)\n","Processed WP_095989442.1 mersacidin/lichenicidin family type 2 lantibiotic [Cystobacter fuscus]: non-AMP/OTHERS (Probabilities: 0.4965, 0.0536, 0.4498)\n","Processed WP_087875939.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9506, 0.0442, 0.0052)\n","Processed WP_002182161.1 lichenicidin A2 family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9513, 0.0090, 0.0397)\n","Processed WP_015159460.1 mersacidin/lichenicidin family type 2 lantibiotic [Chamaesiphon minutus]: non-AMP/OTHERS (Probabilities: 0.9381, 0.0488, 0.0131)\n","Processed WP_075739930.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinoalloteichus]: AMP (Probabilities: 0.2432, 0.6845, 0.0723)\n","Processed WP_075739929.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinoalloteichus]: non-AMP/OTHERS (Probabilities: 0.8049, 0.0255, 0.1696)\n","Processed WP_015147952.1 mersacidin/lichenicidin family type 2 lantibiotic [Oscillatoria acuminata]: non-AMP/OTHERS (Probabilities: 0.9137, 0.0320, 0.0542)\n","Processed WP_075739931.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinoalloteichus]: non-AMP/OTHERS (Probabilities: 0.8781, 0.1059, 0.0160)\n","Processed WP_154803672.1 plantaricin C family lantibiotic [Streptococcus equi]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0038, 0.0043)\n","Processed WP_157747182.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora echinofusca]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0050, 0.0053)\n","Processed WP_000322863.1 type 2 lantibiotic, mersacidin/lichenicidin family, partial [Bacillus]: Neuropeptide (Probabilities: 0.0147, 0.0120, 0.9733)\n","Processed WP_021604107.1 gallidermin/nisin family lantibiotic [Actinomyces graevenitzii]: AMP (Probabilities: 0.1061, 0.8785, 0.0154)\n","Processed WP_152704050.1 lichenicidin A2 family type 2 lantibiotic [Limosilactobacillus reuteri]: AMP (Probabilities: 0.2005, 0.7879, 0.0116)\n","Processed WP_152704051.1 lichenicidin A2 family type 2 lantibiotic [Limosilactobacillus reuteri]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0034, 0.0033)\n","Processed WP_157741520.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora krabiensis]: non-AMP/OTHERS (Probabilities: 0.9769, 0.0084, 0.0148)\n","Processed WP_061523391.1 plantaricin C family lantibiotic [Bacillus nakamurai]: AMP (Probabilities: 0.4088, 0.5773, 0.0139)\n","Processed HAR7304160.1 type 2 lantibiotic [Staphylococcus aureus]: AMP (Probabilities: 0.0666, 0.9189, 0.0144)\n","Processed WP_078747399.1 plantaricin C family lantibiotic [Streptococcus pneumoniae]: Neuropeptide (Probabilities: 0.2300, 0.0983, 0.6717)\n","Processed WP_061521606.1 gallidermin/nisin family lantibiotic [Bacillus nakamurai]: non-AMP/OTHERS (Probabilities: 0.9882, 0.0086, 0.0032)\n","Processed WP_186443353.1 lantibiotic paenibacillin [Staphylococcus agnetis]: AMP (Probabilities: 0.0670, 0.9185, 0.0144)\n","Processed WP_085787919.1 gallidermin/nisin family lantibiotic [Anoxybacillus ayderensis]: non-AMP/OTHERS (Probabilities: 0.7870, 0.1543, 0.0587)\n","Processed KAF6629218.1 mersacidin/lichenicidin family type 2 lantibiotic [Paenibacillus sp. EKM208P]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0036, 0.0031)\n","Processed WP_061735465.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.6133, 0.1932, 0.1934)\n","Processed Q57312.1 Lantibiotic epilancin [Staphylococcus epidermidis]: AMP (Probabilities: 0.0675, 0.9183, 0.0142)\n","Processed P86720.1 Lantibiotic lichenicidin A2 [Bacillus licheniformis DSM 13 = ATCC 14580]: AMP (Probabilities: 0.1316, 0.8556, 0.0128)\n","Processed VED86689.1 Lantibiotic macedocin [Streptococcus vestibularis]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0036, 0.0051)\n","Processed AWX47505.1 lantibiotic transporter [Enterococcus faecium]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed WP_031893546.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.8937, 0.0136, 0.0927)\n","Processed WP_056999992.1 lichenicidin A2 family type 2 lantibiotic [Carnobacterium maltaromaticum]: AMP (Probabilities: 0.0672, 0.9191, 0.0137)\n","Processed EFK82377.1 lantibiotic, gallidermin/nisin family [Staphylococcus aureus subsp. aureus TCH70 TCH70]: Neuropeptide (Probabilities: 0.0112, 0.0108, 0.9780)\n","Processed MNH98119.1 Lantibiotic lacticin A1 precursor [compost metagenome]: Neuropeptide (Probabilities: 0.4146, 0.0327, 0.5527)\n","Processed WP_000416756.1 gallidermin/nisin family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.8897, 0.0135, 0.0968)\n","Processed EHD61543.1 hypothetical protein [Streptococcus pneumoniae GA49447 GA49447]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0039, 0.0029)\n","Processed AJH71610.1 putative lantibiotic cytolysin [Bacillus cereus ATCC 4342 ATCC 4342]: AMP (Probabilities: 0.2889, 0.6997, 0.0115)\n","Processed KMR79974.1 lantibiotic ABC transporter permease, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0068, 0.0028)\n","Processed BAR45151.1 lantibiotic salivaricin A [Streptococcus pyogenes JRS4 JRS4]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0035, 0.0038)\n","Processed KMS35393.1 lantibiotic ABC transporter permease, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9894, 0.0079, 0.0026)\n","Processed WP_197210093.1 plantaricin C family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.8409, 0.0155, 0.1437)\n","Processed WP_222645137.1 gallidermin/nisin family lantibiotic [Clostridium botulinum]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0044, 0.0034)\n","Processed CIW23645.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0058, 0.0029)\n","Processed AIL12211.1 lantibiotic salivaricin-A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0033)\n","Processed WP_139815814.1 gallidermin/nisin family lantibiotic [Clostridium botulinum]: non-AMP/OTHERS (Probabilities: 0.9095, 0.0836, 0.0069)\n","Processed WP_001198626.1 plantaricin C family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9613, 0.0109, 0.0277)\n","Processed WP_071729918.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed WP_136093703.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9688, 0.0057, 0.0254)\n","Processed WP_161556976.1 type A2 lantipeptide [Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.8537, 0.0221, 0.1241)\n","Processed WP_071729920.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0038, 0.0032)\n","Processed WP_001029875.1 hypothetical protein [Streptococcus]: AMP (Probabilities: 0.0819, 0.9057, 0.0124)\n","Processed WP_222317326.1 gallidermin/nisin family lantibiotic [Vagococcus lutrae]: AMP (Probabilities: 0.0721, 0.9149, 0.0130)\n","Processed WP_222225061.1 mersacidin family lantibiotic, partial [Bacillus paranthracis]: non-AMP/OTHERS (Probabilities: 0.5594, 0.4301, 0.0105)\n","Processed WP_038432550.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0038, 0.0031)\n","Processed WP_071732198.1 plantaricin C family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9887, 0.0048, 0.0064)\n","Processed WP_222368712.1 lantibiotic (srt) production protein [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9051, 0.0904, 0.0044)\n","Processed WP_219712143.1 NisI/SpaI family lantibiotic immunity lipoprotein, partial [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0042, 0.0033)\n","Processed WP_219709901.1 NisI/SpaI family lantibiotic immunity lipoprotein, partial [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9882, 0.0066, 0.0052)\n","Processed WP_219706044.1 NisI/SpaI family lantibiotic immunity lipoprotein, partial [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9837, 0.0107, 0.0055)\n","Processed WP_000794683.1 type A2 lantipeptide [Streptococcus agalactiae]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0031, 0.0035)\n","Processed WP_219691885.1 NisI/SpaI family lantibiotic immunity lipoprotein, partial [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0041, 0.0030)\n","Processed WP_145614662.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus licheniformis]: AMP (Probabilities: 0.1120, 0.8744, 0.0135)\n","Processed WP_125180670.1 gallidermin/nisin family lantibiotic [Enterococcaceae]: Neuropeptide (Probabilities: 0.3713, 0.1956, 0.4331)\n","Processed WP_146323895.1 gallidermin/nisin family lantibiotic [Corynebacterium canis]: AMP (Probabilities: 0.1018, 0.8797, 0.0185)\n","Processed WP_071735698.1 gallidermin/nisin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9881, 0.0096, 0.0023)\n","Processed WP_080684088.1 gallidermin/nisin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9834, 0.0141, 0.0025)\n","Processed WP_222149673.1 lantibiotic (srt) production protein, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0056, 0.0027)\n","Processed WP_058915350.1 SCO0268 family class II lantipeptide [Actinomycetia]: non-AMP/OTHERS (Probabilities: 0.8896, 0.0108, 0.0996)\n","Processed WP_222132711.1 lantibiotic (srt) production protein, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0055, 0.0044)\n","Processed WP_019764989.1 thiazolylpeptide-type bacteriocin [unclassified Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0033, 0.0037)\n","Processed WP_191904264.1 lantibiotic dehydratase, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9709, 0.0077, 0.0214)\n","Processed WP_221496435.1 lantibiotic dehydratase [Nonomuraea muscovyensis]: non-AMP/OTHERS (Probabilities: 0.9039, 0.0145, 0.0816)\n","Processed WP_018492818.1 thiazolylpeptide-type bacteriocin [Streptomycetaceae]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed WP_044980251.1 lantibiotic salivaricin M precursor [Streptococcus suis]: AMP (Probabilities: 0.0971, 0.8907, 0.0122)\n","Processed WP_025810438.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9827, 0.0057, 0.0116)\n","Processed WP_003978597.1 SCO0268 family class II lantipeptide [Streptomyces]: Neuropeptide (Probabilities: 0.0479, 0.0186, 0.9335)\n","Processed WP_044688904.1 hypothetical protein [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0080, 0.0025)\n","Processed WP_044980471.1 type A2 lantipeptide [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.8794, 0.0963, 0.0243)\n","Processed WP_018840305.1 thiazolylpeptide-type bacteriocin [unclassified Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0032)\n","Processed WP_044774496.1 class II lanthipeptide, LchA2/BrtA2 family [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0034)\n","Processed WP_079396189.1 type A2 lantipeptide [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9285, 0.0105, 0.0610)\n","Processed WP_044680502.1 hypothetical protein [Streptococcus suis]: AMP (Probabilities: 0.0771, 0.9103, 0.0126)\n","Processed WP_044680499.1 hypothetical protein [Streptococcus suis]: AMP (Probabilities: 0.0662, 0.9191, 0.0147)\n","Processed WP_044680504.1 hypothetical protein [Streptococcus suis]: AMP (Probabilities: 0.0704, 0.9162, 0.0134)\n","Processed WP_044680745.1 hypothetical protein [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.4936, 0.4916, 0.0148)\n","Processed WP_044680500.1 lantibiotic salivaricin M precursor [Streptococcus suis]: AMP (Probabilities: 0.3281, 0.5095, 0.1624)\n","Processed WP_024394058.1 lantibiotic salivaricin M precursor [Streptococcus suis]: AMP (Probabilities: 0.0988, 0.8891, 0.0121)\n","Processed WP_024394059.1 hypothetical protein [Streptococcus suis]: AMP (Probabilities: 0.0638, 0.9215, 0.0146)\n","Processed WP_044980681.1 type A2 lantipeptide [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0040, 0.0049)\n","Processed WP_014636300.1 hypothetical protein [Lactobacillales]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0045, 0.0029)\n","Processed WP_024395819.1 hypothetical protein [Streptococcus suis]: AMP (Probabilities: 0.0663, 0.9195, 0.0142)\n","Processed WP_044771623.1 hypothetical protein [Streptococcus suis]: AMP (Probabilities: 0.0664, 0.9195, 0.0141)\n","Processed WP_024394056.1 hypothetical protein [Streptococcus suis]: AMP (Probabilities: 0.0735, 0.9139, 0.0127)\n","Processed WP_000791200.1 hypothetical protein [Streptococcus agalactiae]: AMP (Probabilities: 0.0650, 0.9201, 0.0149)\n","Processed WP_000240509.1 hypothetical protein [Streptococcus agalactiae]: AMP (Probabilities: 0.0858, 0.9019, 0.0123)\n","Processed WP_001816908.1 hypothetical protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9860, 0.0094, 0.0047)\n","Processed WP_222122580.1 gallidermin/nisin family lantibiotic [Bacillus sp. FJAT-22090]: AMP (Probabilities: 0.2421, 0.6511, 0.1068)\n","Processed WP_024394057.1 hypothetical protein [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.4508, 0.4055, 0.1438)\n","Processed WP_222122479.1 mersacidin family lantibiotic, partial [Bacillus pumilus]: non-AMP/OTHERS (Probabilities: 0.9859, 0.0071, 0.0070)\n","Processed WP_180874067.1 lantibiotic dehydratase [Lactobacillus johnsonii]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0048, 0.0029)\n","Processed WP_186371600.1 mersacidin family lantibiotic [Bacillus mobilis]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0047, 0.0062)\n","Processed WP_145517187.1 type A2 lantipeptide [Streptococcus salivarius]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0039, 0.0031)\n","Processed WP_155731132.1 type A lantibiotic [Bacteria]: AMP (Probabilities: 0.0717, 0.9150, 0.0133)\n","Processed WP_145422688.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus hominis]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed WP_145432843.1 plantaricin C family lantibiotic [Staphylococcus haemolyticus]: AMP (Probabilities: 0.1218, 0.8638, 0.0145)\n","Processed WP_145422687.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus hominis]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0030, 0.0040)\n","Processed WP_144764941.1 mersacidin/lichenicidin family type 2 lantibiotic [Curtobacterium sp. 9128]: non-AMP/OTHERS (Probabilities: 0.7132, 0.0331, 0.2537)\n","Processed WP_144565848.1 lichenicidin A2 family type 2 lantibiotic [Bacillus mobilis]: non-AMP/OTHERS (Probabilities: 0.9804, 0.0111, 0.0085)\n","Processed WP_025172691.1 hypothetical protein [Streptococcus pneumoniae]: AMP (Probabilities: 0.0766, 0.9106, 0.0128)\n","Processed WP_144565851.1 mersacidin family lantibiotic [Bacillus mobilis]: non-AMP/OTHERS (Probabilities: 0.9883, 0.0059, 0.0059)\n","Processed WP_144565850.1 mersacidin family lantibiotic [Bacillus mobilis]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0067, 0.0047)\n","Processed WP_144565853.1 mersacidin family lantibiotic [Bacillus mobilis]: non-AMP/OTHERS (Probabilities: 0.9632, 0.0133, 0.0235)\n","Processed WP_144565797.1 mersacidin family lantibiotic [Bacillus mobilis]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0064, 0.0030)\n","Processed WP_144565846.1 mersacidin family lantibiotic, partial [Bacillus mobilis]: non-AMP/OTHERS (Probabilities: 0.9588, 0.0127, 0.0285)\n","Processed WP_144519357.1 lichenicidin A2 family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0036, 0.0066)\n","Processed WP_144488618.1 gallidermin/nisin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9717, 0.0255, 0.0028)\n","Processed WP_098328442.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9467, 0.0475, 0.0059)\n","Processed WP_098328441.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.6277, 0.0325, 0.3398)\n","Processed WP_144465878.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9779, 0.0129, 0.0092)\n","Processed WP_097832711.1 lichenicidin alpha family lanthipeptide [Bacillus cereus group]: Neuropeptide (Probabilities: 0.4059, 0.0340, 0.5601)\n","Processed WP_016397469.1 hypothetical protein [Streptococcus]: AMP (Probabilities: 0.0644, 0.9210, 0.0146)\n","Processed WP_088364036.1 gallidermin/nisin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9699, 0.0161, 0.0140)\n","Processed WP_088233001.1 gallidermin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.6757, 0.3058, 0.0186)\n","Processed WP_074619570.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0049, 0.0051)\n","Processed WP_076983573.1 gallidermin/nisin family lantibiotic [Bacillus]: AMP (Probabilities: 0.1090, 0.8796, 0.0114)\n","Processed WP_060391312.1 type A2 lantipeptide [Leuconostoc]: AMP (Probabilities: 0.1333, 0.8519, 0.0148)\n","Processed WP_049389114.1 gallidermin family lantibiotic [Staphylococcus capitis]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0029, 0.0042)\n","Processed WP_088364035.1 gallidermin/nisin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9829, 0.0074, 0.0097)\n","Processed WP_017648786.1 hypothetical protein [Streptococcus]: AMP (Probabilities: 0.0681, 0.9183, 0.0136)\n","Processed WP_050197452.1 class II lanthipeptide, LchA2/BrtA2 family [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.7235, 0.1085, 0.1680)\n","Processed WP_018768243.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0054, 0.0025)\n","Processed WP_001233918.1 plantaricin C family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0039, 0.0053)\n","Processed WP_037657426.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0034)\n","Processed WP_000180458.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0080, 0.0024)\n","Processed WP_004936805.1 SCO0268 family class II lantipeptide [Streptomyces]: Neuropeptide (Probabilities: 0.3838, 0.0368, 0.5794)\n","Processed WP_023396278.1 hypothetical protein [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.6076, 0.3108, 0.0816)\n","Processed WP_000180457.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0034)\n","Processed WP_011152951.1 type A2 lantipeptide [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.4882, 0.0307, 0.4810)\n","Processed WP_003154891.1 plantaricin C family lantibiotic [Bacillus]: Neuropeptide (Probabilities: 0.1265, 0.0399, 0.8336)\n","Processed WP_000057443.1 lichenicidin alpha family lanthipeptide [Streptococcus pneumoniae]: AMP (Probabilities: 0.0964, 0.8916, 0.0120)\n","Processed WP_050200815.1 hypothetical protein [Streptococcus pneumoniae]: AMP (Probabilities: 0.0878, 0.8994, 0.0128)\n","Processed WP_015031827.1 class IV lanthipeptide venezuelin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0035, 0.0048)\n","Processed EFU7782059.1 lantibiotic ABC transporter ATP-binding protein [Shigella flexneri]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0046, 0.0034)\n","Processed WP_173609174.1 gallidermin family lantibiotic [Brevibacillus]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0033)\n","Processed WP_171776391.1 plantaricin C family lantibiotic [Priestia]: Neuropeptide (Probabilities: 0.3052, 0.0353, 0.6595)\n","Processed WP_025172692.1 hypothetical protein [Streptococcus]: AMP (Probabilities: 0.0674, 0.9189, 0.0137)\n","Processed WP_023396277.1 hypothetical protein [Streptococcus pneumoniae]: AMP (Probabilities: 0.0647, 0.9207, 0.0146)\n","Processed WP_115057873.1 FDLD family class I lanthipeptide [Paenibacillus thiaminolyticus]: AMP (Probabilities: 0.1397, 0.8393, 0.0210)\n","Processed WP_027968842.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0050, 0.0029)\n","Processed WP_143849181.1 gallidermin/nisin family lantibiotic [Bacillus sp. P16(2019)]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0048, 0.0031)\n","Processed WP_221874071.1 mersacidin family lantibiotic [Mesobacillus maritimus]: AMP (Probabilities: 0.1386, 0.8489, 0.0125)\n","Processed WP_139008740.1 gallidermin/nisin family lantibiotic [Lactococcus lactis]: AMP (Probabilities: 0.0662, 0.9186, 0.0152)\n","Processed WP_162617029.1 lichenicidin A2 family type 2 lantibiotic, partial [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.7306, 0.0130, 0.2563)\n","Processed WP_221874070.1 mersacidin family lantibiotic [Mesobacillus maritimus]: non-AMP/OTHERS (Probabilities: 0.9872, 0.0045, 0.0084)\n","Processed WP_162617017.1 lichenicidin A2 family type 2 lantibiotic, partial [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.7569, 0.0171, 0.2260)\n","Processed WP_011676990.1 hypothetical protein [Lactococcus]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0048, 0.0033)\n","Processed WP_142482228.1 lichenicidin A2 family type 2 lantibiotic [Klebsiella oxytoca]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0043, 0.0032)\n","Processed WP_016397470.1 hypothetical protein [Streptococcus]: AMP (Probabilities: 0.0664, 0.9190, 0.0145)\n","Processed WP_142482227.1 lichenicidin A2 family type 2 lantibiotic [Klebsiella oxytoca]: non-AMP/OTHERS (Probabilities: 0.9827, 0.0147, 0.0026)\n","Processed WP_142482226.1 lichenicidin A2 family type 2 lantibiotic [Klebsiella oxytoca]: non-AMP/OTHERS (Probabilities: 0.9804, 0.0054, 0.0142)\n","Processed WP_142970428.1 type A2 lantipeptide [Dolosigranulum pigrum]: non-AMP/OTHERS (Probabilities: 0.6812, 0.3063, 0.0125)\n","Processed WP_142970427.1 type A2 lantipeptide [Dolosigranulum pigrum]: non-AMP/OTHERS (Probabilities: 0.9853, 0.0066, 0.0081)\n","Processed WP_032874558.1 plantaricin C family lantibiotic [Bacillus]: Neuropeptide (Probabilities: 0.0560, 0.0265, 0.9175)\n","Processed WP_127199363.1 plantaricin C family lantibiotic [Paenibacillus zeisoli]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0047, 0.0036)\n","Processed WP_125743679.1 plantaricin C family lantibiotic [Streptomyces sp. WAC01280]: Neuropeptide (Probabilities: 0.0612, 0.0218, 0.9170)\n","Processed WP_113766098.1 lichenicidin A2 family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0037, 0.0037)\n","Processed WP_031128743.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9874, 0.0092, 0.0034)\n","Processed WP_071584197.1 plantaricin C family lantibiotic [Bacillales]: AMP (Probabilities: 0.1155, 0.8691, 0.0154)\n","Processed MBY0523086.1 mersacidin/lichenicidin family type 2 lantibiotic [Gemmataceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0059, 0.0044)\n","Processed MBY0523085.1 mersacidin/lichenicidin family type 2 lantibiotic [Gemmataceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9811, 0.0140, 0.0048)\n","Processed WP_004952279.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9547, 0.0081, 0.0372)\n","Processed WP_019328243.1 SCO0268 family class II lantipeptide [Streptomyces]: Neuropeptide (Probabilities: 0.4351, 0.0334, 0.5315)\n","Processed WP_019324892.1 lexapeptide family class V lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0044, 0.0033)\n","Processed MBX9792499.1 mersacidin/lichenicidin family type 2 lantibiotic [Pirellulales bacterium]: non-AMP/OTHERS (Probabilities: 0.9828, 0.0098, 0.0073)\n","Processed WP_201783871.1 lichenicidin A2 family type 2 lantibiotic [Carnobacterium maltaromaticum]: AMP (Probabilities: 0.0659, 0.9196, 0.0145)\n","Processed WP_206698159.1 gallidermin family lantibiotic [Cytobacillus firmus]: Neuropeptide (Probabilities: 0.2234, 0.2127, 0.5639)\n","Processed WP_193634948.1 gallidermin family lantibiotic [Vibrio coralliilyticus]: non-AMP/OTHERS (Probabilities: 0.9712, 0.0217, 0.0071)\n","Processed WP_201783870.1 lichenicidin A2 family type 2 lantibiotic [Carnobacterium maltaromaticum]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0038, 0.0028)\n","Processed WP_125398663.1 lantibiotic salivaricin M precursor [Streptococcus oralis]: AMP (Probabilities: 0.0752, 0.9117, 0.0131)\n","Processed WP_125372387.1 plantaricin C family lantibiotic [Streptococcus cristatus]: AMP (Probabilities: 0.0965, 0.8917, 0.0118)\n","Processed WP_007268044.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0036, 0.0036)\n","Processed WP_125177081.1 lantibiotic streptin [Streptococcus suis]: AMP (Probabilities: 0.0665, 0.9185, 0.0150)\n","Processed WP_125828177.1 lantibiotic salivaricin M precursor [Streptococcus]: AMP (Probabilities: 0.0692, 0.9176, 0.0132)\n","Processed WP_118582926.1 gallidermin family lantibiotic [Roseburia inulinivorans]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0056, 0.0044)\n","Processed WP_070846370.1 lantibiotic salivaricin M precursor [Streptococcus]: AMP (Probabilities: 0.0901, 0.8974, 0.0125)\n","Processed WP_069126692.1 type A2 lantipeptide [Brochothrix thermosphacta]: non-AMP/OTHERS (Probabilities: 0.9766, 0.0087, 0.0147)\n","Processed WP_096241054.1 plantaricin C family lantibiotic [Eubacteriales]: Neuropeptide (Probabilities: 0.0091, 0.0098, 0.9810)\n","Processed WP_115596678.1 gallidermin family lantibiotic [Bacillales]: Neuropeptide (Probabilities: 0.0117, 0.0115, 0.9768)\n","Processed WP_115262384.1 hypothetical protein [Streptococcus mitis]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0063, 0.0025)\n","Processed WP_080713125.1 type A2 lantipeptide [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0033, 0.0041)\n","Processed EAA0838320.1 lantibiotic ABC transporter ATP-binding protein [Shigella flexneri]: non-AMP/OTHERS (Probabilities: 0.9892, 0.0060, 0.0048)\n","Processed WP_020647175.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis balhimycina]: Neuropeptide (Probabilities: 0.3310, 0.0434, 0.6256)\n","Processed WP_019710917.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9602, 0.0357, 0.0041)\n","Processed WP_033685037.1 type A2 lantipeptide [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0043, 0.0033)\n","Processed WP_030029258.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0077, 0.0033)\n","Processed WP_061791363.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillales]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0035, 0.0040)\n","Processed WP_020278664.1 ATP-binding cassette domain-containing protein [Geobacillus]: non-AMP/OTHERS (Probabilities: 0.9867, 0.0066, 0.0067)\n","Processed WP_179767042.1 cinnamycin family lantibiotic [Streptomonospora]: non-AMP/OTHERS (Probabilities: 0.5777, 0.0957, 0.3266)\n","Processed WP_141952378.1 cinnamycin family lantibiotic [Actinoallomurus bryophytorum]: non-AMP/OTHERS (Probabilities: 0.8040, 0.1869, 0.0091)\n","Processed WP_001849794.1 hypothetical protein [Streptococcus pneumoniae]: AMP (Probabilities: 0.1229, 0.8641, 0.0130)\n","Processed WP_023522751.1 gallidermin/nisin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0036, 0.0030)\n","Processed WP_185567211.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc sp. 2RC]: non-AMP/OTHERS (Probabilities: 0.9415, 0.0102, 0.0483)\n","Processed WP_185004659.1 mersacidin/lichenicidin family type 2 lantibiotic [Crossiella cryophila]: AMP (Probabilities: 0.0726, 0.9144, 0.0131)\n","Processed WP_185028960.1 cinnamycin family lantibiotic [Actinomadura coerulea]: non-AMP/OTHERS (Probabilities: 0.9580, 0.0336, 0.0084)\n","Processed WP_184986655.1 plantaricin C family lantibiotic [Streptomyces caelestis]: Neuropeptide (Probabilities: 0.4233, 0.0386, 0.5381)\n","Processed WP_124781628.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.8018, 0.0176, 0.1806)\n","Processed WP_184632841.1 plantaricin C family lantibiotic [Streptomonospora salina]: Neuropeptide (Probabilities: 0.1041, 0.0350, 0.8609)\n","Processed WP_184186928.1 plantaricin C family lantibiotic [Paenibacillus sp. JGP012]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0042, 0.0047)\n","Processed WP_184822593.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.5807, 0.3983, 0.0210)\n","Processed WP_185028346.1 gallidermin/nisin family lantibiotic [Streptomyces candidus]: non-AMP/OTHERS (Probabilities: 0.9694, 0.0165, 0.0141)\n","Processed  unnamed protein product [Unknown]: AMP (Probabilities: 0.0895, 0.8979, 0.0126)\n","Processed WP_013060079.1 gallidermin/nisin family lantibiotic [Bacillaceae]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0035)\n","Processed WP_184532092.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora polyrhachis]: non-AMP/OTHERS (Probabilities: 0.7205, 0.0549, 0.2246)\n","Processed WP_102917783.1 thiazolylpeptide-type bacteriocin [Streptomyces eurocidicus]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0040, 0.0036)\n","Processed WP_074842839.1 hypothetical protein [Priestia endophytica]: Neuropeptide (Probabilities: 0.0336, 0.0200, 0.9464)\n","Processed WP_082706110.1 gallidermin/nisin family lantibiotic [Aneurinibacillus]: non-AMP/OTHERS (Probabilities: 0.5054, 0.4839, 0.0107)\n","Processed WP_073827725.1 thiazolylpeptide-type bacteriocin [Micromonospora]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0039, 0.0040)\n","Processed WP_099886165.1 class II lantibiotic LanA [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.7363, 0.0335, 0.2302)\n","Processed WP_221313208.1 mersacidin/lichenicidin family type 2 lantibiotic [Granulicella aggregans]: non-AMP/OTHERS (Probabilities: 0.8853, 0.0137, 0.1010)\n","Processed WP_184478418.1 cinnamycin family lantibiotic [Saccharopolyspora gloriosae]: non-AMP/OTHERS (Probabilities: 0.5322, 0.0530, 0.4147)\n","Processed WP_184223223.1 mersacidin/lichenicidin family type 2 lantibiotic [Granulicella aggregans]: non-AMP/OTHERS (Probabilities: 0.8586, 0.0178, 0.1236)\n","Processed WP_000239231.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9445, 0.0494, 0.0060)\n","Processed WP_221163006.1 plantaricin C family lantibiotic [Staphylococcus caprae]: AMP (Probabilities: 0.0714, 0.9155, 0.0131)\n","Processed WP_184388380.1 cinnamycin family lantibiotic [Nocardiopsis composta]: AMP (Probabilities: 0.0987, 0.8884, 0.0130)\n","Processed WP_055513948.1 lexapeptide family class V lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9472, 0.0091, 0.0437)\n","Processed WP_157308863.1 mersacidin/lichenicidin family type 2 lantibiotic [Chitinophaga tropicalis]: Neuropeptide (Probabilities: 0.2934, 0.0695, 0.6371)\n","Processed WP_123097088.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9290, 0.0113, 0.0597)\n","Processed WP_208679317.1 lantibiotic dehydratase [Streptomyces sp. ID38640]: Neuropeptide (Probabilities: 0.2883, 0.0258, 0.6859)\n","Processed WP_002887196.1 type A2 lantipeptide [Streptococcus salivarius]: non-AMP/OTHERS (Probabilities: 0.8151, 0.1751, 0.0098)\n","Processed WP_220725165.1 gallidermin/nisin family lantibiotic, partial [Clostridium estertheticum]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0034, 0.0042)\n","Processed WP_171299090.1 gallidermin/nisin family lantibiotic [Clostridium estertheticum]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0052, 0.0032)\n","Processed WP_171299247.1 gallidermin/nisin family lantibiotic [Clostridium estertheticum]: non-AMP/OTHERS (Probabilities: 0.9858, 0.0112, 0.0030)\n","Processed WP_113688119.1 thiazolylpeptide-type bacteriocin [unclassified Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0043, 0.0039)\n","Processed WP_121802848.1 thiazolylpeptide-type bacteriocin [Streptomyces griseocarneus]: non-AMP/OTHERS (Probabilities: 0.9845, 0.0125, 0.0030)\n","Processed MBX7072133.1 mersacidin/lichenicidin family type 2 lantibiotic [Pirellulales bacterium]: non-AMP/OTHERS (Probabilities: 0.9884, 0.0070, 0.0046)\n","Processed MBX6766949.1 cinnamycin family lantibiotic [Actinomadura rubrobrunea]: non-AMP/OTHERS (Probabilities: 0.9845, 0.0103, 0.0052)\n","Processed MBX6341530.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermomicrobiaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0037, 0.0040)\n","Processed MBX6342569.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermomicrobiaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.7415, 0.0202, 0.2383)\n","Processed MBX5451287.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermogemmatispora sp.]: Neuropeptide (Probabilities: 0.0175, 0.0128, 0.9697)\n","Processed WP_077323882.1 gallidermin/nisin family lantibiotic [Bifidobacterium longum]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0049, 0.0038)\n","Processed WP_220707766.1 plantaricin C family lantibiotic, partial [Clostridium estertheticum]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0056, 0.0029)\n","Processed WP_220668385.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.8844, 0.0120, 0.1036)\n","Processed EAA1363229.1 lantibiotic ABC transporter ATP-binding protein [Shigella flexneri]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0055, 0.0044)\n","Processed WP_207126352.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinocatenispora comari]: Neuropeptide (Probabilities: 0.0105, 0.0106, 0.9789)\n","Processed WP_035858840.1 thiazolylpeptide-type bacteriocin [Streptomycetaceae]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0039)\n","Processed WP_182543085.1 lantibiotic dehydratase [Saccharopolyspora lacisalsi]: Neuropeptide (Probabilities: 0.1750, 0.0272, 0.7979)\n","Processed WP_169985488.1 cinnamycin family lantibiotic [unclassified Microbispora]: non-AMP/OTHERS (Probabilities: 0.9896, 0.0037, 0.0067)\n","Processed WP_107643592.1 mersacidin family lantibiotic [Staphylococcus saprophyticus]: non-AMP/OTHERS (Probabilities: 0.6075, 0.3150, 0.0775)\n","Processed WP_061642391.1 two-peptide bacteriocin subunit PneA2 [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9794, 0.0153, 0.0053)\n","Processed WP_107643591.1 type 2 lantibiotic [Staphylococcus saprophyticus]: non-AMP/OTHERS (Probabilities: 0.9805, 0.0159, 0.0035)\n","Processed MBX3012719.1 mersacidin/lichenicidin family type 2 lantibiotic [Caldilineaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9279, 0.0202, 0.0519)\n","Processed WP_050204369.1 hypothetical protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0046, 0.0029)\n","Processed MBX3188543.1 mersacidin/lichenicidin family type 2 lantibiotic [Labilithrix sp.]: non-AMP/OTHERS (Probabilities: 0.8957, 0.0184, 0.0859)\n","Processed WP_109031158.1 SCO0268 family class II lantipeptide [Streptomyces rubrogriseus]: non-AMP/OTHERS (Probabilities: 0.5007, 0.0319, 0.4674)\n","Processed WP_182475632.1 plantaricin C family lantibiotic [Staphylococcus gallinarum]: non-AMP/OTHERS (Probabilities: 0.9744, 0.0078, 0.0178)\n","Processed WP_118895795.1 gallidermin/nisin family lantibiotic [Lactobacillus bombicola]: AMP (Probabilities: 0.0649, 0.9197, 0.0154)\n","Processed WP_182055602.1 lantibiotic dehydratase, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0044, 0.0028)\n","Processed WP_119569881.1 plantaricin C family lantibiotic [Staphylococcus succinus]: AMP (Probabilities: 0.0888, 0.8935, 0.0177)\n","Processed WP_180746765.1 gallidermin/nisin family lantibiotic [Lactococcus laudensis]: non-AMP/OTHERS (Probabilities: 0.9265, 0.0576, 0.0160)\n","Processed WP_081605422.1 gallidermin/nisin family lantibiotic [Bacillus subtilis group]: AMP (Probabilities: 0.0707, 0.9144, 0.0149)\n","Processed WP_167744800.1 SCO0268 family class II lantipeptide [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.6909, 0.0223, 0.2868)\n","Processed WP_148876624.1 plantaricin C family lantibiotic [Streptococcus]: AMP (Probabilities: 0.0860, 0.9015, 0.0125)\n","Processed WP_138702611.1 hypothetical protein [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9877, 0.0089, 0.0033)\n","Processed WP_132635289.1 mersacidin/lichenicidin family type 2 lantibiotic [Nonomuraea mesophila]: AMP (Probabilities: 0.1126, 0.8750, 0.0125)\n","Processed WP_132635292.1 mersacidin/lichenicidin family type 2 lantibiotic [Nonomuraea mesophila]: non-AMP/OTHERS (Probabilities: 0.7931, 0.0468, 0.1601)\n","Processed WP_132517964.1 lantibiotic dehydratase [Nonomuraea diastatica]: Neuropeptide (Probabilities: 0.0189, 0.0130, 0.9681)\n","Processed WP_053760578.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0047, 0.0030)\n","Processed WP_059302729.1 SCO0268 family class II lantipeptide [Streptomyces]: Neuropeptide (Probabilities: 0.1178, 0.0239, 0.8583)\n","Processed WP_139212035.1 gallidermin/nisin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9202, 0.0756, 0.0041)\n","Processed WP_144028785.1 elgicin/penisin family lantibiotic [Paenibacillus tyrfis]: AMP (Probabilities: 0.0973, 0.8882, 0.0145)\n","Processed WP_076514037.1 mersacidin family lantibiotic [Priestia flexa]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0034)\n","Processed WP_081890515.1 gallidermin/nisin family lantibiotic [Paenibacillus tyrfis]: AMP (Probabilities: 0.4065, 0.5806, 0.0129)\n","Processed WP_144028786.1 elgicin/penisin family lantibiotic [Paenibacillus tyrfis]: AMP (Probabilities: 0.1655, 0.7857, 0.0488)\n","Processed WP_016129905.1 gallidermin/nisin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9884, 0.0039, 0.0077)\n","Processed WP_052587026.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: AMP (Probabilities: 0.0768, 0.9107, 0.0125)\n","Processed WP_000181086.1 two-peptide bacteriocin subunit PneA1 [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0071, 0.0024)\n","Processed WP_115246312.1 ATP-binding cassette domain-containing protein [Streptococcus dysgalactiae]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0045, 0.0035)\n","Processed WP_031222508.1 hypothetical protein [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0063, 0.0030)\n","Processed EAB9261907.1 lantibiotic ABC transporter ATP-binding protein [Shigella flexneri]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0064, 0.0045)\n","Processed WP_115245854.1 type A2 lantipeptide [Streptococcus dysgalactiae]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0037, 0.0061)\n","Processed WP_220233556.1 lantibiotic dehydratase family protein [Apibacter sp. ESL0432]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0030)\n","Processed WP_048665550.1 type A2 lantipeptide [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0034, 0.0045)\n","Processed WP_048520747.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9412, 0.0524, 0.0064)\n","Processed WP_220264060.1 lantibiotic dehydratase family protein [Apibacter sp. ESL0404]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0051, 0.0027)\n","Processed WP_000447118.1 type A2 lantipeptide [Bacillus cereus group]: AMP (Probabilities: 0.0698, 0.9160, 0.0142)\n","Processed WP_047212565.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0032, 0.0046)\n","Processed WP_115343615.1 plantaricin C family lantibiotic [Staphylococcus saprophyticus]: non-AMP/OTHERS (Probabilities: 0.8563, 0.0227, 0.1210)\n","Processed WP_037601350.1 type A2 lantipeptide [Streptococcus salivarius]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0037)\n","Processed WP_037601366.1 type A2 lantipeptide [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9860, 0.0057, 0.0082)\n","Processed WP_119624632.1 class II lanthipeptide, LchA2/BrtA2 family [Staphylococcus gallinarum]: AMP (Probabilities: 0.0833, 0.9026, 0.0141)\n","Processed MBW8879536.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.9501, 0.0150, 0.0349)\n","Processed MBW8877971.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.4907, 0.1386, 0.3707)\n","Processed WP_037601592.1 lantibiotic streptin [Streptococcus]: AMP (Probabilities: 0.0722, 0.9153, 0.0125)\n","Processed WP_025195114.1 type A2 lantipeptide [Streptococcus agalactiae]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0035)\n","Processed EFA1345489.1 lantibiotic ABC transporter ATP-binding protein [Escherichia coli]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0046, 0.0038)\n","Processed MBW8876143.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.9360, 0.0132, 0.0507)\n","Processed MBW8879538.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: Neuropeptide (Probabilities: 0.4452, 0.0534, 0.5013)\n","Processed MBW8879537.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.9562, 0.0132, 0.0307)\n","Processed MBW8873961.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.9030, 0.0349, 0.0621)\n","Processed MBW8875750.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.7765, 0.1429, 0.0806)\n","Processed WP_220210053.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei]: non-AMP/OTHERS (Probabilities: 0.5567, 0.2360, 0.2073)\n","Processed WP_220203592.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei]: non-AMP/OTHERS (Probabilities: 0.9313, 0.0344, 0.0343)\n","Processed WP_220209762.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei]: Neuropeptide (Probabilities: 0.2489, 0.0243, 0.7268)\n","Processed WP_000378912.1 hypothetical protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0054, 0.0044)\n","Processed WP_018515646.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0033)\n","Processed WP_037601371.1 type A2 lantipeptide [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0039, 0.0042)\n","Processed WP_220203593.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei]: non-AMP/OTHERS (Probabilities: 0.6743, 0.0932, 0.2325)\n","Processed WP_220209760.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei]: Neuropeptide (Probabilities: 0.0594, 0.0232, 0.9174)\n","Processed WP_220203591.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei]: non-AMP/OTHERS (Probabilities: 0.6654, 0.0884, 0.2462)\n","Processed WP_220209763.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei]: Neuropeptide (Probabilities: 0.0687, 0.0243, 0.9070)\n","Processed WP_220194733.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonospora formicarum]: Neuropeptide (Probabilities: 0.0438, 0.0224, 0.9338)\n","Processed WP_220197587.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonospora formicarum]: non-AMP/OTHERS (Probabilities: 0.3954, 0.3631, 0.2414)\n","Processed WP_220209761.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei]: Neuropeptide (Probabilities: 0.0149, 0.0122, 0.9729)\n","Processed WP_181749955.1 plantaricin C family lantibiotic [Paenactinomyces guangxiensis]: AMP (Probabilities: 0.1296, 0.8557, 0.0147)\n","Processed WP_220199012.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonospora formicarum]: non-AMP/OTHERS (Probabilities: 0.4639, 0.1576, 0.3785)\n","Processed WP_220199013.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonospora formicarum]: Neuropeptide (Probabilities: 0.4548, 0.0373, 0.5080)\n","Processed WP_181749956.1 class II lanthipeptide, LchA2/BrtA2 family [Paenactinomyces guangxiensis]: Neuropeptide (Probabilities: 0.1385, 0.0480, 0.8136)\n","Processed EFW8602214.1 lantibiotic ABC transporter ATP-binding protein [Shigella flexneri]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0056, 0.0047)\n","Processed WP_108965343.1 lichenicidin alpha family lanthipeptide [unclassified Actinomyces]: Neuropeptide (Probabilities: 0.4686, 0.0434, 0.4880)\n","Processed WP_201425282.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: non-AMP/OTHERS (Probabilities: 0.5722, 0.0335, 0.3944)\n","Processed WP_112588775.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora noduli]: AMP (Probabilities: 0.0945, 0.8921, 0.0133)\n","Processed WP_201425283.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: Neuropeptide (Probabilities: 0.3516, 0.1235, 0.5249)\n","Processed WP_052826974.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus velezensis]: AMP (Probabilities: 0.0783, 0.9091, 0.0126)\n","Processed WP_112603060.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora]: AMP (Probabilities: 0.0891, 0.8976, 0.0133)\n","Processed WP_111601318.1 plantaricin C family lantibiotic [Streptomyces sp. Amel2xB2]: AMP (Probabilities: 0.0882, 0.8993, 0.0126)\n","Processed WP_111621309.1 gallidermin/nisin family lantibiotic [Paenibacillus pabuli]: Neuropeptide (Probabilities: 0.4626, 0.0618, 0.4757)\n","Processed WP_201422437.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: Neuropeptide (Probabilities: 0.0394, 0.0210, 0.9396)\n","Processed WP_111314546.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.5264, 0.0327, 0.4409)\n","Processed WP_081047461.1 gallidermin/nisin family lantibiotic [Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.8945, 0.0464, 0.0591)\n","Processed WP_113709219.1 mersacidin family lantibiotic [Bacillus sp. SRB_8]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0032, 0.0034)\n","Processed WP_181434431.1 plantaricin C family lantibiotic [Curtobacterium sp. MCLR17_036]: AMP (Probabilities: 0.1001, 0.8862, 0.0137)\n","Processed WP_111138153.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9055, 0.0473, 0.0472)\n","Processed WP_043858027.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9752, 0.0077, 0.0170)\n","Processed WP_220038417.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus plantarum]: AMP (Probabilities: 0.0859, 0.9020, 0.0121)\n","Processed WP_111079677.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.4411, 0.0290, 0.5299)\n","Processed WP_111007059.1 thiazolylpeptide-type bacteriocin [Streptomyces sp. NTH33]: non-AMP/OTHERS (Probabilities: 0.9813, 0.0150, 0.0037)\n","Processed WP_218186923.1 mersacidin family lantibiotic, partial [Bacillus sonorensis]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0032, 0.0044)\n","Processed WP_111131931.1 plantaricin C family lantibiotic [Staphylococcus aureus]: AMP (Probabilities: 0.1152, 0.8511, 0.0337)\n","Processed WP_071640090.1 FDLD family class I lanthipeptide [Paenibacillus]: AMP (Probabilities: 0.0744, 0.9129, 0.0127)\n","Processed WP_218976268.1 mersacidin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0041, 0.0042)\n","Processed WP_217492677.1 mersacidin family lantibiotic, partial [Bacillus cereus]: AMP (Probabilities: 0.2148, 0.7741, 0.0112)\n","Processed WP_218186924.1 mersacidin family lantibiotic, partial [Bacillus sonorensis]: non-AMP/OTHERS (Probabilities: 0.5289, 0.0184, 0.4527)\n","Processed WP_176453815.1 mersacidin family lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.1578, 0.8283, 0.0139)\n","Processed WP_013450132.1 mersacidin family lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.4270, 0.5628, 0.0102)\n","Processed WP_141066300.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.0155, 0.0116, 0.9729)\n","Processed WP_074450945.1 type A2 lantipeptide [Streptococcus equinus]: non-AMP/OTHERS (Probabilities: 0.9746, 0.0070, 0.0183)\n","Processed WP_080990964.1 mersacidin family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0029, 0.0074)\n","Processed WP_013450123.1 lichenicidin A2 family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9829, 0.0140, 0.0031)\n","Processed WP_142322465.1 gallidermin/nisin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.8714, 0.1228, 0.0058)\n","Processed WP_013450121.1 lichenicidin A2 family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0042, 0.0069)\n","Processed WP_018555848.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0038, 0.0033)\n","Processed WP_013450119.1 lichenicidin A2 family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0066, 0.0031)\n","Processed WP_013450118.1 lichenicidin A2 family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0051, 0.0033)\n","Processed WP_013450122.1 lichenicidin A2 family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9880, 0.0082, 0.0039)\n","Processed WP_013450120.1 lichenicidin A2 family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9896, 0.0070, 0.0034)\n","Processed WP_013450111.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0041, 0.0036)\n","Processed WP_013450103.1 LuxR C-terminal-related transcriptional regulator [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0041, 0.0034)\n","Processed WP_012818117.1 class II lanthipeptide, LchA2/BrtA2 family [Staphylococcus epidermidis]: AMP (Probabilities: 0.1424, 0.8379, 0.0197)\n","Processed WP_219947866.1 lantibiotic biosynthesis protein, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0048, 0.0031)\n","Processed WP_216885050.1 mersacidin family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0032)\n","Processed WP_216885049.1 mersacidin family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0031, 0.0043)\n","Processed WP_216885132.1 mersacidin family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0028, 0.0051)\n","Processed WP_012818083.1 plantaricin C family lantibiotic [Staphylococcus epidermidis]: Neuropeptide (Probabilities: 0.0753, 0.0293, 0.8954)\n","Processed WP_216454567.1 mersacidin family lantibiotic, partial [Enterococcus sp. S183_ASV_20]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0047, 0.0055)\n","Processed WP_214971313.1 mersacidin family lantibiotic [Staphylococcus coagulans]: AMP (Probabilities: 0.2114, 0.7731, 0.0155)\n","Processed WP_214725496.1 mersacidin family lantibiotic [unclassified Exiguobacterium]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0035, 0.0056)\n","Processed WP_214691660.1 mersacidin family lantibiotic [unclassified Exiguobacterium]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0041, 0.0056)\n","Processed WP_216885128.1 mersacidin family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0039, 0.0036)\n","Processed WP_216442131.1 mersacidin family lantibiotic, partial [Enterococcus sp. S159_ASV_20]: non-AMP/OTHERS (Probabilities: 0.9740, 0.0043, 0.0218)\n","Processed WP_215178781.1 mersacidin family lantibiotic [Paenibacillus albidus]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0030, 0.0062)\n","Processed WP_215178779.1 mersacidin family lantibiotic [Paenibacillus albidus]: non-AMP/OTHERS (Probabilities: 0.5087, 0.4711, 0.0202)\n","Processed WP_210062333.1 lantibiotic dehydratase, partial [Mycobacterium tuberculosis]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0037, 0.0034)\n","Processed WP_208861136.1 lantibiotic dehydratase, partial [Streptomyces atroolivaceus]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0053, 0.0031)\n","Processed WP_211159114.1 mersacidin family lantibiotic, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9877, 0.0098, 0.0024)\n","Processed WP_214691659.1 mersacidin family lantibiotic [unclassified Exiguobacterium]: AMP (Probabilities: 0.1088, 0.8725, 0.0188)\n","Processed WP_219913713.1 lantibiotic immunity ABC transporter MutG family permease subunit, partial [Bacillus paranthracis]: non-AMP/OTHERS (Probabilities: 0.9874, 0.0098, 0.0028)\n","Processed WP_211161576.1 MFS transporter, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0073, 0.0026)\n","Processed WP_211159198.1 mersacidin family lantibiotic, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9847, 0.0128, 0.0025)\n","Processed WP_121614376.1 mersacidin/lichenicidin family type 2 lantibiotic [Mesobacillus foraminis]: non-AMP/OTHERS (Probabilities: 0.5264, 0.0518, 0.4218)\n","Processed WP_107466734.1 lexapeptide family class V lantibiotic [Streptomyces sp. MA5143a]: non-AMP/OTHERS (Probabilities: 0.8300, 0.0149, 0.1551)\n","Processed WP_106311337.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Chamaesiphon polymorphus]: Neuropeptide (Probabilities: 0.0398, 0.0190, 0.9411)\n","Processed WP_219840867.1 lichenicidin A2 family type 2 lantibiotic [Paeniclostridium sordellii]: non-AMP/OTHERS (Probabilities: 0.9834, 0.0046, 0.0120)\n","Processed WP_062746560.1 gallidermin/nisin family lantibiotic [Bacillaceae]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0047, 0.0030)\n","Processed WP_208954849.1 lichenicidin alpha family lanthipeptide [Dolosigranulum pigrum]: AMP (Probabilities: 0.1278, 0.8582, 0.0140)\n","Processed KMR20294.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0037)\n","Processed WP_140555544.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium sp. UTCIF-39]: AMP (Probabilities: 0.0690, 0.9174, 0.0137)\n","Processed WP_104681709.1 plantaricin C family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.5244, 0.4528, 0.0228)\n","Processed WP_119102891.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sporangiiformans]: non-AMP/OTHERS (Probabilities: 0.7896, 0.1395, 0.0709)\n","Processed WP_002269377.1 type A2 lantipeptide [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0029, 0.0047)\n","Processed WP_015907611.1 plantaricin C family lantibiotic [Caldicellulosiruptor]: AMP (Probabilities: 0.0800, 0.9081, 0.0119)\n","Processed WP_002268661.1 type A2 lantipeptide [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9869, 0.0058, 0.0073)\n","Processed WP_000432035.1 hypothetical protein [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0038)\n","Processed WP_000469595.1 hypothetical protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0065, 0.0025)\n","Processed WP_202500327.1 lantibiotic dehydratase [Streptomyces sp. SID5476]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0032, 0.0045)\n","Processed WP_204730789.1 mersacidin family lantibiotic [Jeotgalibacillus terrae]: AMP (Probabilities: 0.3795, 0.5988, 0.0217)\n","Processed WP_204279792.1 lantibiotic dehydratase, partial [Serratia marcescens]: non-AMP/OTHERS (Probabilities: 0.8576, 0.0719, 0.0704)\n","Processed WP_204268481.1 lantibiotic dehydratase, partial [Escherichia coli]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0041, 0.0069)\n","Processed WP_203091009.1 mersacidin family lantibiotic [Alkalihalobacillus gibsonii]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0036, 0.0050)\n","Processed WP_203008424.1 lantibiotic dehydratase [unclassified Frankia]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0042, 0.0056)\n","Processed WP_203008435.1 lantibiotic dehydratase [unclassified Frankia]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0039, 0.0039)\n","Processed WP_203090953.1 mersacidin family lantibiotic [Alkalihalobacillus gibsonii]: non-AMP/OTHERS (Probabilities: 0.9835, 0.0085, 0.0080)\n","Processed WP_203091008.1 mersacidin family lantibiotic [Alkalihalobacillus gibsonii]: non-AMP/OTHERS (Probabilities: 0.9627, 0.0066, 0.0308)\n","Processed WP_201069645.1 mersacidin family lantibiotic [Bacillus sp. TH13]: non-AMP/OTHERS (Probabilities: 0.9884, 0.0082, 0.0034)\n","Processed WP_201069644.1 mersacidin family lantibiotic [Bacillus sp. TH13]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0037, 0.0037)\n","Processed WP_199816679.1 lantibiotic dehydratase, partial [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0066, 0.0038)\n","Processed WP_199691883.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9893, 0.0079, 0.0028)\n","Processed WP_206649102.1 gallidermin family lantibiotic [Bifidobacterium]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0045, 0.0031)\n","Processed WP_202270760.1 lantibiotic dehydratase [Chryseobacterium sp. KMC2]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0051, 0.0038)\n","Processed WP_199679073.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9193, 0.0162, 0.0646)\n","Processed WP_199679173.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9481, 0.0482, 0.0037)\n","Processed WP_198981730.1 lantibiotic dehydratase, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0040, 0.0029)\n","Processed WP_199679187.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9025, 0.0837, 0.0138)\n","Processed WP_194734578.1 lantibiotic dehydratase, partial [Streptococcus suis]: Neuropeptide (Probabilities: 0.0382, 0.0220, 0.9397)\n","Processed WP_196300200.1 mersacidin family lantibiotic, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0055, 0.0025)\n","Processed WP_197480372.1 gallidermin family lantibiotic [Peptidiphaga gingivicola]: non-AMP/OTHERS (Probabilities: 0.9477, 0.0489, 0.0034)\n","Processed WP_196300662.1 mersacidin family lantibiotic, partial [Streptococcus pneumoniae]: AMP (Probabilities: 0.0843, 0.9023, 0.0135)\n","Processed WP_195318324.1 mersacidin family lantibiotic [Weissella cibaria]: AMP (Probabilities: 0.1252, 0.8620, 0.0127)\n","Processed WP_198435081.1 mersacidin family lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0034, 0.0059)\n","Processed WP_198010258.1 gallidermin family lantibiotic [Actinobaculum sp. oral taxon 183]: AMP (Probabilities: 0.0898, 0.8888, 0.0215)\n","Processed WP_193674184.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.7388, 0.2504, 0.0108)\n","Processed WP_193674183.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.7528, 0.2212, 0.0260)\n","Processed WP_193587759.1 mersacidin family lantibiotic, partial [Bacilli]: non-AMP/OTHERS (Probabilities: 0.9578, 0.0389, 0.0033)\n","Processed WP_167542329.1 mersacidin/lichenicidin family type 2 lantibiotic [Rathayibacter tritici]: Neuropeptide (Probabilities: 0.2188, 0.0452, 0.7360)\n","Processed WP_193441134.1 lantibiotic dehydratase, partial [Streptococcus suis]: Neuropeptide (Probabilities: 0.3604, 0.0310, 0.6086)\n","Processed WP_103653058.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9529, 0.0437, 0.0034)\n","Processed WP_196300073.1 MFS transporter, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0084, 0.0028)\n","Processed WP_146068594.1 gallidermin family lantibiotic [Brevibacillus laterosporus]: non-AMP/OTHERS (Probabilities: 0.7655, 0.0276, 0.2069)\n","Processed WP_198644701.1 lantibiotic dehydratase, partial [Staphylococcus epidermidis]: non-AMP/OTHERS (Probabilities: 0.9883, 0.0080, 0.0037)\n","Processed WP_103653018.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0060, 0.0040)\n","Processed WP_103652896.1 lichenicidin A2 family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0053, 0.0044)\n","Processed WP_030709583.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces]: AMP (Probabilities: 0.0727, 0.9143, 0.0130)\n","Processed WP_023987834.1 FDLD family class I lanthipeptide [Paenibacillus]: AMP (Probabilities: 0.0687, 0.9178, 0.0135)\n","Processed WP_053512377.1 plantaricin C family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.0978, 0.8882, 0.0140)\n","Processed WP_053512376.1 plantaricin C family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.1115, 0.8758, 0.0127)\n","Processed WP_000368259.1 type A2 lantipeptide [Bacillus thuringiensis]: AMP (Probabilities: 0.1121, 0.8607, 0.0271)\n","Processed WP_219715586.1 lantibiotic (srt) production protein, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0034)\n","Processed WP_101501474.1 plantaricin C family lantibiotic [Bacillus]: AMP (Probabilities: 0.0681, 0.9178, 0.0141)\n","Processed WP_030709581.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces]: AMP (Probabilities: 0.1464, 0.8315, 0.0220)\n","Processed WP_202898890.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus intermedius]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0043)\n","Processed WP_180896587.1 mersacidin family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0035, 0.0041)\n","Processed WP_180896589.1 mersacidin family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9905, 0.0034, 0.0061)\n","Processed WP_184793070.1 lantibiotic dehydratase [Phytomonospora endophytica]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed WP_010063176.1 lexapeptide family class V lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9839, 0.0048, 0.0113)\n","Processed WP_180958686.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9581, 0.0166, 0.0253)\n","Processed WP_193287077.1 gallidermin family lantibiotic [Vibrio parahaemolyticus]: non-AMP/OTHERS (Probabilities: 0.9872, 0.0058, 0.0070)\n","Processed WP_180896588.1 mersacidin family lantibiotic [Priestia megaterium]: AMP (Probabilities: 0.3662, 0.6232, 0.0106)\n","Processed WP_180896586.1 mersacidin family lantibiotic [Priestia megaterium]: AMP (Probabilities: 0.1249, 0.8636, 0.0114)\n","Processed WP_010059803.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9840, 0.0125, 0.0035)\n","Processed WP_179811209.1 lantibiotic dehydratase [Nocardiopsis arvandica]: AMP (Probabilities: 0.3989, 0.5388, 0.0624)\n","Processed WP_179144982.1 lantibiotic dehydratase, partial [Escherichia coli]: non-AMP/OTHERS (Probabilities: 0.9316, 0.0604, 0.0080)\n","Processed WP_179140026.1 lantibiotic dehydratase [Chryseobacterium sp. YR221]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0055, 0.0034)\n","Processed WP_176544348.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0047, 0.0041)\n","Processed WP_179191745.1 mersacidin family lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.1258, 0.8463, 0.0280)\n","Processed WP_179023465.1 lantibiotic dehydratase [Streptomyces sp. IMTB 2501]: non-AMP/OTHERS (Probabilities: 0.9460, 0.0399, 0.0141)\n","Processed WP_176271320.1 mersacidin family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.6113, 0.3780, 0.0107)\n","Processed WP_176504929.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0035, 0.0031)\n","Processed WP_171351795.1 gallidermin family lantibiotic [Vibrio coralliilyticus]: non-AMP/OTHERS (Probabilities: 0.9547, 0.0352, 0.0101)\n","Processed WP_178964733.1 lantibiotic dehydratase, partial [Escherichia marmotae]: non-AMP/OTHERS (Probabilities: 0.7280, 0.0669, 0.2051)\n","Processed WP_171346436.1 gallidermin family lantibiotic [Vibrio coralliilyticus]: non-AMP/OTHERS (Probabilities: 0.9465, 0.0470, 0.0065)\n","Processed WP_163170585.1 mersacidin family lantibiotic, partial [Bacillus paranthracis]: non-AMP/OTHERS (Probabilities: 0.9749, 0.0220, 0.0031)\n","Processed WP_157462717.1 mersacidin family lantibiotic, partial [Bacillus sp. JH7]: non-AMP/OTHERS (Probabilities: 0.9149, 0.0810, 0.0041)\n","Processed WP_159863657.1 mersacidin family lantibiotic [Bacillus sp. ZZV12-4809]: non-AMP/OTHERS (Probabilities: 0.9778, 0.0166, 0.0056)\n","Processed WP_157696733.1 gallidermin family lantibiotic [Brevibacillus formosus]: AMP (Probabilities: 0.3497, 0.6203, 0.0300)\n","Processed  unnamed protein product [Unknown]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0042, 0.0029)\n","Processed WP_000426726.1 type A2 lantipeptide [Bacillus]: AMP (Probabilities: 0.3454, 0.3752, 0.2794)\n","Processed WP_155417096.1 mersacidin family lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0981, 0.8886, 0.0132)\n","Processed WP_150008372.1 gallidermin family lantibiotic, partial [Staphylococcus aureus]: AMP (Probabilities: 0.0936, 0.8945, 0.0120)\n","Processed WP_155293891.1 gallidermin family lantibiotic [Rhodococcus fascians]: non-AMP/OTHERS (Probabilities: 0.9364, 0.0097, 0.0539)\n","Processed WP_151614197.1 lichenicidin alpha family lanthipeptide [Bacillus thuringiensis]: AMP (Probabilities: 0.4653, 0.5078, 0.0269)\n","Processed WP_187416707.1 ABC transporter permease, partial [Streptococcus dysgalactiae]: non-AMP/OTHERS (Probabilities: 0.9850, 0.0122, 0.0028)\n","Processed WP_148500019.1 mersacidin family lantibiotic [Paenibacillus ihumii]: AMP (Probabilities: 0.1481, 0.8402, 0.0117)\n","Processed WP_143806301.1 gallidermin family lantibiotic [Pantoea latae]: Neuropeptide (Probabilities: 0.0286, 0.0204, 0.9511)\n","Processed WP_143344272.1 mersacidin family lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9810, 0.0037, 0.0154)\n","Processed WP_142338454.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0035, 0.0031)\n","Processed WP_142323754.1 mersacidin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed WP_187417555.1 ABC transporter permease, partial [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0065, 0.0031)\n","Processed WP_158326862.1 gallidermin family lantibiotic [Brevibacillus laterosporus]: AMP (Probabilities: 0.0757, 0.9118, 0.0125)\n","Processed WP_143527439.1 gallidermin family lantibiotic [Rubrobacter xylanophilus]: non-AMP/OTHERS (Probabilities: 0.9636, 0.0239, 0.0125)\n","Processed WP_140420061.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9705, 0.0265, 0.0030)\n","Processed WP_139353052.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0050, 0.0038)\n","Processed WP_137053910.1 lichenicidin alpha family lanthipeptide, partial [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9871, 0.0093, 0.0037)\n","Processed WP_139889503.1 lichenicidin alpha family lanthipeptide, partial [Bacillus tropicus]: Neuropeptide (Probabilities: 0.0425, 0.0221, 0.9355)\n","Processed WP_119733793.1 gallidermin family lantibiotic [Brevibacillus laterosporus]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0039, 0.0039)\n","Processed WP_129044617.1 mersacidin family lantibiotic [Companilactobacillus metriopterae]: AMP (Probabilities: 0.0685, 0.9174, 0.0141)\n","Processed WP_116777443.1 lichenicidin alpha family lanthipeptide [Bacillus]: AMP (Probabilities: 0.0682, 0.9168, 0.0150)\n","Processed WP_115856393.1 mersacidin family lantibiotic [Staphylococcus felis]: non-AMP/OTHERS (Probabilities: 0.9624, 0.0097, 0.0279)\n","Processed WP_106839070.1 gallidermin family lantibiotic [Brevibacillus fortis]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0038, 0.0032)\n","Processed WP_106383561.1 mersacidin family lantibiotic [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9411, 0.0462, 0.0127)\n","Processed WP_141534179.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0061, 0.0044)\n","Processed WP_114672738.1 gallidermin family lantibiotic [Deinococcus wulumuqiensis]: non-AMP/OTHERS (Probabilities: 0.7210, 0.2117, 0.0673)\n","Processed WP_108466984.1 gallidermin family lantibiotic [Paenibacillus sp. CAA11]: non-AMP/OTHERS (Probabilities: 0.8724, 0.1146, 0.0130)\n","Processed WP_103146399.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9450, 0.0106, 0.0444)\n","Processed WP_103061647.1 mersacidin family lantibiotic [Actinomyces qiguomingii]: AMP (Probabilities: 0.0772, 0.9101, 0.0128)\n","Processed WP_103048208.1 FDLD family class I lanthipeptide [Paenibacillus]: AMP (Probabilities: 0.0695, 0.9173, 0.0132)\n","Processed WP_098775162.1 lichenicidin alpha family lanthipeptide [Bacillus cereus]: Neuropeptide (Probabilities: 0.2423, 0.0339, 0.7239)\n","Processed WP_102205352.1 mersacidin/lichenicidin family type 2 lantibiotic [Fischerella muscicola]: non-AMP/OTHERS (Probabilities: 0.9789, 0.0113, 0.0098)\n","Processed WP_141552552.1 lichenicidin alpha family lanthipeptide, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0927, 0.8877, 0.0196)\n","Processed WP_111762440.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.0104, 0.0104, 0.9792)\n","Processed WP_102210125.1 lichenicidin alpha family lanthipeptide [Streptococcus sp. UMB0029]: AMP (Probabilities: 0.1173, 0.8705, 0.0123)\n","Processed PMB50486.1 mersacidin/lichenicidin family type 2 lantibiotic [Fischerella thermalis CCMEE 5201 CCMEE 5201]: non-AMP/OTHERS (Probabilities: 0.9880, 0.0079, 0.0041)\n","Processed WP_098328036.1 lichenicidin alpha family lanthipeptide, partial [Bacillus cereus]: Neuropeptide (Probabilities: 0.0919, 0.0245, 0.8836)\n","Processed WP_102275103.1 mersacidin family lantibiotic [Bacillus massiliigabonensis]: non-AMP/OTHERS (Probabilities: 0.9055, 0.0224, 0.0721)\n","Processed WP_098854239.1 lichenicidin alpha family lanthipeptide, partial [Bacillus toyonensis]: Neuropeptide (Probabilities: 0.0111, 0.0107, 0.9783)\n","Processed WP_141533480.1 lichenicidin alpha family lanthipeptide, partial [Bacillus cereus]: Neuropeptide (Probabilities: 0.0138, 0.0120, 0.9742)\n","Processed WP_142321224.1 lichenicidin alpha family lanthipeptide, partial [Bacillus cereus]: Neuropeptide (Probabilities: 0.0658, 0.0270, 0.9072)\n","Processed WP_098160134.1 lichenicidin alpha family lanthipeptide, partial [Bacillus pseudomycoides]: Neuropeptide (Probabilities: 0.0345, 0.0217, 0.9438)\n","Processed WP_142316087.1 lichenicidin alpha family lanthipeptide, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.4291, 0.3483, 0.2227)\n","Processed WP_096187401.1 gallidermin family lantibiotic [Bacillus sp. FJAT-44876]: AMP (Probabilities: 0.3664, 0.5560, 0.0775)\n","Processed WP_142310732.1 lichenicidin alpha family lanthipeptide, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.1563, 0.8217, 0.0220)\n","Processed WP_141547190.1 lichenicidin alpha family lanthipeptide, partial [Bacillus cereus group]: AMP (Probabilities: 0.3796, 0.6020, 0.0184)\n","Processed WP_089200866.1 mersacidin family lantibiotic [Paenibacillus xerothermodurans]: non-AMP/OTHERS (Probabilities: 0.9687, 0.0278, 0.0035)\n","Processed WP_097787301.1 lichenicidin alpha family lanthipeptide, partial [Bacillus pseudomycoides]: Neuropeptide (Probabilities: 0.0124, 0.0114, 0.9762)\n","Processed WP_141537911.1 lichenicidin alpha family lanthipeptide, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.5791, 0.0517, 0.3692)\n","Processed WP_088364099.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9117, 0.0821, 0.0062)\n","Processed WP_088355571.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0067, 0.0043)\n","Processed WP_088310178.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.5963, 0.0226, 0.3811)\n","Processed WP_088342444.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9882, 0.0062, 0.0056)\n","Processed WP_088304856.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9273, 0.0680, 0.0048)\n","Processed WP_141523684.1 lichenicidin alpha family lanthipeptide, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.7244, 0.2504, 0.0251)\n","Processed WP_088302732.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8614, 0.1329, 0.0057)\n","Processed WP_142308427.1 lichenicidin alpha family lanthipeptide, partial [Bacillus pseudomycoides]: Neuropeptide (Probabilities: 0.0459, 0.0255, 0.9286)\n","Processed WP_088310001.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8880, 0.0148, 0.0972)\n","Processed WP_087433873.1 mersacidin family lantibiotic [Paenibacillus apiarius]: non-AMP/OTHERS (Probabilities: 0.7028, 0.0275, 0.2698)\n","Processed WP_087433875.1 mersacidin family lantibiotic [Paenibacillus apiarius]: AMP (Probabilities: 0.0680, 0.9174, 0.0146)\n","Processed WP_088297682.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0037, 0.0039)\n","Processed WP_086614240.1 gallidermin family lantibiotic [Actinomyces sp. CtC 72]: non-AMP/OTHERS (Probabilities: 0.9830, 0.0136, 0.0034)\n","Processed WP_082243223.1 gallidermin family lantibiotic [Staphylococcus sp. TE8]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0035)\n","Processed WP_081723725.1 mersacidin family lantibiotic, partial [Bacillus sonorensis]: AMP (Probabilities: 0.0694, 0.9163, 0.0143)\n","Processed WP_080843930.1 mersacidin family lantibiotic [Cytobacillus gottheilii]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0035, 0.0044)\n","Processed WP_179135233.1 lantibiotic dehydratase, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0029, 0.0065)\n","Processed WP_084559628.1 lichenicidin alpha family lanthipeptide [Corynebacterium phocae]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0039, 0.0038)\n","Processed WP_081723720.1 mersacidin family lantibiotic, partial [Bacillus sonorensis]: AMP (Probabilities: 0.0654, 0.9197, 0.0149)\n","Processed WP_080999157.1 mersacidin family lantibiotic [Trueperella pyogenes]: AMP (Probabilities: 0.0949, 0.8927, 0.0124)\n","Processed WP_094206507.1 lantibiotic salivaricin M precursor [Finegoldia magna]: AMP (Probabilities: 0.0687, 0.9179, 0.0133)\n","Processed WP_086413512.1 mersacidin family lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9842, 0.0124, 0.0034)\n","Processed WP_080843929.1 mersacidin family lantibiotic [Cytobacillus gottheilii]: non-AMP/OTHERS (Probabilities: 0.9832, 0.0056, 0.0113)\n","Processed WP_079988711.1 gallidermin family lantibiotic [Candidatus Protochlamydia sp. W-9]: non-AMP/OTHERS (Probabilities: 0.8983, 0.0938, 0.0080)\n","Processed WP_077317465.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0057, 0.0045)\n","Processed WP_069188315.1 mersacidin family lantibiotic [Paenibacillus sp. 79R4]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0034, 0.0048)\n","Processed WP_077317466.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9874, 0.0074, 0.0051)\n","Processed WP_067143630.1 mersacidin family lantibiotic [Oceanivirga salmonicida]: AMP (Probabilities: 0.1561, 0.8327, 0.0112)\n","Processed WP_074619571.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0057, 0.0040)\n","Processed WP_072581944.1 mersacidin family lantibiotic [Bacillus weihaiensis]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0064, 0.0032)\n","Processed WP_064212341.1 mersacidin family lantibiotic [Staphylococcus capitis]: non-AMP/OTHERS (Probabilities: 0.5132, 0.4129, 0.0740)\n","Processed WP_055970083.1 mersacidin family lantibiotic [Exiguobacterium]: AMP (Probabilities: 0.1242, 0.8626, 0.0132)\n","Processed WP_080494188.1 mersacidin family lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0794, 0.9076, 0.0130)\n","Processed WP_057985304.1 mersacidin family lantibiotic [Bacillus cecembensis]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0031, 0.0045)\n","Processed WP_048568715.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.6354, 0.3566, 0.0080)\n","Processed WP_047150429.1 mersacidin family lantibiotic [Aneurinibacillus tyrosinisolvens]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0041, 0.0036)\n","Processed WP_065359317.1 lichenicidin alpha family lanthipeptide [Streptococcus dysgalactiae]: Neuropeptide (Probabilities: 0.0134, 0.0117, 0.9749)\n","Processed WP_047150428.1 mersacidin family lantibiotic [Aneurinibacillus tyrosinisolvens]: non-AMP/OTHERS (Probabilities: 0.5482, 0.4421, 0.0097)\n","Processed WP_036811563.1 mersacidin family lantibiotic [Planomicrobium glaciei]: non-AMP/OTHERS (Probabilities: 0.9510, 0.0061, 0.0428)\n","Processed WP_036737554.1 mersacidin family lantibiotic [Paenibacillus zanthoxyli]: AMP (Probabilities: 0.3799, 0.6100, 0.0100)\n","Processed WP_025692768.1 mersacidin family lantibiotic [Paenibacillus zanthoxyli]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0045, 0.0028)\n","Processed WP_042766370.1 mersacidin family lantibiotic [Streptococcus sp. HSISM1]: AMP (Probabilities: 0.2287, 0.7610, 0.0103)\n","Processed WP_029953015.1 mersacidin family lantibiotic [Bacillus sp. H1a]: AMP (Probabilities: 0.1063, 0.8815, 0.0122)\n","Processed WP_002164696.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0032, 0.0061)\n","Processed WP_048958601.1 mersacidin family lantibiotic [Enterococcus faecalis]: AMP (Probabilities: 0.0695, 0.9166, 0.0139)\n","Processed WP_036811568.1 mersacidin family lantibiotic [Planomicrobium glaciei]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0069, 0.0033)\n","Processed WP_002164697.1 mersacidin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8665, 0.0360, 0.0975)\n","Processed WP_013076756.1 mersacidin family lantibiotic [Kyrpidia tusciae]: AMP (Probabilities: 0.3254, 0.6628, 0.0118)\n","Processed WP_004111352.1 type A2 lantipeptide [Gardnerella]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0034, 0.0041)\n","Processed WP_009454932.1 mersacidin/lichenicidin family type 2 lantibiotic [Fischerella thermalis]: non-AMP/OTHERS (Probabilities: 0.9736, 0.0172, 0.0092)\n","Processed WP_006731991.1 type A2 lantipeptide [Lactobacillus iners]: non-AMP/OTHERS (Probabilities: 0.5156, 0.0221, 0.4623)\n","Processed WP_003086770.1 lichenicidin alpha family lanthipeptide [Streptococcus ratti]: AMP (Probabilities: 0.0799, 0.9080, 0.0121)\n","Processed WP_219605574.1 lantibiotic nisin-A [Lactococcus laudensis]: AMP (Probabilities: 0.0840, 0.9041, 0.0119)\n","Processed WP_006731084.1 type A2 lantipeptide [Lactobacillus iners]: Neuropeptide (Probabilities: 0.3906, 0.0246, 0.5848)\n","Processed WP_025692287.1 mersacidin family lantibiotic [Paenibacillus zanthoxyli]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0033, 0.0053)\n","Processed WP_013076765.1 mersacidin family lantibiotic [Kyrpidia tusciae]: non-AMP/OTHERS (Probabilities: 0.9868, 0.0093, 0.0039)\n","Processed WP_199753706.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus]: Neuropeptide (Probabilities: 0.2698, 0.0291, 0.7011)\n","Processed WP_176718700.1 lantibiotic dehydratase, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9501, 0.0127, 0.0373)\n","Processed WP_180896585.1 mersacidin/lichenicidin family type 2 lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0033)\n","Processed WP_175482411.1 lantibiotic dehydratase [Thermoflavimicrobium dichotomicum]: non-AMP/OTHERS (Probabilities: 0.9876, 0.0058, 0.0066)\n","Processed WP_176471877.1 lantibiotic dehydratase, partial [Alkalihalobacillus clausii]: non-AMP/OTHERS (Probabilities: 0.7362, 0.0206, 0.2432)\n","Processed WP_173224003.1 lantibiotic dehydratase [Kroppenstedtia pulmonis]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0034)\n","Processed WP_107490302.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sp. CB02959]: Neuropeptide (Probabilities: 0.3131, 0.0250, 0.6618)\n","Processed WP_044812438.1 two-peptide bacteriocin subunit PneA1 [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0060, 0.0025)\n","Processed WP_002268652.1 lichenicidin alpha family lanthipeptide [Streptococcus mutans]: AMP (Probabilities: 0.0766, 0.9110, 0.0124)\n","Processed WP_100601676.1 SCO0268 family class II lantipeptide [Streptomyces sp. CB02959]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed WP_102035248.1 plantaricin C family lantibiotic [Rothia dentocariosa]: Neuropeptide (Probabilities: 0.3303, 0.0307, 0.6389)\n","Processed WP_102035247.1 plantaricin C family lantibiotic [Rothia dentocariosa]: non-AMP/OTHERS (Probabilities: 0.9870, 0.0094, 0.0036)\n","Processed WP_101723598.1 lichenicidin A2 family type 2 lantibiotic [Eggerthella timonensis]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0043, 0.0028)\n","Processed WP_101723599.1 lichenicidin A2 family type 2 lantibiotic [Eggerthella timonensis]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0041, 0.0068)\n","Processed WP_099045174.1 lichenicidin A2 family type 2 lantibiotic [Leuconostoc citreum]: non-AMP/OTHERS (Probabilities: 0.9868, 0.0045, 0.0087)\n","Processed WP_099045173.1 lichenicidin A2 family type 2 lantibiotic [Leuconostoc citreum]: AMP (Probabilities: 0.3763, 0.6120, 0.0116)\n","Processed WP_101603837.1 plantaricin C family lantibiotic [Aerococcus sanguinicola]: non-AMP/OTHERS (Probabilities: 0.6957, 0.2622, 0.0420)\n","Processed WP_032871261.1 plantaricin C family lantibiotic [Bacillus]: Neuropeptide (Probabilities: 0.0262, 0.0167, 0.9571)\n","Processed WP_157946658.1 ABC transporter permease, partial [Staphylococcus epidermidis]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0074, 0.0027)\n","Processed WP_155458752.1 lantibiotic dehydratase, partial [Streptococcus pneumoniae]: Neuropeptide (Probabilities: 0.3284, 0.1268, 0.5448)\n","Processed WP_005505332.1 gallidermin/nisin family lantibiotic [Rothia]: Neuropeptide (Probabilities: 0.0079, 0.0092, 0.9828)\n","Processed WP_156780671.1 lantibiotic dehydratase [Peptoniphilus sp. oral taxon 836]: non-AMP/OTHERS (Probabilities: 0.9620, 0.0332, 0.0048)\n","Processed  unnamed protein product [Unknown]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0053, 0.0050)\n","Processed WP_155458462.1 lantibiotic dehydratase, partial [Streptococcus pneumoniae]: Neuropeptide (Probabilities: 0.0747, 0.0321, 0.8932)\n","Processed WP_152424579.1 lantibiotic dehydratase, partial [Elizabethkingia meningoseptica]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0044, 0.0028)\n","Processed WP_133170842.1 ATP-binding cassette domain-containing protein, partial [Staphylococcus cohnii]: non-AMP/OTHERS (Probabilities: 0.9905, 0.0064, 0.0030)\n","Processed WP_143118065.1 lantibiotic dehydratase, partial [Alkalihalobacillus clausii]: non-AMP/OTHERS (Probabilities: 0.9905, 0.0065, 0.0030)\n","Processed WP_143092999.1 lantibiotic dehydratase, partial [Arsenicibacter rosenii]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0044, 0.0027)\n","Processed WP_088778878.1 two-peptide bacteriocin subunit PneA2 [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9772, 0.0106, 0.0122)\n","Processed WP_088839586.1 lantibiotic salivaricin M precursor [Streptococcus pneumoniae]: AMP (Probabilities: 0.0644, 0.9210, 0.0146)\n","Processed WP_088786036.1 two-peptide bacteriocin subunit PneA1 [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9831, 0.0145, 0.0024)\n","Processed WP_153059523.1 lantibiotic dehydratase, partial [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9773, 0.0168, 0.0059)\n","Processed WP_088805185.1 lantibiotic salivaricin M precursor [Streptococcus pneumoniae]: AMP (Probabilities: 0.0704, 0.9162, 0.0135)\n","Processed WP_070835453.1 plantaricin C family lantibiotic [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.9737, 0.0084, 0.0178)\n","Processed WP_142359711.1 lantibiotic dehydratase, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9892, 0.0050, 0.0058)\n","Processed WP_048537146.1 ATP-binding cassette domain-containing protein, partial [Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.9885, 0.0088, 0.0026)\n","Processed EHV2892889.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9663, 0.0055, 0.0282)\n","Processed WP_003746133.1 ATP-binding cassette domain-containing protein, partial [Listeria seeligeri]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0059, 0.0035)\n","Processed WP_025187236.1 gallidermin/nisin family lantibiotic [Enterococcus faecalis]: AMP (Probabilities: 0.0731, 0.9141, 0.0128)\n","Processed WP_025187235.1 gallidermin/nisin family lantibiotic [Enterococcus faecalis]: AMP (Probabilities: 0.0945, 0.8779, 0.0275)\n","Processed WP_219303803.1 gallidermin/nisin family lantibiotic [Bacillus sp. FDAARGOS_1420]: non-AMP/OTHERS (Probabilities: 0.9813, 0.0097, 0.0090)\n","Processed EHV2886393.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9852, 0.0036, 0.0112)\n","Processed WP_042739438.1 ATP-binding cassette domain-containing protein, partial [Staphylococcus gallinarum]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0062, 0.0032)\n","Processed WP_142310207.1 lantibiotic dehydratase, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.1093, 0.8793, 0.0114)\n","Processed WP_219303804.1 gallidermin/nisin family lantibiotic [Bacillus sp. FDAARGOS_1420]: non-AMP/OTHERS (Probabilities: 0.9869, 0.0069, 0.0062)\n","Processed WP_139638388.1 lantibiotic dehydratase [Nonomuraea]: Neuropeptide (Probabilities: 0.0223, 0.0154, 0.9623)\n","Processed WP_093815087.1 lantibiotic salivaricin M precursor [Streptococcus sp. 45]: AMP (Probabilities: 0.1132, 0.8725, 0.0142)\n","Processed MBW4510405.1 mersacidin/lichenicidin family type 2 lantibiotic [Scytonematopsis contorta HA4267-MV1]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0035, 0.0063)\n","Processed WP_143118210.1 lantibiotic dehydratase, partial [Alkalihalobacillus clausii]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0032, 0.0055)\n","Processed WP_090090073.1 gallidermin/nisin family lantibiotic [Leuconostoc gelidum]: AMP (Probabilities: 0.2959, 0.6602, 0.0439)\n","Processed WP_075345895.1 type A2 lantipeptide [Streptococcus sp. 'caviae']: non-AMP/OTHERS (Probabilities: 0.9911, 0.0038, 0.0051)\n","Processed WP_093528619.1 type A2 lantipeptide [Streptococcus macedonicus]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0043, 0.0059)\n","Processed MBW4503087.1 cinnamycin family lantibiotic [Scytonema hyalinum WJT4-NPBG1]: non-AMP/OTHERS (Probabilities: 0.8688, 0.0624, 0.0689)\n","Processed MBW4496784.1 cinnamycin family lantibiotic [Oscillatoria princeps RMCB-10]: non-AMP/OTHERS (Probabilities: 0.8711, 0.0944, 0.0345)\n","Processed MBW4496783.1 cinnamycin family lantibiotic [Oscillatoria princeps RMCB-10]: non-AMP/OTHERS (Probabilities: 0.9305, 0.0486, 0.0209)\n","Processed WP_042740182.1 ATP-binding cassette domain-containing protein, partial [Staphylococcus gallinarum]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0042, 0.0032)\n","Processed MBW4666263.1 cinnamycin family lantibiotic [Cyanomargarita calcarea GSE-NOS-MK-12-04C]: non-AMP/OTHERS (Probabilities: 0.8396, 0.0428, 0.1176)\n","Processed MBW4620720.1 mersacidin/lichenicidin family type 2 lantibiotic [Cyanosarcina radialis HA8281-LM2]: non-AMP/OTHERS (Probabilities: 0.5218, 0.0299, 0.4482)\n","Processed MBW4623876.1 mersacidin/lichenicidin family type 2 lantibiotic [Cyanosarcina radialis HA8281-LM2]: non-AMP/OTHERS (Probabilities: 0.9862, 0.0072, 0.0066)\n","Processed MBW4623885.1 mersacidin/lichenicidin family type 2 lantibiotic [Cyanosarcina radialis HA8281-LM2]: non-AMP/OTHERS (Probabilities: 0.6416, 0.0394, 0.3190)\n","Processed WP_208344623.1 mersacidin/lichenicidin family type 2 lantibiotic [Aetokthonos hydrillicola]: non-AMP/OTHERS (Probabilities: 0.8196, 0.0698, 0.1106)\n","Processed WP_208352317.1 mersacidin/lichenicidin family type 2 lantibiotic [Aetokthonos hydrillicola]: non-AMP/OTHERS (Probabilities: 0.9207, 0.0174, 0.0619)\n","Processed MBW4623882.1 mersacidin/lichenicidin family type 2 lantibiotic [Cyanosarcina radialis HA8281-LM2]: non-AMP/OTHERS (Probabilities: 0.6603, 0.0392, 0.3005)\n","Processed MBW4623883.1 mersacidin/lichenicidin family type 2 lantibiotic [Cyanosarcina radialis HA8281-LM2]: Neuropeptide (Probabilities: 0.4396, 0.0387, 0.5216)\n","Processed MBW4699011.1 mersacidin/lichenicidin family type 2 lantibiotic [Aphanocapsa lilacina HA4352-LM1]: non-AMP/OTHERS (Probabilities: 0.8911, 0.0953, 0.0136)\n","Processed MBW4577389.1 mersacidin/lichenicidin family type 2 lantibiotic [Aphanothece sp. CMT-3BRIN-NPC111]: non-AMP/OTHERS (Probabilities: 0.9442, 0.0273, 0.0286)\n","Processed WP_219184398.1 gallidermin/nisin family lantibiotic [Streptomyces sp. AC542_RSS864]: Neuropeptide (Probabilities: 0.3907, 0.0692, 0.5400)\n","Processed WP_162867105.1 lantibiotic dehydratase, partial [Klebsiella pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9865, 0.0031, 0.0103)\n","Processed WP_163317725.1 lantibiotic dehydratase, partial [Enterobacter hormaechei]: non-AMP/OTHERS (Probabilities: 0.9835, 0.0083, 0.0082)\n","Processed WP_162481184.1 lantibiotic dehydratase, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9149, 0.0177, 0.0674)\n","Processed WP_139889581.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus tropicus]: AMP (Probabilities: 0.1238, 0.8638, 0.0124)\n","Processed WP_082250332.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9862, 0.0109, 0.0029)\n","Processed WP_219137376.1 lantibiotic (srt) production protein [Streptococcus mitis]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0043, 0.0034)\n","Processed WP_090289785.1 FDLD family class I lanthipeptide [Facklamia miroungae]: AMP (Probabilities: 0.3522, 0.4420, 0.2058)\n","Processed WP_163048754.1 lantibiotic dehydratase, partial [Acinetobacter pittii]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0067, 0.0032)\n","Processed WP_042748783.1 hypothetical protein [Lactococcus cremoris]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0051, 0.0031)\n","Processed WP_194949885.1 gallidermin/nisin family lantibiotic [Actinomyces trachealis]: AMP (Probabilities: 0.1042, 0.8807, 0.0150)\n","Processed WP_142256154.1 lantibiotic dehydratase, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0034)\n","Processed WP_153600587.1 type A lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0046, 0.0043)\n","Processed WP_167442757.1 lantibiotic dehydratase, partial [Streptomyces rhizosphaericus]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0034, 0.0044)\n","Processed WP_166857005.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinomyces]: AMP (Probabilities: 0.3567, 0.5754, 0.0679)\n","Processed WP_194949871.1 type 2 lantibiotic [Actinomyces trachealis]: AMP (Probabilities: 0.1392, 0.8477, 0.0131)\n","Processed WP_163334191.1 lantibiotic dehydratase, partial [Enterobacter bugandensis]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0035, 0.0066)\n","Processed WP_162034000.1 lantibiotic dehydratase [Chryseobacterium sp. CECT 9293]: AMP (Probabilities: 0.0779, 0.9095, 0.0127)\n","Processed WP_142345641.1 plantaricin C family lantibiotic [Bacillus sp. AFS051223]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0073, 0.0026)\n","Processed WP_147343401.1 lantibiotic dehydratase, partial [Ruminococcus sp. OM04-4AA]: Neuropeptide (Probabilities: 0.1056, 0.0347, 0.8597)\n","Processed WP_194949870.1 lichenicidin A2 family type 2 lantibiotic [Actinomyces trachealis]: non-AMP/OTHERS (Probabilities: 0.8854, 0.1102, 0.0044)\n","Processed WP_163359452.1 lantibiotic dehydratase, partial [Escherichia coli]: non-AMP/OTHERS (Probabilities: 0.9486, 0.0229, 0.0285)\n","Processed WP_099810709.1 lantibiotic salivaricin M precursor [Streptococcus suis]: Neuropeptide (Probabilities: 0.2555, 0.2915, 0.4530)\n","Processed WP_099810547.1 lantibiotic salivaricin M precursor [Streptococcus suis]: AMP (Probabilities: 0.0773, 0.9103, 0.0123)\n","Processed WP_104929697.1 lantibiotic salivaricin M precursor [Streptococcus suis]: AMP (Probabilities: 0.0883, 0.8993, 0.0125)\n","Processed WP_098830213.1 type A2 lantipeptide [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0036, 0.0037)\n","Processed WP_098801344.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: Neuropeptide (Probabilities: 0.2941, 0.0230, 0.6829)\n","Processed WP_141522203.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.6141, 0.1159, 0.2700)\n","Processed WP_098802602.1 plantaricin C family lantibiotic [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.6380, 0.0208, 0.3413)\n","Processed WP_098999959.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.6626, 0.0386, 0.2988)\n","Processed WP_218963962.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0678, 0.9182, 0.0140)\n","Processed WP_218936829.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0685, 0.9176, 0.0139)\n","Processed WP_218012797.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: Neuropeptide (Probabilities: 0.0503, 0.0201, 0.9296)\n","Processed WP_099498548.1 FxLD family lanthipeptide [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0050, 0.0032)\n","Processed WP_098624264.1 type A2 lantipeptide [Bacillus toyonensis]: Neuropeptide (Probabilities: 0.1226, 0.0255, 0.8519)\n","Processed WP_099070035.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc linckia]: non-AMP/OTHERS (Probabilities: 0.9521, 0.0086, 0.0393)\n","Processed WP_098280502.1 type A2 lantipeptide [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9834, 0.0059, 0.0106)\n","Processed WP_098101392.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.8002, 0.0168, 0.1830)\n","Processed WP_217700407.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus cereus group]: AMP (Probabilities: 0.1247, 0.8627, 0.0126)\n","Processed WP_098124808.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9383, 0.0576, 0.0041)\n","Processed WP_098814126.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed WP_098188482.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.7272, 0.0604, 0.2124)\n","Processed WP_098295762.1 type A2 lantipeptide [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9099, 0.0553, 0.0347)\n","Processed WP_218923082.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus cereus group]: AMP (Probabilities: 0.0650, 0.9198, 0.0152)\n","Processed WP_097892171.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus cereus group]: AMP (Probabilities: 0.1574, 0.8303, 0.0123)\n","Processed WP_205684569.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0740, 0.9124, 0.0136)\n","Processed WP_142314464.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9853, 0.0037, 0.0110)\n","Processed WP_098159029.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.7406, 0.0276, 0.2319)\n","Processed WP_097812070.1 gallidermin/nisin family lantibiotic, partial [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0053, 0.0026)\n","Processed WP_098159030.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9893, 0.0081, 0.0025)\n","Processed WP_088330785.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9775, 0.0172, 0.0053)\n","Processed WP_097795816.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0063, 0.0028)\n","Processed WP_096832221.1 cinnamycin family lantibiotic [Tychonema bourrellyi]: non-AMP/OTHERS (Probabilities: 0.8353, 0.0614, 0.1033)\n","Processed WP_097850015.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0039, 0.0035)\n","Processed WP_097850014.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: Neuropeptide (Probabilities: 0.2641, 0.0330, 0.7028)\n","Processed WP_057487960.1 hypothetical protein [Streptococcus]: AMP (Probabilities: 0.0913, 0.8961, 0.0126)\n","Processed WP_061667779.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9330, 0.0613, 0.0057)\n","Processed WP_057487958.1 hypothetical protein [Streptococcus]: AMP (Probabilities: 0.0717, 0.9152, 0.0131)\n","Processed WP_050775715.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9878, 0.0083, 0.0039)\n","Processed WP_000180649.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9875, 0.0050, 0.0075)\n","Processed WP_088296487.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9849, 0.0109, 0.0042)\n","Processed WP_078156164.1 lantibiotic salivaricin M precursor [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.5610, 0.1473, 0.2917)\n","Processed WP_057487959.1 lantibiotic salivaricin M precursor [Streptococcus]: AMP (Probabilities: 0.0797, 0.9056, 0.0147)\n","Processed WP_057487957.1 hypothetical protein [Streptococcus]: AMP (Probabilities: 0.0635, 0.9217, 0.0147)\n","Processed WP_218939745.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.8668, 0.0146, 0.1187)\n","Processed WP_180229352.1 plantaricin C family lantibiotic [Bacillus pseudomycoides]: AMP (Probabilities: 0.0663, 0.9189, 0.0148)\n","Processed  unnamed protein product [Unknown]: non-AMP/OTHERS (Probabilities: 0.9660, 0.0224, 0.0116)\n","Processed WP_142903337.1 mersacidin/lichenicidin family type 2 lantibiotic [Exilibacterium tricleocarpae]: non-AMP/OTHERS (Probabilities: 0.9802, 0.0058, 0.0141)\n","Processed WP_142337217.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9838, 0.0068, 0.0094)\n","Processed WP_218939161.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus cereus]: AMP (Probabilities: 0.1178, 0.8701, 0.0121)\n","Processed WP_218943451.1 mersacidin/lichenicidin family type 2 lantibiotic [Exilibacterium tricleocarpae]: non-AMP/OTHERS (Probabilities: 0.9782, 0.0056, 0.0162)\n","Processed WP_147776620.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus sp. BF9-10]: AMP (Probabilities: 0.1274, 0.8601, 0.0125)\n","Processed WP_103061646.1 lichenicidin A2 family type 2 lantibiotic [Actinomyces qiguomingii]: Neuropeptide (Probabilities: 0.3172, 0.2166, 0.4662)\n","Processed WP_098883615.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: AMP (Probabilities: 0.1492, 0.8367, 0.0141)\n","Processed WP_003205287.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.7981, 0.0599, 0.1421)\n","Processed WP_141537385.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.4177, 0.5187, 0.0636)\n","Processed WP_098756042.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus toyonensis]: AMP (Probabilities: 0.3011, 0.6881, 0.0108)\n","Processed WP_218937248.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0659, 0.9196, 0.0146)\n","Processed WP_098764186.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9843, 0.0059, 0.0098)\n","Processed WP_098827892.1 gallidermin/nisin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed WP_142338361.1 gallidermin/nisin family lantibiotic, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.1171, 0.8702, 0.0127)\n","Processed WP_098788916.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: AMP (Probabilities: 0.2477, 0.7393, 0.0130)\n","Processed WP_218937003.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0651, 0.9203, 0.0146)\n","Processed WP_218936733.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0669, 0.9186, 0.0146)\n","Processed WP_218936735.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.1203, 0.8678, 0.0119)\n","Processed WP_098855976.1 gallidermin/nisin family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0040, 0.0028)\n","Processed WP_218937082.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus toyonensis]: AMP (Probabilities: 0.1331, 0.8542, 0.0127)\n","Processed WP_098673617.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.7037, 0.0243, 0.2720)\n","Processed WP_218012587.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0639, 0.9212, 0.0149)\n","Processed WP_098624522.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus toyonensis]: non-AMP/OTHERS (Probabilities: 0.9734, 0.0049, 0.0217)\n","Processed WP_098673616.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0054, 0.0039)\n","Processed WP_218936803.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus wiedmannii]: AMP (Probabilities: 0.0661, 0.9188, 0.0151)\n","Processed WP_098651372.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus toyonensis]: Neuropeptide (Probabilities: 0.4333, 0.0241, 0.5426)\n","Processed WP_218937188.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0684, 0.9169, 0.0147)\n","Processed WP_098634525.1 plantaricin C family lantibiotic [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.8879, 0.1053, 0.0068)\n","Processed WP_098623540.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0035)\n","Processed WP_098674702.1 plantaricin C family lantibiotic [Bacillales]: AMP (Probabilities: 0.0902, 0.8980, 0.0118)\n","Processed WP_098674701.1 plantaricin C family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.0677, 0.9180, 0.0143)\n","Processed WP_218938022.1 FDLD family class I lanthipeptide, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9852, 0.0041, 0.0107)\n","Processed WP_098674703.1 plantaricin C family lantibiotic [Bacillales]: non-AMP/OTHERS (Probabilities: 0.5015, 0.4348, 0.0637)\n","Processed WP_142330053.1 gallidermin/nisin family lantibiotic, partial [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0054, 0.0028)\n","Processed WP_142332536.1 AAA family ATPase, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0066, 0.0026)\n","Processed WP_098673615.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0057, 0.0028)\n","Processed WP_098590794.1 gallidermin/nisin family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0050, 0.0044)\n","Processed WP_098610965.1 FDLD family class I lanthipeptide [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0036, 0.0043)\n","Processed PFJ06663.1 hypothetical protein [Bacillus cereus]: Neuropeptide (Probabilities: 0.0107, 0.0104, 0.9789)\n","Processed WP_098623539.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0062, 0.0047)\n","Processed PFJ06660.1 hypothetical protein [Bacillus cereus]: Neuropeptide (Probabilities: 0.0360, 0.0210, 0.9430)\n","Processed WP_098534193.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0033, 0.0052)\n","Processed WP_097879613.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus toyonensis]: AMP (Probabilities: 0.1253, 0.8625, 0.0122)\n","Processed WP_097811970.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0071, 0.0028)\n","Processed WP_097946631.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.7773, 0.0188, 0.2039)\n","Processed WP_097832122.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0039, 0.0056)\n","Processed WP_097946629.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.8782, 0.0136, 0.1081)\n","Processed WP_078211853.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9725, 0.0192, 0.0083)\n","Processed WP_098634526.1 plantaricin C family lantibiotic [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.7860, 0.2070, 0.0070)\n","Processed WP_205684861.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.1521, 0.8350, 0.0129)\n","Processed WP_030733737.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0039, 0.0035)\n","Processed WP_141533607.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0029, 0.0047)\n","Processed WP_098356687.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9841, 0.0134, 0.0025)\n","Processed WP_098437983.1 plantaricin C family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9851, 0.0046, 0.0103)\n","Processed WP_155974693.1 type A2 lantipeptide [Staphylococcus hominis]: Neuropeptide (Probabilities: 0.1436, 0.0233, 0.8331)\n","Processed WP_179862905.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: Neuropeptide (Probabilities: 0.3773, 0.0420, 0.5808)\n","Processed WP_003209900.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9836, 0.0046, 0.0118)\n","Processed WP_142313030.1 plantaricin C family lantibiotic, partial [Bacillus toyonensis]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0040, 0.0033)\n","Processed WP_136097588.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0039, 0.0060)\n","Processed WP_136048034.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9882, 0.0042, 0.0076)\n","Processed WP_179938161.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sp. Rer75]: non-AMP/OTHERS (Probabilities: 0.9023, 0.0098, 0.0879)\n","Processed WP_136296199.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0030, 0.0039)\n","Processed WP_000041632.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0045, 0.0052)\n","Processed WP_136296198.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0035, 0.0078)\n","Processed WP_141538347.1 lichenicidin A2 family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0039, 0.0039)\n","Processed WP_136273135.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9851, 0.0038, 0.0111)\n","Processed WP_050511377.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus group]: AMP (Probabilities: 0.0851, 0.9000, 0.0149)\n","Processed WP_136269773.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0033, 0.0042)\n","Processed WP_136047030.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9494, 0.0098, 0.0408)\n","Processed WP_136291050.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0038, 0.0046)\n","Processed WP_086391186.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9820, 0.0147, 0.0033)\n","Processed WP_136093858.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0030, 0.0037)\n","Processed WP_136261072.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9881, 0.0039, 0.0080)\n","Processed WP_136018286.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0039, 0.0050)\n","Processed WP_132617674.1 lantibiotic dehydratase [Nonomuraea terrae]: non-AMP/OTHERS (Probabilities: 0.9818, 0.0124, 0.0058)\n","Processed WP_098475912.1 plantaricin C family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.7881, 0.0247, 0.1873)\n","Processed WP_098328443.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0032, 0.0036)\n","Processed WP_136075391.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9830, 0.0042, 0.0127)\n","Processed WP_136274160.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.7088, 0.0202, 0.2711)\n","Processed WP_098507581.1 type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9776, 0.0175, 0.0049)\n","Processed WP_098376691.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8426, 0.1512, 0.0062)\n","Processed WP_136274162.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9872, 0.0040, 0.0088)\n","Processed WP_136047031.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0035, 0.0054)\n","Processed WP_098161436.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9842, 0.0133, 0.0025)\n","Processed WP_098377582.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9433, 0.0069, 0.0497)\n","Processed WP_218923826.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus toyonensis]: AMP (Probabilities: 0.0668, 0.9189, 0.0143)\n","Processed WP_142311092.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus toyonensis]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0029, 0.0043)\n","Processed WP_142315536.1 type A2 lantipeptide, partial [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9402, 0.0307, 0.0291)\n","Processed WP_142311813.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus toyonensis]: non-AMP/OTHERS (Probabilities: 0.9496, 0.0071, 0.0433)\n","Processed WP_098127093.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus toyonensis]: non-AMP/OTHERS (Probabilities: 0.8639, 0.1304, 0.0057)\n","Processed WP_098123065.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.7048, 0.2875, 0.0076)\n","Processed WP_218923714.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0676, 0.9180, 0.0145)\n","Processed WP_097853286.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.3170, 0.6531, 0.0299)\n","Processed WP_218923046.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.1157, 0.8720, 0.0123)\n","Processed WP_142308428.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9855, 0.0034, 0.0112)\n","Processed WP_141538287.1 gallidermin/nisin family lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.5139, 0.4774, 0.0088)\n","Processed WP_098217339.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: AMP (Probabilities: 0.1381, 0.8489, 0.0130)\n","Processed WP_095979720.1 mersacidin/lichenicidin family type 2 lantibiotic [Melittangium boletus]: non-AMP/OTHERS (Probabilities: 0.4979, 0.1410, 0.3611)\n","Processed WP_097913740.1 lichenicidin A2 family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9748, 0.0095, 0.0157)\n","Processed WP_097833503.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0034)\n","Processed WP_142311553.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus toyonensis]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0030, 0.0061)\n","Processed WP_089310450.1 cinnamycin family lantibiotic [Actinomadura]: non-AMP/OTHERS (Probabilities: 0.9229, 0.0709, 0.0062)\n","Processed WP_088319973.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9695, 0.0122, 0.0184)\n","Processed WP_083780220.1 mersacidin/lichenicidin family type 2 lantibiotic [Kyrpidia tusciae]: non-AMP/OTHERS (Probabilities: 0.9880, 0.0054, 0.0065)\n","Processed WP_080556864.1 plantaricin C family lantibiotic [Rothia]: AMP (Probabilities: 0.1114, 0.8755, 0.0132)\n","Processed WP_070180740.1 plantaricin C family lantibiotic [Bacillus cereus group]: Neuropeptide (Probabilities: 0.0223, 0.0156, 0.9621)\n","Processed WP_143047207.1 type A2 lantipeptide [Amycolatopsis xylanica]: non-AMP/OTHERS (Probabilities: 0.4963, 0.0182, 0.4855)\n","Processed WP_063549729.1 plantaricin C family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9873, 0.0046, 0.0081)\n","Processed WP_068444983.1 plantaricin C family lantibiotic [Lentibacillus amyloliquefaciens]: non-AMP/OTHERS (Probabilities: 0.9032, 0.0103, 0.0865)\n","Processed WP_063549733.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9642, 0.0322, 0.0036)\n","Processed WP_063549731.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0040, 0.0050)\n","Processed WP_097895861.1 plantaricin C family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9741, 0.0157, 0.0102)\n","Processed WP_013076755.1 mersacidin/lichenicidin family type 2 lantibiotic [Kyrpidia tusciae]: non-AMP/OTHERS (Probabilities: 0.8346, 0.0153, 0.1501)\n","Processed WP_179824887.1 plantaricin C family lantibiotic [Nocardiopsis aegyptia]: AMP (Probabilities: 0.1108, 0.8766, 0.0126)\n","Processed WP_012490912.1 AAA family ATPase [Lacticaseibacillus paracasei]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0042, 0.0034)\n","Processed WP_063484126.1 SCO0268 family class II lantipeptide [Streptomyces]: Neuropeptide (Probabilities: 0.3400, 0.0357, 0.6243)\n","Processed WP_018960897.1 lexapeptide family class V lantibiotic [Streptomyces sp. rh34]: non-AMP/OTHERS (Probabilities: 0.8465, 0.0163, 0.1372)\n","Processed WP_001198625.1 plantaricin C family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.5431, 0.0235, 0.4333)\n","Processed WP_179824889.1 plantaricin C family lantibiotic [Nocardiopsis aegyptia]: AMP (Probabilities: 0.0803, 0.9077, 0.0120)\n","Processed WP_179824893.1 plantaricin C family lantibiotic [Nocardiopsis aegyptia]: non-AMP/OTHERS (Probabilities: 0.6397, 0.3463, 0.0140)\n","Processed WP_123855512.1 lantibiotic dehydratase [Chryseobacterium shandongense]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0055, 0.0025)\n","Processed WP_098507774.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.4169, 0.4415, 0.1416)\n","Processed WP_129536880.1 lantibiotic dehydratase [Chryseobacterium sp. 3008163]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0057, 0.0028)\n","Processed WP_107518459.1 type A lantibiotic [Staphylococcus equorum]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0033, 0.0051)\n","Processed WP_096832598.1 lantibiotic salivaricin M precursor [Streptococcus salivarius]: AMP (Probabilities: 0.0705, 0.9163, 0.0132)\n","Processed WP_096622710.1 lexapeptide family class V lantibiotic [Streptomyces sp. WZ.A104]: non-AMP/OTHERS (Probabilities: 0.9772, 0.0058, 0.0170)\n","Processed WP_136021211.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0030, 0.0039)\n","Processed WP_096632908.1 plantaricin C family lantibiotic [Streptomyces sp. WZ.A104]: Neuropeptide (Probabilities: 0.2041, 0.0388, 0.7572)\n","Processed WP_095267177.1 FDLD family class I lanthipeptide [Paenibacillus]: AMP (Probabilities: 0.1252, 0.8502, 0.0245)\n","Processed WP_179845637.1 cinnamycin family lantibiotic [Actinomadura luteofluorescens]: AMP (Probabilities: 0.3036, 0.6694, 0.0270)\n","Processed WP_218362410.1 mersacidin/lichenicidin family type 2 lantibiotic [Corynebacterium sp. TAE3-ERU12]: non-AMP/OTHERS (Probabilities: 0.8359, 0.0291, 0.1350)\n","Processed WP_024927988.1 type A lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0887, 0.8990, 0.0123)\n","Processed WP_004183534.1 hypothetical protein [Streptococcus salivarius]: AMP (Probabilities: 0.0732, 0.9140, 0.0128)\n","Processed WP_077699971.1 type A lantibiotic [Staphylococcus hominis]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0051, 0.0027)\n","Processed WP_150159044.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9853, 0.0103, 0.0045)\n","Processed WP_024927987.1 type A lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0690, 0.9172, 0.0138)\n","Processed WP_218362411.1 mersacidin/lichenicidin family type 2 lantibiotic [Corynebacterium sp. TAE3-ERU12]: non-AMP/OTHERS (Probabilities: 0.9843, 0.0071, 0.0086)\n","Processed WP_022750734.1 plantaricin C family lantibiotic [Clostridium saccharobutylicum]: AMP (Probabilities: 0.0788, 0.9091, 0.0121)\n","Processed WP_115354508.1 lantibiotic dehydratase [Peptoniphilus lacrimalis]: non-AMP/OTHERS (Probabilities: 0.9227, 0.0709, 0.0063)\n","Processed WP_114317825.1 type A2 lantipeptide [Streptococcus gallolyticus]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0041, 0.0042)\n","Processed WP_110110496.1 type A2 lantipeptide [unclassified Auritidibacter]: Neuropeptide (Probabilities: 0.1167, 0.0259, 0.8575)\n","Processed WP_106006683.1 type A2 lantipeptide [Streptococcus mutans]: Neuropeptide (Probabilities: 0.1796, 0.0401, 0.7803)\n","Processed WP_115856514.1 type A2 lantipeptide [Staphylococcus felis]: non-AMP/OTHERS (Probabilities: 0.9714, 0.0101, 0.0185)\n","Processed WP_101883896.1 type A2 lantipeptide [[Ruminococcus] gnavus]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0035, 0.0051)\n","Processed WP_098849434.1 type A2 lantipeptide [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0035, 0.0043)\n","Processed WP_098909254.1 type A2 lantipeptide, partial [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0031, 0.0050)\n","Processed WP_089506884.1 plantaricin C family lantibiotic [Streptomyces sp. NBS 14/10]: Neuropeptide (Probabilities: 0.0300, 0.0199, 0.9501)\n","Processed WP_088609898.1 plantaricin C family lantibiotic [Bacillus sp. K2I17]: AMP (Probabilities: 0.1142, 0.8723, 0.0135)\n","Processed WP_088308621.1 gallidermin/nisin family lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0664, 0.9192, 0.0144)\n","Processed WP_098491940.1 type A2 lantipeptide [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0059, 0.0038)\n","Processed WP_098491939.1 type A2 lantipeptide [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0044, 0.0048)\n","Processed WP_022750729.1 plantaricin C family lantibiotic [Clostridium saccharobutylicum]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0030)\n","Processed WP_088568403.1 type A2 lantipeptide [Priestia]: non-AMP/OTHERS (Probabilities: 0.7424, 0.1963, 0.0613)\n","Processed WP_089506911.1 SCO0268 family class II lantipeptide [Streptomyces sp. NBS 14/10]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0039, 0.0061)\n","Processed WP_088308619.1 gallidermin/nisin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8588, 0.1333, 0.0078)\n","Processed WP_100046129.1 SCO0268 family class II lantipeptide, partial [Streptomyces sp. Alain-F2R5]: non-AMP/OTHERS (Probabilities: 0.9815, 0.0120, 0.0065)\n","Processed WP_076131461.1 type A2 lantipeptide [Actinomyces naeslundii]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0044, 0.0065)\n","Processed WP_076075532.1 type A2 lantipeptide [Actinomyces naeslundii]: Neuropeptide (Probabilities: 0.1540, 0.0355, 0.8105)\n","Processed WP_076131460.1 type A2 lantipeptide [Actinomyces naeslundii]: AMP (Probabilities: 0.1370, 0.8456, 0.0174)\n","Processed WP_080004879.1 type A2 lantipeptide [Streptococcus equi]: non-AMP/OTHERS (Probabilities: 0.9760, 0.0056, 0.0184)\n","Processed WP_074562142.1 lichenicidin A2 family type 2 lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0705, 0.9158, 0.0137)\n","Processed WP_071528712.1 gallidermin/nisin family lantibiotic [Streptomyces]: Neuropeptide (Probabilities: 0.0898, 0.0310, 0.8792)\n","Processed WP_090903022.1 plantaricin C family lantibiotic [unclassified Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.8062, 0.1870, 0.0068)\n","Processed WP_074596227.1 type A2 lantipeptide [Streptococcus gallolyticus]: non-AMP/OTHERS (Probabilities: 0.8487, 0.0175, 0.1339)\n","Processed WP_074093209.1 type A2 lantipeptide [Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.9893, 0.0037, 0.0070)\n","Processed WP_085634202.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0043)\n","Processed WP_069717239.1 type A2 lantipeptide [Bacillus solimangrovi]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0034, 0.0033)\n","Processed WP_063631441.1 type A2 lantipeptide [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed WP_063539114.1 plantaricin C family lantibiotic [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.5940, 0.0302, 0.3758)\n","Processed WP_074485056.1 type A2 lantipeptide [Streptococcus henryi]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0046, 0.0046)\n","Processed WP_061460398.1 type A2 lantipeptide [Streptococcus gallolyticus]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0030)\n","Processed WP_014181724.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9896, 0.0068, 0.0035)\n","Processed WP_014845285.1 type A2 lantipeptide [Arachnia propionica]: Neuropeptide (Probabilities: 0.0515, 0.0225, 0.9260)\n","Processed WP_060489084.1 type A2 lantipeptide [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0036)\n","Processed WP_031392572.1 type A2 lantipeptide [Kineothrix alysoides]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0036)\n","Processed WP_143353720.1 gallidermin/nisin family lantibiotic, partial [Enterococcus sp. 10A9_DIV0425]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0034, 0.0054)\n","Processed WP_142287892.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.8685, 0.0439, 0.0876)\n","Processed WP_142287183.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9894, 0.0059, 0.0048)\n","Processed WP_087812317.1 SCO0268 family class II lantipeptide [Streptomyces sp. CS113]: Neuropeptide (Probabilities: 0.3041, 0.0332, 0.6626)\n","Processed WP_032489363.1 type A2 lantipeptide [Lactococcus lactis]: non-AMP/OTHERS (Probabilities: 0.9819, 0.0151, 0.0030)\n","Processed WP_041790396.1 type A2 lantipeptide [Streptococcus equi]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0039, 0.0048)\n","Processed WP_048536741.1 lantibiotic dehydratase, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0034)\n","Processed WP_037938180.1 lantibiotic dehydratase, partial [Streptosporangium roseum]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0037, 0.0041)\n","Processed WP_037660238.1 lantibiotic dehydratase, partial [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9834, 0.0116, 0.0050)\n","Processed WP_085876079.1 lantibiotic dehydratase [Peptoniphilus vaginalis]: non-AMP/OTHERS (Probabilities: 0.9872, 0.0091, 0.0037)\n","Processed WP_085573427.1 FxLD family lanthipeptide [Streptomyces sp. 13-12-16]: non-AMP/OTHERS (Probabilities: 0.9542, 0.0333, 0.0124)\n","Processed WP_010988897.1 SapB/AmfS family lanthipeptide [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9499, 0.0075, 0.0426)\n","Processed WP_033305975.1 lantibiotic dehydratase, partial [Streptomyces atroolivaceus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0046, 0.0033)\n","Processed WP_030935350.1 lantibiotic dehydratase [Streptomyces sp. NRRL S-646]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0046, 0.0033)\n","Processed WP_086284962.1 gallidermin/nisin family lantibiotic [Enterococcus]: AMP (Probabilities: 0.0712, 0.9161, 0.0127)\n","Processed WP_037822458.1 lantibiotic dehydratase, partial [Streptomyces sp. NRRL S-146]: non-AMP/OTHERS (Probabilities: 0.9800, 0.0128, 0.0072)\n","Processed WP_077966591.1 FxLD family lanthipeptide [Streptomyces tsukubensis]: non-AMP/OTHERS (Probabilities: 0.9696, 0.0064, 0.0240)\n","Processed HBI1783222.1 gallidermin/nisin family lantibiotic [Enterococcus faecalis]: AMP (Probabilities: 0.0731, 0.9136, 0.0133)\n","Processed WP_179084717.1 lexapeptide family class V lantibiotic [Streptomyces]: Neuropeptide (Probabilities: 0.0150, 0.0122, 0.9728)\n","Processed WP_076243499.1 class II lanthipeptide, LchA2/BrtA2 family [unclassified Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.6818, 0.3067, 0.0115)\n","Processed WP_042132907.1 plantaricin C family lantibiotic [unclassified Paenibacillus]: Neuropeptide (Probabilities: 0.0460, 0.0201, 0.9340)\n","Processed WP_033918111.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9717, 0.0065, 0.0219)\n","Processed WP_072472683.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.0482, 0.0184, 0.9334)\n","Processed WP_072419822.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.0703, 0.0195, 0.9101)\n","Processed WP_168768591.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus aureus]: AMP (Probabilities: 0.0666, 0.9194, 0.0140)\n","Processed WP_001245106.1 hypothetical protein [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0035)\n","Processed HBI0859695.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.2703, 0.0548, 0.6748)\n","Processed WP_073379458.1 plantaricin C family lantibiotic [Nocardiopsis flavescens]: non-AMP/OTHERS (Probabilities: 0.6749, 0.0189, 0.3062)\n","Processed WP_073379440.1 plantaricin C family lantibiotic [Nocardiopsis flavescens]: non-AMP/OTHERS (Probabilities: 0.3839, 0.3827, 0.2334)\n","Processed MBV9690774.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: Neuropeptide (Probabilities: 0.0211, 0.0140, 0.9648)\n","Processed MBV9690605.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.6344, 0.1159, 0.2498)\n","Processed MBV9708024.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: Neuropeptide (Probabilities: 0.3914, 0.0487, 0.5599)\n","Processed WP_023522752.1 gallidermin/nisin family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0038, 0.0039)\n","Processed WP_155910107.1 lichenicidin A2 family type 2 lantibiotic [Bacillus sonorensis]: non-AMP/OTHERS (Probabilities: 0.9872, 0.0104, 0.0024)\n","Processed MBV9708023.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: non-AMP/OTHERS (Probabilities: 0.6843, 0.0342, 0.2815)\n","Processed MBV9708140.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: non-AMP/OTHERS (Probabilities: 0.9293, 0.0246, 0.0461)\n","Processed MBV9787934.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0053, 0.0045)\n","Processed MBV9675530.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteriaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0037, 0.0072)\n","Processed WP_128634886.1 mersacidin/lichenicidin family type 2 lantibiotic [Paenibacillus oralis]: non-AMP/OTHERS (Probabilities: 0.9935, 0.0033, 0.0032)\n","Processed MBV8693865.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.6829, 0.0849, 0.2322)\n","Processed MBV9034251.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteriaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9611, 0.0131, 0.0258)\n","Processed MBV9019692.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.7054, 0.0575, 0.2371)\n","Processed MBV9019691.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.8023, 0.0662, 0.1315)\n","Processed MBV8693864.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: Neuropeptide (Probabilities: 0.1675, 0.0701, 0.7625)\n","Processed MBV9020939.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: Neuropeptide (Probabilities: 0.4708, 0.0368, 0.4924)\n","Processed MBV8822487.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9779, 0.0104, 0.0116)\n","Processed MBV8821398.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.5087, 0.1535, 0.3378)\n","Processed MBV8695395.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.6118, 0.0920, 0.2962)\n","Processed MBV9280102.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0044, 0.0028)\n","Processed MBV9225684.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteriaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0035, 0.0040)\n","Processed MBV9034250.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteriaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0042, 0.0042)\n","Processed MBV9282326.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0034, 0.0040)\n","Processed MBV9020938.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.5399, 0.0372, 0.4228)\n","Processed MBV9232317.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: non-AMP/OTHERS (Probabilities: 0.9625, 0.0134, 0.0241)\n","Processed MBV9232316.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: non-AMP/OTHERS (Probabilities: 0.8265, 0.1476, 0.0258)\n","Processed MBV9307326.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteriaceae bacterium]: Neuropeptide (Probabilities: 0.0600, 0.0185, 0.9215)\n","Processed MBV8228592.1 mersacidin/lichenicidin family type 2 lantibiotic [Planctomycetaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9848, 0.0096, 0.0056)\n","Processed MBV8558273.1 mersacidin/lichenicidin family type 2 lantibiotic [Planctomycetaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9781, 0.0132, 0.0088)\n","Processed MBV9211981.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.8891, 0.0462, 0.0647)\n","Processed  unnamed protein product [Unknown]: non-AMP/OTHERS (Probabilities: 0.5286, 0.1664, 0.3051)\n","Processed WP_143093000.1 lantibiotic dehydratase, partial [Arsenicibacter rosenii]: Neuropeptide (Probabilities: 0.3465, 0.0809, 0.5726)\n","Processed WP_093711265.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0035)\n","Processed MBV8645617.1 mersacidin/lichenicidin family type 2 lantibiotic [Candidatus Eremiobacteraeota bacterium]: Neuropeptide (Probabilities: 0.0093, 0.0096, 0.9811)\n","Processed  unnamed protein product [Unknown]: AMP (Probabilities: 0.1035, 0.8839, 0.0126)\n","Processed WP_071372474.1 FxLD family lanthipeptide [Streptomyces sp. MUSC 14]: non-AMP/OTHERS (Probabilities: 0.9832, 0.0137, 0.0031)\n","Processed WP_079173389.1 gallidermin/nisin family lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0037, 0.0030)\n","Processed WP_141762833.1 gallidermin/nisin family lantibiotic, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.5198, 0.0566, 0.4235)\n","Processed ABS92150.1 Sequence 3 from patent US 7199230 [Unknown]: non-AMP/OTHERS (Probabilities: 0.7756, 0.0635, 0.1609)\n","Processed WP_083326567.1 gallidermin/nisin family lantibiotic [Actinomyces sp. HMSC035G02]: Neuropeptide (Probabilities: 0.0100, 0.0104, 0.9796)\n","Processed WP_093908739.1 lexapeptide family class V lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0037, 0.0060)\n","Processed WP_070002237.1 plantaricin C family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9773, 0.0077, 0.0150)\n","Processed WP_043828845.1 FxLD family lanthipeptide [Pseudonocardiaceae]: non-AMP/OTHERS (Probabilities: 0.6217, 0.0190, 0.3593)\n","Processed WP_069119973.1 hypothetical protein [Brochothrix thermosphacta]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed WP_073790073.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9848, 0.0092, 0.0060)\n","Processed WP_065361259.1 class II lanthipeptide, LchA2/BrtA2 family [Streptococcus dysgalactiae]: non-AMP/OTHERS (Probabilities: 0.9835, 0.0140, 0.0025)\n","Processed WP_141721487.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora citrea]: Neuropeptide (Probabilities: 0.3259, 0.2110, 0.4630)\n","Processed WP_069803490.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermogemmatispora]: non-AMP/OTHERS (Probabilities: 0.9231, 0.0477, 0.0292)\n","Processed WP_161977616.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter kobayashii]: Neuropeptide (Probabilities: 0.4672, 0.0263, 0.5064)\n","Processed WP_155357332.1 lantibiotic protein [Acrocarpospora macrocephala]: Neuropeptide (Probabilities: 0.1795, 0.0293, 0.7912)\n","Processed WP_146896085.1 mersacidin/lichenicidin family type 2 lantibiotic [Adhaeribacter aerolatus]: Neuropeptide (Probabilities: 0.3863, 0.0373, 0.5765)\n","Processed WP_161977615.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter kobayashii]: Neuropeptide (Probabilities: 0.0236, 0.0143, 0.9621)\n","Processed WP_141312463.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces spinoverrucosus]: AMP (Probabilities: 0.0909, 0.8969, 0.0121)\n","Processed WP_025676407.1 FDLD family class I lanthipeptide [Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.9566, 0.0117, 0.0318)\n","Processed WP_164744398.1 plantaricin C family lantibiotic [Paenibacillus]: AMP (Probabilities: 0.3482, 0.6394, 0.0124)\n","Processed WP_141312464.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces spinoverrucosus]: AMP (Probabilities: 0.1107, 0.8760, 0.0132)\n","Processed WP_126598862.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter aurantiacus]: non-AMP/OTHERS (Probabilities: 0.9563, 0.0143, 0.0294)\n","Processed WP_127532363.1 plantaricin C family lantibiotic [Paenibacillus kobensis]: AMP (Probabilities: 0.0869, 0.9000, 0.0131)\n","Processed WP_127489202.1 elgicin/penisin family lantibiotic [Paenibacillus ehimensis]: AMP (Probabilities: 0.1073, 0.8758, 0.0169)\n","Processed WP_126630726.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter alpinus]: Neuropeptide (Probabilities: 0.2957, 0.1463, 0.5579)\n","Processed WP_160145910.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter aurantiacus]: non-AMP/OTHERS (Probabilities: 0.6538, 0.0230, 0.3232)\n","Processed WP_126601206.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter aurantiacus]: AMP (Probabilities: 0.2504, 0.4104, 0.3392)\n","Processed WP_126552689.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter kobayashii]: non-AMP/OTHERS (Probabilities: 0.8486, 0.0145, 0.1370)\n","Processed WP_126596964.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter aurantiacus]: non-AMP/OTHERS (Probabilities: 0.9552, 0.0066, 0.0382)\n","Processed WP_126596963.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter aurantiacus]: non-AMP/OTHERS (Probabilities: 0.9647, 0.0054, 0.0298)\n","Processed WP_126556110.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter kobayashii]: AMP (Probabilities: 0.1966, 0.5424, 0.2611)\n","Processed WP_161977763.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter kobayashii]: non-AMP/OTHERS (Probabilities: 0.9203, 0.0326, 0.0471)\n","Processed WP_111318317.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis]: non-AMP/OTHERS (Probabilities: 0.7281, 0.2366, 0.0352)\n","Processed WP_126554800.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter kobayashii]: AMP (Probabilities: 0.0838, 0.9010, 0.0152)\n","Processed WP_119313480.1 mersacidin/lichenicidin family type 2 lantibiotic [Calidithermus terrae]: non-AMP/OTHERS (Probabilities: 0.8307, 0.0167, 0.1526)\n","Processed WP_111320052.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis]: AMP (Probabilities: 0.0926, 0.8948, 0.0125)\n","Processed WP_111320058.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis]: AMP (Probabilities: 0.1298, 0.7935, 0.0767)\n","Processed WP_111320056.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis]: AMP (Probabilities: 0.2346, 0.5363, 0.2291)\n","Processed WP_111320054.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis]: Neuropeptide (Probabilities: 0.2344, 0.0539, 0.7118)\n","Processed WP_111319848.1 cinnamycin family lantibiotic [Thermosporothrix hazakensis]: non-AMP/OTHERS (Probabilities: 0.9397, 0.0164, 0.0439)\n","Processed WP_111321320.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis]: non-AMP/OTHERS (Probabilities: 0.7068, 0.0252, 0.2680)\n","Processed WP_111323061.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis]: non-AMP/OTHERS (Probabilities: 0.6337, 0.0289, 0.3374)\n","Processed WP_111323434.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis]: AMP (Probabilities: 0.0663, 0.9195, 0.0143)\n","Processed WP_111323438.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis]: AMP (Probabilities: 0.0741, 0.9126, 0.0133)\n","Processed WP_083895979.1 gallidermin/nisin family lantibiotic [Nocardia]: non-AMP/OTHERS (Probabilities: 0.9708, 0.0255, 0.0037)\n","Processed WP_111323432.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis]: non-AMP/OTHERS (Probabilities: 0.5529, 0.0933, 0.3538)\n","Processed WP_084525506.1 gallidermin/nisin family lantibiotic [Nocardia vaccinii]: non-AMP/OTHERS (Probabilities: 0.8517, 0.1413, 0.0071)\n","Processed WP_170143021.1 gallidermin/nisin family lantibiotic [Thermosporothrix hazakensis]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0060, 0.0029)\n","Processed WP_065959517.1 plantaricin C family lantibiotic [Streptomyces sparsogenes]: Neuropeptide (Probabilities: 0.1539, 0.0266, 0.8195)\n","Processed WP_141310909.1 lexapeptide family class V lantibiotic [Streptomyces spinoverrucosus]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0053, 0.0038)\n","Processed WP_111323436.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis]: AMP (Probabilities: 0.0708, 0.9162, 0.0130)\n","Processed WP_063181216.1 elgicin/penisin family lantibiotic [Paenibacillus elgii]: AMP (Probabilities: 0.1242, 0.8340, 0.0418)\n","Processed WP_201823631.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces musisoli]: Neuropeptide (Probabilities: 0.0248, 0.0155, 0.9597)\n","Processed WP_063181218.1 elgicin/penisin family lantibiotic [Paenibacillus elgii]: AMP (Probabilities: 0.0975, 0.8844, 0.0181)\n","Processed WP_094181690.1 thiazolylpeptide-type bacteriocin [Cellulomonas]: non-AMP/OTHERS (Probabilities: 0.7493, 0.2432, 0.0075)\n","Processed WP_063180406.1 gallidermin/nisin family lantibiotic [Lysinibacillus odysseyi]: AMP (Probabilities: 0.0697, 0.9169, 0.0134)\n","Processed WP_023541205.1 thiazolylpeptide-type bacteriocin [Streptomyces niveus]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed MBV6622422.1 mersacidin/lichenicidin family type 2 lantibiotic [Rivularia sp. (in: Bacteria)]: non-AMP/OTHERS (Probabilities: 0.7673, 0.0299, 0.2028)\n","Processed MBV6622421.1 mersacidin/lichenicidin family type 2 lantibiotic [Rivularia sp. (in: Bacteria)]: Neuropeptide (Probabilities: 0.1952, 0.0817, 0.7231)\n","Processed WP_025851113.1 elgicin/penisin family lantibiotic [Paenibacillus ehimensis]: AMP (Probabilities: 0.2827, 0.5271, 0.1902)\n","Processed MBV6622466.1 mersacidin/lichenicidin family type 2 lantibiotic [Rivularia sp. (in: Bacteria)]: AMP (Probabilities: 0.2925, 0.4852, 0.2223)\n","Processed WP_141557059.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.7616, 0.2252, 0.0133)\n","Processed WP_141559676.1 plantaricin C family lantibiotic, partial [Bacillus thuringiensis]: Neuropeptide (Probabilities: 0.0095, 0.0094, 0.9811)\n","Processed WP_218013353.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.1036, 0.8842, 0.0122)\n","Processed WP_176528201.1 lichenicidin A2 family type 2 lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0049, 0.0028)\n","Processed WP_098720203.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: Neuropeptide (Probabilities: 0.0599, 0.0215, 0.9186)\n","Processed WP_141552586.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9892, 0.0031, 0.0078)\n","Processed WP_141547655.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0042, 0.0055)\n","Processed WP_218012173.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.1119, 0.8761, 0.0121)\n","Processed WP_067909139.1 cinnamycin family lantibiotic [Actinomadura rubrobrunea]: non-AMP/OTHERS (Probabilities: 0.9826, 0.0119, 0.0055)\n","Processed WP_067962807.1 cinnamycin family lantibiotic [Nocardiopsis trehalosi]: AMP (Probabilities: 0.0946, 0.8930, 0.0124)\n","Processed WP_098627778.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9054, 0.0102, 0.0844)\n","Processed WP_218009026.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0029, 0.0049)\n","Processed WP_067839410.1 lichenicidin A2 family type 2 lantibiotic [Amphibacillus sediminis]: AMP (Probabilities: 0.0661, 0.9185, 0.0155)\n","Processed WP_067839407.1 lichenicidin A2 family type 2 lantibiotic [Amphibacillus sediminis]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0039, 0.0054)\n","Processed WP_098561344.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed WP_218009025.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.0733, 0.9139, 0.0128)\n","Processed WP_067136986.1 mersacidin/lichenicidin family type 2 lantibiotic [Microtetraspora malaysiensis]: non-AMP/OTHERS (Probabilities: 0.7928, 0.0188, 0.1884)\n","Processed WP_066941594.1 mersacidin/lichenicidin family type 2 lantibiotic [Microtetraspora fusca]: non-AMP/OTHERS (Probabilities: 0.9870, 0.0047, 0.0083)\n","Processed WP_066941592.1 mersacidin/lichenicidin family type 2 lantibiotic [Microtetraspora fusca]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0056, 0.0031)\n","Processed WP_067625256.1 mersacidin/lichenicidin family type 2 lantibiotic [Alicyclobacillus acidiphilus]: AMP (Probabilities: 0.1046, 0.8830, 0.0124)\n","Processed WP_067451807.1 cinnamycin family lantibiotic [Actinomadura macra]: non-AMP/OTHERS (Probabilities: 0.6772, 0.0445, 0.2783)\n","Processed WP_057598023.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacilli]: non-AMP/OTHERS (Probabilities: 0.6363, 0.0302, 0.3335)\n","Processed WP_098300521.1 FDLD family class I lanthipeptide, partial [Bacillus cereus]: AMP (Probabilities: 0.0837, 0.9041, 0.0122)\n","Processed WP_098155368.1 plantaricin C family lantibiotic [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9703, 0.0056, 0.0241)\n","Processed WP_098362637.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: Neuropeptide (Probabilities: 0.2014, 0.0405, 0.7580)\n","Processed WP_029443586.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9790, 0.0178, 0.0032)\n","Processed WP_046199680.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus sp. UMTAT18]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0037, 0.0038)\n","Processed WP_176418485.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Myxococcus]: Neuropeptide (Probabilities: 0.3193, 0.0315, 0.6491)\n","Processed WP_140420103.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8821, 0.0275, 0.0904)\n","Processed WP_098155366.1 plantaricin C family lantibiotic [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0045, 0.0029)\n","Processed WP_050427170.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.5962, 0.3910, 0.0127)\n","Processed WP_140420038.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8823, 0.0352, 0.0825)\n","Processed WP_176417165.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Myxococcus]: Neuropeptide (Probabilities: 0.2685, 0.0648, 0.6667)\n","Processed WP_163869062.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus]: Neuropeptide (Probabilities: 0.1929, 0.0318, 0.7753)\n","Processed WP_176417164.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Myxococcus]: non-AMP/OTHERS (Probabilities: 0.5965, 0.0376, 0.3660)\n","Processed WP_141096717.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.6557, 0.0292, 0.3152)\n","Processed WP_088325178.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.5402, 0.0343, 0.4255)\n","Processed WP_088309454.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus group]: non-AMP/OTHERS (Probabilities: 0.9544, 0.0401, 0.0055)\n","Processed WP_088319975.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9713, 0.0125, 0.0162)\n","Processed WP_088361626.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9434, 0.0387, 0.0180)\n","Processed WP_087433874.1 lichenicidin A2 family type 2 lantibiotic [Paenibacillus apiarius]: non-AMP/OTHERS (Probabilities: 0.9864, 0.0050, 0.0086)\n","Processed WP_088304854.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9841, 0.0087, 0.0072)\n","Processed WP_088339242.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9525, 0.0418, 0.0057)\n","Processed WP_029443585.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0071, 0.0040)\n","Processed WP_088021322.1 gallidermin/nisin family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9612, 0.0209, 0.0178)\n","Processed WP_053512397.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0716, 0.9151, 0.0133)\n","Processed WP_086412513.1 plantaricin C family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.0716, 0.9151, 0.0134)\n","Processed WP_086391185.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9846, 0.0113, 0.0041)\n","Processed WP_086412522.1 plantaricin C family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.0842, 0.9037, 0.0121)\n","Processed WP_000033706.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9872, 0.0074, 0.0054)\n","Processed WP_053512375.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0837, 0.9042, 0.0121)\n","Processed WP_086406557.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0736, 0.9138, 0.0126)\n","Processed HIY40384.1 lantibiotic protein [Candidatus Nocardiopsis merdipullorum]: non-AMP/OTHERS (Probabilities: 0.9539, 0.0144, 0.0317)\n","Processed WP_065211598.1 plantaricin C family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.0887, 0.8993, 0.0120)\n","Processed WP_079528026.1 plantaricin C family lantibiotic [Halobacillus hunanensis]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0040, 0.0041)\n","Processed WP_078206713.1 gallidermin/nisin family lantibiotic, partial [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0035, 0.0068)\n","Processed WP_077701886.1 plantaricin C family lantibiotic [Virgibacillus dokdonensis]: Neuropeptide (Probabilities: 0.1242, 0.0402, 0.8356)\n","Processed WP_073810233.1 thiazolylpeptide-type bacteriocin [Kitasatospora]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0032, 0.0039)\n","Processed WP_083578411.1 gallidermin/nisin family lantibiotic [Bacillus sp. L27]: AMP (Probabilities: 0.0660, 0.9189, 0.0152)\n","Processed WP_143724362.1 gallidermin/nisin family lantibiotic [Bifidobacterium]: non-AMP/OTHERS (Probabilities: 0.4162, 0.2811, 0.3027)\n","Processed WP_076871442.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus group]: AMP (Probabilities: 0.2297, 0.7577, 0.0126)\n","Processed WP_140337868.1 plantaricin C family lantibiotic, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.0670, 0.9190, 0.0140)\n","Processed WP_071709070.1 gallidermin/nisin family lantibiotic [Bacillus sp. L27]: AMP (Probabilities: 0.0662, 0.9188, 0.0150)\n","Processed WP_071709069.1 gallidermin/nisin family lantibiotic [Bacillus sp. L27]: AMP (Probabilities: 0.0681, 0.9184, 0.0135)\n","Processed WP_217569243.1 lantibiotic protein [Streptomyces sp. GbtcB7]: non-AMP/OTHERS (Probabilities: 0.9756, 0.0186, 0.0058)\n","Processed WP_175063860.1 gallidermin/nisin family lantibiotic [Streptococcus salivarius]: AMP (Probabilities: 0.0760, 0.9113, 0.0127)\n","Processed WP_045863412.1 plantaricin C family lantibiotic [Streptomyces sp. WMMB 714]: AMP (Probabilities: 0.1015, 0.8854, 0.0132)\n","Processed WP_029175173.1 plantaricin C family lantibiotic [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.5567, 0.4320, 0.0113)\n","Processed WP_046199679.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Bacillus (in: Bacteria)]: non-AMP/OTHERS (Probabilities: 0.9859, 0.0091, 0.0050)\n","Processed WP_175063859.1 gallidermin/nisin family lantibiotic [Streptococcus salivarius]: AMP (Probabilities: 0.0731, 0.9140, 0.0129)\n","Processed WP_217425351.1 plantaricin C family lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0699, 0.9169, 0.0132)\n","Processed WP_136020033.1 lantibiotic streptin [Streptococcus pyogenes]: AMP (Probabilities: 0.0678, 0.9168, 0.0154)\n","Processed WP_174402585.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0903, 0.8976, 0.0121)\n","Processed WP_161501886.1 lantibiotic streptin [Streptococcus agalactiae]: AMP (Probabilities: 0.0682, 0.9170, 0.0148)\n","Processed WP_217428720.1 ABC transporter permease, partial [Gardnerella vaginalis]: non-AMP/OTHERS (Probabilities: 0.9887, 0.0084, 0.0029)\n","Processed DAZ09722.1 NAG-MUB-ALA-FGA-LYS-DAL-DAL PEPTIDE, NISIN Z-ANTIBIOTIC COMPLEX, LANTIBIOTIC, ANTIMICROBIAL, BACTERIOCIN [Siphoviridae sp.]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0059, 0.0025)\n","Processed WP_217424346.1 lantibiotic (srt) production protein, partial [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0033)\n","Processed WP_128100884.1 gallidermin/nisin family lantibiotic [Paenibacillus sp. DCT19]: Neuropeptide (Probabilities: 0.1877, 0.0310, 0.7813)\n","Processed WP_217210728.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sp. AC550_RSS872]: Neuropeptide (Probabilities: 0.1644, 0.0438, 0.7918)\n","Processed DAX86025.1 Type-A lantibiotic [Siphoviridae sp.]: AMP (Probabilities: 0.0762, 0.9113, 0.0125)\n","Processed DAW89727.1 gallidermin-class lantibiotic protection ABC transporter, ATP-binding subunit [Podoviridae sp.]: non-AMP/OTHERS (Probabilities: 0.9671, 0.0276, 0.0053)\n","Processed WP_149561018.1 lantibiotic salivaricin M precursor [Streptococcus]: AMP (Probabilities: 0.0647, 0.9205, 0.0147)\n","Processed DAV57865.1 Type A lantibiotic family [Myoviridae sp.]: AMP (Probabilities: 0.1272, 0.8610, 0.0118)\n","Processed WP_168491791.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Anabaena]: non-AMP/OTHERS (Probabilities: 0.9853, 0.0056, 0.0091)\n","Processed DAK72859.1 Type A lantibiotic family [Siphoviridae sp.]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0036)\n","Processed WP_167670170.1 elgicin/penisin family lantibiotic [Paenibacillus tianmuensis]: AMP (Probabilities: 0.0735, 0.9127, 0.0138)\n","Processed WP_143006866.1 elgicin/penisin family lantibiotic [Paenibacillus tianmuensis]: AMP (Probabilities: 0.1868, 0.7950, 0.0181)\n","Processed WP_172864306.1 lantipeptide cytolysin subunit CylL-L, partial [Enterococcus faecalis]: AMP (Probabilities: 0.0814, 0.9062, 0.0124)\n","Processed WP_173224275.1 plantaricin C family lantibiotic [Kroppenstedtia pulmonis]: non-AMP/OTHERS (Probabilities: 0.9519, 0.0105, 0.0376)\n","Processed WP_199745882.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus]: Neuropeptide (Probabilities: 0.2923, 0.0291, 0.6786)\n","Processed WP_199749692.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus]: Neuropeptide (Probabilities: 0.1205, 0.0305, 0.8490)\n","Processed WP_143006865.1 elgicin/penisin family lantibiotic [Paenibacillus tianmuensis]: AMP (Probabilities: 0.1083, 0.8770, 0.0146)\n","Processed WP_090670885.1 elgicin/penisin family lantibiotic [Paenibacillus tianmuensis]: AMP (Probabilities: 0.0742, 0.9126, 0.0132)\n","Processed WP_153907241.1 gallidermin/nisin family lantibiotic [Staphylococcus capitis]: AMP (Probabilities: 0.0666, 0.9191, 0.0143)\n","Processed WP_060620598.1 hypothetical protein [Bifidobacterium longum]: non-AMP/OTHERS (Probabilities: 0.7363, 0.1065, 0.1572)\n","Processed WP_049099135.1 lantipeptide cytolysin subunit CylL-L, partial [Enterococcus faecalis]: AMP (Probabilities: 0.0695, 0.9160, 0.0145)\n","Processed WP_216588465.1 plantaricin C family lantibiotic [Streptomyces brasiliscabiei]: non-AMP/OTHERS (Probabilities: 0.7978, 0.1884, 0.0138)\n","Processed WP_143477491.1 NisI/SpaI family lantibiotic immunity lipoprotein, partial [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0063, 0.0035)\n","Processed EGT5258581.1 lantibiotic immunity ABC transporter MutG family permease subunit [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0052, 0.0032)\n","Processed WP_120379436.1 plantaricin C family lantibiotic [Firmicutes]: Neuropeptide (Probabilities: 0.1559, 0.0366, 0.8075)\n","Processed WP_197349700.1 gallidermin/nisin family lantibiotic [Streptomyces bathyalis]: non-AMP/OTHERS (Probabilities: 0.9684, 0.0164, 0.0151)\n","Processed NJB05075.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9881, 0.0095, 0.0024)\n","Processed NJB05074.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0088, 0.0024)\n","Processed WP_081317837.1 gallidermin/nisin family lantibiotic [Paenibacillus]: AMP (Probabilities: 0.0716, 0.9153, 0.0131)\n","Processed EGT5256699.1 lantibiotic immunity ABC transporter MutG family permease subunit [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0061, 0.0029)\n","Processed WP_216831391.1 gallidermin/nisin family lantibiotic [Bacillus sp. MEB199]: AMP (Probabilities: 0.0739, 0.9125, 0.0135)\n","Processed WP_084043486.1 gallidermin/nisin family lantibiotic [Blautia pseudococcoides]: AMP (Probabilities: 0.0971, 0.8907, 0.0122)\n","Processed WP_065541934.1 gallidermin/nisin family lantibiotic [Blautia pseudococcoides]: AMP (Probabilities: 0.1304, 0.8465, 0.0231)\n","Processed WP_216798027.1 elgicin/penisin family lantibiotic [Paenibacillus sp. SM 69]: AMP (Probabilities: 0.2727, 0.6475, 0.0798)\n","Processed WP_169166707.1 plantaricin C family lantibiotic [Cellulomonas taurus]: non-AMP/OTHERS (Probabilities: 0.6245, 0.0430, 0.3325)\n","Processed WP_143630540.1 plantaricin C family lantibiotic [Streptomyces sp. 1-11]: AMP (Probabilities: 0.1369, 0.8243, 0.0388)\n","Processed WP_216759756.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.2414, 0.0340, 0.7246)\n","Processed WP_162952487.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces hundungensis]: non-AMP/OTHERS (Probabilities: 0.9852, 0.0044, 0.0104)\n","Processed WP_058047571.1 lexapeptide family class V lantibiotic [Streptomyces roseifaciens]: Neuropeptide (Probabilities: 0.0124, 0.0112, 0.9765)\n","Processed WP_037796982.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0040)\n","Processed MBU7065573.1 lantibiotic ABC transporter ATP-binding protein [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.0272, 0.0145, 0.9582)\n","Processed DAO82767.1 Lantibiotic dehydratase domain protein, nisin, tRNA-dependent, dehydratase, BIOSYNTHETIC [Siphoviridae sp.]: AMP (Probabilities: 0.1247, 0.8274, 0.0479)\n","Processed WP_216731829.1 lichenicidin A2 family type 2 lantibiotic, partial [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.3437, 0.0300, 0.6263)\n","Processed WP_216731869.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.8337, 0.1067, 0.0596)\n","Processed DAQ58506.1 Lantibiotic dehydratase domain protein, nisin, tRNA-dependent, dehydratase, BIOSYNTHETIC [Podoviridae sp.]: AMP (Probabilities: 0.0680, 0.9175, 0.0145)\n","Processed MBU7117224.1 lantibiotic ABC transporter ATP-binding protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0036, 0.0039)\n","Processed WP_019395593.1 plantaricin C family lantibiotic [Priestia filamentosa]: AMP (Probabilities: 0.0720, 0.9151, 0.0130)\n","Processed WP_002268803.1 mutacin-like lantipeptide [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0053, 0.0038)\n","Processed WP_216734345.1 lichenicidin A2 family type 2 lantibiotic, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0030, 0.0051)\n","Processed WP_199727750.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus]: Neuropeptide (Probabilities: 0.1781, 0.0413, 0.7806)\n","Processed WP_216734346.1 lichenicidin A2 family type 2 lantibiotic, partial [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.3999, 0.0261, 0.5740)\n","Processed WP_084159534.1 gallidermin/nisin family lantibiotic [Paenibacillus ehimensis]: non-AMP/OTHERS (Probabilities: 0.9670, 0.0187, 0.0143)\n","Processed WP_157462714.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus sp. JH7]: non-AMP/OTHERS (Probabilities: 0.7962, 0.0347, 0.1691)\n","Processed WP_015133375.1 mersacidin/lichenicidin family type 2 lantibiotic [Leptolyngbya sp. PCC 7376]: non-AMP/OTHERS (Probabilities: 0.9709, 0.0211, 0.0080)\n","Processed WP_173864623.1 gallidermin/nisin family lantibiotic [Lachnospiraceae]: AMP (Probabilities: 0.0669, 0.9187, 0.0144)\n","Processed WP_155729784.1 gallidermin/nisin family lantibiotic [Paenibacillus polymyxa]: AMP (Probabilities: 0.1828, 0.7690, 0.0482)\n","Processed WP_154778450.1 plantaricin C family lantibiotic [Erwinia sp. CPCC 100877]: non-AMP/OTHERS (Probabilities: 0.9165, 0.0729, 0.0105)\n","Processed WP_154778451.1 plantaricin C family lantibiotic [Erwinia sp. CPCC 100877]: Neuropeptide (Probabilities: 0.1087, 0.0333, 0.8580)\n","Processed WP_082063447.1 lichenicidin A2 family type 2 lantibiotic [unclassified Anoxybacillus]: AMP (Probabilities: 0.2078, 0.7825, 0.0096)\n","Processed WP_061139778.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus sp. JH7]: non-AMP/OTHERS (Probabilities: 0.9700, 0.0262, 0.0038)\n","Processed WP_216399246.1 plantaricin C family lantibiotic [Amphibacillus sp. MSJ-3]: Neuropeptide (Probabilities: 0.0703, 0.0322, 0.8976)\n","Processed WP_216459490.1 gallidermin/nisin family lantibiotic [Blautia sp. MSJ-9]: AMP (Probabilities: 0.0668, 0.9185, 0.0147)\n","Processed WP_063280252.1 gallidermin/nisin family lantibiotic [Lactococcus]: AMP (Probabilities: 0.0683, 0.9172, 0.0145)\n","Processed WP_171426526.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus coralloides]: Neuropeptide (Probabilities: 0.4334, 0.0688, 0.4978)\n","Processed WP_216398064.1 plantaricin C family lantibiotic [Anaerocolumna aminovalerica]: AMP (Probabilities: 0.4118, 0.4255, 0.1627)\n","Processed WP_141252695.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus]: non-AMP/OTHERS (Probabilities: 0.6905, 0.0244, 0.2852)\n","Processed WP_117640231.1 gallidermin/nisin family lantibiotic [Lachnospiraceae]: AMP (Probabilities: 0.0831, 0.9047, 0.0122)\n","Processed WP_041171466.1 hypothetical protein [Streptococcus pneumoniae]: AMP (Probabilities: 0.0641, 0.9211, 0.0147)\n","Processed WP_117640235.1 gallidermin/nisin family lantibiotic [Lachnospiraceae]: AMP (Probabilities: 0.0668, 0.9187, 0.0146)\n","Processed WP_141254677.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus]: non-AMP/OTHERS (Probabilities: 0.5224, 0.0395, 0.4381)\n","Processed WP_141254678.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus]: non-AMP/OTHERS (Probabilities: 0.6974, 0.0487, 0.2539)\n","Processed WP_080120898.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0032, 0.0034)\n","Processed WP_216368849.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter arantiisoli]: Neuropeptide (Probabilities: 0.2670, 0.2163, 0.5166)\n","Processed WP_046648594.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: AMP (Probabilities: 0.2323, 0.7533, 0.0144)\n","Processed WP_019497585.1 mersacidin/lichenicidin family type 2 lantibiotic [Calothrix sp. PCC 7103]: non-AMP/OTHERS (Probabilities: 0.5521, 0.0638, 0.3841)\n","Processed WP_216369478.1 mersacidin/lichenicidin family type 2 lantibiotic [Brevibacillus sp. MCWH]: non-AMP/OTHERS (Probabilities: 0.9543, 0.0107, 0.0351)\n","Processed WP_137073824.1 plantaricin C family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.0831, 0.9049, 0.0119)\n","Processed WP_137073821.1 plantaricin C family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.4626, 0.5193, 0.0181)\n","Processed WP_171441066.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: Neuropeptide (Probabilities: 0.0934, 0.0283, 0.8782)\n","Processed WP_000041631.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0036, 0.0032)\n","Processed WP_094794630.1 gallidermin/nisin family lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.6798, 0.0418, 0.2785)\n","Processed WP_137073822.1 plantaricin C family lantibiotic [Bacillus cereus group]: AMP (Probabilities: 0.0671, 0.9184, 0.0145)\n","Processed WP_085565759.1 FxLD family lanthipeptide [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.7326, 0.0184, 0.2490)\n","Processed WP_000041630.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0038, 0.0036)\n","Processed DAL71937.1 Lantibiotic alpha [Siphoviridae sp.]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0029)\n","Processed DAK83931.1 Lantibiotic dehydratase domain protein, nisin, tRNA-dependent, dehydratase, BIOSYNTHETIC [Siphoviridae sp.]: AMP (Probabilities: 0.1947, 0.7635, 0.0418)\n","Processed HBH1936423.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0071, 0.0029)\n","Processed WP_216116224.1 plantaricin C family lantibiotic [Clostridium gasigenes]: AMP (Probabilities: 0.1389, 0.8392, 0.0219)\n","Processed WP_107421600.1 plantaricin C family lantibiotic [Streptomyces nanshensis]: AMP (Probabilities: 0.1007, 0.8865, 0.0128)\n","Processed WP_074416048.1 lichenicidin A2 family type 2 lantibiotic [Streptococcus suis]: AMP (Probabilities: 0.1076, 0.8809, 0.0115)\n","Processed WP_216126978.1 plantaricin C family lantibiotic [Clostridium estertheticum]: Neuropeptide (Probabilities: 0.1431, 0.0270, 0.8299)\n","Processed DAL90676.1 Type-A lantibiotic [Myoviridae sp.]: Neuropeptide (Probabilities: 0.0080, 0.0091, 0.9829)\n","Processed MBJ0655176.1 lantipeptide cytolysin subunit CylL-S [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed WP_073689034.1 lantibiotic salivaricin M precursor [Streptococcus salivarius]: AMP (Probabilities: 0.2378, 0.7430, 0.0192)\n","Processed MBU5584571.1 lantibiotic ABC transporter ATP-binding protein [Enterococcus sp. S181_ASV_20]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0049, 0.0030)\n","Processed WP_070012661.1 thiazolylpeptide-type bacteriocin [Streptomyces abyssalis]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0047, 0.0040)\n","Processed MBU3806138.1 NisI/SpaI family lantibiotic immunity lipoprotein [Candidatus Fournierella pullistercoris]: non-AMP/OTHERS (Probabilities: 0.7846, 0.2078, 0.0077)\n","Processed DAH58236.1 Immun Lantibiotic streptin immunity protein [Bacteriophage sp.]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0039, 0.0048)\n","Processed MBU5282396.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Anaerostipes hadrus]: non-AMP/OTHERS (Probabilities: 0.9058, 0.0136, 0.0806)\n","Processed WP_015192462.1 mersacidin/lichenicidin family type 2 lantibiotic [Stanieria cyanosphaera]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0034, 0.0054)\n","Processed DAG34568.1 Type-A lantibiotic [Siphoviridae sp.]: non-AMP/OTHERS (Probabilities: 0.8778, 0.0315, 0.0907)\n","Processed MBU3216822.1 lantibiotic ABC transporter ATP-binding protein [Clostridium estertheticum]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0041, 0.0030)\n","Processed WP_015192458.1 mersacidin/lichenicidin family type 2 lantibiotic [Stanieria cyanosphaera]: non-AMP/OTHERS (Probabilities: 0.9868, 0.0082, 0.0050)\n","Processed WP_015192461.1 mersacidin/lichenicidin family type 2 lantibiotic [Stanieria cyanosphaera]: AMP (Probabilities: 0.1033, 0.8848, 0.0119)\n","Processed WP_015192459.1 mersacidin/lichenicidin family type 2 lantibiotic [Stanieria cyanosphaera]: AMP (Probabilities: 0.3050, 0.6811, 0.0139)\n","Processed WP_205145341.1 gallidermin/nisin family lantibiotic [Weissella uvarum]: AMP (Probabilities: 0.0757, 0.9106, 0.0137)\n","Processed WP_137062363.1 FDLD family class I lanthipeptide [Paenibacillus terrae]: AMP (Probabilities: 0.0968, 0.8910, 0.0122)\n","Processed WP_215577533.1 FDLD family class I lanthipeptide [Bacillus mycoides]: AMP (Probabilities: 0.1787, 0.8090, 0.0123)\n","Processed WP_215601155.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus mycoides]: AMP (Probabilities: 0.1104, 0.8771, 0.0125)\n","Processed WP_215601153.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus mycoides]: AMP (Probabilities: 0.1566, 0.8268, 0.0166)\n","Processed WP_064670866.1 plantaricin C family lantibiotic [Bacillus subtilis]: Neuropeptide (Probabilities: 0.0161, 0.0124, 0.9715)\n","Processed WP_090557569.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus subtilis]: non-AMP/OTHERS (Probabilities: 0.5639, 0.4233, 0.0129)\n","Processed HBG0097569.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9839, 0.0129, 0.0033)\n","Processed WP_044491894.1 cinnamycin family lantibiotic [Moorea producens]: non-AMP/OTHERS (Probabilities: 0.9650, 0.0093, 0.0257)\n","Processed MBT9761096.1 lantibiotic ABC transporter ATP-binding protein [Collinsella aerofaciens]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0052, 0.0030)\n","Processed QTR06364.1 lantibiotic dehydratase, partial [Saccharothrix algeriensis]: non-AMP/OTHERS (Probabilities: 0.9122, 0.0830, 0.0048)\n","Processed WP_137365191.1 NisI/SpaI family lantibiotic immunity lipoprotein, partial [Citrobacter sp. TBCS-15]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0040, 0.0031)\n","Processed QTR02238.1 lantibiotic dehydratase, partial [Saccharothrix algeriensis]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0037, 0.0035)\n","Processed  unnamed protein product [Unknown]: AMP (Probabilities: 0.1152, 0.8722, 0.0126)\n","Processed HCW68973.1 lantibiotic ABC transporter [Thalassospira lucentensis]: non-AMP/OTHERS (Probabilities: 0.9876, 0.0047, 0.0077)\n","Processed WP_008178013.1 cinnamycin family lantibiotic [Moorea producens]: non-AMP/OTHERS (Probabilities: 0.9559, 0.0152, 0.0289)\n","Processed WP_124043837.1 plantaricin C family lantibiotic [Bacillus subtilis]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0044, 0.0041)\n","Processed WP_048516917.1 gallidermin/nisin family lantibiotic [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9935, 0.0036, 0.0030)\n","Processed HBF2224879.1 gallidermin/nisin family lantibiotic [Clostridioides difficile]: AMP (Probabilities: 0.0790, 0.9084, 0.0126)\n","Processed QWH03937.1 lantibiotic cytolysin [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9853, 0.0062, 0.0086)\n","Processed WP_078510641.1 plantaricin C family lantibiotic [Streptomyces himastatinicus]: AMP (Probabilities: 0.0842, 0.9035, 0.0123)\n","Processed WP_033717357.1 gallidermin/nisin family lantibiotic [Bacillus]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0058, 0.0027)\n","Processed HBF7166060.1 lantibiotic ABC transporter permease [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0045, 0.0030)\n","Processed HBF2224880.1 gallidermin/nisin family lantibiotic [Clostridioides difficile]: AMP (Probabilities: 0.0685, 0.9184, 0.0131)\n","Processed WP_151190481.1 gallidermin/nisin family lantibiotic [Lachnospiraceae]: AMP (Probabilities: 0.0667, 0.9186, 0.0147)\n","Processed HBF4863944.1 lantibiotic ABC transporter ATP-binding protein [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0040, 0.0029)\n","Processed MBT9260200.1 mersacidin/lichenicidin family type 2 lantibiotic [Clostridiales bacterium]: AMP (Probabilities: 0.1142, 0.8679, 0.0179)\n","Processed WP_004183548.1 hypothetical protein [Streptococcus salivarius]: AMP (Probabilities: 0.0695, 0.9173, 0.0133)\n","Processed WP_214801216.1 plantaricin C family lantibiotic [Bacteria]: Neuropeptide (Probabilities: 0.0218, 0.0152, 0.9630)\n","Processed MBD4203740.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Xanthomonas citri pv. citri LL095-20]: non-AMP/OTHERS (Probabilities: 0.9875, 0.0097, 0.0028)\n","Processed MBD4339284.1 NisI/SpaI family lantibiotic immunity lipoprotein [Xanthomonas citri pv. citri JP541]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0042, 0.0031)\n","Processed MBD4341774.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Xanthomonas citri pv. citri LN005-07]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0066, 0.0025)\n","Processed MBD4371633.1 lantibiotic ABC transporter ATP-binding protein [Xanthomonas citri pv. citri LL174-02]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0037, 0.0048)\n","Processed HBF3623183.1 NisI/SpaI family lantibiotic immunity lipoprotein [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9822, 0.0136, 0.0042)\n","Processed MBD4371634.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Xanthomonas citri pv. citri LL174-02]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0075, 0.0028)\n","Processed MBD4597349.1 lantibiotic dehydratase [Xanthomonas citri pv. citri LL132-01]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0038, 0.0031)\n","Processed WP_017798567.1 plantaricin C family lantibiotic [Oceanobacillus]: non-AMP/OTHERS (Probabilities: 0.5703, 0.0319, 0.3978)\n","Processed WP_215029905.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sp. ISL-66]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0055, 0.0044)\n","Processed WP_215179456.1 plantaricin C family lantibiotic [Paenibacillus albidus]: Neuropeptide (Probabilities: 0.0118, 0.0111, 0.9771)\n","Processed MBD4597440.1 gallidermin/nisin family lantibiotic [Xanthomonas citri pv. citri LL132-01]: non-AMP/OTHERS (Probabilities: 0.9869, 0.0039, 0.0092)\n","Processed WP_055970079.1 lichenicidin A2 family type 2 lantibiotic [Exiguobacterium]: non-AMP/OTHERS (Probabilities: 0.9838, 0.0052, 0.0111)\n","Processed WP_019523030.1 SCO0268 family class II lantipeptide [unclassified Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9177, 0.0094, 0.0729)\n","Processed MBD4375713.1 gallidermin/nisin family lantibiotic [Xanthomonas citri pv. citri LL174-02]: Neuropeptide (Probabilities: 0.0464, 0.0224, 0.9312)\n","Processed WP_164427567.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces rhizosphaericus]: non-AMP/OTHERS (Probabilities: 0.4300, 0.1699, 0.4001)\n","Processed WP_164427571.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces rhizosphaericus]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0049, 0.0053)\n","Processed MBD5075453.1 lantibiotic ABC transporter ATP-binding protein PsdA [Xanthomonas citri pv. citri LK126-02]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0040, 0.0034)\n","Processed WP_164433615.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.7279, 0.0967, 0.1755)\n","Processed WP_164427569.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces rhizosphaericus]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0058, 0.0038)\n","Processed WP_037960079.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces]: Neuropeptide (Probabilities: 0.3896, 0.0679, 0.5425)\n","Processed WP_065436484.1 hypothetical protein [Bifidobacterium]: non-AMP/OTHERS (Probabilities: 0.4345, 0.1378, 0.4277)\n","Processed WP_044579866.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces violaceusniger group]: Neuropeptide (Probabilities: 0.1785, 0.0593, 0.7622)\n","Processed P0DQL4.1 Lantibiotic Flvbeta.b [Ruminococcus flavefaciens]: AMP (Probabilities: 0.2032, 0.7780, 0.0188)\n","Processed P86047.2 Lantibiotic epilancin 15X [Staphylococcus epidermidis]: AMP (Probabilities: 0.0750, 0.9124, 0.0126)\n","Processed P56650.3 Lantibiotic actagardine [Actinoplanes garbadinensis]: non-AMP/OTHERS (Probabilities: 0.9871, 0.0076, 0.0053)\n","Processed WP_088112221.1 plantaricin C family lantibiotic [Bacillus subtilis]: Neuropeptide (Probabilities: 0.0229, 0.0147, 0.9624)\n","Processed P0DQL7.1 Lantibiotic Flvbeta.e [Ruminococcus flavefaciens]: AMP (Probabilities: 0.0825, 0.9043, 0.0131)\n","Processed P0DQL5.1 Lantibiotic Flvbeta.c [Ruminococcus flavefaciens]: AMP (Probabilities: 0.3977, 0.5890, 0.0134)\n","Processed HBE7324529.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0070, 0.0029)\n","Processed WP_161511139.1 type 2 lantibiotic [Staphylococcus haemolyticus]: non-AMP/OTHERS (Probabilities: 0.9839, 0.0118, 0.0043)\n","Processed WP_213433027.1 lichenicidin A2 family type 2 lantibiotic [Lactococcus garvieae]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0052, 0.0038)\n","Processed WP_213829221.1 lantibiotic ABC transporter ATP-binding protein, partial [Escherichia coli]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0051, 0.0037)\n","Processed P0DQL9.1 Lantibiotic Flvbeta.g [Ruminococcus flavefaciens]: non-AMP/OTHERS (Probabilities: 0.9793, 0.0184, 0.0024)\n","Processed WP_213171253.1 mersacidin/lichenicidin family type 2 lantibiotic [Natronoglycomyces albus]: AMP (Probabilities: 0.0711, 0.9157, 0.0132)\n","Processed WP_213171254.1 mersacidin/lichenicidin family type 2 lantibiotic [Natronoglycomyces albus]: AMP (Probabilities: 0.3237, 0.5932, 0.0832)\n","Processed WP_082926755.1 plantaricin C family lantibiotic [Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0052, 0.0036)\n","Processed WP_213451654.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora sp. NBRC 107566]: non-AMP/OTHERS (Probabilities: 0.5551, 0.0341, 0.4108)\n","Processed WP_213451652.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora sp. NBRC 107566]: AMP (Probabilities: 0.1142, 0.8602, 0.0257)\n","Processed WP_213433026.1 lichenicidin A2 family type 2 lantibiotic [Lactococcus garvieae]: AMP (Probabilities: 0.0895, 0.8958, 0.0146)\n","Processed WP_090557565.1 plantaricin C family lantibiotic [Bacillus subtilis]: AMP (Probabilities: 0.2135, 0.7736, 0.0129)\n","Processed WP_191139515.1 plantaricin C family lantibiotic [Hazenella sp. IB182353]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0043, 0.0033)\n","Processed WP_213411923.1 plantaricin C family lantibiotic [Xylanibacillus composti]: non-AMP/OTHERS (Probabilities: 0.9083, 0.0135, 0.0782)\n","Processed WP_191139513.1 plantaricin C family lantibiotic [unclassified Hazenella]: non-AMP/OTHERS (Probabilities: 0.9843, 0.0059, 0.0099)\n","Processed MBS8032810.1 lantibiotic ABC transporter permease [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9743, 0.0225, 0.0033)\n","Processed MBS8097913.1 NisI/SpaI family lantibiotic immunity lipoprotein [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0043, 0.0044)\n","Processed WP_121448753.1 plantaricin C family lantibiotic [Fredinandcohnia aciditolerans]: non-AMP/OTHERS (Probabilities: 0.5470, 0.4385, 0.0145)\n","Processed MBS8087101.1 NisI/SpaI family lantibiotic immunity lipoprotein [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9874, 0.0100, 0.0026)\n","Processed WP_065437246.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium longum]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0040)\n","Processed WP_140419193.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8586, 0.0239, 0.1174)\n","Processed MBS7264532.1 gallidermin family lantibiotic [Eubacteriales bacterium]: Neuropeptide (Probabilities: 0.0237, 0.0157, 0.9606)\n","Processed MBS9394247.1 mersacidin/lichenicidin family type 2 lantibiotic [Dolichospermum sp. OL01]: non-AMP/OTHERS (Probabilities: 0.9831, 0.0107, 0.0063)\n","Processed MBS8016460.1 lantibiotic salivaricin M precursor [Streptococcus suis]: Neuropeptide (Probabilities: 0.2468, 0.3282, 0.4250)\n","Processed MBS8016461.1 lantibiotic salivaricin M precursor [Streptococcus suis]: AMP (Probabilities: 0.0731, 0.9143, 0.0127)\n","Processed MBS6721206.1 plantaricin C family lantibiotic [Peptoniphilus harei]: non-AMP/OTHERS (Probabilities: 0.5329, 0.1242, 0.3429)\n","Processed MBS7214022.1 lantibiotic ABC transporter ATP-binding protein [Streptococcus salivarius]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0051, 0.0028)\n","Processed MBS6961154.1 lantibiotic ABC transporter permease [Roseburia sp.]: non-AMP/OTHERS (Probabilities: 0.9872, 0.0104, 0.0024)\n","Processed MBS5967318.1 plantaricin C family lantibiotic [Finegoldia magna]: AMP (Probabilities: 0.1914, 0.7671, 0.0415)\n","Processed MBS5466800.1 plantaricin C family lantibiotic [Clostridium sp.]: Neuropeptide (Probabilities: 0.1391, 0.0324, 0.8285)\n","Processed MBS4943498.1 gallidermin/nisin family lantibiotic [Actinomyces graevenitzii]: AMP (Probabilities: 0.0810, 0.9053, 0.0137)\n","Processed MBC6308472.1 plantaricin C family lantibiotic [Listeria portnoyi FSL L7-1582]: non-AMP/OTHERS (Probabilities: 0.7655, 0.0368, 0.1977)\n","Processed MBS6579501.1 plantaricin C family lantibiotic [Clostridiales bacterium]: Neuropeptide (Probabilities: 0.0090, 0.0096, 0.9814)\n","Processed MBS4974411.1 NisI/SpaI family lantibiotic immunity lipoprotein [Clostridium celatum]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0039, 0.0038)\n","Processed MBS5040563.1 gallidermin/nisin family lantibiotic [Streptococcus sp.]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0037, 0.0042)\n","Processed MBS4933017.1 gallidermin/nisin family lantibiotic [Clostridiales bacterium]: AMP (Probabilities: 0.0777, 0.9098, 0.0126)\n","Processed WP_148552174.1 gallidermin/nisin family lantibiotic [Paraclostridium bifermentans]: Neuropeptide (Probabilities: 0.0099, 0.0098, 0.9803)\n","Processed MBS5182112.1 lantibiotic ABC transporter permease [Streptococcus salivarius]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0058, 0.0026)\n","Processed WP_086527377.1 hypothetical protein [Clavibacter]: non-AMP/OTHERS (Probabilities: 0.5783, 0.1145, 0.3072)\n","Processed WP_212909505.1 plantaricin C family lantibiotic [Streptomyces sp. TS71-3]: AMP (Probabilities: 0.0990, 0.8881, 0.0129)\n","Processed WP_212925848.1 plantaricin C family lantibiotic [Oceanobacillus sp. J11TS1]: Neuropeptide (Probabilities: 0.0134, 0.0125, 0.9741)\n","Processed WP_197196623.1 plantaricin C family lantibiotic [Bacillus sonorensis]: Neuropeptide (Probabilities: 0.1263, 0.0382, 0.8355)\n","Processed WP_191937749.1 gallidermin/nisin family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0034, 0.0033)\n","Processed MBS5181975.1 lantibiotic ABC transporter permease [Streptococcus salivarius]: non-AMP/OTHERS (Probabilities: 0.9858, 0.0104, 0.0037)\n","Processed WP_194233303.1 gallidermin/nisin family lantibiotic [Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.9669, 0.0299, 0.0032)\n","Processed WP_212844514.1 mersacidin/lichenicidin family type 2 lantibiotic [Catellatospora sp. IY07-71]: Neuropeptide (Probabilities: 0.2381, 0.0304, 0.7314)\n","Processed WP_212846509.1 gallidermin/nisin family lantibiotic [Actinoplanes ianthinogenes]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0045, 0.0029)\n","Processed WP_030445134.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinocatenispora sera]: Neuropeptide (Probabilities: 0.0097, 0.0102, 0.9801)\n","Processed WP_211752180.1 gallidermin/nisin family lantibiotic [Lactococcus lactis]: AMP (Probabilities: 0.0664, 0.9188, 0.0148)\n","Processed WP_212114764.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0043, 0.0060)\n","Processed WP_212697151.1 elgicin/penisin family lantibiotic [Vallitalea pronyensis]: AMP (Probabilities: 0.2069, 0.7787, 0.0144)\n","Processed WP_193774429.1 elgicin/penisin family lantibiotic [Vallitalea guaymasensis]: non-AMP/OTHERS (Probabilities: 0.9446, 0.0513, 0.0041)\n","Processed WP_088291138.1 gallidermin/nisin family lantibiotic [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0032)\n","Processed WP_070195830.1 thiazolylpeptide-type bacteriocin [Streptomyces oceani]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0050, 0.0029)\n","Processed WP_100660920.1 lexapeptide family class V lantibiotic [Streptomyces lavendulae]: AMP (Probabilities: 0.1479, 0.8399, 0.0122)\n","Processed WP_211716011.1 mersacidin/lichenicidin family type 2 lantibiotic [Nocardiopsis sp. MG754419]: non-AMP/OTHERS (Probabilities: 0.9417, 0.0265, 0.0318)\n","Processed WP_211716020.1 mersacidin/lichenicidin family type 2 lantibiotic [Nocardiopsis sp. MG754419]: Neuropeptide (Probabilities: 0.2253, 0.0296, 0.7450)\n","Processed WP_212132806.1 mersacidin/lichenicidin family type 2 lantibiotic [Brevibacillus sp. NL20B1]: non-AMP/OTHERS (Probabilities: 0.9051, 0.0133, 0.0815)\n","Processed WP_093338803.1 plantaricin C family lantibiotic [Salibacterium halotolerans]: Neuropeptide (Probabilities: 0.2311, 0.0325, 0.7364)\n","Processed WP_212323102.1 plantaricin C family lantibiotic [Arachnia rubra]: Neuropeptide (Probabilities: 0.0081, 0.0093, 0.9827)\n","Processed WP_122978795.1 gallidermin/nisin family lantibiotic [Actinoplanes teichomyceticus]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0058, 0.0027)\n","Processed WP_169978858.1 mersacidin/lichenicidin family type 2 lantibiotic [Microbispora sp. H10836]: non-AMP/OTHERS (Probabilities: 0.7746, 0.0891, 0.1363)\n","Processed CAG6378832.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0049, 0.0028)\n","Processed CAG5884543.1 lantibiotic efflux protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9893, 0.0081, 0.0027)\n","Processed CAG5901085.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0077, 0.0035)\n","Processed WP_211334661.1 lichenicidin A2 family type 2 lantibiotic [Vagococcus acidifermentans]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0038, 0.0031)\n","Processed WP_209706947.1 FDLD family class I lanthipeptide [Crossiella equi]: non-AMP/OTHERS (Probabilities: 0.5857, 0.4059, 0.0085)\n","Processed WP_153494469.1 plantaricin C family lantibiotic [Lactococcus hircilactis]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0045, 0.0053)\n","Processed WP_141785537.1 mersacidin/lichenicidin family type 2 lantibiotic [Ornithinicoccus hortensis]: AMP (Probabilities: 0.1106, 0.8759, 0.0135)\n","Processed MBS3030689.1 mersacidin/lichenicidin family type 2 lantibiotic [Dolichospermum sp. DET66]: non-AMP/OTHERS (Probabilities: 0.7990, 0.0325, 0.1685)\n","Processed WP_158645521.1 mersacidin/lichenicidin family type 2 lantibiotic [Stackebrandtia albiflava]: Neuropeptide (Probabilities: 0.3124, 0.0960, 0.5916)\n","Processed WP_141785536.1 mersacidin/lichenicidin family type 2 lantibiotic [Ornithinicoccus hortensis]: Neuropeptide (Probabilities: 0.2037, 0.0798, 0.7164)\n","Processed WP_126823787.1 lichenicidin A2 family type 2 lantibiotic [Vagococcus entomophilus]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0028, 0.0042)\n","Processed CAG5949990.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0064, 0.0029)\n","Processed WP_126830118.1 gallidermin/nisin family lantibiotic [Vagococcus fessus]: AMP (Probabilities: 0.0736, 0.9137, 0.0126)\n","Processed WP_047858556.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium gephyra]: non-AMP/OTHERS (Probabilities: 0.8038, 0.0806, 0.1156)\n","Processed WP_047858559.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium gephyra]: non-AMP/OTHERS (Probabilities: 0.9760, 0.0071, 0.0169)\n","Processed WP_126830116.1 gallidermin/nisin family lantibiotic [Vagococcus fessus]: AMP (Probabilities: 0.1298, 0.8024, 0.0678)\n","Processed WP_121939392.1 mersacidin/lichenicidin family type 2 lantibiotic [Eilatimonas milleporae]: Neuropeptide (Probabilities: 0.2894, 0.0258, 0.6848)\n","Processed WP_121939389.1 mersacidin/lichenicidin family type 2 lantibiotic [Eilatimonas milleporae]: Neuropeptide (Probabilities: 0.0492, 0.0156, 0.9352)\n","Processed WP_109687491.1 lantibiotic paenibacillin [Tumebacillus permanentifrigoris]: non-AMP/OTHERS (Probabilities: 0.9884, 0.0093, 0.0023)\n","Processed WP_047860525.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium gephyra]: non-AMP/OTHERS (Probabilities: 0.5263, 0.0390, 0.4348)\n","Processed WP_047860824.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium gephyra]: Neuropeptide (Probabilities: 0.3566, 0.0605, 0.5829)\n","Processed CAG5275511.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0038, 0.0045)\n","Processed WP_211263393.1 plantaricin C family lantibiotic [Priestia koreensis]: non-AMP/OTHERS (Probabilities: 0.5675, 0.0351, 0.3974)\n","Processed WP_157807626.1 mersacidin/lichenicidin family type 2 lantibiotic [Hymenobacter chitinivorans]: Neuropeptide (Probabilities: 0.3468, 0.0493, 0.6039)\n","Processed WP_139124904.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus rivorum]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed WP_106186683.1 mersacidin/lichenicidin family type 2 lantibiotic [Umezawaea tangerina]: non-AMP/OTHERS (Probabilities: 0.6352, 0.3538, 0.0110)\n","Processed WP_157807625.1 mersacidin/lichenicidin family type 2 lantibiotic [Hymenobacter chitinivorans]: Neuropeptide (Probabilities: 0.3040, 0.0316, 0.6644)\n","Processed WP_084467438.1 gallidermin/nisin family lantibiotic [Actinokineospora inagensis]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0052, 0.0026)\n","Processed WP_081754897.1 gallidermin/nisin family lantibiotic [Paenibacillus durus]: AMP (Probabilities: 0.0903, 0.8975, 0.0122)\n","Processed WP_047856213.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium gephyra]: Neuropeptide (Probabilities: 0.4197, 0.1341, 0.4462)\n","Processed WP_094665038.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium tissieri]: AMP (Probabilities: 0.0689, 0.9176, 0.0135)\n","Processed WP_078971490.1 gallidermin/nisin family lantibiotic [Streptomyces chattanoogensis]: non-AMP/OTHERS (Probabilities: 0.8455, 0.0691, 0.0853)\n","Processed WP_030367889.1 cinnamycin family lantibiotic [Streptomyces roseoverticillatus]: non-AMP/OTHERS (Probabilities: 0.9753, 0.0159, 0.0088)\n","Processed WP_030366991.1 lexapeptide family class V lantibiotic [Actinomycetia]: non-AMP/OTHERS (Probabilities: 0.9732, 0.0060, 0.0208)\n","Processed WP_078577551.1 plantaricin C family lantibiotic [Salipaludibacillus agaradhaerens]: Neuropeptide (Probabilities: 0.1155, 0.0360, 0.8485)\n","Processed WP_076606333.1 mersacidin/lichenicidin family type 2 lantibiotic [Cystobacter fuscus]: non-AMP/OTHERS (Probabilities: 0.9782, 0.0071, 0.0147)\n","Processed WP_067391935.1 FxLD family lanthipeptide [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9867, 0.0076, 0.0056)\n","Processed WP_033317973.1 plantaricin C family lantibiotic [Streptomyces yerevanensis]: Neuropeptide (Probabilities: 0.0284, 0.0159, 0.9556)\n","Processed WP_031040624.1 plantaricin C family lantibiotic [Streptomyces olivaceus]: non-AMP/OTHERS (Probabilities: 0.9078, 0.0120, 0.0802)\n","Processed WP_030879033.1 thiazolylpeptide-type bacteriocin [Streptomyces varsoviensis]: non-AMP/OTHERS (Probabilities: 0.9160, 0.0132, 0.0708)\n","Processed WP_030548305.1 plantaricin C family lantibiotic [Streptomyces exfoliatus]: non-AMP/OTHERS (Probabilities: 0.6477, 0.0336, 0.3187)\n","Processed WP_026413721.1 cinnamycin family lantibiotic [Actinomadura oligospora]: non-AMP/OTHERS (Probabilities: 0.9378, 0.0354, 0.0269)\n","Processed WP_026701915.1 plantaricin C family lantibiotic [Bacillus aidingensis]: Neuropeptide (Probabilities: 0.2260, 0.0512, 0.7228)\n","Processed WP_018165752.1 plantaricin C family lantibiotic [Streptococcus henryi]: non-AMP/OTHERS (Probabilities: 0.9781, 0.0096, 0.0123)\n","Processed WP_014179560.1 plantaricin C family lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9047, 0.0797, 0.0156)\n","Processed WP_083903311.1 lantibiotic streptin [Streptococcus merionis]: AMP (Probabilities: 0.0672, 0.9186, 0.0142)\n","Processed WP_028558719.1 gallidermin/nisin family lantibiotic [Paenibacillus pinihumi]: AMP (Probabilities: 0.0672, 0.9178, 0.0150)\n","Processed WP_028474165.1 plantaricin C family lantibiotic [Nocardioides alkalitolerans]: non-AMP/OTHERS (Probabilities: 0.9880, 0.0040, 0.0080)\n","Processed WP_020557577.1 SCO0268 family class II lantipeptide [Streptomycetaceae]: Neuropeptide (Probabilities: 0.0464, 0.0202, 0.9334)\n","Processed WP_018353405.1 mersacidin/lichenicidin family type 2 lantibiotic [Longispora albida]: Neuropeptide (Probabilities: 0.0919, 0.0444, 0.8637)\n","Processed WP_005585691.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermicanus aegyptius]: non-AMP/OTHERS (Probabilities: 0.8290, 0.0168, 0.1543)\n","Processed MBR8827890.1 mersacidin/lichenicidin family type 2 lantibiotic [Gomphosphaeria aponina SAG 52.96 = DSM 107014]: non-AMP/OTHERS (Probabilities: 0.9485, 0.0167, 0.0348)\n","Processed WP_026411382.1 cinnamycin family lantibiotic [Actinomadura oligospora]: non-AMP/OTHERS (Probabilities: 0.9878, 0.0055, 0.0067)\n","Processed WP_019547644.1 gallidermin/nisin family lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.6221, 0.0429, 0.3349)\n","Processed WP_166639018.1 mersacidin/lichenicidin family type 2 lantibiotic [Maritalea mobilis]: non-AMP/OTHERS (Probabilities: 0.7144, 0.0406, 0.2450)\n","Processed WP_130873165.1 mersacidin/lichenicidin family type 2 lantibiotic [[Pseudopropionibacterium] massiliense]: non-AMP/OTHERS (Probabilities: 0.6317, 0.1779, 0.1904)\n","Processed MBR9274200.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9794, 0.0089, 0.0117)\n","Processed NMN46369.1 mersacidin/lichenicidin family type 2 lantibiotic [Chroococcidiopsis sp. PCC 6712]: non-AMP/OTHERS (Probabilities: 0.6884, 0.0439, 0.2677)\n","Processed MBR8834759.1 mersacidin/lichenicidin family type 2 lantibiotic [Stigonema ocellatum SAG 48.90 = DSM 106950]: non-AMP/OTHERS (Probabilities: 0.8050, 0.0375, 0.1575)\n","Processed HBC8579448.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0074, 0.0028)\n","Processed HAR7580019.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9845, 0.0051, 0.0104)\n","Processed MVL40671.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0063, 0.0028)\n","Processed MBR6404367.1 plantaricin C family lantibiotic [Eubacterium sp.]: AMP (Probabilities: 0.0667, 0.9195, 0.0138)\n","Processed MBR6215896.1 gallidermin family lantibiotic [Spirochaetaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0069, 0.0030)\n","Processed MBR4831112.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9894, 0.0035, 0.0071)\n","Processed MBR4831105.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9874, 0.0031, 0.0095)\n","Processed CZQ74125.1 lantibiotic ABC transporter protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0055, 0.0027)\n","Processed MBR4831109.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9865, 0.0106, 0.0029)\n","Processed MBR4831107.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9836, 0.0055, 0.0109)\n","Processed AWR25453.1 hypothetical protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9873, 0.0092, 0.0035)\n","Processed MBR4831110.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9847, 0.0044, 0.0109)\n","Processed MBR4669349.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9705, 0.0263, 0.0032)\n","Processed MBR4669356.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0033, 0.0058)\n","Processed MBR4669350.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9540, 0.0425, 0.0034)\n","Processed MBR4831106.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9527, 0.0076, 0.0396)\n","Processed MBR4357890.1 plantaricin C family lantibiotic [Butyrivibrio sp.]: Neuropeptide (Probabilities: 0.0092, 0.0100, 0.9808)\n","Processed MBR4669353.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9826, 0.0049, 0.0126)\n","Processed MBR4669351.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9656, 0.0306, 0.0038)\n","Processed MBR1692748.1 plantaricin C family lantibiotic [Lachnospiraceae bacterium]: AMP (Probabilities: 0.0952, 0.8918, 0.0130)\n","Processed NBC46284.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus exiguus]: Neuropeptide (Probabilities: 0.3074, 0.0431, 0.6495)\n","Processed MBR4669355.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0046, 0.0036)\n","Processed MBR4669348.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0031, 0.0050)\n","Processed MBR4831111.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: AMP (Probabilities: 0.0955, 0.8926, 0.0118)\n","Processed MBQ9809408.1 plantaricin C family lantibiotic [Ruminococcus sp.]: AMP (Probabilities: 0.0952, 0.8930, 0.0119)\n","Processed QTE69793.1 plantaricin C family lantibiotic [Clostridiales bacterium FE2011]: non-AMP/OTHERS (Probabilities: 0.9796, 0.0149, 0.0055)\n","Processed MBQ6995446.1 lichenicidin A2 family type 2 lantibiotic [Lachnospiraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9935, 0.0033, 0.0033)\n","Processed WP_147579180.1 gallidermin/nisin family lantibiotic [Corynebacterium propinquum]: non-AMP/OTHERS (Probabilities: 0.5261, 0.3828, 0.0912)\n","Processed MBQ6995447.1 lichenicidin A2 family type 2 lantibiotic [Lachnospiraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0030, 0.0042)\n","Processed MBR4669354.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9481, 0.0479, 0.0041)\n","Processed MBQ6995448.1 lichenicidin A2 family type 2 lantibiotic [Lachnospiraceae bacterium]: AMP (Probabilities: 0.0675, 0.9189, 0.0136)\n","Processed WP_200073465.1 gallidermin/nisin family lantibiotic [Saccharopolyspora sp. HNM0986]: Neuropeptide (Probabilities: 0.1096, 0.0285, 0.8619)\n","Processed MBQ6180269.1 plantaricin C family lantibiotic [Ruminococcus sp.]: AMP (Probabilities: 0.0747, 0.9126, 0.0126)\n","Processed WP_210771589.1 plantaricin C family lantibiotic [Streptomyces albidus (ex Kaewkla and Franco 2021)]: AMP (Probabilities: 0.1168, 0.8659, 0.0173)\n","Processed WP_210947903.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sp. MK37H]: Neuropeptide (Probabilities: 0.3321, 0.0673, 0.6005)\n","Processed WP_210888916.1 gallidermin/nisin family lantibiotic [Clostridium neonatale]: AMP (Probabilities: 0.0668, 0.9177, 0.0155)\n","Processed WP_168696173.1 mersacidin/lichenicidin family type 2 lantibiotic [Dolichospermum flos-aquae]: non-AMP/OTHERS (Probabilities: 0.9864, 0.0072, 0.0064)\n","Processed MBQ3067539.1 gallidermin family lantibiotic [Oscillospiraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9844, 0.0119, 0.0037)\n","Processed WP_082026128.1 gallidermin/nisin family lantibiotic [Caldibacillus thermoamylovorans]: AMP (Probabilities: 0.1641, 0.8240, 0.0119)\n","Processed WP_210729314.1 plantaricin C family lantibiotic [Actinomyces procaprae]: non-AMP/OTHERS (Probabilities: 0.9688, 0.0102, 0.0210)\n","Processed WP_136119572.1 lantibiotic streptin [Streptococcus pyogenes]: AMP (Probabilities: 0.0674, 0.9179, 0.0147)\n","Processed WP_136096304.1 lantibiotic streptin [Streptococcus pyogenes]: AMP (Probabilities: 0.0681, 0.9170, 0.0149)\n","Processed WP_161921747.1 gallidermin/nisin family lantibiotic [Caldibacillus pasinlerensis]: AMP (Probabilities: 0.1645, 0.8158, 0.0197)\n","Processed WP_061079715.1 mersacidin/lichenicidin family type 2 lantibiotic [Nocardiopsis alborubida]: Neuropeptide (Probabilities: 0.0134, 0.0105, 0.9760)\n","Processed WP_164180064.1 FxLD family lanthipeptide [Streptomyces sp. 604F]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0065, 0.0049)\n","Processed WP_111704207.1 lantibiotic streptin [Streptococcus pyogenes]: AMP (Probabilities: 0.0674, 0.9173, 0.0153)\n","Processed WP_082768270.1 plantaricin C family lantibiotic [Nocardiopsis alborubida]: AMP (Probabilities: 0.0760, 0.9108, 0.0131)\n","Processed WP_168520947.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis acididurans]: non-AMP/OTHERS (Probabilities: 0.6128, 0.2845, 0.1028)\n","Processed MBP8947391.1 mersacidin/lichenicidin family type 2 lantibiotic [Candidatus Promineofilum sp.]: non-AMP/OTHERS (Probabilities: 0.9785, 0.0069, 0.0146)\n","Processed WP_136037490.1 lantibiotic streptin [Streptococcus pyogenes]: AMP (Probabilities: 0.0676, 0.9177, 0.0147)\n","Processed MBP8292349.1 mersacidin/lichenicidin family type 2 lantibiotic [Caldilineaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0035, 0.0056)\n","Processed WP_023529351.1 thiazolylpeptide-type bacteriocin [Streptomyces sp. RKND-216]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0040, 0.0034)\n","Processed MBP8292347.1 mersacidin/lichenicidin family type 2 lantibiotic [Caldilineaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0044, 0.0067)\n","Processed MBP8291625.1 mersacidin/lichenicidin family type 2 lantibiotic [Caldilineaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9008, 0.0334, 0.0658)\n","Processed MBP5222816.1 gallidermin/nisin family lantibiotic [Lachnospiraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.5301, 0.0569, 0.4130)\n","Processed MBP8258532.1 cinnamycin family lantibiotic [Verrucomicrobia bacterium]: Neuropeptide (Probabilities: 0.1906, 0.0430, 0.7663)\n","Processed MBP8291627.1 mersacidin/lichenicidin family type 2 lantibiotic [Caldilineaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9405, 0.0212, 0.0383)\n","Processed WP_145434459.1 mersacidin/lichenicidin family type 2 lantibiotic [Lacipirellula limnantheis]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0045, 0.0058)\n","Processed WP_082768272.1 mersacidin/lichenicidin family type 2 lantibiotic [Nocardiopsis alborubida]: non-AMP/OTHERS (Probabilities: 0.6089, 0.0232, 0.3679)\n","Processed WP_165811820.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9686, 0.0221, 0.0094)\n","Processed WP_081584166.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostocales]: non-AMP/OTHERS (Probabilities: 0.9520, 0.0125, 0.0354)\n","Processed WP_210340965.1 lichenicidin A2 family type 2 lantibiotic, partial [Alkalihalobacillus gibsonii]: non-AMP/OTHERS (Probabilities: 0.4850, 0.0490, 0.4659)\n","Processed WP_203091010.1 lichenicidin A2 family type 2 lantibiotic [Alkalihalobacillus gibsonii]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0042, 0.0036)\n","Processed WP_203090954.1 lichenicidin A2 family type 2 lantibiotic [Alkalihalobacillus gibsonii]: non-AMP/OTHERS (Probabilities: 0.9868, 0.0098, 0.0034)\n","Processed WP_203091017.1 lichenicidin A2 family type 2 lantibiotic [Alkalihalobacillus gibsonii]: non-AMP/OTHERS (Probabilities: 0.9885, 0.0082, 0.0032)\n","Processed WP_154755982.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis pithecellobii]: non-AMP/OTHERS (Probabilities: 0.8224, 0.0185, 0.1591)\n","Processed WP_154536788.1 plantaricin C family lantibiotic [Bifidobacterium sp. WCA-178-WT-4B]: Neuropeptide (Probabilities: 0.0092, 0.0096, 0.9812)\n","Processed WP_207649263.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio fibrisolvens]: AMP (Probabilities: 0.1167, 0.8717, 0.0116)\n","Processed MBM0067989.1 lichenicidin A2 family type 2 lantibiotic [Alkalihalobacillus gibsonii]: non-AMP/OTHERS (Probabilities: 0.9762, 0.0081, 0.0156)\n","Processed MBP3489173.1 gallidermin/nisin family lantibiotic [Roseburia sp.]: AMP (Probabilities: 0.0669, 0.9189, 0.0142)\n","Processed WP_139263596.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio fibrisolvens]: non-AMP/OTHERS (Probabilities: 0.9881, 0.0067, 0.0052)\n","Processed WP_189972779.1 plantaricin C family lantibiotic [Streptomyces avidinii]: AMP (Probabilities: 0.0704, 0.9167, 0.0129)\n","Processed WP_135337564.1 class II lantibiotic LanA [Streptomyces palmae]: non-AMP/OTHERS (Probabilities: 0.8997, 0.0330, 0.0673)\n","Processed WP_021281420.1 plantaricin C family lantibiotic [Clostridium]: AMP (Probabilities: 0.0952, 0.8881, 0.0167)\n","Processed WP_030757923.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0076, 0.0029)\n","Processed WP_135338982.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces palmae]: non-AMP/OTHERS (Probabilities: 0.9830, 0.0104, 0.0066)\n","Processed WP_134912015.1 gallidermin/nisin family lantibiotic [Paenibacillus]: AMP (Probabilities: 0.0688, 0.9171, 0.0141)\n","Processed WP_031138450.1 plantaricin C family lantibiotic [Streptomyces xanthophaeus]: Neuropeptide (Probabilities: 0.2622, 0.0438, 0.6940)\n","Processed WP_031172570.1 lexapeptide family class V lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0046, 0.0037)\n","Processed WP_135338980.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces palmae]: Neuropeptide (Probabilities: 0.0700, 0.0261, 0.9039)\n","Processed MBP2072853.1 lantibiotic modifying enzyme [Thermoanaerobacterium butyriciformans]: non-AMP/OTHERS (Probabilities: 0.9670, 0.0182, 0.0148)\n","Processed WP_030319753.1 plantaricin C family lantibiotic [Streptomyces flavochromogenes]: Neuropeptide (Probabilities: 0.0507, 0.0197, 0.9297)\n","Processed EHB47113.1 hypothetical protein [Paenibacillus lactis 154 154]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0046, 0.0033)\n","Processed MBP0473082.1 lantibiotic ABC transporter ATP-binding protein [Pseudomonas aeruginosa]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed WP_135100564.1 lantibiotic dehydratase [Peptoniphilus vaginalis]: non-AMP/OTHERS (Probabilities: 0.9118, 0.0830, 0.0052)\n","Processed WP_004233864.1 hypothetical protein [Lysinibacillus]: AMP (Probabilities: 0.0662, 0.9197, 0.0142)\n","Processed WP_209264061.1 class II lantibiotic LanA [Streptomyces sp. DW4-2]: non-AMP/OTHERS (Probabilities: 0.7843, 0.0172, 0.1985)\n","Processed WP_078983529.1 gallidermin/nisin family lantibiotic [Streptomyces lydicus]: non-AMP/OTHERS (Probabilities: 0.9119, 0.0671, 0.0210)\n","Processed HBA0438792.1 lantibiotic ABC transporter ATP-binding protein [Enterococcus faecium]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed MBP0474069.1 lantibiotic dehydratase [Pseudomonas aeruginosa]: non-AMP/OTHERS (Probabilities: 0.9852, 0.0122, 0.0026)\n","Processed WP_172763328.1 plantaricin C family lantibiotic [Dolosigranulum pigrum]: Neuropeptide (Probabilities: 0.0313, 0.0171, 0.9516)\n","Processed WP_004233859.1 plantaricin C family lantibiotic [Lysinibacillus]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0065, 0.0034)\n","Processed MBO8902365.1 lantibiotic dehydratase [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0040, 0.0029)\n","Processed HBA0438793.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Enterococcus faecium]: non-AMP/OTHERS (Probabilities: 0.9875, 0.0093, 0.0031)\n","Processed CDR14362.1 lantibiotic transport ATP-binding protein [Streptomyces iranensis]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0049, 0.0042)\n","Processed WP_208957739.1 plantaricin C family lantibiotic [Dolosigranulum pigrum]: Neuropeptide (Probabilities: 0.0122, 0.0111, 0.9767)\n","Processed WP_135338981.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces palmae]: non-AMP/OTHERS (Probabilities: 0.8749, 0.1195, 0.0056)\n","Processed WP_208954534.1 plantaricin C family lantibiotic [Dolosigranulum pigrum]: non-AMP/OTHERS (Probabilities: 0.4741, 0.1319, 0.3940)\n","Processed WP_208877364.1 lantibiotic dehydratase [Streptomyces armeniacus]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0040, 0.0069)\n","Processed WP_208924131.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0035, 0.0066)\n","Processed WP_119245821.1 gallidermin/nisin family lantibiotic [Lachnospiraceae]: AMP (Probabilities: 0.1321, 0.8408, 0.0271)\n","Processed WP_016249977.1 gallidermin/nisin family lantibiotic [Enterococcus]: non-AMP/OTHERS (Probabilities: 0.5137, 0.4760, 0.0103)\n","Processed MBO8500548.1 lantibiotic dehydratase [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0043, 0.0035)\n","Processed WP_033799232.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0058, 0.0028)\n","Processed WP_076442089.1 cinnamycin family lantibiotic [Microbispora rosea]: non-AMP/OTHERS (Probabilities: 0.9865, 0.0043, 0.0092)\n","Processed WP_010334371.1 plantaricin C family lantibiotic [Bacillus mojavensis]: non-AMP/OTHERS (Probabilities: 0.8848, 0.0294, 0.0859)\n","Processed WP_003991782.1 plantaricin C family lantibiotic [Streptomyces viridochromogenes]: Neuropeptide (Probabilities: 0.0120, 0.0111, 0.9769)\n","Processed WP_208773770.1 plantaricin C family lantibiotic [Enterococcus faecalis]: Neuropeptide (Probabilities: 0.1815, 0.0348, 0.7837)\n","Processed WP_167721410.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostocales]: non-AMP/OTHERS (Probabilities: 0.9528, 0.0134, 0.0338)\n","Processed WP_208721280.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus aberystwythensis]: Neuropeptide (Probabilities: 0.2396, 0.0503, 0.7101)\n","Processed WP_208647379.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus interemptor]: Neuropeptide (Probabilities: 0.2797, 0.0273, 0.6930)\n","Processed WP_053802155.1 hypothetical protein [Streptomyces]: Neuropeptide (Probabilities: 0.0726, 0.0222, 0.9052)\n","Processed WP_142612812.1 gallidermin/nisin family lantibiotic [Paenibacillus ottowii]: AMP (Probabilities: 0.0662, 0.9198, 0.0140)\n","Processed WP_199832729.1 lantibiotic protein [Streptomyces puniciscabiei]: Neuropeptide (Probabilities: 0.0100, 0.0097, 0.9803)\n","Processed WP_208641392.1 cinnamycin family lantibiotic [Streptomyces africanus]: non-AMP/OTHERS (Probabilities: 0.9022, 0.0559, 0.0419)\n","Processed WP_142612562.1 FDLD family class I lanthipeptide [Paenibacillus]: AMP (Probabilities: 0.0839, 0.9041, 0.0119)\n","Processed WP_120646679.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus llansteffanensis]: Neuropeptide (Probabilities: 0.4527, 0.0416, 0.5057)\n","Processed WP_120529931.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus exercitus]: Neuropeptide (Probabilities: 0.3603, 0.0210, 0.6186)\n","Processed MBO8576196.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0033, 0.0033)\n","Processed WP_053802156.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9823, 0.0104, 0.0073)\n","Processed WP_163581377.1 plantaricin C family lantibiotic [Gracilibacillus]: non-AMP/OTHERS (Probabilities: 0.8191, 0.0319, 0.1490)\n","Processed WP_120556222.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus aberystwythensis]: Neuropeptide (Probabilities: 0.3897, 0.1837, 0.4267)\n","Processed WP_084747561.1 FxLD family lanthipeptide [Kitasatospora albolonga]: non-AMP/OTHERS (Probabilities: 0.8831, 0.0131, 0.1038)\n","Processed WP_090951424.1 gallidermin/nisin family lantibiotic [Parageobacillus thermantarcticus]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0031, 0.0038)\n","Processed WP_082825202.1 gallidermin/nisin family lantibiotic [Paenibacillus]: non-AMP/OTHERS (Probabilities: 0.9616, 0.0255, 0.0129)\n","Processed MBO5522803.1 lantibiotic immunity ABC transporter MutG family permease subunit [Roseburia sp.]: non-AMP/OTHERS (Probabilities: 0.9880, 0.0092, 0.0028)\n","Processed MBO4456624.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9624, 0.0346, 0.0029)\n","Processed MBO4210901.1 lantibiotic dehydratase [Micromonospora echinofusca]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0035, 0.0057)\n","Processed WP_086603600.1 gallidermin/nisin family lantibiotic [Streptomyces swartbergensis]: non-AMP/OTHERS (Probabilities: 0.5244, 0.0507, 0.4249)\n","Processed WP_016249979.1 gallidermin/nisin family lantibiotic [Enterococcus moraviensis]: AMP (Probabilities: 0.0780, 0.9093, 0.0127)\n","Processed MBO4456622.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9825, 0.0079, 0.0096)\n","Processed MBO4207982.1 lantibiotic dehydratase [Micromonospora echinofusca]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0050, 0.0031)\n","Processed MBO4456623.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0035, 0.0038)\n","Processed MBO4456625.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp.]: non-AMP/OTHERS (Probabilities: 0.6007, 0.3808, 0.0185)\n","Processed WP_208344629.1 mersacidin/lichenicidin family type 2 lantibiotic [Aetokthonos hydrillicola]: non-AMP/OTHERS (Probabilities: 0.7674, 0.0466, 0.1860)\n","Processed  unnamed protein product [Unknown]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0053, 0.0046)\n","Processed WP_118404113.1 gallidermin/nisin family lantibiotic [Lachnospiraceae]: AMP (Probabilities: 0.0666, 0.9187, 0.0147)\n","Processed WP_208164693.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0039, 0.0043)\n","Processed  unnamed protein product [Unknown]: non-AMP/OTHERS (Probabilities: 0.7669, 0.0171, 0.2159)\n","Processed WP_208191417.1 plantaricin C family lantibiotic [Lactiplantibacillus plantarum]: AMP (Probabilities: 0.0699, 0.9167, 0.0133)\n","Processed WP_133519699.1 gallidermin/nisin family lantibiotic [Curtobacterium flaccumfaciens]: non-AMP/OTHERS (Probabilities: 0.9682, 0.0291, 0.0027)\n","Processed MBO2469092.1 plantaricin C family lantibiotic [Firmicutes bacterium]: AMP (Probabilities: 0.0827, 0.9004, 0.0169)\n","Processed WP_131847813.1 plantaricin C family lantibiotic [Baia soyae]: non-AMP/OTHERS (Probabilities: 0.6999, 0.1549, 0.1452)\n","Processed WP_131893275.1 cinnamycin family lantibiotic [Actinomadura rubrisoli]: non-AMP/OTHERS (Probabilities: 0.8971, 0.0948, 0.0081)\n","Processed NDU74188.1 cinnamycin family lantibiotic [Actinomadura lepetitiana]: non-AMP/OTHERS (Probabilities: 0.8788, 0.0307, 0.0904)\n","Processed WP_086768465.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc sp. 106C]: non-AMP/OTHERS (Probabilities: 0.9348, 0.0189, 0.0463)\n","Processed WP_086837517.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc sp. RF31YmG]: non-AMP/OTHERS (Probabilities: 0.9762, 0.0121, 0.0116)\n","Processed WP_200274177.1 plantaricin C family lantibiotic [Clostridium sp. YIM B02505]: Neuropeptide (Probabilities: 0.0140, 0.0122, 0.9738)\n","Processed WP_161275792.1 plantaricin C family lantibiotic [Blautia wexlerae]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0039, 0.0041)\n","Processed WP_101514856.1 gallidermin/nisin family lantibiotic [Clostridium botulinum]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0047, 0.0030)\n","Processed WP_024932093.1 plantaricin C family lantibiotic [Clostridium botulinum]: AMP (Probabilities: 0.0662, 0.9192, 0.0146)\n","Processed WP_207663704.1 gallidermin/nisin family lantibiotic [[Ruminococcus] gnavus]: Neuropeptide (Probabilities: 0.0272, 0.0137, 0.9592)\n","Processed WP_191531521.1 gallidermin/nisin family lantibiotic [Eubacterium sp. An11]: non-AMP/OTHERS (Probabilities: 0.9806, 0.0142, 0.0052)\n","Processed WP_186835009.1 gallidermin/nisin family lantibiotic [Eubacteriales]: AMP (Probabilities: 0.0801, 0.9075, 0.0124)\n","Processed WP_010248475.1 lantibiotic dehydratase [Peptoniphilus rhinitidis]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0052, 0.0029)\n","Processed WP_195659585.1 lichenicidin A2 family type 2 lantibiotic [[Ruminococcus] gnavus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0039, 0.0040)\n","Processed WP_195345702.1 lichenicidin A2 family type 2 lantibiotic [Paeniclostridium sordellii]: AMP (Probabilities: 0.1008, 0.8869, 0.0123)\n","Processed WP_037606966.1 plantaricin C family lantibiotic [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.5183, 0.1473, 0.3344)\n","Processed WP_114002469.1 plantaricin C family lantibiotic [Eubacteriales]: non-AMP/OTHERS (Probabilities: 0.9880, 0.0062, 0.0057)\n","Processed WP_195345703.1 lichenicidin A2 family type 2 lantibiotic [Paeniclostridium sordellii]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0076, 0.0029)\n","Processed WP_205606131.1 plantaricin C family lantibiotic [Clostridium sporogenes]: Neuropeptide (Probabilities: 0.2364, 0.2179, 0.5457)\n","Processed WP_170819691.1 gallidermin/nisin family lantibiotic [Hungatella xylanolytica]: Neuropeptide (Probabilities: 0.0130, 0.0123, 0.9747)\n","Processed WP_164952171.1 plantaricin C family lantibiotic [Clostridium sp. FP2]: Neuropeptide (Probabilities: 0.0395, 0.0226, 0.9380)\n","Processed WP_164952019.1 plantaricin C family lantibiotic, partial [Clostridium sp. FP2]: non-AMP/OTHERS (Probabilities: 0.9664, 0.0190, 0.0146)\n","Processed WP_164952022.1 plantaricin C family lantibiotic [Clostridium sp. FP2]: Neuropeptide (Probabilities: 0.0077, 0.0091, 0.9832)\n","Processed WP_164952020.1 plantaricin C family lantibiotic [Clostridium sp. FP2]: Neuropeptide (Probabilities: 0.0080, 0.0092, 0.9828)\n","Processed WP_185658324.1 plantaricin C family lantibiotic, partial [Clostridium saccharobutylicum]: AMP (Probabilities: 0.1041, 0.8834, 0.0125)\n","Processed WP_164952021.1 plantaricin C family lantibiotic [Clostridium sp. FP2]: Neuropeptide (Probabilities: 0.0076, 0.0091, 0.9834)\n","Processed WP_164473178.1 mersacidin/lichenicidin family type 2 lantibiotic [Clostridium sp. E02]: Neuropeptide (Probabilities: 0.0142, 0.0121, 0.9737)\n","Processed WP_158588917.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. XB500-5]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0057, 0.0036)\n","Processed WP_164791580.1 plantaricin C family lantibiotic [Clostridium perfringens]: AMP (Probabilities: 0.0661, 0.9191, 0.0148)\n","Processed WP_154516683.1 gallidermin/nisin family lantibiotic [Velocimicrobium porci]: AMP (Probabilities: 0.0662, 0.9193, 0.0145)\n","Processed WP_142415049.1 plantaricin C family lantibiotic [Hathewaya massiliensis]: AMP (Probabilities: 0.0686, 0.9179, 0.0134)\n","Processed WP_142415051.1 plantaricin C family lantibiotic [Hathewaya massiliensis]: AMP (Probabilities: 0.0695, 0.9169, 0.0136)\n","Processed WP_134217944.1 NisI/SpaI family lantibiotic immunity lipoprotein [Pelotomaculum sp. FP]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0034, 0.0043)\n","Processed WP_158588742.1 lichenicidin A2 family type 2 lantibiotic [Butyrivibrio sp. X503]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0050, 0.0035)\n","Processed WP_154516686.1 gallidermin/nisin family lantibiotic [Velocimicrobium porci]: AMP (Probabilities: 0.0657, 0.9200, 0.0143)\n","Processed WP_121057169.1 NisI/SpaI family lantibiotic immunity lipoprotein [Eubacteriales]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0044, 0.0034)\n","Processed WP_118445437.1 gallidermin/nisin family lantibiotic [[Ruminococcus] gnavus]: Neuropeptide (Probabilities: 0.0129, 0.0115, 0.9756)\n","Processed WP_125152329.1 gallidermin/nisin family lantibiotic [Clostridium rectalis]: AMP (Probabilities: 0.0701, 0.9153, 0.0146)\n","Processed WP_124067896.1 gallidermin/nisin family lantibiotic [Clostridium sp. E02]: AMP (Probabilities: 0.0665, 0.9179, 0.0156)\n","Processed WP_117628691.1 plantaricin C family lantibiotic [Eubacteriales]: Neuropeptide (Probabilities: 0.2699, 0.0486, 0.6815)\n","Processed WP_117357562.1 cinnamycin family lantibiotic [Actinomadura logoneensis]: non-AMP/OTHERS (Probabilities: 0.9486, 0.0187, 0.0327)\n","Processed WP_118445445.1 gallidermin/nisin family lantibiotic [[Ruminococcus] gnavus]: Neuropeptide (Probabilities: 0.0136, 0.0119, 0.9745)\n","Processed WP_117640233.1 gallidermin/nisin family lantibiotic [Lachnospiraceae]: AMP (Probabilities: 0.0666, 0.9188, 0.0146)\n","Processed WP_118445436.1 gallidermin/nisin family lantibiotic [[Ruminococcus] gnavus]: Neuropeptide (Probabilities: 0.0111, 0.0106, 0.9783)\n","Processed WP_070040678.1 lichenicidin A2 family type 2 lantibiotic [Robinsoniella peoriensis]: non-AMP/OTHERS (Probabilities: 0.9745, 0.0213, 0.0042)\n","Processed WP_044289661.1 lichenicidin A2 family type 2 lantibiotic [Robinsoniella]: non-AMP/OTHERS (Probabilities: 0.8829, 0.1114, 0.0057)\n","Processed WP_207658417.1 gallidermin/nisin family lantibiotic, partial [Blautia obeum]: AMP (Probabilities: 0.0715, 0.9148, 0.0137)\n","Processed WP_207652996.1 NisI/SpaI family lantibiotic immunity lipoprotein [Acutalibacter muris]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0039, 0.0038)\n","Processed WP_117416424.1 gallidermin/nisin family lantibiotic [Clostridium indicum]: Neuropeptide (Probabilities: 0.1824, 0.0595, 0.7581)\n","Processed WP_172556369.1 class II lanthipeptide, LchA2/BrtA2 family [Clostridium putrefaciens]: non-AMP/OTHERS (Probabilities: 0.9880, 0.0045, 0.0075)\n","Processed WP_115480874.1 plantaricin C family lantibiotic [Anaerosacchariphilus polymeriproducens]: non-AMP/OTHERS (Probabilities: 0.9845, 0.0099, 0.0056)\n","Processed WP_115480872.1 plantaricin C family lantibiotic [Anaerosacchariphilus polymeriproducens]: AMP (Probabilities: 0.0758, 0.9118, 0.0124)\n","Processed WP_115480873.1 plantaricin C family lantibiotic [Anaerosacchariphilus polymeriproducens]: AMP (Probabilities: 0.0894, 0.8986, 0.0120)\n","Processed WP_170065598.1 class II lanthipeptide, LchA2/BrtA2 family [Clostridium vincentii]: AMP (Probabilities: 0.0656, 0.9197, 0.0147)\n","Processed WP_115639776.1 hypothetical protein [Clostridium perfringens]: non-AMP/OTHERS (Probabilities: 0.9871, 0.0102, 0.0026)\n","Processed WP_115480871.1 plantaricin C family lantibiotic [Anaerosacchariphilus polymeriproducens]: AMP (Probabilities: 0.1129, 0.8732, 0.0139)\n","Processed WP_181573723.1 hypothetical protein [Clostridium tetanomorphum]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0041, 0.0031)\n","Processed WP_155268484.1 gallidermin/nisin family lantibiotic [Clostridium beijerinckii]: AMP (Probabilities: 0.1406, 0.8448, 0.0146)\n","Processed WP_114296517.1 plantaricin C family lantibiotic [Anaerobacterium chartisolvens]: non-AMP/OTHERS (Probabilities: 0.6499, 0.0241, 0.3260)\n","Processed WP_104436664.1 gallidermin/nisin family lantibiotic [Hungatella xylanolytica]: Neuropeptide (Probabilities: 0.0123, 0.0118, 0.9760)\n","Processed WP_110290825.1 lichenicidin A2 family type 2 lantibiotic [Lachnotalea glycerini]: non-AMP/OTHERS (Probabilities: 0.9854, 0.0117, 0.0029)\n","Processed WP_080697001.1 gallidermin/nisin family lantibiotic [Clostridium beijerinckii]: AMP (Probabilities: 0.0705, 0.9149, 0.0146)\n","Processed WP_170138021.1 plantaricin C family lantibiotic [Anaerobacterium chartisolvens]: non-AMP/OTHERS (Probabilities: 0.6972, 0.2859, 0.0169)\n","Processed WP_077895312.1 gallidermin/nisin family lantibiotic [Clostridium felsineum]: AMP (Probabilities: 0.0692, 0.9177, 0.0131)\n","Processed WP_074350647.1 lichenicidin A2 family type 2 lantibiotic [Proteiniborus sp. DW1]: non-AMP/OTHERS (Probabilities: 0.9574, 0.0392, 0.0035)\n","Processed WP_176091561.1 gallidermin/nisin family lantibiotic [Clostridium]: AMP (Probabilities: 0.2651, 0.5957, 0.1392)\n","Processed WP_090418014.1 plantaricin C family lantibiotic [Pseudobutyrivibrio sp. YE44]: AMP (Probabilities: 0.0716, 0.9154, 0.0130)\n","Processed WP_061993722.1 hypothetical protein [Fructobacillus ficulneus]: non-AMP/OTHERS (Probabilities: 0.9841, 0.0127, 0.0032)\n","Processed WP_040470420.1 lichenicidin A2 family type 2 lantibiotic [Eubacterium plexicaudatum]: Neuropeptide (Probabilities: 0.1503, 0.0239, 0.8257)\n","Processed WP_036937457.1 plantaricin C family lantibiotic [Pseudobacteroides cellulosolvens]: AMP (Probabilities: 0.0999, 0.8875, 0.0127)\n","Processed WP_074350646.1 lichenicidin A2 family type 2 lantibiotic [Proteiniborus sp. DW1]: non-AMP/OTHERS (Probabilities: 0.9875, 0.0043, 0.0082)\n","Processed WP_036937455.1 plantaricin C family lantibiotic [Pseudobacteroides cellulosolvens]: AMP (Probabilities: 0.0774, 0.9101, 0.0125)\n","Processed WP_028087812.1 plantaricin C family lantibiotic [Dorea longicatena]: AMP (Probabilities: 0.1861, 0.7795, 0.0343)\n","Processed WP_035773641.1 plantaricin C family lantibiotic [Butyrivibrio sp. VCD2006]: non-AMP/OTHERS (Probabilities: 0.9811, 0.0066, 0.0123)\n","Processed WP_026887542.1 gallidermin/nisin family lantibiotic [Clostridium beijerinckii]: AMP (Probabilities: 0.0819, 0.9050, 0.0131)\n","Processed WP_027422484.1 plantaricin C family lantibiotic [Lachnobacterium bovis]: AMP (Probabilities: 0.0677, 0.9180, 0.0143)\n","Processed WP_080063510.1 plantaricin C family lantibiotic [Ruminiclostridium hungatei]: AMP (Probabilities: 0.0782, 0.9094, 0.0124)\n","Processed WP_014519821.1 class II lanthipeptide, LchA2/BrtA2 family [Clostridium botulinum]: non-AMP/OTHERS (Probabilities: 0.6855, 0.3078, 0.0068)\n","Processed WP_207559138.1 plantaricin C family lantibiotic [Ruminiclostridium hungatei]: AMP (Probabilities: 0.0796, 0.9085, 0.0120)\n","Processed QSZ28270.1 gallidermin/nisin family lantibiotic [thermophilic bacterium 3443-3Ac]: Neuropeptide (Probabilities: 0.1981, 0.0339, 0.7680)\n","Processed WP_131758918.1 cinnamycin family lantibiotic [Actinomadura fibrosa]: non-AMP/OTHERS (Probabilities: 0.9554, 0.0371, 0.0076)\n","Processed WP_131737520.1 cinnamycin family lantibiotic [Actinomadura roseirufa]: non-AMP/OTHERS (Probabilities: 0.9850, 0.0042, 0.0108)\n","Processed WP_027296044.1 lichenicidin A2 family type 2 lantibiotic [Robinsoniella sp. KNHs210]: non-AMP/OTHERS (Probabilities: 0.9550, 0.0402, 0.0047)\n","Processed WP_131220981.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium longum]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0044, 0.0036)\n","Processed WP_131245449.1 gallidermin/nisin family lantibiotic [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0034)\n","Processed WP_165498507.1 hypothetical protein [Streptomonospora sp. M2]: non-AMP/OTHERS (Probabilities: 0.9286, 0.0131, 0.0583)\n","Processed WP_083525003.1 gallidermin/nisin family lantibiotic [Thermotalea metallivorans]: Neuropeptide (Probabilities: 0.0232, 0.0146, 0.9622)\n","Processed WP_068555840.1 gallidermin/nisin family lantibiotic [Thermotalea metallivorans]: non-AMP/OTHERS (Probabilities: 0.9494, 0.0342, 0.0164)\n","Processed MBO1051935.1 mersacidin/lichenicidin family type 2 lantibiotic [Dolichospermum sp. DET73]: AMP (Probabilities: 0.2404, 0.4281, 0.3314)\n","Processed MBO0789449.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0066, 0.0036)\n","Processed MBO0782834.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9811, 0.0056, 0.0133)\n","Processed MBO0782750.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.7555, 0.0376, 0.2069)\n","Processed MBO0782832.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9873, 0.0075, 0.0052)\n","Processed MBO0782833.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0044, 0.0043)\n","Processed MBO0782749.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.5116, 0.0368, 0.4516)\n","Processed MBO0782751.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9787, 0.0052, 0.0161)\n","Processed MBO0801152.1 mersacidin/lichenicidin family type 2 lantibiotic [Blastocatellia bacterium]: Neuropeptide (Probabilities: 0.0254, 0.0160, 0.9587)\n","Processed MBO0912918.1 lantibiotic biosynthesis protein [Streptomyces laculatispora]: non-AMP/OTHERS (Probabilities: 0.9893, 0.0069, 0.0038)\n","Processed WP_207303370.1 lichenicidin A2 family type 2 lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.1720, 0.8147, 0.0133)\n","Processed WP_201050322.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus sp. TH22]: non-AMP/OTHERS (Probabilities: 0.9804, 0.0141, 0.0055)\n","Processed WP_030029256.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9896, 0.0063, 0.0041)\n","Processed WP_016571591.1 FxLD family lanthipeptide [Streptomyces albulus]: non-AMP/OTHERS (Probabilities: 0.9880, 0.0073, 0.0047)\n","Processed WP_011316529.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostocaceae]: non-AMP/OTHERS (Probabilities: 0.9430, 0.0448, 0.0122)\n","Processed WP_201032101.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Bacillus (in: Bacteria)]: non-AMP/OTHERS (Probabilities: 0.9852, 0.0101, 0.0047)\n","Processed WP_182252087.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.8863, 0.0145, 0.0992)\n","Processed WP_130510116.1 plantaricin C family lantibiotic [Krasilnikovia cinnamomea]: Neuropeptide (Probabilities: 0.1784, 0.1028, 0.7188)\n","Processed WP_130133114.1 gallidermin/nisin family lantibiotic, partial [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.1713, 0.0595, 0.7692)\n","Processed WP_129891988.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonosporobacter rubrisoli]: non-AMP/OTHERS (Probabilities: 0.9478, 0.0159, 0.0363)\n","Processed WP_129891986.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonosporobacter rubrisoli]: non-AMP/OTHERS (Probabilities: 0.8712, 0.0231, 0.1057)\n","Processed WP_201032102.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Bacillus (in: Bacteria)]: AMP (Probabilities: 0.2281, 0.7587, 0.0132)\n","Processed WP_129891987.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonosporobacter rubrisoli]: non-AMP/OTHERS (Probabilities: 0.8270, 0.0239, 0.1491)\n","Processed WP_130114895.1 gallidermin/nisin family lantibiotic, partial [Staphylococcus aureus]: AMP (Probabilities: 0.4216, 0.4548, 0.1236)\n","Processed WP_129891990.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonosporobacter rubrisoli]: non-AMP/OTHERS (Probabilities: 0.9614, 0.0094, 0.0292)\n","Processed WP_129891989.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonosporobacter rubrisoli]: non-AMP/OTHERS (Probabilities: 0.5398, 0.0465, 0.4137)\n","Processed WP_113715810.1 type 2 lantibiotic [Priestia endophytica]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0064, 0.0029)\n","Processed WP_073926695.1 thiazolylpeptide-type bacteriocin [Streptomycetaceae]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0037, 0.0039)\n","Processed WP_129888396.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonosporobacter rubrisoli]: Neuropeptide (Probabilities: 0.0874, 0.0235, 0.8891)\n","Processed WP_206848851.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus plantarum]: AMP (Probabilities: 0.0905, 0.8976, 0.0120)\n","Processed WP_206836840.1 lichenicidin A2 family type 2 lantibiotic, partial [Enterococcus plantarum]: non-AMP/OTHERS (Probabilities: 0.9186, 0.0135, 0.0680)\n","Processed WP_206790744.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. NCSPR001]: Neuropeptide (Probabilities: 0.1019, 0.0273, 0.8709)\n","Processed WP_206799957.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis sp. 195334CR]: AMP (Probabilities: 0.0939, 0.8940, 0.0121)\n","Processed WP_206837909.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus plantarum]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0030, 0.0052)\n","Processed WP_103574525.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9838, 0.0073, 0.0090)\n","Processed WP_206799944.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis sp. 195334CR]: non-AMP/OTHERS (Probabilities: 0.9859, 0.0054, 0.0088)\n","Processed WP_206799946.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis sp. 195334CR]: non-AMP/OTHERS (Probabilities: 0.8784, 0.0371, 0.0845)\n","Processed WP_206836700.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus plantarum]: non-AMP/OTHERS (Probabilities: 0.9883, 0.0042, 0.0075)\n","Processed WP_111922658.1 plantaricin C family lantibiotic [Priestia endophytica]: AMP (Probabilities: 0.0729, 0.9143, 0.0128)\n","Processed WP_206799967.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis sp. 195334CR]: AMP (Probabilities: 0.3719, 0.3764, 0.2517)\n","Processed WP_103574544.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9872, 0.0082, 0.0047)\n","Processed WP_103574398.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9502, 0.0192, 0.0306)\n","Processed WP_073784619.1 gallidermin/nisin family lantibiotic [Streptomyces sp. CB01580]: non-AMP/OTHERS (Probabilities: 0.9839, 0.0097, 0.0064)\n","Processed WP_063887479.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sp. Root1310]: non-AMP/OTHERS (Probabilities: 0.8051, 0.1383, 0.0566)\n","Processed WP_061860274.1 plantaricin C family lantibiotic [Priestia megaterium]: Neuropeptide (Probabilities: 0.3735, 0.0210, 0.6055)\n","Processed WP_139032007.1 plantaricin C family lantibiotic [Priestia megaterium]: Neuropeptide (Probabilities: 0.0389, 0.0193, 0.9418)\n","Processed WP_197225946.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus tropicus]: non-AMP/OTHERS (Probabilities: 0.8840, 0.0165, 0.0996)\n","Processed WP_199627653.1 gallidermin/nisin family lantibiotic, partial [Bacillus cereus group sp. N34]: AMP (Probabilities: 0.0951, 0.8927, 0.0122)\n","Processed WP_199691683.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9674, 0.0077, 0.0248)\n","Processed WP_199698042.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8339, 0.0293, 0.1368)\n","Processed WP_018105658.1 thiazolylpeptide-type bacteriocin [unclassified Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0047, 0.0032)\n","Processed WP_103574573.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0061, 0.0040)\n","Processed WP_190908818.1 mersacidin/lichenicidin family type 2 lantibiotic [Anabaena catenula]: non-AMP/OTHERS (Probabilities: 0.7933, 0.0375, 0.1692)\n","Processed WP_190966549.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc spongiaeforme]: non-AMP/OTHERS (Probabilities: 0.8786, 0.1053, 0.0162)\n","Processed WP_197210092.1 plantaricin C family lantibiotic, partial [Bacillus toyonensis]: non-AMP/OTHERS (Probabilities: 0.8293, 0.0115, 0.1592)\n","Processed WP_197230372.1 gallidermin/nisin family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0031)\n","Processed WP_190873712.1 mersacidin/lichenicidin family type 2 lantibiotic [Aulosira sp. FACHB-615]: Neuropeptide (Probabilities: 0.0303, 0.0173, 0.9524)\n","Processed WP_190453189.1 cinnamycin family lantibiotic [Cylindrospermum sp. FACHB-282]: non-AMP/OTHERS (Probabilities: 0.8868, 0.0723, 0.0409)\n","Processed WP_190574770.1 mersacidin/lichenicidin family type 2 lantibiotic [Alkalinema sp. FACHB-956]: Neuropeptide (Probabilities: 0.0100, 0.0097, 0.9803)\n","Processed WP_190574769.1 mersacidin/lichenicidin family type 2 lantibiotic [Alkalinema sp. FACHB-956]: non-AMP/OTHERS (Probabilities: 0.4759, 0.1108, 0.4132)\n","Processed WP_190521178.1 cinnamycin family lantibiotic [Planktothrix sp. FACHB-1365]: Neuropeptide (Probabilities: 0.2858, 0.0353, 0.6789)\n","Processed WP_190585385.1 mersacidin/lichenicidin family type 2 lantibiotic [Calothrix sp. FACHB-156]: non-AMP/OTHERS (Probabilities: 0.8795, 0.0593, 0.0612)\n","Processed WP_190574768.1 mersacidin/lichenicidin family type 2 lantibiotic [Alkalinema sp. FACHB-956]: non-AMP/OTHERS (Probabilities: 0.7856, 0.1654, 0.0490)\n","Processed WP_190587055.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostocales]: non-AMP/OTHERS (Probabilities: 0.9789, 0.0097, 0.0114)\n","Processed WP_190660571.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc linckia]: non-AMP/OTHERS (Probabilities: 0.9097, 0.0416, 0.0486)\n","Processed WP_190734130.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc sp. FACHB-888]: non-AMP/OTHERS (Probabilities: 0.5585, 0.0567, 0.3847)\n","Processed WP_190565339.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc parmelioides]: non-AMP/OTHERS (Probabilities: 0.5284, 0.2085, 0.2631)\n","Processed WP_190565340.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc parmelioides]: non-AMP/OTHERS (Probabilities: 0.9788, 0.0086, 0.0127)\n","Processed WP_190637886.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Calothrix]: non-AMP/OTHERS (Probabilities: 0.8532, 0.1195, 0.0273)\n","Processed WP_190621338.1 mersacidin/lichenicidin family type 2 lantibiotic [Calothrix membranacea]: non-AMP/OTHERS (Probabilities: 0.7531, 0.0500, 0.1969)\n","Processed WP_134570512.1 plantaricin C family lantibiotic [Cryobacterium sp. Sr3]: non-AMP/OTHERS (Probabilities: 0.7665, 0.1780, 0.0555)\n","Processed WP_190730094.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc sp. FACHB-888]: AMP (Probabilities: 0.3608, 0.5269, 0.1123)\n","Processed WP_190565341.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc parmelioides]: Neuropeptide (Probabilities: 0.0802, 0.0321, 0.8877)\n","Processed WP_190637888.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Calothrix]: non-AMP/OTHERS (Probabilities: 0.9867, 0.0065, 0.0069)\n","Processed WP_120754724.1 lexapeptide family class V lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9789, 0.0091, 0.0120)\n","Processed WP_206723358.1 mersacidin/lichenicidin family type 2 lantibiotic [Pyxidicoccus sp. SCPEA002]: Neuropeptide (Probabilities: 0.3904, 0.0380, 0.5716)\n","Processed WP_069969409.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Desertifilum]: non-AMP/OTHERS (Probabilities: 0.9839, 0.0056, 0.0105)\n","Processed WP_107602421.1 plantaricin C family lantibiotic [Mammaliicoccus sciuri]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0037, 0.0049)\n","Processed WP_073943648.1 gallidermin/nisin family lantibiotic [Streptomyces sp. TSRI0107]: Neuropeptide (Probabilities: 0.4314, 0.0393, 0.5293)\n","Processed WP_073725586.1 gallidermin/nisin family lantibiotic [Streptomyces sp. CB02414]: Neuropeptide (Probabilities: 0.0776, 0.0297, 0.8928)\n","Processed WP_073726004.1 thiazolylpeptide-type bacteriocin [Streptomyces sp. TSRI0281]: non-AMP/OTHERS (Probabilities: 0.9858, 0.0082, 0.0061)\n","Processed WP_129292378.1 gallidermin/nisin family lantibiotic [Streptomyces lydicus]: non-AMP/OTHERS (Probabilities: 0.9589, 0.0354, 0.0057)\n","Processed WP_190593185.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostocaceae]: Neuropeptide (Probabilities: 0.0553, 0.0206, 0.9240)\n","Processed WP_061600469.1 hypothetical protein [Streptococcus gordonii]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0058, 0.0026)\n","Processed WP_051972459.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0041)\n","Processed MBN9388160.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: non-AMP/OTHERS (Probabilities: 0.7502, 0.0929, 0.1569)\n","Processed WP_206702357.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9835, 0.0059, 0.0107)\n","Processed MBN9389167.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: Neuropeptide (Probabilities: 0.0097, 0.0099, 0.9804)\n","Processed WP_039690387.1 hypothetical protein [Streptococcus equinus]: AMP (Probabilities: 0.0653, 0.9201, 0.0146)\n","Processed WP_081034542.1 gallidermin/nisin family lantibiotic [Apilactobacillus kunkeei]: AMP (Probabilities: 0.0688, 0.9163, 0.0149)\n","Processed WP_017369960.1 lichenicidin A2 family type 2 lantibiotic [Lactococcus garvieae]: AMP (Probabilities: 0.3806, 0.6049, 0.0145)\n","Processed WP_129044618.1 lichenicidin A2 family type 2 lantibiotic [Companilactobacillus metriopterae]: non-AMP/OTHERS (Probabilities: 0.9656, 0.0312, 0.0032)\n","Processed WP_026063846.1 lichenicidin A2 family type 2 lantibiotic [Lactococcus garvieae]: AMP (Probabilities: 0.1197, 0.8679, 0.0124)\n","Processed WP_197219812.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9842, 0.0077, 0.0082)\n","Processed MBN9390412.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: non-AMP/OTHERS (Probabilities: 0.8343, 0.0231, 0.1426)\n","Processed WP_161067741.1 gallidermin/nisin family lantibiotic [Ligilactobacillus salivarius]: AMP (Probabilities: 0.0661, 0.9196, 0.0143)\n","Processed WP_136983217.1 gallidermin/nisin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0038, 0.0043)\n","Processed WP_206683310.1 hypothetical protein, partial [Salmonella enterica]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0082, 0.0026)\n","Processed WP_080610217.1 lantibiotic salivaricin M precursor [Streptococcus]: AMP (Probabilities: 0.1725, 0.8088, 0.0188)\n","Processed WP_078076938.1 thiazolylpeptide-type bacteriocin [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0033, 0.0037)\n","Processed WP_096666156.1 hypothetical protein [Staphylococcus intermedius group]: non-AMP/OTHERS (Probabilities: 0.9479, 0.0085, 0.0436)\n","Processed WP_006379227.1 FxLD family lanthipeptide [Streptomyces turgidiscabies]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0071, 0.0026)\n","Processed WP_206425350.1 plantaricin C family lantibiotic [Staphylospora marina]: Neuropeptide (Probabilities: 0.0169, 0.0131, 0.9701)\n","Processed WP_077766543.1 thiopeptide-type bacteriocin biosynthesis protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0043, 0.0029)\n","Processed WP_059336728.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus halotolerans]: AMP (Probabilities: 0.0730, 0.9141, 0.0129)\n","Processed WP_048567912.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9786, 0.0075, 0.0139)\n","Processed WP_169089145.1 plantaricin C family lantibiotic [unclassified Streptomyces]: Neuropeptide (Probabilities: 0.1707, 0.0323, 0.7970)\n","Processed WP_168091463.1 plantaricin C family lantibiotic [Streptomyces sp. AA8]: AMP (Probabilities: 0.0719, 0.9152, 0.0129)\n","Processed WP_124727338.1 plantaricin C family lantibiotic [Staphylospora marina]: Neuropeptide (Probabilities: 0.0104, 0.0101, 0.9795)\n","Processed WP_124269439.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sp. ADI96-02]: AMP (Probabilities: 0.1468, 0.8400, 0.0133)\n","Processed WP_017591584.1 cinnamycin family lantibiotic [Nocardiopsis potens]: AMP (Probabilities: 0.1214, 0.8629, 0.0158)\n","Processed WP_126915190.1 gallidermin/nisin family lantibiotic [Streptomyces luteoverticillatus]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0056, 0.0044)\n","Processed WP_124268919.1 lexapeptide family class V lantibiotic [Streptomyces sp. ADI96-02]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0054, 0.0033)\n","Processed WP_117304476.1 lantibiotic salivaricin M precursor [unclassified Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9480, 0.0476, 0.0044)\n","Processed WP_099092178.1 lantibiotic salivaricin M precursor [Streptococcus marmotae]: non-AMP/OTHERS (Probabilities: 0.9905, 0.0034, 0.0061)\n","Processed WP_083192935.1 FxLD family lanthipeptide [Streptomyces violaceoruber]: non-AMP/OTHERS (Probabilities: 0.9672, 0.0287, 0.0041)\n","Processed WP_206157082.1 gallidermin/nisin family lantibiotic [Staphylococcus sp. EG-SA-32]: Neuropeptide (Probabilities: 0.0195, 0.0130, 0.9675)\n","Processed WP_128896104.1 elgicin/penisin family lantibiotic [Longirhabdus pacifica]: AMP (Probabilities: 0.0671, 0.9190, 0.0139)\n","Processed WP_128896103.1 elgicin/penisin family lantibiotic [Longirhabdus pacifica]: AMP (Probabilities: 0.0663, 0.9192, 0.0145)\n","Processed WP_112233228.1 plantaricin C family lantibiotic [Planococcus maitriensis]: AMP (Probabilities: 0.0756, 0.9117, 0.0127)\n","Processed WP_166378761.1 mersacidin/lichenicidin family type 2 lantibiotic [Catellatospora methionotrophica]: AMP (Probabilities: 0.1310, 0.8553, 0.0137)\n","Processed WP_166378759.1 mersacidin/lichenicidin family type 2 lantibiotic [Catellatospora methionotrophica]: non-AMP/OTHERS (Probabilities: 0.5290, 0.2568, 0.2142)\n","Processed WP_037766215.1 plantaricin C family lantibiotic [Streptomyces]: non-AMP/OTHERS (Probabilities: 0.7717, 0.0149, 0.2134)\n","Processed WP_206099050.1 gallidermin/nisin family lantibiotic [Paenibacillus sp. M-152]: AMP (Probabilities: 0.0942, 0.8937, 0.0121)\n","Processed WP_165626760.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.1421, 0.0245, 0.8335)\n","Processed WP_205694646.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus coralloides]: Neuropeptide (Probabilities: 0.1708, 0.0403, 0.7889)\n","Processed WP_166669242.1 lichenicidin A2 family type 2 lantibiotic [Bacillus sp. BH2]: Neuropeptide (Probabilities: 0.0748, 0.0294, 0.8958)\n","Processed WP_205663413.1 plantaricin C family lantibiotic [Priestia megaterium]: Neuropeptide (Probabilities: 0.0379, 0.0158, 0.9463)\n","Processed WP_110513875.1 mersacidin/lichenicidin family type 2 lantibiotic [Herpetosiphon llansteffanensis]: non-AMP/OTHERS (Probabilities: 0.5508, 0.0196, 0.4296)\n","Processed WP_097906119.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9059, 0.0899, 0.0042)\n","Processed WP_124749683.1 plantaricin C family lantibiotic [Bacillus cereus]: Neuropeptide (Probabilities: 0.0514, 0.0173, 0.9312)\n","Processed WP_124749682.1 plantaricin C family lantibiotic [Bacillus cereus]: Neuropeptide (Probabilities: 0.1452, 0.0263, 0.8284)\n","Processed WP_097906120.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0035)\n","Processed WP_082188801.1 plantaricin C family lantibiotic [Bacillus cereus]: Neuropeptide (Probabilities: 0.1887, 0.0558, 0.7555)\n","Processed MBN3905583.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc sp. NMS1]: AMP (Probabilities: 0.2896, 0.6877, 0.0227)\n","Processed WP_100655775.1 mersacidin/lichenicidin family type 2 lantibiotic [Alteromonas flava]: non-AMP/OTHERS (Probabilities: 0.9879, 0.0043, 0.0078)\n","Processed WP_098159533.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9843, 0.0064, 0.0092)\n","Processed WP_098807156.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0068, 0.0025)\n","Processed WP_205016012.1 gallidermin/nisin family lantibiotic [Pseudonocardia sp. EV170527-09]: AMP (Probabilities: 0.2935, 0.6897, 0.0168)\n","Processed WP_204753463.1 mersacidin/lichenicidin family type 2 lantibiotic [Herpetosiphon giganteus]: non-AMP/OTHERS (Probabilities: 0.9697, 0.0047, 0.0256)\n","Processed WP_164215653.1 plantaricin C family lantibiotic [Virgibacillus sp. YIM 98842]: non-AMP/OTHERS (Probabilities: 0.9542, 0.0216, 0.0242)\n","Processed WP_153834672.1 plantaricin C family lantibiotic [Gracilibacillus thailandensis]: Neuropeptide (Probabilities: 0.0164, 0.0142, 0.9694)\n","Processed WP_084170724.1 gallidermin/nisin family lantibiotic [Corynebacterium freiburgense]: non-AMP/OTHERS (Probabilities: 0.6519, 0.2740, 0.0741)\n","Processed WP_093951064.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis vastitatis]: Neuropeptide (Probabilities: 0.4507, 0.0336, 0.5157)\n","Processed WP_079101695.1 gallidermin/nisin family lantibiotic [Streptomyces pathocidini]: non-AMP/OTHERS (Probabilities: 0.9694, 0.0139, 0.0167)\n","Processed WP_075306062.1 mersacidin/lichenicidin family type 2 lantibiotic [Hyalangium minutum]: AMP (Probabilities: 0.1807, 0.7220, 0.0972)\n","Processed WP_044188757.1 mersacidin/lichenicidin family type 2 lantibiotic [Hyalangium minutum]: non-AMP/OTHERS (Probabilities: 0.8969, 0.0847, 0.0184)\n","Processed WP_019631332.1 cinnamycin family lantibiotic [Actinomadura atramentaria]: non-AMP/OTHERS (Probabilities: 0.9101, 0.0350, 0.0550)\n","Processed EHB2504098.1 lantibiotic ABC transporter ATP-binding protein [Shigella flexneri]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0060, 0.0050)\n","Processed WP_205507739.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus vastator]: Neuropeptide (Probabilities: 0.4698, 0.0263, 0.5039)\n","Processed WP_205412440.1 plantaricin C family lantibiotic [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9770, 0.0060, 0.0171)\n","Processed WP_172934544.1 lantibiotic salivaricin M precursor [Streptococcus sp. 2106]: AMP (Probabilities: 0.0714, 0.9159, 0.0127)\n","Processed WP_163782525.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus vastator]: non-AMP/OTHERS (Probabilities: 0.4852, 0.0529, 0.4618)\n","Processed WP_044188755.1 mersacidin/lichenicidin family type 2 lantibiotic [Hyalangium minutum]: non-AMP/OTHERS (Probabilities: 0.6321, 0.0732, 0.2947)\n","Processed WP_116112331.1 gallidermin/nisin family lantibiotic [Austwickia chelonae]: AMP (Probabilities: 0.0751, 0.9122, 0.0127)\n","Processed MBN1207223.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0041, 0.0034)\n","Processed MBN2921610.1 plantaricin C family lantibiotic [Lactobacillus sp.]: non-AMP/OTHERS (Probabilities: 0.7634, 0.0258, 0.2108)\n","Processed MBN2920455.1 plantaricin C family lantibiotic [Lactobacillus sp.]: Neuropeptide (Probabilities: 0.0197, 0.0149, 0.9654)\n","Processed MBN1207224.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.6100, 0.3752, 0.0148)\n","Processed MBN1207222.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.4515, 0.2447, 0.3038)\n","Processed MBN1206722.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.4900, 0.0689, 0.4411)\n","Processed MBN1204906.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.5108, 0.0678, 0.4214)\n","Processed WP_114865143.1 lantibiotic salivaricin M precursor [Streptococcus pneumoniae]: AMP (Probabilities: 0.0671, 0.9192, 0.0137)\n","Processed WP_114865145.1 lantibiotic salivaricin M precursor [Streptococcus pneumoniae]: AMP (Probabilities: 0.0647, 0.9205, 0.0148)\n","Processed WP_205008929.1 lantibiotic salivaricin M precursor [Streptococcus loxodontisalivarius]: AMP (Probabilities: 0.0684, 0.9179, 0.0137)\n","Processed WP_205008928.1 lantibiotic salivaricin M precursor [Streptococcus loxodontisalivarius]: AMP (Probabilities: 0.0719, 0.9149, 0.0131)\n","Processed WP_205008930.1 lantibiotic salivaricin M precursor [Streptococcus loxodontisalivarius]: AMP (Probabilities: 0.0676, 0.9185, 0.0139)\n","Processed WP_114865144.1 lantibiotic salivaricin M precursor [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.7019, 0.2805, 0.0176)\n","Processed WP_143567379.1 class II lanthipeptide, LchA2/BrtA2 family [Streptococcus salivarius]: non-AMP/OTHERS (Probabilities: 0.9675, 0.0111, 0.0214)\n","Processed WP_149666471.1 gallidermin/nisin family lantibiotic [Pseudonocardia sp. EV170527-09]: non-AMP/OTHERS (Probabilities: 0.8158, 0.0356, 0.1487)\n","Processed WP_120326555.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora globbae]: non-AMP/OTHERS (Probabilities: 0.9677, 0.0277, 0.0046)\n","Processed WP_056937921.1 hypothetical protein [Limosilactobacillus gastricus]: non-AMP/OTHERS (Probabilities: 0.9088, 0.0115, 0.0797)\n","Processed WP_076759801.1 plantaricin C family lantibiotic [Bacillus swezeyi]: Neuropeptide (Probabilities: 0.0494, 0.0239, 0.9267)\n","Processed MBM9833680.1 NisI/SpaI family lantibiotic immunity lipoprotein [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0046, 0.0031)\n","Processed MBM7641304.1 gallidermin/nisin family lantibiotic [Paenibacillus nicotianae]: AMP (Probabilities: 0.1997, 0.7778, 0.0225)\n","Processed WP_136312916.1 plantaricin C family lantibiotic [Actinomyces procaprae]: Neuropeptide (Probabilities: 0.0859, 0.0343, 0.8798)\n","Processed WP_095958469.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus macrosporus]: Neuropeptide (Probabilities: 0.3741, 0.0386, 0.5873)\n","Processed WP_013937967.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus macrosporus]: Neuropeptide (Probabilities: 0.0351, 0.0175, 0.9474)\n","Processed WP_080980733.1 gallidermin/nisin family lantibiotic [Streptococcus pseudopneumoniae]: non-AMP/OTHERS (Probabilities: 0.4662, 0.3544, 0.1793)\n","Processed WP_080978351.1 lantibiotic streptin [Streptococcus pseudopneumoniae]: AMP (Probabilities: 0.0683, 0.9183, 0.0134)\n","Processed MBM7580803.1 type 2 lantibiotic (TIGR03893 family) [Jeotgalibacillus terrae]: non-AMP/OTHERS (Probabilities: 0.9730, 0.0233, 0.0037)\n","Processed WP_017626724.1 plantaricin C family lantibiotic [Nocardiopsis chromatogenes]: AMP (Probabilities: 0.0772, 0.9105, 0.0123)\n","Processed WP_017626723.1 plantaricin C family lantibiotic [Nocardiopsis chromatogenes]: non-AMP/OTHERS (Probabilities: 0.6245, 0.3678, 0.0077)\n","Processed WP_041488697.1 plantaricin C family lantibiotic, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0937, 0.8886, 0.0177)\n","Processed MBC8122624.1 mersacidin/lichenicidin family type 2 lantibiotic [Gemmatimonadaceae bacterium]: Neuropeptide (Probabilities: 0.0113, 0.0109, 0.9778)\n","Processed WP_056937914.1 lichenicidin A2 family type 2 lantibiotic [Limosilactobacillus gastricus]: non-AMP/OTHERS (Probabilities: 0.9719, 0.0063, 0.0217)\n","Processed MBM9833322.1 lantibiotic ABC transporter permease [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0062, 0.0028)\n","Processed WP_049387223.1 plantaricin C family lantibiotic [Staphylococcus epidermidis]: non-AMP/OTHERS (Probabilities: 0.7500, 0.0461, 0.2039)\n","Processed WP_161367057.1 gallidermin/nisin family lantibiotic [Streptomyces sp. SID5643]: non-AMP/OTHERS (Probabilities: 0.7635, 0.0366, 0.1998)\n","Processed WP_161354092.1 SCO0268 family class II lantipeptide [Streptomyces sp. SID3343]: Neuropeptide (Probabilities: 0.0815, 0.0220, 0.8965)\n","Processed WP_161114634.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus thuringiensis]: AMP (Probabilities: 0.1392, 0.8398, 0.0210)\n","Processed WP_161114633.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0039, 0.0031)\n","Processed WP_048866583.1 mersacidin/lichenicidin family type 2 lantibiotic [Scytonema tolypothrichoides]: non-AMP/OTHERS (Probabilities: 0.7481, 0.0344, 0.2175)\n","Processed WP_048866582.1 mersacidin/lichenicidin family type 2 lantibiotic [Scytonema tolypothrichoides]: non-AMP/OTHERS (Probabilities: 0.9764, 0.0062, 0.0174)\n","Processed WP_018513356.1 SCO0268 family class II lantipeptide [unclassified Streptomyces]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0040, 0.0048)\n","Processed MBM6979927.1 lantibiotic ABC transporter permease [Actinomyces succiniciruminis]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0049, 0.0039)\n","Processed WP_204228989.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: non-AMP/OTHERS (Probabilities: 0.6709, 0.0365, 0.2926)\n","Processed WP_204223935.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: non-AMP/OTHERS (Probabilities: 0.6862, 0.1172, 0.1966)\n","Processed WP_204226811.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: non-AMP/OTHERS (Probabilities: 0.4175, 0.3949, 0.1876)\n","Processed WP_204219680.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: non-AMP/OTHERS (Probabilities: 0.6749, 0.0358, 0.2893)\n","Processed WP_204220376.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: non-AMP/OTHERS (Probabilities: 0.9434, 0.0157, 0.0408)\n","Processed WP_204219679.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: AMP (Probabilities: 0.1075, 0.8562, 0.0363)\n","Processed WP_193434159.1 lantibiotic (srt) production protein, partial [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0078, 0.0032)\n","Processed WP_163582611.1 gallidermin/nisin family lantibiotic [Gracilibacillus saliphilus]: AMP (Probabilities: 0.2326, 0.7485, 0.0190)\n","Processed WP_204219128.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: non-AMP/OTHERS (Probabilities: 0.5709, 0.0570, 0.3721)\n","Processed WP_153058232.1 NisI/SpaI family lantibiotic immunity lipoprotein, partial [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0036)\n","Processed WP_167593891.1 lichenicidin A2 family type 2 lantibiotic [Carnobacterium divergens]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0031, 0.0050)\n","Processed WP_135014633.1 lichenicidin A2 family type 2 lantibiotic [Carnobacterium divergens]: non-AMP/OTHERS (Probabilities: 0.9806, 0.0047, 0.0147)\n","Processed WP_024408877.1 class II lanthipeptide, LchA2/BrtA2 family [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9935, 0.0036, 0.0030)\n","Processed  unnamed protein product [Unknown]: AMP (Probabilities: 0.0732, 0.9137, 0.0131)\n","Processed WP_109171310.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.3885, 0.0548, 0.5568)\n","Processed WP_153048435.1 NisI/SpaI family lantibiotic immunity lipoprotein, partial [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9735, 0.0229, 0.0036)\n","Processed WP_135014631.1 lichenicidin A2 family type 2 lantibiotic [Carnobacterium divergens]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0037, 0.0038)\n","Processed WP_105264167.1 NisI/SpaI family lantibiotic immunity lipoprotein, partial [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0057, 0.0035)\n","Processed WP_204227037.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: non-AMP/OTHERS (Probabilities: 0.4519, 0.1141, 0.4340)\n","Processed WP_203945573.1 mersacidin/lichenicidin family type 2 lantibiotic [Planotetraspora thailandica]: non-AMP/OTHERS (Probabilities: 0.9111, 0.0835, 0.0053)\n","Processed WP_193208495.1 mersacidin/lichenicidin family type 2 lantibiotic [Microbispora sitophila]: non-AMP/OTHERS (Probabilities: 0.9078, 0.0322, 0.0601)\n","Processed WP_203960207.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinocatenispora thailandica]: Neuropeptide (Probabilities: 0.0112, 0.0109, 0.9779)\n","Processed WP_203655953.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinocatenispora rupis]: non-AMP/OTHERS (Probabilities: 0.8055, 0.0372, 0.1573)\n","Processed WP_164055077.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.7070, 0.0264, 0.2666)\n","Processed WP_153044509.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0033, 0.0079)\n","Processed WP_203750392.1 mersacidin/lichenicidin family type 2 lantibiotic [Catellatospora bangladeshensis]: Neuropeptide (Probabilities: 0.1985, 0.0323, 0.7691)\n","Processed WP_203656419.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinocatenispora rupis]: non-AMP/OTHERS (Probabilities: 0.9855, 0.0058, 0.0088)\n","Processed WP_100655338.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9865, 0.0062, 0.0073)\n","Processed WP_153043837.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9585, 0.0234, 0.0181)\n","Processed WP_100632393.1 plantaricin C family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8527, 0.0227, 0.1246)\n","Processed WP_086095970.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.1246, 0.0234, 0.8521)\n","Processed QRN81059.1 lantibiotic dehydratase, partial [Nocardiopsis sp. BM-2018]: non-AMP/OTHERS (Probabilities: 0.8511, 0.0141, 0.1348)\n","Processed WP_016878212.1 mersacidin/lichenicidin family type 2 lantibiotic [Chlorogloeopsis fritschii]: non-AMP/OTHERS (Probabilities: 0.5903, 0.0383, 0.3715)\n","Processed WP_063648130.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.5928, 0.2357, 0.1714)\n","Processed WP_083389666.1 mersacidin/lichenicidin family type 2 lantibiotic [Trichormus sp. NMC-1]: non-AMP/OTHERS (Probabilities: 0.9831, 0.0037, 0.0132)\n","Processed OHW10474.1 hypothetical protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0053, 0.0026)\n","Processed WP_152911913.1 type 2 lantibiotic [Enterococcus sp. RIT-PI-f]: Neuropeptide (Probabilities: 0.0814, 0.0313, 0.8873)\n","Processed VTY15374.1 Lantibiotic salivaricin-A [Streptococcus salivarius]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0036)\n","Processed MBM3772128.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidimicrobiia bacterium]: AMP (Probabilities: 0.0791, 0.9087, 0.0121)\n","Processed WP_150531403.1 gallidermin/nisin family lantibiotic, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9811, 0.0144, 0.0045)\n","Processed WP_078092829.1 gallidermin/nisin family lantibiotic, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.8334, 0.1259, 0.0407)\n","Processed WP_058223819.1 plantaricin C family lantibiotic [Lactococcus lactis]: AMP (Probabilities: 0.0780, 0.9093, 0.0127)\n","Processed WP_082522658.1 gallidermin/nisin family lantibiotic [Bacillus sp. Leaf75]: Neuropeptide (Probabilities: 0.0393, 0.0190, 0.9416)\n","Processed WP_076983426.1 gallidermin/nisin family lantibiotic, partial [Bacillus amyloliquefaciens]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0042, 0.0029)\n","Processed WP_057985302.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cecembensis]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0040, 0.0054)\n","Processed WP_056699840.1 plantaricin C family lantibiotic [Paenibacillus sp. Root52]: Neuropeptide (Probabilities: 0.3870, 0.0800, 0.5330)\n","Processed WP_054535766.1 mersacidin/lichenicidin family type 2 lantibiotic [Herpetosiphon geysericola]: non-AMP/OTHERS (Probabilities: 0.6548, 0.0158, 0.3294)\n","Processed WP_054116402.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus sp. RIT-PI-f]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0046, 0.0045)\n","Processed WP_054116398.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus sp. RIT-PI-f]: non-AMP/OTHERS (Probabilities: 0.9450, 0.0085, 0.0465)\n","Processed WP_048311475.1 gallidermin/nisin family lantibiotic [Alkalihalobacillus macyae]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0029)\n","Processed WP_014386877.1 plantaricin C family lantibiotic [Lactobacillaceae]: AMP (Probabilities: 0.4522, 0.5194, 0.0284)\n","Processed WP_203268089.1 gallidermin/nisin family lantibiotic [Streptococcus uberis]: AMP (Probabilities: 0.0993, 0.8779, 0.0227)\n","Processed WP_203410568.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: non-AMP/OTHERS (Probabilities: 0.7044, 0.0581, 0.2375)\n","Processed WP_084724180.1 mersacidin/lichenicidin family type 2 lantibiotic [Thalassomonas viridans]: Neuropeptide (Probabilities: 0.0936, 0.0218, 0.8846)\n","Processed WP_084692501.1 mersacidin/lichenicidin family type 2 lantibiotic [Thalassomonas actiniarum]: Neuropeptide (Probabilities: 0.4175, 0.0265, 0.5560)\n","Processed WP_203404436.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: non-AMP/OTHERS (Probabilities: 0.7753, 0.0283, 0.1965)\n","Processed WP_203403106.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: Neuropeptide (Probabilities: 0.4066, 0.1332, 0.4602)\n","Processed WP_152570713.1 elgicin/penisin family lantibiotic [Paenibacillus tyrfis]: AMP (Probabilities: 0.2130, 0.7117, 0.0753)\n","Processed WP_063856615.1 SCO0268 family class II lantipeptide [Kitasatospora]: non-AMP/OTHERS (Probabilities: 0.9160, 0.0105, 0.0735)\n","Processed WP_152643755.1 gallidermin/nisin family lantibiotic [Paenibacillus terrae]: AMP (Probabilities: 0.0673, 0.9180, 0.0147)\n","Processed WP_043402004.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum]: non-AMP/OTHERS (Probabilities: 0.5888, 0.0401, 0.3711)\n","Processed WP_037917023.1 thiazolylpeptide-type bacteriocin [Streptomyces sp. PCS3-D2]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed WP_162647179.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.2635, 0.0350, 0.7014)\n","Processed WP_009630787.1 mersacidin/lichenicidin family type 2 lantibiotic [Synechocystis sp. PCC 7509]: non-AMP/OTHERS (Probabilities: 0.9704, 0.0083, 0.0213)\n","Processed WP_002289978.1 mutacin-like lantipeptide [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9737, 0.0045, 0.0218)\n","Processed WP_024785189.1 mutacin-like lantipeptide [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0039, 0.0043)\n","Processed WP_009630791.1 mersacidin/lichenicidin family type 2 lantibiotic [Synechocystis sp. PCC 7509]: non-AMP/OTHERS (Probabilities: 0.9173, 0.0094, 0.0732)\n","Processed WP_009630790.1 mersacidin/lichenicidin family type 2 lantibiotic [Synechocystis sp. PCC 7509]: non-AMP/OTHERS (Probabilities: 0.9615, 0.0113, 0.0272)\n","Processed  unnamed protein product [Unknown]: AMP (Probabilities: 0.1144, 0.8733, 0.0123)\n","Processed WP_080010376.1 gallidermin/nisin family lantibiotic [Bacillus spizizenii]: AMP (Probabilities: 0.0684, 0.9172, 0.0144)\n","Processed WP_053559940.1 FxLD family lanthipeptide [Streptomyces sp. CFMR 7]: non-AMP/OTHERS (Probabilities: 0.9843, 0.0065, 0.0092)\n","Processed WP_016860135.1 mersacidin/lichenicidin family type 2 lantibiotic [Fischerella muscicola]: non-AMP/OTHERS (Probabilities: 0.9203, 0.0205, 0.0592)\n","Processed WP_002270939.1 lantibiotic mutacin I, partial [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0035, 0.0039)\n","Processed WP_009630786.1 mersacidin/lichenicidin family type 2 lantibiotic [Synechocystis sp. PCC 7509]: non-AMP/OTHERS (Probabilities: 0.9853, 0.0108, 0.0039)\n","Processed WP_009630785.1 mersacidin/lichenicidin family type 2 lantibiotic [Synechocystis sp. PCC 7509]: non-AMP/OTHERS (Probabilities: 0.6890, 0.0275, 0.2835)\n","Processed MBC7593382.1 plantaricin C family lantibiotic [Kineosporiaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.6226, 0.1378, 0.2396)\n","Processed MBC7978644.1 lantibiotic dehydratase [Myxococcales bacterium]: non-AMP/OTHERS (Probabilities: 0.9729, 0.0066, 0.0205)\n","Processed WP_002295970.1 lantibiotic mutacin I, partial [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed WP_100085773.1 elgicin/penisin family lantibiotic [Paenibacillus elgii]: AMP (Probabilities: 0.0954, 0.8916, 0.0130)\n","Processed WP_141121471.1 plantaricin C family lantibiotic [Rothia dentocariosa]: Neuropeptide (Probabilities: 0.0459, 0.0189, 0.9352)\n","Processed WP_008480453.1 mersacidin/lichenicidin family type 2 lantibiotic [Nitrolancea hollandica]: non-AMP/OTHERS (Probabilities: 0.4815, 0.0417, 0.4767)\n","Processed WP_190837386.1 mersacidin/lichenicidin family type 2 lantibiotic [Iningainema tapete]: non-AMP/OTHERS (Probabilities: 0.8201, 0.0141, 0.1657)\n","Processed WP_080513130.1 gallidermin/nisin family lantibiotic [Lactococcus lactis]: non-AMP/OTHERS (Probabilities: 0.8774, 0.1173, 0.0053)\n","Processed WP_037583347.1 mersacidin/lichenicidin family type 2 lantibiotic [Stigmatella aurantiaca]: Neuropeptide (Probabilities: 0.4495, 0.0715, 0.4790)\n","Processed WP_012412843.1 lichenicidin A2 family type 2 lantibiotic [Nostoc punctiforme]: non-AMP/OTHERS (Probabilities: 0.8895, 0.1051, 0.0054)\n","Processed WP_168751399.1 mutacin-like lantipeptide [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed WP_007358547.1 mersacidin/lichenicidin family type 2 lantibiotic [Kamptonema]: non-AMP/OTHERS (Probabilities: 0.9832, 0.0091, 0.0077)\n","Processed WP_002611603.1 mersacidin/lichenicidin family type 2 lantibiotic [Stigmatella aurantiaca]: non-AMP/OTHERS (Probabilities: 0.9307, 0.0465, 0.0228)\n","Processed WP_013939440.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus macrosporus]: Neuropeptide (Probabilities: 0.3339, 0.0359, 0.6303)\n","Processed MBL8206962.1 mersacidin/lichenicidin family type 2 lantibiotic [Blastocatellia bacterium]: non-AMP/OTHERS (Probabilities: 0.9645, 0.0239, 0.0117)\n","Processed MBL8208251.1 mersacidin/lichenicidin family type 2 lantibiotic [Blastocatellia bacterium]: non-AMP/OTHERS (Probabilities: 0.9678, 0.0190, 0.0133)\n","Processed WP_165952514.1 mersacidin/lichenicidin family type 2 lantibiotic [Kribbella albertanoniae]: Neuropeptide (Probabilities: 0.0864, 0.0251, 0.8886)\n","Processed WP_017305343.1 mersacidin/lichenicidin family type 2 lantibiotic [Spirulina subsalsa]: non-AMP/OTHERS (Probabilities: 0.7660, 0.0364, 0.1976)\n","Processed WP_192511255.1 gallidermin/nisin family lantibiotic [Paenibacillus sp. 23TSA30-6]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0041, 0.0030)\n","Processed WP_192511253.1 gallidermin/nisin family lantibiotic [Paenibacillus sp. 23TSA30-6]: AMP (Probabilities: 0.1612, 0.8266, 0.0122)\n","Processed WP_192511254.1 gallidermin/nisin family lantibiotic [Paenibacillus sp. 23TSA30-6]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0031, 0.0036)\n","Processed MBL7062132.1 lantibiotic ABC transporter [Dehalococcoidia bacterium]: non-AMP/OTHERS (Probabilities: 0.9868, 0.0050, 0.0083)\n","Processed WP_014398760.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus coralloides]: Neuropeptide (Probabilities: 0.2530, 0.0272, 0.7198)\n","Processed WP_161923253.1 plantaricin C family lantibiotic [Actinomyces sp. 432]: non-AMP/OTHERS (Probabilities: 0.8869, 0.0286, 0.0844)\n","Processed WP_081495980.1 gallidermin/nisin family lantibiotic [Actinomyces sp. ICM58]: Neuropeptide (Probabilities: 0.1944, 0.0655, 0.7401)\n","Processed WP_076982923.1 gallidermin/nisin family lantibiotic [Bacillus velezensis]: non-AMP/OTHERS (Probabilities: 0.9722, 0.0242, 0.0036)\n","Processed WP_161517816.1 plantaricin C family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9865, 0.0049, 0.0086)\n","Processed MBL3847053.1 mersacidin family lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.1875, 0.7999, 0.0126)\n","Processed WP_161517817.1 plantaricin C family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9735, 0.0047, 0.0218)\n","Processed WP_161387439.1 lantibiotic paenibacillin [Staphylococcus epidermidis]: AMP (Probabilities: 0.0745, 0.9127, 0.0128)\n","Processed WP_161516538.1 lantibiotic salivaricin M precursor [Streptococcus agalactiae]: AMP (Probabilities: 0.2701, 0.6202, 0.1097)\n","Processed WP_079267664.1 gallidermin/nisin family lantibiotic [Streptococcus suis]: AMP (Probabilities: 0.2558, 0.6894, 0.0548)\n","Processed WP_063549727.1 plantaricin C family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9135, 0.0100, 0.0765)\n","Processed WP_161104594.1 cinnamycin family lantibiotic [Actinomadura rayongensis]: non-AMP/OTHERS (Probabilities: 0.5180, 0.0474, 0.4346)\n","Processed MBL3822774.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9536, 0.0146, 0.0318)\n","Processed WP_191261993.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Comamonas]: Neuropeptide (Probabilities: 0.4501, 0.0394, 0.5105)\n","Processed WP_191267182.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Comamonas]: non-AMP/OTHERS (Probabilities: 0.6038, 0.0800, 0.3162)\n","Processed WP_191261994.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Comamonas]: Neuropeptide (Probabilities: 0.2809, 0.0348, 0.6843)\n","Processed MBL3616076.1 lantibiotic ABC transporter ATP-binding protein [Bacillus sp. RHFS18]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0040, 0.0035)\n","Processed WP_144647237.1 plantaricin C family lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9861, 0.0055, 0.0084)\n","Processed WP_161380134.1 plantaricin C family lantibiotic [Staphylococcus hominis]: non-AMP/OTHERS (Probabilities: 0.8023, 0.0264, 0.1712)\n","Processed WP_140862133.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: non-AMP/OTHERS (Probabilities: 0.5864, 0.0349, 0.3787)\n","Processed WP_140877374.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: Neuropeptide (Probabilities: 0.0178, 0.0132, 0.9690)\n","Processed WP_174259345.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: non-AMP/OTHERS (Probabilities: 0.7920, 0.0212, 0.1868)\n","Processed WP_174258912.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: non-AMP/OTHERS (Probabilities: 0.8051, 0.0243, 0.1706)\n","Processed WP_140795328.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: Neuropeptide (Probabilities: 0.0424, 0.0200, 0.9376)\n","Processed WP_174260093.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: Neuropeptide (Probabilities: 0.3478, 0.0295, 0.6227)\n","Processed WP_201855384.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sp. 110]: non-AMP/OTHERS (Probabilities: 0.5444, 0.0508, 0.4048)\n","Processed WP_140790406.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: Neuropeptide (Probabilities: 0.2178, 0.0880, 0.6943)\n","Processed WP_140790404.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: non-AMP/OTHERS (Probabilities: 0.9520, 0.0319, 0.0161)\n","Processed MBL2582222.1 lantibiotic ABC transporter ATP-binding protein [Klebsiella pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9896, 0.0059, 0.0045)\n","Processed WP_201749845.1 hypothetical protein, partial [Micromonospora sp. 4G51]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0038, 0.0040)\n","Processed WP_055426429.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium aesculapii]: Neuropeptide (Probabilities: 0.3831, 0.0269, 0.5900)\n","Processed WP_140790402.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: Neuropeptide (Probabilities: 0.4142, 0.0535, 0.5323)\n","Processed MBL1437701.1 mersacidin/lichenicidin family type 2 lantibiotic [Rhodobacteraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.8672, 0.1186, 0.0142)\n","Processed NGY89472.1 type 2 lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9877, 0.0100, 0.0023)\n","Processed WP_201384293.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacter sp. SOSP1-52]: non-AMP/OTHERS (Probabilities: 0.7699, 0.0190, 0.2110)\n","Processed WP_201394356.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacter sp. SOSP1-52]: Neuropeptide (Probabilities: 0.0112, 0.0105, 0.9783)\n","Processed WP_201384612.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacter sp. SOSP1-85]: Neuropeptide (Probabilities: 0.0338, 0.0184, 0.9478)\n","Processed WP_017747623.1 mersacidin/lichenicidin family type 2 lantibiotic [Scytonema hofmannii]: non-AMP/OTHERS (Probabilities: 0.9816, 0.0073, 0.0111)\n","Processed WP_055426427.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium aesculapii]: non-AMP/OTHERS (Probabilities: 0.9682, 0.0065, 0.0253)\n","Processed WP_201368822.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacter robiniae]: Neuropeptide (Probabilities: 0.0158, 0.0116, 0.9726)\n","Processed WP_201361146.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter formicarum]: Neuropeptide (Probabilities: 0.0352, 0.0206, 0.9442)\n","Processed WP_201365883.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter formicarum]: non-AMP/OTHERS (Probabilities: 0.9672, 0.0076, 0.0252)\n","Processed WP_201365882.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter formicarum]: non-AMP/OTHERS (Probabilities: 0.7878, 0.0245, 0.1877)\n","Processed MBK8989874.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: non-AMP/OTHERS (Probabilities: 0.9720, 0.0076, 0.0204)\n","Processed MBK8252963.1 mersacidin/lichenicidin family type 2 lantibiotic [Polyangiaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.6747, 0.1368, 0.1884)\n","Processed WP_201361239.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter formicarum]: Neuropeptide (Probabilities: 0.2501, 0.0916, 0.6584)\n","Processed WP_201365880.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter formicarum]: non-AMP/OTHERS (Probabilities: 0.9657, 0.0197, 0.0145)\n","Processed WP_159863658.1 lichenicidin A2 family type 2 lantibiotic [Bacillus sp. ZZV12-4809]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0051, 0.0028)\n","Processed WP_048544672.1 thiazolylpeptide-type bacteriocin [Tetrasphaera jenkinsii]: non-AMP/OTHERS (Probabilities: 0.9704, 0.0237, 0.0058)\n","Processed MBK8257190.1 mersacidin/lichenicidin family type 2 lantibiotic [Polyangiaceae bacterium]: AMP (Probabilities: 0.1680, 0.8056, 0.0264)\n","Processed MBK7885353.1 lantibiotic dehydratase [Chitinophagaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0036)\n","Processed WP_061088025.1 hypothetical protein [Streptococcus mitis]: Neuropeptide (Probabilities: 0.2087, 0.0446, 0.7467)\n","Processed WP_026588346.1 plantaricin C family lantibiotic [Bacillus sp. NSP9.1]: Neuropeptide (Probabilities: 0.0247, 0.0167, 0.9586)\n","Processed WP_152663464.1 plantaricin C family lantibiotic [Paenibacillus wulumuqiensis]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0087, 0.0025)\n","Processed WP_083468702.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc piscinale]: non-AMP/OTHERS (Probabilities: 0.7581, 0.2178, 0.0242)\n","Processed WP_038522696.1 FxLD family lanthipeptide [Streptomyces albulus]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0069, 0.0032)\n","Processed WP_000095658.1 plantaricin C family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9815, 0.0063, 0.0122)\n","Processed WP_144399439.1 lichenicidin A2 family type 2 lantibiotic [Pilibacter termitis]: AMP (Probabilities: 0.3272, 0.6629, 0.0099)\n","Processed WP_078762901.1 cinnamycin family lantibiotic [Marinactinospora thermotolerans]: non-AMP/OTHERS (Probabilities: 0.8364, 0.0408, 0.1228)\n","Processed WP_199924955.1 lichenicidin A2 family type 2 lantibiotic [Paenibacillus sp. IHBB 10380]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0041, 0.0035)\n","Processed AYU99978.1 BsaG protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0049, 0.0030)\n","Processed WP_087097274.1 lantibiotic protein [Nocardiopsis sp. JB363]: Neuropeptide (Probabilities: 0.1118, 0.0284, 0.8597)\n","Processed WP_200730597.1 plantaricin C family lantibiotic [Staphylococcus aureus]: AMP (Probabilities: 0.0675, 0.9185, 0.0140)\n","Processed WP_200509781.1 type 2 lantibiotic [Bacillus sp. S56]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0036, 0.0031)\n","Processed WP_079024393.1 plantaricin C family lantibiotic [Streptomyces scabiei]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0071, 0.0029)\n","Processed WP_044878967.1 lichenicidin A2 family type 2 lantibiotic [Paenibacillus sp. IHBB 10380]: AMP (Probabilities: 0.1314, 0.8568, 0.0118)\n","Processed WP_036103586.1 plantaricin C family lantibiotic [Listeria grayi]: non-AMP/OTHERS (Probabilities: 0.7039, 0.0797, 0.2164)\n","Processed WP_107050609.1 gallidermin/nisin family lantibiotic [Streptomyces sp. NRRL F-2580]: non-AMP/OTHERS (Probabilities: 0.9611, 0.0090, 0.0299)\n","Processed WP_084570170.1 mersacidin/lichenicidin family type 2 lantibiotic [Nocardiopsis alba]: non-AMP/OTHERS (Probabilities: 0.8033, 0.1908, 0.0058)\n","Processed MBK3299427.1 lantibiotic ABC transporter permease [Streptococcus oralis]: non-AMP/OTHERS (Probabilities: 0.9859, 0.0112, 0.0029)\n","Processed WP_065004212.1 FxLD family lanthipeptide [Streptomyces sp. H-KF8]: non-AMP/OTHERS (Probabilities: 0.9696, 0.0060, 0.0244)\n","Processed WP_030998976.1 plantaricin C family lantibiotic [Streptomyces sp. NRRL F-5630]: non-AMP/OTHERS (Probabilities: 0.6347, 0.0243, 0.3410)\n","Processed AMV77797.1 epidermin immunity protein F [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0069, 0.0027)\n","Processed WP_105996299.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus agnetis]: non-AMP/OTHERS (Probabilities: 0.9357, 0.0604, 0.0039)\n","Processed  unnamed protein product [Unknown]: non-AMP/OTHERS (Probabilities: 0.9733, 0.0137, 0.0130)\n","Processed WP_082352766.1 gallidermin/nisin family lantibiotic, partial [Bacillus sp. FJAT-21351]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0058, 0.0027)\n","Processed WP_053458365.1 mersacidin/lichenicidin family type 2 lantibiotic [Hapalosiphon sp. MRB220]: non-AMP/OTHERS (Probabilities: 0.9299, 0.0302, 0.0399)\n","Processed WP_199794786.1 plantaricin C family lantibiotic [Paenibacillus sp. Marseille-P3787]: Neuropeptide (Probabilities: 0.0171, 0.0132, 0.9697)\n","Processed WP_199746864.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. AB049A]: Neuropeptide (Probabilities: 0.2117, 0.0406, 0.7477)\n","Processed WP_199738350.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA054B]: Neuropeptide (Probabilities: 0.1463, 0.0404, 0.8132)\n","Processed WP_199733727.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA041A]: Neuropeptide (Probabilities: 0.1614, 0.0334, 0.8052)\n","Processed WP_199560705.1 gallidermin/nisin family lantibiotic [Nocardiopsis sp. L17-MgMaSL7]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0038)\n","Processed WP_122196332.1 cinnamycin family lantibiotic [Actinomadura harenae]: non-AMP/OTHERS (Probabilities: 0.9812, 0.0147, 0.0040)\n","Processed WP_105996303.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus agnetis]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0031, 0.0042)\n","Processed WP_120621866.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA049B]: Neuropeptide (Probabilities: 0.4070, 0.0697, 0.5233)\n","Processed WP_120587568.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA054B]: Neuropeptide (Probabilities: 0.3692, 0.0957, 0.5352)\n","Processed WP_120563712.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Corallococcus]: Neuropeptide (Probabilities: 0.2398, 0.0294, 0.7308)\n","Processed AJP67163.1 membrane protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0064, 0.0027)\n","Processed MBJ8269003.1 type 2 lantibiotic [Staphylococcus pseudintermedius]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0086, 0.0028)\n","Processed MBJ8090925.1 mersacidin family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9806, 0.0049, 0.0145)\n","Processed WP_117401337.1 lantibiotic protein [Actinomadura sp. LHW52907]: non-AMP/OTHERS (Probabilities: 0.8554, 0.1341, 0.0105)\n","Processed WP_110101343.1 gallidermin/nisin family lantibiotic [unclassified Auritidibacter]: non-AMP/OTHERS (Probabilities: 0.7743, 0.0261, 0.1996)\n","Processed MBJ8128298.1 lantibiotic cytolysin [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0035, 0.0051)\n","Processed WP_108076666.1 mersacidin/lichenicidin family type 2 lantibiotic [Vitiosangium sp. GDMCC 1.1324]: AMP (Probabilities: 0.2964, 0.3744, 0.3292)\n","Processed MBJ8269004.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus pseudintermedius]: non-AMP/OTHERS (Probabilities: 0.9529, 0.0424, 0.0046)\n","Processed WP_199246069.1 mersacidin/lichenicidin family type 2 lantibiotic [[Phormidium] sp. ETS-05]: non-AMP/OTHERS (Probabilities: 0.9848, 0.0057, 0.0095)\n","Processed WP_108065533.1 mersacidin/lichenicidin family type 2 lantibiotic [Vitiosangium sp. GDMCC 1.1324]: non-AMP/OTHERS (Probabilities: 0.5332, 0.0461, 0.4208)\n","Processed WP_199036841.1 mersacidin/lichenicidin family type 2 lantibiotic [Glycomyces sp. YIM93776]: Neuropeptide (Probabilities: 0.2082, 0.2364, 0.5554)\n","Processed  unnamed protein product [Unknown]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0052, 0.0032)\n","Processed WP_146071102.1 mersacidin/lichenicidin family type 2 lantibiotic [Cryobacterium sp. Y57]: AMP (Probabilities: 0.1189, 0.8658, 0.0153)\n","Processed MBJ6216362.1 lantibiotic ABC transporter ATP-binding protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0033, 0.0076)\n","Processed MBJ6205775.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0084, 0.0028)\n","Processed WP_199036842.1 mersacidin/lichenicidin family type 2 lantibiotic [Glycomyces sp. YIM93776]: non-AMP/OTHERS (Probabilities: 0.8415, 0.0655, 0.0930)\n","Processed WP_151758635.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter vulcani]: Neuropeptide (Probabilities: 0.2140, 0.2145, 0.5715)\n","Processed WP_088005087.1 gallidermin/nisin family lantibiotic, partial [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9831, 0.0045, 0.0124)\n","Processed WP_142247183.1 gallidermin/nisin family lantibiotic, partial [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.8966, 0.0978, 0.0056)\n","Processed WP_073558439.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium sp. Cb G35]: non-AMP/OTHERS (Probabilities: 0.8632, 0.0524, 0.0844)\n","Processed WP_088005304.1 gallidermin/nisin family lantibiotic, partial [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9431, 0.0537, 0.0032)\n","Processed WP_073558335.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium sp. Cb G35]: non-AMP/OTHERS (Probabilities: 0.9831, 0.0078, 0.0091)\n","Processed WP_149517495.1 lantibiotic streptin [Streptococcus cristatus]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0047, 0.0029)\n","Processed WP_073558340.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium sp. Cb G35]: Neuropeptide (Probabilities: 0.3090, 0.0639, 0.6271)\n","Processed WP_073567086.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium sp. Cb G35]: non-AMP/OTHERS (Probabilities: 0.9605, 0.0109, 0.0287)\n","Processed WP_077412202.1 mersacidin/lichenicidin family type 2 lantibiotic [Marinicella sediminis]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0059, 0.0040)\n","Processed WP_157804293.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Lysinibacillus xylanilyticus]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0042, 0.0044)\n","Processed WP_073560491.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium sp. Cb G35]: non-AMP/OTHERS (Probabilities: 0.9862, 0.0060, 0.0077)\n","Processed WP_101028058.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium longum]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0033, 0.0043)\n","Processed WP_100330120.1 plantaricin C family lantibiotic [Bacillus xiapuensis]: AMP (Probabilities: 0.2389, 0.7459, 0.0152)\n","Processed MBI5928042.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: Neuropeptide (Probabilities: 0.1969, 0.0686, 0.7345)\n","Processed MBI4511131.1 mersacidin/lichenicidin family type 2 lantibiotic [Deltaproteobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.9111, 0.0222, 0.0668)\n","Processed WP_017718744.1 cinnamycin family lantibiotic [Oscillatoria sp. PCC 10802]: non-AMP/OTHERS (Probabilities: 0.9163, 0.0657, 0.0179)\n","Processed MBI5928041.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: AMP (Probabilities: 0.1213, 0.8410, 0.0377)\n","Processed WP_112266928.1 mersacidin/lichenicidin family type 2 lantibiotic [Lentzea terrae]: non-AMP/OTHERS (Probabilities: 0.9706, 0.0252, 0.0042)\n","Processed MBI2839885.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.9905, 0.0040, 0.0056)\n","Processed WP_095984398.1 mersacidin/lichenicidin family type 2 lantibiotic [Cystobacter fuscus]: non-AMP/OTHERS (Probabilities: 0.9827, 0.0062, 0.0111)\n","Processed WP_095564865.1 thiazolylpeptide-type bacteriocin [Plantactinospora sp. KBS50]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0054, 0.0033)\n","Processed WP_198463372.1 lantibiotic streptin [Streptococcus salivarius]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0037, 0.0040)\n","Processed WP_086936211.1 mersacidin/lichenicidin family type 2 lantibiotic [Chamaesiphon minutus]: non-AMP/OTHERS (Probabilities: 0.9828, 0.0055, 0.0116)\n","Processed WP_157260190.1 mersacidin/lichenicidin family type 2 lantibiotic [Chamaesiphon minutus]: non-AMP/OTHERS (Probabilities: 0.9885, 0.0042, 0.0073)\n","Processed WP_095838483.1 mersacidin/lichenicidin family type 2 lantibiotic [Chitinophaga sp. MD30]: non-AMP/OTHERS (Probabilities: 0.7541, 0.1663, 0.0796)\n","Processed WP_000597529.1 plantaricin C family lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.0684, 0.9175, 0.0141)\n","Processed WP_000239232.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9841, 0.0104, 0.0055)\n","Processed WP_066130220.1 hypothetical protein [Gemella asaccharolytica]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0034)\n","Processed WP_066130216.1 hypothetical protein [Gemella asaccharolytica]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0042, 0.0029)\n","Processed WP_080010459.1 gallidermin/nisin family lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9287, 0.0676, 0.0038)\n","Processed WP_080000989.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.7391, 0.0153, 0.2456)\n","Processed WP_000492728.1 plantaricin C family lantibiotic [Bacillus cereus]: AMP (Probabilities: 0.1236, 0.8617, 0.0147)\n","Processed WP_000551056.1 plantaricin C family lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.5156, 0.4738, 0.0106)\n","Processed WP_084369829.1 gallidermin/nisin family lantibiotic [Microbispora sp. ATCC PTA-5024]: Neuropeptide (Probabilities: 0.0589, 0.0203, 0.9208)\n","Processed WP_006544294.1 cinnamycin family lantibiotic [Frankia sp. EUN1f]: AMP (Probabilities: 0.1164, 0.8687, 0.0149)\n","Processed WP_156857985.1 plantaricin C family lantibiotic, partial [Oceanobacillus sp. AG]: non-AMP/OTHERS (Probabilities: 0.6029, 0.0373, 0.3598)\n","Processed WP_036835794.1 plantaricin C family lantibiotic [Pontibacillus litoralis]: non-AMP/OTHERS (Probabilities: 0.9795, 0.0047, 0.0158)\n","Processed WP_156857984.1 plantaricin C family lantibiotic [Oceanobacillus sp. AG]: non-AMP/OTHERS (Probabilities: 0.5738, 0.0383, 0.3879)\n","Processed WP_156857986.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Oceanobacillus sp. AG]: Neuropeptide (Probabilities: 0.0119, 0.0108, 0.9773)\n","Processed WP_084654449.1 gallidermin/nisin family lantibiotic [Paenibacillus zanthoxyli]: AMP (Probabilities: 0.0980, 0.8900, 0.0120)\n","Processed WP_083836188.1 gallidermin/nisin family lantibiotic [Actinomyces sp. ICM47]: non-AMP/OTHERS (Probabilities: 0.9837, 0.0058, 0.0104)\n","Processed WP_006100521.1 mersacidin/lichenicidin family type 2 lantibiotic [Coleofasciculus chthonoplastes]: non-AMP/OTHERS (Probabilities: 0.5314, 0.4484, 0.0203)\n","Processed P80666.1 Lantibiotic mutacin B-Ny266 [Streptococcus mutans]: AMP (Probabilities: 0.0658, 0.9188, 0.0154)\n","Processed WP_197287831.1 plantaricin C family lantibiotic [Streptomyces sp. TRM66233]: non-AMP/OTHERS (Probabilities: 0.4933, 0.0326, 0.4741)\n","Processed WP_035404668.1 plantaricin C family lantibiotic [Bacillus sp. SJS]: non-AMP/OTHERS (Probabilities: 0.9818, 0.0149, 0.0032)\n","Processed WP_013642645.1 plantaricin C family lantibiotic [Streptococcus gallolyticus]: non-AMP/OTHERS (Probabilities: 0.9315, 0.0565, 0.0120)\n","Processed WP_015895698.1 mersacidin/lichenicidin family type 2 lantibiotic [Gemmatimonas aurantiaca]: Neuropeptide (Probabilities: 0.3265, 0.1249, 0.5486)\n","Processed WP_156413137.1 type 2 lantibiotic [Oceanivirga salmonicida]: AMP (Probabilities: 0.0670, 0.9182, 0.0148)\n","Processed WP_164925565.1 lantibiotic salivaricin M precursor [Streptococcus mitis]: non-AMP/OTHERS (Probabilities: 0.9684, 0.0277, 0.0039)\n","Processed WP_083765804.1 mersacidin/lichenicidin family type 2 lantibiotic [Gemmatimonas aurantiaca]: non-AMP/OTHERS (Probabilities: 0.5403, 0.1377, 0.3220)\n","Processed WP_081276761.1 gallidermin/nisin family lantibiotic [Paenibacillus]: AMP (Probabilities: 0.0749, 0.9125, 0.0126)\n","Processed WP_060807398.1 plantaricin C family lantibiotic [Veillonella atypica]: AMP (Probabilities: 0.1282, 0.8565, 0.0153)\n","Processed WP_053615066.1 FxLD family lanthipeptide [Nocardiopsis sp. NRRL B-16309]: Neuropeptide (Probabilities: 0.0163, 0.0122, 0.9716)\n","Processed WP_046563261.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora sp. HK10]: non-AMP/OTHERS (Probabilities: 0.8842, 0.1100, 0.0058)\n","Processed WP_046199690.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus sp. UMTAT18]: non-AMP/OTHERS (Probabilities: 0.9792, 0.0158, 0.0050)\n","Processed WP_156286354.1 lichenicidin A2 family type 2 lantibiotic [Oceanivirga salmonicida]: non-AMP/OTHERS (Probabilities: 0.9771, 0.0193, 0.0036)\n","Processed WP_081752209.1 gallidermin/nisin family lantibiotic [Paenibacillus sp. 1-18]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0080, 0.0025)\n","Processed WP_154803673.1 class II lanthipeptide, LchA2/BrtA2 family [Streptococcus equi]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0041, 0.0034)\n","Processed WP_035298103.1 mersacidin/lichenicidin family type 2 lantibiotic [Brevibacillus thermoruber]: Neuropeptide (Probabilities: 0.4494, 0.0507, 0.4998)\n","Processed MBH0247898.1 lantibiotic dehydratase [Streptomyces cavourensis]: non-AMP/OTHERS (Probabilities: 0.9092, 0.0769, 0.0139)\n","Processed WP_030508207.1 mersacidin/lichenicidin family type 2 lantibiotic [Microbispora rosea]: non-AMP/OTHERS (Probabilities: 0.9817, 0.0118, 0.0065)\n","Processed MBH0351307.1 lichenicidin prepeptide [Bacillus thuringiensis]: AMP (Probabilities: 0.0742, 0.9126, 0.0132)\n","Processed WP_197008977.1 cinnamycin family lantibiotic [Actinomadura viridis]: non-AMP/OTHERS (Probabilities: 0.7078, 0.0409, 0.2513)\n","Processed WP_196497986.1 gallidermin/nisin family lantibiotic [Lactococcus lactis]: non-AMP/OTHERS (Probabilities: 0.9801, 0.0103, 0.0096)\n","Processed MBG3235574.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9781, 0.0046, 0.0173)\n","Processed WP_025720095.1 plantaricin C family lantibiotic [Paenibacillus sp. 1-18]: Neuropeptide (Probabilities: 0.3633, 0.1351, 0.5016)\n","Processed WP_033607255.1 two-peptide bacteriocin subunit PneA1 [Streptococcus sp. BS29a]: non-AMP/OTHERS (Probabilities: 0.9846, 0.0131, 0.0023)\n","Processed WP_080999156.1 lichenicidin A2 family type 2 lantibiotic [Trueperella pyogenes]: AMP (Probabilities: 0.1019, 0.8863, 0.0118)\n","Processed P36500.2 Lantibiotic salivaricin-A [Streptococcus salivarius]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0040)\n","Processed WP_196301930.1 thiopeptide-type bacteriocin biosynthesis protein, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9834, 0.0104, 0.0061)\n","Processed MBF9637156.1 lantibiotic ABC transporter permease [Streptococcus pseudopneumoniae]: non-AMP/OTHERS (Probabilities: 0.9869, 0.0105, 0.0027)\n","Processed WP_196299934.1 thiopeptide-type bacteriocin biosynthesis protein, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0075, 0.0028)\n","Processed WP_155458842.1 lantibiotic salivaricin M precursor, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9096, 0.0840, 0.0064)\n","Processed WP_196245349.1 gallidermin/nisin family lantibiotic [Paenibacillus polymyxa]: Neuropeptide (Probabilities: 0.0366, 0.0180, 0.9454)\n","Processed WP_078168842.1 hypothetical protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0058, 0.0027)\n","Processed WP_080448189.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9261, 0.0114, 0.0625)\n","Processed WP_030029257.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9828, 0.0133, 0.0039)\n","Processed WP_002415200.1 lantipeptide cytolysin subunit CylL-S [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9838, 0.0041, 0.0121)\n","Processed HIO22021.1 mersacidin/lichenicidin family type 2 lantibiotic [Nitrospirales bacterium]: AMP (Probabilities: 0.1772, 0.7606, 0.0622)\n","Processed WP_151379474.1 lantibiotic streptin [Streptococcus]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0058, 0.0026)\n","Processed WP_195865236.1 lichenicidin A2 family type 2 lantibiotic [Exiguobacterium sp. TBG-PICH-001]: non-AMP/OTHERS (Probabilities: 0.9854, 0.0048, 0.0098)\n","Processed WP_074048701.1 lichenicidin A2 family type 2 lantibiotic [Paenibacillus ihumii]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0054, 0.0030)\n","Processed WP_189329732.1 gallidermin/nisin family lantibiotic [Actinoplanes ianthinogenes]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0048, 0.0026)\n","Processed WP_189036675.1 plantaricin C family lantibiotic [Streptomyces daqingensis]: AMP (Probabilities: 0.0668, 0.9181, 0.0151)\n","Processed WP_189286533.1 plantaricin C family lantibiotic [Streptomyces fumigatiscleroticus]: AMP (Probabilities: 0.1167, 0.8609, 0.0224)\n","Processed WP_154687196.1 cinnamycin family lantibiotic [Microlunatus sp. Gsoil 973]: AMP (Probabilities: 0.1943, 0.7328, 0.0730)\n","Processed WP_191256773.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis oliviviridis]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0037, 0.0062)\n","Processed WP_191260058.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis oliviviridis]: AMP (Probabilities: 0.2261, 0.7617, 0.0122)\n","Processed WP_191260060.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis oliviviridis]: non-AMP/OTHERS (Probabilities: 0.8776, 0.1158, 0.0066)\n","Processed WP_191260059.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis oliviviridis]: AMP (Probabilities: 0.0942, 0.8931, 0.0127)\n","Processed HIP24411.1 mersacidin/lichenicidin family type 2 lantibiotic [Rhodobacteraceae bacterium]: AMP (Probabilities: 0.1124, 0.8620, 0.0256)\n","Processed WP_190186651.1 plantaricin C family lantibiotic [Streptomyces cirratus]: non-AMP/OTHERS (Probabilities: 0.9817, 0.0060, 0.0123)\n","Processed WP_189980714.1 plantaricin C family lantibiotic [Streptomyces termitum]: Neuropeptide (Probabilities: 0.0394, 0.0186, 0.9420)\n","Processed WP_188774481.1 gallidermin/nisin family lantibiotic [Paenibacillus hunanensis]: AMP (Probabilities: 0.0824, 0.9030, 0.0145)\n","Processed WP_189542286.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces gelaticus]: AMP (Probabilities: 0.0736, 0.9136, 0.0127)\n","Processed WP_189542284.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces gelaticus]: AMP (Probabilities: 0.1957, 0.7912, 0.0131)\n","Processed WP_189798675.1 plantaricin C family lantibiotic [Streptomyces thermodiastaticus]: Neuropeptide (Probabilities: 0.2723, 0.0304, 0.6973)\n","Processed WP_190143308.1 plantaricin C family lantibiotic [Streptomyces glebosus]: Neuropeptide (Probabilities: 0.3215, 0.0329, 0.6456)\n","Processed WP_188433588.1 mersacidin/lichenicidin family type 2 lantibiotic [Kroppenstedtia guangzhouensis]: AMP (Probabilities: 0.1898, 0.7336, 0.0766)\n","Processed WP_190108184.1 gallidermin/nisin family lantibiotic [Streptomyces cinnamoneus]: non-AMP/OTHERS (Probabilities: 0.8230, 0.0226, 0.1544)\n","Processed WP_188358711.1 mersacidin/lichenicidin family type 2 lantibiotic [Rothia aerolata]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0052, 0.0050)\n","Processed CAC8236107.1 lantibiotic ABC transporter protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0084, 0.0026)\n","Processed WP_188358712.1 mersacidin/lichenicidin family type 2 lantibiotic [Rothia aerolata]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0046, 0.0034)\n","Processed HAR7072755.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.8171, 0.0275, 0.1554)\n","Processed HAR7066494.1 lantibiotic dehydratase [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0040)\n","Processed EGQ1324581.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.4949, 0.0310, 0.4741)\n","Processed CAC6196759.1 lantibiotic ABC transporter protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0084, 0.0027)\n","Processed HAR7056195.1 lantibiotic ABC transporter ATP-binding protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0036)\n","Processed EGQ1361551.1 lantibiotic ABC transporter ATP-binding protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0040, 0.0035)\n","Processed HAR7887866.1 lantibiotic epidermin immunity protein F [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0053, 0.0035)\n","Processed WP_195445102.1 plantaricin C family lantibiotic [Peptoniphilus harei]: non-AMP/OTHERS (Probabilities: 0.5809, 0.1995, 0.2196)\n","Processed EGQ1459810.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9558, 0.0091, 0.0351)\n","Processed EGQ1341744.1 lantibiotic dehydratase [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed WP_000939671.1 hypothetical protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9715, 0.0254, 0.0031)\n","Processed EGQ1431956.1 lantibiotic dehydratase [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0031, 0.0035)\n","Processed WP_195320622.1 gallidermin/nisin family lantibiotic [Streptococcus salivarius]: AMP (Probabilities: 0.0644, 0.9195, 0.0161)\n","Processed HAR4920471.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.0105, 0.0101, 0.9794)\n","Processed EGQ1341567.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0078, 0.0036)\n","Processed HAR5004631.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.2924, 0.0377, 0.6698)\n","Processed HAR4569345.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9491, 0.0087, 0.0422)\n","Processed WP_195318323.1 lichenicidin A2 family type 2 lantibiotic [Weissella cibaria]: non-AMP/OTHERS (Probabilities: 0.9810, 0.0043, 0.0147)\n","Processed EGP5720347.1 lantibiotic transporter [Enterococcus faecium]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0035, 0.0033)\n","Processed EGP5668344.1 lantibiotic transporter [Enterococcus faecium]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0032)\n","Processed EGP5530375.1 lantibiotic transporter [Enterococcus faecium]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0038, 0.0032)\n","Processed EGP5497042.1 lantibiotic transporter [Enterococcus faecium]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0032)\n","Processed EGP5459425.1 lantibiotic ABC transporter [Enterococcus faecium]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0039, 0.0038)\n","Processed EGP5454989.1 lantibiotic transporter [Enterococcus faecium]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0036, 0.0035)\n","Processed WP_194858281.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus sp. AB056]: non-AMP/OTHERS (Probabilities: 0.8253, 0.0176, 0.1571)\n","Processed WP_193573556.1 gallidermin/nisin family lantibiotic [unclassified Staphylococcus]: non-AMP/OTHERS (Probabilities: 0.9689, 0.0072, 0.0239)\n","Processed WP_141591823.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Myxococcus]: non-AMP/OTHERS (Probabilities: 0.6867, 0.0425, 0.2708)\n","Processed EGP5536807.1 lantibiotic transporter [Enterococcus faecium]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0036)\n","Processed WP_175482729.1 gallidermin/nisin family lantibiotic [Actinokineospora iranica]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0060, 0.0028)\n","Processed WP_153507188.1 plantaricin C family lantibiotic [Streptomyces alkaliterrae]: Neuropeptide (Probabilities: 0.4060, 0.0333, 0.5607)\n","Processed WP_153582535.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0041, 0.0043)\n","Processed WP_153593855.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.7662, 0.0217, 0.2121)\n","Processed WP_153593375.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9857, 0.0071, 0.0072)\n","Processed WP_153582559.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9214, 0.0151, 0.0635)\n","Processed WP_153598230.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9411, 0.0138, 0.0451)\n","Processed EGO9466166.1 type 2 lantibiotic [Enterococcus faecalis]: AMP (Probabilities: 0.3286, 0.6589, 0.0125)\n","Processed EGO8197540.1 type 2 lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0047, 0.0041)\n","Processed EGO9043513.1 type 2 lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9687, 0.0052, 0.0261)\n","Processed WP_185203111.1 plantaricin C family lantibiotic [Streptomyces alkaliterrae]: Neuropeptide (Probabilities: 0.2722, 0.1035, 0.6243)\n","Processed MBF1702140.1 lantibiotic ABC transporter permease [Streptococcus sanguinis]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0081, 0.0028)\n","Processed MBF1196461.1 lantibiotic streptin [Fusobacterium periodonticum]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0063, 0.0025)\n","Processed MBF1197689.1 plantaricin C family lantibiotic [Fusobacterium periodonticum]: AMP (Probabilities: 0.0686, 0.9169, 0.0145)\n","Processed MBF1703013.1 lantibiotic ABC transporter permease [Streptococcus sanguinis]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0054, 0.0027)\n","Processed MBF2004360.1 mersacidin/lichenicidin family type 2 lantibiotic [Chlorogloeopsis fritschii C42_A2020_084]: non-AMP/OTHERS (Probabilities: 0.5177, 0.0514, 0.4309)\n","Processed MBF1701147.1 lantibiotic ABC transporter permease [Streptococcus sanguinis]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0064, 0.0026)\n","Processed MBE9543482.1 gallidermin family lantibiotic [Proteobacteria bacterium]: AMP (Probabilities: 0.0671, 0.9170, 0.0159)\n","Processed WP_194063098.1 mersacidin/lichenicidin family type 2 lantibiotic [Tychonema sp. LEGE 07203]: non-AMP/OTHERS (Probabilities: 0.9781, 0.0104, 0.0115)\n","Processed WP_169835568.1 mersacidin/lichenicidin family type 2 lantibiotic [Novibacillus thermophilus]: non-AMP/OTHERS (Probabilities: 0.9148, 0.0452, 0.0401)\n","Processed MBF1049211.1 plantaricin C family lantibiotic [Peptostreptococcus sp.]: non-AMP/OTHERS (Probabilities: 0.6876, 0.1634, 0.1491)\n","Processed MBF0983906.1 lantibiotic ABC transporter [Campylobacter sp.]: non-AMP/OTHERS (Probabilities: 0.9843, 0.0054, 0.0103)\n","Processed MBF0372606.1 lantibiotic ABC transporter [Alphaproteobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0044, 0.0036)\n","Processed WP_152788070.1 plantaricin C family lantibiotic [Streptomyces phyllanthi]: AMP (Probabilities: 0.0961, 0.8910, 0.0128)\n","Processed WP_193998722.1 mersacidin/lichenicidin family type 2 lantibiotic [Nodularia sp. LEGE 06071]: non-AMP/OTHERS (Probabilities: 0.9827, 0.0071, 0.0102)\n","Processed WP_077720412.1 mersacidin/lichenicidin family type 2 lantibiotic [Novibacillus thermophilus]: non-AMP/OTHERS (Probabilities: 0.9857, 0.0039, 0.0104)\n","Processed WP_082472091.1 gallidermin/nisin family lantibiotic [Paenibacillus bovis]: non-AMP/OTHERS (Probabilities: 0.5212, 0.4659, 0.0129)\n","Processed MBE6856935.1 plantaricin C family lantibiotic [Ruminococcus sp.]: AMP (Probabilities: 0.0689, 0.9176, 0.0135)\n","Processed MBE5981079.1 plantaricin C family lantibiotic [Paenibacillaceae bacterium]: AMP (Probabilities: 0.1544, 0.7929, 0.0527)\n","Processed MBE6051035.1 elgicin/penisin family lantibiotic [Clostridium sp.]: non-AMP/OTHERS (Probabilities: 0.6268, 0.3603, 0.0129)\n","Processed MBB1254255.1 plantaricin C family lantibiotic [Streptomyces alkaliterrae OF3]: Neuropeptide (Probabilities: 0.0315, 0.0194, 0.9491)\n","Processed MBE9166110.1 mersacidin/lichenicidin family type 2 lantibiotic [Pleurocapsales cyanobacterium LEGE 06147]: non-AMP/OTHERS (Probabilities: 0.8427, 0.0593, 0.0980)\n","Processed WP_193660179.1 plantaricin C family lantibiotic [Bacillus mojavensis]: Neuropeptide (Probabilities: 0.0104, 0.0102, 0.9794)\n","Processed MBE5974414.1 gallidermin/nisin family lantibiotic [Paenibacillaceae bacterium]: Neuropeptide (Probabilities: 0.0151, 0.0131, 0.9719)\n","Processed MBE6047911.1 plantaricin C family lantibiotic [Clostridium sp.]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0054, 0.0031)\n","Processed MBE6047909.1 lantibiotic cytolysin [Clostridium sp.]: AMP (Probabilities: 0.0667, 0.9196, 0.0137)\n","Processed MBE6051877.1 type 2 lantibiotic [Clostridium sp.]: non-AMP/OTHERS (Probabilities: 0.9863, 0.0056, 0.0081)\n","Processed WP_002635010.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus hansupus]: non-AMP/OTHERS (Probabilities: 0.6734, 0.0415, 0.2851)\n","Processed WP_050215293.1 lantibiotic streptin [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0051, 0.0035)\n","Processed WP_193573557.1 gallidermin/nisin family lantibiotic [Staphylococcus sp. SS21]: non-AMP/OTHERS (Probabilities: 0.8478, 0.0406, 0.1116)\n","Processed WP_002635009.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus hansupus]: Neuropeptide (Probabilities: 0.4085, 0.0270, 0.5645)\n","Processed WP_002635008.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus hansupus]: non-AMP/OTHERS (Probabilities: 0.5082, 0.0393, 0.4525)\n","Processed WP_193064520.1 plantaricin C family lantibiotic [Oceanobacillus oncorhynchi]: Neuropeptide (Probabilities: 0.0668, 0.0267, 0.9065)\n","Processed WP_078726534.1 gallidermin/nisin family lantibiotic, partial [Streptococcus pneumoniae]: AMP (Probabilities: 0.0774, 0.9093, 0.0133)\n","Processed MBE3552560.1 mersacidin/lichenicidin family type 2 lantibiotic [Kyrpidia tusciae]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0030, 0.0039)\n","Processed WP_001820554.1 hypothetical protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9670, 0.0262, 0.0068)\n","Processed WP_192945554.1 plantaricin C family lantibiotic [Staphylococcus cohnii]: AMP (Probabilities: 0.0853, 0.9022, 0.0125)\n","Processed WP_192844040.1 gallidermin/nisin family lantibiotic, partial [Staphylococcus argenteus]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0036, 0.0048)\n","Processed MBE3568791.1 plantaricin C family lantibiotic [Bacillales bacterium]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0033)\n","Processed MBE3566726.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermogemmatispora sp.]: non-AMP/OTHERS (Probabilities: 0.9862, 0.0096, 0.0042)\n","Processed WP_151842906.1 plantaricin C family lantibiotic [unclassified Corynebacterium]: non-AMP/OTHERS (Probabilities: 0.5931, 0.2946, 0.1123)\n","Processed WP_002636956.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus hansupus]: non-AMP/OTHERS (Probabilities: 0.7278, 0.0823, 0.1899)\n","Processed WP_151587959.1 plantaricin C family lantibiotic [Priestia endophytica]: non-AMP/OTHERS (Probabilities: 0.9893, 0.0072, 0.0035)\n","Processed WP_151628943.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9887, 0.0073, 0.0040)\n","Processed WP_151614196.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus thuringiensis]: AMP (Probabilities: 0.0691, 0.9166, 0.0142)\n","Processed WP_061666163.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus sp. RM2(2019)]: non-AMP/OTHERS (Probabilities: 0.9836, 0.0121, 0.0043)\n","Processed PNZ50841.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9698, 0.0137, 0.0166)\n","Processed MBD1823041.1 mersacidin/lichenicidin family type 2 lantibiotic [Cyanobacteria bacterium FACHB-DQ100]: non-AMP/OTHERS (Probabilities: 0.7315, 0.1546, 0.1139)\n","Processed WP_192045167.1 hypothetical protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9876, 0.0085, 0.0038)\n","Processed WP_107164698.1 lantipeptide cytolysin subunit CylL-L [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9807, 0.0159, 0.0033)\n","Processed WP_191239781.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus schweitzeri]: non-AMP/OTHERS (Probabilities: 0.9579, 0.0382, 0.0039)\n","Processed MBD9038876.1 lantibiotic ABC transporter ATP-binding protein [Collinsella aerofaciens]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0040, 0.0046)\n","Processed WP_150854208.1 gallidermin/nisin family lantibiotic, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.7553, 0.1041, 0.1406)\n","Processed MBD6608108.1 hypothetical protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0074, 0.0029)\n","Processed WP_052996903.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.8975, 0.0126, 0.0899)\n","Processed MBD8063585.1 plantaricin C family lantibiotic [Actinomycetaceae bacterium Sa1BUA1]: AMP (Probabilities: 0.0666, 0.9191, 0.0143)\n","Processed WP_191141855.1 plantaricin C family lantibiotic [Hazenella sp. IB182357]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0051, 0.0029)\n","Processed WP_053000964.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.8073, 0.0165, 0.1763)\n","Processed WP_047555146.1 lichenicidin A2 family type 2 lantibiotic [Staphylococcus schweitzeri]: non-AMP/OTHERS (Probabilities: 0.9936, 0.0035, 0.0030)\n","Processed WP_191664142.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0040, 0.0035)\n","Processed WP_150404369.1 gallidermin/nisin family lantibiotic [Nocardia colli]: non-AMP/OTHERS (Probabilities: 0.8163, 0.1779, 0.0058)\n","Processed WP_150224109.1 plantaricin C family lantibiotic [Bacillus atrophaeus]: Neuropeptide (Probabilities: 0.0250, 0.0145, 0.9605)\n","Processed WP_150087159.1 mersacidin/lichenicidin family type 2 lantibiotic [Adhaeribacter rhizoryzae]: Neuropeptide (Probabilities: 0.3241, 0.0461, 0.6299)\n","Processed MBD7742756.1 gallidermin family lantibiotic [Klebsiella pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0037, 0.0042)\n","Processed WP_015792328.1 thiazolylpeptide-type bacteriocin [Catenulispora acidiphila]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0040, 0.0035)\n","Processed WP_031864692.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.0263, 0.0140, 0.9597)\n","Processed WP_116327270.1 gallidermin/nisin family lantibiotic, partial [Bacillus sp. ALD]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0041, 0.0027)\n","Processed WP_148959126.1 plantaricin C family lantibiotic [Bacillus swezeyi]: Neuropeptide (Probabilities: 0.0274, 0.0159, 0.9567)\n","Processed WP_067608779.1 lantibiotic protein [Nocardiopsis listeri]: Neuropeptide (Probabilities: 0.4438, 0.0357, 0.5205)\n","Processed WP_015792793.1 thiazolylpeptide-type bacteriocin [Catenulispora acidiphila]: non-AMP/OTHERS (Probabilities: 0.9712, 0.0157, 0.0131)\n","Processed WP_116327388.1 gallidermin/nisin family lantibiotic, partial [Bacillus sp. ALD]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0035, 0.0040)\n","Processed WP_113710559.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0078, 0.0034)\n","Processed WP_113710558.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9794, 0.0145, 0.0061)\n","Processed WP_108530494.1 elgicin/penisin family lantibiotic [Paenibacillus elgii]: AMP (Probabilities: 0.0808, 0.8983, 0.0209)\n","Processed WP_087115300.1 lichenicidin A2 family type 2 lantibiotic [Carnobacterium maltaromaticum]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0032, 0.0036)\n","Processed WP_155999353.1 plantaricin C family lantibiotic [Streptomonospora sp. PA3]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0085, 0.0025)\n","Processed WP_156002255.1 plantaricin C family lantibiotic [Streptomonospora sp. PA3]: AMP (Probabilities: 0.2536, 0.6856, 0.0608)\n","Processed GGO47189.1 hypothetical protein [Streptomyces daqingensis]: AMP (Probabilities: 0.1019, 0.8825, 0.0156)\n","Processed WP_188281077.1 FxLD family lanthipeptide [Streptomyces sp. CBMA29]: non-AMP/OTHERS (Probabilities: 0.9346, 0.0610, 0.0044)\n","Processed MBD0343996.1 mersacidin/lichenicidin family type 2 lantibiotic [Coleofasciculus sp. Co-bin14]: non-AMP/OTHERS (Probabilities: 0.7394, 0.1865, 0.0741)\n","Processed AHF21242.1 CrnA2 [Carnobacterium maltaromaticum]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0034)\n","Processed WP_149561016.1 lantibiotic salivaricin M precursor [Streptococcus salivarius]: AMP (Probabilities: 0.0683, 0.9183, 0.0134)\n","Processed WP_149561017.1 lantibiotic salivaricin M precursor [Streptococcus salivarius]: AMP (Probabilities: 0.0661, 0.9192, 0.0148)\n","Processed WP_108530493.1 elgicin/penisin family lantibiotic [Paenibacillus elgii]: AMP (Probabilities: 0.0873, 0.8999, 0.0128)\n","Processed MBC9006131.1 lantibiotic modifying enzyme [Micromonospora aurantiaca]: non-AMP/OTHERS (Probabilities: 0.9881, 0.0065, 0.0054)\n","Processed WP_064511516.1 plantaricin C family lantibiotic [Lactiplantibacillus plantarum]: non-AMP/OTHERS (Probabilities: 0.9635, 0.0151, 0.0214)\n","Processed WP_136738428.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces piniterrae]: non-AMP/OTHERS (Probabilities: 0.8382, 0.0198, 0.1420)\n","Processed WP_148767233.1 cinnamycin family lantibiotic [Actinomadura decatromicini]: non-AMP/OTHERS (Probabilities: 0.9741, 0.0158, 0.0101)\n","Processed WP_148855137.1 lantibiotic streptin [Streptococcus pyogenes]: AMP (Probabilities: 0.0702, 0.9167, 0.0131)\n","Processed WP_119217143.1 plantaricin C family lantibiotic [Dorea formicigenerans]: AMP (Probabilities: 0.1255, 0.8477, 0.0268)\n","Processed WP_187399537.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora sp. AP08]: non-AMP/OTHERS (Probabilities: 0.9327, 0.0291, 0.0382)\n","Processed WP_163509955.1 mersacidin/lichenicidin family type 2 lantibiotic [Fodinicola acaciae]: Neuropeptide (Probabilities: 0.1060, 0.0234, 0.8706)\n","Processed WP_148349799.1 cinnamycin family lantibiotic [Actinomadura syzygii]: non-AMP/OTHERS (Probabilities: 0.9671, 0.0157, 0.0173)\n","Processed WP_187354179.1 hypothetical protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0050, 0.0027)\n","Processed WP_187159389.1 gallidermin/nisin family lantibiotic [Lactobacillus helsingborgensis]: AMP (Probabilities: 0.0749, 0.9126, 0.0125)\n","Processed WP_148251489.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.0093, 0.0095, 0.9811)\n","Processed WP_147126017.1 mersacidin/lichenicidin family type 2 lantibiotic [Shimia sp. WX04]: non-AMP/OTHERS (Probabilities: 0.6139, 0.3709, 0.0152)\n","Processed MBC6480600.1 cinnamycin family lantibiotic [Hormoscilla sp. GM7CHS1pb]: non-AMP/OTHERS (Probabilities: 0.7860, 0.0875, 0.1264)\n","Processed MBB1572298.1 plantaricin C family lantibiotic [Propionibacterium sp.]: Neuropeptide (Probabilities: 0.0177, 0.0148, 0.9675)\n","Processed WP_097996468.1 plantaricin C family lantibiotic [Bacillus tropicus]: non-AMP/OTHERS (Probabilities: 0.9880, 0.0089, 0.0031)\n","Processed VED92238.1 lantibiotic transport ATP-binding protein [Streptococcus equinus]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0056, 0.0028)\n","Processed KAF6614952.1 lantibiotic biosynthesis protein, partial [Paenibacillus sp. EKM208P]: non-AMP/OTHERS (Probabilities: 0.9677, 0.0057, 0.0266)\n","Processed CIO18263.1 lantibiotic modifying enzyme [Streptococcus pseudopneumoniae]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0034)\n","Processed KAF6606748.1 lantibiotic biosynthesis protein, partial [Paenibacillus sp. EKM208P]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0052, 0.0040)\n","Processed VTS12533.1 lantibiotic salivaricin A [Streptococcus pseudoporcinus]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0032, 0.0040)\n","Processed CKF87098.1 lantibiotic ABC transporter ATP-binding protein [Bacillus paranthracis]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0042, 0.0033)\n","Processed CAC7004492.1 ABC transporter ATP-binding protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0037, 0.0037)\n","Processed WP_037593174.1 lantibiotic streptin [Streptococcus anginosus]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0056, 0.0026)\n","Processed VED86690.1 putative lantibiotic modifying protein [Streptococcus vestibularis]: non-AMP/OTHERS (Probabilities: 0.9866, 0.0065, 0.0069)\n","Processed EFX7079126.1 lantibiotic ABC transporter ATP-binding protein [Shigella flexneri]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0063, 0.0048)\n","Processed MBA4106396.1 mersacidin/lichenicidin family type 2 lantibiotic [Pirellula sp.]: non-AMP/OTHERS (Probabilities: 0.9839, 0.0052, 0.0108)\n","Processed MBA3711874.1 mersacidin/lichenicidin family type 2 lantibiotic [Pyrinomonadaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.6819, 0.0580, 0.2601)\n","Processed HGZ87768.1 mersacidin/lichenicidin family type 2 lantibiotic [Gloeobacter violaceus SpSt-379]: Neuropeptide (Probabilities: 0.0095, 0.0095, 0.9810)\n","Processed EFZ2507501.1 lantibiotic ABC transporter ATP-binding protein [Shigella flexneri]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0061, 0.0050)\n","Processed MBA2766242.1 mersacidin/lichenicidin family type 2 lantibiotic [Solirubrobacterales bacterium]: AMP (Probabilities: 0.1584, 0.8280, 0.0137)\n","Processed HGZ85506.1 mersacidin/lichenicidin family type 2 lantibiotic [Gloeobacter violaceus SpSt-379]: Neuropeptide (Probabilities: 0.0148, 0.0116, 0.9736)\n","Processed HGZ87767.1 mersacidin/lichenicidin family type 2 lantibiotic [Gloeobacter violaceus SpSt-379]: Neuropeptide (Probabilities: 0.1768, 0.0274, 0.7958)\n","Processed QLH35163.1 gallidermin/nisin family lantibiotic [Parachlamydiaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0048, 0.0036)\n","Processed CAD0120928.1 protein of unknown function [Streptococcus thermophilus]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0034, 0.0032)\n","Processed EFX95424.1 hypothetical protein [Streptococcus vestibularis ATCC 49124 ATCC 49124]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0054, 0.0032)\n","Processed NUR58374.1 mersacidin/lichenicidin family type 2 lantibiotic [Catenulispora sp.]: Neuropeptide (Probabilities: 0.0219, 0.0143, 0.9638)\n","Processed NUT47504.1 mersacidin/lichenicidin family type 2 lantibiotic [Saccharothrix sp.]: non-AMP/OTHERS (Probabilities: 0.9706, 0.0060, 0.0234)\n","Processed NUT47916.1 lantibiotic dehydratase [Saccharothrix sp.]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0035, 0.0050)\n","Processed NUT30257.1 gallidermin/nisin family lantibiotic [Streptomyces sp.]: non-AMP/OTHERS (Probabilities: 0.9802, 0.0167, 0.0031)\n","Processed NUU25491.1 lantibiotic modifying enzyme [Streptomycetaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9659, 0.0074, 0.0266)\n","Processed P85065.1 Lantibiotic 107891 [Microbispora sp. 107891]: AMP (Probabilities: 0.2794, 0.7066, 0.0140)\n","Processed P36503.1 Lantibiotic duramycin C [Streptomyces griseoluteus]: non-AMP/OTHERS (Probabilities: 0.7907, 0.2032, 0.0061)\n","Processed Q2QBT0.1 Lantibiotic nisin-U [Streptococcus uberis]: non-AMP/OTHERS (Probabilities: 0.7318, 0.2325, 0.0357)\n","Processed H2A7G5.1 Lantibiotic macedovicin [Streptococcus macedonicus ACA-DC 198]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0041, 0.0058)\n","Processed P36504.1 Lantibiotic duramycin [Streptomyces griseoverticillatus]: AMP (Probabilities: 0.0791, 0.9076, 0.0133)\n","Processed NVJ03721.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus sp. AM001]: Neuropeptide (Probabilities: 0.0087, 0.0096, 0.9817)\n","Processed P36502.1 Lantibiotic duramycin B [Streptomyces sp. (strain R2075)]: AMP (Probabilities: 0.1797, 0.7990, 0.0213)\n","Processed P38655.1 Lantibiotic ancovenin [Streptomyces sp. A647P-2]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0036, 0.0041)\n","Processed CAC6075160.1 type 2 lantibiotic, SP_1948 family [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0052, 0.0034)\n","Processed NSQ70741.1 plantaricin C family lantibiotic [Enterococcus faecalis]: Neuropeptide (Probabilities: 0.2621, 0.0518, 0.6861)\n","Processed NSF27194.1 lantibiotic ABC transporter permease [Blautia wexlerae]: non-AMP/OTHERS (Probabilities: 0.9894, 0.0078, 0.0028)\n","Processed NSI60225.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [[Ruminococcus] gnavus]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0043, 0.0034)\n","Processed NRT74869.1 type 2 lantibiotic (TIGR03893 family) [Clostridium beijerinckii]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0050, 0.0042)\n","Processed NRG12807.1 lantibiotic ABC transporter ATP-binding protein [Bacillus velezensis]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0061, 0.0030)\n","Processed NRD97783.1 type 2 lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.5601, 0.4264, 0.0136)\n","Processed NRD95109.1 type 2 lantibiotic [Enterococcus faecalis]: non-AMP/OTHERS (Probabilities: 0.9219, 0.0069, 0.0712)\n","Processed NRD96455.1 type 2 lantibiotic [Enterococcus faecalis]: Neuropeptide (Probabilities: 0.0097, 0.0097, 0.9806)\n","Processed NNB84144.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus exiguus]: Neuropeptide (Probabilities: 0.3012, 0.0448, 0.6541)\n","Processed NQY37536.1 mersacidin/lichenicidin family type 2 lantibiotic [Alteromonadaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.6562, 0.0238, 0.3200)\n","Processed NQM73659.1 gallidermin/nisin family lantibiotic [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9858, 0.0040, 0.0102)\n","Processed NQJ22897.1 plantaricin C family lantibiotic [Streptococcus suis]: AMP (Probabilities: 0.4264, 0.5604, 0.0131)\n","Processed NQK80980.1 lantibiotic salivaricin M precursor [Streptococcus suis]: AMP (Probabilities: 0.0739, 0.9136, 0.0125)\n","Processed NQJ60634.1 plantaricin C family lantibiotic [Streptococcus suis]: Neuropeptide (Probabilities: 0.1095, 0.0489, 0.8416)\n","Processed NPC70299.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus exiguus]: Neuropeptide (Probabilities: 0.2042, 0.0382, 0.7576)\n","Processed NQN67325.1 gallidermin/nisin family lantibiotic [Streptococcus suis]: AMP (Probabilities: 0.0706, 0.9153, 0.0141)\n","Processed NQK96712.1 lantibiotic (srt) production protein [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0047, 0.0067)\n","Processed NQK65237.1 lantibiotic salivaricin M precursor [Streptococcus suis]: AMP (Probabilities: 0.1426, 0.8184, 0.0390)\n","Processed NQK80981.1 lantibiotic salivaricin M precursor [Streptococcus suis]: Neuropeptide (Probabilities: 0.2967, 0.0905, 0.6129)\n","Processed NQF18095.1 lantibiotic ABC transporter ATP-binding protein [Enterobacter hormaechei]: non-AMP/OTHERS (Probabilities: 0.9905, 0.0047, 0.0048)\n","Processed EGS83774.1 hypothetical protein [Staphylococcus aureus subsp. aureus 21235 21235]: non-AMP/OTHERS (Probabilities: 0.9858, 0.0117, 0.0024)\n","Processed NOK60968.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium AL-N1]: non-AMP/OTHERS (Probabilities: 0.5890, 0.0498, 0.3611)\n","Processed NOJ78701.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: Neuropeptide (Probabilities: 0.0926, 0.0271, 0.8803)\n","Processed NOR64254.1 mersacidin/lichenicidin family type 2 lantibiotic [Rhodobacteraceae bacterium]: AMP (Probabilities: 0.1069, 0.8732, 0.0199)\n","Processed NOJ94756.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus coralloides]: Neuropeptide (Probabilities: 0.2809, 0.0630, 0.6561)\n","Processed NNV03594.1 mersacidin/lichenicidin family type 2 lantibiotic [Brevibacillus sp. MCWH]: non-AMP/OTHERS (Probabilities: 0.6816, 0.0260, 0.2924)\n","Processed EFN5149602.1 lantibiotic ABC transporter ATP-binding protein [Escherichia coli]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0051, 0.0041)\n","Processed NMN47917.1 lantibiotic modifying enzyme [Chroococcidiopsis sp. PCC 6712]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0038)\n","Processed NMG98238.1 lantibiotic transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0052, 0.0033)\n","Processed NMH19107.1 lantibiotic transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9821, 0.0152, 0.0027)\n","Processed ABF87463.1 hypothetical protein [Myxococcus xanthus DK 1622 DK 1622]: Neuropeptide (Probabilities: 0.0922, 0.0258, 0.8819)\n","Processed NMH09352.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0049, 0.0030)\n","Processed NMG63225.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0049, 0.0027)\n","Processed HHX60578.1 lantibiotic salivaricin M precursor [Epulopiscium sp.]: AMP (Probabilities: 0.0738, 0.9136, 0.0127)\n","Processed NMB87221.1 lantibiotic ABC transporter [Chloroflexi bacterium]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0045, 0.0046)\n","Processed AUO14642.1 type 2 lantibiotic [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0071, 0.0032)\n","Processed HHW22955.1 plantaricin C family lantibiotic [Clostridiaceae bacterium]: AMP (Probabilities: 0.1265, 0.8570, 0.0165)\n","Processed NLQ33332.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9866, 0.0105, 0.0029)\n","Processed NJK73197.1 mersacidin/lichenicidin family type 2 lantibiotic [Microcoleus sp. SU_5_6]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0043, 0.0072)\n","Processed NMG62781.1 lantibiotic transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9846, 0.0124, 0.0029)\n","Processed NJR66509.1 mersacidin/lichenicidin family type 2 lantibiotic [Leptolyngbyaceae cyanobacterium CRU_2_3]: non-AMP/OTHERS (Probabilities: 0.9131, 0.0797, 0.0072)\n","Processed NJN09340.1 mersacidin/lichenicidin family type 2 lantibiotic [Richelia sp. RM1_1_1]: non-AMP/OTHERS (Probabilities: 0.5262, 0.0764, 0.3974)\n","Processed NJK69080.1 mersacidin/lichenicidin family type 2 lantibiotic [Microcoleus sp. SU_5_3]: non-AMP/OTHERS (Probabilities: 0.9825, 0.0087, 0.0087)\n","Processed NJK52945.1 mersacidin/lichenicidin family type 2 lantibiotic [Leptolyngbyaceae cyanobacterium SU_3_3]: non-AMP/OTHERS (Probabilities: 0.7362, 0.0322, 0.2317)\n","Processed NJK52943.1 mersacidin/lichenicidin family type 2 lantibiotic [Leptolyngbyaceae cyanobacterium SU_3_3]: Neuropeptide (Probabilities: 0.0429, 0.0219, 0.9352)\n","Processed NJK52942.1 mersacidin/lichenicidin family type 2 lantibiotic [Leptolyngbyaceae cyanobacterium SU_3_3]: non-AMP/OTHERS (Probabilities: 0.9893, 0.0048, 0.0059)\n","Processed NJL57877.1 mersacidin/lichenicidin family type 2 lantibiotic [bacterium]: AMP (Probabilities: 0.1183, 0.8388, 0.0429)\n","Processed MNY21730.1 Lantibiotic lichenicidin VK21 A1 precursor [compost metagenome]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0049, 0.0042)\n","Processed MNL72545.1 Lantibiotic lichenicidin VK21 A2 precursor [compost metagenome]: non-AMP/OTHERS (Probabilities: 0.9795, 0.0181, 0.0024)\n","Processed NJK52944.1 mersacidin/lichenicidin family type 2 lantibiotic [Leptolyngbyaceae cyanobacterium SU_3_3]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0043, 0.0042)\n","Processed HFC11660.1 mersacidin/lichenicidin family type 2 lantibiotic [Anaerolineae bacterium]: non-AMP/OTHERS (Probabilities: 0.6813, 0.1677, 0.1510)\n","Processed NGW97582.1 lantibiotic dehydratase [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0046, 0.0028)\n","Processed NGQ35581.1 lantibiotic dehydratase [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0060, 0.0027)\n","Processed HHS01949.1 type 2 lantibiotic [Caldicellulosiruptor owensensis]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed HGB38655.1 mersacidin/lichenicidin family type 2 lantibiotic [Candidatus Hydrogenedentes bacterium]: non-AMP/OTHERS (Probabilities: 0.6262, 0.0293, 0.3445)\n","Processed HFC00693.1 lantibiotic ABC transporter [Phaeodactylibacter sp.]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0048, 0.0032)\n","Processed HEC78528.1 mersacidin/lichenicidin family type 2 lantibiotic [candidate division WOR-3 bacterium]: Neuropeptide (Probabilities: 0.0944, 0.0332, 0.8724)\n","Processed NFA60701.1 plantaricin C family lantibiotic [Clostridium botulinum]: AMP (Probabilities: 0.3053, 0.4837, 0.2110)\n","Processed NFY75456.1 lantibiotic dehydratase [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0037)\n","Processed NGG26416.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9896, 0.0077, 0.0027)\n","Processed NEQ09357.1 cinnamycin family lantibiotic [Moorea sp. SIO4E2]: non-AMP/OTHERS (Probabilities: 0.8275, 0.0720, 0.1005)\n","Processed NEO39949.1 cinnamycin family lantibiotic [Moorea sp. SIOASIH]: non-AMP/OTHERS (Probabilities: 0.7055, 0.2279, 0.0667)\n","Processed NEO27782.1 mersacidin/lichenicidin family type 2 lantibiotic [Kamptonema sp. SIO4C4]: non-AMP/OTHERS (Probabilities: 0.9723, 0.0127, 0.0150)\n","Processed NEP35942.1 cinnamycin family lantibiotic [Moorea sp. SIO3B2]: non-AMP/OTHERS (Probabilities: 0.5871, 0.3572, 0.0557)\n","Processed NEZ76783.1 gallidermin/nisin family lantibiotic [Clostridium botulinum]: non-AMP/OTHERS (Probabilities: 0.9040, 0.0856, 0.0105)\n","Processed NEP08410.1 mersacidin/lichenicidin family type 2 lantibiotic [Okeania sp. SIO4D6]: non-AMP/OTHERS (Probabilities: 0.8438, 0.0129, 0.1434)\n","Processed NEO97085.1 cinnamycin family lantibiotic [Moorea sp. SIO3G5]: non-AMP/OTHERS (Probabilities: 0.9377, 0.0314, 0.0309)\n","Processed NER91960.1 cinnamycin family lantibiotic [Moorea sp. SIO3A2]: non-AMP/OTHERS (Probabilities: 0.8520, 0.0670, 0.0810)\n","Processed NEP60518.1 mersacidin/lichenicidin family type 2 lantibiotic [Symploca sp. SIO2G7]: non-AMP/OTHERS (Probabilities: 0.9675, 0.0084, 0.0241)\n","Processed NEQ21862.1 mersacidin/lichenicidin family type 2 lantibiotic [Microcoleus sp. SIO2G3]: non-AMP/OTHERS (Probabilities: 0.6121, 0.0233, 0.3646)\n","Processed NES02212.1 mersacidin/lichenicidin family type 2 lantibiotic [Okeania sp. SIO2F4]: non-AMP/OTHERS (Probabilities: 0.7524, 0.0165, 0.2310)\n","Processed NES95844.1 mersacidin/lichenicidin family type 2 lantibiotic [Desertifilum sp. SIO1I2]: non-AMP/OTHERS (Probabilities: 0.9829, 0.0061, 0.0110)\n","Processed NET01229.1 mersacidin/lichenicidin family type 2 lantibiotic [Sphaerospermopsis sp. SIO1G1]: Neuropeptide (Probabilities: 0.0308, 0.0166, 0.9526)\n","Processed NED78112.1 lantibiotic dehydratase [Streptomyces sp. SID9944]: non-AMP/OTHERS (Probabilities: 0.6824, 0.2068, 0.1107)\n","Processed NEE60560.1 lantibiotic biosynthesis protein [Streptomyces sp. SID8455]: non-AMP/OTHERS (Probabilities: 0.9510, 0.0121, 0.0369)\n","Processed NED84812.1 lantibiotic dehydratase [Streptomyces sp. SID11233]: non-AMP/OTHERS (Probabilities: 0.9116, 0.0113, 0.0770)\n","Processed NEA17398.1 lantibiotic dehydratase [Streptomyces halstedii]: non-AMP/OTHERS (Probabilities: 0.9822, 0.0046, 0.0132)\n","Processed AVF28182.1 putative lantibiotic biosynthesis protein [Paenibacillus larvae subsp. larvae Eric_III]: non-AMP/OTHERS (Probabilities: 0.9875, 0.0098, 0.0027)\n","Processed NEE61497.1 lantibiotic biosynthesis protein [Streptomyces sp. SID8455]: Neuropeptide (Probabilities: 0.0316, 0.0216, 0.9468)\n","Processed NED34005.1 lantibiotic dehydratase [Streptomyces sp. SID8499]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0044, 0.0034)\n","Processed AQR76285.1 hypothetical protein [Paenibacillus larvae subsp. larvae ATCC 9545]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0060, 0.0032)\n","Processed NDQ55574.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidipila sp.]: AMP (Probabilities: 0.1139, 0.8739, 0.0122)\n","Processed MZI14701.1 lantibiotic ABC transporter permease [Collinsella aerofaciens]: non-AMP/OTHERS (Probabilities: 0.9881, 0.0084, 0.0034)\n","Processed MZJ61899.1 lantibiotic ABC transporter permease [Collinsella aerofaciens]: non-AMP/OTHERS (Probabilities: 0.9841, 0.0124, 0.0035)\n","Processed MZJ67395.1 lantibiotic ABC transporter permease [Collinsella aerofaciens]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0051, 0.0034)\n","Processed AQP95668.1 hypothetical protein [Bacillus sp. 275]: Neuropeptide (Probabilities: 0.4190, 0.0481, 0.5329)\n","Processed AFQ13329.1 lantibiotic cytolysin [Bacillus cereus FRI-35 FRI-35]: AMP (Probabilities: 0.1574, 0.8175, 0.0252)\n","Processed MYR90262.1 lantibiotic dehydratase [Streptomyces sp. SID685]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0043, 0.0036)\n","Processed MYS41035.1 lantibiotic dehydratase [Streptomyces sp. SID5998]: non-AMP/OTHERS (Probabilities: 0.7054, 0.0589, 0.2357)\n","Processed MYT34797.1 lantibiotic dehydratase [Streptomyces sp. SID8356]: non-AMP/OTHERS (Probabilities: 0.6175, 0.0366, 0.3459)\n","Processed KAF0802962.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit, partial [Pediococcus acidilactici]: non-AMP/OTHERS (Probabilities: 0.9859, 0.0101, 0.0039)\n","Processed MXO81256.1 lantibiotic ABC transporter ATP-binding protein [Paenibacillus sp. OT2-17]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0042, 0.0039)\n","Processed MTV80896.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0053, 0.0028)\n","Processed MYR62274.1 lantibiotic biosynthesis protein [Streptomyces sp. SID625]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0049, 0.0035)\n","Processed MTV97253.1 lantibiotic transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9839, 0.0134, 0.0027)\n","Processed MTV72341.1 lantibiotic transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9882, 0.0093, 0.0025)\n","Processed MTV72360.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0060, 0.0026)\n","Processed MTV85678.1 lantibiotic (srt) production protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0039, 0.0040)\n","Processed MTV61731.1 plantaricin C family lantibiotic [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9845, 0.0042, 0.0113)\n","Processed MTV72209.1 lantibiotic (srt) production protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9749, 0.0217, 0.0034)\n","Processed MTV91566.1 lantibiotic ABC transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0060, 0.0032)\n","Processed MTW00285.1 lantibiotic ABC transporter permease [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9815, 0.0155, 0.0031)\n","Processed MTW38257.1 lantibiotic ABC transporter permease [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0052, 0.0031)\n","Processed MTV61797.1 lantibiotic ABC transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0067, 0.0042)\n","Processed MTW38134.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9838, 0.0134, 0.0028)\n","Processed MTV85739.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9832, 0.0138, 0.0030)\n","Processed MTW25944.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0052, 0.0059)\n","Processed MTV67254.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0044, 0.0028)\n","Processed MTV94424.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0066, 0.0026)\n","Processed MTW38258.1 lantibiotic ABC transporter permease [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9882, 0.0092, 0.0026)\n","Processed MTW25875.1 lantibiotic transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9868, 0.0106, 0.0026)\n","Processed MTV58710.1 lantibiotic transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9823, 0.0150, 0.0027)\n","Processed MTV94376.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9820, 0.0136, 0.0044)\n","Processed MTV94312.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0039, 0.0036)\n","Processed MTW28398.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0059, 0.0025)\n","Processed MTW31162.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0069, 0.0027)\n","Processed MTW31091.1 lantibiotic (srt) production protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0040, 0.0037)\n","Processed MTW28137.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9694, 0.0113, 0.0193)\n","Processed MTV59180.1 lantibiotic transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0051, 0.0027)\n","Processed MTW31096.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0068, 0.0031)\n","Processed MTW28534.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0056, 0.0027)\n","Processed MTV74609.1 lantibiotic (srt) production protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0033)\n","Processed MTW31042.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0070, 0.0026)\n","Processed MTV64681.1 lantibiotic (srt) production protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0055, 0.0036)\n","Processed MTV64511.1 lantibiotic ABC transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0054, 0.0031)\n","Processed MTV35705.1 type 2 lantibiotic [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0039, 0.0066)\n","Processed MTW08771.1 lantibiotic transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0071, 0.0026)\n","Processed MTV35953.1 lantibiotic transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0043, 0.0027)\n","Processed MTK09987.1 gallidermin/nisin family lantibiotic [Hungatella sp.]: Neuropeptide (Probabilities: 0.0125, 0.0120, 0.9755)\n","Processed MRC88178.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0987, 0.8827, 0.0186)\n","Processed MTV64418.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9885, 0.0088, 0.0027)\n","Processed WP_096385066.1 mersacidin/lichenicidin family type 2 lantibiotic [Stanieria sp. NIES-3757]: AMP (Probabilities: 0.1833, 0.8017, 0.0151)\n","Processed KAB2340034.1 lantibiotic protein [Actinomadura rudentiformis]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0064, 0.0046)\n","Processed WP_097996467.1 plantaricin C family lantibiotic [Bacillus tropicus]: non-AMP/OTHERS (Probabilities: 0.9835, 0.0139, 0.0026)\n","Processed WP_151190484.1 NisI/SpaI family lantibiotic immunity lipoprotein [Blautia obeum]: non-AMP/OTHERS (Probabilities: 0.9005, 0.0947, 0.0048)\n","Processed WP_096385069.1 mersacidin/lichenicidin family type 2 lantibiotic [Stanieria sp. NIES-3757]: non-AMP/OTHERS (Probabilities: 0.9769, 0.0089, 0.0142)\n","Processed WP_096385068.1 mersacidin/lichenicidin family type 2 lantibiotic [Stanieria sp. NIES-3757]: AMP (Probabilities: 0.1791, 0.8071, 0.0138)\n","Processed KAB2364261.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: Neuropeptide (Probabilities: 0.0128, 0.0119, 0.9753)\n","Processed KAB1635969.1 lantibiotic ABC transporter permease, partial [Ellagibacter isourolithinifaciens]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0053, 0.0037)\n","Processed MRB12070.1 plantaricin C family lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.0695, 0.9170, 0.0134)\n","Processed CJI07061.1 lantibiotic ABC transporter ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0036, 0.0040)\n","Processed KAA0805645.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9846, 0.0117, 0.0037)\n","Processed TYK79856.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit, partial [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0062, 0.0024)\n","Processed TYL03908.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit, partial [Streptococcus dysgalactiae]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0075, 0.0026)\n","Processed ARC21856.1 lantibiotic salivaricin M precursor [Streptococcus sp. FDAARGOS_192]: AMP (Probabilities: 0.1202, 0.8664, 0.0134)\n","Processed KAA1011510.1 gallidermin/nisin family lantibiotic [Pseudonocardia sp. EV170527-09]: non-AMP/OTHERS (Probabilities: 0.8538, 0.1279, 0.0183)\n","Processed TXO09174.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0068, 0.0026)\n","Processed TXK91598.1 type 2 lantibiotic [Parageobacillus sp. SY1]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0037, 0.0040)\n","Processed ASF13193.1 gallidermin/nisin family lantibiotic [Nocardia brasiliensis]: non-AMP/OTHERS (Probabilities: 0.5726, 0.2138, 0.2135)\n","Processed KAA9180841.1 plantaricin C family lantibiotic [Enterococcus durans]: Neuropeptide (Probabilities: 0.0508, 0.0185, 0.9307)\n","Processed TXO24278.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0076, 0.0029)\n","Processed TXN89611.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0033)\n","Processed PDM39144.1 type 2 lantibiotic [Parageobacillus yumthangensis]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0033)\n","Processed EPZ46713.1 mutacin ABC transporter protein, MutE domain protein [Streptococcus pyogenes GA41345 GA41345]: non-AMP/OTHERS (Probabilities: 0.9905, 0.0069, 0.0025)\n","Processed GEN97028.1 hypothetical protein [Streptococcus cristatus]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0060, 0.0032)\n","Processed TXO71123.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0063, 0.0030)\n","Processed TVW95185.1 lantibiotic salivaricin M precursor, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0035, 0.0032)\n","Processed TVW83991.1 lantibiotic biosynthesis protein, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0034, 0.0033)\n","Processed GEN97027.1 hypothetical protein [Streptococcus cristatus]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0041, 0.0034)\n","Processed TVX28780.1 lantibiotic salivaricin M precursor, partial [Streptococcus pneumoniae]: AMP (Probabilities: 0.0872, 0.8998, 0.0129)\n","Processed TVW94359.1 lantibiotic salivaricin M precursor [Streptococcus pneumoniae]: AMP (Probabilities: 0.0679, 0.9179, 0.0142)\n","Processed TVW79802.1 lantibiotic ABC transporter, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0071, 0.0032)\n","Processed TVV96755.1 lantibiotic transporter, partial [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0088, 0.0027)\n","Processed TVQ58099.1 mersacidin/lichenicidin family type 2 lantibiotic [Spirulina sp. DLM2.Bin59]: non-AMP/OTHERS (Probabilities: 0.9840, 0.0121, 0.0039)\n","Processed TVP62108.1 mersacidin/lichenicidin family type 2 lantibiotic [Nodularia sp. (in: Bacteria)]: non-AMP/OTHERS (Probabilities: 0.9826, 0.0051, 0.0123)\n","Processed TVW15624.1 lantibiotic salivaricin M precursor, partial [Streptococcus pneumoniae]: AMP (Probabilities: 0.0819, 0.9056, 0.0125)\n","Processed QDP90305.1 plantaricin C family lantibiotic [Bacillus amyloliquefaciens]: Neuropeptide (Probabilities: 0.0411, 0.0195, 0.9394)\n","Processed EBP3979502.1 lantibiotic ABC transporter ATP-binding protein [Salmonella enterica subsp. enterica CFSAN002856]: non-AMP/OTHERS (Probabilities: 0.9883, 0.0065, 0.0051)\n","Processed QDE92925.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus]: Neuropeptide (Probabilities: 0.1140, 0.0274, 0.8586)\n","Processed TOC10720.1 lantibiotic biosynthesis protein, partial [Vibrio parahaemolyticus]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0065, 0.0027)\n","Processed TNY43897.1 lantibiotic immunity ABC transporter MutG family permease subunit, partial [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9752, 0.0221, 0.0026)\n","Processed TNY43896.1 lantibiotic immunity ABC transporter MutG family permease subunit, partial [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9561, 0.0153, 0.0286)\n","Processed TNY43416.1 lantibiotic ABC transporter ATP-binding protein, partial [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0045, 0.0031)\n","Processed EBF5116770.1 type 2 lantibiotic [Listeria monocytogenes]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0058, 0.0030)\n","Processed TNP10486.1 type 2 lantibiotic, partial [Bacillus tropicus]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0033, 0.0058)\n","Processed TNP10151.1 lantibiotic cytolysin, partial [Bacillus tropicus]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0032, 0.0034)\n","Processed TMR91129.1 lantibiotic modifying-like protein, partial [Nonomuraea sp. 160415]: non-AMP/OTHERS (Probabilities: 0.5849, 0.0431, 0.3720)\n","Processed TMO90880.1 lantibiotic biosynthesis protein, partial [Pseudoalteromonas sp. S3260]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0040, 0.0031)\n","Processed TQV84324.1 mersacidin/lichenicidin family type 2 lantibiotic [Exilibacterium tricleocarpae]: non-AMP/OTHERS (Probabilities: 0.9846, 0.0062, 0.0092)\n","Processed PJZ19998.1 type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9807, 0.0152, 0.0041)\n","Processed WP_096727142.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc carneum]: non-AMP/OTHERS (Probabilities: 0.8849, 0.0497, 0.0654)\n","Processed TMQ64855.1 mersacidin/lichenicidin family type 2 lantibiotic [Candidatus Eisenbacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.6994, 0.0547, 0.2460)\n","Processed TMD58138.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Chloroflexi bacterium]: Neuropeptide (Probabilities: 0.0501, 0.0183, 0.9316)\n","Processed TMC23698.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: Neuropeptide (Probabilities: 0.0338, 0.0212, 0.9450)\n","Processed TMC23699.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: Neuropeptide (Probabilities: 0.1050, 0.0240, 0.8710)\n","Processed TMC23711.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: Neuropeptide (Probabilities: 0.4348, 0.0259, 0.5393)\n","Processed TLF57701.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit, partial [Bacillus intestinalis]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0061, 0.0031)\n","Processed TKY01899.1 lantibiotic ABC transporter ATP-binding protein, partial [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0029, 0.0043)\n","Processed TKY00216.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit, partial [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0069, 0.0027)\n","Processed TKX97949.1 lantibiotic immunity ABC transporter MutG family permease subunit, partial [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9869, 0.0079, 0.0051)\n","Processed TKV34746.1 lantibiotic ABC transporter ATP-binding protein, partial [Citrobacter sp. TBCS-11]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0054, 0.0030)\n","Processed TKV28308.1 lantibiotic ABC transporter ATP-binding protein, partial [Citrobacter sp. TBCS-11]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0062, 0.0027)\n","Processed VHR94085.1 lantibiotic ABC transporter ATP-binding protein [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0063, 0.0031)\n","Processed TMC19290.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium]: Neuropeptide (Probabilities: 0.3430, 0.0626, 0.5944)\n","Processed TLM97727.1 mersacidin/lichenicidin family type 2 lantibiotic [bacterium]: non-AMP/OTHERS (Probabilities: 0.9727, 0.0207, 0.0065)\n","Processed VJY24319.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0066, 0.0030)\n","Processed VRJ79307.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: Neuropeptide (Probabilities: 0.0308, 0.0187, 0.9505)\n","Processed VQS44876.1 type 2 lantibiotic, SP_1948 family [Streptococcus pneumoniae]: AMP (Probabilities: 0.3613, 0.6273, 0.0114)\n","Processed VSI63744.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0037, 0.0041)\n","Processed VMR07669.1 Lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0035, 0.0059)\n","Processed VMY09973.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9865, 0.0109, 0.0027)\n","Processed VPK98598.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0045, 0.0054)\n","Processed VSW78074.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0047, 0.0037)\n","Processed VRQ81171.1 type 2 lantibiotic, SP_1948 family [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9887, 0.0065, 0.0048)\n","Processed VIX54386.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0038, 0.0035)\n","Processed VKX57039.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0033)\n","Processed VOD08312.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0031)\n","Processed VQA59273.1 lantibiotic efflux protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0080, 0.0032)\n","Processed VIV08811.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0037)\n","Processed VOP84061.1 type 2 lantibiotic, SP_1948 family [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0035, 0.0031)\n","Processed VSU04631.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0044, 0.0053)\n","Processed VOW25294.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0082, 0.0032)\n","Processed VIS14123.1 lantibiotic export protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0053, 0.0032)\n","Processed VLI98501.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0062, 0.0031)\n","Processed VJD88580.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0068, 0.0033)\n","Processed VSP82565.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0077, 0.0034)\n","Processed VKK32681.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: Neuropeptide (Probabilities: 0.0951, 0.0304, 0.8745)\n","Processed VKZ74662.1 type 2 lantibiotic, SP_1948 family [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9885, 0.0069, 0.0046)\n","Processed VRR58084.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: AMP (Probabilities: 0.4653, 0.5134, 0.0213)\n","Processed VMT92795.1 lantibiotic ABC transporter ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0037, 0.0037)\n","Processed VQI99445.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0060, 0.0030)\n","Processed VPA34923.1 type 2 lantibiotic, SP_1948 family [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0051, 0.0027)\n","Processed VQI11257.1 lantibiotic ABC transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9878, 0.0091, 0.0031)\n","Processed VJR69320.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9899, 0.0067, 0.0034)\n","Processed VJR19066.1 type 2 lantibiotic, SP_1948 family [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0054, 0.0026)\n","Processed VNX46824.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9463, 0.0478, 0.0059)\n","Processed VFC13029.1 lantibiotic ABC transporter ATP-binding protein [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0033)\n","Processed VPA06075.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0052, 0.0027)\n","Processed VJB48793.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0030)\n","Processed CGF54512.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0085, 0.0025)\n","Processed CIO56818.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9892, 0.0083, 0.0025)\n","Processed AXU26491.1 lantibiotic ABC transporter ATP-binding protein [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0044, 0.0038)\n","Processed CZD16116.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: AMP (Probabilities: 0.3305, 0.6564, 0.0131)\n","Processed CJV53296.1 type 2 lantibiotic%2C SP_1948 family [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0033, 0.0033)\n","Processed CGF11061.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9751, 0.0206, 0.0043)\n","Processed THG25565.1 lantibiotic immunity ABC transporter MutG family permease subunit, partial [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0082, 0.0027)\n","Processed CEX10115.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0060, 0.0033)\n","Processed TGY91458.1 type 2 lantibiotic [Lachnospiraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0049, 0.0035)\n","Processed EHE21452.1 subtilisin-like serine protease domain protein [Streptococcus pneumoniae GA41565 GA41565]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0053, 0.0026)\n","Processed TGY91457.1 type 2 lantibiotic [Lachnospiraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.8707, 0.1238, 0.0055)\n","Processed TGY91466.1 type 2 lantibiotic [Lachnospiraceae bacterium]: Neuropeptide (Probabilities: 0.3645, 0.0255, 0.6099)\n","Processed TGY91459.1 type 2 lantibiotic [Lachnospiraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9282, 0.0670, 0.0049)\n","Processed CFP78278.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9867, 0.0104, 0.0028)\n","Processed VHN11423.1 Type-A lantibiotic [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9893, 0.0041, 0.0066)\n","Processed VHN09947.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0034, 0.0041)\n","Processed VGU24277.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0039, 0.0037)\n","Processed VHF40320.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0039)\n","Processed VGW39033.1 lantibiotic transport permease [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9883, 0.0088, 0.0029)\n","Processed VGS14146.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0037, 0.0048)\n","Processed VHC84879.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0035, 0.0041)\n","Processed VHD42351.1 Lantibiotic macedocin [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed VGR89131.1 lantibiotic transport ATP-binding protein [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9396, 0.0329, 0.0275)\n","Processed VHE95157.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0043, 0.0031)\n","Processed VGQ41888.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0048, 0.0055)\n","Processed VGX08813.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0035, 0.0046)\n","Processed VGV21488.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0032)\n","Processed VGQ53806.1 putative lantibiotic modifying protein [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0031, 0.0034)\n","Processed VGQ19409.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0035, 0.0041)\n","Processed SQF14396.1 lantibiotic streptin [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0040, 0.0037)\n","Processed VHC71739.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0038, 0.0030)\n","Processed VGR89097.1 lantibiotic salivaricin A [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0057, 0.0033)\n","Processed  unnamed protein product [Unknown]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0054, 0.0035)\n","Processed TCP70223.1 two-component lantibiotic alpha peptide [Baia soyae]: Neuropeptide (Probabilities: 0.3225, 0.0465, 0.6310)\n","Processed SQF25986.1 lantibiotic biosynthesis protein [Streptococcus thermophilus]: non-AMP/OTHERS (Probabilities: 0.9896, 0.0079, 0.0025)\n","Processed TFH55350.1 lantibiotic ABC transporter, partial [candidate division Zixibacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0044, 0.0053)\n","Processed TEA45671.1 type 2 lantibiotic [Bacillus sp. BH2]: Neuropeptide (Probabilities: 0.3476, 0.0208, 0.6316)\n","Processed TBW68542.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus epidermidis]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0046, 0.0035)\n","Processed TAG95058.1 cinnamycin family lantibiotic [Oscillatoriales cyanobacterium]: non-AMP/OTHERS (Probabilities: 0.8997, 0.0614, 0.0389)\n","Processed VGV21454.1 lantibiotic transport ATP-binding protein [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0033)\n","Processed TAE61612.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostocales cyanobacterium]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0035, 0.0054)\n","Processed TAG17337.1 mersacidin/lichenicidin family type 2 lantibiotic [Oscillatoriales cyanobacterium]: non-AMP/OTHERS (Probabilities: 0.9738, 0.0088, 0.0174)\n","Processed TAF05567.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostocales cyanobacterium]: non-AMP/OTHERS (Probabilities: 0.9853, 0.0066, 0.0081)\n","Processed VFH49739.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9910, 0.0056, 0.0034)\n","Processed VFC10996.1 lantibiotic two-component sensor histidine kinase [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9840, 0.0119, 0.0041)\n","Processed RZU51344.1 two-component lantibiotic alpha peptide [Krasilnikovia cinnamomea]: Neuropeptide (Probabilities: 0.0812, 0.0350, 0.8838)\n","Processed RZK31215.1 lantibiotic ABC transporter, partial [Hymenobacter sp.]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0052, 0.0032)\n","Processed RZI15487.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0038, 0.0039)\n","Processed SJN91991.1 nodulation ABC transporter NodI [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0056, 0.0034)\n","Processed RZI12819.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9882, 0.0089, 0.0029)\n","Processed RZI06353.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0040, 0.0034)\n","Processed RZH84540.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0035, 0.0040)\n","Processed CDQ30217.1 putative integrative and conjugative element protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0061, 0.0033)\n","Processed RZH79776.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0040, 0.0031)\n","Processed RZH82734.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0040, 0.0043)\n","Processed RZI11151.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed KMR18445.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed QBD82986.1 lantibiotic protein [Ktedonosporobacter rubrisoli]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0049, 0.0032)\n","Processed VDZ12601.1 lantibiotic ABC transporter protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9892, 0.0077, 0.0031)\n","Processed RSU09959.1 type 2 lantibiotic [Vagococcus acidifermentans]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0036, 0.0030)\n","Processed MNU09291.1 Lantibiotic lichenicidin VK21 A2 precursor [compost metagenome]: non-AMP/OTHERS (Probabilities: 0.5230, 0.4587, 0.0183)\n","Processed MNV34094.1 Lantibiotic lichenicidin VK21 A1 precursor [compost metagenome]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0054, 0.0043)\n","Processed RQX36334.1 lantibiotic ABC transporter, partial [Micromonospora chalcea]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0041, 0.0034)\n","Processed AZV44330.1 lantibiotic biosynthesis protein, HFCD family [Peribacillus asahii]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0030, 0.0052)\n","Processed OFN94305.1 lantibiotic salivaricin M precursor [Streptococcus sp. HMSC077D04]: AMP (Probabilities: 0.1543, 0.8308, 0.0149)\n","Processed RQN27779.1 lantibiotic ABC transporter ATP-binding protein, partial [Paraburkholderia tropica]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0042, 0.0033)\n","Processed RWZ58686.1 plantaricin C family lantibiotic [Halobacillus sp. SKP4-6]: AMP (Probabilities: 0.1066, 0.8799, 0.0135)\n","Processed OAN11929.1 lacticin family bacteriocin [Streptococcus sp. CCUG 49591]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed AZH28847.1 gallidermin/nisin family lantibiotic [Paenibacillus sp. M-152]: AMP (Probabilities: 0.1022, 0.8860, 0.0118)\n","Processed CDQ30350.1 putative integrative and conjugative element protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0048, 0.0050)\n","Processed RPJ84259.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: Neuropeptide (Probabilities: 0.0307, 0.0157, 0.9536)\n","Processed RPG21504.1 mersacidin/lichenicidin family type 2 lantibiotic [Phycisphaera sp. TMED9]: non-AMP/OTHERS (Probabilities: 0.9894, 0.0030, 0.0076)\n","Processed ROV59924.1 type 2 lantibiotic [Limosilactobacillus reuteri]: AMP (Probabilities: 0.1338, 0.8544, 0.0118)\n","Processed EGI11534.1 L-arabinose ABC transporter, ATP-binding protein AraG [Escherichia coli H736 H736]: non-AMP/OTHERS (Probabilities: 0.9875, 0.0061, 0.0064)\n","Processed ROX80814.1 plantaricin C family lantibiotic [Enterococcus durans]: Neuropeptide (Probabilities: 0.0867, 0.0208, 0.8925)\n","Processed ROV59923.1 type 2 lantibiotic [Limosilactobacillus reuteri]: Neuropeptide (Probabilities: 0.0102, 0.0098, 0.9800)\n","Processed AOW88171.1 lantibiotic lacticin [Streptomyces olivaceus]: non-AMP/OTHERS (Probabilities: 0.9882, 0.0044, 0.0073)\n","Processed RLU56229.1 lantibiotic ABC transporter permease, partial [Streptococcus iniae]: non-AMP/OTHERS (Probabilities: 0.9878, 0.0095, 0.0027)\n","Processed RLM76129.1 lantibiotic ABC transporter ATP-binding protein, partial [Halorubrum sp. Atlit-8R]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0057, 0.0030)\n","Processed RKW68568.1 plantaricin C family lantibiotic [Veillonella sp.]: non-AMP/OTHERS (Probabilities: 0.9793, 0.0173, 0.0034)\n","Processed RKW56431.1 lantibiotic ABC transporter permease, partial [Prevotella sp.]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0065, 0.0026)\n","Processed RKW32020.1 lantibiotic immunity ABC transporter MutG family permease subunit, partial [Lachnospiraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9359, 0.0600, 0.0041)\n","Processed RKW14544.1 lantibiotic ABC transporter permease, partial [Catonella sp.]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0074, 0.0025)\n","Processed RKV79916.1 lantibiotic ABC transporter permease, partial [Streptococcus sp.]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0078, 0.0031)\n","Processed RKM59697.1 type 2 lantibiotic [Butyrivibrio sp. XB500-5]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0056, 0.0037)\n","Processed RKW05977.1 lantibiotic immunity ABC transporter MutG family permease subunit, partial [Streptococcus sp.]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0057, 0.0027)\n","Processed RKV70500.1 lantibiotic ABC transporter permease, partial [Streptococcus sp.]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0059, 0.0028)\n","Processed RKM55315.1 type 2 lantibiotic [Butyrivibrio sp. X503]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0049, 0.0047)\n","Processed RKI73647.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. AB049A]: Neuropeptide (Probabilities: 0.3522, 0.0463, 0.6015)\n","Processed RKH68135.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus aberystwythensis]: non-AMP/OTHERS (Probabilities: 0.4743, 0.0729, 0.4528)\n","Processed RKH30201.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA041A]: Neuropeptide (Probabilities: 0.2949, 0.0422, 0.6630)\n","Processed RKG71000.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA054B]: Neuropeptide (Probabilities: 0.2155, 0.0582, 0.7263)\n","Processed RKH54076.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. AB050B]: Neuropeptide (Probabilities: 0.2751, 0.0437, 0.6811)\n","Processed RIO54763.1 lantibiotic ABC transporter ATP-binding protein, partial [Mammaliicoccus sciuri]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0038, 0.0035)\n","Processed RIM62027.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus xylosus]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0037, 0.0033)\n","Processed RIM74344.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus arlettae]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0033)\n","Processed RIM61251.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus xylosus]: non-AMP/OTHERS (Probabilities: 0.9866, 0.0034, 0.0099)\n","Processed RIY20685.1 lantibiotic ABC transporter permease, partial [Gardnerella vaginalis]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0045, 0.0030)\n","Processed RIY22755.1 lantibiotic ABC transporter permease, partial [Gardnerella vaginalis]: non-AMP/OTHERS (Probabilities: 0.9885, 0.0089, 0.0026)\n","Processed HCX70376.1 lantibiotic ABC transporter ATP-binding protein [Firmicutes bacterium]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0047, 0.0046)\n","Processed HCT90750.1 lantibiotic ABC transporter permease [Lachnospiraceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0080, 0.0031)\n","Processed ABX06376.1 hypothetical protein [Herpetosiphon aurantiacus DSM 785]: non-AMP/OTHERS (Probabilities: 0.8740, 0.0115, 0.1144)\n","Processed HBT71644.1 lantibiotic ABC transporter ATP-binding protein [Lysinibacillus sp.]: non-AMP/OTHERS (Probabilities: 0.9879, 0.0054, 0.0067)\n","Processed HBT16501.1 lantibiotic ABC transporter permease [Firmicutes bacterium]: non-AMP/OTHERS (Probabilities: 0.9882, 0.0081, 0.0037)\n","Processed HBL13915.1 mersacidin/lichenicidin family type 2 lantibiotic [Cyanobacteria bacterium UBA11162]: non-AMP/OTHERS (Probabilities: 0.9748, 0.0072, 0.0179)\n","Processed HCE12917.1 type 2 lantibiotic [Enterococcus sp.]: non-AMP/OTHERS (Probabilities: 0.9035, 0.0110, 0.0855)\n","Processed HBR81277.1 lantibiotic transporter [Enterococcus sp.]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0032)\n","Processed HBF45725.1 lantibiotic ABC transporter permease [Shewanella frigidimarina]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0044, 0.0030)\n","Processed HCB88017.1 lantibiotic ABC transporter [Porphyromonadaceae bacterium]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0065, 0.0033)\n","Processed HBF45723.1 lantibiotic ABC transporter permease [Shewanella frigidimarina]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0037, 0.0033)\n","Processed RHM68438.1 gallidermin/nisin family lantibiotic [[Ruminococcus] gnavus]: Neuropeptide (Probabilities: 0.0640, 0.0176, 0.9184)\n","Processed HBB62253.1 lantibiotic ABC transporter ATP-binding protein [Ruminococcus sp.]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0048, 0.0051)\n","Processed RHV00049.1 NisI/SpaI family lantibiotic immunity lipoprotein [Firmicutes bacterium OM07-11]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0042, 0.0030)\n","Processed RHQ10679.1 gallidermin/nisin family lantibiotic [Ruminococcus sp. AM50-15BH]: AMP (Probabilities: 0.0702, 0.9167, 0.0131)\n","Processed RIL41018.1 plantaricin C family lantibiotic [Staphylococcus gallinarum]: non-AMP/OTHERS (Probabilities: 0.9775, 0.0071, 0.0154)\n","Processed HAX81003.1 lantibiotic ABC transporter ATP-binding protein [Actinobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0040, 0.0039)\n","Processed RHH35385.1 lantibiotic ABC transporter permease, partial [Collinsella sp. AM18-10]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0055, 0.0030)\n","Processed RHF95935.1 lantibiotic ABC transporter permease, partial [Collinsella sp. AM23-17]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0051, 0.0043)\n","Processed RHI07812.1 lantibiotic ABC transporter permease, partial [Collinsella sp. AM16-21]: non-AMP/OTHERS (Probabilities: 0.9888, 0.0083, 0.0029)\n","Processed RGZ03622.1 plantaricin C family lantibiotic [Blautia obeum]: non-AMP/OTHERS (Probabilities: 0.9104, 0.0161, 0.0735)\n","Processed RGQ29348.1 lantibiotic ABC transporter permease, partial [Collinsella sp. AF28-5AC]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0042, 0.0033)\n","Processed RHB14202.1 lantibiotic ABC transporter permease, partial [Collinsella sp. AM41-2BH]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0055, 0.0031)\n","Processed RHB09745.1 lantibiotic ABC transporter permease, partial [Collinsella sp. AM41-2BH]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0048, 0.0032)\n","Processed RHA65007.1 lantibiotic ABC transporter permease, partial [Collinsella sp. AM43-1]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0059, 0.0034)\n","Processed RHS75421.1 NisI/SpaI family lantibiotic immunity lipoprotein [Firmicutes bacterium AM43-11BH]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0046, 0.0049)\n","Processed RGQ66375.1 type 2 lantibiotic [[Ruminococcus] gnavus]: non-AMP/OTHERS (Probabilities: 0.9636, 0.0061, 0.0303)\n","Processed RHE23605.1 lantibiotic ABC transporter permease, partial [Collinsella sp. AM29-10AC]: non-AMP/OTHERS (Probabilities: 0.9905, 0.0053, 0.0042)\n","Processed HAD67421.1 plantaricin C family lantibiotic [Roseburia sp.]: Neuropeptide (Probabilities: 0.0158, 0.0132, 0.9711)\n","Processed HAJ49387.1 lantibiotic ABC transporter ATP-binding protein [Eubacterium sp.]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0068, 0.0030)\n","Processed RGI93988.1 gallidermin/nisin family lantibiotic [Blautia obeum]: AMP (Probabilities: 0.0688, 0.9171, 0.0142)\n","Processed HAD67420.1 lantibiotic salivaricin M precursor [Roseburia sp.]: AMP (Probabilities: 0.0682, 0.9183, 0.0135)\n","Processed AKO91019.1 hypothetical protein [Priestia filamentosa]: AMP (Probabilities: 0.0764, 0.9092, 0.0144)\n","Processed ETT80164.1 lantibiotic precursor peptide, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0036, 0.0043)\n","Processed RGH38932.1 NisI/SpaI family lantibiotic immunity lipoprotein [Firmicutes bacterium AM41-5BH]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0036, 0.0044)\n","Processed AKO91018.1 hypothetical protein [Priestia filamentosa]: AMP (Probabilities: 0.0812, 0.9049, 0.0139)\n","Processed RFT24015.1 lantibiotic ABC transporter permease, partial [Gardnerella vaginalis]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0070, 0.0033)\n","Processed RFT31743.1 lantibiotic ABC transporter permease, partial [Gardnerella vaginalis]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0060, 0.0030)\n","Processed RFT38982.1 lantibiotic ABC transporter permease, partial [Bifidobacteriaceae bacterium NR003]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0050, 0.0031)\n","Processed RFT37719.1 lantibiotic ABC transporter permease, partial [Bifidobacteriaceae bacterium NR015]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0085, 0.0026)\n","Processed RFT24016.1 lantibiotic ABC transporter permease, partial [Gardnerella vaginalis]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0039, 0.0028)\n","Processed RFS83503.1 lantibiotic protein [Actinomadura sp. LHW52907]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0060, 0.0050)\n","Processed REA65563.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0069, 0.0032)\n","Processed AKJ04875.1 Hypothetical protein [Archangium gephyra]: non-AMP/OTHERS (Probabilities: 0.9115, 0.0189, 0.0696)\n","Processed RDV23370.1 lantibiotic nisin-A, partial [Parageobacillus toebii]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0046, 0.0055)\n","Processed SUM43748.1 type 2 lantibiotic, SP_1948 family [Staphylococcus intermedius NCTC 11048 NCTC 11048]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed CEC09170.1 Hydrophobic transmembrane protein [Salmonella enterica subsp. enterica serovar Paratyphi A]: non-AMP/OTHERS (Probabilities: 0.9851, 0.0124, 0.0025)\n","Processed RDU24246.1 lantibiotic salivaricin M precursor [Anaerosacchariphilus polymeriproducens]: AMP (Probabilities: 0.0898, 0.8983, 0.0119)\n","Processed SUA96117.1 type 2 lantibiotic, SP_1948 family [Paenibacillus thiaminolyticus]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0035, 0.0040)\n","Processed SUL36574.1 lantibiotic immunity protein F [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0051, 0.0031)\n","Processed SUL36571.1 lantibiotic immunity protein F [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0039, 0.0034)\n","Processed SUL36565.1 lantibiotic ABC transporter protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9753, 0.0220, 0.0026)\n","Processed SUL36572.1 lantibiotic immunity protein F [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0046, 0.0034)\n","Processed OLF49190.1 hypothetical protein [Streptococcus acidominimus]: non-AMP/OTHERS (Probabilities: 0.9896, 0.0073, 0.0031)\n","Processed CCE59500.1 putative lantibiotic ABC transporter protein (fragment) [Staphylococcus argenteus]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0076, 0.0026)\n","Processed SUK45342.1 putative lantibiotic ABC transporter protein (fragment) [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9892, 0.0083, 0.0025)\n","Processed SUK96192.1 lantibiotic gallidermin [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0040)\n","Processed SUN61092.1 lantibiotic ABC transporter permease [Streptococcus dysgalactiae subsp. equisimilis NCTC11564]: non-AMP/OTHERS (Probabilities: 0.9886, 0.0087, 0.0027)\n","Processed EFX35883.1 hypothetical protein [Streptococcus infantis ATCC 700779 ATCC 700779]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0043)\n","Processed ABB05512.1 salivaricin A1 precursor [Streptococcus agalactiae]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed RCK13063.1 plantaricin C family lantibiotic [Bacillus licheniformis]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0033, 0.0033)\n","Processed RBN36892.1 lantibiotic biosynthesis protein, partial [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0033)\n","Processed SUN61096.1 lantibiotic ABC transporter permease [Streptococcus dysgalactiae subsp. equisimilis NCTC11564]: non-AMP/OTHERS (Probabilities: 0.9840, 0.0132, 0.0029)\n","Processed EGS91203.1 hypothetical protein [Staphylococcus aureus subsp. aureus 21269 21269]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0067, 0.0030)\n","Processed RCK13137.1 type 2 lantibiotic [Bacillus licheniformis]: AMP (Probabilities: 0.3396, 0.6491, 0.0113)\n","Processed RBN36516.1 lantibiotic biosynthesis protein, partial [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9883, 0.0084, 0.0033)\n","Processed RAN71050.1 type 2 lantibiotic [Bacillus sp. SRB_8]: non-AMP/OTHERS (Probabilities: 0.9601, 0.0362, 0.0037)\n","Processed RAJ69875.1 two-component lantibiotic alpha peptide [Streptomyces sp. Amel2xB2]: AMP (Probabilities: 0.0848, 0.9015, 0.0137)\n","Processed RBN36240.1 lantibiotic biosynthesis protein, partial [Priestia megaterium]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0054, 0.0028)\n","Processed PZL76953.1 type 2 lantibiotic [Enterococcus plantarum]: AMP (Probabilities: 0.0935, 0.8945, 0.0120)\n","Processed KFA43237.1 epidermin decarboxylase [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0034)\n","Processed SQI08564.1 two-component system response regulator lantibiotic associated [Streptococcus pasteurianus]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0058, 0.0044)\n","Processed SQF82094.1 lantibiotic synthetase protein [Streptococcus equi subsp. zooepidemicus NCTC6176]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0043, 0.0040)\n","Processed SQF42301.1 lantibiotic biosynthesis DNA-binding response regulator [Streptococcus lutetiensis]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0043, 0.0030)\n","Processed PYT18031.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: Neuropeptide (Probabilities: 0.3196, 0.1046, 0.5759)\n","Processed PYT07918.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: Neuropeptide (Probabilities: 0.0147, 0.0123, 0.9730)\n","Processed PYT03204.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: AMP (Probabilities: 0.1442, 0.8308, 0.0250)\n","Processed PYT18032.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.7514, 0.2219, 0.0267)\n","Processed PYS96078.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.5425, 0.4188, 0.0388)\n","Processed PYS96079.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: Neuropeptide (Probabilities: 0.2891, 0.0902, 0.6206)\n","Processed PYS12131.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: Neuropeptide (Probabilities: 0.2435, 0.0486, 0.7079)\n","Processed PYS89840.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.7070, 0.0236, 0.2694)\n","Processed PYS89839.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.9656, 0.0126, 0.0218)\n","Processed PYQ63345.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.8849, 0.0371, 0.0779)\n","Processed PYQ59993.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.8223, 0.1539, 0.0237)\n","Processed PYR98312.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: Neuropeptide (Probabilities: 0.4380, 0.0719, 0.4901)\n","Processed PYQ59992.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: non-AMP/OTHERS (Probabilities: 0.8602, 0.1200, 0.0199)\n","Processed PYQ09556.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: Neuropeptide (Probabilities: 0.3825, 0.0549, 0.5625)\n","Processed PWG68138.1 lantibiotic ABC transporter ATP-binding protein, partial [Enterobacter mori]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0059, 0.0033)\n","Processed PVW16807.1 lantibiotic dehydratase, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0045, 0.0032)\n","Processed EEK72523.1 hypothetical protein [Bacillus mycoides]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0037, 0.0038)\n","Processed PYQ12947.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium]: Neuropeptide (Probabilities: 0.1038, 0.0266, 0.8696)\n","Processed PYP90029.1 mersacidin/lichenicidin family type 2 lantibiotic [Blastocatellia bacterium AA13]: non-AMP/OTHERS (Probabilities: 0.9649, 0.0077, 0.0274)\n","Processed PWV48517.1 gallidermin/nisin family lantibiotic [Nocardiopsis sp. L17-MgMaSL7]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0036)\n","Processed PVC86288.1 lantibiotic protein, partial [Streptomyces sp. CS131]: Neuropeptide (Probabilities: 0.0651, 0.0210, 0.9138)\n","Processed AVN84861.1 lantibiotic mersacidin, partial [Bacillus subtilis]: Neuropeptide (Probabilities: 0.3162, 0.0462, 0.6376)\n","Processed PPJ95081.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed PRP60939.1 lantibiotic ABC transporter ATP-binding protein, partial [Bacillus halotolerans]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0037, 0.0032)\n","Processed PNZ60859.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0062, 0.0026)\n","Processed PNZ59920.1 gallidermin/nisin family lantibiotic, partial [Staphylococcus aureus]: Neuropeptide (Probabilities: 0.0590, 0.0413, 0.8997)\n","Processed PKQ14295.1 lantibiotic biosynthesis protein, partial [Actinobacteria bacterium HGW-Actinobacteria-8]: non-AMP/OTHERS (Probabilities: 0.9771, 0.0077, 0.0152)\n","Processed PKM75344.1 lantibiotic ABC transporter ATP-binding protein, partial [Firmicutes bacterium HGW-Firmicutes-17]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0055, 0.0037)\n","Processed POP40054.1 lantibiotic immunity ABC transporter MutE/EpiE family permease subunit, partial [Blautia producta]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0041, 0.0027)\n","Processed PNZ49523.1 type 2 lantibiotic [Staphylococcus intermedius NCTC 11048 NCTC 11048]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0038)\n","Processed PJJ52929.1 mersacidin/lichenicidin family type 2 lantibiotic [Hymenobacter chitinivorans DSM 11115 DSM 11115]: non-AMP/OTHERS (Probabilities: 0.5836, 0.0376, 0.3788)\n","Processed PDP85838.1 mersacidin/lichenicidin family type 2 lantibiotic [Glycomyces fuscus]: Neuropeptide (Probabilities: 0.4636, 0.0257, 0.5107)\n","Processed PKE42958.1 lantibiotic ABC transporter ATP-binding protein, partial [Macrococcus caseolyticus]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0047, 0.0035)\n","Processed PJZ17800.1 type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9876, 0.0053, 0.0071)\n","Processed ATL75566.1 lantibiotic dehydratase, partial [Lactococcus lactis]: AMP (Probabilities: 0.0884, 0.9000, 0.0116)\n","Processed ATL75574.1 lantibiotic dehydratase, partial [Lactococcus lactis]: AMP (Probabilities: 0.1005, 0.8877, 0.0118)\n","Processed PDP85830.1 lantibiotic lacticin [Glycomyces fuscus]: AMP (Probabilities: 0.0757, 0.9113, 0.0130)\n","Processed PIH06223.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus epidermidis]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0033)\n","Processed PDP85837.1 mersacidin/lichenicidin family type 2 lantibiotic [Glycomyces fuscus]: Neuropeptide (Probabilities: 0.0128, 0.0110, 0.9763)\n","Processed PJD94815.1 lantibiotic epidermin [Parachlamydia sp.]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0058, 0.0033)\n","Processed PHG10863.1 lantibiotic cytolysin, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.1299, 0.8588, 0.0113)\n","Processed PHG10864.1 lantibiotic cytolysin [Bacillus pseudomycoides]: AMP (Probabilities: 0.1572, 0.8213, 0.0215)\n","Processed PHE82938.1 lantibiotic cytolysin, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0038, 0.0050)\n","Processed PHE61017.1 lantibiotic cytolysin [Bacillus pseudomycoides]: AMP (Probabilities: 0.1288, 0.8554, 0.0159)\n","Processed PGZ27754.1 type 2 lantibiotic [Bacillus anthracis]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0050, 0.0031)\n","Processed ATL75584.1 lantibiotic dehydratase, partial [Lactococcus lactis]: AMP (Probabilities: 0.0944, 0.8940, 0.0116)\n","Processed PGX42690.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.7840, 0.2069, 0.0092)\n","Processed PGQ30920.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.1447, 0.8422, 0.0131)\n","Processed PGU21491.1 type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9234, 0.0136, 0.0630)\n","Processed PGP34776.1 lantibiotic cytolysin, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.8546, 0.0101, 0.1353)\n","Processed PFT24507.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.0851, 0.9027, 0.0122)\n","Processed PFF57997.1 lantibiotic cytolysin [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9852, 0.0058, 0.0089)\n","Processed PGC73695.1 lantibiotic salivaricin A, partial [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0032, 0.0053)\n","Processed PEZ83843.1 lantibiotic cytolysin [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9684, 0.0108, 0.0208)\n","Processed PFA58577.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8536, 0.1220, 0.0243)\n","Processed PEF29980.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.1091, 0.8786, 0.0122)\n","Processed PGO41331.1 lantibiotic lacticin [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9395, 0.0117, 0.0488)\n","Processed PEF08708.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: Neuropeptide (Probabilities: 0.2393, 0.0797, 0.6810)\n","Processed PEF08709.1 type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9717, 0.0065, 0.0219)\n","Processed PDZ75828.1 type 2 lantibiotic, partial [Bacillus cereus]: AMP (Probabilities: 0.2170, 0.7680, 0.0150)\n","Processed PEC70250.1 type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9700, 0.0069, 0.0231)\n","Processed PDZ75827.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0036, 0.0043)\n","Processed PEC69548.1 type 2 lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.3399, 0.6500, 0.0102)\n","Processed PGE01485.1 lantibiotic cytolysin, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.1305, 0.8548, 0.0147)\n","Processed PEB38295.1 lantibiotic cytolysin, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0032, 0.0061)\n","Processed PDZ19764.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.5406, 0.3722, 0.0872)\n","Processed OTW81043.1 type 2 lantibiotic [Bacillus thuringiensis serovar jinghongiensis BGSC 4AR1]: non-AMP/OTHERS (Probabilities: 0.9530, 0.0085, 0.0385)\n","Processed EOO82395.1 hypothetical protein, partial [Bacillus cereus K-5975c K-5975c]: non-AMP/OTHERS (Probabilities: 0.9892, 0.0035, 0.0073)\n","Processed PGE94576.1 lantibiotic cytolysin, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0037)\n","Processed PFZ12298.1 lantibiotic cytolysin, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0703, 0.9157, 0.0140)\n","Processed PFU72930.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0071, 0.0034)\n","Processed PGD87167.1 lantibiotic cytolysin, partial [Bacillus pseudomycoides]: AMP (Probabilities: 0.0727, 0.9145, 0.0128)\n","Processed PGB11248.1 lantibiotic cytolysin, partial [Bacillus toyonensis]: AMP (Probabilities: 0.1022, 0.8854, 0.0123)\n","Processed PFU57454.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0039, 0.0039)\n","Processed PFR12159.1 type 2 lantibiotic, partial [Bacillus anthracis]: non-AMP/OTHERS (Probabilities: 0.6463, 0.3456, 0.0081)\n","Processed PFQ22352.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0036, 0.0042)\n","Processed PFK33797.1 lantibiotic salivaricin A, partial [Bacillus cereus]: AMP (Probabilities: 0.4736, 0.5045, 0.0219)\n","Processed PFN84793.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed PEY58207.1 lantibiotic nukacin [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9241, 0.0150, 0.0609)\n","Processed PEW26420.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.6676, 0.1067, 0.2257)\n","Processed PEV16977.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.1238, 0.8638, 0.0124)\n","Processed PDX97457.1 lantibiotic cytolysin [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9849, 0.0094, 0.0057)\n","Processed PEA49082.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0038, 0.0043)\n","Processed PFI80111.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0029, 0.0059)\n","Processed PEA13104.1 type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0035, 0.0032)\n","Processed EEM83103.1 hypothetical protein [Bacillus thuringiensis serovar huazhongensis BGSC 4BD1 BGSC 4BD1]: AMP (Probabilities: 0.1043, 0.8831, 0.0126)\n","Processed PEI86162.1 lantibiotic cytolysin, partial [Bacillus pseudomycoides]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0039, 0.0064)\n","Processed PEA46881.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.1023, 0.8854, 0.0123)\n","Processed PFT24128.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9892, 0.0055, 0.0053)\n","Processed PFD97243.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9175, 0.0764, 0.0061)\n","Processed PFC80268.1 type 2 lantibiotic, partial [Bacillus anthracis]: non-AMP/OTHERS (Probabilities: 0.9228, 0.0163, 0.0608)\n","Processed PEZ24879.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.4537, 0.1519, 0.3944)\n","Processed PEY48513.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0034, 0.0037)\n","Processed PEX03416.1 type 2 lantibiotic [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9898, 0.0054, 0.0048)\n","Processed PEY46832.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8720, 0.1116, 0.0165)\n","Processed PFE39189.1 type 2 lantibiotic, partial [Bacillus cereus]: AMP (Probabilities: 0.0949, 0.8929, 0.0121)\n","Processed PFE36798.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.7705, 0.1975, 0.0319)\n","Processed PFD86408.1 type 2 lantibiotic, partial [Bacillus anthracis]: non-AMP/OTHERS (Probabilities: 0.7749, 0.2179, 0.0072)\n","Processed PEU67206.1 lantibiotic cytolysin, partial [Bacillus cereus]: AMP (Probabilities: 0.0841, 0.9001, 0.0158)\n","Processed PES89631.1 lantibiotic cytolysin, partial [Bacillus anthracis]: AMP (Probabilities: 0.0998, 0.8878, 0.0124)\n","Processed PEW28378.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0040, 0.0038)\n","Processed PES89633.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus anthracis]: Neuropeptide (Probabilities: 0.1440, 0.0432, 0.8129)\n","Processed PES74638.1 lantibiotic cytolysin [Bacillus anthracis]: AMP (Probabilities: 0.0788, 0.9068, 0.0144)\n","Processed PEL13682.1 lantibiotic cytolysin, partial [Bacillus wiedmannii]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0035)\n","Processed PEF83314.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9862, 0.0040, 0.0098)\n","Processed PES89632.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus anthracis]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0034, 0.0050)\n","Processed PES75855.1 type 2 lantibiotic [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0054, 0.0045)\n","Processed PER22420.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8567, 0.0130, 0.1304)\n","Processed PEA49057.1 type 2 lantibiotic, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.6867, 0.2560, 0.0573)\n","Processed PDZ63916.1 type 2 lantibiotic, partial [Bacillus cereus]: non-AMP/OTHERS (Probabilities: 0.8202, 0.0195, 0.1603)\n","Processed PDZ62118.1 type 2 lantibiotic, partial [Bacillus cereus]: AMP (Probabilities: 0.1064, 0.8807, 0.0129)\n","Processed PDM41724.1 type 2 lantibiotic, partial [Parageobacillus yumthangensis]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0045, 0.0028)\n","Processed PEF87618.1 type 2 lantibiotic [Bacillus thuringiensis]: AMP (Probabilities: 0.1651, 0.8160, 0.0189)\n","Processed PDH70752.1 lantibiotic ABC transporter ATP-binding protein, partial [Bacillus licheniformis]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0054, 0.0036)\n","Processed ATF27291.1 lantibiotic lacticin [Brochothrix thermosphacta]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0033, 0.0046)\n","Processed BBA24449.1 lantibiotic ABC transporter protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0062, 0.0029)\n","Processed OZX29908.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0038, 0.0039)\n","Processed SBS02255.1 LanC lantibiotic synthetase component C-like 2 (bacterial), partial [Nothobranchius pienaari]: Neuropeptide (Probabilities: 0.0191, 0.0142, 0.9667)\n","Processed PAD23141.1 lantibiotic lacticin [Terribacillus saccharophilus]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0052, 0.0026)\n","Processed OZX29909.1 lantibiotic ABC transporter permease, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9894, 0.0078, 0.0028)\n","Processed SNI32089.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0077, 0.0034)\n","Processed SNH77726.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0041, 0.0044)\n","Processed SNC83946.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0069, 0.0030)\n","Processed OWA35270.1 type 2 lantibiotic [Saccharibacillus sp. O16]: AMP (Probabilities: 0.4194, 0.5708, 0.0098)\n","Processed OTY52016.1 type 2 lantibiotic [Bacillus thuringiensis serovar graciosensis BGSC 4CD1]: non-AMP/OTHERS (Probabilities: 0.5218, 0.0499, 0.4283)\n","Processed OTX89614.1 type 2 lantibiotic [Bacillus thuringiensis serovar londrina BGSC 4BF1]: non-AMP/OTHERS (Probabilities: 0.6909, 0.0660, 0.2430)\n","Processed OTW81042.1 type 2 lantibiotic [Bacillus thuringiensis serovar jinghongiensis BGSC 4AR1]: non-AMP/OTHERS (Probabilities: 0.9483, 0.0114, 0.0403)\n","Processed OSC73494.1 lantibiotic protein [Streptomyces sp. BF-3]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0033)\n","Processed ORP07981.1 lantibiotic salivaricin M precursor [Streptococcus mitis]: AMP (Probabilities: 0.0684, 0.9180, 0.0135)\n","Processed ORO17315.1 lantibiotic epidermin immunity protein F [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9876, 0.0098, 0.0026)\n","Processed ORN91437.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0039, 0.0030)\n","Processed OTP39067.1 lantibiotic protein CylL-L, partial [Enterococcus faecalis]: Neuropeptide (Probabilities: 0.0092, 0.0096, 0.9813)\n","Processed OLO17448.1 lantibiotic ABC transporter, partial [Bacillus licheniformis]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0036, 0.0032)\n","Processed SJU69835.1 lantibiotic ABC transporter permease [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0074, 0.0025)\n","Processed OOK82254.1 lantibiotic modifying enzyme domain protein [Mycobacterium kansasii]: non-AMP/OTHERS (Probabilities: 0.8721, 0.0725, 0.0554)\n","Processed OJT85981.1 lantibiotic ABC transporter permease, partial [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9905, 0.0067, 0.0028)\n","Processed KEQ32241.1 type-A lantibiotic family protein [Streptococcus mitis]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0038, 0.0038)\n","Processed APO33423.1 lanti_perm_MutG: lantibiotic protection ABC transporter permease subunit, MutG, partial [uncultured bacterium]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0068, 0.0024)\n","Processed SIJ40885.1 type 2 lantibiotic, SP_1948 family [Mycobacteroides abscessus subsp. abscessus 1079]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0045, 0.0031)\n","Processed SIJ40855.1 type 2 lantibiotic, SP_1948 family [Mycobacteroides abscessus subsp. abscessus 1079]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0031, 0.0040)\n","Processed OJT81074.1 lantibiotic ABC transporter permease, partial [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed BAQ25094.1 lantibiotic mersacidin modifying enzyme [Streptococcus troglodytae]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0036)\n","Processed OJT73877.1 lantibiotic ABC transporter permease, partial [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9901, 0.0068, 0.0031)\n","Processed OJT72128.1 lantibiotic ABC transporter permease, partial [Clostridioides difficile]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0048, 0.0025)\n","Processed SHH08738.1 type 2 lantibiotic, SP_1948 family [Butyrivibrio fibrisolvens DSM 3071 DSM 3071]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0065, 0.0035)\n","Processed SHC67496.1 lantibiotic ABC transporter protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0049, 0.0036)\n","Processed SHH08690.1 type 2 lantibiotic, SP_1948 family [Butyrivibrio fibrisolvens DSM 3071 DSM 3071]: AMP (Probabilities: 0.1200, 0.8682, 0.0117)\n","Processed SFA53938.1 lantibiotic, gallidermin/nisin family [Parageobacillus thermantarcticus]: non-AMP/OTHERS (Probabilities: 0.9664, 0.0260, 0.0075)\n","Processed OJD57917.1 lantibiotic salivaricin A, partial [Bacillus sp. N35-10-4]: non-AMP/OTHERS (Probabilities: 0.9455, 0.0512, 0.0033)\n","Processed OII63764.1 lantibiotic protein [Streptomyces sp. CC77]: non-AMP/OTHERS (Probabilities: 0.9725, 0.0239, 0.0035)\n","Processed SFL42827.1 Lantibiotic alpha [Lachnospiraceae bacterium KH1T2]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0060, 0.0033)\n","Processed SDJ22508.1 Lantibiotic biosynthesis dehydratase C-term [Nonomuraea jiangxiensis]: non-AMP/OTHERS (Probabilities: 0.9769, 0.0059, 0.0171)\n","Processed SDC53842.1 lantibiotic, gallidermin/nisin family [Actinokineospora iranica]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0044, 0.0030)\n","Processed SDD41851.1 type 2 lantibiotic, SP_1948 family [Exiguobacterium enclense]: AMP (Probabilities: 0.1091, 0.8787, 0.0121)\n","Processed BAD74579.1 lantibiotic precursor [Geobacillus kaustophilus HTA426 HTA426]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0035, 0.0048)\n","Processed OEV30460.1 lantibiotic precursor [Streptomyces nanshensis]: Neuropeptide (Probabilities: 0.0323, 0.0158, 0.9519)\n","Processed OJE27707.1 lantibiotic cytolysin, partial [Bacillus thuringiensis]: AMP (Probabilities: 0.0930, 0.8948, 0.0123)\n","Processed ABQ49654.1 hypothetical protein [Staphylococcus aureus subsp. aureus JH9 JH9]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0059, 0.0027)\n","Processed SAO99659.1 lantibiotic ABC transporter protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0060, 0.0027)\n","Processed OEV07533.1 lantibiotic lacticin [Streptomyces nanshensis]: AMP (Probabilities: 0.0808, 0.9068, 0.0125)\n","Processed ODJ63176.1 lantibiotic lacticin [Brochothrix thermosphacta]: non-AMP/OTHERS (Probabilities: 0.9580, 0.0082, 0.0338)\n","Processed SCG09812.1 Lantibiotic alpha [Streptomyces sp. Ncost-T6T-2b]: non-AMP/OTHERS (Probabilities: 0.8773, 0.0200, 0.1027)\n","Processed OCR00530.1 mersacidin/lichenicidin family type 2 lantibiotic [Oscillatoriales cyanobacterium USR001]: non-AMP/OTHERS (Probabilities: 0.9789, 0.0088, 0.0123)\n","Processed AJH71620.1 putative lantibiotic cytolysin [Bacillus cereus ATCC 4342 ATCC 4342]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0036, 0.0073)\n","Processed AAL15567.1 lantibiotic ericin Sa [Bacillus subtilis]: AMP (Probabilities: 0.1959, 0.7921, 0.0120)\n","Processed SCU54605.1 lantibiotic modifying enzyme [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0034, 0.0034)\n","Processed ANB57459.1 type 2 lantibiotic, mersacidin/lichenicidin family protein [Anoxybacillus sp. B2M1]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0044, 0.0028)\n","Processed SCL44176.1 type 2 lantibiotic, mersacidin/lichenicidin family [Micromonospora citrea]: Neuropeptide (Probabilities: 0.3397, 0.1605, 0.4997)\n","Processed SAN45979.1 Lantibiotic transport ATP-binding protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0062, 0.0031)\n","Processed KXT73631.1 Protein involved in lantibiotic (srt) production / relaxase [Streptococcus sp. DD10]: non-AMP/OTHERS (Probabilities: 0.9878, 0.0034, 0.0088)\n","Processed KXO02963.1 lantibiotic cytolysin, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9929, 0.0037, 0.0034)\n","Processed CXS90711.1 Lantibiotic transport ATP-binding protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0033)\n","Processed CYB84144.1 Lantibiotic transport ATP-binding protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9885, 0.0086, 0.0029)\n","Processed SAN93872.1 lantibiotic ABC transporter protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0048, 0.0027)\n","Processed ANC27738.1 salivaricin lantibiotic precursor protein SalA [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0031, 0.0043)\n","Processed KXO02962.1 lantibiotic cytolysin, partial [Bacillus thuringiensis]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0038, 0.0038)\n","Processed CXP15165.1 Lantibiotic transport ATP-binding protein [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0036, 0.0031)\n","Processed CXH04982.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0047, 0.0030)\n","Processed CYK25284.1 Lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0042, 0.0032)\n","Processed CXG43177.1 lantibiotic export protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0058, 0.0032)\n","Processed CYK53058.1 lantibiotic leader peptide processing serine protease [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9900, 0.0068, 0.0031)\n","Processed CYZ88697.1 lantibiotic efflux protein [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9879, 0.0092, 0.0029)\n","Processed CYV71839.1 lantibiotic efflux protein [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9903, 0.0070, 0.0028)\n","Processed CYT82013.1 lantibiotic efflux protein [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0055, 0.0029)\n","Processed CVK73891.1 lantibiotic export protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0050, 0.0030)\n","Processed CYK83691.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0036, 0.0031)\n","Processed CVP61807.1 lantibiotic leader peptide processing serine protease [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0062, 0.0029)\n","Processed CVX44746.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0065, 0.0030)\n","Processed KWX88349.1 lantibiotic ABC transporter ATP-binding protein, partial [Paenibacillus riograndensis]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0050, 0.0028)\n","Processed CVK15321.1 Lantibiotic dehydratase, C terminus [Apibacter mensalis]: non-AMP/OTHERS (Probabilities: 0.9881, 0.0090, 0.0029)\n","Processed KQV12091.1 lantibiotic precursor [Kitasatospora sp. Root107]: non-AMP/OTHERS (Probabilities: 0.8634, 0.0777, 0.0589)\n","Processed CUH89921.1 type 2 lantibiotic biosynthesis protein LanM [Phaeobacter sp. CECT 5382]: non-AMP/OTHERS (Probabilities: 0.9884, 0.0084, 0.0032)\n","Processed CDR25809.1 lantibiotic protein [Staphylococcus schweitzeri]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0037, 0.0039)\n","Processed KPC81937.1 lantibiotic biosynthesis protein, partial [Streptomyces sp. NRRL WC-3753]: non-AMP/OTHERS (Probabilities: 0.9858, 0.0062, 0.0080)\n","Processed KPA61566.1 lantibiotic (srt) production protein, partial [Streptococcus suis]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0056, 0.0027)\n","Processed CDR21191.1 putative lantibiotic ABC transporter protein (fragment) [Staphylococcus argenteus]: non-AMP/OTHERS (Probabilities: 0.9891, 0.0082, 0.0027)\n","Processed KOO46502.1 lantibiotic lacticin [Priestia koreensis]: non-AMP/OTHERS (Probabilities: 0.9760, 0.0088, 0.0152)\n","Processed GAD15076.1 lantibiotic ABC transporter, permease [Geobacillus kaustophilus GBlys GBlys]: non-AMP/OTHERS (Probabilities: 0.9846, 0.0127, 0.0027)\n","Processed COQ61520.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9909, 0.0049, 0.0042)\n","Processed KOG87342.1 lantibiotic protein, partial [Streptomyces varsoviensis]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0035, 0.0037)\n","Processed KMS01292.1 lantibiotic ABC transporter ATP-binding protein, partial [Staphylococcus aureus]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0039, 0.0033)\n","Processed CJF78143.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0042, 0.0039)\n","Processed CJV61543.1 lantibiotic ABC transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0051, 0.0028)\n","Processed CJG39226.1 lantibiotic ABC transporter ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0044, 0.0034)\n","Processed CJH98636.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9908, 0.0048, 0.0044)\n","Processed CRH71363.1 lantibiotic protection ABC transporter%2C ATP-binding subunit [Chlamydia trachomatis]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0045, 0.0036)\n","Processed CIV31488.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0074, 0.0036)\n","Processed CKF99531.1 lantibiotic transport ATP-binding protein srtF [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0042, 0.0038)\n","Processed CEX43870.1 lantibiotic ABC transporter ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0041, 0.0033)\n","Processed CRG68836.1 lantibiotic efflux protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9895, 0.0078, 0.0027)\n","Processed CEW81709.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9824, 0.0104, 0.0072)\n","Processed CGF77759.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0062, 0.0033)\n","Processed CEX42554.1 lantibiotic ABC transporter ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9921, 0.0043, 0.0036)\n","Processed COE47912.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9862, 0.0100, 0.0037)\n","Processed CPS52596.1 type 2 lantibiotic biosynthesis protein LanM [Chlamydia trachomatis]: non-AMP/OTHERS (Probabilities: 0.9890, 0.0083, 0.0027)\n","Processed COD64146.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9914, 0.0051, 0.0035)\n","Processed CKE72091.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0037)\n","Processed CJI32508.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0050, 0.0044)\n","Processed CKA55057.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0056, 0.0033)\n","Processed CJB75595.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0034)\n","Processed CKG25777.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9907, 0.0051, 0.0042)\n","Processed CJD52543.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0051, 0.0036)\n","Processed CIV96852.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0051, 0.0029)\n","Processed CJC45358.1 lantibiotic ABC transporter ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0039)\n","Processed CJF15529.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9500, 0.0402, 0.0098)\n","Processed CIV96817.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9795, 0.0168, 0.0036)\n","Processed CIW23248.1 lantibiotic export protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9889, 0.0085, 0.0026)\n","Processed CIW07679.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0075, 0.0022)\n","Processed CIW24421.1 lantibiotic export protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9915, 0.0056, 0.0029)\n","Processed CJP81744.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9871, 0.0094, 0.0035)\n","Processed CJH63553.1 lantibiotic ABC transporter ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0044, 0.0033)\n","Processed CIV96832.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9924, 0.0046, 0.0030)\n","Processed CIV84789.1 lantibiotic synthetase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0034, 0.0041)\n","Processed CKG00101.1 lantibiotic ABC transporter ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0035, 0.0042)\n","Processed CJH57712.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9840, 0.0122, 0.0037)\n","Processed CGF60088.1 lantibiotic transport ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9117, 0.0165, 0.0718)\n","Processed CJA37713.1 truncated MobA family endonuclease relaxase [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0042, 0.0041)\n","Processed COE15343.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9784, 0.0166, 0.0051)\n","Processed CIW43188.1 lantibiotic ABC transporter [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0037)\n","Processed GAO21878.1 lantibiotic dehydratase domain protein [Alicycliphilus sp. B1]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0042, 0.0041)\n","Processed KKC19582.1 lantibiotic (srt) production protein, partial [Streptococcus dysgalactiae subsp. equisimilis UT-SS1069]: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0039)\n","Processed CRG03904.1 lantibiotic ABC transporter ATP-binding protein [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0034)\n","Processed BAQ51270.1 lantibiotic ABC superfamily ATP binding cassette transporter permease protein [Streptococcus pyogenes]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0049, 0.0031)\n","Processed EOP79153.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus BAG5X12-1 BAG5X12-1]: non-AMP/OTHERS (Probabilities: 0.9825, 0.0138, 0.0037)\n","Processed EOP01528.1 gallidermin/nisin family lantibiotic, partial [Bacillus cereus MC118 MC118]: non-AMP/OTHERS (Probabilities: 0.9878, 0.0098, 0.0024)\n","Processed ERH18710.1 lantibiotic, gallidermin/nisin family [Actinobaculum sp. oral taxon 183 str. F0552 F0552]: AMP (Probabilities: 0.0810, 0.8988, 0.0202)\n","Processed EOO05426.1 type 2 lantibiotic [Bacillus cereus str. Schrouff Schrouff]: non-AMP/OTHERS (Probabilities: 0.9816, 0.0055, 0.0129)\n","Processed EJS16086.1 type 2 lantibiotic [Bacillus cereus VDM062 VDM062]: non-AMP/OTHERS (Probabilities: 0.9918, 0.0036, 0.0047)\n","Processed KJJ41464.1 lantibiotic lacticin 3147 A1 [Bacillus subtilis]: non-AMP/OTHERS (Probabilities: 0.8512, 0.1410, 0.0078)\n","Processed EGF56602.1 lantibiotic mutacin-2 [Actinomyces sp. oral taxon 170 str. F0386 F0386]: AMP (Probabilities: 0.3012, 0.3931, 0.3057)\n","Processed EEZ78766.1 lantibiotic, gallidermin/nisin family [Actinomyces sp. oral taxon 848 str. F0332 F0332]: AMP (Probabilities: 0.0647, 0.9209, 0.0144)\n","Processed KIR10353.1 lantibiotic ABC transporter permease, partial [Staphylococcus gallinarum]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0056, 0.0025)\n","Processed COP26862.1 lantibiotic modifying enzyme [Streptococcus pneumoniae]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0080, 0.0023)\n","Processed EXY33061.1 lantibiotic biosynthesis domain protein, partial [Bacteroides fragilis str. 3397 T10 3397 T10]: non-AMP/OTHERS (Probabilities: 0.9897, 0.0075, 0.0029)\n","Processed KFM89805.1 putative lantibiotic cytolysin [Bacillus cereus ATCC 4342 ATCC 4342]: non-AMP/OTHERS (Probabilities: 0.9932, 0.0030, 0.0038)\n","Processed KEP71914.1 lantibiotic ABC transporter ATP-binding protein, partial [Microbacterium sp. SUBG005]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0044, 0.0039)\n","Processed EZI22806.1 lantibiotic, gallidermin/nisin family [Staphylococcus aureus subsp. aureus CO-85 CO-85]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0033, 0.0042)\n","Processed AGU55491.1 lantibiotic ABC transporter protein, putative [Staphylococcus aureus subsp. aureus 6850 6850]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0051, 0.0026)\n","Processed ABF32110.1 lantibiotic transport permease protein [Streptococcus pyogenes MGAS9429 MGAS9429]: non-AMP/OTHERS (Probabilities: 0.7938, 0.1996, 0.0066)\n","Processed KEQ33839.1 lantibiotic streptin immunity family protein [Streptococcus mitis]: non-AMP/OTHERS (Probabilities: 0.9912, 0.0052, 0.0036)\n","Processed EIF37885.1 hypothetical protein [Streptococcus sp. SK643]: non-AMP/OTHERS (Probabilities: 0.9917, 0.0034, 0.0049)\n","Processed ADA64985.1 Nisin precursor, C-terminal fragment [Lactococcus lactis subsp. lactis KF147 KF147]: AMP (Probabilities: 0.1161, 0.8718, 0.0121)\n","Processed KEQ41663.1 type-A lantibiotic family protein [Streptococcus mitis]: non-AMP/OTHERS (Probabilities: 0.9846, 0.0038, 0.0116)\n","Processed ETJ38849.1 MutG family lantibiotic protection ABC superfamily ATP binding cassette transporter permease, partial [human gut metagenome]: AMP (Probabilities: 0.3346, 0.6557, 0.0097)\n","Processed  unnamed protein product [Unknown]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0052, 0.0035)\n","Processed EGF18908.1 lantibiotic ABC superfamily ATP binding cassette transporter permease protein [Streptococcus sanguinis SK408 SK408]: non-AMP/OTHERS (Probabilities: 0.9904, 0.0071, 0.0025)\n","Processed EQJ61331.1 putative lantibiotic ABC transporter, permease domain protein [Clostridioides difficile P32 P32]: non-AMP/OTHERS (Probabilities: 0.8994, 0.0955, 0.0050)\n","Processed EQJ04104.1 putative lantibiotic ABC transporter, permease domain protein, partial [Clostridioides difficile P6 P6]: non-AMP/OTHERS (Probabilities: 0.9913, 0.0060, 0.0027)\n","Processed AAT87775.1 Lantibiotic salivaricin A [Streptococcus pyogenes MGAS10394 MGAS10394]: non-AMP/OTHERS (Probabilities: 0.9934, 0.0033, 0.0033)\n","Processed EUA92428.1 lantibiotic modifying enzyme domain protein [Mycobacterium ulcerans str. Harvey Harvey]: non-AMP/OTHERS (Probabilities: 0.9911, 0.0053, 0.0035)\n","Processed ETJ39238.1 MutG family lantibiotic protection ABC superfamily ATP binding cassette transporter permease, partial [human gut metagenome]: non-AMP/OTHERS (Probabilities: 0.9925, 0.0043, 0.0032)\n","Processed EQE69435.1 putative lantibiotic ABC transporter, permease domain protein, partial [Clostridioides difficile CD44 CD44]: non-AMP/OTHERS (Probabilities: 0.9920, 0.0052, 0.0028)\n","Processed EPZ61722.1 putative lantibiotic biosynthesis regulatory protein [Paeniclostridium sordellii ATCC 9714 ATCC 9714]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0048, 0.0030)\n","Processed EPU25318.1 lantibiotic streptin [Streptococcus agalactiae LMG 14609 LMG 14609]: non-AMP/OTHERS (Probabilities: 0.9873, 0.0091, 0.0036)\n","Processed EPC76021.1 Lantibiotic efflux protein, partial [Lacticaseibacillus paracasei subsp. paracasei Lpp71 Lpp71]: non-AMP/OTHERS (Probabilities: 0.9876, 0.0047, 0.0077)\n","Processed EFQ54446.1 lantibiotic system transcriptional regulator, partial [Streptococcus parasanguinis F0405 F0405]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0032, 0.0035)\n","Processed EOR32047.1 lantibiotic epidermin biosynthesis protein EpiB, partial [Staphylococcus aureus subsp. aureus 103564 103564]: non-AMP/OTHERS (Probabilities: 0.9933, 0.0039, 0.0028)\n","Processed EMZ23840.1 type 2 lantibiotic [Eubacterium plexicaudatum ASF492 ASF492]: Neuropeptide (Probabilities: 0.3276, 0.0245, 0.6479)\n","Processed EMP60925.1 lantibiotic precursor peptide [Streptococcus mutans KK23 KK23]: non-AMP/OTHERS (Probabilities: 0.9803, 0.0094, 0.0103)\n","Processed EPZ57426.1 putative lantibiotic ABC transporter, ATP-binding protein [Paeniclostridium sordellii ATCC 9714 ATCC 9714]: non-AMP/OTHERS (Probabilities: 0.9844, 0.0078, 0.0078)\n","Processed EOH96631.1 gallidermin/nisin family lantibiotic, partial [Enterococcus moraviensis ATCC BAA-383 ATCC BAA-383]: non-AMP/OTHERS (Probabilities: 0.9824, 0.0146, 0.0030)\n","Processed EOB24898.1 lantibiotic mersacidin transporter system, partial [Streptococcus pneumoniae 357 357]: non-AMP/OTHERS (Probabilities: 0.9922, 0.0049, 0.0029)\n","Processed EMP60923.1 lantibiotic precursor peptide [Streptococcus mutans KK23 KK23]: non-AMP/OTHERS (Probabilities: 0.9864, 0.0062, 0.0074)\n","Processed EHD46927.1 lantibiotic modifying enzyme domain protein [Streptococcus pneumoniae GA44452 GA44452]: non-AMP/OTHERS (Probabilities: 0.9902, 0.0074, 0.0024)\n","Processed EFL65044.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae BS455 BS455]: non-AMP/OTHERS (Probabilities: 0.9847, 0.0114, 0.0040)\n","Processed ELW76278.1 lantibiotic efflux protein, partial [Streptococcus thermophilus MTCC 5461 MTCC 5461]: non-AMP/OTHERS (Probabilities: 0.9926, 0.0048, 0.0026)\n","Processed AFX69055.1 lantibiotic mersacidin, partial [Bacillus subtilis]: non-AMP/OTHERS (Probabilities: 0.5881, 0.0219, 0.3900)\n","Processed EKC80082.1 lantibiotic ABC transporter, permease protein, partial [human gut metagenome]: non-AMP/OTHERS (Probabilities: 0.9919, 0.0055, 0.0026)\n","Processed EKC48343.1 lantibiotic ABC transporter, permease protein, partial [human gut metagenome]: AMP (Probabilities: 0.4694, 0.5198, 0.0107)\n","Processed AFX69057.1 lantibiotic ericin, partial [Bacillus subtilis]: non-AMP/OTHERS (Probabilities: 0.9906, 0.0065, 0.0029)\n","Processed EKC45266.1 lantibiotic ABC superfamily ATP binding cassette transporter, membrane protein, partial [human gut metagenome]: non-AMP/OTHERS (Probabilities: 0.9855, 0.0118, 0.0028)\n","Processed EHD55949.1 lantibiotic mersacidin modifying enzyme [Streptococcus pneumoniae NP070 NP070]: non-AMP/OTHERS (Probabilities: 0.9916, 0.0051, 0.0033)\n","Processed EKU40925.1 lantibiotic ABC transporter ATP-binding protein [Lysinibacillus fusiformis ZB2 ZB2]: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0033)\n","Processed EFB97951.1 lantibiotic precursor, partial [Staphylococcus aureus A9765 A9765]: non-AMP/OTHERS (Probabilities: 0.6094, 0.1058, 0.2848)\n","Processed JC6527 lantibiotic mutacin II protein MutM [Streptococcus mutans]: non-AMP/OTHERS (Probabilities: 0.9923, 0.0042, 0.0035)\n","Processed EFI65092.1 lantibiotic modifying enzyme [Bacillus cereus SJ1 SJ1]: non-AMP/OTHERS (Probabilities: 0.9645, 0.0267, 0.0089)\n"]}]},{"cell_type":"markdown","source":["## SELECCIÓN DE LAS SECUENCIAS CON POTENCIAL NEUROPEPTÍDICO POSITIVO\n"],"metadata":{"id":"6V3JzlughCiu"}},{"cell_type":"code","source":["# Función para leer un archivo de predicciones y filtrar aquellos con etiqueta 'Neuropeptide'\n","def print_neuropeptides(file_path):\n","    i=0\n","    with open(file_path, 'r') as f:\n","        next(f)  # Saltar la primera línea (encabezado)\n","        for line in f:\n","            columns = line.strip().split(\"\\t\")  # Dividir las columnas por tabulador\n","            identifier, sequence, prediction, prob_class1, prob_class2, prob_class3 = columns\n","            # Imprimir solo aquellas líneas con la etiqueta 'Neuropeptide'\n","            if prediction == 'Neuropeptide':\n","                print(f\"Identifier: {identifier}, Sequence: {sequence}, Probabilities: {prob_class3}\")\n","                i=i+1\n","        print(f\"\\nEL NÚMERO DE SECUENCIAS CON POTENCIAL NEUROPEPTÍDICO ES: {i}\")\n","# Ruta del archivo de resultados con predicciones\n","result_file = '/content/drive/MyDrive/Datos Finales para Memoria/resultados/RESULTADO_prediccion_bacteriocinas.tsv'  # Archivo de salida generado anteriormente\n","\n","# Llamar a la función para imprimir los neuropeptidos\n","print_neuropeptides(result_file)\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"tdnzIlpms3Wn","executionInfo":{"status":"ok","timestamp":1733928712994,"user_tz":-60,"elapsed":443,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"649e62d0-3b05-46d2-bace-b19a91c147bf"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Identifier: 116.1;SGR_1514_putative_Linaridin, Sequence: MSMSPTPAALRGAGGLSESDPGRALSSLAPVTATPGVVAGVALGVALVNAFAAGYNHCGGNVELPM, Probabilities: 0.7664\n","Identifier: 13.1;Butyrivibriocin, Sequence: MNKDLNALTNPIDEKELEQILGGGDGVFRTISHECHMNTWMFIFTCCS, Probabilities: 0.3600\n","Identifier: 133.1;Thiomuracin_A_(Thiomuracin_B)_(Thiomuracin_C)_(Thiomuracin_D)_(Thiomuracin_E)_(Thiomuracin_F)_(Thiomuracin_G)_(Thiomuracin_H)_(Thiomuracin_I), Sequence: MDLSDLPMDVFELADDGVAVESLTAGHGMTEVGASCNCFCYICCSCSSA, Probabilities: 0.5838\n","Identifier: 139.1;Trunkamide, Sequence: MNKKNILPQLGQPVIRLTAGQLSSQLAELSEEALGGVDASTSIAPFCSYDGVDASTSIAPFCSYDGVDASTSIAPFCSYDD, Probabilities: 0.7242\n","Identifier: 144.1;Thiocillin_GE2270, Sequence: SCNCVCGFCCSCSP, Probabilities: 0.8699\n","Identifier: 15.1;Catenulipeptin, Sequence: MTEEMTLLDLQGMEQTETDSWGGSGHGGGGDSGLSVTGCNGHSGISLLCDL, Probabilities: 0.7596\n","Identifier: 197.1;Planosporicin, Sequence: MGISSPALPQNTADLFQLDLEIGVEQSLASPAITSVSWCTPGCTSEGGGSGCSHCC, Probabilities: 0.6302\n","Identifier: 203.1;Prochlorosin 4.3, Sequence: MSEEQLKAFIAKVQADTSLQEQLKAEGADVVAIAKAAGFTITTEDLNSHRQNLTDDELEGVAGGTASGGCDTSMFCY, Probabilities: 0.4875\n","Identifier: 207.1;Stackepeptin A, Sequence: MALLDLQGLESPADLSATRGGSSGSGHSCPSNLSATLCGGTSGLSVLICH, Probabilities: 0.9722\n","Identifier: 208.1;Stackepeptin B, Sequence: MALLDLQGLESPADLSATRGSSGSGHSCPSNLSATLCGGTSGLSVLICH, Probabilities: 0.9720\n","Identifier: 209.1;Stackepeptin C, Sequence: MALLDLQGLESPADLSATRGGSSGSGHSCPSNLSATLCGGTSGLSVLICH, Probabilities: 0.9722\n","Identifier: 210.1;Stackepeptin D, Sequence: MALLDLQGLESPADLSATRGGSSGSGHSCPSNLSATLCGGTSGLSVLICH, Probabilities: 0.9722\n","Identifier: 211.1;Bottromycin B2, Sequence: MGPVVVFDCMTADFLNDDPNNAELSALEMEELESWGAWDGEATS, Probabilities: 0.9741\n","Identifier: 212.1;Bottromycin D2, Sequence: MGPVVVFDCMTADFLNDDPNNAELSALEMEELESWGAWDGEATS, Probabilities: 0.9741\n","Identifier: 213.1;Bottromycin E2, Sequence: MGPVVVFDCMTADFLNDDPNNAELSALEMEELESWGAWDGEATS, Probabilities: 0.9741\n","Identifier: 214.1;Venturamide A, Sequence: ATACVC, Probabilities: 0.9831\n","Identifier: 220.1;Cyclothiazomycin, Sequence: MEKELVLDLADLSVDELDVLPTSPGAGLESINVGHAMVEIGASNCTSTGTPASCCSCCCC, Probabilities: 0.9482\n","Identifier: 221.1;GE2270_A_(GE2270_B1)_(GE2270_B2)_(GE2270_C1)_(GE2270_C2a)_(GE2270_C2b)_(GE2270_D1)_(GE2270_D2)_(GE2270_E)_(GE2270_T), Sequence: MSELESKLNLSDLPMDVFEMADSGMEVESLTAGHGMPEVGASCNCVCGFCCSCSPSA, Probabilities: 0.8193\n","Identifier: 23.1;Enterocin_W_alfa, Sequence: MKKEELVGMAKEDFLNVICENDNKLENSGAKCPWWNLSCHLGNDGKICTYSHECTAGCNA, Probabilities: 0.8804\n","Identifier: 231.1;Amythiamicin_D, Sequence: SCNCVCGVCCSC, Probabilities: 0.9788\n","Identifier: 245.1;Promothiocin_A, Sequence: SCVGSACASSS, Probabilities: 0.9701\n","Identifier: 246.1;Promothiocin_B, Sequence: SCVGSACASSSSS, Probabilities: 0.4998\n","Identifier: 257.1;68-1147, Sequence: MSISVDPQTELGVEELTGLDIDTLEISDYLDESMLDTHDLTETMIASASCTTCICTCSCSS, Probabilities: 0.9102\n","Identifier: 265.1;Chaxapeptin, Sequence: MEPQMTELQPEAYEAPSLIEVGEFSEDTLGFGSKPLDSFGLNFF, Probabilities: 0.9514\n","Identifier: 267.1;Burhizin, Sequence: MNKQQQESGLLLAEESLMELCASSETLGGAGQYKEVEAGRWSDRIDSDDE, Probabilities: 0.9720\n","Identifier: 270.1;Cyclothiazomycin_B, Sequence: MEAKLELDLGDLSVEELDVLPTSPGAGLESINVGHAMVEIGASNCTSRGTPASCCSCCCC, Probabilities: 0.9674\n","Identifier: 271.1;Cyclothiazomycin_C, Sequence: MDANLVLDLDDLSVDQLDILPTAPGSTLESINVGHAMVEIGASNCTSKGSPASCCSCCCC, Probabilities: 0.7863\n","Identifier: 275.1;Lactazole, Sequence: MSDITASRVESLDLQDLDLSELTVTSLRDTVALPENGASWGSCSCQASSSCAQPQDM, Probabilities: 0.7957\n","Identifier: 276.1;Lactocillin, Sequence: MKGGVKMKDLFDIDEKSLENLDLGDFKILDSEDLSEKDEKSILGASCTTCTCCCSCCA, Probabilities: 0.9144\n","Identifier: 278.1;Marinostatin, Sequence: MKTTPFFANLLASQTRELTENELEMTAGGTASQQSPVQEVPEQPFATMRYPSDSDEDGFNFPV, Probabilities: 0.8542\n","Identifier: 279.1;Microcin_C7, Sequence: MRTGNAD, Probabilities: 0.9851\n","Identifier: 298.1;Thiomuracins, Sequence: MDLSDLPMDVFELADDGVAVESLTAGHGMTEVGASCNCFCYICCSCSSA, Probabilities: 0.5838\n","Identifier: 302.1;Syanodin, Sequence: MERNSEDRRDDVVELGAVSVETKGISGGTVDAPAGQGLAGILDD, Probabilities: 0.9576\n","Identifier: 303.1;Caulonodin_I, Sequence: MERIEDHIDDELIDLGAASVETQGDVLNAPEPGIGREPTGLSRD, Probabilities: 0.9833\n","Identifier: 305.1;Caulonodin_III, Sequence: MEFEGIPSPDARIDLGLASEETCGQIYDHPEVGIGAYGCEGLQR, Probabilities: 0.9809\n","Identifier: 306.1;Rubrivinodin, Sequence: MKEFAMDEELELEIVDLGDAKELTQGAPSLINSEDNPAFPQRV, Probabilities: 0.8647\n","Identifier: 308.1;Sphingonodin_II, Sequence: MDRHDNSEVDEIIDLGTASAVTQGMGSGSTDQNGQPKNLIGGISDD, Probabilities: 0.9795\n","Identifier: 313.1;Caulosegnin_III, Sequence: MTSRFQLLRLGKADRLTRGALVGLLLEDITVARYDPM, Probabilities: 0.9041\n","Identifier: 32.1;Erythreapeptin, Sequence: MEMVLELQELDAPNELAYGDPSHGGGSNLSLLASCANSTVSLLTCH, Probabilities: 0.5094\n","Identifier: 326.1;Astexin_3, Sequence: MRTYNRSLPARAGLTDLGKVTTHTKGPTPMVGLDSVSGQYWDQHAPLAD, Probabilities: 0.9803\n","Identifier: 329.1;Caulonodin_V, Sequence: MTQVSPSPLRLIRVGRALDLTRSIGDSGLRESMSSQTYWP, Probabilities: 0.7873\n","Identifier: 334.1;Paeninodin, Sequence: AGPGTSTPDAFQPDPDEDVHYDS, Probabilities: 0.9761\n","Identifier: 337.1;RES_701_1_(RES_701_2), Sequence: GNWHGTAPDWFFNYYW, Probabilities: 0.9832\n","Identifier: 340.1;xanthomonin_II, Sequence: MDTSNNDARTTALDQDLIVLGVASLDTQGGPLAGEEMGGITTLGISQD, Probabilities: 0.9485\n","Identifier: 343.1;Sungsanpin, Sequence: GFGSKPIDSFGLSWL, Probabilities: 0.9202\n","Identifier: 346.1;Carnocin_UI49, Sequence: GSEIQPR, Probabilities: 0.9852\n","Identifier: 348.1;Aerucyclamide_A_McaE1_(Aerucyclamide_B_McaE1)_(Aerucyclamide_C_McaE1)_(Aerucyclamide_D_McaE1), Sequence: MRITPMDKKNLLPNQGAPVIRGISGKLPSHLAELSEEALGGNGAEASATVSICAFDGAEASFTGCMCAFDGAEASITGCICAFDGAEASITGCICAFDGDEA, Probabilities: 0.8382\n","Identifier: 349.1;aeruginosamide_B_AgeE, Sequence: MDKKNILPHQGKPVLRTTNGKLPSHLAELSEEALGGAGMDASFFPCSYDGADASFFPVCSYDGADASFFPCSYDDGDA, Probabilities: 0.9062\n","Identifier: 350.1;aeruginosamide_C_AgeE, Sequence: MDKKNILPHQGKPVLRTTNGKLPSHLAELSEEALGGAGMDASFFPCSYDGADASFFPVCSYDGADASFFPCSYDDGDA, Probabilities: 0.9062\n","Identifier: 351.1;Aesturamide_A_LynE_(Aesturamide_B_LynE)_(Aesturamide_C_LynE)_(Aesturamide_D_LynE)_(Aesturamide_E_LynE)_(Aesturamide_F_LynE), Sequence: MDKKNILPHQGKPVLRTTNGKLPSHLAELSEEALGGNGVDASACMPCYPSYDGVDASVCMPCYPSYDGVDASVCMPCYPSYDDAE, Probabilities: 0.5800\n","Identifier: 352.1;Aesturamide_G_LynE, Sequence: MDKKNILPHQGKPVLRTTNGKLPSHLAELSEEALGGNGVDASACMPCYPSYDGVDASVCMPCYPSYDGVDASVCMPCYPSYDDAE, Probabilities: 0.5800\n","Identifier: 353.1;Aesturamide_H_LynE_(Aesturamide_I_LynE)_(Aesturamide_J_LynE)_(Aesturamide_K_LynE)_(Aesturamide_L_LynE), Sequence: MDKKNILPHQGKPVRLTTNGKLPSHLAELSEEALGGNGVDASACMPCYPSYDGVDASVCMPCYPSYDGVDASVCMPCYPSYDDAE, Probabilities: 0.6165\n","Identifier: 354.1;Agardhipeptin_A, Sequence: HGWPWGL, Probabilities: 0.9848\n","Identifier: 355.1;Agardhipeptin_B, Sequence: WLPWAPWV, Probabilities: 0.9851\n","Identifier: 371.1;arthrospiramides_A_ArtE3, Sequence: MDKKNITPNPQQPVDRVPSGQLPSALAELSEEAIGSLALASGLPGMPCFCSYEGDDE, Probabilities: 0.8295\n","Identifier: 372.1;arthrospiramides_B_ArtE5, Sequence: MDKKNISPNPQQPVDRVPSGQLPSALAELSEEAIGSLALASGLVLMPCFCSYEGDDE, Probabilities: 0.7871\n","Identifier: 374.1;Axinastatin_1, Sequence: FVVPVNP, Probabilities: 0.9851\n","Identifier: 375.1;Axinastatin_2, Sequence: FVLPVNP, Probabilities: 0.9851\n","Identifier: 376.1;Axinastatin_3, Sequence: FILPVNP, Probabilities: 0.9851\n","Identifier: 377.1;Axinellin_C, Sequence: TVPTPFPL, Probabilities: 0.9850\n","Identifier: 382.1;Bistratamide_C, Sequence: VSACVC, Probabilities: 0.9829\n","Identifier: 383.1;Bistratamide_D_(Bistratamide_F)_(Bistratamide_G)_(Bistratamide_I), Sequence: VTVSVC, Probabilities: 0.9851\n","Identifier: 385.1;Comoramide_A, Sequence: ITFTAC, Probabilities: 0.9851\n","Identifier: 386.1;Cyanothecamide_A_ThcE2, Sequence: MDLQNLLPQQANPIHRTTTGQLPAELAELSDETLCNSGVLPSGFEAPVLYKDQCSYEGDDAE, Probabilities: 0.9574\n","Identifier: 387.1;Cyanothecamide_B_ThcE2, Sequence: MDLQNLLPQQANPIHRTTTGQLPAELAELSDETLCNSGVLPSGFEAPVLYKDQCSYEGDDAE, Probabilities: 0.9574\n","Identifier: 390.1;Cyclodidemnamide_B, Sequence: ITVPCLC, Probabilities: 0.9850\n","Identifier: 391.1;Cyclonellin, Sequence: YTANPRYP, Probabilities: 0.9851\n","Identifier: 397.1;Dolastatin_3, Sequence: VPLQCGC, Probabilities: 0.9851\n","Identifier: 398.1;Dolastatin_E, Sequence: ASACIC, Probabilities: 0.9842\n","Identifier: 399.1;Dolastatin_I, Sequence: ITVSAC, Probabilities: 0.6133\n","Identifier: 400.1;Haliclonamide_A_(Haliclonamide_B)_(Haliclonamide_D)_(Haliclonamide_E), Sequence: PASYPTIP, Probabilities: 0.9850\n","Identifier: 401.1;Hexamollamide, Sequence: VVCTFP, Probabilities: 0.9850\n","Identifier: 402.1;Homodolastatin_3, Sequence: IPLQCGC, Probabilities: 0.9850\n","Identifier: 403.1;Hymenamide_A, Sequence: VPFWRPP, Probabilities: 0.9851\n","Identifier: 404.1;Hymenamide_B, Sequence: NFVEFPP, Probabilities: 0.9850\n","Identifier: 405.1;Hymenamide_C, Sequence: FGPELWP, Probabilities: 0.9850\n","Identifier: 406.1;Hymenamide_D, Sequence: YDPLAIP, Probabilities: 0.9852\n","Identifier: 407.1;Hymenamide_E, Sequence: TTPYFFP, Probabilities: 0.9851\n","Identifier: 408.1;Hymenamide_F, Sequence: AVMLRPP, Probabilities: 0.9849\n","Identifier: 409.1;Hymenamide_G, Sequence: YVPLILPP, Probabilities: 0.9851\n","Identifier: 410.1;Hymenamide_H, Sequence: LPWVPLTP, Probabilities: 0.9850\n","Identifier: 411.1;Hymenamide_J, Sequence: YDFWKVYP, Probabilities: 0.9850\n","Identifier: 412.1;Hymenamide_K, Sequence: YDFWKAVP, Probabilities: 0.9852\n","Identifier: 414.1;Kawaguchipeptin_B, Sequence: WLNGDNNWSTP, Probabilities: 0.9853\n","Identifier: 415.1;Kororamide, Sequence: LYCNPSLC, Probabilities: 0.9852\n","Identifier: 416.1;Lissoclinamide_1, Sequence: ICFPTVC, Probabilities: 0.4805\n","Identifier: 418.1;Lissoclinamide_4_(Lissoclinamide_5)_(Lissoclinamide_7)_(Lissoclinamide_8), Sequence: FCFPTVC, Probabilities: 0.9824\n","Identifier: 419.1;Lissoclinamide_9, Sequence: VCFPTIC, Probabilities: 0.6955\n","Identifier: 424.1;microcyclamide_McaE2, Sequence: MDKKNILPQQGKPVFRTTTGKLPSYLAELSEEALGGNGLEASHCATICAFDGAEASHCATICAFDGDEA, Probabilities: 0.9511\n","Identifier: 425.1;Microphycin_AL828, Sequence: EIGVYGLP, Probabilities: 0.9852\n","Identifier: 426.1;Mollamide, Sequence: IPISFPC, Probabilities: 0.9850\n","Identifier: 427.1;Mollamide_B, Sequence: VFPTVC, Probabilities: 0.9851\n","Identifier: 428.1;Mollamide_C, Sequence: IPGSLC, Probabilities: 0.9848\n","Identifier: 429.1;Muscoride_A, Sequence: VPTT, Probabilities: 0.9848\n","Identifier: 432.1;Nostocyclamide, Sequence: ATGCVC, Probabilities: 0.9851\n","Identifier: 433.1;Nostocyclamide_M, Sequence: ATGCMC, Probabilities: 0.9852\n","Identifier: 434.1;Oscillacyclin, Sequence: FTTSIAYNP, Probabilities: 0.9836\n","Identifier: 436.1;patellamide_A_patE4, Sequence: MNKKNILPQQGQPVIRLTAGQLSSQLAELSEEALGDAGLEASLTACITFCAYDGVEPSITVCISVCAYDGE, Probabilities: 0.4953\n","Identifier: 437.1;patellamide_B_patE4, Sequence: MNKKNILPQQGQPVIRLTAGQLSSQLAELSEEALGDAGLEASLTACITFCAYDGVEPSITVCISVCAYDGE, Probabilities: 0.4953\n","Identifier: 438.1;patellamide_B_patE5, Sequence: MNKKNILPQQGQPVIRLTAGQLSSQLAELSEEALGDAGLEASLTACITFCAYDGVEPSCTLCCTLCAYDGE, Probabilities: 0.6681\n","Identifier: 445.1;Patellin_1, Sequence: TVPTLC, Probabilities: 0.9850\n","Identifier: 448.1;Patellin_4, Sequence: TLPVPTVC, Probabilities: 0.9776\n","Identifier: 449.1;Patellin_5, Sequence: TVPVPSFC, Probabilities: 0.9851\n","Identifier: 45.1;Labyrinthopeptin_A1/A3, Sequence: MASILELQDLEVERASSAADSNASVWECCSTGSWVPFTCC, Probabilities: 0.5336\n","Identifier: 451.1;Phakellistatin_1, Sequence: YPIPIFP, Probabilities: 0.9850\n","Identifier: 453.1;Phakellistatin_13, Sequence: LWPFGPT, Probabilities: 0.9850\n","Identifier: 454.1;Phakellistatin_2, Sequence: YPFPIIP, Probabilities: 0.9850\n","Identifier: 455.1;Phakellistatin_7, Sequence: YIPPIFALPP, Probabilities: 0.5640\n","Identifier: 458.1;Planktocyclin, Sequence: GLVMFGVP, Probabilities: 0.9850\n","Identifier: 46.1;Labyrinthopeptin_A2, Sequence: MASILELQNLDVEHARGENRSDWSLWECCSTGSLFACC, Probabilities: 0.9363\n","Identifier: 460.1;Prenylagaramide_A, Sequence: YGTGEFFNP, Probabilities: 0.9850\n","Identifier: 467.1;Raocyclamide_A_(Raocyclamide_B), Sequence: FSASIC, Probabilities: 0.9839\n","Identifier: 468.1;Stylisin_1, Sequence: IFPLPYP, Probabilities: 0.9851\n","Identifier: 469.1;Stylisin_2, Sequence: FPIPYPP, Probabilities: 0.9851\n","Identifier: 47.1;Lacticin_3147_A1, Sequence: MNKNEIETQPVTWLEEVSDQNFDEDVFGACSTNTFSLSDYWGNNGAWCTLTHECMAWCK, Probabilities: 0.9522\n","Identifier: 470.1;Stylissamide_A, Sequence: VYPYKPP, Probabilities: 0.9849\n","Identifier: 471.1;Stylissamide_B, Sequence: IYPFPPP, Probabilities: 0.9851\n","Identifier: 472.1;Stylissamide_C, Sequence: FIPYPFP, Probabilities: 0.9850\n","Identifier: 473.1;Stylissamide_D, Sequence: FIPYPLP, Probabilities: 0.9851\n","Identifier: 476.1;Tenuecyclamide_C, Sequence: ATGCMC, Probabilities: 0.9852\n","Identifier: 478.1;Tenuecyclamide_D, Sequence: ATGCMC, Probabilities: 0.9852\n","Identifier: 479.1;ThcE4, Sequence: MDLQNLLPQQSQPIQRATAGQLPTELAELTEEALNNESAVLASSCDCSLYGGCESCSYEGDEAE, Probabilities: 0.9637\n","Identifier: 482.1;trunkamide_TruE3, Sequence: MNKKNILPQLGQPVIRLTAGQLSSQLAELSEEALGGVDASTSIAPFCSYDGVDASTSIAPFCSYDGVDASTSIAPFCSYDD, Probabilities: 0.7242\n","Identifier: 485.1;ulithiacyclamide_patE5, Sequence: MNKKNILPQQGQPVIRLTAGQLSSQLAELSEEALGDAGLEASLTACITFCAYDGVEPSCTLCCTLCAYDGE, Probabilities: 0.6681\n","Identifier: 491.1;Lantibiotic nukacin, Sequence: MENSKIMKDIEVANLLEEVQEDELNEVLGAKKKSGVIPTVSHDCHMNTFQFMFTCCS, Probabilities: 0.8331\n","Identifier: 62.1;Mersacidin, Sequence: MSQEAIIRSWKDPFSRENSTQNPAGNPFSELKEAQMDKLVGAGDMEAACTFTLPGGGGVCTLTSECIC, Probabilities: 0.9652\n","Identifier: 64.1;Microbisporicin_(NAI-107)_(Microbisporicin_A1)_(Microbisporicin_A2), Sequence: MPADILETRTSETEDLLDLDLSIGVEEITAGPAVTSWSLCTPGCTSPGGGSNCSFCC, Probabilities: 0.9208\n","Identifier: 71.1;Microcyclamide, Sequence: MRITPMDKKNILPQQGKPVFRTTTGKLPSYLAELSEEALGGNGLEASHCATICAFDGAEASHCATICAFDGDEA, Probabilities: 0.8011\n","Identifier: 8.1;Bottromycin_A2, Sequence: MGPAVVFDCMTADFLNDDPNNAELSSLEMEELESWGAWSDDTDQSV, Probabilities: 0.9600\n","Identifier: 82.1;Nmul_A1818_putative_Linaridin, Sequence: MMTKLAEAELAGLDAVIEARRMTSDTSDKYEQIVIPIGNAGTAVAGDNAGLSGGEGLSLGVLMKLRENALS, Probabilities: 0.8598\n","Identifier: 83.1;Nosiheptide, Sequence: MDAAHLSDLDIDALEISEFLDESRLEDSEVVAKVMSASCTTCECCCSCSS, Probabilities: 0.9773\n","Identifier: 87.1;Patellamide_B, Sequence: MNKKNILPQQGQPVIRLTAGQLSSQLAELSEEALGDAGLEASLTACITFCAYDGVEPSCTLCCTLCAYDGE, Probabilities: 0.6681\n","Identifier: 9.1;Bottromycin_D, Sequence: MGPVVVFDCMTADFLNDDPNNAELSALEMEELESWGAWDGEATS, Probabilities: 0.9741\n","Identifier: 96.1;putative_lantibiotic, Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHLFCSFGCEKTGSFNSFCC, Probabilities: 0.9670\n","Identifier: 489.1;Pinensin_01, Sequence: MNNSMDKLKLNLEDLQLESFVTSIDSEVAKRLSGGLGNVSEPTHTVPTDDHHPTEQVICTTVQCLGNVSEPTHTVPTDDHHPTEQVICTTVQC, Probabilities: 0.6485\n","Identifier: 490.1;Pinensin_02, Sequence: MNNSMDKLKLNLEDLQLESETTSIDSEVAKRLSGGLGNVSEPTHTVPTDDHHPIEQVICTTVQCLGNVSEPTHTVPTDDHHPIEQVICTTVQC, Probabilities: 0.9189\n","Identifier: 491.1;Pinensin_03, Sequence: MNNSLDKLELNLEDLQLESFVTSIDSEVANRLSGGLGNVSEPTHTEPTDDHHTPPVKCTTVVCLGNVSEPTHTEPTDDHHTPPVKCTTVVC, Probabilities: 0.6370\n","Identifier: 494.1;Pinensin_06, Sequence: MKNHAEHLKLNLEDLQLESFVTSIDSEVASRLAGGLGNLSHPTHTEPTDDHHNPPVOCTTVICLGNLSHPTHTEPTDDHHNPPVOCTTVIC, Probabilities: 0.8872\n","Identifier: 495.1;Pinensin_07, Sequence: MKTNSENLKLSLEDLQLESFVTSIDSEMAARLSGGLGALEHPTHTEPTDTDDTPHAQCTTIVCLGALEHPTHTEPTDTDDTPHAQCTTIVC, Probabilities: 0.7499\n","Identifier: 497.1;Pinensin_09, Sequence: MKEQSNNLQLSIDDLKIDSFVTSLDNEMNMRLAGGLAGQSHPTHTLQTDDEHHLCTTIICLAGQSHPTHTLQTDDEHHLCTTIIC, Probabilities: 0.8777\n","Identifier: 101.2;EP-20, Sequence: EGPVGLADPDGPASAPLGAP, Probabilities: 0.9801\n","Identifier: 139.2;Lichenin, Sequence: ISLEICXIFHDN, Probabilities: 0.9619\n","Identifier: 140.2;lichenin, Sequence: ISLEICXIFHDN, Probabilities: 0.9619\n","Identifier: 143.2;LSEI_2386, Sequence: MKQFDEQKMVNMSDEELLGFIGGDSIRDVSPTFNKIRRWFDGLFK, Probabilities: 0.8422\n","Identifier: 148.2;Microcin_C7, Sequence: MRTGNAN, Probabilities: 0.9853\n","Identifier: 214.2;Serracin-P, Sequence: DYHHGVRVL, Probabilities: 0.9852\n","Identifier: 3.2;Acidocin_8912, Sequence: MISSHQKTLTDKELALISGGKTHYPTNAWKSLWKGFWESLRYTDGF, Probabilities: 0.8751\n","Identifier: 38.2;BlpO, Sequence: MDTKMMSQFAVMDNEMLACVEGGDIDWGRKISCAAGVAYGAIDGCATTV, Probabilities: 0.9570\n","Identifier: 4.2;Acidocin_8912_N-terminal, Sequence: KTHYPTNAXKSLRKGFXESLRXTD, Probabilities: 0.3395\n","Identifier: 58.2;Curvalicin-28a, Sequence: TPVVNPPFLQQT, Probabilities: 0.9840\n","Identifier: 59.2;Curvalicin-28b, Sequence: VAPFPEQFLX, Probabilities: 0.9852\n","Identifier: sp|Q47505|MCCC7_ECOLX Microcin C7 OS=Escherichia coli OX=562 GN=mccA PE=1 SV=1, Sequence: MRTGNAN, Probabilities: 0.9853\n","Identifier: WP_020449898.1 plantaricin C family lantibiotic [Bacillus], Sequence: MSKIEAWKNPVARMNSQIVSPAGDLMDELSDSEMEMLAGGCEWYNISCQLGNKGQWCTLTKECQRSCK, Probabilities: 0.9478\n","Identifier: WP_029325769.1 plantaricin C family lantibiotic [Bacillus], Sequence: MSQEAIIRSWKDPFSRENSTQNPAGNPFSELKEAQMDKLVGAGDMEAACTFTLPGGGGVCTLTSECIC, Probabilities: 0.9652\n","Identifier: WP_001286517.1 plantaricin C family lantibiotic [Streptococcus], Sequence: MSDWKKPIYESLSTEGNPAGDILRELSDSEMDQAMAGGTARSYCACYSGEHSCGRGCTITTECPFLTFICC, Probabilities: 0.8177\n","Identifier: WP_010824504.1 plantaricin C family lantibiotic [Enterococcus faecalis], Sequence: MRDEENNETLINNEQVAWFEEVADQSFDDDVFGACTTNTFSLSDYWGNKGNWCTATHECMGWCK, Probabilities: 0.9728\n","Identifier: WP_000416760.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERLTSYSLCSFGCGKTGSFNSFCC, Probabilities: 0.7507\n","Identifier: WP_049522134.1 plantaricin C family lantibiotic [Streptococcus], Sequence: MSDWKKPIYESLSTEGNPAGDILRELSDSEMDQTMAGGTARSYCACYSGENSCGRGCTITTECPFLTFICC, Probabilities: 0.9574\n","Identifier: WP_050197451.1 plantaricin C family lantibiotic [Streptococcus], Sequence: MDKHSEVTLENEILFQEFDEDVFGACTTNTFSLSDYWGNKGNWCTLTHECMNWCK, Probabilities: 0.9750\n","Identifier: WP_004635387.1 plantaricin C family lantibiotic [Dolosigranulum pigrum], Sequence: MKNKVNNWKTPVYEKEEDYDNPAGDIFRELKSDDIDKVMASGTANTYCRCYSGRHSCGRACTITAECPVFTVACC, Probabilities: 0.8586\n","Identifier: WP_069995861.1 plantaricin C family lantibiotic [Staphylococcus], Sequence: MESTFLEKDVEEQVTWFEEVSDQEFDDDVFGACSTNTFSLSDYWGNNGNWCTASHECMSWCK, Probabilities: 0.8434\n","Identifier: WP_002296231.1 type A2 lantipeptide [Streptococcus mutans], Sequence: MKQSNEMLELIQEVSLDELDQVIGGMGKGAVGTISHECRYNSWAFLATCCS, Probabilities: 0.5190\n","Identifier: WP_017237622.1 plantaricin C family lantibiotic [Streptomyces], Sequence: MNNFDGDISLIEEIAEQDFDGVAYGACSTNTFSLSDFLGNQGGWCTLTKECQPNCN, Probabilities: 0.6018\n","Identifier: WP_003955817.1 FxLD family lanthipeptide [Streptomyces clavuligerus], Sequence: MTNATLAQAELPMALDEDDFAPLDVRVVIAEHPYGKLMCSTGDGCGPTCQGGASACNSFVEDSA, Probabilities: 0.5152\n","Identifier: WP_024388713.1 plantaricin C family lantibiotic [Streptococcus suis], Sequence: MSKKLENWKKAIYFKDENIDNPAGDIFRELSDSELDGMMAGGTANSFCHCYSGKDSCGRGCTITTECPIATIICC, Probabilities: 0.5216\n","Identifier: WP_125532690.1 plantaricin C family lantibiotic [Streptomyces sp. WAC 06725], Sequence: MFFEGDASLVEEIADQDFEGVAYGACTTNTFSLSNALGNNGGWCTLTKECQPNCN, Probabilities: 0.7537\n","Identifier: WP_000416759.1 gallidermin/nisin family lantibiotic [Staphylococcus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHLFCSFGCEKTGSFNSFCC, Probabilities: 0.9670\n","Identifier: WP_123965962.1 plantaricin C family lantibiotic [Streptomyces sp. TLI_185], Sequence: MSFDGDVSLLEEIAEQDFDGVAYGACTTNTFSLSDGLGNNGGWCTLTKECQPNCN, Probabilities: 0.9119\n","Identifier: WP_107368141.1 mersacidin family lantibiotic [Staphylococcus chromogenes], Sequence: MSEKAFTEEIVGKSLHSLTQEEMHSFYGDFDDNAEVRASPVASAVVKETVKQGAKASSAKCAGVISLSGIGGAASSANDCLG, Probabilities: 0.6385\n","Identifier: WP_024343838.1 gallidermin/nisin family lantibiotic [Streptococcus equinus], Sequence: MSNEDFNLDLVSVSKNSNSGASPRITSKSLCTPGCKTGALMTCPIKTATCGCHFGN, Probabilities: 0.5181\n","Identifier: WP_060674405.1 plantaricin C family lantibiotic [Bacillus], Sequence: MSIIETWKNPALRMNSQVVNPAGDLMEELSDDEMEMLAGGCAWYDISCKLGNKGAWCTLTVECQSSCN, Probabilities: 0.9784\n","Identifier: WP_044276571.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus], Sequence: MSHSDKNHILRAWRNADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEATCPATPTPNELTCLYTNCGRIACCR, Probabilities: 0.9727\n","Identifier: WP_011552917.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus], Sequence: MSQKKDHILRAWRDPEYFNSLSSEERAALPANPAAELELGDDLLEVITGGDFCLPGQSSAQCTPCPPRHCL, Probabilities: 0.7264\n","Identifier: WP_133377367.1 mersacidin family lantibiotic [Jeotgalibacillus sp. S-D1], Sequence: MSYMSREEKINALKNPEVRNQGIENPAGKTMNELSSEELAGIQGASDVNPETTPLCIAASIGATIIFSVRNC, Probabilities: 0.8490\n","Identifier: WP_101485617.1 gallidermin family lantibiotic [Actinomyces sp. UMB0138], Sequence: METALELDIDISTDDLVTADDEALITSFILCTAGCTSKGGGSLCSWCC, Probabilities: 0.6462\n","Identifier: WP_141520739.1 lichenicidin alpha family lanthipeptide, partial [Bacillus], Sequence: GEGDVTPEGLSSWLGNNGGYCTLTKECMPSCN, Probabilities: 0.8495\n","Identifier: WP_002192303.1 mersacidin family lantibiotic, partial [Bacillus cereus], Sequence: MTNYAGPTFEELTGEEMQAIQGSGDVQAETTP, Probabilities: 0.9736\n","Identifier: WP_047437384.1 gallidermin/nisin family lantibiotic [Staphylococcus argenteus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERLTSYSLCSFGCEKTGSFNSFCC, Probabilities: 0.7534\n","Identifier: WP_005522761.1 lichenicidin alpha family lanthipeptide [Corynebacterium matruchotii], Sequence: MNASTAVLDSTILDELDKELEAQIIKELEPEMFGLSRLLGNNGRWCTITKECMPSCN, Probabilities: 0.9824\n","Identifier: WP_098690898.1 plantaricin C family lantibiotic, partial [Bacillus toyonensis], Sequence: MSSSKKVVEAWKNPVERSKNEDAPNHPAGEVDSKEIQSLFGAGEGDVTPEGLSSWLGNNGGYCTLTKECMPS, Probabilities: 0.7743\n","Identifier: WP_091040467.1 mersacidin/lichenicidin family type 2 lantibiotic [Glycomyces harbinensis], Sequence: MDIVRSWKEPEYRRSLGDDAPEHPAGTELTEVSELSLTSVGGAAGTQHMGTFGCCWCLPWYSGWTVCGSLCSPKYVCD, Probabilities: 0.5354\n","Identifier: WP_126639118.1 plantaricin C family lantibiotic [Embleya hyalina], Sequence: MNPMYEGDISLLEEIADQDLQHDAAYYAGTGSITITTITIFSNDLGNHGNWCTLTKECQRSCN, Probabilities: 0.4928\n","Identifier: WP_065957033.1 plantaricin C family lantibiotic [Streptococcus dysgalactiae], Sequence: MDKHSEITLENEILFQEFDEDVFGACTTNTFSLSDYWGNKGNWCTLTHECMNWCK, Probabilities: 0.9643\n","Identifier: WP_098033951.1 plantaricin C family lantibiotic, partial [Bacillus thuringiensis], Sequence: MSSSKKVVEAWKNPVERSKNEDAPNHPAGEVDSKEIQSLIGAGEGDVTPEGLSSWLGNNGGYCTLTKECMPS, Probabilities: 0.7580\n","Identifier: WP_086444718.1 plantaricin C family lantibiotic [Enterococcus sp. 12C11_DIV0727], Sequence: MRNEENKETLLSNEQVTWFEEVADQSFDDDVFGACTTNTFSLSDYWGNKGNWCTATHECMGWCK, Probabilities: 0.9402\n","Identifier: WP_093230868.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermoflavimicrobium dichotomicum], Sequence: MTKMVKSWVDPVYRASLSAEEKAAMPVNPAGDVFAELSDAELTNIAGAGCGEVCSWTKDCAACPSWSCWNWSCW, Probabilities: 0.5669\n","Identifier: WP_093230867.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermoflavimicrobium dichotomicum], Sequence: MSKVVKSWVDPVYRASLSAEEKAAMPVNPAGDVFAELSDAELTNIAGAGCGEVCSWTKDCAACPSWSCWNWSCW, Probabilities: 0.8758\n","Identifier: WP_086396028.1 plantaricin C family lantibiotic [Bacillus thuringiensis], Sequence: MKNEISKEVQLAQVTWLEEVVDQEFDDDVFGACTTNTFSLSDYWGNDGSWCSLSHECMSWCK, Probabilities: 0.9620\n","Identifier: WP_197683974.1 mersacidin/lichenicidin family type 2 lantibiotic [Allokutzneria albata], Sequence: MSEEELAALPANPAGAVELTDRALDNVRGLGDIAYGGTPVSPMTVFPPRPSVPV, Probabilities: 0.9818\n","Identifier: WP_055056069.1 plantaricin C family lantibiotic [Eubacteriales], Sequence: MSEMRNPITREENFSNPAGNVMMELQEADLNKFVAGAGETRNSGGVLCTSTAECYFGTLLKPCCPKK, Probabilities: 0.9778\n","Identifier: WP_174257142.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus], Sequence: MSHSDKNHILRAWRDADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEATCPATPTPNELTCLYTNCGRIACCR, Probabilities: 0.9230\n","Identifier: WP_115642317.1 plantaricin C family lantibiotic [Clostridium putrefaciens], Sequence: MKNSQDVVQAEQVTWLEEVQDQSFDDDVFGACTTNTFSLSDYWGNDGGWCSISHECMAWCK, Probabilities: 0.8663\n","Identifier: WP_003470161.1 gallidermin/nisin family lantibiotic [Clostridium perfringens], Sequence: MLKDFDLNLKMETGSSDDKEPRITSVAYCTPGCLTGELCGSSECGLTRSCTNTWLCS, Probabilities: 0.9749\n","Identifier: WP_103565569.1 cinnamycin family lantibiotic [Actinomadura rubteroloni], Sequence: MSPSTILRQAAADAEFRAELLDNPEVFGVAAGTVPESVERQDEASLNYWTEGVAAVDAYACSSTCTSGPFTFACDGTTKG, Probabilities: 0.4848\n","Identifier: WP_091822691.1 plantaricin C family lantibiotic [Butyrivibrio sp. ob235], Sequence: MGDLNAIKDPVLRQKMEILTDPNANVSIELTEQDMQALAGAGWYSEMSAALGNRGSYCTLTKECQACCN, Probabilities: 0.5510\n","Identifier: WP_111330805.1 plantaricin C family lantibiotic [unclassified Streptomyces], Sequence: MYEGDISLLEEIQEQEFDAVGARMACSTNTFSLSDYLGNKGGWCTLTVECQPNCN, Probabilities: 0.4483\n","Identifier: WP_078787334.1 plantaricin C family lantibiotic [Eubacterium ruminantium], Sequence: MNTENKNEVTWLDEVSAQEFDEEVYGACSTNTFSLSDYMGNNGNWCTISHECMAWCK, Probabilities: 0.9734\n","Identifier: WP_207486578.1 lichenicidin A2 family type 2 lantibiotic [Enterococcus avium], Sequence: MSQIGNRFEDLSDYEMQYIAGGNGEARLTPTTIIAASASAASSFVTSYVLSAYTKCR, Probabilities: 0.6654\n","Identifier: WP_128799169.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus coralloides], Sequence: MMTQDMIIRAWKDPSFRATLTVEQLAELPASPSGPTMNELNEGDLMDVVGGIQAVSHRNSCGYVCTLTTECGC, Probabilities: 0.4854\n","Identifier: WP_143567378.1 plantaricin C family lantibiotic [Streptococcus salivarius], Sequence: MDKHSEVTLENEILFQEFDEDVFGACTTNTFSMSDYWGNKGNWCTLTHECMNWCR, Probabilities: 0.9635\n","Identifier: WP_056937913.1 lichenicidin A2 family type 2 lantibiotic [Limosilactobacillus gastricus], Sequence: MEKDINTVVGDPFEELTIDEMTKIQGNGTQSAQSTPVVESLAESFALSYAISRAFKHSK, Probabilities: 0.7902\n","Identifier: WP_084946468.1 plantaricin C family lantibiotic [Streptococcus mitis], Sequence: MSDWKKPIYESLSTEGNPAGDILRELSDSEMDQTMAGGTARSYCACYSGEHSCGRGCTITTECPFLTFICC, Probabilities: 0.8943\n","Identifier: WP_010890685.1 plantaricin C family lantibiotic [Lactococcus lactis], Sequence: MNKNEIETQPVTWLEEVSDQNFDEDVFGACSTNTFSLSDYWGNNGAWCTLTHECMAWCK, Probabilities: 0.9522\n","Identifier: WP_139555170.1 plantaricin C family lantibiotic [Fructilactobacillus sanfranciscensis], Sequence: MDKFNEKVNSNEKEQVTWLEEVSDQNFDDDVFGACSTNTFSLSDYWGNRGNWCTLSHECMAWCK, Probabilities: 0.9040\n","Identifier: WP_000322863.1 type 2 lantibiotic, mersacidin/lichenicidin family, partial [Bacillus], Sequence: VLRSKNEDAPNHPAGEVDSNEIKALFGAGEGDVTPEGLSSWLGNKGGYCTLTKECMSSCN, Probabilities: 0.9733\n","Identifier: WP_078747399.1 plantaricin C family lantibiotic [Streptococcus pneumoniae], Sequence: MSDWKKPIYESLSTEGNPAGDILRELSDSEMNQAMAGGTARSYCACYSGEHSCGRGCTITTECPFLTFICC, Probabilities: 0.6717\n","Identifier: EFK82377.1 lantibiotic, gallidermin/nisin family [Staphylococcus aureus subsp. aureus TCH70 TCH70], Sequence: MQVKGNNNTNDSAGDERITSHLFCSFGCEKTGSFNSFCC, Probabilities: 0.9780\n","Identifier: MNH98119.1 Lantibiotic lacticin A1 precursor [compost metagenome], Sequence: MLENRNPFIVFNKDSKMEALSPVLAEELIEVDLSIQGGCTTNTFTLSNYLGNNGGWCTLTKECQRACN, Probabilities: 0.5527\n","Identifier: WP_125180670.1 gallidermin/nisin family lantibiotic [Enterococcaceae], Sequence: MPKYDDFDLNLKQNVSAPNKEPRVTSLSWCTPGTCNNTCKGDSTLKSNCCGGSIICSLGGC, Probabilities: 0.4331\n","Identifier: WP_003978597.1 SCO0268 family class II lantipeptide [Streptomyces], Sequence: MRTEIVLSHEAPELDLDLDLRVSDLPEQAASFGEGTFTSPSSYAIGTRCPICC, Probabilities: 0.9335\n","Identifier: WP_097832711.1 lichenicidin alpha family lanthipeptide [Bacillus cereus group], Sequence: MSSSKKVVEAWKNPVERSKNEDAPNHPAGEVDSKEIQSLFGAGEGDVTPEGLSSWLGNNGGYCTLTKECMPSCNG, Probabilities: 0.5601\n","Identifier: WP_004936805.1 SCO0268 family class II lantipeptide [Streptomyces], Sequence: MRTEIVLSHEAPELDLDLDLRVSDLPEQAESFGQGTYTSPSSYAIGTRCPICC, Probabilities: 0.5794\n","Identifier: WP_003154891.1 plantaricin C family lantibiotic [Bacillus], Sequence: MNEKMYRFAGDLREELEEISLSEFSGGGGAEQRGISQGNDGKLCTLTWECGLCPTHTCWC, Probabilities: 0.8336\n","Identifier: WP_171776391.1 plantaricin C family lantibiotic [Priestia], Sequence: MEKFLKNPVLQSKLSLNVDNPSGNLLEEISEQDMSAMAGGTTDAVSKTLAFQHGLGRNYGETCTWSAECAATISCGD, Probabilities: 0.6595\n","Identifier: WP_032874558.1 plantaricin C family lantibiotic [Bacillus], Sequence: MNEKMYRFAGDLREELEEISLNEFSGGGGAEQRGISQGNDGKLCTLTWECGLCPTHTCWC, Probabilities: 0.9175\n","Identifier: WP_125743679.1 plantaricin C family lantibiotic [Streptomyces sp. WAC01280], Sequence: MNNFDGDISLIEEIAEQDFDGVAYGACSTNTFSLSDFLGNHGGWCTLTKECQPNCN, Probabilities: 0.9170\n","Identifier: WP_019328243.1 SCO0268 family class II lantipeptide [Streptomyces], Sequence: MRTEIVLSHEAPELDLELDLRVSDLPEQAESFGQGTFTSPSSYAIGTRCPICC, Probabilities: 0.5315\n","Identifier: WP_206698159.1 gallidermin family lantibiotic [Cytobacillus firmus], Sequence: MFDLDVQVLKAASGEQPDIKSKLFCTPGCVDQTTKTSCCLKCKTSLCNSNMSCW, Probabilities: 0.5639\n","Identifier: WP_096241054.1 plantaricin C family lantibiotic [Eubacteriales], Sequence: MSRNVKNWKLPVYEMNEVESSNPAGNSFQELKEMELEDYTGAGTANTHCNCYSEDASCGRLCTGTAECPIMTIICC, Probabilities: 0.9810\n","Identifier: WP_115596678.1 gallidermin family lantibiotic [Bacillales], Sequence: MFVKENMFDLDVQVLKAASGEQPDIKSKLFCTPGCVDQTTKTSCCLKCKTSLCNSNMSCW, Probabilities: 0.9768\n","Identifier: WP_020647175.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis balhimycina], Sequence: MSDHEIIRSWKDEEFRNSLGPAQRAVLPGNPAGLIELTDEALDDLIAGAAALDSCCWSSCNSTQ, Probabilities: 0.6256\n","Identifier: WP_184986655.1 plantaricin C family lantibiotic [Streptomyces caelestis], Sequence: MYQGDISLLEEIAEQDFDGVAYGACTTNTFSLSDSLGNNGGWCTLTKECQPNCS, Probabilities: 0.5381\n","Identifier: WP_184632841.1 plantaricin C family lantibiotic [Streptomonospora salina], Sequence: MDSHVLDPAGSMIEELTDQDLSGKEGGQSGGLWTVPGTHTIGAVCTISWECNGGTVCGN, Probabilities: 0.8609\n","Identifier: WP_074842839.1 hypothetical protein [Priestia endophytica], Sequence: MEERLYGFAGKLEEELEEIALKEISGGAEQRGISQGNDGHQCTITWECSICPTHTCWC, Probabilities: 0.9464\n","Identifier: WP_157308863.1 mersacidin/lichenicidin family type 2 lantibiotic [Chitinophaga tropicalis], Sequence: MKHSNVDVIRAWKDAEYRNQLSAEQLATLPANPIESADIDGSLLKGGIAREIEAPAVSGGWVCTISGECNGGSCCNPFTSR, Probabilities: 0.6371\n","Identifier: WP_208679317.1 lantibiotic dehydratase [Streptomyces sp. ID38640], Sequence: MRESGDVPEPPAVGLHLDAEVQLPHAVAWEAEAAATVLARLTPYPYGTPSWQTYRHRHRFLEHYGPHVLIPPLECTEPVTGISLPDGFHTTAAV, Probabilities: 0.6859\n","Identifier: MBX5451287.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermogemmatispora sp.], Sequence: DEAYRQSLSEEELSLLPESPIGEVELTDADLEAVQGGHGNTDECNNTVAVLCLQSLAVLLGLCNTNAAGGCL, Probabilities: 0.9697\n","Identifier: WP_207126352.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinocatenispora comari], Sequence: MSAEELVRAWRDEDFRLELESAVAHPAGDVDRELYVSTGAASTCNYASVTCSHICSCAI, Probabilities: 0.9789\n","Identifier: WP_182543085.1 lantibiotic dehydratase [Saccharopolyspora lacisalsi], Sequence: MTSLVQHGFLVTSLRAPSTVTDPSGHVLDRLRETNVEALPVAPLIRQLDQVHRDIDAHNEAPSEHARDEYAGLAGPGHDPRA, Probabilities: 0.7979\n","Identifier: WP_132517964.1 lantibiotic dehydratase [Nonomuraea diastatica], Sequence: MAALPFQPRVRYRSTILTPARWPLPATVTGARGSRLGRRPAHLADDHGSPAAGHRRHRP, Probabilities: 0.9681\n","Identifier: WP_059302729.1 SCO0268 family class II lantipeptide [Streptomyces], Sequence: MRTEMILSHEAPELDLDLDLRVSDLPEQAESFGQGTFTSPSSYAIGTRCPICC, Probabilities: 0.8583\n","Identifier: MBW8879538.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium], Sequence: MNRVDIVRAWKDPLYRASLTADERDRLPGHPSGLMEIEDELLKSVAAGALTTALKCTESSFNNFRSCCPK, Probabilities: 0.5013\n","Identifier: WP_220209762.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei], Sequence: MSADEIIRVWKAEDDEEKGKDAPANPAGEELSDEDLEQVEGGTEPGNGNPTSPYCASYQTLCPE, Probabilities: 0.7268\n","Identifier: WP_220209760.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei], Sequence: MSTEEIIRAWKAENDEEKNEDAPANPAGEELSDEELGGVEGGSEAPPDGSAGTACSCAKFSC, Probabilities: 0.9174\n","Identifier: WP_220209763.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei], Sequence: MSIDEIIRAWKAEDEEEKNGNAPANPAGEELSDEDLEQVEGGSEAPGDADMSQNICTCAKISCL, Probabilities: 0.9070\n","Identifier: WP_220194733.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonospora formicarum], Sequence: MQYDIVRAWKDEEYNQSMPQQAENPVGYAELSDADLELMSGCGEGAPGKSIVQGEKADNEVINNFSFKKKTNAGLIIIDLTLVDLSCAMFNNVYF, Probabilities: 0.9338\n","Identifier: WP_220209761.1 mersacidin/lichenicidin family type 2 lantibiotic [Reticulibacter mediterranei], Sequence: MSTDEIIRAWKAENDEEKNEDAPANPAGEELSDEELEKVEGGSEDPGDAEGSYVCSCLKTSC, Probabilities: 0.9729\n","Identifier: WP_220199013.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonospora formicarum], Sequence: MNIDIVRTWKDEAYRQNLQRKGMAFPENPVGDLELTDEELLYVVGGARNETDSCTAGCTDLCSNSCTVA, Probabilities: 0.5080\n","Identifier: WP_181749956.1 class II lanthipeptide, LchA2/BrtA2 family [Paenactinomyces guangxiensis], Sequence: MSKQDVVRALRDPEYRAQLNGFNHPAGLVTDEELKRIAGAADAEPQSTIPCGVVSATVSLVSAAGCPTTKCTSQCTNVDGLC, Probabilities: 0.8136\n","Identifier: WP_108965343.1 lichenicidin alpha family lanthipeptide [unclassified Actinomyces], Sequence: MESQAEVTWIEEVGEQSFDTDVFGACSTNTFSLSDYWGNRGYWCTLTHECMSWCK, Probabilities: 0.4880\n","Identifier: WP_201425283.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus], Sequence: MSQKEHILRAWRDPEYFNNLSSEERAALPANPAAELELGDDLLEVITGGDFCPTGQSSPYCTPCPPLHCY, Probabilities: 0.5249\n","Identifier: WP_111621309.1 gallidermin/nisin family lantibiotic [Paenibacillus pabuli], Sequence: MNKDFDLNLKIAKGTDNGVQPRITSVAYCTPGCNTGSTCGTSECGLTRNCTNTWLCG, Probabilities: 0.4757\n","Identifier: WP_201422437.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus], Sequence: MGHSDKNHILRAWRDADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEATCPATPTPNELTCLYTNCGRVACCR, Probabilities: 0.9396\n","Identifier: WP_111079677.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHLFCSCGCEKTGSFNSFCC, Probabilities: 0.5299\n","Identifier: WP_141066300.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERLTSHLFCSFGCEKTGSFNSFCC, Probabilities: 0.9729\n","Identifier: WP_012818083.1 plantaricin C family lantibiotic [Staphylococcus epidermidis], Sequence: MRSNLERNPYLRNEAQQSVDLPLDNPIRELKEDELLKLNGAKNRFIPGAEVAVSTLGCYGGSVALGNDGFMCTTTVECQNQCK, Probabilities: 0.8954\n","Identifier: WP_106311337.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Chamaesiphon polymorphus], Sequence: SQENIIRSWKSLDFRNGLSAKERAFLPENPAGSIELTDAELKSATGAARNNPCSGACNLCVGH, Probabilities: 0.9411\n","Identifier: WP_194734578.1 lantibiotic dehydratase, partial [Streptococcus suis], Sequence: RSLYESLQKYYSGDSMKRKKINQLSESVYKYYKRSKERSTPFGLFSETSIGSFSSTEKLNLNGKTLKKVLLDSEWLIR, Probabilities: 0.9397\n","Identifier: WP_167542329.1 mersacidin/lichenicidin family type 2 lantibiotic [Rathayibacter tritici], Sequence: MTEEIFWKDPEYRFTRSNDRDFSSHPSGVTLLTDLELDEAFGGTVSVTPVPSITGVVGCWTVGNTVCNGSCAAFGTNGCCH, Probabilities: 0.7360\n","Identifier: WP_193441134.1 lantibiotic dehydratase, partial [Streptococcus suis], Sequence: VKRTYLDDYKEKFLDKYGVNREVQLMELFDSNLGIGSPYGYQHPKNDFWESSPSTVYFSEKEELEYLNNFEKALETGGNIQLYNEEDF, Probabilities: 0.6086\n","Identifier: WP_143806301.1 gallidermin family lantibiotic [Pantoea latae], Sequence: MCAEICSNFDSVYRVIAASRANASSRILAWYLCRPGCPSGFLLIIGTNPIRLLKFALKHDNGLVHPLIRVSCIKFA, Probabilities: 0.9511\n","Identifier: WP_139889503.1 lichenicidin alpha family lanthipeptide, partial [Bacillus tropicus], Sequence: GEVDSKEIQSLFGAGEGDVTPEGLSSWLGNNGGYCTLTKECMPSCN, Probabilities: 0.9355\n","Identifier: WP_098775162.1 lichenicidin alpha family lanthipeptide [Bacillus cereus], Sequence: MGSSKKVVEAWKNPVERSKNEDAPNHPAGEVDSKEIQSLFGAGEGDVTPEGLSSWLGNNGGYCTLTKECMPSCNG, Probabilities: 0.7239\n","Identifier: WP_111762440.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHLLCSFGCGKTGSFNSFCC, Probabilities: 0.9792\n","Identifier: WP_098328036.1 lichenicidin alpha family lanthipeptide, partial [Bacillus cereus], Sequence: SSKKVVEAWKNPVERSKNEDAPNHPAGEVDSKEIQSLFGAGEGDVTPEGLSSWLGNNGGYCTLTKECMPSCNG, Probabilities: 0.8836\n","Identifier: WP_098854239.1 lichenicidin alpha family lanthipeptide, partial [Bacillus toyonensis], Sequence: PVERSKNEDAPNHPAGEVDSKEIQSLFGAGEGDVTPEGLSSWLGNNGGYCTLTKECMPSCN, Probabilities: 0.9783\n","Identifier: WP_141533480.1 lichenicidin alpha family lanthipeptide, partial [Bacillus cereus], Sequence: GEVDSKEIQSLFGAGEGDVTPEGLSSWLGNNGGYCTLTKECMPSCNG, Probabilities: 0.9742\n","Identifier: WP_142321224.1 lichenicidin alpha family lanthipeptide, partial [Bacillus cereus], Sequence: DVTPEGLSSWLGNNGGYCTLTKECMPSCNG, Probabilities: 0.9072\n","Identifier: WP_098160134.1 lichenicidin alpha family lanthipeptide, partial [Bacillus pseudomycoides], Sequence: DAPNHPAGEVDSNEIKALFGAGEGDVTPEGLSSWLGNKGGYCTLTKECMSSCN, Probabilities: 0.9438\n","Identifier: WP_097787301.1 lichenicidin alpha family lanthipeptide, partial [Bacillus pseudomycoides], Sequence: SKNEDAPNHPAGEVDSNEIKALFGAGEGDVTPEGLSSWLGNKGGYCTLTKECMSSCN, Probabilities: 0.9762\n","Identifier: WP_142308427.1 lichenicidin alpha family lanthipeptide, partial [Bacillus pseudomycoides], Sequence: LFGAGEGDVTPEGLSSWLGNKGGYCTLTKECMSSCN, Probabilities: 0.9286\n","Identifier: WP_065359317.1 lichenicidin alpha family lanthipeptide [Streptococcus dysgalactiae], Sequence: MDKHSEVTLENEILFQEFDEDIFGACTTNTFSLSDYWGNKGNWCTLTHECMNWCK, Probabilities: 0.9749\n","Identifier: WP_006731084.1 type A2 lantipeptide [Lactobacillus iners], Sequence: MEKLNHNAVTSLNEVSDSELDTILGGSRWWQGVLPTVSHECRMNSFQHIFTCC, Probabilities: 0.5848\n","Identifier: WP_199753706.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus], Sequence: MMTPDMIIRAWKDPSFRESLTAEQRTQLPANPSGATLNELSETELMDVVGGRIDIGFTATNSCEYICTNLTFSLC, Probabilities: 0.7011\n","Identifier: WP_107490302.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sp. CB02959], Sequence: MDSTVNVWKNTMYREGLSMDPSHPAGVVDLTADLSDEGDVVAAKTLSTQPFFNCCG, Probabilities: 0.6618\n","Identifier: WP_102035248.1 plantaricin C family lantibiotic [Rothia dentocariosa], Sequence: MSDITLLEEISEQDLHQANGGSLFASPADPAGVILSAGYTAYSWLKGNEGIACTATKECNCN, Probabilities: 0.6389\n","Identifier: WP_032871261.1 plantaricin C family lantibiotic [Bacillus], Sequence: MNEKMYRFAGDLREELEEISLNEISGGGGAEQRGISQGNDGKLCTLTWECGLCPTHTCWC, Probabilities: 0.9571\n","Identifier: WP_155458752.1 lantibiotic dehydratase, partial [Streptococcus pneumoniae], Sequence: MLMVANPRFFNELTKEKIYQNSTFRNYAKRSLTRATPFGLFSSVGVGSFSKVSYPQQIRENYRKK, Probabilities: 0.5448\n","Identifier: WP_005505332.1 gallidermin/nisin family lantibiotic [Rothia], Sequence: MVTVDLDTRVEEVEEAVCPHITSKFLCAPGCPTGGIACFTAQCANGCGISVGCKWLVALSQQSALEGRHPRPYLQGRFL, Probabilities: 0.9828\n","Identifier: WP_155458462.1 lantibiotic dehydratase, partial [Streptococcus pneumoniae], Sequence: MLMVANPRFFNELTKEKIYQNSTFRNYAKRSLTRATPFGLFSSVGVGSFSKVSYPQQIRENY, Probabilities: 0.8932\n","Identifier: WP_139638388.1 lantibiotic dehydratase [Nonomuraea], Sequence: MGEVSTCEGDGPSRIGSQGRRRDPADFGAANLLPHLPRIRHGRTVLSPARWIFAAGTAAGVGEDLARLAGPVAGAAHDGAA, Probabilities: 0.9623\n","Identifier: MBW4623883.1 mersacidin/lichenicidin family type 2 lantibiotic [Cyanosarcina radialis HA8281-LM2], Sequence: MSQENIIRAWKDENFRQQLSEKERALLPEHPAGSIELTDEEIDNAAGGLMPPPTRLCSHSSSC, Probabilities: 0.5216\n","Identifier: WP_219184398.1 gallidermin/nisin family lantibiotic [Streptomyces sp. AC542_RSS864], Sequence: MGITPTTGQNDANLFDLDLEIGVEKSVAGPAITSVSWCTPGCTSEGGGSGCSHCC, Probabilities: 0.5400\n","Identifier: WP_147343401.1 lantibiotic dehydratase, partial [Ruminococcus sp. OM04-4AA], Sequence: KECEKVLNCFSKISEAAETDFLSKCEKVLNCFSKISEAAETDFLSKYKNDFIEKYGYHRVVPVMEVLDNDIGIGIPAGIQEQNAIEIRKLVMKQHCAKG, Probabilities: 0.8597\n","Identifier: WP_099810709.1 lantibiotic salivaricin M precursor [Streptococcus suis], Sequence: MDTLDFEISTQELEGTSGAGGWYSAFKMTLAGRCGRCFTCSYECTSNNVNC, Probabilities: 0.4530\n","Identifier: WP_098801344.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides], Sequence: MSKNEKLNKLRDQEFDTKELIGSVDENDLKQVAGAGDVNPETTPATP, Probabilities: 0.6829\n","Identifier: WP_218012797.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides], Sequence: MSKNEKLNKLRDQEFDTKELIGSVDENDLKQVAGAGDVNPET, Probabilities: 0.9296\n","Identifier: WP_098624264.1 type A2 lantipeptide [Bacillus toyonensis], Sequence: MEKNFELKALEAINDLTDTELEHVVGADSGQINTLTHECHWNTFNSLFTCCS, Probabilities: 0.8519\n","Identifier: WP_097850014.1 class II lanthipeptide, LchA2/BrtA2 family [Bacillus pseudomycoides], Sequence: MSKNEKLNKLRDQEFDTKELIGSIDENELKKVAGAGDVNPETTPVINPGFTNRLCPTIPCTLRA, Probabilities: 0.7028\n","Identifier: WP_103061646.1 lichenicidin A2 family type 2 lantibiotic [Actinomyces qiguomingii], Sequence: MTQDYNCGPSFESLSQEEMDFVTGANGDIQPAATPTLSSVVCGGVSKFVGSAIVSFVGSFVASASSRCRD, Probabilities: 0.4662\n","Identifier: WP_098651372.1 mersacidin/lichenicidin family type 2 lantibiotic [Bacillus toyonensis], Sequence: MSSSKKVVEAWKNPVERSKNEDAPNHPAGEVDSKEIQSLFGAGEGDVTPXXMYAFL, Probabilities: 0.5426\n","Identifier: PFJ06663.1 hypothetical protein [Bacillus cereus], Sequence: MNNQNFIKNPIAQNKLSMSSNHPSGNLFEEISEQDMSLVAGGTNHWASKTAAQIMGLGDNYGRVCTVSAECNFTSPCGGTQ, Probabilities: 0.9789\n","Identifier: PFJ06660.1 hypothetical protein [Bacillus cereus], Sequence: MKNSLKNPVLQHKYSSREKNPAGDLLQEINEQEMKLARGGYEFTHADSVTLAQMQGLGNNYGTKCTVSAECLFTAQCGGWW, Probabilities: 0.9430\n","Identifier: WP_155974693.1 type A2 lantipeptide [Staphylococcus hominis], Sequence: MENSKIMKDIEVANLLEEVQEDELNEVLGAKKKSGVIPTVSHDCHMNTFQFMFTCCS, Probabilities: 0.8331\n","Identifier: WP_179862905.1 plantaricin C family lantibiotic [Bacillus thuringiensis], Sequence: MKNSLKNPVLQHKYSAREKNPAGDLLQEITEQEMKLASGGYEYTHGDSVTLARMQGLGDNYGTKCTVSAECLFTAQCGGWW, Probabilities: 0.5808\n","Identifier: WP_070180740.1 plantaricin C family lantibiotic [Bacillus cereus group], Sequence: MKTEISKEVQLAQVTWLEEVVDQEFDDDVFGACTTNTFSLSDYWGNDGSWCSLSHECMSWCK, Probabilities: 0.9621\n","Identifier: WP_063484126.1 SCO0268 family class II lantipeptide [Streptomyces], Sequence: MRTEIVLSHEAPELDLDLDLRVSDLPEQAESFGQGTFTSPSSYAIGTRCPICC, Probabilities: 0.6243\n","Identifier: WP_096632908.1 plantaricin C family lantibiotic [Streptomyces sp. WZ.A104], Sequence: MANPNVGILEEIDDQNLDSVSAAGQGWGTLVSSLSCYGASYVLGNNGNFCTATAECQNNCK, Probabilities: 0.7572\n","Identifier: WP_110110496.1 type A2 lantipeptide [unclassified Auritidibacter], Sequence: MKAINHEAATALQELTDNELDQILGADGVISTLTHECHMNSWQGLLTCCS, Probabilities: 0.8575\n","Identifier: WP_106006683.1 type A2 lantipeptide [Streptococcus mutans], Sequence: MKNTTNEMLELIQEVSLDELDQVIGGMGKGTVGTISHECRYNSWAFLATCCS, Probabilities: 0.7803\n","Identifier: WP_089506884.1 plantaricin C family lantibiotic [Streptomyces sp. NBS 14/10], Sequence: MREEFEGDLPLMEELAEQDFHDAAQNGGTIGPEQLAPLIMRVSAHYGGKGGFGMFCTLTKECQPNCN, Probabilities: 0.9501\n","Identifier: WP_076075532.1 type A2 lantipeptide [Actinomyces naeslundii], Sequence: MSNINMQAAAALDELSDAELDQVLGAGGDGVKWSITHECYMNSLQVLLTCCS, Probabilities: 0.8105\n","Identifier: WP_071528712.1 gallidermin/nisin family lantibiotic [Streptomyces], Sequence: MGIHAPTTGQNDANLFDLDLEIGVEKSVAGPAITSVSWCTPGCTSEGGGSGCSHCC, Probabilities: 0.8792\n","Identifier: WP_014845285.1 type A2 lantipeptide [Arachnia propionica], Sequence: MQSNYDDQILDNLAELSDTEIEELLGAGWGSIFSFTHECNTNTMTQLFTCCF, Probabilities: 0.9260\n","Identifier: WP_087812317.1 SCO0268 family class II lantipeptide [Streptomyces sp. CS113], Sequence: MRTEMILSHEAPELDLDLDLRVSDIPEQAESFGQGTFTSPSSYAIGTRCPICC, Probabilities: 0.6626\n","Identifier: WP_179084717.1 lexapeptide family class V lantibiotic [Streptomyces], Sequence: MQNVTEKDLFDGYTAYTSAEELGLYDGKDAAPAFSPTIPWAIRAGLITARSSQQCAAAIGSFTARTIESKC, Probabilities: 0.9728\n","Identifier: WP_042132907.1 plantaricin C family lantibiotic [unclassified Paenibacillus], Sequence: MFFSKEMVSAWKNPVARRNLGDVPHPSGDVMAELKDEDLQNFAGGWDTNTLTTSTSIFISRGLGNDGWVCTWSKECMPSCN, Probabilities: 0.9340\n","Identifier: WP_072472683.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHLFCSFGCEKTGTFNSFCC, Probabilities: 0.9334\n","Identifier: WP_072419822.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHLFCSFGCEKAGSFNSFCC, Probabilities: 0.9101\n","Identifier: HBI0859695.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: NNNSNDSAGDERITSHSLCTPGCAKTGSFNSFCC, Probabilities: 0.6748\n","Identifier: MBV9690774.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium], Sequence: AQYRESLSSEELALVPANPAGEVELVEEDLLSVMGGGGSETCCQHVSVCVNTVLCSVACSVVC, Probabilities: 0.9648\n","Identifier: MBV9708024.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium], Sequence: MDIVRAWKDAEYRESLSSEELAQVPAHPAGEVELAEGDLAGVVGGLSAVVGGGCQSTASTGSCVALTVCLALSCELL, Probabilities: 0.5599\n","Identifier: MBV8693864.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium], Sequence: MNFNIERAWKDEVYFNSLSEEERALLPECPVGAVDLDGDDMDLVHGTTPFSRFCSNGCLNSFSGGGCNSNNGSCNISNAVICAQSFALVCISNGNC, Probabilities: 0.7625\n","Identifier: MBV9020939.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacteraceae bacterium], Sequence: MNIDIVRAWKDAEYRESLSSEELAQVPAHPAGEVELAEGDLAGVVGGLSAVVGGGCQSTASTGSCVALTVCLALSCELL, Probabilities: 0.4924\n","Identifier: MBV9307326.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteriaceae bacterium], Sequence: MSHQNIIKAWKNPAYRNTLSPAEREALPANPAGTIEISDEVLGQFAGGANNAAPHTVVCKTQIACTMFEECSNLGCPSRLCSLIDCGPPILTF, Probabilities: 0.9215\n","Identifier: WP_143093000.1 lantibiotic dehydratase, partial [Arsenicibacter rosenii], Sequence: MLDKETNFRAGNARTLRTHVRVDMYYLLAIRKWLITRPEIRAQLRLFPNSSIYSVGAS, Probabilities: 0.5726\n","Identifier: MBV8645617.1 mersacidin/lichenicidin family type 2 lantibiotic [Candidatus Eremiobacteraeota bacterium], Sequence: MVRAWRDPLFRAALPVDTLRELPPHPAGVADDDDDAIVRDITFGGGCGTAGCSRVVCTDPCT, Probabilities: 0.9811\n","Identifier: WP_083326567.1 gallidermin/nisin family lantibiotic [Actinomyces sp. HMSC035G02], Sequence: MVTVDLDTRVEEVEEAVCPQITSKFLCAPGCPTGGIACFTTQCTNGCSISVGCKWLVALSQQSALEVKHPRPYLQGRFLRKVGHASYIGN, Probabilities: 0.9796\n","Identifier: WP_141721487.1 mersacidin/lichenicidin family type 2 lantibiotic [Micromonospora citrea], Sequence: MDLPHVLGVLMNDKTLRAWKDPAFRTTLDDTDIALLPESPVGAVGGLDADFADIFGGDRAFTFHTCFTCQWGSLGCTNTCWIMTAGCTSGSC, Probabilities: 0.4630\n","Identifier: WP_161977616.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter kobayashii], Sequence: MSIEDIVKAWKSGETLENDSLVNPAGEELSDEELREITGGMLCGYTCYWGCHGNTTCDESVLV, Probabilities: 0.5064\n","Identifier: WP_155357332.1 lantibiotic protein [Acrocarpospora macrocephala], Sequence: MSKIRNELAALESEVFEIDEIPDLMEALASAEMPIASSTDGLSCTSTNSCTSSASTNSCCSG, Probabilities: 0.7912\n","Identifier: WP_146896085.1 mersacidin/lichenicidin family type 2 lantibiotic [Adhaeribacter aerolatus], Sequence: MKKNVDVIRAWKDAEYRATLTEEQLKSLPENPVNDSNALLDNIALRNISGGRPSSGWFCSVSGECNGGYSCNPYYW, Probabilities: 0.5765\n","Identifier: WP_161977615.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter kobayashii], Sequence: MNIEEIVKAWKSEEETADVNMPANPVGEELSDEELQEVSGGFPCPIISKCLTNWTGNL, Probabilities: 0.9621\n","Identifier: WP_126630726.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter alpinus], Sequence: MSVDIVRAWTDEAYRRSLGPAELALLPTNPAGEIELVDADLATIYGGTGIGIGVGVGRPNEQFSHGCSADCLRSSYCFTFGPCC, Probabilities: 0.5579\n","Identifier: WP_111320054.1 mersacidin/lichenicidin family type 2 lantibiotic [Thermosporothrix hazakensis], Sequence: MKLDVIRAWKDADYRADLSPEALAALPENPVGEAKELTEEELHEVVGGSSSYWSSHSHYSSSYCSCDSWDGWDYWW, Probabilities: 0.7118\n","Identifier: WP_065959517.1 plantaricin C family lantibiotic [Streptomyces sparsogenes], Sequence: MNSEFEGDLSLVEELAEQDFHDVAQHGGTIDLAHRISQYFGGGGYGIFCTLTKECQPNCN, Probabilities: 0.8195\n","Identifier: WP_201823631.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces musisoli], Sequence: MSEPTAAEVIRAWRDPLFRAALGDLARAALPPHPAGDIEVPGLDRE, Probabilities: 0.9597\n","Identifier: MBV6622421.1 mersacidin/lichenicidin family type 2 lantibiotic [Rivularia sp. (in: Bacteria)], Sequence: MSNIDIIRAWKDEEYRNSLSEEQRSQLPENPAGMVELSDEDMESMAGGVYLYPTSAVGFTTKASVYNCPLPPKPKPTYPSYPSIGIIFPFRY, Probabilities: 0.7231\n","Identifier: WP_141559676.1 plantaricin C family lantibiotic, partial [Bacillus thuringiensis], Sequence: MNNELKSWKNPYLRNSQNHATNLPIEDPLMELAEEDIKSINGGV, Probabilities: 0.9811\n","Identifier: WP_098720203.1 class II lanthipeptide, LchA2/BrtA2 family, partial [Bacillus pseudomycoides], Sequence: MSKNEKLNKLRDQEFDTKELIGSVDENDLKQVAGAGDVNPETTPAT, Probabilities: 0.9186\n","Identifier: WP_098362637.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus cereus], Sequence: SSKKVVEAWKNPVERSKNEDAPNHPAGEVDSKEIQSLFGAGEGDVTPEGLSSWLGNNGGYCTLIKECISSCN, Probabilities: 0.7580\n","Identifier: WP_176418485.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Myxococcus], Sequence: MSHSDKNHILRAWRDADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEATCPATPTPNELTCLYTACGMSACCR, Probabilities: 0.6491\n","Identifier: WP_176417165.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Myxococcus], Sequence: MSQKKEHILRAWRDPEYFNSLSSEERAALPANPAAELELGDDLLEVITGGDACLPGQSSPYCTPCPPFHCF, Probabilities: 0.6667\n","Identifier: WP_163869062.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus], Sequence: MSDSVKSEIIRAWKDEEFRNNLSESERDLIPANPAGILELTDEVLGVASGGLAAASCDWCSC, Probabilities: 0.7753\n","Identifier: WP_077701886.1 plantaricin C family lantibiotic [Virgibacillus dokdonensis], Sequence: MKEKDLQLYNTAGDFLSELEEASLENISGGAAEPMGISQGNDGKWCTVTWECSICPTHTCWC, Probabilities: 0.8356\n","Identifier: WP_128100884.1 gallidermin/nisin family lantibiotic [Paenibacillus sp. DCT19], Sequence: MNKDFDLNLKIAKGKDDSVQPQITSVAYCTPGCNTGSTCGSSECGLTRQCTNTWLCA, Probabilities: 0.7813\n","Identifier: WP_217210728.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sp. AC550_RSS872], Sequence: MDLHKIVRAWEDREYRSGLSAAELADLPDNPAGVIELTDAELGQVLGGAQSGNSVGCNTRTCVTTRGNSQNPCRSC, Probabilities: 0.7918\n","Identifier: WP_199745882.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus], Sequence: MMTPDMIIRAWKDPSFRESLTAEQRTQLPANPSGATLNELSETELMDVVGGRIDVGFTATNSCEYICTNLTFSLC, Probabilities: 0.6786\n","Identifier: WP_199749692.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus], Sequence: MMTPDMIIRAWKDPSFRESLTAEQRTQLPANPSGATLNELSETELMDVVGGRIDVGFTATNSCEYICTNLTFSMC, Probabilities: 0.8490\n","Identifier: WP_120379436.1 plantaricin C family lantibiotic [Firmicutes], Sequence: MKEKDLQLYNTAGDFMSELEEASLDNISGGAAEPMGISQGNDGKYCTITWECSICPTHTCWC, Probabilities: 0.8075\n","Identifier: WP_216759756.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHLFCSFGCAKTGSFNSFCC, Probabilities: 0.7246\n","Identifier: WP_058047571.1 lexapeptide family class V lantibiotic [Streptomyces roseifaciens], Sequence: MQNVNEKDLFDGYTAYTSAEELGLYDGKDAAPAFSPTIPWAIRAGLITARSSQQCAAAIGSFTARTIESKC, Probabilities: 0.9765\n","Identifier: MBU7065573.1 lantibiotic ABC transporter ATP-binding protein [Staphylococcus aureus], Sequence: VTETSEEELKQFKDNDLENVLLEIIEREDQA, Probabilities: 0.9582\n","Identifier: WP_216731829.1 lichenicidin A2 family type 2 lantibiotic, partial [Staphylococcus aureus], Sequence: MKNNNIPLEISSLVGNSFEEMSLEEMALTQGTGDVQPQSTPGC, Probabilities: 0.6263\n","Identifier: WP_199727750.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus], Sequence: MMTPDMIIRAWKDPSFRESLTAEQRAQLPANPSGTTVNEMSETELMDVVGGKIDIGFTATNSCEYICTNLTFVC, Probabilities: 0.7806\n","Identifier: WP_216734346.1 lichenicidin A2 family type 2 lantibiotic, partial [Staphylococcus aureus], Sequence: LVGNSFEEMSLEEMALTQGTGDVQPQSTPGCFFASLGIGVTISAWKC, Probabilities: 0.5740\n","Identifier: WP_154778451.1 plantaricin C family lantibiotic [Erwinia sp. CPCC 100877], Sequence: MNEKKEVLTKLEQEEQDVLLQEINEQQLEHEVGGAWWEISKNLGNKGRFCTLTKECQANCR, Probabilities: 0.8580\n","Identifier: WP_216399246.1 plantaricin C family lantibiotic [Amphibacillus sp. MSJ-3], Sequence: MNMVETWKNPLSRMEHQVESPVGNVIDELSDSEMEMIAGACKWYNISCRLGNKGGWCTLTVECQRSCN, Probabilities: 0.8976\n","Identifier: WP_171426526.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus coralloides], Sequence: MMTQDMIIRAWKDPSFRATLTVEQLAELPASPSGPTMNELNEGDLMDVVGGMQAVSHRNSCGYICTLTTECGC, Probabilities: 0.4978\n","Identifier: WP_216368849.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter arantiisoli], Sequence: MSIDIVRAWTDEAYRRSLSPDDLARLPAHPAGQMELLDVDLATICGGAGKGLIEHFSLVCSSDCLKSSYCFTFGPCCQS, Probabilities: 0.5166\n","Identifier: WP_171441066.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus], Sequence: MSHSDKNHILRAWRDADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEATCPATPTPNELTCLYTNCGRVACCR, Probabilities: 0.8782\n","Identifier: WP_216126978.1 plantaricin C family lantibiotic [Clostridium estertheticum], Sequence: MSNNSIVRAWKNPMYRSKLNQNVVNPAGNAMVELNELELDELTGAASSGAICTATTECTYFSAICC, Probabilities: 0.8299\n","Identifier: DAL90676.1 Type-A lantibiotic [Myoviridae sp.], Sequence: MIECMRNAALKPQTNADRIRSMTDEELAEFLPIASNFICQPTEECIRNTVMNHCGECERTEECAMKWLRAESEE, Probabilities: 0.9829\n","Identifier: WP_064670866.1 plantaricin C family lantibiotic [Bacillus subtilis], Sequence: MENNFKNPVLRSKKLNEKHPAGNLLEEINEQDMSSIAGGTNDAYSKSLANRAGLGNNYGKYCTVSAECFGTISCGS, Probabilities: 0.9715\n","Identifier: WP_214801216.1 plantaricin C family lantibiotic [Bacteria], Sequence: MNEKMYCFAGDLREELEEISLSEFSGGGGAEQRGISQGNDGKLCTLTWECGLCPTHTCFC, Probabilities: 0.9630\n","Identifier: WP_215179456.1 plantaricin C family lantibiotic [Paenibacillus albidus], Sequence: MEERLYSLAGNLSEEMEEISMKAVRGGAEQRGISQGNDGKLCTLTWECSICPTLTCFWC, Probabilities: 0.9771\n","Identifier: MBD4375713.1 gallidermin/nisin family lantibiotic [Xanthomonas citri pv. citri LL174-02], Sequence: NDSAGDERITSHSLCTPGCAKTGSFNSFCC, Probabilities: 0.9312\n","Identifier: WP_037960079.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces], Sequence: MNRTTKAWKDPVFRAGLSSEELSALPAHPAGAIELTDQRLDSVVGGGLTDTWVTCGHKTCRTCFTCDWLCG, Probabilities: 0.5425\n","Identifier: WP_044579866.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces violaceusniger group], Sequence: MNRTTKAWKDPVFRAGLSSEELSALPAHPAGAIELTDQRLDNVVGGGLTDTWVTCGHKTCRTCFTCDWLCG, Probabilities: 0.7622\n","Identifier: WP_088112221.1 plantaricin C family lantibiotic [Bacillus subtilis], Sequence: MENNFKNPVLRSKKLNEKHPAGNLLEEINEQDMSSIAGGTTDAYSKSLANRAGLGNNYGQYCTVSAECFGTISCGS, Probabilities: 0.9624\n","Identifier: MBS7264532.1 gallidermin family lantibiotic [Eubacteriales bacterium], Sequence: MFFRHGSFCGGLVLRHFFELCTEDNAVNKQSEDNEMNKRFKSKFFCKDGDEKYAVD, Probabilities: 0.9606\n","Identifier: MBS8016460.1 lantibiotic salivaricin M precursor [Streptococcus suis], Sequence: MDTLDFEVSTQELEGTSGAGGWYSAFKMTLAGRCGRCFTCSYECTSNNVNC, Probabilities: 0.4250\n","Identifier: MBS5466800.1 plantaricin C family lantibiotic [Clostridium sp.], Sequence: MSKKSLRNPMKHVDNNFNPAGDSLIELQEAELNQVVGAGDHNSKLTSCGIFCTFTAECEGRYTISCC, Probabilities: 0.8285\n","Identifier: MBS6579501.1 plantaricin C family lantibiotic [Clostridiales bacterium], Sequence: MSINRNASMRPETNPAGNLLQEISEAELNQLSAGAGEPRASWGDFCTSTLECTFGTMICCPPET, Probabilities: 0.9814\n","Identifier: WP_148552174.1 gallidermin/nisin family lantibiotic [Paraclostridium bifermentans], Sequence: MNKDFDLNLKMDKAPSNDGKQGITSVAYCTPGCVTGQLCGSSECGLTRNCTGSIACA, Probabilities: 0.9803\n","Identifier: WP_212925848.1 plantaricin C family lantibiotic [Oceanobacillus sp. J11TS1], Sequence: MSKEAMINSWKDPLARENASTQNPAGNPFNELEEAQMDKVVGAGDMEAACTWTLPGGGGVCTLTSECIC, Probabilities: 0.9741\n","Identifier: WP_197196623.1 plantaricin C family lantibiotic [Bacillus sonorensis], Sequence: MSKIEAWKNPVARMNSQIVSPAGDLMDELSDSEMEMLAGGCEWYNISCKLGNKGQWCTLTKECQRSCK, Probabilities: 0.8355\n","Identifier: WP_212844514.1 mersacidin/lichenicidin family type 2 lantibiotic [Catellatospora sp. IY07-71], Sequence: MSISDVVRNWTDEFHRATSPELSQAAHPAGLVEMSRDELQTVAGGAAIDGHTSMCCHCVC, Probabilities: 0.7314\n","Identifier: WP_030445134.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinocatenispora sera], Sequence: MSAEELVRAWRDEDFRLELESAVAHPAGEVDRELYVSTGAASTCNYASVTCAHICSCAA, Probabilities: 0.9801\n","Identifier: WP_211716020.1 mersacidin/lichenicidin family type 2 lantibiotic [Nocardiopsis sp. MG754419], Sequence: MDIAKAWKNPLYRASLSEKQRDSLPEHPAGGIVVSDEEMDSVNGGTTVPCSAVTAAVCNTFTDTVCHGSCGSWSVGCC, Probabilities: 0.7450\n","Identifier: WP_093338803.1 plantaricin C family lantibiotic [Salibacterium halotolerans], Sequence: MEEKFYAVAGDVEQELEDISLSSISGGADAEPRGISQGNDGDYCTITWECGMCPTHTCWC, Probabilities: 0.7364\n","Identifier: WP_212323102.1 plantaricin C family lantibiotic [Arachnia rubra], Sequence: MQDQNGFQDVTWMQEVAEQEFDTDVFGACTTNTFSLSDYWGNRGHLCSITSECMSWCR, Probabilities: 0.9827\n","Identifier: WP_158645521.1 mersacidin/lichenicidin family type 2 lantibiotic [Stackebrandtia albiflava], Sequence: MSTDTLTRAWTDVEFRATLSDAELDLLANPAGDLDAEMNELAVQEATAVSTACTKWSGLTPPRRCCC, Probabilities: 0.5916\n","Identifier: WP_141785536.1 mersacidin/lichenicidin family type 2 lantibiotic [Ornithinicoccus hortensis], Sequence: MDIVRSWKDPQSRSADMAHPSGSAEIAELTELEMDEVSGAGSGLVGTFGCCWCVPWYSGWTVCGLVCDHPGC, Probabilities: 0.7164\n","Identifier: WP_121939392.1 mersacidin/lichenicidin family type 2 lantibiotic [Eilatimonas milleporae], Sequence: MKTKFQAESYTDGKVSADETVQDAVNPAGAMLDIDDDALASVAGGCGCPPGSGSAMCSPCPPLQCY, Probabilities: 0.6848\n","Identifier: WP_121939389.1 mersacidin/lichenicidin family type 2 lantibiotic [Eilatimonas milleporae], Sequence: MKTKFQAESYADGKVSADETVQDAIDPAGAMLDIEDEALASVAGGCGCPGGATSALCSPCPPRDCF, Probabilities: 0.9352\n","Identifier: WP_047860824.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium gephyra], Sequence: MRADTIIRAWKDPAFRASLTSEERAALPELPSGKPVTELEEDELADAVGGMRHYPPRGCGGPREFTDFPCNAVDACPSALACTRFDC, Probabilities: 0.5829\n","Identifier: WP_157807626.1 mersacidin/lichenicidin family type 2 lantibiotic [Hymenobacter chitinivorans], Sequence: MTIRAWKDAEYRATLTPEQLAQVEPNPVGSALTEEELTQNTGGLMVGPGWWPTISGECNGGYCCNPFGSNIG, Probabilities: 0.6039\n","Identifier: WP_157807625.1 mersacidin/lichenicidin family type 2 lantibiotic [Hymenobacter chitinivorans], Sequence: MEKQLMTIRAWKDAEYRATLTPEQLALIEPNPIGAALTEEELTQHTGGEAFEDVVGPGWWPTISGECNGGYCCNPFGSGR, Probabilities: 0.6644\n","Identifier: WP_047856213.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium gephyra], Sequence: MSNIDVVRAWKDEQYRMSLTTEERAQLPQNPAGMVELTDSDLEGVAGGYAEAVDIDVSVTKTSCCTYSTTATACC, Probabilities: 0.4462\n","Identifier: WP_078577551.1 plantaricin C family lantibiotic [Salipaludibacillus agaradhaerens], Sequence: MEELYAIAGNVAEEMEEISLNSVSGGADVQGISQGNDGKYCTITWECSICPTHTCWC, Probabilities: 0.8485\n","Identifier: WP_033317973.1 plantaricin C family lantibiotic [Streptomyces yerevanensis], Sequence: MNPSYQGDVSLLEEIAEQDFDGVAYGACTTNTFSLSDALGNNGGWCTLTKECQPNCN, Probabilities: 0.9556\n","Identifier: WP_026701915.1 plantaricin C family lantibiotic [Bacillus aidingensis], Sequence: MTEKFYAAAGDVEQELEDISLKSINGGADVETRGLSQGNDGDYCTITWECGMCPTHTCWC, Probabilities: 0.7228\n","Identifier: WP_020557577.1 SCO0268 family class II lantipeptide [Streptomycetaceae], Sequence: MNDKILLEETTETRDFADFDLDVRIAEGELEAADFGAAGGGWTSPSTYGVQSRCPLCCA, Probabilities: 0.9334\n","Identifier: WP_018353405.1 mersacidin/lichenicidin family type 2 lantibiotic [Longispora albida], Sequence: MTALLESTDIDVAQIWRDPAFRASLGMDELETLPASPIGELRAVNVGEITSGESAVVSILSPAGCLISFACTWPCCAW, Probabilities: 0.8637\n","Identifier: MBR4357890.1 plantaricin C family lantibiotic [Butyrivibrio sp.], Sequence: MSRFENKLDRADINLACGTVAEEIDAQNMNEEAVGGASSSDLGNNGSACSLTNECQKICSIIRYSQWGFWCK, Probabilities: 0.9808\n","Identifier: NBC46284.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus exiguus], Sequence: MTPDMIIRAWKDPSFRESLTAEQRTQLPANPSGATLNELSETELMDVVGGRIDIGFTATNSCEYICTNLTFSLC, Probabilities: 0.6495\n","Identifier: WP_200073465.1 gallidermin/nisin family lantibiotic [Saccharopolyspora sp. HNM0986], Sequence: MSAATDQAVAEAEVDLFDLDLEVGFEATMANPAVTSVSYCTPGCTSEGGGSNCSHCC, Probabilities: 0.8619\n","Identifier: WP_210947903.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces sp. MK37H], Sequence: MNRTTKAWKDPVFRAGLSSEELSALPAHPAGAIELTDQRLGNVVGGGLTDTWVTCGHKTCRTCFTCDWLCG, Probabilities: 0.6005\n","Identifier: WP_061079715.1 mersacidin/lichenicidin family type 2 lantibiotic [Nocardiopsis alborubida], Sequence: MSTERAWKDPDYRDRLTEDEAADMPENPAGTSPLDNRRLNEIHGGTVVTTRPVNTVALCPLTLVPSLCGTLGRD, Probabilities: 0.9760\n","Identifier: MBP8258532.1 cinnamycin family lantibiotic [Verrucomicrobia bacterium], Sequence: MQTITSPARSPRLQQVMFKAAIDPEFRRDLSDHPETMLDSEELSLLRAMGAALPEAVQVGDLEIADQLDSVDLFGANSTHCRCTCISGFTIKCDGRSL, Probabilities: 0.7663\n","Identifier: WP_154536788.1 plantaricin C family lantibiotic [Bifidobacterium sp. WCA-178-WT-4B], Sequence: MDEQSFDPTYLEDISEQTLDTDAFGACTTNTFSLSDYWGNKGDWCTLTHECMNWCK, Probabilities: 0.9812\n","Identifier: WP_031138450.1 plantaricin C family lantibiotic [Streptomyces xanthophaeus], Sequence: MANPNVGILEEITDQNLDAVSAAGQGWGTVMSSLSCYGASYVLGNNGNFCTATAECQNNCK, Probabilities: 0.6940\n","Identifier: WP_135338980.1 mersacidin/lichenicidin family type 2 lantibiotic [Streptomyces palmae], Sequence: MDIVRAWKDPEYRSTLPASPQHPSGVPGLVRLEAPELAGVAGAGTQSLLSFGCCQSPRTFTLTSPIFCSISLAIC, Probabilities: 0.9039\n","Identifier: WP_030319753.1 plantaricin C family lantibiotic [Streptomyces flavochromogenes], Sequence: MNNYDGDISLIEEIAEQDFDGVAYGACSTNTFSLSDFLGNQGGWCTLTKECQPNCN, Probabilities: 0.9297\n","Identifier: WP_172763328.1 plantaricin C family lantibiotic [Dolosigranulum pigrum], Sequence: MKKEELVGTAKENFLNVICENDNKLENSGAKCPWWNLSCHLGNDGKICTYSHECTAGCNV, Probabilities: 0.9516\n","Identifier: WP_208957739.1 plantaricin C family lantibiotic [Dolosigranulum pigrum], Sequence: MKKEELVGTAKEDFLNVICENDNKLENSGAKCPWWNLSCHLGNDGKICTYSHECTAGCNA, Probabilities: 0.9767\n","Identifier: WP_003991782.1 plantaricin C family lantibiotic [Streptomyces viridochromogenes], Sequence: MCRGSLSRDPAAPAINLEETVYRGDISLLEEIEEQDFAFGAGCSANFTTFSLSGIFGNDGDCCTLTKECQASCN, Probabilities: 0.9769\n","Identifier: WP_208773770.1 plantaricin C family lantibiotic [Enterococcus faecalis], Sequence: MKKEELVGLAKEDFLNVICENDNKLENSGAKCPWWNLSCHLGNDGKICTYSHECTAGCNA, Probabilities: 0.7837\n","Identifier: WP_208721280.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus aberystwythensis], Sequence: MMTPDMIIRAWKDSSYRASLTAEQRAQLPANPSGVTVNELSESELRSVVGGKIDIGFTATNSCEYICTNLTFVC, Probabilities: 0.7101\n","Identifier: WP_208647379.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus interemptor], Sequence: MMTPDMIIRAWKDPSFRESLTAEQRAQLPANPSGATLNELNETELMDVVGGRIDIGFTATNSCEYICTNLTFSLC, Probabilities: 0.6930\n","Identifier: WP_053802155.1 hypothetical protein [Streptomyces], Sequence: MTPKAELADLARDMLALESETFEINDYSEFDGKLLGVSSTSCDSSSCSSSTTSTTSCCTA, Probabilities: 0.9052\n","Identifier: WP_199832729.1 lantibiotic protein [Streptomyces puniciscabiei], Sequence: MTPKTDLADLAQEMLALESETFEINDYPDTQGVLLNATAPAATAAARPARWSPACACSSTRSCRTFDRR, Probabilities: 0.9803\n","Identifier: WP_120646679.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus llansteffanensis], Sequence: MSDNVKSEIIRAWKDEEFRNQLSASERDLIPANPAGILELTDEVLEVASGGVIAASCDWCSC, Probabilities: 0.5057\n","Identifier: WP_120529931.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus exercitus], Sequence: MSNKDIIEYWKNSEHRLNLTAEERAKLPENPAGLVELTDEALDDLISGGVAEDVEVSSCGWDSCN, Probabilities: 0.6186\n","Identifier: WP_120556222.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus aberystwythensis], Sequence: MMTQDMIIRAWKDPSFRATLTAEQLAELPANPSGLTMNELNEGDLMDVVGGRIQAPVSHANSCGYICTLTTECGC, Probabilities: 0.4267\n","Identifier: WP_200274177.1 plantaricin C family lantibiotic [Clostridium sp. YIM B02505], Sequence: MRNIEMLKNPVLSNKYSVSENNPAGDLLVEIREQDFTDSVTGGYDSEGLGNQGKKCSLTRECQRLCNWISYGSGGVFGC, Probabilities: 0.9738\n","Identifier: WP_207663704.1 gallidermin/nisin family lantibiotic [[Ruminococcus] gnavus], Sequence: MQMNKDFDLNLKADKSTTEGPAPRITSVAYCTPGCITGDLCGSSECCLTRNCTGSILCA, Probabilities: 0.9592\n","Identifier: WP_205606131.1 plantaricin C family lantibiotic [Clostridium sporogenes], Sequence: MSNVSMLKNPVLRNKFSNNEENPAGDLLVEVSEQDFSLNISGGYDSKGLGNTGKRCSWSRECQRLCNWISYGSGGVIGC, Probabilities: 0.5457\n","Identifier: WP_170819691.1 gallidermin/nisin family lantibiotic [Hungatella xylanolytica], Sequence: MNKDFDLDLKMDKGSVDVGESKIASVAYCTPGCLTGSTCGTSECGMTRNCTNTWLCY, Probabilities: 0.9747\n","Identifier: WP_164952171.1 plantaricin C family lantibiotic [Clostridium sp. FP2], Sequence: MSIIKNPLLRNAKNYNEDLASGNLLKEITDNQLETISGGTNDWVSKQAANIAGLGTNYGRVCTVSAECDFTNLCAGTK, Probabilities: 0.9380\n","Identifier: WP_164952022.1 plantaricin C family lantibiotic [Clostridium sp. FP2], Sequence: MSIIKNPLLRNAKNYNEDLASGNLLKEITDNQLETISGGTNDWLSKKAANMAGLGNNYGRICSVSAECEGFLCGDTK, Probabilities: 0.9832\n","Identifier: WP_164952020.1 plantaricin C family lantibiotic [Clostridium sp. FP2], Sequence: MSIIKNPLLRNAKNYNEDLASGNLLKEITDNQLETISGGTNDWVSKQAANIAGLGTNYGRVCTVSAECDFTNLCGGTK, Probabilities: 0.9828\n","Identifier: WP_164952021.1 plantaricin C family lantibiotic [Clostridium sp. FP2], Sequence: MSIIKNPLLRNAQNYNEDLASGNLLKEITDNQLETISGGTNDWLSKKAANMAGLGNNYGRVCTVSAECDFTNLCGDTK, Probabilities: 0.9834\n","Identifier: WP_164473178.1 mersacidin/lichenicidin family type 2 lantibiotic [Clostridium sp. E02], Sequence: MSKNSMINAWKNPMHRLADIENPAGNAIVELNDLELDDLTGAASSGNICTATTECTYWSAICC, Probabilities: 0.9737\n","Identifier: WP_118445437.1 gallidermin/nisin family lantibiotic [[Ruminococcus] gnavus], Sequence: MNKDFDLNLKTDKNTTEGSAPRITSVAYCTPGCITGELCGSSECGLTRNCTGSILCA, Probabilities: 0.9756\n","Identifier: WP_117628691.1 plantaricin C family lantibiotic [Eubacteriales], Sequence: MKDLRNPIERRNFTHPAGNTMTELKEAELNNFAAGAGEPRNSGGMFCTSTGECNMGTLNIFCC, Probabilities: 0.6815\n","Identifier: WP_118445445.1 gallidermin/nisin family lantibiotic [[Ruminococcus] gnavus], Sequence: MNKDFDLNLKADKSATEGPAPRITRVAYCTPGCVTGELCGSSECGLTRNCTGSILCA, Probabilities: 0.9745\n","Identifier: WP_118445436.1 gallidermin/nisin family lantibiotic [[Ruminococcus] gnavus], Sequence: MNKDFDLNLKADKSATEGPAPRITSVAYCTPGCVTGELCGSSECGLTRNCTGSILWA, Probabilities: 0.9783\n","Identifier: WP_117416424.1 gallidermin/nisin family lantibiotic [Clostridium indicum], Sequence: MNKDFDLDLKMDKGSVDVGESKIASIVYCTSGCLTGTTCETSECGITRSCTNTVLCY, Probabilities: 0.7581\n","Identifier: WP_104436664.1 gallidermin/nisin family lantibiotic [Hungatella xylanolytica], Sequence: MNKDFDLDLKMDKGSVDVGESKIASVAYCTPGCLTGSTCGTSECGLTRNCTNTWLCY, Probabilities: 0.9760\n","Identifier: WP_040470420.1 lichenicidin A2 family type 2 lantibiotic [Eubacterium plexicaudatum], Sequence: MAMDNEKKSQLQEESSDMDILGKKFEDLTQEEMKKIQGSGDVSPEDIGAFTTILTSKNPYNCK, Probabilities: 0.8257\n","Identifier: QSZ28270.1 gallidermin/nisin family lantibiotic [thermophilic bacterium 3443-3Ac], Sequence: MKDFNDFDLDVQIKKNNQKIEPDITSKSLCTPGCITGILMCFTQGCFSNHSCIACRD, Probabilities: 0.7680\n","Identifier: WP_083525003.1 gallidermin/nisin family lantibiotic [Thermotalea metallivorans], Sequence: MPKYDDFDLDIQVKKVDNLVEPNITSVSLCTPGTCWNTCGGTSTLNTNCCIATYLCSLAESCR, Probabilities: 0.9622\n","Identifier: MBO0801152.1 mersacidin/lichenicidin family type 2 lantibiotic [Blastocatellia bacterium], Sequence: MLETDKIIRAWKDDDYRMSLSADEQRSLPANPAGMVEFSSGADLNHAAGSYYCFSYFRSLMVLVCICA, Probabilities: 0.9587\n","Identifier: WP_130510116.1 plantaricin C family lantibiotic [Krasilnikovia cinnamomea], Sequence: MYEGDVALLEEIDEQDFEGVAYGACTTNTFSLSNVLGNNGGWCTLTRECQRSCN, Probabilities: 0.7188\n","Identifier: WP_130133114.1 gallidermin/nisin family lantibiotic, partial [Staphylococcus aureus], Sequence: NNNTNDSAGDERITSHLFCSFGCEKTGSFNSFCC, Probabilities: 0.7692\n","Identifier: WP_129888396.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonosporobacter rubrisoli], Sequence: MMNIESIIQAWKAEEEARDGWTSLMPALPVGEELTEQELEEVVGGFCGQHAFTGRQCADGQEW, Probabilities: 0.8891\n","Identifier: WP_206790744.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. NCSPR001], Sequence: MMTPDLIIRAWKDPSFRESLTAEQRAQLPANPSGATLNELSETELMDVVGGRIDIGFTATNSCEYICTNLTFSMC, Probabilities: 0.8709\n","Identifier: WP_061860274.1 plantaricin C family lantibiotic [Priestia megaterium], Sequence: MREKFLKNPVLQSKHSLDVENPSGNLFEEISEQDMSAMAGGTNDWASKQAANLTGLGNNYGRTCTLSAECNGTDSY, Probabilities: 0.6055\n","Identifier: WP_139032007.1 plantaricin C family lantibiotic [Priestia megaterium], Sequence: MMEKFLKNPVLQSKHSLDVENPSGNLFEEISEQDMSAMAGGTNDWASKQAANLTGLGNNYGRVCTLSAECNGTDPCGGTK, Probabilities: 0.9418\n","Identifier: WP_190873712.1 mersacidin/lichenicidin family type 2 lantibiotic [Aulosira sp. FACHB-615], Sequence: MSNFDIIRAWKDEDYRNSLSDEQRSQLPENPAGLIELPDAESNALSGGGCSLFTGTCGRVCERLTPQYGCTGCDNGGLTT, Probabilities: 0.9524\n","Identifier: WP_190574770.1 mersacidin/lichenicidin family type 2 lantibiotic [Alkalinema sp. FACHB-956], Sequence: MSQENVIRFWKDESFRNRLKDRDQTELPAHPAGLIQLNDADLGSVAGGFNSCLAGSHQPVCQDRCGI, Probabilities: 0.9803\n","Identifier: WP_190521178.1 cinnamycin family lantibiotic [Planktothrix sp. FACHB-1365], Sequence: MSTTELLENSVDEQVDEEWLLRSVVDFDFRNEFSSIDFSTLPASVLPQDMSFAEIVKNANQAACVETCISGFTIVCDGRSYECRSTCITGWTMRCDGTSL, Probabilities: 0.6789\n","Identifier: WP_190565341.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostoc parmelioides], Sequence: MSNFDIIRAWKDEDYRNSLSDEQRSQLPENPAGMIELPNEVTQTLAGGGFSKGVFCTGLKGFTKGVICDIFTRN, Probabilities: 0.8877\n","Identifier: WP_206723358.1 mersacidin/lichenicidin family type 2 lantibiotic [Pyxidicoccus sp. SCPEA002], Sequence: MSDTVKSEIIRAWKDEEFRNNLSESERDLIPANPAGILELTDEVLEVASGGVVAASCDWCSC, Probabilities: 0.5716\n","Identifier: WP_073943648.1 gallidermin/nisin family lantibiotic [Streptomyces sp. TSRI0107], Sequence: MGTHGPAQYDDADLFDLDLEIGVEKSVAGPAITSVSYCTPGCTSDGGGSGCSHCC, Probabilities: 0.5293\n","Identifier: WP_073725586.1 gallidermin/nisin family lantibiotic [Streptomyces sp. CB02414], Sequence: MGIHAPATGQNDANLFDLDLEIGVEKSVAGPAITSVSWCTPGCTSEGGGSGCSHCC, Probabilities: 0.8928\n","Identifier: WP_190593185.1 mersacidin/lichenicidin family type 2 lantibiotic [Nostocaceae], Sequence: MSNFDIIRAWKDEDYRNSLSDEQRSQLPENPAGLIELPDAESNALSGGGCSIFTGTCGRVCERLTPQYGCKGCDNGGLTT, Probabilities: 0.9240\n","Identifier: MBN9389167.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium], Sequence: MEEQLGLKGVDVARALKDEEYRASLGSEARLALQNLMEEGEFSEQDLDNVSGGFIMKDSIIVRTGGFTKPVPQTDTLR, Probabilities: 0.9804\n","Identifier: WP_206425350.1 plantaricin C family lantibiotic [Staphylospora marina], Sequence: MSREIRMWKDPVFRLQSEGSMMEHPSGNVMEELREQDLLSVAGGCDWWNISCHLGNTGRFCTLTKECQPNCNY, Probabilities: 0.9701\n","Identifier: WP_169089145.1 plantaricin C family lantibiotic [unclassified Streptomyces], Sequence: MTNYDGDISLIEEIAEQDFDGVAYGACSTNTFSLSDFLGNQGGWCTLTKECQPNCN, Probabilities: 0.7970\n","Identifier: WP_124727338.1 plantaricin C family lantibiotic [Staphylospora marina], Sequence: MNQLVRSWKDPLFRVQTEQNRMEHPSGNVLEELREQDLLSVAGGCDWWNISCHLGNTGQFCTLTKECQPSCNR, Probabilities: 0.9795\n","Identifier: WP_206157082.1 gallidermin/nisin family lantibiotic [Staphylococcus sp. EG-SA-32], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHLFCSFGCEKTGNFNSFCC, Probabilities: 0.9675\n","Identifier: WP_165626760.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHLFCSFGCEKTGSFNNFCC, Probabilities: 0.8335\n","Identifier: WP_205694646.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus coralloides], Sequence: MMTPDLIIRAWKDPSFRESLTAEQRAQLPANPSGTTVNEMSETELMDVVGGKIDIGFTATNSCEYICTNMTFVC, Probabilities: 0.7889\n","Identifier: WP_166669242.1 lichenicidin A2 family type 2 lantibiotic [Bacillus sp. BH2], Sequence: MLKKEGNRLNNFEDKVFKNMEDRFENVGDAFEDMTLEDMALAQGSGDIQPRTTPGCFFASLGVSVTISAWSC, Probabilities: 0.8958\n","Identifier: WP_205663413.1 plantaricin C family lantibiotic [Priestia megaterium], Sequence: MGQDLIKHWKNPMSRTSLGANNLPQHPAGDVMSELSDAELNNVMGAGEMNAAAKSGGNVCTYSVECDFCPTRNCTGIPWFGC, Probabilities: 0.9463\n","Identifier: WP_124749683.1 plantaricin C family lantibiotic [Bacillus cereus], Sequence: MKELLKNPVLQNKYSMKEKNPAGDLIQEVNEQEMQEVNGGIGPAESRSQARRMGLGDDYGSLCTISAECLFTISCGS, Probabilities: 0.9312\n","Identifier: WP_124749682.1 plantaricin C family lantibiotic [Bacillus cereus], Sequence: MKELLKNPVLQNKYSMKEKNPAGDLLQEVNEQEMQEVNGGVLGPAESRAIARKMGLGNNYGSMCTVSAECMFTISCGS, Probabilities: 0.8284\n","Identifier: WP_082188801.1 plantaricin C family lantibiotic [Bacillus cereus], Sequence: MGKMSEKMYELAGDLSKELEEISIQSINGGSDVDVRGIGISSGNDGGVCTLTWECGKCPTVTCWSWSC, Probabilities: 0.7555\n","Identifier: WP_153834672.1 plantaricin C family lantibiotic [Gracilibacillus thailandensis], Sequence: MSEKMYELAGDLSTELDEISLESINGGSDVDVRGISEGNDGDFCTLTWECGKCPTVTCWNWSC, Probabilities: 0.9694\n","Identifier: WP_093951064.1 mersacidin/lichenicidin family type 2 lantibiotic [Amycolatopsis vastitatis], Sequence: MSDHEIIRSWKDEEFRNSLGPAQRAVLPGNPAGLIELTDEALDDLIAGAAALDSCCWSSCTSTQ, Probabilities: 0.5157\n","Identifier: WP_205507739.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus vastator], Sequence: MSHSDKNHILRAWRDADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEATCPTTPTPNELTCLNTACGMSACCR, Probabilities: 0.5039\n","Identifier: MBN2920455.1 plantaricin C family lantibiotic [Lactobacillus sp.], Sequence: MSEMRNPIKRNAGFNHPSGNVMTELQEADLNKFVAGAGETRDSGGVLCTSTAECYLGTVLFFCCP, Probabilities: 0.9654\n","Identifier: WP_076759801.1 plantaricin C family lantibiotic [Bacillus swezeyi], Sequence: MSKENIVLSWKDPLSRKNNLQNPAGNPFTELEEAQMDKLAGAGDMEAACTWTLPGGGGVCTVTSECIC, Probabilities: 0.9267\n","Identifier: WP_136312916.1 plantaricin C family lantibiotic [Actinomyces procaprae], Sequence: MNAKFTSTSTAKNDLLDEITEQDYITANGGVSWLSRKLGNRGFFCTLTKECQSMCN, Probabilities: 0.8798\n","Identifier: WP_095958469.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus macrosporus], Sequence: MSQKKDHILRAWRDPEYFNNLSAEARASLPANPAAELELSDEVLESISGADSCERFGSYHCTPCPPYVCL, Probabilities: 0.5873\n","Identifier: WP_013937967.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus macrosporus], Sequence: MSHSDKNHILRAWRDADYYDSLSDAERAALPASPASVMELEDATLAFITGGDTESTCPATPTPNELTCLYTNCGRIACCR, Probabilities: 0.9474\n","Identifier: MBC8122624.1 mersacidin/lichenicidin family type 2 lantibiotic [Gemmatimonadaceae bacterium], Sequence: MQSKKSDYIPAKFGKLTGLIQRYAQDAEFRSSDMTPNNDRTERSWKDEDFRKQLSPEEQAALAPSPVEPQGELLDEDELDEIAGGAPSLELGCGNLL, Probabilities: 0.9778\n","Identifier: WP_161354092.1 SCO0268 family class II lantipeptide [Streptomyces sp. SID3343], Sequence: MAEPTDLDLDVRIVESDLASTNFGTSGDYTSPSTYGVQSRCPLCC, Probabilities: 0.8965\n","Identifier: WP_109171310.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHFLCSFGCGKTGSFNSFFC, Probabilities: 0.5568\n","Identifier: WP_203960207.1 mersacidin/lichenicidin family type 2 lantibiotic [Actinocatenispora thailandica], Sequence: MSAEELVRAWRDEDFRLELESAVAHPAGEVDRELYVSTGAASTCNYASVTCSHICSCAI, Probabilities: 0.9779\n","Identifier: WP_203750392.1 mersacidin/lichenicidin family type 2 lantibiotic [Catellatospora bangladeshensis], Sequence: MSISDVVRNWTDEFHRATTPELSQAAHPAGLVEMSRDELQTVAGGAAIDGHTSMCCHCVC, Probabilities: 0.7691\n","Identifier: WP_086095970.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSLLFCSFGCEKTGSFNSFCC, Probabilities: 0.8521\n","Identifier: WP_152911913.1 type 2 lantibiotic [Enterococcus sp. RIT-PI-f], Sequence: MSDKELQEVEEYEESFERLDISEMEQVQGAGDLNGELISVNTLPIKLSTSIINLFKK, Probabilities: 0.8873\n","Identifier: WP_082522658.1 gallidermin/nisin family lantibiotic [Bacillus sp. Leaf75], Sequence: MNNVKNLFDLDVQVTTASSDVDPQITSVSLCTPGCGDTGXFDLDVQVTTASSDVDPQITSVSLCTPGCGDTGSWNSFCC, Probabilities: 0.9416\n","Identifier: WP_056699840.1 plantaricin C family lantibiotic [Paenibacillus sp. Root52], Sequence: MHKNILRNPLKRNAEFENPAGNIMIELSEAELNKVHGAANSPNTYNSCYSKQKSCGNFCTFTTECPFLSFCC, Probabilities: 0.5330\n","Identifier: WP_084724180.1 mersacidin/lichenicidin family type 2 lantibiotic [Thalassomonas viridans], Sequence: MSVSQIVKSWKDPAYRRSLSQGQLQKIPGHPAATNELDKEVMDGNPQAFTAMGSAICSPCPPRHCF, Probabilities: 0.8846\n","Identifier: WP_084692501.1 mersacidin/lichenicidin family type 2 lantibiotic [Thalassomonas actiniarum], Sequence: MSVSQIIKTWKDPAYRRSLSQGQLEKTPGHPAASNELDQEVIDGNPEALTMNGSAICSPCPPRHCL, Probabilities: 0.5560\n","Identifier: WP_203403106.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium violaceum], Sequence: MDIVRAWKDADYRMSLTQEQLAQLPENPVGAVELSDEQLQGAGGVAAQSASFYSCPTKSICSYCDCI, Probabilities: 0.4602\n","Identifier: WP_162647179.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAADERLTSYSLCSFGCGKTGSFNSFCC, Probabilities: 0.7014\n","Identifier: WP_141121471.1 plantaricin C family lantibiotic [Rothia dentocariosa], Sequence: MSKDITLLEEIAEQDLERRSGGAASVGSWILGDPDIICTWTLECQKNCR, Probabilities: 0.9352\n","Identifier: WP_037583347.1 mersacidin/lichenicidin family type 2 lantibiotic [Stigmatella aurantiaca], Sequence: MNPKNIIRAWKDPAFRASLSPEERAALPESPSGQPLSELSEEELLGITGGQVSKGEIGPSTGCTGPVRPTCGIVICPITRDMAI, Probabilities: 0.4790\n","Identifier: WP_013939440.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus macrosporus], Sequence: MSQKKDHILRAWRDPEYFNSLSAEARASLPANPAAELELSDEVLESISGADSCERFGSYHCTPCPPYVCL, Probabilities: 0.6303\n","Identifier: WP_165952514.1 mersacidin/lichenicidin family type 2 lantibiotic [Kribbella albertanoniae], Sequence: MPTLEVISAWTDLDVRAALSEAELAQVPVHPAGDLDEELRQMLPENSGATTHCTQWSGRPVCCC, Probabilities: 0.8886\n","Identifier: WP_014398760.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus coralloides], Sequence: MMTPDMIIRAWKDPSFRESLTAEQRAQLPANPSGATLNELSETELMDVVGGRIDIGFTATNSCEYICTNLTFSLC, Probabilities: 0.7198\n","Identifier: WP_081495980.1 gallidermin/nisin family lantibiotic [Actinomyces sp. ICM58], Sequence: MVTVDLGTRVEEVEETVRPQITSKFLCTPGCPTGGIACFTTQCTNGYSISVGCKWLVALSQQSALEVKHPRPNLQGRFL, Probabilities: 0.7401\n","Identifier: WP_191261993.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Comamonas], Sequence: MSHKKDHILRAWRDPEYFNSLTAEERAALPTNPAAELELGDDLLESISGGRSEPEQCSALCTPCPPYHCL, Probabilities: 0.5105\n","Identifier: WP_191261994.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Comamonas], Sequence: MNQNEHILRAWRDPQYFNALTAEERAALPANPAAELELGDDVLESISGADSCEQFGSPYCTPCPPIECY, Probabilities: 0.6843\n","Identifier: WP_140877374.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus], Sequence: MSHSDKNHILRAWRDADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEASCPASPTPNELTCLYTNCGRIACCR, Probabilities: 0.9690\n","Identifier: WP_140795328.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus], Sequence: MSHSDKNHILRAWRDADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEASCPATPTPNELTCLYTNCGRIACCR, Probabilities: 0.9376\n","Identifier: WP_174260093.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus], Sequence: MSDNNKSHILRAWRDPDYHDSLSDAERAALPDSPASVLELDDATLAFITGGDGASTCPNTPPPSTPRLVSGRSAGESPAASERRPAMT, Probabilities: 0.6227\n","Identifier: WP_140790406.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus], Sequence: MSQKKDHILRAWRDPEYFNSLSSEERASLPANPAAELELGDDLLEVITGGDFCLPGQSSPQCTPCPPRHCF, Probabilities: 0.6943\n","Identifier: WP_055426429.1 lichenicidin A2 family type 2 lantibiotic [Bifidobacterium aesculapii], Sequence: MSMNEKQIVGESFEDLSDKDMAMLTGRGDDEVAPASFTPTMGEFVTLTLISIQVCTRTLYKCK, Probabilities: 0.5900\n","Identifier: WP_140790402.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus], Sequence: MSQKKEHILRAWRDPEYFNSLSSEERASLPANPAAELELSDEVLENISGADTCEQFGSYYCTPCPPYHCL, Probabilities: 0.5323\n","Identifier: WP_201394356.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacter sp. SOSP1-52], Sequence: MSIEEIIRAWKSSEGYDESILPASPVGSELSDEELKEVTGGMPCAVSCADNNTCMRYLSCIGNSSVA, Probabilities: 0.9783\n","Identifier: WP_201384612.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacter sp. SOSP1-85], Sequence: MSIEEIIKAWKSEEQALEVSVPANPVGQELSEEELQEVVGGERCTITCEEISLQCNPTCYTSYA, Probabilities: 0.9478\n","Identifier: WP_201368822.1 mersacidin/lichenicidin family type 2 lantibiotic [Ktedonobacter robiniae], Sequence: MSIEEIVKAWKSEEMALEAHIPANPVGQELTEKELQEVVGGMRCTVTCEETSLVCNPTCRLSSLVNN, Probabilities: 0.9726\n","Identifier: WP_201361146.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter formicarum], Sequence: MNNIEQDYRTSLTSEEQVELPQHPSSVRELAEEELAGVHGAFGPYGPNYGGGYGYGGGGYGEGYGYGGGGYGEGYGYGGGDGGGIGIGLGLGIGLGDLL, Probabilities: 0.9442\n","Identifier: WP_201361239.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter formicarum], Sequence: MSSVDVVRAWTDEAYRRSLGPAELAQLPANPAGEIELIDADLATICGGDGIGLGIGIGLDRSHDQQFSLGCSADCLKSSYCFTFGPCC, Probabilities: 0.6584\n","Identifier: WP_061088025.1 hypothetical protein [Streptococcus mitis], Sequence: MAKDAERPVIDSLDFEVANQELAGKSGSGWFTAIQLTLAGRCGLCFTCSYECTSNNVHC, Probabilities: 0.7467\n","Identifier: WP_026588346.1 plantaricin C family lantibiotic [Bacillus sp. NSP9.1], Sequence: MGKDFKNPILRRNNNENHPSGNLLEEIREQDMSEIAGGTTDAVSAAIARSVGLGNNFGKYCTWSAECAGTVSCGK, Probabilities: 0.9586\n","Identifier: WP_087097274.1 lantibiotic protein [Nocardiopsis sp. JB363], Sequence: MSRTTALSNLQGELELLESETFEVLDLVEAREEMLGSTTSTTSCCSSCSSSSCSSTSCAGG, Probabilities: 0.8597\n","Identifier: WP_199794786.1 plantaricin C family lantibiotic [Paenibacillus sp. Marseille-P3787], Sequence: MVDNRNPFIVYNRVEKMESLSPVLAEELIEIDQSIQGGCTTNTFTLSDYLGNKGGWCTLTKECQRSCN, Probabilities: 0.9697\n","Identifier: WP_199746864.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. AB049A], Sequence: MMTPDMIIRAWKDPSFRESLTAEQRAQLPANPSGATLNELNETELMDVVGGRIDIGFTATNSCEYICTNLTFSQC, Probabilities: 0.7477\n","Identifier: WP_199738350.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA054B], Sequence: MMTPDMIIRAWKDPSFRESLTSEQRAQLPANPSGETLNELSETELLDVVGGKIDIGFTATNSCEYICTNLTFVC, Probabilities: 0.8132\n","Identifier: WP_199733727.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA041A], Sequence: MMTPDMIIRAWKDPSFRESLTAEQRTQLPANPSGATLNELSEAELMDVVGGRIDVGFTATNSCEYICTNLTFSLC, Probabilities: 0.8052\n","Identifier: WP_120621866.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA049B], Sequence: MMTQDMIIRAWKDPSFRATLTVEQLAELPASPSGPTMNELNEGDLMDVVGGLQAVSHRNSCGYICTLTTECGC, Probabilities: 0.5233\n","Identifier: WP_120587568.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA054B], Sequence: MMTQDMIIRAWKDPSFRATLTVEQLAELPASPSGPTMNELNEGDLMDVVGGAQAVSHRNSCGYICTLTTECGC, Probabilities: 0.5352\n","Identifier: WP_120563712.1 mersacidin/lichenicidin family type 2 lantibiotic [unclassified Corallococcus], Sequence: MMTPDMIIRAWKDPSFRESLTAEQRAQLPSNPSGATLNELSETELMDVVGGRIDIGFTATNSCEYICTNLTFSLC, Probabilities: 0.7308\n","Identifier: WP_199036841.1 mersacidin/lichenicidin family type 2 lantibiotic [Glycomyces sp. YIM93776], Sequence: MNDIRAWKDPERRHELGGDVQDNPAGLVDLNDDAVLDVTGGTSWWCATVALTATFCSPTGTMCGSCQFGTRGCC, Probabilities: 0.5554\n","Identifier: WP_151758635.1 mersacidin/lichenicidin family type 2 lantibiotic [Dictyobacter vulcani], Sequence: MSVDIVRAWTDEAYRRSLGPADLAQLPVNPAGEMELIDADLATIYGGTGIGIGVGLGRPNDHFSLGCSADCLRSSYCFTFGPCC, Probabilities: 0.5715\n","Identifier: WP_073558340.1 mersacidin/lichenicidin family type 2 lantibiotic [Archangium sp. Cb G35], Sequence: MIVRAWKDPAYRASLTDEERSSLPESPSGRAMTELEDDELLGISGGKRGPDYSVDTFRCPAPTFFNCQDTVNSCGIVACV, Probabilities: 0.6271\n","Identifier: MBI5928042.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium], Sequence: MSNVDVVRAWKDAEYRNSLTAEQRASLPENPAGLVELDDAMLDASVGGGTELVASLGCCGGLTNYNTCFGLTQCGFLCTRAILSNCC, Probabilities: 0.7345\n","Identifier: WP_084369829.1 gallidermin/nisin family lantibiotic [Microbispora sp. ATCC PTA-5024], Sequence: MPADILETRTSETEDLLDLDLSIGVEEITAGPAVTSWSLCTPGCTSPGGGSNCSFCC, Probabilities: 0.9208\n","Identifier: WP_156857986.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Oceanobacillus sp. AG], Sequence: MTNGNLVRAWKDPMYRMSVDGEVNHPAGDTFTELSEAEMSGIAGAADV, Probabilities: 0.9773\n","Identifier: WP_015895698.1 mersacidin/lichenicidin family type 2 lantibiotic [Gemmatimonas aurantiaca], Sequence: MTDTLIRAWKDQHFRDSLPHATRAALPANPAGELANIDPEVGAAVGGESRTEFLLTLGCCQGFTQFCGAFTGGSVCTSDCMTIWLTTDAICAMK, Probabilities: 0.5486\n","Identifier: WP_053615066.1 FxLD family lanthipeptide [Nocardiopsis sp. NRRL B-16309], Sequence: MDTNRGSSMALAEQPAPSFGAVEDDFSLDVRVVVAAHPVGKLMCNTGDGCGNTCQGSASACNSYVSDPV, Probabilities: 0.9716\n","Identifier: WP_035298103.1 mersacidin/lichenicidin family type 2 lantibiotic [Brevibacillus thermoruber], Sequence: MITDQKGMVEVNGRKIRVDELIRAWKDPDYRTQLSHLLGRVPHPSGYPLDELSEEILDQE, Probabilities: 0.4998\n","Identifier: WP_025720095.1 plantaricin C family lantibiotic [Paenibacillus sp. 1-18], Sequence: MKNELRNPISRGEFMHPAGNVMEELGEAELQNFAAGGGIPWTSGGAICTLTHECAGTHWYSCC, Probabilities: 0.5016\n","Identifier: WP_196245349.1 gallidermin/nisin family lantibiotic [Paenibacillus polymyxa], Sequence: MAENLFDLDVQVNKSQGSVEPKIWSIWNCSIGCGGGNTSKTCVETCSPCFSNLGTLC, Probabilities: 0.9454\n","Identifier: WP_189980714.1 plantaricin C family lantibiotic [Streptomyces termitum], Sequence: MMNYDGDISLIEEIAEQDFDGVAYGACSTNTFSLSDFLGNHGGWCTLTKECQPNCN, Probabilities: 0.9420\n","Identifier: WP_189798675.1 plantaricin C family lantibiotic [Streptomyces thermodiastaticus], Sequence: MYQGDLSLLEEIEEQDFDGVAYGACTTNTFSLSDALGNSGGWCTLTKECQPNCN, Probabilities: 0.6973\n","Identifier: WP_190143308.1 plantaricin C family lantibiotic [Streptomyces glebosus], Sequence: MPFEGDVSLIEEIAEQDFDGVAYGACTTNTFSLSDAVGNHGGWCTLTKECQPNCN, Probabilities: 0.6456\n","Identifier: HAR4920471.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNINDSAGDERITSHLFCSFGCEKTGSFNSFCC, Probabilities: 0.9794\n","Identifier: HAR5004631.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHFLCSFGYGKTGSFNSFCC, Probabilities: 0.6698\n","Identifier: WP_153507188.1 plantaricin C family lantibiotic [Streptomyces alkaliterrae], Sequence: MEEISEQDLDGVAHGAGGVNAFSFSDKLGNDGKWCTLTKECQRSCN, Probabilities: 0.5607\n","Identifier: WP_185203111.1 plantaricin C family lantibiotic [Streptomyces alkaliterrae], Sequence: MLEEISEQDLDGVAHGAGGVNAFSFSDKLGNDGKWCTLTKECQRSCN, Probabilities: 0.6243\n","Identifier: MBB1254255.1 plantaricin C family lantibiotic [Streptomyces alkaliterrae OF3], Sequence: MSHEYEGDIALLEEISEQDLDGVAHGAGGVNAFSFSDKLGNDGKWCTLTKECQRSCN, Probabilities: 0.9491\n","Identifier: WP_193660179.1 plantaricin C family lantibiotic [Bacillus mojavensis], Sequence: MSIIEAWKNPVLRMNSQVANPAGDVMEELSDNEMEVLAGGCAWYDVSCRLGNKGAWCTLTVECQNNCN, Probabilities: 0.9794\n","Identifier: MBE5974414.1 gallidermin/nisin family lantibiotic [Paenibacillaceae bacterium], Sequence: MNKDFDLDLKMDKGSVDVGESKIASVAYCTPGCLTGTTCGTSECGMTRNCTNTWLCY, Probabilities: 0.9719\n","Identifier: WP_002635009.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus hansupus], Sequence: MSQNEHILRAWRDPQYFNNLTAEERAALPANPAAELELSDDLLESVSGADSCETIGSPYCTPCPPIECY, Probabilities: 0.5645\n","Identifier: WP_193064520.1 plantaricin C family lantibiotic [Oceanobacillus oncorhynchi], Sequence: MKEKDLQLFNTAGDFMSELEEASLKNISGGAAEPMGISEDNDGKYCTLTWECSICPTHTCWC, Probabilities: 0.9065\n","Identifier: WP_150224109.1 plantaricin C family lantibiotic [Bacillus atrophaeus], Sequence: MSQEKIIRSWKDPFALENSSSQNPAGNPFSELEEAQMDKLVGAGDMEAACTFTLPGGGGVCTLTSECIC, Probabilities: 0.9605\n","Identifier: WP_150087159.1 mersacidin/lichenicidin family type 2 lantibiotic [Adhaeribacter rhizoryzae], Sequence: MKNNVDVIRAWKDAAYRATLTEEQLKSLPENPVNDSNALLDNIALRNISGGRPSSGWFCSVSGECNGGYSCNPYYW, Probabilities: 0.6299\n","Identifier: WP_031864692.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MEKVLDLDVQVKGNNNTNDSAGDERITSHLFCSFGCEKTGSSNSFCC, Probabilities: 0.9597\n","Identifier: WP_148959126.1 plantaricin C family lantibiotic [Bacillus swezeyi], Sequence: MSKENIVLSWKDPLSRENNLQNPAGNPFTELEEAQMDKLAGAGDMEAACTWTLPGGGGVCTVTSECIC, Probabilities: 0.9567\n","Identifier: WP_067608779.1 lantibiotic protein [Nocardiopsis listeri], Sequence: MRKSTGLSDLQGELELLESETFEVLDLVEAREEMLGSTTSTTSCCSSCSSSSCSSTSCAGG, Probabilities: 0.5205\n","Identifier: WP_163509955.1 mersacidin/lichenicidin family type 2 lantibiotic [Fodinicola acaciae], Sequence: MSVAEMVRAWKDEDYRLSLPADALPEHPAGVVESELYASTGKSLNTCEHRSSCGRICSA, Probabilities: 0.8706\n","Identifier: WP_148251489.1 gallidermin/nisin family lantibiotic [Staphylococcus aureus], Sequence: MDKVLDLDVQVKGNNNTNDSAGDERITSHLFCSFGCEKTGSFNSFCC, Probabilities: 0.9811\n","Identifier: MBB1572298.1 plantaricin C family lantibiotic [Propionibacterium sp.], Sequence: MQDQNGFQYVTWMQEVAEQEFDTDVFGACTTNTFSLSDYWGNRGHLCSITSECMSWCR, Probabilities: 0.9675\n","Identifier: HGZ87768.1 mersacidin/lichenicidin family type 2 lantibiotic [Gloeobacter violaceus SpSt-379], Sequence: MSLTPEQIVRAWKDEEYRNSLSPEERAQLPAAPAETDSLSDSDLEEIAGGGSCAALSCYAAER, Probabilities: 0.9810\n","Identifier: HGZ85506.1 mersacidin/lichenicidin family type 2 lantibiotic [Gloeobacter violaceus SpSt-379], Sequence: MTPEQIVRAWKDEEYRNSLSPEELAQLPATPAEADSLSDSDLEEIAGGGSCTVLSCYSAVSEV, Probabilities: 0.9736\n","Identifier: HGZ87767.1 mersacidin/lichenicidin family type 2 lantibiotic [Gloeobacter violaceus SpSt-379], Sequence: MSLTPEQIVRAWKDEEYRNSLSPGELAQLSATSAEVEY, Probabilities: 0.7958\n","Identifier: NUR58374.1 mersacidin/lichenicidin family type 2 lantibiotic [Catenulispora sp.], Sequence: MYDIIRAWKDEDYREEVHADPGTHPAGTIDLAAAQPASSMPLLSQSTSLDAAQSACDISHCADICSATTCFDAVGSISWDDPVLTGV, Probabilities: 0.9638\n","Identifier: NVJ03721.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus sp. AM001], Sequence: MSHSDKNHLLRAWRDADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEATCPATPTPNELTCLYSNCGLIPCCR, Probabilities: 0.9817\n","Identifier: NSQ70741.1 plantaricin C family lantibiotic [Enterococcus faecalis], Sequence: MSKKVENWKKAIYSKSENLDNPAGDIFRELSDSELDGIMAGGTANTYCHCYSGKHSCGRACTVTTECPIETIICC, Probabilities: 0.6861\n","Identifier: NRD96455.1 type 2 lantibiotic [Enterococcus faecalis], Sequence: MENLSVVPSFEELSVEEMEAIQGSGD, Probabilities: 0.9806\n","Identifier: NNB84144.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus exiguus], Sequence: MTPDMIIRAWKDPSFRESLTAEQRTQLPANPSGATLNELSETELMDVVGGRIDVGFTATNSCEYICTNLTFSLC, Probabilities: 0.6541\n","Identifier: NQJ60634.1 plantaricin C family lantibiotic [Streptococcus suis], Sequence: MSDWKKPIYESLSTEGNPAGDILRELSESEMDQAMAGGTARSYCGCYSGEHSCGRGCTITTECPFLTFICC, Probabilities: 0.8416\n","Identifier: NPC70299.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus exiguus], Sequence: MTPDMIIRAWKDPSFRESLTAEQRTQLPANPSGATLNELSETELMDVVGGRIDVGFTATNSCEYICTNLTFSMC, Probabilities: 0.7576\n","Identifier: NQK80981.1 lantibiotic salivaricin M precursor [Streptococcus suis], Sequence: MDTLDFEISTQELEGTSGAGGWYSAFKMTLAGRCDRCFTCSYECTSNNVNC, Probabilities: 0.6129\n","Identifier: NOJ78701.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus], Sequence: MAVPQQPKRSSTMSHSDKNHILRAWRDADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEATCPATPTPNELTCLYTNCGRVACCR, Probabilities: 0.8803\n","Identifier: NOJ94756.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus coralloides], Sequence: MTPDMIIRAWKDPSFRESLTAEQRAQLPANPSGTTVNEMSETELMDVVGGKIDIGFTATNSCEYICTNLTFVC, Probabilities: 0.6561\n","Identifier: ABF87463.1 hypothetical protein [Myxococcus xanthus DK 1622 DK 1622], Sequence: MAVPQQPKRSSTMSHSDKNHILRAWRNADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEATCPATPTPNELTCLYTNCGRIACCR, Probabilities: 0.8819\n","Identifier: NJK52943.1 mersacidin/lichenicidin family type 2 lantibiotic [Leptolyngbyaceae cyanobacterium SU_3_3], Sequence: MPNENIIRAWKDTSFRDGLSDVERSLLPENPAGLMQLRDDQLNAAAGGAKPLTREQCSEDFCPGPGPGF, Probabilities: 0.9352\n","Identifier: HEC78528.1 mersacidin/lichenicidin family type 2 lantibiotic [candidate division WOR-3 bacterium], Sequence: MSTEDIIRAWKDPEYRDKLYSKQQMDIPQNPAGTITEITDTALDSLANRTASLCTCIRSAASCLCCR, Probabilities: 0.8724\n","Identifier: NET01229.1 mersacidin/lichenicidin family type 2 lantibiotic [Sphaerospermopsis sp. SIO1G1], Sequence: MSNVDIIRAWKDEEYRQSLSAEQQEQLPVNPAGLVELDDEDMSSVSGGCTTCGDPFHTLLPTCKPIGIETIGIDDFSVG, Probabilities: 0.9526\n","Identifier: NEE61497.1 lantibiotic biosynthesis protein [Streptomyces sp. SID8455], Sequence: TEQAVDAFLAERPALYEVDSGFLNDFYNTLFEIEFPGSERGHYMDEQGRMNLRPNNSRHREAYEPEFGKYGGPAGTELAEWHFRH, Probabilities: 0.9468\n","Identifier: AQP95668.1 hypothetical protein [Bacillus sp. 275], Sequence: MKMSIIETWKNPALRMNSQVVNPAGDLMEELSDDEMEMLAGGCAWYDISCKLGNKGAWCTLTVECQSSCN, Probabilities: 0.5329\n","Identifier: MTK09987.1 gallidermin/nisin family lantibiotic [Hungatella sp.], Sequence: MNKDFDLDLKMDKGSIDVGESKIASVAYCTPGCLTGSTCGTSECGLTRNCTNTWLCY, Probabilities: 0.9755\n","Identifier: KAB2364261.1 plantaricin C family lantibiotic [Bacillus thuringiensis], Sequence: MLGGVNGEMFKFKKENISELAKESYLNQLDKNNTRLENSFGADCNFWNISCHLGNDGSVCTLTKECRPACW, Probabilities: 0.9753\n","Identifier: KAA9180841.1 plantaricin C family lantibiotic [Enterococcus durans], Sequence: MNKRDAFETYQEKGNLKEMSPMMEEEIMDIETGIQAGCTTNTFSLSDYLGNTGGWCTATKECMPSCN, Probabilities: 0.9307\n","Identifier: QDP90305.1 plantaricin C family lantibiotic [Bacillus amyloliquefaciens], Sequence: MLNKGVNTMSQEAIIRSWKDPFSRENSTQNPAGNPFSELKEAQMDKLVGAGDMEAACTFTLPGGGGVCTLTSECIC, Probabilities: 0.9394\n","Identifier: QDE92925.1 mersacidin/lichenicidin family type 2 lantibiotic [Myxococcus xanthus], Sequence: MAVPQQPKRSSTMSHSDKNHILRAWRDADYYDSLSDAERAALPASPASVMELEDATLAFITGGDAEATCPATPTPNELTCLYTNCGRIACCR, Probabilities: 0.8586\n","Identifier: TMD58138.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Chloroflexi bacterium], Sequence: MKIDIVRAWKDEAYRQSLSEEERLTLPENPVGEIELTDADLEAVYGGQPSGILACSAVCNSAALCNSGICHSGVCHVTV, Probabilities: 0.9316\n","Identifier: TMC23698.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium], Sequence: MKFDVVRAWKDALYRHSLSCEELATLPACPAGDFELTDADLQAIQGGWRQEQEQECCEREERETNMTGVGFGICLSFVGNCLNNPTMTTGG, Probabilities: 0.9450\n","Identifier: TMC23699.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium], Sequence: MKLDVIRAWKDALYRHSLSSEELATLPACPAGDFELSDADLQAVQGSWGLEHECCEREECERTNMTGVGALGCISFGGSCGNGVSAITVTPA, Probabilities: 0.8710\n","Identifier: TMC23711.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium], Sequence: MMTFDVVRVWKDAMYRHSLSCEEQATLSACPVGGFELTDADLEAVQGGCDQDEGGECSQCSPCERCEHCEHCETNRTRCGRFAFFSFMESCF, Probabilities: 0.5393\n","Identifier: TMC19290.1 mersacidin/lichenicidin family type 2 lantibiotic [Chloroflexi bacterium], Sequence: MSLNVVNTVRAWKDEMYRKRLSSEEQAMLPVCPAGALELTDADLEAVQGGWGNRFSGCRCRSIFGGMGHGGFLFGFF, Probabilities: 0.5944\n","Identifier: VRJ79307.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae], Sequence: MVANPRFFNELTKEKIYQNSTFRNYAKRSLTRATPFGLFSSVGVGSFSKVSYPQQIRENYSKKVSVSGEWISSLCMMLE, Probabilities: 0.9505\n","Identifier: VKK32681.1 lantibiotic biosynthesis protein [Streptococcus pneumoniae], Sequence: MVANPRFFNELTKEKIYQNSTFRNYAKRSLTRATPFGLFSSVGVGSFSKVSYPQQIRENYSKKVSVSGEWISSLCMMLENEDSVLLQLHL, Probabilities: 0.8745\n","Identifier: TGY91466.1 type 2 lantibiotic [Lachnospiraceae bacterium], Sequence: MKPDKEEKKYNREKQIEDGDVIGKKFEDLSEEEMKKIQGSGSDMNPEFTTSISNLVLSKNGYNC, Probabilities: 0.6099\n","Identifier: TCP70223.1 two-component lantibiotic alpha peptide [Baia soyae], Sequence: MGKDVFMKNPVLRKKVSPQFSNPSGEVLEEVKDHSIVGAGCTKWDFLVTSLVCCGASYVLGNKGHLCTGTVECQNNCRD, Probabilities: 0.6310\n","Identifier: TEA45671.1 type 2 lantibiotic [Bacillus sp. BH2], Sequence: MEDRFENVGDAFEDMTLEDMALAQGSGDIQPRTTPGCFFASLGVSVTISAWSC, Probabilities: 0.6316\n","Identifier: RZU51344.1 two-component lantibiotic alpha peptide [Krasilnikovia cinnamomea], Sequence: MARIQGEGMYEGDVALLEEIDEQDFEGVAYGACTTNTFSLSNVLGNNGGWCTLTRECQRSCN, Probabilities: 0.8838\n","Identifier: RPJ84259.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium], Sequence: MKIDIVRAWKDPEYRSSLTPEELASVPANPSGPTPTSDEELRAVTGGLRRSLRNSSTTVYPDIGCCDNSTVGPKTTVYPDIGCCDNSTIGPKLSMY, Probabilities: 0.9536\n","Identifier: ROX80814.1 plantaricin C family lantibiotic [Enterococcus durans], Sequence: MNKRDAFETYQEKGNLKEMSPMMEEEIMDIETEIQAGCTTNTFSLSDYLGNTGGWCTATKECMPSCN, Probabilities: 0.8925\n","Identifier: ROV59923.1 type 2 lantibiotic [Limosilactobacillus reuteri], Sequence: MEEKELEGVIGNSFESMTVEEMTKIQGMGEYQVDSTPGYFMESAAFSALTANITRHAMHHH, Probabilities: 0.9800\n","Identifier: RKI73647.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. AB049A], Sequence: MTPDMIIRAWKDPSFRESLTAEQRAQLPANPSGATLNELNETELMDVVGGRIDIGFTATNSCEYICTNLTFSQC, Probabilities: 0.6015\n","Identifier: RKH30201.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA041A], Sequence: MTPDMIIRAWKDPSFRESLTAEQRTQLPANPSGATLNELSEAELMDVVGGRIDVGFTATNSCEYICTNLTFSLC, Probabilities: 0.6630\n","Identifier: RKG71000.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. CA054B], Sequence: MTPDMIIRAWKDPSFRESLTSEQRAQLPANPSGETLNELSETELLDVVGGKIDIGFTATNSCEYICTNLTFVC, Probabilities: 0.7263\n","Identifier: RKH54076.1 mersacidin/lichenicidin family type 2 lantibiotic [Corallococcus sp. AB050B], Sequence: MTPDMIIRAWKDPSFRESLTAEQRAQLPANPSGATLNELNETELMDVVGGRIDIGFTATNSCEYICTNLTFSLC, Probabilities: 0.6811\n","Identifier: RHM68438.1 gallidermin/nisin family lantibiotic [[Ruminococcus] gnavus], Sequence: MNKDFDLNLKADKSTTEGPAPRITSVAYCTPGCITGDLCGSSECCLTRNCTGSILCA, Probabilities: 0.9184\n","Identifier: HAD67421.1 plantaricin C family lantibiotic [Roseburia sp.], Sequence: MKGEIIMSEMRNPITREENFSNPAGNVMMELQEADLNKFVAGAGETRNSGGVLCTSTAECYFGTLLKPCCPKK, Probabilities: 0.9711\n","Identifier: PYT18031.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium], Sequence: MKIDVVRAWKDAAYRSSLSADEVAFLPSNPAGLVDLTDAELKEASGLTAVAQTTCICCTDTNRPRRCCP, Probabilities: 0.5759\n","Identifier: PYT07918.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium], Sequence: MCKFDVVRAWKDEDYRMSLSDAERRMLPTNPAGLIEISDSELGAVAGGEAGGISFFGVCSCFGDCMSELFQVCSGDWMPCTIGPLFCPVFPIE, Probabilities: 0.9730\n","Identifier: PYS96079.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium], Sequence: MQIDVVRAWKDAAYRSSLSADEVAFLPSNPAGLVDLTDAELKEASGLTAVAQTTCICCTDTNRPRRCCP, Probabilities: 0.6206\n","Identifier: PYS12131.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium], Sequence: MAIDITRAWKDPEYRKSLTAEERASLPPNPTGEPPASEEELRKIVGGLGSTITLGGPTRCRTEYPDCCYNSTIGPRLG, Probabilities: 0.7079\n","Identifier: PYR98312.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium], Sequence: MAIDITRAWKDPEYRKSLTAEERASLPPNPTGEPPASEEELRKIVGGLGSTVTLGGPTRCRTEYPDCCYNSTIGPRLG, Probabilities: 0.4901\n","Identifier: PYQ09556.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium], Sequence: MDVVRAWKDPGYRAGLGAGEFALIPPNPAGAVELSDAQLKEATGLSAAPVTTNLACTMYTFNRFRCCP, Probabilities: 0.5625\n","Identifier: PYQ12947.1 mersacidin/lichenicidin family type 2 lantibiotic [Acidobacteria bacterium], Sequence: MDVVRAWKDPSYRACLGPEELALAPPNPAGMVDLSDDELKRASGVNGAVGTTAIDCTEYTFRRFRCCPKP, Probabilities: 0.8696\n","Identifier: PVC86288.1 lantibiotic protein, partial [Streptomyces sp. CS131], Sequence: MTPKTELATLADEILELESETFEISDYSDASEVVLAGSTS, Probabilities: 0.9138\n","Identifier: AVN84861.1 lantibiotic mersacidin, partial [Bacillus subtilis], Sequence: MSQEAIIRSWKDPFSRENSTQNPAGNPFSELKEAQMDKLVGAGDMEAACTFTLPGGGGVCTLTSECI, Probabilities: 0.6376\n","Identifier: PNZ59920.1 gallidermin/nisin family lantibiotic, partial [Staphylococcus aureus], Sequence: TPGCGKTGSFNSFCC, Probabilities: 0.8997\n","Identifier: PDP85838.1 mersacidin/lichenicidin family type 2 lantibiotic [Glycomyces fuscus], Sequence: MSIEKAWKDPDYRDGLTEIELDDLPENPAGATPLSSQSLNSVRGGRDPFEMSVLPYVTCGQPTRSVIMTCAGPA, Probabilities: 0.5107\n","Identifier: PDP85837.1 mersacidin/lichenicidin family type 2 lantibiotic [Glycomyces fuscus], Sequence: MNTERAWKDPDYRDGLSEDETADLPENPAGTSSLDNRRLNEVHGGTVVTTQPVHTLALCPMTLRPALCGSPSGG, Probabilities: 0.9763\n","Identifier: PEF08708.1 type 2 lantibiotic, partial [Bacillus thuringiensis], Sequence: IYGASGVDPRTTPSPLLASFVASYIASAEFRCGKDNKGK, Probabilities: 0.6810\n","Identifier: PES89633.1 mersacidin/lichenicidin family type 2 lantibiotic, partial [Bacillus anthracis], Sequence: VTPEGLSSWLGNNGGYCTLTKECMPSCN, Probabilities: 0.8129\n","Identifier: SBS02255.1 LanC lantibiotic synthetase component C-like 2 (bacterial), partial [Nothobranchius pienaari], Sequence: ALCRFTMSHRRRNTCTEPTSLRSGVWTTGPMAVASPTDRTRYLKGWLGPFTTCQRWKGRKLPAFLPLNFDLLTDCLGVEGSLEVNLPPGILDWM, Probabilities: 0.9667\n","Identifier: OTP39067.1 lantibiotic protein CylL-L, partial [Enterococcus faecalis], Sequence: MENLSVVPSFEELSVEEMEAIQGSGDVQAETT, Probabilities: 0.9813\n","Identifier: OEV30460.1 lantibiotic precursor [Streptomyces nanshensis], Sequence: MTSATLTPTAELAEWEDDFAPLDVKVVVAEHPYGKLMCATGDGCGQTCATNSSACNSFVEDPA, Probabilities: 0.9519\n","Identifier: SCL44176.1 type 2 lantibiotic, mersacidin/lichenicidin family [Micromonospora citrea], Sequence: MNDKTLRAWKDPAFRTTLDDTDIALLPESPVGAVGGLDADFADIFGGDRAFTFHTCFTCQWGSLGCTNTCWIMTAGCTSGSC, Probabilities: 0.4997\n","Identifier: EMZ23840.1 type 2 lantibiotic [Eubacterium plexicaudatum ASF492 ASF492], Sequence: MDNEKKSQLQEESSDMDILGKKFEDLTQEEMKKIQGSGDVSPEDIGAFTTILTSKNPYNCK, Probabilities: 0.6479\n","\n","EL NÚMERO DE SECUENCIAS CON POTENCIAL NEUROPEPTÍDICO ES: 585\n"]}]},{"cell_type":"markdown","source":["### TEST DEL HOSPITAL RAMÓN Y CAJAL (not completed yet...)"],"metadata":{"id":"SoVBEIAQiZWW"}},{"cell_type":"code","source":["import re\n","import torch\n","\n","# Función para leer un archivo FASTA\n","def read_fasta(file_path):\n","    sequences = []\n","    with open(file_path, 'r') as f:\n","        identifier = \"\"\n","        sequence = \"\"\n","        for line in f:\n","            line = line.strip()\n","            if line.startswith(\">\"):  # Es un identificador de secuencia\n","                if sequence:  # Guarda la secuencia anterior si existe\n","                    sequences.append((identifier, sequence))\n","                    sequence = \"\"\n","                identifier = line[1:]  # Guarda el identificador sin el \">\"\n","            else:\n","                sequence += line  # Agrega las líneas de la secuencia\n","        if sequence:  # Guarda la última secuencia\n","            sequences.append((identifier, sequence))\n","    return sequences\n","\n","# Función para hacer predicciones de AMP/non-AMP para múltiples secuencias y guardar probabilidades\n","def predict_amp(sequences, output_file):\n","    i=0\n","    a=0\n","    n=0\n","    total=0\n","    with open(output_file, 'w') as out_f:\n","        out_f.write(\"Identifier\\tSequence\\tPrediction\\tProbability\\n\")  # Encabezados\n","        for identifier, input_seq in sequences:\n","            input_seq_spaced = ' '.join([input_seq[i:i+1] for i in range(0, len(input_seq), 1)])\n","            input_seq_spaced = re.sub(r'[UZOB]', 'X', input_seq_spaced)  # Reemplazar aminoácidos no estándar\n","            input_seq_tok = tokenizer(input_seq_spaced, return_tensors='pt')\n","            #print(input_seq_tok)\n","\n","            output = model(**input_seq_tok)\n","            logits = output[0]\n","           # print(f\"Logits for {identifier}: {logits}\")\n","\n","            # Extraer la probabilidad de clase AMP\n","            #y_prob = torch.sigmoid(logits)[:, 1].detach().numpy()\n","            #y_pred = y_prob > 0.84  # Predicción binaria con umbral de 0.5\n","            #prob = y_prob[0]  # Probabilidad real (sin umbral)\n","\n","            # Aplicar softmax para obtener probabilidades de cada clase\n","            y_prob = torch.softmax(logits, dim=1).detach().numpy()\n","\n","            # Obtener la clase con la probabilidad más alta\n","            y_pred_class = y_prob.argmax(axis=1)[0]\n","            prob_class1, prob_class2, prob_class3 = y_prob[0]  # Probabilidades para las tres clases\n","            # Determinar la etiqueta de la predicción según la clase\n","            if y_pred_class == 0:\n","                input_class = 'non-AMP/OTHERS'  # Cambia estos nombres según tus etiquetas\n","                n=n+1\n","                total=total+1\n","            elif y_pred_class == 1:\n","                input_class = 'AMP'\n","                a=a+1\n","                total=total+1\n","            else:\n","                input_class = 'Neuropeptide'\n","                i=i+1\n","                total=total+1\n","\n","            # Escribir el identificador, secuencia, predicción y probabilidades en el archivo de salida\n","            out_f.write(f\"{identifier}\\t{input_seq}\\t{input_class}\\t{prob_class1:.4f}\\t{prob_class2:.4f}\\t{prob_class3:.4f}\\n\")\n","            print(f\"Processed {identifier}: {input_class} (Probabilities: {prob_class1:.4f}, {prob_class2:.4f}, {prob_class3:.4f})\")\n","    print(f\"El número de secuencias de AMPs es: {a}\")\n","    print(f\"El número de secuencias de non-AMP es: {n}\")\n","    print(f\"El número de secuencias de neuropéptidos es: {i}\")\n","    print(f\"El número total de secuencias es: {total}\")\n","\n","\n","            # Determinar clase (AMP o non-AMP)\n","            #if y_pred:\n","            #    input_class = 'AMP'\n","            #else:\n","            #    input_class = 'non-AMP'\n","\n","            # Escribir el identificador, secuencia, predicción y probabilidad en el archivo de salida\n","           # out_f.write(f\"{identifier}\\t{input_seq}\\t{input_class}\\t{prob:.4f}\\n\")\n","           # print(f\"Processed {identifier}: {input_class} (Probability: {prob:.4f})\")\n","\n","# Ruta del archivo FASTA de entrada y del archivo de salida\n","fasta_file = '/content/drive/MyDrive/data/parsed_combined_peptide.fasta'  # Nombre del archivo FASTA con secuencias\n","output_file = '/content/drive/MyDrive/Datos Finales para Memoria/resultados/RESULTADO_RyC.tsv'  # Archivo de salida con predicciones y probabilidades\n","\n","# Leer las secuencias del archivo FASTA\n","sequences = read_fasta(fasta_file)\n","\n","# Realizar predicciones y escribir resultados en un archivo\n","predict_amp(sequences, output_file)\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":1000},"id":"GzwJh0CPigQG","executionInfo":{"status":"error","timestamp":1734260890009,"user_tz":-60,"elapsed":618586,"user":{"displayName":"Javier Col Dom","userId":"08183675435831495369"}},"outputId":"d09818a5-c4d9-49bb-c31d-c0bae2350beb"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["\u001b[1;30;43mSe han truncado las últimas 5000 líneas del flujo de salida.\u001b[0m\n","Processed MH0424_GL0140559: non-AMP/OTHERS (Probabilities: 0.9834, 0.0092, 0.0074)\n","Processed 890402.BBMN68_1789: non-AMP/OTHERS (Probabilities: 0.9259, 0.0116, 0.0625)\n","Processed MH0345_GL0067062: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0074_GL0044846: non-AMP/OTHERS (Probabilities: 0.9921, 0.0039, 0.0040)\n","Processed MH0086_GL0042171: Neuropeptide (Probabilities: 0.0513, 0.0215, 0.9272)\n","Processed MH0086_GL0019990: Neuropeptide (Probabilities: 0.0066, 0.0088, 0.9846)\n","Processed MH0131_GL0028902: non-AMP/OTHERS (Probabilities: 0.9045, 0.0391, 0.0563)\n","Processed MH0104_GL0084786: non-AMP/OTHERS (Probabilities: 0.9774, 0.0049, 0.0177)\n","Processed N013A_GL0001357: non-AMP/OTHERS (Probabilities: 0.9814, 0.0117, 0.0068)\n","Processed MH0153_GL0063928: non-AMP/OTHERS (Probabilities: 0.9812, 0.0146, 0.0042)\n","Processed MH0184_GL0000442: non-AMP/OTHERS (Probabilities: 0.9914, 0.0043, 0.0042)\n","Processed MH0003_GL0056148: AMP (Probabilities: 0.4076, 0.5164, 0.0760)\n","Processed 764224817-stool1_revised_scaffold7255_2_gene28062: Neuropeptide (Probabilities: 0.0182, 0.0146, 0.9672)\n","Processed MH0008_GL0040485: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0292_GL0022866: Neuropeptide (Probabilities: 0.0063, 0.0087, 0.9850)\n","Processed MH0131_GL0062065: non-AMP/OTHERS (Probabilities: 0.9921, 0.0034, 0.0045)\n","Processed MH0319_GL0146778: Neuropeptide (Probabilities: 0.0082, 0.0092, 0.9826)\n","Processed MH0044_GL0067497: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9849)\n","Processed SZEY-103A_GL0074872: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0011_GL0031928: non-AMP/OTHERS (Probabilities: 0.9747, 0.0109, 0.0144)\n","Processed MH0011_GL0058597: non-AMP/OTHERS (Probabilities: 0.9911, 0.0038, 0.0052)\n","Processed MH0287_GL0029725: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed tr|A0A5C2G6I5|A0A5C2G6I5_HUMAN: non-AMP/OTHERS (Probabilities: 0.9844, 0.0120, 0.0037)\n","Processed tr|A0A5C2G8W2|A0A5C2G8W2_HUMAN: non-AMP/OTHERS (Probabilities: 0.9892, 0.0073, 0.0035)\n","Processed tr|A0A5C2GGX3|A0A5C2GGX3_HUMAN: non-AMP/OTHERS (Probabilities: 0.9686, 0.0279, 0.0035)\n","Processed O2.UC41-0_GL0043774: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed MH0191_GL0057943: Neuropeptide (Probabilities: 0.3863, 0.0265, 0.5873)\n","Processed MH0058_GL0022375: Neuropeptide (Probabilities: 0.0063, 0.0086, 0.9851)\n","Processed MH0191_GL0099022: non-AMP/OTHERS (Probabilities: 0.9906, 0.0057, 0.0037)\n","Processed 763577454-stool2_revised_scaffold34723_1_gene146642: Neuropeptide (Probabilities: 0.0159, 0.0121, 0.9720)\n","Processed MH0003_GL0107715: non-AMP/OTHERS (Probabilities: 0.7084, 0.0615, 0.2301)\n","Processed MH0085_GL0007682: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0188_GL0188237: non-AMP/OTHERS (Probabilities: 0.9924, 0.0038, 0.0037)\n","Processed MH0119_GL0002515: Neuropeptide (Probabilities: 0.0122, 0.0118, 0.9760)\n","Processed NOM005_GL0028923: Neuropeptide (Probabilities: 0.1004, 0.0249, 0.8747)\n","Processed ED50A_GL0050988: Neuropeptide (Probabilities: 0.0068, 0.0087, 0.9845)\n","Processed O2.UC44-0_GL0016421: Neuropeptide (Probabilities: 0.0924, 0.0222, 0.8854)\n","Processed SZEY-10A_GL0049767: Neuropeptide (Probabilities: 0.1579, 0.0257, 0.8164)\n","Processed MH0330_GL0101936: Neuropeptide (Probabilities: 0.0063, 0.0089, 0.9849)\n","Processed MH0188_GL0049750: non-AMP/OTHERS (Probabilities: 0.9861, 0.0045, 0.0094)\n","Processed MH0003_GL0100648: Neuropeptide (Probabilities: 0.0082, 0.0092, 0.9826)\n","Processed MH0142_GL0107840: Neuropeptide (Probabilities: 0.0103, 0.0106, 0.9791)\n","Processed MH0267_GL0220657: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed V1.UC26-4_GL0163542: Neuropeptide (Probabilities: 0.4491, 0.0365, 0.5144)\n","Processed O2.UC22-2_GL0082655: non-AMP/OTHERS (Probabilities: 0.9835, 0.0054, 0.0110)\n","Processed MH0348_GL0117196: non-AMP/OTHERS (Probabilities: 0.9899, 0.0036, 0.0065)\n","Processed NOM010_GL0041204: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed O2.UC17-1_GL0125989: Neuropeptide (Probabilities: 0.0077, 0.0089, 0.9834)\n","Processed V1.CD3-0-PN_GL0110819: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0276_GL0233441: Neuropeptide (Probabilities: 0.0086, 0.0092, 0.9822)\n","Processed MH0252_GL0159578: non-AMP/OTHERS (Probabilities: 0.9915, 0.0035, 0.0049)\n","Processed MH0006_GL0191140: non-AMP/OTHERS (Probabilities: 0.9525, 0.0067, 0.0409)\n","Processed V1.UC51-4_GL0058561: non-AMP/OTHERS (Probabilities: 0.9783, 0.0048, 0.0169)\n","Processed T2D-132A_GL0016535: non-AMP/OTHERS (Probabilities: 0.9845, 0.0043, 0.0112)\n","Processed MH0274_GL0006183: Neuropeptide (Probabilities: 0.0072, 0.0089, 0.9839)\n","Processed MH0330_GL0018044: non-AMP/OTHERS (Probabilities: 0.9688, 0.0228, 0.0084)\n","Processed 1042403.BALAC2494_01051: Neuropeptide (Probabilities: 0.0086, 0.0092, 0.9823)\n","Processed MH0131_GL0045133: non-AMP/OTHERS (Probabilities: 0.9425, 0.0168, 0.0407)\n","Processed MH0131_GL0045133: non-AMP/OTHERS (Probabilities: 0.9498, 0.0133, 0.0368)\n","Processed O2.UC19-1_GL0019088: non-AMP/OTHERS (Probabilities: 0.9305, 0.0187, 0.0508)\n","Processed MH0193_GL0188340: Neuropeptide (Probabilities: 0.0076, 0.0088, 0.9836)\n","Processed MH0002_GL0034060: non-AMP/OTHERS (Probabilities: 0.9747, 0.0072, 0.0181)\n","Processed MH0010_GL0015260: non-AMP/OTHERS (Probabilities: 0.9759, 0.0126, 0.0115)\n","Processed MH0012_GL0018527: non-AMP/OTHERS (Probabilities: 0.9870, 0.0071, 0.0059)\n","Processed DOF004_GL0077564: non-AMP/OTHERS (Probabilities: 0.9918, 0.0049, 0.0033)\n","Processed MH0002_GL0058070: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0092_GL0125010: non-AMP/OTHERS (Probabilities: 0.9920, 0.0033, 0.0047)\n","Processed MH0128_GL0021374: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed 159268001-stool1_revised_scaffold40157_5_gene53870: Neuropeptide (Probabilities: 0.0064, 0.0088, 0.9848)\n","Processed MH0276_GL0233441: non-AMP/OTHERS (Probabilities: 0.6970, 0.0560, 0.2470)\n","Processed MH0192_GL0058513: Neuropeptide (Probabilities: 0.0132, 0.0117, 0.9750)\n","Processed T2D-25A_GL0081229: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed V1.UC53-0_GL0076489: Neuropeptide (Probabilities: 0.0117, 0.0121, 0.9762)\n","Processed MH0006_GL0216213: Neuropeptide (Probabilities: 0.0061, 0.0086, 0.9853)\n","Processed O2.UC17-1_GL0096530: Neuropeptide (Probabilities: 0.0076, 0.0093, 0.9831)\n","Processed O2.UC58-2_GL0049625: Neuropeptide (Probabilities: 0.0065, 0.0085, 0.9850)\n","Processed O2.CD1-0-PT_GL0071365: Neuropeptide (Probabilities: 0.0065, 0.0085, 0.9850)\n","Processed MH0120_GL0089989: Neuropeptide (Probabilities: 0.0070, 0.0087, 0.9843)\n","Processed MH0041_GL0056023: non-AMP/OTHERS (Probabilities: 0.9895, 0.0050, 0.0056)\n","Processed MH0124_GL0038646: Neuropeptide (Probabilities: 0.0440, 0.0161, 0.9400)\n","Processed MH0277_GL0010072: Neuropeptide (Probabilities: 0.1587, 0.0184, 0.8230)\n","Processed MH0131_GL0034718: non-AMP/OTHERS (Probabilities: 0.7974, 0.0202, 0.1824)\n","Processed MH0087_GL0040944: non-AMP/OTHERS (Probabilities: 0.5522, 0.0314, 0.4164)\n","Processed O2.UC47-1_GL0197962: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0039)\n","Processed MH0211_GL0119662: Neuropeptide (Probabilities: 0.0528, 0.0266, 0.9207)\n","Processed BGI-34A_GL0045096: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9851)\n","Processed V1.FI20_GL0218752: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0035)\n","Processed 765074482-stool1_revised_C1280564_1_gene159645: Neuropeptide (Probabilities: 0.0077, 0.0089, 0.9834)\n","Processed T2D-25A_GL0014035: Neuropeptide (Probabilities: 0.0105, 0.0098, 0.9797)\n","Processed MH0325_GL0083561: Neuropeptide (Probabilities: 0.0141, 0.0112, 0.9746)\n","Processed MH0341_GL0087567: Neuropeptide (Probabilities: 0.0103, 0.0101, 0.9796)\n","Processed 457391.HMPREF0105_2160: non-AMP/OTHERS (Probabilities: 0.9900, 0.0044, 0.0056)\n","Processed 435590.BVU_1005: non-AMP/OTHERS (Probabilities: 0.7625, 0.0359, 0.2015)\n","Processed MH0011_GL0034332: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed MH0011_GL0034332: Neuropeptide (Probabilities: 0.0090, 0.0088, 0.9822)\n","Processed O2.UC4-1_GL0214797: non-AMP/OTHERS (Probabilities: 0.7110, 0.0274, 0.2616)\n","Processed MH0010_GL0035374: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0006_GL0133657: Neuropeptide (Probabilities: 0.0559, 0.0166, 0.9275)\n","Processed MH0314_GL0005726: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9848)\n","Processed MH0314_GL0056519: Neuropeptide (Probabilities: 0.0061, 0.0091, 0.9848)\n","Processed V1.FI35_GL0107338: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed V1.UC35-4_GL0156025: Neuropeptide (Probabilities: 0.0067, 0.0086, 0.9847)\n","Processed MH0188_GL0017894: non-AMP/OTHERS (Probabilities: 0.9921, 0.0032, 0.0047)\n","Processed V1.FI29_GL0056834: Neuropeptide (Probabilities: 0.0960, 0.0476, 0.8564)\n","Processed V1.FI29_GL0056834: non-AMP/OTHERS (Probabilities: 0.6663, 0.0878, 0.2458)\n","Processed V1.UC42-0_GL0033145: Neuropeptide (Probabilities: 0.0065, 0.0089, 0.9847)\n","Processed V1.UC15-3_GL0139095: Neuropeptide (Probabilities: 0.0066, 0.0088, 0.9846)\n","Processed 764487809-stool1_revised_scaffold28188_2_gene200920: non-AMP/OTHERS (Probabilities: 0.9571, 0.0124, 0.0305)\n","Processed 457391.HMPREF0105_1616: non-AMP/OTHERS (Probabilities: 0.8487, 0.0240, 0.1274)\n","Processed O2.UC22-1_GL0095645: Neuropeptide (Probabilities: 0.0073, 0.0090, 0.9837)\n","Processed MH0440_GL0104404: non-AMP/OTHERS (Probabilities: 0.9869, 0.0071, 0.0060)\n","Processed DOM012_GL0058147: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0006_GL0163943: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9851)\n","Processed MH0274_GL0103868: Neuropeptide (Probabilities: 0.0284, 0.0148, 0.9568)\n","Processed 160502038-stool1_revised_C1104120_1_gene135499: non-AMP/OTHERS (Probabilities: 0.9912, 0.0042, 0.0046)\n","Processed MH0007_GL0033939: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0045_GL0021824: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed SZEY-07A_GL0033149: non-AMP/OTHERS (Probabilities: 0.9737, 0.0098, 0.0165)\n","Processed O2.UC2-2_GL0128667: non-AMP/OTHERS (Probabilities: 0.7611, 0.0400, 0.1990)\n","Processed MH0062_GL0007381: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed MH0060_GL0025377: Neuropeptide (Probabilities: 0.0062, 0.0088, 0.9849)\n","Processed MH0150_GL0062730: non-AMP/OTHERS (Probabilities: 0.9559, 0.0099, 0.0342)\n","Processed MH0122_GL0052895: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0188_GL0176742: non-AMP/OTHERS (Probabilities: 0.9926, 0.0036, 0.0038)\n","Processed DOF008_GL0049544: non-AMP/OTHERS (Probabilities: 0.9841, 0.0084, 0.0075)\n","Processed V1.CD8-0-PN_GL0073924: Neuropeptide (Probabilities: 0.0190, 0.0121, 0.9689)\n","Processed V1.UC48-0_GL0168329: Neuropeptide (Probabilities: 0.0060, 0.0090, 0.9850)\n","Processed MH0326_GL0093213: Neuropeptide (Probabilities: 0.0095, 0.0097, 0.9807)\n","Processed MH0111_GL0071025: Neuropeptide (Probabilities: 0.0152, 0.0123, 0.9725)\n","Processed MH0002_GL0032397: Neuropeptide (Probabilities: 0.0100, 0.0098, 0.9802)\n","Processed 159268001-stool2_revised_scaffold17156_1_gene14159: Neuropeptide (Probabilities: 0.0060, 0.0090, 0.9851)\n","Processed MH0032_GL0029361: non-AMP/OTHERS (Probabilities: 0.7945, 0.1831, 0.0224)\n","Processed MH0006_GL0028898: non-AMP/OTHERS (Probabilities: 0.9894, 0.0041, 0.0065)\n","Processed NLM023_GL0083745: non-AMP/OTHERS (Probabilities: 0.9922, 0.0041, 0.0036)\n","Processed MH0233_GL0114673: Neuropeptide (Probabilities: 0.0277, 0.0157, 0.9566)\n","Processed MH0011_GL0075800: Neuropeptide (Probabilities: 0.1325, 0.0341, 0.8334)\n","Processed MH0131_GL0167935: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0002_GL0000401: non-AMP/OTHERS (Probabilities: 0.9837, 0.0055, 0.0108)\n","Processed MH0062_GL0017613: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9851)\n","Processed O2.UC7-1_GL0174619: non-AMP/OTHERS (Probabilities: 0.9791, 0.0102, 0.0107)\n","Processed MH0252_GL0238240: non-AMP/OTHERS (Probabilities: 0.9858, 0.0035, 0.0107)\n","Processed sp|Q12913|PTPRJ_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed O2.UC48-2_GL0102753: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed tr|A0A5C2GX23|A0A5C2GX23_HUMAN: non-AMP/OTHERS (Probabilities: 0.9891, 0.0065, 0.0045)\n","Processed O2.UC28-1_GL0117204: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0138_GL0003413: non-AMP/OTHERS (Probabilities: 0.9733, 0.0152, 0.0115)\n","Processed MH0058_GL0077304: Neuropeptide (Probabilities: 0.2273, 0.0283, 0.7444)\n","Processed MH0203_GL0160157: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed O2.UC55-0_GL0095793: non-AMP/OTHERS (Probabilities: 0.9885, 0.0078, 0.0037)\n","Processed MH0438_GL0036209: non-AMP/OTHERS (Probabilities: 0.8612, 0.0279, 0.1108)\n","Processed MH0203_GL0160157: non-AMP/OTHERS (Probabilities: 0.9921, 0.0042, 0.0037)\n","Processed MH0150_GL0017002: Neuropeptide (Probabilities: 0.0081, 0.0093, 0.9825)\n","Processed MH0372_GL0035490: non-AMP/OTHERS (Probabilities: 0.9919, 0.0042, 0.0039)\n","Processed MH0023_GL0030355: non-AMP/OTHERS (Probabilities: 0.9875, 0.0049, 0.0075)\n","Processed MH0150_GL0017002: non-AMP/OTHERS (Probabilities: 0.9917, 0.0033, 0.0050)\n","Processed O2.UC40-1_GL0169380: Neuropeptide (Probabilities: 0.0081, 0.0093, 0.9826)\n","Processed O2.UC40-0_GL0127803: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0032)\n","Processed MH0131_GL0045133: non-AMP/OTHERS (Probabilities: 0.9887, 0.0037, 0.0076)\n","Processed MH0238_GL0166123: non-AMP/OTHERS (Probabilities: 0.9746, 0.0054, 0.0200)\n","Processed O2.UC19-1_GL0019088: non-AMP/OTHERS (Probabilities: 0.9802, 0.0048, 0.0149)\n","Processed MH0239_GL0088549: Neuropeptide (Probabilities: 0.0088, 0.0096, 0.9816)\n","Processed 324831.LGAS_0778: Neuropeptide (Probabilities: 0.0066, 0.0090, 0.9845)\n","Processed MH0124_GL0047168: non-AMP/OTHERS (Probabilities: 0.9868, 0.0099, 0.0033)\n","Processed MH0417_GL0100560: Neuropeptide (Probabilities: 0.0408, 0.0326, 0.9266)\n","Processed MH0131_GL0044733: Neuropeptide (Probabilities: 0.0086, 0.0105, 0.9810)\n","Processed MH0062_GL0009063: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed MH0086_GL0050799: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0239_GL0131911: Neuropeptide (Probabilities: 0.0058, 0.0088, 0.9853)\n","Processed MH0144_GL0088442: Neuropeptide (Probabilities: 0.2885, 0.0677, 0.6438)\n","Processed V1.FI25_GL0072091: Neuropeptide (Probabilities: 0.0061, 0.0086, 0.9853)\n","Processed O2.UC17-1_GL0099902: Neuropeptide (Probabilities: 0.0096, 0.0104, 0.9800)\n","Processed O2.UC17-1_GL0099902: Neuropeptide (Probabilities: 0.0091, 0.0098, 0.9811)\n","Processed V1.CD52-0_GL0095458: Neuropeptide (Probabilities: 0.0306, 0.0154, 0.9539)\n","Processed MH0169_GL0061538: Neuropeptide (Probabilities: 0.0063, 0.0093, 0.9844)\n","Processed HT14A_GL0028770: Neuropeptide (Probabilities: 0.1239, 0.0605, 0.8155)\n","Processed MH0192_GL0067831: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 765074482-stool2_revised_C1084384_1_gene137130: non-AMP/OTHERS (Probabilities: 0.9906, 0.0032, 0.0062)\n","Processed tr|A0A5C2G6C5|A0A5C2G6C5_HUMAN: non-AMP/OTHERS (Probabilities: 0.9889, 0.0083, 0.0029)\n","Processed sp|A0A075B6R9|KVD24_HUMAN: non-AMP/OTHERS (Probabilities: 0.9022, 0.0396, 0.0583)\n","Processed tr|A0A5C2FYQ1|A0A5C2FYQ1_HUMAN: non-AMP/OTHERS (Probabilities: 0.9686, 0.0147, 0.0167)\n","Processed tr|A0A5C2FUC2|A0A5C2FUC2_HUMAN: non-AMP/OTHERS (Probabilities: 0.9666, 0.0305, 0.0029)\n","Processed tr|A0A5C2G420|A0A5C2G420_HUMAN: Neuropeptide (Probabilities: 0.0068, 0.0087, 0.9845)\n","Processed tr|A0A5C2G420|A0A5C2G420_HUMAN: non-AMP/OTHERS (Probabilities: 0.9626, 0.0137, 0.0237)\n","Processed tr|A0A5C2FYV9|A0A5C2FYV9_HUMAN: non-AMP/OTHERS (Probabilities: 0.8953, 0.0389, 0.0657)\n","Processed tr|A0A5C2GN85|A0A5C2GN85_HUMAN: non-AMP/OTHERS (Probabilities: 0.9890, 0.0062, 0.0047)\n","Processed tr|Q0KKI6|Q0KKI6_HUMAN: non-AMP/OTHERS (Probabilities: 0.9634, 0.0131, 0.0236)\n","Processed tr|A0A5C2GBP7|A0A5C2GBP7_HUMAN: non-AMP/OTHERS (Probabilities: 0.7771, 0.0335, 0.1894)\n","Processed tr|A0A5C2FWZ4|A0A5C2FWZ4_HUMAN: non-AMP/OTHERS (Probabilities: 0.9896, 0.0066, 0.0038)\n","Processed tr|A0A5C2FZH8|A0A5C2FZH8_HUMAN: non-AMP/OTHERS (Probabilities: 0.9234, 0.0261, 0.0505)\n","Processed tr|A0A5C2FU34|A0A5C2FU34_HUMAN: non-AMP/OTHERS (Probabilities: 0.9913, 0.0050, 0.0037)\n","Processed tr|A0A5C2FZN7|A0A5C2FZN7_HUMAN: non-AMP/OTHERS (Probabilities: 0.9816, 0.0117, 0.0067)\n","Processed tr|A0A5C2H2C4|A0A5C2H2C4_HUMAN: non-AMP/OTHERS (Probabilities: 0.5640, 0.0586, 0.3774)\n","Processed tr|A0A5C2G1F7|A0A5C2G1F7_HUMAN: non-AMP/OTHERS (Probabilities: 0.9915, 0.0053, 0.0032)\n","Processed tr|A0A5C2G086|A0A5C2G086_HUMAN: non-AMP/OTHERS (Probabilities: 0.9900, 0.0073, 0.0027)\n","Processed tr|A0A5C2FVM1|A0A5C2FVM1_HUMAN: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0032)\n","Processed tr|A0A5C2G6S6|A0A5C2G6S6_HUMAN: non-AMP/OTHERS (Probabilities: 0.9899, 0.0062, 0.0039)\n","Processed tr|A0A5C2FU65|A0A5C2FU65_HUMAN: non-AMP/OTHERS (Probabilities: 0.9859, 0.0063, 0.0077)\n","Processed tr|A0A5C2G252|A0A5C2G252_HUMAN: non-AMP/OTHERS (Probabilities: 0.9329, 0.0278, 0.0393)\n","Processed tr|A0A5C2GAT8|A0A5C2GAT8_HUMAN: non-AMP/OTHERS (Probabilities: 0.9147, 0.0327, 0.0526)\n","Processed tr|A0A5C2G8C2|A0A5C2G8C2_HUMAN: non-AMP/OTHERS (Probabilities: 0.9898, 0.0062, 0.0040)\n","Processed tr|A0A5C2GIQ1|A0A5C2GIQ1_HUMAN: non-AMP/OTHERS (Probabilities: 0.9863, 0.0111, 0.0026)\n","Processed tr|A0A5C2GKU3|A0A5C2GKU3_HUMAN: non-AMP/OTHERS (Probabilities: 0.9510, 0.0447, 0.0043)\n","Processed tr|A0A5C2GK57|A0A5C2GK57_HUMAN: non-AMP/OTHERS (Probabilities: 0.9885, 0.0088, 0.0028)\n","Processed tr|A0A5C2H1L1|A0A5C2H1L1_HUMAN: non-AMP/OTHERS (Probabilities: 0.9920, 0.0054, 0.0026)\n","Processed tr|A0A5C2FVX9|A0A5C2FVX9_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed tr|S6C4R7|S6C4R7_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed tr|A0A5C2G5Q9|A0A5C2G5Q9_HUMAN: non-AMP/OTHERS (Probabilities: 0.9889, 0.0084, 0.0027)\n","Processed tr|A0A5C2G276|A0A5C2G276_HUMAN: non-AMP/OTHERS (Probabilities: 0.9828, 0.0142, 0.0030)\n","Processed tr|A0A5C2G1G5|A0A5C2G1G5_HUMAN: non-AMP/OTHERS (Probabilities: 0.9797, 0.0169, 0.0034)\n","Processed tr|A0A7T0LN96|A0A7T0LN96_HUMAN: non-AMP/OTHERS (Probabilities: 0.9819, 0.0153, 0.0029)\n","Processed tr|A0N5G3|A0N5G3_HUMAN: non-AMP/OTHERS (Probabilities: 0.9231, 0.0723, 0.0046)\n","Processed tr|A0A5C2GB75|A0A5C2GB75_HUMAN: Neuropeptide (Probabilities: 0.0170, 0.0120, 0.9709)\n","Processed tr|A0A5C2G7V5|A0A5C2G7V5_HUMAN: non-AMP/OTHERS (Probabilities: 0.9917, 0.0045, 0.0038)\n","Processed V1.UC43-0_GL0006015: non-AMP/OTHERS (Probabilities: 0.9929, 0.0038, 0.0033)\n","Processed 159632143-stool1_revised_scaffold46407_2_gene23189: Neuropeptide (Probabilities: 0.0077, 0.0094, 0.9829)\n","Processed 1042403.BALAC2494_01051: non-AMP/OTHERS (Probabilities: 0.9869, 0.0035, 0.0095)\n","Processed sp|A0A075B6R9|KVD24_HUMAN: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9851)\n","Processed sp|A0A075B6R9|KVD24_HUMAN: non-AMP/OTHERS (Probabilities: 0.9923, 0.0042, 0.0035)\n","Processed sp|A0A075B6S2|KVD29_HUMAN: non-AMP/OTHERS (Probabilities: 0.9904, 0.0061, 0.0035)\n","Processed MH0067_GL0026670: non-AMP/OTHERS (Probabilities: 0.5716, 0.1346, 0.2937)\n","Processed MH0093_GL0008960: Neuropeptide (Probabilities: 0.0094, 0.0097, 0.9809)\n","Processed MH0003_GL0043606: Neuropeptide (Probabilities: 0.0064, 0.0087, 0.9849)\n","Processed 159247771-stool1_revised_scaffold718_3_gene72089: Neuropeptide (Probabilities: 0.0097, 0.0103, 0.9800)\n","Processed MH0406_GL0196004: Neuropeptide (Probabilities: 0.1114, 0.0331, 0.8554)\n","Processed MH0187_GL0108143: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed 158337416-stool1_revised_C1274042_1_gene15182: non-AMP/OTHERS (Probabilities: 0.9932, 0.0037, 0.0031)\n","Processed MH0277_GL0048949: non-AMP/OTHERS (Probabilities: 0.9933, 0.0036, 0.0032)\n","Processed MH0345_GL0154635: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0034)\n","Processed 658659.HMPREF0983_02260: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed 763860675-stool1_revised_scaffold57833_1_gene103553: non-AMP/OTHERS (Probabilities: 0.9877, 0.0051, 0.0072)\n","Processed 763860675-stool1_revised_scaffold57833_1_gene103553: non-AMP/OTHERS (Probabilities: 0.9895, 0.0046, 0.0060)\n","Processed O2.UC30-0_GL0042208: Neuropeptide (Probabilities: 0.0062, 0.0088, 0.9850)\n","Processed MH0272_GL0046008: non-AMP/OTHERS (Probabilities: 0.9889, 0.0038, 0.0072)\n","Processed MH0011_GL0033287: non-AMP/OTHERS (Probabilities: 0.9934, 0.0033, 0.0032)\n","Processed sp|O60844|ZG16_HUMAN: non-AMP/OTHERS (Probabilities: 0.9461, 0.0207, 0.0332)\n","Processed MH0002_GL0010048: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed V1.UC13-3_GL0052191: non-AMP/OTHERS (Probabilities: 0.9917, 0.0035, 0.0048)\n","Processed MH0006_GL0221704: non-AMP/OTHERS (Probabilities: 0.9925, 0.0033, 0.0042)\n","Processed MH0277_GL0054431: Neuropeptide (Probabilities: 0.1639, 0.0242, 0.8119)\n","Processed sp|P05154|IPSP_HUMAN: non-AMP/OTHERS (Probabilities: 0.9685, 0.0087, 0.0228)\n","Processed O2.UC46-2_GL0083035: Neuropeptide (Probabilities: 0.1099, 0.0298, 0.8603)\n","Processed 764062976-stool1_revised_scaffold5388_1_gene105628: non-AMP/OTHERS (Probabilities: 0.9908, 0.0062, 0.0031)\n","Processed MH0089_GL0021917: non-AMP/OTHERS (Probabilities: 0.9815, 0.0055, 0.0130)\n","Processed MH0314_GL0034617: non-AMP/OTHERS (Probabilities: 0.9732, 0.0130, 0.0138)\n","Processed HT14A_GL0087897: Neuropeptide (Probabilities: 0.0072, 0.0087, 0.9841)\n","Processed sp|P17538|CTRB1_HUMAN: non-AMP/OTHERS (Probabilities: 0.9916, 0.0033, 0.0051)\n","Processed sp|P17538|CTRB1_HUMAN: non-AMP/OTHERS (Probabilities: 0.9916, 0.0033, 0.0051)\n","Processed MH0097_GL0078063: non-AMP/OTHERS (Probabilities: 0.9859, 0.0091, 0.0050)\n","Processed O2.UC11-2_GL0086546: non-AMP/OTHERS (Probabilities: 0.9823, 0.0102, 0.0075)\n","Processed O2.UC53-0_GL0201464: non-AMP/OTHERS (Probabilities: 0.9900, 0.0039, 0.0061)\n","Processed sp|P29622|KAIN_HUMAN: Neuropeptide (Probabilities: 0.2756, 0.0903, 0.6341)\n","Processed 717962.CC1_19330: non-AMP/OTHERS (Probabilities: 0.7444, 0.0789, 0.1767)\n","Processed MH0349_GL0096701: non-AMP/OTHERS (Probabilities: 0.7731, 0.0506, 0.1763)\n","Processed O2.UC30-1_GL0015147: non-AMP/OTHERS (Probabilities: 0.9919, 0.0041, 0.0040)\n","Processed O2.UC41-1_GL0091400: non-AMP/OTHERS (Probabilities: 0.9870, 0.0069, 0.0061)\n","Processed O2.UC17-1_GL0094730: non-AMP/OTHERS (Probabilities: 0.7733, 0.0146, 0.2121)\n","Processed MH0131_GL0054159: Neuropeptide (Probabilities: 0.0074, 0.0088, 0.9838)\n","Processed 411485.FAEPRAM212_00741: AMP (Probabilities: 0.1185, 0.8556, 0.0259)\n","Processed MH0383_GL0119529: Neuropeptide (Probabilities: 0.2663, 0.2871, 0.4466)\n","Processed V1.FI22_GL0004518: non-AMP/OTHERS (Probabilities: 0.8510, 0.0261, 0.1230)\n","Processed DLM009_GL0022990: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed MH0086_GL0001131: non-AMP/OTHERS (Probabilities: 0.9574, 0.0062, 0.0364)\n","Processed MH0268_GL0049189: Neuropeptide (Probabilities: 0.0076, 0.0094, 0.9830)\n","Processed MH0022_GL0024475: non-AMP/OTHERS (Probabilities: 0.7667, 0.2105, 0.0229)\n","Processed V1.UC17-2_GL0123387: non-AMP/OTHERS (Probabilities: 0.8998, 0.0121, 0.0881)\n","Processed MH0065_GL0046295: Neuropeptide (Probabilities: 0.0181, 0.0143, 0.9676)\n","Processed MH0014_GL0074141: Neuropeptide (Probabilities: 0.0127, 0.0116, 0.9756)\n","Processed MH0083_GL0060271: non-AMP/OTHERS (Probabilities: 0.9691, 0.0123, 0.0186)\n","Processed MH0277_GL0036860: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0014_GL0037528: non-AMP/OTHERS (Probabilities: 0.9586, 0.0115, 0.0299)\n","Processed sp|O43280|TREA_HUMAN: non-AMP/OTHERS (Probabilities: 0.8146, 0.0481, 0.1373)\n","Processed MH0336_GL0131615: non-AMP/OTHERS (Probabilities: 0.8789, 0.0542, 0.0669)\n","Processed O2.UC17-1_GL0025081: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed T2D-12A_GL0008102: non-AMP/OTHERS (Probabilities: 0.9855, 0.0068, 0.0077)\n","Processed MH0009_GL0049988: non-AMP/OTHERS (Probabilities: 0.9922, 0.0034, 0.0044)\n","Processed MH0161_GL0155640: non-AMP/OTHERS (Probabilities: 0.9916, 0.0040, 0.0044)\n","Processed MH0193_GL0113741: Neuropeptide (Probabilities: 0.0069, 0.0086, 0.9844)\n","Processed MH0002_GL0053267: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9853)\n","Processed 159268001-stool2_revised_scaffold16596_1_gene15032: Neuropeptide (Probabilities: 0.0089, 0.0102, 0.9809)\n","Processed 159369152-stool2_revised_scaffold10192_1_gene1461: AMP (Probabilities: 0.3214, 0.4766, 0.2019)\n","Processed DOM016_GL0090143: non-AMP/OTHERS (Probabilities: 0.7060, 0.2057, 0.0883)\n","Processed DOM016_GL0090143: Neuropeptide (Probabilities: 0.1979, 0.1559, 0.6462)\n","Processed O2.UC20-1_GL0134067: non-AMP/OTHERS (Probabilities: 0.9913, 0.0042, 0.0045)\n","Processed MH0003_GL0073731: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0011_GL0083021: non-AMP/OTHERS (Probabilities: 0.9912, 0.0037, 0.0051)\n","Processed MH0039_GL0027075: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed O2.UC37-0_GL0046038: non-AMP/OTHERS (Probabilities: 0.9674, 0.0096, 0.0230)\n","Processed MH0288_GL0145856: non-AMP/OTHERS (Probabilities: 0.9924, 0.0033, 0.0043)\n","Processed 764588959-stool1_revised_C845271_1_gene89525: non-AMP/OTHERS (Probabilities: 0.9933, 0.0036, 0.0032)\n","Processed MH0003_GL0115607: Neuropeptide (Probabilities: 0.0529, 0.0187, 0.9284)\n","Processed O2.UC31-0_GL0109539: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed V1.CD25-4_GL0098718: Neuropeptide (Probabilities: 0.0065, 0.0088, 0.9848)\n","Processed V1.UC35-4_GL0058948: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed 158883629-stool1_revised_scaffold1422_2_gene30854: Neuropeptide (Probabilities: 0.0957, 0.0494, 0.8549)\n","Processed V1.UC54-0_GL0187989: non-AMP/OTHERS (Probabilities: 0.9839, 0.0121, 0.0040)\n","Processed MH0277_GL0026044: non-AMP/OTHERS (Probabilities: 0.9918, 0.0055, 0.0027)\n","Processed MH0272_GL0009792: non-AMP/OTHERS (Probabilities: 0.9914, 0.0048, 0.0038)\n","Processed MH0138_GL0028293: non-AMP/OTHERS (Probabilities: 0.9748, 0.0120, 0.0132)\n","Processed MH0141_GL0087530: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed T2D-26A_GL0015226: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed V1.FI04_GL0112320: non-AMP/OTHERS (Probabilities: 0.9924, 0.0045, 0.0031)\n","Processed MH0341_GL0049323: non-AMP/OTHERS (Probabilities: 0.9932, 0.0039, 0.0029)\n","Processed MH0443_GL0134565: Neuropeptide (Probabilities: 0.0064, 0.0088, 0.9848)\n","Processed V1.UC58-0_GL0003356: Neuropeptide (Probabilities: 0.0071, 0.0088, 0.9841)\n","Processed V1.CD3-0-PN_GL0128525: Neuropeptide (Probabilities: 0.1889, 0.0512, 0.7599)\n","Processed MH0088_GL0018296: non-AMP/OTHERS (Probabilities: 0.6219, 0.0684, 0.3097)\n","Processed MH0196_GL0117151: non-AMP/OTHERS (Probabilities: 0.9857, 0.0066, 0.0076)\n","Processed MH0038_GL0013094: Neuropeptide (Probabilities: 0.0066, 0.0086, 0.9848)\n","Processed MH0006_GL0174419: Neuropeptide (Probabilities: 0.0274, 0.0248, 0.9478)\n","Processed MH0203_GL0248186: Neuropeptide (Probabilities: 0.0095, 0.0101, 0.9804)\n","Processed MH0092_GL0036248: non-AMP/OTHERS (Probabilities: 0.9782, 0.0060, 0.0158)\n","Processed MH0192_GL0075120: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0034)\n","Processed MH0245_GL0134619: Neuropeptide (Probabilities: 0.0672, 0.0196, 0.9133)\n","Processed V1.UC38-4_GL0108753: non-AMP/OTHERS (Probabilities: 0.9901, 0.0038, 0.0061)\n","Processed MH0104_GL0054429: non-AMP/OTHERS (Probabilities: 0.9917, 0.0041, 0.0042)\n","Processed MH0104_GL0054429: non-AMP/OTHERS (Probabilities: 0.9913, 0.0037, 0.0050)\n","Processed O2.UC47-1_GL0088469: non-AMP/OTHERS (Probabilities: 0.9839, 0.0047, 0.0114)\n","Processed MH0006_GL0006783: Neuropeptide (Probabilities: 0.0216, 0.0205, 0.9579)\n","Processed O2.UC28-2_GL0179590: Neuropeptide (Probabilities: 0.0322, 0.0148, 0.9530)\n","Processed MH0101_GL0075310: non-AMP/OTHERS (Probabilities: 0.9885, 0.0058, 0.0056)\n","Processed MH0319_GL0148976: non-AMP/OTHERS (Probabilities: 0.9927, 0.0032, 0.0041)\n","Processed MH0250_GL0244988: non-AMP/OTHERS (Probabilities: 0.9826, 0.0045, 0.0129)\n","Processed V1.FI32_GL0077256: Neuropeptide (Probabilities: 0.0092, 0.0096, 0.9812)\n","Processed MH0004_GL0002393: Neuropeptide (Probabilities: 0.0088, 0.0096, 0.9816)\n","Processed MH0227_GL0139866: Neuropeptide (Probabilities: 0.0090, 0.0096, 0.9814)\n","Processed MH0199_GL0015980: Neuropeptide (Probabilities: 0.0063, 0.0086, 0.9851)\n","Processed MH0101_GL0059430: Neuropeptide (Probabilities: 0.0158, 0.0171, 0.9671)\n","Processed MH0011_GL0070329: non-AMP/OTHERS (Probabilities: 0.5220, 0.0386, 0.4394)\n","Processed MH0131_GL0107430: Neuropeptide (Probabilities: 0.0096, 0.0106, 0.9798)\n","Processed V1.CD46-0_GL0086187: non-AMP/OTHERS (Probabilities: 0.9703, 0.0083, 0.0215)\n","Processed V1.CD46-0_GL0086187: non-AMP/OTHERS (Probabilities: 0.9874, 0.0044, 0.0082)\n","Processed 763901136-stool2_revised_scaffold40528_1_gene22478: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0034)\n","Processed T2D-26A_GL0068903: non-AMP/OTHERS (Probabilities: 0.9885, 0.0053, 0.0062)\n","Processed N082A_GL0072780: non-AMP/OTHERS (Probabilities: 0.9909, 0.0039, 0.0052)\n","Processed N082A_GL0072780: non-AMP/OTHERS (Probabilities: 0.9917, 0.0040, 0.0044)\n","Processed MH0270_GL0181700: non-AMP/OTHERS (Probabilities: 0.9278, 0.0556, 0.0166)\n","Processed MH0003_GL0081497: non-AMP/OTHERS (Probabilities: 0.9904, 0.0035, 0.0061)\n","Processed MH0230_GL0150630: AMP (Probabilities: 0.2824, 0.6708, 0.0468)\n","Processed O2.UC18-1_GL0194814: non-AMP/OTHERS (Probabilities: 0.9754, 0.0111, 0.0135)\n","Processed O2.UC48-0_GL0120234: non-AMP/OTHERS (Probabilities: 0.9899, 0.0071, 0.0030)\n","Processed sp|Q8WZ42|TITIN_HUMAN: Neuropeptide (Probabilities: 0.0062, 0.0088, 0.9850)\n","Processed sp|Q8WZ42|TITIN_HUMAN: non-AMP/OTHERS (Probabilities: 0.9921, 0.0054, 0.0025)\n","Processed MH0188_GL0147305: Neuropeptide (Probabilities: 0.0134, 0.0109, 0.9757)\n","Processed O2.UC52-2_GL0007300: non-AMP/OTHERS (Probabilities: 0.9887, 0.0060, 0.0053)\n","Processed V1.UC2-0_GL0118449: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9851)\n","Processed O2.UC48-0_GL0283946: non-AMP/OTHERS (Probabilities: 0.9905, 0.0062, 0.0033)\n","Processed MH0239_GL0114637: Neuropeptide (Probabilities: 0.2043, 0.0380, 0.7577)\n","Processed O2.UC50-2_GL0144333: non-AMP/OTHERS (Probabilities: 0.9718, 0.0247, 0.0034)\n","Processed MH0388_GL0140018: Neuropeptide (Probabilities: 0.0061, 0.0089, 0.9851)\n","Processed MH0388_GL0140018: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0103_GL0042412: non-AMP/OTHERS (Probabilities: 0.9914, 0.0037, 0.0049)\n","Processed MH0260_GL0038426: Neuropeptide (Probabilities: 0.0623, 0.0240, 0.9137)\n","Processed MH0346_GL0079802: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0014_GL0003070: Neuropeptide (Probabilities: 0.0062, 0.0087, 0.9852)\n","Processed MH0274_GL0016819: non-AMP/OTHERS (Probabilities: 0.9136, 0.0246, 0.0618)\n","Processed MH0011_GL0000337: Neuropeptide (Probabilities: 0.1012, 0.0270, 0.8718)\n","Processed MH0003_GL0054998: non-AMP/OTHERS (Probabilities: 0.9288, 0.0180, 0.0532)\n","Processed 326426.Bbr_1738: non-AMP/OTHERS (Probabilities: 0.9926, 0.0034, 0.0039)\n","Processed 1042403.BALAC2494_00828: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0036)\n","Processed O2.UC44-2_GL0054699: non-AMP/OTHERS (Probabilities: 0.9896, 0.0048, 0.0056)\n","Processed DLM001_GL0071826: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed MH0050_GL0029332: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed 1042403.BALAC2494_00156: Neuropeptide (Probabilities: 0.0069, 0.0087, 0.9844)\n","Processed MH0020_GL0012550: Neuropeptide (Probabilities: 0.0068, 0.0086, 0.9846)\n","Processed O2.UC32-0_GL0015967: non-AMP/OTHERS (Probabilities: 0.8850, 0.0721, 0.0429)\n","Processed MH0220_GL0194209: non-AMP/OTHERS (Probabilities: 0.9592, 0.0335, 0.0073)\n","Processed MH0221_GL0073013: non-AMP/OTHERS (Probabilities: 0.9218, 0.0278, 0.0504)\n","Processed V1.CD55-0_GL0034218: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9852)\n","Processed V1.CD55-0_GL0034218: Neuropeptide (Probabilities: 0.0929, 0.0317, 0.8754)\n","Processed MH0406_GL0049046: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed MH0088_GL0083345: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0211_GL0171952: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0062_GL0069108: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed 518635.BIFANG_02039: Neuropeptide (Probabilities: 0.0357, 0.0189, 0.9454)\n","Processed MH0062_GL0041947: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed MH0058_GL0045778: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed 158337416-stool1_revised_scaffold30271_2_gene67334: non-AMP/OTHERS (Probabilities: 0.9926, 0.0040, 0.0034)\n","Processed MH0188_GL0025030: Neuropeptide (Probabilities: 0.0131, 0.0122, 0.9748)\n","Processed MH0012_GL0011758: Neuropeptide (Probabilities: 0.0069, 0.0086, 0.9845)\n","Processed MH0346_GL0027857: non-AMP/OTHERS (Probabilities: 0.9900, 0.0059, 0.0041)\n","Processed O2.UC35-1_GL0090557: Neuropeptide (Probabilities: 0.0184, 0.0130, 0.9686)\n","Processed MH0272_GL0009333: Neuropeptide (Probabilities: 0.4405, 0.0206, 0.5389)\n","Processed MH0147_GL0006920: Neuropeptide (Probabilities: 0.0062, 0.0088, 0.9850)\n","Processed MH0111_GL0037407: non-AMP/OTHERS (Probabilities: 0.9295, 0.0402, 0.0303)\n","Processed MH0006_GL0111266: non-AMP/OTHERS (Probabilities: 0.9892, 0.0033, 0.0075)\n","Processed MH0006_GL0111266: Neuropeptide (Probabilities: 0.1393, 0.0269, 0.8338)\n","Processed T2D-187A_GL0086078: non-AMP/OTHERS (Probabilities: 0.8864, 0.0120, 0.1017)\n","Processed V1.UC39-4_GL0076825: non-AMP/OTHERS (Probabilities: 0.9920, 0.0034, 0.0046)\n","Processed O2.UC47-1_GL0181965: Neuropeptide (Probabilities: 0.0184, 0.0141, 0.9675)\n","Processed MH0229_GL0008693: Neuropeptide (Probabilities: 0.0061, 0.0086, 0.9852)\n","Processed 765560005-stool1_revised_scaffold22340_1_gene17944: Neuropeptide (Probabilities: 0.0068, 0.0087, 0.9844)\n","Processed MH0010_GL0025233: Neuropeptide (Probabilities: 0.0881, 0.0207, 0.8912)\n","Processed MH0233_GL0066057: Neuropeptide (Probabilities: 0.0065, 0.0089, 0.9846)\n","Processed MH0002_GL0044056: non-AMP/OTHERS (Probabilities: 0.4850, 0.0612, 0.4537)\n","Processed sp|P20929|NEBU_HUMAN: non-AMP/OTHERS (Probabilities: 0.9573, 0.0284, 0.0143)\n","Processed sp|P35527|K1C9_HUMAN: non-AMP/OTHERS (Probabilities: 0.5101, 0.2895, 0.2005)\n","Processed MH0010_GL0042018: non-AMP/OTHERS (Probabilities: 0.9924, 0.0046, 0.0030)\n","Processed MH0188_GL0183755: non-AMP/OTHERS (Probabilities: 0.4929, 0.2284, 0.2787)\n","Processed MH0006_GL0021287: non-AMP/OTHERS (Probabilities: 0.9841, 0.0032, 0.0128)\n","Processed MH0002_GL0074907: non-AMP/OTHERS (Probabilities: 0.6222, 0.3696, 0.0083)\n","Processed MH0016_GL0052727: non-AMP/OTHERS (Probabilities: 0.9592, 0.0082, 0.0325)\n","Processed MH0026_GL0008864: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed MH0017_GL0010462: non-AMP/OTHERS (Probabilities: 0.9657, 0.0125, 0.0218)\n","Processed MH0003_GL0022627: Neuropeptide (Probabilities: 0.0591, 0.0398, 0.9011)\n","Processed SZEY-69A_GL0020037: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0054_GL0026456: Neuropeptide (Probabilities: 0.0125, 0.0110, 0.9764)\n","Processed 764588959-stool1_revised_scaffold122_3_gene75866: Neuropeptide (Probabilities: 0.0078, 0.0091, 0.9831)\n","Processed MH0185_GL0002854: Neuropeptide (Probabilities: 0.0105, 0.0098, 0.9797)\n","Processed MH0003_GL0043498: non-AMP/OTHERS (Probabilities: 0.9918, 0.0038, 0.0044)\n","Processed MH0020_GL0044510: non-AMP/OTHERS (Probabilities: 0.5703, 0.0349, 0.3948)\n","Processed MH0375_GL0126199: Neuropeptide (Probabilities: 0.0076, 0.0090, 0.9835)\n","Processed sp|P15144|AMPN_HUMAN: Neuropeptide (Probabilities: 0.3777, 0.0410, 0.5813)\n","Processed sp|P15144|AMPN_HUMAN: Neuropeptide (Probabilities: 0.3417, 0.0486, 0.6097)\n","Processed V1.FI28_GL0017018: Neuropeptide (Probabilities: 0.2189, 0.0590, 0.7221)\n","Processed MH0203_GL0011997: non-AMP/OTHERS (Probabilities: 0.9917, 0.0052, 0.0031)\n","Processed MH0199_GL0034725: non-AMP/OTHERS (Probabilities: 0.9919, 0.0050, 0.0031)\n","Processed sp|P48052|CBPA2_HUMAN: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed sp|P15085|CBPA1_HUMAN: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0370_GL0042621: Neuropeptide (Probabilities: 0.2866, 0.0529, 0.6605)\n","Processed BGI-33A_GL0063085: non-AMP/OTHERS (Probabilities: 0.7072, 0.0848, 0.2080)\n","Processed MH0074_GL0003362: non-AMP/OTHERS (Probabilities: 0.9450, 0.0219, 0.0331)\n","Processed MH0188_GL0103778: non-AMP/OTHERS (Probabilities: 0.9906, 0.0040, 0.0054)\n","Processed MH0327_GL0022594: non-AMP/OTHERS (Probabilities: 0.9925, 0.0036, 0.0040)\n","Processed 823052294-stool1_revised_C732513_1_gene75864: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0054_GL0019354: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0199_GL0034725: Neuropeptide (Probabilities: 0.0069, 0.0086, 0.9846)\n","Processed MH0212_GL0052331: Neuropeptide (Probabilities: 0.0070, 0.0086, 0.9845)\n","Processed MH0003_GL0010264: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0131_GL0110091: Neuropeptide (Probabilities: 0.0072, 0.0087, 0.9841)\n","Processed MH0113_GL0027089: Neuropeptide (Probabilities: 0.0063, 0.0087, 0.9851)\n","Processed NOM028_GL0070900: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0045)\n","Processed O2.UC20-1_GL0085446: Neuropeptide (Probabilities: 0.2196, 0.0284, 0.7520)\n","Processed 334390.LAF_1502: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0356_GL0133306: non-AMP/OTHERS (Probabilities: 0.9911, 0.0044, 0.0044)\n","Processed T2D-70A_GL0031741: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed O2.UC40-1_GL0251294: non-AMP/OTHERS (Probabilities: 0.8801, 0.1041, 0.0158)\n","Processed MH0023_GL0024415: non-AMP/OTHERS (Probabilities: 0.7152, 0.2788, 0.0060)\n","Processed MH0018_GL0014580: non-AMP/OTHERS (Probabilities: 0.9681, 0.0063, 0.0257)\n","Processed MH0250_GL0115028: Neuropeptide (Probabilities: 0.0107, 0.0101, 0.9792)\n","Processed MH0044_GL0067497: non-AMP/OTHERS (Probabilities: 0.9921, 0.0033, 0.0046)\n","Processed MH0161_GL0019990: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed MH0200_GL0138862: Neuropeptide (Probabilities: 0.0115, 0.0123, 0.9762)\n","Processed DOM010_GL0088706: Neuropeptide (Probabilities: 0.0081, 0.0094, 0.9824)\n","Processed MH0145_GL0055249: non-AMP/OTHERS (Probabilities: 0.5660, 0.0342, 0.3999)\n","Processed MH0086_GL0042171: Neuropeptide (Probabilities: 0.0110, 0.0096, 0.9794)\n","Processed MH0446_GL0103208: Neuropeptide (Probabilities: 0.0320, 0.0205, 0.9475)\n","Processed DOM026_GL0007628: non-AMP/OTHERS (Probabilities: 0.9767, 0.0077, 0.0156)\n","Processed V1.CD40-0_GL0133600: Neuropeptide (Probabilities: 0.1556, 0.1025, 0.7419)\n","Processed 457412.RSAG_02950: non-AMP/OTHERS (Probabilities: 0.9931, 0.0031, 0.0038)\n","Processed V1.CD2-0-PN_GL0017891: non-AMP/OTHERS (Probabilities: 0.4836, 0.0758, 0.4406)\n","Processed MH0369_GL0032418: Neuropeptide (Probabilities: 0.0396, 0.0174, 0.9430)\n","Processed MH0187_GL0059024: non-AMP/OTHERS (Probabilities: 0.9886, 0.0045, 0.0069)\n","Processed 763860675-stool1_revised_scaffold31428_1_gene3042: non-AMP/OTHERS (Probabilities: 0.9921, 0.0044, 0.0035)\n","Processed MH0188_GL0026540: Neuropeptide (Probabilities: 0.0063, 0.0088, 0.9849)\n","Processed 159713063-stool1_revised_C504757_1_gene51584: non-AMP/OTHERS (Probabilities: 0.9920, 0.0040, 0.0040)\n","Processed MH0003_GL0069935: non-AMP/OTHERS (Probabilities: 0.9934, 0.0036, 0.0031)\n","Processed MH0238_GL0065563: non-AMP/OTHERS (Probabilities: 0.9052, 0.0115, 0.0833)\n","Processed SZEY-103A_GL0053874: Neuropeptide (Probabilities: 0.0223, 0.0167, 0.9610)\n","Processed NLM021_GL0081278: AMP (Probabilities: 0.2865, 0.4856, 0.2279)\n","Processed MH0133_GL0090494: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0244_GL0062774: non-AMP/OTHERS (Probabilities: 0.9900, 0.0051, 0.0049)\n","Processed MH0400_GL0071356: non-AMP/OTHERS (Probabilities: 0.9914, 0.0058, 0.0028)\n","Processed MH0074_GL0058556: non-AMP/OTHERS (Probabilities: 0.9887, 0.0041, 0.0072)\n","Processed MH0311_GL0057761: non-AMP/OTHERS (Probabilities: 0.9908, 0.0038, 0.0054)\n","Processed MH0381_GL0139415: Neuropeptide (Probabilities: 0.0085, 0.0086, 0.9828)\n","Processed 1042403.BALAC2494_00245: non-AMP/OTHERS (Probabilities: 0.9866, 0.0068, 0.0066)\n","Processed MH0159_GL0105010: non-AMP/OTHERS (Probabilities: 0.9816, 0.0078, 0.0106)\n","Processed MH0055_GL0009975: non-AMP/OTHERS (Probabilities: 0.9048, 0.0271, 0.0681)\n","Processed O2.UC17-1_GL0103202: non-AMP/OTHERS (Probabilities: 0.9935, 0.0033, 0.0032)\n","Processed O2.UC19-1_GL0089391: non-AMP/OTHERS (Probabilities: 0.9901, 0.0071, 0.0028)\n","Processed 547043.BIFPSEUDO_02724: non-AMP/OTHERS (Probabilities: 0.9921, 0.0036, 0.0043)\n","Processed 547043.BIFPSEUDO_02724: non-AMP/OTHERS (Probabilities: 0.8644, 0.0200, 0.1156)\n","Processed 411481.BIFADO_00476: Neuropeptide (Probabilities: 0.0467, 0.0211, 0.9322)\n","Processed 411481.BIFADO_00476: non-AMP/OTHERS (Probabilities: 0.8909, 0.0214, 0.0876)\n","Processed 537937.BLIG_02144: Neuropeptide (Probabilities: 0.0103, 0.0099, 0.9797)\n","Processed 537937.BLIG_02144: Neuropeptide (Probabilities: 0.0110, 0.0106, 0.9784)\n","Processed MH0222_GL0093396: non-AMP/OTHERS (Probabilities: 0.9642, 0.0081, 0.0277)\n","Processed MH0391_GL0162846: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed V1.FI07_GL0102201: Neuropeptide (Probabilities: 0.0067, 0.0085, 0.9848)\n","Processed O2.UC8-1_GL0031823: non-AMP/OTHERS (Probabilities: 0.9910, 0.0042, 0.0049)\n","Processed MH0149_GL0106021: non-AMP/OTHERS (Probabilities: 0.9913, 0.0047, 0.0040)\n","Processed MH0006_GL0144221: Neuropeptide (Probabilities: 0.0178, 0.0124, 0.9697)\n","Processed MH0026_GL0025845: non-AMP/OTHERS (Probabilities: 0.9893, 0.0048, 0.0058)\n","Processed O2.CD1-0-PT_GL0056649: non-AMP/OTHERS (Probabilities: 0.9910, 0.0034, 0.0055)\n","Processed 158337416-stool2_revised_C1045768_1_gene102471: Neuropeptide (Probabilities: 0.3033, 0.0508, 0.6459)\n","Processed MH0433_GL0182416: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0251_GL0055910: Neuropeptide (Probabilities: 0.2089, 0.0331, 0.7579)\n","Processed V1.UC13-0_GL0054961: non-AMP/OTHERS (Probabilities: 0.8507, 0.0344, 0.1149)\n","Processed MH0083_GL0063758: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0013_GL0018442: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0011_GL0033290: non-AMP/OTHERS (Probabilities: 0.9930, 0.0040, 0.0030)\n","Processed O2.UC48-0_GL0237359: Neuropeptide (Probabilities: 0.3874, 0.0623, 0.5503)\n","Processed O2.UC17-1_GL0083797: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed MH0092_GL0020057: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0271_GL0029746: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed O2.UC30-0_GL0000699: Neuropeptide (Probabilities: 0.0069, 0.0087, 0.9844)\n","Processed MH0131_GL0067130: non-AMP/OTHERS (Probabilities: 0.9925, 0.0041, 0.0034)\n","Processed V1.CD2-0-PT_GL0000436: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0239_GL0157898: non-AMP/OTHERS (Probabilities: 0.9891, 0.0062, 0.0046)\n","Processed O2.UC17-1_GL0040485: Neuropeptide (Probabilities: 0.0080, 0.0092, 0.9828)\n","Processed DOF004_GL0090170: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0003_GL0073370: non-AMP/OTHERS (Probabilities: 0.9922, 0.0032, 0.0046)\n","Processed O2.UC27-0_GL0046636: Neuropeptide (Probabilities: 0.2432, 0.0270, 0.7299)\n","Processed MH0441_GL0206408: non-AMP/OTHERS (Probabilities: 0.9862, 0.0042, 0.0096)\n","Processed N027A_GL0049059: non-AMP/OTHERS (Probabilities: 0.9845, 0.0059, 0.0096)\n","Processed MH0367_GL0090768: non-AMP/OTHERS (Probabilities: 0.9924, 0.0034, 0.0042)\n","Processed MH0124_GL0081077: Neuropeptide (Probabilities: 0.4058, 0.1061, 0.4881)\n","Processed O2.UC50-2_GL0134714: Neuropeptide (Probabilities: 0.0522, 0.0343, 0.9136)\n","Processed MH0277_GL0013178: non-AMP/OTHERS (Probabilities: 0.9608, 0.0224, 0.0167)\n","Processed MH0006_GL0093930: non-AMP/OTHERS (Probabilities: 0.9880, 0.0034, 0.0086)\n","Processed V1.FI08_GL0038671: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9851)\n","Processed 908612.HMPREF9720_2004: non-AMP/OTHERS (Probabilities: 0.8675, 0.1222, 0.0103)\n","Processed O2.UC20-1_GL0169349: Neuropeptide (Probabilities: 0.0086, 0.0097, 0.9817)\n","Processed MH0087_GL0008588: Neuropeptide (Probabilities: 0.0151, 0.0113, 0.9736)\n","Processed 158337416-stool1_revised_scaffold29327_1_gene92715: Neuropeptide (Probabilities: 0.0074, 0.0086, 0.9840)\n","Processed MH0236_GL0049453: Neuropeptide (Probabilities: 0.0378, 0.0144, 0.9479)\n","Processed MH0028_GL0066735: Neuropeptide (Probabilities: 0.0184, 0.0136, 0.9681)\n","Processed MH0191_GL0072140: Neuropeptide (Probabilities: 0.0271, 0.0272, 0.9457)\n","Processed MH0002_GL0066664: non-AMP/OTHERS (Probabilities: 0.9860, 0.0057, 0.0083)\n","Processed 160218816-stool1_revised_C881900_1_gene84963: non-AMP/OTHERS (Probabilities: 0.9603, 0.0331, 0.0066)\n","Processed O2.UC15-0_GL0016463: non-AMP/OTHERS (Probabilities: 0.9929, 0.0042, 0.0028)\n","Processed MH0014_GL0090940: Neuropeptide (Probabilities: 0.0065, 0.0085, 0.9850)\n","Processed MH0039_GL0055871: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed N037A_GL0052806: Neuropeptide (Probabilities: 0.0299, 0.0152, 0.9549)\n","Processed MH0301_GL0022139: AMP (Probabilities: 0.3305, 0.4509, 0.2186)\n","Processed MH0089_GL0026063: non-AMP/OTHERS (Probabilities: 0.9920, 0.0042, 0.0037)\n","Processed MH0113_GL0099888: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9851)\n","Processed MH0006_GL0057816: non-AMP/OTHERS (Probabilities: 0.9665, 0.0085, 0.0250)\n","Processed MH0110_GL0082558: non-AMP/OTHERS (Probabilities: 0.9628, 0.0162, 0.0209)\n","Processed MH0398_GL0069409: non-AMP/OTHERS (Probabilities: 0.9912, 0.0057, 0.0032)\n","Processed MH0398_GL0069409: non-AMP/OTHERS (Probabilities: 0.9111, 0.0844, 0.0045)\n","Processed MH0002_GL0035883: non-AMP/OTHERS (Probabilities: 0.9741, 0.0060, 0.0199)\n","Processed MH0281_GL0141686: non-AMP/OTHERS (Probabilities: 0.7005, 0.1222, 0.1773)\n","Processed NOM019_GL0078489: non-AMP/OTHERS (Probabilities: 0.9901, 0.0038, 0.0061)\n","Processed MH0051_GL0048904: non-AMP/OTHERS (Probabilities: 0.9881, 0.0051, 0.0068)\n","Processed N033A_GL0007242: Neuropeptide (Probabilities: 0.0091, 0.0094, 0.9815)\n","Processed V1.CD46-0_GL0018942: Neuropeptide (Probabilities: 0.0067, 0.0085, 0.9847)\n","Processed O2.UC38-1_GL0113728: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0345_GL0137422: non-AMP/OTHERS (Probabilities: 0.9869, 0.0058, 0.0074)\n","Processed T2D-78A_GL0015703: non-AMP/OTHERS (Probabilities: 0.9901, 0.0066, 0.0033)\n","Processed SZEY-62A_GL0070012: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0044)\n","Processed O2.CD3-0-PT_GL0079343: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0001_GL0015817: Neuropeptide (Probabilities: 0.0652, 0.0220, 0.9128)\n","Processed 479437.Elen_1043: non-AMP/OTHERS (Probabilities: 0.9881, 0.0074, 0.0044)\n","Processed MH0239_GL0157898: non-AMP/OTHERS (Probabilities: 0.9398, 0.0135, 0.0467)\n","Processed MH0270_GL0121925: non-AMP/OTHERS (Probabilities: 0.9862, 0.0053, 0.0085)\n","Processed MH0197_GL0017710: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0402_GL0134720: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed MH0330_GL0207275: non-AMP/OTHERS (Probabilities: 0.8699, 0.0220, 0.1081)\n","Processed O2.UC30-0_GL0091402: Neuropeptide (Probabilities: 0.0208, 0.0153, 0.9639)\n","Processed MH0002_GL0047966: non-AMP/OTHERS (Probabilities: 0.9243, 0.0235, 0.0522)\n","Processed MH0251_GL0174421: Neuropeptide (Probabilities: 0.0108, 0.0100, 0.9792)\n","Processed MH0003_GL0040136: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0003_GL0031524: non-AMP/OTHERS (Probabilities: 0.9904, 0.0032, 0.0063)\n","Processed MH0187_GL0127824: non-AMP/OTHERS (Probabilities: 0.9811, 0.0152, 0.0037)\n","Processed MH0370_GL0058834: non-AMP/OTHERS (Probabilities: 0.9905, 0.0040, 0.0054)\n","Processed V1.UC33-0_GL0020824: non-AMP/OTHERS (Probabilities: 0.9911, 0.0044, 0.0045)\n","Processed 537937.BLIG_01296: non-AMP/OTHERS (Probabilities: 0.9902, 0.0036, 0.0063)\n","Processed V1.CD25-0_GL0054229: non-AMP/OTHERS (Probabilities: 0.9915, 0.0044, 0.0041)\n","Processed 158458797-stool1_revised_scaffold28810_1_gene9011: non-AMP/OTHERS (Probabilities: 0.7916, 0.1824, 0.0260)\n","Processed MH0009_GL0067278: non-AMP/OTHERS (Probabilities: 0.9808, 0.0072, 0.0120)\n","Processed MH0085_GL0031382: non-AMP/OTHERS (Probabilities: 0.9886, 0.0052, 0.0062)\n","Processed N088A_GL0077420: non-AMP/OTHERS (Probabilities: 0.9886, 0.0074, 0.0040)\n","Processed 469593.HMPREF9011_01321: Neuropeptide (Probabilities: 0.0068, 0.0085, 0.9846)\n","Processed MH0314_GL0118797: non-AMP/OTHERS (Probabilities: 0.9748, 0.0056, 0.0196)\n","Processed V1.FI01_GL0193945: non-AMP/OTHERS (Probabilities: 0.9932, 0.0039, 0.0029)\n","Processed MH0006_GL0001542: non-AMP/OTHERS (Probabilities: 0.7846, 0.0453, 0.1700)\n","Processed V1.UC16-0_GL0074812: Neuropeptide (Probabilities: 0.0074, 0.0086, 0.9840)\n","Processed V1.UC16-0_GL0074812: Neuropeptide (Probabilities: 0.0242, 0.0215, 0.9543)\n","Processed V1.CD7-0-PN_GL0042618: non-AMP/OTHERS (Probabilities: 0.9927, 0.0037, 0.0035)\n","Processed MH0356_GL0057918: non-AMP/OTHERS (Probabilities: 0.9925, 0.0038, 0.0037)\n","Processed MH0205_GL0155933: Neuropeptide (Probabilities: 0.0076, 0.0088, 0.9837)\n","Processed O2.UC38-1_GL0062113: Neuropeptide (Probabilities: 0.0253, 0.0186, 0.9561)\n","Processed MH0128_GL0018202: Neuropeptide (Probabilities: 0.0065, 0.0085, 0.9850)\n","Processed 158337416-stool1_revised_C1271394_1_gene70210: non-AMP/OTHERS (Probabilities: 0.9883, 0.0048, 0.0069)\n","Processed MH0002_GL0058940: non-AMP/OTHERS (Probabilities: 0.9441, 0.0409, 0.0150)\n","Processed O2.UC15-2_GL0097856: non-AMP/OTHERS (Probabilities: 0.9129, 0.0349, 0.0521)\n","Processed MH0130_GL0033950: non-AMP/OTHERS (Probabilities: 0.9853, 0.0075, 0.0072)\n","Processed MH0211_GL0067590: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed HT8A_GL0058729: Neuropeptide (Probabilities: 0.0070, 0.0085, 0.9845)\n","Processed MH0055_GL0036804: Neuropeptide (Probabilities: 0.0484, 0.0377, 0.9139)\n","Processed NLF006_GL0014278: non-AMP/OTHERS (Probabilities: 0.9900, 0.0073, 0.0027)\n","Processed MH0131_GL0160692: Neuropeptide (Probabilities: 0.0098, 0.0095, 0.9808)\n","Processed DLF001_GL0024631: non-AMP/OTHERS (Probabilities: 0.9880, 0.0038, 0.0082)\n","Processed 765094712-stool1_revised_scaffold3690_2_gene47385: non-AMP/OTHERS (Probabilities: 0.9918, 0.0044, 0.0039)\n","Processed O2.CD1-0-PT_GL0069667: non-AMP/OTHERS (Probabilities: 0.5015, 0.4178, 0.0807)\n","Processed 264199.stu0692: non-AMP/OTHERS (Probabilities: 0.5972, 0.1875, 0.2153)\n","Processed ED9A_GL0030837: non-AMP/OTHERS (Probabilities: 0.9908, 0.0057, 0.0035)\n","Processed 709991.Odosp_3575: non-AMP/OTHERS (Probabilities: 0.9923, 0.0031, 0.0046)\n","Processed O2.UC37-2_GL0045315: non-AMP/OTHERS (Probabilities: 0.9911, 0.0048, 0.0041)\n","Processed MH0086_GL0088724: non-AMP/OTHERS (Probabilities: 0.9630, 0.0081, 0.0290)\n","Processed V1.UC35-0_GL0069105: non-AMP/OTHERS (Probabilities: 0.9913, 0.0043, 0.0044)\n","Processed MH0074_GL0014887: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed T2D-68A_GL0035724: non-AMP/OTHERS (Probabilities: 0.9564, 0.0216, 0.0220)\n","Processed DOF012_GL0004220: non-AMP/OTHERS (Probabilities: 0.9271, 0.0627, 0.0102)\n","Processed V1.FI31_GL0135429: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 823052294-stool1_revised_C732513_1_gene75864: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0003_GL0047575: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9851)\n","Processed MH0088_GL0017906: non-AMP/OTHERS (Probabilities: 0.9585, 0.0349, 0.0066)\n","Processed V1.UC12-4_GL0157002: non-AMP/OTHERS (Probabilities: 0.9786, 0.0103, 0.0111)\n","Processed MH0211_GL0055401: non-AMP/OTHERS (Probabilities: 0.9914, 0.0044, 0.0042)\n","Processed BGI-27A_GL0076293: non-AMP/OTHERS (Probabilities: 0.9313, 0.0263, 0.0424)\n","Processed MH0006_GL0127001: Neuropeptide (Probabilities: 0.0136, 0.0110, 0.9754)\n","Processed MH0251_GL0005451: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0275_GL0144248: Neuropeptide (Probabilities: 0.0076, 0.0089, 0.9835)\n","Processed MH0200_GL0038670: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0386_GL0071409: non-AMP/OTHERS (Probabilities: 0.9240, 0.0138, 0.0622)\n","Processed MH0092_GL0088682: non-AMP/OTHERS (Probabilities: 0.9630, 0.0112, 0.0258)\n","Processed MH0002_GL0028113: non-AMP/OTHERS (Probabilities: 0.9574, 0.0223, 0.0203)\n","Processed MH0316_GL0190431: non-AMP/OTHERS (Probabilities: 0.8596, 0.0333, 0.1071)\n","Processed MH0062_GL0014969: Neuropeptide (Probabilities: 0.0063, 0.0086, 0.9851)\n","Processed 246515023-stool1_revised_scaffold4703_1_gene50269: non-AMP/OTHERS (Probabilities: 0.9917, 0.0036, 0.0047)\n","Processed 158499257-stool1_revised_C1271806_1_gene207293: non-AMP/OTHERS (Probabilities: 0.9832, 0.0086, 0.0082)\n","Processed MH0188_GL0056321: non-AMP/OTHERS (Probabilities: 0.7091, 0.0194, 0.2715)\n","Processed O2.UC11-1_GL0150059: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed O2.UC53-0_GL0274354: non-AMP/OTHERS (Probabilities: 0.8556, 0.0456, 0.0989)\n","Processed MH0448_GL0060458: non-AMP/OTHERS (Probabilities: 0.8076, 0.0333, 0.1591)\n","Processed V1.FI24_GL0108913: non-AMP/OTHERS (Probabilities: 0.9853, 0.0057, 0.0090)\n","Processed MH0086_GL0092003: Neuropeptide (Probabilities: 0.0929, 0.0226, 0.8844)\n","Processed MH0086_GL0010554: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed V1.CD45-0_GL0016359: Neuropeptide (Probabilities: 0.0129, 0.0120, 0.9751)\n","Processed T2D-6A_GL0097996: Neuropeptide (Probabilities: 0.2680, 0.0271, 0.7048)\n","Processed O2.UC23-2_GL0034778: non-AMP/OTHERS (Probabilities: 0.5073, 0.0716, 0.4211)\n","Processed MH0443_GL0005679: Neuropeptide (Probabilities: 0.0106, 0.0098, 0.9796)\n","Processed MH0441_GL0089727: Neuropeptide (Probabilities: 0.0079, 0.0090, 0.9832)\n","Processed MH0345_GL0125947: Neuropeptide (Probabilities: 0.0087, 0.0097, 0.9816)\n","Processed V1.CD21-0_GL0006059: Neuropeptide (Probabilities: 0.0217, 0.0134, 0.9649)\n","Processed O2.UC8-2_GL0106576: non-AMP/OTHERS (Probabilities: 0.9908, 0.0052, 0.0040)\n","Processed V1.UC38-4_GL0094780: non-AMP/OTHERS (Probabilities: 0.8016, 0.0346, 0.1638)\n","Processed MH0026_GL0058653: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9849)\n","Processed V1.CD43-0_GL0057588: non-AMP/OTHERS (Probabilities: 0.9886, 0.0064, 0.0050)\n","Processed O2.UC11-2_GL0041450: Neuropeptide (Probabilities: 0.0073, 0.0086, 0.9841)\n","Processed MH0150_GL0015844: non-AMP/OTHERS (Probabilities: 0.9916, 0.0055, 0.0028)\n","Processed MH0045_GL0032209: non-AMP/OTHERS (Probabilities: 0.9779, 0.0076, 0.0145)\n","Processed MH0023_GL0007622: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed 159268001-stool1_revised_C1015785_1_gene47829: non-AMP/OTHERS (Probabilities: 0.9927, 0.0039, 0.0034)\n","Processed 565042.BLLJ_1804: non-AMP/OTHERS (Probabilities: 0.7540, 0.0246, 0.2214)\n","Processed V1.CD44-0_GL0007623: non-AMP/OTHERS (Probabilities: 0.9904, 0.0046, 0.0050)\n","Processed DLM027_GL0037652: non-AMP/OTHERS (Probabilities: 0.8874, 0.0159, 0.0967)\n","Processed O2.UC30-0_GL0093651: non-AMP/OTHERS (Probabilities: 0.9924, 0.0033, 0.0043)\n","Processed MH0277_GL0045973: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed 763759525-stool2_revised_C499976_1_gene13017: non-AMP/OTHERS (Probabilities: 0.9922, 0.0034, 0.0044)\n","Processed MH0088_GL0132621: Neuropeptide (Probabilities: 0.0077, 0.0091, 0.9833)\n","Processed sp|P98073|ENTK_HUMAN: Neuropeptide (Probabilities: 0.0171, 0.0145, 0.9684)\n","Processed 158499257-stool1_revised_scaffold9312_2_gene138791: Neuropeptide (Probabilities: 0.2605, 0.0865, 0.6529)\n","Processed T2D-103A_GL0042076: Neuropeptide (Probabilities: 0.0068, 0.0089, 0.9843)\n","Processed NOM016_GL0015381: Neuropeptide (Probabilities: 0.0070, 0.0087, 0.9843)\n","Processed NOM016_GL0015381: non-AMP/OTHERS (Probabilities: 0.9229, 0.0229, 0.0542)\n","Processed MH0032_GL0009850: non-AMP/OTHERS (Probabilities: 0.9843, 0.0045, 0.0112)\n","Processed MH0239_GL0078811: non-AMP/OTHERS (Probabilities: 0.9867, 0.0057, 0.0077)\n","Processed MH0011_GL0017878: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed O2.UC43-1_GL0053675: non-AMP/OTHERS (Probabilities: 0.9905, 0.0039, 0.0056)\n","Processed MH0305_GL0011259: Neuropeptide (Probabilities: 0.0086, 0.0099, 0.9815)\n","Processed MH0176_GL0019913: non-AMP/OTHERS (Probabilities: 0.5835, 0.0373, 0.3791)\n","Processed MH0399_GL0249530: Neuropeptide (Probabilities: 0.0131, 0.0116, 0.9754)\n","Processed MH0375_GL0070847: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed MH0016_GL0085869: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed 428126.CLOSPI_02309: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed BGI003A_GL0026319: non-AMP/OTHERS (Probabilities: 0.4214, 0.2553, 0.3233)\n","Processed MH0211_GL0033014: non-AMP/OTHERS (Probabilities: 0.9515, 0.0158, 0.0327)\n","Processed MH0122_GL0026668: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 404239096-stool1_revised_scaffold6512_11_gene17378: non-AMP/OTHERS (Probabilities: 0.9917, 0.0045, 0.0038)\n","Processed T2D-133A_GL0004968: non-AMP/OTHERS (Probabilities: 0.9907, 0.0060, 0.0033)\n","Processed MH0188_GL0118581: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed V1.CD6-4_GL0055248: Neuropeptide (Probabilities: 0.3679, 0.0565, 0.5756)\n","Processed MH0006_GL0142269: non-AMP/OTHERS (Probabilities: 0.9843, 0.0099, 0.0059)\n","Processed MH0338_GL0164043: non-AMP/OTHERS (Probabilities: 0.9922, 0.0035, 0.0043)\n","Processed O2.UC17-1_GL0038070: non-AMP/OTHERS (Probabilities: 0.8830, 0.0395, 0.0775)\n","Processed MH0125_GL0030562: non-AMP/OTHERS (Probabilities: 0.9915, 0.0034, 0.0051)\n","Processed MH0277_GL0000971: non-AMP/OTHERS (Probabilities: 0.9901, 0.0074, 0.0025)\n","Processed V1.CD6-0-PN_GL0090532: non-AMP/OTHERS (Probabilities: 0.9920, 0.0053, 0.0027)\n","Processed MH0333_GL0121497: Neuropeptide (Probabilities: 0.0093, 0.0100, 0.9806)\n","Processed NOM009_GL0092368: non-AMP/OTHERS (Probabilities: 0.9919, 0.0040, 0.0041)\n","Processed MH0425_GL0149565: Neuropeptide (Probabilities: 0.0098, 0.0095, 0.9807)\n","Processed O2.UC59-2_GL0042274: non-AMP/OTHERS (Probabilities: 0.9655, 0.0072, 0.0273)\n","Processed MH0102_GL0031449: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed 159632143-stool1_revised_scaffold46407_2_gene23189: Neuropeptide (Probabilities: 0.0152, 0.0145, 0.9703)\n","Processed 764143897-stool1_revised_C889242_1_gene105398: non-AMP/OTHERS (Probabilities: 0.9883, 0.0054, 0.0063)\n","Processed MH0360_GL0079565: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0036)\n","Processed O2.UC7-0_GL0079668: non-AMP/OTHERS (Probabilities: 0.9904, 0.0047, 0.0049)\n","Processed MH0071_GL0063116: non-AMP/OTHERS (Probabilities: 0.9884, 0.0054, 0.0062)\n","Processed MH0339_GL0124734: Neuropeptide (Probabilities: 0.0855, 0.0298, 0.8847)\n","Processed MH0088_GL0034073: Neuropeptide (Probabilities: 0.0269, 0.0192, 0.9539)\n","Processed MH0327_GL0079702: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0033)\n","Processed MH0188_GL0141556: Neuropeptide (Probabilities: 0.3128, 0.1921, 0.4951)\n","Processed O2.UC11-2_GL0047134: non-AMP/OTHERS (Probabilities: 0.9753, 0.0127, 0.0119)\n","Processed 553973.CLOHYLEM_06673: non-AMP/OTHERS (Probabilities: 0.5942, 0.0479, 0.3579)\n","Processed MH0054_GL0073289: non-AMP/OTHERS (Probabilities: 0.9831, 0.0084, 0.0085)\n","Processed MH0088_GL0012814: non-AMP/OTHERS (Probabilities: 0.9926, 0.0040, 0.0033)\n","Processed MH0370_GL0032782: non-AMP/OTHERS (Probabilities: 0.6445, 0.1111, 0.2444)\n","Processed MH0002_GL0046841: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0003_GL0018734: non-AMP/OTHERS (Probabilities: 0.9901, 0.0060, 0.0040)\n","Processed MH0006_GL0057599: Neuropeptide (Probabilities: 0.0380, 0.0226, 0.9394)\n","Processed MH0012_GL0150149: Neuropeptide (Probabilities: 0.0736, 0.0558, 0.8706)\n","Processed O2.UC32-0_GL0043392: non-AMP/OTHERS (Probabilities: 0.9920, 0.0047, 0.0033)\n","Processed MH0233_GL0079949: Neuropeptide (Probabilities: 0.0066, 0.0087, 0.9847)\n","Processed ED13A_GL0036684: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0003_GL0075427: Neuropeptide (Probabilities: 0.0073, 0.0088, 0.9839)\n","Processed MH0236_GL0016844: Neuropeptide (Probabilities: 0.0068, 0.0086, 0.9845)\n","Processed V1.UC58-0_GL0003356: Neuropeptide (Probabilities: 0.0082, 0.0091, 0.9827)\n","Processed V1.UC52-1_GL0178638: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0003_GL0048640: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0396_GL0183408: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9849)\n","Processed MH0054_GL0019355: non-AMP/OTHERS (Probabilities: 0.9167, 0.0496, 0.0337)\n","Processed MH0088_GL0070845: non-AMP/OTHERS (Probabilities: 0.3780, 0.3742, 0.2478)\n","Processed MH0018_GL0017414: non-AMP/OTHERS (Probabilities: 0.9440, 0.0465, 0.0095)\n","Processed MH0334_GL0118374: Neuropeptide (Probabilities: 0.0073, 0.0085, 0.9843)\n","Processed MH0188_GL0079775: non-AMP/OTHERS (Probabilities: 0.9846, 0.0064, 0.0091)\n","Processed tr|A0A5C2GLK2|A0A5C2GLK2_HUMAN: non-AMP/OTHERS (Probabilities: 0.9818, 0.0153, 0.0029)\n","Processed MH0043_GL0024560: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed 272621.LBA0169: non-AMP/OTHERS (Probabilities: 0.9913, 0.0043, 0.0044)\n","Processed T2D-20A_GL0000966: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed O2.UC49-0_GL0069483: Neuropeptide (Probabilities: 0.0112, 0.0099, 0.9789)\n","Processed 764143897-stool1_revised_scaffold622_3_gene45079: non-AMP/OTHERS (Probabilities: 0.9228, 0.0133, 0.0639)\n","Processed MH0205_GL0063348: Neuropeptide (Probabilities: 0.0084, 0.0095, 0.9821)\n","Processed O2.UC24-2_GL0024588: non-AMP/OTHERS (Probabilities: 0.9797, 0.0074, 0.0129)\n","Processed V1.FI22_GL0136429: AMP (Probabilities: 0.2314, 0.7579, 0.0108)\n","Processed V1.UC30-0_GL0127415: non-AMP/OTHERS (Probabilities: 0.9800, 0.0165, 0.0035)\n","Processed 158256496-stool1_revised_scaffold10996_1_gene28058: non-AMP/OTHERS (Probabilities: 0.9731, 0.0102, 0.0167)\n","Processed V1.FI04_GL0211510: non-AMP/OTHERS (Probabilities: 0.9675, 0.0130, 0.0195)\n","Processed MH0410_GL0157934: Neuropeptide (Probabilities: 0.0059, 0.0087, 0.9853)\n","Processed MH0013_GL0032047: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed V1.UC52-1_GL0103474: non-AMP/OTHERS (Probabilities: 0.8534, 0.0388, 0.1078)\n","Processed MH0239_GL0017274: Neuropeptide (Probabilities: 0.0381, 0.0198, 0.9421)\n","Processed MH0382_GL0111772: non-AMP/OTHERS (Probabilities: 0.9883, 0.0058, 0.0060)\n","Processed MH0150_GL0127769: non-AMP/OTHERS (Probabilities: 0.9894, 0.0043, 0.0062)\n","Processed MH0259_GL0125589: non-AMP/OTHERS (Probabilities: 0.9771, 0.0082, 0.0146)\n","Processed MH0001_GL0005568: Neuropeptide (Probabilities: 0.0176, 0.0126, 0.9698)\n","Processed MH0418_GL0037069: Neuropeptide (Probabilities: 0.0201, 0.0134, 0.9666)\n","Processed MH0416_GL0175200: non-AMP/OTHERS (Probabilities: 0.9808, 0.0057, 0.0135)\n","Processed MH0370_GL0112692: non-AMP/OTHERS (Probabilities: 0.9874, 0.0053, 0.0073)\n","Processed 764002286-stool1_revised_scaffold2122_1_gene10000: Neuropeptide (Probabilities: 0.0122, 0.0106, 0.9773)\n","Processed MH0087_GL0035200: Neuropeptide (Probabilities: 0.0661, 0.0185, 0.9153)\n","Processed MH0284_GL0144352: non-AMP/OTHERS (Probabilities: 0.9929, 0.0029, 0.0043)\n","Processed MH0277_GL0040049: non-AMP/OTHERS (Probabilities: 0.9885, 0.0052, 0.0063)\n","Processed MH0233_GL0095038: non-AMP/OTHERS (Probabilities: 0.9866, 0.0035, 0.0099)\n","Processed V1.UC40-0_GL0130575: Neuropeptide (Probabilities: 0.0075, 0.0089, 0.9836)\n","Processed MH0062_GL0028982: non-AMP/OTHERS (Probabilities: 0.8736, 0.0169, 0.1096)\n","Processed MH0329_GL0074523: non-AMP/OTHERS (Probabilities: 0.9907, 0.0043, 0.0050)\n","Processed 521002.METSMIF1_02270: non-AMP/OTHERS (Probabilities: 0.9924, 0.0034, 0.0042)\n","Processed MH0193_GL0102881: Neuropeptide (Probabilities: 0.0204, 0.0155, 0.9641)\n","Processed V1.UC11-0_GL0020196: non-AMP/OTHERS (Probabilities: 0.6848, 0.0632, 0.2520)\n","Processed V1.UC11-0_GL0020196: non-AMP/OTHERS (Probabilities: 0.9923, 0.0048, 0.0029)\n","Processed MH0222_GL0076490: non-AMP/OTHERS (Probabilities: 0.9923, 0.0032, 0.0045)\n","Processed MH0014_GL0019222: non-AMP/OTHERS (Probabilities: 0.8918, 0.0160, 0.0922)\n","Processed MH0014_GL0080693: non-AMP/OTHERS (Probabilities: 0.9902, 0.0034, 0.0064)\n","Processed MH0345_GL0107228: non-AMP/OTHERS (Probabilities: 0.9645, 0.0077, 0.0277)\n","Processed MH0001_GL0015313: non-AMP/OTHERS (Probabilities: 0.9899, 0.0041, 0.0060)\n","Processed 158337416-stool2_revised_C968591_1_gene114129: non-AMP/OTHERS (Probabilities: 0.9920, 0.0049, 0.0031)\n","Processed MH0002_GL0050951: non-AMP/OTHERS (Probabilities: 0.9931, 0.0030, 0.0039)\n","Processed MH0002_GL0050951: non-AMP/OTHERS (Probabilities: 0.9933, 0.0034, 0.0033)\n","Processed O2.UC41-1_GL0089426: non-AMP/OTHERS (Probabilities: 0.9861, 0.0042, 0.0097)\n","Processed 763759525-stool1_revised_scaffold12737_1_gene81174: non-AMP/OTHERS (Probabilities: 0.9923, 0.0035, 0.0042)\n","Processed SZEY-81A_GL0036080: non-AMP/OTHERS (Probabilities: 0.9841, 0.0049, 0.0109)\n","Processed MH0188_GL0088109: non-AMP/OTHERS (Probabilities: 0.9926, 0.0030, 0.0044)\n","Processed MH0441_GL0090117: non-AMP/OTHERS (Probabilities: 0.9934, 0.0034, 0.0033)\n","Processed MH0113_GL0046892: non-AMP/OTHERS (Probabilities: 0.9926, 0.0035, 0.0039)\n","Processed 515619.EUBREC_1472: non-AMP/OTHERS (Probabilities: 0.9926, 0.0042, 0.0032)\n","Processed MH0086_GL0046721: non-AMP/OTHERS (Probabilities: 0.9923, 0.0038, 0.0040)\n","Processed MH0206_GL0180285: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9850)\n","Processed MH0131_GL0105124: AMP (Probabilities: 0.1910, 0.7211, 0.0880)\n","Processed MH0277_GL0006823: non-AMP/OTHERS (Probabilities: 0.9901, 0.0039, 0.0059)\n","Processed MH0003_GL0089553: non-AMP/OTHERS (Probabilities: 0.8212, 0.0366, 0.1423)\n","Processed T2D-22A_GL0041016: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0038)\n","Processed V1.FI29_GL0077596: non-AMP/OTHERS (Probabilities: 0.9927, 0.0038, 0.0034)\n","Processed V1.FI20_GL0040175: non-AMP/OTHERS (Probabilities: 0.9919, 0.0043, 0.0037)\n","Processed MH0318_GL0075991: Neuropeptide (Probabilities: 0.0068, 0.0086, 0.9846)\n","Processed MH0143_GL0103835: non-AMP/OTHERS (Probabilities: 0.9686, 0.0102, 0.0212)\n","Processed DLM008_GL0027121: AMP (Probabilities: 0.2769, 0.5714, 0.1516)\n","Processed MH0230_GL0089844: non-AMP/OTHERS (Probabilities: 0.9901, 0.0049, 0.0050)\n","Processed sp|O15162|PLS1_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed MH0086_GL0092513: non-AMP/OTHERS (Probabilities: 0.9896, 0.0044, 0.0060)\n","Processed DOF008_GL0021589: non-AMP/OTHERS (Probabilities: 0.9815, 0.0070, 0.0115)\n","Processed MH0004_GL0016946: non-AMP/OTHERS (Probabilities: 0.9841, 0.0066, 0.0093)\n","Processed MH0446_GL0032744: non-AMP/OTHERS (Probabilities: 0.9786, 0.0094, 0.0120)\n","Processed O2.UC58-0_GL0093768: non-AMP/OTHERS (Probabilities: 0.9928, 0.0043, 0.0029)\n","Processed MH0118_GL0117499: non-AMP/OTHERS (Probabilities: 0.9882, 0.0048, 0.0071)\n","Processed MH0073_GL0041082: Neuropeptide (Probabilities: 0.0070, 0.0086, 0.9843)\n","Processed MH0196_GL0117151: non-AMP/OTHERS (Probabilities: 0.9869, 0.0066, 0.0065)\n","Processed MH0196_GL0056900: Neuropeptide (Probabilities: 0.0080, 0.0089, 0.9832)\n","Processed MH0406_GL0019760: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0359_GL0127021: non-AMP/OTHERS (Probabilities: 0.7399, 0.0357, 0.2244)\n","Processed O2.UC18-2_GL0123767: Neuropeptide (Probabilities: 0.0067, 0.0085, 0.9848)\n","Processed MH0011_GL0089936: Neuropeptide (Probabilities: 0.0245, 0.0135, 0.9620)\n","Processed 159166850-stool1_revised_C1048569_1_gene100957: Neuropeptide (Probabilities: 0.0855, 0.0283, 0.8862)\n","Processed V1.FI19_GL0125587: non-AMP/OTHERS (Probabilities: 0.9873, 0.0097, 0.0030)\n","Processed MH0010_GL0041047: non-AMP/OTHERS (Probabilities: 0.9887, 0.0048, 0.0064)\n","Processed MH0092_GL0101117: non-AMP/OTHERS (Probabilities: 0.9914, 0.0043, 0.0043)\n","Processed MH0011_GL0034505: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0230_GL0070003: non-AMP/OTHERS (Probabilities: 0.9918, 0.0035, 0.0048)\n","Processed O2.UC13-1_GL0000924: Neuropeptide (Probabilities: 0.0538, 0.0217, 0.9245)\n","Processed O2.UC17-2_GL0115696: non-AMP/OTHERS (Probabilities: 0.9917, 0.0035, 0.0048)\n","Processed MH0012_GL0085806: non-AMP/OTHERS (Probabilities: 0.9854, 0.0118, 0.0027)\n","Processed MH0286_GL0046152: non-AMP/OTHERS (Probabilities: 0.9904, 0.0069, 0.0028)\n","Processed MH0001_GL0015029: non-AMP/OTHERS (Probabilities: 0.9912, 0.0058, 0.0030)\n","Processed MH0382_GL0095535: non-AMP/OTHERS (Probabilities: 0.9832, 0.0041, 0.0128)\n","Processed 411485.FAEPRAM212_01761: non-AMP/OTHERS (Probabilities: 0.9620, 0.0077, 0.0303)\n","Processed 411485.FAEPRAM212_01761: non-AMP/OTHERS (Probabilities: 0.9926, 0.0038, 0.0035)\n","Processed MH0212_GL0020739: non-AMP/OTHERS (Probabilities: 0.9919, 0.0038, 0.0043)\n","Processed MH0342_GL0136612: non-AMP/OTHERS (Probabilities: 0.9864, 0.0050, 0.0086)\n","Processed MH0342_GL0136612: non-AMP/OTHERS (Probabilities: 0.9924, 0.0046, 0.0030)\n","Processed V1.FI09_GL0152930: non-AMP/OTHERS (Probabilities: 0.9900, 0.0040, 0.0060)\n","Processed V1.CD20-4_GL0011904: non-AMP/OTHERS (Probabilities: 0.9924, 0.0034, 0.0041)\n","Processed MH0088_GL0071814: non-AMP/OTHERS (Probabilities: 0.9886, 0.0040, 0.0074)\n","Processed MH0406_GL0195445: non-AMP/OTHERS (Probabilities: 0.9589, 0.0213, 0.0198)\n","Processed O2.CD2-0_GL0089367: Neuropeptide (Probabilities: 0.1893, 0.2627, 0.5480)\n","Processed MH0011_GL0046826: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed N003A_GL0033412: Neuropeptide (Probabilities: 0.1463, 0.0281, 0.8255)\n","Processed MH0088_GL0094606: non-AMP/OTHERS (Probabilities: 0.9867, 0.0062, 0.0071)\n","Processed sp|Q12864|CAD17_HUMAN: non-AMP/OTHERS (Probabilities: 0.9915, 0.0052, 0.0034)\n","Processed T2D-176A_GL0032937: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed MH0058_GL0057070: non-AMP/OTHERS (Probabilities: 0.8810, 0.0480, 0.0711)\n","Processed N032A_GL0033225: Neuropeptide (Probabilities: 0.0067, 0.0085, 0.9847)\n","Processed MH0200_GL0181671: non-AMP/OTHERS (Probabilities: 0.9934, 0.0034, 0.0033)\n","Processed MH0203_GL0160944: Neuropeptide (Probabilities: 0.0072, 0.0089, 0.9839)\n","Processed MH0101_GL0039547: Neuropeptide (Probabilities: 0.0066, 0.0086, 0.9848)\n","Processed V1.UC40-1_GL0172302: Neuropeptide (Probabilities: 0.0071, 0.0089, 0.9840)\n","Processed 158337416-stool1_revised_C1288592_1_gene123468: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed MH0012_GL0232564: Neuropeptide (Probabilities: 0.0068, 0.0087, 0.9844)\n","Processed MH0131_GL0129109: non-AMP/OTHERS (Probabilities: 0.9818, 0.0114, 0.0068)\n","Processed MH0432_GL0138977: Neuropeptide (Probabilities: 0.0080, 0.0089, 0.9831)\n","Processed MH0006_GL0024198: non-AMP/OTHERS (Probabilities: 0.9884, 0.0040, 0.0077)\n","Processed MH0088_GL0031463: non-AMP/OTHERS (Probabilities: 0.9905, 0.0041, 0.0055)\n","Processed V1.FI28_GL0011296: Neuropeptide (Probabilities: 0.2098, 0.0501, 0.7401)\n","Processed MH0002_GL0051478: non-AMP/OTHERS (Probabilities: 0.9428, 0.0087, 0.0485)\n","Processed MH0002_GL0030417: non-AMP/OTHERS (Probabilities: 0.7136, 0.0269, 0.2595)\n","Processed 866777.HMPREF9228_0175: non-AMP/OTHERS (Probabilities: 0.9925, 0.0037, 0.0039)\n","Processed MH0086_GL0013846: Neuropeptide (Probabilities: 0.0135, 0.0108, 0.9757)\n","Processed MH0289_GL0078169: Neuropeptide (Probabilities: 0.0171, 0.0127, 0.9702)\n","Processed MH0003_GL0091064: Neuropeptide (Probabilities: 0.0163, 0.0119, 0.9718)\n","Processed MH0008_GL0014566: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9850)\n","Processed DOM018_GL0064623: Neuropeptide (Probabilities: 0.0301, 0.0170, 0.9529)\n","Processed MH0274_GL0054360: Neuropeptide (Probabilities: 0.0100, 0.0098, 0.9803)\n","Processed O2.UC27-0_GL0120410: non-AMP/OTHERS (Probabilities: 0.9776, 0.0090, 0.0134)\n","Processed MH0391_GL0124006: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 159490532-stool1_revised_scaffold8888_2_gene50955: non-AMP/OTHERS (Probabilities: 0.8318, 0.0464, 0.1218)\n","Processed V1.UC13-3_GL0052191: non-AMP/OTHERS (Probabilities: 0.9895, 0.0041, 0.0063)\n","Processed V1.FI13_GL0149995: non-AMP/OTHERS (Probabilities: 0.9889, 0.0046, 0.0066)\n","Processed MH0297_GL0024147: Neuropeptide (Probabilities: 0.0067, 0.0085, 0.9848)\n","Processed V1.FI15_GL0168284: non-AMP/OTHERS (Probabilities: 0.9721, 0.0126, 0.0153)\n","Processed V1.FI15_GL0168284: non-AMP/OTHERS (Probabilities: 0.9732, 0.0131, 0.0137)\n","Processed V1.UC35-4_GL0170425: non-AMP/OTHERS (Probabilities: 0.9775, 0.0141, 0.0084)\n","Processed V1.UC53-0_GL0066150: Neuropeptide (Probabilities: 0.0655, 0.0299, 0.9046)\n","Processed 764447348-stool2_revised_scaffold1306_1_gene116552: non-AMP/OTHERS (Probabilities: 0.9920, 0.0037, 0.0043)\n","Processed MH0188_GL0190994: Neuropeptide (Probabilities: 0.0894, 0.0257, 0.8850)\n","Processed MH0002_GL0040484: Neuropeptide (Probabilities: 0.0101, 0.0104, 0.9794)\n","Processed MH0012_GL0143300: non-AMP/OTHERS (Probabilities: 0.9912, 0.0036, 0.0052)\n","Processed MH0068_GL0046675: Neuropeptide (Probabilities: 0.0073, 0.0086, 0.9842)\n","Processed MH0274_GL0013721: non-AMP/OTHERS (Probabilities: 0.9876, 0.0048, 0.0076)\n","Processed MH0013_GL0012176: Neuropeptide (Probabilities: 0.0186, 0.0123, 0.9691)\n","Processed MH0053_GL0071436: Neuropeptide (Probabilities: 0.0319, 0.0196, 0.9485)\n","Processed MH0003_GL0090598: non-AMP/OTHERS (Probabilities: 0.9730, 0.0068, 0.0202)\n","Processed MH0003_GL0100023: non-AMP/OTHERS (Probabilities: 0.9912, 0.0063, 0.0025)\n","Processed V1.UC61-0_GL0085243: non-AMP/OTHERS (Probabilities: 0.9913, 0.0040, 0.0047)\n","Processed O2.UC47-1_GL0109843: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0251_GL0005451: Neuropeptide (Probabilities: 0.0302, 0.0144, 0.9554)\n","Processed 160704339-stool1_revised_scaffold9618_1_gene27378: non-AMP/OTHERS (Probabilities: 0.8898, 0.1008, 0.0094)\n","Processed V1.FI14_GL0140086: non-AMP/OTHERS (Probabilities: 0.4191, 0.2280, 0.3529)\n","Processed MH0246_GL0129256: Neuropeptide (Probabilities: 0.0073, 0.0087, 0.9841)\n","Processed MH0089_GL0027868: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed MH0006_GL0095533: Neuropeptide (Probabilities: 0.0185, 0.0120, 0.9694)\n","Processed MH0193_GL0018424: non-AMP/OTHERS (Probabilities: 0.9915, 0.0046, 0.0040)\n","Processed MH0188_GL0160346: non-AMP/OTHERS (Probabilities: 0.4493, 0.1215, 0.4292)\n","Processed V1.UC47-4_GL0056608: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed O2.UC15-0_GL0105862: non-AMP/OTHERS (Probabilities: 0.9698, 0.0165, 0.0136)\n","Processed MH0087_GL0012536: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed 763577454-stool2_revised_C1048050_1_gene171791: Neuropeptide (Probabilities: 0.0068, 0.0086, 0.9845)\n","Processed MH0375_GL0062328: non-AMP/OTHERS (Probabilities: 0.9932, 0.0035, 0.0033)\n","Processed DOM005_GL0011016: Neuropeptide (Probabilities: 0.0529, 0.0221, 0.9250)\n","Processed 469592.HMPREF0104_03148: non-AMP/OTHERS (Probabilities: 0.9910, 0.0051, 0.0039)\n","Processed MH0422_GL0006644: non-AMP/OTHERS (Probabilities: 0.9932, 0.0032, 0.0036)\n","Processed MH0204_GL0156652: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0041)\n","Processed O2.UC32-0_GL0085839: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0036)\n","Processed N005A_GL0071976: non-AMP/OTHERS (Probabilities: 0.9847, 0.0065, 0.0089)\n","Processed O2.UC32-1_GL0072478: non-AMP/OTHERS (Probabilities: 0.7053, 0.0330, 0.2617)\n","Processed MH0150_GL0139170: non-AMP/OTHERS (Probabilities: 0.9876, 0.0036, 0.0088)\n","Processed MH0407_GL0078163: non-AMP/OTHERS (Probabilities: 0.9770, 0.0116, 0.0113)\n","Processed MH0086_GL0119127: non-AMP/OTHERS (Probabilities: 0.9835, 0.0069, 0.0096)\n","Processed MH0441_GL0137342: Neuropeptide (Probabilities: 0.0071, 0.0087, 0.9842)\n","Processed MH0199_GL0146261: non-AMP/OTHERS (Probabilities: 0.9666, 0.0135, 0.0199)\n","Processed 1075106.BLC1_0532: non-AMP/OTHERS (Probabilities: 0.9930, 0.0033, 0.0037)\n","Processed O2.UC7-1_GL0012366: Neuropeptide (Probabilities: 0.0081, 0.0096, 0.9823)\n","Processed MH0041_GL0025906: non-AMP/OTHERS (Probabilities: 0.9697, 0.0141, 0.0162)\n","Processed MH0239_GL0126780: non-AMP/OTHERS (Probabilities: 0.9587, 0.0300, 0.0114)\n","Processed MH0239_GL0126780: non-AMP/OTHERS (Probabilities: 0.7025, 0.2125, 0.0850)\n","Processed T2D-123A_GL0010094: Neuropeptide (Probabilities: 0.0069, 0.0086, 0.9845)\n","Processed MH0153_GL0109757: Neuropeptide (Probabilities: 0.0106, 0.0104, 0.9790)\n","Processed MH0052_GL0032291: Neuropeptide (Probabilities: 0.1900, 0.0299, 0.7801)\n","Processed MH0002_GL0073573: Neuropeptide (Probabilities: 0.2942, 0.1075, 0.5983)\n","Processed MH0002_GL0016290: non-AMP/OTHERS (Probabilities: 0.9605, 0.0198, 0.0197)\n","Processed MH0050_GL0064982: non-AMP/OTHERS (Probabilities: 0.9597, 0.0166, 0.0237)\n","Processed MH0188_GL0079775: non-AMP/OTHERS (Probabilities: 0.9886, 0.0047, 0.0066)\n","Processed MH0239_GL0157962: Neuropeptide (Probabilities: 0.2309, 0.1465, 0.6227)\n","Processed O2.UC28-2_GL0179102: non-AMP/OTHERS (Probabilities: 0.9921, 0.0044, 0.0035)\n","Processed MH0147_GL0029411: Neuropeptide (Probabilities: 0.0084, 0.0095, 0.9821)\n","Processed MH0188_GL0198465: non-AMP/OTHERS (Probabilities: 0.9766, 0.0139, 0.0095)\n","Processed MH0002_GL0005122: non-AMP/OTHERS (Probabilities: 0.9929, 0.0033, 0.0039)\n","Processed MH0007_GL0000262: Neuropeptide (Probabilities: 0.0797, 0.0340, 0.8863)\n","Processed O2.UC22-1_GL0071212: non-AMP/OTHERS (Probabilities: 0.9925, 0.0039, 0.0036)\n","Processed MH0011_GL0019972: non-AMP/OTHERS (Probabilities: 0.9614, 0.0119, 0.0267)\n","Processed V1.FI28_GL0007248: non-AMP/OTHERS (Probabilities: 0.9799, 0.0058, 0.0143)\n","Processed V1.FI01_GL0053459: non-AMP/OTHERS (Probabilities: 0.5573, 0.1002, 0.3425)\n","Processed MH0330_GL0213978: Neuropeptide (Probabilities: 0.4024, 0.1457, 0.4519)\n","Processed MH0330_GL0213978: Neuropeptide (Probabilities: 0.0324, 0.0179, 0.9497)\n","Processed 823052294-stool1_revised_C730961_1_gene40926: non-AMP/OTHERS (Probabilities: 0.9885, 0.0070, 0.0045)\n","Processed V1.FI02_GL0084000: Neuropeptide (Probabilities: 0.1435, 0.0271, 0.8295)\n","Processed MH0152_GL0052535: Neuropeptide (Probabilities: 0.4045, 0.0261, 0.5695)\n","Processed MH0152_GL0052535: non-AMP/OTHERS (Probabilities: 0.6243, 0.0324, 0.3433)\n","Processed MH0007_GL0016509: Neuropeptide (Probabilities: 0.0076, 0.0090, 0.9834)\n","Processed MH0277_GL0037031: non-AMP/OTHERS (Probabilities: 0.9913, 0.0044, 0.0043)\n","Processed MH0012_GL0070453: non-AMP/OTHERS (Probabilities: 0.9772, 0.0173, 0.0055)\n","Processed 367928.BAD_0287: Neuropeptide (Probabilities: 0.0075, 0.0088, 0.9837)\n","Processed MH0012_GL0001575: non-AMP/OTHERS (Probabilities: 0.5322, 0.0231, 0.4446)\n","Processed O2.UC37-1_GL0049895: non-AMP/OTHERS (Probabilities: 0.9915, 0.0032, 0.0053)\n","Processed MH0003_GL0104512: non-AMP/OTHERS (Probabilities: 0.9857, 0.0059, 0.0085)\n","Processed DLM007_GL0059779: Neuropeptide (Probabilities: 0.0094, 0.0095, 0.9811)\n","Processed DLM007_GL0059779: non-AMP/OTHERS (Probabilities: 0.9479, 0.0183, 0.0337)\n","Processed MH0188_GL0092730: Neuropeptide (Probabilities: 0.0651, 0.0301, 0.9048)\n","Processed MH0014_GL0106383: non-AMP/OTHERS (Probabilities: 0.9840, 0.0079, 0.0081)\n","Processed MH0131_GL0070884: Neuropeptide (Probabilities: 0.0087, 0.0090, 0.9824)\n","Processed MH0011_GL0103472: non-AMP/OTHERS (Probabilities: 0.9920, 0.0030, 0.0050)\n","Processed MH0104_GL0035273: Neuropeptide (Probabilities: 0.0068, 0.0086, 0.9846)\n","Processed O2.UC1-2_GL0056841: Neuropeptide (Probabilities: 0.0093, 0.0104, 0.9804)\n","Processed MH0141_GL0105124: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9849)\n","Processed DOF014_GL0033006: Neuropeptide (Probabilities: 0.0063, 0.0087, 0.9851)\n","Processed O2.UC23-2_GL0008714: Neuropeptide (Probabilities: 0.0146, 0.0148, 0.9706)\n","Processed MH0203_GL0145867: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed V1.CD1-3-PN_GL0130101: non-AMP/OTHERS (Probabilities: 0.7704, 0.0459, 0.1837)\n","Processed V1.CD1-3-PN_GL0130101: non-AMP/OTHERS (Probabilities: 0.9918, 0.0043, 0.0039)\n","Processed O2.CD1-0-PT_GL0079916: Neuropeptide (Probabilities: 0.0081, 0.0104, 0.9815)\n","Processed 702446.CUU_1118: non-AMP/OTHERS (Probabilities: 0.9876, 0.0065, 0.0058)\n","Processed V1.FI05_GL0111842: Neuropeptide (Probabilities: 0.0070, 0.0087, 0.9843)\n","Processed V1.FI05_GL0111842: Neuropeptide (Probabilities: 0.0836, 0.0239, 0.8924)\n","Processed MH0161_GL0001233: Neuropeptide (Probabilities: 0.0081, 0.0089, 0.9830)\n","Processed V1.FI17_GL0225567: Neuropeptide (Probabilities: 0.0252, 0.0132, 0.9616)\n","Processed MH0341_GL0028604: Neuropeptide (Probabilities: 0.0231, 0.0201, 0.9568)\n","Processed O2.UC23-2_GL0066537: Neuropeptide (Probabilities: 0.0497, 0.0258, 0.9245)\n","Processed MH0131_GL0152022: non-AMP/OTHERS (Probabilities: 0.9738, 0.0121, 0.0141)\n","Processed T2D-114A_GL0095369: non-AMP/OTHERS (Probabilities: 0.9930, 0.0043, 0.0028)\n","Processed MH0131_GL0160071: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed T2D-110A_GL0105184: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0054_GL0014211: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0371_GL0099531: Neuropeptide (Probabilities: 0.0476, 0.0161, 0.9364)\n","Processed DLF012_GL0011816: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed V1.CD51-0_GL0006732: non-AMP/OTHERS (Probabilities: 0.9897, 0.0035, 0.0068)\n","Processed V1.FI09_GL0160283: non-AMP/OTHERS (Probabilities: 0.9819, 0.0153, 0.0028)\n","Processed V1.CD16-0_GL0010188: non-AMP/OTHERS (Probabilities: 0.9785, 0.0173, 0.0041)\n","Processed O2.UC7-0_GL0126702: non-AMP/OTHERS (Probabilities: 0.9900, 0.0068, 0.0032)\n","Processed 553973.CLOHYLEM_06529: Neuropeptide (Probabilities: 0.0062, 0.0088, 0.9850)\n","Processed V1.UC3-0_GL0073355: non-AMP/OTHERS (Probabilities: 0.9924, 0.0043, 0.0033)\n","Processed MH0143_GL0056429: non-AMP/OTHERS (Probabilities: 0.9916, 0.0046, 0.0038)\n","Processed 160178356-stool1_revised_scaffold11920_1_gene76369: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0252_GL0143897: non-AMP/OTHERS (Probabilities: 0.6837, 0.2872, 0.0291)\n","Processed MH0203_GL0209321: Neuropeptide (Probabilities: 0.0081, 0.0092, 0.9826)\n","Processed MH0412_GL0146022: non-AMP/OTHERS (Probabilities: 0.4770, 0.0694, 0.4536)\n","Processed MH0462_GL0178179: non-AMP/OTHERS (Probabilities: 0.4429, 0.3012, 0.2559)\n","Processed MH0318_GL0131092: non-AMP/OTHERS (Probabilities: 0.9891, 0.0058, 0.0051)\n","Processed 763820215-stool1_revised_C756724_1_gene51486: non-AMP/OTHERS (Probabilities: 0.9922, 0.0048, 0.0030)\n","Processed MH0399_GL0088310: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed T2D-123A_GL0080108: non-AMP/OTHERS (Probabilities: 0.9726, 0.0135, 0.0138)\n","Processed V1.UC39-4_GL0065830: non-AMP/OTHERS (Probabilities: 0.8813, 0.0238, 0.0949)\n","Processed MH0083_GL0091317: non-AMP/OTHERS (Probabilities: 0.9689, 0.0072, 0.0239)\n","Processed MH0147_GL0090911: non-AMP/OTHERS (Probabilities: 0.6269, 0.0294, 0.3438)\n","Processed MH0319_GL0147619: non-AMP/OTHERS (Probabilities: 0.9288, 0.0205, 0.0507)\n","Processed DLF013_GL0061173: non-AMP/OTHERS (Probabilities: 0.9928, 0.0042, 0.0030)\n","Processed MH0342_GL0130397: non-AMP/OTHERS (Probabilities: 0.9550, 0.0326, 0.0124)\n","Processed MH0345_GL0165132: non-AMP/OTHERS (Probabilities: 0.9883, 0.0076, 0.0041)\n","Processed O2.UC4-1_GL0193399: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0196_GL0082268: Neuropeptide (Probabilities: 0.0193, 0.0127, 0.9681)\n","Processed MH0012_GL0029606: Neuropeptide (Probabilities: 0.0126, 0.0116, 0.9758)\n","Processed MH0049_GL0018494: non-AMP/OTHERS (Probabilities: 0.9877, 0.0073, 0.0050)\n","Processed MH0088_GL0109611: Neuropeptide (Probabilities: 0.0075, 0.0087, 0.9839)\n","Processed MH0023_GL0023745: non-AMP/OTHERS (Probabilities: 0.9930, 0.0033, 0.0037)\n","Processed O2.UC22-1_GL0006998: Neuropeptide (Probabilities: 0.0093, 0.0101, 0.9806)\n","Processed MH0186_GL0075458: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0002_GL0067069: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9852)\n","Processed O2.UC40-1_GL0003434: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0040)\n","Processed 206672.BL1811: non-AMP/OTHERS (Probabilities: 0.9910, 0.0047, 0.0043)\n","Processed O2.UC40-0_GL0044906: non-AMP/OTHERS (Probabilities: 0.9880, 0.0038, 0.0081)\n","Processed O2.UC17-1_GL0099902: non-AMP/OTHERS (Probabilities: 0.9910, 0.0033, 0.0057)\n","Processed MH0003_GL0003181: non-AMP/OTHERS (Probabilities: 0.9338, 0.0170, 0.0492)\n","Processed NLM023_GL0004314: non-AMP/OTHERS (Probabilities: 0.9919, 0.0040, 0.0041)\n","Processed V1.CD8-0-PN_GL0097357: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed HT8A_GL0053911: AMP (Probabilities: 0.0760, 0.9109, 0.0131)\n","Processed MH0006_GL0124546: non-AMP/OTHERS (Probabilities: 0.9932, 0.0037, 0.0031)\n","Processed MH0188_GL0026540: non-AMP/OTHERS (Probabilities: 0.9867, 0.0040, 0.0093)\n","Processed MH0316_GL0183256: Neuropeptide (Probabilities: 0.1771, 0.0693, 0.7535)\n","Processed T2D-123A_GL0082886: non-AMP/OTHERS (Probabilities: 0.9900, 0.0066, 0.0033)\n","Processed MH0414_GL0067767: non-AMP/OTHERS (Probabilities: 0.9921, 0.0038, 0.0041)\n","Processed MH0325_GL0059915: non-AMP/OTHERS (Probabilities: 0.9914, 0.0039, 0.0047)\n","Processed MH0119_GL0033639: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0188_GL0146567: non-AMP/OTHERS (Probabilities: 0.9898, 0.0057, 0.0044)\n","Processed MH0012_GL0050906: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed 809635352-stool1_revised_scaffold26288_1_gene152999: non-AMP/OTHERS (Probabilities: 0.9905, 0.0039, 0.0056)\n","Processed DOF004_GL0018363: non-AMP/OTHERS (Probabilities: 0.9916, 0.0035, 0.0049)\n","Processed MH0049_GL0013601: non-AMP/OTHERS (Probabilities: 0.9922, 0.0030, 0.0048)\n","Processed 469592.HMPREF0104_03148: Neuropeptide (Probabilities: 0.0193, 0.0156, 0.9652)\n","Processed DOM005_GL0011016: non-AMP/OTHERS (Probabilities: 0.9904, 0.0029, 0.0067)\n","Processed 469591.HMPREF9008_04396: non-AMP/OTHERS (Probabilities: 0.9379, 0.0109, 0.0512)\n","Processed MH0358_GL0059698: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0035)\n","Processed O2.UC47-1_GL0090469: Neuropeptide (Probabilities: 0.2160, 0.0657, 0.7183)\n","Processed MH0318_GL0131092: non-AMP/OTHERS (Probabilities: 0.9907, 0.0053, 0.0040)\n","Processed MH0147_GL0127639: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0188_GL0154167: non-AMP/OTHERS (Probabilities: 0.8623, 0.1114, 0.0263)\n","Processed MH0002_GL0075547: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0002_GL0075547: non-AMP/OTHERS (Probabilities: 0.8964, 0.0284, 0.0752)\n","Processed MH0090_GL0071346: non-AMP/OTHERS (Probabilities: 0.9929, 0.0030, 0.0041)\n","Processed MH0145_GL0011423: non-AMP/OTHERS (Probabilities: 0.9881, 0.0032, 0.0087)\n","Processed DOF008_GL0032512: non-AMP/OTHERS (Probabilities: 0.9911, 0.0048, 0.0040)\n","Processed V1.FI05_GL0137010: non-AMP/OTHERS (Probabilities: 0.9924, 0.0036, 0.0040)\n","Processed O2.UC15-0_GL0077925: non-AMP/OTHERS (Probabilities: 0.8157, 0.1226, 0.0617)\n","Processed MH0364_GL0018036: Neuropeptide (Probabilities: 0.0093, 0.0108, 0.9799)\n","Processed MH0453_GL0164777: non-AMP/OTHERS (Probabilities: 0.9923, 0.0035, 0.0042)\n","Processed MH0161_GL0061147: non-AMP/OTHERS (Probabilities: 0.9914, 0.0039, 0.0048)\n","Processed MH0122_GL0073303: non-AMP/OTHERS (Probabilities: 0.9862, 0.0039, 0.0099)\n","Processed O2.UC44-2_GL0054699: non-AMP/OTHERS (Probabilities: 0.9890, 0.0036, 0.0075)\n","Processed MH0088_GL0058291: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9849)\n","Processed MH0292_GL0113922: non-AMP/OTHERS (Probabilities: 0.5691, 0.0358, 0.3951)\n","Processed O2.CD2-0-PT_GL0004300: non-AMP/OTHERS (Probabilities: 0.8795, 0.1098, 0.0106)\n","Processed O2.UC28-1_GL0000577: non-AMP/OTHERS (Probabilities: 0.9902, 0.0038, 0.0060)\n","Processed MH0104_GL0054429: AMP (Probabilities: 0.1024, 0.8836, 0.0140)\n","Processed N022A_GL0021154: non-AMP/OTHERS (Probabilities: 0.9928, 0.0031, 0.0041)\n","Processed MH0322_GL0157766: non-AMP/OTHERS (Probabilities: 0.9905, 0.0035, 0.0060)\n","Processed V1.FI29_GL0039996: non-AMP/OTHERS (Probabilities: 0.9877, 0.0075, 0.0048)\n","Processed ED50A_GL0047405: non-AMP/OTHERS (Probabilities: 0.9897, 0.0061, 0.0042)\n","Processed MH0028_GL0007732: Neuropeptide (Probabilities: 0.0140, 0.0117, 0.9743)\n","Processed 159166850-stool1_revised_scaffold2783_1_gene25708: non-AMP/OTHERS (Probabilities: 0.9915, 0.0033, 0.0052)\n","Processed DLM012_GL0048730: non-AMP/OTHERS (Probabilities: 0.9868, 0.0049, 0.0083)\n","Processed 299768.str0334: non-AMP/OTHERS (Probabilities: 0.9927, 0.0035, 0.0038)\n","Processed 160765029-stool1_revised_C1040480_1_gene46665: non-AMP/OTHERS (Probabilities: 0.9859, 0.0049, 0.0092)\n","Processed MH0079_GL0015201: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0244_GL0053181: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed MH0244_GL0022860: non-AMP/OTHERS (Probabilities: 0.9920, 0.0048, 0.0032)\n","Processed O2.UC40-1_GL0003434: non-AMP/OTHERS (Probabilities: 0.9903, 0.0051, 0.0045)\n","Processed O2.UC17-0_GL0002897: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed MH0078_GL0004582: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0244_GL0072054: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed DOM018_GL0031247: Neuropeptide (Probabilities: 0.0942, 0.0425, 0.8633)\n","Processed MH0198_GL0076209: non-AMP/OTHERS (Probabilities: 0.9904, 0.0036, 0.0060)\n","Processed MH0161_GL0015504: non-AMP/OTHERS (Probabilities: 0.9863, 0.0071, 0.0066)\n","Processed MH0010_GL0010650: non-AMP/OTHERS (Probabilities: 0.9916, 0.0048, 0.0036)\n","Processed MH0062_GL0036371: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed V1.CD25-0_GL0023251: non-AMP/OTHERS (Probabilities: 0.9872, 0.0042, 0.0086)\n","Processed MH0293_GL0066180: non-AMP/OTHERS (Probabilities: 0.9856, 0.0046, 0.0098)\n","Processed MH0131_GL0101824: non-AMP/OTHERS (Probabilities: 0.9711, 0.0114, 0.0175)\n","Processed O2.UC9-0_GL0075145: non-AMP/OTHERS (Probabilities: 0.9417, 0.0201, 0.0381)\n","Processed sp|Q93050|VPP1_HUMAN: non-AMP/OTHERS (Probabilities: 0.9913, 0.0044, 0.0043)\n","Processed MH0293_GL0116698: Neuropeptide (Probabilities: 0.3218, 0.0456, 0.6327)\n","Processed MH0141_GL0092652: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0002_GL0071847: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed V1.FI07_GL0070055: non-AMP/OTHERS (Probabilities: 0.9869, 0.0036, 0.0095)\n","Processed MH0211_GL0143356: non-AMP/OTHERS (Probabilities: 0.9593, 0.0225, 0.0182)\n","Processed 159268001-stool1_revised_C1106135_1_gene4442: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9851)\n","Processed MH0028_GL0068382: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed SZEY-06A_GL0068002: Neuropeptide (Probabilities: 0.0149, 0.0126, 0.9725)\n","Processed MH0088_GL0104916: non-AMP/OTHERS (Probabilities: 0.9851, 0.0089, 0.0061)\n","Processed MH0188_GL0111314: non-AMP/OTHERS (Probabilities: 0.9730, 0.0235, 0.0035)\n","Processed MH0191_GL0107421: Neuropeptide (Probabilities: 0.0074, 0.0095, 0.9831)\n","Processed BGI-27A_GL0101813: non-AMP/OTHERS (Probabilities: 0.9911, 0.0029, 0.0059)\n","Processed O2.UC48-0_GL0175158: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0039)\n","Processed MH0012_GL0071082: Neuropeptide (Probabilities: 0.0219, 0.0132, 0.9648)\n","Processed MH0119_GL0081440: Neuropeptide (Probabilities: 0.0069, 0.0085, 0.9846)\n","Processed MH0203_GL0062131: Neuropeptide (Probabilities: 0.0072, 0.0085, 0.9843)\n","Processed MH0088_GL0092590: Neuropeptide (Probabilities: 0.0288, 0.0182, 0.9531)\n","Processed DOM022_GL0024801: Neuropeptide (Probabilities: 0.2753, 0.0259, 0.6989)\n","Processed MH0006_GL0090216: Neuropeptide (Probabilities: 0.0066, 0.0087, 0.9847)\n","Processed MH0161_GL0120502: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0330_GL0195205: AMP (Probabilities: 0.1280, 0.8602, 0.0118)\n","Processed MH0397_GL0158981: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0035)\n","Processed N046A_GL0001049: Neuropeptide (Probabilities: 0.0381, 0.0153, 0.9466)\n","Processed MH0006_GL0061052: non-AMP/OTHERS (Probabilities: 0.9578, 0.0204, 0.0218)\n","Processed 823052294-stool1_revised_C729179_1_gene30830: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed MH0006_GL0003713: non-AMP/OTHERS (Probabilities: 0.9925, 0.0038, 0.0038)\n","Processed MH0318_GL0234144: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9851)\n","Processed MH0318_GL0234144: non-AMP/OTHERS (Probabilities: 0.9912, 0.0038, 0.0050)\n","Processed MH0003_GL0016745: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed MH0088_GL0110736: non-AMP/OTHERS (Probabilities: 0.9707, 0.0085, 0.0207)\n","Processed MH0188_GL0042823: non-AMP/OTHERS (Probabilities: 0.9618, 0.0103, 0.0279)\n","Processed O2.CD1-0-PT_GL0070216: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0039)\n","Processed 246515023-stool2_revised_scaffold7571_2_gene35472: non-AMP/OTHERS (Probabilities: 0.9900, 0.0042, 0.0059)\n","Processed O2.UC15-0_GL0095744: non-AMP/OTHERS (Probabilities: 0.9856, 0.0056, 0.0088)\n","Processed MH0127_GL0004387: non-AMP/OTHERS (Probabilities: 0.9915, 0.0038, 0.0046)\n","Processed 764669880-stool1_revised_C419629_1_gene48193: non-AMP/OTHERS (Probabilities: 0.9922, 0.0029, 0.0049)\n","Processed MH0188_GL0042823: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed MH0006_GL0056289: non-AMP/OTHERS (Probabilities: 0.9924, 0.0031, 0.0045)\n","Processed MH0197_GL0154481: non-AMP/OTHERS (Probabilities: 0.9911, 0.0033, 0.0055)\n","Processed T2D-62A_GL0043407: non-AMP/OTHERS (Probabilities: 0.9892, 0.0038, 0.0070)\n","Processed O2.UC38-0_GL0039189: non-AMP/OTHERS (Probabilities: 0.9916, 0.0033, 0.0051)\n","Processed 404239096-stool1_revised_scaffold26117_2_gene23128: non-AMP/OTHERS (Probabilities: 0.5093, 0.0211, 0.4696)\n","Processed MH0458_GL0142660: non-AMP/OTHERS (Probabilities: 0.9918, 0.0052, 0.0029)\n","Processed MH0010_GL0021286: non-AMP/OTHERS (Probabilities: 0.9912, 0.0035, 0.0053)\n","Processed MH0110_GL0047810: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0115_GL0111825: Neuropeptide (Probabilities: 0.0228, 0.0142, 0.9629)\n","Processed MH0115_GL0111825: non-AMP/OTHERS (Probabilities: 0.9890, 0.0063, 0.0048)\n","Processed 1042403.BALAC2494_00614: non-AMP/OTHERS (Probabilities: 0.9922, 0.0048, 0.0031)\n","Processed O2.UC4-1_GL0166577: Neuropeptide (Probabilities: 0.0066, 0.0087, 0.9846)\n","Processed V1.CD53-0_GL0038153: non-AMP/OTHERS (Probabilities: 0.9665, 0.0057, 0.0278)\n","Processed MH0211_GL0012457: non-AMP/OTHERS (Probabilities: 0.9927, 0.0037, 0.0036)\n","Processed MH0381_GL0058712: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0034)\n","Processed V1.UC21-0_GL0053085: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed MH0373_GL0023905: non-AMP/OTHERS (Probabilities: 0.9909, 0.0057, 0.0034)\n","Processed O2.UC9-1_GL0099020: non-AMP/OTHERS (Probabilities: 0.9901, 0.0062, 0.0036)\n","Processed MH0071_GL0056232: non-AMP/OTHERS (Probabilities: 0.9929, 0.0040, 0.0031)\n","Processed MH0396_GL0047332: non-AMP/OTHERS (Probabilities: 0.9930, 0.0041, 0.0029)\n","Processed NLF007_GL0099807: non-AMP/OTHERS (Probabilities: 0.9844, 0.0114, 0.0042)\n","Processed O2.UC28-1_GL0062794: non-AMP/OTHERS (Probabilities: 0.9914, 0.0047, 0.0040)\n","Processed MH0375_GL0026277: non-AMP/OTHERS (Probabilities: 0.9770, 0.0159, 0.0072)\n","Processed MH0002_GL0001425: non-AMP/OTHERS (Probabilities: 0.9926, 0.0043, 0.0031)\n","Processed MH0441_GL0101761: Neuropeptide (Probabilities: 0.0118, 0.0106, 0.9777)\n","Processed V1.FI12_GL0198636: non-AMP/OTHERS (Probabilities: 0.9912, 0.0041, 0.0048)\n","Processed O2.UC7-0_GL0071226: Neuropeptide (Probabilities: 0.3933, 0.0364, 0.5703)\n","Processed 556261.HMPREF0240_00328: non-AMP/OTHERS (Probabilities: 0.9800, 0.0167, 0.0033)\n","Processed MH0198_GL0180812: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed DOF004_GL0053162: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0034)\n","Processed V1.FI36_GL0177470: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed V1.CD8-0-PN_GL0091132: Neuropeptide (Probabilities: 0.0462, 0.0267, 0.9272)\n","Processed MH0251_GL0055910: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed V1.UC53-0_GL0060838: Neuropeptide (Probabilities: 0.0070, 0.0085, 0.9845)\n","Processed O2.UC35-2_GL0104332: Neuropeptide (Probabilities: 0.0075, 0.0085, 0.9841)\n","Processed T2D-114A_GL0002973: non-AMP/OTHERS (Probabilities: 0.9918, 0.0037, 0.0045)\n","Processed V1.UC60-0_GL0052142: Neuropeptide (Probabilities: 0.0068, 0.0086, 0.9846)\n","Processed T2D-112A_GL0030632: non-AMP/OTHERS (Probabilities: 0.9895, 0.0079, 0.0026)\n","Processed O2.UC52-2_GL0027082: non-AMP/OTHERS (Probabilities: 0.9917, 0.0033, 0.0051)\n","Processed MH0407_GL0121504: non-AMP/OTHERS (Probabilities: 0.9934, 0.0031, 0.0035)\n","Processed MH0057_GL0041917: non-AMP/OTHERS (Probabilities: 0.6301, 0.1185, 0.2514)\n","Processed O2.CD1-0_GL0100549: Neuropeptide (Probabilities: 0.0796, 0.0223, 0.8981)\n","Processed 264199.stu0194: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed 264199.stu0194: non-AMP/OTHERS (Probabilities: 0.9584, 0.0072, 0.0345)\n","Processed MH0368_GL0085981: non-AMP/OTHERS (Probabilities: 0.9920, 0.0032, 0.0048)\n","Processed O2.UC40-1_GL0125568: non-AMP/OTHERS (Probabilities: 0.8839, 0.0274, 0.0888)\n","Processed V1.FI25_GL0057739: non-AMP/OTHERS (Probabilities: 0.9212, 0.0585, 0.0203)\n","Processed O2.UC14-2_GL0061213: non-AMP/OTHERS (Probabilities: 0.8956, 0.0771, 0.0272)\n","Processed MH0227_GL0003913: Neuropeptide (Probabilities: 0.0687, 0.0302, 0.9010)\n","Processed DLM023_GL0069622: Neuropeptide (Probabilities: 0.0087, 0.0093, 0.9820)\n","Processed V1.FI37_GL0141216: Neuropeptide (Probabilities: 0.0131, 0.0109, 0.9761)\n","Processed O2.UC49-0_GL0010552: non-AMP/OTHERS (Probabilities: 0.9911, 0.0038, 0.0052)\n","Processed MH0037_GL0050217: non-AMP/OTHERS (Probabilities: 0.9922, 0.0043, 0.0034)\n","Processed T2D-122A_GL0015238: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed 367928.BAD_1620: Neuropeptide (Probabilities: 0.0078, 0.0093, 0.9828)\n","Processed MH0363_GL0078871: AMP (Probabilities: 0.2036, 0.7393, 0.0571)\n","Processed MH0280_GL0137988: Neuropeptide (Probabilities: 0.0152, 0.0131, 0.9717)\n","Processed O2.UC50-2_GL0137833: non-AMP/OTHERS (Probabilities: 0.5192, 0.0647, 0.4161)\n","Processed MH0188_GL0145135: non-AMP/OTHERS (Probabilities: 0.9438, 0.0150, 0.0412)\n","Processed O2.CD2-0-PT_GL0062968: Neuropeptide (Probabilities: 0.0759, 0.0267, 0.8974)\n","Processed MH0197_GL0069925: Neuropeptide (Probabilities: 0.0979, 0.0303, 0.8718)\n","Processed MH0277_GL0010184: non-AMP/OTHERS (Probabilities: 0.9925, 0.0038, 0.0036)\n","Processed O2.UC17-1_GL0125990: non-AMP/OTHERS (Probabilities: 0.9893, 0.0062, 0.0045)\n","Processed MH0277_GL0019858: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0131_GL0173462: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0122_GL0041725: non-AMP/OTHERS (Probabilities: 0.9926, 0.0034, 0.0040)\n","Processed O2.UC53-0_GL0177576: non-AMP/OTHERS (Probabilities: 0.9873, 0.0049, 0.0077)\n","Processed MH0188_GL0045399: non-AMP/OTHERS (Probabilities: 0.9929, 0.0031, 0.0040)\n","Processed MH0338_GL0040878: non-AMP/OTHERS (Probabilities: 0.9888, 0.0043, 0.0069)\n","Processed 1048332.SALIVB_0571: Neuropeptide (Probabilities: 0.0104, 0.0142, 0.9754)\n","Processed 763820215-stool2_revised_scaffold1780_4_gene9731: Neuropeptide (Probabilities: 0.0359, 0.0201, 0.9441)\n","Processed MH0397_GL0186038: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0370_GL0042621: non-AMP/OTHERS (Probabilities: 0.9745, 0.0087, 0.0168)\n","Processed MH0002_GL0032782: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed MH0457_GL0198515: Neuropeptide (Probabilities: 0.4798, 0.0258, 0.4944)\n","Processed N039A_GL0072487: non-AMP/OTHERS (Probabilities: 0.9907, 0.0036, 0.0057)\n","Processed N052A_GL0003037: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0131_GL0107430: Neuropeptide (Probabilities: 0.0978, 0.0256, 0.8766)\n","Processed DOF014_GL0051135: non-AMP/OTHERS (Probabilities: 0.9928, 0.0044, 0.0028)\n","Processed MH0277_GL0013194: Neuropeptide (Probabilities: 0.0214, 0.0147, 0.9640)\n","Processed O2.UC11-2_GL0010505: non-AMP/OTHERS (Probabilities: 0.9931, 0.0029, 0.0040)\n","Processed MH0318_GL0059809: Neuropeptide (Probabilities: 0.0083, 0.0087, 0.9830)\n","Processed 764325968-stool1_revised_C587037_1_gene57487: non-AMP/OTHERS (Probabilities: 0.6967, 0.0266, 0.2767)\n","Processed MH0329_GL0031185: Neuropeptide (Probabilities: 0.0079, 0.0091, 0.9830)\n","Processed DOF004_GL0055269: non-AMP/OTHERS (Probabilities: 0.9907, 0.0058, 0.0035)\n","Processed V1.FI18_GL0001371: non-AMP/OTHERS (Probabilities: 0.9869, 0.0039, 0.0092)\n","Processed MH0065_GL0020891: Neuropeptide (Probabilities: 0.0109, 0.0106, 0.9784)\n","Processed V1.UC13-3_GL0014892: non-AMP/OTHERS (Probabilities: 0.9318, 0.0091, 0.0591)\n","Processed MH0001_GL0001615: non-AMP/OTHERS (Probabilities: 0.9903, 0.0040, 0.0057)\n","Processed MH0001_GL0001615: non-AMP/OTHERS (Probabilities: 0.5979, 0.0431, 0.3590)\n","Processed 159551223-stool2_revised_scaffold46565_2_gene70158: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed V1.CD12-0_GL0044687: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed V1.CD1-3-PN_GL0114759: non-AMP/OTHERS (Probabilities: 0.8999, 0.0441, 0.0560)\n","Processed MH0013_GL0019938: non-AMP/OTHERS (Probabilities: 0.9899, 0.0051, 0.0050)\n","Processed V1.FI04_GL0078901: Neuropeptide (Probabilities: 0.0096, 0.0099, 0.9805)\n","Processed V1.FI35_GL0114916: non-AMP/OTHERS (Probabilities: 0.9906, 0.0039, 0.0054)\n","Processed MH0041_GL0063176: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9852)\n","Processed MH0293_GL0045633: non-AMP/OTHERS (Probabilities: 0.9933, 0.0031, 0.0036)\n","Processed MH0244_GL0022860: Neuropeptide (Probabilities: 0.2497, 0.0302, 0.7201)\n","Processed MH0147_GL0006920: non-AMP/OTHERS (Probabilities: 0.9844, 0.0043, 0.0113)\n","Processed BGI002A_GL0060090: non-AMP/OTHERS (Probabilities: 0.9914, 0.0043, 0.0042)\n","Processed NLM021_GL0109942: non-AMP/OTHERS (Probabilities: 0.9868, 0.0068, 0.0065)\n","Processed MH0230_GL0080024: non-AMP/OTHERS (Probabilities: 0.8371, 0.1333, 0.0296)\n","Processed O2.UC36-2_GL0008526: Neuropeptide (Probabilities: 0.0682, 0.0211, 0.9107)\n","Processed V1.CD20-4_GL0118363: non-AMP/OTHERS (Probabilities: 0.8399, 0.1210, 0.0391)\n","Processed V1.FI13_GL0044190: non-AMP/OTHERS (Probabilities: 0.9847, 0.0069, 0.0084)\n","Processed V1.UC41-0_GL0045739: non-AMP/OTHERS (Probabilities: 0.9887, 0.0040, 0.0072)\n","Processed MH0192_GL0015721: non-AMP/OTHERS (Probabilities: 0.9904, 0.0044, 0.0052)\n","Processed O2.UC56-0_GL0035626: non-AMP/OTHERS (Probabilities: 0.9896, 0.0055, 0.0049)\n","Processed MH0188_GL0148624: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed 159753524-stool1_revised_scaffold13416_1_gene37586: Neuropeptide (Probabilities: 0.3050, 0.0522, 0.6428)\n","Processed DOM019_GL0096212: Neuropeptide (Probabilities: 0.0076, 0.0089, 0.9836)\n","Processed MH0446_GL0097533: non-AMP/OTHERS (Probabilities: 0.7511, 0.1428, 0.1061)\n","Processed MH0370_GL0065902: non-AMP/OTHERS (Probabilities: 0.9405, 0.0090, 0.0505)\n","Processed MH0188_GL0053529: non-AMP/OTHERS (Probabilities: 0.9803, 0.0084, 0.0114)\n","Processed V1.CD6-0-PT_GL0007182: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0124_GL0076377: Neuropeptide (Probabilities: 0.0065, 0.0085, 0.9850)\n","Processed MH0041_GL0017839: non-AMP/OTHERS (Probabilities: 0.9905, 0.0069, 0.0026)\n","Processed DOF013_GL0053897: non-AMP/OTHERS (Probabilities: 0.9891, 0.0038, 0.0071)\n","Processed MH0281_GL0061703: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0034)\n","Processed MH0072_GL0022881: Neuropeptide (Probabilities: 0.2844, 0.0325, 0.6830)\n","Processed T2D-56A_GL0051420: non-AMP/OTHERS (Probabilities: 0.9922, 0.0034, 0.0044)\n","Processed MH0086_GL0082494: non-AMP/OTHERS (Probabilities: 0.6532, 0.0190, 0.3278)\n","Processed MH0457_GL0025117: Neuropeptide (Probabilities: 0.0094, 0.0090, 0.9817)\n","Processed V1.FI27_GL0072285: non-AMP/OTHERS (Probabilities: 0.6621, 0.2664, 0.0715)\n","Processed V1.FI27_GL0072285: non-AMP/OTHERS (Probabilities: 0.9905, 0.0061, 0.0034)\n","Processed MH0203_GL0160944: non-AMP/OTHERS (Probabilities: 0.8073, 0.1520, 0.0407)\n","Processed MH0203_GL0160944: non-AMP/OTHERS (Probabilities: 0.9901, 0.0067, 0.0032)\n","Processed MH0370_GL0042621: non-AMP/OTHERS (Probabilities: 0.9871, 0.0063, 0.0066)\n","Processed MH0052_GL0015164: non-AMP/OTHERS (Probabilities: 0.9913, 0.0047, 0.0040)\n","Processed MH0055_GL0009975: non-AMP/OTHERS (Probabilities: 0.9919, 0.0035, 0.0047)\n","Processed MH0055_GL0009975: non-AMP/OTHERS (Probabilities: 0.9920, 0.0037, 0.0042)\n","Processed N038A_GL0001480: non-AMP/OTHERS (Probabilities: 0.9666, 0.0129, 0.0205)\n","Processed MH0012_GL0038839: non-AMP/OTHERS (Probabilities: 0.9834, 0.0082, 0.0084)\n","Processed DOF004_GL0033601: non-AMP/OTHERS (Probabilities: 0.9916, 0.0043, 0.0041)\n","Processed MH0424_GL0048658: non-AMP/OTHERS (Probabilities: 0.9890, 0.0040, 0.0070)\n","Processed MH0424_GL0048658: non-AMP/OTHERS (Probabilities: 0.9855, 0.0050, 0.0095)\n","Processed V1.CD7-4_GL0037690: non-AMP/OTHERS (Probabilities: 0.9918, 0.0035, 0.0046)\n","Processed V1.FI28_GL0239313: non-AMP/OTHERS (Probabilities: 0.9331, 0.0122, 0.0547)\n","Processed MH0002_GL0009883: non-AMP/OTHERS (Probabilities: 0.9917, 0.0031, 0.0051)\n","Processed MH0153_GL0127041: non-AMP/OTHERS (Probabilities: 0.9908, 0.0035, 0.0057)\n","Processed MH0003_GL0077283: Neuropeptide (Probabilities: 0.0076, 0.0089, 0.9835)\n","Processed MH0230_GL0028792: non-AMP/OTHERS (Probabilities: 0.9930, 0.0033, 0.0036)\n","Processed MH0097_GL0078063: non-AMP/OTHERS (Probabilities: 0.9910, 0.0032, 0.0058)\n","Processed MH0012_GL0175016: Neuropeptide (Probabilities: 0.0078, 0.0091, 0.9832)\n","Processed O2.UC50-0_GL0031629: AMP (Probabilities: 0.0791, 0.9075, 0.0134)\n","Processed MH0370_GL0134155: non-AMP/OTHERS (Probabilities: 0.9905, 0.0036, 0.0060)\n","Processed 763597684-stool1_revised_scaffold53529_1_gene58354: Neuropeptide (Probabilities: 0.0074, 0.0090, 0.9836)\n","Processed DOM018_GL0062637: Neuropeptide (Probabilities: 0.0347, 0.0170, 0.9483)\n","Processed T2D-25A_GL0081229: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0267_GL0172504: non-AMP/OTHERS (Probabilities: 0.9810, 0.0073, 0.0117)\n","Processed MH0251_GL0014519: Neuropeptide (Probabilities: 0.2411, 0.0449, 0.7141)\n","Processed MH0062_GL0001737: non-AMP/OTHERS (Probabilities: 0.5980, 0.0308, 0.3712)\n","Processed MH0251_GL0129971: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0159_GL0001056: Neuropeptide (Probabilities: 0.0192, 0.0130, 0.9678)\n","Processed MH0424_GL0112817: Neuropeptide (Probabilities: 0.0360, 0.0148, 0.9493)\n","Processed MH0270_GL0142116: non-AMP/OTHERS (Probabilities: 0.9774, 0.0058, 0.0167)\n","Processed 1042403.BALAC2494_00922: Neuropeptide (Probabilities: 0.0072, 0.0089, 0.9838)\n","Processed MH0410_GL0049220: Neuropeptide (Probabilities: 0.0097, 0.0097, 0.9806)\n","Processed MH0382_GL0099490: Neuropeptide (Probabilities: 0.1490, 0.0208, 0.8302)\n","Processed SZEY-101A_GL0068335: non-AMP/OTHERS (Probabilities: 0.9923, 0.0035, 0.0042)\n","Processed O2.UC22-1_GL0036114: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0039)\n","Processed MH0161_GL0071760: non-AMP/OTHERS (Probabilities: 0.9683, 0.0157, 0.0161)\n","Processed MH0003_GL0012718: non-AMP/OTHERS (Probabilities: 0.5911, 0.3574, 0.0515)\n","Processed O2.UC15-0_GL0020430: non-AMP/OTHERS (Probabilities: 0.9923, 0.0043, 0.0033)\n","Processed MH0356_GL0112525: non-AMP/OTHERS (Probabilities: 0.9874, 0.0060, 0.0066)\n","Processed MH0188_GL0104239: non-AMP/OTHERS (Probabilities: 0.9810, 0.0097, 0.0094)\n","Processed 411462.DORLON_00739: non-AMP/OTHERS (Probabilities: 0.9585, 0.0234, 0.0181)\n","Processed MH0004_GL0025010: non-AMP/OTHERS (Probabilities: 0.9869, 0.0063, 0.0068)\n","Processed MH0087_GL0021648: non-AMP/OTHERS (Probabilities: 0.9828, 0.0063, 0.0109)\n","Processed O2.UC48-2_GL0069932: non-AMP/OTHERS (Probabilities: 0.9862, 0.0050, 0.0089)\n","Processed T2D-42A_GL0082631: non-AMP/OTHERS (Probabilities: 0.9923, 0.0039, 0.0038)\n","Processed V1.FI16_GL0012531: non-AMP/OTHERS (Probabilities: 0.9889, 0.0056, 0.0055)\n","Processed MH0049_GL0007523: Neuropeptide (Probabilities: 0.0061, 0.0089, 0.9850)\n","Processed MH0334_GL0096615: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0039)\n","Processed O2.CD3-0-PT_GL0026187: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed 160400887-stool1_revised_scaffold29136_1_gene78865: non-AMP/OTHERS (Probabilities: 0.9931, 0.0030, 0.0039)\n","Processed MH0236_GL0116391: non-AMP/OTHERS (Probabilities: 0.9925, 0.0040, 0.0036)\n","Processed MH0236_GL0116391: non-AMP/OTHERS (Probabilities: 0.9924, 0.0042, 0.0034)\n","Processed MH0089_GL0111378: Neuropeptide (Probabilities: 0.0457, 0.0206, 0.9338)\n","Processed MH0251_GL0068513: non-AMP/OTHERS (Probabilities: 0.9929, 0.0037, 0.0034)\n","Processed V1.FI04_GL0014289: non-AMP/OTHERS (Probabilities: 0.9872, 0.0040, 0.0087)\n","Processed O2.UC22-1_GL0143101: non-AMP/OTHERS (Probabilities: 0.9915, 0.0033, 0.0052)\n","Processed MH0188_GL0096271: non-AMP/OTHERS (Probabilities: 0.9930, 0.0029, 0.0041)\n","Processed V1.UC55-0_GL0220226: non-AMP/OTHERS (Probabilities: 0.9914, 0.0048, 0.0039)\n","Processed 158499257-stool1_revised_C1426365_1_gene218397: non-AMP/OTHERS (Probabilities: 0.6706, 0.2330, 0.0964)\n","Processed T2D-135A_GL0121453: non-AMP/OTHERS (Probabilities: 0.9765, 0.0128, 0.0107)\n","Processed MH0453_GL0024375: non-AMP/OTHERS (Probabilities: 0.9918, 0.0032, 0.0049)\n","Processed MH0041_GL0017839: Neuropeptide (Probabilities: 0.0755, 0.0254, 0.8991)\n","Processed V1.CD41-0_GL0104555: non-AMP/OTHERS (Probabilities: 0.9895, 0.0033, 0.0072)\n","Processed V1.CD41-0_GL0104555: non-AMP/OTHERS (Probabilities: 0.9923, 0.0033, 0.0043)\n","Processed MH0086_GL0049581: non-AMP/OTHERS (Probabilities: 0.9408, 0.0149, 0.0443)\n","Processed 158458797-stool1_revised_C1274612_1_gene27583: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed O2.UC17-1_GL0005463: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0147_GL0108522: Neuropeptide (Probabilities: 0.0122, 0.0104, 0.9774)\n","Processed MH0432_GL0050497: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed V1.CD6-4_GL0055248: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0060_GL0000031: non-AMP/OTHERS (Probabilities: 0.9882, 0.0042, 0.0076)\n","Processed O2.UC46-1_GL0084476: non-AMP/OTHERS (Probabilities: 0.9723, 0.0050, 0.0227)\n","Processed MH0080_GL0067765: Neuropeptide (Probabilities: 0.0069, 0.0088, 0.9842)\n","Processed O2.UC35-1_GL0052110: non-AMP/OTHERS (Probabilities: 0.9547, 0.0170, 0.0283)\n","Processed T2D-110A_GL0101647: Neuropeptide (Probabilities: 0.0072, 0.0089, 0.9839)\n","Processed MH0131_GL0167816: non-AMP/OTHERS (Probabilities: 0.9906, 0.0046, 0.0048)\n","Processed MH0345_GL0133962: non-AMP/OTHERS (Probabilities: 0.9804, 0.0064, 0.0132)\n","Processed MH0101_GL0025174: non-AMP/OTHERS (Probabilities: 0.9923, 0.0035, 0.0042)\n","Processed MH0073_GL0002640: Neuropeptide (Probabilities: 0.0068, 0.0086, 0.9846)\n","Processed V1.CD6-0-PT_GL0039159: non-AMP/OTHERS (Probabilities: 0.9905, 0.0033, 0.0062)\n","Processed O2.UC48-0_GL0304849: Neuropeptide (Probabilities: 0.0344, 0.0168, 0.9488)\n","Processed V1.CD4-0-PT_GL0019723: non-AMP/OTHERS (Probabilities: 0.9919, 0.0039, 0.0042)\n","Processed V1.UC41-0_GL0024987: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0040)\n","Processed MH0006_GL0149370: non-AMP/OTHERS (Probabilities: 0.9926, 0.0036, 0.0038)\n","Processed NOM016_GL0075239: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9850)\n","Processed MH0037_GL0008631: non-AMP/OTHERS (Probabilities: 0.9777, 0.0056, 0.0167)\n","Processed O2.UC16-1_GL0064415: non-AMP/OTHERS (Probabilities: 0.9900, 0.0037, 0.0063)\n","Processed MH0375_GL0070661: Neuropeptide (Probabilities: 0.0348, 0.0243, 0.9409)\n","Processed O2.UC11-2_GL0013257: non-AMP/OTHERS (Probabilities: 0.9889, 0.0039, 0.0072)\n","Processed O2.UC11-2_GL0013257: non-AMP/OTHERS (Probabilities: 0.9915, 0.0041, 0.0044)\n","Processed MH0230_GL0016557: Neuropeptide (Probabilities: 0.0112, 0.0101, 0.9787)\n","Processed V1.UC52-1_GL0075016: Neuropeptide (Probabilities: 0.1230, 0.0182, 0.8588)\n","Processed MH0396_GL0182819: Neuropeptide (Probabilities: 0.0071, 0.0085, 0.9844)\n","Processed MH0140_GL0149137: Neuropeptide (Probabilities: 0.0068, 0.0086, 0.9846)\n","Processed MH0006_GL0021178: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed MH0188_GL0097972: non-AMP/OTHERS (Probabilities: 0.9918, 0.0031, 0.0051)\n","Processed MH0346_GL0112122: non-AMP/OTHERS (Probabilities: 0.9919, 0.0035, 0.0045)\n","Processed MH0423_GL0103049: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed MH0253_GL0134623: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0038)\n","Processed O2.UC19-1_GL0089391: non-AMP/OTHERS (Probabilities: 0.9853, 0.0065, 0.0082)\n","Processed MH0373_GL0099098: Neuropeptide (Probabilities: 0.0816, 0.0401, 0.8783)\n","Processed MH0051_GL0050191: non-AMP/OTHERS (Probabilities: 0.9921, 0.0042, 0.0037)\n","Processed MH0412_GL0180640: non-AMP/OTHERS (Probabilities: 0.9553, 0.0140, 0.0307)\n","Processed MH0272_GL0214683: non-AMP/OTHERS (Probabilities: 0.9092, 0.0236, 0.0673)\n","Processed O2.UC11-2_GL0070235: non-AMP/OTHERS (Probabilities: 0.9359, 0.0143, 0.0498)\n","Processed MH0127_GL0016116: Neuropeptide (Probabilities: 0.0105, 0.0106, 0.9789)\n","Processed MH0131_GL0100298: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9849)\n","Processed MH0309_GL0146813: non-AMP/OTHERS (Probabilities: 0.9468, 0.0180, 0.0352)\n","Processed MH0011_GL0033337: non-AMP/OTHERS (Probabilities: 0.8719, 0.0163, 0.1117)\n","Processed SZEY-104A_GL0070582: non-AMP/OTHERS (Probabilities: 0.9892, 0.0038, 0.0070)\n","Processed MH0292_GL0113922: non-AMP/OTHERS (Probabilities: 0.9919, 0.0035, 0.0046)\n","Processed 763840445-stool2_revised_scaffold52674_1_gene133491: Neuropeptide (Probabilities: 0.0089, 0.0114, 0.9797)\n","Processed MH0203_GL0160944: Neuropeptide (Probabilities: 0.2114, 0.2021, 0.5864)\n","Processed MH0131_GL0011494: non-AMP/OTHERS (Probabilities: 0.9167, 0.0467, 0.0366)\n","Processed MH0002_GL0043905: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed V1.FI32_GL0055432: non-AMP/OTHERS (Probabilities: 0.9746, 0.0064, 0.0190)\n","Processed MH0071_GL0053861: non-AMP/OTHERS (Probabilities: 0.9223, 0.0082, 0.0695)\n","Processed MH0456_GL0056782: non-AMP/OTHERS (Probabilities: 0.9920, 0.0053, 0.0027)\n","Processed DLF002_GL0040171: non-AMP/OTHERS (Probabilities: 0.9882, 0.0043, 0.0075)\n","Processed MH0176_GL0056810: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0176_GL0056810: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed MH0002_GL0059239: non-AMP/OTHERS (Probabilities: 0.9872, 0.0043, 0.0085)\n","Processed MH0189_GL0028464: non-AMP/OTHERS (Probabilities: 0.9922, 0.0035, 0.0042)\n","Processed MH0161_GL0062489: non-AMP/OTHERS (Probabilities: 0.9907, 0.0044, 0.0048)\n","Processed V1.FI05_GL0136948: non-AMP/OTHERS (Probabilities: 0.9923, 0.0035, 0.0042)\n","Processed O2.UC11-1_GL0143871: Neuropeptide (Probabilities: 0.0068, 0.0088, 0.9845)\n","Processed V1.UC56-0_GL0048603: non-AMP/OTHERS (Probabilities: 0.9894, 0.0036, 0.0071)\n","Processed MH0197_GL0016912: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed MH0086_GL0049543: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0036)\n","Processed MH0275_GL0021402: Neuropeptide (Probabilities: 0.0092, 0.0099, 0.9809)\n","Processed MH0086_GL0043556: non-AMP/OTHERS (Probabilities: 0.9924, 0.0031, 0.0045)\n","Processed MH0239_GL0143854: non-AMP/OTHERS (Probabilities: 0.9903, 0.0042, 0.0055)\n","Processed MH0040_GL0078917: non-AMP/OTHERS (Probabilities: 0.9908, 0.0044, 0.0048)\n","Processed MH0160_GL0005567: non-AMP/OTHERS (Probabilities: 0.9917, 0.0046, 0.0037)\n","Processed sp|P55259|GP2_HUMAN: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed MH0401_GL0113465: Neuropeptide (Probabilities: 0.0410, 0.0225, 0.9365)\n","Processed MH0006_GL0190389: Neuropeptide (Probabilities: 0.0496, 0.0285, 0.9219)\n","Processed V1.UC1-0_GL0031408: non-AMP/OTHERS (Probabilities: 0.9832, 0.0039, 0.0130)\n","Processed T2D-45A_GL0146633: non-AMP/OTHERS (Probabilities: 0.8822, 0.0166, 0.1012)\n","Processed MH0122_GL0079504: non-AMP/OTHERS (Probabilities: 0.9909, 0.0035, 0.0056)\n","Processed O2.UC34-0_GL0001194: Neuropeptide (Probabilities: 0.0065, 0.0087, 0.9848)\n","Processed MH0396_GL0133140: Neuropeptide (Probabilities: 0.0059, 0.0093, 0.9848)\n","Processed MH0168_GL0012168: Neuropeptide (Probabilities: 0.0180, 0.0132, 0.9689)\n","Processed MH0082_GL0047574: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9849)\n","Processed sp|P21397|AOFA_HUMAN: Neuropeptide (Probabilities: 0.0230, 0.0154, 0.9616)\n","Processed MH0200_GL0198105: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0034)\n","Processed MH0086_GL0064754: non-AMP/OTHERS (Probabilities: 0.9925, 0.0040, 0.0035)\n","Processed MH0247_GL0029943: Neuropeptide (Probabilities: 0.0067, 0.0086, 0.9847)\n","Processed MH0197_GL0012634: non-AMP/OTHERS (Probabilities: 0.9717, 0.0154, 0.0129)\n","Processed NOM002_GL0000647: Neuropeptide (Probabilities: 0.0073, 0.0089, 0.9838)\n","Processed MH0011_GL0072665: Neuropeptide (Probabilities: 0.0200, 0.0131, 0.9669)\n","Processed MH0011_GL0072665: Neuropeptide (Probabilities: 0.1010, 0.0191, 0.8798)\n","Processed MH0200_GL0193325: non-AMP/OTHERS (Probabilities: 0.9905, 0.0062, 0.0033)\n","Processed MH0100_GL0045221: non-AMP/OTHERS (Probabilities: 0.9883, 0.0042, 0.0075)\n","Processed MH0277_GL0020688: non-AMP/OTHERS (Probabilities: 0.9895, 0.0065, 0.0040)\n","Processed MH0088_GL0073500: Neuropeptide (Probabilities: 0.2457, 0.1144, 0.6399)\n","Processed NLM027_GL0082523: non-AMP/OTHERS (Probabilities: 0.9404, 0.0482, 0.0114)\n","Processed MH0003_GL0098273: non-AMP/OTHERS (Probabilities: 0.9705, 0.0056, 0.0239)\n","Processed MH0003_GL0098273: non-AMP/OTHERS (Probabilities: 0.9908, 0.0039, 0.0053)\n","Processed MH0161_GL0064248: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed V1.UC55-0_GL0084059: non-AMP/OTHERS (Probabilities: 0.9884, 0.0052, 0.0064)\n","Processed MH0305_GL0015340: Neuropeptide (Probabilities: 0.0066, 0.0085, 0.9849)\n","Processed MH0088_GL0054024: Neuropeptide (Probabilities: 0.0292, 0.0236, 0.9472)\n","Processed MH0441_GL0128719: non-AMP/OTHERS (Probabilities: 0.8993, 0.0298, 0.0709)\n","Processed O2.UC8-1_GL0093851: non-AMP/OTHERS (Probabilities: 0.9916, 0.0051, 0.0033)\n","Processed V1.UC33-0_GL0171120: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed MH0002_GL0007633: non-AMP/OTHERS (Probabilities: 0.9933, 0.0035, 0.0032)\n","Processed MH0204_GL0204382: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9850)\n","Processed MH0018_GL0017414: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9849)\n","Processed MH0003_GL0097415: non-AMP/OTHERS (Probabilities: 0.5439, 0.0346, 0.4215)\n","Processed MH0443_GL0137285: non-AMP/OTHERS (Probabilities: 0.9397, 0.0238, 0.0365)\n","Processed 717962.CC1_13270: non-AMP/OTHERS (Probabilities: 0.9871, 0.0094, 0.0035)\n","Processed MH0277_GL0042312: Neuropeptide (Probabilities: 0.0068, 0.0097, 0.9835)\n","Processed 158337416-stool2_revised_scaffold13846_1_gene18690: Neuropeptide (Probabilities: 0.0100, 0.0132, 0.9768)\n","Processed MH0346_GL0160252: non-AMP/OTHERS (Probabilities: 0.9087, 0.0236, 0.0677)\n","Processed DOM012_GL0051744: Neuropeptide (Probabilities: 0.0067, 0.0094, 0.9839)\n","Processed MH0069_GL0078130: Neuropeptide (Probabilities: 0.1096, 0.0240, 0.8664)\n","Processed MH0069_GL0078130: non-AMP/OTHERS (Probabilities: 0.9898, 0.0059, 0.0043)\n","Processed 435590.BVU_1005: non-AMP/OTHERS (Probabilities: 0.9454, 0.0117, 0.0429)\n","Processed MH0054_GL0019359: Neuropeptide (Probabilities: 0.0064, 0.0085, 0.9851)\n","Processed MH0011_GL0092124: non-AMP/OTHERS (Probabilities: 0.9923, 0.0039, 0.0037)\n","Processed O2.UC22-1_GL0164376: non-AMP/OTHERS (Probabilities: 0.9856, 0.0053, 0.0091)\n","Processed MH0361_GL0003681: Neuropeptide (Probabilities: 0.0076, 0.0091, 0.9833)\n","Processed O2.CD2-0-PT_GL0071257: Neuropeptide (Probabilities: 0.0753, 0.0205, 0.9041)\n","Processed MH0088_GL0034073: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0277_GL0049367: Neuropeptide (Probabilities: 0.0606, 0.0182, 0.9212)\n","Processed 1042403.BALAC2494_00245: non-AMP/OTHERS (Probabilities: 0.9880, 0.0086, 0.0034)\n","Processed DLF012_GL0006636: non-AMP/OTHERS (Probabilities: 0.9473, 0.0414, 0.0112)\n","Processed MH0356_GL0007431: non-AMP/OTHERS (Probabilities: 0.9574, 0.0317, 0.0109)\n","Processed BGI003A_GL0085603: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed T2D-6A_GL0170904: non-AMP/OTHERS (Probabilities: 0.9928, 0.0040, 0.0032)\n","Processed MH0100_GL0108102: non-AMP/OTHERS (Probabilities: 0.4959, 0.0270, 0.4771)\n","Processed 1042403.BALAC2494_00215: non-AMP/OTHERS (Probabilities: 0.9761, 0.0084, 0.0155)\n","Processed MH0100_GL0008285: Neuropeptide (Probabilities: 0.0069, 0.0096, 0.9835)\n","Processed 435591.BDI_1920: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0356_GL0162057: non-AMP/OTHERS (Probabilities: 0.9762, 0.0069, 0.0169)\n","Processed MH0203_GL0064918: non-AMP/OTHERS (Probabilities: 0.9891, 0.0038, 0.0071)\n","Processed V1.UC12-0_GL0055568: non-AMP/OTHERS (Probabilities: 0.9899, 0.0070, 0.0031)\n","Processed MH0302_GL0019301: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9852)\n","Processed MH0435_GL0123496: Neuropeptide (Probabilities: 0.3714, 0.0319, 0.5967)\n","Processed MH0435_GL0123496: non-AMP/OTHERS (Probabilities: 0.9923, 0.0029, 0.0048)\n","Processed V1.UC26-4_GL0113470: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0096_GL0055481: non-AMP/OTHERS (Probabilities: 0.8688, 0.0203, 0.1108)\n","Processed MH0096_GL0055481: Neuropeptide (Probabilities: 0.2959, 0.0302, 0.6739)\n","Processed MH0303_GL0082287: non-AMP/OTHERS (Probabilities: 0.5673, 0.0320, 0.4007)\n","Processed DLF013_GL0030242: Neuropeptide (Probabilities: 0.0110, 0.0105, 0.9785)\n","Processed O2.UC37-2_GL0026574: Neuropeptide (Probabilities: 0.0095, 0.0100, 0.9805)\n","Processed O2.UC37-2_GL0026574: Neuropeptide (Probabilities: 0.0110, 0.0110, 0.9779)\n","Processed V1.FI18_GL0011533: Neuropeptide (Probabilities: 0.1831, 0.0323, 0.7846)\n","Processed MH0043_GL0012691: Neuropeptide (Probabilities: 0.0107, 0.0101, 0.9792)\n","Processed 469603.HMPREF0406_01991: non-AMP/OTHERS (Probabilities: 0.5296, 0.0758, 0.3946)\n","Processed MH0229_GL0163719: Neuropeptide (Probabilities: 0.3022, 0.0485, 0.6493)\n","Processed MH0277_GL0022590: Neuropeptide (Probabilities: 0.2996, 0.0447, 0.6557)\n","Processed 435842.HMPREF0848_00547: non-AMP/OTHERS (Probabilities: 0.9878, 0.0067, 0.0056)\n","Processed MH0345_GL0177949: non-AMP/OTHERS (Probabilities: 0.9472, 0.0376, 0.0152)\n","Processed MH0003_GL0040418: Neuropeptide (Probabilities: 0.0380, 0.0189, 0.9431)\n","Processed MH0203_GL0120151: non-AMP/OTHERS (Probabilities: 0.9930, 0.0043, 0.0028)\n","Processed 547043.BIFPSEUDO_04241: non-AMP/OTHERS (Probabilities: 0.9929, 0.0038, 0.0032)\n","Processed MH0206_GL0207124: non-AMP/OTHERS (Probabilities: 0.9804, 0.0098, 0.0098)\n","Processed DLM016_GL0081646: non-AMP/OTHERS (Probabilities: 0.9921, 0.0041, 0.0039)\n","Processed 428126.CLOSPI_02119: non-AMP/OTHERS (Probabilities: 0.9912, 0.0034, 0.0054)\n","Processed MH0188_GL0043557: non-AMP/OTHERS (Probabilities: 0.9930, 0.0033, 0.0037)\n","Processed O2.UC17-0_GL0009803: non-AMP/OTHERS (Probabilities: 0.9925, 0.0034, 0.0041)\n","Processed MH0011_GL0034504: non-AMP/OTHERS (Probabilities: 0.9496, 0.0239, 0.0265)\n","Processed MH0259_GL0131606: non-AMP/OTHERS (Probabilities: 0.9130, 0.0398, 0.0471)\n","Processed MH0062_GL0045853: non-AMP/OTHERS (Probabilities: 0.9898, 0.0049, 0.0053)\n","Processed MH0277_GL0034687: non-AMP/OTHERS (Probabilities: 0.9880, 0.0049, 0.0072)\n","Processed MH0318_GL0131092: non-AMP/OTHERS (Probabilities: 0.9924, 0.0034, 0.0042)\n","Processed MH0203_GL0100417: Neuropeptide (Probabilities: 0.4206, 0.0230, 0.5564)\n","Processed MH0003_GL0005201: Neuropeptide (Probabilities: 0.3275, 0.0440, 0.6285)\n","Processed MH0203_GL0120151: non-AMP/OTHERS (Probabilities: 0.9931, 0.0037, 0.0032)\n","Processed O2.UC6-1_GL0157896: non-AMP/OTHERS (Probabilities: 0.9906, 0.0030, 0.0064)\n","Processed MH0111_GL0006613: Neuropeptide (Probabilities: 0.0077, 0.0088, 0.9835)\n","Processed DLM012_GL0044340: Neuropeptide (Probabilities: 0.1197, 0.0408, 0.8395)\n","Processed MH0072_GL0022881: non-AMP/OTHERS (Probabilities: 0.9919, 0.0032, 0.0049)\n","Processed MH0099_GL0058602: Neuropeptide (Probabilities: 0.2692, 0.0991, 0.6317)\n","Processed MH0037_GL0050217: Neuropeptide (Probabilities: 0.0098, 0.0095, 0.9807)\n","Processed MH0323_GL0012858: non-AMP/OTHERS (Probabilities: 0.9902, 0.0046, 0.0051)\n","Processed MH0277_GL0023021: Neuropeptide (Probabilities: 0.2504, 0.3110, 0.4387)\n","Processed MH0438_GL0191196: non-AMP/OTHERS (Probabilities: 0.9667, 0.0126, 0.0207)\n","Processed 158944319-stool2_revised_scaffold27192_1_gene64328: non-AMP/OTHERS (Probabilities: 0.9928, 0.0040, 0.0032)\n","Processed 457421.CBFG_02757: non-AMP/OTHERS (Probabilities: 0.9913, 0.0038, 0.0049)\n","Processed 944560.HMPREF9058_1644: Neuropeptide (Probabilities: 0.0938, 0.0415, 0.8647)\n","Processed V1.UC40-1_GL0156300: non-AMP/OTHERS (Probabilities: 0.9920, 0.0046, 0.0034)\n","Processed MH0211_GL0059692: non-AMP/OTHERS (Probabilities: 0.9914, 0.0028, 0.0058)\n","Processed O2.UC2-1_GL0086763: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0033)\n","Processed 321967.LSEI_0877: Neuropeptide (Probabilities: 0.0155, 0.0127, 0.9717)\n","Processed MH0131_GL0051643: non-AMP/OTHERS (Probabilities: 0.9917, 0.0042, 0.0041)\n","Processed MH0095_GL0089723: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0131_GL0107430: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed O2.UC4-1_GL0041942: Neuropeptide (Probabilities: 0.0061, 0.0089, 0.9850)\n","Processed MH0128_GL0012238: non-AMP/OTHERS (Probabilities: 0.9920, 0.0033, 0.0047)\n","Processed MH0244_GL0029239: non-AMP/OTHERS (Probabilities: 0.9912, 0.0034, 0.0054)\n","Processed MH0003_GL0063796: non-AMP/OTHERS (Probabilities: 0.9929, 0.0038, 0.0033)\n","Processed MH0188_GL0116650: non-AMP/OTHERS (Probabilities: 0.9874, 0.0048, 0.0078)\n","Processed DOM014_GL0024811: non-AMP/OTHERS (Probabilities: 0.9702, 0.0079, 0.0219)\n","Processed MH0179_GL0032520: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed MH0117_GL0096473: Neuropeptide (Probabilities: 0.0415, 0.0214, 0.9371)\n","Processed MH0088_GL0089819: non-AMP/OTHERS (Probabilities: 0.9863, 0.0048, 0.0089)\n","Processed MH0131_GL0096652: non-AMP/OTHERS (Probabilities: 0.9909, 0.0058, 0.0033)\n","Processed MH0003_GL0109716: non-AMP/OTHERS (Probabilities: 0.9217, 0.0113, 0.0670)\n","Processed MH0062_GL0027690: non-AMP/OTHERS (Probabilities: 0.9912, 0.0041, 0.0047)\n","Processed MH0119_GL0002515: Neuropeptide (Probabilities: 0.1263, 0.0298, 0.8439)\n","Processed MH0422_GL0022608: non-AMP/OTHERS (Probabilities: 0.9701, 0.0122, 0.0177)\n","Processed O2.UC44-2_GL0142709: non-AMP/OTHERS (Probabilities: 0.9788, 0.0181, 0.0031)\n","Processed 367928.BAD_0853: non-AMP/OTHERS (Probabilities: 0.9875, 0.0059, 0.0066)\n","Processed O2.CD1-0-PT_GL0052675: non-AMP/OTHERS (Probabilities: 0.9918, 0.0035, 0.0047)\n","Processed 1035817.BLNIAS_01784: non-AMP/OTHERS (Probabilities: 0.9710, 0.0051, 0.0239)\n","Processed 367928.BAD_0991: non-AMP/OTHERS (Probabilities: 0.9914, 0.0033, 0.0053)\n","Processed NLM027_GL0039233: non-AMP/OTHERS (Probabilities: 0.9778, 0.0161, 0.0061)\n","Processed V1.UC6-0_GL0083590: non-AMP/OTHERS (Probabilities: 0.9809, 0.0075, 0.0117)\n","Processed 809635352-stool1_revised_scaffold55643_1_gene118260: non-AMP/OTHERS (Probabilities: 0.9750, 0.0077, 0.0173)\n","Processed MH0008_GL0038621: non-AMP/OTHERS (Probabilities: 0.9926, 0.0035, 0.0040)\n","Processed MH0025_GL0060205: Neuropeptide (Probabilities: 0.0127, 0.0097, 0.9777)\n","Processed MH0025_GL0060205: non-AMP/OTHERS (Probabilities: 0.6231, 0.0220, 0.3548)\n","Processed sp|P30101|PDIA3_HUMAN: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9853)\n","Processed O2.UC48-0_GL0254486: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0040)\n","Processed MH0375_GL0023587: non-AMP/OTHERS (Probabilities: 0.9914, 0.0037, 0.0049)\n","Processed MH0193_GL0052517: non-AMP/OTHERS (Probabilities: 0.9922, 0.0030, 0.0048)\n","Processed MH0193_GL0052517: non-AMP/OTHERS (Probabilities: 0.9920, 0.0035, 0.0045)\n","Processed O2.UC1-2_GL0137618: non-AMP/OTHERS (Probabilities: 0.9482, 0.0302, 0.0216)\n","Processed O2.UC43-2_GL0030526: non-AMP/OTHERS (Probabilities: 0.9691, 0.0110, 0.0199)\n","Processed 428126.CLOSPI_01979: non-AMP/OTHERS (Probabilities: 0.9778, 0.0170, 0.0052)\n","Processed MH0062_GL0065556: non-AMP/OTHERS (Probabilities: 0.9883, 0.0051, 0.0067)\n","Processed 404239096-stool1_revised_scaffold6512_11_gene17378: non-AMP/OTHERS (Probabilities: 0.9540, 0.0097, 0.0363)\n","Processed MH0002_GL0033407: Neuropeptide (Probabilities: 0.0127, 0.0105, 0.9769)\n","Processed MH0104_GL0107281: non-AMP/OTHERS (Probabilities: 0.9924, 0.0035, 0.0041)\n","Processed MH0230_GL0010727: Neuropeptide (Probabilities: 0.0532, 0.0223, 0.9245)\n","Processed MH0131_GL0034022: non-AMP/OTHERS (Probabilities: 0.9907, 0.0035, 0.0058)\n","Processed MH0287_GL0037000: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0003_GL0012425: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed DLF012_GL0038373: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9850)\n","Processed NLM027_GL0008562: non-AMP/OTHERS (Probabilities: 0.9841, 0.0079, 0.0080)\n","Processed MH0131_GL0117744: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9850)\n","Processed MH0188_GL0022128: non-AMP/OTHERS (Probabilities: 0.6465, 0.0999, 0.2536)\n","Processed O2.CD1-0-PT_GL0014459: non-AMP/OTHERS (Probabilities: 0.8485, 0.0531, 0.0985)\n","Processed V1.FI09_GL0193739: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0015_GL0016930: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0015_GL0016930: non-AMP/OTHERS (Probabilities: 0.9885, 0.0070, 0.0045)\n","Processed ED50A_GL0062661: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0447_GL0229233: Neuropeptide (Probabilities: 0.0092, 0.0106, 0.9802)\n","Processed O2.UC53-0_GL0220189: Neuropeptide (Probabilities: 0.0061, 0.0090, 0.9849)\n","Processed O2.UC53-0_GL0220189: Neuropeptide (Probabilities: 0.1977, 0.0446, 0.7576)\n","Processed MH0089_GL0061840: non-AMP/OTHERS (Probabilities: 0.9860, 0.0088, 0.0051)\n","Processed MH0338_GL0054998: non-AMP/OTHERS (Probabilities: 0.9911, 0.0038, 0.0051)\n","Processed MH0062_GL0057194: non-AMP/OTHERS (Probabilities: 0.9868, 0.0069, 0.0063)\n","Processed MH0133_GL0011222: non-AMP/OTHERS (Probabilities: 0.9669, 0.0220, 0.0110)\n","Processed MH0277_GL0029966: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed SZEY-27A_GL0054702: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed O2.UC35-1_GL0083268: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed O2.UC19-1_GL0089391: non-AMP/OTHERS (Probabilities: 0.9925, 0.0030, 0.0044)\n","Processed O2.UC20-1_GL0142222: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed O2.CD1-0-PT_GL0133582: non-AMP/OTHERS (Probabilities: 0.9923, 0.0045, 0.0032)\n","Processed 457412.RSAG_00843: non-AMP/OTHERS (Probabilities: 0.9800, 0.0174, 0.0026)\n","Processed sp|O96008|TOM40_HUMAN: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9853)\n","Processed MH0395_GL0208434: non-AMP/OTHERS (Probabilities: 0.9921, 0.0052, 0.0027)\n","Processed 158458797-stool1_revised_scaffold25252_2_gene79145: non-AMP/OTHERS (Probabilities: 0.9887, 0.0087, 0.0026)\n","Processed MH0020_GL0050757: Neuropeptide (Probabilities: 0.0062, 0.0089, 0.9849)\n","Processed MH0345_GL0178971: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed 765094712-stool1_revised_scaffold3296_2_gene29312: non-AMP/OTHERS (Probabilities: 0.9879, 0.0076, 0.0046)\n","Processed O2.UC1-2_GL0049277: Neuropeptide (Probabilities: 0.0343, 0.0144, 0.9513)\n","Processed V1.UC58-4_GL0161109: Neuropeptide (Probabilities: 0.0587, 0.0152, 0.9261)\n","Processed 823052294-stool1_revised_C729179_1_gene30830: Neuropeptide (Probabilities: 0.0372, 0.0188, 0.9440)\n","Processed MH0413_GL0050424: non-AMP/OTHERS (Probabilities: 0.6448, 0.3029, 0.0523)\n","Processed MH0002_GL0007277: non-AMP/OTHERS (Probabilities: 0.9912, 0.0046, 0.0042)\n","Processed V1.UC58-0_GL0128364: non-AMP/OTHERS (Probabilities: 0.9876, 0.0072, 0.0052)\n","Processed V1.UC49-0_GL0193802: Neuropeptide (Probabilities: 0.0295, 0.0140, 0.9564)\n","Processed T2D-83A_GL0021375: non-AMP/OTHERS (Probabilities: 0.9762, 0.0054, 0.0184)\n","Processed O2.CD1-0-PT_GL0031848: Neuropeptide (Probabilities: 0.0605, 0.0507, 0.8888)\n","Processed MH0119_GL0067011: non-AMP/OTHERS (Probabilities: 0.9924, 0.0042, 0.0034)\n","Processed MH0019_GL0040024: non-AMP/OTHERS (Probabilities: 0.9914, 0.0033, 0.0053)\n","Processed O2.UC46-0_GL0108472: non-AMP/OTHERS (Probabilities: 0.9928, 0.0034, 0.0038)\n","Processed MH0198_GL0018777: non-AMP/OTHERS (Probabilities: 0.9923, 0.0033, 0.0045)\n","Processed O2.UC20-1_GL0154389: non-AMP/OTHERS (Probabilities: 0.9757, 0.0164, 0.0079)\n","Processed MH0087_GL0020675: AMP (Probabilities: 0.4037, 0.5608, 0.0355)\n","Processed SZEY-64A_GL0041008: non-AMP/OTHERS (Probabilities: 0.9924, 0.0037, 0.0039)\n","Processed MH0006_GL0148368: Neuropeptide (Probabilities: 0.0067, 0.0086, 0.9846)\n","Processed MH0050_GL0034425: non-AMP/OTHERS (Probabilities: 0.6155, 0.1327, 0.2518)\n","Processed MH0406_GL0119792: non-AMP/OTHERS (Probabilities: 0.8031, 0.0509, 0.1460)\n","Processed DLM016_GL0007149: non-AMP/OTHERS (Probabilities: 0.9889, 0.0043, 0.0069)\n","Processed 158944319-stool1_revised_scaffold8892_1_gene82172: non-AMP/OTHERS (Probabilities: 0.9593, 0.0120, 0.0287)\n","Processed O2.UC40-1_GL0149825: non-AMP/OTHERS (Probabilities: 0.9928, 0.0031, 0.0041)\n","Processed ED50A_GL0021590: non-AMP/OTHERS (Probabilities: 0.8583, 0.0162, 0.1254)\n","Processed MH0003_GL0030590: Neuropeptide (Probabilities: 0.0062, 0.0086, 0.9852)\n","Processed MH0150_GL0127769: non-AMP/OTHERS (Probabilities: 0.9923, 0.0031, 0.0046)\n","Processed MH0441_GL0236753: non-AMP/OTHERS (Probabilities: 0.7571, 0.0186, 0.2243)\n","Processed MH0128_GL0069184: non-AMP/OTHERS (Probabilities: 0.9925, 0.0033, 0.0043)\n","Processed O2.UC53-0_GL0106303: non-AMP/OTHERS (Probabilities: 0.9728, 0.0083, 0.0189)\n","Processed MH0435_GL0037515: Neuropeptide (Probabilities: 0.0122, 0.0111, 0.9767)\n","Processed MH0172_GL0107956: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed 479437.Elen_0412: Neuropeptide (Probabilities: 0.0066, 0.0088, 0.9847)\n","Processed 657315.ROI_19420: non-AMP/OTHERS (Probabilities: 0.9698, 0.0054, 0.0247)\n","Processed 206672.BL0440: non-AMP/OTHERS (Probabilities: 0.9545, 0.0159, 0.0295)\n","Processed MH0203_GL0117758: Neuropeptide (Probabilities: 0.0065, 0.0084, 0.9851)\n","Processed MH0280_GL0186345: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9850)\n","Processed MH0440_GL0206260: non-AMP/OTHERS (Probabilities: 0.9932, 0.0032, 0.0036)\n","Processed MH0162_GL0140658: non-AMP/OTHERS (Probabilities: 0.9274, 0.0236, 0.0490)\n","Processed MH0270_GL0044430: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0251_GL0137853: non-AMP/OTHERS (Probabilities: 0.7522, 0.2273, 0.0205)\n","Processed DLF002_GL0049811: non-AMP/OTHERS (Probabilities: 0.9210, 0.0465, 0.0325)\n","Processed MH0002_GL0003865: non-AMP/OTHERS (Probabilities: 0.9898, 0.0046, 0.0056)\n","Processed MH0274_GL0009078: non-AMP/OTHERS (Probabilities: 0.8050, 0.1594, 0.0357)\n","Processed DOF013_GL0005261: non-AMP/OTHERS (Probabilities: 0.9816, 0.0087, 0.0097)\n","Processed DLF002_GL0049811: non-AMP/OTHERS (Probabilities: 0.8322, 0.1408, 0.0270)\n","Processed MH0193_GL0052698: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0002_GL0020591: Neuropeptide (Probabilities: 0.0482, 0.0180, 0.9338)\n","Processed DLF012_GL0021195: non-AMP/OTHERS (Probabilities: 0.5165, 0.0422, 0.4412)\n","Processed MH0211_GL0147826: Neuropeptide (Probabilities: 0.1187, 0.0719, 0.8095)\n","Processed MH0128_GL0068720: non-AMP/OTHERS (Probabilities: 0.9783, 0.0055, 0.0162)\n","Processed MH0006_GL0161706: non-AMP/OTHERS (Probabilities: 0.9876, 0.0047, 0.0077)\n","Processed O2.UC48-0_GL0251434: non-AMP/OTHERS (Probabilities: 0.9748, 0.0106, 0.0146)\n","Processed 367928.BAD_0874: Neuropeptide (Probabilities: 0.0193, 0.0139, 0.9668)\n","Processed sp|P35579|MYH9_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0360_GL0079565: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0432_GL0075237: Neuropeptide (Probabilities: 0.0135, 0.0107, 0.9758)\n","Processed MH0406_GL0097277: non-AMP/OTHERS (Probabilities: 0.9317, 0.0154, 0.0529)\n","Processed MH0372_GL0112226: non-AMP/OTHERS (Probabilities: 0.9713, 0.0145, 0.0143)\n","Processed MH0193_GL0178152: non-AMP/OTHERS (Probabilities: 0.9877, 0.0094, 0.0029)\n","Processed MH0131_GL0008268: non-AMP/OTHERS (Probabilities: 0.9893, 0.0079, 0.0028)\n","Processed MH0131_GL0008268: non-AMP/OTHERS (Probabilities: 0.9833, 0.0138, 0.0029)\n","Processed MH0161_GL0006861: non-AMP/OTHERS (Probabilities: 0.8559, 0.0196, 0.1245)\n","Processed MH0016_GL0087621: non-AMP/OTHERS (Probabilities: 0.9927, 0.0043, 0.0030)\n","Processed MH0127_GL0067595: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0036)\n","Processed MH0127_GL0067595: non-AMP/OTHERS (Probabilities: 0.9910, 0.0059, 0.0030)\n","Processed V1.CD48-0_GL0078074: non-AMP/OTHERS (Probabilities: 0.9921, 0.0042, 0.0037)\n","Processed MH0001_GL0018270: non-AMP/OTHERS (Probabilities: 0.9931, 0.0031, 0.0038)\n","Processed MH0092_GL0150929: non-AMP/OTHERS (Probabilities: 0.9927, 0.0032, 0.0041)\n","Processed MH0211_GL0086319: non-AMP/OTHERS (Probabilities: 0.9860, 0.0059, 0.0082)\n","Processed MH0288_GL0182934: non-AMP/OTHERS (Probabilities: 0.9931, 0.0031, 0.0037)\n","Processed MH0131_GL0151752: non-AMP/OTHERS (Probabilities: 0.9110, 0.0255, 0.0635)\n","Processed O2.UC22-1_GL0051458: non-AMP/OTHERS (Probabilities: 0.9925, 0.0030, 0.0045)\n","Processed MH0002_GL0050697: non-AMP/OTHERS (Probabilities: 0.9921, 0.0035, 0.0044)\n","Processed O2.UC50-2_GL0078453: Neuropeptide (Probabilities: 0.0062, 0.0090, 0.9849)\n","Processed O2.UC50-2_GL0078453: non-AMP/OTHERS (Probabilities: 0.7270, 0.0752, 0.1978)\n","Processed MH0197_GL0124370: non-AMP/OTHERS (Probabilities: 0.9239, 0.0277, 0.0485)\n","Processed MH0446_GL0103208: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed tr|A0A5C2FUE5|A0A5C2FUE5_HUMAN: Neuropeptide (Probabilities: 0.0080, 0.0103, 0.9817)\n","Processed O2.UC15-0_GL0064972: non-AMP/OTHERS (Probabilities: 0.9933, 0.0032, 0.0035)\n","Processed O2.UC49-0_GL0010552: non-AMP/OTHERS (Probabilities: 0.9911, 0.0032, 0.0057)\n","Processed HT14A_GL0118202: non-AMP/OTHERS (Probabilities: 0.9921, 0.0031, 0.0047)\n","Processed V1.FI29_GL0077807: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0015_GL0039438: non-AMP/OTHERS (Probabilities: 0.9874, 0.0046, 0.0080)\n","Processed MH0406_GL0180442: non-AMP/OTHERS (Probabilities: 0.9821, 0.0122, 0.0057)\n","Processed MH0369_GL0114400: non-AMP/OTHERS (Probabilities: 0.6184, 0.0229, 0.3587)\n","Processed MH0211_GL0175339: non-AMP/OTHERS (Probabilities: 0.9816, 0.0092, 0.0092)\n","Processed MH0217_GL0140221: non-AMP/OTHERS (Probabilities: 0.9926, 0.0030, 0.0044)\n","Processed HT14A_GL0030136: Neuropeptide (Probabilities: 0.0373, 0.0173, 0.9455)\n","Processed MH0233_GL0019448: non-AMP/OTHERS (Probabilities: 0.9865, 0.0047, 0.0088)\n","Processed sp|O75629|CREG1_HUMAN: non-AMP/OTHERS (Probabilities: 0.9910, 0.0039, 0.0052)\n","Processed MH0131_GL0079282: non-AMP/OTHERS (Probabilities: 0.9874, 0.0045, 0.0080)\n","Processed V1.CD55-0_GL0193200: Neuropeptide (Probabilities: 0.0391, 0.0166, 0.9443)\n","Processed MH0305_GL0015340: non-AMP/OTHERS (Probabilities: 0.9933, 0.0034, 0.0034)\n","Processed NOM002_GL0000647: non-AMP/OTHERS (Probabilities: 0.9935, 0.0032, 0.0033)\n","Processed NOM020_GL0046604: Neuropeptide (Probabilities: 0.0066, 0.0086, 0.9849)\n","Processed T2D-41A_GL0051749: non-AMP/OTHERS (Probabilities: 0.9806, 0.0073, 0.0121)\n","Processed MH0293_GL0038915: Neuropeptide (Probabilities: 0.0085, 0.0094, 0.9821)\n","Processed 158499257-stool2_revised_scaffold67679_1_gene167855: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed O2.UC28-2_GL0021260: non-AMP/OTHERS (Probabilities: 0.9662, 0.0172, 0.0166)\n","Processed MH0138_GL0039744: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0297_GL0021679: Neuropeptide (Probabilities: 0.1214, 0.0681, 0.8106)\n","Processed DOF012_GL0004220: non-AMP/OTHERS (Probabilities: 0.9927, 0.0038, 0.0035)\n","Processed MH0003_GL0004366: non-AMP/OTHERS (Probabilities: 0.9102, 0.0238, 0.0660)\n","Processed MH0162_GL0083240: non-AMP/OTHERS (Probabilities: 0.6904, 0.0462, 0.2634)\n","Processed MH0089_GL0097026: non-AMP/OTHERS (Probabilities: 0.5780, 0.0982, 0.3239)\n","Processed O2.CD3-0-PT_GL0027257: non-AMP/OTHERS (Probabilities: 0.5433, 0.0196, 0.4371)\n","Processed 158458797-stool1_revised_C1248028_1_gene48202: Neuropeptide (Probabilities: 0.0258, 0.0135, 0.9608)\n","Processed MH0428_GL0198057: Neuropeptide (Probabilities: 0.0385, 0.0149, 0.9467)\n","Processed 264199.stu1654: non-AMP/OTHERS (Probabilities: 0.9921, 0.0038, 0.0041)\n","Processed MH0236_GL0049453: Neuropeptide (Probabilities: 0.0152, 0.0116, 0.9732)\n","Processed O2.UC27-0_GL0046636: Neuropeptide (Probabilities: 0.0186, 0.0128, 0.9686)\n","Processed 764285508-stool1_revised_scaffold26604_1_gene113646: non-AMP/OTHERS (Probabilities: 0.9875, 0.0046, 0.0079)\n","Processed MH0188_GL0103025: non-AMP/OTHERS (Probabilities: 0.9844, 0.0127, 0.0029)\n","Processed MH0002_GL0071398: non-AMP/OTHERS (Probabilities: 0.9917, 0.0040, 0.0042)\n","Processed MH0122_GL0063105: non-AMP/OTHERS (Probabilities: 0.9904, 0.0063, 0.0033)\n","Processed MH0086_GL0015768: Neuropeptide (Probabilities: 0.1740, 0.0516, 0.7744)\n","Processed MH0330_GL0213978: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed MH0356_GL0086898: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed T2D-51A_GL0152264: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed O2.UC48-0_GL0278979: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0062_GL0034116: non-AMP/OTHERS (Probabilities: 0.9917, 0.0036, 0.0047)\n","Processed V1.FI18_GL0093238: Neuropeptide (Probabilities: 0.2270, 0.0621, 0.7109)\n","Processed MH0006_GL0174420: non-AMP/OTHERS (Probabilities: 0.9833, 0.0052, 0.0115)\n","Processed SZEY-69A_GL0070022: non-AMP/OTHERS (Probabilities: 0.9881, 0.0043, 0.0076)\n","Processed MH0006_GL0071814: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed O2.UC17-2_GL0128124: non-AMP/OTHERS (Probabilities: 0.9907, 0.0035, 0.0058)\n","Processed MH0127_GL0027709: non-AMP/OTHERS (Probabilities: 0.9920, 0.0029, 0.0051)\n","Processed MH0014_GL0025066: non-AMP/OTHERS (Probabilities: 0.9871, 0.0036, 0.0094)\n","Processed MH0014_GL0025066: non-AMP/OTHERS (Probabilities: 0.9907, 0.0031, 0.0061)\n","Processed MH0330_GL0001729: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed MH0272_GL0186415: non-AMP/OTHERS (Probabilities: 0.9923, 0.0031, 0.0046)\n","Processed MH0203_GL0229393: non-AMP/OTHERS (Probabilities: 0.9920, 0.0028, 0.0052)\n","Processed MH0088_GL0001694: non-AMP/OTHERS (Probabilities: 0.9904, 0.0034, 0.0062)\n","Processed MH0110_GL0025297: non-AMP/OTHERS (Probabilities: 0.9909, 0.0037, 0.0054)\n","Processed MH0230_GL0008635: non-AMP/OTHERS (Probabilities: 0.9847, 0.0056, 0.0098)\n","Processed 158337416-stool1_revised_C1242610_1_gene200612: Neuropeptide (Probabilities: 0.4408, 0.0466, 0.5126)\n","Processed V1.FI29_GL0061338: non-AMP/OTHERS (Probabilities: 0.8319, 0.0300, 0.1381)\n","Processed MH0430_GL0170087: non-AMP/OTHERS (Probabilities: 0.9907, 0.0041, 0.0051)\n","Processed O2.UC13-1_GL0230652: non-AMP/OTHERS (Probabilities: 0.9629, 0.0120, 0.0251)\n","Processed sp|P54296|MYOM2_HUMAN: non-AMP/OTHERS (Probabilities: 0.9899, 0.0045, 0.0056)\n","Processed V1.UC58-4_GL0166559: Neuropeptide (Probabilities: 0.0232, 0.0191, 0.9577)\n","Processed V1.UC58-4_GL0166559: non-AMP/OTHERS (Probabilities: 0.9899, 0.0042, 0.0059)\n","Processed MH0062_GL0059294: Neuropeptide (Probabilities: 0.2492, 0.0401, 0.7108)\n","Processed MH0230_GL0088143: non-AMP/OTHERS (Probabilities: 0.9924, 0.0035, 0.0040)\n","Processed MH0188_GL0089789: non-AMP/OTHERS (Probabilities: 0.9923, 0.0034, 0.0043)\n","Processed MH0356_GL0130200: non-AMP/OTHERS (Probabilities: 0.9923, 0.0036, 0.0040)\n","Processed MH0356_GL0171909: non-AMP/OTHERS (Probabilities: 0.9912, 0.0041, 0.0048)\n","Processed MH0086_GL0083850: non-AMP/OTHERS (Probabilities: 0.9859, 0.0045, 0.0096)\n","Processed MH0002_GL0012963: non-AMP/OTHERS (Probabilities: 0.9897, 0.0035, 0.0068)\n","Processed MH0012_GL0036452: Neuropeptide (Probabilities: 0.0061, 0.0090, 0.9849)\n","Processed MH0163_GL0106300: non-AMP/OTHERS (Probabilities: 0.9558, 0.0130, 0.0313)\n","Processed MH0096_GL0073471: non-AMP/OTHERS (Probabilities: 0.9894, 0.0040, 0.0066)\n","Processed MH0161_GL0011973: non-AMP/OTHERS (Probabilities: 0.9918, 0.0034, 0.0048)\n","Processed MH0443_GL0018845: non-AMP/OTHERS (Probabilities: 0.9925, 0.0031, 0.0043)\n","Processed T2D-48A_GL0071276: non-AMP/OTHERS (Probabilities: 0.9914, 0.0045, 0.0041)\n","Processed 764285508-stool1_revised_C995492_1_gene152378: non-AMP/OTHERS (Probabilities: 0.9884, 0.0033, 0.0083)\n","Processed V1.CD34-0_GL0042016: Neuropeptide (Probabilities: 0.0161, 0.0152, 0.9687)\n","Processed V1.CD34-0_GL0042016: non-AMP/OTHERS (Probabilities: 0.9761, 0.0055, 0.0184)\n","Processed MH0117_GL0041413: non-AMP/OTHERS (Probabilities: 0.9910, 0.0034, 0.0055)\n","Processed MH0211_GL0040264: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed MH0177_GL0007234: non-AMP/OTHERS (Probabilities: 0.9355, 0.0129, 0.0516)\n","Processed V1.UC2-0_GL0065638: Neuropeptide (Probabilities: 0.1200, 0.0279, 0.8521)\n","Processed NLM031_GL0096614: Neuropeptide (Probabilities: 0.0796, 0.0337, 0.8867)\n","Processed MH0211_GL0023139: Neuropeptide (Probabilities: 0.0066, 0.0094, 0.9840)\n","Processed MH0217_GL0035863: non-AMP/OTHERS (Probabilities: 0.9917, 0.0036, 0.0047)\n","Processed MH0293_GL0090794: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0293_GL0090794: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed T2D-68A_GL0056640: non-AMP/OTHERS (Probabilities: 0.9906, 0.0044, 0.0050)\n","Processed MH0122_GL0047434: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0131_GL0004011: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0188_GL0104007: non-AMP/OTHERS (Probabilities: 0.9562, 0.0219, 0.0219)\n","Processed MH0188_GL0053702: non-AMP/OTHERS (Probabilities: 0.8858, 0.0385, 0.0757)\n","Processed V1.CD55-0_GL0259012: non-AMP/OTHERS (Probabilities: 0.9871, 0.0057, 0.0072)\n","Processed MH0227_GL0114479: non-AMP/OTHERS (Probabilities: 0.9754, 0.0099, 0.0147)\n","Processed O2.UC15-0_GL0043865: non-AMP/OTHERS (Probabilities: 0.9327, 0.0196, 0.0477)\n","Processed T2D-53A_GL0103379: non-AMP/OTHERS (Probabilities: 0.9885, 0.0054, 0.0061)\n","Processed MH0118_GL0121396: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0118_GL0121396: Neuropeptide (Probabilities: 0.0108, 0.0138, 0.9754)\n","Processed 763577454-stool1_revised_scaffold22315_2_gene10974: non-AMP/OTHERS (Probabilities: 0.9928, 0.0036, 0.0036)\n","Processed N020A_GL0023120: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9853)\n","Processed O2.UC28-2_GL0002087: Neuropeptide (Probabilities: 0.0078, 0.0093, 0.9829)\n","Processed 158499257-stool2_revised_C1325001_1_gene115353: non-AMP/OTHERS (Probabilities: 0.9908, 0.0032, 0.0060)\n","Processed DLF014_GL0043096: non-AMP/OTHERS (Probabilities: 0.9839, 0.0059, 0.0103)\n","Processed MH0332_GL0058383: non-AMP/OTHERS (Probabilities: 0.9915, 0.0044, 0.0041)\n","Processed MH0006_GL0220817: non-AMP/OTHERS (Probabilities: 0.9832, 0.0066, 0.0102)\n","Processed MH0203_GL0172920: non-AMP/OTHERS (Probabilities: 0.9722, 0.0197, 0.0081)\n","Processed MH0050_GL0034391: Neuropeptide (Probabilities: 0.1732, 0.0675, 0.7592)\n","Processed MH0011_GL0033287: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed MH0004_GL0022670: non-AMP/OTHERS (Probabilities: 0.7639, 0.0711, 0.1650)\n","Processed V1.FI34_GL0079347: non-AMP/OTHERS (Probabilities: 0.6929, 0.0345, 0.2726)\n","Processed V1.FI34_GL0079347: non-AMP/OTHERS (Probabilities: 0.5910, 0.0450, 0.3640)\n","Processed MH0022_GL0005735: Neuropeptide (Probabilities: 0.0081, 0.0092, 0.9827)\n","Processed O2.CD2-0_GL0075042: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed 763860675-stool1_revised_C1154688_1_gene140954: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed MH0415_GL0184174: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0006_GL0110176: non-AMP/OTHERS (Probabilities: 0.9908, 0.0046, 0.0046)\n","Processed 470146.COPCOM_01056: AMP (Probabilities: 0.0784, 0.9089, 0.0127)\n","Processed NLM029_GL0062158: non-AMP/OTHERS (Probabilities: 0.9756, 0.0103, 0.0142)\n","Processed O2.UC40-1_GL0110927: AMP (Probabilities: 0.1370, 0.8504, 0.0126)\n","Processed MH0119_GL0106827: Neuropeptide (Probabilities: 0.0693, 0.0276, 0.9031)\n","Processed MH0004_GL0013476: AMP (Probabilities: 0.0745, 0.9124, 0.0131)\n","Processed MH0157_GL0098365: non-AMP/OTHERS (Probabilities: 0.9910, 0.0052, 0.0038)\n","Processed MH0062_GL0041395: non-AMP/OTHERS (Probabilities: 0.9865, 0.0059, 0.0076)\n","Processed MH0450_GL0024861: non-AMP/OTHERS (Probabilities: 0.9867, 0.0053, 0.0079)\n","Processed O2.UC44-1_GL0058189: Neuropeptide (Probabilities: 0.2073, 0.0497, 0.7430)\n","Processed MH0071_GL0018535: non-AMP/OTHERS (Probabilities: 0.9756, 0.0059, 0.0185)\n","Processed MH0196_GL0160045: Neuropeptide (Probabilities: 0.0092, 0.0103, 0.9806)\n","Processed MH0062_GL0026707: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed O2.UC8-1_GL0049804: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed SZEY-78A_GL0068642: Neuropeptide (Probabilities: 0.0951, 0.0259, 0.8790)\n","Processed MH0002_GL0042649: Neuropeptide (Probabilities: 0.0190, 0.0131, 0.9678)\n","Processed MH0002_GL0000402: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed DLF003_GL0054309: non-AMP/OTHERS (Probabilities: 0.7506, 0.2116, 0.0378)\n","Processed MH0062_GL0001730: non-AMP/OTHERS (Probabilities: 0.9856, 0.0055, 0.0090)\n","Processed MH0284_GL0134959: Neuropeptide (Probabilities: 0.0084, 0.0088, 0.9828)\n","Processed MH0001_GL0003056: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed T2D-41A_GL0009930: non-AMP/OTHERS (Probabilities: 0.9934, 0.0035, 0.0032)\n","Processed O2.UC17-2_GL0005547: non-AMP/OTHERS (Probabilities: 0.9916, 0.0035, 0.0050)\n","Processed MH0089_GL0033080: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed MH0360_GL0085446: non-AMP/OTHERS (Probabilities: 0.9926, 0.0034, 0.0040)\n","Processed 457412.RSAG_00053: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0034)\n","Processed ED9A_GL0087076: non-AMP/OTHERS (Probabilities: 0.9914, 0.0040, 0.0046)\n","Processed MH0086_GL0009216: non-AMP/OTHERS (Probabilities: 0.9918, 0.0040, 0.0042)\n","Processed MH0435_GL0143716: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed MH0205_GL0011763: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed V1.CD6-4_GL0055248: non-AMP/OTHERS (Probabilities: 0.9808, 0.0132, 0.0060)\n","Processed O2.UC31-0_GL0124055: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0036)\n","Processed MH0006_GL0057816: Neuropeptide (Probabilities: 0.0101, 0.0104, 0.9795)\n","Processed MH0050_GL0056108: non-AMP/OTHERS (Probabilities: 0.9877, 0.0058, 0.0065)\n","Processed MH0277_GL0002017: Neuropeptide (Probabilities: 0.0119, 0.0108, 0.9773)\n","Processed MH0358_GL0049685: Neuropeptide (Probabilities: 0.0057, 0.0089, 0.9853)\n","Processed DLM001_GL0028724: non-AMP/OTHERS (Probabilities: 0.9869, 0.0062, 0.0069)\n","Processed MH0458_GL0000637: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0006_GL0090615: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0355_GL0038120: non-AMP/OTHERS (Probabilities: 0.9595, 0.0107, 0.0298)\n","Processed MH0355_GL0038120: non-AMP/OTHERS (Probabilities: 0.9903, 0.0051, 0.0047)\n","Processed MH0188_GL0088109: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0036)\n","Processed MH0188_GL0088109: non-AMP/OTHERS (Probabilities: 0.9719, 0.0128, 0.0152)\n","Processed MH0342_GL0136612: non-AMP/OTHERS (Probabilities: 0.9923, 0.0043, 0.0034)\n","Processed MH0147_GL0101414: non-AMP/OTHERS (Probabilities: 0.9674, 0.0096, 0.0229)\n","Processed MH0193_GL0120581: non-AMP/OTHERS (Probabilities: 0.9913, 0.0047, 0.0040)\n","Processed V1.CD10-0_GL0018126: Neuropeptide (Probabilities: 0.2279, 0.2659, 0.5063)\n","Processed O2.UC8-0_GL0155520: non-AMP/OTHERS (Probabilities: 0.9925, 0.0046, 0.0029)\n","Processed MH0055_GL0033606: non-AMP/OTHERS (Probabilities: 0.7728, 0.1204, 0.1068)\n","Processed NOM009_GL0041857: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0062_GL0028594: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed V1.UC30-0_GL0100642: Neuropeptide (Probabilities: 0.0264, 0.0155, 0.9580)\n","Processed V1.CD2-0-PN_GL0028276: non-AMP/OTHERS (Probabilities: 0.9632, 0.0136, 0.0232)\n","Processed sp|Q92485|ASM3B_HUMAN: non-AMP/OTHERS (Probabilities: 0.9656, 0.0120, 0.0224)\n","Processed MH0011_GL0087251: non-AMP/OTHERS (Probabilities: 0.9880, 0.0054, 0.0065)\n","Processed 160178356-stool1_revised_scaffold11920_1_gene76369: non-AMP/OTHERS (Probabilities: 0.9822, 0.0078, 0.0099)\n","Processed MH0230_GL0066816: non-AMP/OTHERS (Probabilities: 0.9925, 0.0037, 0.0038)\n","Processed V1.CD12-3_GL0112792: Neuropeptide (Probabilities: 0.0069, 0.0087, 0.9844)\n","Processed MH0059_GL0015459: non-AMP/OTHERS (Probabilities: 0.6397, 0.2098, 0.1505)\n","Processed MH0045_GL0017496: Neuropeptide (Probabilities: 0.0646, 0.0450, 0.8904)\n","Processed O2.UC8-0_GL0133121: Neuropeptide (Probabilities: 0.0386, 0.0300, 0.9314)\n","Processed O2.UC32-2_GL0075315: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed MH0089_GL0033080: Neuropeptide (Probabilities: 0.0074, 0.0088, 0.9838)\n","Processed T2D-41A_GL0009930: Neuropeptide (Probabilities: 0.0099, 0.0097, 0.9804)\n","Processed 457412.RSAG_00053: Neuropeptide (Probabilities: 0.0090, 0.0094, 0.9816)\n","Processed MH0211_GL0034404: Neuropeptide (Probabilities: 0.0097, 0.0107, 0.9796)\n","Processed MH0031_GL0010460: Neuropeptide (Probabilities: 0.0115, 0.0105, 0.9780)\n","Processed MH0205_GL0011763: Neuropeptide (Probabilities: 0.3402, 0.0648, 0.5950)\n","Processed O2.UC46-2_GL0096495: Neuropeptide (Probabilities: 0.0197, 0.0148, 0.9655)\n","Processed MH0001_GL0003056: Neuropeptide (Probabilities: 0.2691, 0.0495, 0.6813)\n","Processed DLF013_GL0043571: Neuropeptide (Probabilities: 0.0562, 0.0264, 0.9174)\n","Processed V1.FI36_GL0253878: non-AMP/OTHERS (Probabilities: 0.4997, 0.0346, 0.4657)\n","Processed 765620695-stool1_revised_scaffold26148_1_gene110238: non-AMP/OTHERS (Probabilities: 0.9738, 0.0122, 0.0140)\n","Processed 547043.BIFPSEUDO_04210: AMP (Probabilities: 0.0726, 0.9131, 0.0143)\n","Processed MH0252_GL0113441: non-AMP/OTHERS (Probabilities: 0.9751, 0.0072, 0.0177)\n","Processed V1.FI14_GL0214940: non-AMP/OTHERS (Probabilities: 0.9797, 0.0066, 0.0136)\n","Processed 158944319-stool2_revised_C1429391_1_gene46428: Neuropeptide (Probabilities: 0.0078, 0.0099, 0.9823)\n","Processed O2.UC48-0_GL0289865: non-AMP/OTHERS (Probabilities: 0.9908, 0.0055, 0.0038)\n","Processed V1.UC5-3_GL0098783: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed V1.UC5-3_GL0098783: non-AMP/OTHERS (Probabilities: 0.9929, 0.0036, 0.0036)\n","Processed MH0333_GL0078441: non-AMP/OTHERS (Probabilities: 0.9561, 0.0253, 0.0186)\n","Processed V1.UC39-4_GL0076825: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 158479027-stool1_revised_scaffold21726_1_gene110618: Neuropeptide (Probabilities: 0.0107, 0.0112, 0.9781)\n","Processed MH0061_GL0047301: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0074_GL0030149: non-AMP/OTHERS (Probabilities: 0.9572, 0.0224, 0.0205)\n","Processed MH0290_GL0003592: non-AMP/OTHERS (Probabilities: 0.9343, 0.0229, 0.0428)\n","Processed DOF004_GL0046385: Neuropeptide (Probabilities: 0.3675, 0.1521, 0.4804)\n","Processed BGI-06A_GL0072944: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed sp|P05164|PERM_HUMAN: Neuropeptide (Probabilities: 0.0073, 0.0093, 0.9835)\n","Processed sp|P27487|DPP4_HUMAN: Neuropeptide (Probabilities: 0.0079, 0.0099, 0.9822)\n","Processed 159632143-stool1_revised_scaffold17083_1_gene147977: non-AMP/OTHERS (Probabilities: 0.9931, 0.0039, 0.0030)\n","Processed N085A_GL0065028: Neuropeptide (Probabilities: 0.1292, 0.0339, 0.8369)\n","Processed MH0006_GL0017869: Neuropeptide (Probabilities: 0.1930, 0.0497, 0.7572)\n","Processed MH0315_GL0055634: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed MH0092_GL0029477: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9849)\n","Processed MH0341_GL0093732: non-AMP/OTHERS (Probabilities: 0.9630, 0.0090, 0.0279)\n","Processed MH0293_GL0116698: Neuropeptide (Probabilities: 0.0534, 0.0430, 0.9036)\n","Processed MH0002_GL0009830: Neuropeptide (Probabilities: 0.0077, 0.0091, 0.9832)\n","Processed 158458797-stool1_revised_scaffold25252_2_gene79145: non-AMP/OTHERS (Probabilities: 0.9918, 0.0051, 0.0031)\n","Processed MH0114_GL0049218: non-AMP/OTHERS (Probabilities: 0.9911, 0.0044, 0.0045)\n","Processed MH0003_GL0056148: Neuropeptide (Probabilities: 0.0474, 0.0657, 0.8869)\n","Processed 763496533-stool2_revised_C1189528_1_gene182949: Neuropeptide (Probabilities: 0.0105, 0.0141, 0.9754)\n","Processed V1.UC52-0_GL0150183: non-AMP/OTHERS (Probabilities: 0.4997, 0.3381, 0.1622)\n","Processed 765074482-stool1_revised_scaffold42053_1_gene199697: Neuropeptide (Probabilities: 0.0119, 0.0144, 0.9737)\n","Processed sp|P27797|CALR_HUMAN: Neuropeptide (Probabilities: 0.0070, 0.0095, 0.9835)\n","Processed MH0337_GL0100466: non-AMP/OTHERS (Probabilities: 0.9513, 0.0355, 0.0132)\n","Processed O2.UC20-1_GL0176539: non-AMP/OTHERS (Probabilities: 0.9199, 0.0383, 0.0417)\n","Processed MH0028_GL0067833: non-AMP/OTHERS (Probabilities: 0.9890, 0.0054, 0.0055)\n","Processed MH0059_GL0033088: Neuropeptide (Probabilities: 0.0488, 0.0284, 0.9228)\n","Processed MH0092_GL0026398: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed MH0290_GL0081558: Neuropeptide (Probabilities: 0.1150, 0.0220, 0.8630)\n","Processed MH0290_GL0081558: non-AMP/OTHERS (Probabilities: 0.9865, 0.0040, 0.0094)\n","Processed MH0196_GL0042240: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0003_GL0088975: non-AMP/OTHERS (Probabilities: 0.9934, 0.0032, 0.0034)\n","Processed MH0010_GL0009891: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed 411479.BACUNI_02644: Neuropeptide (Probabilities: 0.0523, 0.0189, 0.9288)\n","Processed MH0045_GL0013273: Neuropeptide (Probabilities: 0.0069, 0.0085, 0.9846)\n","Processed O2.UC28-2_GL0211970: non-AMP/OTHERS (Probabilities: 0.9894, 0.0043, 0.0063)\n","Processed MH0016_GL0068439: non-AMP/OTHERS (Probabilities: 0.8621, 0.0406, 0.0973)\n","Processed MH0055_GL0028521: Neuropeptide (Probabilities: 0.0066, 0.0087, 0.9847)\n","Processed 763982056-stool2_revised_C405479_1_gene59550: Neuropeptide (Probabilities: 0.4293, 0.0893, 0.4814)\n","Processed O2.UC6-1_GL0157896: Neuropeptide (Probabilities: 0.2550, 0.0353, 0.7097)\n","Processed MH0011_GL0034486: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed 764143897-stool1_revised_scaffold33057_1_gene17514: non-AMP/OTHERS (Probabilities: 0.9903, 0.0053, 0.0044)\n","Processed 717962.CC1_34350: non-AMP/OTHERS (Probabilities: 0.9805, 0.0118, 0.0077)\n","Processed 763901136-stool1_revised_C1062383_1_gene48952: non-AMP/OTHERS (Probabilities: 0.9911, 0.0032, 0.0058)\n","Processed O2.UC17-1_GL0099902: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0131_GL0133151: non-AMP/OTHERS (Probabilities: 0.9927, 0.0034, 0.0039)\n","Processed MH0368_GL0055974: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0095_GL0046103: Neuropeptide (Probabilities: 0.3110, 0.0763, 0.6126)\n","Processed 764447348-stool2_revised_scaffold31028_1_gene20691: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed 515619.EUBREC_1472: Neuropeptide (Probabilities: 0.0126, 0.0107, 0.9767)\n","Processed MH0433_GL0176712: Neuropeptide (Probabilities: 0.0068, 0.0085, 0.9847)\n","Processed MH0003_GL0089792: Neuropeptide (Probabilities: 0.1752, 0.0720, 0.7529)\n","Processed MH0011_GL0004594: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0012_GL0200149: non-AMP/OTHERS (Probabilities: 0.9922, 0.0041, 0.0037)\n","Processed MH0002_GL0063912: non-AMP/OTHERS (Probabilities: 0.9924, 0.0042, 0.0033)\n","Processed MH0002_GL0063912: non-AMP/OTHERS (Probabilities: 0.9922, 0.0039, 0.0040)\n","Processed MH0039_GL0041357: non-AMP/OTHERS (Probabilities: 0.9732, 0.0224, 0.0044)\n","Processed MH0086_GL0014310: non-AMP/OTHERS (Probabilities: 0.8452, 0.1395, 0.0153)\n","Processed MH0195_GL0023297: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed V1.CD54-0_GL0084523: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed N064A_GL0039476: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0074_GL0043980: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0080_GL0067765: non-AMP/OTHERS (Probabilities: 0.9742, 0.0050, 0.0208)\n","Processed MH0003_GL0063425: non-AMP/OTHERS (Probabilities: 0.9622, 0.0306, 0.0072)\n","Processed MH0281_GL0077203: non-AMP/OTHERS (Probabilities: 0.7568, 0.0542, 0.1890)\n","Processed MH0071_GL0060761: non-AMP/OTHERS (Probabilities: 0.9610, 0.0208, 0.0182)\n","Processed V1.FI27_GL0139364: non-AMP/OTHERS (Probabilities: 0.9897, 0.0055, 0.0048)\n","Processed 158337416-stool1_revised_C1281344_1_gene79007: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed MH0183_GL0124311: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed O2.UC48-0_GL0270997: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed MH0188_GL0160360: Neuropeptide (Probabilities: 0.0098, 0.0108, 0.9793)\n","Processed MH0188_GL0067785: non-AMP/OTHERS (Probabilities: 0.9871, 0.0078, 0.0051)\n","Processed MH0197_GL0027033: Neuropeptide (Probabilities: 0.0073, 0.0086, 0.9841)\n","Processed MH0197_GL0027033: non-AMP/OTHERS (Probabilities: 0.9925, 0.0033, 0.0041)\n","Processed T2D-87A_GL0007628: non-AMP/OTHERS (Probabilities: 0.9550, 0.0104, 0.0346)\n","Processed O2.UC37-0_GL0012229: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9853)\n","Processed MH0026_GL0002842: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0008_GL0019596: non-AMP/OTHERS (Probabilities: 0.7284, 0.1213, 0.1504)\n","Processed DOF004_GL0080016: Neuropeptide (Probabilities: 0.0861, 0.0581, 0.8559)\n","Processed MH0088_GL0053882: non-AMP/OTHERS (Probabilities: 0.9797, 0.0161, 0.0042)\n","Processed V1.UC43-0_GL0134438: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed MH0002_GL0018323: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed sp|P08183|MDR1_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0162_GL0107380: Neuropeptide (Probabilities: 0.0220, 0.0179, 0.9601)\n","Processed MH0221_GL0028677: non-AMP/OTHERS (Probabilities: 0.9617, 0.0149, 0.0234)\n","Processed MH0188_GL0024227: non-AMP/OTHERS (Probabilities: 0.9932, 0.0032, 0.0037)\n","Processed MH0138_GL0022513: Neuropeptide (Probabilities: 0.0063, 0.0086, 0.9852)\n","Processed MH0138_GL0022513: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed O2.CD3-0-PT_GL0120399: AMP (Probabilities: 0.3178, 0.3907, 0.2915)\n","Processed MH0166_GL0050601: Neuropeptide (Probabilities: 0.3068, 0.1618, 0.5314)\n","Processed MH0003_GL0103217: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0034)\n","Processed DOM010_GL0088706: Neuropeptide (Probabilities: 0.0067, 0.0086, 0.9847)\n","Processed V1.UC58-0_GL0026568: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0286_GL0111197: non-AMP/OTHERS (Probabilities: 0.9917, 0.0041, 0.0043)\n","Processed V1.FI07_GL0062185: Neuropeptide (Probabilities: 0.0068, 0.0088, 0.9844)\n","Processed MH0333_GL0056429: Neuropeptide (Probabilities: 0.0068, 0.0088, 0.9844)\n","Processed MH0410_GL0017925: Neuropeptide (Probabilities: 0.0065, 0.0089, 0.9846)\n","Processed MH0113_GL0000286: non-AMP/OTHERS (Probabilities: 0.9932, 0.0035, 0.0033)\n","Processed O2.UC41-1_GL0005350: Neuropeptide (Probabilities: 0.4089, 0.0992, 0.4920)\n","Processed NOM028_GL0008171: Neuropeptide (Probabilities: 0.0101, 0.0114, 0.9785)\n","Processed V1.UC47-4_GL0131628: Neuropeptide (Probabilities: 0.0095, 0.0120, 0.9785)\n","Processed MH0277_GL0007840: non-AMP/OTHERS (Probabilities: 0.8484, 0.0393, 0.1123)\n","Processed MH0001_GL0005437: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0036)\n","Processed O2.UC53-2_GL0013344: Neuropeptide (Probabilities: 0.0060, 0.0090, 0.9850)\n","Processed O2.CD3-0-PT_GL0072639: Neuropeptide (Probabilities: 0.0935, 0.0299, 0.8766)\n","Processed MH0277_GL0037689: Neuropeptide (Probabilities: 0.0101, 0.0103, 0.9796)\n","Processed MH0002_GL0020653: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0036)\n","Processed O2.UC21-2_GL0003699: Neuropeptide (Probabilities: 0.0071, 0.0090, 0.9839)\n","Processed 763840445-stool2_revised_scaffold53604_1_gene29147: Neuropeptide (Probabilities: 0.0083, 0.0093, 0.9824)\n","Processed 159632143-stool1_revised_scaffold9477_1_gene148992: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed V1.UC16-0_GL0087226: non-AMP/OTHERS (Probabilities: 0.7234, 0.0180, 0.2586)\n","Processed MH0443_GL0073978: non-AMP/OTHERS (Probabilities: 0.9811, 0.0045, 0.0144)\n","Processed 457421.CBFG_05673: non-AMP/OTHERS (Probabilities: 0.7418, 0.0531, 0.2051)\n","Processed O2.UC2-1_GL0171936: Neuropeptide (Probabilities: 0.0458, 0.0212, 0.9330)\n","Processed MH0441_GL0004757: Neuropeptide (Probabilities: 0.0202, 0.0125, 0.9673)\n","Processed MH0006_GL0169975: Neuropeptide (Probabilities: 0.0437, 0.0426, 0.9137)\n","Processed MH0092_GL0017589: Neuropeptide (Probabilities: 0.0079, 0.0092, 0.9829)\n","Processed 367928.BAD_1197: Neuropeptide (Probabilities: 0.0791, 0.0311, 0.8897)\n","Processed MH0037_GL0019396: Neuropeptide (Probabilities: 0.0118, 0.0127, 0.9755)\n","Processed 552398.HMPREF0866_02762: Neuropeptide (Probabilities: 0.0083, 0.0104, 0.9813)\n","Processed T2D-195A_GL0059200: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed O2.UC2-1_GL0073958: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed O2.UC2-1_GL0073958: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9853)\n","Processed MH0236_GL0057333: non-AMP/OTHERS (Probabilities: 0.9926, 0.0038, 0.0035)\n","Processed MH0371_GL0002477: Neuropeptide (Probabilities: 0.0066, 0.0090, 0.9845)\n","Processed ED12A_GL0079344: Neuropeptide (Probabilities: 0.2427, 0.2440, 0.5133)\n","Processed MH0014_GL0005680: non-AMP/OTHERS (Probabilities: 0.9402, 0.0517, 0.0081)\n","Processed MH0108_GL0078573: non-AMP/OTHERS (Probabilities: 0.9912, 0.0038, 0.0050)\n","Processed 457396.CSBG_01633: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed T2D-139A_GL0031638: non-AMP/OTHERS (Probabilities: 0.8389, 0.0383, 0.1228)\n","Processed MH0310_GL0036108: non-AMP/OTHERS (Probabilities: 0.9875, 0.0055, 0.0070)\n","Processed O2.UC60-2_GL0083891: non-AMP/OTHERS (Probabilities: 0.9535, 0.0274, 0.0191)\n","Processed V1.FI08_GL0123573: Neuropeptide (Probabilities: 0.0084, 0.0093, 0.9823)\n","Processed V1.UC53-0_GL0115417: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0003_GL0044261: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0318_GL0138806: non-AMP/OTHERS (Probabilities: 0.4730, 0.1366, 0.3904)\n","Processed MH0092_GL0102248: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed O2.UC6-1_GL0157896: non-AMP/OTHERS (Probabilities: 0.9879, 0.0036, 0.0085)\n","Processed MH0155_GL0029986: non-AMP/OTHERS (Probabilities: 0.9228, 0.0337, 0.0436)\n","Processed MH0191_GL0107199: Neuropeptide (Probabilities: 0.0093, 0.0095, 0.9812)\n","Processed MH0023_GL0024479: non-AMP/OTHERS (Probabilities: 0.5899, 0.0333, 0.3769)\n","Processed MH0131_GL0108368: Neuropeptide (Probabilities: 0.0061, 0.0089, 0.9850)\n","Processed MH0006_GL0140949: non-AMP/OTHERS (Probabilities: 0.7696, 0.0293, 0.2011)\n","Processed O2.UC28-2_GL0048618: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0267_GL0117850: non-AMP/OTHERS (Probabilities: 0.9750, 0.0126, 0.0124)\n","Processed V1.CD1-0-PN_GL0033157: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0416_GL0219398: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0305_GL0074716: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9850)\n","Processed MH0086_GL0011887: non-AMP/OTHERS (Probabilities: 0.9873, 0.0043, 0.0084)\n","Processed MH0001_GL0019798: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed HT25A_GL0121540: non-AMP/OTHERS (Probabilities: 0.9679, 0.0190, 0.0131)\n","Processed MH0069_GL0060610: Neuropeptide (Probabilities: 0.0223, 0.0273, 0.9504)\n","Processed 765701615-stool1_revised_C1357082_1_gene106611: Neuropeptide (Probabilities: 0.0070, 0.0087, 0.9843)\n","Processed MH0337_GL0093201: AMP (Probabilities: 0.2976, 0.3685, 0.3338)\n","Processed SZEY-62A_GL0098562: non-AMP/OTHERS (Probabilities: 0.9926, 0.0046, 0.0028)\n","Processed MH0191_GL0033492: non-AMP/OTHERS (Probabilities: 0.9916, 0.0050, 0.0034)\n","Processed MH0003_GL0106972: non-AMP/OTHERS (Probabilities: 0.8591, 0.0224, 0.1185)\n","Processed 160158126-stool1_revised_C323048_1_gene45242: non-AMP/OTHERS (Probabilities: 0.9812, 0.0080, 0.0108)\n","Processed 159632143-stool1_revised_C952203_1_gene53988: non-AMP/OTHERS (Probabilities: 0.9608, 0.0186, 0.0207)\n","Processed T2D-113A_GL0141265: non-AMP/OTHERS (Probabilities: 0.7744, 0.0495, 0.1761)\n","Processed MH0400_GL0032534: non-AMP/OTHERS (Probabilities: 0.9591, 0.0257, 0.0152)\n","Processed MH0131_GL0044184: Neuropeptide (Probabilities: 0.0102, 0.0102, 0.9796)\n","Processed MH0131_GL0044184: Neuropeptide (Probabilities: 0.0131, 0.0116, 0.9753)\n","Processed HT25A_GL0084681: non-AMP/OTHERS (Probabilities: 0.9933, 0.0035, 0.0032)\n","Processed SZEY-69A_GL0027522: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0034)\n","Processed MH0014_GL0075909: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed O2.UC13-2_GL0016834: Neuropeptide (Probabilities: 0.0406, 0.0193, 0.9401)\n","Processed MH0010_GL0015260: Neuropeptide (Probabilities: 0.0091, 0.0095, 0.9814)\n","Processed MH0345_GL0045473: Neuropeptide (Probabilities: 0.0069, 0.0085, 0.9846)\n","Processed MH0230_GL0039709: Neuropeptide (Probabilities: 0.0116, 0.0118, 0.9766)\n","Processed MH0006_GL0154117: non-AMP/OTHERS (Probabilities: 0.9692, 0.0164, 0.0144)\n","Processed MH0402_GL0076304: Neuropeptide (Probabilities: 0.2747, 0.1349, 0.5904)\n","Processed MH0026_GL0024394: non-AMP/OTHERS (Probabilities: 0.9902, 0.0052, 0.0046)\n","Processed MH0026_GL0024394: non-AMP/OTHERS (Probabilities: 0.9922, 0.0042, 0.0037)\n","Processed MH0247_GL0077972: non-AMP/OTHERS (Probabilities: 0.9910, 0.0043, 0.0048)\n","Processed 862965.PARA_07540: Neuropeptide (Probabilities: 0.0103, 0.0106, 0.9791)\n","Processed MH0020_GL0072349: non-AMP/OTHERS (Probabilities: 0.5898, 0.1091, 0.3011)\n","Processed O2.UC6-1_GL0073139: Neuropeptide (Probabilities: 0.0126, 0.0110, 0.9764)\n","Processed O2.UC6-1_GL0073139: Neuropeptide (Probabilities: 0.0197, 0.0140, 0.9663)\n","Processed MH0062_GL0022255: Neuropeptide (Probabilities: 0.1350, 0.0302, 0.8348)\n","Processed DLF013_GL0001020: Neuropeptide (Probabilities: 0.0082, 0.0102, 0.9816)\n","Processed MH0014_GL0090940: Neuropeptide (Probabilities: 0.0063, 0.0086, 0.9851)\n","Processed O2.UC19-1_GL0068112: Neuropeptide (Probabilities: 0.0368, 0.0159, 0.9473)\n","Processed O2.UC19-1_GL0068112: non-AMP/OTHERS (Probabilities: 0.9674, 0.0095, 0.0231)\n","Processed O2.CD1-0-PT_GL0112159: Neuropeptide (Probabilities: 0.0305, 0.0208, 0.9487)\n","Processed N037A_GL0054348: non-AMP/OTHERS (Probabilities: 0.9920, 0.0036, 0.0044)\n","Processed MH0293_GL0038915: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0003_GL0088975: Neuropeptide (Probabilities: 0.0067, 0.0086, 0.9847)\n","Processed 435590.BVU_1005: Neuropeptide (Probabilities: 0.0130, 0.0160, 0.9710)\n","Processed 742723.HMPREF9477_00960: non-AMP/OTHERS (Probabilities: 0.9757, 0.0079, 0.0164)\n","Processed N005A_GL0071976: non-AMP/OTHERS (Probabilities: 0.9916, 0.0043, 0.0041)\n","Processed MH0356_GL0176196: Neuropeptide (Probabilities: 0.0083, 0.0095, 0.9822)\n","Processed MH0007_GL0009357: non-AMP/OTHERS (Probabilities: 0.9906, 0.0055, 0.0039)\n","Processed V1.FI34_GL0033764: non-AMP/OTHERS (Probabilities: 0.9923, 0.0038, 0.0039)\n","Processed MH0081_GL0028554: non-AMP/OTHERS (Probabilities: 0.9118, 0.0504, 0.0378)\n","Processed MH0020_GL0047245: non-AMP/OTHERS (Probabilities: 0.9858, 0.0080, 0.0062)\n","Processed MH0030_GL0013880: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0374_GL0117483: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0062_GL0064814: non-AMP/OTHERS (Probabilities: 0.8526, 0.0601, 0.0873)\n","Processed MH0008_GL0019484: non-AMP/OTHERS (Probabilities: 0.5324, 0.3111, 0.1565)\n","Processed MH0161_GL0153261: non-AMP/OTHERS (Probabilities: 0.9902, 0.0053, 0.0044)\n","Processed MH0196_GL0154004: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0265_GL0168160: non-AMP/OTHERS (Probabilities: 0.9915, 0.0038, 0.0047)\n","Processed V1.UC55-0_GL0036366: non-AMP/OTHERS (Probabilities: 0.9922, 0.0037, 0.0041)\n","Processed DLM018_GL0048484: Neuropeptide (Probabilities: 0.0111, 0.0124, 0.9765)\n","Processed DOM015_GL0051213: non-AMP/OTHERS (Probabilities: 0.9924, 0.0035, 0.0041)\n","Processed MH0088_GL0125320: non-AMP/OTHERS (Probabilities: 0.9928, 0.0036, 0.0036)\n","Processed MH0089_GL0055851: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed MH0003_GL0088975: non-AMP/OTHERS (Probabilities: 0.9935, 0.0034, 0.0031)\n","Processed MH0196_GL0152466: non-AMP/OTHERS (Probabilities: 0.9934, 0.0033, 0.0033)\n","Processed DOF004_GL0052864: non-AMP/OTHERS (Probabilities: 0.7904, 0.0551, 0.1545)\n","Processed MH0006_GL0016147: non-AMP/OTHERS (Probabilities: 0.9912, 0.0054, 0.0034)\n","Processed MH0355_GL0038120: Neuropeptide (Probabilities: 0.0089, 0.0119, 0.9793)\n","Processed 717962.CC1_34350: non-AMP/OTHERS (Probabilities: 0.9886, 0.0050, 0.0064)\n","Processed 158802708-stool2_revised_C990436_1_gene15357: non-AMP/OTHERS (Probabilities: 0.9861, 0.0065, 0.0074)\n","Processed MH0104_GL0018449: non-AMP/OTHERS (Probabilities: 0.9907, 0.0056, 0.0038)\n","Processed MH0008_GL0009154: non-AMP/OTHERS (Probabilities: 0.9910, 0.0058, 0.0032)\n","Processed MH0008_GL0009154: non-AMP/OTHERS (Probabilities: 0.9814, 0.0119, 0.0067)\n","Processed 765094712-stool1_revised_scaffold4546_1_gene48753: Neuropeptide (Probabilities: 0.1557, 0.1864, 0.6580)\n","Processed V1.CD19-0_GL0177345: non-AMP/OTHERS (Probabilities: 0.9904, 0.0045, 0.0050)\n","Processed V1.CD19-0_GL0177345: non-AMP/OTHERS (Probabilities: 0.9905, 0.0060, 0.0035)\n","Processed O2.UC9-1_GL0045684: non-AMP/OTHERS (Probabilities: 0.9916, 0.0058, 0.0026)\n","Processed MH0088_GL0096599: Neuropeptide (Probabilities: 0.2397, 0.0691, 0.6913)\n","Processed V1.FI05_GL0135770: non-AMP/OTHERS (Probabilities: 0.6416, 0.3073, 0.0511)\n","Processed MH0229_GL0202255: non-AMP/OTHERS (Probabilities: 0.9785, 0.0079, 0.0136)\n","Processed MH0006_GL0024527: non-AMP/OTHERS (Probabilities: 0.9893, 0.0034, 0.0073)\n","Processed O2.UC20-1_GL0091933: Neuropeptide (Probabilities: 0.0075, 0.0090, 0.9836)\n","Processed MH0359_GL0124487: Neuropeptide (Probabilities: 0.0071, 0.0088, 0.9841)\n","Processed MH0266_GL0088396: non-AMP/OTHERS (Probabilities: 0.9855, 0.0078, 0.0067)\n","Processed MH0325_GL0146643: non-AMP/OTHERS (Probabilities: 0.9867, 0.0054, 0.0079)\n","Processed sp|P16233|LIPP_HUMAN: non-AMP/OTHERS (Probabilities: 0.8724, 0.0449, 0.0826)\n","Processed MH0239_GL0140353: non-AMP/OTHERS (Probabilities: 0.8814, 0.0356, 0.0830)\n","Processed MH0062_GL0003852: non-AMP/OTHERS (Probabilities: 0.9745, 0.0082, 0.0173)\n","Processed MH0247_GL0054734: Neuropeptide (Probabilities: 0.0093, 0.0099, 0.9808)\n","Processed V1.FI05_GL0105060: non-AMP/OTHERS (Probabilities: 0.8699, 0.0209, 0.1092)\n","Processed MH0244_GL0115989: Neuropeptide (Probabilities: 0.0074, 0.0090, 0.9836)\n","Processed 158742018-stool1_revised_C808427_1_gene118421: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0038)\n","Processed MH0054_GL0053143: non-AMP/OTHERS (Probabilities: 0.9920, 0.0038, 0.0042)\n","Processed T2D-19A_GL0114267: non-AMP/OTHERS (Probabilities: 0.9903, 0.0066, 0.0031)\n","Processed V1.FI03_GL0021058: non-AMP/OTHERS (Probabilities: 0.8973, 0.0137, 0.0889)\n","Processed V1.FI03_GL0021058: Neuropeptide (Probabilities: 0.0324, 0.0240, 0.9436)\n","Processed 763496533-stool2_revised_scaffold46824_1_gene23000: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0006_GL0219013: non-AMP/OTHERS (Probabilities: 0.9574, 0.0206, 0.0221)\n","Processed 411481.BIFADO_02111: Neuropeptide (Probabilities: 0.0069, 0.0085, 0.9846)\n","Processed sp|A8K7I4|CLCA1_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed NOF005_GL0069305: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed N013A_GL0019255: non-AMP/OTHERS (Probabilities: 0.9604, 0.0157, 0.0239)\n","Processed V1.CD19-0_GL0177345: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0131_GL0142784: Neuropeptide (Probabilities: 0.2946, 0.0401, 0.6653)\n","Processed MH0211_GL0148886: non-AMP/OTHERS (Probabilities: 0.9456, 0.0147, 0.0397)\n","Processed 518634.BIFBRE_04338: non-AMP/OTHERS (Probabilities: 0.9626, 0.0217, 0.0157)\n","Processed V1.CD42-0_GL0129731: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed V1.UC13-3_GL0108099: Neuropeptide (Probabilities: 0.0205, 0.0140, 0.9655)\n","Processed MH0211_GL0140994: Neuropeptide (Probabilities: 0.0272, 0.0145, 0.9583)\n","Processed MH0027_GL0025484: non-AMP/OTHERS (Probabilities: 0.9779, 0.0123, 0.0099)\n","Processed DOF008_GL0025936: non-AMP/OTHERS (Probabilities: 0.6866, 0.0295, 0.2839)\n","Processed MH0092_GL0077945: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0250_GL0153477: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed sp|Q8WUM4|PDC6I_HUMAN: Neuropeptide (Probabilities: 0.1421, 0.1019, 0.7560)\n","Processed V1.UC26-4_GL0192052: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9851)\n","Processed MH0012_GL0085114: Neuropeptide (Probabilities: 0.0529, 0.0240, 0.9232)\n","Processed MH0009_GL0054881: Neuropeptide (Probabilities: 0.0066, 0.0086, 0.9848)\n","Processed 518634.BIFBRE_04992: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed MH0073_GL0005556: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9850)\n","Processed 367928.BAD_0874: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9851)\n","Processed O2.CD2-0-PT_GL0075729: Neuropeptide (Probabilities: 0.0077, 0.0095, 0.9828)\n","Processed V1.UC32-0_GL0187132: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed DOM024_GL0018259: non-AMP/OTHERS (Probabilities: 0.8494, 0.0543, 0.0963)\n","Processed MH0375_GL0058231: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed O2.UC22-1_GL0014828: Neuropeptide (Probabilities: 0.0061, 0.0086, 0.9853)\n","Processed 1042403.BALAC2494_00010: Neuropeptide (Probabilities: 0.3225, 0.0343, 0.6433)\n","Processed 159207311-stool2_revised_C1433448_1_gene39941: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed sp|Q16819|MEP1A_HUMAN: Neuropeptide (Probabilities: 0.3133, 0.0841, 0.6026)\n","Processed MH0371_GL0046682: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0012_GL0007537: non-AMP/OTHERS (Probabilities: 0.9924, 0.0041, 0.0035)\n","Processed MH0006_GL0057816: AMP (Probabilities: 0.3099, 0.6647, 0.0254)\n","Processed MH0345_GL0088699: non-AMP/OTHERS (Probabilities: 0.9912, 0.0033, 0.0055)\n","Processed DOF004_GL0078268: non-AMP/OTHERS (Probabilities: 0.9865, 0.0060, 0.0074)\n","Processed MH0233_GL0020760: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed V1.FI27_GL0176797: non-AMP/OTHERS (Probabilities: 0.9916, 0.0040, 0.0044)\n","Processed sp|Q9Y6R7|FCGBP_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0003_GL0025445: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed MH0008_GL0033516: non-AMP/OTHERS (Probabilities: 0.9035, 0.0129, 0.0836)\n","Processed MH0009_GL0041096: Neuropeptide (Probabilities: 0.0540, 0.0198, 0.9261)\n","Processed SZEY-41A_GL0063124: Neuropeptide (Probabilities: 0.0084, 0.0094, 0.9822)\n","Processed T2D-132A_GL0034521: non-AMP/OTHERS (Probabilities: 0.9796, 0.0066, 0.0138)\n","Processed MH0221_GL0031690: non-AMP/OTHERS (Probabilities: 0.9518, 0.0070, 0.0412)\n","Processed sp|P35237|SPB6_HUMAN: non-AMP/OTHERS (Probabilities: 0.6465, 0.0293, 0.3242)\n","Processed MH0069_GL0063799: non-AMP/OTHERS (Probabilities: 0.9819, 0.0068, 0.0114)\n","Processed 160319967-stool1_revised_C562686_1_gene31843: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed V1.FI26_GL0069248: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed 159227541-stool2_revised_scaffold26509_1_gene79822: Neuropeptide (Probabilities: 0.0242, 0.0158, 0.9600)\n","Processed MH0418_GL0050645: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed O2.UC3-0_GL0122518: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0131_GL0151752: Neuropeptide (Probabilities: 0.1963, 0.0997, 0.7040)\n","Processed DOF012_GL0009633: non-AMP/OTHERS (Probabilities: 0.9915, 0.0051, 0.0034)\n","Processed MH0277_GL0029001: non-AMP/OTHERS (Probabilities: 0.9875, 0.0034, 0.0091)\n","Processed MH0024_GL0029752: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed MH0101_GL0021414: non-AMP/OTHERS (Probabilities: 0.9882, 0.0047, 0.0071)\n","Processed MH0287_GL0075705: non-AMP/OTHERS (Probabilities: 0.7058, 0.0818, 0.2125)\n","Processed MH0131_GL0003171: non-AMP/OTHERS (Probabilities: 0.8703, 0.0291, 0.1006)\n","Processed MH0006_GL0095533: Neuropeptide (Probabilities: 0.0062, 0.0088, 0.9850)\n","Processed O2.UC20-1_GL0134067: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0089_GL0081699: non-AMP/OTHERS (Probabilities: 0.9891, 0.0057, 0.0052)\n","Processed MH0196_GL0230468: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed sp|P29508|SPB3_HUMAN: non-AMP/OTHERS (Probabilities: 0.9804, 0.0041, 0.0155)\n","Processed O2.UC31-2_GL0067716: Neuropeptide (Probabilities: 0.0063, 0.0088, 0.9850)\n","Processed 445972.ANACOL_01426: Neuropeptide (Probabilities: 0.0066, 0.0085, 0.9848)\n","Processed V1.CD27-0_GL0031092: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9850)\n","Processed MH0003_GL0041541: Neuropeptide (Probabilities: 0.0066, 0.0088, 0.9845)\n","Processed 764143897-stool1_revised_scaffold622_3_gene45079: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0446_GL0161835: non-AMP/OTHERS (Probabilities: 0.9921, 0.0038, 0.0041)\n","Processed MH0293_GL0021553: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed MH0441_GL0205856: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0203_GL0165115: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed BGI-28A_GL0083427: non-AMP/OTHERS (Probabilities: 0.9922, 0.0034, 0.0045)\n","Processed MH0010_GL0005244: non-AMP/OTHERS (Probabilities: 0.9601, 0.0057, 0.0342)\n","Processed MH0211_GL0010425: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed V1.CD41-0_GL0143792: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed O2.UC35-0_GL0046532: Neuropeptide (Probabilities: 0.0153, 0.0147, 0.9700)\n","Processed MH0319_GL0003871: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed T2D-135A_GL0121453: Neuropeptide (Probabilities: 0.2091, 0.0613, 0.7296)\n","Processed MH0198_GL0045866: non-AMP/OTHERS (Probabilities: 0.9924, 0.0041, 0.0035)\n","Processed MH0045_GL0023728: Neuropeptide (Probabilities: 0.0905, 0.0197, 0.8898)\n","Processed MH0045_GL0023728: non-AMP/OTHERS (Probabilities: 0.6352, 0.0308, 0.3340)\n","Processed V1.UC58-0_GL0003356: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed O2.UC7-1_GL0008287: non-AMP/OTHERS (Probabilities: 0.9821, 0.0133, 0.0046)\n","Processed T2D-178A_GL0036531: non-AMP/OTHERS (Probabilities: 0.9919, 0.0046, 0.0035)\n","Processed V1.UC42-0_GL0130017: non-AMP/OTHERS (Probabilities: 0.9923, 0.0037, 0.0040)\n","Processed MH0002_GL0000402: Neuropeptide (Probabilities: 0.0389, 0.0173, 0.9438)\n","Processed MH0062_GL0007643: Neuropeptide (Probabilities: 0.3085, 0.0235, 0.6680)\n","Processed MH0412_GL0224798: non-AMP/OTHERS (Probabilities: 0.9716, 0.0138, 0.0146)\n","Processed T2D-11A_GL0028727: non-AMP/OTHERS (Probabilities: 0.9921, 0.0037, 0.0042)\n","Processed MH0131_GL0105443: non-AMP/OTHERS (Probabilities: 0.9897, 0.0039, 0.0064)\n","Processed 411476.BACOVA_04505: non-AMP/OTHERS (Probabilities: 0.6410, 0.3226, 0.0365)\n","Processed MH0003_GL0109718: non-AMP/OTHERS (Probabilities: 0.9639, 0.0137, 0.0224)\n","Processed MH0203_GL0207775: Neuropeptide (Probabilities: 0.0127, 0.0105, 0.9769)\n","Processed MH0203_GL0207775: non-AMP/OTHERS (Probabilities: 0.5004, 0.0240, 0.4756)\n","Processed 159207311-stool2_revised_C1404759_1_gene171515: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed BGI002A_GL0118416: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9851)\n","Processed MH0117_GL0052773: non-AMP/OTHERS (Probabilities: 0.9880, 0.0078, 0.0041)\n","Processed 764062976-stool2_revised_scaffold67_1_gene60912: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9851)\n","Processed MH0006_GL0082463: non-AMP/OTHERS (Probabilities: 0.8909, 0.0391, 0.0700)\n","Processed MH0222_GL0076490: Neuropeptide (Probabilities: 0.0070, 0.0086, 0.9844)\n","Processed MH0406_GL0030612: Neuropeptide (Probabilities: 0.0141, 0.0159, 0.9699)\n","Processed sp|P12273|PIP_HUMAN: non-AMP/OTHERS (Probabilities: 0.7855, 0.1710, 0.0435)\n","Processed sp|P12273|PIP_HUMAN: non-AMP/OTHERS (Probabilities: 0.9607, 0.0123, 0.0271)\n","Processed O2.UC27-0_GL0107877: Neuropeptide (Probabilities: 0.0069, 0.0090, 0.9841)\n","Processed 469588.HMPREF0102_00212: non-AMP/OTHERS (Probabilities: 0.9917, 0.0037, 0.0046)\n","Processed MH0101_GL0077103: non-AMP/OTHERS (Probabilities: 0.5971, 0.1221, 0.2808)\n","Processed MH0137_GL0094047: non-AMP/OTHERS (Probabilities: 0.9915, 0.0043, 0.0041)\n","Processed 159207311-stool2_revised_C1433448_1_gene39941: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0347_GL0013304: Neuropeptide (Probabilities: 0.0077, 0.0088, 0.9835)\n","Processed O2.UC14-1_GL0091402: AMP (Probabilities: 0.4459, 0.5447, 0.0094)\n","Processed O2.UC11-1_GL0068784: non-AMP/OTHERS (Probabilities: 0.9804, 0.0164, 0.0032)\n","Processed MH0014_GL0099017: AMP (Probabilities: 0.2163, 0.6229, 0.1608)\n","Processed V1.UC38-4_GL0144905: non-AMP/OTHERS (Probabilities: 0.8412, 0.1341, 0.0247)\n","Processed MH0002_GL0066314: Neuropeptide (Probabilities: 0.0545, 0.0345, 0.9110)\n","Processed MH0019_GL0019578: non-AMP/OTHERS (Probabilities: 0.9834, 0.0042, 0.0123)\n","Processed MH0002_GL0001425: Neuropeptide (Probabilities: 0.0149, 0.0151, 0.9700)\n","Processed MH0139_GL0041455: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0008_GL0036612: Neuropeptide (Probabilities: 0.0062, 0.0087, 0.9851)\n","Processed MH0131_GL0093405: non-AMP/OTHERS (Probabilities: 0.8631, 0.0185, 0.1185)\n","Processed MH0006_GL0209486: non-AMP/OTHERS (Probabilities: 0.9676, 0.0203, 0.0121)\n","Processed BGI-06A_GL0052462: non-AMP/OTHERS (Probabilities: 0.8754, 0.0501, 0.0745)\n","Processed MH0170_GL0056715: non-AMP/OTHERS (Probabilities: 0.9906, 0.0049, 0.0046)\n","Processed MH0006_GL0219552: non-AMP/OTHERS (Probabilities: 0.9422, 0.0244, 0.0334)\n","Processed MH0005_GL0016860: non-AMP/OTHERS (Probabilities: 0.9882, 0.0077, 0.0041)\n","Processed MH0442_GL0137318: Neuropeptide (Probabilities: 0.4340, 0.0633, 0.5027)\n","Processed MH0262_GL0074290: non-AMP/OTHERS (Probabilities: 0.9792, 0.0075, 0.0133)\n","Processed 435590.BVU_1005: Neuropeptide (Probabilities: 0.4168, 0.1600, 0.4232)\n","Processed O2.UC2-0_GL0025716: Neuropeptide (Probabilities: 0.0063, 0.0087, 0.9850)\n","Processed MH0341_GL0072009: non-AMP/OTHERS (Probabilities: 0.8829, 0.1012, 0.0159)\n","Processed MH0147_GL0108957: non-AMP/OTHERS (Probabilities: 0.9176, 0.0401, 0.0423)\n","Processed MH0074_GL0059889: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0277_GL0042679: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0014_GL0099017: non-AMP/OTHERS (Probabilities: 0.9884, 0.0046, 0.0069)\n","Processed MH0016_GL0084603: non-AMP/OTHERS (Probabilities: 0.6252, 0.3082, 0.0666)\n","Processed O2.UC48-1_GL0076755: Neuropeptide (Probabilities: 0.0068, 0.0087, 0.9845)\n","Processed 411476.BACOVA_04392: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed sp|P29622|KAIN_HUMAN: Neuropeptide (Probabilities: 0.0484, 0.0317, 0.9199)\n","Processed MH0166_GL0001729: Neuropeptide (Probabilities: 0.3479, 0.0814, 0.5707)\n","Processed sp|Q9Y6X5|ENPP4_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0277_GL0033217: non-AMP/OTHERS (Probabilities: 0.9839, 0.0127, 0.0034)\n","Processed MH0191_GL0127010: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed T2D-76A_GL0038848: Neuropeptide (Probabilities: 0.0072, 0.0088, 0.9839)\n","Processed O2.UC6-1_GL0020061: non-AMP/OTHERS (Probabilities: 0.9924, 0.0030, 0.0046)\n","Processed MH0343_GL0037396: non-AMP/OTHERS (Probabilities: 0.9928, 0.0031, 0.0041)\n","Processed V1.UC11-0_GL0085451: non-AMP/OTHERS (Probabilities: 0.9878, 0.0043, 0.0079)\n","Processed V1.UC11-0_GL0085451: non-AMP/OTHERS (Probabilities: 0.9893, 0.0048, 0.0058)\n","Processed DLF013_GL0021448: Neuropeptide (Probabilities: 0.0091, 0.0097, 0.9812)\n","Processed DLF013_GL0021448: Neuropeptide (Probabilities: 0.0084, 0.0093, 0.9823)\n","Processed MH0003_GL0047894: Neuropeptide (Probabilities: 0.0087, 0.0094, 0.9819)\n","Processed MH0003_GL0047894: Neuropeptide (Probabilities: 0.0081, 0.0091, 0.9828)\n","Processed O2.UC9-1_GL0085814: Neuropeptide (Probabilities: 0.0093, 0.0098, 0.9809)\n","Processed V1.FI35_GL0047142: non-AMP/OTHERS (Probabilities: 0.9816, 0.0037, 0.0147)\n","Processed MH0205_GL0144892: non-AMP/OTHERS (Probabilities: 0.9913, 0.0052, 0.0035)\n","Processed tr|K7ERL9|K7ERL9_HUMAN: Neuropeptide (Probabilities: 0.0327, 0.0204, 0.9469)\n","Processed MH0356_GL0140244: Neuropeptide (Probabilities: 0.0078, 0.0088, 0.9834)\n","Processed 206672.BL1295: Neuropeptide (Probabilities: 0.0082, 0.0088, 0.9830)\n","Processed V1.UC50-1_GL0064402: non-AMP/OTHERS (Probabilities: 0.9927, 0.0032, 0.0041)\n","Processed MH0143_GL0153122: non-AMP/OTHERS (Probabilities: 0.9929, 0.0033, 0.0038)\n","Processed N032A_GL0048212: non-AMP/OTHERS (Probabilities: 0.9924, 0.0031, 0.0045)\n","Processed MH0055_GL0014387: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed MH0221_GL0048612: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed V1.UC38-0_GL0012021: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed MH0272_GL0233897: non-AMP/OTHERS (Probabilities: 0.9929, 0.0032, 0.0039)\n","Processed MH0420_GL0007634: non-AMP/OTHERS (Probabilities: 0.9925, 0.0032, 0.0043)\n","Processed MH0433_GL0193889: non-AMP/OTHERS (Probabilities: 0.9928, 0.0031, 0.0041)\n","Processed O2.UC30-2_GL0065804: Neuropeptide (Probabilities: 0.0075, 0.0089, 0.9836)\n","Processed MH0367_GL0121791: Neuropeptide (Probabilities: 0.2814, 0.0459, 0.6727)\n","Processed 158337416-stool2_revised_scaffold16676_1_gene161551: Neuropeptide (Probabilities: 0.0074, 0.0090, 0.9836)\n","Processed 435590.BVU_0563: Neuropeptide (Probabilities: 0.0067, 0.0088, 0.9846)\n","Processed MH0187_GL0062282: non-AMP/OTHERS (Probabilities: 0.9869, 0.0041, 0.0090)\n","Processed 518634.BIFBRE_03753: Neuropeptide (Probabilities: 0.0094, 0.0097, 0.9809)\n","Processed MH0131_GL0004010: non-AMP/OTHERS (Probabilities: 0.9735, 0.0191, 0.0074)\n","Processed MH0356_GL0057914: non-AMP/OTHERS (Probabilities: 0.9842, 0.0080, 0.0078)\n","Processed MH0014_GL0104218: Neuropeptide (Probabilities: 0.0168, 0.0127, 0.9705)\n","Processed MH0004_GL0026812: Neuropeptide (Probabilities: 0.0062, 0.0090, 0.9848)\n","Processed MH0154_GL0028983: non-AMP/OTHERS (Probabilities: 0.9924, 0.0036, 0.0040)\n","Processed MH0006_GL0174077: non-AMP/OTHERS (Probabilities: 0.9920, 0.0030, 0.0050)\n","Processed MH0006_GL0174077: non-AMP/OTHERS (Probabilities: 0.9923, 0.0036, 0.0041)\n","Processed MH0401_GL0018868: non-AMP/OTHERS (Probabilities: 0.9924, 0.0043, 0.0034)\n","Processed V1.UC33-0_GL0139214: Neuropeptide (Probabilities: 0.0282, 0.0165, 0.9553)\n","Processed MH0106_GL0101064: Neuropeptide (Probabilities: 0.0078, 0.0085, 0.9837)\n","Processed MH0003_GL0019783: non-AMP/OTHERS (Probabilities: 0.9925, 0.0029, 0.0045)\n","Processed MH0211_GL0171725: non-AMP/OTHERS (Probabilities: 0.8501, 0.1095, 0.0404)\n","Processed MH0211_GL0171725: non-AMP/OTHERS (Probabilities: 0.9000, 0.0151, 0.0850)\n","Processed MH0008_GL0041246: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9852)\n","Processed V1.CD7-0-PN_GL0040276: Neuropeptide (Probabilities: 0.0443, 0.0234, 0.9322)\n","Processed O2.UC44-1_GL0166896: Neuropeptide (Probabilities: 0.0613, 0.0305, 0.9083)\n","Processed sp|P56470|LEG4_HUMAN: non-AMP/OTHERS (Probabilities: 0.8082, 0.0401, 0.1518)\n","Processed DOM018_GL0021162: Neuropeptide (Probabilities: 0.3035, 0.0286, 0.6679)\n","Processed NOM002_GL0000647: Neuropeptide (Probabilities: 0.0085, 0.0094, 0.9821)\n","Processed MH0124_GL0120196: non-AMP/OTHERS (Probabilities: 0.5820, 0.2636, 0.1544)\n","Processed 763840445-stool2_revised_scaffold29212_1_gene39220: non-AMP/OTHERS (Probabilities: 0.9921, 0.0040, 0.0039)\n","Processed MH0002_GL0008545: non-AMP/OTHERS (Probabilities: 0.9914, 0.0050, 0.0036)\n","Processed V1.UC27-0_GL0150585: non-AMP/OTHERS (Probabilities: 0.9916, 0.0049, 0.0035)\n","Processed MH0305_GL0015340: non-AMP/OTHERS (Probabilities: 0.9919, 0.0036, 0.0045)\n","Processed MH0277_GL0046481: non-AMP/OTHERS (Probabilities: 0.9847, 0.0051, 0.0103)\n","Processed O2.UC32-0_GL0085839: Neuropeptide (Probabilities: 0.0219, 0.0167, 0.9614)\n","Processed MH0367_GL0119658: Neuropeptide (Probabilities: 0.0241, 0.0171, 0.9588)\n","Processed MH0246_GL0048416: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0032)\n","Processed MH0419_GL0178471: Neuropeptide (Probabilities: 0.0553, 0.0295, 0.9152)\n","Processed MH0091_GL0043719: Neuropeptide (Probabilities: 0.0153, 0.0117, 0.9729)\n","Processed MH0006_GL0154117: non-AMP/OTHERS (Probabilities: 0.9907, 0.0046, 0.0047)\n","Processed MH0251_GL0055910: non-AMP/OTHERS (Probabilities: 0.9782, 0.0181, 0.0037)\n","Processed MH0012_GL0156581: non-AMP/OTHERS (Probabilities: 0.9830, 0.0064, 0.0106)\n","Processed MH0012_GL0110152: non-AMP/OTHERS (Probabilities: 0.9873, 0.0056, 0.0071)\n","Processed MH0099_GL0119848: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed MH0375_GL0100399: non-AMP/OTHERS (Probabilities: 0.9929, 0.0030, 0.0041)\n","Processed O2.UC32-2_GL0063089: non-AMP/OTHERS (Probabilities: 0.9908, 0.0046, 0.0047)\n","Processed MH0101_GL0018469: non-AMP/OTHERS (Probabilities: 0.9926, 0.0040, 0.0034)\n","Processed MH0356_GL0135583: non-AMP/OTHERS (Probabilities: 0.9924, 0.0033, 0.0043)\n","Processed 1042403.BALAC2494_01286: Neuropeptide (Probabilities: 0.0697, 0.0253, 0.9050)\n","Processed O2.CD3-0-PT_GL0063205: non-AMP/OTHERS (Probabilities: 0.9922, 0.0033, 0.0044)\n","Processed MH0192_GL0103671: non-AMP/OTHERS (Probabilities: 0.9921, 0.0032, 0.0047)\n","Processed 159146620-stool1_revised_scaffold6613_1_gene8505: non-AMP/OTHERS (Probabilities: 0.9910, 0.0056, 0.0034)\n","Processed T2D-68A_GL0010155: non-AMP/OTHERS (Probabilities: 0.9924, 0.0033, 0.0043)\n","Processed MH0006_GL0212607: non-AMP/OTHERS (Probabilities: 0.9923, 0.0029, 0.0048)\n","Processed 547043.BIFPSEUDO_02361: non-AMP/OTHERS (Probabilities: 0.5487, 0.0298, 0.4215)\n","Processed 1048332.SALIVB_2004: Neuropeptide (Probabilities: 0.0208, 0.0134, 0.9658)\n","Processed MH0011_GL0084113: Neuropeptide (Probabilities: 0.0063, 0.0086, 0.9851)\n","Processed V1.UC29-0_GL0026874: Neuropeptide (Probabilities: 0.0114, 0.0099, 0.9787)\n","Processed 518635.BIFANG_03540: non-AMP/OTHERS (Probabilities: 0.9923, 0.0033, 0.0044)\n","Processed MH0131_GL0167555: non-AMP/OTHERS (Probabilities: 0.9919, 0.0033, 0.0048)\n","Processed MH0014_GL0078654: Neuropeptide (Probabilities: 0.0072, 0.0086, 0.9842)\n","Processed MH0003_GL0016785: non-AMP/OTHERS (Probabilities: 0.8968, 0.0187, 0.0845)\n","Processed 160765029-stool1_revised_C1040480_1_gene46665: non-AMP/OTHERS (Probabilities: 0.9903, 0.0072, 0.0025)\n","Processed 748224.HMPREF9436_02584: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed O2.UC11-2_GL0071604: Neuropeptide (Probabilities: 0.0097, 0.0097, 0.9806)\n","Processed V1.FI32_GL0109135: non-AMP/OTHERS (Probabilities: 0.9935, 0.0033, 0.0031)\n","Processed MH0086_GL0074214: non-AMP/OTHERS (Probabilities: 0.8724, 0.0221, 0.1055)\n","Processed O2.UC7-0_GL0105244: Neuropeptide (Probabilities: 0.1849, 0.0443, 0.7708)\n","Processed O2.UC49-0_GL0025488: non-AMP/OTHERS (Probabilities: 0.9927, 0.0035, 0.0038)\n","Processed O2.UC49-0_GL0025488: non-AMP/OTHERS (Probabilities: 0.9925, 0.0035, 0.0040)\n","Processed MH0188_GL0092730: non-AMP/OTHERS (Probabilities: 0.9926, 0.0035, 0.0039)\n","Processed MH0370_GL0128897: non-AMP/OTHERS (Probabilities: 0.9927, 0.0034, 0.0039)\n","Processed MH0251_GL0009283: non-AMP/OTHERS (Probabilities: 0.9923, 0.0035, 0.0042)\n","Processed MH0131_GL0070884: non-AMP/OTHERS (Probabilities: 0.9924, 0.0034, 0.0042)\n","Processed V1.FI25_GL0177509: non-AMP/OTHERS (Probabilities: 0.9739, 0.0077, 0.0183)\n","Processed MH0233_GL0093246: Neuropeptide (Probabilities: 0.0129, 0.0108, 0.9762)\n","Processed 479437.Elen_1387: non-AMP/OTHERS (Probabilities: 0.9889, 0.0037, 0.0074)\n","Processed MH0002_GL0046841: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed MH0211_GL0087691: non-AMP/OTHERS (Probabilities: 0.9908, 0.0064, 0.0028)\n","Processed MH0028_GL0029239: non-AMP/OTHERS (Probabilities: 0.9893, 0.0032, 0.0075)\n","Processed 908612.HMPREF9720_2076: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0036)\n","Processed O2.UC52-0_GL0008084: Neuropeptide (Probabilities: 0.0062, 0.0087, 0.9851)\n","Processed V1.UC22-1_GL0055260: Neuropeptide (Probabilities: 0.3882, 0.0461, 0.5657)\n","Processed O2.UC17-2_GL0115696: non-AMP/OTHERS (Probabilities: 0.9929, 0.0042, 0.0029)\n","Processed MH0239_GL0075317: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0034)\n","Processed MH0175_GL0008042: non-AMP/OTHERS (Probabilities: 0.9882, 0.0070, 0.0048)\n","Processed 765701615-stool1_revised_scaffold48317_2_gene56161: non-AMP/OTHERS (Probabilities: 0.9923, 0.0044, 0.0033)\n","Processed V1.UC58-0_GL0004917: non-AMP/OTHERS (Probabilities: 0.9929, 0.0036, 0.0036)\n","Processed 367928.BAD_0742: non-AMP/OTHERS (Probabilities: 0.9922, 0.0037, 0.0041)\n","Processed MH0062_GL0075827: non-AMP/OTHERS (Probabilities: 0.9922, 0.0032, 0.0046)\n","Processed MH0458_GL0050178: non-AMP/OTHERS (Probabilities: 0.9915, 0.0031, 0.0053)\n","Processed MH0356_GL0176224: non-AMP/OTHERS (Probabilities: 0.9919, 0.0034, 0.0047)\n","Processed MH0095_GL0004206: non-AMP/OTHERS (Probabilities: 0.9929, 0.0032, 0.0039)\n","Processed NOF001_GL0019750: non-AMP/OTHERS (Probabilities: 0.9925, 0.0031, 0.0044)\n","Processed MH0337_GL0086069: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0040)\n","Processed SZEY-44A_GL0015657: non-AMP/OTHERS (Probabilities: 0.9929, 0.0032, 0.0039)\n","Processed MH0275_GL0080637: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed MH0436_GL0110086: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9852)\n","Processed MH0203_GL0237122: non-AMP/OTHERS (Probabilities: 0.9835, 0.0077, 0.0088)\n","Processed O2.UC17-2_GL0033904: Neuropeptide (Probabilities: 0.0076, 0.0094, 0.9830)\n","Processed 864570.HMPREF9189_0837: non-AMP/OTHERS (Probabilities: 0.9453, 0.0103, 0.0444)\n","Processed MH0119_GL0113736: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed V1.CD3-0-PN_GL0027907: non-AMP/OTHERS (Probabilities: 0.9929, 0.0032, 0.0038)\n","Processed MH0002_GL0042386: non-AMP/OTHERS (Probabilities: 0.9715, 0.0052, 0.0233)\n","Processed MH0002_GL0042386: non-AMP/OTHERS (Probabilities: 0.9203, 0.0100, 0.0696)\n","Processed MH0011_GL0033289: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0452_GL0078331: Neuropeptide (Probabilities: 0.0074, 0.0091, 0.9835)\n","Processed MH0274_GL0173210: Neuropeptide (Probabilities: 0.0073, 0.0091, 0.9836)\n","Processed MH0131_GL0029451: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed MH0086_GL0030269: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed V1.UC47-4_GL0106990: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0040)\n","Processed MH0108_GL0000360: Neuropeptide (Probabilities: 0.3439, 0.0300, 0.6261)\n","Processed NOM014_GL0039342: Neuropeptide (Probabilities: 0.0433, 0.0154, 0.9413)\n","Processed V1.FI17_GL0161218: Neuropeptide (Probabilities: 0.0062, 0.0086, 0.9852)\n","Processed MH0188_GL0150361: non-AMP/OTHERS (Probabilities: 0.9888, 0.0035, 0.0077)\n","Processed MH0238_GL0124775: Neuropeptide (Probabilities: 0.0083, 0.0089, 0.9828)\n","Processed DOM012_GL0040138: non-AMP/OTHERS (Probabilities: 0.9103, 0.0175, 0.0722)\n","Processed MH0453_GL0127534: non-AMP/OTHERS (Probabilities: 0.9866, 0.0064, 0.0070)\n","Processed MH0377_GL0088622: non-AMP/OTHERS (Probabilities: 0.9893, 0.0060, 0.0047)\n","Processed MH0003_GL0096402: Neuropeptide (Probabilities: 0.0064, 0.0087, 0.9848)\n","Processed O2.UC37-0_GL0042279: Neuropeptide (Probabilities: 0.0079, 0.0091, 0.9830)\n","Processed MH0088_GL0118986: non-AMP/OTHERS (Probabilities: 0.9925, 0.0031, 0.0044)\n","Processed MH0314_GL0030447: Neuropeptide (Probabilities: 0.0096, 0.0095, 0.9809)\n","Processed 367928.BAD_1532: Neuropeptide (Probabilities: 0.0074, 0.0086, 0.9840)\n","Processed 763860675-stool1_revised_scaffold51668_1_gene141411: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0244_GL0022860: non-AMP/OTHERS (Probabilities: 0.5741, 0.0362, 0.3897)\n","Processed O2.UC31-0_GL0109539: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed 158742018-stool1_revised_scaffold23471_1_gene92491: Neuropeptide (Probabilities: 0.0064, 0.0085, 0.9851)\n","Processed MH0203_GL0172920: non-AMP/OTHERS (Probabilities: 0.9820, 0.0065, 0.0115)\n","Processed 159611913-stool2_revised_scaffold16836_1_gene39838: non-AMP/OTHERS (Probabilities: 0.9826, 0.0057, 0.0117)\n","Processed O2.UC7-0_GL0020160: Neuropeptide (Probabilities: 0.0256, 0.0139, 0.9605)\n","Processed MH0009_GL0061190: non-AMP/OTHERS (Probabilities: 0.9911, 0.0057, 0.0031)\n","Processed MH0281_GL0060511: non-AMP/OTHERS (Probabilities: 0.9932, 0.0033, 0.0036)\n","Processed MH0341_GL0088523: non-AMP/OTHERS (Probabilities: 0.9934, 0.0031, 0.0035)\n","Processed 411481.BIFADO_02111: non-AMP/OTHERS (Probabilities: 0.9681, 0.0076, 0.0244)\n","Processed MH0217_GL0014133: non-AMP/OTHERS (Probabilities: 0.9910, 0.0057, 0.0033)\n","Processed MH0131_GL0094987: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9852)\n","Processed MH0377_GL0006076: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0356_GL0057591: non-AMP/OTHERS (Probabilities: 0.9927, 0.0043, 0.0030)\n","Processed 391904.BLIJ_0450: non-AMP/OTHERS (Probabilities: 0.9925, 0.0034, 0.0041)\n","Processed MH0292_GL0069120: non-AMP/OTHERS (Probabilities: 0.8795, 0.0120, 0.1084)\n","Processed MH0062_GL0045881: Neuropeptide (Probabilities: 0.4795, 0.0369, 0.4836)\n","Processed MH0161_GL0008156: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0040)\n","Processed MH0122_GL0026668: non-AMP/OTHERS (Probabilities: 0.9925, 0.0032, 0.0044)\n","Processed MH0006_GL0174080: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0035)\n","Processed O2.CD1-0_GL0006497: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0128_GL0019168: Neuropeptide (Probabilities: 0.0139, 0.0119, 0.9742)\n","Processed sp|Q96A32|MLRS_HUMAN: non-AMP/OTHERS (Probabilities: 0.9898, 0.0063, 0.0039)\n","Processed SZEY-37A_GL0056413: Neuropeptide (Probabilities: 0.0137, 0.0112, 0.9750)\n","Processed DOF004_GL0077229: non-AMP/OTHERS (Probabilities: 0.9694, 0.0082, 0.0223)\n","Processed O2.CD2-0-PT_GL0004300: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed O2.UC6-1_GL0073139: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0161_GL0140735: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0113_GL0079842: Neuropeptide (Probabilities: 0.0313, 0.0157, 0.9530)\n","Processed O2.UC27-0_GL0052333: non-AMP/OTHERS (Probabilities: 0.4957, 0.0217, 0.4826)\n","Processed T2D-83A_GL0082037: Neuropeptide (Probabilities: 0.0123, 0.0109, 0.9768)\n","Processed T2D-106A_GL0006657: non-AMP/OTHERS (Probabilities: 0.9901, 0.0040, 0.0059)\n","Processed MH0123_GL0065857: non-AMP/OTHERS (Probabilities: 0.9809, 0.0036, 0.0155)\n","Processed MH0203_GL0083057: non-AMP/OTHERS (Probabilities: 0.9915, 0.0041, 0.0044)\n","Processed 763678604-stool1_revised_scaffold58348_1_gene150086: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed NLF008_GL0035123: Neuropeptide (Probabilities: 0.3066, 0.0527, 0.6408)\n","Processed MH0092_GL0009462: Neuropeptide (Probabilities: 0.0074, 0.0085, 0.9842)\n","Processed O2.UC11-1_GL0004818: non-AMP/OTHERS (Probabilities: 0.9901, 0.0043, 0.0057)\n","Processed MH0211_GL0035208: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0037)\n","Processed DLM028_GL0023551: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed NLM023_GL0007606: non-AMP/OTHERS (Probabilities: 0.9887, 0.0034, 0.0079)\n","Processed MH0058_GL0056069: non-AMP/OTHERS (Probabilities: 0.8312, 0.0129, 0.1559)\n","Processed MH0251_GL0017806: non-AMP/OTHERS (Probabilities: 0.9924, 0.0034, 0.0042)\n","Processed MH0188_GL0001615: non-AMP/OTHERS (Probabilities: 0.9924, 0.0031, 0.0045)\n","Processed V1.FI28_GL0214411: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0036)\n","Processed V1.CD6-4_GL0133420: non-AMP/OTHERS (Probabilities: 0.9925, 0.0029, 0.0046)\n","Processed T2D-122A_GL0050726: non-AMP/OTHERS (Probabilities: 0.9932, 0.0036, 0.0033)\n","Processed sp|P05109|S10A8_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0309_GL0146813: non-AMP/OTHERS (Probabilities: 0.9455, 0.0083, 0.0463)\n","Processed O2.UC15-1_GL0072279: non-AMP/OTHERS (Probabilities: 0.9882, 0.0057, 0.0061)\n","Processed MH0220_GL0175640: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed O2.UC28-0_GL0089898: non-AMP/OTHERS (Probabilities: 0.9918, 0.0034, 0.0047)\n","Processed V1.UC42-0_GL0014160: Neuropeptide (Probabilities: 0.0104, 0.0106, 0.9790)\n","Processed T2D-178A_GL0000566: non-AMP/OTHERS (Probabilities: 0.9923, 0.0046, 0.0031)\n","Processed 411902.CLOBOL_05372: non-AMP/OTHERS (Probabilities: 0.9573, 0.0172, 0.0256)\n","Processed MH0330_GL0169023: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0040)\n","Processed DLF007_GL0016583: non-AMP/OTHERS (Probabilities: 0.9494, 0.0070, 0.0435)\n","Processed MH0454_GL0013007: non-AMP/OTHERS (Probabilities: 0.9875, 0.0035, 0.0090)\n","Processed MH0288_GL0119974: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed 765701615-stool1_revised_C1380714_1_gene165857: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0037)\n","Processed MH0131_GL0099453: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed MH0188_GL0093557: Neuropeptide (Probabilities: 0.0107, 0.0096, 0.9797)\n","Processed MH0002_GL0025922: Neuropeptide (Probabilities: 0.0073, 0.0088, 0.9839)\n","Processed O2.UC32-0_GL0072495: Neuropeptide (Probabilities: 0.0120, 0.0101, 0.9779)\n","Processed MH0189_GL0129052: Neuropeptide (Probabilities: 0.0201, 0.0123, 0.9676)\n","Processed MH0045_GL0056768: Neuropeptide (Probabilities: 0.0104, 0.0101, 0.9795)\n","Processed MH0003_GL0010255: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0191_GL0137920: Neuropeptide (Probabilities: 0.0080, 0.0089, 0.9831)\n","Processed 764285508-stool1_revised_scaffold11097_1_gene111470: Neuropeptide (Probabilities: 0.0112, 0.0102, 0.9786)\n","Processed 764285508-stool1_revised_scaffold11097_1_gene111470: Neuropeptide (Probabilities: 0.0617, 0.0201, 0.9181)\n","Processed MH0211_GL0190372: non-AMP/OTHERS (Probabilities: 0.9695, 0.0067, 0.0239)\n","Processed N084A_GL0072565: non-AMP/OTHERS (Probabilities: 0.7037, 0.0469, 0.2494)\n","Processed MH0277_GL0054423: non-AMP/OTHERS (Probabilities: 0.9921, 0.0035, 0.0044)\n","Processed MH0188_GL0176365: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0039)\n","Processed MH0058_GL0059005: non-AMP/OTHERS (Probabilities: 0.8711, 0.0345, 0.0945)\n","Processed MH0441_GL0097774: non-AMP/OTHERS (Probabilities: 0.9874, 0.0044, 0.0082)\n","Processed 763982056-stool1_revised_scaffold11720_1_gene80511: Neuropeptide (Probabilities: 0.0617, 0.0165, 0.9218)\n","Processed MH0270_GL0127009: Neuropeptide (Probabilities: 0.0323, 0.0211, 0.9466)\n","Processed MH0081_GL0036550: Neuropeptide (Probabilities: 0.0635, 0.0196, 0.9170)\n","Processed MH0197_GL0170840: non-AMP/OTHERS (Probabilities: 0.9927, 0.0034, 0.0039)\n","Processed MH0457_GL0226231: Neuropeptide (Probabilities: 0.0065, 0.0088, 0.9847)\n","Processed MH0457_GL0226231: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed MH0161_GL0060525: non-AMP/OTHERS (Probabilities: 0.9866, 0.0040, 0.0095)\n","Processed V1.CD32-4_GL0002548: non-AMP/OTHERS (Probabilities: 0.9914, 0.0051, 0.0035)\n","Processed MH0270_GL0121925: non-AMP/OTHERS (Probabilities: 0.9914, 0.0035, 0.0051)\n","Processed MH0011_GL0017881: Neuropeptide (Probabilities: 0.2729, 0.0332, 0.6939)\n","Processed MH0011_GL0017881: Neuropeptide (Probabilities: 0.0597, 0.0275, 0.9128)\n","Processed MH0003_GL0010257: Neuropeptide (Probabilities: 0.2319, 0.0209, 0.7472)\n","Processed MH0058_GL0069876: Neuropeptide (Probabilities: 0.0078, 0.0089, 0.9833)\n","Processed O2.UC17-1_GL0103202: non-AMP/OTHERS (Probabilities: 0.9929, 0.0028, 0.0043)\n","Processed O2.UC17-1_GL0103202: non-AMP/OTHERS (Probabilities: 0.9928, 0.0028, 0.0044)\n","Processed O2.UC20-2_GL0122705: Neuropeptide (Probabilities: 0.0117, 0.0102, 0.9781)\n","Processed MH0293_GL0190676: non-AMP/OTHERS (Probabilities: 0.8266, 0.0209, 0.1525)\n","Processed V1.CD27-0_GL0122637: non-AMP/OTHERS (Probabilities: 0.9847, 0.0048, 0.0105)\n","Processed MH0002_GL0072797: non-AMP/OTHERS (Probabilities: 0.9554, 0.0072, 0.0374)\n","Processed MH0002_GL0035068: non-AMP/OTHERS (Probabilities: 0.9929, 0.0033, 0.0039)\n","Processed MH0193_GL0186495: non-AMP/OTHERS (Probabilities: 0.9920, 0.0032, 0.0048)\n","Processed MH0161_GL0161223: Neuropeptide (Probabilities: 0.0508, 0.0225, 0.9267)\n","Processed MH0161_GL0161223: non-AMP/OTHERS (Probabilities: 0.9906, 0.0041, 0.0054)\n","Processed MH0230_GL0010688: Neuropeptide (Probabilities: 0.0429, 0.0223, 0.9347)\n","Processed MH0206_GL0059301: non-AMP/OTHERS (Probabilities: 0.9929, 0.0037, 0.0033)\n","Processed V1.FI27_GL0074041: non-AMP/OTHERS (Probabilities: 0.9916, 0.0031, 0.0053)\n","Processed 737052003-stool1_revised_scaffold22585_4_gene30774: non-AMP/OTHERS (Probabilities: 0.9507, 0.0218, 0.0275)\n","Processed MH0230_GL0071172: non-AMP/OTHERS (Probabilities: 0.9799, 0.0076, 0.0125)\n","Processed O2.UC3-2_GL0142105: non-AMP/OTHERS (Probabilities: 0.9915, 0.0029, 0.0056)\n","Processed MH0416_GL0075412: non-AMP/OTHERS (Probabilities: 0.6781, 0.0164, 0.3055)\n","Processed sp|P21589|5NTD_HUMAN: Neuropeptide (Probabilities: 0.0144, 0.0135, 0.9721)\n","Processed MH0008_GL0046039: non-AMP/OTHERS (Probabilities: 0.9923, 0.0029, 0.0048)\n","Processed 158458797-stool1_revised_scaffold25252_2_gene79145: non-AMP/OTHERS (Probabilities: 0.7581, 0.0205, 0.2215)\n","Processed V1.CD2-0-PN_GL0107891: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0044)\n","Processed MH0051_GL0050191: non-AMP/OTHERS (Probabilities: 0.9882, 0.0044, 0.0074)\n","Processed 764042746-stool2_revised_scaffold4910_1_gene5708: Neuropeptide (Probabilities: 0.0065, 0.0088, 0.9848)\n","Processed MH0011_GL0094282: Neuropeptide (Probabilities: 0.0419, 0.0170, 0.9411)\n","Processed MH0091_GL0081366: Neuropeptide (Probabilities: 0.0101, 0.0099, 0.9800)\n","Processed MH0239_GL0050110: non-AMP/OTHERS (Probabilities: 0.7236, 0.0362, 0.2402)\n","Processed 765094712-stool1_revised_scaffold781_1_gene14523: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0442_GL0146762: non-AMP/OTHERS (Probabilities: 0.9804, 0.0080, 0.0115)\n","Processed MH0020_GL0020073: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed MH0227_GL0114504: non-AMP/OTHERS (Probabilities: 0.9916, 0.0052, 0.0032)\n","Processed 411479.BACUNI_01823: Neuropeptide (Probabilities: 0.0066, 0.0088, 0.9846)\n","Processed 411479.BACUNI_01823: non-AMP/OTHERS (Probabilities: 0.9590, 0.0166, 0.0244)\n","Processed O2.UC22-2_GL0138063: Neuropeptide (Probabilities: 0.0078, 0.0091, 0.9831)\n","Processed N040A_GL0064889: non-AMP/OTHERS (Probabilities: 0.9933, 0.0032, 0.0036)\n","Processed MH0088_GL0050991: Neuropeptide (Probabilities: 0.2594, 0.0299, 0.7107)\n","Processed MH0188_GL0092730: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0419_GL0006281: non-AMP/OTHERS (Probabilities: 0.9928, 0.0045, 0.0027)\n","Processed V1.CD48-0_GL0007362: Neuropeptide (Probabilities: 0.4000, 0.0550, 0.5450)\n","Processed MH0188_GL0031829: Neuropeptide (Probabilities: 0.0097, 0.0097, 0.9807)\n","Processed O2.UC2-1_GL0059921: Neuropeptide (Probabilities: 0.0068, 0.0087, 0.9846)\n","Processed V1.UC58-4_GL0108396: non-AMP/OTHERS (Probabilities: 0.8684, 0.0175, 0.1141)\n","Processed V1.UC23-0_GL0104454: non-AMP/OTHERS (Probabilities: 0.9852, 0.0040, 0.0108)\n","Processed MH0172_GL0086622: non-AMP/OTHERS (Probabilities: 0.9908, 0.0040, 0.0052)\n","Processed V1.CD4-0-PT_GL0015627: Neuropeptide (Probabilities: 0.0070, 0.0086, 0.9843)\n","Processed T2D-25A_GL0043369: non-AMP/OTHERS (Probabilities: 0.7840, 0.0163, 0.1997)\n","Processed V1.FI13_GL0132301: Neuropeptide (Probabilities: 0.0082, 0.0091, 0.9827)\n","Processed MH0358_GL0118417: non-AMP/OTHERS (Probabilities: 0.9896, 0.0035, 0.0069)\n","Processed T2D-11A_GL0030436: non-AMP/OTHERS (Probabilities: 0.9921, 0.0034, 0.0045)\n","Processed MH0244_GL0076969: non-AMP/OTHERS (Probabilities: 0.9935, 0.0030, 0.0035)\n","Processed 765620695-stool1_revised_scaffold4221_2_gene118611: non-AMP/OTHERS (Probabilities: 0.9738, 0.0052, 0.0210)\n","Processed MH0012_GL0148859: non-AMP/OTHERS (Probabilities: 0.9927, 0.0040, 0.0032)\n","Processed MH0062_GL0038009: non-AMP/OTHERS (Probabilities: 0.8166, 0.0203, 0.1631)\n","Processed MH0147_GL0006920: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0002_GL0003859: Neuropeptide (Probabilities: 0.0097, 0.0101, 0.9803)\n","Processed MH0238_GL0057245: non-AMP/OTHERS (Probabilities: 0.7292, 0.0273, 0.2435)\n","Processed BGI-28A_GL0084390: Neuropeptide (Probabilities: 0.0069, 0.0089, 0.9843)\n","Processed MH0272_GL0004555: Neuropeptide (Probabilities: 0.0320, 0.0184, 0.9496)\n","Processed MH0277_GL0055243: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed SZEY-09A_GL0075141: Neuropeptide (Probabilities: 0.0168, 0.0130, 0.9702)\n","Processed SZEY-09A_GL0075141: non-AMP/OTHERS (Probabilities: 0.9778, 0.0093, 0.0129)\n","Processed O2.UC11-2_GL0041407: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed MH0088_GL0047835: non-AMP/OTHERS (Probabilities: 0.9926, 0.0030, 0.0044)\n","Processed MH0419_GL0023346: non-AMP/OTHERS (Probabilities: 0.9916, 0.0033, 0.0051)\n","Processed O2.UC15-1_GL0073124: non-AMP/OTHERS (Probabilities: 0.9922, 0.0031, 0.0046)\n","Processed MH0433_GL0038703: non-AMP/OTHERS (Probabilities: 0.9916, 0.0038, 0.0046)\n","Processed DOF004_GL0090409: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0033)\n","Processed DOF004_GL0090409: non-AMP/OTHERS (Probabilities: 0.9931, 0.0037, 0.0032)\n","Processed MH0441_GL0017861: Neuropeptide (Probabilities: 0.0830, 0.0268, 0.8902)\n","Processed DOM016_GL0070733: non-AMP/OTHERS (Probabilities: 0.9315, 0.0092, 0.0593)\n","Processed MH0287_GL0083644: Neuropeptide (Probabilities: 0.0062, 0.0087, 0.9851)\n","Processed MH0370_GL0034841: Neuropeptide (Probabilities: 0.0065, 0.0087, 0.9848)\n","Processed V1.UC29-0_GL0094257: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9850)\n","Processed O2.UC48-0_GL0000051: Neuropeptide (Probabilities: 0.0081, 0.0091, 0.9828)\n","Processed MH0217_GL0170323: non-AMP/OTHERS (Probabilities: 0.9917, 0.0058, 0.0025)\n","Processed V1.CD19-0_GL0177345: non-AMP/OTHERS (Probabilities: 0.9719, 0.0114, 0.0167)\n","Processed V1.CD19-0_GL0177345: non-AMP/OTHERS (Probabilities: 0.9774, 0.0155, 0.0071)\n","Processed V1.CD36-0_GL0033795: non-AMP/OTHERS (Probabilities: 0.9820, 0.0103, 0.0077)\n","Processed 367928.BAD_1011: Neuropeptide (Probabilities: 0.3484, 0.0299, 0.6217)\n","Processed MH0012_GL0189634: Neuropeptide (Probabilities: 0.0085, 0.0093, 0.9823)\n","Processed MH0010_GL0026622: Neuropeptide (Probabilities: 0.0128, 0.0125, 0.9746)\n","Processed MH0426_GL0151268: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed MH0426_GL0151268: non-AMP/OTHERS (Probabilities: 0.9737, 0.0117, 0.0146)\n","Processed MH0135_GL0081331: non-AMP/OTHERS (Probabilities: 0.9905, 0.0042, 0.0052)\n","Processed MH0193_GL0179491: non-AMP/OTHERS (Probabilities: 0.8723, 0.0222, 0.1055)\n","Processed V1.FI29_GL0101205: Neuropeptide (Probabilities: 0.0085, 0.0091, 0.9824)\n","Processed MH0062_GL0000895: non-AMP/OTHERS (Probabilities: 0.9870, 0.0064, 0.0066)\n","Processed MH0062_GL0041928: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed V1.CD8-0-PN_GL0085765: Neuropeptide (Probabilities: 0.0075, 0.0088, 0.9837)\n","Processed O2.UC11-1_GL0069979: non-AMP/OTHERS (Probabilities: 0.9904, 0.0037, 0.0060)\n","Processed sp|P04040|CATA_HUMAN: non-AMP/OTHERS (Probabilities: 0.9424, 0.0232, 0.0345)\n","Processed 338793263-stool1_revised_C203677_1_gene18783: non-AMP/OTHERS (Probabilities: 0.4961, 0.0241, 0.4798)\n","Processed MH0078_GL0004581: non-AMP/OTHERS (Probabilities: 0.5629, 0.1806, 0.2565)\n","Processed MH0276_GL0032729: non-AMP/OTHERS (Probabilities: 0.9670, 0.0182, 0.0148)\n","Processed MH0010_GL0042333: Neuropeptide (Probabilities: 0.0772, 0.0260, 0.8968)\n","Processed MH0198_GL0063059: Neuropeptide (Probabilities: 0.0114, 0.0101, 0.9786)\n","Processed MH0062_GL0070765: Neuropeptide (Probabilities: 0.0234, 0.0169, 0.9597)\n","Processed MH0016_GL0068440: Neuropeptide (Probabilities: 0.0072, 0.0086, 0.9842)\n","Processed MH0086_GL0052534: AMP (Probabilities: 0.1197, 0.8681, 0.0122)\n","Processed MH0086_GL0052534: non-AMP/OTHERS (Probabilities: 0.9911, 0.0061, 0.0029)\n","Processed O2.UC46-1_GL0164726: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0041)\n","Processed O2.UC46-1_GL0164726: non-AMP/OTHERS (Probabilities: 0.9921, 0.0036, 0.0043)\n","Processed MH0393_GL0098327: non-AMP/OTHERS (Probabilities: 0.9923, 0.0038, 0.0040)\n","Processed MH0088_GL0007602: non-AMP/OTHERS (Probabilities: 0.9921, 0.0038, 0.0041)\n","Processed O2.UC26-1_GL0070493: Neuropeptide (Probabilities: 0.0109, 0.0105, 0.9786)\n","Processed MH0233_GL0070818: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed V1.CD17-4_GL0116222: non-AMP/OTHERS (Probabilities: 0.9911, 0.0034, 0.0055)\n","Processed V1.CD32-0_GL0014024: non-AMP/OTHERS (Probabilities: 0.9918, 0.0034, 0.0048)\n","Processed MH0014_GL0009871: non-AMP/OTHERS (Probabilities: 0.9926, 0.0030, 0.0045)\n","Processed 264199.stu0809: Neuropeptide (Probabilities: 0.0277, 0.0149, 0.9574)\n","Processed DLF013_GL0046441: non-AMP/OTHERS (Probabilities: 0.9917, 0.0042, 0.0041)\n","Processed O2.UC40-1_GL0138266: non-AMP/OTHERS (Probabilities: 0.9921, 0.0041, 0.0039)\n","Processed MH0442_GL0023173: non-AMP/OTHERS (Probabilities: 0.9921, 0.0042, 0.0037)\n","Processed MH0087_GL0010788: Neuropeptide (Probabilities: 0.0092, 0.0096, 0.9811)\n","Processed V1.UC11-0_GL0047701: Neuropeptide (Probabilities: 0.1845, 0.0334, 0.7821)\n","Processed MH0203_GL0010969: Neuropeptide (Probabilities: 0.0071, 0.0090, 0.9838)\n","Processed V1.UC63-0_GL0238124: non-AMP/OTHERS (Probabilities: 0.9922, 0.0032, 0.0046)\n","Processed MH0410_GL0014913: non-AMP/OTHERS (Probabilities: 0.9933, 0.0035, 0.0032)\n","Processed 467705.SGO_0665: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9850)\n","Processed MH0203_GL0120151: non-AMP/OTHERS (Probabilities: 0.9926, 0.0043, 0.0030)\n","Processed MH0131_GL0162486: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0040)\n","Processed MH0196_GL0026163: non-AMP/OTHERS (Probabilities: 0.9904, 0.0041, 0.0055)\n","Processed MH0161_GL0140735: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed V1.FI21_GL0041033: non-AMP/OTHERS (Probabilities: 0.9916, 0.0029, 0.0056)\n","Processed MH0066_GL0038551: non-AMP/OTHERS (Probabilities: 0.9879, 0.0044, 0.0077)\n","Processed MH0060_GL0018946: non-AMP/OTHERS (Probabilities: 0.9932, 0.0037, 0.0031)\n","Processed MH0150_GL0033381: Neuropeptide (Probabilities: 0.0178, 0.0119, 0.9703)\n","Processed MH0305_GL0011259: non-AMP/OTHERS (Probabilities: 0.9908, 0.0034, 0.0057)\n","Processed O2.UC13-2_GL0048227: non-AMP/OTHERS (Probabilities: 0.4446, 0.1737, 0.3817)\n","Processed O2.UC19-2_GL0107489: non-AMP/OTHERS (Probabilities: 0.9929, 0.0030, 0.0042)\n","Processed V1.CD7-4_GL0038595: non-AMP/OTHERS (Probabilities: 0.9928, 0.0030, 0.0042)\n","Processed MH0150_GL0077036: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed V1.CD10-0_GL0037831: non-AMP/OTHERS (Probabilities: 0.9910, 0.0051, 0.0038)\n","Processed MH0172_GL0085427: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0172_GL0085427: Neuropeptide (Probabilities: 0.0063, 0.0087, 0.9850)\n","Processed O2.UC27-2_GL0025063: Neuropeptide (Probabilities: 0.0083, 0.0090, 0.9827)\n","Processed MH0234_GL0087310: Neuropeptide (Probabilities: 0.0069, 0.0088, 0.9844)\n","Processed MH0002_GL0008496: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed 435590.BVU_3291: non-AMP/OTHERS (Probabilities: 0.9921, 0.0038, 0.0041)\n","Processed MH0253_GL0198191: AMP (Probabilities: 0.1131, 0.8560, 0.0310)\n","Processed MH0089_GL0033973: AMP (Probabilities: 0.1300, 0.8035, 0.0665)\n","Processed N046A_GL0046392: non-AMP/OTHERS (Probabilities: 0.9441, 0.0209, 0.0350)\n","Processed O2.CD3-0-PT_GL0149386: Neuropeptide (Probabilities: 0.0065, 0.0085, 0.9850)\n","Processed MH0002_GL0001401: non-AMP/OTHERS (Probabilities: 0.9904, 0.0041, 0.0055)\n","Processed MH0445_GL0023299: non-AMP/OTHERS (Probabilities: 0.9893, 0.0047, 0.0060)\n","Processed O2.UC58-2_GL0001136: Neuropeptide (Probabilities: 0.0082, 0.0091, 0.9827)\n","Processed MH0012_GL0232564: non-AMP/OTHERS (Probabilities: 0.5663, 0.0259, 0.4077)\n","Processed MH0457_GL0072774: non-AMP/OTHERS (Probabilities: 0.9716, 0.0116, 0.0168)\n","Processed DOM019_GL0096212: Neuropeptide (Probabilities: 0.1161, 0.0482, 0.8357)\n","Processed MH0083_GL0081429: Neuropeptide (Probabilities: 0.0081, 0.0093, 0.9826)\n","Processed V1.UC50-1_GL0011136: non-AMP/OTHERS (Probabilities: 0.6851, 0.0368, 0.2781)\n","Processed MH0230_GL0095457: non-AMP/OTHERS (Probabilities: 0.9912, 0.0036, 0.0052)\n","Processed O2.UC9-2_GL0160421: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9851)\n","Processed MH0328_GL0051223: Neuropeptide (Probabilities: 0.0071, 0.0086, 0.9843)\n","Processed MH0188_GL0157833: Neuropeptide (Probabilities: 0.0070, 0.0091, 0.9839)\n","Processed O2.UC19-1_GL0014221: AMP (Probabilities: 0.3360, 0.6522, 0.0118)\n","Processed sp|P20073|ANXA7_HUMAN: Neuropeptide (Probabilities: 0.0062, 0.0086, 0.9853)\n","Processed MH0111_GL0006613: non-AMP/OTHERS (Probabilities: 0.9840, 0.0046, 0.0114)\n","Processed V1.CD41-0_GL0114304: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed MH0147_GL0085305: non-AMP/OTHERS (Probabilities: 0.9901, 0.0037, 0.0062)\n","Processed O2.UC35-2_GL0147005: non-AMP/OTHERS (Probabilities: 0.9919, 0.0036, 0.0045)\n","Processed V1.UC31-4_GL0059802: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0036)\n","Processed O2.UC46-1_GL0109746: Neuropeptide (Probabilities: 0.0189, 0.0141, 0.9670)\n","Processed MH0305_GL0038208: Neuropeptide (Probabilities: 0.0127, 0.0113, 0.9760)\n","Processed MH0100_GL0086056: Neuropeptide (Probabilities: 0.0176, 0.0131, 0.9694)\n","Processed MH0321_GL0036415: non-AMP/OTHERS (Probabilities: 0.5714, 0.1036, 0.3250)\n","Processed 871237.SIN_1899: Neuropeptide (Probabilities: 0.0067, 0.0086, 0.9847)\n","Processed sp|P12429|ANXA3_HUMAN: non-AMP/OTHERS (Probabilities: 0.9889, 0.0038, 0.0073)\n","Processed MH0119_GL0113736: non-AMP/OTHERS (Probabilities: 0.7204, 0.0388, 0.2408)\n","Processed V1.UC21-0_GL0017564: Neuropeptide (Probabilities: 0.0076, 0.0089, 0.9834)\n","Processed V1.UC11-5_GL0154867: non-AMP/OTHERS (Probabilities: 0.9933, 0.0040, 0.0027)\n","Processed MH0003_GL0025445: non-AMP/OTHERS (Probabilities: 0.9904, 0.0067, 0.0029)\n","Processed V1.FI28_GL0064117: non-AMP/OTHERS (Probabilities: 0.9886, 0.0058, 0.0056)\n","Processed O2.UC16-1_GL0132332: non-AMP/OTHERS (Probabilities: 0.9806, 0.0087, 0.0107)\n","Processed V1.UC42-0_GL0049062: non-AMP/OTHERS (Probabilities: 0.9931, 0.0037, 0.0033)\n","Processed V1.UC16-0_GL0018225: non-AMP/OTHERS (Probabilities: 0.9925, 0.0029, 0.0045)\n","Processed MH0370_GL0059748: non-AMP/OTHERS (Probabilities: 0.9930, 0.0029, 0.0041)\n","Processed O2.UC15-0_GL0079291: Neuropeptide (Probabilities: 0.0064, 0.0085, 0.9850)\n","Processed V1.CD14-0_GL0079954: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed V1.CD14-0_GL0079954: Neuropeptide (Probabilities: 0.0087, 0.0093, 0.9820)\n","Processed MH0340_GL0173335: non-AMP/OTHERS (Probabilities: 0.9886, 0.0036, 0.0078)\n","Processed V1.UC53-0_GL0058779: Neuropeptide (Probabilities: 0.0923, 0.0235, 0.8843)\n","Processed MH0432_GL0050497: Neuropeptide (Probabilities: 0.1371, 0.0224, 0.8405)\n","Processed NLF012_GL0035086: non-AMP/OTHERS (Probabilities: 0.9932, 0.0030, 0.0038)\n","Processed O2.CD3-0-PT_GL0149386: non-AMP/OTHERS (Probabilities: 0.9791, 0.0147, 0.0062)\n","Processed T2D-25A_GL0016689: non-AMP/OTHERS (Probabilities: 0.9914, 0.0034, 0.0051)\n","Processed MH0287_GL0011695: Neuropeptide (Probabilities: 0.2584, 0.0311, 0.7104)\n","Processed 411481.BIFADO_00398: non-AMP/OTHERS (Probabilities: 0.9767, 0.0050, 0.0183)\n","Processed MH0440_GL0091102: non-AMP/OTHERS (Probabilities: 0.9934, 0.0033, 0.0032)\n","Processed 763840445-stool2_revised_scaffold26029_1_gene114144: non-AMP/OTHERS (Probabilities: 0.9892, 0.0031, 0.0077)\n","Processed V1.CD36-0_GL0160831: non-AMP/OTHERS (Probabilities: 0.9917, 0.0033, 0.0050)\n","Processed O2.UC37-2_GL0002887: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed O2.UC23-2_GL0044659: Neuropeptide (Probabilities: 0.0951, 0.0481, 0.8569)\n","Processed MH0003_GL0053226: Neuropeptide (Probabilities: 0.0066, 0.0086, 0.9848)\n","Processed MH0131_GL0023268: Neuropeptide (Probabilities: 0.0167, 0.0131, 0.9702)\n","Processed O2.UC19-2_GL0050407: Neuropeptide (Probabilities: 0.0068, 0.0087, 0.9846)\n","Processed O2.UC19-2_GL0050407: non-AMP/OTHERS (Probabilities: 0.8294, 0.0192, 0.1515)\n","Processed 159611913-stool1_revised_C741254_1_gene27612: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed 411461.DORFOR_01605: Neuropeptide (Probabilities: 0.0322, 0.0255, 0.9423)\n","Processed MH0422_GL0219120: Neuropeptide (Probabilities: 0.2509, 0.1398, 0.6093)\n","Processed MH0001_GL0015204: non-AMP/OTHERS (Probabilities: 0.9878, 0.0049, 0.0073)\n","Processed MH0159_GL0017208: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed 483217.BACDOR_00668: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed 205913.Blon03001720: Neuropeptide (Probabilities: 0.0086, 0.0094, 0.9820)\n","Processed MH0433_GL0247999: non-AMP/OTHERS (Probabilities: 0.9933, 0.0030, 0.0037)\n","Processed MH0230_GL0014390: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0036)\n","Processed O2.UC37-1_GL0100927: non-AMP/OTHERS (Probabilities: 0.9929, 0.0030, 0.0040)\n","Processed MH0131_GL0117745: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0039)\n","Processed V1.CD25-4_GL0109648: non-AMP/OTHERS (Probabilities: 0.9902, 0.0036, 0.0062)\n","Processed V1.CD22-0_GL0159239: non-AMP/OTHERS (Probabilities: 0.9916, 0.0037, 0.0047)\n","Processed MH0348_GL0068636: non-AMP/OTHERS (Probabilities: 0.9919, 0.0042, 0.0039)\n","Processed MH0088_GL0070744: non-AMP/OTHERS (Probabilities: 0.9901, 0.0042, 0.0057)\n","Processed V1.UC26-0_GL0068088: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0037)\n","Processed MH0129_GL0119986: non-AMP/OTHERS (Probabilities: 0.9928, 0.0030, 0.0042)\n","Processed MH0435_GL0172791: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0088_GL0110249: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0033)\n","Processed MH0012_GL0083373: Neuropeptide (Probabilities: 0.0120, 0.0112, 0.9768)\n","Processed O2.UC48-1_GL0163502: non-AMP/OTHERS (Probabilities: 0.9760, 0.0158, 0.0082)\n","Processed MH0397_GL0003656: non-AMP/OTHERS (Probabilities: 0.3691, 0.3109, 0.3201)\n","Processed MH0318_GL0020934: non-AMP/OTHERS (Probabilities: 0.9866, 0.0060, 0.0074)\n","Processed MH0003_GL0042541: non-AMP/OTHERS (Probabilities: 0.6236, 0.0962, 0.2801)\n","Processed MH0003_GL0042541: Neuropeptide (Probabilities: 0.0495, 0.0245, 0.9260)\n","Processed 457395.BSBG_00304: Neuropeptide (Probabilities: 0.0987, 0.0542, 0.8471)\n","Processed ED14A_GL0028024: Neuropeptide (Probabilities: 0.0137, 0.0127, 0.9737)\n","Processed MH0045_GL0037079: non-AMP/OTHERS (Probabilities: 0.9930, 0.0039, 0.0031)\n","Processed MH0099_GL0058602: Neuropeptide (Probabilities: 0.0991, 0.0225, 0.8783)\n","Processed MH0012_GL0175262: Neuropeptide (Probabilities: 0.0439, 0.0257, 0.9304)\n","Processed MH0062_GL0050125: non-AMP/OTHERS (Probabilities: 0.7099, 0.0275, 0.2625)\n","Processed MH0062_GL0050125: non-AMP/OTHERS (Probabilities: 0.8735, 0.0310, 0.0955)\n","Processed MH0176_GL0147448: Neuropeptide (Probabilities: 0.3615, 0.0263, 0.6122)\n","Processed MH0196_GL0010727: non-AMP/OTHERS (Probabilities: 0.9889, 0.0049, 0.0062)\n","Processed T2D-15A_GL0164279: Neuropeptide (Probabilities: 0.4352, 0.0239, 0.5409)\n","Processed MH0147_GL0120111: Neuropeptide (Probabilities: 0.0069, 0.0085, 0.9846)\n","Processed O2.UC30-2_GL0033604: non-AMP/OTHERS (Probabilities: 0.9921, 0.0039, 0.0040)\n","Processed V1.CD2-0-PT_GL0032823: non-AMP/OTHERS (Probabilities: 0.9933, 0.0032, 0.0036)\n","Processed SZEY-81A_GL0030339: non-AMP/OTHERS (Probabilities: 0.9785, 0.0043, 0.0172)\n","Processed MH0150_GL0106941: Neuropeptide (Probabilities: 0.0667, 0.0294, 0.9039)\n","Processed MH0188_GL0199062: non-AMP/OTHERS (Probabilities: 0.9882, 0.0049, 0.0069)\n","Processed MH0293_GL0098428: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0044)\n","Processed MH0287_GL0123541: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed MH0199_GL0037795: non-AMP/OTHERS (Probabilities: 0.9234, 0.0208, 0.0558)\n","Processed V1.FI08_GL0038671: non-AMP/OTHERS (Probabilities: 0.9821, 0.0069, 0.0110)\n","Processed MH0062_GL0066940: non-AMP/OTHERS (Probabilities: 0.9924, 0.0036, 0.0040)\n","Processed MH0003_GL0029107: non-AMP/OTHERS (Probabilities: 0.9851, 0.0099, 0.0050)\n","Processed 445972.ANACOL_01524: AMP (Probabilities: 0.1323, 0.8515, 0.0162)\n","Processed 445972.ANACOL_01524: AMP (Probabilities: 0.2824, 0.7021, 0.0154)\n","Processed MH0211_GL0137684: non-AMP/OTHERS (Probabilities: 0.9919, 0.0053, 0.0028)\n","Processed MH0006_GL0167941: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed MH0121_GL0012378: non-AMP/OTHERS (Probabilities: 0.9890, 0.0080, 0.0030)\n","Processed NOM026_GL0025553: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed O2.UC36-1_GL0029055: non-AMP/OTHERS (Probabilities: 0.9865, 0.0047, 0.0089)\n","Processed sp|Q16819|MEP1A_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed MH0141_GL0105124: non-AMP/OTHERS (Probabilities: 0.9933, 0.0030, 0.0037)\n","Processed 158337416-stool1_revised_scaffold1491_3_gene189446: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0071_GL0064176: Neuropeptide (Probabilities: 0.0067, 0.0087, 0.9846)\n","Processed V1.CD12-3_GL0111491: non-AMP/OTHERS (Probabilities: 0.8637, 0.0252, 0.1112)\n","Processed O2.UC2-2_GL0010076: non-AMP/OTHERS (Probabilities: 0.9528, 0.0096, 0.0376)\n","Processed V1.UC16-0_GL0022731: non-AMP/OTHERS (Probabilities: 0.9838, 0.0045, 0.0118)\n","Processed MH0086_GL0000688: non-AMP/OTHERS (Probabilities: 0.9925, 0.0043, 0.0032)\n","Processed MH0131_GL0050108: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed O2.UC22-1_GL0091094: non-AMP/OTHERS (Probabilities: 0.9669, 0.0107, 0.0224)\n","Processed MH0002_GL0001931: Neuropeptide (Probabilities: 0.3627, 0.0211, 0.6162)\n","Processed O2.UC2-2_GL0062355: non-AMP/OTHERS (Probabilities: 0.7467, 0.0754, 0.1779)\n","Processed MH0329_GL0122426: Neuropeptide (Probabilities: 0.0123, 0.0113, 0.9764)\n","Processed MH0001_GL0014429: Neuropeptide (Probabilities: 0.3994, 0.1517, 0.4489)\n","Processed MH0196_GL0113604: non-AMP/OTHERS (Probabilities: 0.9902, 0.0035, 0.0063)\n","Processed MH0012_GL0224638: non-AMP/OTHERS (Probabilities: 0.9924, 0.0034, 0.0041)\n","Processed O2.UC24-2_GL0024588: non-AMP/OTHERS (Probabilities: 0.9908, 0.0032, 0.0059)\n","Processed MH0131_GL0034178: non-AMP/OTHERS (Probabilities: 0.5425, 0.0544, 0.4031)\n","Processed O2.UC58-0_GL0135500: non-AMP/OTHERS (Probabilities: 0.9929, 0.0033, 0.0038)\n","Processed 763536994-stool2_revised_scaffold58331_1_gene173483: non-AMP/OTHERS (Probabilities: 0.9924, 0.0034, 0.0042)\n","Processed MH0142_GL0052272: non-AMP/OTHERS (Probabilities: 0.9700, 0.0153, 0.0147)\n","Processed 763536994-stool1_revised_scaffold24663_2_gene74792: non-AMP/OTHERS (Probabilities: 0.9926, 0.0034, 0.0040)\n","Processed 445972.ANACOL_04414: non-AMP/OTHERS (Probabilities: 0.9926, 0.0034, 0.0040)\n","Processed MH0011_GL0023955: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed MH0113_GL0001634: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0041)\n","Processed MH0088_GL0128177: non-AMP/OTHERS (Probabilities: 0.9890, 0.0040, 0.0071)\n","Processed MH0422_GL0178685: non-AMP/OTHERS (Probabilities: 0.9927, 0.0034, 0.0039)\n","Processed MH0101_GL0022285: non-AMP/OTHERS (Probabilities: 0.9923, 0.0037, 0.0040)\n","Processed MH0127_GL0018360: non-AMP/OTHERS (Probabilities: 0.9917, 0.0038, 0.0045)\n","Processed MH0198_GL0045866: non-AMP/OTHERS (Probabilities: 0.9789, 0.0069, 0.0142)\n","Processed MH0003_GL0085853: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed MH0003_GL0085853: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed 871237.SIN_1083: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed 871237.SIN_1083: Neuropeptide (Probabilities: 0.0067, 0.0086, 0.9847)\n","Processed ED12A_GL0088878: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed 518635.BIFANG_02554: Neuropeptide (Probabilities: 0.0065, 0.0085, 0.9849)\n","Processed MH0298_GL0023289: AMP (Probabilities: 0.1607, 0.8257, 0.0136)\n","Processed MH0414_GL0119044: non-AMP/OTHERS (Probabilities: 0.9917, 0.0046, 0.0038)\n","Processed MH0414_GL0119044: AMP (Probabilities: 0.1084, 0.8791, 0.0125)\n","Processed MH0013_GL0037461: AMP (Probabilities: 0.0951, 0.8918, 0.0130)\n","Processed NOM028_GL0070900: non-AMP/OTHERS (Probabilities: 0.8908, 0.0182, 0.0910)\n","Processed 763840445-stool2_revised_C893489_1_gene156020: non-AMP/OTHERS (Probabilities: 0.9918, 0.0037, 0.0045)\n","Processed NOM025_GL0054406: Neuropeptide (Probabilities: 0.4366, 0.0409, 0.5225)\n","Processed MH0385_GL0178922: non-AMP/OTHERS (Probabilities: 0.9820, 0.0052, 0.0128)\n","Processed 1042403.BALAC2494_00812: non-AMP/OTHERS (Probabilities: 0.7582, 0.0340, 0.2077)\n","Processed 206672.BL1792: non-AMP/OTHERS (Probabilities: 0.9733, 0.0078, 0.0188)\n","Processed V1.UC48-0_GL0002861: Neuropeptide (Probabilities: 0.0065, 0.0087, 0.9848)\n","Processed MH0268_GL0016092: Neuropeptide (Probabilities: 0.0736, 0.0256, 0.9008)\n","Processed MH0173_GL0078100: Neuropeptide (Probabilities: 0.0072, 0.0090, 0.9838)\n","Processed MH0173_GL0078100: non-AMP/OTHERS (Probabilities: 0.8810, 0.0632, 0.0558)\n","Processed O2.UC22-2_GL0142083: Neuropeptide (Probabilities: 0.0181, 0.0134, 0.9685)\n","Processed MH0020_GL0017062: non-AMP/OTHERS (Probabilities: 0.9923, 0.0038, 0.0039)\n","Processed MH0407_GL0189820: non-AMP/OTHERS (Probabilities: 0.9805, 0.0081, 0.0114)\n","Processed MH0077_GL0012539: non-AMP/OTHERS (Probabilities: 0.9843, 0.0128, 0.0029)\n","Processed MH0345_GL0178314: Neuropeptide (Probabilities: 0.0086, 0.0098, 0.9816)\n","Processed DOF004_GL0015130: Neuropeptide (Probabilities: 0.0084, 0.0093, 0.9823)\n","Processed T2D-135A_GL0106289: Neuropeptide (Probabilities: 0.0184, 0.0131, 0.9685)\n","Processed 158256496-stool1_revised_C962397_1_gene145221: non-AMP/OTHERS (Probabilities: 0.9739, 0.0157, 0.0104)\n","Processed MH0026_GL0018680: Neuropeptide (Probabilities: 0.1572, 0.0287, 0.8141)\n","Processed MH0012_GL0202239: non-AMP/OTHERS (Probabilities: 0.9391, 0.0117, 0.0492)\n","Processed MH0205_GL0036173: non-AMP/OTHERS (Probabilities: 0.9693, 0.0245, 0.0062)\n","Processed MH0006_GL0223975: non-AMP/OTHERS (Probabilities: 0.9910, 0.0045, 0.0045)\n","Processed MH0028_GL0028835: Neuropeptide (Probabilities: 0.0125, 0.0108, 0.9768)\n","Processed V1.CD29-0_GL0085594: Neuropeptide (Probabilities: 0.0066, 0.0087, 0.9847)\n","Processed MH0454_GL0027486: Neuropeptide (Probabilities: 0.0069, 0.0086, 0.9845)\n","Processed sp|P04406|G3P_HUMAN: non-AMP/OTHERS (Probabilities: 0.9872, 0.0101, 0.0027)\n","Processed MH0011_GL0034332: Neuropeptide (Probabilities: 0.0061, 0.0086, 0.9853)\n","Processed V1.FI24_GL0163422: Neuropeptide (Probabilities: 0.0105, 0.0099, 0.9797)\n","Processed MH0188_GL0007207: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0044)\n","Processed 1042403.BALAC2494_00034: non-AMP/OTHERS (Probabilities: 0.9931, 0.0030, 0.0039)\n","Processed MH0003_GL0096731: non-AMP/OTHERS (Probabilities: 0.9927, 0.0040, 0.0033)\n","Processed MH0230_GL0023581: non-AMP/OTHERS (Probabilities: 0.9930, 0.0029, 0.0041)\n","Processed V1.FI05_GL0057846: Neuropeptide (Probabilities: 0.0092, 0.0094, 0.9814)\n","Processed MH0416_GL0075697: non-AMP/OTHERS (Probabilities: 0.9929, 0.0030, 0.0042)\n","Processed MH0411_GL0067231: non-AMP/OTHERS (Probabilities: 0.9900, 0.0061, 0.0039)\n","Processed MH0010_GL0027989: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed O2.CD1-0-PT_GL0095055: non-AMP/OTHERS (Probabilities: 0.9817, 0.0044, 0.0139)\n","Processed sp|P21796|VDAC1_HUMAN: Neuropeptide (Probabilities: 0.0623, 0.0328, 0.9049)\n","Processed V1.UC49-0_GL0091607: non-AMP/OTHERS (Probabilities: 0.9920, 0.0053, 0.0028)\n","Processed MH0165_GL0060751: non-AMP/OTHERS (Probabilities: 0.8567, 0.0146, 0.1287)\n","Processed 479437.Elen_1276: Neuropeptide (Probabilities: 0.0149, 0.0121, 0.9730)\n","Processed MH0011_GL0048160: non-AMP/OTHERS (Probabilities: 0.7900, 0.0190, 0.1910)\n","Processed V1.FI32_GL0137497: Neuropeptide (Probabilities: 0.0069, 0.0088, 0.9842)\n","Processed MH0341_GL0016081: Neuropeptide (Probabilities: 0.0080, 0.0090, 0.9830)\n","Processed MH0006_GL0137798: non-AMP/OTHERS (Probabilities: 0.9534, 0.0165, 0.0300)\n","Processed MH0287_GL0104799: non-AMP/OTHERS (Probabilities: 0.9931, 0.0031, 0.0038)\n","Processed MH0224_GL0182052: non-AMP/OTHERS (Probabilities: 0.9882, 0.0047, 0.0071)\n","Processed O2.CD3-0-PT_GL0147991: non-AMP/OTHERS (Probabilities: 0.9918, 0.0039, 0.0043)\n","Processed MH0110_GL0071098: non-AMP/OTHERS (Probabilities: 0.9471, 0.0079, 0.0450)\n","Processed MH0086_GL0111107: non-AMP/OTHERS (Probabilities: 0.8712, 0.0118, 0.1169)\n","Processed MH0327_GL0102821: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0034)\n","Processed MH0002_GL0024598: non-AMP/OTHERS (Probabilities: 0.9767, 0.0081, 0.0152)\n","Processed MH0104_GL0094526: non-AMP/OTHERS (Probabilities: 0.9929, 0.0033, 0.0037)\n","Processed MH0345_GL0166263: Neuropeptide (Probabilities: 0.0121, 0.0112, 0.9768)\n","Processed V1.FI31_GL0135429: non-AMP/OTHERS (Probabilities: 0.9826, 0.0065, 0.0109)\n","Processed MH0188_GL0032497: non-AMP/OTHERS (Probabilities: 0.9643, 0.0187, 0.0170)\n","Processed MH0089_GL0030176: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0330_GL0081737: non-AMP/OTHERS (Probabilities: 0.9921, 0.0050, 0.0028)\n","Processed DOF011_GL0006893: non-AMP/OTHERS (Probabilities: 0.9915, 0.0044, 0.0041)\n","Processed V1.FI13_GL0004941: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0039)\n","Processed MH0382_GL0179377: non-AMP/OTHERS (Probabilities: 0.9871, 0.0047, 0.0081)\n","Processed DLF003_GL0026525: non-AMP/OTHERS (Probabilities: 0.9826, 0.0045, 0.0129)\n","Processed MH0012_GL0004024: Neuropeptide (Probabilities: 0.4356, 0.0228, 0.5416)\n","Processed MH0089_GL0054942: non-AMP/OTHERS (Probabilities: 0.9752, 0.0079, 0.0169)\n","Processed MH0012_GL0133682: non-AMP/OTHERS (Probabilities: 0.6722, 0.2085, 0.1192)\n","Processed 264199.stu1593: Neuropeptide (Probabilities: 0.0071, 0.0091, 0.9838)\n","Processed MH0037_GL0054052: non-AMP/OTHERS (Probabilities: 0.9679, 0.0055, 0.0266)\n","Processed MH0002_GL0049221: non-AMP/OTHERS (Probabilities: 0.9904, 0.0042, 0.0054)\n","Processed 585054.EFER_0954: non-AMP/OTHERS (Probabilities: 0.8879, 0.0294, 0.0827)\n","Processed MH0147_GL0072408: non-AMP/OTHERS (Probabilities: 0.5280, 0.4054, 0.0666)\n","Processed MH0002_GL0071847: non-AMP/OTHERS (Probabilities: 0.9205, 0.0747, 0.0048)\n","Processed MH0002_GL0038337: non-AMP/OTHERS (Probabilities: 0.9824, 0.0143, 0.0034)\n","Processed MH0006_GL0020034: non-AMP/OTHERS (Probabilities: 0.9840, 0.0070, 0.0089)\n","Processed V1.CD49-0_GL0047974: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0045_GL0001950: Neuropeptide (Probabilities: 0.0063, 0.0086, 0.9850)\n","Processed MH0277_GL0026995: non-AMP/OTHERS (Probabilities: 0.9919, 0.0029, 0.0051)\n","Processed 367928.BAD_1197: non-AMP/OTHERS (Probabilities: 0.9888, 0.0047, 0.0066)\n","Processed MH0277_GL0015228: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0131_GL0088903: non-AMP/OTHERS (Probabilities: 0.9934, 0.0033, 0.0034)\n","Processed MH0233_GL0020760: non-AMP/OTHERS (Probabilities: 0.9930, 0.0039, 0.0031)\n","Processed 411481.BIFADO_02111: Neuropeptide (Probabilities: 0.0092, 0.0096, 0.9812)\n","Processed MH0023_GL0013462: non-AMP/OTHERS (Probabilities: 0.9925, 0.0045, 0.0031)\n","Processed MH0203_GL0126315: Neuropeptide (Probabilities: 0.0075, 0.0089, 0.9836)\n","Processed sp|P12821|ACE_HUMAN: non-AMP/OTHERS (Probabilities: 0.9916, 0.0048, 0.0036)\n","Processed MH0401_GL0060836: non-AMP/OTHERS (Probabilities: 0.5007, 0.0383, 0.4610)\n","Processed MH0060_GL0019704: Neuropeptide (Probabilities: 0.0074, 0.0087, 0.9839)\n","Processed O2.UC46-0_GL0120745: non-AMP/OTHERS (Probabilities: 0.9933, 0.0033, 0.0034)\n","Processed O2.UC41-1_GL0081163: non-AMP/OTHERS (Probabilities: 0.9933, 0.0037, 0.0030)\n","Processed 763860675-stool1_revised_scaffold19053_2_gene151702: non-AMP/OTHERS (Probabilities: 0.9911, 0.0040, 0.0049)\n","Processed MH0002_GL0074407: non-AMP/OTHERS (Probabilities: 0.9837, 0.0046, 0.0116)\n","Processed MH0002_GL0033031: Neuropeptide (Probabilities: 0.0069, 0.0085, 0.9846)\n","Processed MH0252_GL0055637: non-AMP/OTHERS (Probabilities: 0.9831, 0.0062, 0.0107)\n","Processed MH0358_GL0153367: non-AMP/OTHERS (Probabilities: 0.9917, 0.0039, 0.0044)\n","Processed MH0198_GL0025621: non-AMP/OTHERS (Probabilities: 0.9594, 0.0079, 0.0327)\n","Processed MH0118_GL0106669: non-AMP/OTHERS (Probabilities: 0.9166, 0.0138, 0.0696)\n","Processed V1.CD17-4_GL0091217: non-AMP/OTHERS (Probabilities: 0.9897, 0.0031, 0.0072)\n","Processed MH0086_GL0062472: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0034)\n","Processed MH0086_GL0062472: non-AMP/OTHERS (Probabilities: 0.9934, 0.0037, 0.0029)\n","Processed 158499257-stool1_revised_C1447578_1_gene148027: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0062_GL0014965: non-AMP/OTHERS (Probabilities: 0.9931, 0.0038, 0.0031)\n","Processed MH0297_GL0133030: non-AMP/OTHERS (Probabilities: 0.9929, 0.0032, 0.0039)\n","Processed NOM027_GL0034897: non-AMP/OTHERS (Probabilities: 0.9912, 0.0036, 0.0052)\n","Processed MH0277_GL0006824: non-AMP/OTHERS (Probabilities: 0.9928, 0.0031, 0.0041)\n","Processed T2D-22A_GL0017772: non-AMP/OTHERS (Probabilities: 0.9903, 0.0036, 0.0061)\n","Processed N074A_GL0004859: Neuropeptide (Probabilities: 0.0071, 0.0085, 0.9844)\n","Processed MH0333_GL0069425: non-AMP/OTHERS (Probabilities: 0.9776, 0.0189, 0.0035)\n","Processed V1.UC5-3_GL0258268: non-AMP/OTHERS (Probabilities: 0.9887, 0.0087, 0.0026)\n","Processed O2.CD3-0-PT_GL0111408: non-AMP/OTHERS (Probabilities: 0.9932, 0.0032, 0.0036)\n","Processed V1.UC47-4_GL0056608: non-AMP/OTHERS (Probabilities: 0.6524, 0.0190, 0.3286)\n","Processed MH0002_GL0006448: Neuropeptide (Probabilities: 0.0329, 0.0151, 0.9520)\n","Processed MH0131_GL0167816: Neuropeptide (Probabilities: 0.0292, 0.0157, 0.9551)\n","Processed V1.CD7-4_GL0002018: Neuropeptide (Probabilities: 0.0488, 0.0195, 0.9318)\n","Processed sp|O14983|AT2A1_HUMAN: Neuropeptide (Probabilities: 0.0069, 0.0085, 0.9846)\n","Processed O2.UC59-2_GL0017609: non-AMP/OTHERS (Probabilities: 0.8400, 0.0640, 0.0960)\n","Processed MH0055_GL0046639: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0403_GL0178841: non-AMP/OTHERS (Probabilities: 0.9928, 0.0039, 0.0033)\n","Processed MH0264_GL0060731: Neuropeptide (Probabilities: 0.0091, 0.0094, 0.9815)\n","Processed O2.CD2-0_GL0096950: non-AMP/OTHERS (Probabilities: 0.9925, 0.0034, 0.0041)\n","Processed NOF013_GL0075115: non-AMP/OTHERS (Probabilities: 0.9918, 0.0057, 0.0026)\n","Processed MH0127_GL0041063: non-AMP/OTHERS (Probabilities: 0.9918, 0.0035, 0.0048)\n","Processed MH0373_GL0065303: Neuropeptide (Probabilities: 0.0093, 0.0094, 0.9813)\n","Processed MH0115_GL0015831: Neuropeptide (Probabilities: 0.0097, 0.0094, 0.9808)\n","Processed T2D-49A_GL0041344: non-AMP/OTHERS (Probabilities: 0.9925, 0.0043, 0.0032)\n","Processed O2.UC17-1_GL0130225: non-AMP/OTHERS (Probabilities: 0.9476, 0.0072, 0.0452)\n","Processed MH0356_GL0075468: non-AMP/OTHERS (Probabilities: 0.9922, 0.0034, 0.0044)\n","Processed V1.FI05_GL0136548: Neuropeptide (Probabilities: 0.0069, 0.0084, 0.9848)\n","Processed sp|P51148|RAB5C_HUMAN: non-AMP/OTHERS (Probabilities: 0.9930, 0.0036, 0.0035)\n","Processed MH0327_GL0089278: Neuropeptide (Probabilities: 0.0401, 0.0169, 0.9430)\n","Processed O2.UC18-1_GL0011138: non-AMP/OTHERS (Probabilities: 0.9929, 0.0029, 0.0042)\n","Processed MH0434_GL0012411: non-AMP/OTHERS (Probabilities: 0.9925, 0.0032, 0.0043)\n","Processed sp|P43098|FAS2_CANAX: non-AMP/OTHERS (Probabilities: 0.9931, 0.0029, 0.0040)\n","Processed O2.UC15-0_GL0087797: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed MH0122_GL0071151: non-AMP/OTHERS (Probabilities: 0.9931, 0.0036, 0.0032)\n","Processed MH0293_GL0177015: non-AMP/OTHERS (Probabilities: 0.9926, 0.0029, 0.0045)\n","Processed 763860675-stool1_revised_C1065643_1_gene58219: Neuropeptide (Probabilities: 0.3985, 0.0271, 0.5743)\n","Processed MH0101_GL0104134: Neuropeptide (Probabilities: 0.0074, 0.0090, 0.9836)\n","Processed 763496533-stool1_revised_scaffold4356_2_gene85099: Neuropeptide (Probabilities: 0.0083, 0.0092, 0.9825)\n","Processed MH0006_GL0013004: non-AMP/OTHERS (Probabilities: 0.8239, 0.0453, 0.1308)\n","Processed MH0003_GL0001491: non-AMP/OTHERS (Probabilities: 0.6497, 0.2991, 0.0512)\n","Processed O2.UC8-1_GL0111443: non-AMP/OTHERS (Probabilities: 0.9458, 0.0376, 0.0166)\n","Processed HT25A_GL0130354: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0041)\n","Processed MH0180_GL0079047: non-AMP/OTHERS (Probabilities: 0.9923, 0.0036, 0.0041)\n","Processed MH0122_GL0045642: Neuropeptide (Probabilities: 0.3425, 0.0407, 0.6168)\n","Processed MH0172_GL0103369: Neuropeptide (Probabilities: 0.3851, 0.0395, 0.5753)\n","Processed MH0088_GL0102957: non-AMP/OTHERS (Probabilities: 0.6058, 0.0368, 0.3573)\n","Processed O2.CD3-0-PT_GL0102446: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0301_GL0036285: Neuropeptide (Probabilities: 0.2816, 0.0345, 0.6839)\n","Processed MH0003_GL0024116: Neuropeptide (Probabilities: 0.0070, 0.0085, 0.9844)\n","Processed MH0187_GL0083240: non-AMP/OTHERS (Probabilities: 0.9927, 0.0035, 0.0037)\n","Processed MH0191_GL0060572: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0002_GL0018323: non-AMP/OTHERS (Probabilities: 0.9926, 0.0028, 0.0046)\n","Processed MH0370_GL0082221: Neuropeptide (Probabilities: 0.0838, 0.0267, 0.8895)\n","Processed MH0188_GL0060604: non-AMP/OTHERS (Probabilities: 0.9767, 0.0048, 0.0185)\n","Processed 763961826-stool2_revised_C699374_1_gene68595: Neuropeptide (Probabilities: 0.4396, 0.0392, 0.5213)\n","Processed MH0131_GL0100298: Neuropeptide (Probabilities: 0.0087, 0.0094, 0.9819)\n","Processed O2.UC48-0_GL0157955: non-AMP/OTHERS (Probabilities: 0.9928, 0.0031, 0.0041)\n","Processed V1.FI10_GL0093205: Neuropeptide (Probabilities: 0.0573, 0.0248, 0.9179)\n","Processed MH0452_GL0102636: Neuropeptide (Probabilities: 0.0367, 0.0187, 0.9445)\n","Processed O2.UC48-1_GL0111889: non-AMP/OTHERS (Probabilities: 0.9905, 0.0070, 0.0026)\n","Processed T2D-139A_GL0031638: Neuropeptide (Probabilities: 0.0079, 0.0104, 0.9817)\n","Processed MH0396_GL0188923: non-AMP/OTHERS (Probabilities: 0.9929, 0.0031, 0.0039)\n","Processed MH0337_GL0096873: non-AMP/OTHERS (Probabilities: 0.9482, 0.0155, 0.0363)\n","Processed MH0043_GL0021427: Neuropeptide (Probabilities: 0.0066, 0.0087, 0.9848)\n","Processed T2D-176A_GL0009034: non-AMP/OTHERS (Probabilities: 0.9922, 0.0036, 0.0042)\n","Processed 435842.HMPREF0848_00568: non-AMP/OTHERS (Probabilities: 0.9932, 0.0035, 0.0033)\n","Processed MH0050_GL0029332: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0368_GL0109275: Neuropeptide (Probabilities: 0.0149, 0.0122, 0.9729)\n","Processed 518634.BIFBRE_03804: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed O2.UC52-0_GL0060410: Neuropeptide (Probabilities: 0.0136, 0.0117, 0.9748)\n","Processed MH0055_GL0036804: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0203_GL0247970: non-AMP/OTHERS (Probabilities: 0.9919, 0.0046, 0.0035)\n","Processed MH0188_GL0025245: non-AMP/OTHERS (Probabilities: 0.9925, 0.0042, 0.0033)\n","Processed MH0356_GL0183680: non-AMP/OTHERS (Probabilities: 0.9925, 0.0042, 0.0033)\n","Processed MH0277_GL0016630: non-AMP/OTHERS (Probabilities: 0.9897, 0.0046, 0.0057)\n","Processed MH0038_GL0013094: non-AMP/OTHERS (Probabilities: 0.9926, 0.0038, 0.0036)\n","Processed 556263.FSEG_00611: Neuropeptide (Probabilities: 0.0132, 0.0113, 0.9755)\n","Processed DLM016_GL0034808: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed MH0188_GL0070948: non-AMP/OTHERS (Probabilities: 0.9927, 0.0038, 0.0035)\n","Processed MH0001_GL0014686: non-AMP/OTHERS (Probabilities: 0.9928, 0.0036, 0.0036)\n","Processed O2.UC20-0_GL0011298: non-AMP/OTHERS (Probabilities: 0.9929, 0.0037, 0.0033)\n","Processed 367928.BAD_0575: Neuropeptide (Probabilities: 0.0068, 0.0089, 0.9843)\n","Processed O2.UC2-2_GL0018056: non-AMP/OTHERS (Probabilities: 0.9795, 0.0054, 0.0151)\n","Processed MH0002_GL0063437: Neuropeptide (Probabilities: 0.2500, 0.0462, 0.7038)\n","Processed O2.UC48-0_GL0247841: non-AMP/OTHERS (Probabilities: 0.9925, 0.0040, 0.0034)\n","Processed 159207311-stool2_revised_C1433448_1_gene39941: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed 717962.CC1_16210: non-AMP/OTHERS (Probabilities: 0.9907, 0.0030, 0.0063)\n","Processed 717962.CC1_16210: non-AMP/OTHERS (Probabilities: 0.9926, 0.0034, 0.0040)\n","Processed MH0408_GL0082441: non-AMP/OTHERS (Probabilities: 0.9388, 0.0197, 0.0414)\n","Processed V1.FI02_GL0012023: non-AMP/OTHERS (Probabilities: 0.9865, 0.0057, 0.0078)\n","Processed MH0002_GL0028621: non-AMP/OTHERS (Probabilities: 0.9924, 0.0046, 0.0030)\n","Processed MH0139_GL0022533: non-AMP/OTHERS (Probabilities: 0.9773, 0.0046, 0.0181)\n","Processed MH0327_GL0058421: Neuropeptide (Probabilities: 0.0405, 0.0178, 0.9416)\n","Processed MH0005_GL0016199: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0230_GL0037000: non-AMP/OTHERS (Probabilities: 0.9928, 0.0031, 0.0041)\n","Processed 553973.CLOHYLEM_04918: Neuropeptide (Probabilities: 0.3708, 0.0319, 0.5973)\n","Processed 1042403.BALAC2494_00537: non-AMP/OTHERS (Probabilities: 0.9909, 0.0032, 0.0060)\n","Processed MH0274_GL0001684: Neuropeptide (Probabilities: 0.0070, 0.0088, 0.9842)\n","Processed MH0188_GL0079775: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed MH0062_GL0046828: non-AMP/OTHERS (Probabilities: 0.9895, 0.0046, 0.0060)\n","Processed MH0033_GL0055185: non-AMP/OTHERS (Probabilities: 0.9894, 0.0066, 0.0041)\n","Processed O2.UC22-1_GL0114932: Neuropeptide (Probabilities: 0.0079, 0.0101, 0.9819)\n","Processed O2.UC47-2_GL0022408: non-AMP/OTHERS (Probabilities: 0.9833, 0.0058, 0.0109)\n","Processed V1.CD34-0_GL0135713: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed V1.CD25-4_GL0004006: AMP (Probabilities: 0.3267, 0.6589, 0.0144)\n","Processed V1.CD6-0-PT_GL0177604: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed 160643649-stool1_revised_scaffold2415_3_gene14690: non-AMP/OTHERS (Probabilities: 0.7657, 0.0212, 0.2131)\n","Processed V1.UC50-1_GL0141695: non-AMP/OTHERS (Probabilities: 0.9914, 0.0031, 0.0056)\n","Processed MH0341_GL0040268: non-AMP/OTHERS (Probabilities: 0.9921, 0.0033, 0.0045)\n","Processed MH0012_GL0015236: Neuropeptide (Probabilities: 0.0080, 0.0091, 0.9829)\n","Processed O2.UC28-2_GL0009789: Neuropeptide (Probabilities: 0.0088, 0.0092, 0.9820)\n","Processed DOF004_GL0050832: non-AMP/OTHERS (Probabilities: 0.9906, 0.0030, 0.0064)\n","Processed MH0188_GL0013474: Neuropeptide (Probabilities: 0.0065, 0.0085, 0.9849)\n","Processed MH0193_GL0042210: non-AMP/OTHERS (Probabilities: 0.9610, 0.0153, 0.0236)\n","Processed O2.UC32-0_GL0077634: non-AMP/OTHERS (Probabilities: 0.6648, 0.2543, 0.0808)\n","Processed DOF004_GL0038283: non-AMP/OTHERS (Probabilities: 0.5165, 0.2603, 0.2232)\n","Processed MH0131_GL0144302: Neuropeptide (Probabilities: 0.0907, 0.0273, 0.8820)\n","Processed MH0086_GL0115171: non-AMP/OTHERS (Probabilities: 0.7375, 0.0769, 0.1856)\n","Processed MH0270_GL0003430: non-AMP/OTHERS (Probabilities: 0.9902, 0.0050, 0.0048)\n","Processed MH0145_GL0082459: non-AMP/OTHERS (Probabilities: 0.4844, 0.0953, 0.4203)\n","Processed V1.FI04_GL0093297: non-AMP/OTHERS (Probabilities: 0.6168, 0.0487, 0.3344)\n","Processed O2.UC2-2_GL0139406: non-AMP/OTHERS (Probabilities: 0.6654, 0.0564, 0.2782)\n","Processed MH0188_GL0123509: non-AMP/OTHERS (Probabilities: 0.4708, 0.0589, 0.4703)\n","Processed 159268001-stool2_revised_scaffold539_1_gene111678: non-AMP/OTHERS (Probabilities: 0.6093, 0.1318, 0.2589)\n","Processed V1.UC28-0_GL0101853: non-AMP/OTHERS (Probabilities: 0.5792, 0.0602, 0.3607)\n","Processed N052A_GL0031351: Neuropeptide (Probabilities: 0.3812, 0.0533, 0.5656)\n","Processed 765640925-stool1_revised_scaffold30229_5_gene41263: non-AMP/OTHERS (Probabilities: 0.9566, 0.0183, 0.0251)\n","Processed MH0211_GL0033018: non-AMP/OTHERS (Probabilities: 0.4777, 0.0636, 0.4587)\n","Processed MH0275_GL0131386: non-AMP/OTHERS (Probabilities: 0.9932, 0.0036, 0.0032)\n","Processed O2.CD3-0-PT_GL0011601: non-AMP/OTHERS (Probabilities: 0.9448, 0.0153, 0.0399)\n","Processed MH0131_GL0079282: Neuropeptide (Probabilities: 0.4577, 0.0422, 0.5001)\n","Processed V1.FI08_GL0002261: non-AMP/OTHERS (Probabilities: 0.7432, 0.0613, 0.1955)\n","Processed SZEY-39A_GL0006935: non-AMP/OTHERS (Probabilities: 0.9930, 0.0033, 0.0037)\n","Processed O2.UC30-0_GL0148258: non-AMP/OTHERS (Probabilities: 0.9868, 0.0040, 0.0092)\n","Processed MH0188_GL0089797: non-AMP/OTHERS (Probabilities: 0.9933, 0.0036, 0.0031)\n","Processed MH0440_GL0111783: non-AMP/OTHERS (Probabilities: 0.9924, 0.0034, 0.0043)\n","Processed 765074482-stool1_revised_scaffold6054_3_gene190414: non-AMP/OTHERS (Probabilities: 0.6330, 0.3102, 0.0568)\n","Processed V1.UC21-4_GL0029378: non-AMP/OTHERS (Probabilities: 0.9906, 0.0060, 0.0034)\n","Processed MH0188_GL0007212: non-AMP/OTHERS (Probabilities: 0.9916, 0.0049, 0.0035)\n","Processed V1.CD7-0-PN_GL0017295: non-AMP/OTHERS (Probabilities: 0.9788, 0.0161, 0.0051)\n","Processed O2.UC48-0_GL0012195: non-AMP/OTHERS (Probabilities: 0.9150, 0.0093, 0.0757)\n","Processed T2D-25A_GL0016689: non-AMP/OTHERS (Probabilities: 0.9922, 0.0031, 0.0047)\n","Processed MH0003_GL0075872: non-AMP/OTHERS (Probabilities: 0.9810, 0.0049, 0.0142)\n","Processed O2.UC22-1_GL0046220: non-AMP/OTHERS (Probabilities: 0.4685, 0.2004, 0.3311)\n","Processed MH0066_GL0058173: non-AMP/OTHERS (Probabilities: 0.9929, 0.0031, 0.0040)\n","Processed MH0410_GL0030258: non-AMP/OTHERS (Probabilities: 0.8381, 0.0209, 0.1410)\n","Processed MH0250_GL0074259: Neuropeptide (Probabilities: 0.2702, 0.0403, 0.6895)\n","Processed MH0184_GL0068950: Neuropeptide (Probabilities: 0.0106, 0.0104, 0.9790)\n","Processed T2D-105A_GL0017842: non-AMP/OTHERS (Probabilities: 0.9930, 0.0030, 0.0041)\n","Processed sp|Q9Y6R7|FCGBP_HUMAN: Neuropeptide (Probabilities: 0.3996, 0.0488, 0.5515)\n","Processed MH0389_GL0016915: non-AMP/OTHERS (Probabilities: 0.9774, 0.0088, 0.0139)\n","Processed MH0159_GL0001056: Neuropeptide (Probabilities: 0.0073, 0.0087, 0.9840)\n","Processed O2.CD2-0-PT_GL0062968: non-AMP/OTHERS (Probabilities: 0.9920, 0.0034, 0.0045)\n","Processed MH0002_GL0014894: non-AMP/OTHERS (Probabilities: 0.9605, 0.0120, 0.0275)\n","Processed MH0002_GL0014894: non-AMP/OTHERS (Probabilities: 0.9894, 0.0046, 0.0060)\n","Processed MH0446_GL0134151: non-AMP/OTHERS (Probabilities: 0.9925, 0.0040, 0.0035)\n","Processed MH0124_GL0076378: non-AMP/OTHERS (Probabilities: 0.9908, 0.0058, 0.0035)\n","Processed MH0356_GL0130219: non-AMP/OTHERS (Probabilities: 0.9918, 0.0034, 0.0048)\n","Processed 160502038-stool1_revised_C1109617_1_gene82814: non-AMP/OTHERS (Probabilities: 0.9919, 0.0030, 0.0050)\n","Processed MH0199_GL0094040: non-AMP/OTHERS (Probabilities: 0.9933, 0.0030, 0.0037)\n","Processed N062A_GL0075599: non-AMP/OTHERS (Probabilities: 0.6012, 0.0606, 0.3383)\n","Processed MH0161_GL0032551: non-AMP/OTHERS (Probabilities: 0.9923, 0.0030, 0.0047)\n","Processed MH0003_GL0009127: non-AMP/OTHERS (Probabilities: 0.9802, 0.0100, 0.0098)\n","Processed MH0088_GL0109387: non-AMP/OTHERS (Probabilities: 0.9841, 0.0082, 0.0077)\n","Processed MH0119_GL0102704: Neuropeptide (Probabilities: 0.0068, 0.0086, 0.9846)\n","Processed DOF012_GL0009489: Neuropeptide (Probabilities: 0.4265, 0.0270, 0.5465)\n","Processed MH0088_GL0090980: non-AMP/OTHERS (Probabilities: 0.9888, 0.0051, 0.0061)\n","Processed 391904.BLIJ_2272: non-AMP/OTHERS (Probabilities: 0.9935, 0.0034, 0.0031)\n","Processed MH0161_GL0140039: non-AMP/OTHERS (Probabilities: 0.9935, 0.0034, 0.0031)\n","Processed MH0108_GL0077260: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed DOM014_GL0024811: Neuropeptide (Probabilities: 0.0062, 0.0085, 0.9852)\n","Processed MH0188_GL0049375: non-AMP/OTHERS (Probabilities: 0.9863, 0.0071, 0.0066)\n","Processed MH0230_GL0079910: Neuropeptide (Probabilities: 0.0098, 0.0094, 0.9808)\n","Processed MH0179_GL0032520: Neuropeptide (Probabilities: 0.0064, 0.0085, 0.9850)\n","Processed sp|Q04609|FOLH1_HUMAN: non-AMP/OTHERS (Probabilities: 0.9744, 0.0101, 0.0155)\n","Processed sp|Q9UQQ1|NALDL_HUMAN: Neuropeptide (Probabilities: 0.0091, 0.0097, 0.9813)\n","Processed V1.FI32_GL0055432: non-AMP/OTHERS (Probabilities: 0.9922, 0.0037, 0.0041)\n","Processed MH0199_GL0034725: Neuropeptide (Probabilities: 0.3428, 0.0277, 0.6295)\n","Processed MH0152_GL0024976: non-AMP/OTHERS (Probabilities: 0.9643, 0.0060, 0.0296)\n","Processed MH0156_GL0057004: non-AMP/OTHERS (Probabilities: 0.9887, 0.0053, 0.0060)\n","Processed 518635.BIFANG_03749: non-AMP/OTHERS (Probabilities: 0.9918, 0.0036, 0.0046)\n","Processed 1042403.BALAC2494_01398: non-AMP/OTHERS (Probabilities: 0.8819, 0.0280, 0.0900)\n","Processed MH0196_GL0042812: AMP (Probabilities: 0.1736, 0.8060, 0.0204)\n","Processed MH0002_GL0073923: non-AMP/OTHERS (Probabilities: 0.9914, 0.0051, 0.0035)\n","Processed MH0011_GL0105280: non-AMP/OTHERS (Probabilities: 0.9702, 0.0058, 0.0240)\n","Processed MH0211_GL0139898: non-AMP/OTHERS (Probabilities: 0.9929, 0.0037, 0.0033)\n","Processed T2D-149A_GL0004747: non-AMP/OTHERS (Probabilities: 0.5123, 0.0510, 0.4368)\n","Processed MH0065_GL0032077: non-AMP/OTHERS (Probabilities: 0.6796, 0.0535, 0.2669)\n","Processed MH0233_GL0034680: non-AMP/OTHERS (Probabilities: 0.9904, 0.0030, 0.0065)\n","Processed MH0012_GL0015236: non-AMP/OTHERS (Probabilities: 0.9859, 0.0065, 0.0076)\n","Processed MH0012_GL0009665: Neuropeptide (Probabilities: 0.0068, 0.0087, 0.9845)\n","Processed V1.CD31-0_GL0134962: non-AMP/OTHERS (Probabilities: 0.9862, 0.0041, 0.0097)\n","Processed MH0100_GL0108102: Neuropeptide (Probabilities: 0.3141, 0.0207, 0.6653)\n","Processed MH0006_GL0189559: non-AMP/OTHERS (Probabilities: 0.9416, 0.0177, 0.0407)\n","Processed MH0110_GL0043950: Neuropeptide (Probabilities: 0.0174, 0.0134, 0.9691)\n","Processed MH0462_GL0181619: Neuropeptide (Probabilities: 0.0091, 0.0094, 0.9815)\n","Processed O2.UC17-1_GL0001092: non-AMP/OTHERS (Probabilities: 0.9929, 0.0031, 0.0040)\n","Processed 206672.BL0951: non-AMP/OTHERS (Probabilities: 0.9932, 0.0032, 0.0036)\n","Processed O2.UC26-2_GL0054152: Neuropeptide (Probabilities: 0.0073, 0.0092, 0.9835)\n","Processed O2.UC14-1_GL0004346: Neuropeptide (Probabilities: 0.0070, 0.0085, 0.9844)\n","Processed O2.UC15-0_GL0016309: non-AMP/OTHERS (Probabilities: 0.9772, 0.0041, 0.0186)\n","Processed MH0196_GL0067240: non-AMP/OTHERS (Probabilities: 0.9929, 0.0031, 0.0041)\n","Processed MH0199_GL0081608: non-AMP/OTHERS (Probabilities: 0.9922, 0.0039, 0.0040)\n","Processed MH0416_GL0075412: non-AMP/OTHERS (Probabilities: 0.5411, 0.0360, 0.4229)\n","Processed V1.UC17-2_GL0030336: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0037)\n","Processed MH0211_GL0124964: Neuropeptide (Probabilities: 0.0133, 0.0108, 0.9759)\n","Processed V1.UC60-0_GL0050439: non-AMP/OTHERS (Probabilities: 0.9921, 0.0034, 0.0044)\n","Processed 518634.BIFBRE_03753: non-AMP/OTHERS (Probabilities: 0.9910, 0.0048, 0.0043)\n","Processed MH0008_GL0041482: non-AMP/OTHERS (Probabilities: 0.9915, 0.0041, 0.0044)\n","Processed 435842.HMPREF0848_01240: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0131_GL0095173: non-AMP/OTHERS (Probabilities: 0.9904, 0.0060, 0.0035)\n","Processed HT8A_GL0040877: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0044)\n","Processed MH0428_GL0029105: non-AMP/OTHERS (Probabilities: 0.9927, 0.0032, 0.0041)\n","Processed 411481.BIFADO_02111: non-AMP/OTHERS (Probabilities: 0.9922, 0.0043, 0.0035)\n","Processed O2.UC50-2_GL0249423: non-AMP/OTHERS (Probabilities: 0.5638, 0.0314, 0.4047)\n","Processed MH0233_GL0028698: non-AMP/OTHERS (Probabilities: 0.9923, 0.0032, 0.0046)\n","Processed V1.CD34-0_GL0158468: non-AMP/OTHERS (Probabilities: 0.9655, 0.0083, 0.0262)\n","Processed MH0211_GL0040265: Neuropeptide (Probabilities: 0.1631, 0.0235, 0.8134)\n","Processed MH0006_GL0038634: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0037)\n","Processed MH0211_GL0140994: non-AMP/OTHERS (Probabilities: 0.9927, 0.0034, 0.0039)\n","Processed MH0196_GL0207991: Neuropeptide (Probabilities: 0.3291, 0.0928, 0.5781)\n","Processed MH0006_GL0055426: non-AMP/OTHERS (Probabilities: 0.4801, 0.2414, 0.2785)\n","Processed DOF013_GL0043682: non-AMP/OTHERS (Probabilities: 0.9680, 0.0073, 0.0247)\n","Processed 908612.HMPREF9720_2076: non-AMP/OTHERS (Probabilities: 0.9737, 0.0067, 0.0196)\n","Processed V1.CD6-0-PN_GL0123164: non-AMP/OTHERS (Probabilities: 0.9888, 0.0035, 0.0077)\n","Processed 160704339-stool1_revised_scaffold32330_1_gene175163: non-AMP/OTHERS (Probabilities: 0.6804, 0.0221, 0.2974)\n","Processed sp|P20618|PSB1_HUMAN: Neuropeptide (Probabilities: 0.2109, 0.0495, 0.7396)\n","Processed MH0143_GL0050947: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed V1.FI08_GL0002261: Neuropeptide (Probabilities: 0.0112, 0.0103, 0.9786)\n","Processed MH0131_GL0096652: Neuropeptide (Probabilities: 0.3470, 0.0557, 0.5973)\n","Processed DOM014_GL0024811: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0338_GL0053273: non-AMP/OTHERS (Probabilities: 0.9879, 0.0034, 0.0088)\n","Processed T2D-146A_GL0042295: non-AMP/OTHERS (Probabilities: 0.9912, 0.0033, 0.0055)\n","Processed MH0348_GL0122167: non-AMP/OTHERS (Probabilities: 0.4962, 0.0439, 0.4599)\n","Processed MH0440_GL0073289: Neuropeptide (Probabilities: 0.2921, 0.0423, 0.6656)\n","Processed V1.CD18-3_GL0016916: non-AMP/OTHERS (Probabilities: 0.9800, 0.0063, 0.0137)\n","Processed MH0268_GL0138137: non-AMP/OTHERS (Probabilities: 0.9781, 0.0069, 0.0149)\n","Processed MH0462_GL0130961: non-AMP/OTHERS (Probabilities: 0.9591, 0.0093, 0.0317)\n","Processed MH0236_GL0091042: non-AMP/OTHERS (Probabilities: 0.9732, 0.0087, 0.0182)\n","Processed MH0211_GL0050697: non-AMP/OTHERS (Probabilities: 0.9903, 0.0042, 0.0056)\n","Processed MH0089_GL0026058: non-AMP/OTHERS (Probabilities: 0.9260, 0.0112, 0.0628)\n","Processed O2.UC13-1_GL0011684: non-AMP/OTHERS (Probabilities: 0.9661, 0.0069, 0.0269)\n","Processed MH0425_GL0012276: non-AMP/OTHERS (Probabilities: 0.9880, 0.0052, 0.0068)\n","Processed V1.UC2-0_GL0046032: non-AMP/OTHERS (Probabilities: 0.9758, 0.0075, 0.0167)\n","Processed MH0002_GL0018942: non-AMP/OTHERS (Probabilities: 0.8713, 0.0266, 0.1021)\n","Processed MH0003_GL0030510: non-AMP/OTHERS (Probabilities: 0.7665, 0.0219, 0.2116)\n","Processed 367928.BAD_1197: non-AMP/OTHERS (Probabilities: 0.9837, 0.0050, 0.0113)\n","Processed 160582958-stool1_revised_scaffold9477_1_gene8746: Neuropeptide (Probabilities: 0.0351, 0.0198, 0.9451)\n","Processed 763597684-stool1_revised_C1139432_1_gene107939: Neuropeptide (Probabilities: 0.1767, 0.0382, 0.7852)\n","Processed MH0039_GL0001091: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9849)\n","Processed N003A_GL0016608: non-AMP/OTHERS (Probabilities: 0.9839, 0.0049, 0.0113)\n","Processed MH0068_GL0040086: Neuropeptide (Probabilities: 0.0787, 0.0428, 0.8785)\n","Processed MH0145_GL0017882: non-AMP/OTHERS (Probabilities: 0.5089, 0.0585, 0.4325)\n","Processed 246515023-stool1_revised_scaffold5616_2_gene26719: non-AMP/OTHERS (Probabilities: 0.9507, 0.0341, 0.0151)\n","Processed MH0008_GL0046039: non-AMP/OTHERS (Probabilities: 0.9911, 0.0040, 0.0049)\n","Processed MH0397_GL0091242: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0409_GL0180768: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed T2D-83A_GL0041107: non-AMP/OTHERS (Probabilities: 0.9875, 0.0034, 0.0091)\n","Processed MH0008_GL0024623: Neuropeptide (Probabilities: 0.1619, 0.0230, 0.8151)\n","Processed MH0277_GL0007547: non-AMP/OTHERS (Probabilities: 0.9371, 0.0571, 0.0058)\n","Processed 457416.HMPREF0873_01787: Neuropeptide (Probabilities: 0.0121, 0.0112, 0.9767)\n","Processed O2.UC4-1_GL0166577: Neuropeptide (Probabilities: 0.0093, 0.0096, 0.9810)\n","Processed MH0151_GL0005778: non-AMP/OTHERS (Probabilities: 0.7466, 0.0246, 0.2288)\n","Processed MH0198_GL0102182: Neuropeptide (Probabilities: 0.0445, 0.0274, 0.9282)\n","Processed MH0193_GL0004423: non-AMP/OTHERS (Probabilities: 0.9925, 0.0039, 0.0036)\n","Processed O2.UC22-2_GL0133686: Neuropeptide (Probabilities: 0.0110, 0.0098, 0.9792)\n","Processed MH0087_GL0030880: Neuropeptide (Probabilities: 0.0432, 0.0145, 0.9423)\n","Processed MH0012_GL0127755: Neuropeptide (Probabilities: 0.0074, 0.0093, 0.9833)\n","Processed MH0388_GL0030289: non-AMP/OTHERS (Probabilities: 0.8723, 0.0286, 0.0991)\n","Processed MH0365_GL0124917: Neuropeptide (Probabilities: 0.0068, 0.0089, 0.9844)\n","Processed MH0238_GL0182265: Neuropeptide (Probabilities: 0.0207, 0.0138, 0.9655)\n","Processed MH0238_GL0182265: non-AMP/OTHERS (Probabilities: 0.9403, 0.0137, 0.0460)\n","Processed V1.UC6-0_GL0012925: Neuropeptide (Probabilities: 0.0726, 0.0253, 0.9020)\n","Processed MH0371_GL0055922: non-AMP/OTHERS (Probabilities: 0.9347, 0.0110, 0.0543)\n","Processed MH0100_GL0046759: non-AMP/OTHERS (Probabilities: 0.9912, 0.0059, 0.0029)\n","Processed MH0100_GL0046759: non-AMP/OTHERS (Probabilities: 0.9927, 0.0045, 0.0029)\n","Processed MH0086_GL0047393: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed V1.CD8-0-PN_GL0054249: non-AMP/OTHERS (Probabilities: 0.9909, 0.0042, 0.0049)\n","Processed MH0272_GL0009333: non-AMP/OTHERS (Probabilities: 0.9875, 0.0036, 0.0089)\n","Processed NLF009_GL0077737: non-AMP/OTHERS (Probabilities: 0.5144, 0.0295, 0.4561)\n","Processed MH0002_GL0040735: non-AMP/OTHERS (Probabilities: 0.5676, 0.0706, 0.3618)\n","Processed MH0147_GL0083186: Neuropeptide (Probabilities: 0.3960, 0.0252, 0.5788)\n","Processed MH0011_GL0102674: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed V1.UC47-4_GL0013011: Neuropeptide (Probabilities: 0.0631, 0.0212, 0.9157)\n","Processed MH0028_GL0081182: Neuropeptide (Probabilities: 0.0070, 0.0087, 0.9843)\n","Processed T2D-76A_GL0043878: non-AMP/OTHERS (Probabilities: 0.9873, 0.0062, 0.0065)\n","Processed V1.CD51-0_GL0181853: non-AMP/OTHERS (Probabilities: 0.9177, 0.0230, 0.0592)\n","Processed MH0198_GL0018776: non-AMP/OTHERS (Probabilities: 0.9896, 0.0034, 0.0070)\n","Processed O2.UC50-0_GL0340073: non-AMP/OTHERS (Probabilities: 0.9910, 0.0029, 0.0061)\n","Processed O2.UC22-2_GL0038235: non-AMP/OTHERS (Probabilities: 0.9922, 0.0031, 0.0046)\n","Processed MH0322_GL0083309: non-AMP/OTHERS (Probabilities: 0.9911, 0.0032, 0.0057)\n","Processed MH0022_GL0023576: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0044)\n","Processed MH0088_GL0073250: non-AMP/OTHERS (Probabilities: 0.9925, 0.0030, 0.0045)\n","Processed MH0130_GL0013716: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0038)\n","Processed V1.FI17_GL0215860: non-AMP/OTHERS (Probabilities: 0.9921, 0.0034, 0.0045)\n","Processed ED13A_GL0065402: non-AMP/OTHERS (Probabilities: 0.9882, 0.0040, 0.0078)\n","Processed V1.CD18-0_GL0025638: non-AMP/OTHERS (Probabilities: 0.9883, 0.0038, 0.0079)\n","Processed O2.UC19-1_GL0081112: Neuropeptide (Probabilities: 0.1564, 0.0528, 0.7908)\n","Processed MH0003_GL0079782: non-AMP/OTHERS (Probabilities: 0.9908, 0.0044, 0.0048)\n","Processed 537937.BLIG_01343: non-AMP/OTHERS (Probabilities: 0.9882, 0.0033, 0.0085)\n","Processed O2.UC27-1_GL0024439: non-AMP/OTHERS (Probabilities: 0.6796, 0.1694, 0.1510)\n","Processed MH0203_GL0075813: Neuropeptide (Probabilities: 0.2287, 0.0393, 0.7320)\n","Processed V1.FI26_GL0199729: Neuropeptide (Probabilities: 0.0192, 0.0133, 0.9675)\n","Processed V1.UC26-0_GL0080589: non-AMP/OTHERS (Probabilities: 0.5624, 0.1488, 0.2888)\n","Processed MH0277_GL0039553: non-AMP/OTHERS (Probabilities: 0.9919, 0.0031, 0.0050)\n","Processed N083A_GL0010421: non-AMP/OTHERS (Probabilities: 0.9924, 0.0044, 0.0032)\n","Processed MH0008_GL0044786: non-AMP/OTHERS (Probabilities: 0.9912, 0.0059, 0.0029)\n","Processed MH0055_GL0031824: Neuropeptide (Probabilities: 0.0124, 0.0116, 0.9760)\n","Processed MH0055_GL0031824: Neuropeptide (Probabilities: 0.0094, 0.0094, 0.9812)\n","Processed MH0002_GL0001930: Neuropeptide (Probabilities: 0.0333, 0.0204, 0.9464)\n","Processed MH0233_GL0013435: Neuropeptide (Probabilities: 0.2735, 0.0287, 0.6977)\n","Processed MH0297_GL0133030: Neuropeptide (Probabilities: 0.0837, 0.0218, 0.8945)\n","Processed MH0297_GL0133030: non-AMP/OTHERS (Probabilities: 0.9794, 0.0157, 0.0049)\n","Processed O2.UC35-1_GL0114693: non-AMP/OTHERS (Probabilities: 0.9891, 0.0045, 0.0064)\n","Processed MH0197_GL0035102: non-AMP/OTHERS (Probabilities: 0.9926, 0.0038, 0.0037)\n","Processed MH0251_GL0078655: non-AMP/OTHERS (Probabilities: 0.9763, 0.0042, 0.0195)\n","Processed MH0359_GL0113801: Neuropeptide (Probabilities: 0.1012, 0.0243, 0.8746)\n","Processed MH0359_GL0113801: non-AMP/OTHERS (Probabilities: 0.6054, 0.0243, 0.3703)\n","Processed 809635352-stool1_revised_C1593856_1_gene115785: non-AMP/OTHERS (Probabilities: 0.7518, 0.0259, 0.2223)\n","Processed O2.UC48-0_GL0172641: Neuropeptide (Probabilities: 0.0078, 0.0089, 0.9833)\n","Processed MH0062_GL0031466: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed N011A_GL0006474: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0341_GL0070958: non-AMP/OTHERS (Probabilities: 0.9886, 0.0041, 0.0073)\n","Processed 160319967-stool1_revised_C575601_1_gene50298: Neuropeptide (Probabilities: 0.0194, 0.0134, 0.9671)\n","Processed MH0002_GL0071095: Neuropeptide (Probabilities: 0.0186, 0.0125, 0.9689)\n","Processed O2.UC23-2_GL0061055: non-AMP/OTHERS (Probabilities: 0.9928, 0.0036, 0.0036)\n","Processed T2D-109A_GL0059171: non-AMP/OTHERS (Probabilities: 0.9399, 0.0120, 0.0481)\n","Processed MH0058_GL0066702: non-AMP/OTHERS (Probabilities: 0.9888, 0.0049, 0.0062)\n","Processed MH0116_GL0055143: Neuropeptide (Probabilities: 0.0106, 0.0098, 0.9796)\n","Processed MH0416_GL0231701: Neuropeptide (Probabilities: 0.0067, 0.0087, 0.9846)\n","Processed MH0203_GL0081518: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed O2.UC19-1_GL0085858: non-AMP/OTHERS (Probabilities: 0.9894, 0.0038, 0.0068)\n","Processed MH0161_GL0150015: non-AMP/OTHERS (Probabilities: 0.9917, 0.0051, 0.0032)\n","Processed MH0061_GL0002633: non-AMP/OTHERS (Probabilities: 0.9931, 0.0030, 0.0040)\n","Processed 556268.OFAG_01714: non-AMP/OTHERS (Probabilities: 0.9007, 0.0167, 0.0825)\n","Processed O2.UC40-2_GL0032577: non-AMP/OTHERS (Probabilities: 0.9921, 0.0034, 0.0045)\n","Processed MH0006_GL0190435: non-AMP/OTHERS (Probabilities: 0.9902, 0.0034, 0.0063)\n","Processed MH0126_GL0108383: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9848)\n","Processed V1.UC25-0_GL0051029: non-AMP/OTHERS (Probabilities: 0.9923, 0.0043, 0.0033)\n","Processed V1.CD22-0_GL0142658: non-AMP/OTHERS (Probabilities: 0.9919, 0.0032, 0.0049)\n","Processed MH0233_GL0050390: non-AMP/OTHERS (Probabilities: 0.9919, 0.0029, 0.0052)\n","Processed MH0440_GL0069139: non-AMP/OTHERS (Probabilities: 0.9876, 0.0038, 0.0086)\n","Processed SZEY-90A_GL0036257: non-AMP/OTHERS (Probabilities: 0.9904, 0.0059, 0.0037)\n","Processed SZEY-90A_GL0036257: non-AMP/OTHERS (Probabilities: 0.9907, 0.0038, 0.0055)\n","Processed V1.FI01_GL0193945: Neuropeptide (Probabilities: 0.3957, 0.0194, 0.5849)\n","Processed MH0002_GL0050951: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed MH0230_GL0032016: non-AMP/OTHERS (Probabilities: 0.9330, 0.0264, 0.0406)\n","Processed V1.UC27-0_GL0112645: non-AMP/OTHERS (Probabilities: 0.9644, 0.0200, 0.0156)\n","Processed 763860675-stool1_revised_scaffold14158_1_gene108376: non-AMP/OTHERS (Probabilities: 0.9571, 0.0057, 0.0371)\n","Processed MH0122_GL0077613: non-AMP/OTHERS (Probabilities: 0.9924, 0.0030, 0.0046)\n","Processed MH0244_GL0062774: Neuropeptide (Probabilities: 0.0061, 0.0086, 0.9852)\n","Processed MH0337_GL0097066: non-AMP/OTHERS (Probabilities: 0.9911, 0.0042, 0.0047)\n","Processed O2.UC15-1_GL0042148: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0045)\n","Processed 435842.HMPREF0848_01851: Neuropeptide (Probabilities: 0.0070, 0.0084, 0.9846)\n","Processed 521003.COLINT_02525: Neuropeptide (Probabilities: 0.0069, 0.0085, 0.9846)\n","Processed O2.UC31-2_GL0031904: non-AMP/OTHERS (Probabilities: 0.9897, 0.0048, 0.0055)\n","Processed MH0188_GL0069484: non-AMP/OTHERS (Probabilities: 0.9503, 0.0125, 0.0372)\n","Processed MH0452_GL0028253: Neuropeptide (Probabilities: 0.0211, 0.0140, 0.9649)\n","Processed MH0192_GL0139005: Neuropeptide (Probabilities: 0.0326, 0.0171, 0.9503)\n","Processed V1.CD28-0_GL0022846: Neuropeptide (Probabilities: 0.0095, 0.0094, 0.9810)\n","Processed MH0370_GL0127611: non-AMP/OTHERS (Probabilities: 0.9792, 0.0050, 0.0158)\n","Processed T2D-49A_GL0019577: non-AMP/OTHERS (Probabilities: 0.9904, 0.0031, 0.0065)\n","Processed V1.UC38-0_GL0113297: non-AMP/OTHERS (Probabilities: 0.9922, 0.0033, 0.0045)\n","Processed MH0074_GL0058556: non-AMP/OTHERS (Probabilities: 0.9248, 0.0169, 0.0583)\n","Processed MH0268_GL0123508: non-AMP/OTHERS (Probabilities: 0.9900, 0.0033, 0.0067)\n","Processed sp|P01008|ANT3_HUMAN: non-AMP/OTHERS (Probabilities: 0.9860, 0.0082, 0.0058)\n","Processed 763820215-stool1_revised_scaffold2191_1_gene5563: non-AMP/OTHERS (Probabilities: 0.9902, 0.0032, 0.0066)\n","Processed 763820215-stool1_revised_scaffold2191_1_gene5563: non-AMP/OTHERS (Probabilities: 0.9893, 0.0039, 0.0067)\n","Processed 158499257-stool2_revised_scaffold68682_1_gene222426: Neuropeptide (Probabilities: 0.0075, 0.0085, 0.9840)\n","Processed DOM010_GL0012419: non-AMP/OTHERS (Probabilities: 0.9885, 0.0044, 0.0071)\n","Processed MH0247_GL0055399: Neuropeptide (Probabilities: 0.0070, 0.0087, 0.9843)\n","Processed MH0203_GL0143400: Neuropeptide (Probabilities: 0.0848, 0.0316, 0.8836)\n","Processed 158924089-stool1_revised_scaffold12016_1_gene56879: Neuropeptide (Probabilities: 0.0366, 0.0181, 0.9454)\n","Processed MH0277_GL0034239: Neuropeptide (Probabilities: 0.0117, 0.0102, 0.9782)\n","Processed O2.UC29-1_GL0050127: Neuropeptide (Probabilities: 0.0773, 0.0348, 0.8879)\n","Processed V1.UC55-4_GL0115910: Neuropeptide (Probabilities: 0.2362, 0.0237, 0.7400)\n","Processed MH0274_GL0061161: Neuropeptide (Probabilities: 0.0128, 0.0111, 0.9762)\n","Processed 210007.SMU_1691: non-AMP/OTHERS (Probabilities: 0.9914, 0.0046, 0.0040)\n","Processed O2.UC46-1_GL0109746: Neuropeptide (Probabilities: 0.1248, 0.0353, 0.8399)\n","Processed MH0437_GL0011266: Neuropeptide (Probabilities: 0.0159, 0.0114, 0.9727)\n","Processed MH0150_GL0131282: Neuropeptide (Probabilities: 0.0066, 0.0086, 0.9849)\n","Processed V1.CD12-0_GL0029180: Neuropeptide (Probabilities: 0.0445, 0.0193, 0.9362)\n","Processed MH0088_GL0109912: Neuropeptide (Probabilities: 0.0089, 0.0096, 0.9815)\n","Processed MH0088_GL0109912: non-AMP/OTHERS (Probabilities: 0.8862, 0.0130, 0.1008)\n","Processed MH0188_GL0125761: Neuropeptide (Probabilities: 0.0174, 0.0127, 0.9700)\n","Processed T2D-133A_GL0087234: non-AMP/OTHERS (Probabilities: 0.9919, 0.0045, 0.0036)\n","Processed 159753524-stool2_revised_C592050_1_gene31357: non-AMP/OTHERS (Probabilities: 0.9348, 0.0094, 0.0558)\n","Processed MH0148_GL0141995: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0416_GL0101205: Neuropeptide (Probabilities: 0.0124, 0.0103, 0.9773)\n","Processed O2.UC59-0_GL0062682: non-AMP/OTHERS (Probabilities: 0.9664, 0.0106, 0.0230)\n","Processed MH0188_GL0102916: non-AMP/OTHERS (Probabilities: 0.9701, 0.0074, 0.0225)\n","Processed V1.CD36-0_GL0147854: non-AMP/OTHERS (Probabilities: 0.9928, 0.0038, 0.0034)\n","Processed V1.FI01_GL0076253: Neuropeptide (Probabilities: 0.0186, 0.0136, 0.9678)\n","Processed HT14A_GL0010456: Neuropeptide (Probabilities: 0.2286, 0.0239, 0.7475)\n","Processed MH0088_GL0076838: Neuropeptide (Probabilities: 0.1547, 0.0199, 0.8253)\n","Processed MH0002_GL0048893: non-AMP/OTHERS (Probabilities: 0.9922, 0.0030, 0.0048)\n","Processed 764062976-stool2_revised_C1084475_1_gene7504: non-AMP/OTHERS (Probabilities: 0.9918, 0.0036, 0.0046)\n","Processed O2.UC40-1_GL0170069: non-AMP/OTHERS (Probabilities: 0.9918, 0.0039, 0.0043)\n","Processed MH0003_GL0015556: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed NLM003_GL0009172: Neuropeptide (Probabilities: 0.0080, 0.0090, 0.9830)\n","Processed V1.UC33-0_GL0020824: Neuropeptide (Probabilities: 0.0173, 0.0124, 0.9702)\n","Processed 537937.BLIG_01296: Neuropeptide (Probabilities: 0.0059, 0.0087, 0.9854)\n","Processed 159571453-stool2_revised_C1173406_1_gene60413: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed N002A_GL0030844: non-AMP/OTHERS (Probabilities: 0.8898, 0.0153, 0.0949)\n","Processed 159268001-stool1_revised_C1097967_1_gene2706: Neuropeptide (Probabilities: 0.0109, 0.0104, 0.9786)\n","Processed MH0369_GL0103744: non-AMP/OTHERS (Probabilities: 0.5837, 0.3707, 0.0456)\n","Processed DOM019_GL0003205: non-AMP/OTHERS (Probabilities: 0.9784, 0.0135, 0.0081)\n","Processed MH0002_GL0038002: non-AMP/OTHERS (Probabilities: 0.9559, 0.0244, 0.0197)\n","Processed MH0011_GL0030608: non-AMP/OTHERS (Probabilities: 0.9297, 0.0567, 0.0136)\n","Processed 638754422-stool1_revised_scaffold18996_1_gene85170: non-AMP/OTHERS (Probabilities: 0.9032, 0.0509, 0.0459)\n","Processed MH0188_GL0103778: Neuropeptide (Probabilities: 0.0069, 0.0085, 0.9846)\n","Processed V1.CD40-0_GL0174569: non-AMP/OTHERS (Probabilities: 0.9918, 0.0033, 0.0049)\n","Processed O2.CD1-0-PT_GL0028760: non-AMP/OTHERS (Probabilities: 0.9920, 0.0043, 0.0037)\n","Processed MH0092_GL0118497: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0034)\n","Processed DLM027_GL0012356: Neuropeptide (Probabilities: 0.0066, 0.0085, 0.9849)\n","Processed O2.UC27-1_GL0024718: Neuropeptide (Probabilities: 0.3881, 0.0250, 0.5869)\n","Processed 658659.HMPREF0983_02046: non-AMP/OTHERS (Probabilities: 0.9889, 0.0056, 0.0054)\n","Processed 1042403.BALAC2494_01286: Neuropeptide (Probabilities: 0.0826, 0.0187, 0.8986)\n","Processed 763536994-stool2_revised_scaffold24753_1_gene6178: non-AMP/OTHERS (Probabilities: 0.9919, 0.0044, 0.0037)\n","Processed sp|P14410|SUIS_HUMAN: Neuropeptide (Probabilities: 0.0065, 0.0084, 0.9851)\n","Processed 158944319-stool1_revised_scaffold29754_1_gene139329: non-AMP/OTHERS (Probabilities: 0.9927, 0.0039, 0.0033)\n","Processed MH0301_GL0075879: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0035)\n","Processed MH0203_GL0036222: non-AMP/OTHERS (Probabilities: 0.9934, 0.0032, 0.0034)\n","Processed T2D-135A_GL0099393: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0002_GL0028621: Neuropeptide (Probabilities: 0.3031, 0.0675, 0.6294)\n","Processed DOM005_GL0002909: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed DLM013_GL0022410: AMP (Probabilities: 0.0803, 0.9073, 0.0124)\n","Processed MH0203_GL0079869: non-AMP/OTHERS (Probabilities: 0.9890, 0.0044, 0.0066)\n","Processed MH0233_GL0020760: Neuropeptide (Probabilities: 0.0075, 0.0086, 0.9839)\n","Processed V1.UC39-0_GL0097911: non-AMP/OTHERS (Probabilities: 0.9906, 0.0035, 0.0060)\n","Processed NOM001_GL0012793: Neuropeptide (Probabilities: 0.0107, 0.0112, 0.9780)\n","Processed NOM001_GL0012793: non-AMP/OTHERS (Probabilities: 0.7147, 0.2420, 0.0433)\n","Processed V1.UC15-3_GL0025093: non-AMP/OTHERS (Probabilities: 0.9905, 0.0039, 0.0057)\n","Processed MH0003_GL0028371: non-AMP/OTHERS (Probabilities: 0.9783, 0.0077, 0.0140)\n","Processed MH0131_GL0138470: non-AMP/OTHERS (Probabilities: 0.9617, 0.0096, 0.0287)\n","Processed MH0203_GL0247821: non-AMP/OTHERS (Probabilities: 0.9918, 0.0037, 0.0045)\n","Processed DOF004_GL0033364: non-AMP/OTHERS (Probabilities: 0.9909, 0.0053, 0.0038)\n","Processed 765620695-stool1_revised_C756383_1_gene38173: non-AMP/OTHERS (Probabilities: 0.9911, 0.0051, 0.0038)\n","Processed MH0358_GL0129560: Neuropeptide (Probabilities: 0.0064, 0.0085, 0.9851)\n","Processed 657315.ROI_06730: non-AMP/OTHERS (Probabilities: 0.9884, 0.0084, 0.0031)\n","Processed MH0139_GL0020226: non-AMP/OTHERS (Probabilities: 0.9917, 0.0035, 0.0048)\n","Processed T2D-149A_GL0035030: non-AMP/OTHERS (Probabilities: 0.9879, 0.0050, 0.0071)\n","Processed V1.CD19-0_GL0177345: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed MH0198_GL0009056: non-AMP/OTHERS (Probabilities: 0.9536, 0.0147, 0.0317)\n","Processed MH0054_GL0020279: Neuropeptide (Probabilities: 0.2960, 0.0236, 0.6803)\n","Processed O2.UC40-1_GL0183557: Neuropeptide (Probabilities: 0.0069, 0.0088, 0.9843)\n","Processed MH0068_GL0040050: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0257_GL0091184: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed SZEY-30A_GL0017454: non-AMP/OTHERS (Probabilities: 0.9924, 0.0031, 0.0045)\n","Processed MH0229_GL0094180: non-AMP/OTHERS (Probabilities: 0.9927, 0.0032, 0.0040)\n","Processed V1.UC10-2_GL0005801: non-AMP/OTHERS (Probabilities: 0.8463, 0.0220, 0.1318)\n","Processed MH0188_GL0030481: Neuropeptide (Probabilities: 0.0119, 0.0105, 0.9776)\n","Processed MH0073_GL0052733: non-AMP/OTHERS (Probabilities: 0.9920, 0.0033, 0.0047)\n","Processed MH0229_GL0028943: non-AMP/OTHERS (Probabilities: 0.9921, 0.0043, 0.0036)\n","Processed MH0188_GL0055383: non-AMP/OTHERS (Probabilities: 0.9928, 0.0029, 0.0043)\n","Processed V1.UC56-0_GL0043400: Neuropeptide (Probabilities: 0.0085, 0.0098, 0.9817)\n","Processed 763840445-stool2_revised_scaffold30987_1_gene152238: non-AMP/OTHERS (Probabilities: 0.9000, 0.0111, 0.0889)\n","Processed MH0026_GL0009654: non-AMP/OTHERS (Probabilities: 0.8884, 0.0173, 0.0943)\n","Processed O2.UC13-1_GL0115288: non-AMP/OTHERS (Probabilities: 0.9929, 0.0033, 0.0038)\n","Processed NOM027_GL0035999: non-AMP/OTHERS (Probabilities: 0.9930, 0.0030, 0.0040)\n","Processed sp|A0A075B6H9|LV469_HUMAN: non-AMP/OTHERS (Probabilities: 0.5359, 0.0277, 0.4364)\n","Processed tr|A0A5C2GB75|A0A5C2GB75_HUMAN: non-AMP/OTHERS (Probabilities: 0.7603, 0.0178, 0.2218)\n","Processed MH0399_GL0103611: non-AMP/OTHERS (Probabilities: 0.9916, 0.0045, 0.0039)\n","Processed O2.UC31-2_GL0067716: Neuropeptide (Probabilities: 0.3955, 0.0322, 0.5723)\n","Processed N013A_GL0001357: Neuropeptide (Probabilities: 0.0186, 0.0144, 0.9670)\n","Processed T2D-83A_GL0064726: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed MH0002_GL0062277: non-AMP/OTHERS (Probabilities: 0.9924, 0.0030, 0.0047)\n","Processed 264199.stu0692: non-AMP/OTHERS (Probabilities: 0.9335, 0.0094, 0.0572)\n","Processed 160502038-stool1_revised_C1090702_1_gene138294: non-AMP/OTHERS (Probabilities: 0.9854, 0.0057, 0.0088)\n","Processed sp|P00441|SODC_HUMAN: non-AMP/OTHERS (Probabilities: 0.5483, 0.0305, 0.4212)\n","Processed O2.UC12-1_GL0042202: Neuropeptide (Probabilities: 0.0477, 0.0158, 0.9365)\n","Processed T2D-118A_GL0021935: non-AMP/OTHERS (Probabilities: 0.9711, 0.0066, 0.0223)\n","Processed O2.UC22-1_GL0071753: non-AMP/OTHERS (Probabilities: 0.8668, 0.0149, 0.1184)\n","Processed MH0211_GL0137678: non-AMP/OTHERS (Probabilities: 0.9761, 0.0047, 0.0192)\n","Processed V1.FI26_GL0267199: Neuropeptide (Probabilities: 0.0067, 0.0088, 0.9844)\n","Processed MH0433_GL0164193: non-AMP/OTHERS (Probabilities: 0.9916, 0.0037, 0.0048)\n","Processed T2D-33A_GL0031227: non-AMP/OTHERS (Probabilities: 0.9922, 0.0041, 0.0037)\n","Processed T2D-11A_GL0081018: non-AMP/OTHERS (Probabilities: 0.9923, 0.0043, 0.0034)\n","Processed MH0333_GL0121497: Neuropeptide (Probabilities: 0.1926, 0.0217, 0.7858)\n","Processed MH0222_GL0109027: Neuropeptide (Probabilities: 0.1093, 0.0423, 0.8484)\n","Processed MH0002_GL0035895: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed MH0319_GL0148976: non-AMP/OTHERS (Probabilities: 0.9843, 0.0051, 0.0106)\n","Processed MH0341_GL0101890: non-AMP/OTHERS (Probabilities: 0.9864, 0.0059, 0.0078)\n","Processed NOF008_GL0055581: non-AMP/OTHERS (Probabilities: 0.6644, 0.0214, 0.3142)\n","Processed 206672.BL0530: non-AMP/OTHERS (Probabilities: 0.7560, 0.0633, 0.1806)\n","Processed 1042403.BALAC2494_00976: non-AMP/OTHERS (Probabilities: 0.8983, 0.0163, 0.0854)\n","Processed MH0113_GL0097271: non-AMP/OTHERS (Probabilities: 0.9901, 0.0038, 0.0062)\n","Processed MH0101_GL0087795: non-AMP/OTHERS (Probabilities: 0.9919, 0.0039, 0.0042)\n","Processed MH0188_GL0183755: non-AMP/OTHERS (Probabilities: 0.6024, 0.0544, 0.3432)\n","Processed MH0139_GL0116967: Neuropeptide (Probabilities: 0.0230, 0.0144, 0.9627)\n","Processed O2.UC34-2_GL0081781: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0036)\n","Processed 457391.HMPREF0105_1616: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed T2D-114A_GL0092053: Neuropeptide (Probabilities: 0.0288, 0.0154, 0.9558)\n","Processed MH0131_GL0038390: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed O2.UC32-1_GL0105973: non-AMP/OTHERS (Probabilities: 0.9848, 0.0057, 0.0095)\n","Processed MH0062_GL0036388: non-AMP/OTHERS (Probabilities: 0.9740, 0.0061, 0.0199)\n","Processed 159207311-stool1_revised_C1161236_1_gene26364: non-AMP/OTHERS (Probabilities: 0.9892, 0.0057, 0.0051)\n","Processed MH0425_GL0012276: non-AMP/OTHERS (Probabilities: 0.9913, 0.0031, 0.0056)\n","Processed MH0062_GL0032341: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0212_GL0110534: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0039)\n","Processed MH0147_GL0089969: non-AMP/OTHERS (Probabilities: 0.9923, 0.0033, 0.0044)\n","Processed MH0088_GL0062250: non-AMP/OTHERS (Probabilities: 0.9884, 0.0033, 0.0083)\n","Processed V1.FI32_GL0139411: non-AMP/OTHERS (Probabilities: 0.9898, 0.0064, 0.0038)\n","Processed MH0095_GL0046572: non-AMP/OTHERS (Probabilities: 0.8966, 0.0671, 0.0363)\n","Processed MH0178_GL0044140: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed V1.FI29_GL0077807: non-AMP/OTHERS (Probabilities: 0.9040, 0.0145, 0.0814)\n","Processed V1.UC48-0_GL0001632: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed MH0410_GL0011479: Neuropeptide (Probabilities: 0.0145, 0.0135, 0.9720)\n","Processed MH0012_GL0085267: non-AMP/OTHERS (Probabilities: 0.8576, 0.0108, 0.1316)\n","Processed NLM008_GL0007879: Neuropeptide (Probabilities: 0.0107, 0.0100, 0.9793)\n","Processed V1.CD5-0-PN_GL0043791: non-AMP/OTHERS (Probabilities: 0.9855, 0.0033, 0.0112)\n","Processed O2.UC43-1_GL0089746: non-AMP/OTHERS (Probabilities: 0.7280, 0.0325, 0.2395)\n","Processed ED50A_GL0021590: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed MH0288_GL0118018: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0037)\n","Processed MH0006_GL0020175: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0037)\n","Processed V1.FI15_GL0168284: non-AMP/OTHERS (Probabilities: 0.9380, 0.0101, 0.0519)\n","Processed MH0372_GL0100009: Neuropeptide (Probabilities: 0.0110, 0.0105, 0.9785)\n","Processed T2D-6A_GL0075772: non-AMP/OTHERS (Probabilities: 0.9930, 0.0039, 0.0032)\n","Processed MH0012_GL0094896: Neuropeptide (Probabilities: 0.0911, 0.0351, 0.8738)\n","Processed MH0233_GL0075508: non-AMP/OTHERS (Probabilities: 0.8691, 0.0142, 0.1167)\n","Processed O2.CD1-0_GL0006497: non-AMP/OTHERS (Probabilities: 0.9902, 0.0040, 0.0058)\n","Processed MH0131_GL0047641: non-AMP/OTHERS (Probabilities: 0.9923, 0.0047, 0.0030)\n","Processed NLF002_GL0038999: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9852)\n","Processed MH0423_GL0065899: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0257_GL0156757: non-AMP/OTHERS (Probabilities: 0.9920, 0.0039, 0.0041)\n","Processed 764447348-stool2_revised_scaffold32125_1_gene27934: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0034)\n","Processed MH0147_GL0020425: non-AMP/OTHERS (Probabilities: 0.9793, 0.0089, 0.0118)\n","Processed MH0131_GL0023527: non-AMP/OTHERS (Probabilities: 0.9799, 0.0048, 0.0154)\n","Processed MH0089_GL0026058: non-AMP/OTHERS (Probabilities: 0.9703, 0.0044, 0.0252)\n","Processed 160319967-stool1_revised_C572990_1_gene6093: AMP (Probabilities: 0.0858, 0.9006, 0.0136)\n","Processed MH0115_GL0003727: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0277_GL0029048: non-AMP/OTHERS (Probabilities: 0.9702, 0.0089, 0.0209)\n","Processed T2D-65A_GL0081754: Neuropeptide (Probabilities: 0.0267, 0.0155, 0.9577)\n","Processed 160704339-stool1_revised_C1391361_1_gene91800: non-AMP/OTHERS (Probabilities: 0.9307, 0.0308, 0.0385)\n","Processed 1035817.BLNIAS_00999: non-AMP/OTHERS (Probabilities: 0.9826, 0.0037, 0.0137)\n","Processed MH0093_GL0032443: non-AMP/OTHERS (Probabilities: 0.9886, 0.0037, 0.0077)\n","Processed MH0093_GL0032443: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0034)\n","Processed MH0055_GL0009975: Neuropeptide (Probabilities: 0.0087, 0.0094, 0.9819)\n","Processed MH0055_GL0031453: Neuropeptide (Probabilities: 0.0168, 0.0126, 0.9707)\n","Processed MH0056_GL0010804: Neuropeptide (Probabilities: 0.0068, 0.0085, 0.9846)\n","Processed V1.CD3-0-PN_GL0067694: Neuropeptide (Probabilities: 0.0341, 0.0145, 0.9514)\n","Processed MH0335_GL0074583: non-AMP/OTHERS (Probabilities: 0.9915, 0.0032, 0.0053)\n","Processed V1.UC11-5_GL0178943: non-AMP/OTHERS (Probabilities: 0.6698, 0.0176, 0.3125)\n","Processed MH0396_GL0182819: non-AMP/OTHERS (Probabilities: 0.7693, 0.0196, 0.2111)\n","Processed T2D-83A_GL0048226: non-AMP/OTHERS (Probabilities: 0.9843, 0.0049, 0.0108)\n","Processed T2D-83A_GL0064726: non-AMP/OTHERS (Probabilities: 0.9920, 0.0035, 0.0045)\n","Processed MH0443_GL0049450: non-AMP/OTHERS (Probabilities: 0.9919, 0.0052, 0.0029)\n","Processed MH0009_GL0045815: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0044)\n","Processed 321967.LSEI_0970: non-AMP/OTHERS (Probabilities: 0.9895, 0.0053, 0.0052)\n","Processed V1.UC34-0_GL0074693: non-AMP/OTHERS (Probabilities: 0.9903, 0.0037, 0.0061)\n","Processed MH0341_GL0049323: Neuropeptide (Probabilities: 0.0081, 0.0088, 0.9831)\n","Processed O2.UC46-2_GL0146529: non-AMP/OTHERS (Probabilities: 0.9915, 0.0050, 0.0035)\n","Processed T2D-135A_GL0025333: non-AMP/OTHERS (Probabilities: 0.9920, 0.0043, 0.0037)\n","Processed MH0431_GL0064120: non-AMP/OTHERS (Probabilities: 0.9868, 0.0037, 0.0095)\n","Processed V1.UC49-0_GL0091607: non-AMP/OTHERS (Probabilities: 0.9911, 0.0041, 0.0048)\n","Processed MH0020_GL0056756: non-AMP/OTHERS (Probabilities: 0.6237, 0.0169, 0.3593)\n","Processed MH0198_GL0062112: non-AMP/OTHERS (Probabilities: 0.8241, 0.1686, 0.0072)\n","Processed MH0003_GL0046930: non-AMP/OTHERS (Probabilities: 0.9904, 0.0047, 0.0048)\n","Processed MH0002_GL0059686: non-AMP/OTHERS (Probabilities: 0.8346, 0.1304, 0.0350)\n","Processed V1.UC36-0_GL0021896: non-AMP/OTHERS (Probabilities: 0.9929, 0.0032, 0.0039)\n","Processed MH0133_GL0037821: non-AMP/OTHERS (Probabilities: 0.9920, 0.0040, 0.0040)\n","Processed MH0293_GL0038915: Neuropeptide (Probabilities: 0.0206, 0.0122, 0.9672)\n","Processed MH0356_GL0161232: non-AMP/OTHERS (Probabilities: 0.9237, 0.0172, 0.0591)\n","Processed MH0001_GL0013123: non-AMP/OTHERS (Probabilities: 0.8659, 0.0093, 0.1249)\n","Processed 467705.SGO_1593: non-AMP/OTHERS (Probabilities: 0.9914, 0.0051, 0.0035)\n","Processed DOM016_GL0035281: non-AMP/OTHERS (Probabilities: 0.9913, 0.0032, 0.0055)\n","Processed NOM005_GL0067735: non-AMP/OTHERS (Probabilities: 0.9914, 0.0044, 0.0042)\n","Processed V1.UC40-1_GL0006720: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed V1.UC40-1_GL0006720: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed 759350.BLJ_1465: Neuropeptide (Probabilities: 0.0243, 0.0146, 0.9612)\n","Processed 158337416-stool1_revised_scaffold30271_2_gene67334: Neuropeptide (Probabilities: 0.0070, 0.0085, 0.9845)\n","Processed MH0188_GL0178299: non-AMP/OTHERS (Probabilities: 0.9903, 0.0048, 0.0049)\n","Processed MH0002_GL0057195: non-AMP/OTHERS (Probabilities: 0.9921, 0.0032, 0.0048)\n","Processed MH0195_GL0142479: non-AMP/OTHERS (Probabilities: 0.9896, 0.0032, 0.0072)\n","Processed O2.UC11-1_GL0089718: non-AMP/OTHERS (Probabilities: 0.9908, 0.0032, 0.0059)\n","Processed MH0426_GL0151268: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0426_GL0151268: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9850)\n","Processed MH0411_GL0145251: non-AMP/OTHERS (Probabilities: 0.9925, 0.0030, 0.0045)\n","Processed MH0011_GL0097968: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0360_GL0094201: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0002_GL0031638: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed V1.UC47-0_GL0067080: non-AMP/OTHERS (Probabilities: 0.9880, 0.0050, 0.0070)\n","Processed DOM020_GL0030194: non-AMP/OTHERS (Probabilities: 0.8875, 0.0151, 0.0974)\n","Processed DOM017_GL0002143: non-AMP/OTHERS (Probabilities: 0.9563, 0.0134, 0.0303)\n","Processed MH0274_GL0071360: non-AMP/OTHERS (Probabilities: 0.9796, 0.0086, 0.0118)\n","Processed V1.UC31-0_GL0019111: non-AMP/OTHERS (Probabilities: 0.9909, 0.0039, 0.0052)\n","Processed MH0131_GL0047971: non-AMP/OTHERS (Probabilities: 0.9891, 0.0050, 0.0060)\n","Processed MH0002_GL0013099: Neuropeptide (Probabilities: 0.0288, 0.0153, 0.9558)\n","Processed MH0006_GL0013215: Neuropeptide (Probabilities: 0.3660, 0.0229, 0.6111)\n","Processed MH0191_GL0182998: non-AMP/OTHERS (Probabilities: 0.9299, 0.0104, 0.0596)\n","Processed BGI-06A_GL0076090: non-AMP/OTHERS (Probabilities: 0.9777, 0.0051, 0.0172)\n","Processed MH0095_GL0015563: non-AMP/OTHERS (Probabilities: 0.9906, 0.0030, 0.0064)\n","Processed V1.UC40-0_GL0153388: non-AMP/OTHERS (Probabilities: 0.9659, 0.0085, 0.0256)\n","Processed MH0192_GL0099199: non-AMP/OTHERS (Probabilities: 0.9925, 0.0029, 0.0045)\n","Processed MH0345_GL0180051: Neuropeptide (Probabilities: 0.0139, 0.0120, 0.9741)\n","Processed MH0419_GL0179037: non-AMP/OTHERS (Probabilities: 0.6214, 0.0188, 0.3598)\n","Processed MH0126_GL0049795: non-AMP/OTHERS (Probabilities: 0.8076, 0.0295, 0.1629)\n","Processed MH0055_GL0069853: Neuropeptide (Probabilities: 0.0070, 0.0097, 0.9832)\n","Processed MH0131_GL0041769: non-AMP/OTHERS (Probabilities: 0.9738, 0.0067, 0.0194)\n","Processed O2.UC46-2_GL0041426: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0086_GL0050799: Neuropeptide (Probabilities: 0.0097, 0.0097, 0.9807)\n","Processed MH0259_GL0027769: Neuropeptide (Probabilities: 0.0285, 0.0159, 0.9556)\n","Processed MH0188_GL0168164: non-AMP/OTHERS (Probabilities: 0.9902, 0.0050, 0.0049)\n","Processed MH0092_GL0059548: non-AMP/OTHERS (Probabilities: 0.9880, 0.0042, 0.0078)\n","Processed MH0233_GL0060857: non-AMP/OTHERS (Probabilities: 0.9925, 0.0044, 0.0031)\n","Processed O2.CD3-0-PT_GL0183235: non-AMP/OTHERS (Probabilities: 0.9916, 0.0051, 0.0033)\n","Processed 210007.SMU_474: Neuropeptide (Probabilities: 0.0067, 0.0086, 0.9847)\n","Processed DLF012_GL0031284: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0037)\n","Processed MH0356_GL0118049: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0101_GL0086946: non-AMP/OTHERS (Probabilities: 0.9850, 0.0064, 0.0086)\n","Processed NOM009_GL0072011: non-AMP/OTHERS (Probabilities: 0.9908, 0.0040, 0.0052)\n","Processed MH0088_GL0062250: non-AMP/OTHERS (Probabilities: 0.9776, 0.0084, 0.0139)\n","Processed O2.UC23-2_GL0149943: non-AMP/OTHERS (Probabilities: 0.9369, 0.0234, 0.0398)\n","Processed O2.UC48-0_GL0030973: Neuropeptide (Probabilities: 0.0128, 0.0112, 0.9760)\n","Processed MH0188_GL0146567: non-AMP/OTHERS (Probabilities: 0.9931, 0.0031, 0.0038)\n","Processed O2.CD3-0-PT_GL0195180: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed DLM013_GL0019764: non-AMP/OTHERS (Probabilities: 0.9897, 0.0054, 0.0050)\n","Processed MH0192_GL0070156: non-AMP/OTHERS (Probabilities: 0.9918, 0.0038, 0.0044)\n","Processed O2.UC41-1_GL0071713: non-AMP/OTHERS (Probabilities: 0.9844, 0.0061, 0.0095)\n","Processed 159268001-stool2_revised_scaffold34258_3_gene67794: non-AMP/OTHERS (Probabilities: 0.9860, 0.0045, 0.0095)\n","Processed 159227541-stool3_revised_C740006_1_gene110282: non-AMP/OTHERS (Probabilities: 0.9897, 0.0044, 0.0058)\n","Processed MH0257_GL0120737: non-AMP/OTHERS (Probabilities: 0.9840, 0.0045, 0.0115)\n","Processed MH0006_GL0161792: non-AMP/OTHERS (Probabilities: 0.9907, 0.0036, 0.0057)\n","Processed MH0239_GL0088549: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed O2.UC38-2_GL0048055: Neuropeptide (Probabilities: 0.0103, 0.0107, 0.9790)\n","Processed 158499257-stool1_revised_C1350391_1_gene170149: Neuropeptide (Probabilities: 0.0121, 0.0107, 0.9771)\n","Processed MH0062_GL0036388: Neuropeptide (Probabilities: 0.0058, 0.0093, 0.9849)\n","Processed V1.UC48-0_GL0060046: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0038)\n","Processed 158337416-stool2_revised_scaffold12181_1_gene138773: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0191_GL0172709: Neuropeptide (Probabilities: 0.0071, 0.0087, 0.9842)\n","Processed O2.UC17-1_GL0007264: non-AMP/OTHERS (Probabilities: 0.9578, 0.0086, 0.0335)\n","Processed MH0088_GL0092590: non-AMP/OTHERS (Probabilities: 0.9116, 0.0090, 0.0794)\n","Processed V1.FI19_GL0192756: non-AMP/OTHERS (Probabilities: 0.9693, 0.0048, 0.0260)\n","Processed MH0220_GL0225242: Neuropeptide (Probabilities: 0.1773, 0.0367, 0.7859)\n","Processed O2.UC8-0_GL0007096: Neuropeptide (Probabilities: 0.0078, 0.0088, 0.9834)\n","Processed MH0103_GL0021811: non-AMP/OTHERS (Probabilities: 0.9930, 0.0039, 0.0031)\n","Processed O2.UC17-1_GL0130225: non-AMP/OTHERS (Probabilities: 0.9925, 0.0032, 0.0043)\n","Processed MH0354_GL0133068: non-AMP/OTHERS (Probabilities: 0.9422, 0.0090, 0.0488)\n","Processed MH0203_GL0167339: non-AMP/OTHERS (Probabilities: 0.9933, 0.0029, 0.0038)\n","Processed MH0131_GL0090036: non-AMP/OTHERS (Probabilities: 0.9933, 0.0031, 0.0036)\n","Processed 160704339-stool1_revised_C1358053_1_gene127654: non-AMP/OTHERS (Probabilities: 0.9894, 0.0040, 0.0066)\n","Processed V1.FI26_GL0107344: Neuropeptide (Probabilities: 0.3853, 0.0846, 0.5301)\n","Processed MH0058_GL0057298: non-AMP/OTHERS (Probabilities: 0.9920, 0.0050, 0.0030)\n","Processed MH0131_GL0082233: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0037)\n","Processed 622312.ROSEINA2194_00512: non-AMP/OTHERS (Probabilities: 0.9870, 0.0103, 0.0027)\n","Processed O2.UC17-1_GL0027263: non-AMP/OTHERS (Probabilities: 0.9916, 0.0040, 0.0044)\n","Processed MH0005_GL0006043: non-AMP/OTHERS (Probabilities: 0.9917, 0.0039, 0.0045)\n","Processed MH0233_GL0082914: Neuropeptide (Probabilities: 0.1384, 0.0243, 0.8373)\n","Processed MH0052_GL0012113: non-AMP/OTHERS (Probabilities: 0.9911, 0.0035, 0.0054)\n","Processed O2.UC47-1_GL0006392: non-AMP/OTHERS (Probabilities: 0.9776, 0.0039, 0.0185)\n","Processed DOF004_GL0033563: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed O2.UC5-1_GL0039191: non-AMP/OTHERS (Probabilities: 0.9876, 0.0033, 0.0092)\n","Processed MH0382_GL0031961: non-AMP/OTHERS (Probabilities: 0.9239, 0.0075, 0.0685)\n","Processed O2.UC17-1_GL0128382: non-AMP/OTHERS (Probabilities: 0.9707, 0.0053, 0.0240)\n","Processed MH0367_GL0103639: Neuropeptide (Probabilities: 0.4134, 0.0226, 0.5640)\n","Processed 159551223-stool1_revised_C1036541_1_gene109536: non-AMP/OTHERS (Probabilities: 0.9907, 0.0031, 0.0062)\n","Processed 321967.LSEI_1365: non-AMP/OTHERS (Probabilities: 0.9715, 0.0065, 0.0220)\n","Processed V1.CD7-0-PN_GL0024023: non-AMP/OTHERS (Probabilities: 0.9502, 0.0111, 0.0387)\n","Processed MH0188_GL0152900: non-AMP/OTHERS (Probabilities: 0.9900, 0.0031, 0.0069)\n","Processed MH0277_GL0029045: non-AMP/OTHERS (Probabilities: 0.9840, 0.0039, 0.0121)\n","Processed 1042403.BALAC2494_00863: non-AMP/OTHERS (Probabilities: 0.9321, 0.0079, 0.0600)\n","Processed MH0305_GL0107881: non-AMP/OTHERS (Probabilities: 0.9761, 0.0050, 0.0189)\n","Processed MH0412_GL0025033: Neuropeptide (Probabilities: 0.0114, 0.0100, 0.9786)\n","Processed O2.UC28-2_GL0196451: Neuropeptide (Probabilities: 0.0280, 0.0161, 0.9559)\n","Processed O2.UC11-2_GL0118321: Neuropeptide (Probabilities: 0.0230, 0.0137, 0.9633)\n","Processed DLM018_GL0016638: Neuropeptide (Probabilities: 0.0263, 0.0151, 0.9586)\n","Processed MH0110_GL0009571: non-AMP/OTHERS (Probabilities: 0.9864, 0.0083, 0.0053)\n","Processed MH0002_GL0017508: non-AMP/OTHERS (Probabilities: 0.7001, 0.0411, 0.2588)\n","Processed MH0047_GL0012567: non-AMP/OTHERS (Probabilities: 0.9878, 0.0042, 0.0081)\n","Processed MH0003_GL0078036: non-AMP/OTHERS (Probabilities: 0.9914, 0.0039, 0.0047)\n","Processed MH0319_GL0090304: Neuropeptide (Probabilities: 0.0098, 0.0092, 0.9810)\n","Processed NLM022_GL0073680: non-AMP/OTHERS (Probabilities: 0.9770, 0.0056, 0.0174)\n","Processed T2D-135A_GL0099393: non-AMP/OTHERS (Probabilities: 0.7369, 0.0213, 0.2418)\n","Processed T2D-135A_GL0099393: non-AMP/OTHERS (Probabilities: 0.6357, 0.0219, 0.3423)\n","Processed O2.UC49-2_GL0057394: non-AMP/OTHERS (Probabilities: 0.9065, 0.0092, 0.0843)\n","Processed MH0006_GL0154660: non-AMP/OTHERS (Probabilities: 0.9924, 0.0031, 0.0045)\n","Processed ED20A_GL0026873: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DOM003_GL0014762: non-AMP/OTHERS (Probabilities: 0.9562, 0.0202, 0.0237)\n","Processed MH0002_GL0067069: Neuropeptide (Probabilities: 0.0070, 0.0089, 0.9841)\n","Processed MH0184_GL0001534: Neuropeptide (Probabilities: 0.2595, 0.0248, 0.7157)\n","Processed N074A_GL0095852: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0034)\n","Processed SZEY-09A_GL0048131: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0035)\n","Processed MH0425_GL0012276: non-AMP/OTHERS (Probabilities: 0.5197, 0.0342, 0.4462)\n","Processed MH0089_GL0026058: non-AMP/OTHERS (Probabilities: 0.6332, 0.0224, 0.3445)\n","Processed 160765029-stool1_revised_scaffold62_1_gene51067: non-AMP/OTHERS (Probabilities: 0.9866, 0.0036, 0.0099)\n","Processed 160765029-stool1_revised_scaffold62_1_gene51067: non-AMP/OTHERS (Probabilities: 0.9877, 0.0033, 0.0089)\n","Processed MH0302_GL0039116: non-AMP/OTHERS (Probabilities: 0.9916, 0.0031, 0.0053)\n","Processed MH0355_GL0038120: non-AMP/OTHERS (Probabilities: 0.9915, 0.0058, 0.0027)\n","Processed sp|Q9UKN1|MUC12_HUMAN: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed MH0016_GL0068440: non-AMP/OTHERS (Probabilities: 0.9920, 0.0032, 0.0048)\n","Processed MH0057_GL0013967: Neuropeptide (Probabilities: 0.0072, 0.0089, 0.9839)\n","Processed MH0417_GL0230157: non-AMP/OTHERS (Probabilities: 0.9833, 0.0039, 0.0128)\n","Processed O2.UC4-2_GL0101766: non-AMP/OTHERS (Probabilities: 0.9734, 0.0115, 0.0151)\n","Processed 764325968-stool2_revised_scaffold885_1_gene120403: non-AMP/OTHERS (Probabilities: 0.9398, 0.0081, 0.0521)\n","Processed MH0062_GL0012750: Neuropeptide (Probabilities: 0.0079, 0.0095, 0.9826)\n","Processed MH0014_GL0101523: non-AMP/OTHERS (Probabilities: 0.9843, 0.0037, 0.0120)\n","Processed MH0014_GL0101523: non-AMP/OTHERS (Probabilities: 0.9862, 0.0039, 0.0100)\n","Processed 159268001-stool1_revised_scaffold21270_2_gene30296: Neuropeptide (Probabilities: 0.0087, 0.0089, 0.9824)\n","Processed MH0138_GL0039744: non-AMP/OTHERS (Probabilities: 0.9908, 0.0039, 0.0053)\n","Processed DLF002_GL0036840: Neuropeptide (Probabilities: 0.0143, 0.0109, 0.9748)\n","Processed DLF002_GL0036840: non-AMP/OTHERS (Probabilities: 0.8156, 0.0149, 0.1695)\n","Processed MH0006_GL0057815: Neuropeptide (Probabilities: 0.0111, 0.0107, 0.9782)\n","Processed 160178356-stool1_revised_scaffold15747_1_gene106453: Neuropeptide (Probabilities: 0.0211, 0.0136, 0.9653)\n","Processed MH0230_GL0051987: Neuropeptide (Probabilities: 0.4248, 0.0520, 0.5232)\n","Processed 160765029-stool1_revised_scaffold20037_1_gene72972: non-AMP/OTHERS (Probabilities: 0.5105, 0.0243, 0.4653)\n","Processed MH0252_GL0179035: non-AMP/OTHERS (Probabilities: 0.7855, 0.0155, 0.1991)\n","Processed MH0277_GL0039552: non-AMP/OTHERS (Probabilities: 0.9924, 0.0035, 0.0041)\n","Processed MH0359_GL0072193: Neuropeptide (Probabilities: 0.0096, 0.0101, 0.9803)\n","Processed MH0011_GL0095988: non-AMP/OTHERS (Probabilities: 0.9894, 0.0052, 0.0055)\n","Processed 158944319-stool2_revised_scaffold17476_1_gene14895: non-AMP/OTHERS (Probabilities: 0.9924, 0.0038, 0.0038)\n","Processed 764285508-stool1_revised_scaffold42267_1_gene50825: non-AMP/OTHERS (Probabilities: 0.9925, 0.0029, 0.0046)\n","Processed MH0104_GL0113945: Neuropeptide (Probabilities: 0.0645, 0.0241, 0.9115)\n","Processed V1.CD37-4_GL0093429: non-AMP/OTHERS (Probabilities: 0.9928, 0.0040, 0.0033)\n","Processed MH0233_GL0033226: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0033)\n","Processed MH0154_GL0041273: non-AMP/OTHERS (Probabilities: 0.9921, 0.0039, 0.0040)\n","Processed MH0367_GL0177323: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed MH0293_GL0038915: Neuropeptide (Probabilities: 0.0081, 0.0091, 0.9828)\n","Processed sp|P08123|CO1A2_HUMAN: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed MH0066_GL0055296: Neuropeptide (Probabilities: 0.0149, 0.0125, 0.9726)\n","Processed V1.UC13-3_GL0052191: non-AMP/OTHERS (Probabilities: 0.9925, 0.0035, 0.0041)\n","Processed MH0068_GL0016032: non-AMP/OTHERS (Probabilities: 0.9879, 0.0039, 0.0082)\n","Processed MH0030_GL0001686: Neuropeptide (Probabilities: 0.0368, 0.0194, 0.9439)\n","Processed MH0188_GL0019382: Neuropeptide (Probabilities: 0.0085, 0.0085, 0.9830)\n","Processed MH0068_GL0023804: non-AMP/OTHERS (Probabilities: 0.9907, 0.0042, 0.0051)\n","Processed O2.UC59-0_GL0106896: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed T2D-148A_GL0078076: Neuropeptide (Probabilities: 0.4715, 0.0355, 0.4929)\n","Processed MH0131_GL0062123: non-AMP/OTHERS (Probabilities: 0.9880, 0.0038, 0.0083)\n","Processed 160502038-stool1_revised_scaffold15990_1_gene23414: non-AMP/OTHERS (Probabilities: 0.9904, 0.0065, 0.0031)\n","Processed MH0360_GL0079565: non-AMP/OTHERS (Probabilities: 0.9911, 0.0054, 0.0036)\n","Processed V1.UC13-3_GL0052191: Neuropeptide (Probabilities: 0.0066, 0.0085, 0.9849)\n","Processed 445972.ANACOL_01426: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9849)\n","Processed V1.CD41-0_GL0027347: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed DLM022_GL0002083: non-AMP/OTHERS (Probabilities: 0.9867, 0.0064, 0.0069)\n","Processed DLF002_GL0048704: non-AMP/OTHERS (Probabilities: 0.9274, 0.0626, 0.0100)\n","Processed MH0113_GL0039803: non-AMP/OTHERS (Probabilities: 0.9422, 0.0092, 0.0486)\n","Processed O2.UC8-1_GL0096881: non-AMP/OTHERS (Probabilities: 0.9923, 0.0034, 0.0043)\n","Processed MH0230_GL0161037: non-AMP/OTHERS (Probabilities: 0.9920, 0.0042, 0.0039)\n","Processed MH0086_GL0075813: non-AMP/OTHERS (Probabilities: 0.9920, 0.0041, 0.0038)\n","Processed O2.UC5-1_GL0123280: non-AMP/OTHERS (Probabilities: 0.9926, 0.0035, 0.0039)\n","Processed MH0124_GL0033309: non-AMP/OTHERS (Probabilities: 0.9758, 0.0064, 0.0178)\n","Processed sp|P04083|ANXA1_HUMAN: Neuropeptide (Probabilities: 0.0072, 0.0090, 0.9837)\n","Processed V1.UC25-1_GL0162008: non-AMP/OTHERS (Probabilities: 0.9901, 0.0029, 0.0069)\n","Processed MH0131_GL0123461: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0036)\n","Processed 158337416-stool2_revised_scaffold16676_1_gene161551: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed 483216.BACEGG_00870: non-AMP/OTHERS (Probabilities: 0.9881, 0.0064, 0.0054)\n","Processed V1.CD6-0-PN_GL0131477: Neuropeptide (Probabilities: 0.0202, 0.0130, 0.9668)\n","Processed MH0199_GL0001383: Neuropeptide (Probabilities: 0.1991, 0.0227, 0.7782)\n","Processed MH0073_GL0003406: non-AMP/OTHERS (Probabilities: 0.9900, 0.0039, 0.0061)\n","Processed V1.CD40-0_GL0126218: non-AMP/OTHERS (Probabilities: 0.9926, 0.0036, 0.0038)\n","Processed O2.UC54-2_GL0000495: Neuropeptide (Probabilities: 0.0090, 0.0094, 0.9816)\n","Processed MH0011_GL0095189: non-AMP/OTHERS (Probabilities: 0.9925, 0.0037, 0.0037)\n","Processed DLM012_GL0010473: non-AMP/OTHERS (Probabilities: 0.8420, 0.0111, 0.1469)\n","Processed MH0003_GL0012840: Neuropeptide (Probabilities: 0.0068, 0.0086, 0.9846)\n","Processed O2.UC22-2_GL0028283: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed O2.UC22-2_GL0028283: Neuropeptide (Probabilities: 0.1798, 0.0332, 0.7870)\n","Processed V1.CD17-4_GL0091217: Neuropeptide (Probabilities: 0.0068, 0.0085, 0.9847)\n","Processed V1.FI34_GL0034716: non-AMP/OTHERS (Probabilities: 0.9924, 0.0038, 0.0039)\n","Processed MH0440_GL0005194: non-AMP/OTHERS (Probabilities: 0.9921, 0.0032, 0.0046)\n","Processed MH0188_GL0054470: non-AMP/OTHERS (Probabilities: 0.9893, 0.0038, 0.0069)\n","Processed MH0127_GL0008590: non-AMP/OTHERS (Probabilities: 0.9893, 0.0048, 0.0059)\n","Processed V1.FI02_GL0135168: non-AMP/OTHERS (Probabilities: 0.9922, 0.0034, 0.0044)\n","Processed V1.FI02_GL0135168: Neuropeptide (Probabilities: 0.1508, 0.0206, 0.8286)\n","Processed MH0012_GL0139599: non-AMP/OTHERS (Probabilities: 0.9649, 0.0061, 0.0289)\n","Processed MH0002_GL0050528: non-AMP/OTHERS (Probabilities: 0.9890, 0.0035, 0.0075)\n","Processed ED20A_GL0078797: non-AMP/OTHERS (Probabilities: 0.9857, 0.0040, 0.0103)\n","Processed sp|P36952|SPB5_HUMAN: non-AMP/OTHERS (Probabilities: 0.9492, 0.0070, 0.0438)\n","Processed MH0003_GL0049708: non-AMP/OTHERS (Probabilities: 0.9894, 0.0035, 0.0071)\n","Processed MH0012_GL0070213: non-AMP/OTHERS (Probabilities: 0.9895, 0.0037, 0.0068)\n","Processed MH0062_GL0070627: non-AMP/OTHERS (Probabilities: 0.9391, 0.0077, 0.0532)\n","Processed O2.UC23-2_GL0066537: non-AMP/OTHERS (Probabilities: 0.9788, 0.0060, 0.0152)\n","Processed O2.UC8-1_GL0049804: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed tr|Q6MZW0|Q6MZW0_HUMAN: non-AMP/OTHERS (Probabilities: 0.9700, 0.0086, 0.0213)\n","Processed MH0196_GL0219356: non-AMP/OTHERS (Probabilities: 0.7964, 0.0137, 0.1899)\n","Processed MH0377_GL0001124: non-AMP/OTHERS (Probabilities: 0.9924, 0.0030, 0.0046)\n","Processed V1.UC38-0_GL0111951: non-AMP/OTHERS (Probabilities: 0.9911, 0.0038, 0.0051)\n","Processed MH0076_GL0022599: Neuropeptide (Probabilities: 0.0186, 0.0132, 0.9683)\n","Processed O2.UC40-2_GL0164737: Neuropeptide (Probabilities: 0.0124, 0.0107, 0.9769)\n","Processed O2.UC40-2_GL0164737: Neuropeptide (Probabilities: 0.0521, 0.0217, 0.9262)\n","Processed 159753524-stool2_revised_scaffold18742_1_gene47581: non-AMP/OTHERS (Probabilities: 0.9926, 0.0042, 0.0032)\n","Processed MH0325_GL0059915: Neuropeptide (Probabilities: 0.1193, 0.0269, 0.8537)\n","Processed V1.CD41-0_GL0122590: non-AMP/OTHERS (Probabilities: 0.7416, 0.0224, 0.2360)\n","Processed V1.FI12_GL0198636: non-AMP/OTHERS (Probabilities: 0.9911, 0.0052, 0.0037)\n","Processed MH0131_GL0030085: non-AMP/OTHERS (Probabilities: 0.9918, 0.0034, 0.0048)\n","Processed NLM023_GL0025489: Neuropeptide (Probabilities: 0.0076, 0.0085, 0.9839)\n","Processed V1.UC5-3_GL0261262: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9850)\n","Processed MH0345_GL0127590: non-AMP/OTHERS (Probabilities: 0.9919, 0.0030, 0.0051)\n","Processed V1.CD17-4_GL0091217: non-AMP/OTHERS (Probabilities: 0.9919, 0.0043, 0.0038)\n","Processed MH0086_GL0040596: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0037)\n","Processed O2.CD3-0-PT_GL0033875: Neuropeptide (Probabilities: 0.4436, 0.0680, 0.4884)\n","Processed O2.UC40-2_GL0164737: non-AMP/OTHERS (Probabilities: 0.9314, 0.0099, 0.0587)\n","Processed V1.UC40-1_GL0017984: Neuropeptide (Probabilities: 0.0229, 0.0158, 0.9613)\n","Processed DOM026_GL0035760: Neuropeptide (Probabilities: 0.1206, 0.0281, 0.8513)\n","Processed 159268001-stool1_revised_C1027725_1_gene13407: non-AMP/OTHERS (Probabilities: 0.9906, 0.0041, 0.0052)\n","Processed 1042403.BALAC2494_00903: non-AMP/OTHERS (Probabilities: 0.5501, 0.0171, 0.4328)\n","Processed V1.FI05_GL0069756: Neuropeptide (Probabilities: 0.0061, 0.0089, 0.9850)\n","Processed 367928.BAD_0094: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0143_GL0022673: non-AMP/OTHERS (Probabilities: 0.9857, 0.0050, 0.0093)\n","Processed O2.UC47-1_GL0034730: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0370_GL0053023: non-AMP/OTHERS (Probabilities: 0.9919, 0.0047, 0.0034)\n","Processed T2D-12A_GL0063129: non-AMP/OTHERS (Probabilities: 0.9920, 0.0042, 0.0038)\n","Processed MH0086_GL0023034: non-AMP/OTHERS (Probabilities: 0.8354, 0.0568, 0.1078)\n","Processed MH0007_GL0003396: non-AMP/OTHERS (Probabilities: 0.9925, 0.0030, 0.0045)\n","Processed MH0131_GL0110908: non-AMP/OTHERS (Probabilities: 0.9827, 0.0087, 0.0086)\n","Processed 411479.BACUNI_00259: non-AMP/OTHERS (Probabilities: 0.9901, 0.0037, 0.0062)\n","Processed T2D-33A_GL0082962: non-AMP/OTHERS (Probabilities: 0.9912, 0.0036, 0.0052)\n","Processed MH0145_GL0086827: non-AMP/OTHERS (Probabilities: 0.9922, 0.0030, 0.0048)\n","Processed MH0277_GL0045938: non-AMP/OTHERS (Probabilities: 0.9862, 0.0046, 0.0092)\n","Processed 158802708-stool1_revised_scaffold38525_1_gene8307: non-AMP/OTHERS (Probabilities: 0.9901, 0.0044, 0.0056)\n","Processed O2.UC11-2_GL0093066: non-AMP/OTHERS (Probabilities: 0.9898, 0.0038, 0.0064)\n","Processed MH0277_GL0055058: non-AMP/OTHERS (Probabilities: 0.9840, 0.0044, 0.0116)\n","Processed MH0188_GL0022139: Neuropeptide (Probabilities: 0.0175, 0.0111, 0.9714)\n","Processed MH0003_GL0111821: non-AMP/OTHERS (Probabilities: 0.9919, 0.0030, 0.0051)\n","Processed sp|P43304|GPDM_HUMAN: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0040)\n","Processed MH0277_GL0002658: non-AMP/OTHERS (Probabilities: 0.9748, 0.0091, 0.0161)\n","Processed MH0211_GL0033936: non-AMP/OTHERS (Probabilities: 0.9920, 0.0032, 0.0048)\n","Processed O2.CD2-0-PT_GL0067932: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0038)\n","Processed MH0392_GL0162415: non-AMP/OTHERS (Probabilities: 0.9933, 0.0031, 0.0036)\n","Processed MH0088_GL0017025: non-AMP/OTHERS (Probabilities: 0.9861, 0.0054, 0.0085)\n","Processed 159551223-stool1_revised_scaffold38999_1_gene80098: Neuropeptide (Probabilities: 0.0321, 0.0210, 0.9469)\n","Processed 765094712-stool1_revised_scaffold5072_2_gene22305: Neuropeptide (Probabilities: 0.1102, 0.0364, 0.8535)\n","Processed V1.CD52-0_GL0140662: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0041)\n","Processed 159268001-stool2_revised_C679271_1_gene123369: non-AMP/OTHERS (Probabilities: 0.8198, 0.0267, 0.1535)\n","Processed MH0113_GL0047584: non-AMP/OTHERS (Probabilities: 0.9899, 0.0036, 0.0065)\n","Processed MH0001_GL0013053: non-AMP/OTHERS (Probabilities: 0.9913, 0.0038, 0.0049)\n","Processed V1.UC58-4_GL0082366: Neuropeptide (Probabilities: 0.0144, 0.0124, 0.9732)\n","Processed MH0006_GL0169048: Neuropeptide (Probabilities: 0.0317, 0.0178, 0.9505)\n","Processed O2.UC20-2_GL0033084: non-AMP/OTHERS (Probabilities: 0.9775, 0.0054, 0.0171)\n","Processed MH0203_GL0167339: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed O2.UC9-1_GL0016380: non-AMP/OTHERS (Probabilities: 0.9918, 0.0034, 0.0048)\n","Processed 195103.CPF_1068: Neuropeptide (Probabilities: 0.1051, 0.0231, 0.8718)\n","Processed 195103.CPF_1068: Neuropeptide (Probabilities: 0.0256, 0.0155, 0.9589)\n","Processed MH0455_GL0141278: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0040)\n","Processed O2.UC59-0_GL0106896: non-AMP/OTHERS (Probabilities: 0.9895, 0.0080, 0.0026)\n","Processed sp|O00159|MYO1C_HUMAN: non-AMP/OTHERS (Probabilities: 0.9865, 0.0051, 0.0084)\n","Processed MH0376_GL0087616: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0039)\n","Processed MH0428_GL0101880: non-AMP/OTHERS (Probabilities: 0.5673, 0.0214, 0.4112)\n","Processed SZEY-20A_GL0021595: non-AMP/OTHERS (Probabilities: 0.9907, 0.0052, 0.0041)\n","Processed MH0359_GL0124487: non-AMP/OTHERS (Probabilities: 0.9703, 0.0088, 0.0209)\n","Processed MH0003_GL0042548: non-AMP/OTHERS (Probabilities: 0.9917, 0.0044, 0.0039)\n","Processed MH0128_GL0041269: non-AMP/OTHERS (Probabilities: 0.9921, 0.0037, 0.0042)\n","Processed MH0012_GL0107054: non-AMP/OTHERS (Probabilities: 0.9842, 0.0098, 0.0060)\n","Processed MH0442_GL0234374: Neuropeptide (Probabilities: 0.0749, 0.0212, 0.9040)\n","Processed MH0014_GL0034995: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed MH0014_GL0034995: non-AMP/OTHERS (Probabilities: 0.9814, 0.0056, 0.0130)\n","Processed MH0020_GL0063748: Neuropeptide (Probabilities: 0.0475, 0.0232, 0.9293)\n","Processed MH0012_GL0233286: non-AMP/OTHERS (Probabilities: 0.9891, 0.0060, 0.0049)\n","Processed 1042403.BALAC2494_00156: non-AMP/OTHERS (Probabilities: 0.9901, 0.0034, 0.0066)\n","Processed 159207311-stool2_revised_scaffold27937_2_gene147442: non-AMP/OTHERS (Probabilities: 0.9917, 0.0043, 0.0039)\n","Processed MH0395_GL0006299: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0012_GL0029177: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed 159814214-stool1_revised_scaffold15032_1_gene11449: non-AMP/OTHERS (Probabilities: 0.9827, 0.0089, 0.0084)\n","Processed T2D-178A_GL0083524: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0037)\n","Processed N075A_GL0027846: non-AMP/OTHERS (Probabilities: 0.9893, 0.0034, 0.0074)\n","Processed MH0211_GL0088041: non-AMP/OTHERS (Probabilities: 0.9872, 0.0042, 0.0086)\n","Processed O2.UC17-2_GL0046248: non-AMP/OTHERS (Probabilities: 0.9672, 0.0061, 0.0267)\n","Processed MH0203_GL0263695: Neuropeptide (Probabilities: 0.0732, 0.0190, 0.9078)\n","Processed T2D-178A_GL0066484: Neuropeptide (Probabilities: 0.1618, 0.0417, 0.7964)\n","Processed HT25A_GL0137351: non-AMP/OTHERS (Probabilities: 0.9876, 0.0050, 0.0074)\n","Processed O2.UC37-2_GL0002887: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0003_GL0110880: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed V1.FI18_GL0093238: Neuropeptide (Probabilities: 0.1968, 0.0224, 0.7808)\n","Processed MH0113_GL0054054: Neuropeptide (Probabilities: 0.2134, 0.0211, 0.7655)\n","Processed MH0271_GL0147225: non-AMP/OTHERS (Probabilities: 0.9932, 0.0038, 0.0030)\n","Processed MH0441_GL0002880: non-AMP/OTHERS (Probabilities: 0.9921, 0.0036, 0.0043)\n","Processed MH0462_GL0016042: Neuropeptide (Probabilities: 0.0113, 0.0110, 0.9777)\n","Processed MH0315_GL0081464: Neuropeptide (Probabilities: 0.0076, 0.0092, 0.9833)\n","Processed 759350.BLJ_0947: Neuropeptide (Probabilities: 0.0120, 0.0117, 0.9762)\n","Processed O2.UC22-1_GL0136922: non-AMP/OTHERS (Probabilities: 0.9924, 0.0043, 0.0033)\n","Processed MH0230_GL0136230: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed 158944319-stool1_revised_scaffold20541_1_gene57924: non-AMP/OTHERS (Probabilities: 0.9816, 0.0066, 0.0118)\n","Processed MH0014_GL0019222: non-AMP/OTHERS (Probabilities: 0.8248, 0.0268, 0.1484)\n","Processed MH0370_GL0046631: non-AMP/OTHERS (Probabilities: 0.9881, 0.0035, 0.0084)\n","Processed MH0127_GL0067595: non-AMP/OTHERS (Probabilities: 0.8353, 0.0170, 0.1478)\n","Processed MH0327_GL0006401: non-AMP/OTHERS (Probabilities: 0.7709, 0.0223, 0.2068)\n","Processed O2.UC48-0_GL0120234: non-AMP/OTHERS (Probabilities: 0.8116, 0.0194, 0.1691)\n","Processed T2D-45A_GL0146633: Neuropeptide (Probabilities: 0.0066, 0.0086, 0.9848)\n","Processed MH0442_GL0231073: Neuropeptide (Probabilities: 0.2668, 0.0650, 0.6682)\n","Processed MH0002_GL0012963: Neuropeptide (Probabilities: 0.4134, 0.0589, 0.5277)\n","Processed 763435843-stool1_revised_scaffold13726_1_gene42064: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0364_GL0080481: non-AMP/OTHERS (Probabilities: 0.9545, 0.0153, 0.0302)\n","Processed 763496533-stool1_revised_scaffold12966_1_gene128340: Neuropeptide (Probabilities: 0.0063, 0.0086, 0.9851)\n","Processed T2D-175A_GL0052020: non-AMP/OTHERS (Probabilities: 0.9854, 0.0076, 0.0069)\n","Processed MH0012_GL0003700: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed 159268001-stool1_revised_scaffold207_1_gene117000: Neuropeptide (Probabilities: 0.0117, 0.0104, 0.9779)\n","Processed T2D-62A_GL0025792: Neuropeptide (Probabilities: 0.0087, 0.0096, 0.9817)\n","Processed sp|P02788|TRFL_HUMAN: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed 518635.BIFANG_03439: Neuropeptide (Probabilities: 0.0215, 0.0130, 0.9655)\n","Processed O2.UC50-0_GL0054803: Neuropeptide (Probabilities: 0.0209, 0.0151, 0.9640)\n","Processed MH0326_GL0039571: non-AMP/OTHERS (Probabilities: 0.9153, 0.0738, 0.0109)\n","Processed 548480.HMPREF0175_0864: Neuropeptide (Probabilities: 0.0111, 0.0099, 0.9791)\n","Processed BGI-33A_GL0064365: Neuropeptide (Probabilities: 0.0157, 0.0111, 0.9732)\n","Processed HT14A_GL0019427: non-AMP/OTHERS (Probabilities: 0.9862, 0.0052, 0.0086)\n","Processed DOM012_GL0051744: non-AMP/OTHERS (Probabilities: 0.7568, 0.0197, 0.2235)\n","Processed MH0041_GL0056023: non-AMP/OTHERS (Probabilities: 0.9926, 0.0030, 0.0044)\n","Processed O2.UC11-2_GL0041450: non-AMP/OTHERS (Probabilities: 0.9887, 0.0036, 0.0077)\n","Processed MH0138_GL0032179: non-AMP/OTHERS (Probabilities: 0.9745, 0.0054, 0.0202)\n","Processed O2.UC47-1_GL0127210: non-AMP/OTHERS (Probabilities: 0.9688, 0.0040, 0.0273)\n","Processed O2.UC47-1_GL0127210: non-AMP/OTHERS (Probabilities: 0.9923, 0.0033, 0.0044)\n","Processed MH0008_GL0020565: Neuropeptide (Probabilities: 0.0078, 0.0090, 0.9832)\n","Processed MH0019_GL0043917: Neuropeptide (Probabilities: 0.0235, 0.0171, 0.9594)\n","Processed 765640925-stool1_revised_scaffold12412_4_gene944: Neuropeptide (Probabilities: 0.0113, 0.0112, 0.9775)\n","Processed MH0429_GL0121865: non-AMP/OTHERS (Probabilities: 0.9900, 0.0061, 0.0039)\n","Processed O2.UC46-0_GL0120745: non-AMP/OTHERS (Probabilities: 0.9931, 0.0033, 0.0036)\n","Processed MH0402_GL0131889: non-AMP/OTHERS (Probabilities: 0.9894, 0.0077, 0.0029)\n","Processed MH0260_GL0139736: Neuropeptide (Probabilities: 0.0110, 0.0107, 0.9783)\n","Processed MH0239_GL0172619: Neuropeptide (Probabilities: 0.4585, 0.0281, 0.5133)\n","Processed MH0053_GL0076410: non-AMP/OTHERS (Probabilities: 0.9901, 0.0050, 0.0048)\n","Processed O2.UC31-2_GL0132443: non-AMP/OTHERS (Probabilities: 0.9859, 0.0045, 0.0096)\n","Processed T2D-192A_GL0010669: Neuropeptide (Probabilities: 0.0096, 0.0098, 0.9806)\n","Processed N027A_GL0001776: non-AMP/OTHERS (Probabilities: 0.9926, 0.0039, 0.0035)\n","Processed DOF014_GL0052797: non-AMP/OTHERS (Probabilities: 0.8035, 0.0340, 0.1625)\n","Processed DOF014_GL0052797: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed O2.UC22-1_GL0052774: Neuropeptide (Probabilities: 0.0107, 0.0098, 0.9795)\n","Processed SZEY-10A_GL0011015: non-AMP/OTHERS (Probabilities: 0.9917, 0.0027, 0.0056)\n","Processed V1.CD15-3_GL0050352: non-AMP/OTHERS (Probabilities: 0.8581, 0.0126, 0.1293)\n","Processed O2.UC58-0_GL0121696: non-AMP/OTHERS (Probabilities: 0.9920, 0.0034, 0.0046)\n","Processed MH0003_GL0104512: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0033_GL0055185: non-AMP/OTHERS (Probabilities: 0.9926, 0.0034, 0.0040)\n","Processed MH0327_GL0092311: non-AMP/OTHERS (Probabilities: 0.9592, 0.0175, 0.0233)\n","Processed MH0080_GL0000626: non-AMP/OTHERS (Probabilities: 0.5594, 0.0290, 0.4116)\n","Processed MH0188_GL0067094: non-AMP/OTHERS (Probabilities: 0.9893, 0.0052, 0.0055)\n","Processed T2D-25A_GL0081229: non-AMP/OTHERS (Probabilities: 0.9861, 0.0048, 0.0091)\n","Processed 469591.HMPREF9008_01484: Neuropeptide (Probabilities: 0.0118, 0.0105, 0.9777)\n","Processed MH0422_GL0220989: non-AMP/OTHERS (Probabilities: 0.9895, 0.0033, 0.0073)\n","Processed MH0389_GL0069308: non-AMP/OTHERS (Probabilities: 0.9931, 0.0030, 0.0040)\n","Processed O2.UC41-1_GL0017471: Neuropeptide (Probabilities: 0.0119, 0.0109, 0.9772)\n","Processed V1.UC13-3_GL0014892: non-AMP/OTHERS (Probabilities: 0.9923, 0.0046, 0.0031)\n","Processed MH0011_GL0087451: Neuropeptide (Probabilities: 0.0066, 0.0087, 0.9847)\n","Processed O2.UC28-1_GL0185618: Neuropeptide (Probabilities: 0.0090, 0.0096, 0.9813)\n","Processed SZEY-09A_GL0048131: Neuropeptide (Probabilities: 0.1391, 0.0236, 0.8373)\n","Processed MH0161_GL0074749: non-AMP/OTHERS (Probabilities: 0.9807, 0.0089, 0.0104)\n","Processed V1.UC40-0_GL0130575: non-AMP/OTHERS (Probabilities: 0.9872, 0.0038, 0.0089)\n","Processed MH0093_GL0052241: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed O2.UC48-0_GL0260591: Neuropeptide (Probabilities: 0.0057, 0.0091, 0.9851)\n","Processed MH0033_GL0055185: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0220_GL0175640: non-AMP/OTHERS (Probabilities: 0.9794, 0.0067, 0.0139)\n","Processed V1.CD12-3_GL0108555: Neuropeptide (Probabilities: 0.0097, 0.0098, 0.9805)\n","Processed O2.UC5-1_GL0070103: Neuropeptide (Probabilities: 0.0072, 0.0086, 0.9842)\n","Processed V1.CD31-0_GL0134962: non-AMP/OTHERS (Probabilities: 0.9571, 0.0067, 0.0363)\n","Processed MH0244_GL0021081: non-AMP/OTHERS (Probabilities: 0.9577, 0.0244, 0.0178)\n","Processed 908612.HMPREF9720_2076: Neuropeptide (Probabilities: 0.0087, 0.0091, 0.9822)\n","Processed 908612.HMPREF9720_2076: Neuropeptide (Probabilities: 0.0076, 0.0088, 0.9835)\n","Processed MH0251_GL0173209: Neuropeptide (Probabilities: 0.0125, 0.0108, 0.9767)\n","Processed MH0293_GL0038915: non-AMP/OTHERS (Probabilities: 0.9922, 0.0042, 0.0036)\n","Processed O2.UC21-2_GL0103309: non-AMP/OTHERS (Probabilities: 0.9921, 0.0038, 0.0041)\n","Processed V1.UC46-0_GL0057208: Neuropeptide (Probabilities: 0.0176, 0.0117, 0.9707)\n","Processed 547043.BIFPSEUDO_02937: Neuropeptide (Probabilities: 0.1242, 0.0270, 0.8489)\n","Processed MH0002_GL0025348: Neuropeptide (Probabilities: 0.0137, 0.0122, 0.9741)\n","Processed MH0161_GL0108235: Neuropeptide (Probabilities: 0.0601, 0.0197, 0.9203)\n","Processed MH0089_GL0026062: non-AMP/OTHERS (Probabilities: 0.9922, 0.0030, 0.0048)\n","Processed T2D-14A_GL0021714: non-AMP/OTHERS (Probabilities: 0.9862, 0.0034, 0.0103)\n","Processed T2D-14A_GL0021714: Neuropeptide (Probabilities: 0.0224, 0.0129, 0.9647)\n","Processed V1.FI13_GL0093264: non-AMP/OTHERS (Probabilities: 0.7798, 0.0238, 0.1963)\n","Processed MH0119_GL0022168: non-AMP/OTHERS (Probabilities: 0.9924, 0.0041, 0.0035)\n","Processed MH0277_GL0044725: Neuropeptide (Probabilities: 0.0068, 0.0087, 0.9846)\n","Processed 391904.BLIJ_0194: Neuropeptide (Probabilities: 0.0073, 0.0085, 0.9842)\n","Processed MH0124_GL0081077: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed 764669880-stool1_revised_scaffold2519_5_gene19842: non-AMP/OTHERS (Probabilities: 0.9883, 0.0052, 0.0065)\n","Processed MH0212_GL0057268: Neuropeptide (Probabilities: 0.3562, 0.0230, 0.6208)\n","Processed MH0309_GL0088956: Neuropeptide (Probabilities: 0.0101, 0.0106, 0.9794)\n","Processed MH0439_GL0156341: Neuropeptide (Probabilities: 0.0078, 0.0086, 0.9836)\n","Processed MH0039_GL0047521: non-AMP/OTHERS (Probabilities: 0.6241, 0.0585, 0.3174)\n","Processed 763860675-stool1_revised_scaffold48034_4_gene110306: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9850)\n","Processed 748224.HMPREF9436_02298: Neuropeptide (Probabilities: 0.0070, 0.0085, 0.9845)\n","Processed O2.UC28-1_GL0110191: Neuropeptide (Probabilities: 0.4027, 0.0251, 0.5722)\n","Processed O2.UC17-1_GL0099902: non-AMP/OTHERS (Probabilities: 0.9921, 0.0031, 0.0047)\n","Processed O2.UC17-1_GL0099902: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed MH0002_GL0018942: Neuropeptide (Probabilities: 0.0233, 0.0135, 0.9633)\n","Processed MH0420_GL0069287: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed MH0277_GL0054420: non-AMP/OTHERS (Probabilities: 0.9908, 0.0034, 0.0058)\n","Processed T2D-19A_GL0077960: non-AMP/OTHERS (Probabilities: 0.9921, 0.0032, 0.0047)\n","Processed MH0037_GL0049232: non-AMP/OTHERS (Probabilities: 0.9879, 0.0059, 0.0062)\n","Processed MH0293_GL0034149: Neuropeptide (Probabilities: 0.0432, 0.0201, 0.9367)\n","Processed 763034.HMPREF9446_03954: non-AMP/OTHERS (Probabilities: 0.9902, 0.0035, 0.0063)\n","Processed MH0011_GL0097388: non-AMP/OTHERS (Probabilities: 0.9213, 0.0105, 0.0682)\n","Processed 547043.BIFPSEUDO_04362: non-AMP/OTHERS (Probabilities: 0.9775, 0.0050, 0.0175)\n","Processed MH0402_GL0178708: non-AMP/OTHERS (Probabilities: 0.9605, 0.0195, 0.0200)\n","Processed MH0257_GL0061015: Neuropeptide (Probabilities: 0.0309, 0.0195, 0.9496)\n","Processed MH0277_GL0054719: Neuropeptide (Probabilities: 0.0380, 0.0204, 0.9416)\n","Processed MH0277_GL0021675: non-AMP/OTHERS (Probabilities: 0.9841, 0.0061, 0.0098)\n","Processed MH0440_GL0176679: Neuropeptide (Probabilities: 0.4843, 0.0237, 0.4920)\n","Processed MH0277_GL0030402: Neuropeptide (Probabilities: 0.0084, 0.0090, 0.9825)\n","Processed MH0116_GL0050315: Neuropeptide (Probabilities: 0.0079, 0.0089, 0.9832)\n","Processed NOM014_GL0023889: non-AMP/OTHERS (Probabilities: 0.9839, 0.0066, 0.0096)\n","Processed MH0086_GL0064567: non-AMP/OTHERS (Probabilities: 0.5992, 0.0392, 0.3616)\n","Processed T2D-135A_GL0000682: Neuropeptide (Probabilities: 0.0871, 0.0264, 0.8865)\n","Processed MH0127_GL0037698: Neuropeptide (Probabilities: 0.3323, 0.0214, 0.6463)\n","Processed 159268001-stool2_revised_scaffold31272_1_gene122605: Neuropeptide (Probabilities: 0.0070, 0.0090, 0.9840)\n","Processed MH0086_GL0064754: non-AMP/OTHERS (Probabilities: 0.9797, 0.0049, 0.0154)\n","Processed NLM027_GL0030600: non-AMP/OTHERS (Probabilities: 0.9901, 0.0032, 0.0067)\n","Processed MH0443_GL0183189: Neuropeptide (Probabilities: 0.0064, 0.0087, 0.9849)\n","Processed MH0233_GL0060576: non-AMP/OTHERS (Probabilities: 0.9788, 0.0048, 0.0163)\n","Processed MH0193_GL0112161: non-AMP/OTHERS (Probabilities: 0.9912, 0.0035, 0.0053)\n","Processed O2.UC37-0_GL0050503: non-AMP/OTHERS (Probabilities: 0.5005, 0.0283, 0.4712)\n","Processed V1.CD17-4_GL0084563: non-AMP/OTHERS (Probabilities: 0.9461, 0.0086, 0.0453)\n","Processed MH0454_GL0013007: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed V1.CD8-0-PN_GL0128754: non-AMP/OTHERS (Probabilities: 0.7014, 0.0432, 0.2554)\n","Processed DOM017_GL0050578: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0039)\n","Processed 657323.CK1_21680: non-AMP/OTHERS (Probabilities: 0.7278, 0.0146, 0.2576)\n","Processed MH0172_GL0038751: non-AMP/OTHERS (Probabilities: 0.9702, 0.0047, 0.0251)\n","Processed MH0108_GL0106886: non-AMP/OTHERS (Probabilities: 0.9928, 0.0032, 0.0040)\n","Processed 159510762-stool1_revised_C328748_1_gene17106: non-AMP/OTHERS (Probabilities: 0.9759, 0.0054, 0.0187)\n","Processed MH0003_GL0005021: Neuropeptide (Probabilities: 0.0334, 0.0147, 0.9519)\n","Processed MH0259_GL0123532: non-AMP/OTHERS (Probabilities: 0.9610, 0.0057, 0.0333)\n","Processed MH0098_GL0009663: non-AMP/OTHERS (Probabilities: 0.6072, 0.0189, 0.3740)\n","Processed 765701615-stool1_revised_scaffold36681_2_gene49805: Neuropeptide (Probabilities: 0.1290, 0.0209, 0.8502)\n","Processed MH0188_GL0036590: non-AMP/OTHERS (Probabilities: 0.9881, 0.0051, 0.0068)\n","Processed MH0088_GL0027555: non-AMP/OTHERS (Probabilities: 0.9904, 0.0046, 0.0049)\n","Processed MH0088_GL0027555: non-AMP/OTHERS (Probabilities: 0.9919, 0.0046, 0.0035)\n","Processed MH0246_GL0073032: non-AMP/OTHERS (Probabilities: 0.9933, 0.0036, 0.0031)\n","Processed V1.UC50-1_GL0095874: Neuropeptide (Probabilities: 0.0077, 0.0092, 0.9831)\n","Processed MH0001_GL0016004: Neuropeptide (Probabilities: 0.2111, 0.0281, 0.7608)\n","Processed 765094712-stool1_revised_scaffold3296_2_gene29312: Neuropeptide (Probabilities: 0.0580, 0.0290, 0.9130)\n","Processed 159632143-stool1_revised_scaffold7162_2_gene140920: Neuropeptide (Probabilities: 0.0109, 0.0104, 0.9787)\n","Processed 1042403.BALAC2494_00770: non-AMP/OTHERS (Probabilities: 0.7923, 0.0300, 0.1777)\n","Processed MH0147_GL0143781: Neuropeptide (Probabilities: 0.0472, 0.0183, 0.9345)\n","Processed MH0271_GL0029746: non-AMP/OTHERS (Probabilities: 0.9723, 0.0070, 0.0206)\n","Processed 159268001-stool2_revised_scaffold17156_1_gene14159: non-AMP/OTHERS (Probabilities: 0.8228, 0.0446, 0.1325)\n","Processed O2.UC17-1_GL0029675: Neuropeptide (Probabilities: 0.0449, 0.0170, 0.9381)\n","Processed MH0161_GL0053213: Neuropeptide (Probabilities: 0.0919, 0.0366, 0.8715)\n","Processed MH0377_GL0022788: Neuropeptide (Probabilities: 0.2962, 0.0396, 0.6642)\n","Processed MH0457_GL0222516: Neuropeptide (Probabilities: 0.2225, 0.0251, 0.7523)\n","Processed MH0020_GL0063748: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed O2.CD3-0-PT_GL0046945: non-AMP/OTHERS (Probabilities: 0.9892, 0.0040, 0.0067)\n","Processed O2.UC30-1_GL0183212: Neuropeptide (Probabilities: 0.0158, 0.0117, 0.9724)\n","Processed 158479027-stool1_revised_C768791_1_gene3365: non-AMP/OTHERS (Probabilities: 0.7454, 0.0213, 0.2334)\n","Processed O2.UC8-0_GL0029670: non-AMP/OTHERS (Probabilities: 0.9555, 0.0063, 0.0383)\n","Processed MH0003_GL0107482: non-AMP/OTHERS (Probabilities: 0.7795, 0.0381, 0.1824)\n","Processed O2.UC7-0_GL0116301: non-AMP/OTHERS (Probabilities: 0.9925, 0.0032, 0.0042)\n","Processed MH0012_GL0154419: non-AMP/OTHERS (Probabilities: 0.9918, 0.0036, 0.0046)\n","Processed MH0329_GL0068542: Neuropeptide (Probabilities: 0.1047, 0.0215, 0.8738)\n","Processed MH0012_GL0237776: non-AMP/OTHERS (Probabilities: 0.8770, 0.0217, 0.1012)\n","Processed V1.FI28_GL0017018: Neuropeptide (Probabilities: 0.2340, 0.0288, 0.7373)\n","Processed MH0089_GL0006156: Neuropeptide (Probabilities: 0.0093, 0.0100, 0.9806)\n","Processed MH0316_GL0144268: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed V1.CD51-0_GL0027106: Neuropeptide (Probabilities: 0.0066, 0.0094, 0.9841)\n","Processed V1.FI28_GL0168275: non-AMP/OTHERS (Probabilities: 0.4319, 0.3364, 0.2317)\n","Processed V1.FI29_GL0149073: non-AMP/OTHERS (Probabilities: 0.9925, 0.0046, 0.0030)\n","Processed MH0131_GL0169325: non-AMP/OTHERS (Probabilities: 0.9926, 0.0041, 0.0033)\n","Processed MH0339_GL0136203: non-AMP/OTHERS (Probabilities: 0.9929, 0.0044, 0.0027)\n","Processed MH0006_GL0191102: non-AMP/OTHERS (Probabilities: 0.9928, 0.0030, 0.0042)\n","Processed MH0131_GL0022802: non-AMP/OTHERS (Probabilities: 0.9912, 0.0036, 0.0051)\n","Processed MH0003_GL0104512: non-AMP/OTHERS (Probabilities: 0.9908, 0.0047, 0.0045)\n","Processed MH0011_GL0033934: non-AMP/OTHERS (Probabilities: 0.9921, 0.0033, 0.0047)\n","Processed MH0251_GL0016511: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed ED9A_GL0082226: non-AMP/OTHERS (Probabilities: 0.9223, 0.0124, 0.0652)\n","Processed V1.FI09_GL0088324: non-AMP/OTHERS (Probabilities: 0.7656, 0.0230, 0.2114)\n","Processed O2.UC11-2_GL0109840: non-AMP/OTHERS (Probabilities: 0.9855, 0.0044, 0.0101)\n","Processed MH0293_GL0021553: non-AMP/OTHERS (Probabilities: 0.9930, 0.0033, 0.0037)\n","Processed 521002.METSMIF1_02270: non-AMP/OTHERS (Probabilities: 0.9915, 0.0035, 0.0049)\n","Processed MH0131_GL0130279: Neuropeptide (Probabilities: 0.0078, 0.0088, 0.9834)\n","Processed MH0086_GL0024967: non-AMP/OTHERS (Probabilities: 0.9781, 0.0051, 0.0168)\n","Processed MH0431_GL0138040: Neuropeptide (Probabilities: 0.1119, 0.0257, 0.8624)\n","Processed O2.UC9-2_GL0027655: non-AMP/OTHERS (Probabilities: 0.7166, 0.0189, 0.2645)\n","Processed MH0341_GL0091658: non-AMP/OTHERS (Probabilities: 0.5337, 0.0670, 0.3993)\n","Processed 764447348-stool2_revised_scaffold32769_1_gene127380: non-AMP/OTHERS (Probabilities: 0.7823, 0.0148, 0.2028)\n","Processed MH0426_GL0151268: non-AMP/OTHERS (Probabilities: 0.9876, 0.0051, 0.0074)\n","Processed MH0281_GL0011141: non-AMP/OTHERS (Probabilities: 0.9888, 0.0048, 0.0064)\n","Processed SZEY-35A_GL0116858: non-AMP/OTHERS (Probabilities: 0.9928, 0.0037, 0.0035)\n","Processed V1.UC4-5_GL0020741: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed 763901136-stool1_revised_scaffold18727_1_gene129113: non-AMP/OTHERS (Probabilities: 0.9669, 0.0146, 0.0185)\n","Processed MH0010_GL0037706: non-AMP/OTHERS (Probabilities: 0.9916, 0.0051, 0.0033)\n","Processed MH0003_GL0090597: non-AMP/OTHERS (Probabilities: 0.9854, 0.0056, 0.0090)\n","Processed V1.UC47-0_GL0070676: non-AMP/OTHERS (Probabilities: 0.9873, 0.0050, 0.0077)\n","Processed V1.UC47-0_GL0070676: Neuropeptide (Probabilities: 0.0119, 0.0115, 0.9766)\n","Processed NLM031_GL0070563: Neuropeptide (Probabilities: 0.0077, 0.0092, 0.9830)\n","Processed MH0329_GL0068542: non-AMP/OTHERS (Probabilities: 0.9883, 0.0039, 0.0078)\n","Processed MH0012_GL0237776: non-AMP/OTHERS (Probabilities: 0.9863, 0.0045, 0.0093)\n","Processed MH0067_GL0070485: Neuropeptide (Probabilities: 0.0087, 0.0097, 0.9816)\n","Processed MH0333_GL0017496: Neuropeptide (Probabilities: 0.4310, 0.0333, 0.5357)\n","Processed MH0277_GL0032156: non-AMP/OTHERS (Probabilities: 0.6103, 0.0235, 0.3662)\n","Processed 435842.HMPREF0848_01496: Neuropeptide (Probabilities: 0.3205, 0.0588, 0.6206)\n","Processed BGI-33A_GL0041789: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed V1.UC55-4_GL0224385: non-AMP/OTHERS (Probabilities: 0.9852, 0.0099, 0.0048)\n","Processed MH0149_GL0128198: non-AMP/OTHERS (Probabilities: 0.9608, 0.0288, 0.0104)\n","Processed MH0011_GL0057108: Neuropeptide (Probabilities: 0.0092, 0.0092, 0.9816)\n","Processed tr|A0A8H6BTV7|A0A8H6BTV7_CANAX: non-AMP/OTHERS (Probabilities: 0.9786, 0.0061, 0.0153)\n","Processed MH0260_GL0144570: non-AMP/OTHERS (Probabilities: 0.8841, 0.0099, 0.1060)\n","Processed V1.UC31-4_GL0099327: non-AMP/OTHERS (Probabilities: 0.8667, 0.0248, 0.1085)\n","Processed 518635.BIFANG_03244: Neuropeptide (Probabilities: 0.4383, 0.0416, 0.5201)\n","Processed O2.UC30-1_GL0003421: non-AMP/OTHERS (Probabilities: 0.9866, 0.0052, 0.0081)\n","Processed MH0016_GL0053213: non-AMP/OTHERS (Probabilities: 0.9912, 0.0030, 0.0058)\n","Processed MH0005_GL0005098: non-AMP/OTHERS (Probabilities: 0.9925, 0.0034, 0.0041)\n","Processed MH0293_GL0060100: non-AMP/OTHERS (Probabilities: 0.9800, 0.0126, 0.0074)\n","Processed MH0088_GL0011153: Neuropeptide (Probabilities: 0.0293, 0.0220, 0.9486)\n","Processed 159268001-stool1_revised_scaffold14208_2_gene118930: Neuropeptide (Probabilities: 0.0572, 0.0339, 0.9089)\n","Processed sp|Q07837|SLC31_HUMAN: non-AMP/OTHERS (Probabilities: 0.9808, 0.0059, 0.0133)\n","Processed MH0200_GL0180919: non-AMP/OTHERS (Probabilities: 0.9906, 0.0035, 0.0058)\n","Processed NOM023_GL0022793: non-AMP/OTHERS (Probabilities: 0.9806, 0.0043, 0.0151)\n","Processed MH0377_GL0032996: Neuropeptide (Probabilities: 0.0157, 0.0109, 0.9734)\n","Processed V1.UC40-0_GL0106638: non-AMP/OTHERS (Probabilities: 0.9892, 0.0050, 0.0058)\n","Processed 158499257-stool1_revised_C1331047_1_gene18730: Neuropeptide (Probabilities: 0.0224, 0.0154, 0.9621)\n","Processed MH0086_GL0106459: non-AMP/OTHERS (Probabilities: 0.8927, 0.0102, 0.0971)\n","Processed MH0247_GL0089030: Neuropeptide (Probabilities: 0.0145, 0.0117, 0.9738)\n","Processed V1.CD8-0-PN_GL0073924: non-AMP/OTHERS (Probabilities: 0.9797, 0.0036, 0.0167)\n","Processed MH0188_GL0048449: Neuropeptide (Probabilities: 0.0908, 0.0183, 0.8909)\n","Processed MH0188_GL0048449: non-AMP/OTHERS (Probabilities: 0.9072, 0.0152, 0.0776)\n","Processed O2.UC40-2_GL0031829: non-AMP/OTHERS (Probabilities: 0.9916, 0.0046, 0.0038)\n","Processed MH0356_GL0195253: Neuropeptide (Probabilities: 0.0270, 0.0148, 0.9582)\n","Processed MH0330_GL0087566: non-AMP/OTHERS (Probabilities: 0.8808, 0.0102, 0.1090)\n","Processed V1.CD12-3_GL0112792: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0040)\n","Processed MH0006_GL0122694: non-AMP/OTHERS (Probabilities: 0.9915, 0.0037, 0.0048)\n","Processed 411490.ANACAC_00272: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0033)\n","Processed MH0104_GL0003552: Neuropeptide (Probabilities: 0.0523, 0.0202, 0.9275)\n","Processed MH0083_GL0029700: Neuropeptide (Probabilities: 0.0097, 0.0097, 0.9806)\n","Processed T2D-140A_GL0069425: non-AMP/OTHERS (Probabilities: 0.9923, 0.0031, 0.0046)\n","Processed MH0286_GL0124209: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed MH0355_GL0049271: non-AMP/OTHERS (Probabilities: 0.9902, 0.0034, 0.0064)\n","Processed MH0346_GL0144134: non-AMP/OTHERS (Probabilities: 0.9864, 0.0046, 0.0090)\n","Processed MH0161_GL0061147: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed V1.CD45-0_GL0117139: non-AMP/OTHERS (Probabilities: 0.8837, 0.0335, 0.0828)\n","Processed MH0196_GL0095203: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0060_GL0026101: Neuropeptide (Probabilities: 0.0082, 0.0090, 0.9828)\n","Processed MH0222_GL0093136: non-AMP/OTHERS (Probabilities: 0.9908, 0.0034, 0.0058)\n","Processed V1.UC33-0_GL0064936: non-AMP/OTHERS (Probabilities: 0.9930, 0.0028, 0.0042)\n","Processed V1.UC11-0_GL0065896: non-AMP/OTHERS (Probabilities: 0.9854, 0.0046, 0.0100)\n","Processed 411479.BACUNI_01476: Neuropeptide (Probabilities: 0.0219, 0.0190, 0.9592)\n","Processed MH0323_GL0130000: Neuropeptide (Probabilities: 0.4621, 0.0442, 0.4937)\n","Processed O2.UC50-0_GL0215845: Neuropeptide (Probabilities: 0.1646, 0.0435, 0.7919)\n","Processed MH0088_GL0083345: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed MH0251_GL0068384: non-AMP/OTHERS (Probabilities: 0.9914, 0.0053, 0.0033)\n","Processed MH0119_GL0002515: non-AMP/OTHERS (Probabilities: 0.9928, 0.0044, 0.0028)\n","Processed O2.UC40-1_GL0183557: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9852)\n","Processed N040A_GL0065006: Neuropeptide (Probabilities: 0.3120, 0.0412, 0.6468)\n","Processed O2.UC13-2_GL0111542: Neuropeptide (Probabilities: 0.1805, 0.0337, 0.7858)\n","Processed V1.FI29_GL0056834: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9852)\n","Processed MH0199_GL0066911: Neuropeptide (Probabilities: 0.0268, 0.0155, 0.9576)\n","Processed MH0272_GL0035167: Neuropeptide (Probabilities: 0.0148, 0.0109, 0.9742)\n","Processed MH0272_GL0035167: Neuropeptide (Probabilities: 0.0161, 0.0121, 0.9718)\n","Processed MH0012_GL0062455: Neuropeptide (Probabilities: 0.0064, 0.0087, 0.9850)\n","Processed MH0382_GL0055623: non-AMP/OTHERS (Probabilities: 0.9933, 0.0032, 0.0035)\n","Processed MH0006_GL0033658: Neuropeptide (Probabilities: 0.2680, 0.0258, 0.7062)\n","Processed MH0006_GL0033658: non-AMP/OTHERS (Probabilities: 0.8735, 0.0120, 0.1145)\n","Processed MH0104_GL0054429: non-AMP/OTHERS (Probabilities: 0.9878, 0.0034, 0.0089)\n","Processed V1.CD52-0_GL0031668: non-AMP/OTHERS (Probabilities: 0.9656, 0.0103, 0.0241)\n","Processed V1.UC1-0_GL0008565: non-AMP/OTHERS (Probabilities: 0.9924, 0.0037, 0.0039)\n","Processed MH0131_GL0085583: non-AMP/OTHERS (Probabilities: 0.5832, 0.0188, 0.3979)\n","Processed V1.FI17_GL0196642: Neuropeptide (Probabilities: 0.3769, 0.0259, 0.5971)\n","Processed MH0006_GL0052353: Neuropeptide (Probabilities: 0.0151, 0.0123, 0.9726)\n","Processed MH0131_GL0164712: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0037)\n","Processed MH0355_GL0051604: non-AMP/OTHERS (Probabilities: 0.9903, 0.0050, 0.0047)\n","Processed MH0442_GL0033056: non-AMP/OTHERS (Probabilities: 0.9920, 0.0037, 0.0043)\n","Processed N029A_GL0048149: non-AMP/OTHERS (Probabilities: 0.9788, 0.0177, 0.0035)\n","Processed MH0045_GL0023728: Neuropeptide (Probabilities: 0.0078, 0.0092, 0.9830)\n","Processed MH0277_GL0015364: non-AMP/OTHERS (Probabilities: 0.9908, 0.0039, 0.0053)\n","Processed MH0250_GL0268447: non-AMP/OTHERS (Probabilities: 0.9470, 0.0132, 0.0398)\n","Processed MH0316_GL0180652: Neuropeptide (Probabilities: 0.2872, 0.0292, 0.6836)\n","Processed MH0188_GL0067094: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9851)\n","Processed V1.UC16-0_GL0026399: non-AMP/OTHERS (Probabilities: 0.9795, 0.0057, 0.0148)\n","Processed MH0183_GL0044804: Neuropeptide (Probabilities: 0.0094, 0.0096, 0.9810)\n","Processed MH0012_GL0069804: non-AMP/OTHERS (Probabilities: 0.9882, 0.0046, 0.0072)\n","Processed N009A_GL0027799: non-AMP/OTHERS (Probabilities: 0.9904, 0.0040, 0.0056)\n","Processed MH0055_GL0033633: Neuropeptide (Probabilities: 0.0074, 0.0091, 0.9835)\n","Processed MH0131_GL0076986: non-AMP/OTHERS (Probabilities: 0.9293, 0.0439, 0.0268)\n","Processed 158337416-stool1_revised_scaffold5632_1_gene160091: non-AMP/OTHERS (Probabilities: 0.9865, 0.0086, 0.0049)\n","Processed T2D-62A_GL0004525: non-AMP/OTHERS (Probabilities: 0.9932, 0.0036, 0.0032)\n","Processed V1.FI26_GL0199729: non-AMP/OTHERS (Probabilities: 0.9930, 0.0029, 0.0041)\n","Processed V1.FI26_GL0199729: non-AMP/OTHERS (Probabilities: 0.9917, 0.0029, 0.0054)\n","Processed MH0418_GL0192656: non-AMP/OTHERS (Probabilities: 0.9922, 0.0031, 0.0048)\n","Processed MH0131_GL0036883: non-AMP/OTHERS (Probabilities: 0.9890, 0.0040, 0.0071)\n","Processed O2.UC12-2_GL0085855: non-AMP/OTHERS (Probabilities: 0.9846, 0.0041, 0.0112)\n","Processed MH0188_GL0008392: non-AMP/OTHERS (Probabilities: 0.8287, 0.0189, 0.1524)\n","Processed O2.UC46-0_GL0108472: non-AMP/OTHERS (Probabilities: 0.9276, 0.0096, 0.0628)\n","Processed MH0422_GL0001063: non-AMP/OTHERS (Probabilities: 0.6707, 0.0256, 0.3037)\n","Processed MH0408_GL0192340: non-AMP/OTHERS (Probabilities: 0.8391, 0.0119, 0.1490)\n","Processed MH0408_GL0192340: non-AMP/OTHERS (Probabilities: 0.9916, 0.0033, 0.0051)\n","Processed 764447348-stool2_revised_scaffold32270_2_gene133602: non-AMP/OTHERS (Probabilities: 0.9911, 0.0038, 0.0051)\n","Processed MH0187_GL0057988: non-AMP/OTHERS (Probabilities: 0.9930, 0.0029, 0.0041)\n","Processed MH0147_GL0122740: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0044)\n","Processed 479437.Elen_2237: non-AMP/OTHERS (Probabilities: 0.9914, 0.0040, 0.0047)\n","Processed 638301.HMPREF0444_1001: non-AMP/OTHERS (Probabilities: 0.9907, 0.0036, 0.0058)\n","Processed MH0197_GL0105431: non-AMP/OTHERS (Probabilities: 0.9478, 0.0123, 0.0399)\n","Processed MH0238_GL0078987: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed T2D-33A_GL0137257: non-AMP/OTHERS (Probabilities: 0.9920, 0.0036, 0.0044)\n","Processed MH0125_GL0005726: non-AMP/OTHERS (Probabilities: 0.9018, 0.0086, 0.0895)\n","Processed T2D-195A_GL0059200: non-AMP/OTHERS (Probabilities: 0.9883, 0.0034, 0.0083)\n","Processed O2.UC8-2_GL0090833: non-AMP/OTHERS (Probabilities: 0.5614, 0.0590, 0.3795)\n","Processed MH0071_GL0029234: non-AMP/OTHERS (Probabilities: 0.8793, 0.0273, 0.0933)\n","Processed MH0131_GL0082809: non-AMP/OTHERS (Probabilities: 0.9924, 0.0034, 0.0042)\n","Processed MH0131_GL0082809: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0035)\n","Processed ED14A_GL0006953: non-AMP/OTHERS (Probabilities: 0.7560, 0.0202, 0.2238)\n","Processed MH0277_GL0027646: Neuropeptide (Probabilities: 0.0071, 0.0086, 0.9843)\n","Processed O2.UC17-1_GL0102839: non-AMP/OTHERS (Probabilities: 0.9854, 0.0049, 0.0097)\n","Processed 435842.HMPREF0848_01251: non-AMP/OTHERS (Probabilities: 0.9908, 0.0038, 0.0054)\n","Processed MH0184_GL0107971: Neuropeptide (Probabilities: 0.0120, 0.0119, 0.9760)\n","Processed MH0003_GL0065028: non-AMP/OTHERS (Probabilities: 0.9899, 0.0039, 0.0062)\n","Processed MH0099_GL0008513: non-AMP/OTHERS (Probabilities: 0.9634, 0.0101, 0.0265)\n","Processed NOM002_GL0000647: non-AMP/OTHERS (Probabilities: 0.9905, 0.0038, 0.0058)\n","Processed MH0189_GL0171183: non-AMP/OTHERS (Probabilities: 0.9869, 0.0045, 0.0086)\n","Processed V1.CD35-1_GL0088353: non-AMP/OTHERS (Probabilities: 0.9720, 0.0047, 0.0233)\n","Processed O2.UC12-1_GL0043848: Neuropeptide (Probabilities: 0.1821, 0.0203, 0.7976)\n","Processed MH0014_GL0008746: Neuropeptide (Probabilities: 0.2221, 0.0210, 0.7570)\n","Processed V1.UC56-0_GL0048603: non-AMP/OTHERS (Probabilities: 0.9924, 0.0031, 0.0045)\n","Processed O2.UC15-0_GL0052903: non-AMP/OTHERS (Probabilities: 0.9917, 0.0031, 0.0052)\n","Processed V1.FI19_GL0125587: non-AMP/OTHERS (Probabilities: 0.7624, 0.0505, 0.1871)\n","Processed O2.UC56-2_GL0117888: Neuropeptide (Probabilities: 0.1448, 0.0239, 0.8313)\n","Processed T2D-57A_GL0107451: non-AMP/OTHERS (Probabilities: 0.8202, 0.0246, 0.1553)\n","Processed MH0062_GL0066940: Neuropeptide (Probabilities: 0.1246, 0.0260, 0.8494)\n","Processed MH0131_GL0074702: Neuropeptide (Probabilities: 0.4509, 0.0463, 0.5029)\n","Processed MH0188_GL0053015: Neuropeptide (Probabilities: 0.0064, 0.0086, 0.9850)\n","Processed O2.UC20-1_GL0116897: non-AMP/OTHERS (Probabilities: 0.9822, 0.0069, 0.0108)\n","Processed 1069533.Sinf_0283: Neuropeptide (Probabilities: 0.0072, 0.0089, 0.9839)\n","Processed MH0270_GL0037468: Neuropeptide (Probabilities: 0.0200, 0.0141, 0.9660)\n","Processed DLF004_GL0014611: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed MH0127_GL0099575: Neuropeptide (Probabilities: 0.0616, 0.0307, 0.9077)\n","Processed MH0183_GL0112110: Neuropeptide (Probabilities: 0.0095, 0.0094, 0.9811)\n","Processed 552396.HMPREF0863_01322: non-AMP/OTHERS (Probabilities: 0.9590, 0.0060, 0.0350)\n","Processed MH0086_GL0048271: Neuropeptide (Probabilities: 0.0072, 0.0089, 0.9840)\n","Processed MH0370_GL0062167: Neuropeptide (Probabilities: 0.0077, 0.0090, 0.9833)\n","Processed O2.UC41-1_GL0129471: non-AMP/OTHERS (Probabilities: 0.9843, 0.0034, 0.0123)\n","Processed O2.CD3-0-PT_GL0079343: non-AMP/OTHERS (Probabilities: 0.9858, 0.0035, 0.0107)\n","Processed 158883629-stool1_revised_C120575_1_gene11350: non-AMP/OTHERS (Probabilities: 0.9886, 0.0033, 0.0081)\n","Processed O2.UC59-0_GL0035519: non-AMP/OTHERS (Probabilities: 0.9617, 0.0062, 0.0321)\n","Processed MH0244_GL0046654: non-AMP/OTHERS (Probabilities: 0.9050, 0.0123, 0.0827)\n","Processed O2.CD2-0-PT_GL0067932: non-AMP/OTHERS (Probabilities: 0.9899, 0.0035, 0.0066)\n","Processed 763496533-stool1_revised_scaffold17300_1_gene161567: non-AMP/OTHERS (Probabilities: 0.7906, 0.0239, 0.1855)\n","Processed V1.UC47-4_GL0158789: Neuropeptide (Probabilities: 0.0504, 0.0243, 0.9253)\n","Processed N046A_GL0046391: non-AMP/OTHERS (Probabilities: 0.9933, 0.0031, 0.0037)\n","Processed tr|Q8TE24|Q8TE24_HUMAN: Neuropeptide (Probabilities: 0.0525, 0.0231, 0.9245)\n","Processed sp|O43451|MGA_HUMAN: Neuropeptide (Probabilities: 0.1377, 0.0524, 0.8099)\n","Processed MH0277_GL0004805: non-AMP/OTHERS (Probabilities: 0.9155, 0.0140, 0.0705)\n","Processed MH0161_GL0069344: Neuropeptide (Probabilities: 0.3031, 0.0215, 0.6754)\n","Processed DOF013_GL0053897: Neuropeptide (Probabilities: 0.0232, 0.0166, 0.9601)\n","Processed MH0346_GL0049242: non-AMP/OTHERS (Probabilities: 0.9616, 0.0073, 0.0311)\n","Processed MH0203_GL0105210: Neuropeptide (Probabilities: 0.3224, 0.0419, 0.6357)\n","Processed SZEY-35A_GL0124608: Neuropeptide (Probabilities: 0.0561, 0.0194, 0.9245)\n","Processed MH0045_GL0001950: non-AMP/OTHERS (Probabilities: 0.9085, 0.0152, 0.0764)\n","Processed MH0003_GL0014858: Neuropeptide (Probabilities: 0.0103, 0.0100, 0.9797)\n","Processed MH0098_GL0123687: non-AMP/OTHERS (Probabilities: 0.9913, 0.0034, 0.0053)\n","Processed 445972.ANACOL_00335: Neuropeptide (Probabilities: 0.0280, 0.0149, 0.9570)\n","Processed O2.UC56-2_GL0015554: Neuropeptide (Probabilities: 0.0059, 0.0087, 0.9854)\n","Processed MH0055_GL0009975: non-AMP/OTHERS (Probabilities: 0.9858, 0.0044, 0.0098)\n","Processed MH0197_GL0121100: Neuropeptide (Probabilities: 0.4859, 0.0254, 0.4887)\n","Processed V1.UC23-1_GL0090652: non-AMP/OTHERS (Probabilities: 0.9745, 0.0075, 0.0179)\n","Processed V1.UC23-1_GL0090652: non-AMP/OTHERS (Probabilities: 0.9914, 0.0047, 0.0039)\n","Processed MH0178_GL0000001: Neuropeptide (Probabilities: 0.4861, 0.0257, 0.4882)\n","Processed 763840445-stool2_revised_scaffold52861_1_gene3593: Neuropeptide (Probabilities: 0.0278, 0.0156, 0.9566)\n","Processed MH0188_GL0008392: non-AMP/OTHERS (Probabilities: 0.9839, 0.0050, 0.0110)\n","Processed MH0002_GL0007970: non-AMP/OTHERS (Probabilities: 0.9829, 0.0055, 0.0115)\n","Processed 585501.HMPREF6123_0872: non-AMP/OTHERS (Probabilities: 0.9927, 0.0035, 0.0038)\n","Processed BGI-33A_GL0005884: non-AMP/OTHERS (Probabilities: 0.9645, 0.0074, 0.0281)\n","Processed 1000570.HMPREF9966_1235: non-AMP/OTHERS (Probabilities: 0.9916, 0.0034, 0.0050)\n","Processed MH0088_GL0029939: non-AMP/OTHERS (Probabilities: 0.6179, 0.0269, 0.3552)\n","Processed O2.UC19-1_GL0019088: non-AMP/OTHERS (Probabilities: 0.8918, 0.0177, 0.0904)\n","Processed MH0277_GL0006763: non-AMP/OTHERS (Probabilities: 0.9304, 0.0123, 0.0572)\n","Processed V1.UC40-1_GL0006720: non-AMP/OTHERS (Probabilities: 0.8587, 0.0160, 0.1253)\n","Processed V1.UC40-1_GL0006720: non-AMP/OTHERS (Probabilities: 0.9752, 0.0060, 0.0188)\n","Processed MH0007_GL0001609: non-AMP/OTHERS (Probabilities: 0.9909, 0.0054, 0.0038)\n","Processed MH0422_GL0045782: non-AMP/OTHERS (Probabilities: 0.7944, 0.0147, 0.1910)\n","Processed MH0150_GL0065851: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed MH0002_GL0026945: Neuropeptide (Probabilities: 0.3971, 0.0374, 0.5655)\n","Processed MH0187_GL0043905: Neuropeptide (Probabilities: 0.0087, 0.0094, 0.9820)\n","Processed 159268001-stool2_revised_scaffold29216_3_gene162161: non-AMP/OTHERS (Probabilities: 0.9798, 0.0051, 0.0151)\n","Processed MH0176_GL0014450: Neuropeptide (Probabilities: 0.0665, 0.0263, 0.9072)\n","Processed MH0446_GL0029409: non-AMP/OTHERS (Probabilities: 0.8895, 0.0114, 0.0991)\n","Processed MH0185_GL0027169: Neuropeptide (Probabilities: 0.2647, 0.0222, 0.7131)\n","Processed V1.CD21-0_GL0017821: Neuropeptide (Probabilities: 0.0066, 0.0085, 0.9849)\n","Processed 864570.HMPREF9189_1425: non-AMP/OTHERS (Probabilities: 0.9804, 0.0064, 0.0133)\n","Processed O2.UC23-2_GL0008714: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0189_GL0085562: non-AMP/OTHERS (Probabilities: 0.9931, 0.0031, 0.0039)\n","Processed O2.UC57-0_GL0070922: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0037)\n","Processed V1.FI37_GL0214509: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0036)\n","Processed MH0001_GL0018220: non-AMP/OTHERS (Probabilities: 0.9934, 0.0036, 0.0030)\n","Processed MH0001_GL0018220: non-AMP/OTHERS (Probabilities: 0.9935, 0.0035, 0.0030)\n","Processed O2.UC22-1_GL0047312: Neuropeptide (Probabilities: 0.0120, 0.0108, 0.9772)\n","Processed NOM020_GL0027694: non-AMP/OTHERS (Probabilities: 0.9877, 0.0034, 0.0089)\n","Processed MH0119_GL0104585: non-AMP/OTHERS (Probabilities: 0.9931, 0.0031, 0.0038)\n","Processed V1.FI20_GL0149374: non-AMP/OTHERS (Probabilities: 0.9929, 0.0030, 0.0041)\n","Processed V1.UC29-0_GL0091932: Neuropeptide (Probabilities: 0.0138, 0.0099, 0.9763)\n","Processed MH0188_GL0111302: Neuropeptide (Probabilities: 0.0175, 0.0114, 0.9711)\n","Processed MH0233_GL0107880: non-AMP/OTHERS (Probabilities: 0.9725, 0.0059, 0.0217)\n","Processed V1.UC31-4_GL0150221: non-AMP/OTHERS (Probabilities: 0.9617, 0.0203, 0.0181)\n","Processed T2D-176A_GL0034912: non-AMP/OTHERS (Probabilities: 0.9921, 0.0033, 0.0046)\n","Processed V1.CD25-0_GL0063271: non-AMP/OTHERS (Probabilities: 0.9841, 0.0043, 0.0116)\n","Processed O2.UC7-1_GL0020513: non-AMP/OTHERS (Probabilities: 0.8209, 0.0330, 0.1460)\n","Processed T2D-163A_GL0003254: non-AMP/OTHERS (Probabilities: 0.9909, 0.0030, 0.0061)\n","Processed MH0096_GL0015875: non-AMP/OTHERS (Probabilities: 0.9904, 0.0036, 0.0060)\n","Processed MH0110_GL0003128: non-AMP/OTHERS (Probabilities: 0.9922, 0.0034, 0.0044)\n","Processed MH0203_GL0133087: non-AMP/OTHERS (Probabilities: 0.9862, 0.0055, 0.0082)\n","Processed MH0003_GL0083869: non-AMP/OTHERS (Probabilities: 0.9901, 0.0044, 0.0055)\n","Processed 158499257-stool2_revised_scaffold66901_1_gene211726: Neuropeptide (Probabilities: 0.0066, 0.0087, 0.9847)\n","Processed MH0251_GL0016511: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed MH0086_GL0009190: Neuropeptide (Probabilities: 0.0139, 0.0111, 0.9749)\n","Processed MH0002_GL0061142: Neuropeptide (Probabilities: 0.2917, 0.1045, 0.6038)\n","Processed MH0332_GL0065635: non-AMP/OTHERS (Probabilities: 0.9906, 0.0046, 0.0049)\n","Processed MH0333_GL0019870: non-AMP/OTHERS (Probabilities: 0.9919, 0.0045, 0.0036)\n","Processed MH0089_GL0027868: Neuropeptide (Probabilities: 0.0069, 0.0088, 0.9843)\n","Processed MH0272_GL0014356: non-AMP/OTHERS (Probabilities: 0.9265, 0.0157, 0.0578)\n","Processed MH0189_GL0021744: Neuropeptide (Probabilities: 0.0138, 0.0119, 0.9743)\n","Processed MH0274_GL0106788: non-AMP/OTHERS (Probabilities: 0.9716, 0.0046, 0.0238)\n","Processed MH0188_GL0146214: Neuropeptide (Probabilities: 0.0231, 0.0137, 0.9632)\n","Processed MH0272_GL0192805: non-AMP/OTHERS (Probabilities: 0.8777, 0.0242, 0.0982)\n","Processed MH0267_GL0070059: non-AMP/OTHERS (Probabilities: 0.9865, 0.0091, 0.0044)\n","Processed MH0003_GL0104512: non-AMP/OTHERS (Probabilities: 0.6301, 0.0337, 0.3362)\n","Processed MH0206_GL0150155: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0035)\n","Processed T2D-177A_GL0019614: non-AMP/OTHERS (Probabilities: 0.9932, 0.0031, 0.0038)\n","Processed sp|P00338|LDHA_HUMAN: non-AMP/OTHERS (Probabilities: 0.5796, 0.0452, 0.3752)\n","Processed MH0229_GL0072915: non-AMP/OTHERS (Probabilities: 0.9254, 0.0143, 0.0603)\n","Processed 764143897-stool1_revised_scaffold24988_1_gene58660: non-AMP/OTHERS (Probabilities: 0.9885, 0.0044, 0.0072)\n","Processed O2.UC35-2_GL0066288: non-AMP/OTHERS (Probabilities: 0.9922, 0.0032, 0.0045)\n","Processed MH0277_GL0023728: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9849)\n","Processed MH0153_GL0000584: non-AMP/OTHERS (Probabilities: 0.9790, 0.0054, 0.0156)\n","Processed MH0230_GL0010025: non-AMP/OTHERS (Probabilities: 0.9672, 0.0054, 0.0274)\n","Processed V1.CD10-0_GL0022498: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed MH0012_GL0011046: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed O2.UC19-2_GL0076209: Neuropeptide (Probabilities: 0.0062, 0.0088, 0.9850)\n","Processed MH0065_GL0020891: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed MH0446_GL0032744: non-AMP/OTHERS (Probabilities: 0.9926, 0.0041, 0.0033)\n","Processed MH0131_GL0070876: Neuropeptide (Probabilities: 0.0257, 0.0170, 0.9572)\n","Processed MH0131_GL0070876: non-AMP/OTHERS (Probabilities: 0.9161, 0.0129, 0.0709)\n","Processed MH0131_GL0044184: non-AMP/OTHERS (Probabilities: 0.9801, 0.0049, 0.0151)\n","Processed MH0068_GL0025194: non-AMP/OTHERS (Probabilities: 0.9909, 0.0048, 0.0044)\n","Processed MH0002_GL0029261: non-AMP/OTHERS (Probabilities: 0.9691, 0.0047, 0.0262)\n","Processed MH0002_GL0029261: non-AMP/OTHERS (Probabilities: 0.6093, 0.0235, 0.3672)\n","Processed MH0006_GL0205611: non-AMP/OTHERS (Probabilities: 0.4998, 0.0224, 0.4778)\n","Processed MH0086_GL0010554: Neuropeptide (Probabilities: 0.0066, 0.0087, 0.9847)\n","Processed SZEY-66A_GL0066418: non-AMP/OTHERS (Probabilities: 0.8051, 0.0145, 0.1805)\n","Processed SZEY-66A_GL0066418: non-AMP/OTHERS (Probabilities: 0.9248, 0.0083, 0.0669)\n","Processed MH0144_GL0122172: non-AMP/OTHERS (Probabilities: 0.9916, 0.0041, 0.0042)\n","Processed MH0281_GL0091133: non-AMP/OTHERS (Probabilities: 0.9890, 0.0064, 0.0046)\n","Processed N051A_GL0045736: Neuropeptide (Probabilities: 0.0087, 0.0103, 0.9809)\n","Processed O2.CD1-0-PT_GL0052648: non-AMP/OTHERS (Probabilities: 0.8753, 0.0202, 0.1045)\n","Processed MH0100_GL0112378: non-AMP/OTHERS (Probabilities: 0.9921, 0.0034, 0.0045)\n","Processed 160765029-stool1_revised_C1103680_1_gene68887: Neuropeptide (Probabilities: 0.0080, 0.0089, 0.9830)\n","Processed MH0088_GL0084225: non-AMP/OTHERS (Probabilities: 0.9516, 0.0175, 0.0309)\n","Processed V1.CD31-0_GL0134962: non-AMP/OTHERS (Probabilities: 0.9875, 0.0045, 0.0080)\n","Processed MH0211_GL0033395: non-AMP/OTHERS (Probabilities: 0.9841, 0.0073, 0.0086)\n","Processed MH0288_GL0064060: non-AMP/OTHERS (Probabilities: 0.9873, 0.0063, 0.0064)\n","Processed 764588959-stool1_revised_C741214_1_gene35265: non-AMP/OTHERS (Probabilities: 0.9879, 0.0069, 0.0052)\n","Processed MH0014_GL0079550: non-AMP/OTHERS (Probabilities: 0.9901, 0.0052, 0.0046)\n","Processed O2.UC30-0_GL0072593: non-AMP/OTHERS (Probabilities: 0.9912, 0.0050, 0.0038)\n","Processed MH0277_GL0039636: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed O2.UC17-1_GL0097332: non-AMP/OTHERS (Probabilities: 0.9927, 0.0029, 0.0045)\n","Processed MH0276_GL0123335: Neuropeptide (Probabilities: 0.0638, 0.0196, 0.9167)\n","Processed O2.UC31-1_GL0115264: non-AMP/OTHERS (Probabilities: 0.6350, 0.0289, 0.3361)\n","Processed MH0211_GL0050697: Neuropeptide (Probabilities: 0.0274, 0.0139, 0.9587)\n","Processed MH0188_GL0186218: Neuropeptide (Probabilities: 0.0628, 0.0182, 0.9190)\n","Processed MH0150_GL0143313: non-AMP/OTHERS (Probabilities: 0.9238, 0.0123, 0.0638)\n","Processed MH0403_GL0178841: Neuropeptide (Probabilities: 0.0125, 0.0112, 0.9763)\n","Processed MH0310_GL0036108: non-AMP/OTHERS (Probabilities: 0.8462, 0.0263, 0.1275)\n","Processed MH0120_GL0064060: Neuropeptide (Probabilities: 0.0625, 0.0210, 0.9165)\n","Processed V1.FI15_GL0127882: Neuropeptide (Probabilities: 0.0059, 0.0092, 0.9849)\n","Processed T2D-62A_GL0052960: Neuropeptide (Probabilities: 0.0092, 0.0094, 0.9814)\n","Processed MH0015_GL0029020: Neuropeptide (Probabilities: 0.0685, 0.0252, 0.9064)\n","Processed MH0233_GL0018529: non-AMP/OTHERS (Probabilities: 0.9583, 0.0073, 0.0344)\n","Processed MH0443_GL0138790: Neuropeptide (Probabilities: 0.1651, 0.0289, 0.8061)\n","Processed MH0199_GL0001383: non-AMP/OTHERS (Probabilities: 0.9928, 0.0031, 0.0041)\n","Processed DLM027_GL0037652: non-AMP/OTHERS (Probabilities: 0.9911, 0.0053, 0.0036)\n","Processed 158479027-stool1_revised_scaffold27522_1_gene8866: non-AMP/OTHERS (Probabilities: 0.9916, 0.0042, 0.0042)\n","Processed MH0016_GL0003996: non-AMP/OTHERS (Probabilities: 0.9914, 0.0042, 0.0044)\n","Processed MH0298_GL0016710: non-AMP/OTHERS (Probabilities: 0.9915, 0.0042, 0.0043)\n","Processed V1.UC48-0_GL0211858: non-AMP/OTHERS (Probabilities: 0.8793, 0.0114, 0.1094)\n","Processed O2.UC20-1_GL0040812: non-AMP/OTHERS (Probabilities: 0.9924, 0.0029, 0.0047)\n","Processed MH0003_GL0098970: non-AMP/OTHERS (Probabilities: 0.9874, 0.0054, 0.0072)\n","Processed MH0277_GL0038937: non-AMP/OTHERS (Probabilities: 0.9903, 0.0064, 0.0033)\n","Processed MH0118_GL0091085: non-AMP/OTHERS (Probabilities: 0.9900, 0.0053, 0.0048)\n","Processed MH0277_GL0054420: non-AMP/OTHERS (Probabilities: 0.9915, 0.0049, 0.0036)\n","Processed O2.UC11-1_GL0051769: non-AMP/OTHERS (Probabilities: 0.9913, 0.0050, 0.0037)\n","Processed MH0277_GL0029545: Neuropeptide (Probabilities: 0.0089, 0.0099, 0.9812)\n","Processed N052A_GL0031351: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9850)\n","Processed DLF005_GL0087140: non-AMP/OTHERS (Probabilities: 0.5052, 0.0271, 0.4678)\n","Processed N032A_GL0013223: non-AMP/OTHERS (Probabilities: 0.9933, 0.0030, 0.0037)\n","Processed 367928.BAD_0287: non-AMP/OTHERS (Probabilities: 0.9926, 0.0036, 0.0038)\n","Processed sp|P08123|CO1A2_HUMAN: non-AMP/OTHERS (Probabilities: 0.9897, 0.0056, 0.0047)\n","Processed MH0002_GL0016344: Neuropeptide (Probabilities: 0.1926, 0.0363, 0.7711)\n","Processed 206672.BL1146: Neuropeptide (Probabilities: 0.1066, 0.0246, 0.8688)\n","Processed SZEY-09A_GL0037074: non-AMP/OTHERS (Probabilities: 0.9544, 0.0097, 0.0360)\n","Processed 679189.HMPREF9019_1914: non-AMP/OTHERS (Probabilities: 0.8077, 0.0177, 0.1746)\n","Processed MH0440_GL0073289: non-AMP/OTHERS (Probabilities: 0.9931, 0.0031, 0.0038)\n","Processed V1.CD46-0_GL0074297: non-AMP/OTHERS (Probabilities: 0.9865, 0.0040, 0.0095)\n","Processed 245018.CL2_05410: Neuropeptide (Probabilities: 0.0096, 0.0098, 0.9806)\n","Processed 764143897-stool1_revised_scaffold8269_1_gene46889: non-AMP/OTHERS (Probabilities: 0.5559, 0.0227, 0.4214)\n","Processed DOM012_GL0039444: Neuropeptide (Probabilities: 0.4782, 0.0236, 0.4982)\n","Processed MH0236_GL0091042: Neuropeptide (Probabilities: 0.0064, 0.0089, 0.9847)\n","Processed 160704339-stool1_revised_scaffold9618_1_gene27378: Neuropeptide (Probabilities: 0.0117, 0.0105, 0.9778)\n","Processed O2.UC46-0_GL0108472: non-AMP/OTHERS (Probabilities: 0.5323, 0.0259, 0.4418)\n","Processed MH0371_GL0087622: Neuropeptide (Probabilities: 0.0441, 0.0176, 0.9383)\n","Processed MH0270_GL0179185: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9853)\n","Processed MH0370_GL0009991: non-AMP/OTHERS (Probabilities: 0.6404, 0.0635, 0.2961)\n","Processed MH0267_GL0014081: non-AMP/OTHERS (Probabilities: 0.7467, 0.0377, 0.2155)\n","Processed MH0191_GL0117571: non-AMP/OTHERS (Probabilities: 0.9872, 0.0035, 0.0093)\n","Processed MH0371_GL0002477: Neuropeptide (Probabilities: 0.0099, 0.0098, 0.9803)\n","Processed SZEY-24A_GL0023853: non-AMP/OTHERS (Probabilities: 0.9468, 0.0073, 0.0459)\n","Processed MH0322_GL0094299: non-AMP/OTHERS (Probabilities: 0.8197, 0.0222, 0.1581)\n","Processed SZEY-07A_GL0113829: Neuropeptide (Probabilities: 0.1774, 0.0286, 0.7940)\n","Processed V1.CD38-0_GL0063722: Neuropeptide (Probabilities: 0.2534, 0.0283, 0.7183)\n","Processed MH0003_GL0032234: non-AMP/OTHERS (Probabilities: 0.6824, 0.0183, 0.2992)\n","Processed DLM019_GL0086684: non-AMP/OTHERS (Probabilities: 0.9921, 0.0031, 0.0048)\n","Processed O2.CD1-0-PT_GL0078644: non-AMP/OTHERS (Probabilities: 0.9440, 0.0145, 0.0415)\n","Processed MH0090_GL0020758: non-AMP/OTHERS (Probabilities: 0.9928, 0.0044, 0.0029)\n","Processed MH0198_GL0050498: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0034)\n","Processed MH0193_GL0140286: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed MH0113_GL0097271: non-AMP/OTHERS (Probabilities: 0.9888, 0.0036, 0.0076)\n","Processed O2.CD2-0-PT_GL0048593: non-AMP/OTHERS (Probabilities: 0.8697, 0.0135, 0.1168)\n","Processed MH0229_GL0072054: non-AMP/OTHERS (Probabilities: 0.9922, 0.0034, 0.0044)\n","Processed MH0104_GL0101416: Neuropeptide (Probabilities: 0.0741, 0.0272, 0.8987)\n","Processed MH0276_GL0227646: Neuropeptide (Probabilities: 0.0658, 0.0209, 0.9133)\n","Processed N039A_GL0072487: non-AMP/OTHERS (Probabilities: 0.9887, 0.0059, 0.0054)\n","Processed 765094712-stool2_revised_scaffold17101_1_gene34809: non-AMP/OTHERS (Probabilities: 0.9928, 0.0033, 0.0039)\n","Processed 159247771-stool1_revised_scaffold718_3_gene72089: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed MH0191_GL0183947: non-AMP/OTHERS (Probabilities: 0.9927, 0.0033, 0.0040)\n","Processed O2.UC40-0_GL0018375: non-AMP/OTHERS (Probabilities: 0.9878, 0.0046, 0.0076)\n","Processed MH0415_GL0106961: non-AMP/OTHERS (Probabilities: 0.9571, 0.0086, 0.0344)\n","Processed MH0161_GL0035596: non-AMP/OTHERS (Probabilities: 0.9770, 0.0043, 0.0187)\n","Processed MH0071_GL0040079: non-AMP/OTHERS (Probabilities: 0.9768, 0.0047, 0.0185)\n","Processed V1.CD10-0_GL0037831: non-AMP/OTHERS (Probabilities: 0.9911, 0.0031, 0.0057)\n","Processed T2D-35A_GL0004763: non-AMP/OTHERS (Probabilities: 0.9934, 0.0030, 0.0037)\n","Processed MH0387_GL0029039: non-AMP/OTHERS (Probabilities: 0.9121, 0.0179, 0.0700)\n","Processed 764325968-stool1_revised_C587037_1_gene57487: non-AMP/OTHERS (Probabilities: 0.8817, 0.0218, 0.0965)\n","Processed V1.UC16-0_GL0086389: non-AMP/OTHERS (Probabilities: 0.9919, 0.0041, 0.0040)\n","Processed 518635.BIFANG_02645: non-AMP/OTHERS (Probabilities: 0.9888, 0.0051, 0.0061)\n","Processed MH0077_GL0064107: non-AMP/OTHERS (Probabilities: 0.9872, 0.0058, 0.0070)\n","Processed O2.UC20-1_GL0158276: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed MH0059_GL0067641: Neuropeptide (Probabilities: 0.0162, 0.0122, 0.9716)\n","Processed MH0086_GL0070888: Neuropeptide (Probabilities: 0.0231, 0.0138, 0.9630)\n","Processed MH0433_GL0037078: non-AMP/OTHERS (Probabilities: 0.9931, 0.0031, 0.0039)\n","Processed MH0188_GL0131572: non-AMP/OTHERS (Probabilities: 0.7543, 0.0317, 0.2140)\n","Processed MH0131_GL0050088: non-AMP/OTHERS (Probabilities: 0.9921, 0.0034, 0.0044)\n","Processed MH0277_GL0055021: Neuropeptide (Probabilities: 0.0251, 0.0142, 0.9607)\n","Processed MH0274_GL0007230: Neuropeptide (Probabilities: 0.0079, 0.0091, 0.9830)\n","Processed 411481.BIFADO_01551: non-AMP/OTHERS (Probabilities: 0.9759, 0.0054, 0.0188)\n","Processed MH0014_GL0013311: Neuropeptide (Probabilities: 0.0150, 0.0112, 0.9738)\n","Processed MH0086_GL0089125: Neuropeptide (Probabilities: 0.0062, 0.0087, 0.9851)\n","Processed MH0368_GL0125902: non-AMP/OTHERS (Probabilities: 0.9922, 0.0043, 0.0035)\n","Processed MH0203_GL0099345: Neuropeptide (Probabilities: 0.0298, 0.0172, 0.9530)\n","Processed MH0022_GL0016287: Neuropeptide (Probabilities: 0.1938, 0.0401, 0.7661)\n","Processed MH0454_GL0013007: Neuropeptide (Probabilities: 0.2090, 0.0319, 0.7591)\n","Processed MH0086_GL0071397: Neuropeptide (Probabilities: 0.0077, 0.0090, 0.9832)\n","Processed MH0002_GL0039425: non-AMP/OTHERS (Probabilities: 0.9908, 0.0037, 0.0055)\n","Processed V1.UC4-5_GL0108751: non-AMP/OTHERS (Probabilities: 0.9923, 0.0033, 0.0044)\n","Processed V1.UC4-5_GL0108751: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0031)\n","Processed O2.UC2-0_GL0037764: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0039)\n","Processed MH0277_GL0012081: non-AMP/OTHERS (Probabilities: 0.8478, 0.0165, 0.1357)\n","Processed DOF004_GL0038070: Neuropeptide (Probabilities: 0.1917, 0.0198, 0.7885)\n","Processed O2.UC50-0_GL0230434: Neuropeptide (Probabilities: 0.3132, 0.0250, 0.6618)\n","Processed O2.UC32-0_GL0058052: Neuropeptide (Probabilities: 0.0093, 0.0096, 0.9811)\n","Processed N024A_GL0063714: Neuropeptide (Probabilities: 0.0095, 0.0097, 0.9808)\n","Processed MH0086_GL0026524: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0104_GL0031064: Neuropeptide (Probabilities: 0.0091, 0.0094, 0.9815)\n","Processed 246515023-stool1_revised_scaffold4703_1_gene50269: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0114_GL0049218: Neuropeptide (Probabilities: 0.0161, 0.0118, 0.9721)\n","Processed MH0012_GL0068496: Neuropeptide (Probabilities: 0.0171, 0.0117, 0.9712)\n","Processed sp|Q8NCR9|CLRN3_HUMAN: Neuropeptide (Probabilities: 0.0240, 0.0131, 0.9629)\n","Processed MH0396_GL0048530: non-AMP/OTHERS (Probabilities: 0.5640, 0.0276, 0.4083)\n","Processed MH0396_GL0097376: non-AMP/OTHERS (Probabilities: 0.9928, 0.0030, 0.0042)\n","Processed O2.UC22-1_GL0182512: non-AMP/OTHERS (Probabilities: 0.9896, 0.0032, 0.0072)\n","Processed MH0222_GL0101100: Neuropeptide (Probabilities: 0.3588, 0.0277, 0.6135)\n","Processed MH0086_GL0095466: Neuropeptide (Probabilities: 0.0144, 0.0123, 0.9733)\n","Processed MH0088_GL0054800: non-AMP/OTHERS (Probabilities: 0.9390, 0.0097, 0.0513)\n","Processed MH0184_GL0001534: Neuropeptide (Probabilities: 0.0060, 0.0086, 0.9853)\n","Processed MH0020_GL0055752: Neuropeptide (Probabilities: 0.0344, 0.0151, 0.9505)\n","Processed MH0020_GL0055752: non-AMP/OTHERS (Probabilities: 0.9911, 0.0030, 0.0059)\n","Processed MH0003_GL0030541: non-AMP/OTHERS (Probabilities: 0.7508, 0.0208, 0.2284)\n","Processed 764487809-stool2_revised_scaffold49165_1_gene164467: Neuropeptide (Probabilities: 0.0965, 0.0198, 0.8837)\n","Processed MH0003_GL0048087: Neuropeptide (Probabilities: 0.0268, 0.0146, 0.9586)\n","Processed MH0062_GL0032065: Neuropeptide (Probabilities: 0.0078, 0.0088, 0.9834)\n","Processed MH0227_GL0102901: Neuropeptide (Probabilities: 0.2754, 0.0308, 0.6937)\n","Processed SZEY-78A_GL0098701: Neuropeptide (Probabilities: 0.0555, 0.0227, 0.9218)\n","Processed MH0170_GL0116780: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0212_GL0074922: Neuropeptide (Probabilities: 0.0067, 0.0087, 0.9846)\n","Processed MH0012_GL0084211: non-AMP/OTHERS (Probabilities: 0.9905, 0.0037, 0.0059)\n","Processed MH0012_GL0084211: non-AMP/OTHERS (Probabilities: 0.8375, 0.0176, 0.1449)\n","Processed MH0425_GL0012276: Neuropeptide (Probabilities: 0.0081, 0.0090, 0.9828)\n","Processed MH0348_GL0122167: Neuropeptide (Probabilities: 0.0079, 0.0089, 0.9832)\n","Processed MH0211_GL0050697: Neuropeptide (Probabilities: 0.0327, 0.0142, 0.9531)\n","Processed T2D-114A_GL0007999: Neuropeptide (Probabilities: 0.0063, 0.0087, 0.9850)\n","Processed MH0086_GL0119665: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0431_GL0064120: non-AMP/OTHERS (Probabilities: 0.9917, 0.0053, 0.0031)\n","Processed MH0230_GL0036758: non-AMP/OTHERS (Probabilities: 0.9929, 0.0039, 0.0032)\n","Processed MH0088_GL0077901: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed sp|P02452|CO1A1_HUMAN: AMP (Probabilities: 0.3146, 0.6654, 0.0200)\n","Processed O2.CD2-0-PT_GL0041547: non-AMP/OTHERS (Probabilities: 0.9743, 0.0079, 0.0178)\n","Processed 404239096-stool1_revised_scaffold26117_2_gene23128: non-AMP/OTHERS (Probabilities: 0.9923, 0.0028, 0.0049)\n","Processed BGI-27A_GL0054907: non-AMP/OTHERS (Probabilities: 0.9789, 0.0066, 0.0145)\n","Processed MH0002_GL0006448: non-AMP/OTHERS (Probabilities: 0.9931, 0.0035, 0.0035)\n","Processed DLF013_GL0053478: Neuropeptide (Probabilities: 0.2592, 0.0311, 0.7097)\n","Processed V1.FI01_GL0076253: Neuropeptide (Probabilities: 0.2097, 0.0199, 0.7703)\n","Processed MH0211_GL0070524: non-AMP/OTHERS (Probabilities: 0.9918, 0.0044, 0.0038)\n","Processed MH0377_GL0129393: non-AMP/OTHERS (Probabilities: 0.9886, 0.0049, 0.0065)\n","Processed MH0277_GL0039635: non-AMP/OTHERS (Probabilities: 0.9924, 0.0039, 0.0037)\n","Processed MH0003_GL0088223: Neuropeptide (Probabilities: 0.2423, 0.0562, 0.7014)\n","Processed MH0159_GL0050552: non-AMP/OTHERS (Probabilities: 0.9795, 0.0067, 0.0138)\n","Processed N038A_GL0078372: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed MH0133_GL0011222: Neuropeptide (Probabilities: 0.0072, 0.0087, 0.9840)\n","Processed O2.UC27-1_GL0111545: Neuropeptide (Probabilities: 0.0094, 0.0096, 0.9810)\n","Processed 349741.Amuc_1039: non-AMP/OTHERS (Probabilities: 0.9853, 0.0054, 0.0093)\n","Processed MH0451_GL0030877: non-AMP/OTHERS (Probabilities: 0.9724, 0.0082, 0.0193)\n","Processed MH0002_GL0042701: non-AMP/OTHERS (Probabilities: 0.9134, 0.0171, 0.0695)\n","Processed sp|Q14108|SCRB2_HUMAN: Neuropeptide (Probabilities: 0.0961, 0.0215, 0.8824)\n","Processed V1.FI29_GL0006683: non-AMP/OTHERS (Probabilities: 0.9933, 0.0034, 0.0034)\n","Processed 409438.ECSE_2080: non-AMP/OTHERS (Probabilities: 0.9851, 0.0079, 0.0070)\n","Processed MH0221_GL0061395: non-AMP/OTHERS (Probabilities: 0.9460, 0.0074, 0.0467)\n","Processed 764062976-stool2_revised_scaffold5551_1_gene59321: non-AMP/OTHERS (Probabilities: 0.9779, 0.0052, 0.0170)\n","Processed NOM026_GL0004303: non-AMP/OTHERS (Probabilities: 0.9332, 0.0198, 0.0470)\n","Processed MH0277_GL0015228: non-AMP/OTHERS (Probabilities: 0.7346, 0.0218, 0.2436)\n","Processed O2.UC13-1_GL0104205: non-AMP/OTHERS (Probabilities: 0.8009, 0.0234, 0.1757)\n","Processed MH0131_GL0101467: non-AMP/OTHERS (Probabilities: 0.9906, 0.0031, 0.0063)\n","Processed V1.CD19-0_GL0085138: Neuropeptide (Probabilities: 0.2283, 0.0220, 0.7497)\n","Processed SZEY-44A_GL0015657: Neuropeptide (Probabilities: 0.0126, 0.0110, 0.9764)\n","Processed MH0108_GL0087557: Neuropeptide (Probabilities: 0.0903, 0.0230, 0.8866)\n","Processed MH0243_GL0062285: non-AMP/OTHERS (Probabilities: 0.9543, 0.0154, 0.0303)\n","Processed N038A_GL0001480: non-AMP/OTHERS (Probabilities: 0.9851, 0.0047, 0.0102)\n","Processed O2.UC17-1_GL0117257: Neuropeptide (Probabilities: 0.3028, 0.0222, 0.6750)\n","Processed MH0037_GL0049232: non-AMP/OTHERS (Probabilities: 0.9920, 0.0039, 0.0042)\n","Processed MH0146_GL0001011: Neuropeptide (Probabilities: 0.0375, 0.0210, 0.9415)\n","Processed MH0104_GL0099888: non-AMP/OTHERS (Probabilities: 0.9381, 0.0184, 0.0435)\n","Processed 763982056-stool1_revised_scaffold155_1_gene34227: non-AMP/OTHERS (Probabilities: 0.9136, 0.0190, 0.0674)\n","Processed V1.UC38-0_GL0024858: non-AMP/OTHERS (Probabilities: 0.9902, 0.0046, 0.0051)\n","Processed MH0327_GL0079702: Neuropeptide (Probabilities: 0.0517, 0.0223, 0.9260)\n","Processed MH0327_GL0079702: Neuropeptide (Probabilities: 0.0139, 0.0129, 0.9732)\n","Processed 158458797-stool1_revised_scaffold25252_2_gene79145: non-AMP/OTHERS (Probabilities: 0.9916, 0.0031, 0.0053)\n","Processed MH0028_GL0088541: non-AMP/OTHERS (Probabilities: 0.9920, 0.0030, 0.0050)\n","Processed MH0200_GL0043366: non-AMP/OTHERS (Probabilities: 0.9928, 0.0042, 0.0029)\n","Processed MH0359_GL0124487: non-AMP/OTHERS (Probabilities: 0.9934, 0.0034, 0.0032)\n","Processed MH0086_GL0103503: Neuropeptide (Probabilities: 0.3579, 0.0460, 0.5961)\n","Processed ED14A_GL0038280: Neuropeptide (Probabilities: 0.0997, 0.0213, 0.8790)\n","Processed MH0131_GL0160070: Neuropeptide (Probabilities: 0.0411, 0.0165, 0.9425)\n","Processed MH0188_GL0186841: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed V1.FI22_GL0017171: Neuropeptide (Probabilities: 0.0067, 0.0085, 0.9848)\n","Processed sp|P05556|ITB1_HUMAN: Neuropeptide (Probabilities: 0.3080, 0.1349, 0.5571)\n","Processed V1.UC25-0_GL0051029: non-AMP/OTHERS (Probabilities: 0.9767, 0.0054, 0.0179)\n","Processed MH0006_GL0144991: non-AMP/OTHERS (Probabilities: 0.9897, 0.0037, 0.0066)\n","Processed sp|Q9Y6R7|FCGBP_HUMAN: non-AMP/OTHERS (Probabilities: 0.9116, 0.0191, 0.0693)\n","Processed O2.UC17-1_GL0001756: non-AMP/OTHERS (Probabilities: 0.9831, 0.0066, 0.0102)\n","Processed MH0206_GL0077457: non-AMP/OTHERS (Probabilities: 0.9820, 0.0071, 0.0110)\n","Processed MH0012_GL0069804: non-AMP/OTHERS (Probabilities: 0.9928, 0.0036, 0.0035)\n","Processed MH0131_GL0076986: non-AMP/OTHERS (Probabilities: 0.8277, 0.0192, 0.1531)\n","Processed O2.UC29-1_GL0162594: non-AMP/OTHERS (Probabilities: 0.9901, 0.0031, 0.0068)\n","Processed MH0107_GL0039912: non-AMP/OTHERS (Probabilities: 0.9909, 0.0028, 0.0063)\n","Processed O2.UC15-1_GL0063817: non-AMP/OTHERS (Probabilities: 0.9906, 0.0028, 0.0066)\n","Processed O2.UC48-0_GL0164587: non-AMP/OTHERS (Probabilities: 0.9747, 0.0091, 0.0162)\n","Processed V1.CD52-0_GL0096148: Neuropeptide (Probabilities: 0.0061, 0.0086, 0.9853)\n","Processed V1.CD52-0_GL0096148: non-AMP/OTHERS (Probabilities: 0.9878, 0.0050, 0.0072)\n","Processed MH0243_GL0069564: non-AMP/OTHERS (Probabilities: 0.9918, 0.0038, 0.0044)\n","Processed 765094712-stool1_revised_scaffold6107_1_gene43525: non-AMP/OTHERS (Probabilities: 0.9894, 0.0029, 0.0077)\n","Processed T2D-109A_GL0157344: Neuropeptide (Probabilities: 0.0113, 0.0101, 0.9786)\n","Processed 160502038-stool1_revised_C1110141_1_gene1223: non-AMP/OTHERS (Probabilities: 0.9896, 0.0040, 0.0064)\n","Processed MH0229_GL0033129: Neuropeptide (Probabilities: 0.1140, 0.0211, 0.8649)\n","Processed MH0020_GL0021820: non-AMP/OTHERS (Probabilities: 0.9917, 0.0037, 0.0047)\n","Processed MH0064_GL0018815: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed MH0113_GL0079842: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9851)\n","Processed MH0150_GL0161143: Neuropeptide (Probabilities: 0.0086, 0.0093, 0.9821)\n","Processed MH0002_GL0018743: Neuropeptide (Probabilities: 0.0060, 0.0093, 0.9847)\n","Processed MH0099_GL0120733: non-AMP/OTHERS (Probabilities: 0.9882, 0.0044, 0.0075)\n","Processed MH0197_GL0110860: non-AMP/OTHERS (Probabilities: 0.9930, 0.0029, 0.0042)\n","Processed MH0011_GL0078407: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9852)\n","Processed 764669880-stool1_revised_C445541_1_gene32191: Neuropeptide (Probabilities: 0.0074, 0.0089, 0.9838)\n","Processed V1.FI34_GL0161608: non-AMP/OTHERS (Probabilities: 0.9923, 0.0028, 0.0050)\n","Processed 160218816-stool1_revised_C874352_1_gene112716: non-AMP/OTHERS (Probabilities: 0.9921, 0.0029, 0.0050)\n","Processed 160218816-stool1_revised_C874352_1_gene112716: non-AMP/OTHERS (Probabilities: 0.9930, 0.0030, 0.0040)\n","Processed MH0293_GL0102897: non-AMP/OTHERS (Probabilities: 0.9930, 0.0029, 0.0042)\n","Processed N035A_GL0009872: non-AMP/OTHERS (Probabilities: 0.9881, 0.0030, 0.0089)\n","Processed MH0062_GL0017613: Neuropeptide (Probabilities: 0.2802, 0.0474, 0.6724)\n","Processed MH0092_GL0020057: non-AMP/OTHERS (Probabilities: 0.9911, 0.0052, 0.0037)\n","Processed sp|Q92211|G3P_CANAW: non-AMP/OTHERS (Probabilities: 0.9892, 0.0036, 0.0072)\n","Processed DOF014_GL0051135: Neuropeptide (Probabilities: 0.0098, 0.0097, 0.9804)\n","Processed 470146.COPCOM_00496: Neuropeptide (Probabilities: 0.2883, 0.0326, 0.6791)\n","Processed T2D-31A_GL0100041: non-AMP/OTHERS (Probabilities: 0.9911, 0.0037, 0.0052)\n","Processed V1.FI33_GL0077973: non-AMP/OTHERS (Probabilities: 0.9885, 0.0029, 0.0086)\n","Processed 763982056-stool2_revised_scaffold4027_1_gene69962: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0037)\n","Processed 763982056-stool2_revised_scaffold4027_1_gene69962: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0034)\n","Processed MH0188_GL0067976: non-AMP/OTHERS (Probabilities: 0.9911, 0.0055, 0.0034)\n","Processed MH0250_GL0268447: Neuropeptide (Probabilities: 0.0171, 0.0129, 0.9700)\n","Processed V1.UC1-0_GL0031408: non-AMP/OTHERS (Probabilities: 0.9908, 0.0056, 0.0036)\n","Processed MH0055_GL0043277: non-AMP/OTHERS (Probabilities: 0.9923, 0.0033, 0.0044)\n","Processed V1.CD27-0_GL0120505: non-AMP/OTHERS (Probabilities: 0.9797, 0.0048, 0.0155)\n","Processed 159915365-stool1_revised_C245139_1_gene10215: non-AMP/OTHERS (Probabilities: 0.9778, 0.0045, 0.0177)\n","Processed DOM010_GL0006099: non-AMP/OTHERS (Probabilities: 0.9932, 0.0030, 0.0038)\n","Processed MH0088_GL0109611: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed O2.UC50-0_GL0215845: Neuropeptide (Probabilities: 0.0107, 0.0101, 0.9793)\n","Processed MH0161_GL0055799: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9852)\n","Processed MH0161_GL0055799: Neuropeptide (Probabilities: 0.0070, 0.0089, 0.9841)\n","Processed V1.UC55-0_GL0151642: Neuropeptide (Probabilities: 0.0074, 0.0093, 0.9833)\n","Processed O2.UC12-1_GL0053606: Neuropeptide (Probabilities: 0.0101, 0.0094, 0.9805)\n","Processed MH0019_GL0013605: Neuropeptide (Probabilities: 0.1367, 0.0302, 0.8331)\n","Processed O2.UC48-1_GL0202419: non-AMP/OTHERS (Probabilities: 0.9904, 0.0035, 0.0061)\n","Processed V1.CD51-0_GL0201913: non-AMP/OTHERS (Probabilities: 0.9905, 0.0039, 0.0056)\n","Processed 762984.HMPREF9445_02509: Neuropeptide (Probabilities: 0.0080, 0.0095, 0.9825)\n","Processed BGI-33A_GL0020094: non-AMP/OTHERS (Probabilities: 0.9558, 0.0134, 0.0308)\n","Processed V1.UC29-0_GL0089797: Neuropeptide (Probabilities: 0.0322, 0.0209, 0.9469)\n","Processed MH0403_GL0038429: Neuropeptide (Probabilities: 0.0093, 0.0099, 0.9808)\n","Processed O2.UC6-1_GL0133356: non-AMP/OTHERS (Probabilities: 0.9655, 0.0065, 0.0280)\n","Processed MH0088_GL0033186: non-AMP/OTHERS (Probabilities: 0.4766, 0.0735, 0.4499)\n","Processed V1.CD17-4_GL0091217: non-AMP/OTHERS (Probabilities: 0.9857, 0.0037, 0.0106)\n","Processed NOM026_GL0076117: Neuropeptide (Probabilities: 0.2201, 0.0416, 0.7382)\n","Processed MH0110_GL0035441: Neuropeptide (Probabilities: 0.3375, 0.0499, 0.6127)\n","Processed MH0329_GL0079140: non-AMP/OTHERS (Probabilities: 0.9732, 0.0169, 0.0099)\n","Processed O2.UC22-1_GL0129375: Neuropeptide (Probabilities: 0.0106, 0.0100, 0.9794)\n","Processed MH0019_GL0028873: non-AMP/OTHERS (Probabilities: 0.9905, 0.0035, 0.0060)\n","Processed MH0006_GL0083870: non-AMP/OTHERS (Probabilities: 0.9918, 0.0037, 0.0045)\n","Processed sp|Q14CN2|CLCA4_HUMAN: Neuropeptide (Probabilities: 0.0227, 0.0173, 0.9600)\n","Processed sp|Q14CN2|CLCA4_HUMAN: non-AMP/OTHERS (Probabilities: 0.6856, 0.1097, 0.2047)\n","Processed O2.UC31-0_GL0026596: non-AMP/OTHERS (Probabilities: 0.9844, 0.0059, 0.0097)\n","Processed MH0161_GL0147055: Neuropeptide (Probabilities: 0.1616, 0.0322, 0.8062)\n","Processed 367928.BAD_0378: non-AMP/OTHERS (Probabilities: 0.9896, 0.0038, 0.0066)\n","Processed MH0383_GL0040621: non-AMP/OTHERS (Probabilities: 0.9933, 0.0036, 0.0031)\n","Processed O2.UC9-2_GL0090528: non-AMP/OTHERS (Probabilities: 0.9176, 0.0528, 0.0296)\n","Processed 404239096-stool1_revised_scaffold26117_2_gene23128: Neuropeptide (Probabilities: 0.0084, 0.0094, 0.9822)\n","Processed MH0184_GL0124020: non-AMP/OTHERS (Probabilities: 0.9934, 0.0035, 0.0031)\n","Processed DOF004_GL0048377: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0036_GL0023903: Neuropeptide (Probabilities: 0.1890, 0.0205, 0.7905)\n","Processed MH0009_GL0034627: non-AMP/OTHERS (Probabilities: 0.9854, 0.0049, 0.0097)\n","Processed SZEY-26A_GL0052978: Neuropeptide (Probabilities: 0.3686, 0.0426, 0.5888)\n","Processed MH0136_GL0027923: non-AMP/OTHERS (Probabilities: 0.7112, 0.0764, 0.2124)\n","Processed 670892.EC3431_2376: non-AMP/OTHERS (Probabilities: 0.9135, 0.0131, 0.0734)\n","Processed MH0211_GL0099673: non-AMP/OTHERS (Probabilities: 0.8510, 0.0473, 0.1017)\n","Processed MH0196_GL0042240: non-AMP/OTHERS (Probabilities: 0.4972, 0.0257, 0.4771)\n","Processed T2D-47A_GL0016721: non-AMP/OTHERS (Probabilities: 0.9923, 0.0041, 0.0036)\n","Processed MH0350_GL0189843: non-AMP/OTHERS (Probabilities: 0.9926, 0.0040, 0.0034)\n","Processed MH0205_GL0093021: non-AMP/OTHERS (Probabilities: 0.9927, 0.0037, 0.0036)\n","Processed O2.UC48-1_GL0202419: Neuropeptide (Probabilities: 0.0806, 0.0208, 0.8986)\n","Processed O2.UC37-1_GL0120283: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0058_GL0077304: Neuropeptide (Probabilities: 0.0076, 0.0094, 0.9830)\n","Processed V1.CD10-0_GL0011536: Neuropeptide (Probabilities: 0.0138, 0.0127, 0.9735)\n","Processed 1042403.BALAC2494_00308: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed O2.UC24-1_GL0140125: Neuropeptide (Probabilities: 0.3324, 0.0262, 0.6415)\n","Processed 159632143-stool1_revised_C901680_1_gene49165: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed 158802708-stool1_revised_C887271_1_gene104864: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0058_GL0065727: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0101_GL0025527: non-AMP/OTHERS (Probabilities: 0.9931, 0.0032, 0.0037)\n","Processed N011A_GL0087204: Neuropeptide (Probabilities: 0.0092, 0.0100, 0.9808)\n","Processed N056A_GL0046831: Neuropeptide (Probabilities: 0.0063, 0.0086, 0.9851)\n","Processed N056A_GL0046831: Neuropeptide (Probabilities: 0.0087, 0.0095, 0.9818)\n","Processed MH0197_GL0014476: non-AMP/OTHERS (Probabilities: 0.9930, 0.0034, 0.0036)\n","Processed MH0277_GL0004866: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9853)\n","Processed MH0188_GL0126874: Neuropeptide (Probabilities: 0.0065, 0.0088, 0.9847)\n","Processed 910311.HMPREF9404_3897: Neuropeptide (Probabilities: 0.0063, 0.0089, 0.9849)\n","Processed NOM026_GL0075750: non-AMP/OTHERS (Probabilities: 0.9872, 0.0096, 0.0033)\n","Processed V1.CD51-0_GL0006732: non-AMP/OTHERS (Probabilities: 0.6862, 0.0296, 0.2842)\n","Processed MH0011_GL0058597: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0233_GL0031149: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0110_GL0006507: Neuropeptide (Probabilities: 0.1239, 0.0263, 0.8498)\n","Processed MH0230_GL0146263: non-AMP/OTHERS (Probabilities: 0.9269, 0.0164, 0.0567)\n","Processed MH0119_GL0024850: non-AMP/OTHERS (Probabilities: 0.5370, 0.0320, 0.4310)\n","Processed sp|P40879|S26A3_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed V1.FI05_GL0134650: Neuropeptide (Probabilities: 0.0084, 0.0106, 0.9809)\n","Processed MH0129_GL0079437: non-AMP/OTHERS (Probabilities: 0.9862, 0.0057, 0.0082)\n","Processed NOM009_GL0110521: non-AMP/OTHERS (Probabilities: 0.9921, 0.0031, 0.0048)\n","Processed MH0086_GL0019990: Neuropeptide (Probabilities: 0.3104, 0.0602, 0.6294)\n","Processed V1.UC26-4_GL0050523: Neuropeptide (Probabilities: 0.1909, 0.0536, 0.7556)\n","Processed MH0003_GL0096360: non-AMP/OTHERS (Probabilities: 0.9932, 0.0029, 0.0038)\n","Processed MH0188_GL0007184: non-AMP/OTHERS (Probabilities: 0.9926, 0.0030, 0.0044)\n","Processed ED12A_GL0081079: non-AMP/OTHERS (Probabilities: 0.9925, 0.0029, 0.0045)\n","Processed N025A_GL0032541: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed MH0211_GL0167950: non-AMP/OTHERS (Probabilities: 0.9575, 0.0250, 0.0174)\n","Processed MH0193_GL0184676: non-AMP/OTHERS (Probabilities: 0.9829, 0.0103, 0.0068)\n","Processed MH0074_GL0044846: non-AMP/OTHERS (Probabilities: 0.9881, 0.0045, 0.0074)\n","Processed MH0277_GL0046340: non-AMP/OTHERS (Probabilities: 0.9908, 0.0035, 0.0057)\n","Processed T2D-26A_GL0024418: Neuropeptide (Probabilities: 0.0121, 0.0099, 0.9781)\n","Processed O2.UC48-0_GL0123343: Neuropeptide (Probabilities: 0.4317, 0.0228, 0.5455)\n","Processed O2.UC11-1_GL0033944: non-AMP/OTHERS (Probabilities: 0.6837, 0.0219, 0.2944)\n","Processed MH0199_GL0034774: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9853)\n","Processed MH0193_GL0140286: non-AMP/OTHERS (Probabilities: 0.8917, 0.0872, 0.0211)\n","Processed 473819.BIFDEN_02100: Neuropeptide (Probabilities: 0.0153, 0.0118, 0.9729)\n","Processed MH0117_GL0042427: Neuropeptide (Probabilities: 0.0063, 0.0086, 0.9851)\n","Processed V1.CD25-4_GL0011492: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0035)\n","Processed MH0103_GL0091658: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed MH0445_GL0156284: non-AMP/OTHERS (Probabilities: 0.9896, 0.0045, 0.0059)\n","Processed MH0011_GL0028073: Neuropeptide (Probabilities: 0.1242, 0.0349, 0.8409)\n","Processed MH0011_GL0028073: Neuropeptide (Probabilities: 0.3681, 0.0555, 0.5764)\n","Processed MH0002_GL0071794: non-AMP/OTHERS (Probabilities: 0.9881, 0.0086, 0.0034)\n","Processed N047A_GL0035390: Neuropeptide (Probabilities: 0.0358, 0.0180, 0.9462)\n","Processed N047A_GL0035390: non-AMP/OTHERS (Probabilities: 0.8030, 0.0484, 0.1487)\n","Processed 657315.ROI_06730: non-AMP/OTHERS (Probabilities: 0.9892, 0.0078, 0.0030)\n","Processed MH0275_GL0149996: non-AMP/OTHERS (Probabilities: 0.9918, 0.0040, 0.0042)\n","Processed MH0069_GL0071776: non-AMP/OTHERS (Probabilities: 0.9884, 0.0083, 0.0032)\n","Processed MH0188_GL0168121: non-AMP/OTHERS (Probabilities: 0.9866, 0.0094, 0.0040)\n","Processed MH0058_GL0056069: non-AMP/OTHERS (Probabilities: 0.9894, 0.0070, 0.0036)\n","Processed DOM003_GL0005454: non-AMP/OTHERS (Probabilities: 0.9853, 0.0116, 0.0031)\n","Processed MH0358_GL0022108: non-AMP/OTHERS (Probabilities: 0.9926, 0.0041, 0.0034)\n","Processed O2.UC17-1_GL0007264: non-AMP/OTHERS (Probabilities: 0.9927, 0.0036, 0.0037)\n","Processed MH0244_GL0046654: non-AMP/OTHERS (Probabilities: 0.9925, 0.0033, 0.0042)\n","Processed V1.FI14_GL0250005: non-AMP/OTHERS (Probabilities: 0.9925, 0.0034, 0.0041)\n","Processed 888833.HMPREF9961_1930: Neuropeptide (Probabilities: 0.3098, 0.0279, 0.6623)\n","Processed MH0089_GL0070884: non-AMP/OTHERS (Probabilities: 0.9740, 0.0119, 0.0140)\n","Processed MH0092_GL0074084: Neuropeptide (Probabilities: 0.0177, 0.0120, 0.9703)\n","Processed 763577454-stool2_revised_C997474_1_gene74487: Neuropeptide (Probabilities: 0.3867, 0.0231, 0.5902)\n","Processed MH0176_GL0056810: non-AMP/OTHERS (Probabilities: 0.9352, 0.0102, 0.0546)\n","Processed MH0176_GL0056810: non-AMP/OTHERS (Probabilities: 0.9906, 0.0043, 0.0051)\n","Processed MH0001_GL0003214: non-AMP/OTHERS (Probabilities: 0.9901, 0.0031, 0.0068)\n","Processed MH0150_GL0011745: Neuropeptide (Probabilities: 0.4074, 0.0328, 0.5598)\n","Processed DLM014_GL0004997: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0011_GL0105294: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0088_GL0056851: non-AMP/OTHERS (Probabilities: 0.9921, 0.0037, 0.0043)\n","Processed NOM025_GL0019567: Neuropeptide (Probabilities: 0.0086, 0.0093, 0.9821)\n","Processed BGI-28A_GL0104596: non-AMP/OTHERS (Probabilities: 0.9581, 0.0161, 0.0259)\n","Processed MH0119_GL0106651: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0038)\n","Processed MH0122_GL0028871: non-AMP/OTHERS (Probabilities: 0.6873, 0.0274, 0.2854)\n","Processed MH0337_GL0109696: non-AMP/OTHERS (Probabilities: 0.9909, 0.0030, 0.0060)\n","Processed MH0292_GL0067872: non-AMP/OTHERS (Probabilities: 0.8382, 0.0415, 0.1203)\n","Processed MH0026_GL0043838: Neuropeptide (Probabilities: 0.3412, 0.1611, 0.4977)\n","Processed DOF013_GL0053897: Neuropeptide (Probabilities: 0.0068, 0.0090, 0.9843)\n","Processed V1.FI31_GL0135429: Neuropeptide (Probabilities: 0.0882, 0.0608, 0.8510)\n","Processed MH0006_GL0024198: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed V1.UC58-4_GL0041455: Neuropeptide (Probabilities: 0.0258, 0.0206, 0.9536)\n","Processed MH0003_GL0029363: non-AMP/OTHERS (Probabilities: 0.9921, 0.0036, 0.0043)\n","Processed V1.CD53-0_GL0038153: Neuropeptide (Probabilities: 0.0130, 0.0132, 0.9738)\n","Processed 160704339-stool1_revised_scaffold7296_1_gene26176: non-AMP/OTHERS (Probabilities: 0.9847, 0.0092, 0.0061)\n","Processed MH0370_GL0042621: non-AMP/OTHERS (Probabilities: 0.9070, 0.0301, 0.0628)\n","Processed DLM022_GL0024109: Neuropeptide (Probabilities: 0.0607, 0.0391, 0.9002)\n","Processed MH0402_GL0024006: Neuropeptide (Probabilities: 0.1722, 0.0520, 0.7757)\n","Processed MH0196_GL0067754: Neuropeptide (Probabilities: 0.0068, 0.0086, 0.9846)\n","Processed MH0004_GL0007700: non-AMP/OTHERS (Probabilities: 0.9882, 0.0045, 0.0073)\n","Processed MH0191_GL0137920: Neuropeptide (Probabilities: 0.0562, 0.0381, 0.9058)\n","Processed MH0007_GL0044520: non-AMP/OTHERS (Probabilities: 0.9918, 0.0036, 0.0046)\n","Processed MH0313_GL0148476: non-AMP/OTHERS (Probabilities: 0.9914, 0.0036, 0.0050)\n","Processed MH0074_GL0014631: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed 159571453-stool2_revised_C1174584_1_gene39013: Neuropeptide (Probabilities: 0.0775, 0.0607, 0.8619)\n","Processed V1.UC4-0_GL0069741: Neuropeptide (Probabilities: 0.2283, 0.0948, 0.6769)\n","Processed MH0067_GL0032307: Neuropeptide (Probabilities: 0.1725, 0.0889, 0.7387)\n","Processed MH0006_GL0133694: non-AMP/OTHERS (Probabilities: 0.5989, 0.1830, 0.2181)\n","Processed MH0002_GL0071847: Neuropeptide (Probabilities: 0.0127, 0.0120, 0.9753)\n","Processed 435590.BVU_1157: Neuropeptide (Probabilities: 0.0076, 0.0092, 0.9833)\n","Processed T2D-25A_GL0081229: non-AMP/OTHERS (Probabilities: 0.9886, 0.0038, 0.0076)\n","Processed MH0389_GL0175840: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed MH0068_GL0000252: non-AMP/OTHERS (Probabilities: 0.9522, 0.0240, 0.0238)\n","Processed T2D-41A_GL0008713: Neuropeptide (Probabilities: 0.0087, 0.0104, 0.9809)\n","Processed MH0011_GL0057108: Neuropeptide (Probabilities: 0.0889, 0.0663, 0.8447)\n","Processed MH0212_GL0074922: non-AMP/OTHERS (Probabilities: 0.7309, 0.0528, 0.2162)\n","Processed NLM008_GL0027897: non-AMP/OTHERS (Probabilities: 0.9911, 0.0040, 0.0049)\n","Processed MH0006_GL0224905: non-AMP/OTHERS (Probabilities: 0.9448, 0.0166, 0.0386)\n","Processed O2.UC22-1_GL0047403: non-AMP/OTHERS (Probabilities: 0.8932, 0.0165, 0.0903)\n","Processed 537013.CLOSTMETH_03248: non-AMP/OTHERS (Probabilities: 0.9919, 0.0038, 0.0043)\n","Processed V1.UC5-3_GL0054181: non-AMP/OTHERS (Probabilities: 0.9923, 0.0045, 0.0032)\n","Processed MH0136_GL0027923: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed V1.FI37_GL0044600: non-AMP/OTHERS (Probabilities: 0.8922, 0.0212, 0.0867)\n","Processed O2.UC22-1_GL0016243: AMP (Probabilities: 0.1152, 0.8690, 0.0158)\n","Processed MH0443_GL0134242: Neuropeptide (Probabilities: 0.0062, 0.0087, 0.9851)\n","Processed NLM008_GL0027897: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed O2.UC36-0_GL0086723: Neuropeptide (Probabilities: 0.0067, 0.0087, 0.9846)\n","Processed MH0043_GL0032446: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9852)\n","Processed MH0028_GL0031124: non-AMP/OTHERS (Probabilities: 0.9917, 0.0043, 0.0040)\n","Processed MH0244_GL0062774: non-AMP/OTHERS (Probabilities: 0.6795, 0.0154, 0.3051)\n","Processed MH0161_GL0160910: Neuropeptide (Probabilities: 0.0061, 0.0088, 0.9851)\n","Processed MH0286_GL0114879: Neuropeptide (Probabilities: 0.0082, 0.0090, 0.9828)\n","Processed V1.UC58-0_GL0004917: non-AMP/OTHERS (Probabilities: 0.9514, 0.0082, 0.0404)\n","Processed MH0233_GL0122560: Neuropeptide (Probabilities: 0.0444, 0.0227, 0.9330)\n","Processed O2.UC23-2_GL0043224: non-AMP/OTHERS (Probabilities: 0.9831, 0.0074, 0.0095)\n","Processed MH0002_GL0018165: Neuropeptide (Probabilities: 0.0108, 0.0106, 0.9786)\n","Processed V1.CD12-0_GL0001661: non-AMP/OTHERS (Probabilities: 0.9896, 0.0037, 0.0066)\n","Processed O2.UC17-1_GL0086258: non-AMP/OTHERS (Probabilities: 0.9793, 0.0045, 0.0162)\n","Processed MH0233_GL0084239: Neuropeptide (Probabilities: 0.0062, 0.0087, 0.9851)\n","Processed O2.UC23-2_GL0153123: non-AMP/OTHERS (Probabilities: 0.9913, 0.0045, 0.0041)\n","Processed MH0442_GL0126331: non-AMP/OTHERS (Probabilities: 0.9583, 0.0061, 0.0355)\n","Processed V1.FI30_GL0091785: non-AMP/OTHERS (Probabilities: 0.9923, 0.0038, 0.0039)\n","Processed N085A_GL0017046: non-AMP/OTHERS (Probabilities: 0.9921, 0.0043, 0.0036)\n","Processed O2.UC27-1_GL0024439: non-AMP/OTHERS (Probabilities: 0.9683, 0.0082, 0.0234)\n","Processed V1.CD22-0_GL0034362: non-AMP/OTHERS (Probabilities: 0.9920, 0.0033, 0.0046)\n","Processed N085A_GL0021877: non-AMP/OTHERS (Probabilities: 0.9933, 0.0030, 0.0037)\n","Processed sp|P08236|BGLR_HUMAN: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9850)\n","Processed MH0089_GL0041666: Neuropeptide (Probabilities: 0.0391, 0.0232, 0.9377)\n","Processed N074A_GL0022313: non-AMP/OTHERS (Probabilities: 0.9877, 0.0056, 0.0067)\n","Processed MH0131_GL0068396: Neuropeptide (Probabilities: 0.0069, 0.0088, 0.9843)\n","Processed MH0211_GL0163373: Neuropeptide (Probabilities: 0.0068, 0.0088, 0.9844)\n","Processed V1.FI26_GL0205845: Neuropeptide (Probabilities: 0.3557, 0.0235, 0.6208)\n","Processed 717962.CC1_29610: non-AMP/OTHERS (Probabilities: 0.9892, 0.0068, 0.0040)\n","Processed V1.UC25-0_GL0051029: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed NLF006_GL0026981: non-AMP/OTHERS (Probabilities: 0.7328, 0.0220, 0.2452)\n","Processed O2.UC48-0_GL0030973: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0006_GL0033117: non-AMP/OTHERS (Probabilities: 0.9935, 0.0029, 0.0035)\n","Processed O2.UC36-0_GL0066790: non-AMP/OTHERS (Probabilities: 0.9912, 0.0040, 0.0048)\n","Processed MH0012_GL0145002: Neuropeptide (Probabilities: 0.4709, 0.0368, 0.4924)\n","Processed O2.CD3-0-PT_GL0142447: non-AMP/OTHERS (Probabilities: 0.9899, 0.0046, 0.0054)\n","Processed NOM015_GL0059565: Neuropeptide (Probabilities: 0.2486, 0.0376, 0.7138)\n","Processed MH0341_GL0102431: non-AMP/OTHERS (Probabilities: 0.8482, 0.0363, 0.1155)\n","Processed MH0056_GL0008736: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed T2D-6A_GL0105380: non-AMP/OTHERS (Probabilities: 0.9607, 0.0058, 0.0335)\n","Processed 411477.PARMER_03867: non-AMP/OTHERS (Probabilities: 0.8767, 0.0139, 0.1094)\n","Processed MH0087_GL0013022: non-AMP/OTHERS (Probabilities: 0.9393, 0.0497, 0.0110)\n","Processed MH0178_GL0049744: non-AMP/OTHERS (Probabilities: 0.9910, 0.0044, 0.0046)\n","Processed MH0002_GL0039500: Neuropeptide (Probabilities: 0.0908, 0.0292, 0.8800)\n","Processed MH0333_GL0092661: Neuropeptide (Probabilities: 0.2685, 0.0580, 0.6735)\n","Processed 717962.CC1_16390: non-AMP/OTHERS (Probabilities: 0.6749, 0.0867, 0.2384)\n","Processed MH0011_GL0055430: Neuropeptide (Probabilities: 0.0058, 0.0092, 0.9850)\n","Processed MH0298_GL0040531: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed NLM010_GL0033811: Neuropeptide (Probabilities: 0.0069, 0.0088, 0.9844)\n","Processed MH0288_GL0148802: Neuropeptide (Probabilities: 0.0066, 0.0088, 0.9845)\n","Processed MH0014_GL0090940: non-AMP/OTHERS (Probabilities: 0.9312, 0.0229, 0.0459)\n","Processed MH0179_GL0045930: non-AMP/OTHERS (Probabilities: 0.9874, 0.0048, 0.0077)\n","Processed 1042403.BALAC2494_00679: Neuropeptide (Probabilities: 0.3292, 0.0354, 0.6354)\n","Processed MH0440_GL0054051: non-AMP/OTHERS (Probabilities: 0.6908, 0.0276, 0.2816)\n","Processed MH0002_GL0056756: Neuropeptide (Probabilities: 0.0167, 0.0134, 0.9700)\n","Processed MH0438_GL0138082: non-AMP/OTHERS (Probabilities: 0.9917, 0.0035, 0.0048)\n","Processed MH0004_GL0012914: Neuropeptide (Probabilities: 0.0060, 0.0090, 0.9850)\n","Processed MH0004_GL0012914: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9850)\n","Processed MH0011_GL0100221: non-AMP/OTHERS (Probabilities: 0.9600, 0.0361, 0.0039)\n","Processed MH0088_GL0111634: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0193_GL0163590: Neuropeptide (Probabilities: 0.3667, 0.0391, 0.5942)\n","Processed MH0211_GL0170472: Neuropeptide (Probabilities: 0.0993, 0.0293, 0.8715)\n","Processed MH0193_GL0142950: Neuropeptide (Probabilities: 0.0107, 0.0104, 0.9788)\n","Processed MH0104_GL0021386: non-AMP/OTHERS (Probabilities: 0.9744, 0.0044, 0.0212)\n","Processed MH0441_GL0134139: non-AMP/OTHERS (Probabilities: 0.9475, 0.0139, 0.0386)\n","Processed MH0069_GL0080003: Neuropeptide (Probabilities: 0.0120, 0.0118, 0.9762)\n","Processed DLM001_GL0098804: Neuropeptide (Probabilities: 0.0301, 0.0153, 0.9546)\n","Processed T2D-146A_GL0038706: Neuropeptide (Probabilities: 0.0099, 0.0101, 0.9800)\n","Processed DOF008_GL0044701: non-AMP/OTHERS (Probabilities: 0.9910, 0.0032, 0.0058)\n","Processed MH0100_GL0011223: non-AMP/OTHERS (Probabilities: 0.9926, 0.0031, 0.0043)\n","Processed MH0275_GL0066122: Neuropeptide (Probabilities: 0.1893, 0.0226, 0.7881)\n","Processed V1.UC57-0_GL0137313: non-AMP/OTHERS (Probabilities: 0.9933, 0.0035, 0.0033)\n","Processed MH0350_GL0153085: non-AMP/OTHERS (Probabilities: 0.9682, 0.0101, 0.0217)\n","Processed MH0131_GL0110091: non-AMP/OTHERS (Probabilities: 0.6349, 0.0425, 0.3225)\n","Processed MH0030_GL0029779: non-AMP/OTHERS (Probabilities: 0.9916, 0.0034, 0.0050)\n","Processed MH0002_GL0014846: Neuropeptide (Probabilities: 0.0073, 0.0090, 0.9837)\n","Processed MH0010_GL0001659: Neuropeptide (Probabilities: 0.2597, 0.0459, 0.6944)\n","Processed sp|Q10588|BST1_HUMAN: Neuropeptide (Probabilities: 0.0144, 0.0132, 0.9725)\n","Processed MH0432_GL0170651: non-AMP/OTHERS (Probabilities: 0.5558, 0.0839, 0.3604)\n","Processed MH0275_GL0049176: Neuropeptide (Probabilities: 0.0241, 0.0165, 0.9594)\n","Processed V1.UC53-0_GL0058779: non-AMP/OTHERS (Probabilities: 0.9908, 0.0034, 0.0058)\n","Processed MH0345_GL0171329: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0131_GL0079918: Neuropeptide (Probabilities: 0.0079, 0.0090, 0.9831)\n","Processed MH0003_GL0030545: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed MH0002_GL0063069: non-AMP/OTHERS (Probabilities: 0.9924, 0.0040, 0.0035)\n","Processed NOF005_GL0001797: non-AMP/OTHERS (Probabilities: 0.9929, 0.0036, 0.0035)\n","Processed MH0003_GL0048460: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9853)\n","Processed MH0129_GL0097175: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0035)\n","Processed O2.UC22-1_GL0022393: Neuropeptide (Probabilities: 0.4426, 0.0264, 0.5310)\n","Processed MH0230_GL0083200: Neuropeptide (Probabilities: 0.0089, 0.0094, 0.9817)\n","Processed MH0401_GL0119233: Neuropeptide (Probabilities: 0.0069, 0.0087, 0.9844)\n","Processed MH0314_GL0151298: Neuropeptide (Probabilities: 0.0545, 0.0242, 0.9213)\n","Processed O2.UC20-2_GL0039172: Neuropeptide (Probabilities: 0.0254, 0.0168, 0.9577)\n","Processed O2.UC20-2_GL0039172: Neuropeptide (Probabilities: 0.3187, 0.0552, 0.6262)\n","Processed V1.CD6-0-PT_GL0182210: non-AMP/OTHERS (Probabilities: 0.5599, 0.0532, 0.3870)\n","Processed V1.CD6-0-PT_GL0182210: Neuropeptide (Probabilities: 0.0865, 0.0391, 0.8743)\n","Processed N029A_GL0049168: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed MH0185_GL0142512: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed O2.UC12-2_GL0049373: non-AMP/OTHERS (Probabilities: 0.6523, 0.2527, 0.0949)\n","Processed MH0337_GL0135190: Neuropeptide (Probabilities: 0.0781, 0.0536, 0.8684)\n","Processed MH0099_GL0080558: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed V1.UC33-0_GL0101279: Neuropeptide (Probabilities: 0.0071, 0.0089, 0.9841)\n","Processed DOM010_GL0012419: Neuropeptide (Probabilities: 0.0095, 0.0092, 0.9812)\n","Processed MH0021_GL0011245: non-AMP/OTHERS (Probabilities: 0.9615, 0.0127, 0.0257)\n","Processed O2.UC48-0_GL0103416: non-AMP/OTHERS (Probabilities: 0.9871, 0.0042, 0.0087)\n","Processed HT14A_GL0106301: non-AMP/OTHERS (Probabilities: 0.6021, 0.0325, 0.3654)\n","Processed V1.FI17_GL0119839: non-AMP/OTHERS (Probabilities: 0.9768, 0.0069, 0.0163)\n","Processed V1.UC47-4_GL0146516: non-AMP/OTHERS (Probabilities: 0.9901, 0.0036, 0.0063)\n","Processed O2.UC27-1_GL0024439: Neuropeptide (Probabilities: 0.1581, 0.0264, 0.8155)\n","Processed MH0188_GL0022128: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed DOF009_GL0050886: non-AMP/OTHERS (Probabilities: 0.9917, 0.0034, 0.0048)\n","Processed MH0170_GL0102091: Neuropeptide (Probabilities: 0.4478, 0.0434, 0.5088)\n","Processed MH0277_GL0034532: non-AMP/OTHERS (Probabilities: 0.9928, 0.0040, 0.0032)\n","Processed MH0133_GL0006923: non-AMP/OTHERS (Probabilities: 0.9861, 0.0062, 0.0077)\n","Processed T2D-62A_GL0051664: non-AMP/OTHERS (Probabilities: 0.9926, 0.0043, 0.0031)\n","Processed 764285508-stool1_revised_C994638_1_gene39117: non-AMP/OTHERS (Probabilities: 0.9930, 0.0038, 0.0032)\n","Processed MH0011_GL0011557: non-AMP/OTHERS (Probabilities: 0.9069, 0.0156, 0.0774)\n","Processed MH0239_GL0035583: non-AMP/OTHERS (Probabilities: 0.9927, 0.0031, 0.0042)\n","Processed sp|P09651|ROA1_HUMAN: non-AMP/OTHERS (Probabilities: 0.9889, 0.0056, 0.0055)\n","Processed 764447348-stool2_revised_scaffold27644_2_gene114884: non-AMP/OTHERS (Probabilities: 0.6654, 0.0306, 0.3040)\n","Processed sp|P0DTE7|AMY1B_HUMAN: non-AMP/OTHERS (Probabilities: 0.9782, 0.0065, 0.0154)\n","Processed sp|P04746|AMYP_HUMAN: non-AMP/OTHERS (Probabilities: 0.9768, 0.0069, 0.0162)\n","Processed T2D-29A_GL0106551: non-AMP/OTHERS (Probabilities: 0.4981, 0.0682, 0.4337)\n","Processed MH0329_GL0079140: Neuropeptide (Probabilities: 0.0070, 0.0088, 0.9841)\n","Processed MH0030_GL0016264: Neuropeptide (Probabilities: 0.1371, 0.0315, 0.8314)\n","Processed O2.UC40-2_GL0098045: Neuropeptide (Probabilities: 0.0065, 0.0086, 0.9849)\n","Processed 457394.BSFG_01024: non-AMP/OTHERS (Probabilities: 0.9861, 0.0042, 0.0097)\n","Processed 457394.BSFG_01024: non-AMP/OTHERS (Probabilities: 0.9926, 0.0036, 0.0038)\n","Processed T2D-135A_GL0058455: non-AMP/OTHERS (Probabilities: 0.9919, 0.0040, 0.0041)\n","Processed V1.UC39-0_GL0110075: non-AMP/OTHERS (Probabilities: 0.9929, 0.0034, 0.0037)\n","Processed MH0005_GL0006544: non-AMP/OTHERS (Probabilities: 0.9925, 0.0043, 0.0031)\n","Processed MH0217_GL0075646: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0083_GL0000785: non-AMP/OTHERS (Probabilities: 0.9626, 0.0091, 0.0283)\n","Processed MH0083_GL0000785: non-AMP/OTHERS (Probabilities: 0.8035, 0.0287, 0.1677)\n","Processed MH0062_GL0032426: non-AMP/OTHERS (Probabilities: 0.9898, 0.0044, 0.0058)\n","Processed MH0161_GL0032808: non-AMP/OTHERS (Probabilities: 0.9761, 0.0130, 0.0109)\n","Processed MH0109_GL0013042: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0006_GL0211764: Neuropeptide (Probabilities: 0.0068, 0.0085, 0.9847)\n","Processed V1.UC54-4_GL0135518: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed MH0301_GL0106172: Neuropeptide (Probabilities: 0.4756, 0.0431, 0.4813)\n","Processed O2.UC37-1_GL0147271: non-AMP/OTHERS (Probabilities: 0.9479, 0.0195, 0.0326)\n","Processed MH0417_GL0137304: non-AMP/OTHERS (Probabilities: 0.7472, 0.0399, 0.2130)\n","Processed MH0337_GL0144236: non-AMP/OTHERS (Probabilities: 0.7474, 0.0685, 0.1842)\n","Processed MH0347_GL0014331: Neuropeptide (Probabilities: 0.3285, 0.0348, 0.6367)\n","Processed MH0305_GL0031217: non-AMP/OTHERS (Probabilities: 0.5710, 0.0854, 0.3436)\n","Processed MH0003_GL0000250: Neuropeptide (Probabilities: 0.0748, 0.0272, 0.8980)\n","Processed MH0068_GL0040050: Neuropeptide (Probabilities: 0.0058, 0.0090, 0.9852)\n","Processed 764062976-stool1_revised_C1217437_1_gene39662: Neuropeptide (Probabilities: 0.0059, 0.0087, 0.9853)\n","Processed 764224817-stool1_revised_scaffold43840_1_gene150135: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed 160765029-stool1_revised_C1101688_1_gene189603: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0371_GL0090830: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed 547043.BIFPSEUDO_03536: non-AMP/OTHERS (Probabilities: 0.6968, 0.0360, 0.2672)\n","Processed sp|P20073|ANXA7_HUMAN: non-AMP/OTHERS (Probabilities: 0.9924, 0.0031, 0.0045)\n","Processed MH0143_GL0111571: Neuropeptide (Probabilities: 0.0075, 0.0087, 0.9838)\n","Processed MH0060_GL0030354: Neuropeptide (Probabilities: 0.0066, 0.0086, 0.9848)\n","Processed 763860675-stool2_revised_scaffold56423_1_gene25742: Neuropeptide (Probabilities: 0.0071, 0.0086, 0.9843)\n","Processed V1.FI04_GL0070936: non-AMP/OTHERS (Probabilities: 0.9922, 0.0037, 0.0041)\n","Processed MH0393_GL0125746: non-AMP/OTHERS (Probabilities: 0.9926, 0.0037, 0.0037)\n","Processed MH0133_GL0011222: non-AMP/OTHERS (Probabilities: 0.9718, 0.0143, 0.0140)\n","Processed O2.UC52-0_GL0101885: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9852)\n","Processed O2.UC27-0_GL0120410: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0150_GL0036777: non-AMP/OTHERS (Probabilities: 0.9452, 0.0181, 0.0367)\n","Processed O2.UC7-0_GL0141880: non-AMP/OTHERS (Probabilities: 0.7288, 0.0581, 0.2131)\n","Processed MH0284_GL0153103: non-AMP/OTHERS (Probabilities: 0.9561, 0.0112, 0.0327)\n","Processed MH0011_GL0080339: Neuropeptide (Probabilities: 0.0764, 0.0367, 0.8869)\n","Processed MH0088_GL0045946: non-AMP/OTHERS (Probabilities: 0.9909, 0.0036, 0.0056)\n","Processed O2.UC40-0_GL0130413: non-AMP/OTHERS (Probabilities: 0.6976, 0.0320, 0.2705)\n","Processed N006A_GL0021905: non-AMP/OTHERS (Probabilities: 0.8493, 0.0200, 0.1307)\n","Processed 763034.HMPREF9446_03768: Neuropeptide (Probabilities: 0.0061, 0.0087, 0.9852)\n","Processed 159268001-stool2_revised_scaffold17730_1_gene18950: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed MH0016_GL0068437: Neuropeptide (Probabilities: 0.1212, 0.1266, 0.7522)\n","Processed MH0089_GL0014266: Neuropeptide (Probabilities: 0.0074, 0.0091, 0.9834)\n","Processed MH0244_GL0072054: Neuropeptide (Probabilities: 0.1663, 0.0328, 0.8009)\n","Processed O2.UC13-1_GL0161182: Neuropeptide (Probabilities: 0.0104, 0.0097, 0.9800)\n","Processed O2.UC13-1_GL0161182: non-AMP/OTHERS (Probabilities: 0.8031, 0.0410, 0.1559)\n","Processed MH0230_GL0109570: non-AMP/OTHERS (Probabilities: 0.8584, 0.1170, 0.0246)\n","Processed MH0239_GL0147745: Neuropeptide (Probabilities: 0.0058, 0.0089, 0.9852)\n","Processed MH0275_GL0117295: non-AMP/OTHERS (Probabilities: 0.9869, 0.0061, 0.0070)\n","Processed MH0277_GL0047239: Neuropeptide (Probabilities: 0.0774, 0.0183, 0.9044)\n","Processed T2D-135A_GL0072427: non-AMP/OTHERS (Probabilities: 0.9774, 0.0107, 0.0118)\n","Processed MH0016_GL0004296: non-AMP/OTHERS (Probabilities: 0.9298, 0.0337, 0.0365)\n","Processed MH0161_GL0057640: non-AMP/OTHERS (Probabilities: 0.9922, 0.0039, 0.0039)\n","Processed MH0192_GL0102794: AMP (Probabilities: 0.1417, 0.8453, 0.0130)\n","Processed 764669880-stool1_revised_C436697_1_gene18828: non-AMP/OTHERS (Probabilities: 0.5768, 0.0488, 0.3744)\n","Processed MH0318_GL0107296: non-AMP/OTHERS (Probabilities: 0.9655, 0.0121, 0.0224)\n","Processed O2.UC22-1_GL0006065: Neuropeptide (Probabilities: 0.2540, 0.1100, 0.6361)\n","Processed MH0126_GL0033083: non-AMP/OTHERS (Probabilities: 0.9926, 0.0032, 0.0042)\n","Processed MH0211_GL0039182: non-AMP/OTHERS (Probabilities: 0.8727, 0.0501, 0.0771)\n","Processed O2.UC15-1_GL0034574: non-AMP/OTHERS (Probabilities: 0.9536, 0.0352, 0.0113)\n","Processed 763840445-stool2_revised_C947629_1_gene54091: non-AMP/OTHERS (Probabilities: 0.4919, 0.2335, 0.2746)\n","Processed MH0161_GL0082654: non-AMP/OTHERS (Probabilities: 0.9502, 0.0370, 0.0128)\n","Processed MH0444_GL0038916: non-AMP/OTHERS (Probabilities: 0.9787, 0.0163, 0.0050)\n","Processed O2.UC53-0_GL0131589: non-AMP/OTHERS (Probabilities: 0.9921, 0.0044, 0.0035)\n","Processed MH0131_GL0158205: non-AMP/OTHERS (Probabilities: 0.6406, 0.0259, 0.3335)\n","Processed MH0337_GL0117100: Neuropeptide (Probabilities: 0.0191, 0.0159, 0.9649)\n","Processed 158499257-stool2_revised_C1255796_1_gene205707: Neuropeptide (Probabilities: 0.0065, 0.0089, 0.9846)\n","Processed MH0229_GL0005604: AMP (Probabilities: 0.3223, 0.6446, 0.0331)\n","Processed MH0086_GL0037948: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed MH0002_GL0069115: Neuropeptide (Probabilities: 0.0059, 0.0092, 0.9850)\n","Processed MH0103_GL0013757: non-AMP/OTHERS (Probabilities: 0.9757, 0.0070, 0.0173)\n","Processed 160502038-stool1_revised_scaffold9377_1_gene139584: non-AMP/OTHERS (Probabilities: 0.9908, 0.0031, 0.0060)\n","Processed 210007.SMU_1200: Neuropeptide (Probabilities: 0.0072, 0.0087, 0.9842)\n","Processed MH0099_GL0032698: non-AMP/OTHERS (Probabilities: 0.5700, 0.1227, 0.3072)\n","Processed 546273.VEIDISOL_01258: non-AMP/OTHERS (Probabilities: 0.7228, 0.1190, 0.1581)\n","Processed MH0128_GL0012238: Neuropeptide (Probabilities: 0.0067, 0.0089, 0.9844)\n","Processed MH0244_GL0029239: non-AMP/OTHERS (Probabilities: 0.9907, 0.0051, 0.0042)\n","Processed MH0128_GL0012238: non-AMP/OTHERS (Probabilities: 0.5512, 0.0853, 0.3635)\n","Processed MH0144_GL0136978: non-AMP/OTHERS (Probabilities: 0.9901, 0.0050, 0.0049)\n","Processed MH0144_GL0136978: non-AMP/OTHERS (Probabilities: 0.9909, 0.0044, 0.0047)\n","Processed MH0008_GL0019487: non-AMP/OTHERS (Probabilities: 0.5863, 0.0762, 0.3375)\n","Processed DLF014_GL0007698: Neuropeptide (Probabilities: 0.0155, 0.0136, 0.9710)\n","Processed DOF004_GL0080794: Neuropeptide (Probabilities: 0.0090, 0.0091, 0.9820)\n","Processed MH0272_GL0149637: non-AMP/OTHERS (Probabilities: 0.9908, 0.0063, 0.0029)\n","Processed MH0221_GL0018859: non-AMP/OTHERS (Probabilities: 0.9898, 0.0077, 0.0025)\n","Processed V1.FI02_GL0135168: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0039)\n","Processed 159207311-stool2_revised_scaffold31365_1_gene17238: Neuropeptide (Probabilities: 0.0667, 0.0360, 0.8973)\n","Processed MH0012_GL0106453: Neuropeptide (Probabilities: 0.0197, 0.0142, 0.9661)\n","Processed O2.UC17-1_GL0129703: Neuropeptide (Probabilities: 0.0252, 0.0167, 0.9581)\n","Processed MH0277_GL0030036: non-AMP/OTHERS (Probabilities: 0.5095, 0.0247, 0.4658)\n","Processed T2D-11A_GL0086067: non-AMP/OTHERS (Probabilities: 0.6490, 0.0286, 0.3224)\n","Processed O2.UC22-1_GL0005981: non-AMP/OTHERS (Probabilities: 0.9837, 0.0101, 0.0062)\n","Processed MH0012_GL0176248: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed V1.FI37_GL0035669: Neuropeptide (Probabilities: 0.0088, 0.0091, 0.9821)\n","Processed O2.UC31-0_GL0080980: Neuropeptide (Probabilities: 0.0564, 0.0264, 0.9172)\n","Processed MH0200_GL0147938: non-AMP/OTHERS (Probabilities: 0.7304, 0.0705, 0.1991)\n","Processed MH0452_GL0028253: non-AMP/OTHERS (Probabilities: 0.9901, 0.0054, 0.0045)\n","Processed O2.CD3-0-PT_GL0165637: non-AMP/OTHERS (Probabilities: 0.9929, 0.0029, 0.0042)\n","Processed MH0277_GL0035625: non-AMP/OTHERS (Probabilities: 0.9924, 0.0031, 0.0045)\n","Processed O2.UC17-1_GL0056989: Neuropeptide (Probabilities: 0.2890, 0.0713, 0.6396)\n","Processed V1.CD46-0_GL0087852: Neuropeptide (Probabilities: 0.0161, 0.0117, 0.9722)\n","Processed O2.UC54-0_GL0090310: Neuropeptide (Probabilities: 0.0064, 0.0088, 0.9848)\n","Processed 160400887-stool1_revised_C1533145_1_gene192223: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9850)\n","Processed DOF004_GL0050807: non-AMP/OTHERS (Probabilities: 0.9473, 0.0164, 0.0363)\n","Processed MH0062_GL0050493: non-AMP/OTHERS (Probabilities: 0.9933, 0.0032, 0.0035)\n","Processed 158944319-stool2_revised_scaffold3460_2_gene84843: Neuropeptide (Probabilities: 0.2970, 0.0328, 0.6702)\n","Processed MH0440_GL0024113: AMP (Probabilities: 0.1558, 0.7843, 0.0599)\n","Processed MH0370_GL0106319: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed O2.UC34-0_GL0063416: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0006_GL0150124: non-AMP/OTHERS (Probabilities: 0.9908, 0.0031, 0.0061)\n","Processed O2.UC29-0_GL0014852: non-AMP/OTHERS (Probabilities: 0.9888, 0.0040, 0.0072)\n","Processed 556261.HMPREF0240_00626: Neuropeptide (Probabilities: 0.0472, 0.0179, 0.9349)\n","Processed MH0388_GL0139264: non-AMP/OTHERS (Probabilities: 0.9895, 0.0060, 0.0045)\n","Processed O2.UC3-2_GL0004711: non-AMP/OTHERS (Probabilities: 0.9921, 0.0030, 0.0050)\n","Processed MH0011_GL0080484: AMP (Probabilities: 0.0892, 0.8957, 0.0152)\n","Processed MH0002_GL0050870: non-AMP/OTHERS (Probabilities: 0.9838, 0.0047, 0.0115)\n","Processed MH0002_GL0050870: non-AMP/OTHERS (Probabilities: 0.9915, 0.0041, 0.0044)\n","Processed MH0281_GL0031658: Neuropeptide (Probabilities: 0.1792, 0.0293, 0.7915)\n","Processed O2.CD1-0-PT_GL0078644: Neuropeptide (Probabilities: 0.1118, 0.0269, 0.8614)\n","Processed MH0236_GL0098622: non-AMP/OTHERS (Probabilities: 0.5535, 0.0257, 0.4208)\n","Processed 764447348-stool2_revised_scaffold30476_2_gene104329: Neuropeptide (Probabilities: 0.0537, 0.0271, 0.9192)\n","Processed V1.CD6-0-PN_GL0144444: non-AMP/OTHERS (Probabilities: 0.9923, 0.0036, 0.0041)\n","Processed MH0233_GL0064342: non-AMP/OTHERS (Probabilities: 0.9083, 0.0283, 0.0633)\n","Processed V1.UC59-0_GL0146781: non-AMP/OTHERS (Probabilities: 0.9923, 0.0043, 0.0034)\n","Processed O2.UC17-2_GL0124642: non-AMP/OTHERS (Probabilities: 0.9921, 0.0047, 0.0031)\n","Processed MH0088_GL0034073: non-AMP/OTHERS (Probabilities: 0.9929, 0.0043, 0.0028)\n","Processed O2.UC41-1_GL0141950: non-AMP/OTHERS (Probabilities: 0.8844, 0.0125, 0.1031)\n","Processed MH0239_GL0023667: non-AMP/OTHERS (Probabilities: 0.6818, 0.0290, 0.2891)\n","Processed MH0131_GL0037106: Neuropeptide (Probabilities: 0.2147, 0.0243, 0.7610)\n","Processed MH0014_GL0030719: non-AMP/OTHERS (Probabilities: 0.9913, 0.0046, 0.0041)\n","Processed MH0188_GL0125761: non-AMP/OTHERS (Probabilities: 0.9925, 0.0040, 0.0036)\n","Processed MH0006_GL0220817: non-AMP/OTHERS (Probabilities: 0.9914, 0.0047, 0.0039)\n","Processed MH0326_GL0177693: non-AMP/OTHERS (Probabilities: 0.9923, 0.0043, 0.0034)\n","Processed MH0088_GL0109912: non-AMP/OTHERS (Probabilities: 0.9899, 0.0061, 0.0040)\n","Processed MH0131_GL0173617: non-AMP/OTHERS (Probabilities: 0.9923, 0.0049, 0.0028)\n","Processed V1.UC56-0_GL0048581: non-AMP/OTHERS (Probabilities: 0.9904, 0.0057, 0.0039)\n","Processed MH0287_GL0029725: non-AMP/OTHERS (Probabilities: 0.9927, 0.0041, 0.0032)\n","Processed MH0003_GL0052726: non-AMP/OTHERS (Probabilities: 0.9673, 0.0084, 0.0243)\n","Processed O2.UC17-1_GL0125990: non-AMP/OTHERS (Probabilities: 0.8728, 0.0179, 0.1093)\n","Processed V1.UC60-0_GL0043101: Neuropeptide (Probabilities: 0.0778, 0.0298, 0.8924)\n","Processed MH0288_GL0155245: non-AMP/OTHERS (Probabilities: 0.9928, 0.0035, 0.0037)\n","Processed T2D-6A_GL0197081: non-AMP/OTHERS (Probabilities: 0.8329, 0.0271, 0.1400)\n","Processed O2.UC11-1_GL0012919: Neuropeptide (Probabilities: 0.4382, 0.0774, 0.4843)\n","Processed O2.CD1-0-PT_GL0014900: non-AMP/OTHERS (Probabilities: 0.6420, 0.0803, 0.2778)\n","Processed MH0345_GL0154637: non-AMP/OTHERS (Probabilities: 0.9907, 0.0034, 0.0060)\n","Processed MH0277_GL0004218: non-AMP/OTHERS (Probabilities: 0.9927, 0.0035, 0.0038)\n","Processed MH0003_GL0097601: non-AMP/OTHERS (Probabilities: 0.9894, 0.0056, 0.0050)\n","Processed MH0330_GL0122654: Neuropeptide (Probabilities: 0.0069, 0.0085, 0.9847)\n","Processed MH0203_GL0143400: AMP (Probabilities: 0.2677, 0.5537, 0.1786)\n","Processed MH0088_GL0012079: non-AMP/OTHERS (Probabilities: 0.9685, 0.0119, 0.0196)\n","Processed MH0002_GL0022212: non-AMP/OTHERS (Probabilities: 0.9801, 0.0095, 0.0104)\n","Processed MH0453_GL0169238: non-AMP/OTHERS (Probabilities: 0.9862, 0.0063, 0.0075)\n","Processed MH0092_GL0077945: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed V1.CD46-0_GL0072562: Neuropeptide (Probabilities: 0.0076, 0.0096, 0.9827)\n","Processed V1.FI26_GL0213289: Neuropeptide (Probabilities: 0.0768, 0.0444, 0.8788)\n","Processed NOF013_GL0035509: non-AMP/OTHERS (Probabilities: 0.5569, 0.1095, 0.3335)\n","Processed V1.CD6-0-PN_GL0131477: Neuropeptide (Probabilities: 0.1911, 0.0428, 0.7661)\n","Processed O2.UC32-1_GL0130202: non-AMP/OTHERS (Probabilities: 0.9912, 0.0030, 0.0058)\n","Processed V1.UC38-4_GL0036516: Neuropeptide (Probabilities: 0.0066, 0.0086, 0.9848)\n","Processed MH0011_GL0058137: non-AMP/OTHERS (Probabilities: 0.9918, 0.0039, 0.0043)\n","Processed MH0351_GL0040136: non-AMP/OTHERS (Probabilities: 0.9840, 0.0076, 0.0084)\n","Processed V1.CD3-0-PN_GL0078615: Neuropeptide (Probabilities: 0.1703, 0.0931, 0.7366)\n","Processed MH0003_GL0105268: Neuropeptide (Probabilities: 0.0072, 0.0090, 0.9838)\n","Processed MH0188_GL0136773: Neuropeptide (Probabilities: 0.0062, 0.0089, 0.9849)\n","Processed MH0062_GL0048592: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed O2.UC2-0_GL0090805: non-AMP/OTHERS (Probabilities: 0.6964, 0.0454, 0.2583)\n","Processed O2.UC1-2_GL0041736: non-AMP/OTHERS (Probabilities: 0.9528, 0.0064, 0.0409)\n","Processed O2.UC17-1_GL0127169: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed O2.CD1-0-PT_GL0028970: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed MH0002_GL0062277: Neuropeptide (Probabilities: 0.3456, 0.0282, 0.6262)\n","Processed sp|O43451|MGA_HUMAN: non-AMP/OTHERS (Probabilities: 0.6395, 0.1293, 0.2312)\n","Processed MH0103_GL0054645: non-AMP/OTHERS (Probabilities: 0.6696, 0.0777, 0.2527)\n","Processed MH0103_GL0054645: non-AMP/OTHERS (Probabilities: 0.8095, 0.1011, 0.0894)\n","Processed MH0019_GL0014967: non-AMP/OTHERS (Probabilities: 0.7509, 0.0267, 0.2223)\n","Processed MH0002_GL0039389: non-AMP/OTHERS (Probabilities: 0.9702, 0.0089, 0.0209)\n","Processed sp|P08697|A2AP_HUMAN: non-AMP/OTHERS (Probabilities: 0.9908, 0.0036, 0.0057)\n","Processed O2.UC48-0_GL0301445: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed MH0011_GL0101437: non-AMP/OTHERS (Probabilities: 0.9926, 0.0033, 0.0041)\n","Processed MH0407_GL0010808: Neuropeptide (Probabilities: 0.0072, 0.0089, 0.9840)\n","Processed DLF007_GL0025922: non-AMP/OTHERS (Probabilities: 0.9919, 0.0037, 0.0044)\n","Processed O2.UC53-0_GL0131589: non-AMP/OTHERS (Probabilities: 0.9909, 0.0046, 0.0045)\n","Processed NOF006_GL0047668: Neuropeptide (Probabilities: 0.2141, 0.0417, 0.7442)\n","Processed MH0037_GL0008893: non-AMP/OTHERS (Probabilities: 0.9385, 0.0173, 0.0442)\n","Processed MH0373_GL0165006: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed DOM005_GL0016066: Neuropeptide (Probabilities: 0.0519, 0.0229, 0.9252)\n","Processed DOF008_GL0048356: Neuropeptide (Probabilities: 0.0140, 0.0119, 0.9741)\n","Processed MH0026_GL0030158: Neuropeptide (Probabilities: 0.0198, 0.0130, 0.9671)\n","Processed 537937.BLIG_01018: Neuropeptide (Probabilities: 0.0059, 0.0090, 0.9851)\n","Processed V1.UC25-1_GL0017972: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9851)\n","Processed O2.UC16-1_GL0071658: non-AMP/OTHERS (Probabilities: 0.9274, 0.0135, 0.0591)\n","Processed V1.UC49-1_GL0131780: Neuropeptide (Probabilities: 0.3420, 0.0473, 0.6107)\n","Processed O2.CD3-0-PT_GL0112405: non-AMP/OTHERS (Probabilities: 0.9916, 0.0040, 0.0044)\n","Processed MH0003_GL0100980: non-AMP/OTHERS (Probabilities: 0.4205, 0.3167, 0.2628)\n","Processed T2D-59A_GL0176321: Neuropeptide (Probabilities: 0.0154, 0.0149, 0.9697)\n","Processed DOF008_GL0046958: Neuropeptide (Probabilities: 0.2861, 0.1291, 0.5848)\n","Processed O2.UC41-1_GL0166822: non-AMP/OTHERS (Probabilities: 0.9925, 0.0031, 0.0044)\n","Processed MH0002_GL0008495: non-AMP/OTHERS (Probabilities: 0.8723, 0.0455, 0.0822)\n","Processed MH0203_GL0222883: non-AMP/OTHERS (Probabilities: 0.9789, 0.0096, 0.0115)\n","Processed MH0446_GL0028582: non-AMP/OTHERS (Probabilities: 0.9894, 0.0064, 0.0042)\n","Processed 411467.BACCAP_02044: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9851)\n","Processed 159551223-stool1_revised_C947473_1_gene153564: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0446_GL0010140: non-AMP/OTHERS (Probabilities: 0.9823, 0.0071, 0.0107)\n","Processed 763961826-stool1_revised_C508627_1_gene26599: Neuropeptide (Probabilities: 0.0405, 0.0270, 0.9325)\n","Processed MH0006_GL0216537: Neuropeptide (Probabilities: 0.0069, 0.0090, 0.9841)\n","Processed MH0356_GL0072582: non-AMP/OTHERS (Probabilities: 0.9820, 0.0121, 0.0059)\n","Processed V1.UC5-3_GL0256305: non-AMP/OTHERS (Probabilities: 0.9915, 0.0030, 0.0055)\n","Processed 159490532-stool1_revised_scaffold8888_2_gene50955: non-AMP/OTHERS (Probabilities: 0.9759, 0.0056, 0.0185)\n","Processed MH0411_GL0115290: Neuropeptide (Probabilities: 0.2240, 0.0234, 0.7526)\n","Processed O2.UC15-1_GL0076084: Neuropeptide (Probabilities: 0.0083, 0.0092, 0.9826)\n","Processed MH0087_GL0019460: non-AMP/OTHERS (Probabilities: 0.9890, 0.0046, 0.0063)\n","Processed V1.CD2-0-PN_GL0046612: Neuropeptide (Probabilities: 0.0511, 0.0204, 0.9286)\n","Processed BGI-33A_GL0016671: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0197_GL0068001: Neuropeptide (Probabilities: 0.1590, 0.0273, 0.8138)\n","Processed MH0002_GL0041136: Neuropeptide (Probabilities: 0.0113, 0.0117, 0.9770)\n","Processed MH0069_GL0084953: Neuropeptide (Probabilities: 0.0318, 0.0207, 0.9475)\n","Processed V1.UC12-4_GL0044083: Neuropeptide (Probabilities: 0.4195, 0.0362, 0.5444)\n","Processed NOM002_GL0081763: Neuropeptide (Probabilities: 0.0063, 0.0089, 0.9848)\n","Processed MH0379_GL0178093: Neuropeptide (Probabilities: 0.0063, 0.0088, 0.9850)\n","Processed O2.UC40-1_GL0169380: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0002_GL0049991: Neuropeptide (Probabilities: 0.0060, 0.0089, 0.9852)\n","Processed MH0002_GL0072127: Neuropeptide (Probabilities: 0.0060, 0.0087, 0.9853)\n","Processed MH0131_GL0082829: Neuropeptide (Probabilities: 0.0066, 0.0085, 0.9849)\n","Processed MH0275_GL0011966: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9851)\n","Processed MH0014_GL0030719: Neuropeptide (Probabilities: 0.3503, 0.0664, 0.5833)\n","Processed MH0452_GL0028253: Neuropeptide (Probabilities: 0.2803, 0.0550, 0.6647)\n","Processed MH0193_GL0023653: Neuropeptide (Probabilities: 0.0106, 0.0110, 0.9784)\n","Processed MH0442_GL0125857: non-AMP/OTHERS (Probabilities: 0.9880, 0.0056, 0.0064)\n","Processed MH0011_GL0102686: Neuropeptide (Probabilities: 0.3562, 0.1213, 0.5225)\n","Processed MH0124_GL0125153: non-AMP/OTHERS (Probabilities: 0.9896, 0.0057, 0.0047)\n","Processed MH0012_GL0227664: non-AMP/OTHERS (Probabilities: 0.9797, 0.0148, 0.0055)\n","Processed NOF008_GL0105665: Neuropeptide (Probabilities: 0.0066, 0.0089, 0.9845)\n","Processed sp|P0DOX2|IGA2_HUMAN: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9852)\n","Processed MH0277_GL0006824: non-AMP/OTHERS (Probabilities: 0.6640, 0.0234, 0.3126)\n","Processed MH0227_GL0102901: Neuropeptide (Probabilities: 0.1824, 0.0291, 0.7885)\n","Processed MH0002_GL0046092: non-AMP/OTHERS (Probabilities: 0.9391, 0.0129, 0.0480)\n","Processed MH0203_GL0112272: non-AMP/OTHERS (Probabilities: 0.9525, 0.0090, 0.0385)\n","Processed sp|P35908|K22E_HUMAN: non-AMP/OTHERS (Probabilities: 0.8056, 0.1835, 0.0109)\n","Processed MH0002_GL0054101: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed O2.UC12-2_GL0134094: non-AMP/OTHERS (Probabilities: 0.5893, 0.0486, 0.3620)\n","Processed MH0057_GL0013967: Neuropeptide (Probabilities: 0.0258, 0.0181, 0.9560)\n","Processed MH0229_GL0039660: non-AMP/OTHERS (Probabilities: 0.9881, 0.0038, 0.0081)\n","Processed MH0062_GL0008503: non-AMP/OTHERS (Probabilities: 0.9727, 0.0199, 0.0074)\n","Processed MH0002_GL0060297: non-AMP/OTHERS (Probabilities: 0.9715, 0.0135, 0.0150)\n","Processed MH0305_GL0136397: Neuropeptide (Probabilities: 0.0087, 0.0097, 0.9816)\n","Processed NLM007_GL0046379: Neuropeptide (Probabilities: 0.0064, 0.0088, 0.9848)\n","Processed MH0346_GL0159816: non-AMP/OTHERS (Probabilities: 0.8741, 0.0586, 0.0673)\n","Processed MH0006_GL0089747: Neuropeptide (Probabilities: 0.4168, 0.0583, 0.5249)\n","Processed T2D-22A_GL0017772: Neuropeptide (Probabilities: 0.0302, 0.0160, 0.9538)\n","Processed O2.UC17-1_GL0040485: Neuropeptide (Probabilities: 0.0060, 0.0088, 0.9852)\n","Processed V1.FI36_GL0215811: non-AMP/OTHERS (Probabilities: 0.9449, 0.0074, 0.0477)\n","Processed O2.CD3-0-PT_GL0173376: non-AMP/OTHERS (Probabilities: 0.8396, 0.0406, 0.1198)\n","Processed MH0131_GL0095432: Neuropeptide (Probabilities: 0.0075, 0.0087, 0.9839)\n","Processed 160704339-stool1_revised_C1411991_1_gene36239: Neuropeptide (Probabilities: 0.0713, 0.0258, 0.9029)\n","Processed MH0441_GL0053560: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0197_GL0065823: non-AMP/OTHERS (Probabilities: 0.9918, 0.0029, 0.0053)\n","Processed MH0014_GL0108471: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0012_GL0082767: non-AMP/OTHERS (Probabilities: 0.9569, 0.0162, 0.0269)\n","Processed 411481.BIFADO_02238: non-AMP/OTHERS (Probabilities: 0.9716, 0.0072, 0.0212)\n","Processed V1.CD32-0_GL0008228: Neuropeptide (Probabilities: 0.0059, 0.0088, 0.9853)\n","Processed MH0141_GL0037895: Neuropeptide (Probabilities: 0.0106, 0.0110, 0.9785)\n","Processed MH0141_GL0037895: non-AMP/OTHERS (Probabilities: 0.8545, 0.0262, 0.1193)\n","Processed MH0037_GL0029307: non-AMP/OTHERS (Probabilities: 0.9849, 0.0051, 0.0100)\n","Processed MH0193_GL0192617: non-AMP/OTHERS (Probabilities: 0.9923, 0.0035, 0.0041)\n","Processed MH0179_GL0005531: non-AMP/OTHERS (Probabilities: 0.9907, 0.0040, 0.0053)\n","Processed T2D-65A_GL0057861: non-AMP/OTHERS (Probabilities: 0.9932, 0.0034, 0.0033)\n","Processed sp|P42704|LPPRC_HUMAN: Neuropeptide (Probabilities: 0.0066, 0.0085, 0.9849)\n","Processed MH0006_GL0186216: Neuropeptide (Probabilities: 0.0211, 0.0143, 0.9646)\n","Processed MH0003_GL0041550: Neuropeptide (Probabilities: 0.0064, 0.0084, 0.9852)\n","Processed MH0086_GL0070039: non-AMP/OTHERS (Probabilities: 0.9873, 0.0039, 0.0088)\n","Processed MH0006_GL0174419: Neuropeptide (Probabilities: 0.0328, 0.0135, 0.9536)\n","Processed MH0277_GL0011646: non-AMP/OTHERS (Probabilities: 0.5672, 0.0737, 0.3591)\n","Processed DOM017_GL0041190: non-AMP/OTHERS (Probabilities: 0.9892, 0.0033, 0.0075)\n","Processed MH0356_GL0154723: Neuropeptide (Probabilities: 0.0108, 0.0098, 0.9794)\n","Processed MH0087_GL0037616: non-AMP/OTHERS (Probabilities: 0.9289, 0.0414, 0.0296)\n","Processed 158337416-stool1_revised_scaffold196_1_gene216864: non-AMP/OTHERS (Probabilities: 0.9855, 0.0072, 0.0073)\n","Processed MH0013_GL0019938: Neuropeptide (Probabilities: 0.0424, 0.0241, 0.9335)\n","Processed MH0277_GL0043326: non-AMP/OTHERS (Probabilities: 0.6972, 0.0372, 0.2656)\n","Processed MH0040_GL0053754: non-AMP/OTHERS (Probabilities: 0.7354, 0.0185, 0.2461)\n","Processed V1.UC14-0_GL0068351: non-AMP/OTHERS (Probabilities: 0.8001, 0.0154, 0.1844)\n","Processed MH0366_GL0124223: Neuropeptide (Probabilities: 0.3451, 0.0285, 0.6264)\n","Processed 158742018-stool1_revised_C770219_1_gene42889: non-AMP/OTHERS (Probabilities: 0.8555, 0.0164, 0.1281)\n","Processed MH0277_GL0010072: non-AMP/OTHERS (Probabilities: 0.9930, 0.0032, 0.0038)\n","Processed MH0092_GL0043774: non-AMP/OTHERS (Probabilities: 0.9180, 0.0095, 0.0725)\n","Processed V1.CD18-0_GL0057018: non-AMP/OTHERS (Probabilities: 0.9354, 0.0156, 0.0490)\n","Processed MH0230_GL0150630: non-AMP/OTHERS (Probabilities: 0.9757, 0.0088, 0.0155)\n","Processed T2D-175A_GL0052020: Neuropeptide (Probabilities: 0.0059, 0.0089, 0.9852)\n","Processed MH0002_GL0003461: non-AMP/OTHERS (Probabilities: 0.9911, 0.0045, 0.0044)\n","Processed 367928.BAD_1112: Neuropeptide (Probabilities: 0.0357, 0.0266, 0.9377)\n","Processed MH0071_GL0036119: Neuropeptide (Probabilities: 0.0094, 0.0094, 0.9812)\n","Processed MH0277_GL0033147: non-AMP/OTHERS (Probabilities: 0.8765, 0.0109, 0.1126)\n","Processed MH0003_GL0015139: non-AMP/OTHERS (Probabilities: 0.9930, 0.0037, 0.0033)\n","Processed 457421.CBFG_05788: Neuropeptide (Probabilities: 0.0059, 0.0091, 0.9850)\n","Processed MH0437_GL0040350: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9850)\n","Processed MH0441_GL0002880: Neuropeptide (Probabilities: 0.0545, 0.0184, 0.9271)\n","Processed MH0292_GL0113922: non-AMP/OTHERS (Probabilities: 0.9891, 0.0043, 0.0067)\n","Processed MH0292_GL0113922: non-AMP/OTHERS (Probabilities: 0.9930, 0.0031, 0.0039)\n","Processed O2.UC41-1_GL0006555: non-AMP/OTHERS (Probabilities: 0.9914, 0.0043, 0.0043)\n","Processed MH0131_GL0120714: AMP (Probabilities: 0.0770, 0.9097, 0.0133)\n","Processed O2.UC9-2_GL0090528: Neuropeptide (Probabilities: 0.2141, 0.0480, 0.7379)\n","Processed MH0440_GL0094644: non-AMP/OTHERS (Probabilities: 0.7115, 0.0374, 0.2511)\n","Processed MH0150_GL0010605: non-AMP/OTHERS (Probabilities: 0.9860, 0.0089, 0.0051)\n","Processed MH0088_GL0023189: non-AMP/OTHERS (Probabilities: 0.9929, 0.0036, 0.0035)\n","Processed V1.UC47-4_GL0081143: non-AMP/OTHERS (Probabilities: 0.9929, 0.0035, 0.0036)\n","Processed MH0087_GL0030432: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0035)\n","Processed O2.UC15-1_GL0033816: non-AMP/OTHERS (Probabilities: 0.9930, 0.0035, 0.0035)\n","Processed MH0003_GL0057165: non-AMP/OTHERS (Probabilities: 0.9931, 0.0034, 0.0036)\n","Processed MH0188_GL0161354: Neuropeptide (Probabilities: 0.0589, 0.0265, 0.9146)\n","Processed MH0011_GL0100040: Neuropeptide (Probabilities: 0.0058, 0.0091, 0.9850)\n","Processed MH0265_GL0139382: non-AMP/OTHERS (Probabilities: 0.4919, 0.0393, 0.4687)\n","Processed MH0447_GL0118504: non-AMP/OTHERS (Probabilities: 0.9708, 0.0096, 0.0197)\n","Processed MH0277_GL0002935: non-AMP/OTHERS (Probabilities: 0.9682, 0.0063, 0.0256)\n","Processed MH0062_GL0001109: Neuropeptide (Probabilities: 0.0444, 0.0288, 0.9268)\n","Processed MH0118_GL0118004: Neuropeptide (Probabilities: 0.2168, 0.1737, 0.6095)\n","Processed MH0161_GL0108235: Neuropeptide (Probabilities: 0.2710, 0.1293, 0.5997)\n","Processed O2.UC12-1_GL0060944: Neuropeptide (Probabilities: 0.0214, 0.0171, 0.9615)\n","Processed MH0161_GL0039629: non-AMP/OTHERS (Probabilities: 0.7330, 0.0302, 0.2368)\n","Processed V1.FI03_GL0052602: non-AMP/OTHERS (Probabilities: 0.9772, 0.0081, 0.0147)\n","Processed MH0411_GL0067231: non-AMP/OTHERS (Probabilities: 0.9889, 0.0073, 0.0038)\n","Processed O2.UC28-2_GL0141358: Neuropeptide (Probabilities: 0.0362, 0.0191, 0.9448)\n","Processed MH0011_GL0060922: Neuropeptide (Probabilities: 0.2543, 0.1787, 0.5670)\n","Processed MH0011_GL0102498: Neuropeptide (Probabilities: 0.2461, 0.0445, 0.7094)\n","Processed O2.UC18-2_GL0031006: non-AMP/OTHERS (Probabilities: 0.8355, 0.0173, 0.1472)\n","Processed MH0161_GL0013249: non-AMP/OTHERS (Probabilities: 0.9643, 0.0096, 0.0261)\n","Processed NOM028_GL0075584: Neuropeptide (Probabilities: 0.1166, 0.0290, 0.8544)\n","Processed O2.UC12-2_GL0058024: Neuropeptide (Probabilities: 0.0118, 0.0107, 0.9775)\n","Processed 457391.HMPREF0105_1254: non-AMP/OTHERS (Probabilities: 0.6400, 0.2070, 0.1530)\n","Processed O2.UC8-0_GL0092453: non-AMP/OTHERS (Probabilities: 0.9919, 0.0047, 0.0034)\n","Processed V1.CD27-0_GL0111625: non-AMP/OTHERS (Probabilities: 0.9925, 0.0044, 0.0031)\n"]},{"output_type":"error","ename":"KeyboardInterrupt","evalue":"","traceback":["\u001b[0;31m---------------------------------------------------------------------------\u001b[0m","\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)","\u001b[0;32m<ipython-input-5-10c3cc1b1a8d>\u001b[0m in \u001b[0;36m<cell line: 93>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     91\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     92\u001b[0m \u001b[0;31m# Realizar predicciones y escribir resultados en un archivo\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 93\u001b[0;31m \u001b[0mpredict_amp\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0msequences\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0moutput_file\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m","\u001b[0;32m<ipython-input-5-10c3cc1b1a8d>\u001b[0m in \u001b[0;36mpredict_amp\u001b[0;34m(sequences, output_file)\u001b[0m\n\u001b[1;32m     35\u001b[0m             \u001b[0;31m#print(input_seq_tok)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     36\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 37\u001b[0;31m             \u001b[0moutput\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mmodel\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m**\u001b[0m\u001b[0minput_seq_tok\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     38\u001b[0m             \u001b[0mlogits\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0moutput\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     39\u001b[0m            \u001b[0;31m# print(f\"Logits for {identifier}: {logits}\")\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_wrapped_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1734\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_compiled_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m  \u001b[0;31m# type: ignore[misc]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1735\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1736\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1737\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1738\u001b[0m     \u001b[0;31m# torchrec tests the code consistency with the following code\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1745\u001b[0m                 \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_pre_hooks\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_hooks\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1746\u001b[0m                 or _global_forward_hooks or _global_forward_pre_hooks):\n\u001b[0;32m-> 1747\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mforward_call\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1748\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1749\u001b[0m         \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/transformers/models/bert/modeling_bert.py\u001b[0m in \u001b[0;36mforward\u001b[0;34m(self, input_ids, attention_mask, token_type_ids, position_ids, head_mask, inputs_embeds, labels, output_attentions, output_hidden_states, return_dict)\u001b[0m\n\u001b[1;32m   1666\u001b[0m         \u001b[0mreturn_dict\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mreturn_dict\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0mreturn_dict\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32mNone\u001b[0m \u001b[0;32melse\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mconfig\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0muse_return_dict\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1667\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1668\u001b[0;31m         outputs = self.bert(\n\u001b[0m\u001b[1;32m   1669\u001b[0m             \u001b[0minput_ids\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1670\u001b[0m             \u001b[0mattention_mask\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mattention_mask\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_wrapped_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1734\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_compiled_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m  \u001b[0;31m# type: ignore[misc]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1735\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1736\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1737\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1738\u001b[0m     \u001b[0;31m# torchrec tests the code consistency with the following code\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1745\u001b[0m                 \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_pre_hooks\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_hooks\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1746\u001b[0m                 or _global_forward_hooks or _global_forward_pre_hooks):\n\u001b[0;32m-> 1747\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mforward_call\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1748\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1749\u001b[0m         \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/transformers/models/bert/modeling_bert.py\u001b[0m in \u001b[0;36mforward\u001b[0;34m(self, input_ids, attention_mask, token_type_ids, position_ids, head_mask, inputs_embeds, encoder_hidden_states, encoder_attention_mask, past_key_values, use_cache, output_attentions, output_hidden_states, return_dict)\u001b[0m\n\u001b[1;32m   1140\u001b[0m         \u001b[0mhead_mask\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget_head_mask\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mhead_mask\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mconfig\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mnum_hidden_layers\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1141\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1142\u001b[0;31m         encoder_outputs = self.encoder(\n\u001b[0m\u001b[1;32m   1143\u001b[0m             \u001b[0membedding_output\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1144\u001b[0m             \u001b[0mattention_mask\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mextended_attention_mask\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_wrapped_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1734\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_compiled_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m  \u001b[0;31m# type: ignore[misc]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1735\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1736\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1737\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1738\u001b[0m     \u001b[0;31m# torchrec tests the code consistency with the following code\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1745\u001b[0m                 \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_pre_hooks\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_hooks\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1746\u001b[0m                 or _global_forward_hooks or _global_forward_pre_hooks):\n\u001b[0;32m-> 1747\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mforward_call\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1748\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1749\u001b[0m         \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/transformers/models/bert/modeling_bert.py\u001b[0m in \u001b[0;36mforward\u001b[0;34m(self, hidden_states, attention_mask, head_mask, encoder_hidden_states, encoder_attention_mask, past_key_values, use_cache, output_attentions, output_hidden_states, return_dict)\u001b[0m\n\u001b[1;32m    693\u001b[0m                 )\n\u001b[1;32m    694\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 695\u001b[0;31m                 layer_outputs = layer_module(\n\u001b[0m\u001b[1;32m    696\u001b[0m                     \u001b[0mhidden_states\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    697\u001b[0m                     \u001b[0mattention_mask\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_wrapped_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1734\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_compiled_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m  \u001b[0;31m# type: ignore[misc]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1735\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1736\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1737\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1738\u001b[0m     \u001b[0;31m# torchrec tests the code consistency with the following code\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1745\u001b[0m                 \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_pre_hooks\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_hooks\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1746\u001b[0m                 or _global_forward_hooks or _global_forward_pre_hooks):\n\u001b[0;32m-> 1747\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mforward_call\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1748\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1749\u001b[0m         \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/transformers/models/bert/modeling_bert.py\u001b[0m in \u001b[0;36mforward\u001b[0;34m(self, hidden_states, attention_mask, head_mask, encoder_hidden_states, encoder_attention_mask, past_key_value, output_attentions)\u001b[0m\n\u001b[1;32m    583\u001b[0m         \u001b[0;31m# decoder uni-directional self-attention cached key/values tuple is at positions 1,2\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    584\u001b[0m         \u001b[0mself_attn_past_key_value\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpast_key_value\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;36m2\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0mpast_key_value\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32mNone\u001b[0m \u001b[0;32melse\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 585\u001b[0;31m         self_attention_outputs = self.attention(\n\u001b[0m\u001b[1;32m    586\u001b[0m             \u001b[0mhidden_states\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    587\u001b[0m             \u001b[0mattention_mask\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_wrapped_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1734\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_compiled_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m  \u001b[0;31m# type: ignore[misc]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1735\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1736\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1737\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1738\u001b[0m     \u001b[0;31m# torchrec tests the code consistency with the following code\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1745\u001b[0m                 \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_pre_hooks\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_hooks\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1746\u001b[0m                 or _global_forward_hooks or _global_forward_pre_hooks):\n\u001b[0;32m-> 1747\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mforward_call\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1748\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1749\u001b[0m         \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/transformers/models/bert/modeling_bert.py\u001b[0m in \u001b[0;36mforward\u001b[0;34m(self, hidden_states, attention_mask, head_mask, encoder_hidden_states, encoder_attention_mask, past_key_value, output_attentions)\u001b[0m\n\u001b[1;32m    513\u001b[0m         \u001b[0moutput_attentions\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mOptional\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mbool\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mFalse\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    514\u001b[0m     ) -> Tuple[torch.Tensor]:\n\u001b[0;32m--> 515\u001b[0;31m         self_outputs = self.self(\n\u001b[0m\u001b[1;32m    516\u001b[0m             \u001b[0mhidden_states\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    517\u001b[0m             \u001b[0mattention_mask\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_wrapped_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1734\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_compiled_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m  \u001b[0;31m# type: ignore[misc]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1735\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1736\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1737\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1738\u001b[0m     \u001b[0;31m# torchrec tests the code consistency with the following code\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1745\u001b[0m                 \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_pre_hooks\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_hooks\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1746\u001b[0m                 or _global_forward_hooks or _global_forward_pre_hooks):\n\u001b[0;32m-> 1747\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mforward_call\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1748\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1749\u001b[0m         \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/transformers/models/bert/modeling_bert.py\u001b[0m in \u001b[0;36mforward\u001b[0;34m(self, hidden_states, attention_mask, head_mask, encoder_hidden_states, encoder_attention_mask, past_key_value, output_attentions)\u001b[0m\n\u001b[1;32m    406\u001b[0m             \u001b[0mkey_layer\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mvalue_layer\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpast_key_value\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    407\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 408\u001b[0;31m             \u001b[0mkey_layer\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtranspose_for_scores\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcurrent_states\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    409\u001b[0m             \u001b[0mvalue_layer\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtranspose_for_scores\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mvalue\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcurrent_states\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    410\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mpast_key_value\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32mNone\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0mis_cross_attention\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_wrapped_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1734\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_compiled_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m  \u001b[0;31m# type: ignore[misc]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1735\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1736\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_call_impl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1737\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1738\u001b[0m     \u001b[0;31m# torchrec tests the code consistency with the following code\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_call_impl\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m   1745\u001b[0m                 \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_pre_hooks\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0m_global_backward_hooks\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1746\u001b[0m                 or _global_forward_hooks or _global_forward_pre_hooks):\n\u001b[0;32m-> 1747\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mforward_call\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1748\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1749\u001b[0m         \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;32m/usr/local/lib/python3.10/dist-packages/torch/nn/modules/linear.py\u001b[0m in \u001b[0;36mforward\u001b[0;34m(self, input)\u001b[0m\n\u001b[1;32m    123\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    124\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0mforward\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0minput\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mTensor\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m->\u001b[0m \u001b[0mTensor\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 125\u001b[0;31m         \u001b[0;32mreturn\u001b[0m \u001b[0mF\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mlinear\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mweight\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mbias\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    126\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    127\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0mextra_repr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m->\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;31mKeyboardInterrupt\u001b[0m: "]}]}]}